# DNA SEQUENCES CODING FOR THE DR BETA CHAIN LOCUS OF THE HUMAN LYMPHOCYTE ANTIGEN COMPLEX AND POLYPEPTIDES, DIAGNOSTIC TYPING PROCESSES AND PRODUCTS RELATED THERETO

## Claims
DNA Sequenz, die ein Produkt codiert, das eine immunologische oder biologische Aktivität eine β Kette Antigens des HLA DR Locus des menschlichen Lymphozyten Antigen Komplexes aufweist, wobei die Sequenz ausgewählt ist aus den DNA Sequenzen DR β A ATCC 39164 , DR β B ATCC 39163 und

## Description
This invention relates to DNA sequences that code for the DR β chain locus of the human lymphocyte antigen complex. More particularly, it relates to the use of those DNA sequences in diagnostic typing processes and products. Such processes and products are useful in determining an individual s susceptibility to a wide variety of diseases and an individual s characteristics as a donor or acceptor of a tissue or organ transplant. The DNA sequences of this invnetion are also useful in the expression of polypeptides encoded by them. The human lymphocyte antigen HLA system is the major histocompatibility complex in man. It, therefore, constitutes the strongest barrier for tissue and organ transplants between individuals, apparently distinguishing between self and non self. In addition, HLA factors have been demonstrated to be associated with increased susceptibility to a wide variety of diseases. Therefore, the antigens of the HLA system have found use in diagnostic typing processes and products for determining an individual s susceptibility to a wide variety of diseases and his characteristics as a donor or acceptor of a tissue or organ transplant F. H. Bach and J. J. Van Rood, From a genetic point of view the HLA system is fairly well characterized. See e.g., L. P. Ryder et al., Genetics of HLA Disease Association , The biology of the HLA system is less well understood. Class 1 factors are distributed in all tissues except erthrocytes. Class 2 factors are substantially restricted to β lymphocytes and mononuclear phagocytic cells and the Class 3 complement factors are directly involved in the activation of the C3 factor, the key component in the complement system. The HLA DR antigens appear to be involved in immunological phenomena immune responsiveness, T cell suppression, T cell and β cell cooperation and T cell and macrophage presentation B. Benacerraf in The Role of The Major Histocompatibility Complex In Immunobiology , M. E. Dorf, ed., Garland SPTM Press, pp. 255 69 1981 . The HLA DR antigens are composed of two non covalently linked glycosylated peptide chains, a heavy or α chain of about 35000 molecular weight and a light or β chain of about 29000 molecular weight, that span the cellular membrane Strominger et al., Several other loci also exist that encode polymorphic Ia like antigens that are closely linked but not identical, to HLA DR G. Corte et al., At present the HLA DR antigens are isolated serologically by precipitation with antisera. Therefore, the exact nature of the HLA DR determinants is uncertain. However, these antigens have found use in typing processes and products to determine the compatibility of donors and acceptors for tissue or organ transplants and to determine susceptibility of an individual to a wide variety of diseases. For example, Ryder et al., From these typings, it can be seen that an individual typed positive for D DR4 has a 6.4 times higher risk of developing insulin dependent diabetes than individuals typed negative for D DR4. In some cases it has also been demonstrated that a disease is more severe in patients having the disease associated antigen than in those who do not have that antigen. For example, multiple sclerosis progresses more rapidly in D DR2 positive patients than in D DR2 negative patients. Moreover, relapses in certain diseases are more common in patients positive for the disease associated antigens. Plainly, then, HLA DR typing has great diagnostic and prognostic value. However, the use of such typing processes and products and, therefore, the attainment of the important advantages that they would provide in identifying acceptable transplant donors and recipients and disease susceptible individuals, has been severely restricted because the present typing procedure is complex and time consuming and because there are not sufficient HLA DR antigens available to provide a useful and economical source for such processes and products. The present invention solves the problems referred to by providing DNA sequences coding for the DR β chains, the major polymorphic regions of the DR locus of the human lymphocyte antigen complex, and diagnostic typing processes and products related thereto. By virtue of this invention, the DNA sequences encoding the HLA DR light or β chains are now for the first time made available for use in HLA DR typing processes and products. Not only are the DNA sequences of this invention able to be produced economically and in large amount, their use in typing processes and products substantially simplifies and reduces the cost of the former HLA DR antigen based typing processes and products. For example, the DNA typing process of this invention is simple, can be performed with as little as 10 20 ml of blod and can easily be scaled up to several thousand typings. Finally, the DNA sequences of this invention permit the expression of those sequences in appropriate hosts and the production of the specific DR β chain antigens encoded by them, uncontaminated by other HLA DR factors, for use as diagnostic, preventive or therapeutic agents. Figure 1 is a schematic outline of chromosome 6 and the location of HLA loci on the short arm. Figure 2 is a schematic outline of one embodiment of a cloning process of this invention. Figure 3 is a partial restriction map of clones 83 7, 68 6, DR β Figure 4 depicts a partial restriction map of the cDNA sequences of HLA DR β A, HLA DR β B, HLA DR β C and HLA DR β D. Figure 5 depicts the sequencing strategy and the nucleotide and amino acid sequences of the cDNA sequence HLA DR β A. Figure 6 depicts a comparison of the amino acid sequence deduced from the cDNA sequence HLA DR β A, the amino acid sequence determined by Kratzin for an Ia antigen β chain isolated from a DR2 homozygous line and the amino acid sequence deduced from the cDNA clone isolated by Larhammar from a DR3, w6 cell line. Figure 7 depicts the nucleotide and amino acid sequences of cDNA sequence HLA DR β B. Figure 8 is a Southern blot of DNA from four individuals DR 7 7, 6 6, 3 6 and 1 1 typed using one embodiment of a typing process of this invention. Figure 9 depicts three regions of nucleotide sequence mismatch between the coding regions of cDNA clones HLA DR β A and HLA DR β B. In Figure 9, the black circles designate the nucleotide mismatches and the boxes the 19 mers prepared from these sequences. In order that the invention herein described may be more fully understood, the following detailed description is set forth. In the description the following terms are employed Referring now to Figure 1, we have shown therein a simplified diagram of chromosome 6 and the location of HLA loci on the short arm of that chromosome. In view of the complexity of the HLA system, it was important to develop a mRNA translation assay that would distinguish between the various Ia like antigens and the various HLA DR antigens themselves. Cell free translation systems, like the rabbit reticulocyte lysate system, will not process or assemble multimeric proteins. On the other hand, oocytes of the clawed toad Using mRNA encoding DR antigen rich fractions, identified in the above assay, we prepared cDNA from the mRNA, cloned it and selected and isolated clones containing the DNA sequences encoding the DR β chain antigens of this invention. These DNA sequences were then employed in the processes and products of this invention to determine compatibility for tissue and organ transplants and to determine increased susceptibility of an individual to a wide variety of diseases. These DNA sequences are also useful in appropriate hosts to produce the antigens for which they encode, substantially uncontaminated by other HLA DR factors, for use in diagnosis, therapy and the prevention of disease. We grew a human β lymphoblastoid cell line, Raji cells, a heterozygous cell line having two DR genes, DR3 and DR6 in RPMI 1640 medium, supplemented with 10 fetal calf serum, glutamine and gentamicin, substantially as described by S. Carrell et al., We lysed the frozen cell pellets in ice cold lysis buffer 10 mM Tris HCl pH 7.6 , 0.1 M NaCl, 1 Nonidet P40 1 ml buffer 10 To recover the RNA from the tubes, we inverted the tubes and cut off the bottoms. We then dissolved the RNA in 10 mM Tris HCl pH 7.4 , 1 mM EDTA, adjusted the mixture to 0.3 M sodium acetate pH 5.0 and precipitated the RNA with 2 vol ethanol. We again dissolved the RNA in 10 mM Tris HCl pH 7.4 , 1 mM EDTA and 1 SDS, heated it at 100 C for 2 min and cooled the mixture to room temperature. After addition of 1 vol 10 mM Tris HCl pH 7.4 , 1 mM EDTA, 1 M NaCl, we loaded the RNA onto an oligo dT cellulose column Collaborative Research and eluted the poly A We size fractionated the poly A To carry out the poly A To assay the HLA DR activity if any of the various poly A We then adjusted the supernatant from the oocyte homogenate to 2 ml with 0.15 M NaCl, 0.25 Nonidet P40 and loaded it over a 1 ml column of lentil lectin Sepharose Pharmacia . After washing the column extensively with that buffer, we eluted the glycosylated material with the same buffer, containing 0.1 M α methyl mannoside 1.3 The glycosylated material from the lentil lectin column was then adjusted to pH 8.0 with Tris HCl pH 7.0 and to 1 Aprotenin Sigma and we added 20 ul PX63 ascites per ml. After incubation for more than 2 h in the cold and incubation for another 2 h in the presence of excess protein A Sepharose Pharmacia , we added 20 µl per ml of a mixture of anti DR monoclonal antibodies D1 12, D4 22, BT 2.2 in the form of ascites fluid. This corresponds to 1 µl of ascites per injected ooctyte. After incubation overnight at 4 C, we spun down the samples for 3 min Eppendorf microfuge and discarded the pellets. This centrifugation is important in avoiding high background in the assay due to aggregated material. We then added Protein A Sepharose to the supernatant and continued incubation for 4 h. The immunoprecipitates were collected by centrifugation microfuge and washed twice in about 400 µl 50 mM Tris HCl pH 7.4 , 5 mM EDTA, 0.15 M NaCl, 1 Nonidet P40, 10 mM methionine, 1 Aprotenin, three times with about 400 µl of the same buffer without Aprotenin and 0.15 M NaCl, but with 0.5 M NaCl, and twice in about 400 µl 10 mM Tris HCl pH 7.4 , 1 mM EDTA, 0.15 M NaCl, 0.5 Nonidet P40. We then dissolved the immunoprecipitates in 25 µl 0.5 M Tris HCl pH 8.8 , 1 M sucrose, 5 mM EDTA, 0.01 bromophenol blue, 3 SDS and 8.3 mM dithiothreitol by heating at 100 C for 3 min and loaded the solution onto a 12 polyacrylamide SDS gel. We ran the gel in two dimensions, with non equilibrium pH gradient electrophoresis in the first dimension, substantially as described by P. Z. O Farrell et al., Using this assay, we identified fraction 31, containing mRNA s 1200 1300 nucleotides long as the fraction containing RNAs coding for the α, intermediate and β chains of HLA DR. The RNA of this fraction was enriched about 20 fold over total poly A Our assay of the size fractionated RNA was based on a number of previous analyses of translated RNAs and DR antigens from Raji cells. From these analyses we had determined that the oocytes translated the RNA coding for the α, intermediate and β chains of HLA DR, glycosylated those antigens and assembled them. We had also determined that the assembly was immunoprecipitated by monoclonal antibodies D1 12, D4 22 and BT 2.2, but that only the β chains were immunoprecipitated with BT 2.2, after the antigen assemblies were denatured. We had also determined that the α chain had an apparent molecular weight of 35000 36000, the intermediate chain had about a 33000 apparent molecular weight and the β chains had apparent molecular weights of 31000 and 29000 in the SDS polyacrylamide gels. In addition, the non glycosylated species appeared as follows 30000 and 29000 α chain , 27000 intermediate and 27000 and 26000 β chains . To prepare a single stranded cDNA copy of the poly A We prepared double stranded cDNA from the above pooled fractions by denaturing the cDNA by heating it at 68 C for 90 sec and quick chilling it in ice. We then prepared the following reaction mixture single stranded cDNA 40 µg ml , 50 mM Tris HCl pH 8.3 , 10 mM MgCl We nicked the hairpin loop in our double stranded cDNA preparation with S A wide variety of host cloning vehicle combinations may be employed in cloning double stranded cDNA. In addition, within each specific cloning vehicle various sites may be selected for insertions of the double stranded cDNA. It should be understood that the particular selection from among these various alternatives for cloning the DNA sequences of this invnetion may be made by one of skill in the art without departing from the scope of this invention. For our initial cloning work, we chose the bacterial plasmid pBR322 F. Bolivar et al., Construction And Characterization Of New Cloning Vehicles II. A Multi Purpose Cloning System , We digested pBR322 with We added dC tails to the cDNA, prepared above, in a reaction mixture containing 200 mM K cacodylate, 50 mM Tris HCl pH 6.9 , 1 mM dCTP, 100 µg ml BSA Pentex , 2 µg ml cDNA and 125 units ml terminal deoxynucleotidyl transferase P L Biochemicals by heating the mixture at 37 C for between 1 and 6 min. We selected the optimal reaction time usually about 4 min by using small aliquots. We then used that time to tail the cDNA. We again stopped the reaction by adding EDTA to 10 mM and SDS to 0.5 and by chilling the mixture in ice for 15 min. We isolated the dC tailed cDNA for annealing to the dG tailed We combined 40 ng of the dC tailed cDNA prepared above and 250 ng of the dG tailed, It should be understood that only a few of the recombinant DNA molecules prepared above will actually contain a DNA sequence coding for the β or light chains of HLA DR, the chains encoding the major polymorphic region of the HLA DR locus. In fact, the majority of the cloned species will be unrelated to HLA DR or to the β chains thereof. We transformed competent Since plasmid pBR322 includes the genes coding for ampicillin resistance and tetracycline resistance and since the former gene is inactivated by cDNA insertion at the There are several approaches to screen a library of clones for a clone containing a particular recombinant DNA molecule, i.e., one containing an HLA DR β chain related DNA insert. These methods are well known in the art. For our initial clone screening we chose to use high criteria positive hybridization selection to poly A We divided 550 selected clones into 11 groups of 50 clones each and grew the pools in L broth, supplemented with 10 µg ml tetracycline. We then amplified the plasmids with chloramphenicol 50 µg ml overnight and prepared plasmid DNA from the pools using the conventional cleared lysate CsCl gradient procedure. We then treated the plasmid DNA with 0.5 diethylpyrocarbonate and passed it over a Sepharose B column 10 mM Tris HCl pH 7.6 , 1 mM EDTA to remove any contaminating small RNA molecules. We partially depurinated the plasmid DNA in 0.25 N HCl for 10 min at room temperature, adjusted the mixture to 0.5 N NaOH, 0.5 M NaCl incubated it for 20 min, neutralized it with HCl and precipitated the DNA twice with EtOH. We then prepared diazobenzyloxymethyl paper Schleicher Schuell and covalently bound to it the above prepared DNA, substantially as described by Goldberg et al., We prehybridized the filters in 50 formamide recrystallized twice and deionized , 20 mM PIPES pH 6.4 , 0.75 M NaCl, 2 mM EDTA, 0.4 SDS, 1 glycine, 0.3 mg ml and We eluted the hybridized RNA in two portions with 150 µl 5 mM Tris HCl pH 7.4 , 0.5 mM EDTA, 6 µg ml rabbit tRNA by heating the filter containing solution at 98 C for 75 sec. We then adjusted the mixture to 0.3 M NaOAc pH 5.0 and precipitated the RNA twice with EtOH. We complemented the RNA from above with mRNA for the HLA DR α chain and intermediate chain selected from 25 µg poly A We divided each of the two positive pools into 5 groups of 10 clones each and hybridized and assayed them as before. One out of the 5 in groups derived from each of the original two positive pools was again positive. We then divided the two positive pools into 10 groups of single clones each and hybridized and assayed them as before. We selected two positive clones clone 68 6 and clone 83 7. Clone 83 7 selected DR β chain mRNA very efficiently under the conditions of hybridization. This mRNA produced in oocytes an antigen that was immunoprecipitated with the pool of anti DR monoclonal antibodies D1 12, D4 22, BT 2.2 in the absence of complementation with α and intermediate chain RNA. Conversely, clone 68 6 was much less efficient in selecting DR β chain mRNA. Clone 83 7 had an insert of 180 bp and clone 68 6 an insert of 470 bp. The inserts did not cross hybridize. Referring now to Figure 3, we have displayed therein the location of the cDNA insert of clone 83 7 in the DR domain and the location of the cDNA insert of clone 68 6 in the Ia like domain. The Ia like domain refers to a region of the HLA loci Figure 1 . Clone 68 6 is designated Ia because it represents a region that is related to, but is not identical to, HLA DR. We also analyzed RNA homologous to these two cDNA clones by gel transfer hybridization. Both cDNA clones hybridized with poly A We employed the DNA inserts of clones 83 7 and 68 6 as probes to screen more extensive libraries of total poly A We excised the inserts from the plasmid DNA of the two clones by From this screening we isolated a number of clones containing longer cDNA inserts. The inserts of these clones were designated DR β We also carried out cross hybridization experiments with various fragments of these clones at several hybridization criteria to determine the degree of homology between the different cDNA clones that we had selected. DNA sequences from the 3 untranslated portion of the cDNA clones did not cross hybridize at a high criterium 5 C below Tm , at an intermediate criterium 24 C below Tm or even at a low criterium 43 C below Tm . Conversely, DNA sequences, at the 5 end of clones, encoding the first domain of Ia like region, rather than the DR β chain loci, did cross hybridize at the intermediate criteria. Therefore DR related DNA sequences do not cross hybridize to Ia related sequences, but Ia related sequences cross hybridize α to other Ia related sequences. We mapped the HLA related inserts of our various cDNA clones by restriction analysis using single and double digestions with various restriction endonucleases. We employed the conditions and buffers recommended by the endonuclease suppliers New England Biolabs, Bethesda Research Lab, Boehringer and analyzed the resulting fragments on agarose gels. Referring again to Figure 3, we have depicted therein the partial restriction maps of various cDNA inserts located in our screening process. The actual location of the restriction sites depicted in Figure 3, of course, is inexact. Nucleotide sequencing using conventional methods will properly locate the particular sites as well as other predicted sites. As noted previously, Raji cells are heterozygous, i.e., DR 3 6. Therefore, the fact that two different sequences DR β We employed the DNA insert DR β We had employed what we thought was a homozygous cell line to avoid the aforementioned possibility of the difficulties in detecting any allelic polymorphism that may be present in heterozygous cell lines, like Raji cells. In contrast to heterozygous lines, β chain clones detected in clones from homozygous cells lines will, by definition, represent different β chain gene families. However, as noted above, the line employed by us was in fact heterozygous. As a result of our screening of this heterozygous cell line derived library, we located four families of HLA DR related DNA sequences. We designated these families of coding sequences DR β A, DR β B, DR β C and DRβ D on the basis of restriction mapping.It should, of course, be understood that other β chain families may also exist. For examle, Accolla, Clones in our four families of DR β clones cross hybridize α well throughout their coding and non coding regions. They may be distinguished by restriction mapping and also by cross hybridization at very high stringencies Figure 4 . Therefore, they most likely represent four mRNAs derived from four different DR genes. Because they are derived from a cell line heterozygous for DR 4, w6 , the four DR β genes are believed to represent at least two non allelic loci encoding DR β chains. This conclusion is also supported by an analysis of genomic DNA clones that we isolated from the same β cell line using our β For nucleotide sequencing, we prepared restriction fragments, as above, from the DNA inserts DR β A, DR β B, DR β C and DR β D, extracted them from acrylamide gels and purified them over DEAE cellulose columns. We 3 labelled the fragments with α Referring now to Figure 5, we have depicted therein the sequencing strategy and the nucleotide and amino acid sequences of the coding strand of cDNA clone HLA DR β A.In clone HLA DR β A thirty five nucleotides precede the first ATG triplet. This ATG is the first codon of an open reading frame 266 amino acids long. The first 29 amino acids, having a core of 11 consecutive hydrophobic residues, precede a sequence which has a high homology with partial amino acid sequences determined for the β chain of human Ia antigens D. A. Shackelford et al., Referring now to Figure 6, we have depicted therein an amino acid sequence comparison of the amino acid sequence that we deduced from clone HLA DR β A, the sequence determined by Kratzin for an Ia antigen β chain isolated from a DR2 homozygous line H. Kratzin et al., Referring now to Figure 7, we have depicted the nucleotide and amino acid sequences of another HLA DR β clone HLA DR β B . Again, the amino acid sequence deduced from this clone has a 29 amino acid putative signal sequence and 237 other amino acids in the coding region. The cDNA inserts coding for families of HLA DR β chain antigens or fragments thereof may be used in DR typing processes and kits. In general such typing processes comprise the steps of 1 restricting an individual s DNA using conventional endonucleases and conditions, 2 size fractionating the restricted DNA, for example on conventional gels, 3 hybridizing the size fractionated DNA to the HLA DR β chain related probes of this invention or fragments thereof and 4 detecting the areas of hybridization. For example, as one illustration of such a process, we obtained high molecular weight DNA from four different individuals 3 homozygous 1 1, 6 6, 7 7 and 1 heterozygous 3 6 for HLA DR from established cells lines. We digested the DNA at 37 C overnight with After transfer, we rinsed the filters in 4 SSC SSC is 150 mM NaCl, 15 mM trisodium citrate and then baked them for 2 h at 80 C in a vacuum oven. We then incubated the filters successively in 5 SSC, 5 Denhardt s reagent for 1 2 h at 65 C with gentle shaking and for 2 h at 65 C in 1 Denhardt s reagent, 0.75 M NaCl, 5 mM EDTA, 50 mM sodium phosphate buffer pH 7 10 dextran sulfate, 0.1 SDS, 50 µg ml poly G and 250 µg ml sonicated denatured herring DNA. We then hybridized the filter bound DNA for 8 12 h at 65 C in 1 Denhardt s reagent, 0.75 M Na Cl, 5 mM EDTA, 50 mM sodium phosphate buffer pH 7 , and 1 10 After hybridization we washed the filters twice 65 C, 30 min with each of 5 SSC, 1 Denhardt s reagent, 0.1 SDS, 0.1 sodium pyrophosphate 2 SSC, 0.1 SDS 0.5 SSC and 0.1 SSC. We then exposed the dried filters to preflashed Kodak X AR film with intensifying screens Cawo at 70 C for 48 h. Referring now to Figure 8, we have displayed the results of the hybridization. As can be seen in Figure 5, each different human DNA DR 7 7 lane 1 , DR 6 6 lane 2 , DR 3 6 lane 3 and DR 1 1 lane 4 exhibits a different electrophoretic pattern for each restriction endonuclease set. Therefore, Southern blots of DNA from various HLA DR typed individuals using the probes of this invention can distinguish among individuals of different HLA DR specificities simply and economically. Moreover, the simple blot patterns obtained in these typing processes and products may permit typing refinements, not possible in classical typing procedures, so as to identify and to distinguish various subgroups in conventional HLA DR groups and better to determine the susceptibility of those subgroups to various diseases. It should of course be understood that the detection of the hybridizing portions of the restricted particular DNA need not be done by a Hybridization under conditions of Southern blotting with short 19 base oligonucleotide DNA fragments has been shown to allow the discrimination of perfect matching sequences identical or allele from mismatching sequences a different sequence or allele . See, e.g., B. J. Conner et al. We have analyzed the nucleotide sequences of our HLA DR β cDNAs and identified at least three regions within those sequences that display sequence differences including polymorphic differences . These three regions are 1 the coding sequence for amino acids 8 14 2 the coding sequence for amino acids 26 32 and 3 the coding sequence for amino acids 72 78 Figure 9 . We also identified a region the coding sequence for amino acids 39 45 that is identical among the different DR β chain genes, and also among the DC and SB β chain genes. We prepared synthetic oligonucleotide 19 mer probes spanning the three regions of mismatch black circles in Figure 9 . The blocked areas of Figure 9 depict the particular 19 mers prepared for each of the three regions of the two HLA DR β cDNA clones. Because each of these 19 mers has more than one mismatch, an unambiguous distinction among HLA DR sequences can be made with each probe. Moreover, a 19 mer may be prepared form the homologous region, described above, to act as a positive hybridization control. In like manner, a collection of 19 mer DNA probes from regions of mismatch and identity among the other HLA DR β chain genes may be prepared. Each of the probes will then be specific for a given DR specificity. Hybridization with the collection of probes and controls would, accordingly, allow the rapid and accurate DR typing of large numbers of individuals. The level of production of a protein is governed by two major factors the number of copies of its gene within the cell and the efficiency with which those gene copies are transcribed and translated. Efficiency of transcription and translation which together comprise expression is in turn dependent upon nucleotide sequences, normally situated ahead of the desired coding sequence. These nucleotide sequences or expression control sequences define, inter alia, the location at which RNA polymerase interacts to initiate transcription the promoter sequence and at which ribosomes bind and interact with the mRNA the product of transcription to initiate translation. Not all such expression control sequences function with equal efficiency. It is thus of advantage to separate the specific coding sequences for a desired protein from their adjacent nucleotide sequences and to fuse them instead to other expression control sequences so as to favor higher levels of expression. This having been achieved, the newly engineered DNA fragment may be inserted into a multicopy plasmid or a bacteriophage derivative in order to increase the number of gene copies within the cell and thereby further to improve the yield of expressed protein. A wide variety of host expression control sequence vector combinations may, therefore, be employed in producing HLA DR β chain like polypeptides in accordance with the processes of this invention by inserting the appropriate coding sequences therein. For example, useful vectors may consist of segments of chromosomal, non chromosomal and synthetic DNA sequences, such as various known bacterial plasmids from Of course, not all host expression control sequence vector combinations may be equally efficient with a particular HLA DR coding sequence. However, as described in this invention and giving due consideration to biosafety, the sites available in the HLA DR β coding sequences of this invention for particular constructions, the size of the HLA DR β chain polypeptides to be expressed, the susceptibility of those polypeptides to proteolytic degradation by host cell enzymes, the possible contamination of those polypeptides by host cell proteins difficult to remove during purification, the expression characteristics of HLA DR β coding sequences, such as the structure of the DNA coding sequence and the location of the start and stop codons with respect to the expression control sequences and other factors recognized by those skilled in the art, an appropriate combination may be selected wherein the HLA DR β chain coding sequences of this invention are operatively linked to an expression control sequence in a vector and there employed to transform a host such that culturing the host produces the polypeptide coded for by the inserted coding sequence. There are also various methods known in the art for inserting a DNA sequence and expression control sequence into a vector. These include, for example, direct ligation, synthetic linkers exonuclease and polymerase linked repair reactions followed by ligation, or extension of the DNA strand with DNA polymerase and an appropriate single stranded template followed by ligation. Again, those of skill in the art may select one or more of such methods to express the DNA sequences of this invention without departing from the scope hereof. It should also be understood that the actual HLA DR β chain coding sequences expressed in a chosen host expression control sequence vector combination of this invention may result in products which are not identical to the authentic HLA DR β chain antigens. For example, the coding sequence expressed might code for HLA DR β chains plus a methionine or other amino acids unrelated to HLA DR β chains. The DNA sequences expressed might alternatively code for only a part or parts of HLA DR β chains alone or together with methionine or other amino acids. These constructions and products are encompassed by this invention. For example, a host transformed with a nucleotide sequence coding for a HLA DR β chain like polypeptide might product that compound alone or fused to other amino acids or it might secrete that product. All that is necessary is that the product, either after isolation from the fermentation culture or after conventional treatment such as cleavage, synthetic linking or other well known processes displays an immunological or biological activity of the HLA DR β chain antigens. The above described HLA DR polypeptides after purification or antibodies raised against them may be employed to type individuals in conventional HLA DR typing processes or kits or may be employed in other diagnostic, preventive or therapeutic agents or processes. Microorganisms and recombinant DNA molecules prepared by the processes of this invention are exemplified by cultures deposited in the American Type Culture Collection in Rockville, Maryland on July 28, 1982, and identified as DR β A, DR β B and DR β C While we have hereinbefore presented a number of embodiments of this invention, it is apparent that our basic construction can be altered to provide other embodiments which utilize the processes and compositions of this invention. Therefore, it will be appreciated that the scope of this invention is to be defined by the claims appended hereto rather than by the specific embodiments which have been presented hereinbefore by way of example.