# Novel antigens and vaccines containing them.

## Abstract
Novel antigenic peptides effective in stimulating immun ity to foot and mouth disease virus FMDV are disclosed, together with methods for their preparation, vaccines con taining the peptides and to their use in the prophylaxis of FMDV.

## Claims
CLAIMS 1. A peptide characterised in that it comprises the amino acid sequence of an antiqenic site of a VP1 protein selected from serotypes A24 Cruzeiro, C3 Indaial or Ol BFS, fragments thereof and derivatives thereof. 2. A peotide as claimed in claim 1 characterised in that it corresponds to all or part of a reqion of VP1, selected from 42 61 7945 95 102 129 160 or 162 170 176 193 204 or 206 . 3. A peptide as claimed in either claim 1 or claim 2 characterised in that it corresponds to a reqion of VP1 selected from 79 85 129 160 or 162 and 193 204 or 209 . 4. A peptide as claimed in any one of claims 1 to 3 characterised in that it corresponds to a reqion of VP1 selected from 42 51 56 61 129 147 151 160 and 143 162. 5. A peptide as daimed in any one of claims 1 to 4 and selected from a VVRHEGN b AVTHTGK c AVKHERD d VYNGTSKYAVGGSGRR e TYTGTTAYTASSARR f VYNGECRYSRNAVPNLR q GSLAARVVKQ h AHLAAAHARH i VLPVQPTGDRH j IQSLSPTHVI k VHVSGNQHTL l VHKDSI m VPSHTL n ESALLNTS q VKAETIT r GSGRRGDMGSLAARVVKQLP s NLRGDLQVLAQKVARTLP t ASARRGDLAHLAAAHAR u ARRGDLAHLAAAHARHLP v VPNLRGDLQVLAQKVARTLP w VLPVQPTGDRHKQPLI o VSALDNTA p IKADAIH 6. The peptide havinq the sequence GSGRRGDMGSLAARVVKQLP. 7. The peptide havinq the sequence ARGDLQVLAQKVARJLP. 8. A protein having the structure of VP1 of FMDV serotypes A24 Cruzeiro, C3 Indaial or O, BFS as shown in Fiqure 3. 9. An immunoqen comprising a peptide as defined in any one of claims 1 to 8 linked to a carrier. 10. A vaccine comprising a peptide as defined in any one of claims 1 to 8 or an immunoqen as defined in claim 9 toqether with a pharmaceutically acceptable vehicle therefor. 11. A method of preparing a peptide or protein as defined in any one of claims 1 to 8 comprisinq A. Synthetic preparation by known techniques from individual amino acids, from linking of smaller peptides or a combination thereof B. Cleavage, and, optionally subsequent modification, of larqer polypeptides or proteins orC. a transformino a host cell with recombinant DNA, said recombinant DNA comprising 1 a doubled. stranded DNA sequence coding for or including a sequence codinq for a Drotein or for a peptide as defined in any one of claims 1 to 8 2 a cloninq vehicle DNA sequence and 3 initiator and terminator sequences for expression of DNA b culturinq the host cell under conditions suitable for expression of FMD peptide c harvestinq the cell culture, and d separatinq said peptide 12. A DNA molecule comprising a nucleotide sequence for a protein or peotide as defined in any one of claims 1 to 8 or a DNA molecule codinq for a polypeptide which polypeptide includes a peptide or protein as defined in any one of claims 1 to 8. 13. A recombinant DNA molecule comprising an operon having initiator sequences as herein defined, and a nucleotide sequence substantially coding for protein or peptide as defined in any one of claims 1 to 8 or a nucleotide sequence including such a sequence, the nucleotide sequence being located between the initiator sequences and the terminator sequences of the operon. 14. A recombinant DNA cloning vehicle capable of expressing all or part of a protein or peptide as defined in any one of claims 1 to 8 comprising an operon having initiator sequences and terminator sequences, and a nucleotide sequence substantially coding for said peptide or protein the nucleotide sequence being located between the initiator sequences and terminator sequences of the operon. 15. A host cell containing a recombinant DNA cloning vehicle and or a recombinant DNA molecule as defined in any one of claims 12 to 14. 16. An antigen for stimulating the production of antibodies against FMDV in a mammal comprising a protein or a peptide as defined in any one of claims 1 to 8 prepared by the expression of a DNA molecule as hereinbefore defined and produced by a host cell transformed with a recombinat DNA cloning vehicle as hereinbefore defined and produced by a host cell transformed with a recombinant DNA cloning vehicle as hereinbefore defined. 17. A method of preparing a DNA molecule substantially coding for a peptide as defined in any one of claims 1 to 8 comprising a isolating FMDV single stranded RNA b preparing a first single strand of DNA complementary to the single strand of FMDV RNA and c preparing a second single strand of DNA complementary to and hydrogen bonded to the first DNA strand so as to produce a double strand of DNA. 18. A peptide or protein as defined in any one of claims 1 to 8 for use in therapy.

## Description
DESCRIPTION NOVEL ANTIGENS AND VACCINES CONTAINlNC THEMThe present invention relates to novel antigenic polypeptides effective in stimulating immunity to foot and mouth disease virus FMDV , to methods for the preparation thereof, to vaccines containing the antigen and to the use of the antigen and vaccines in the prophylaxis of FMD.Foot and mouth disease FMD is one of the most virulent and contagious diseases of farm animals. The disease is endemic In several areas of the world and can be found in many countries of Africa, Asia and South America where it is controlled to varying degrees by immunisation programmes. Countries which are free of the disease remain so only by strict import and quarantine controls, together with the use of slaughter when outbreaks occur.FMDV is an RNA ribonucleic acid virus classified as a member of the genusApthovirus of the family Picornaviridae see Cooper, P.D., et al, Interviroloqy, 10, 165 180, 1978 . There are seven known serotypes of FMDV, the European serotypes A,O and C, the South Africa Territories serotypes SAT 1, SAT 2, and SAT 3, and the Asia 1 serotype. A number of antigenically distinct subtypes are recognised within each of these serotypes and, as the subtypes are so distinct, immunologically specific subtype vaccines are required. For each serotype or subtype several genetically distinct variants exist.Vaccines currently used for the prophylaxis of FMD employ inactivated whole virus particles.Foot and mouth disease virus comprises a single strand of RNA and four polypeptides, namely VP1, VP2,. VP3 and VP4, which form the capsid proteins of the virus. The protein referred to here as VP1 is variously referred to by other workers in the field as VP3, VPThr, and VPT see Bachrach, H.L. at al, J.Immunoloqy, 115, 1636 1641, 1975 Strohmaier, K. et al, Biochem. Biophys. Res. Comm., 85, 1640 1645, 1978 Bachrach, H.L. et al, Interviroloqy, 12, 65 72, 1979 .Each of the polypeptides VP1, VP , and VP3 has a molecular weight of about 26,000, whilst VP4 has a molecular weight of about 8,000 and It is generally considered that there are approximately 60 copies of each of them in the virus.The other polypeptides translated from the virus RNA probably have a role in virus replication. The single strand of FMDV RNA has a molecular weight of about 2.6 x 106 which is equivalent to about 8000 nucleotides, and is of positive polarity acting as a template for both translation into polypeptides and RNA synthesis. One of the primary translational products is a protein designated P88 which is subsequently cleaved to produce the four capsid proteins VP1 to VP4.Capsid protein VP1, mentioned above, is susceptible to cleavage when intact virus is treated with trypsin, resulting in a large decrease in infectivity of most strains of FMDV Wild, T.F. and Brown, F., J. Gen. Virology, 1, 247 250, 1967 . Trypsin treatment may also reduce the capacity of virus to stimulate the production of neutralising antibody. Thus it appears that VP1 is likely to be the primary immunogen capable of eliciting effective protection against infection byFMDV and indeed VP1 separated from virus particles produces neutralising antibody and elicits effective protection against the virus Laporte, J. et al, C. R Acad. Sc. Paris, t. 276 Serie p, 3399 5401, 1973 Bachrach, H.L. et al, a. Immunology, 115, 1636 1641, 1975. . Separation of the naturally occurringFMDV capsid proteins, particularly VP, in order to provide a safer vaccine has necessitated the use of strongly denaturing conditions and has generally been held by those skilled in the art to be disadvantageous for the maintenance of optimum immunogenicity and production of an effective vaccine.More recently it has been shown that certain regions within VP1 have been shown to be of major significance in providing the antigenic variation of the sub types ofFMDV It has further been shown that short peptides corresponding to certain of these regions are, in their own right, effective as immunogenic against FMDV see for example J.L.Bittle et. al, Nature, 1982, 298, 30 33 .There have now been identified a number of novel pep tides which have useful immunogenic properties or confer such properties upon larger peptides or polypeptides when incorporated therein. The invention therefor provides, in a first aspect, a novel peptide characterised in that it comprises the amino acid sequence of an antigenic site of a VP1 protein selected from the serotypes A24 Cruzeiro, C3 Indaial or O1BFS. Included within the scope of the invention are fragments of the peptides containing at least 6 amino acids and derivatives of said peptides or fragments. In particular the invention is concerned with novel peptides corresponding to the following regions, the numbering corresponding to that shown in figure 4, 42 61, 79 85, 95 102, 129 160, or 162 170z176 and 193 204 01209 and fragments and derivatives thereof.Of these peptides those corresponding to regions 79 85, 129 160 or 162 and 193 204 are preferred.Within the peptides corresponding to regions 42 61 and 129 160 or 162 fragments and derivatives thereof which correspond to regions 42 51 and 56 61 and 129 147, 151 160 and 143 162 respectively are of particular interest.Novel peptides within the scope of the invention include for example a VVRHEGN hereinafter referred to as Peptide 1 b AVTHTGK hereinafter referred to as Peptide 2 c AVKHERD hereinafter referred to as Peptide 3 d VYNGTSKYAVGGSGRR hereinafter referred to as Peptide 4 e TYTGTTAYTASARR hereinafter referred to as Peptide 5 f VYNGECRYSRNAVPNLR hereinafter referred to as Peptide 6 g CSLAARVVKO hereinafter referred to as Peptide 7 h AHLAAAHARH hereinafter referred to as Peptide 8 i VLPVQPTCDRH hereinafter referred to as Peptide 9 j IOSLSPTHVI hereinafter referred to as Peptide 10 k VHVSGNQHTL hereinafter referred to as Peptide 11 RTI ID 3.11 l VHKDSI hereinafter referred to as Peptide 12 m VPSHTL hereinafter referred to as Peptide 13 n ESALLNTS hereinafter referred to as Peptide 14 o VSALDNTA hereinafter referred to as Peptide 15 p IKADAIH hereinafter referred to as Peptide 16 q VKAETIT hereinafter referred to as Peptide 17 r GSGRRGDMGSLAARVVKQLP hereinafter referred to as Peptide 18 s NLRGDLOVLAQKVARTLP hereinafter referred to as Pep tide 19 t ASARRCDLAHLAAAHAR hereinafter referred to as Peptide 20 u ARRCDLAHLAAAHARHLP hereinafter referred to as Peptide 21 v VPNLRGDLQVLAQKVARTLP hereinafter referred to as Peptide 22 w VLPVQPTGDRHKQPLI hereinafter referred to as Peptide 23 fragments thereof containing at least 6 amino acids and derivatives of said peptides or fragments.Amino acids forming the components of the peptides and polypeptides referred to herein are each symbolised a single letter as follows A alanine C cysteine D aspartic acid E glutamic acid F phenylalanine G glycine H histidine I isoleucine K lysine L leucine M methionine N asparagine P proline Q glutamine R arginine S serine T threonine V valine W tryptophan Y tyrosineThe term fragment as used herein refers to any peptide containing at least 6 amino acids, the sequence of the fragments being found within the sequence of the peptides defined above ie. comprising a partial sequence of said peptides .Of the peptides defined above those referred to as Pep tides 1 to 9 and fragments thereof are preferred.The peptides of the invention may be used in their own right as antigens for use in vaccination against FMDV. However when used as antigen it is preferred that the peptides of the invention be coupled to a suitable carrier, preferably a polypeptide, for example bovine serum albumin BSA or keyhole limpet haemocyanin KLH . The invention thus provides, in a further aspect, an antigen to FMDV characterised in that it comprises a peptide as defined hereinabove coupled to a carrier therefor.The peptides of the invention are also useful in confering advantageous properties to larger peptides and polypeptides. Thus the peptides of the invention may comprise a sequence within a larger peptide or polypeptide. Such larger peptides may comprise only one or a few additional amino acid residues or may contain many more amino acid residues. Such larger peptides or polypeptides may comprise for example 2 or more peptides or polypeptides of the invention linked either directly or through one or more additional amino acid residues. Such larger peptides or polypeptides may comprise all or part of a capsid protein from FMDV, in particular VP1 either alone or as part of a larger protein additionally containing, for example, VP2, VP3, VP4 or p88.Advantageously the polypeptide will comprise all or part of a VP1 protein or a polypeptide similar thereto and processing similar antigenic characterisitics.The invention thus provides in a further aspect a peptide or polypeptide characterised in that it is immunogenic for FMDV and comprises or includes as a partial sequence a peptide as defined hereinabove or a fragment thereof as defined above.The peptides and polypeptides of the invention may be prepared by any method known in the art for the preparation of peptides or polypeptides.Thus in a first method the peptides and polypeptides may be prepared synthetically by known techniques from individual amino acids, from linking of smaller peptides or a combination thereof.In an second method the peptides and polypeptides may be obtained by cleavage, and optionally subsequent modification, of larger polypeptides or proteins.In a further method the peptides and polypeptides of the invention may be prepared by the technique of genetic engineering as described for example with particular respect to FMDV antigens in European Patent Application Publication No. 0048455, which is incorporated herein by reference.Thus the invention provides in a further aspect a method for the preparation of a peptide or polypeptide of the invention which method comprises a transforming a host cell with recombinant DNA, said recombinant DNA comprising 1 a doubled stranded DNA sequence coding for or including a sequence for coding for a peptide of the invention 2 a cloning vehicle DNA sequence and 3 initiator and terminator sequences for expression of DNA b culturing the host cell under conditions suitable for expression of FMD peptide. c harvesting the cell culture, and d separating said pep tideThere is further provided a DNA molecule comprising a nucleotide sequence for the peptides of the invention or a DNA molecule coding for a polypeptide which polypeptide includes a peptide of the invention.There is additionally provided a recombinant DNA molecule comprising an operon having initiator sequences as hereinafter defined, and a nucleotide sequence substantially coding for a pep tide of the invention or a nucleotide sequence including such a sequence, the nucleotide sequence being located between the initiator sequences and the terminator sequences of the operon.The invention also provides a recombinant DNA cloning vehicle capable of expressing all or part of a pep tide of the invention comprising an operon having initiator sequences and terminator sequences, and a nucleotide sequence substantially coding for said pep tide the nucleotide sequence being located between the initiator sequences and terminator sequences of the operon.In a further aspect of the invention there is provided a host cell containing a recombinant DNA cloning vehicle and or a recombinant DNA molecule as defined above.The invention also provides an antigen for stimulating the production of antibodies against FMDV in a mammal comprising a peptide of the invention prepared by the expression of a DNA molecule as hereinbefore defined and produced by a host cell transformed with a recombinat DNA cloning vehicle as hereinbefore defined and produced by a host cell transformed with a recombinant DNA cloning vehicle as hereinbefore defined. The invention further provides a method of preparing a DNA molecule substantially coding for a peptide of the invention comprising a isolating FMDV single stranded RNA b preparing a first single strand of DNA complementary to the single strand of FMDV RNA c preparing a second single strand of DNA complementary to and hydrogen bonded to the first DNA strand so as to produce a double strand of DNA.The double strand of DNA so produced can be inserted into a cloning vehicle following digestion of the cloning vehicle with a restriction endonuclease, free ends of the cloning vehicle being joined to the DNA molecule so as to form a recombinant cloning vehicle. This in turn can be inserted into a suitable host cell by for example transformation.As used herein the terms listed below have the following meanings Nucleotide a unit of DNA or RNA comprising a sugar moiety pentose , a phosphate and a nitrogenous heterocyclic base. The base is joined to the sugar moiety via the glycosidic carbon 1 carbon of the pentose and the base characterizes the nucleotide. The four DNA bases are adenine A , guanine G , cytosine C and thymine T . The four RNA bases are AE Gt C and uracil U .Recombinant DNA a hybrid double stranded DNA sequence comprising at least two double stranded DNA nucleotide sequences, the first sequence not being found together in nature with the second sequence.Cloning Vehicle non chromosomal double stranded DNA capable of replicating when placed within a unicellular micro organism. Plasmid a cloning vehicle derived from viruses or bacteria.Structural Gene a sequence of DNA nucleotides which codes for a sequence of amino acids characteristic of a specific polypeptide.Initiator Seauences sequences of DNA nucleotides which control the initiation of transcription or translation.Terminator Sequences sequences of DNA nucleotides at which transcription or translation ceases.Transcription the process whereby RNA polymerase is caused to move along theDNA sequence forming messenger RNA.Translation the process of producing a polypeptide from messenger RNA.Operon a structural gene s coding for polypeptide expression which is preceded by initiator sequences and succeeded by terminator sequences.Expression the process involved in producinq a polypeptide from a structural gene. The invention will be further described by way of reference to the accompanying drawings in which Figure 1 shows physical maps and sequencing strategy used Figure 2 shows the nucleotide and predicted amino acid sequence for VP1 regions of 3 serotypes of FMDV Figure 3 shows a comparison of the amino acid sequences of Figure 2 with 3 published sequences for VP1 andFigure 4 shows the variability of three sequences of VP1 In Figure 1 the fragments used for sequencing are indicated byEMI8.1 where indicates the labelled end. Restriction enzyme sites used to generate the labelled end are shown.The abbreviations for the sites are A, Ava iI B, Bam HI E EcoRl G, Bgl iI H, Hind III M Sma I P, Pst I P , expected Pst I site which was lost presumably by slight exonuclease contamination R, Rsa I, S, Sst I. The polypeptides encoded by each region are shown.In Figure 2 the nucleotide and predicted amino acid sequences for the VP1 regions of a A24 Cruzeiro, b C3 Indaial, c O1BFS are shown a The first eleven codons of VPI in A24 are presumed missing. b The C was read as a T in the plasmid pFC3301 but is presumed to be a C in the parental plasmid pFC3r3. c The underlined sequence was derived from both plasmids pFO1t351 and pFOlr2.tThe C was derived from pFO1r2 and was replaced by a T in pFO1t351, which would predict a methionine residue.In Figure 3 a comparison of the amino acid sequences between a A12, A24 andA1o, b A1o, C3 and O1BfS, c O1BFS and O1K is shown. The amino acids were aligned for maximum, homology with the minimum introduction of gaps, each indicated by a . Each position in VPI is numbered from 1 to 216, including the gaps. All difference within a comparison group are boxed. t The threonine residue was predicted in pFO1r2 but a methionine was predicted in pFO1t351.In Figure 4 the variability of A1061, C3 Indaial and O1BF5 plotted with a c hydrophilicity of A10, C3 or 01 or with d linear representations of pep tides tested for immunogenicity. Each type of plot is generated by combining the scores of five positions in the sequence, each point representing the middle i.e. third position of each group of five. The variability plotEMI8.2 uses a score of 0 for every identical comparison for each position and a score of 1 for every different comparison for each position. This gives a maximum score of 3 for each position and 15 for each group of 5 positions.The hydrophilicity plots using the scoring system of Hopp and Woods P.N.A.S., 1981, 78, 3824 3828 for each amino acid, the scores ranging from 3.4 to 3.0. This gives a maximum score of 15 and a minimum score of 17 for each group of 5 positions. In order to align the two plots gaps were introduced in the hydrophilicity plots corresponding to gaps in the appropriate sequence in Fig. 3 which were used in the variability plot. The major hydrophilicity peaks are numbered 1 5 d immunogenic peptide, non immunogenic peptide. A,B peptides generated by trypsin c,D peptides generated by mouse submaxiliary gland protease E H peptides generated by cyanogen bromide Strohmaier et al.,J. Gen.Virol., 1982 59, 295 306 I P peptides produced by chemical synthesis Bittle et al., 1982 .Further characteristics and features of the invention are described in the followingExamples which are presented by way of illustration only and are not to be considered as limiting the scope of the present invention in any way.EXAMPLE 1 a Preparation of FMDV RNAApproximately 10 ml of a recent harvest of FMDV type A24 Cruzeiro in Eagle s medium were added to each of ten Roux bottles containing monolayers of approximately 108 BHK21 cells. After gentle shaking for 30 min the medium was decanted and 20 ml fresh medium added to each bottle. After the virus infection had destroyed the cell monolayers 3 4h , the medium was decanted and the cell debris was removed by centrifugation at 12,000xg for 15 min atA C.The virus was then pellet by spinning at 90,000xg for lhr at 40C. The virus pellet was resuspended in 2 ml of TNE buffer 10mM Tris HCl pH 7.5 150mM NaCI and lmM EDTA ethylenediamino tetracetic acid and the suspension was cleared by centrifuging for 10 min at 2000 at about 5,000xg.The cleared supernatant was made 1 w v in sodium dodecyl sulphate SDS and loaded onto a preformed gradient sucrose 15 45 w v in TN buffer 100mM Tris HCl pH 7.6, 100mMNaCl and spun at 100,000x9 for 2hr at 100C. Fractions 1 mi were monitored at 260nm and the fractions containing virus were extracted once with an equal volume of phenol chloroform 1 1 and the aqueous phase precipitated by the addition of 2 volumes ethanol, incubating overnight at 200C. The RNA precipitate was pelleted by spinning at 5,000xg for 30min at 40C. The supernatant was discarded, the pellet drained and then dissolved in 0.5 ml TNES buffer 10mM Tris HCl pH 7.6, 150mMNaCl, lmM EDTA, and 0.2 w v SDS .This solution was loaded onto a preformed sucrose gradient 5 25 in TNES and centrifuged at 200,000xg for 3.5hr at 200C. Fractions 0.5 ml containing RNA sedimenting at 355 were pooled, phenol chloroform extracted once, ethanol precipitated and redissolved as described for virus above. The resulting solution was ethanol precipitated and finally dissolved in 0.05 ml of double distilled water.By a similar technique FMDV RNA was also obtained from FMDV types C3 Indaial and O1BFS. b Synthesis of double stranded complementary DNA DScDNA FMDV RNA 10 ug and either oligo dTt 12 18 primer 0.5 ug obtained fromCollaborative Research or a calf thymus DNA DNAaseI digest were incubated at 30 0C for 2 hr in a final volume of 100 ul containing 0.4mM dithiothreitol, 8mM MgC12, 50mM Tris HCl pH Z 8.1 148mM Nail, 0.2mM dATP 2.5Ci m mole I 32P dATP obtained from Radiochemical Centre, Amersham , 0.2mM dTTP, 0.2mM dCTP, 0.2mM dGTP, 4mM Na4P207 and 28 units AMV reverse transcriptase supplied by Dr. J. Beard, Life Sciences Inc., Florida . The reaction was stopped by adding EDTA 20mM and SDS 0.2 w v .A sample 2 of this terminated reaction mixture was spotted on 2.5cm DE81 paper discs obtained from Whatman . These were washed extensively in 5 w v Na2HPO4, followed by a brief wash 5 min in distilled water before drying and scintillation counting. From this procedure, a total yield of approximately 2 ug of cDNA was calculated. The remainder of the reaction mixture was phenol chloroform extracted and ethanol precipitated as described above except that sodium acetate was added to 0.2M before addition of the ethanol. The desiccated pellet was dissolved in 100 ul 0.1MNaOH and incubated at 700 C for 20 min to remove the template i.e. FMDV RNA.The solution was neutralised with acetic acid and the cDNA resolved from degraded RNA by passing through a column 100 x 15mm of bead f phenolchloroform. The desiccated pellet was resuspended to a final volume of 90 ul in 50mM Tris HCl pH 8.3 20mM dithiothreitol, 10mM MgC12, 0.4mM dCTP, 0.4mM dCTP, 0.4mM dATP 5.5Ci m mole 32P dATP Radiochemical Centre, Amersham , 0.4mM dTTP, and 36 units AMV reverse transcriptase. After 4 hr incubation at 450C the reaction was stopped by the addition of EDTA 20mM and SDS 0.2 w v . The mixture was phenol chloroform extracted, NaCI was added to 0.2M and the mixture precipitated with 2 volumes of ethanol ovemight at 200C. The desiccated pellet was resuspended in 200 ul of 50mM NaCI, 0.1 w v SDS and passed through a Sephadex G100 column as described above. The excluded peak was ethanol precipitated in the presence of 200 ug glycogen carrier. DE81 paper disc analysis, as described above, indicated the second strand synthesis was about 60 of the maximum theoretical yield. The desiccated pellet from ethanol precipitation was resuspended in a final volume of 200 ul S1 buffer 25mM sodium acetate pH 4.6 , 150mM NaCI, and 1mM ZnSO4 . A 28 ul sample of this sampleA was stored at 200C while the remainder sample B was incubated with 5 units 51 nuclease obtained from Sigma for 30 min at 37 C.The reaction was stopped by extraction with phenol chloroform and the aqueous phase precipitated with ethanol. The desiccated pellet was resuspended in 20 ul H20. To check the effectiveness of the S1 nuclease treatment in removing the loops in the doublestranded cDNA, a small amount 2 of samples A and B was heated in S1 buffer at.100 C for 6 min, cooled to 60 C and 3 units of S1 added at varying times 0, 0.25, 1 and 14 hr followed by incubation at 3fOC for 30 min. Resistance to 51 was assayed by DE81 paper disc binding as described above.The results showed that greater than 53 of sample A was resistant to S1 digestion compared with less than 2 of sample B when both were incubated with 51 immediately upon cooling to 600C. If S1 was added 0.25h after the samples were placed at 600C these figures rose to 81 and 15 , respectively. Hence the initial S1 treatment had effectively cleaved the loops present after second strand synthesis. c Homooolymer tailinq of plasmid and double stranded cDNA ds cDNA i dStailina of the vector. Approximately 7 ug of the vector, plasmid pAT153, were digested with Pst 1 endonuclease under conditions recommended by the enzyme suppliers Boehringer . The reaction was stopped by the addition of sodium acetate to 0.3M and ethanol precipitated.The desiccated pellet was resuspended in 100 ul of dG tailing buffer 100mM sodium cacodylate HCl pH 7.1 , 5mM MgCl2, 50ug ml bovine albumin and lmM dGTP 200mCi 83 H dGTP m mole . To this, 2 ul 38 units of terminal transferase TT were added and the reaction incubated at 370C. Samples were removed after 2.5, 5 and 10 min.In each case the reaction was stopped by chilling on ice and adding EDTA to 20my 1. Analysis by trichloroacetic acid precipitation of a 5 ul sample from each showed that no further incorporation of 3H dGMP occurred after 2.5 min and that the average length of the homopolymer tail by this time was about 25 nucleotides. The remainder of the 2.5 min sample was diluted to 100 ul with TE buffer 10mM Tris HCl pH 8.0, lmM EDTA and extracted once with an equal volume of TE saturated phenol. The aqueous phase was extracted four times with ether sodium acetate was added to 0.3M and the sample ethanol precipitated. The desiccated pellet was resuspended in 40 ul H20 and stored at 100C. The concentration of DNA in this final solution was estimated to be 15ug ml. II dC tailinq of dscDNA. Approximately 0.4ug of dscDNA in dC tailing buffer 100mM sodium cacodylate HCl pH 7.1 , 1mM CoC12, 0.1mM DTT, 50ug ml bovine serum albumin 0.5mM 3H dCTP 600mCi 5 3H CTP m mole was incubated with 1 ul TT C 19 units . Aliquots were removed after 2 and 5 min and assayed for incorporation by TCA precipitation of a 5 ul sample of each. This indicated the average length of the homopolymer tail to be about 70 nucleotides after 2 min and 160 nucleotides after 5 min. These two aliquots were pooled, phenol extracted once, ether extracted four times, ethanol precipitated, redissolved in 50 ul H2O and stored at 100C. d Transformation with annealed plasmid and dscDNAApproximately 0.12ug of detailed plasmid pAT153 0.O5pmoles and 0.lug dCtailed dscDNA 0.2 0.4pmoles were incubated at 650C for 0.5h in 100ul TNEBuffer lOmM Tris Cl pH 8.0, 200mM NaCI, lmM EDTA . The temperature of the incubation was steadily dropped to 200 C over a 4 hour period and then rapidly brought to 0 C. The solution was diluted to 200ul containing, finally, 15mM TrisHCl pH 7.5, 100mM NaCI, 10mM MgC12, l0mM CaC12, 0.5mM EDTA and stored at 0 C for about 3 hours. A transformation competent culture of E.coli HB101 was prepared by inoculating lml of a stationary phase culture into 65ml L broth 1 Difco Bacto Tryptone, 0.5 Difco Bacto Yeast Extract and 0.5 NaCI, pH 7.2 and shaking at 370C for about 3h after which time the A650 was 0.4. The cells were pelleted by spinning at 3,000xg for 5 min at 40C. The cells were resuspended in 25ml 0.1M MgC12 OOC and repelleted. The pellet was resuspended in 2ml 0.1M CaCl2 0 C and left for 30min at 0 C.Approximately 0.2ml of such transformation competent cells were gently mixed with 0.lml of annealed pATl53 dscDNA and left for a further 30min at OOC, followed by 2min at 420C, and a final 30min at OOC. To this was added lml L broth and the transformation mix then incubated at 370C for 25min. Ten 0.lml samples of this were spread onto L Tc plates CL broth plus 1.5 agar and 15ug ml tetracycline obtained from Sigma and incubated ovemight at 37 0C. From these ten plates 150 tetracycline resistant colonies were obtained of which about 87 were sensitive to ampicillin 100ug ml Sigma suggesting insertion of cDNA at the Pst 1 site of pATl53. e Larqe scale recombinant plasmid preparationThe transformants containing the plasmids were grown in bulk by inoculating lûml stationary culture into 500ml L broth and shaking at 370C for about 4h until A590 reached about 1.0. Chloramphenicol obtained from Sigma was then added to Orlmg ml and the cultures shaken at 370C overnight. After this amplification step the cells were pelleted by centrifuging at 5000xg for Smin at 40C.The supernatant was disinfected and discarded, and the cell pellet resuspended in 12ml R buffer 50mM Tris HCl pH 8,0 , 40mM EDTA, 25 sucrose . Lysosyme and EDTA were then added to 1.4mg ml and 60mM respectively and the suspension left on ice for 5min. To 16ml of this were added 30ml Triton mix 0.1 Triton X 100, 62.5mMEDTA, 50mM Tris HCl pH 8.0 and the mixture left on ice about 10min or until viscous. The mixture was then cleared by spinning at 48,000xg for 15min at 40C.The resulting supernatant was carefully decanted and 0.959 CsCl and 0.lml of a 10mg ml solution of ethidium bromide added per ml. This was spun at 12000xg in a Beckman SOTi rotor at 20 C for 40h. The plasmid band was visualised by long wave U.V. fluoresence and removed by syringe by piercing the side of the tube. The plasmid DNA was spun to equilibrium in a second CsCl ethidium bromide gradient. The resulting band was extracted 4 times with propan 2 ol saturated with Na Cl and H20, and then ethanol precipitated, redissolved in TE and reprecipitated. The final desiccated pellet was resuspended in 200ul TE buffer and stored at 40C. f Screening of Recombinants The transformed cells obtained above were screened by the colony hybridisation of Hanahan and Meselon Gene 10, 63 67 1980 . The C3 and A12 recombinant plasmid transformed cells were probed with a Pvu U fragment of pFA1Ût76 Boothroyd et L1 Gene 17, 153 161 1982 which contains DNA.sequences derived from A1061, covering most of both VP3 and VP1. The Ol transformed cells were probed with a small RsaI Hinfl fragment of pFO1t351 which was derived from the plasmid pFO t251 Boothroyd et al Nature, 290, 800 802 1981 and comprises mainly the 3 end of VP1. The positive colonies were further screened by restriction mapping of their plasmid DNA. These maps were aligned with a linear map of the protein coding units of the FMDV genome, by analogy with the cDNA sequences of A1061 Boothroyd et al Nature, 290, 800 802 1981 , of A12119 Kleid at. al, Science, 214, 1125 1129 1981 and of O1K Kurz et. al., Nucleic Acids Research, 9, 1919 1931 1981 . In some cases, Southern blotting of appropriate restriction fragments was necessary to resolve ambiguities. Sequenceing and Analysis of Sequence DataDNA was seauenced bv the method of Maxam and Gilbert Methods in Enzymoloqy Vol. 65, 499 560, Grossman, L. and Moldave, K. Eds , Academic Press, NewYork as described bv Boothroyd et. al. Gene, L7, 153 161 1982 .An Apple U microcomputer was used to search DNA sequences for restriction sites and to predict the corresponding protein sequences, using a modification of a proaramme written by Larson and Messing Nucleic Acids Research, 10, 39 49 1982 . It was also used to make hydrophilicity plots using the programme in Kyte and Doolittle J.mol. Biol., 157, 105 132 1982 . A PDP II minicomputer was wed to determine dinucleotide frequencies and codon usage, usina programme written by Staden Nucleic Acids Research 4, 4037 4051 1977 and Nucleic AcidsResearch, 8, 3673 3694 1980 .The VP1 coding sequences for the three serotypes A24, C3 and O,BFS were thus determined and the predicted amino acid sequences were compared with three known VP1 sequences A10 Boothroyd et. al. Gene I7, 153 161 1982 . A12 Kleid et. al. Science 214, 1125 1129 1981 and 01K Kurz et. al., Nucleic AcidsResearch, 9 1919 1931 1981 . The results are shown in Fiqures 2 and 3.Comparison of Amino Acid Seauences Amino acid sequences for VPI from the three sarotypes A24, C3 and O1BFS were predicted from the nucleotide sequences determined above and are shown in Fiaure 2. The sequences were compared to known amino acid sequences for VP1 of three other strains.The sequences were aligned to ovive maximum horology which necessitated the introduction of paps indicated by dashes but these were kept to a minimum. They are arranqed to enable comparisons to be made between b different serotypes A10, C3 O1BFS , a different subtypes of the same serotype A10, A12, A24 and b different isolates of the same subtype O1BFS, 01K . Some of the flanking sequence is included, but only VP1 is numbered. In order to maintain consistency between different sequences, the gaps have been included in the numbering system.The results are shown in Figure 3.Determination of Immunogenic Amino Acid Sequences Comparisnn of VPl sequences form different strains of FMDV revealed a number of variable reqions interspersed by conserved regions. It is thouqht likely that immunoqenic regions are under stronq selection pressure to vary in order to evade the host immune system. Therefore some of the variable regions are likely to be immunogenic.Since immunoqenic reqions are almost certainly located on the surface of the virus they are likely to be themselves hydrophilic or situated near to hydrophilic reckons. Fiqure 4 shows plots of predicted hvdrophilicity VP1 from each of the 3 serotypes studied, super imosed on a plot of variability of VP1 from A1o, C3, O. This identified those variable reqions which overlapped hydrophilic reqions and are therefore more likely to be immunoqenic.EXAMPLE 2. Preparation of Peptide Fraoments. a A Deotide fragment corresponding to the reaion 143 162 of A24 Cruzeiro VP1 which has the structure GSGRRGDMGSLAARVVKQLP, was synthesized by the solid phase method Maralin,A, Merrifield, RB. Ann. Review Biochem. 39, p.841 866, 1970 on chloromethvl polystyrene resin usinq a custom built automatic synthesiser.tBoc pro Resin was purchased from Siqma Poole, Enqland . All the couplinq reactions used a 3 fold excess of tBoc amino acid and dicyclohexyl carbodinimide and were tested bv ninhvdrin 11 . Where the reaction was less than 99 completes the couplinq procedure was repeated.The peptide was removed from the support and deprotected with 20ml hydrogen fluoride plus 5ml anisole as scavenger and 25ml ethyl methyl sulphide, the latter to protect the methionine at 40C for 30 minutes. The hydrogen fluoride was removed with a stream of nitroqen. The peptide and resin were then dried under vacuum to remove all traces of HF, then washed with ether to remove the anisole and remaining ethyl methyl sulphide. The resin was then air dried to remove the ether. The peptide was extracted with 10 acetic acid. The purity was assayed by amino acid analYsis and by HPLC on a reverse phase column of Zorbax C8 DuPont usinq 0.1 trifluoroacetic acid and an acetonitrile aradient of 0 60 . The peptide was desalted on Sephadex GlO and evaporated to dryness to qive the required product. b Similarly prepared was a peptide fragment corresponding to reqion 143162 of O1BFS 1860 VP1 and having to seauence NLRGDLQVLAQKVARTLP.