# High efficiency yeast vector plasmid and method of its use.

## Abstract
A yeast vector having a gene for the target peptide combined with a promoter for an operon which controls glyceraldehyde 3 phophate dehydrogenase which is one of the enzymes taking part in the glycolysis of a yeast strain, Saccharomyces, a process for preparing the vector and a process for producing the target peptide from yeast in high yield by using such vector.

## Claims
Claims 1. A plasmid containing a promoter for the gene controlling glyceraldehyde 3 phosphate dehydrogenase and which expresses a gene for the target peptide in yeast. 2. A process for producing the target peptide by preparing a plasmid containing a promoter for the gene controlling glyceraldehyde 3 phosphate dehydrogenase and a gene for the target peptide, and culturing a yeast that has been transformed by said plasmid.

## Description
HIGH EFFICIENCY YEAST VECTOR PLASM IT AND METHOD OF ITS USE The present invention relates to a yeast vector having a gene for the target peptide combined with a promoter for a gene operon which controls glyceraldehyde 3 phosphate dehydrogenase hereunder referred to as GAP DH which is one of the enzymes taking part in the glycolysis of a yeast strain, Saccharomyces. More particularly, the invention relates to a process for producing the target peptide from yeast in high yield by using such promoter. The recent development of the technology of genetic engineering is so remarkable that today, pharmaceutically useful peptides such as somatostatin, insulin, growth hormones, interferons alpha, beta and gamma , influenza viruses, hepatitis B viral protein, thymosin alpha 1, betaendorphin, alpha neoendorphin, secretin, urokinase and plasminogen activating materials can be synthesized in microorganisms and animal cells. In most cases, these materials are produced by using procaryotic E. coli as host cells. Recently, the use of eukaryotic yeast cells is drawing the attention of many researchers for such reasons as easy cultivation of yeast, rapid cell division, low hazard, and advanced analysis of the genetic and biochemical aspects of Saccharomyces yeast.However, no exogenous peptides have been produced in yeast except for alpha interferon, hepatitis viral B surface antigen protein and alpha neoendorphin. The expressing of exogenous genes such as chemically synthesized genes, complementary DNA cDNA genes prepared from mRNA by reverse transcriptases, and genes prepared from chromosomes by a suitable treatment requires the presence of, a promoter region adapted to the specific host cells, and the efficiency of the expressing of peptide genes greatly depends on the promoter used. Therefore, intensive efforts have been made to develop expressing plasmid vectors using various promoters. However, the vectors so far reported for expressing exogenous genes using yeast cells as the host are not highly efficient in expressing the desired exogenous gene as compared with conventional vectors using E. coli cells as th host. Noting that glyceraldehyde 3 phosphate dehydrogenase GAP DH which is an enzyme controlling the glycolysis of Saccharomyces is produced in yeast cells in a large amount, the present inventors thought that good results would be obtained by using a promoter gene for the operon coding for this enzyme. As a result of our studies made on the basis of this observation, we found that this promoter was very useful in the production of exogenous or endogenous peptides in the yeast.We then prepared a yeast vector having an exogenous or endogenous peptide gene inserted at a site downstream of the promoter, and used the so prepared vector in transforming the yeast. By analyzing the separated yeast cells, we confirmed that they had been transformed. This has lead to the accomplishment of the present invention. Figs. 1 to 3 illustrate the sequence of constructing a plasmid bearing both GAP DH gene and aNE gene. Fig. 1 specifically shows the cloning of GAP DH gene. Fig. 2 shows the method of constructing a plasmid pYaNE53 bearing both GAP DH gene and aNE gene. Fig. 3 shows the method of constructing a plasmid having the 3 terminal non translational region of GAP DH gene inserted into pYaNE53 at a point downstream of the aNE gene. The yeast vector according to the present invention is prepared by the following procedure. Nucleic DNA inSaccharomyces XS 16 5C cairo cells is cleaved by a restriction enzyme Hind III to produce a DNA fragment having a length of 1.9 to 2.2 kb. This fragment is cloned to a HindIII site.of a suitable vector plasmid, say, plasmid pBR322, to obtain a yeast gene library. From this library, a clone is obtained which has a yeast DNA fragment having a length of 2.1 kb, one SalI cleavage site and one HpaI cleavage site.The DNA fragment in this plasmid is determined for its base sequence by the Maxam Gilbert method PNAS 74, 560 564 1977 . By comparing the results with the data reported in Holland et al., J. Biol. Chem. , 254, 9839 1979 , the insertion of the GAP DH gene can be confirmed. The plasmid having the desired GAP DH gene is cut with a specific restriction enzyme, and a DNA fragment harboring the GAP DH gene is inserted into a yeast vector carrying a selection marker, preferably E. coli yeast shuttle vector which is used only for the purpose of facilitating the subsequent manipulation. Then, a fragment bearing the target peptide gene is inserted into the vector in such a manner that the reading frame fits the SalI cleavage site in the vicinity of C terminal of the GAP DH gene. The resulting plasmid carrying both the GAP DH gene and the target peptide gene is used to transform yeast cells. For the purposes of the present invention, the target peptide gene includes both an exogeous peptide gene and an endogenous peptide gene in the yeast other than the GAP DH gene. We confirmed that the GAP DH gene promoter in the transformed yeast is capable of expressing the target peptide gene in a very efficient manner. It is known that the production of the target peptide in yeast can be significantly increased by incorporating 3 terminal non translational region of the GAP DH gene in the plasmid at a point downstream of the target peptide gene.The effect of adding the 3 terminal non translational region is specifically described in Japanese Patent ApplicationNo. 167615 81. The method of using this 3 terminal nontranslational region, a plasmid having this region inserted downstream the target peptide gene, and a method of using such a plasmid are also included in the scope of the present invention. The present invention is described in greater detail by reference to the following non limiting example. In the example, alpha neoendorphin aNE which is the target peptide was produced as GAP aNE chimeric protein from sequences such as GAP DH promoter GAP DH structual gene aNE gene abbreviated as pGAP GAP aNE and pGAP GAP aNE 3 GAP with the 3 terminal non translational region of the GAP DH gene attached , wherein aNE gene was the target peptide gene.Another target peptide, for example, interferon can be directly produced by attaching a gene coding for the target peptide to the GAP DH promoter with a suitable base sequence being interposed. Therefore, the GAP DH promoter can be extensively used not only to express various exogenous genes but also to produce exogenous or endogenous peptides other than alpha neoendorphin.Enxample 1. Cloning of GAP DH gene see Fig. 1 Saccharomyces XS 16 5C cairo cells Saccharomyces cerevisiae XS 16 5C MATa leu2 his3 trpl sir0 were cultured in 1,000 ml of YPD medium 1t yeast extract, 2 polypeptone and 2 glucose at 300C for 24 hours, and allDNA were separated from the cultured cells by the method ofCryer et al. described in Isolation of Yeast DNA , Methods in Cell Biology, vol. 12, Academic Press, New York, 1975, pp. 39 44. The yeast DNA 50 Vg was cleaved by HindIII 100 units by heating at 370C for 2 hours. The reaction solution was subjected to 0.8 agarose gel electrophoresis, and the obtained DNA HindIII fragments having a length of 1.92.2 kb were purified. Plasmid pBR322 0.5 pg was cleaved with HindIII 1 unit by reaction in TA buffer 33 mM trisCH3COOH buffered at pH 7.9, 66 mM CH3COOK, 10 iM CH3COO 2.Ng, 0.5 mM DTT at 370C for 1 hour. The two kinds of DNA were dissolved in T4 DNA ligase buffer, 20 mM tris HCl buffered at pH 7.5, 10 mM MgC12, 10 mM DDT and 0.5 mM ATP and after adding 2 units of T4 DNA ligase, and subjected to ligation at 150C for 18 hours. The reaction solution 10 p1 was added to E. coli JA221 that had been treated with CaCl2 0.3 ml .From the transformed ampicillin resistant clones, 400 clones which were sensitive to tetracycline were selected and analyz ed for the plasmid DNA by the alkali denaturation method. According to the analytical data reported by Holland et al., a 2.1 kb fragment bearing the GAP DH gene has two restriction sites, one by HpaI and the other by Sall. A 2.1 kb HindIII fragment is cleaved into two sub fragments 0.14 kb and 1.96 kb by SalI, and is also cut into two sub fragments 1.39 kb and 0.71 kb by HpaI. Our analysis of the 400 clones which were resistant to ampicillin and sensitive to tetracycline showed that one of them contained a plasmid having the same restriction sites as in the case of the above reported fragment containingGAP DH gene. We named this plasmid pYgap87. The plasmid pYgap87 was cut by HindIII at a point closer to Sal I restriction site, and the resulting cohesive end was labelled with gamma 32R ATP and polynucleotide kinase, and the DNA base sequence was determined by theMaxam Gilbert method. By comparing the results with the data of Holland et al., we confirmed that pYgapE7 had theGAP DH gene. E. coli K 12 cells that were transformed by pYgap87 were named E. coli SBM 151 and have been deposited with the Fermentation Research Institute, Japan depositeNo. FERM P 6762 International Deposition Acceptance No. FERMBP 382 under the Budapest Treaty .2. Preparation of pYE237 plasmid vector Plasmid vector pYE237 is the same as pYE227 vector described in Japanese Patent Application No. 167615 81 except that the former does not contain SalI cleavage site.This plasmid was prepared by the following method. Plasmid pYE227 5 ug was cleaved by SalI 10 units by heating in TA buffer at 370C for 1.5 hours. By subsequent heating at 650 C, SalI was inactivated, and thereafter, 0.3 mM of four kinds of dNTP and 8 mM of 2 mercaptoethanol were added. The solution was heated at 370C for 30 minutes in the presence of T4 DNA polymerase 1 unit so as to fill in the cohesive ends produced by cleavage with SalI.After one extraction with phenol, DNA was precipitated by ethanol twice the volume of the reaction solution. The DNA precipitate was dissolved in T4 DNA ligase buffer 20 p1 , and subjected to ligation at 150C for 18 hours in the presence of T4 DNA ligase 5 units . The reaction solution was introduced into E. coli JA 221 as donor DNA according to the conventional method. From the transformed ampicillin resistant clones, DNA was separated and analyzed for confirming the production of pYE 237 having no SalI restriction site.3. Preparation of pYE1201 yeast E. coli shuttle vector see Fig. 2 Plasmid pYE237 5 ug was cleaved with HindIII by heating in TA buffer at 37eC for 1 hour. Throughout the example, all reactions with restriction enzymes were conducted in TA buffer at 370C for 1 hour. The already prepared plasmid pYgap87 5 ug was cleaved with HindIII 20 units and subjected to agar electrophoresis to separate 2.1 kb DNA fragments GAP DH gene , followed by elution and purification of the DNA. The purified 2.1 kb DNA fragments and DNA fragments obtained by cutting pYE237 with HindIII were dissolved in DNA 20 ul of DNA ligase buffer and subjected to ligation at 150C for 16 hours in the presence of T4 DNA ligase 1 unit .The reaction solution 10 p1 was introduced into CaC12 0.3 ml treated E. coli JA221 so as to transform the latter. Plasmid DNA was isolated from the ampicillin resistant transformants by the conventional method and analyzed for confirming the production of pie1201. Yeast XS 16 5C cells transformed by this plasmid were named Saccharomyces cerevisiae SBM 321 and have been deposited with the Fermentation Research Institute depositNo. FERM P 6766 International Deposition Acceptance No.FERM BP 381 under the Budapest Treaty .4. Preparation of plasmid DNA bearing aNE gene paNES Sal I c According to the method described in Nucleic AcidResearch, 10, 1741 1754 . 1982 , an aNE gene EcoRI BamHIDNA fragment made of 46 base pairs was prepared. The aNE gene was inserted into the larger fragment 4 kb of the two fragments obtained by cutting pBR322 with EcoRI and BamHI. By the procedure described below, SalI linker composed of 5 GGGTCGACCC 3 was inserted to the resulting pYaNE to form plasmid pYaNE5 SalI c. This was for the purpose of inserting aNE so that the reading frame would fit Sal I site in the GAP DH gene. Plasmid pYaNE5 5 pg was cleaved with EcoRI 10 units by heating in TA buffer at 370C for 1 hour.After additional heating at 650C, 0.3 mM of two kinds of dNTP dATP.and dTTP and 8 mM of 2 mercaptoethanol were added, and the solution was heated at 370C for 30 munites to fill in the cohesive ends formed by cleavage with EcoRI. After one extraction with phenol, DNA was recovered by precipitation with ethanol. SalI linker 1 Vg had its 5 terminal phosphorylated by heating in a buffer 50 mM tris HCl buffered at pH 7.5 and 10 mM MgC12 at 370C for 45 minutes in the presence of ATP 25 nmol and polynucleotide kinase 10 units . The two DNA fractions were mixed and reacted in T4 DNA ligase buffer at 150C for 18 hours in the presence of T4 DNA ligase 5 units .The reaction solution 10 p1 was introduced into CaC12 0.3 ml treated E. coli JAS21. Plasmid DNA was separated from the ampicillin resistant transformants by the conventional manner and analyzed to confirm the construction of pYaNE 5 SalI c.5. Insertion of alpha neoendorphin aNE gene see Fig. 2 Plasmid pYEl2Ol 5 Ug was cut into two fragments byBamHI 20 units and SalI 20 units and the two fragments were separated by agar electrophoresis. The larger fragment was eluted from the agar and purified. Plasmid paNE5 SalI c 50 pg was also cut into two fragments by BamHI 100 units and SalI 100 units and the two fragments were separated by 5 polyacrylamide gel electrophoresis. A DNA fragment made of 50 base pairs and containing the aNE gene was eluted from the gel and purified. The two DNA fragments were dissolved in DNA ligase buffer 20 p1 and heated at 150C for 16 hours in the presence of T4 DNA ligase 2 units . The reaction solution 10 p1 was introduced into CaC12 0.3 ml treated E. coli JA221. Plasmid DNA was separated from the ampicillin resistant transformants by the conventional manner and analyzed to confirm the production of pYaNE53.6. Addition of 3 terminal region of GAP DH gene to pYaNE53 see Fig. 3 The 3 terminal region of GAP DH gene was introduced into pYaNE53 at a point downstream of the aNE gene in the following manner. Plasmid pYaNE53 10 pg was cut with BamHI 30 units and heated in a T4 DNA polymerase buffer 67 mM tris HCl buffered at pH 8.8, 6.7 mM MgCl2, 10 mM 2 mercaptoethanol, 16.6 mM ammonium sulfate, 6.7 pM EDTA, 0.3 mM of four kinds of dNTP at 370C for 30 minutes in the presence of T4 DNA polymerase 1 unit so as to fill in the cohesive ends formed by cutting with BamHI. The 3 terminal region of GAP DH gene was obtained from pYE1201 by the following procedure. Plasmid pYE1201 10 pg was cut with SalI 30 units and heated in a T4 DNA polymerase buffer at 370C for 30 minutes in the presence ofT4 DNA polymerase 1 unit so as to fill in the cohesive ends formed by cutting with SaIl. The two plasmids were cleaved with PstI 30 units , and by agarose electrophoresis, a fraction containing aNE gene was separated from pY NE53, and a fraction containing the 3 terminal region of GAP DH gene was separated from pYE1201, and the two fractions were purified. They were ligated to each other by reacting in T4DNA ligase buffer 20 NIl at 150C for 17 hours in the presence of T4 DNA ligase 5 units .The reaction solution 10 p1 was introduced into CaC12 0.3 ml treated E. coliJA221. Plasmid DNA was separated from the ampicillinresistant transformants by the conventional manner and analyzed to confirm the construction of pYaNElS5. 7. Transformation and culture of yeast Plasmids pYE1201, pYaNE53 and pYaNE155 were introduced into Saccharomyces cerevisiae XS 16 5C according to the method described in Beggs J.D., Nature, 27, 5104 1978 .The resulting transformants were shake cultured in a YPD medium 1 yeast extract, 2 polypeptone and 2 glucose at 300C for 24 hours and the cells were recovered by centrifugation. Cells recovered from 1 ml of the culture were mixed with 0.5 ml of cold acetone and the mixture was left to stand at 20 C for 1 24 hours, and freeze dried. The dried cells were suspended in 70 formic acid containing cyanogen bromide 5 mg ml and the suspension was left to stand in a dark place for 24 hours. The suspension was freeze dried and aNE was eluted with 0.1N acetic acid 0.1 ml . The eluted aNE was neutralized with 1.3M tris HC1 pH 8.0 and subjected to radioimmunoassay RIA according to the method described in N. Minamino et al., BBRC, vol.102, 226 1981 . The results are shown in Table 1. Plasmid pYaNE53 having no 3 terminal region of GAP DH gene expressed two million molecules per cell of chimeric protein having aNE attached right after the N terminal 323 rd amino acid residue of GAP DH Plasmid pYaNE155 having 3 terminal region of GAP DH gene attached to pYaNE53 expressed 20 million molecules of the same chimeric protein per cell.This was equivalent to the productivity of about 2 g of the chimeric protein per ml of the culture. To determine the stability of the three plasmids, they were cultured through 10 generations under non selective conditions in YPD medium.The stability factor of pYE1201 having a complete GAP DH was relatively low 10 20 . On the other hand, pYaNE53 and pYaNE155 having the aNE gene attached in the vicinity of Cterminal of GAP DH has somewhat imporved stability. Table 1 Productivity of aNE in the presence of GAP DH promoterEMI9.1 tb SEP aNE SEP productivity tb Cell SEP Cell SEP count SEP Plasmid tb Plasmid SEP per SEP 1 SEP ml SEP ng ml SEP no. SEP of SEP mole SEP stability tb SEP culesper SEP cel SEP tb pYEl20l SEP 1.83 SEP x SEP 108 SEP 0.01 SEP SEP 15.1 tb SEP 1.28 SEP x SEP 108 SEP 445 SEP 1.73 SEP x SEP 106 SEP 42.0 tb pyaNE53 SEP 0.6 SEP x SEP 108 SEP 337 SEP 2.8 SEP x SEP 106 SEP 78.9 tb SEP 1.12 SEP x SEP 108 SEP 368 SEP 1.6 SEP x SEP 106 SEP N.D. SEP tb SEP 0.99 SEP x SEP 108 SEP 2049 SEP 1.0 SEP x SEP 107 SEP 43.2 SEP tb pY alpha NE155 SEP SEP 0.20 SEP x SEP 108 SEP 702 SEP 1.7 SEP x SEP 10 SEP tb SEP 0.25 SEP x SEP 108 SEP 1260 SEP 2.5 SEP x SEP 7 SEP 25.5 tb SEP 0.42 SEP x SEP 108 SEP 1706 SEP 2.0 SEP x SEP 10 SEP tb Not measured.