# The cloning and expression of nucleotide sequences encoding bovine growth hormone.

## Abstract
Methods and compositions are provided for the cloning and expression of bovine growth hormone BGH genetic sequences in single cell host organisms. Also described are methods for culturing these novel single cell organisms to produce polypeptides having growth hormone activity and methods for purification of the gene product. Recombinant plasmids were constructed so as to provide an optimal system for the expression of BGH gene sequences in compatible host cells. The BGH related proteins produced by the recombinant DNA techniques described herein may be used in animal husbandry and in veterinary medicine.

## Claims
WE CLAIM 1. A process for producing a polypeptide having bovine growth hormone activity, characterized by a culturing a unicellular organism containing a recombinant vector comprising a DNA sequence coding for a polypeptide having bovine growth hormone activity and capable of being replicated, transcribed and translated in the unicellular organism and b isolating said polypeptide from the culture. 2. The process according to claim 1, wherein the polypeptide is a bovine growth hormone fusion protein, an unfused bovine growth hormone, or a mature bovine growth hormone. 3. The process according to claim 1, wherein the DNA sequence is obtained by isolating mRNA coding for a bovine growth hormone protein and using reverse transcriptase to contruct a cDNA sequence having one strand complementary to the isolated mRNA. 4. The process according to claim 1, wherein said unicellular organism is a procaryotic organism such as Escherichia coli. 5. A process for preparing a unicellular organism having a DNA sequence coding for a polypeptide having bovine growth hormone activity, said process characterized by introducing into the unicellular organism by such methods as transformation, transduction or transfection, a recombinant vector comprising a DNA sequence coding for a polypeptide having bovine growth hormone activity and capable of being replicated, transcribed and translated in the unicellular organism. 6. A recombinant DNA vector comprising theBGH DNA sequence as depicted in FIG. 2, or any portion thereof coding.for a polypeptide having bovine growth hormone activity wherein the BGH DNA sequence is under the control of a transcriptional promoter, such as the lac promoter, such as the one designated p29 112 412c the trp promoter also containing the trp operator the trp promoter containing the trp operator and the lac operator, such as the one designated pMTE 407 29 the tac promoter, such as the one designated p153 5 or the tac promoter, such as the one designated p141 2. 7. The.recombinant DNA vector according to claim 6, wherein the BGH DNA sequence is connected in the correct translational reading frame to a second DNA sequence coding for a protein, such as the vectors designated pRedl3, pN 37, or pMet 27. 8. The recombinant DNA vector according to claim 6, wherein the BGH DNA sequence further comprises a synthetic front end, such as the vectors designated plO2 1, pSFE ll, or plO4 14. 9. The recombinant DNA vector according to claim 6 further comprising an A T rich DNA region located between the promoter and the BGH DNA sequence, such as said vectors designated p128 7 2, p146 1, or p179 3. 10. The recombinant DNA vector according to claim 6 further comprising a transcription attenuatorDNA sequence located at the 3 terminus of the BGH DNA sequence. 11. The recombinant DNA vector according to claim 6 further comprising tandem translation termination sequences located at the 3 terminus of the BGHDNA sequence wherein the termination sequences are in the correct translational reading frame with the BGHDNA sequence. 12. The recombinant DNA vector according to claim 10 further comprisinandem translation termination sequences located at the 3 terminus of the BCH DNA sequence, wherein the termination sequences are in the correct translational reading frame with the BGH DNA sequence, such as said vectors designated p214 1 or p217 1. 13. The.recombinant DNA vector according to claim 6 further comprising a second DNA sequence encoding a fusion protein wherein the second DNA sequence is under the control of its own transcriptional promoter, such that the.BGH DNA sequence and the second DNA sequence are capable of being replicated, transcribed and translated in a unicellular organism, such as said vectors designated p230 2 3, p230 5 3, p230 6 2, p230 6 1, p195 4 or p231 3. 14. The recombinant DNA vector of claim 6 further comprising a DNA ori sequence which allows for production of a high copy number of the recombinant DNA vector in a host cell, such as said vectors designated p238 D2 1, p239 D5 1, or p237 D3 1. 15. Unicellular organisms, such as Escherichia coli bacteria, each such organism containing one of the recombinant DNA vectors of claim 6, such as the lac promoter vector containing Escherichia coil bacterium, such as the one deposited with the NRRL and assigned accession No. B 15654 the trp promoter also containing the trp operator vector containing Escherichia coli bacterium the trp promoter containing the trp operator and the lac operator vector containing Escherichia coli bacterium, such as the one deposited with the NRRL and assigned accession No. BY15681 the tac promoter vectorcontaining Escherichia coli bacterium, such as the one deposited with the NRRL and assigned accession No. B 15662 and mutants, recombinants, and genetically engineered derivatives thereof. 16. Unicellular organisms, such as Escherichia coli bacteria, each such organism containing one of the vectors of claim 7, such as the vector containingEscherichia coli deposited with the NRRL and assigned accession No. B15655, B 15190, and B 15667 and the mutants, recombinants and genetically engineered derivatives thereof. 17. Unicellular organisms, such as Escherichia call bacteria, each such organisms containing one of the recombinant DNA vectors of claim 8, such as the recombinant DNA vector containing Escherichia coli bacteria deposited with the NRRL and assigned accession No. B 15659, B 15668, B 15664 and the mutants, recombinants, and genetically engineered derivatives thereof. 18. Unicellular organisms, such as Escherichia coli bacteria, each such organism containing one of theA T rich DNA region vectors of claim 9, such as AlT richDNA region vector containing Escherichia coli bacteria deposited with the NRRL and assigned accession No. B 15657, B 15663, B 15658 and mutants, recombinnts, ind genetically engineered derivatives thereof. 19. A unicellular organism, such asEscherichia coli bacterium, each such organism containing one of the recombinant DNA vectors of claim 10. 20. A unicellular organism, such asEscherichia coli bacterium, each such organism containing one of the recombinant DNA vectors of Claim 11. 21. Unicellular organisms, such as Escherichia coli bacteria, each such organism containing one of the recombinant DNA vectors of claim 12, such as the recombinant DNA vector containing Escherichia coli bacteria deposited with the NRRL and assigned accession No. B 15656, B 15.661 and mutants, recombinants, and genetically engineered derivatives thereof. 22. Unicellular organisms, such as Escherichia coli bacteria, each such organism containing one of the recombinant DNA vectors of claim 13, such as the recombinant DNA vector containing Escherichia coli bacteria deposited with the NRRL and assigned accession No. B 15670, B 15671, B 15674, B 15675, B 15676, B 15678 and mutants, recombinants, and genetically engineered derivatives thereof. 23. Unicellular organisms, such as Escherichia coli bacteria, each such organism containing one of the recombinant DNA vectors of claim 14, such as the recombinant DNA vector containing Escherichia coli bacteria deposited with the NRRL and assigned accession No. B 15669, B 15672, B 15673 and mutants, recombinants, and genetically engineered derivatives thereof.

## Description
THE CLONING AND EXPRESSION OF NUCLEOTIDE SEQUENCES ENCODING BOVINE GROWTH HORMONE 1 TABLE OF CONTENTS Page 1. Field of the Invention........................... 4 2. Background of the Invention...................... 5 2.1. Recombinant DNA Technology and Gene Expression................................. 5 2.1.1. Efficient Gene Expression.......... 8 2.1.2. The Lactose and Tryptophan Operons. 10 2.1.3. Ori Mutations 15 2.2. Bovine Growth Hormone..................0..0 17 3. Summary of the Invention....................... 18 4. Brief Description of the Figures 20 5. Description of the Invention 24 5.1. Identification and Isolation of BGH Genes.. 25 5.2. Insertion of the BGH Gene into An Expression vector.......................... 26 5.2.1.Construction of an Hybrid Promoter 29 5.2.2. Construction of BGH with Synthetic Front End................. 30 5.2.3. Insertion of the SFE BGH into an Expression Vector 31 5.2.4. Insertion of A T Rich Segment 32 5.2.5. Transcriptional and Translational Termination. 0 0 0 0 0 0 0 0 0 0 0 0 32 5.3. Construction of pTRP Vectors Regulated by the LAC Operator LacO ................. 33 5.4. Preparation of Fusion Protains............... 34 5.5. Identification of the Gene Product 36 5.6. Purification of Gene Product 36 Page 5.7. Preparation of Unfused BGH................. 39 5.7.1. Co Transformation of Host Cells.... 40 5.7.2. Transformation of Host Cells with One Plasmid Carrying Two Genes..... 41 5.7.3. Modification of the FusionProtein Gene. .0... 00 00 0 0 0 000 0 0 00 41 5.8.Insertion of BGH Sequences into High Copy Number Vectors......................... 45 6. Example Bovine Growth Hormone........00......... 45 6.1. General Procedures Used for Preparation of the Plasmids....................... 46 6.1.1. Plasmid DNA Isolation................ 47 6.1.2. Conditions for Restriction Enzyme Digestions...................... 47 6.1.3. Restriction Enzyme Buffers......... 48 6.1.4. Modification of DNA.................. 48 6.1.5. DNA Polymerase Reaction........... 50 6.1.6. Gel Purification of DNA Fragments 52 6.1.7. DNA Ligation...................... 52 6.1.8. Identification of BGH Proteins by Immunoprecipitation with Anti BGH SERA.......................... 53 6.2. Isolation and Molecular Cloning of the BGH Gene..................................... 54 6.2.1.Isolation of mRNA..................... 54 6.2.2. Preparation of cDNA from BGH mRNA.. 55 6.2.3. Cloning of BGH cDNA.................. 58 6.3. Cloning and Expression of BGH DNA Sequences................................... 61 6.3.1. Cloning and Expression of BGH in pJS413........................ 61 6.3.2. Cloning and Expression of BGH In pDN572 Met Plasmids........... 64 6.3.3. Expression Vector pDN6l2........... 66 Page 6.3.4. Generation of Met BGH by Employing A Synthetic SindIII NcoI Linker.... 67 6.3.5. The Construction of the Plasmid pN 37 BY Employing a Synthetic BgIII NcoI Linker................. 69 6.3.6. Method for the Generation of Met BGH by Employing Primer Repair............................ 70 6.4. Construction of Synthetic Front End................ 72 6.4.1.Construction of Plasmids pSFEll, p104 14 and p102 1............. 73 6.4.2. Construction of ptac Clones........ 74 6.4.3. Insertion of A T Rich Sequences.... 76 6.4.4 . Insertion of Translational Termination Sequences............. 78 6.4.5. Insertion of Transcriptional Termination Sequences.............. 79 6.4.6. Construction of ptac A T SFE BGH TERM Plasmid......................... 80 6.5. Insertion of BGH into PPV Fusion Protein Plasmid.......................... 81 6.5.1.Insertion of pTAC BGH Sequences into the PPV Fusion Protein Plasmid................................. 82 6.5.2 Insertion of p214 1 into the PPV Fusion Protein Plasmid.............. 82 6.5.3 Insertion of pTAC A T TTS att BGH into the PPV Fusion Protein Plasmid................................. 83 6.5.4 Analysis of Proteins Produced by Transformants................... 84 6.5.5. Quantitation of Protein................. 85 6.6 Insertion of BGH Sequences into the high Copy Number Plasmid pORI TET 1....... 86 Page 6.7. Construction of pTRP Expression Vectors.... 87 6.7.1. Constructlon of Plasmid pTC503...... 87 6.7.2. Construction of Plasmid pTNA771.... 88 6.7.3. Construction of Plasmid Polink 23456.............................. 88 6.7.4.Construction of Plasmid pTNAP771... 89 6.7.5. Construction of pTRP Vector pSFE 1 Which Expresses BGH.................. 89 6.7.6. Construction of pTRP Vectors Which . are Regulated by the LAC Operator 90 6.7.7. Construction of Plasmid p407 29 Which is Regulated by LACO............... 91 6.8. Plasmids that Express the BGH Gene......... 92 Deposit of Microorganisms0 ...00................. 96 1. FIELD OF THE INVENTION This invention is directed to processes for the production of proteins related to bovine growth hormone BGH and to processes for production of novel plasmid vehicles, both eucaryotic and procaryotic, which can be used to produce such proteins. The present invention utilizes recombinant DNA techniques to insert a DNA sequence coding for BGH or a portion thereof into a DNA vector, such as viral DNA, plasmid DNA or bacteriophage DNA, such that the vector is capable of replicating and directing expression of the BGHDNA sequence in a bacterial host or other single cell system. The resulting recombinant DNA molecule is introduced into host cells to enable production of BGH by the host cells. The protein produced is then isolated, purified and modified for use as a biologically active growth promoting molecule. 2. BACKGROUND OF THE INVENTION 2.1. RECOMBINANT DNA TECHNOLOGY AND GENE EXPRESSION Recombinant DNA technology involves insertion of specific DNA sequences into a DNA vehicle vector to form a recombinant DNA molecule which is capable of replication in a host cell. Generally, the inserted DNA sequence is foreign to the recipient DNA vehicle, i.e., the insertedDNA sequence and the DNA vector are derived from organisms which do not exchange genetic information in nature, or. the inserted DNA sequence may be wholly or partially synthetically made. In recent years several general methods have been developed which enable construction of recombinant DNA molecules. For example, U.S. Pat. No.4,237,224 to Cohen and Boyer describes production of such recombinant plasmids using restriction enzymes and methods known as ligation. These recombinant plasmids are then introduced and replicated in unicellular organisms by means of transformation. Because of the general applicability of the techniques described therein, U.S.Pat. No. 4,237,224 is hereby incorporated by reference into the present specification. Another method for introducing recombinant DNA molecules into unicellular organisms is described byCollins and Hohn in U.S. Pat. No. 4,304,863 which is also incorporated herein by reference. This method utilizes a packaging transduction system with bacteriophage vectors cosmids . Regardless of the method used for construction, the recombinant DNA molecule must be compatible with the host cell, iXe., capable of autonomous replication in the host cell. The recombinant DNA molecule should also have a marker function which allows the selection of host cells so transformed by the recombinant DNA molecule. In addition, if if..ali of the. proper replication, transcription and translation signals are correctly arranged on the plasmid, the foreign gene will be properly expressed in the transformed cells and their progeny. These different genetic signals and processing events control many levels of gene expression, for instance, DNA transcription and messenger RNA translation. Transcription of DNA is dependent upon the presence of a promoter which is a DNA sequence that directs the binding of RNA polymerase and thereby promotes transcription. The DNA sequences of eucaryotic promoters differ from those of procaryotic promoters. Furthermore, eucaryotic promoters and accompanying genetic signals may not be recognized in or may not function in a procaryotic system. Similarly, translation of messenger aNA mRNA in procaryotes depends upon the presence of the proper procaryotic signals which differ from those of eucaryotes. Efficient translation of mRNA in procaryotes requires a ribosome binding site called the Shine Dalgarno SD sequence on the mRNA. This sequence is a short nucleotide sequence of mRNA that is located before the chain initiation codon AUG which encodes the amino terminal methionine of the protein. The SD sequences are complementary to the 3 end of the 16S rRNA tribosomal RNA and probably promote binding of mRNA to ribosomes by duplexing with the rRNA to allow correct positioning of the ribosome.For a review on maximizing gene expression, see Roberts and Lauer, 1979, Methods inEnzymology 68 473 Many factors complicate the expression of eucaryotic genes in procaryotes even after the proper signals are inserted and appropriately positioned. A clear understanding of the nature of these factors and the mechanisms by which they operate is presently lacking.One such factor is the presence of an active proteolytic system in E. coli another bacteria. This protein degrading system appears to selectively destroy 1abnormal or foreign proteins such as eucaryotic proteins. A tremendous utility,. therefore, would be afforded by the development of a means to protect eucaryotic proteins produced in bacteria from proteolytic degradation. One strategy is to construct hybrid genes in which the eucaryotic sequence is1igated in phase i.e., in the correct reading frame with a procaryotic gene.Expression of this hybrid gene results in a fusion protein product a protein that is a hybrid of procaryotic and foreign or eucaryotic amino acid sequences . Construction of hybrid genes was the approach used in the molecular cloning of genes encoding a number of eucaryotic proteins, such as somatostatin, rat proinsulin, growth hormone, and ovalbumin like protein. Additionally, procaryotic promoters have been ligated to such fusion protein gene sequences in the case of ovalbumin and ss globin Guarente et al., 1980, Cell 20 543 . The Guarente et al. system involves inserting the lac promoter, including the SD sequence, at varying distances in front of the ATG of the fusion protein gene.Although the molecular cloning and expression of several eucaryotic genes has been accomplished, this has not heretofore been done for the BGH gene. Nor is the state of the art such that expression of foreign or eucaryotic gene at procaryotic host cells may be routinely performed. Successful expression of a cloned gene requires efficient transcription of DNA, translation of the mRNA and in some in st anceslpost translational modification of the protein. Expression vectors have been used to express genes in a suitable host and to increase protein production. The cloned gene should be placed next to a strong promoter which is controllable so that transcription can be turned on when necessary. Cells can be grown to a high density and then the promoter can be induced to increase the number of transcripts. These, if efficiently translated will result in high yields of protein production. This is an especially efficient system if the foreign protein is deleterious to the host cell. 2.1.1. EFFICIENT GENE EXPRESSION Recent studies have shown that the major outer membrane proteins of E. coli are present in large quantities in the cell. The lipoprotein lpp gene of E.coli for example, is expressed at levels reaching 700,000 to 750,000 molecules cell, demonstrating an extremely efficient transcriptional and translational system. Other outer membrane proteins OMP , such as A and OmpF are each produced in similar quantities indicating that these systems also have very efficient machinery for gene expression. Although the mechanisms which are responsible for the highly efficient expression of these 2. coli genes are not yet fully understood, it is believed that several factors must contribute to the abundance of such proteins in the host cell. It has been found that in comparison with other known promoter regions of E. coli genes, that the promoter regions of the outer membrane protein genes are extremelyA T rich which is believed to be essential for efficient transcription of the genes. These OMP gene sequences also have regions which strongly bind RNA polymerase and are important for the initiation and efficiency of transcription of these genes. Further analysis has revealed that A T rich sequences are found between the SD sequence and the ATG of the Omp genes. This region A U on the mRNA transcript is thought to play an important role in the initiation of translation. The A U richness of this segment in the mRNA transcript may be needed to destabilize the secondary structure of the mRNA within this region and thereby facilitate ribosome binding and subsequent translation along the message. Nucleotlde sequence information of highly expressed genes in B. coli and yeast, has also shown a preferred codon bias which correlates well with the intracellular amounts of tRNA s for those preferred codonsv i.e.r a high frequency of cordon usage correlates with large high tRNA pools of tRNA for that amino acid. Being the most highly expressed genes in bacteria, the OMP sequences show an extreme bias for certain codons for most of the amino acids, thereby providing another factor for the highly efficient expression of these proteins within the bacterial cell. Resolution of the DNA sequence at the 3 end of the coding region of the OMP genes, has also disclosed a consensus type bacterial transcription termination signal. The DNA immediately preceeding the site of transcription termination is GC rich and possesses dyad symmetry. The 3 terminus of the transcript typically contains a series of uridine residues. These factors contribute to the overall efficiency of transcription by hastening then rate of the mRNA production and by limiting the overall size of the mRNk transcribed from the DNA. Finally, the presence of all three translational stop codons in the 3 untranslated region of the mRNA transcript, all three of which are in the same translational reading frame as the 1coding1 frame of the mRNA, provide a sequence of tandem terminators of translation. Again, this factor probably contributes to the overall efficiency of translation by assuring proper translational termination with little chance of read through into the,untranslated region of the mRNA. 2.1.2. TE LACTOSE AND TRYPTOPHAN OPERONS An operon is a coordinated unit of gene expression, The genetic elements of an operon are a regulatory gene, an operator, and the associated structural genes which are under the control of a single promoter. The regulatory gene directs the production of a repressor protein that interacts with the operator to prevent transcription of the associated structural genes. Binding of the repressor protein to the operator gene interferes with the binding of RNA polymerase to the promoter and, therefore, prevents initiation of transcription. Induction of transcription derepression can be accomplished by preventing the interaction between the repressor protein and the operator so that transcription is initiated. The resulting transcript is a single mRNA which encodes all of the associated structural genes. For example, the genes controlling the synthesis of enzymes sequentially linked in a metabolic pathway such as the lactose lac genes of Ev coli, z, y, a are induced or repressed together. The z, X, and a genes code for the enzyme a galactosidase, which catalyzes the hydrolysis of ss galactosides e.g., lactose , S galactoside permease a membrane protein which transports galactosides into the bacterial cell , and a galactoside transacetylase which functions in vitro to transfer an acetyl group from acetyl CoA to galactosides , respectively.These genes can be induced only by molecules with an intact unsubstituted galactoside residue such as lactose and thiogalactosides isopropyl ss D thiogalactoside . The inducer need not be metabolized by its enzymes i.e., a gratuitous inducer . The regulator i governs the synthesis of the lac repressor protein. In the absence of inducer, the regulator i directs the synthesis of a repressor protein which binds to the operator and prevents the structural genes Qz, yr a from functioning. When an inducer is added to the bacterial culture, it binds to the repressor protein converting it to its inactive form i.e., changing its conformation and therefore the repressor can no longer bind to the operator. The structural genes can now express their protein products. In the lac operon, this interaction between repressor protein and operator occurs at tie level of transcription. When the repressor protein binds to the operator region of the DNA, no transcription will take place until an inducer is added to the cells. For a more complete review of the lactose operon see Lewin, 1974, Gene Expression, John Wiley and Sons, pp. 2n 309. In a wild type E. coli cell, the tryptophan operon encodes five genes which code for enzymes that participate in the biosynthesis of tryptophan from chorismic acid. When there is not sufficient tryptophan in the cell to support normal growth, the tryptophan operon is induced and the five enzymes are synthesized and begin to convert chorismic acid to tryptophan. When tryptophan levels in the cell are high, the operon is repressed, and synthesis of the five enzymes is inhibited. When an operon such as the tryptophan operon is repressed, transcription of mRNA is prevented and thus no proteins encoded by that operon can be synthesized. The repressor for the tryptophan operator is a protein coded for by the trp R gene, a gene that is not physically a part of the tryptophan operon, but is located quite a distance from the tryptophan operon on the E. coli chromosome. In order to repress trancription from the trp operon, the repressor protein must first complex with the amino acid tryptophan which is referred to as the corepressor . The repressor corepressor complex binds to the trp operator region and physically prevents the binding of RNA polymerase to the promoter site.Since RNA polymerase cannot bind to the promoter, no mRNA can be transcribed and, therefore, no proteins encoded by the structural genes of the tryptophan operon can be made.Since two components the repressor protein and tryptophan are required for repression, a diminution of either will result in at least partial induction i.e., derepression of the tryptophan operon. The tryptophan operon is really a very fine tuned system since the entity which the operon is programmed to help produce tryptophan is one of the components that turns the operon off. In addition, transcription of the trp operon is also regulated by a transcription termination site called the attenuator, that is located between the operator and the first gene in the operon i.e., the first gene expressed in the enzyme pathway . The attenuator site complements the operator site in regulating transcription of the trP genes as flollowsr when tryptophan is abundant, transcription of the trp operon is blocked by the binding of the tryptophan repressor complex to the operator. As the concentration of tryptophan in the cell decreases, the trn operon is derepressed and transcription is initiated. However, some of the RNA polymerase molecules fall off the template DNA strand at the attenuator site, whereas other aNA polymerase molecules continue to transcribe the entire trp message. The proportion of RNA polynierase molecules that proceed past the attenuator site increases as the tryptophan concentration decreases. A number of researchers have reported the use of the tryptophan operon regulatory region including the promoter, the operator, a ribosome binding site sequence rbs , and an initiation ATG in plasmid vectors designed to express foreign genes in E. coli Christie and Platt, 1980, J. Molecular Biology, 143 335 European Patent Application No. 0036776 . The ribosome binding site plays a crucial role in determining the level of translation into protein of mRNA transcribed from the gene sequences associated with the regulator region of an operon such as the tryptophan operon.The majority of expression vectors containing the tryptophan operon which are in use at the present time contain the natural ribosome binding site of either the tryptophan leader sequence Edxan et al., 1981,Nature, 291 503 Miozzari et al., 1978, J. Bacteriology, 133 1457 Kleid et al., European Patent Application EP O 036 770 A2 Goeddel et al., 1980, Nature, 287 411 or the tryptophan E gene Hallewell and Emtage, 1980, Gene, 9 27 . Two basic methods have been employed rather extensively to induce the transcription of 1foreign1 genes ligated to the controlling region of the tryptophan operon 1 the addition of tryptophan analogs to the culture medium and 2 the dilution of tryptophan in the medium to low levels such that there will be insufficient repreasor corepressor complex to prevent transcription.Both of these methods described below attempt to manipulate repression by altering some function of the co repressor. Tryptophan analogs, such as 3 indolylacrylic acid, can bind to the tryptophan repressor protein just as tryptophan does, however the resulting repressor indolylacrylic acid complex cannot bind to the operator and therefore does not prevent transcription of the operon. Although this method makes it possible to induce1 transcription of the operon, it disrupts translation and thus reduces gene expression.Specifically, although transcription proceeds rather well, the translation proce is disrupted 1 because the translational machinery attempts to insert the tryptophan analog into a growing polypeptide chain at the point where the nucleic acid code calls for the addition of a tryptophan molecule and 2 because translation is inhibited none of five enzymes are produced, therefore, no indole is converted to tryptophan thus effectively starving the cells of tryptophan . The second method of induction, dilution of tryptophan in the cell culture medium is not deleterious to the cell but requires that an actively growing culture harboring a recombinant plasmid carrying a foreign gene under control of the tryptophan operon be diluted 10 or 20 fold in order to reduce the tryptophan levels sufficiently to obtain significant induction. This method requires growth in large volumes of media and or reduced yield of gene product per volume of culture. Thus, while it is possible to obtain expression of proteins using the tryptophan operon, it would be advantageous to provide a method for optimal gene expression of using methods that neither require the addition of harmful tryptophan analogs nor the dilution of cells in culture. 2.1.3. ORI MUTATIONS Most plasmid cloning vectors commonly used in E.coli are derivatives of ColEl type replicons for additional information see Oka, et al., 1979, Mol. Gen.Genet. 172 151 159 . The ColEl plasmids are stably maintained in E. coli strains as monomeric molecules with a copy number of about 15 20 copies per chromosome.Various levels of expression of human and animal health care protein products of foreign genes inserted into these plasmids has been obtained. However, very high expression levels must be obtained in order for it to become economically feasible to produce these protein products. One way to obtain large amounts of a given gene product is to clone a gene on a plasmid which has a very high copy number within the bacterial cell. In theory, by increasing the number of copies of a particular gene, the cell would also increase its IRNA and therefore its production of protein. using et al. 1981, Cell, 24 235 242 have isolated the recessive copy number mutant,Cop plasmid pOPl 6, from ColEl. This plasmid, which exists at levels of about 200 300 copies per chromosome, is chatacterizeduby xutzt ion on the plasmid genome about 400 b.p. from the origin of replication of a single base pair alteration a G C to T A transversion in the copA gene.The copA sequence is thought to code for a small RNA transcript RNAL which may act as a negative modulator of the Co1E1 DNA replication machinery. It is believed that a disruption of the secondary structure of the RNA by the mutation is responsible for the loss of its negative controlling effects. Uhlin et al. 1983, Gene, 22 255 265 have reported a series of plasmids which are double cop copy number mutants of the plasmid R1. These plasmids are unique in that they carry a temperature sensitive mutation which allows the plasmid to be present in moderate numbers per cell at temperatures below 35 C. Above 35 C, all control of plasmid replication is apparently lost and the number of plasmid copies per cell increases continuously to nearly 2000. Cell growth and protein synthesis continue at normal rates for 2 3 hours at the higher temperature. During this period, products of genes present on the plasmid may be overproduced. Inhibition of cell growth occurs after 2 3 hours and the cell loses viability. Eventually, plasmid DNA may account for as much as 75 of the total DNA present in the cell. 2.2. BOVINE GROWTH HORMONE Growth hormone sometimes called somatotropin is a protein that is secreted from the adenohypophysis of the pituitary glad of mammals and affects the rate of skeletal growth and gain in body weight. A defect in growth hormone secretion or secretion of an inactive growth hormone molecule in young animals results in a greatly retarded growth rate or even a complete failure to grow.In young animal that are suffering from a lack of endogenous growth hormone, continuous administration of exogenous growth hormone results in a complete restoration of normal development. Although the most obvious role of growth hormone in animal development is its promotion of skeletal growth, the hormone also has an effect on many other metabolic functions including increased insulin release from the pancreas, increased glucagon secretion, a lipid mobilizing effect, stimulation of reticulocytosis, a lactopoietic effect and stimulation of chondrogenesis and osteogenesis. Another valuable benefit of growth hormone is in the cattle industry. Erogenous administration of bovine growth hormone increases the growth rate, decreases the fattening time, increases the meat to fat ratio and increases milk production in domestic cattle. Until the present invention, supplies of bovine growth hormone could only be obtained by tedious and expensive extraction and isolation of excised bovine pituitary glands, and the quantities of hormone that would be required for a large commercial scale adnlinistration to cattle were simply not available. Bovine growth hormone is a protein composed of 191 amino acids having the a molecular weight of approximately 22,000 daltons. The hormone is synthesized by the cell first as a pre growth hormones that contains a 26 amino acid 1leader1 or signal sequence which may be involved in transport to the cellular membrane. This signal sequence is enzymatically removed to produce the nature protein biologically active molecule. The production of modified bovine growth hormone is described in U.K. Patent Application GB 2,073,245 and in European Application No. 47,600, and also by Keshet et al., Nucleic Acid Res., 9 1 t19 30 1981 . The molecular cloning but not the expression of DNA complementary to bovine growth hormone mRNA has been described tiller et al., 1980, J. of Biol. Chem. 255 7521 7524 . 3. SUMMARY or THE INVENTION Methods and compositions are provided for the cloning and expression of growth hormone genetic sequences in single cell host organisms. Also described are methods for culturing these novel single cell organisms to produce polypeptides having growth hormone activity herein after referred to as GH proteins and methods for the purification of the gene product. Recombinant plasmids were constructed so as to provide an optimal1 system for the expression of bovine growth hormone BGH gene sequences in compatible host cells. The BGH related proteins produced by the recombinant DNA techniques described herein may be used in animal husbandry and in veterinary medicine. The genetic sequence which encodes BGH may be obtained by isolating BGR mRNA and preparing cDNA using the mRNA as a template. The cDNA sequences can subsequently be processed and inserted into plasmid vectors to form recombinant plasmids which serve as biologically functional replication units. These recombinant plasmids are constructed so as to facilitate both the replication and expression of the BGH genes upon transformation of compatible host cells. Additionally, thuplasmids provide for a one step identification of transformed microorganisms actively expressing the BGH genes. Methods and compositions are provided for construction of a plasmid cloning vector which has a very high copy number within the bacterial. cell. By inserting the BGH gene into this expression vector the number of copies of the gene is increased and therefore production levels of protein are increased. Methods and compositions are provided for the construction and synthesis of an expression vector containing the tryptophan operon controlling region.Expression of genetic sequences inserted into this plasmid is obtained by transforming a host cell. The trp vectors described herein are unique in that they contain a trp promoter which is regulated by a lac operator region.This method allows for the manipulation of repression by altering the repressor molecule. Finally, methods are described for the isolation, stabilization, and purification of the expressed gene products. These methods take advantage of the fact that fusion proteins are resistant to degradation in the host cell. Methods are provided for the co production of both the fusion protein and the unfused product. Methods are also provided for the isolation and purification of the co aggregate forming proteins. 4. BRIEF DESCRIPTION OF THE FIGURES The present invention may be more fully understood by reference to the following detailed descriptions of the invention, examples of specific embodiments of the invention, and the appended figures. In all figures within this text, restriction endonuclease enzyme restriction sites are indicated by the following abbreviations AhaIII, BamEI, BqlII, ClaIEcoRI, EcoRV, HaeII, HaeIII, HincII, HindIII, HinfI,HpaII, VarI, NcoI, PstI, PvuII, RsaI, SacI, SalI, Sau3A 1,SmaI, I, XbaI, XmaI, XmnI, Unless otherwise indicated, the figures which follow are not drawn to scale. FIG. 1 is a diagrammatic representation and partial restriction endonuclease cleavage map of BGH cDNA clone pBH27 which contains the entire coding sequence for pre BGH and BGH. Only relevant restriction sites are indicated. Fragment sizes are denoted in base pairs b.p. . FIG. 2 represents the nucleotide sequence of thePstI BGH DNA insert of the positive strand of pBH27. The differences between the reported mRNA sequence of Miller et al. 1980, J. Biol, Chem., 225 7521 7524 and the pBH27 sequence are indicated above the pBH27 sequence nucleotides present in the pBH27 sequence but not found in the unpublished sequence are indicated by a single asterisk . Only relevant restriction sites are indicated. FIG. 3 is a diagrammatic representation of the construction of the p3 30 IM Bal 31 plasmid population. FIG. 4 is a diagrammatic representation of the construction of pRedl3. FIG. 5 is a diagrammatic representation of the construction of p572 13 1 modified met BGH fusion plasmid series and p572 13 1 Bg1II NcoI heterogenous population. FIG. 6 is a diagrammatic representaion of the reconstructed mature BGH. PIG, 7 shows pertinent nucleotide and corresponding amino acid sequences of several met plasmids. FIG. 8 and 8A represents the construction of modified plasmids pBG18Cl and pJS413 612N met BGE. FIG. 9 represents the construction of pN 37. FIG. 10 is a scheme showing the generation of met BGH using the method of primer repair. The construction of plasmid p27 is shown in FIG. 10A. The construction of plasmid p27 112 4 is shown in FIG. 10B. FIG. 11 is a sequence comparison of the NaturalFront End NFE sequences from the cDNA clone pBH27 of BGH to those of the Synthetic Front End SFE construction. FIG. 12 is a diagrammatic representation of pSFE ll. FIG. 12A is a diagrammatic representation of p104 14 and plO2 1. FIG. 13 shows the construction of Ptac from PtrP and Placuv5. The 35 and 10 regions of each promoter are underlined and their respective binding sites SD are overlined. The initiation of transcription is labeled at the first nucleotide. FIG. 14 and 14A represent the construction of various Ptac BGH plasmids. FIG. 143 shows the distance between the SD1ac and SDCrO regions of two ptac BGH plasmids. The SD regions are underlined. FIG. 15 represents the construction of p146 1 which contains an A T rich intervening sequence between the SDcro and the ATG of 3GH. FIG. 16 represents a comparison of the intervening sequences between the SDCro and ATG of BGE constructs p102 l, p128 7 2, and pl46 1. PIGS. 17 and 17A represent the construction of p166 1 and p179 3. FIG. 18 is a diagrammatic representation of the insertion of a DNA oligomer containing three translational stop sequences TTS to generate p175 12. FIG. 19 represents the insertion of the trp attenuator sequences 3 to the TTS of BGH to generate p211 4 FIG. 20 is a diagrammatic construction of p217 1 and p214 1. FIG. 21 represents the construction of pPPV 1000tx. FIG. 22 is a diagrammatic representation of the construction of p158 1 amd p195 4. FIG. 23 is a diagrammatic representation of the construction of various recombinant plasmids which contain the PPV fusion protein and BGH coding sequences on one plasmid. FIG. 24 represents the construction of pOTI tet 1. FIG. 25 represents the construction of various recombinant plasmids which contain ptac A T SPE BGH TTS att and the ori mutation. FIG. 26 represents the construction of pTC503.The abbreviation p o stands for the promoter operator sequences of trp. FIG.27 represents the construction of pTNA nl. FIG. 28 represents the construction of plasmidPolink 23456. FIG. 29 is a diagrammatic representation of the construction of pTNAP771. FIG. 30 represents the nucleotide sequence of the pvuII BgIII fragment from pTNAP771. FIG. 31 is a diagrammatic representation of pSEE l. FIG. 32 is a diagrammatic representation of plo 6 15. FIG. 33 is a diagrammatic representation of the construction of pT248 ll. PIGS. 34 and 34A represent the construction of pMTE132 from plasmids pT248 1l, pl06 15, and the synthetic lacO 25 mer fragment. FIG. 34B represents the construction of pMTE40 29 from pMTE132 and pSFE 1. FIG.34C represents the nucleotide sequence of the promoter operator, lacO, and SD region of pMTE 407 29. 5. DESCRIPTION OF TE INVENTION This invention relates to the use of recombinantDNA techniques to produce polypeptides related to BGH proteins which can be used as biologically active hormones. More specifically, the production of EGH related proteins and partially synthetic BGH related proteins is described. The recombinant plasmids, constructed as described herein, provide for host cell production of a protein which is a BGH related polypeptide and which is stable and resistant to host cell degradation such plasmids enable the generation of large quantities of aBGH related proteins or fusion proteins. However, the DNA compositions described herein are not limited to the production of a BGE related protein and may be used to produce polypeptides related to any of the bovine hormone proteins. The method of this invention may be divided into the following stages for the purpose of description a isolation and identification of the BGH genetic sequences b preparation and molecular cloning of the BGE genetic sequences in order to generate multiple copies of the BGH sequencesor portions of the BGH sequences c insertion of various portions of the cloned BGH sequence into a cloning expression vector to form a recombinant DNA molecule which is used to transform single celi organisms and d identification and purification of the gene product. 5.1. IDENTIFICATION AND ISOLATION OF BGH GENES The coding sequence for BGH may be obtained from any cell or tissue which produces BGH the most commonly known source of BGH is pituitary glands. Accordingly, pituitary glands may be homogenized and the mRNA which encodes BGE may be isolated. Reverse transcriptase may be used to convert the mRNA to cDNA and the cDNA may then be inserted into a DNA cloning vector that is used to transform host cells. This enables the generation of multiple copies of the BGH DNA sequence so that an ample supply is available for analysis. Subsequently, the BGHDNA sequence or portions thereof may be subcloned into expression vectors which may then be used to transform host cells that are capable of expressing the inserted BGH gene. When cloning the BGH gene in procaryotes, isolation of mRNA and subsequent cloning of cDNA encodingBGH is advantageous because the gene sequence is obtained in an uninterrupted form. However, alternatives to this technique include, but are not limited to, chemically synthesizing the gene DNA sequence itself or a portion thereof , or isolating the genomic DNA that encodes BGH, or isolating the BGH gene from a recombinant vector in which the gene had previously been cloned.5.2. INSERTION OF THE BGH GENE INTO AN EXPRESSION VECTOR Once BGE cDNA containing recombinant plasmids are identified, the BGE DNA sequences hereinafter referred to as a BGH gene are subcloned into appropriate expression vectors, i.e., vectors which contain the necessary elements for transcription and translation of the inserted genes To obtain efficient expression of the BGR gene or a portion of the gene , a promoter located 5 to the inserted BGH gene must be present in the expression vector. RNA polymerase normally binds to the promoter and initiates transcription of a gene or a group of linked genes and regulatory elements. This group is called an operon. Promoters vary in their strength , i.e., their ability to promote transcription and thus produce large quantities of gene product.Depending upon the host cell system utilized, any one of a number of suitable promoters may be used. For instance, when cloning in an E. coli, its bacteriophages or plasmids promoters such as the lac promoter, trp promoter, recA promoter, ribosomal RNA promoter, the PR and PL promoters of coliphage lambda and others including but not limited to lacuv5, trp lacuv5 tac hybrid promoter, ompF, bla, lpp and the like, may be used to direct high levels of transcription of adjacentDNA segments. Additionally, E. coli promoters produced by recombinant DNA or other synthetic DNA techniques may be used to provide for transcription of the inserted gene. When cloning in a eucaryotic host cell, enhancer sequences e.g., the 72 b.p. tandem repeat of SV40 DNA, retroviral long terminal repeats or LTRs, etc. may be inserted to increase transcriptional effeciency. Enhancer sequences are a set of eucaryotic promoter elements that appear to increase transcriptional efficiency in a manner relatively independent of their position and orientation with respect to a nearby gene. Unlike the classic promoter elements e.g., the polymerase binding site and the Goldberg Hogness TATAw box which must be located immediately 5 to the gene, enhancer sequences have a remarkable ability to function upstream from, within, or downstream from eucaryotic genes therefore, the position of the enhancer sequence with respect to the inserted BGH gene is less critical. Specific initiation signals are also required for efficient translation in procaryotic and eucaryotic cells. Protein translation is initiated as the result of interaction between two mRNA 8itcs the ribosome binding site SD sequence on the DNA complement and the initiation codon AUG ATG on the DNA complement . These translation initiation signals may vary in strength1 as measured by the quantity of gene specific protein synthesized. The DNA expression vector, in addition to containing a promoter for transcription, may also contain any combination of various strong translation initiation signals.Thus, any SD ATG combination that can be utilized by host cell ribosomes may be employed such cominations include, but are not limited to, the SD ATG combination from the cro gene or the N gene of coliphage lambda, or from the E. coli tryptophan E, D, C, B, or A genes. Additionally, any SD ATG combination produced by recombinant DNA or other synthetic technique may be used. Any of the methods previously described for the insertion of DNA fragments into a vector may be used to ligate a promoter and other control elements into specific sites within the vector. Accordingly, the BGH gene or any portion thereof can be ligated into an expression vector at a specific site in relation to the vector promoter and control elements so that the BGH gene sequence is in the correct translational reading frame i.e., in phase with respect to the vector ATG sequence. Alternatively, the vector and or gene sequence may be modified so that theATG or GTG or CTG of the gene sequence is used as the initiation signal. The resultant recombinant DNA molecule is then introduced into appropriate host cells by transformation, transduction or transfection depending on the vector host cell system . Transformants are selected based upon the expression of appropriate gene markers normally present in the vectors, such as ampicillin resistance or tetracycline resistance in pBR322, or thymidine kinase activity in eucaryotic host systems.Expression of such marker genes indicates that the recombinant DNA molecule is intact and is replicating.Expression vectors may be derived from cloning vectors, which usually contain a marker function such cloning vectors may include, but are not limited to the following SV40 and adenovirus vectors, yeast vectors, bacteriophage vectors such as lambda gt WES lambda B,Charon 28, Charon 4A, lambda. gt l lambda BC, lambda gt l lambda B, M13mp7, M13mp8, Ml3mp9, or plasmid DNA vectors such as pBR322, pAClOS, pVA51, pACYC177, pRE47, pACYC184, pUBllO, pMB9, pBR32S, Col El, pSC101, pBR313, pML2l, RSF2124, pCRl or RP4. In addition, host cell strains may be chosen which inhibit the action of the promoter unless specifically induced. In this way, greater than 95 of the vector s promoter s effectiveness may be inhibited in uninduced cells. As previously explained, in certain operons the addition of specific inducers is necessary for efficient transcription and translation of the inserted DNAt for example, the lac operon is induced by the addition of lactose or IPTG i.e., isopropylthio B D galactoside , A variety of other operons, such as trp, pro, tac etc., are under different controls.The trp operon is induced when tryptophan is absent in the growth media and the PL promoter of lambda is induced by an increase in temperature in host cells containing a temperature sensitive lambda repressor protein, e.g., c1857. Thus, expression of the genetically engineered GH protein may be controlled. This is important if production of the protein product of the cloned gene is lethal or otherwise detrimental to the host cells. In such cases, the foreign gene may be replicated but not expressed dpring growth of the transformants.After the cells reach a suitable density in the growth medium, the promoter can be induced for production of the protein. 5.2.1. CoNSTRUCTION OF AN HYBRID PROMOTER One such promoter operator system is the so called 1tac or tr lac promoter operator system referred.to as tac1 . This hybrid promoter is constructed by combining the 35 b.p. 35 region of the trp promoter and the 10 b.p. 10 region or Pribnow box of the lac promoter the sequences of DNA which are theRNA polymerase binding site . In addition to maintaining the strong promoter characteristics of the tryptophan promoter, tac is also easily controlled by the lac repressor from lac Iq . This construction may explain the higher efficiency of expression of this hybrid promoter with respect to either one of the parental promoters. 5.2.2. CONSTRUCTION OF BGH WITH SYNTHETIC FRONT END Expression levels of BGH DNA sequences derived from the cDNA library containing natural eucaryotic coding sequences were determined to be very low less than 0.011 of the total protein expressed by the cell in a number of expression vector constructions. In order to increase production of BGH, a DNA fragment coding for theN terminus of BGH i.e., amino acids 1 methionine through 24 is synthesized using state of the art procedures. This fragment SPE or synthetic front end is inserted in place of the natural front end NFE coding sequences of the BGE genome. The SFE is constructed so that the non preferred codons of 13 of the 24 amino acids are replaced with preferred1 codons for those amino acids.This is done for two reasons. First, in all highly expressed B. coli genes there is a certain codon bias for most of the amino acids of the corresponding protein. This bias correlates well with the intracellular pools of tRNA s for those preferred codons. By changing the codons in BGH which are non preferred to preferred codons, translational efficiency should increase thereby increasing overall expression. Second, certain regions on the mRNA transcript are very important for the initiation and subsequent translation of the protein which are recognized by the host cell ribosomes. Since mRNA is single stranded it has a tendency to fold and base pair with the complement sequences anywhere along the entire length of the transcript. If the regions critical for initiation are buried within this now folded molecule, translation, and therefore expression, will be decreased.However, if undesirable folding of the mRNA does not occur and these regions are readily accessible to the host cell ribosomes, translation initiation and elongation of the polypeptide are subsequently increased. 5.2.3 INSERTION OF THE SFE BGH INTO AN EXPRESSION VECTOR When the SFE BGE construction is inserted into the expression vector pDN6l2, a significant increase in expression levels of the met BGE protein is found BGH comprising greater than 0.1 of total host cell protein . When this construction SFE BGH was inserted into the expression vector p141 2, a pDR540 modified plasmid which utilize the tac trp lac promoter with lac and cro SD sequences a large increase BGH comprising greater than 1 of total host cell protein expressed in the amount of BGE being produced was observed. Finally, the SFE met BGE construct was inserted into a tryptophan expression vector pTNAP771 which utilizes the trp operon promoter and trp SD sequences for expression. Again, this construct showed a significant increase 1 in the amount of BGR being expressed by host cells. Since SPE constructs produced relatively stable proteins, they were modified further in order to increase stability and thus increase protein expression. 5.2.4. INSERTION OF A T RICH SEGMENT In order to improve the initiation of translation for BGE, an A T DNA segment can be inserted between the SDcrO and the ATG of BGH. A DNA oligomer which contains an A T rich sequence may be synthesized using state of the art nucleic acid chemistry. A particularly useful method is described in U. S. Patent ApplicationSerial No. 491,099 filed May 3, 1983, by Kempe et al., which is incorporated herein by reference. This A T rich segment may be needed to destabilize the secondary structure of the mRNA within this region and thereby facilitate ribosome binding and subsequent translation along the message. 5.2.5. TRANSCRIPTIONAL AND TRANSLATIONAL TERMINATION Sequence analysis of the cDNA clone of BGH pBE27 has revealed that only one translational stop codon TAG is present for termination of translation, and that no transcriptional stop like sequences are found in the 3 untranslated region of the cDNA clone. Therefore a triple translational stop sequence, with all 3 codons can be inserted in phase with the reading frame of the protein. Again, this can be accomplished by insertion of a synthetic linker using current nucleic acid chemistry. This construction should provide for efficient termination of translation of theBGH message by assuring proper translational termination with little change of read through into the normally untranslated region of the mRNA. Recent studies have shown that the tryptophan attenuator region of the tryptophan operon provides for the efficient termination of mRNA transcription from theDNA. This extremely stable stem and loop structure is characteristically GC rich and po8sesses dyad symmetry, followed by an oligo T stretch at the site of termination similar to the well known features of rho independent transcription termination sites in procaryotes. A sequence coding for the attenuator portion of the tryptophan operon may be synthetically constructed or isolated from E. coli chromosomal DNA and inserted into an appropriate expression vector. In the example of the present invention, the tryptophan attenuator region was placed approximately 100 b.p. 3 to the translational stopcodons of the BGH genome. This construction may now allow for the effective termination of transcription of the mRNA from the BGH genome. 5.3. CONSTRUCTION or pTRP VECTORS REGULATED BY THE LAC OPERATOR LacO Expression of cloned genes in bacteria can generally be controlled by strong promoters. The ptrp vector system did not provide for significant regulation of the trp promoter by the co repressor L tryptophan when used for the expression of BGH. Nucleotide sequence analysis of the trP operator region demonstrated that no sequence abnormalities existed, suggesting that the lack of regulation was probably due to the low number of tro repressor molecules relative to the high number of plasmids containing trp operator regions. Since high intracellular concentrations of BGH might be toxic to the bacterial cell, the cloned BGH was placed adjacent to a strong regulatable promoter operator region. In order to efficiently regulate the trp promoter, a unique trp vector System was constructed in which a synthetic lacO operator is inserted 17 nucleotides 3 to the 10 region of the trP promoter. The expression vector includes not only a trp promoter, but in addition, a lac operator which can be regulated by lac repressor in a lack9 strain of E. coli and thus efficiently repressBGH expression. The induction of this hybrid promoter operator system is easily and efficiently accomplished by the addition of lactose or IPTG to the bacterial culture at a phase of growth where BGH synthesis is maximal.This mechanism of trp promoter regulation circumvents many of the problems associated with the derepression of the tryptophan promoter by either thz addition of tryptophan analogues or by dilution of tryptophan. 5.4 iREPARATION OF FUSION PROTEINS To maximize the level of gene expression in a specific transformant it may be desirable to ligate the gene in question to a gene encoding another protein, such as a host cell protein. An additional advantage ifs obtained if the host cell protein inherently contains an assayable function. The expression of the ligated genes results in a fusion protein product hereinafter referred to as BGH fusion proteins that can be identified on the basis of its large molecular weight and assayable function. For example, production of a BGH galactosidasefusion protein offers several advantages for cloning and expression of BGR in an E. coli host. First, this allows for an approximation of the level of protein production hence expression directed by the vector using a colorimetric assay specific for a galactosidase activity according to the method of Miller pages 47 55 and 352 355 in Experiments in MolecularGenetics, Cold Spring Harbor Press, New York, N.Y., 1972 . Second, fusion protein production simplifies the identification and isolation of the protein product. The unfused BGH protein produced by E. coli transformants is smaller than the BGE ss galactosidase fusion protein and as such may co migrate with several other host proteins analyzed by SDS polyacrylamide gel electrophoresis.However, the fusion protein produced can be easily detected and identified by SDS polyacrylamide electrophoresis SDS PAGE due to its unique large molecular weight. The present invention is not limited to the production of a S galactosidase fusion protein any gene of either eucaryotic or procaryotic origin may be ligated in phase with the BGH gene to provide advantages similar to the O galactosidase fusion protein product. Examples include, but are not limited to, galactokinase trp D, E or leader pilus genes and eucaryotic genes, such as thymidine kinase, B globin, SV 40 T antigen, or RousSarcoma Virus transforming gene. In order to construct a gene which encodes a fusion protein, the two genes must be joined within their coding sequence such that the translational reading frame is maintained and uninterrupted by termination signals.Also, as previously explained, if the host cell is a strain which inhibits the action of the promoter, the fusion protein will be produced only in response to induction. 5.5. IDENTIFICATION OF TES GENE PRODUCT Identification of the protein described in this invention is based upon two requirements. First, theBGH related protein should be produced only in response to the induction of the promoter. Second, the BGH related protein can be detected immunologically using a variety of polyclonal or monoclonal antibodies directed against BGE the protein should be immunoreactive whether it results from the expression of the entire gene sequence, a portion of the gene seqAehnce or from two or more gene sequences which are ligated to direct the production of a fusion protein.This reactivity may be demonstrated by standard immunological techniques, such as immunoprecipitations, immunodiffusion, radio immune competition, immunoelectrophoresis or the immunological detection of proteins which were separated by polyacrylamide gel electrophoresis and then transferred to nitrocellulose. 5.6 PURIFICATION OF GENE PRODUCT Cells containing the cloned gene are grown up in a large volume, and the protein produced after induction of the promoter is isolated from such cells or from the medium if the protein is excreted. The protein may be isolated and purified either by standard chromatography including ion exchange, affinity or sizing resins, by centrifugation, or by any other standard technique for the purification of proteins. Since certain fusion proteins form aggregates when overproduced in cells and when in solution, a method for isolating aggregate forming proteins is particularly useful for isolating the fusion proteins produced in the present invention. These fusion proteins can then be used in vaccine formulations. Purification of fusion proteins hereinafter referred to as aggregate purification involves the extraction, separation and or purification of aggregate forming proteins from protein mixtures such as lysates of cell cultures by disruption of cells followed by washing the aggregated material.Additionally, the aggregated material may be solubilized by the addition of a solvent that is both a strong hydrogen acceptor and a strong hydrogen donor or a combination of solvents each having one of these properties the aggregate forming proteins may then be precipitated by dilution with a compatible buffer. Suitable protein solvents include, but are not limited to urea from about 4M to about 8M , formamide at least about 80 , volume volume basis , and guanidine hydrochloride tfr om about 4M to about 8M . Some solvents which are capable of solubilizing aggregate forming proteins, for example SDS sodium dodecyl sulfate , 708 formic acid, are inappropriate for use in this procedure because subsequent precipitation of the solubilized aggregate forming proteins has proved to be inefficient.Although guanidine hydrochloride and other similar agents are denaturants, studies indicate that this denaturation is not irreversible and that renaturation may occur upon removal by dialysis, for example or dilution of the denaturant, allowing re formation of immunologically and or biologically active protein. One embodiment of the fusion protein isolation technique is outlined as follows hereinafter referred to as the non denaturing aggregate purification procedure the cell pellet is quick frozen using dry ice methanol, weighed, and 3 4 g of cells are resuspended in at least 25 mQ of a buffer solution e.g., 50 mM Tris HCl tris bydroxymethylaminomethane hydrochloride , pH 8.0, 2mM EDTA ethylenediaminetetraacetic acid and 200 mM NaCll. That is, the cells are suspended in a buffer solution at an approximate concentration of from about 100 to 200 grams of cells liter. Concentrations less than about 160 grams liter are preferred.To this suspension lysozyme is added to a final concentration of about 130 yg mt and the resulting mixture is allowed to stand at 4vC for 20 minutes with occasional shaking. Nonidet P40 NP 40,Shell trademark, polyoxyethylene 9 R tert octylphenol , a non ionic detergent used to solubilize membranes, is added to a final concentration of about 0.1 and the solution mixed. Then, the suspension is ground for approximately 1 minute using a Polytron grinder BrinkmanInstruments, Westbury, N.Y. or equivalent. The suspension is centrifuged at 8,000 x g for 30 minutes, and the pellet.resuspended in a wash buffer, e.g., 20mM Tris HCl pH 7.2 , 1mm EDTA, 150mM NaCl and 2 Triton X 100 polyoxyethylene 9 10 p tert octylphenol, a non ionic detergentl, and ground with the Polytron grinder. This step of centrifugation, washing, and grinding may be repeated to further wash the pellet and remove as much cellular debris as possible. Alternatively, the pellet of aggregates may be further treated by the addition of a denaturant hereinafter referred to as the denaturing aggregate purification procedure as follows The suspension is centrifuged, the supernatant removed completely and the pellet resuspended in about one fifth volume of 6M guanidine hydrochloride in distilled water . For instance, 3 g of cells washed with 25 me of buffer should be resuspended at this step in 5 mll of 6M guanidine hydrochloride solution. It may be difficult to resuspend the pellet at this stage and sonication or homogenization may be required in order to obtain a homogenous solution.The solution is allowed to stand at 220C for 20 minutes and is then centrifuged at 8,000 x g for 30 minutes to remove debris, saving the supernatant which at this point contains the fusion protein. The fusion protein is precipitated from the guanidine hydrochloride supernatant by the addition of about four volumes of aqueous buffer. Almost any aqueous buffer may be used at this stageS however, the addition of a small amount of non ionic detergent, such as 0.58 NP 40 or Triton X 100 may be helpful. This suspension is allowed to stand at 40C for 30 minutes and is then centrifuged at 8,000 x g for 10 minutes. The supernatant is discarded, the pellet containing the fusion protein precipitate is resuspended in an appropriate buffer, e.g., Phosphate Buffered Saline PBS in any appropriate volume. Brief sonication or homogenization may aid in obtaining a homogenous suspension or flurry since aggregate forming proteins are insoluble in water . 5.7 PREPARATIQN OF INFUSED BGH As previously explained, transformants which produce unfused BGH proteins, generally do so in smaller quantities than transformants which produce EGH fusion proteins this is true even when the gene sequence for the unfused protein is under the control of an inducible promoter. In addition, the unfused BGE proteins produced by bacterial transformants may be less stable than BGH fusion proteins. In an alternate embodiment of the present invention, a host cell transformant can be engineered to produce large quantities of both fused and unfused BGH related proteins which will coaggregate and can be purified easily. Three embodiments of this mode of the invention are described in the subsections below. 5.7.1. CO TRANSFORMATION OF HOST CELLS Any appropriate host cell can be co transformed with two or more plasmids. One plasmid carries the desired unfused protein gene and one plasmid carries a fusion protein gene. Each gene should be under the control of a promoter and translational control elements.The same or different promoter systems may be used to control expression of each gene. If a different promoter is used for each gene sequence then the ratio of the expression of the sequences may be controlled by varying the degree of induction of each promoter. In addition, the two plasmids may carry different selectable drug markers for antibiotic resistance e.g., tetracycline and ampicillin resistance and may be maintained within the same cell if grown in the presence of the two antibiotics. With both plasmids in the cell, expression of the fusion protein and the labile unfused protein can occur. These transformed cells produce insoluble aggregates which contain both the unfused proteins and the large fusion protein. There is no requirement that the fusion protein be a BGH fusion protein. In fact, any combination of fusion protein with unfused protein will have the aggregation and stabilization properties. A significant stabilization of the unfused BGH protein was observed when two plasmids were used to transform one host cell, each plasmid directing the expression of its respective protein unfused BGH and porcine parvovirus a galactosidase fusion protein PPV 8 galactosidase . Pulse chase experiments indicated that the half life of the unfused modified BGH proteins had increased from a 1 2 minute level to between 30 minutes and 1 hour. Additionally, when these proteins were highly expressed within the cell, they form insoluble aggregates which contain both the unfused BGE protein and the large PPV B galactosidase fusion protein. Subsequent purification of the aggregate proteins indicated that the protein could be. readily obtained in gram quantities. 5. 7. 2. TRANSFORMATION OF BOST CELLS WITH ONE PLASMID CARRYING TWO GENES In one embodiment of the present invention, plasmids which carry both the unfused protein gene and the larger fusion protein gene on a single vector are designed and constructed. As described supra for the co infection scheme, this embodiment of the invention is suitable for any combination of fusion protein and any unfused protein. The plasmids are constructed such that each gene on the plasmid has its own promoter and expression control elements. The same or different promoter systems may be used to control expression of each gene, therefore this embodiment. of the present invention also allows for the regulation of the ratio of the fusion protein. and the desired gene product. 5.7.3. MODIFICATION OF TE FUSION PROTEIN GENE In an alternate embodiment of the present invention, a recombinant plasmid which encodes a BGH fusion protein is altered at the junction of the two gene sequences which comprise the fusion protein gene. A chain termination sequence such as amber TAG , ochre TAA , or opal TGA is located between the two gene sequences the chain terminator must be in phase with the translational reading frames of both gene sequences. Such an alteration may be accomplished by cleaving the plasnid at a restriction site or sites located between the two gene sequences and then ligating a nucleotide linker sequence encoding a chain terminator such as amber, ochre, or opal into the cleaved site on the plasmid so that the chain terminator is in phase with the translational reading frames of both gene sequences. Introduction of these amber, ochre, or opal modified plasmids into a host cell containing the appropriate tRNA suppressors results in the synthesis of both an unfused protein as well as a fusion protein sin suppression is significantly less than 1008 . A tRNA suppressor is a tRNA that has undergone a mutation that aLlows the tRNA to recognize the termination codon and results in the occasional but not regular insertion of an amino acid at the site of the termination codon. Therefore, host cells carrying the suppressor tRNA characteristically produce both the unfused protein and a fusion protein of normal length. Every nonsense or termination suppressor has a characteristic efficiency, indicated by the fraction of protein chains that are completed. There are at least two ways to introduce the amber, ochre or opal modified plasmids into a suppressor cell background 1 the transformant i.e., a host cell transformed with amber, ochre or opal modified plasmid can be infected with a lysogenic transducing phage that carries the appropriate suppressor tRNA gene e.g., gso pSU3 carries the SupF suppressor of amber mutations or 2 the amber, ochre or opal modified plasmids can be used to transform cell lines which contain suppressor tRNAs of amber, ochre, or opal respectively. Examples of such strains include but are not limited to LE392 containing supE and supF suppressors of amber mutations , ThC containing supF , and C600 supE .The various amber suppressor tRNA genes in E. coli include but are not limited to supB, supC, supD, supE, supF, supG, supL, supM, supN, supo, supP, supU, supV the various ochre suppressor tRNA genes in B. coli include but are not limited to supB, supC, supG, supL, supM, supN, supO, supV and the various opal suppreAsor tRNA genes in E.coli incude but are not limited to supK see Bachmann and Low, 1980, Microbiological Reviews 44 1 1 56 . The host cells containing the appropriate suppressor tRNA gene are transformed with the amber, ochre, or opal modified plasmids and can produce the.protein as both a fusion protein and as an unfused protein the proportions of fused to unfused protein produced depends upon the extent of suppression in the host cell . The amber, ochre, or opal modified plasmids may be further modified to ensure that translation of the corresponding mRNA transcript will terminate at the 3 terminus of the fusion protein gene. Proper chain termination is important because it contributes to the overall efficiency of translation. A number of approaches which may be used to effect chain termination are described below. In one embodiment, all three chain termination sequences may be inserted in tandem at the 3 region of the fusion protein gene so that they are in phase with the translational reading frame of the gene sequence. The presence of the chain terminate sequences in tandem will reduce the chance of read through, consequently, translation will terminate at the chain termination codons on the mRNA transcript. Alternatively, one or more chain termination sequences of the appropriate type may be inserted into the 3 region of the fusion protein gene. For example, an amber modified plasmid may be further modified by the insertion in phase of one or more opal or ochre sequences at the 3 end of the fusion protein gene. When this plasmid is inserted into a host cell containing an amber tRNA suppressor such as supD, supE, supF, supP, supU, supV in E. coli that does not suppress ochre or opal, the host cell will produce both the fusion protein and the unfused protein and translation of the fusion protein will terminate at the location of the opal and or ochre codons located at the 3 end of the mRNA transcript. In a similar fashion, an opal modified plasmid may be further modified by the insertion in phase of one or more amber or ochre sequences at the 3 end of the fusion protein gene. When this plasmid is inserted into a host cell containing an opal tRNA suppressor such as supK in E. coli that does not suppress amber or ochre, the host cell will produce both the fusion protein and the unfused protein and translation of the fusion protein will terminate at the location of the amber and or ochre codons located at the 3 end of the mRNA transcript. Similarly, an ochre modified plasmid may be further modified by the insertion in phase of one or more opal sequences at the 3 end of the fusion protein gene. When this plasmid is inserted into a host cell containing an ochre tRNA suppressor such as supB, supC, supG, supL, sup, supN, supO, supV in E. coli that does not suppress opal, the host cell will produce both the fusion protein and the unfused protein and translation of the fusion protein will terminate at the location of the opal codon or codons located at the 3 end of the mRNA transcript. 5.8. INSERTION OF BGH SEQUENCES INTO HIGH COPY NUMBER VECTORS The high copy number ori vector pOP1 6, described suPra in Sec. 2.1.3 can be used in order to increase the expression levels of the BGH gene by amplification of the number of plasmid and therefore gene copies, thereby increasing the BGE mRNA and hence the BGH protein levels within the cell. These plasmids are referred to as ori, high copy number, or runaway plasmids. When the SFE met BGH construction p217 1, was inserted into the. runaway plasmid vector, pORI tetEl a tetracycline resistant derivative of the ampicillin resistant pOP1 , plasmid copy numbers were determined to exist at levels of approximately 200 300 copies per cell as compared to 20 40 copies per cell for p217 1.Expression levels of BGH were determined by SDS PAGE to range from 10 to 18 of the total bacterial cell protein. BGH protein in these constructions was found to be extremely stable i.e., half life greater than one hour within the cell, probably due to the production of insoluble aggregate forming proteins. 6. EXAMPLE Bovine Growth Hormone According to this embodiment of the present invention the BGH mRNA was isolated and identified and a cDNA was synthesized using the mRNA as a template. The cDNA fragments of the BGH gene were then inserted into a pBR322 vector to form several recombinant plasmids. These plasmids were then used to transform E. coli bacterial hosts. Those clones positive for the insert were then identified. The plasmid containing the BGH gene was identified using two techniques not necessarily in the order listed a DNA sequencing and b BGE specific mRNA hybridization experiments. After the BGH gene was identified, it was excised from the non expression plasmid. To increase expression efficiency, a synthetic amino terminus was inserted in place of the natural amino terminus. This DGE gene with the synthetic front end AtFE met BGE was then inserted into an expression plasmid pJS413 or pDN6l2. To decrease the proteolytic destruction of the newly synthesized BGE by the host cell and thus increase the yield of gene product, the BGH expression plasmid was co infected into a host along with an expression plasmid encoding the fusion protein PPV ss galactosidase.Alternatively, host cells were transformed with one expression plasmid encoding both the SFE met BGH andPPV ss galactosidase. The co expression of BGH and fusion protein decreased the rate of proteolytic degradation and simplified purification of the BGH protein. 6.1. GENERAL PROCEDURES USED FOR PREPARATION OF THE PLASMIDS The following subsections describe the general procedures and materials used for DNA isolation, enzyme reactions and ligation reactions. 6.1.1. PLASMID DNA ISOLATION Host cell bacteria Escherichia coli, E. coli were transformed Kushner, p. 17 in Genetic Engineering, ed. H.B. Boyer and S. Nicosia, Blsevier North Bolland,Amsterdam, 1978 , and large tmicrogram quantities of plasmid DNA were isolated from cultures of E. coli transformants grown in LB medium Maniatis et al., p. 88 in Molecular Cloning A Laboratory Manual, Cold SpringHarbor Press, New York, N.Y., 1983 . Plasmids were isolated from cells in late logarithmic stage of growth using a modification of. the method of Godson and Vapnek 1973, Biochim. Biophys. Acta. 229 516 . 6.1.2. CONDITIONS FOR RESTRICTION ENZYME DIGESTIONS Restriction enzymes used in the present application were obtained from New England Biolabs, Inc.,Beverly, MA, unless otherwise indicated. An enzyme unit is defined as the amount required to digest 1.0 pg of lambda DNA in 60 minutes at 370C in a total reaction mixture of 50 pt. All restriction enzyme total digestions were accomplished under the following conditions each 1 g DNA was incubated with 2 5 units of enzyme at 370C for 60 minutes in 20 pX buffer. Partial digestions were accomplished by modifying the conditions used for total digestion as follows each 1 Vg DNA was incubated with 0.1 units of enzyme at 370C for 15 minutes. Reactions were terminated by the addition of 0.18 sodium dodecyl sulfate SDS . Thus, the reaction conditions were adjusted to obtain an average of one cleavage per DNA molecule. 6.1.3. RESTRICTION ENZYME BUFFERS The buffer used for BpaI or SmaI digestions consisted of 2OmM KCl, lOzaM Tris ECl pE 7.4 , lOmM MgC12 and 1mM DTT. The buffer used for AhaIII, Aval, BamHI, BglII,ClaI, EcoRI, EcoRV, HaeII,, HaeIII, HincII, HindIII, Hinfl, NarI, NcoI, PstI, PvuII, RsaI, Sac, SalI, Sau3A 1,TagI, XbaI or XmnI digestions consisted of 50mM NaCl, lOmM Tris HCl pH 7.4 , 10mM MgC12 and lmM DTT. It should be noted that whenever two or more restriction endonuclease reactions are performed on DNA, the reaction in low salt buffer is accomplished before the reaction in high salt buffer. If two enzymes require the same buffer, then the reactions may be performed simultaneously. 6.1.4. MODIFICATION OF DNA Bal 31 nuclease is a multifunctional enzyme that contains a highly specific single stranded endodeoxyribonuclease activity and a processive exonuclease activity that simultaneously degrades both the 3 and 5 termini of double stranded DNA dsDNA . One unit of nuclease Bal 31 New England Biolabs, Inc.Beverly, MA is defined as the amount required to release 1.0 g of acid soluble nucleotides from denatured calf thymus DNA 650 g ml in one minute at 30 C. The reaction buffer used for Bal 31 consisted of 20mM Tris ECI pH 8.0 , 600mM NaCl, 12mM CaC12, 12mM MgCl2, l.OmM EDTA and DNA. Incubations were at 300C Bal 31 digests were accomplished by incubating 30 g of DNA with 0.5 units Bal 31 for 1 to 10 minutes. Reactions were terminated by the addition of EDTA to 50mM or heat inactivation of Bal 31 i.e., 650C for 10 minutes . The enzyme terminal deoxynucleotidyl transferase TdT catalyzes the polymerization of deoxynucleoside triphosphates at the 3 termini of single stranded DNA or oligodeoxynucleotides. By using cobalt ion instead of magnesium ion as cofactor, the specificity of the enzyme is changed so that it can be used for terminal labeling and addition of homopolymer tails to duplex DNA fragments. Such incorporation can be made at the 3 OH terminus of duplex DNA with either base paired or staggered ends. The homopolymeric tail may be composed of any one of the four nucleotides as the repeating unit dCTP, dGTP, dATP or dTTP. One unit of activity of TdT Bethesda Research Laboratories, Gaithersberg, MD is defined as the amount of enzyme that incorporates Inmol dATP into acid precipitable material in one hour at 370C, using d pA 50 as primer. S1 nuclease and mung bean nuclease degrade RNA or denatured DNA i.e., single stranded DNA into mononucleotides, but will not under proper conditions degrade double stranded DNA or DNA RNA hybrids. One unit of Sl nuclease Boehringer Mannheim, Indianapolis, Ind. is defined as the amount of enzyme required to acid solubilize 1 Vg of denatured calf thymus DNA in 30 minutes at 37 C. The reaction buffer used for S1 nuclease consisted of 25mM sodium acetate pH4.6 100mM NaCl, 0.lmM ZnSO4 and 5 glycerol. SI digestions were accomplished by incubating 2000 units of enzyme with 0.1. Vg of DNA at 37QC for 30 minutes.One unit of mung bean nuclease P LBiochemicals, Milwaukee, Wits. is defined as that amount of enzyme which produces 1 Ug of acid soluble material per minute at 37 C. The reaction buffer for mung bean nuclease consisted of 30mM NaAc pH5.0 200mM Nail, 5 glycerol and lmM ZnS04. Mung bean digestions were accomplished by incubating 20 units of enzyme with 1.0 Vg DNA at 370C for 30 minutes. 6.1.5. DNA POLYMERASE REACTION DNA polymerases catalyze the stepwise elongation of a DNA chain. Chain growth is in the 5 to 3 direction i.e., addition of nucleotides occurs at the 3 terminus and the sequence of the polymer is determined by that of the template because the incoming nucleotide must be complementary to the template, i.e., form the correct base pair with the template strand. The known DNA polymerases cannot initiate chain synthesis, thus DNA synthesis requires a primer strand with a free 3 hydroxyl terminus annealed to a template strand. Cbain elongation proceeds by the nucleophilic attack of the 3 hydroxyl terminus of the primer on the 5 phosphate of the incoming nucleotide. The new chain is base paired and antiparallel to the template strand. As a result of the DNA polymerase reaction the single stranded template strand is filled in or converted to a double stranded DNA molecule. A second important feature of the DNA polymerases is the proof reading function. A 3 to 5 exonuclease activity associated with DNA polymerase I acts on nonbase paired termini and can degrade both single and double stranded DNA in the 3 to 5 direction yielding mononucleotides. Thus an incorrectly added nucleotide is removed before polymerization continues, and the fidelity of the enzyme is further enhanced. DNA polymerase I also has a 5 to 3 exonuclease activity specific for double stranded DNA yielding 5 mononucleotides and oligonucleotides which can also excise mismatched regions in DNA. Proteolytic treatment of DNA polymerase I by the method of Klenow Klenow et al., 1971, Bur. J. Biochem. 22 371 yields two fragments, large and small. The large fragment or Klenow fragment 76,000 daltons retains the polymerase and 3 5 exonuclease activity whereas the small fragment retains the 5 3 exonuclease activity.One unit of DNA polymerase I or DNA polymerase I large fragment New England Biolabs, Beverly, MA is defined as the amount converting 10 nmoles of deoxyribonucleotides to an acid insoluble form in 30 minutes at 370C. The assay conditions for both enzymes are the same except that the reaction mixture for DNA polymerase I contains 67mM KPO4 pH 7.5 while the reaction mixture for the Klenow fragment contains 40mM KPO4 pH. 7.5 in the following buffer 6.6mM MgCl2, 1.0mM ss ME, 2mM dAT copolymer, 33 M dATP, 33 M H dTTP and enzyme. In the present application, when DNA polymerase large Klenow fragment was used to fill in single stranded DNA or cohesive ends that result from restriction enzyme cleavage, the following procedure was employed restriction enzyme reactions were terminated by heating at 680C for 10 minutes. To the same reaction mixture a final concentration of 50gM of each deoxynucleosidt triphosphate was added and for each microgram of DEN in the reaction mixture 10 100 units of DNA polymerase Klenow fragment was added. In other words an excess of DEA.polymerase Klenow fragment was used to ensure that sir.isle stranded ends were completely filled in. 6.1.6. GEL PURIFICATION OF DNA FRAGMENTS After restriction enzyme or nuclease treatment,DNA fragments of varying sizes were separated by gel electrophoresis in either agarose or polyacrylamide slab gels at low voltage approximately 2 volts cm for agarose gels and 10 volts cm for polyacrylamide gels , stained with ethidium bromide and visualized under ultraviolet light Southern, 1979, Methods in Enzymology 68 152 . In order to recover particular DNA fragments from gels, the appropriate bands were excised from the gel and the DNA was electroeluted into dialysis tubing. The DNA was then isolated on DEAE cellulose, or ethanol precipitated, and resuspended in the appropriate buffer Smith, 1980, Methods in Enzymology 65 371 . Alternatively, DNA restriction digests were separated by low melting point agarose LMP agarose gel electrophoresis Bethesda Research Laboratories, Inc.,Gaithersburg, MD . After electrophoresis the particularDNA fragment of interest was excised from the gel and theDNA was separated from the LMP agarose by heating at 650C, followed by phenol extraction, ethanol precspitation, and resuspension in the apprdprfate buffer Weislander, 1979,Anal. Biochem. 98 305 . 6.1.7. DNA LIGATION All ligations were accomplished using T4 DNA ligase New England Biolabs, Inc., Beverly, MA . One unit of T4 DNA ligase is defined as the amount required to yield 50 ligation of HindIII fragments of bacteriophage lambda DNA in 30 minutes at 160C in 20 D of ligase buffer at a 5 DNA termini concentration of 0.12 M 300 pg mQ . DNA ligations were carried out in ligase buffer consisting of 60mM Tris HCl pH 7.8 , lOmM MgCl2, 20mM dithiothreitol DTT , l.OmM ATP and a DNA concentration ranging from 15 50 g ml. Ligation reactions were incubated 4 to 24 hours at room temperature using approximately 300 units of T4 DNA ligase per 10 l reaction volume. 6.1.8. IDENTIFICATION OF BGH PROTEINS BY IMMUNOPRECIPITATION WITH ANTI BGH SERA Cultures of clones were grown overnight in AF media Krzyzek, R., and Rogers, P., 1972, J. Bacteriol.110 945 954 with 1 glucose and without added methionine at 300C. Aliquots 0.4 ml were removed, washed once inAF medium with 1 glucose without added methionine and resuspended in 1.0 n of the above medium. After incubation at 300C for 30 minutes, IPTG was added to lmM.The cultures were further incubated for 30 minutes and then 35S methionine 1440 Ci mmol was added to a concentration of 50 Ci ml for 1.5 minutes. Pulse labeling was terminated by placing the cultures in an ice water bath. Bacteria were. collected by centrifugation and resuspended in 50 l of solution containing 25 sucrose and 0.05M Tris HCl pH 8.0 . A 50 p solution containing 0.02M EDTA pH 8.0 and 500 g lysozyme was added.The cell suspension was then incubated at 40C for 10 minutes, 400 l of TET buffer 10.LM Tris ECl pH 8.0 , 0.05M EDTA, and 2 Triton X 100 polyoxyethylene 9 10 p tert octylphenol was then added for 5 minutes and lysis was completed by the addition of 500 pA of RIPA buffer 0.OlM Tris SCl pH 7.5 , 0.15M NaCl, 1 deoxycholate, 1 Triton X 100 and 0.1 SDS .Formaldehyde fixed Staphylococcus cells 100 l were added Pansorbin Staphylococcus Aureus Cells, Calbiochem,La Jolla, CA . The cell lysate was incubated at 40C for an additional 5 minutes and then centrifuged. The supernatant was removed and incubated with 10 iL of anti BGE sera for 30 minutes at 40C. Formaldehyde f ixed Staphylococcus cells 200 pt were added, and after 10 minutes the immune complex was collected by centrifugation Kessler, S.W., 1976, J. Immunol., 177 1482 1490 . The immune complex was then washed and analyzed by SDS PAGE Krzyzek, R.A, et al., 1980, J. Virol., 36 805 815 . 6.2. ISOLATION AND MOLECULAR CLONING OF THE BGH GENE The following subsections describe the molecular cloning of BGH cDNA. A recombinant plasmid, designated pBE27 contained the entire BGH coding sequence. 6.2.1. ISOLATION OF mRNA Messenger RNA was extracted in several steps.Bovine pituitaries were disrupted with a VirTis tissue homogenizer VirTis Company, Gardiner, N.Y. in a solution containing 4M guanidinium thiocyanate, 0.5 sodiumN lauroyl sarcosine, 0.lM ss mercaptoethanol, 0.1 of 30 antifoam A Sigma, St. Louis, Missouri , and 0.025M sodium citrate PR 7.0 . Cellular debris was removed by centrifugation in a Sorvall HB 4 rotor at 10,000 X g for 10 minutes at 40C and the supernatant was layered onto a 5.7M CsCl cushion buffered with 0.025M sodium acetate pH 5.0 and containing 0.2 diethyl pyrocarbonate. Chirgwin,J.M. et al., 1979, Biochem., 18 5294 5299 . The RNA was pelleted by centrifugation in a SW28 rotor at 90,000 X g for 16 hours at 150C and the resultant pellet was resuspended in a solution containing 0.05M sodium acetate PR 5.1 , 0.01M ethylenediaminetetraacetic acid EDTA , and 1 sodium dodecylsulfate SDS , and then extracted at 560C with an equal volume of phenol saturated with the above buffer. The RNA was precipitated twice in the presence of 0.2M sodium acetate with 2 volumes of ethanol, collected by centrifugation at 13,000 X g for 30 minutes, and suspended in TE buffer 10.01M Tris HCl pH 7.4 0.01M EDTA . Following the extraction of RNA from bovine pituitary glands, poly A containing RNA was isolated from the mixture of RNAs extracted. Accordingly, an aliquot of bovine pituitary RNA was adjusted to 0.5M NaCl and 0.5 SDS and applied to an oligo dT cellulose column. Miller, W.L. and McCarthy, B.J., 1979, J. Biol. Chem.254 742 748 . The column was washed extensively with loading buffer 0.0lM Tris HCl pH 7.5 , 0.5M NaC1, 0.00lM EDTA, and 0.5 SDS and the poly A containing RNA was.eluted with buffer containing 0.01M Tris ECl pH 7.5 and 0,56 SDS and concentrated by ethanol precipitation. 6.2.2. PREPARATION OF cDNA FROM BGH mRNA Following the preparation of poly A containingRNA, double stranded complementary DNA cDNA was prepared for cloning. The poly A RNA was enzymatically converted to cDNA using the enzyme RNA dependent DNA polymerase reverse transcriptase . Reverse transcriptase is used chiefly to transcribe RNA into DNA, which can then be inserted into procaryotic vectors. The primer for this reaction can be oligo dT or a collection of randomly generated oligodeoxynucleotides. If randomly generated oligodeoxynucleotides are used as primers, the diversity of these molecules must be so large as to guarantee that some of them will be complementary to sequences in the template nucleic acid. Because different oligonucleotides bind to different sequences in the template, all parts of the template are represented in the resulting DNA at equal frequency. Furthermore, random oligonucleotides can be used as primers on any single stranded RNA template.By contrast, olsgo dT binds only to poly A located at the 3 end of the RNA molecule, and therefore primes the synthesis of DNA that is heavily biased toward sequences at the 3 end of the mRNA template. Treatment of the RNA with reverse transcriptase results in a RNA cDNA hybrid which must be separated or the RNA strand destroyed in order to retrieve the single stranded cDNA molecule representing the gene to be cloned. The separation or RNA destruction may be accomplished by treatment with hot alkali or with ribonuclease which will solubilize or digest the RNA component the resulting cDNA is a single stranded DNA molecule which is then isolated from the reaction mixture using standard techniques such as rate zonal sedimentation, electrophoresis, chromatography, etc. Single stranded cDNA was prepared in several steps. Twenty micrograms of polyadenylated bovine pituitary RNA was reverse transcribed into single stranded complementary DNA in a 200 pL reaction containing 4 Vg of oligo dT 12 18 primer, 0.05M Tris HCl pH 8.0 , 0.00sum MgC12, 0.Q28M KCl, 0.004M sodium pyrophosphate, 0.001M B mercaptoethanol O ME , 200us each of dGTP, dCTP, dTTP and dATP, and 40 units AMV reverse transcriptase.One unit of AMV reverse transcriptase Bethesda ResearchLaboratories, Inc., Gaithersburg, MD is defined as the amount of enzyme which incorporates one pmol of deoxythymidine 5 triphosphate into acid precipitable product in 1 minute at 370C using poly rA oligo dT 12 18 as the template primer. After incubation at 370C for 60 minutes, EDTA was added to 0.02M, the mixture was then heated for 1 minute at 1000C, treated with pancreatic ribonuclease 40 pg m for 30 minutes at 37 C, and extracted with phenol three times. The single stranded cDNA was then purified by Sepharose 4B chromatography and concentrated by ethanol precipitation. After the preparation of single stranded cDNA, double stranded cDNA was obtained. This was done by treating the cDNA species with E. coli DNA polymerase I.The reaction was carried out at pE 6.9 to minimize the 5 to 3 exonuclease activity of DNA polymerase I and at 150C to minimize the possibility of synthesizing snap back DNA. The Klenow fragment of DNA polymerase I has also been succeqsfully employed to synthesize the second cDNA strand. Reverse transcriptase may also be utilized to synthesize the second cDNA strand using conditions similar to those already described for the synthesis of the first cDNA strand. Both DNA polymerase I and reverse transcriptase may be used together in the conversion of single stranded to double stranded DNA. After synthesis of cDNA was complete, the first and second strands were still covalently joined by the hairpin loop that was used to prime the second strand synthesis. This loop is susceptible to cleavage by the single strand specific nuclease, S1. However, the resulting termini are not always perfectly blunt ended, and the efficiency of cloning is improved if they are filled in with the Klenow fragment of E. coli DNA polymerase I. Alternatively, mung bean nuclease may be used to cleave the two strands. Although mung bean nuclease and nuclease 51 are similar to one another in their physical and catalytic properties, there are indications that mung bean nuclease may be less severe in its action than nuclease Sl. Second strand synthesis was performed in a 40 l reaction solution containing 200 500 ng of single stranded cDNA, E. coli DNA polymerase I Klenow fragment, 7 units , 300 M each of dGTP, dCTP, dTTP and dATP, 0.01M MgC12 O.1M KC1 and 0.02M HEPES pH 6.9 and incubated for 18 hours at 150C. Five volumes of solution containing 0. 03M sodium acetate pE 4.5 , 0.3M NaCl, 0.003M ZnCl2 andmung bean nuclease 1 unit was added to digest single stranded hairpins. The mixture was incubated for 30 minutes at room temperature, heated to 650C for 5 minutes and extracted three times with phenol.Double stranded DNA was purified by Sepharose 4B chromatography and concentrated by ethanol precipiFatio. 6.2.3. CLONING OF BGH cDNA After preparation of the BGH double stranded cDNA, the cDNA was prepared for insertion into a suitable vector. This was accomplished by using the enzyme terminal deoxynucleotidyl transferase TdT to attach homopolymer tails to the double stranded cDNA molecule.The tailed BGH cDNA was then inserted into a cloning vector that has been cleaved at a unique restriction site and tailed with a homopolymer that is complementary to the 3 tails of the cDNA molecule. In the present invention, the cDNA was poly dC tailed. The circular plasmid pBR322 was then cut with the restriction enzyme PstI a unique site on the plasmid and dG tailed as described above.At least 10 dGMP nucleotides were added to each 3 end of the double stranded plasmid DNA. The dC tailed BGH cDNA was allowed to anneal to the PstI cleaved, dG tailed plasmid DNA to form a recircularized recombinant plasmid containing the BGH cDNA insert. The recombinant molecules were used to transform appropriate host cells. The recombinant molecules were then isolated from transformants, analyzed by restriction endonuclease digestion and gel electrophoresis in order to map or characterize the inserted BGH DNA sequence.Details are described below. Single stranded tracts of homopolymeric deoxycytidine were added to the 3 ends of the double stranded cDNA in a 50 l reaction containing 10 to 100 ng of double stranded cDNA, 0.1M sodium cacodylate pH 7.2 , 0.006M CoCl2, 0.002M dCTP and 20 units TdT. The mixture was incubated for 1 minute at 370C and the reaction was terminated by the addition of EDTA to 0.02M and heating at 650C for 3 minutes. Plasmid pBR322 was cut with PstI and 500 ng was similarly tailed with deoxyguanosine by incubation at 379C for 5 minutes. In order to anneal the cDNA to the plasmid, both reaction mixtures were combined, heated to 65 C for 5 minutes, incubated for 2 hours at 429C, and then allowed to cool slowly to room temperature overnight. E. coli K 12 MC1000 2 mt of cells in the logarithmic phase of growth was rendered permeable to the annealed DNA mixture by incubation in 0.1M MOPS morpholinopropane sulfonic acid, pH 6.5 , 0.05M CaCl2, and 0.01M RbC12 for 30 minutes at 40C then 3 pQ dimethylsulfoxide DMSO and 1 200 ng of annealed DNA in a volume of 10 l or less of TE 10mm Tris ECl, lmm EDTA pH 8.0 was added and incubated for 30 minutes at 40C. The mixture was warmed at 420C for 1 minute, mixed with 10 volumes of L broth, incubated for 1 hour at 370C without shaking and finally plated onto L agar containing 20 g l tetracycline.Tetracycline resistant, ampicillin sensitive transformants containing BGH DNA were identified by in situ hybridization using 1251 labeled pituitary poly A containing ENA as a probe Soreq, E., et al., 1979, Nucleic Acids Res., 6s2471 2481 . The plasmid designated pBH27 which was isolated from one clone was found by restriction enzyme analysis to contain a largeBGH cDNA insert FIG. 1 .The BGH DNA insert of the pBH27 plasmid was sequenced by the method of Maxam and Gilbert 1980, Methods in Enzvmology, 65 499 580 . Sequencing data FIG. 2 indicate that the BGH DNA insert is 795 nucleotides in length and, more importantly, contains the entire coding sequence for pre BGH as well as additional nucleotides corresponding to the 5 and 3 untranslated regions of BGH poly A containing RNA. The nucleotide sequence of BGH mRNA was reported by Miller et al. 1980, J. Biol. Chem., 225 7521 7524 . The differences between the reported mRNA sequence and the pBE27 sequence are indicated in FIG. 2. The nucleotides of the published mRNA sequence that differ from those of pBE27 are indicated above the pBH27 sequence and nucleotides present in the pBH27 sequence but not found in the published sequence are indicated above the pBH27 sequence by a single asterisk The nucleotide changes described above in the coding region of the mature BGH are all silent changes.These differences may represent allelic variability and or are changes associated with the infidelity of reverse transcription. Within the text that follows, DNA fragments are identified according to their length in base pairs b.p. and according to their termini generated by restriction enzymes. All DNA fragment sizes which are represented in terms of base pairs are determined from DNA sequence data FIG. 1 . If a DNA fragment has a single stranded cohesive end, the b.p. number indicated for that fragment includes the nucleotides comprising the single stranded cohesive end. The termini of each DNA fragment containingBGH sequences are indicated on either side of a slash in the order of the direction of transcription of the BGH DNA sequence. For example, a 907 b.p. HindIII BglII DNA fragment signifies a DNA fragment which has a SindIII cohesive end at its amino coding terminus, a BglII cohesive end at its carboxy coding terminus, and is 907 nucleotides in length. 6.3. CLONING AND EXPRESSION OF BGH DNA SEQUENCES Portions of the BGH DNA of. pBH27 were subcloned into expression vectors so that the BGH DNA was placed under the control of various promoters. Some of the plasmids described below are described in U.S. applicationSerial No. 449,187 by J. Salstrom et al., filed December 13, 1982, which is herein incorporated by reference. 6..3.1. CLONING AND EXPRESSION OF BGH IN pJS413 The expression vector, pJS413 FIG. 3 , is a pBR322 derivative which contains the ampr ss lactamase gene, a lac promoter Placuv5 , lac and cro ribosome binding sites SDlacZ and SDcro are represented in all figures as SDZ and SDcro, respectively , a chain initiation ATG with 65 nucleotides of cro cro , and a modified L3 gala ctosidase gene the lac i z gene, hereinafter referred to as the z gene .The plasmid has unique cloning sites which span the cro A junction BctlII, SmaI, and BamHI. The z gene is not in the same translational reading frame with the cro ATG, thus the intact plasmid does not direct the production of a galactosidase. However, when a DNA fragment of the appropriate length is inserted into any one of the cloning sites in this plasmid 3n l , the reading frame of the z gene may be readjusted with respect to the cro ATG. If the inserted DNA sequence contains no termination signals e.g., TGA, TAA, or TAG that are in phase with the croATG or z gene, a galactosidase fusion protein will be produced by host cell transformants. Portions of the BGE DNA sequence from pBH27 were inserted into the expression vector pJS413 according to the following procedure. A Sau3A fragment approximately 798 b.p. which encodes approximately 280 b.p. of pBR322 and two thirds of BGE the amino terminus were isolated from the cDNA clone pBH27 see FIG. 3 . The Sau3A fragment was then digested with SmaI which generated a small 10 b.p. fragment from the BGH 3 end and a 785 b.p.Sau3A SmaI fragment the 785 b.p. fragment contains approximately 505 b.p. of the BGE DNA sequence. The expression vector, pJS413 see FIG. 3 was digested with Smal and BqlII to generate a SmaI BglIn backbone. The 785 b.p. Sau3A SmaI pBH27 fragment was ligated to the SmaI BqlII pJS413 backbone. N.B. the Sau3A cohesive ends are complementary to the BqlII cohesive ends. This vector was used to transform E. coli MC1000 and plated onL agar containing 100 ygZmD of ampicillin. Ampicillin resistant transformants containing theBGH DNA were identified by in situ hybridization using a radiolabeled PvuII PvuII fragment of pBH27. One clone which contained the BGH insert was amplified and plasmidDNA was purified. This plasmid was designated p3 30 IM. FIG. 3 shows the construction of p3 30 IM containing approximately 280 b.p. of pBR322 and 505 b.p. encoding the amino terminal portion of BGH. In order to remove the pBR322 portion of the inserted DNA, p3 30 IM was linearized with BglII and digested with Bal 31 for varying times see FIG. 3 . A number of linearized p3 30 IM molecules of varying lengths were generated. These were recircularized by ligation in the presence of DNA linkers which encode the BglII restriction endonuclease recognition sequence 5 CAGATCTG 3 which resulted in a population of p3 30 IM plasmids of different sizes wherein the BqlII site was restored. The ligation mixture was used to transform E. coli MC1000 and the plasmids were amplified and DNA purified.The isolated plasmids were digested with Smal and BqlII DNA fragments of approximately 350 to 500 b.p. encoding BGH sequences were isolated by agarose gel electrophoresis see FIG, 4 These BlII SmaI BGH fragments were ligated into pJS413 which had been linearized by cleavage with SmaI and BqlII FIG. 4 . The ligation mixture was used to transform E. coli MC1000. Cells were plated on MacConkey agar and red colonies were isolated Apte, B.W., et al., 1975, Nature, 257 329 331 . The red colonies were screened for BGH DNA by in situ hybridization Grunstein,M. and Hogness, D.S., 1975, Proc. Natl. Acad.Sci., 72 3961 3965 and then assayed for the expression of BGH B galactosidase fusion proteins by an in situ immunoassay Kemp, D.J. and Cowman, A.E. 1981, Proc. Natl.Acad. Sci. 78 4520 4524 . Plasmid DNA isolate from several expressing clones was cleaved with BlI. SmaI and analyzed by agarose or polyacrylamide gel electrophoresis to determine the size of the BGH DNA inserts. The Bg1II smaI DNA fragment of one clone, pRedl3 FIG. 4 , was slightly larger than aDNA sequence that could encode a mature BGH protein. The cro BGH B galactosidase fusion protein expressed by E.coli transformed with pRedl3 contained 135 amino acid residues of BGE i.e., amino acid residue number 3 to 132 in FIG. 2 which included 3 amino acid residues of the BGH leader sequence which are normally not present in the mature BGH protein. Plasmid pRedl3 was then employed to generate several BGH DNA constructions as described in the following examples. 6.3.2. CLONING AND EXPRESSION OF BGH IN pDN572 MET PLASMIDS A series of treatments was devised starting with plasmids pJS413 and pRedl3 to obtain the expression of several modified met BGH proteins. The experimental scheme follows. The construction of cro deletion derivatives of pJS413 is discussed in detail in U.S. application SerialNo. 449,187 supra . Briefly, one such expression vector, pDN572 is a pJS413 derivative which was prepared as follows varying size deletions of the cro gene sequences in the plasmid pJS413 were generated by digestion with BqlII followed by limited Bal 31 treatment and ligation in the presence of BqlII linkers. One of the pJS413 derivatives, designated pDN572, is depicted in FIG. 5. pDN572 includes the cro ATG plus the next 7 nucleotides of cro. As discussed below, pDN572 was used to construct a series of modified met BGH plasmids. A BglII SmaI BGH DNA fragment approximately 405 b.p. was isolated from pl3EX. This fragment was then ligated to the BqlII SmaI backbone of pDN572 FIG. 5 .Transformed E. coli NF1829 were plated on L agar containing 100 yg mQ of ampicillin. Clones were screened for BGH DNA by in situ hybridization utilizing a radiolabeled PvulI PsuII fragment of pBH27. Plasmid DNA designated p572 13 1 was amplified and isolated from one clone see FIG. 5 . In order to remove the BGH leader sequence and cro coding sequence, p572 13 1 was linearized by digestion with BqlII and treated with Bal 31 for time periods of 20, 40, and 60 seconds to remove approximately 5 to 20b.p. in each direction from the BqlZI site. The DNA was then ligated the BqlII recognition site is now removed and used to transform E. coli NF1829 which was plated onMacConkey lactose agar plates. 8 galactosidase positive colonies were selected and red colonies were isolated. Plasmids isolated from transformants expressing S galactosidase were screened by restriction analysis using the restriction enzyme HaeII. Plasmids found to be HaeII a site at the start of the mature BGH sequence see FIG. 2 were sequenced by the dideoxy chain termination method Sanger et al., 1977, Proc. Natl. Acad. Sci. 74 5436 5467 . The carboxyl coding terminus of the BGH was ligated into the met plasmids in order to reconstruct theBGH gene by inserting a Smal EcoRI fragment from pBH27 approximately 1042 b.p. into the homologous sites in the met plasmids FIG. 6 . The expression of the reconstructed BGH by host cells transformed with the modified met plasmids was determined by immunoprecipitation with anti BGH sera and SDS PAGE analysis. Reconstruction of the 3 carboxy coding terminus of BGH as provided herein results in no fusion protein production by transformants the i z gene is removed from the expression plasmids during this procedure . FIG. 7 shows pertinent nucleotide and corresponding amino acid sequences of several HaeII plasmids.The method described above provides a general method for the insertion of the carboxy terminal coding segment of BGH wherein the nucleotides encode amino acid residues argy33 through phe191 as shown in FIG. 2 . One BGH construct designated clone pl 40 C was determined to produce BGH protein at a higher level thanany other previous construct greater than 1 . As shown in FIG. 7, the 5 end of this clone has a number of changes from the mature BGH sequences. It is interesting to note that the first five codons in the pl 40 C sequence are preferred type codons i.e., those which code for the major isoaccepting species of tNk s within the E. coli cell. Plasmid DNA was also isolated from clone pl 40 C which contained the gene sequences that code for the unfused modified BGE protein. This plasmid contains the gene sequences that code for ampiciuin resistance.Hence, bacteria which are transformed with this plasmid are resistant to ampicillin and can produce the modifiedBGH protein. 6.3.3. EXPRESSION VECTOR pDN612 The expression vector pDN612 which is used below is a derivative of the expression vector pJS413. This vector was constructed by the removal from pJS413 of those nucleotides which corresponded to the cro gene, including, removal of the cro ATG and the two nucleotides 5 to the cro ATG. Vector pJS413 was digested with BQ1II followed by limited Bal 31 treatment and recircularization by ligatio n in the presence of BglII linkers. All other characteristics of the pJS413 vector are retained within pDN612. The important feature of this construction is that it provides for a unique cloning site 3 to the croSD region.Any DNA fragment may be inserted at this site, and when provided with its own ATG, may allow for subsequent expression of the protein for which it codes by transformants. 6.3.4. GENERATION OF NET BGH BY EMPLOYING A SYNTHETIC HINDIII NCOI LINKER The strategy for expression cloning described in the following scheme involves inserting BGH DNA into an expression vector such that a met codon is inserted 5 to the mature BGH gene sequence. The pUC9 expression vector which contains the lac promoter, SDlaC, a linker sequence for 10 restriction endonuclease sites, and the amino terminus of the a galactosedase gene was digested with PstI to linearize the plasmid Messing et al., Gene, 1982, 19 259 268 .The linker sequence provides numerous convenient restriction sites 5 to the BGH sequence. Plasmid pBH27 was partially digested with PstI and the large 795 b.p. BGH fragment was isolated by agarose gel electrophoresis and ligated with the linearized plasmid pUC9 as described in FIG. 8. The ligation mixture was used to transform E.coli strain NF1829. Ampicillin resistant colonies were amplified and plasmid DNA was isolated. Plasmid DNA was analyzed by restriction digestion and one clone, pBG18Cl contained the entire leader sequence and mature BGB sequence with the exception of the first two codons ATG ATG of the leader sequence. pBG18Cl produced a fusion protein consisting of the first 8 terminals amino codons of a galactosidase fused to amino acid 24 ala of the BGH leader sequence. In order to remove the leader sequence, plasmid pBG18Cl was linearized by digestion with NarI. The linearized plasmid was treated with S1 nuclease to remove the 5 overhangs and subsequently digested with HindIlI to remove the BGH leader sequence. The plasmid was then ligated in the presence of the synthetic HindIII NcoI linker which introduces an ATG codon, and the mixture was used to transform E. coli NF1829. Plasmid DNA from one clone, pBG18N was isolated. Plasmid pBG18N FIG. 8A was digested wih NcoI and Smal and an approximately 370 b.p. fragment containing the amino terminus of the BGH sequence was isolated by agarose gel electrophoresis. The plasmid pDN612 was linearized by digestion with Bq1II. The linearized plasmid was digested with S1 nuclease to create blunt ends and ligated in the presence of the synthetic NcoI linker.E. coli strain NF1829 was transformed and the plasmid DNA from pJS413 612N was isolated. pJS413 61ZN was digested with NcoI and Smal and the backbone was ligated with theNcoI SmaI fragment isolated from pBG18N. The ligation mixture was used to transform E. coli NF1829 and was plated on MacConkey agar. Plasmids isolated from transformants expressing ss galactosidase were screened and one plasmid, pJS413 612N met BGH FIG. 8A was found to be expressed as a fusion protein which contained the amino terminus of mature BGH amino acids 1 132 fused in phase with 8 galactosidase.The carboxy terminus of the BGH was ligated into the met plasmid by inserting a SmaI BamEI fragment from pBE27 approximately 1417 b.p. into the homologous sites in the met BGH plasmid as described supra Sec. 6.3.1. . FIG. 8A shows the steps leading to the construction of plasmid pBG511020 containing the nucleotide sequence encoding met BGE using the synthetic HindIII NcoI linker strategy. 6.3.5. THE CONSTRUCTION OF TEE PLASMID pN 37 BY EMPLOYING A SYNTHETIC BglI I NcoI LINER The modified met BGH plasmid pN 37 was constructed in order to insert an ATG methionine codon 5 to the mature BGH sequence FIG. 9 . The pJS413 derivative p572 13 1 was digested with BglII, subjected to limited Bal 31 treatment for 20 to 60 seconds and then ligated in the presence of the synthetic B4lII NcoI linker See FIG. 5 . This procedure was used to remove the BGH leader sequence. The randomized DNA mixture was used to transform E. coli strain NF1829.Transformants were amplified and plasmid DNA isolated. N.B., no individual colony was first isolated, the entire randomized1 DNA population was amplified and purified an NcoI site would have been regenerated only if the linker was ligated next to a corresponding G nucleotide in the randomized sequence . Plasmid DNA from this randomized population was digested with SmaI BqlII and approximately 400 b.p.fragments were isolated by agarose gel electrophoresis.The fragments were ligated into the Smal BglII backbone of pDN612 see FIG. 9 . The ligation mixture was then used to transform E. coli strain NF1829 and plated on LB plates containing 100 g ml ampicillin. Individual colonies were screened for the pretence of an NcoI site. The DNA of one plasmid, pll2 Nco , was isolated and found to contain anNcoI site. Plasmid DNA from the p572 13 1 BqlII NcoI heterogeneous population was digested with NcoI and SmaI and fragments of approximately 400 b.p. were isolated by agarose gel electrophoresis. These NcoI SmaI fragments were then ligated to the approximately 6.2 kb NcoI SmaI backbone of pll2 NcoI . The ligation mixture was used to transform B. coli strain NF1829 and plated ontoMacConkey lactose agar plates containing 100g mQ ampicillin and ss galactosidase positive colonies were isolated. One such isolate, pN 37, contains the BqlIIJNcoI linker and amino acids 9 to 132 of the matureBGH sequence See FIG. 9 . 6.3.6. METHOD FOR TEE GENERATION OF met BGH BY EMPLOYING PRIMER REPAIR This procedure uses the method of primer repair as described by Goeddel 1980, Nucleic Acids Res., 8 4057 4074 and Messing et al. 1982, Gene, 19 269 276 to generate a met BGE plasmid. Plasmid p13EX FIG. 4 was digested with Smal and Bv1II. A 405 b.p. fragment was then isolated by agarose gel electrophoresis. A 25 mer DNA single strand primer was then synthesized. The primer contains the nucleotides TATG at the 5 end and 21 nucleotides corresponding to amino acids 1 to 7 of the mature BGH See FIG. 10 . The primer reaction was performed as follows 80 l of double distilled H20 containing 0.5 ug of the 405 b.p. BglII SmaI of fragment of pl3EX, 1.0 g of primer treated with kinase, and 10 g of bacteriophage M13 N 37DNA were boiled for 5 minutes and then allowed to anneal at room temperature for 30 minutes. The addition of M13 N 37DNA described below was a modification of the basic primer repair reaction to minimize the reannealing of the complementary strands of the BqlII SmaI fragment, and thus maximize the reannealing of the 25 mer primer to the negative strand. The M13 N 37DNA was prepared by inserting the BglII SmaI fragment of pN 37 into bacteriophage M13.Since the BqlII SmaI fragment of pN 37 see section 6.3.5. is deleted for the nucleotides encoding the first eight amino acids of BGH and the primer contains the nucleotide sequences corresponding to only the first seven amino acids of BGH, no reannealing will occur between these two DNA species. Instead, theM13 N 37DNA will hybridize to the positive strand of the BglII SmaI fragment which competes with the primer for the negative strand see FIG. 10 . The solution was adjusted to 6mN Tris HC1 pS 7.5 , 6mM MgCl2, 100mM NaCl, lmM dithiothreitol, and 0.5mM of each dATP, dCTP, dTTP and dGTP and 10 units of the Klenow fragment of DNA polymerase was then added to create blunt ends. The reaction was allowed to proceed at 300C for 2 hours and then placed at 650C for 10 minutes.Two units of PstI were added to digest the primer repaired fragment and the resultant 280 b.p. fragment was isolated by agarose gel electrophoresis. In order to generate a vector having one blunt end and a PstI cohesive end backbone, the vector pCJ2 a derivative of pJS413 in which all the nucleotide sequences of the cro gene are deleted except those coding for the first two amino acids was digested with 8qlI SacI and the approximately 4.5 kb fragment was isolated FIG.10A . Plasmid pN 37 was digested with BqlII SacI and the approximately 2.4 kb fragment which contained BGH sequences was isolated by gel electrophoresis. The pCJ2 BalII SacI fragment and the BqlII SacI fragment of pN 37 were then ligated, the mixture was used to transform E.coli NF1829, and the plasmid DNA purified. One clone, plasmid pCJ2 2 4 contained a 370 b.p.BGH insert from amino acids 9 to 132 See FIG. 10A , pCJ2 2 4 was digested with BglII and the overhangs were filled in using Klenow fragment of DNA polymerase I and four dNTP s. The DNA was then digested with PstI and isolated by agarose gel electrophoresis. The approximately 7.5 kb DNA backbone was ligated to the 280 b.p. primer repaired PstI digested fragment and used to transform E. coli strain NF1829. The resulting sequence of the met BGH construct designated p27 see FIG. lOA contains cro sequences including cro ATG followed by the mature BGE sequence ATG and amino acids 1 132 . In order to express met BGH, the plasmid p27 was digested with SacI and BglII and the approximately 2.4 kbBGH containing fragment was isolated by agarose gel electrophoresis FIG. lOB . Similarly, the 4.5 kbBglII SacI fragment from the pJS413 derivative pDN612 which contains the gene for Ampr and the site of origin was isolated and ligated with the BqlII SacI .fragment from p27 See FIG. lOB . By this construction a plasmid was generated such that the met initiation codon was located 5 to amino acid number 1 GCC of the matureBGH sequence and 3 to the SDcro sequences with no cro initiator ATG codon present. The carboxy terminus of BGH was then reconstructed as previously described, in Sec.6.3.1 supra, and this construct was identified as p27 112 4 C. 6,4, CONSTRUCTION OF SYNTHETIC FRONT END In order to increase expression levels by optimizing translational initiation starts for the met BGH constructs, a DNA fragment encoding for the N terminus ofBGH amino acids 1 24, with the first amino acid encoding for methionine was chemically synthesized using state of the art procedures. Eight different oligonucleotides, varying in length from 15 to 25 bases, were synthesized by the phosphite triester method of Kempe et al. 1982, Nucleic Acids Res. 10 21 s 6695 6714 .Overlapping regions of complimentarity were used in order to facilitate in the assembly of the various fragments.All oligonucleotide fragments were then phosphorylated using T4 polynucleotide kinase and t32P ATP. Phosphorylated fragments were then ligated using T4 DNA ligase, and subsequently purified by gel electrophoresis.FIG. 11 shows the nucleotide sequence of the synthetic front end hereinafter referred to as SFE DNA fragment compared to the natural codons of the BGH gene. Codon changes for 13 of the 24 amino acids were made in this fragment. 6.4.1. 1. CONSTRUCTION OF PLASMIDS pSFEll, p104 14 and P102 l The construction of these plasmids is depicted inFIGS. 12 and l2A. An approximately 200 b.p. DNA fragment containing BGH sequences was generated by digesting pBH27 with PvuII and PstI. The DNA fragment coding for amino acids 24 91 was isolated by gel electrophoresis. The expression plasmid, pCJ2 2 4 , was digested with Bx1II andPstI and the approximately 7 kb fragment was isolated by gel electrophoresis. The SFE sequences, the PvuII PstI 200 b.p. BGH fragment, and the 7 kb BQlII pstI backbone of pCJ2 2 4 fragments were ligated at a 20 5 1 molar ratio using T4 DNA ligase.The ligation mixture was used to transform Ev coli strain NF1929. A chloramphenicol resistant recombinant expression plasmid was identified as pSFE ll See FIG. 12 . Plasmid DNA from pSFE ll was digested with BqlII and Smal and approximately 400 b.p. BGE fragment was isolated.by gel electrophoresis see FIG. 12A . The BGH plasmid p27 112 4 C was digested with BglII and SmaI and the 4.5 kb plasmid backbone was isolated by gel electrophoresis. The 400 b.p. BglII SmaI BGH fragment from pSPE 1l and the 4.5 kb BqlII SmaI backbone of p27 112 4 C were ligated at a 5 1 molar ratio using T4 DNA ligase. The ligation mixture was used to transform E.coli strain NP1829. The recombinant BGH plasmid p104 14 was isolated from transformants See FIG. 12A . plO 4 14 was partially digested with HincII to generate HincII blunt ends and an approximately 3640 be. fragment was purified by gel electrophoresis. This fragment was then ligated in the presence of linkers which encode a HindIlI recognition sequence See FIG. l2A cont d at a molar ratio of 50 1 linker fragment using T4DNA ligase. E. coli strain NF1829 was transformed and the recombinant BGH plasmid 102 1 was identified. Plasmids p104 14 and pl02 1 are identical except for the removal af a 950 b.p. sequence at the 3 end of the BGH genome from plasmid p104 14 thereby generating plO 2 1. 6.4.2. CONSTRUCTION OF ptac CLONES Various tac or trw lac promoter operator region clones were constructed. The hybrid promoter was constructed by combining the 35 region and the 10 region the sequences of DNA which are the RNA polymerase binding site of and tryptophan and the lactose promoters See FIG. 13 . Plasmid p104 14 was digested with the enzyme RsaI to generate a 549 b.p. fragment containing SDcro BGH sequences see FIG. 14 . The plasmid pDR540 obtained from PL Biochemicals, Milwaukee, WI contains the strong consensus tac or trP lac promoter. This plasmid was linearized by digestion with BamEI and the cohesive ends filled in using the Klenow fragment of DNA polymerase I and dNTP s. The linearized DNA was isolated by agarose gel electrophoresis and ligated to the 549 b.p. RsaI fragment. The ligation mixture was used to transform E.coli NF1829. Plasmids were studied by restriction analyses to select for the insertion of the RsaI fragment in the desired orientation see FIG. 14 . One clone selected, pl21 1, contained the ptac SDcro BGH sequences. Plasmid p191 2 was constructed in the identical manner. Plasmid plO 4 14 was then digested with XmnI andBqlII and the plasmid backbone purified and isolated by agarose gel electrophoresis see FIG. 14A . The ptac containing fragment of p121 1 was isolated by digestion with XmnI and Ba1II and separated by gel electrophoresis.DNA of the plasmid plO 4 14 backbone and the p121 1 containing fragment were ligated and used to transform E.coli strain NF1829. Clones were analyzed for the presence of an HindIlI site. One clone was selected and designated p141 2 see FIG. 14A . Note that the distance between the SDla and the SDCr regions in this clone was 12 nucleotides See FIG. 14B . p153 l was constructed by the same procedure described for p141 1 supra except that after linearizing the pDR540 plasmid with BamHI, the 5 overhang was treated with mung bean nuclease to create blunt ends. The tac containing fragment from plasmid pl27 10 was ligated to the p104 14 backbone and used to transform E. coliNF1829. One plasmid p153 5 was isolated and analyzed. The distance between the two SD regions was 9 nucleotides See FIG. 14B . 6.4.3. INSERTION OF A T RICH SEQUENCES An A T rich DNA segment was inserted between the SDcro and the ATG of BGH. Expression vector plO 2 1 DNA was digested withBqlII and treated with mung bean nuclease to remove the 5l overhang and to create blunt ends see FIG. 15 . A DNA oligomer which contains within it the recognition site for the restriction endonuclease AhaIII, was synthesized using the phosphite triester method of Kempe et al., supra Sec.6.4. . This linker was then ligated with the BglII Mung treated linearized backbone of p102 l in a ratio of 50 1 using T4 DNA ligase. The ligation mixture was used to transform E. coli strain NF1829, plated.on LB agar plates that contained 100 vg mQ of ampicillin and individual colonies were selected for the presence of the AhaIlI restriction site. One such clone was designated p128 7 2 See FIG. 15 . Following the successful insertion of this linker, the distance or number of nucleotides between the SDcro and the ATG of the BGH genome was reduced by restriction digestion of plasmid p128 7 2 with AhaIII andHindIII followed by isolation of a 900 b.p. BGH containing fragment by agarose gel electrophoresis. An approximately 2.4 kb BqlII Hung treated HindIII fragment containing amps and site of origin was also isolated from plasmid p128 7 2. Ligation of the 900 b.p. fragment with the 2.4 kb backbone was performed using T4 DNA ligase and theDNA mixture was used to transform E. coli strain NF1829 and plated onto LB agar plates containing 100 ug mQ of ampicillin.Individual colonies were isolated and screened for the absence of the AhalIl site between the SDcrO and the ATG of BGH. One such clone found has been given the designation p146 1 see PIG. 15 . FIG. 16 shows the number of nucleotides between he SDcro and the ATG of BGH in clone p128 7 2 ii 12 nucleotides and the distance between the SDCrO and the ATG of BGH in clone p146 1 is only 8 nucleotides. Plasmid p146 1 was digested with RsaI and a 549 b.p. BGH fragment was isolated by agarose gel electrophoresis and purified see FIG. 17 . This fragment was then ligated to the BamHI filled in fragment of pDR540 as described sura in Sec. 6.4.2. The ligation mixture was then used to transform E. coli strain NF1829, plated onto LB plates containing 100 g ml ampicillin and individual colonies were screened for the correct orientation of the BGH fragment. One such clone, designated here as p166 1 was isolated and plasmid DNA purified See FIG. 17 . DNA from p166 1 was digested withXmnI and SmaI and an approximately 800 b.p. fragment containing BGH sequences was isolated see FIG. 17A .p102 1 was also digested with XmnI and SmaI and an approximately 3.0 kb fragment containing ampr and the site of origin was isolated by agarose gel electrophoresis and purified. The 800 b.p. BGH fragment and the 3.0 kb ampicillin ori backbone were ligated in a ratio of 5 1 using T4 DNA ligase and this mixture was used to transformE. coli NF1829, plated onto B plates with 100 g ml of ampicillin and individual colonies were selected for theBGH insert and the tac promoter sequences. Plasmid p179 3 contains the sequences for ptac SD A T BGH. 6.4.4. INSERTION OF TRANSLATIONAL TERMINATION SEQUENCES Sequence analysis of the cDNA clone pBH27 revealed that only one stop codon TAG was present for the termination of translation, and that no transcriptional stop like sequences were found in the 3 untranslated region of the clone. Therefore, a synthetic oligomer containing a triple translational stop hereinafter referred to as TTS sequence of all 3 stop codons TAG, TAAr TGA in phase with the reading frame of the protein was synthesized using the phosphite triester method of Kempe et al. described supra . In order to insert the TTS linker into the BGH sequences, p104 14 was digested with PvuII FIG. 18 . The approximately 3.4 kb backbone and the 500 b.p. EGE containing fragment were isolated by agarose gel electrophoresis and purified. The TTS linker which encodes all three stop codons followed by the HinaIII recognition sequence was ligated to the backbone of p104 14 at a ratio of 50 1 using T4 DNA ligase. The ligation mixture was used to transform E. coli strainNF1829, plated onto LB agar plates containing 100 pg me of ampicillin and individual colonies were selected and screened for the insert. Clones were also selected for the proper orientation of the linker as described in FIG. 18. One such clone has been given the designation of p164 3a This plasmid, was linearized by digestion withPvuII and isolated by agarose gel electrophoresis. TheBGH PvuII 500 b.p. fragment from p104 14 was then ligated to the backbone of p164 3 at a ratio of 10 1. The ligation mixture was used to transform E. coli strainNF1829, plated onto LB agar plates with 100 pg mQ ampicillin and individual colonies screened for the insertion and proper orientation of the BGH insert. One plasmid, p175 12 contains the 3 translational stop codons and provides for the efficient termination of translation of the BGH message. 6. 4. 5. INSERTION OF TRANSCRIPTIONAL TERMINATION SEQUENCES Since no transcriptional attenuator region was found in the cDNA clone which provided for the efficient termination of transcription, a sequence coding for the attenuator portion of the tryptophan operon was placed approximately 100 b.p. 3 to the translational stop codons of the BGH sequence See FIG. 19 . Plasmid p175 12 was digested with HindIlI, the recessed ends of the DNA were filled in with dNTP s using the Klenow fragment of DNA polymerase I to create blunt ended fragments and then further digested with Xmnl. An approximately 1100 b.p. fragment containing BGH sequences was isolated by agarose gel electrophoresis and purified. Another preparation of p175 12 was digested with HindIll and XmnI and the 2000 b.p. fragment which contained the ampicillin resistance gene and the sequences for the site of origin were isolated by agarose gel electrophoresis and purified.A third plasmid, ptrpED5 l Hallewell, R.A. et al., 1980, Gene, 9s 27 47 , was digested. with SPaI and EindIII and an approximately 2400 b.p. fragment which contains the tryptophan attenuator sequences was isolated by agarose gel electrophoresis and purified. This fragment also contains sequences for the trpE an the trpD genes. The three fragments were ligated at a ratio of 5 2 1 using T4 DNA ligase and the ligation mixture was used to transform E. coli strain NF1829, plated onto LB plates containing 100 vg mD ampicillin, and individual colonies were isolated and screened for the insertion of the BGH sequences and the trP attenuator sequences.Plasmid p211 4 was shown to contain the BGH fragment followed by the trP attenuator region 3 to theTTS. 6.4.6. CONSTRUCTION OF ptac A T SFE BGH TERM PLASMID Plasmid p217 1 was constructed to contain the following features 1 a tac promoter to provide for a very strong promoter of mRNA transcription in E. coli 2 an A T rich intervening region to optimize translational starts 3 a synthetic front end to optimize for a lack of secondary structure and to replace non preferred codons with preferred codons for maximum translational efficiency 4 a triple translational stop sequences TTS to provide for a unique sequence of tandem translational stops to assure proper termination and 5 a trP attenuator to provide for effective termination of transcription which may, in addition contribute to the overall stability of the mRNA by limiting its size. Plasmid p179 3 which contains ptac, an A T rich region and BGH sequences as described supra, was digested with EcoRI and SmaI and the 500 b.p. ptac containing fragment was isolated by agarose gel electrophoresis and purified See FIG. 20 . Plasmid p211 4 described supra was also digested with EcoRI and SmaI and the approximately 5.5 kb plasmid backbone which contains theTTS and trP attenuator sequences was isolated by gel electrophoresis and purified. These fragments were then ligated at a molar ratio of 5 1 using T4 DNA ligase and the ligation mixture used to transform E. coli NF1829.Individual colonies were isolated and screened for the insertion of the ptac BGH sequences into the TTS trP attenuator backbone. Plasmid p217 1 was analyzed by restriction mapping and shown to contain ptac A T BGH TTS trP attenuator sequences. Plasmid p214 1 was constructed in the identical manner described supra. This plasmid lacks the A T rich region and the SFE. 6.5. INSERTION OF BGH INTO PPV FUSION PROTEIN PLASMID Plasmids were constructed to carry and express the unfused BGH protein and the larger PPV 8 galactosidase fusion protein. The construction of plasmid pPPVlOOOt was described in U. S. Patent Application Serial No. 456,423 by Halling et al. filed January 7, 1983. This plasmid contains the unique restriction site NcoI CCATGG near the end of the 8 galactosidase gene. This site was replaced by opening the plasmid at its unique NcoI site, treating the DNA with a limited amount of the nuclease Bal 31 in order to create flush ends, and inserting a linker which contains the restriction endonuclease XmnI site FIG. 21 . Clones which contain the unique XmnI site were isolated and plasmid DNA purified.One clone, designated pPPVlO0Otx contained the cro PPV ss galactosidase fusion protein. Plasmid DNA was isolated from clone pPPVlOOOtx. This plasmid carries the gene for tetracycline resistance, therefore bacteria transformed with this plasmid will be both tetracycline resistant and able to produce the PPV galactosid ase fusion protein. 6.5.1. INSERTION OF pTAC BGH SEQUENCES INTO THE PPV FUSION PROTEIN PLASMID Plasmid from BGH construct p141 2 was digested with the restriction endonuclease HindIII and the 3 recessed ends were filled in with dNTP s using the Klenow fragment of DNA polymerase I see FIG. 22 . An approximately 1000 b.p. fragment containing the ptac andBGH sequences were isolated by agarose gel electrophoresis and purified. The plasmid pPPVlO0Otx described supra was digested with the restriction endonuclease XmnI and the approximately 7.2 kb linearized plasmid was isolated by agarose gel electrophoresis and purified. The ptac BGH fragment and the pPPVlOOOtx linearized fragment were then ligated at molar ratio of 10 1 using T4 DNA ligase. This mixture used to transform E. coli strain NF1829.The transformants plated onto LB agar plates containing 20 ug LI tetracycline, individual colonies were isolated and clones were selected for the insertion of the ptac BGH sequences into the PPV B galactosidase fusion protein vector. This procedure was also repeated using the ptac A T 13GH clone p179 3 See FIG. 22 . A clone isolated from the p141 2 construct was designated p158 1. A clone isolated from the p179 3 construction was designated p195 4. Clone p158 1 and p195 4 were assayed for the production of both the PPV l3 galactosidase fusion protein and the BGH protein. 6.5.2 INSERTION OF p214 l INTO THE PPV FUSION PROTEIN PLASMID Plasmid p214 l described supra, was digested .with the restriction endonuclease EcoRV and the approximately 800 b.p. fragment containing the ptac BGH TTS trp attenuator sequences was isolated by agarose gel electrophoresis and purified see FIG. 23 . Plasmid pPPVlOOOtx described supra, was digested with XmnI and the linearized plasmid was isolated by agarose gel electrophoresis and purified. The 800 b.p. ptac BGH fragment and the approximately 7.2 k.b. linearized plasmid pPPVlOO0tx were ligated in a molar ratio of 10 1 using T4DNA Ligase. This mixture was used to transform E. coli strain NF1829 and plated onto LB agar plates that contained 20 pg mD tetracycline.Individual colonies were isolated and ceened for the insertion of the ptac BGH sequences into the PPV backbone. Clone p231 3 was isolated and assayed for the production of both the PPV ss galactosidose fusion protein and the modified BGH protein See FIG. 23 . 6.5.3 INSERTION OF pTAC A T TTS att BGH INTO TEE PPV FUSION PROTEIN PLASMID An approximately 900 b.p. fragment which contains the ptac A T BGH TTS trp attenuator sequences from plasmid p217 1 described supra, was isolated by gel electrophoresis after digestion with EcoRV as described inFIG. 23. Plasmid pPPVlOOOtx was digested with XmnI and linears were isolated by agarose gel electrophoresis and purified. The ptac A T BGh TTS att EcoRV fragment and the 7.2 kb pPPVlOOfftx linear were ligated at a molar ratio of 10 1 using T4 DNA ligase and this mixture was used to transform 2. coli strain F SK107 which contains theF iq mutation F factor plasmid which carries on it the it mutation which leads to the overproduction of the lac repressor protein .Transformants were then plated ontoLB agar plates which contain 20 Vg mQ tracycline and individual colonies were isolated and screened for the insertion of the BGH sequences. Clones p230 2 3, p230 5 3 and p230 6 2 were isolated and assayed for the production of both the PPV B galactosidase fusion protein and the BGH protein. Plasmid DNA from clone p230 6 2 was isolated and used to transform the E. coli strain SK107 no iq mutation . Transformants were plated onto LB agar plates that contain 20 vg mss tetracycline and individual colonies were isolated. One isolate, p230 6 1, was assayed for its production of both the PPV ss galactosidase fusion protein and the BGH polypeptide in the absence of any inducer i.e. IPTG or lactose . 6.5.4 ANALYSIS OF PROTEINS PRODUCED BY TRANSFORMANTS All pPPV recombinant plasmids were used to transform B. coli strain NF1829, SK107, or E SK107. Transformants were isolated and assayed by restriction analysis for the insertion of the fragment.Clones which were positive for the insert were then assayed for the production of both the PPV ss galactosidase fusion protein and the BGH protein after induction of the promoter with IPTG or lactose. A number of clones were identified as expression positive for both proteins. Furthermore, a significant amount of stabilization of the BGH protein was evident.Purification of the aggregate proteins also enabled the purification of the unfused protein. The unfused protein and fusion proteins produced by E. coli transformants were isolated and purified by the non denaturing co aggregate purification method which is outlined in detail in Section 5.6, supra. Briefly,L broth containing, as appropriate, either 100 vg mQ ampicillin and or 20 g ml tetracycline and 1 lactose or lmM IPTG was inoculated with select transformants. After incubation at 300C overnight the cells were pelleted and quick frozen in dry ice methanol. The unfused protein and fusion protein were then isolated according to the embodiment described in Section 5.6. The unfused protein can be separated from the fusion protein on the basis of size or charge. The isolated co aggregate proteins were resuspended in 50 pX distilled H20 to which 50 l of 2xSample Buffer 2x Sample Buffer is composed of 108 SDS, 4OmM Tris HCI, pH6.8, 29 8 ME, and 0.02 volumes saturated solution of bromphenol blue was added and mixed well.The mixture was boiled for 5 minutes and 25Q aliquots were applied to 10 SDS polyacrylamide gels or electrophoresis. After SDS PAGE, the unfused protein was present in sufficient quantity to form a prominent stainable band. The protein can therefore be isolated and purified from the large aggregate forming protein by standard aeparation techniques. Amino acid analysis of this band was performed using state of the art techniques to confirm that this protein was indeed the desired product. 6.5.5. QUANTITATION OF PROTEIN Analysis via SDS PAGE indicated that in clones which contain and express the PPV ss galactosidase fusion protein that spproximiltely 10 to 15 of the total bacterial cell protein was fusion protein. The minimum tuaount of expression need to cause aggregation of the fusion protein is probably dependent on the nature of the fusion peptide utilized. For several BGH PPV B galactosidase fusion constructs, a significant amount of expression greater than 3 was required to cause the co aggregation of these unfused protein and fusion protein. 6.6 INSERTION OF BGH SEQUENCES INTO TEE HIGH COPY NUMBER PLASMID pORI TET tl Derivatives of the runaway plasmid pOPIX6 were constructed in order to enhance the expression levels ofBGH. Plasmid pOP6 was digested with EcoRI PvuII and a 1.7 kb fragment which contains the ori mutation site was isolated by agarose gel electrophoresis and subsequently purified see FIG. 24 . A 2065 b.p. EcoRI PvuII fragment which contains the tetracycline resistance gene was isolated from pBR322. The two fragments were ligated at a 1 1 ratio and the mixture was used to transform E. coli strain NF1829. A tetracycline resistant, ampicillin sensitive colony was isolated and the plasmid DNA amplified and purified. The construction of the runaway plasmid pORI tetEl is outlined in FIG. 24. Plasmid pORI tetEl was digested with AvaI, EcoRI of PvuII. The AvaI cohesive ends and the EcoRI cohesive ends were filled in with dNTP s using the Klenow fragment of DNA polymerase I see FIG. 25 . The blunt ended EcoRV fragment of p217 I see section 6.4.6. supra was isolated by agarose gel electrophoresis and purified. The approximately 900 b.p. BGH fragment which contains the 1tac1 promoter sequences 5 to the BGH gene and the trp attenuator sequences 3 to the BGH gene, were ligated in three sepatate reactions to the three isolated linears from pORI tetEl. The ligation mixture was used to transform E. coli So107 which contains the F factor carrying the iq mutation for the lac repressor protein. Transformants were plated on LB agar containing 100 pg mQ ampicillin. Individual colonies were selected, plasmidDNA was isolated, and plasmids were selected for both the presence of the BGH insert and the high copy number phenotype after analysis by agarose gel electrophoresis. Three constructions of the runaway plasmids p237 D2 1, p238 D3 1, p239 D5 1 are shown in FIG. 25. Aggregated BGH protein was subsequently purified from these clones using the non denaturing aggregate purification method as described supra Sec. 5.6. .Purified BGH protein was determined to be identical to its pituitary counterpart by comparison of tryptic peptide mapping procedures and subsequent amino acid sequence determination. 6.7. CONSTRUCTION OF pTRP EXPRESSION VECTORS The operator, lacO was inserted 5 to theSFE BGH plasmids in order to efficiently regulate the trp promoter and increase expression of BGH. The construction of an expression vector carrying a trp promoter fragment and lacO fragment is described below. 6.7.1. CONSTRUCTION OF PLASMID pTC503 Plasmid pGC301 Christie et al., 1981, Proc.Natl. Acad. Schiz 78 4180 is a trp vector which contains the trp promoter region, operator region, and SDtrp as seen in FIG. 26. The vector was modified and used to express DGH. Plasmid pGC301 was digested with restriction endonuclease HinfI and the ends were made blunt by wfilling inw using the Klenow fragment of DNA Polymerase I. The blunt end fragments were electrophoresed in 1 LMP and the approximately 965 b.p. fragment was removed from the gel and purified by phenol extraction. The fragment contains the promoter operator regions and approximately 630 b.p. of coding sequences for the trpC gene.This fragment was ligated to pBR322 DNA which was digested withEcoRI and also rendered blunt ended by treatment with the Klenow fragment DNA polymerase I. The ligation mixture was used to transform E. coli MC 1000 and ampicillin resistant colonies were isolated and DNA of plasmid pTCSO3 was purified. 6.7.2. CONSTRUCTION OF PLASMID PTNA771 Plasmid pTC503 was used to construct plasmid pTNA771 as described in FIG. 27. Plasmid pTC503 was cleaved with HpaI and Clal and an approximately 4.3 kb fragment was isolated from LMP, thus removing the C gene including the ATG from the plasmid. Since there are several TagI restriction sites in pTC503, a second plasmid, pTC503 was first cleaved with HpaI and BqlII and the small fragment was isolated from LMP. The HpaI BglII fragment was then cleaved with Tali, and the 346 b.p.HpaI TagI fragment was isolated and ligated to the largeHpaI ClaI fragment to generate plasmid pTNA771. The plasmid pTNA771 contains the tro promoter, operator, region of initiation of transcription, and SDtrP sequences the trPC has been deleted . 6.7.3. CONSTRUCTION OF PLASMXD POLINK 23456 A synthetic 71 b.p. multi restriction site fragment polink 23456 was inserted into pBR328. Plasmid pBR328 Bethesda Research Laboratories, Gaitheraburg, MD was digested with EcoRI and Salt, and an approximately 3100 b.p. fragment was isolated from LMP and ligated to the polink23456 DNA. The construction of the Polink23456 vector is described in FIG. 28. 6.7.4. CONSTRUCTION OF PLASMID PTNAP771 Plasmid pTNAP771 was constructed to include the multi restriction site synthetic linker described supra.Plasmid pTNA771 was cleaved with PstI and ClaI and the approximately 1650 b.p. fragment was isolated from LMP see FIG. 29 . Plasmid Polink23456 was cleaved with PstI and XbaI, the fragments were then partially filled in using Klenow fragment of DNA polymerase I in the presence of dCTP and dTTP, and the 1950 b.p. fragment was isolated by agarose gel electrophoresis as described in FIG. 29.The modified PstI Xbat fragment of plasmid Polink23456 was ligated to the Pst ClaI fragment of pTNA771 and the DNA mixture used to transform E. coli MC1000. Plasmids were screened for the presence of an XbaI site resulting fom cellular repair of the T C mismatch to a T A. One plasmid generated, pTNAP771 shown in FIG. 29 contained the nucleotide sequence of the tryptophan promoter, operator, mRNA initiation site, and ribosome binding site in addition to several unique restriction sites as shown inFIG. 30, 6.7.5. CONSTRUCTION OF pTRP VECTOR pSFE l WHICH EXPRESSES BGH The construction of plasmid pSPE l from pSFE ll FIG. 12 is shown in FIG 31. Plasmid pSFE 11 was digested with XmnI and Bq1II and the 6300 b.p. fragment was isolated by agarose gel electrophoresis.Plasmid pTNAP771 was also cleaved with XmnI and sqlII and the approximately 750 b.p. fragment was isolated and ligated to the large fragment of pSFE l1. Tne ligation mixture was used to transform E. coli NF1829. Ampicillin resistant colonies were examined for production of fusion protein on indicator agar plates. Plasmid pSFE l produced a BGH ss gaactosidase fusion protein. To express BGE as an unfused protein, i.e., to remove the i z fusion portion and to reconstruct the 3 carboxyl coding terminus of BGH the following procedure was used see FIG. 32 . Plasmid pSFE l was cleaved with EcoRI and SmaI and the approximately 4.3 kb fragment was isolated by agarose gel electrophoresis.Plasmid p27 112 4 2C described supra was also cleaved with EcoRI and Smal, the approximately 1042 b.p. fragment containing the carboxyl terminus of BGH was isolated by agarose gel electrophoresis and ligated to the isolated fragment of pSFE l. The ligation mixture was used to transform E. coli NF1829. The procedure used to generate plasmid plo 6 15 is shown in FIG. 32 Plasmid plO 6 15 contains the sequences for P ç SDtrP BGH. 6.7.6. CONSTRUCTION OF pTRP VECTORS WHICH ARE REGULATED BY TEE LAC OPERATOR The expression of the BGH 8 galactosidase fusion protein or metBGH in various trpR strains i.e., strains which produce an active trp repressor containing plasmids pSFE l or plO 6 15, respectively, was similar in both the presence and absence of the co repressor L tryptophan. These results indicated that the trp vector system lacked the ability to significantly regulate BGH expression.Therefore, the trp promoter was placed under the control of lacO by inserting a synthetic lacO nucleotide sequence seventeen nucleotides 3 to the 10 region of ptrp. The inclusion of an F lac episome with the lac Iq mutation for lac repressor overproduction in the host strain assured sufficient lac repressor molecules to efficiently repress BGH synthesis. Induction of BGH synthesis was achieved by the addition of the inducer lactose or IPTG to the bacterial culture at a phase of growth where BGH synthesis was maximal. 6.7.7. CONSTRUCTION OF PLASMID p407 29 WHICH IS REGULATED BY LACO The construction of plasmid pT248 11 which contains ptrp, SDtrp, and SDcro sequences is described in FIG. 33. Plasmid SFE 11 was digested with RsaI and SmaI and the approximately 400 b.p. fragment which contains SDcro and the amino terminal BGH sequences including amino acids 1 132 was isolated by agarose gel electrophoresis. Plasmid pSFE l was linearized by digestion with Bq1XI and made blunt ended using the Klenow fragment of DNA polymerase I. The two fragments were ligated and used to transform E. coli NF1829 and plasmid pT248 ll was isolated. The construction of the ptrP vector pMTE 407 29 which is regulated by lacO is shown in FIG. 34. Plasmid pT248 11 was cleaved at the unique XbaI site and the resulting four base 5 overhangs were made blunt ended by treatment with mung bean nuclease. In order to remove most of the DNA sequences between the 10 region and theSD region, the modified plasmid DNA was then subjected to limited 3 to 5 exonuclease digestion by treatment withT4 DNA polymerase in the presence of dCTP. The thirteen nucleotide 5 overhang remaining was then removed by digestion with mung bean nuclease. The resultant DNA was then cleaved with EcoRI and the approximately 3.3 kb fragment was isolated by a agarose gel electrophoresis.Plasmid plO 6 15 described supra was digested with HpaI and EcoRI, and the 1500 b.p. fragment which contains theBGH sequence was isolated by agarose gel electrophoresis.and ligated to the EcoRI fragment of pT248 ll in the presence of a 25 mer synthetic oligomer which encodes the sequence for lacO PL Biochemicals, Milwaukee, Wis . The ligation mixture was used to transform E.coli NF1829. The plasmids were then modified to encode fusion proteins in order to assay for ss galactosidase activity on i ndicato aga r plates. Several colonies were selected and the DNA from one plasmid, pSTE 132 FIG. 34A was digested with BclII and EcoRI. A 3.4 kb fragment which contained sequences for ptrp, lacO, and SDtrP in the correct orientation was isolated by agarose gel electrophoresis. Plasmid pSFE 1 was then digested withBqlII, EcoRI, and XmnI and the 3.6 kb fragment containingBGE sequences was isolated by agarose gel electrophoresis and ligated to the B1II EcoRI fragment from pMTE 132 seeFIG. 34B . The ligation mixture was used to transform E.coli NP1829. FIG. 34C shows the sequence of the 10 region to the BGH of pMEE 407 29 construction which contains ptrp lacO SDtrp BGH. The regulation of ptrp by lacO was analyzed in the presence and absence of the inducer IPTG. In the absence of inducer, expression of BGH 8 galactosidase fusion protein was reduced 10 15 fold, thus indicating efficient regulation of ptrv by lacO. 6.8. PLASMIDS THAT EXPRESS THE BGH GENE The plasmids isolated from E. coli transformants described supra were analyzed for their ability to direct the expression of the BGH protein either as a fusion protein or as an unfused protein. Plasmids were transformed into E. coli NF1829, F SK107, SK107, orMC1000. Since the lac promoter was transcriptionally inactive in NF1829 and F SK107 due to overproduction of the lac repressor , the BGH related protein could only be detected upon induction of the promoter with either lmMIPTG or l lOmM lactose. Fusion proteins were examined by assaying for ss galactosidase activity on indicator plates. Positive colonies were tested for the presence of BGH 8 galactosidase fusion protein by SDS PAGE analysis of total lysates of transformants induced with IPTG. The unfused proteins were identified and purified by the non denaturing ca aggregate procedure described supra in Sec. 5.6. To determine if the recombinant expression plasmids containing BGH sequences were capable of synthesizing the predicted BGH B galactosidase fusion protein or unfused protein , total E. coli proteins from cells harboring these expression plasmids were analyzed bySDS PAGE, followed by Coomassie Brilliant Blue staining.For example, E. coli strain F 5K107 containing either plasmid p238 D2 1 or pSFE ll, were grown in L broth nutrient medium in the absence or presence of inducer 18 lactose . Overnight cultures were harvested, lysed directly in SDS gel sample buffer 2X sample buffer 0.14M Tris ECl, pE 7.8, 0.0011 bromophenol blue, glycerol, 6 SDS, 10 ss ME by boiling for 5 minutes and aliquots were subjected to 109 SDS PAGE. The bacterial strains utilized synthesize high molecular weight polypeptides only upon induction of the lac promoter. The proteins were the size predicted for fusion proteins of the inserted BGH sequences and or z gene Table I .Proteins produced by p238 D2 1, and pSFE ll are approximately 22,000 daltons and 120,000 daltons respectively. After SDS PAGE, amino acid analysis of the purified BGH broth protein was performed using state of the art procedures to confirm the identity of the BGH protein product. The recombinant plasmids encoding BGE fusion protein or unfused BGH are shown in Table I. The level of expression of protein is given in terms of percent of total host cell proteins. TABLE I Size of Protein Clone Description Expression kilodaltons p27 112 4 2c cro BGH 0.1 22 pRedl3 cro BGH ss gal 10 120 pN 37 mod BGH ss gal 1 3 120 pMet 27 BGH ss gal 1 120 pl 40 c mod BGH 0.5 1 21 p102 1 SFE BGH 0.5 1 22 pSFE 11 BGH ss gal 1 3 120 p104 14 SFE BGH 0.5 1 22 p128 7 2 A T 1 2 22 p153 5 tac BGH I 5 22 p141 2 ptac 1 5 22 p146 1 A T SFE 1 2 22 p179 3 tac A T BGH 2 5 22 p214 1 tac BGH TTS att 5 10 21 p217 1 tac A T. SFE BGH TTS att 5 15 22 p230 2 3 co agg. 5 10 22 p230 5 3 co agg. 5 10 22 p230 6 2 co agg. 5 10 22 p230 6 1 co agg 5 10 22 p195 4 co agg 5 10 22 p231 3 co agg 5 10 21 p238 D2 1 ori 10 18 22 p239 D5 1 ori 10 18 22 .p237 D3 1 ori 10 18 22 pMTE 407 29 lacO 5 10 120 7.DEPOSIT OF MICROORGANISMS It is to be understood that all base pair sizes given for nucleotide sequences are approximate and are used for purposes of description. Furthermore, it is apparent that many modifications and variations of this invention as hereinbefore set forth may be made without departing from the spirit and scope thereof. The specific embodiments described are given by way of example only and the invention is1imited only by the appended claims. The following E. coli strains carrying the listed plasmids have been deposited with the AgriculturalResearch Culture Collection NRRL , Peoria, IL and have been assigned the listed accession numbers E. Coli Strain Plasmid Accession NumbersK12, MC1000 pBH27 B 15648K12, NF1829 p226 22 B 15649K12, NF1829 p119 1 B 15650 X12, NF1829 p344 3 B 15651K12, NF1829 p106 15 B 15652K12, NF1829 pBG511020 B 15653K12, NF1829 p27 112 4 2c B 15654K12, NF1829 pRedl3 B 15655K12, NF1829 p214 1 B 15656K12, NF1829 p128 7 2 B 15657 X12, NF1829 p179 3 B 15658 K12, NF1829 plO 2 1 B 15659K12, NF1829 p225 1 B 15660K12, NF1829 p217 1 B 15661K12, NF1829 p153 5 B 15662K12, NF1829 p146 1 B 15663K12, NF1829 p104 14 B 15664K12, NF1829 p141 2 B 15665 R12, NF1829 pMet27 B 15667K12, NF1829 pSFE 11 B 15668K12, F SK107 p238 D2 1 B 15669 R12, F SK107 p230 2 3 B 15670K12, F SK107 p230 5 3 B 15671 K12, F SK107 p239 D5 1 B 15672K12, F SK107 p237 D3 1 B 15673K12, F SK107 p230 6 2 B 15674K12, SK107 p230 6 1 B 15675R12, SK107 p195 4 B 15676K12, SK107 pori tet 1 B 15677K12, SK107 p231 3 B 15678K12, NF1829 pSFE l B 15679K12, NF1829 pMT 433 26 B 15680 X12, NF1829 pMTE 407 29 B 15681R12, NF1829 pM425 5 B 15682K12, NF1829 pME474 1 B 15683K12, NF1829 pll B 15177X12, NF1829 p83 B 15178K12, NF1829 pl 20F B 15179 K12, NF1829 pl 40 B 15180K12, NF1829 pl 40 C B 15181K12, NF1829 pl 60A B 15182 Kl2, NF1829 pl 60B B 15183K12, NF1829 p112 R D B 15184K12, NF1829 p112R G B 15185K12, NF1829 p112 R L B 15186K12, NF1829 p112 R O B 15187K12, NF1829 p112 R15 B 15188K12, NF1829 pN 1 B 15189 R12, NF1829 pN37 B 15190, B 15666K12, NF1829 pNB9 B 15191K12, NF1829 pDN112 B 15192. A culture of the deposited microorganisms will be made available to the public upon the grant of patent based upon the present application. It is to be understood that the availability of a deposit does not constitute a license to practice the subject invention in derogation of patent rights granted by the United States government. Furthermore, the present invention is not to be limited in scope by the microorganism deposited, since the deposited embodiment is intended as a single illustration of one aspect of the invention. Indeed, various modificatons of the invention in addition to those shown and described herein will become apparent to those skilled in the art from the foregoing description and accompanying drawings. Such modifications are intended to fall within the scope of the appended claims.