# Methods and compositions for expression of competent eukaryotic gene products.

## Abstract
Heat shock gene control elements from different eukary otic cells are utilized to provide expression of a competent gene products in the same or similar eukaryotic as well as in procaryotic host cells. The control elements can be joined to a suitable eukaryotic replication system to form an expres sion vector, or may be joined to the gene of interest and introduced directly into the host genome.

## Claims
CLAIMS 1. A gene expression unit host cell combination system wherein the gene expression unit comprises a an expression control region of less than 100 kb. derived from a eucaryotic heat shock protein gene b at least one site with which a structural gene of interest c is associated in order to come under the transcriptional and or translational control of said expression control region a , and wherein the host cell system corrports cells which are either from bacterial origin or from at least one species similar to those from which the said expression control region was isolated and in which control region a linked functionally to the said structural gene c of interest is present in order to produce the product of said structural gene. 2. A gene expression unit host cell system as in claim 1, wherein the structural gene of interest is associated by being inserted into said at least one site of the gene expression unit. 3. A gene expression unit host cell system as in claim 1, wherein the heat shock gene control region is derived from heat shock genes from eucaryotic organisms. 4. A gene expression unit host cell system as in claim 3, wherein the heat shock gene control region is derived from the genomic DNA of Droscphila melanogaster. 5. A gene expression unit host cell system as in claim 4, wherein the heat shock gene control region is derived from any one of the genes encoding the 70 kilo dalton heat shock proteins of Dro scohila melanogaster. 6. A gene expression unit host cell system as in claim I, wherein at least part of said functional heat shock gene control region is of synthetic origin. 7. A gene expression unit host cell system as in claim 1, in which said expression unit further comprises at least one selectable marker linked to the expression unit which allows for selection of transformed hosts. 8. A gene expression unit host cell system as in claim 1, in which said expression unit further conprises a procaryotic replication system which allows propagation of the expression unit, and at least one antibiotic resistance gene. 9. A gene expression unit host cell system as in claim 1, wherein said expression unit is linked to a fragment containing a eucaryotic extrachomosomal replication system. 10. A gene expression unit host cell system as in claim 1 wherein the eucaryotic heat shock gene control region is linked to a cellular or viral transcription enhancer element to allow for consthitutive expression of the gene of interest. 11. A gene expression unit bost cellsystem as in claim 1, wherein a heat shock promoter element is linked to the complete RNA coding region of a structural gene of interest. 12. A gene expression unit host cell system as in claim 1, wherein a eucaryotic heat shock gene prcmoter RNA leader segment is linked to the protein coding region of a structural gene of interest. 13. A gene expression unit host cell system as in claim 1, wherein a eucaryotic heat shock gene control element is linked to at least part of the DNA coding region of a structural gene of interest. 14. Plasmid pRV15 as a new conposition of matter. 15. Plasmid PRS4 as a new composition of matter. 16. A method for preparing the gene expression host cell comer bfration of claim 1, comprising introducing the expression unit into host cells by cotransformation with a selectable marker. 17. A method as in claim 16, wherein the expression unit is in troduced into host cells with an amplifiable gene 18. A method as in claim 16, wherein the expression unit is in troduced into host cells by transformation or transfection. 19. An expression vector comprising a eukaryotic extrachroaosanal replication system an expression control region derived from a eukaryotic heat shock protein and at least one insertion site under the transcriptional control of the expression control region. 20. An expression vector as in claim 19, said vector being free of a translational start codon between the expression control region and the insertion site. 21. An expression vector as in claim 19, having a translational start codon upstream from the insertion site. 22. An expression vector as in claim 19, wherein the heat shock gene control region is derived from the genomic DNA of Drosophila melanogaster. 23. An expression vector as in claim 22, wherein the heat shock gene control region is derived from the heat shock gene which encodes the 70 kilodalton heat shock protein of Drosophila melanogaster. 24. An expression vector as in claim 19, further comprising a prokaryotic replication system which allows stable maintenance in prokaryotic hosts. 25. An expression vector as in claim 19, further carprising at least one selectable marker which allows for selection of transformed hosts. 26. An expression vector carprising a eukaryotic extrachrowosomal replication system a heat shock gene control region substantially homologous to a 650 base pair sequence which controls transcription of the 70 kilodalton heat shock protein in Drosophila melanogaster and at least one insertion site under the transcriptional control of the heat shock gene control region. 27. A DNA construct of less than 15 kbp which includes an expression control region derived from a eukaryotic heat shock gene and at least one insertion site under the transcriptional control of the heat shock gene control region. 28. A DNA construct as in claim 27, wherein the heat shock gene which encodes the 70 kilodalton heat shock protein of Drosophila mela nogaster. 29. A method for producing competent gene products, said method coirprising joining a structural gene to an expression control region derived from a eukaryotic heat shock gene introducing said structural gene and control region into a eukaryotic host suitable for expression of the structural gene and growing said host, whereby said product is produced. 30. A method as in claim 29, wherein the structural gene and control region are joined together in an expression vector having an eukaryotic extrachrorroscaaI replication system 31. A method as in claim 29, wherein the structural gene and control region are introduced to the host by co transformation with a selectable marker. 32. A method as in claim 29, wherein the expression control region is derived from a heat shock gene which encodes a 70 kilodalton heat shock protein in Drosophila melanogaster. 33. A method as in claim 29, wherein the heat shcck gene is derived from a different eukaryotic species from the host. 34. A method as in claim 29, wherein the structural gene is a mammalian gene. 35. A method as in claim 34, wherein the host is a mammalian cell culture 36. A method as in claim 29, wherein the structural gene is a human gene. 37. A method as in claim 36, wherein the host is a mammalian cell culture. 38. A method as in claim 37, wherein the host is a COS I cell culture.

## Description
METHODS AND COMPOSITIONS FOR EXPRESSION OF COMPETENT ErSvRgDrIc GENE PRODUCTS Background of the Invention 1. Field of the Invention The industrial production of proteins coded by specific genes such as those encoding hormones, clotting factors, virus proteins, insulin, interferon and others, involves selecting and isolating gene sequences from viral, eukaryotic and other RNA or DNA, splicing of the sequences in the form of DNA into DNA vectors to provide recombinant DNA, and introducing said recombinant DNA into host cells capable of expressing these genes. The vectors may inpart to the transformed host cells a phenotypic trait used for isolation and cloning purposes. The gene products are isolated from cell cultures by usual techniques. Up to now, efforts in this field have usually been based on the adaptation of microorganisms as host cells for expressing genes of interest. Indeed, bacteria are often the organisms of choice since they can be grown rapidly, in large quantities and at low cost. Foreign DNA can be introduced easily into bacterial cells by using vectors such as plasmids, cosmids, viruses and the like. Use of bacterial hosts, however, is not always sufficient to obtain expression of mature eukaryotic proteins. Many important eukaryotic proteins are modified by glycosylation, acetylation, phosphorylation, specific proteolytic cleavage and other forms of processing. In many instances post transcriptional and post translational modifications are crucial in determining the final biological properties of protein products. Non proteolytic post translational modifications of certain proteins such as glycosylation, acetylation, phosphorylation and others may not occur correctly, if at all, in bacteria or cell types significantly different from the cell type in which the gene product of interest is normally produced in the organism.The correct form of these modifications may prove criti cal in the synthesis of fully competent gene products by molecular cloning techniques. For example, when the gene product is a glycoprotein, such as the haemagglutinin of influenza virus, the precise nature of glycosylation may influence the efficiency of antibodrles raised against this synthetic protein to protect human beings or animals against influenza virus infections. Moreover, when glycosylation, acetylation or phosphorylation or other modifications are required to stabilize, activate, or mediate intracellular transport or excretion of a protein, the precision of these modifications may be critical to the utility of these genetically engineered protein pro, ducts in practice. Because of the above mentioned shortcomings of using bacteria for the synthesis of complex gene products, there have been a number of attempts to introduce DNA encoding specific eukaryotic proteins into eukaryotic cells. DNA can be introduced by co transformation into suitable mutant cells which are deficient in the production of a particular enzyme, such as thymidine kinase or hypoxanthine phosphoribosyl transferase. Ten by culturing the cells in a selective medium deficient in the enzyme product, transformed cells may be selected for by their ability to grow, i.e., their ability to produce the enzyme. Alternatively, cells can be co transfected in a positive sense to add a gene that will transduce cells to become selectively resistant to a drug or selective medium such methotrexate, neomycin, or others.Although many of these attests have enjoyed some measure of success, in most cases the yield of mature gene products has been quite limited. In order to maximize the yields of expression of fully carpe tent gene products of interest, it would be desirable that the host expression unit system employed consists of an expression vector derived from one cell type or organism that is capable of synthesizing the said fully competent gene products, and that this expression vec tor directs the synthesis of said gene products in the same or similar cell type or organism as host. 2. Description of the Prior Art The e heat shock phenomenon has been studied most extensively inDrosophila melanogaster. For a review, see Ashburner and Bonner 1979 Cell 17 241 254. When Drosophila cells or organs, normally at about 250C, are exposed to a heat treatment at 35 370C, a family of heatshock genes is activated and most of the genes active at 250C are no longer transcribed. Seven genes code for polypeptides with molecular weights between 22,000 and 84,000 daltons. During heat treatment these heat shock polypeptides are synthesized almost exclusively and after 8 hrs, represent 10 of the total cellular protein Arrigo, P. 1979 Ph. D. Thesis, University of Geneva .During heat treatment of Drosophila cells, much of the polysome bound mRNA codes for heat shock proteins McKenzie et al. 1975 72 117 1121 Mi rault et al. 1978 Cold Spring Harbor Syitp. Quant. Biol. 42 819 827 . All seven Drosophila heat shock protein genes have been cloned.See, Livak et al. 1978 PNAS 75 5613 5617 Schedl et al. 1978 Cell 14 921 929 Craig et ai. 1979 Cell 16 575 588 Holngren et al. 1979 Cell 18 1359 1370 Wadsworth et al. 1980 PNAS 77 2134 2137 forces et al. 1980 PNAS 77 5390 5393 Voellmy et al. 1981 Cell 23 261 270. A number of the genes have been sequenced. See Karch and Torok 1980 Nucleic Acid Res. 8 3105 3123, and Ingola and Craig 1982 PNAS 79 2360 2364. All eukaryotic organisms appear to possess heat shock genes.See Relly and Schlesinger 1978 Cell 15 1277 1288. Many of the heat shock genes appear to be conserved throughout widely diverse species, and Drosophila heat shock genes have been shown to be transcribed in mouse cells Corces et al. 1981 PNAS 78 7038 7042 , frog cells OVoellmy and Ranger 1982 PNkS 79 1776 1780 , and monkey cells Pelham 1982 Cell 30 517 528 . Fusion genes consisting of Droso phila heat shock gene regions and Herpes Simple virus thymidine kinase gene regions are also transcribed in these heterologous cell systems. No evidence has been presented which would suggest that the protein products of these genes are formed. Pelham, H. and Bienz,M. 1982 p. 43 48 in Heat Shock from Bacteria to Man.Ed. Schlesinr ger, Ashburner and Tissieres. Cold Spring Harbour Press. Coerces et al. 1982 p. 27 34 in Eeat Shock from Bacteria to Man. Ed. Schlesinger, Ashburner and Tissieres. Cold Spring Harbour Press. Summary of the Invention DNA constructions are provided comprising control elements deriv ed fran heat shock genes associated with genes of interest, which permit the expression of the gene of interest in both procaryotic and eucaryotic cells. In particular, for the efficient synthesis of fully competent gene products, a homologous combination of expression unit and host cell for expression is used and the nature of this homologous system is that of the same or similar cell type or organism that normally produces the said gene product. Brief Description of the Drawings Fig. la is a schematic representation of a partial restriction map of part of plasmid 132E3 containing two 70 kdal heat shock protein hsp genes heavy lines and their orientation arrows . The lower part of Fig. la represents the relevant partial restriction map of plasmid 51 for comparison purposes. Fig. Ib represents on an enlarged scale a more detailed restriction map of part of plasmid 51 with additional restriction sites, including the two characteristic Sau3A sites. Fig. 2 represents the complete DNA sequence of one strand of the DNA preceding the 70 kdal hsp gene in plasmid 51 except for the white sequences on a black background, which are present in another 70 kdal gene sequence, as determined by the procedure of Maxam andGilbert 1977 , PNAS 74 560 , and in addition some inportant restriction sequences as well as the site of the start of transcription and translation arrows . Figs 3a 3d represent a diagrammatic representation of the procedure for the construction of plasmid pRV15. In particular, Fig. 3a shows a restriction map of plasmid 51 which after digestion with Sau3A gives rise to an approximately 650 bp fragment referred to henceforth as the 650 bp fragment, one end of which is indicated in detail in Fig. 3c left part . Fig. 3b is a restriction map of plasmid pMC1403 which, after restriction with BailiEl, leaves one end with the structure indicated in detail in Fig. 3c right part . Fig. 3c represents details of the sequence of the ends produced by the restriction digests of plasmids 51 and pMC1403. Fig. 3d represents the structure of plasmid pRV15 showing the linkage of the Sau3A fragment of plasmid 51 in the BamBI site of plasmid pMC1403. The position of the 3 galactosidase gene and of the Drosophila DNA are indicated in the plasmid by heavy lines. Same restriction sites are also indicated. Fig. 4a is a photograph showing characterization of the structure of plasmid pRV15 indicated in Fig. 3d by the colony hybridization assay of Grunstein and Hogness from 1975 PNAS 72 3961. This was performed using radioactive probes prepared by nick translation Maniatis et al., 1975 PNAS 72 1184 of either a 2 kbp XbaI fragment excised Grossmann plasmid 132E3 see Fig. la or a labelled 650 bp Sau3A fragment from plasmid 51 Fig. 4b . A number of transformants produced using the construction scheme of Fig. 3 were tested as were some colonies of FMC1403 as a control these colonies are indicated by arrows in Fig. 4a. Fig. 4b represents a similar colony hybridization assay but using the Sau3A 650 kbp fragment from plasmid 51 as a probe. Control colonies of pMK1403 are indicated by arrows. Fig. 5 represents the characterization, by restriction analy sis, of the structure for plasmid pRV15 indicated in Fig. 3d. Fragment sized in bps are indicated. In particular Fig. 5a is a photograph showing the migration of various DNA fragments under electrophoresis. DNA was prepared from mini plasmid preparations Davis et al., 1980 , methods in Enzymology, pRV15 and Mbldave, eds. 65 404 414 for plasmid pMC1403, pRV5. and two similar isolates, pRV25 and after These plasmids were digested with restriction enzymes as indicated below and the fragments of DNA produced were electrophoresed on 0.85 agarose gels at 150 v for 4 hours whiclt bands were visualized using ethidium bromide staining and U.V. photography. Identification of the lanes is as follows Lane 1 pMC1403 Sau3A XhoI 2 SalI XbaI 3 SalI EcoRI 4 pRV25 SalI XhoI 5 SalI XbaI 6 SalI EcoRI 7 pRV15 SalI XhoI 8 SalI XbaI 9 SalI EcoRI 10 pRV5 SalI XhoI 11 SalI Xbal 12 SalI EcoRI Fig. 5b is similar to Fig. 5a for a series of digests and analyses performed identically.The lanes are as follows lane 1 pRV15 XhoI 2 XbaI 3 SalI XhoI 4 SalI XbaI 5 SalI 6 pRV5 XhoI 7 11 8 SalI XhoI 9 SalI XbaI 10 SalI Fig. 5c is similar toFigs 5a and 5b and concerns restriction digests and analyses performed for fragments separated on gels of 5 polyacrylamide. Electrcphoresis was at 40 v for 14 hours. lane 1 51 Sau3A XbaI 2 51 Sau3A2XhoI 3 51 Sau3A 4 pMC1403 Sau3A 5 pRV15 Sau3A XbaI 6 pRV15 Sau3A XhoI 7 pRV15 Sau3A 8 Diagrammatic representation of 51 Sau3A 9 Diagrammatic representation of pBR322 Sau3A Figs 6a 6b refer to an analysis of the orientation of the Drosophila 70 kdal hsp gene Sau3A fragment in plasmid pRVlS. In particular Fig. 6a is a diagrammatic representation of the sizes of predicted restriction fragments that would result if the desired orientation of Drosophila elements and the E. coli ss galactosidase gene is indeed the case in plasmid pRV15. Fig. 6b is a photograph of an acrylamide gel 58 analysis of the results of restriction of plasmid pRV15 and pBR322 to test the prediction presented in the diagram of Fig. 6a. Analyses were performed as described for the cases represented by Fig. Sc. lane 1 pBR322 HinfI 2 pRMl5 EccRI 3 pRV15 XbaI EccRI 4 pRV15 XhoI Eco 5 pRV15 XhoI The position of bands of sizes 517, 220 and 154 bps are indicated by arrows and are derived from the. known sizes of fragments derived by restriction of plasmid pBR322 by Hindi. Figs 7a 7e, which show plasmids or fragments of plasmids after digestion with restriction enzymes, represent schematically the cons truction of plasmid 520, which consists of incorporating the SmaI SalI fragment of pRV15 containing the lac operon fragment and theDrosophila 650 bp control element into plasmid pSVod using the BamHI SalI restriction site. In particular Fig. 7a represents pRV15, Fig. 7c represents p5Vod Fig. 7b represents a Smal SalI fragment of pRV15 Fig. 7d represents pSVod after excision of a BamHI Sail fragment and Fig. 7e represents p520. Fig. 8a is a restriction map for plasmid vector 520 indicating the position of selected restriction sites in the structure of the plasmid. Fig. 8b is a photograph of an agarose gel analysis of restriction enzyme digestions of plasmid 520 performed as described in the case of pRV15 see Fig. 5a . lane I HindIII 2 XhoI EcoRI 3 HindIII EcoRI 4 PstI 5 HindIII PstI 6 PstI HindIII EcoRI 7 PstI EcoRI Fig. 8c is a photograph also referring to the characterization of plasmid 520. It shows agarose gel analysis of various restriction digest of plasmids pSVod, pRV15 and 520 like plasmids, i.e., plasmids 639, 520, 519, X, Y, Z. lane 1 639 SalI EcoRI 2 520 3 519 4 516 5 X n n 6 Y 7 Z n n II 8 pSVod 9 pRV15 10 pSVod XhoI EcoRI ll pRV15 Note that the digests in lanes 8 and 9 are incomplete. Fig. 8d is a photograph showing further restriction digests of plasmid 520 by comparison with digests of plasmids pSVod and pRV15. lane 1 pSVod PstI EcoRI 2 pSVod HindIII SalI 3 pRV15 PstI 4 pRV15 HindIII SalI 5 pE 15 SalI XbaI 6 520 PstI EcoRI 7 520 HindIII SalI 8 520 SalI XbaI Note that the digests in lanes 2 and 7 are incomplete. Figs 9a 9c refer to the construction of plasmid PR81. In particular Fig. 9a represents the Mount Sinai A PR 8 ms 4.76 plasmid fram which a fragment ref. no. 20 was excised with BamHI and HindIII Fig. 9b represents pSVod less a HindIII SalI fragment and Fig.9c represents plasmid PR81, i.e., the ligation product of the BamHI HindIII fragment 20 with a Sal HindIII digest. Figs 9d and 9e are photographs of a Grunstein colony hybridization analysis of PR81 like plasmids, performed as described for pRV15 see Fig. 4 . The nick translated probe used was the BamHI HindIII fragment for A PR 8 ms 4.76. As a control some pSVod containing colonies are indicated by the arrows. Fig. 9f is a photograph of a restriction analysis of PR81 like plasmids shown by colony hybridization analysis in Figs 9d and 9e. lane 1 filter 21, Sample 30 EcoRI 2 n 21, Sample 44 3 n 22, Sample 11 4 22, Sample 24 5 23, Sample 10 6 n 23, Sample 31 7 24,Sample 8 8 pSVod n n 9 PR 8 ms 4.76 n n II 10 filter 21, Sample 30 HindIII SalI 11 21, Sample 44 n II 12 n 22, Sample 11 n 13 22,Sample 24 14 23, Sample 10 n U 15 23, Sample 31 II 16 24, Sample 8 17 pSVod 18 PR 8 ms 4.76 Figs 10a 10e refer to the construction scheme of plasmid PR84.This plasmid was constructed as shown in the figure and consists of substituting the haemagglutinin HA gene 20 present in plasmid PR81 for thess galactosidase gene in plasmid 520. In particular Fig. 10a represents plasmid 520 Fig. loc represents plasmidPR81 Fig. 10b represents an NcoI PvuII fragment excised from 520 Fig. lod represents PR81 after digestion with HindIII and NcoI Fig.lOe represents schematically PR84. Fig. lOf is a more detailed representation of PR84 with one strand of base pairs including the sequence originating from PR 8 ms only the crucial joining region between the Drosophila HS control element and the HA start sequence is detailed. Fig. 11 refers to checking the results fran the construction of PR84. Fig. Ila is a photograph referring to the restriction digestion of plasmids PR81, PR 8 ms 34, pSVod and 520. lane 1 PR81 HindIII EcoRI 2 PR81 XhoI EcoRI 3 PR81 NcoI 4 PR81 NcOI XhoI 5 PR 8 ms 34 HindIII EcoRI 6 PR 8 ms 34 XhoI EcoRI 7 P3l 8 ns 34 NcoI 8 PR 8 ms 34 NcoI XhoI 9 pSVod NkoI SalI 10 pSVod PvuII SalI 11 pSVod HindIII EcoRl 12 pSVod HinfI STANDARD 13 520 PvuII 14 520 PvuII XbaI 15 520 PvuII XhoI 16 520 PvuII EcoRI 17 520 PvuII NcoI 18 520 NcoI 19 520 NcoI XbaI 20 520 NcoIjThoI Figs lIb l and llb 2 refer to a Grunstein colony hybridization analysis of PR 84 like plasmids.A nick translated radioactive probe approximately 108 cpm g of the plasmid 51 Sau 3A Xho I Drosophila control element fragment was used in these assays. Fig. Ile refers to a 5 polyacrylamide gel analysis of the XhoI digestion products of PR84 like plasmids selected from the positive colonies identified in Fig. lib lane 1 filter 3, Sample 12 XhoI 2 3, n 15 PR82 3 3, 22 4 3, 42 PR84 5 3, 45 6 4, 19 7 n 4, H 40 1t n n 8 5, 16 II 9 5, 28 10 5, n 39 n 11 pSVod Rinfi STANDARD Fig. ild refers to further restriction analysis of plasmids giving the XhoI digestion pattern shown in lanes 2, 4, 6 and 8 of Fig.llc. Analysis was performed as described for Fig. llc. lane 1 X 174 RF HaeIII STANDARD 2 filter 5, Sample 16 XbaI NcoI 3 n 5, n 16 XhoI NcoI 4 4, 19 XbaI NcoI 5 4, n 19 XhoI NcoI 6 3, 42 XbaI NcoI 7 3, 42 XhoI NcoI 8 3, 15 XbaI NcoI 9 n 3, n 15 XhoI NcoI Fig. 12 desoribes the characterization of plasmid p629. Fig. 12a refers to a 5 polyacrylamide gel analysis of some candidate 629 series plasmids. Pl mid lane 1 629 8 BglII EcoRI 2 629 7 n 3 629 6 4 629 4 n 5 DNA standard Fig. 12b refers to another 5 polyacrylamide gel analysis of all eight candidate 629 series plasmids. Plasmid lane 1 629 1 XhoI 2 629 2 n 3 629 3 II 4 629 4 5 629 5 n 6 629 6 U 7 629 7 1l 8 629 8 n Fig. 12c shows a Grunstein colony hybridization assay using the nick translated Xho BqlII fragment from plasmid p622b Fig. 12d shows a Grunstein colony hybridization assay using the nick translated HindIII BamHI.HA gene fragment from plasmid A PR 8 mx 34. Description of the Specific Embodiments Methods and caipositions are provided for the controlled expression of a gene in a eukaryotic host. The present invention is particularly useful for expressing mammalian genes in mammalian hosts, typically in a host cell culture similar or identical to the host in which the gene is naturally expressed. In this way, post transcriptional and post translational modifications of the gene products will be achieved, thus assuring the competence of such resulting gene products. DNA sequences which comprise an inducible expression control region derived from a eukaryotic heat shock protein hsp gene are employed. Such control regions are inducible for example by elevated temperatures and include sequences responsible for both transcrip tional and translational control of the hsp gene activity, e.g., the promoter region RNA polymerase recognition and binding sites , and possibly also heat shock protein gene control sequences which have been modified in such a way that they allow the constitutive expression of downstream coding sequences. The control region may also include other portions of the translated and untranslated regions of the hsp gene which participate in transcriptional and or translational regulation including 3 untranslated sequences.Furthermore, sequences derived from the 3 untranslated region flanking the structural gene i.e. thenoncoding portion of the gene of interest may be included. cOnveniently, the entire hsp gene including the structural region as well as the 5 and 3 flanking regions may be employed. In the latter case, the gene of interest may be inserted internal to the structural region to produce a fused protein upon translation.The protein of interest may then be recovered by conventional means. The e term heat shock protein hsp gene will be employed in the specification and the claims to denote the entire gene locus responsible for the inducible expression of heat shock proteins, specifically including the 5 regulatory sequences upstream of the structural region, the structural region which encodes the mRNk transcript including untranslated leader sequences and the 3 flanking region downstream of the structural gene. Heat shock protein genes may be obtained from most higher eukarycr tic organisms. Particular heat shock proteins have been identified in fruit flies Drosophila , mice, frogs, monkeys, and man. Heat shock genes suitable as a source for the control regions of the present invention may be obtained from any of these or other eukaryotic organisms. Although heat shock genes derived from one species can be expressed in other species and even in procaryotes, it is generally desirable that in order to optimize the expression advantages of heatshock control elements at both the transcriptional and the translational levels, these elements should be derived from the same or similar organism or cell type that will be used for expression of the gene of commercial interest.For instance, constructions including genes under control of Drosophila heat shock elements can be most advantageously expressed in Drosophila cells in culture. In fhe Experimental section hereinafter, an inducible expres sion control region obtained from a heat shock protein gene which encodes for a 70 kdal heat shock protein found in Drosophila was used to express E. coli In galactosidase gene in E. coli, COS I Monkey cells, and in Xencous occytes. The same region was used to control the expression of the influenza haemagglutinin gene in COS I cells. The inducible expression control region of the present invention may be combined with an extrachrcmasomal replication system for a predetermined host to provide an expression vector for that host.Such vectors will include DNA sequences having restriction site s for insertion of gene s 3 to said control regions to provide the regulated transcription and translation of the inserted genes. The vector can also include markers for selection in bacteria and in eukaryotic host cells, a prokaryotic replication system allowing cloning of the vector in a prokaryotic host, and other DNA regions of interest. Alternatively, the expression control region can be joined to a desired structural gene and the resulting DNA constructs introduced directly into the host cells. Methods for such direct transfer include injection of the DNA into nuclei Ctppechi 1980 cell 22 479 488 and co transformation by calcium phosphate precipitation fi gler et al. 1979 Cell 16 777 785 or DEAE dextran Cutchan, J.H.and Pagano J.S. 1968 J. Nat. Cancer Inst. 41 351 357 . As stated above, the DNA constructs of the present invention may include differing portions of the hsp gene. The constructs will include at least the 5 regulatory region which carries the promoter, regulatory sequences such as operators, activators, cap signals, signals enhancing ribosomal binding, and other sequences as well as additional DNA leader sequences responsible for transcriptional and translational control. The DNA constructs may also include part of the protein coding sequence of the hsp gene, resulting in the production of fused proteins when the foreign gene to be expressed is inserted downstream of the hsp sequences. If a fused protein is not desired, it will be necessary to resve the hsp coding sequences from the isolated hsp DNA by usual methods such as restriction by enzymes and exonuclease digestion. It may be desirable to leave at least a portion of the hsp structural gene downstream from the natural translational initiation codon. In this way, a fused protein including the amino terminal amino acid sequences of the heat shock protein is provided. When such fused pro teins are produced, it may be desirable to introduce selective cleavage sites so that the desired protein can be separated from the precursor protein. See U.S. Patent No. 4,366,246 to Riggs which teaches how such cleavage sites may be introduced. The expression control region of the present invention will usually be combined with a terminator for complete transcriptional control of the inserted structural gene. Cbnveniently, the terminator can be derived from the heat shock gene itself, although the inserted structural gene may carry its own or any other suitable terminator sequence. Extrachromoscmal replication systems may also be used. Suitable replication systems include autonomously replicating sequences as described by Struhl et al. 1979 PNAS 73 1471 1475 and the 2 um plasmid for replication in yeast. Mammalian replication systems could be derived from papovaviruses, such as simian virus 40 and bovine papillata virus adenoviruses avian retroviruses, such as avian sar coma virus and mammalian retroviruses such as Moloney leukemia virus. In addition to the optional eukaryotic replication system, it is advantageous to provide a prokaryotic replication system to allow for cloning of the vector in a bacterial host. This allows large quantities of the vector to be grown in well characterized bacterial systems prior to transforming a eukaryotic host. Suitable prokaryotic replication systems are well known and include plasmids such as pBRB22, pRK290, ColEl, and bacteriophages, e.g. iSdv. The e prokaryotic replication systems will necessarily include an origin of replication recognizable by a prokaryotic host, and will usually include one or more markers for the selection of transformants in the prokaryotic host. Such markers include biocide resistance, toxin resistance, and the like. Alternatively, coqplementation allowing the growth of an auxotrophic host in a selective medium may be employed. Such techniques are well known in the art and need not be described further. Usually, the markers employed will be different for selection in prokaryotic and eukaryotic hosts. Various dominantly acting markers are useful in selecting for transformed mammalian cell lines. They usually comprise a specific gene whose expression confers a new drug resistant phenotype to the mammalian cells in an appropriate selective medium. Specific markers include the bacterial xanthine guanine phosphoribosyl transferase gene which can be selected in medium containing mycophenolic acid and xanthine Mulligan et al. 1981 PNAS 78 2072 2076 transformants with vectors carrying a mouse cDNA fragment coding for dihydrofolate reductase may be selected for using medium containing aminopterin Subramani et al. 1981 Mol. Cell Biol.1 854 861 and a bacterial plasmid gene specifying an amino glycoside phosphotransferase that inactivates the anti bacterial action of neomycin kanamycin derivatives may be selected for using medium containing G418 a neomycin derivative toxic for most mammalian cell lines Colbere Garapin et al. 1981 J. Mol. Biol. 150 1 14 . It is evident that the number of copies of gene expression units introduced into a host cell may vary and that by proximal combina tion of one of the amplifiable genes such as the mouse dihydrofolate reductase gene the number of copies of integrated gene expression units may likewise be ampli by induction of the amplification of the associated amplifiable gene and the proximnl DNA. See for exams ple the co amplification of dihydrofolate reductase cDNA and theE. coli XGPRT xanthine guanine phorphoribosyl transferase gene inChinese ovary cells RLingold, G. et al., 1981 . J. Mol. and Appl.Genetics, 1, 165 175 . In the exemplary method for preparing the vectors of the sub ject invention, both the expression control region of the heat shock protein gene and the eukaryotic replication system are inserted into a suitable prokaryotic plasmid. The manner and order of the insertion are not critical, and it is necessary only that the resulting vector retains viable replication systems for prokaryotic and when necessary eukaryotic hosts. The DNk constructs containing Drosophila heat shock gene control segments functionally linked to a structural gene of interest can be used to synthesize products of said gene in Drosophila cells or cultured cells closely related to Drosophila. Since the structural gene is under the transcription and if necessary translation control of the heat shock control element, its expression can be enhanced by for example increasing the ambient teirperature of the cells for instance in the range 37 420C. By such induction a large fraction of the newly made mRNA will be derived from said structural gene.Now if the structural gene is a human gene encoding a protein requiring complex processing, the expression vector preferably comprises, in addition to said structural gene, replication elements and marker genes, a heat shock control element derived from a human heatshock gene. This construct will be introduced advantageously into cultured human cells by procedures described above to produce products of said human gene. As necessary the choice of recipient human cells will be made on the basis of their competence in correctly expressing the fully processed gene product. A wide variety of structural genes may be introduced into the subject vectors to permit the production of various gene products including polypeptides, such as enzymes, proteins, hormones, novel protein structures and the like. Experimental The following examples are offered by way of illustration. 1. The construction of plasmid pRV15 which contains the E. coli 2 galactosidase gene under the control of a Drosophila heat shock control element. A 650 base pair bp DNA fragment from one of the two Droscphila 70 kdal heat shock protein genes, containing a heat shock gene transcription control element, a complete RNA leader sequence, a translation initiation signal and a sequence coding for the first few amino acids of the Drosophila 70 kdal heat shock protein, was obtained from a sub clone of part of plasmid 132E3 Schedl et al. 1978 Cell 14 921 929 see Fig. 1 . Plasmid 132E3 contains two complete genes for the 70 kdal heat shock protein, and the aforementioned isolated sub clone, plasmid 51 Karch et al. 1981 J.Mol Biol. 148 219 230 , contains a fragment of the first heat shock gene in plasmid 132E3. This fragment was isolated by digestion of plasmid p51 with the restriction endonucleases BolII and BanrHI. The upper part of Fig. la is a representation of a portion of plasmid 132E3 indicating the two genes encoding 70 kdal heat shock proteins thick line segments and sane of the identified restric tion sites. The e lower part of Fig. la represents a portion of p51 containing the BglTI BamHI segment containing the aforesaid 650 bp fragment, bounded by the Sau3A cleavage sites. Fig. Ib is a more detailed representation of the portion of interest contained in p51 with additional restriction sites indicated, and also showing the position of the 650 bp fragment between twoSau3A sites. Fig. 2 provides the DNk sequence data for one strand contained in the aforementioned 650 base pair fragment. The limits of this sequence are indicated by the position of the Sau3A restriction sites GATC . Also, XbaI, XhoI and PstI recognition sites for restriction are indicated, and the position of the transcription and translation start sequence are indicated by arrows la and Ib, respectively. The 650 bp fragment was functionally inserted into plasmid pMCll403 in a position to control the expression of test genes in this plasmid.Plasmid pMC1403 Casadaban et al. 1980 J. Bacteriol. 143 971 980 is a derivative of plasmid pBR322 Bolivar et al. 1977 Gene 2 95 113 containing the entire E. coli lac operon with the exception of the sequences coding for the first seven amino acids of p galac tosidase and all sequences 5 to the ss galactosidase protein coding region promoter, riboscmal binding site, translation initiation codon . Consequently lac strains of E. coli such as MC 1061 Casada ban et al. 1980 J. Mol. Biol 138 179 207 carrying plasmid pMC1403 do not produce ss galactosidase.A polylinker EcoRI, Srnai, Bam9I at the 51 end of the lac sequences in pMC1403 permits the introduction of foreign DNA sequences upstream from the ss galactosidase cod ing region. Insertion of a segment containing a functional promoter and the RNA leader sequences will result in the production of ss galac tosidase activity. It should be noted that the amino terminal end of ss 3galactosidase is not essential for its enzymatic activity ul ler Hill B. and Kania, J. 1974 Nature 249 561 563 . All that is therefore required is that the inserted promoter PNA leader sequence permits reading in the correct frame with the incomplete 3galac tosidase coding region. Hence the 650 bp segment was ligated into the BamHI site of plasmid pMK1403, in front of the incomplete num galac tosidase gene. The e new recombinant plasmid thus obtained see Fig.3 was designated pRV15. The procedure of this plasmid construction is shown in a diagrammatic form in Fig. 3. In this figure, the up per left part Fig. 3a represents a section of the aforementioned plasmid 51 containing the 650 bp fragment 2 to be cleaved out withSau3A and to be inserted into the BamHI site of plasmid pMC1403 shown on Fig. 3b. The truncated lac operon of the plasmid pMK1403 is represented by the heavy line with numeral 3 on the diagram of Fig. 3b.It also comprises a structural gene for ampicillin resistance. Plasmid p51 is digested with Sau3A to excise the aforesaid 650 bp fragment represented on the left of Fig. 3c by numeral 2 this fragment was purified from polyacrylamide gels see Fig. 5c by electroelution. This fragment was inserted into the BartHI site of pMK1403 in either orientation. The ligation mixture was then used to transform the lac strain of E. coli MC 1061. Transformants were plated on media containing ampicillin and Xgal Xgal is 5 bromo 4 chloro 3 indolyl N D gelactoside which is a subs trate for 0 ss galactosidase, and which after cleavage by the enzyme produces an identifiable colored product see Miller, J. Experiments in Molecular Genetics, pp. 47 55, Cold Spring Harbor, N.Y. 1972 .Plasmid pMC1403 did not produce galactosidase activity in this assay while plasmid pRV15 and a large number of other transformants, prepared in the aforesaid manner, were found to produce substantial amounts of 0 galactosidase as determined by the color change produced on Xgal plates. In addition, ss galactosidase coding plasmid constructions such as pRV15 were analyzed further by the colony hybridization assay ofGrunstein Grunstein and Hogness 1975 PNAS 72 3961 3966 using either radioactively labelled 650 bp Sau3A fragments from p51 seeFig. 4b or the 2 kbp XbaI gene fragment see Figs la, 4a from plasmid 132E3 as hybridization probes. These fragments were radioactively labelled by the process of nick translation Maniatis et al. 1975 PNAS 72 1184 . As seen in Figs 4a and 4b, all selected trans formats hybridized to both radioactive probe DNAs demonstrating the presence of both DNA sequences in recombinant plasmids such as pRV15. The presence of the 650 bp fragment containing the Drosophila 70 kdal heat shock gene control element was confirmed by restriction analysis, see Figs 5a, b and c. Plasmid pMC1403 contains unique restriction sites for EccRI and Sal, but no XhoI or XbaI sites. TheSau3A 650 bp fragment from plasmid p51 hcwever contains unique sites for restriction by Thol and XbaI but has no sites for restriction enzymes such as EcoRI and SalI. In contrast, recombinant plasmids such as pRV15 should contain unique sites for all four aforementioned enzymes see Fig. 3 . Restriction of recombinant DEA from plasmids such as pRV15 with XhoI and SalI or with XbaI and SalI should produce two fragments of about 4 and 6.5 7 kbp if the structure shown for pRV15 in Fig. 3 is correct.Plasmid pMC1403 however should only be linearized. Fig. 5a shows a photograph of DNA fragments produced by the aforementioned restriction enzymes after electrophoresis on agarose gels, and confirms the structure of plasmid pRV15 shown inFig. 3. These results are presented again in Fig. 5b where it is also demonstrated that digestion with SalI, XhoI or XbaI individually only linearize pRlS, further confirming the structure shown in Fig. 3. Further, if pRV15 contains the 650 bp Sau3A fragment of plasmid p51 inserted into the BamHI site of pMC1403, then it should be possible to recover the 650 bp Sau3A fragment by digestion of recom binant DNAs such as pRV15 with the restriction enzyme Sau3A. That this is the case is shown in Fig. Sc lane 7 by the arrows. In addition, the identity of the 650 bp excised fragment is confirmed by its restriction by Thai or by XhoI. See Fig. Sc lanes 5 and 6. The demonstration that the Sau3A 650 bp fragment and th ss galac tosidase coding sequences are in the orientation shown for pRV15 inFig. 3 is presented in Figs 6a and b. In Fig. 6a, the correct orientation is shown in a diagrammatic form from which the respective double digestions with EcoRI and XbaI or by EcoRI and XhoI would predict the excisicn of fragments from the 650 bp fragment, correctly oriented with respect to the p galactosidase gene 12, of respecti vely 150 and 220 bp. In Fig. 6a, the arrow 11 indicates the transcriptional direction of the heat shock control elements.The above prediction is demonstrated by the restriction gel analysis shown inFig. 6b where EcoRI and Thai doubie digestion liberate an approximately 150 bp fragment, and EcoRI, XhoI double digestion, a 220 bp fragment as evidenced by electropheresis on 5 acrylamide gels. These experiments confirm that plasmid pRV15 has the desired orientation of control and coding elements shown in Fig. 3. The position of bands of sizes 515, 220 and 154 bp are indicated by arrows marked with the corresponding numbering. The hybrid plasmid pRV15 and other identical isolates constructed as described in the preceding section, produce substantial amounts of ss galactosidase as determined by examining the color changes on Xgal plates. The e presence of galactosidase protein produced in E. coli under the control of a Drosophila heat shock control element has also been clearly demonstrated by immune precipitation of protein extracts of E. coli containing plasmid pRV15 after radioactive labelling a newly synthesized protein with 35S methionine. A polypeptide of molecular weight approximately 120,000 daltons was precipitated from such protein extracts by immune sera directed against authentic E. coli ss galactosidase. The e details of the construction and expression of pRV15 are now set forth. A. Construction of Plasmid EBRV15 10 pig of plasmid p51 Fig. la, lower part were digested for 4 hours at 370C with 10 units of Sau3A incubation buffers for this, and all other digestion described below as suggested by the supplier of the restriction enzymes New England Biolabs Cat. 1982 . The concentration of DNA during digestion with Sau3A was 40 tg ml. The digestion products were electrophoresed on a non denaturing 5 polyacrylamide gel in 1 x TBE 10.9 g Tris base, 5.5 g boric acid, 0.93 g Na2EDTA per liter H20 buffer. A Sau3A digest of pBR322 was electrophoresed in parallel and served to identify the 650 bp 51 Sau3A promoter fragment fig. 5c . DNA fragments were visualized with ethidium bromide EtBr . The region containing the promoter fragment, indicated by the arrows in Fig. 5c, was cut out of the gel and the fragment was electroeluted into a dialysis bag. Electroelution was carried out for 5 hours at 200v in 1 x TBE buffer. The eluate was collected in a 15 ml siliconized Cbrex tube and ethanol precipatated overnight at 200C. The precipitate was collected by centrifugation Sorvall, 30 min., 10,000 rprn, SS34 rotor , dried in a lyophi lizer and resuspended in 200 l of TE buffer 10 mM Tris. HCl, pH 7.5, 1 mM Na2EDTA . The DNA Was then extracted twice with TE saturated phenol and twice with ether. The solution was then passed through a Sephadex G75 minicolurrn in a Pasteur pipet .DNA in the column eluate was ethanol precipitated twice, dried and resuspended in 20 jil of TE buffer. A 10 l portion was incubated with severalUnits of XbaT in the appropriate digestion buffer for 1 hour at 37 C,The partially digested DNA was electrophoresed on a 5 polyacrylamide gel. This experiment Fig. 5C demonstrated that the isolated segment was the 650 bp Sau3A fragment, since the promoter fragment includes the only XbaI site of plasmid p51. One g of plasmid pMC1403 was digested with 4 units of BanHI for 30 min. at 37 C. The digested DNA was extracted 3 times with phenol and then 3 times with ether. The DNA was further purified by 2 subsequent ethanol precipitations. The pelleted DNA was dried and then resuspended in 20 l of TE buffer. Ten P aliquots of the solutions containing the 650 bp promoter fragment and digested pMC1403 were combined and then incubated overnight at 14 C with an excess of T4 DNA ligase in a total volume of 25 1 incubation buffer as recommended by New England Biolabs 1982 Cat. . The ligation mixture was then incubated for several hours with 8 units of BamHI at 140C. Aliquots 1, 3 and 7 1 were then used to transform 0.5 ml aliquots of CaC12 treated E. coli MC 1061.Aliquots 0.1 ml of the transformation suspension were then placed on IB agar containing 10 Fg ml apicillin and 40 ug ml of Xgal.Blue colonies were observed after overnight incubation of the plates at 37 C, Approximately 80 pwgalactosidase producing colonies were isolated. The presence of the p51 Sau3A promoter fragment in the recombinants was established by Grunstein colony hybridization Fig.4b . The hybridization probes were prepared as follows the p51 Sau3A promoter fragment was isolated as described above. 50 pg of 132E3 were digested with 50 units of XbaI for 2 hours at 37 C. The digestion prcducts were separated on a 0.98 agarose gel. The 2 kbp 70 kdal heat shock protein gene fragment was eluted electrophoretically from the gel and was purified as described above. The latter fragment and the p51 Sau3A fragment were then 32P labelled by nick translation to a specific radioactivity of 2x108 cpmfpg. The two probes were then denatured and hybridized to two nitrccellulose filters containing DNA of the 80 selected transformants and of pMCl403 Figs 4a, b .Both probes strongly hybridized to DNAs of all 80 recombinants, suggesting that all recombinante contained the 650 bp promoter fragment.Clones 5, 15, 25, 35, 45 and 55 were selected for further studies.Small quantities of DNA were prepared from each of these clones Davis et al. 1980 Methods in Fnzymology, Grossmann and Moldave, eds. 65 404 414 . These DNAs and pMK1403 DNA were compared further by restriction digestion and electrophoresis on 0.9 agarose gels Figs 5a, b . The e results indicate that restriction of the recombinant plasmids such as pRV15 with XhoI and SalI or with XbaI and SalI produce two fragments of 4 and 6.5 7 kbp, thus confirming the structure of plasmid pRV15 as shown in Fig. 3d. B. Expression of the Drosophilia Heat Shock E coli ss galacto sidase Fusion Gene in E. coli Bacteria containing the hybrid plasmid pRV15 and other isolates constructed as described in the preceding section, produce substantial amounts of galactosidase as determined by examining the color changes on Xgal plates. The presence of a 2 galactosidase protein produced in E. coli under the control of a Drosophila heat shock control element was also demonstrated by imrmne precipitation Bromley et al. 1979 J. Virol. 31 86 93 of protein extracts of E. coli containing plasmid pRV15 after radioactive labelling of newly synthe sized proteins with 35S methionine.A polypeptide of molecular weight about 120,000 daltons was clearly precipitated from such protein extracts by immune sera directed against authentic E. coli ss galacto sidase. C. Expression of a Heat Shock ss galactosidase Hybrid Gene inXenopus Ocaytes Plasmid pRV15 50 100 g was digested with an excess of SalI and EcoRI. The two resulting restriction fragments were then separated on 0.85 agarose gels. The 7kb fragment containing the hybrid gene but no vector sequences was purified by electroelution and gel filtration on Sephadex G75. This fragment was then incubated with an excess of T4 DNk ligase in a total volume of 25 1 as described above, to permit circle formation. The ligated fragments were injected into oocytes as described by Voollmy and Rungger 1982 PRas 79 1776 1780.Following a 6 20 hours preincubation at 200C, the oocytes were injected a second time with alpha 32P GTP and either heat treated for 2 hours at 37 C or kept at 200C for the same period. Total RNE prepared from these oocytes or from oocytes that did not contain foreign DNA was hybridized to Southern blots of Sal I Eco RI digests of plasmid pMC1403. Oocytes that did not contain the hybrid gene did not produce measurable amounts of RNA complementary to the ss galac tosidase gene. Both heat treated and untreated oocytes containing the pR7l5 fragment were found to have made galactosidase RNA in approximately similar quantities. In another series of experiments, unlabelled RNA was isolated from oocytes containing the hybrid gene which had been either heattreated for 2 hours or incubated at 200C for the same time in the presence or absence of low concentration of alpha Amanitin Voellmy andRungger 1982 PRAS 79 1776 1780 . The occyte RNAs were labelled by reverse transcription as described by Branley et al. 1979 J.Virol. 31 86 93 and hybridized to pMC1403 Southern blots. Again 3 galactosidase transcripts were present in RNAs for heat treated and untreated cocytes. alpha Amanitin did stop transoription of the hy brid gene at both temperatures indicating that RNA polymerase B i.e. II was responsible for all observed transcriptional activity. 2. The Construction and use in Eukaryotic Cells of Plasmid PR84 which contains a Buman Influenza Virus Haemagglutinin Gene under theControl ofr a Drosophila Heat sbock Control eloment A plasmid denoted 520 was constructed from plasmid pRi15 and plasmid pSvod Mellon et al. 1981 Cell 27 279 288 which is capable of replicating either in procaryotic cells or in certain eukaryotic cells. Plasmid 520 allows rapid analysis of transcriptional control since it includes the SV40 virus origin of replication and is able to replicate efficiently in SV40 mstant transformed, transformation antigen positive COS cells Gluzman 1981 Cell 23 175 82 . The schematic form of the construction of plasmid 520 is shown in Fig. 7, Plasmid pRV15 Fig. 7a, representation in linear form was digested with Seal, subsequently digested with SalI and the resulting 7 kbp fragment Fig. 7c containing the 650 bp Drosophila control element 2 and the lac gene fragment 3 originating from pMK1403 present in pRV15 was isolated by electrcphoretic elution from a preparative agarose gel. Plasmid pSVod Fig. 7c was digested with BamHI, the cohesive ends were filled in with DNk polymerase Klenow fragment and the DNk finally digested with SalI. In Fig. 7c, numeral 10 indicates the site of the deletion of the lkb pBR322 sequence which has been claimed to be inhibitory for replication in eukaryotic cells. The truncated pSVod fragment Fig. 7d so formed was ligated to the 7 kbp fragment isolated from pRV15, and the ligation mixture was used to transform E. coli MC1061. The structure of plasmid 520 is shown in Fig.7e in which a more detailed representation of the 650 bp segment also appears with the XbaI, XhoI and PstI restriction sites. Another restriction site PvuII is also indicated and corresponds to the CAGCrG sequence between 60 and 70 bp, as denoted in Fig. 2. The e significance of the PvuII site will be mentioned hereinafter. Plasmid 520 is shown in circularized form in Fig. 8a where the segments a, y and z represent genes of the lac operon. Numeral 11 designates the fragment containing the SV40 origin. Transformants were plated out on Xgal Ampicillin plates as described previously, and blue transformants suspected of containing plasmid 520 were isolated. From the structure shown in Fig. 8a, plasmid 520 should provide the following size fragments when digested with the named restriction enzyme combinations. Fragment sized in kbp Enzymes 0.75 1.1 8.5 Pstl EcoR1 2.6 7.2 HindIII SalI 3.1 6.7 SalI XbaI The 520 plasmids indeed produced fragments of the correct sizes when digested as evidenced in the gel analyses of Figs 8b, c and d.other fragments produced by various combinations of enzyme digestions serve as controls for the interpretion of the digestions noted above. Plasmid p81 places a gene which encodes for a eukaryotic protein in the correct reading frame with the 650 bp heat shock control element of p520. As starting material for this construction, plasmid A PR 8 ms 34 IMbunt Sinai Clone No. 4.76 containing a human influenza haemagglutinin gene 1775 bp, numeral 20 inserted into thePvuII site of vector PAT 153 Pvu II 8 was obtained from Dr. G. Brcwn lee University of Oxford, Department of Pathology U.K. . The plasmid containing the haemagglutinin HA gene insert 20, see Fig. 9 was digested with BamHI, and the ends were filled in with DNk polymerase Klenow fragment. Following digestion with HindIII, the haemagglutinin gene fragment was purified on agarose gels.The insert 20 contains a complete HA protein coding sequence, an RNA leader segment, and 40 bp of 3 nontranslated sequence. The HA gene contain ing fragment was inserted into plasmid vector pSVod to produce plasmid PRS1 as follows pSVod DNk was digested with SalI, ends were filled in as described above, and the DNA was further digested with HindlIl see structure in Fig. 9b . The HA fragment 20 and the SalI BindlII digest of pSood were ligated tcgether using T4 DNA ligase in the presence of BamHI and SalI to give PRB1 see Fig. 9c , and the ligation mixture was used to transform E. coli C 600. Transformants were analyzed for the presence of Per like plasmids for predicted structure, see Fig. 9c as follows Grunstein colony hybridization was performed in a manner similar to that described earlier in connection with pRV15 see Fig. 4 . A nick trans lated prcbe of the HA gene BamHI HindIII fragment from pA PR 8 34 ms 4.76 kb was prepared and hybridized to the PRBl like transformants see Figs 9d and 9e . Figs 9d and 9e show the autoradiograms of eight colonies. It can be seen from Figs 9d and 9e that this DNA probe hybridizes to most of the transformants. Negative controls are provided by the pSVod containing colonies indicated in the figure by arrows. PRBl like plasmids were further characterized by restriction analysis, as shown in Fig. 9f. The structure indicated for PRS1 inFig. 9c is verified by the formation of DNA fragments of the following sizes after digestion with the restriction enzymes indicated below Plasmids Sizes of fragments kbp Enzymes used PRS1 3.2, 1.5 1.6 EcoRIPR 8 34 3.8, 1.3 EcoRI pSVod 3.3 EcoRIPR81 4.5 HindIII SalIPR 8 34 3.3, 1.7 1.8 HindIII SalI Psvod 2.7, 0.6 0.7 HindIII SalI The results provided in Fig. 9f indicate the presence of the above fragments. The e construction scheme for plasmid PR84 is presented in Figs 10a to 10e. Briefly, plasmid PRs4 places the HA gene under the control of expression control region of the Droscohila heat shock gene described heretofore. The hybrid gene itself is placed under the replication control of plasmid pSVod. Digestion of plasmid 520 Fig. 10a with PvuPI and NcoI results in the formation of two fragments of about 1 kbp in length, one of which Fig. l0b contains part of the SV40 origin of replication sequence and part of the 650 bp heat shock control element, including 400 bp of 5 nontranscribed sequence and 60 bp of the RNA leader sequence.The two 1 kbp fragments were isolated by electrophoresis on preparative gels of 1.2 low melting agarose. Plasmid PRBl Fig. lOc was digested with HindIII, the ends were filled up with DNA polymerase Klenow fragment, and the DNA was further digested with NcoI which has a unique cutting site in this plasmid situated within the SV40 origin of replication sequence see Fig.lOd . The resulting PR81 fragments were ligated with the 1 kbp fragments isolated from plasmid 520 and the ligation mixture was used to transform E. coli C 600. The resulting transformants include plasmids such as PR84 whose predicted structure is shown in Fig. 10e. A more detailed structure for PR34 including partial DNA sequence predictions are presented in Fig. lOf in the form of a single strand representation of plasmid sequences. As shown in Fig. lOf, PR84 comprises from the 5 end, successively, a 250 bp origin of replication segment of SV40 origin, a 346 bp sequence from pBR322, a 400 bp Drosophila 5 nontranscribed sequence p51 p132E3, see Karch et al. 1981 J. Mol. Biol 148 219 230 originally included in the 650 bp heat shock control sequence, a Drosophila 65 bp hsp gene RNA leader fragment base pairs numbered 1 to 65 , a linker segment of 8 basepairs, a haemagglutinin HA RNA leader segment base pairs numbered 1 32 , a HA signal peptide coding segment base pairs numbered 33 83 and a HA protein coding segment bp 84 onwards . The rather complicated construction scheme chosen for plasmidPR84 was checked by restriction digest analysis of the parent plasmids used, the results of which are shown in Fig. ila. The following restriction sites are predicted for the plasmids PRSl 1 NcoI site, 1 HindIII site, the NcoI site in the SV 40 origin of replication sequence. PR 8 ms 34 1 NcoI site, no site in HA gene. 520 1 Ncol site, several PvuII sites in the lac operon a pair of 1 kbp NcoI PvuII fragments, one of them containing the origin hsp 70 prompter fragment. That these predictions fit with the structures of these plasmids illustrated in the construction schemes is shown by the results of the restriction analyses indicated by the gel patterns in Fig. Ila. The two 1 kbp NcoI PvuII fragments are clearly seen and are in dictated in Fig. ll lane 17 by an arrow. Further characterization of PR84 like plasmids was performed using the Grunstein colony hybridization procedure described before see Fig. 4 . In this case, a nick translated probe approximately 108 cpm g of the Sau3A XhoI fragment from plasmid p51 containing the Drosophila hsp control element was employed, and the results are presented in Fig. llb A number of the positive colonies identified in this way were grown up for the preparation of sma11 emounts of each plasmid as desoribed previously, and a series of restriction analyses were perform ed. The gel patterns so obtained are shown in Figs llc and d.The expected pattern of digestion products formed by XhoI digestion as predicted From the structure of PR84 presented in Fig. 10 are one ShoI fragment of 405 bps and one very large fragment, Fig. llc shows the results of this analysis and the plasmid 5 clones in lanes 2, 4, 6 and 8 have the expected pattern as corpared to the HinfI standard digest pattern of pSVod. These plasmids were selected for further analysis where they were digested either with XhoI NcoI or withXcoI NcoI. The results obtained are shown in Fig. lld. The expect ed sizes of fragments produced from the map shown in Fig. 10 would be one fragment of about 700 bp and one of 405 bp with XhoI NcoI and one large fragment, and with XbaI NcoI, one fragment of 650 bp and one large fragment.Fig. lld shows the validity of these predictions for all of the plasmids analyzed. DNA from plasmid PR84 was used to transfect eukaryotic cells.As suitable cells for bringing about such experiments COS cells Gluzman 1981 Cell 23 175 182 were selected because of their usefulness for the rapid expression analysis of gene constructions Gething and Sambrook 1981 Nature 293 620 625 . After transfection, the cells were incubated at either 370C or 420C in the presence of labelled methionine 35S . Then the contents of the cells were analyzed by immune precipitation using rabbit antiserum raised againstPR 8 ms influenza virus a gift of Dr. J. Skehel, M.R.C. Laboratories, Mill Hill, London U.R. . The complexes purified on protein A Sepharose were finally subjected to electrophoresis on polyacry lamide gels and identified by fluorography.The analytical results indicated the presence in good yield of a protein Mr 75,000 indistinguishable from authentic glycosylated haemagglutinin Elder et al. 1979 Virology 95 343 350 . Synthesis of haemagglutinin occurred both at 370C and to a lesser extent at 420C. No such specific polypeptides were observed using any other transfecting DNA than of PR84 nor in immune complexes other than with anti PR 8 antisera. The E details of the construction and expression of the eukaryotic expression vectors are now set forth. A. Cbnstruction of plasmid 520 Fifty pg of plasmid pRV15 were digested first with 50 units of SmaI for 2 hours at 370C and subsequently with 50 units of SalI for 2 hours at 370C buffers as recommended by New England Biolabs . The digest was then electrcphoresed on a 0.9 agarose gel. The region containing the 7 kb lac fragment was cut out of the gel, and the DNA was recovered by electroelution 200 v, 6 hours . DNA in the eluate was collected by ethanol precipitation. The fragment was then dried and resuspended in 200 of TE buffer, phenol and ether extracted twice, and then passed through a small Sepha dex G75 column as described before. The DNA in the column eluate was collected by ethanol precipitation. Ten jug of plasmid pSVod were digested with 10 units of BamHI for 2 hours at 370C. The e digested DNA was purified by 3 phenol extractions, 3 ether extractions and 2 ethanol precipitations. The dried D5k was then resuspended in TE buffer and incubated with several units of DNA polymerase Kienow fragment in the presence of 0.5 mM deoxya denosine triphosphate dATP , deoxycytidine triphosphate dCTP , deoxyguanosine triphosphate dGTP and deoxythymidine triphosphate dTTP at 6 70C for one hour. The DNA was purified again by repeated phenol and ether extraction and ethanol precipitation. The purifiedDNA was then digested with 10 units of Sail for 2 hours at 370C. The digestion products were purified again as described above.The e 7 kbp pRV15 SmaI SalI fragment and the above described pSVod fragments were then incubated overnight in a molar ratio of 10 1 with an excess of T4 DNA ligase at 140C. This ligation mixture was used to transform E. coli MC 1061. SmaI and BamHI were included in the ligation mixture. Transformants were plated on IB plates containing 10 g ml aweicillin and 40 yug ml Xgal. Blue transformants were isolated. DEN from such recombinants was prepared as described by Schedl et al. 1978 Cell 14 921 929. Recombinants were identified by restriction analyses. B. Construction of plasmid PR31 Fifty pg of plasmid AgPR 8 34 Clone No. 4.76 obtained from Prof. G. Brcwnlee, University of Oxford, U.K. were digested with 30 units of HindIII and BamHI for 2 hours at 370C. The digest was electropho resed on a O.858 agarose gel. The HindIII BamHI haemagglutinin gene fragment was purified by electroelution as described above. Aliquots of this DNA were nick translated to serve as probes for identifying PR31 and similar recombinants by Grunstein colony hybridization specivic radioactivity of probes about 108 cpm g . Fifty fug of A PR 8 34 4.76 were digested with 50 units of BamHI for 2 hours at 370C. The DNA was phenol extracted three times, ether extracted and ethanol precipitated. The dried DNA was then re suspended in 25 l of TE buffer and incubated with several units of DNA polymerase Klenow fragment and 0.3 rrM of all four deoxynucleotide triphosphates in a total volume of 100 p1 for 90 minutes at 6 70C. The DNA was then purified by phenol and ether extractions as above. Following ethanol precipitation, the DNA was dried and then resuspended in 25 1 of TE buffer. It was digested subsequently with 54 units of HindIII for one hour at 370C. The digested DNA was repurified by phenol extraction. Ten ug of plasmid pSVod Mellon et al. 1981 Cell 27 279 288 were digested with 30 units of SalI for one hour at 370C. The DNk was purified by phenol extraction as above and was then incubated with several units of Klenow fragment in 100 pl in the presence of 0.5 mM, dSrP, dCTP, dGTP and dTTP for one hour at 370C. Following repurification by phenol extraction, the DNA was digested further with 25 units of HindIII for one hour at 370C and repurified by phenol extraction. An A PR 8 34 digest 1.5 g and a pSVod digest 2 Zg were incubated overnight at 140C with an excess of T4 DNA ligase in a total volume of 30 l. Several units of BamHI and SalI were included in the ligation mixture. The e ligated DNA was purified by phenol extraction as described above. It was then digested with 6 units of BamRI for one hour at 370C, followed by incubation with 12 units of Sail for one hour at 370C. Aliquots of this reaction mixture were used to transform CaCl2 treated E. coli C 600. Transformants were isolated on LB plates containing 10 g m. ampicillin. About 350 clones were purified and analyzed by colony hybridization using the nick translated HindIII BamHI haemagglutinin gene fragment described above as probe.Approximately 80 90 of the clones were found to contain a haemagglutinin gene insert. Seven of the positive clones were grown up, and DNA was prepared from them. These DNAs were compared with the original clones A PR 8 34 and pSVod by restriction digestion and electrcphoresis on a 0.98 agarose gel. Predicted fragment sizes Digestion with EcoRI Recombinants 3.2 kbp, 1.5 1.6 kbp A PR 8 34 3.8 kbp, 1.3 kbp pSVod 3.3 kbp Digestion with HindIII and SalI Recombinants 4.5 kbp A PR 8 34 3.3 k5p, 1.7 1.8 kbp pSVod 2.7 kbp, 0.6 0.7 kbp Five out of the seven recombinants that were analyzed showed the expected digestion pattern. PR81 is one of these five clones. PR81 DNk was prepared as described by the method of Schedl et al. 1978 Cell 14 921 929 and was analyzed further by restriction enzyme digestion. C. Construction of plasmid PR84 p520 DNA 75 Xg was digested with 25 units of PvuII for one hour at 370C and subsequently with 20 units of NcoI for 2 hours at 370C. The digest was electrophoresed on a 1.2 low melting agarose gel containing 0.05 g ml EtBr. The region including the two 1 kbpWcoI PvuII fragments one of them is a hsp 70 gene promoter fragment was cut out of the gel. The agarose piece and 10 volumes of TE buf fer were heated to 650C for 10 minutes, and the DNA was extracted subsequently with an equal volume of phenol. After one additional phenol chloroform and 3 ether extractions, the DNA was precipitated two times with ethanol. PRS1 DNA 10 g was digested with 15 units of HindIII for one hour at 370C. Following purification by phenol extraction the DNA was incubated with several units of Rlenow fragment and 0.5 mM of all four deoxyribonucleotide triphosphates for one hour at 6 7 C. After repurification, the DNA was digested With 10 units of NcoI for 90 minutes at 370C and was repurified by ph iL extraction. Aliquots 0.25 g of the p520 1 k McoI PvuII fragments and aliquots 0.05 0.1 g of the PR81 fragmen were incubated overnight at 14 C with an excess of T4 DNA ligase This mixture was used to transform E. coli C 600. About 600 ampicill In resistant transformants were isolated and examined by Grunstein DIOny hybridization using a nick translated p51 Sau3AChoI promoter fragment as a probe approximately 108 cp pg Fig. llb . Ten positives were grown up and DNA was prepared from these clones. Aliquots of their DNAs were digested with XhoI and were analyzed on 5 polyacrylamide gels. The presence and sizes of XhoI fragments provided good evidence for the presence in the recombinant DNAs of both the hsp 70 kdal gene promoter and the haemagglutinin gene. Four out of the 10 clones tested showed the expected restriction pattern two fragments one large, about 5 kbp, and one of 400 bp Figs llc and d . Additional digestions with NcoI XhoI and XbaI NcoI confirmed our initial analysis. PRS4 is one of the four plasmids which have the correct structure. D. Expression experiments using plasmid PR84 COS I cells Gluzman 1981 Cell 23 175 182 were maintained in Dulbecco s MEM with 10 new born calf serum and were subcultured one day prior to transfection to give cultures containing about 106 cells per 6 cm diameter petri dish after a further one day of culture. Transfection of cells was performed as follows The medium was removed from cell cultures at room terrperature and the cultures were washed twice with 5 ml of phosphate buffered saline PBS . Subsequently one ml of the following preparations per dish were added. 1. 1 ml of Dulbecco s MEM with no serum, but containing DEAEDextran 500 tug and chloroquine 200 g . 2. As 1, but containing in addition, 10 jig of plasmid 520 DNA. 3. As 1, but containing in addition, 10 g of plasmid PR84 DNA. Transfections were performed on 5 dishes of cells for each of the three conditions stated above. Cell cultures were maintained at room temperature for 30 minutes with occasional gentle rocking to distribute the transfection solution over the entire cell culture. After removal of the trans fection solution, 5 ml per dish of Dulbecco s MEM with 10 serum was added. Cell cultures were further incubated at 370C for 36 hours.Half of the two cultures were incubated at 420C for the last 4 hours of this period. After this period, the medium was removed and cultures were washed once with a Dulbecco s MEM containing 3 mg l ofL methionine and 1 new born calf serum. Cell cultures were labelled by addition of 1 ml of this same low methionine medium per dish but in the presence of 35S Methionine 50 Ci ml. 968 Ci mmal . Cultures were incubated for one hour at either 370C or 420C in the case of the heat shock samples. After this period of labelling, the 35S Methionine medium was removed, the cultures washed three times with ice cold PBS, and cells were scraped off in 1 ml of NET buffer 0.1 M NaCl, 0.01 M Tris HCl pH 7.8, 0.001 M EDTA, 0.5 NP 40 .After vigorous pipetting to break down the outer cell membranes, nuclei and cellular debris were removed by centrifugation at 5000 x g for 5 minutes at 40C. The cytoplasmic supernatants were removed and aliquots were taken for immune precipitation as follows Aliquots 500 l of cytoplasmic extracts were kept on ice, 5 11 of specific antibodies were added and the solutions were mixed gently at 40C for two hours. The antibodies used were the following Nor mal rabbit antiserum, antiserum raised against purified E. coli p galactosidase and antiserum raised against PRS influenza virus obtained from Dr. J. Shekel, MRC Laboratories, Mill Hill, London,U.K.Anti HA antisera was preadsorbed for 30 minutes in NET buffer using cytoplasmic extracts of unlabelled COS cells prior to use for immone precipitation of labelled samples. Antigen antibody carplexes were isolated by the addition of 50 1 of a 50 suspension of protein A Sepharose Pharmacia , continuing the shaking at 40C for one hour, and recovering the bound complexes by centrifugation at 15,000 x g for one minute. The complexes on protein A Sepharosewere washed and centrifuged four times using 1 ml of NET and three times using 1 ml of RIPA buffer 0.15 M NaCl, 0.05M Tris HCl pH 7.5, 0.02 M BOTA, 1 M Urrea, 1 Triton x 100, 1 sodium deoxycholate . The final Sepharose pellets were boiled for 3 minutes in 100 ul of sample buffer Lacmmli 1970 Nature 227 680 685 .After centrifugation at 15,000 x g fcr 1 minute, the samples were subjeted to electrophoresis on 10 polyacrylamide gels as indicated by Lammli 1970 supra. In the case of cell cultures transfected with plasmid PR84 DNA and in which the labelled cytoplasmic extracts were immune precipi tated with antiserum raised against PR8 influenza virus, the gel analysis by fluorography Laskey and Mills 1975 Eur. J. Biochem. 56 335 341 clearly indicated the presence of a protein which is indistinguishable in size Mr 75,000 from authentic glycosylated haemagglutinin Elder et al. 1979 Virology 95 343 350 . Some possible processing to give polypeptides of Mr approximately 42,000 and 36,000 was observed in some of these experiments. Synthesis of haemagglutinin occurred both when labelling was performed at 370C and to a lesser extent at 420C.No such specific polypeptides were observed using any other transfecting DNA than PR84, nor in immune precipitations other than with anti PR8 antisera. In a second series of similar transfection experiments, COS cells transfected with PFE4 were subjected to a period of heat shock of five hours at 420C and were subsequently labelled with 35Sthio nine as described above but at 370C for 14 hrs. Subsequent i rtrrLlno precipitations with anti HA antisera indicated a larger quantity ofHA product than in non heat treated samples labelled under similar conditions. These results suggest that the Drosophila heat shock prorrotor functions better at the heat shock terrperature in monkey cells than at the 370C control temperature. Some activity of this promotor at 370C is perhaps to be expected as we have previously noted the use of 370C as the heat shock temperature for Drosophila cells. The im proved synthesis of HA when cells are returned to 370C after a period of heat shock suggests that the translational control of theDrosophila heat shock promotor fragment, in particular the Drosophi la ribosorne hinding site is non functional or poorly functional in monkey cells at least at the higher tempetature.Thus it is possible that although promotion of transcription occurs at 420C using the Drosophila DNA sequences, ribosome binding and initiation of translation may occur using the HA ribosome hinding site, optimally at 370C. According to the present invention, expression vectors are provided which yield efficient transformation of eukaryotic hosts and high levels of expression within the host. In particular, the transcriptional translational control region of a 70 kdal heat shock protein has been isolated and joined to a replication system derived fran a simian virus. A human influenza virus haemagglutinin gene was inserted into the resulting vector under the control of the said control region, and the resulting plasmid used to transfect a mammalian cell culture and apparently glycosylated haemagglutinin was produced. Construction of plasmid p629 A further Drosophila control element HA gene construction was made to extend the utility of plasmid constructions such as PRS4.The HA gene being a viral gene is rather specialized as a model for any eucaryotic gene with its 3 end signals, and its RNA leader sequence, hence the PRS4 construction contains both an HA RNA leader and a Drosophila RNA leader. As a more general example of the construction of a gene expression unit using heat shock control elements plasmid p629 was constructed. This plasmid contained in addition to the Drosophila heatshock promotor the entire heat shock RNA leader region, but was attached to a HA gene lacking the first 40 amino acid codons. For convenience, a plasmid p622b a gift and personal communication of R. Voellmy was employed this plasmid contains a 450 bp XhoI BglII it frag ment carrying the Drosophila hsp 70 promotor and RA leader sequence represented by the XhoI Sau3A fragment of plasmid pRV15 described ed earlier . p622b is a derivative of the CCE cell vector pSVod. The sequence around the BglII site is given below EMI36.1 Experimental construction of p629 IS ug of p622b were digested with 12 units per 15 min of II, and 15 ug of PR 8 ms 34 with 15 units of XhoI. Digests were for 2 hrs at 370C. The DNAs were then extracted three times with phenol, three times with ether and prcopitated two times with EtOH. BothDNAs were resuspended in 20 ul of TE ffer and ends were filled in a reaction involving several units of 3Nk polymerase KIenow frag ment and 0.5 mM of each of the four dNTPS total volume 100 ul reaction at 60C for 2 hrs . Both DNAs were again phenol extracted and purified as above and again dissolved in 20 ul of TE buffer. They were then digested individually with 15 units of BamHI total volume 100 ul 370C for 2 hrs . Following phenol extraction the DNAs were collected by EtOH precipitation and dissolved in 15 ul of TE buffer.Different sets of ligations containing 1 ul of 622b digest and 1 5 ul of PR 8 ms 34 digest in a total volume of 25 ul were carried out enzyme 1 2 ul, T4 DWA ligase, New England Biolabs incubated overnight at 14 . The ligase was inactivated at 650C for 10 min and the samples were then digested with several units of ClaI and SalI successively, using the conditions suggested by New England Biolabs for 2 hrs at 370C in volumes of 50 ul. The DNA mixtures were then used to transform E. coli MCl061. About 300 individual transformants were characterized further using the Grunstein colony hybridization assay. Two separate sets of hybridizations were carried out probes used were a the 450 bp Xho BqlII fragment from p622b and b the 1775 bp HindIII BamHI gene fragment from PRB.4.76. Fragments had been separated in low melting agarose gels and were subsequently eluted from these gels. They were then labelled by nick translation 107 108 cpmzug 106 cp filter . 8 colonies that hybridized to both probes were selected, see Figs 12c and d and DNk was prepared from them using the mini DNk preparation procedure described previously. The DNAs were digested withXhoI or with num lII EcoRI and the digests were analyzed on 5 polyacrylamide gels. XhoI was expected to yield two fragments with sizes of 450bp and of about 5kb. The BglII RI digest should give fragments of 1130 bps, 1100 bps and of about 2.6 kb. Three of the eight clones tested had the correct restriction patterns see Figs 12a and b . Expression experiments using plasmid p629 Transfection of COS I cells was performed in a manner analogous to that described for the expression experiments using plasmid PRS4. Immune precipitations performed as described previously, precipitated one major and a number of minor polypeptides of M.Wt around 75,000 daltons. When 35S MeEhionine labelling was performed in the presence of 1 ug ml tunicaRycin, one major band Mr 75,000 alone was observ edr suggesting that the minor bands were the result of glycosylation. Synthesis of HA protein was greater during a labelling period of 14 hrs at 370C following a period of heat shock than in parallel labelling periods in the absence of heat shock. Although the foregoing invention has been described in some detail, by way of illustration and example for purposes of clarity and understanding, it will be obvious that certain changes and modifications may be practiced within the scope of the appended claims. The cultures of E. Coli containing plasmids pRV 15, PR 84 and 629, respectively, cited in the present disclosure have been registered on February 2, 1984 under Nos. 39597, 39596, 39598 at the AMERI CAN E CULTURE COLLECTION, 12301 Parklawn Drive, Rockville, Maryland 20852 U.S.A.