# In vivo labelling of polynucleotide sequences

## Claims
Verfahren zur

## Description
This invention relates to In the description, the following terms are employed The analysis and detection of minute quantities of substances in biological and non biological samples has become a routine practice in clinical and analytical laboratories. These detection techniques can be divided into two major classes 1 those based on ligand receptor interactions e.g., immunoassay based techniques , and 2 those based on nucleic acid hybridization polynucleotide sequence based techniques . Immunoassay based techniques are characterized by a sequence of steps comprising the non covalent binding of an antibody and antigen complementary to it. See for example, T. Chard, Polynucleotide sequence based detection techniques are characterized by a sequence of steps comprising the non covalent binding of a labelled polynucleotide sequence or probe to a complementary sequence of the analyte under hybridization conditions in accordance with the Watson Crick base pairing of adenine A and thymidine T , and guanine G and cytidine C , and the detection of that hybridization. M. Grunstein and D. S. Hogness, Colony Hybridization A Method For The Isolation Of Cloned DNAs That Contain A Specific Gene , In a generalized sense, the non covalent binding of a labelled sequence or probe to a complementary sequence of an analyte is the primary recognition event of polynucleotide sequence based detection techniques. This binding event is brought about by a precise molecular alignment and interaction of complementary nucleotides of the probe and analyte. It is energetically favored by the release of non covalent bonding free energy, e.g., hydrogen bonding, stacking free energy and the like. In addition to the primary recognition event, it is also necessary to detect when binding takes place between the labelled polynucleotide sequence and the complementary sequence of the analyte. This detection is effected through a signalling step or event. A signalling step or event allows detection in some quantitative or qualitative manner, e.g., a human or instrument detection system, of the occurrence of the primary recognition event. The primary recognition event and the signalling event of polynucleotide sequence based detection techniques may be coupled either directly or indirectly, proportionately or inversely proportionately. Thus, in such systems as nucleic acid hybridizations with sufficient quantities of radiolabeled probes, the amount of radio activity is usually directly proportional to the amount of analyte present. Inversely proportional techniques include, for example, competitive immuno assays, wherein the amount of detected signal decreases with the greater amount of analyte that is present in the sample. Amplification techniques are also employed for enhancing detection wherein the signalling event is related to the primary recognition event in a ratio greater than 1 1. For example, the signalling component of the assay may be present in a ratio of 10 1 to each recognition component, thereby providing a 10 fold increase in sensitivity. A wide variety of signalling events may be employed to detect the occurrence of the primary recognition event. The signalling event chosen depends on the particular signal that characterizes the label or signalling moiety of the polynucleotide sequence employed in the primary recognition event. Although the label itself, without further treatment to attach or to bind to it a signalling moiety or a combination of bridging moiety and signalling moiety, may be detectable, it is more usual either to attach covalently or to bind non covalently to the label a signalling moiety or a combination of bridging moiety and signalling moiety that is itself detectable or that becomes detectable after further modification. It should, of course, be understood that the combination of bridging moiety and signalling moiety, described above, may be constructed before attachment or binding to the label, or it may be sequentially attached or bound to the label. For example, the bridging moiety may be first bound or attached to the label and then the signalling moiety combined with that bridging moiety. In addition, it should be understood that several bridging moieties and or signalling moieties may be employed together in any one combination of bridging moiety and signalling moiety. Examples of the covalent attachment of a signalling moiety or a combination of bridging moiety and signalling moiety to a label are the chemical modification of the label with signalling moieties, such as radioactive moieties, fluorescent moieties or other moieties that themselves provide signals to available detection means or the chemical modification of the label with at least one combination of bridging moiety and signalling moiety to provide that signal. Examples of the non covalent binding of a signalling moiety or a combination of bridging moiety and signalling moiety to a label are the non covalent binding to the label of a signalling moiety that itself can be detected by appropriate means, or the non covalent binding to the label of a combination of bridging moiety and signalling moiety to provide a signal that may be detected by one of those means. For example, the label of the polynucleotide sequence may be non covalently bound to an antibody, a fluorescent moiety or another moiety which is detectable by appropriate means. Alternatively, the label could be bound to a bridging moiety, e.g., a lectin, and then bound through the lectin, or bridging moiety, to another moiety that is detectable by appropriate means. There are a wide variety of signalling moieties and bridging moieties that may be employed for covalent attachment or non covalent binding to the labels of polynucleotide sequences useful as probes in analyte detection systems. They include both a wide variety of radioactive and non radioactive signalling moieties and a wide variety of non radioactive bridging moieties. All that is required is that the signalling moiety provide a signal that may be detected by appropriate means and that the bridging moiety, if any, be characterized by the ability to attach covalently or to bind non covalently to the label and also the ability to combine with a signalling moiety. Radioactive signalling moieties and combinations of various bridging moieties and radio active signalling moieties are characterized by one or more radioisotopes such as ³²P, ¹³¹I, ¹⁴C, ³H, ⁶⁰Co, ⁵⁹Ni, ⁶³Ni and the like. Preferably, the isotope employed emits β or γ radiation and has a long half life. Detection of the radioactive signal is then, most usually, accomplished by means of a radioactivity detector, such as exposure to a film. Non radioactive signalling moieties and combinations of bridging moieties and non radioactive signalling moieties are being increasingly used both in research and clinical settings. Because these signalling and bridging moieties do not involve radio activity, the techniques and labelled probes using them are safer, cleaner, generally more stable when stored, and consequently cheaper to use. Detection sensitivities of the non radioactive signalling moieties also are as high or higher than radio labelling techniques. Among the preferred non radioactive signalling moieties or combinations of bridging signalling moieties useful with non radioactive labels are those based on the biotin avidin binding system. P. R. Langer et al., Enzymatic Synthesis Of Biotin Labeled Polynucleotides Novel Nucleic Acid Affinity Probes , The use of non radioactively labelled polynucleotides in detection systems has been somewhat disadvantaged by the expense of the attachement of a label that does not interfere with hybridization to complementary polynucleotide sequences. First, the chemical reaction conditions that might be useful for modification of a polynucleotide polymer to add to it a label are often too vigorous to be sufficiently selective for a particular nucleotide. More importantly, chemical labelling of polynucleotide sequences often interferes with hybridization because the label interferes with the hydrogen bonding necessary for hybridization. For example, dicarbonyl reagents, such as kethoxal or glyoxal, react with guanine residues Shapiro et al., Accordingly, in order to label a polynucleotide sequence for use as a probe, a labelled monomeric nucleotide must be synthesized and then incorporated into a polynucleotide sequence. Various methods are available to label an individual nucleotide in such a way that the label does not interfere with hybridization. Various methods, both chemical and enzymatic, are also available to attach those labelled monomeric nucleotides to a polynucleotide probe. For example, a labelled nucleotide, such as 2 deoxyuridine 5 triphosphate 5 allylamine biotin may be substituted in DNA probes by nick translation P. R. Langer et al., Enzymatic Synthesis Of Biotin Labeled Polynucleotides Novel Nucleic Acid Affinity Probes , There are, however, production limitations with these processes. For example, it is necessary to synthesize the labelled monomeric nucleotides prior to incorporating them into the polynucleotide probes. This synthesis may sometimes involve expensive chemical processes. The coupling of the labelled monomeric nucleotides into a polynucleotide is also expensive. For example, the enzymes employed in enzymatic coupling are costly. Related limitations include the difficulties and cost in scale up of such processes to commercial levels. As a result, these processes currently produce non radioactively labelled polynucleotides that are more costly than are desired. N.J. Casna and D.A. Stub in Gene The present invention solves the production limitations referred to above by providing The processes for preparing the labelled DNA sequences of this invention generally involve, the The present invention relates to a process for More specifically, the embodiment of the process for In one preferred embodiment of this invention a heterologous DNA sequence that is desired to be labelled is inserted into the T4 bacteriophage genome which normally contains hydroxymethylated and glucosylated DNA . The host T4 phage is grown under conditions wherein it produces hydroxymethylated and glucosylated DNA. The T4 phage containing the inserted probe DNA sequence is harvested, and the probe DNA sequence is preferably excised for use as a hybridization probe in any of the well known The presence of the probe hybridized to the analyte is then detected, for example, by using a combination of bridging moiety and signalling moiety. For example, Concanavalin A Con A is bound to the glucosylated probe DNA sequence and there acts as a bridge to a naturally glycosylated enzyme. The enzyme, e.g., horseradish peroxidase, upon contact to the proper substrate, e.g., H₂O₂ and diaminobenzidine, produces colorimetric products which can be detected. In addition, other lectin detection systems or antibody or other detection systems utilizing well known processes may also be used to detect the hybridized DNA sequence. Figure 1 is a schematic outline of one embodiment of a process for attaching an In order that the invention herein described may be more fully understood, the following detailed description is set forth. Any of a large number of polynucleotide sequences may be employed in the processes of this invention to be labelled and to be used in the detection of analytes. Included, for example, are polynucleotide sequences that characterize various viral, viroid, fungal, parasitic or bacterial infections, genetic disorders or other sequences in analytes that it is desired to detect. They may be of synthetic, semi synthetic or natural origin. Any of a large number of available sources may be used to produce The polynucleotide sequence that is desired to be labelled and the source of the label may be combined in the host in a large variety of ways. The source of the label may also be present in the host in a variety of ways. For example, when the source of the label is a DNA sequence that expresses a product that labels the desired polynucleotide sequence on its replication in a host characterized by that polynucleotide sequence, that DNA sequence may be present in the host because it was originally part of the natural genome of the host or because it was inserted into that genome. Alternatively, the DNA sequence that is the source of the label may be part of a cloning vehicle, phage DNA or other DNA sequence used to transfect the host for replication therein. In one preferred embodiment of this invention, the polynucleotide sequence to be labelled is present in the same cloning vehicle or phage DNA as the DNA sequence that is the source of the label. Alternatively, the two DNA sequences may be separately present in the host. Most preferably, the polynucleotide sequence to be labelled is cloned into the DNA sequence that is the source of the label. In any event, on culturing the host the desired DNA sequence is labelled during or after replication as a result of the product expressed by the DNA sequence that is the source of the label. Hosts useful in the processes of this invention may be selected from a wide variety of known organisms. They include, for example, various microorganisms, such as The biologically produced probe DNA sequences of this invention have many practical uses. One use is in the detection of analytes. The analyte to be detected can be present in any biological or non biological sample, such as clinical samples, for example, blood, urine, feces, saliva, pus, semen, serum, other tissues, fermentation broths, culture media, and the like. If necessary, the analyte may be pre extracted or purified by known methods to concentrate its nucleic acids. Such nucleic acid concentration procedures include, for example, phenol extraction, treatment with chloroform isoamyl alcohol or chloroform octanol, column chromatography e.g., Sephadex, hydroxyl apatite , and CsCl equilibrium centrifugation The analyte, together with contaminating materials, if present, may be tested in the mixture, as purified, or the analyte may be immobilized before analysis. There are also many applications for the detection methods and kits of this invention. Any analyte desired to be detected and analyzed in any sample can be subject to the methods and kits of the invention. For example, the methods and kits may be employed to detect and to identify viral and bacterial DNA sequences, e.g., the detection of herpes virus. The methods and kits of this invention can also be utilized to diagnose human genetic disorders by preparing a probe complementary to a DNA sequence which is associated with the genetic disorder and detecting the presence or absence of any primary recognition events. Among these genetic disorders, for example, is thalassemia. The diagnosis of thalassemia can be made by hybridization of probe polynucleotide sequences to genomic DNA. Another use for the methods and kits of this invention is in chromosomal karyotyping, which comprises using a series of labelled polynucleotide sequences, corresponding to a series of defined DNA sequences uniquely located on each of the chromosomes, and then detecting primary recognition events thereon. For testing, the composition suspected of containing the analyte is incubated with the labelled probe polynucleotide sequence for a time and under conditions sufficient to allow hybridization between the polynucleotide sequence of the analyte and the recognizing polynucleotide sequence on the probe. These conditions will vary depending on the nature and amount of the analyte and of the probe D. E. Kennell, Principles And Practices Of Nucleic Acid Hybridization , A wide variety of signalling events may be employed to detect the occurrence of the primary recognition event the hybridization of the labelled DNA sequence to a complementary sequence in the analyte. The particular signalling event chosen depends on the particular signal that characterizes the label or modified label of the polynucleotide sequence. For example, the label carried by the polynucleotide sequence may, without further treatment to attach or to bind to it a signalling moiety or at least one combination of bridging moiety and signalling moiety, be detectable. However, it is more usual, either to attach covalently or to bind non covalently to the label a signalling moiety, or at least one combination of bridging moiety and signalling moiety that is itself detectable or that becomes detectable after further modification Examples of signalling and bridging moieties that may be covalently attached to the labels of polynucleotide sequences include radioactive compounds, fluorescent compounds, fluorescein, rhodamine, dansyl, magnetic compounds, chelating agents and other signalling, and bridging moieties which may be covalently attached to those labels. Examples of signalling and bridging moieties that may be non covalently bound to the labels of polynucleotide sequences include polypeptides, proteins, lectins, Concanavalin A, enzymes, alkaline phosphatases, acid phosphatases, antigens, antibodies, polypeptides having strepatvidin groups attached thereto, β galactosidase, glucose oxidase, horseradish peroxidase, chelating agents and other signalling and bridging moieties that may be non covalently bound to those labels. For example, an enzyme might be non covalently bound as a signalling moiety to the label on the polynucleotide probe sequence. Then substrate would be added to obtain color development fluorescence, radioactivity or chelation detection systems may also be used . Alternatively, if the moiety bound to the label were a biotin moiety, for example, then a biotin binding molecule such as avidin, streptavidin or anti biotin antibody, would then be added thereto. The biotin binding molecule would then be conjugated to an enzyme, a fluorescent com pound, an electron dense compound, or an insoluble solid phase, and detection carried out by appropriate means. In order that this invention may be better understood, the following examples are set forth. These examples are for purposes of illustration only and this invention should not be considered to be limited by any recitation used therein. Using the techniques of recombinant DNA technology, a probe DNA sequence can be inserted into a T4 bacteriophage for labelling with glucosyl residues. T4 DNA is naturally glucosylated. Because most of the commonly used restriction endonucleases do not hydrolyze glucosylated DNA, it may be necessary to remove the glucose residues from the T4 DNA prior to endonucleolytic cleavage and insertion of the probe DNA. Hydroxymethyl cytidine glucosylase, a phage induced enzyme, removes glucose moieties from glucosylated DNA and transfers them to UDP in its reverse reaction J. Josse et al., Once the T4 DNA has been deglucosylated as set forth above, the probe DNA sequence is inserted into the deglucosylated T4 genome. Although insertion of the probe DNA sequence into the deglucosylated T4 genome may be effected in a variety of known ways, it is preferable to insert it in such a manner that it may be removed from the T4 DNA subsequent to labelling. Because in T4 phages only the C residues are glucosylated, the restriction endonuclease A general protocol for the preparation and insertion of a DNA probe into T4 DNA and its removal after labelling is described below and illustrated in Figure 1 Con A, a bridging moiety, binds to both glucosylated DNA and to glycosylated proteins a signalling moiety . At pH 5 and temperatures below room temperature, Con A is a dimer having two glycosyl binding sites. At physiological pH and room temperature, or slightly above 37 C , native Con A is a tetramer and contains four binding sites. At alkaline pH 8.5 and above and higher temperatures, Con A dissociates into inactive subunits. Manganese or magnesium and calcium are required for Con A to bind glycosyl residues. Stock solutions of Con A are kept in glass tubes at a concentration of 1 5 mg ml in TCMN buffer 5 mM Tris HCl, pH 7.0, 1 mM CaCl₂, 1 mM MnCl₂, 1 M NaCl because Con A adheres to plastic surfaces and aggregates at concentrations greater than 5 mg ml. Con A can be stored at 4 C for about 3 months. Nitrocellulose paper containing glucosylated T4 DNA dots was blocked overnight at 42 C in a humidity chamber in a buffer containing 2 acidified bovine serum albumin BSA , 1 X TCMN and 0.1 V V Triton X 100. The nitrocellulose filters were then rinsed three times, for 5 minutes each time, with a buffer containing 1 BSA and 1 X TCMN. The nitrocellulose filters were then incubated in a Con A solution consisting of 100 200 µg Con A ml in 0.1 BSA and 1 X TCMN. The solution was applied at 0.018 ml cm² of nitrocellulose filter or 0.2 ml cm² of, Whatman 3 MM paper, and was incubated for 1 h at 37 C. Following incubation, the filters were rinsed 3 4 times, for 5 min each time, in a 0.1 BSA and 1 X TCMN buffer. The filters were incubated in a solution of 0.1 BSA and 1 X TCMN containing 2 10 units of enzyme. The solutions were applied at 0.018 ml cm² of nitrocellulose and 0.2 ml cm² of Whatman 3 MM filters. Incubation was at 37 C for 1 h. The filters were then rinsed 3 times, for 5 min each time, in NBTT buffer 0.5 M NaCl, 0.1 BSA, 0.05 Tween 20, 10 mM Tris HCl, pH 7.2 and 2 times, for 5 min each time, in NCBT buffer 0.3 M NaCl, 0.03 M Na citrate, 0.1 BSA, and 0.05 Tween 20 . The glycosylated enzyme Con A derivatized DNA sequences were detected as follows This protocol for Con A detection was found to be non optimal. The sequential addition of Con A followed by enzyme was found to give excessive background in the assay detection system. The procedure was also relatively time consuming and the sensitivity of detection was lower than desired. Therefore a second method of contacting Con A, glucosylated DNA and enzyme was employed. Con A and glycosylated enzyme were mixed in a 1 1 molar ratio in TCMN buffer and incubated at 37 C for 2 h or at 25 C for 4 6 h or at 4 C overnight to 48 h. At the end of the incubation period the mixture should be clear. If it is not, Con A is present in excess and more enzyme must be added. The blocking of nitrocellulose filters containing T4 glucosylated DNA was done as set forth above. The filters were then rinsed 3 times, for 5 min each time, in TCMN buffer and 1 BSA. The Con A enzyme complex prepared above was then contacted to the filter at a volume of 0.018 ml cm², or in an amount containing about 2 10 units of enzyme. Incubation was at 37 C for 1 h. After incubation the filters were first rinsed 3 times, for 5 min each time, in NBTT and then 2 times, for 5 min each time, in NCBT. The horseradish peroxidase reaction was performed as set forth above. With this procedure it was possible to detect 31.25 picograms of DNA and there was significantly less background than with the previously described detection technique. Similar detection procedures can also be performed utilizing a Con A acid phosphotase complex, a Con A glucose oxidase complex, as well as other similiar Con A enzyme complexes. Lectin antibody and other detection systems may also be used see Ward et al., supra. . In order to aid in the detection of hybridization reactions between glucosylated probe DNA and analytes, the glucosylated probe DNA may be further derivatized with biotin moieties, for which standard detection systems are known. One ml of 1 mg ml T4 DNA in 0.1 M sodium acetate buffer pH 4.3 , was mixed with 0.1 ml of a freshly prepared 1 M NaIO₄ solution. The mixture was incubated for 3 h at room temperature in the dark. After the oxidation reaction was completed, the solution was dialyzed at 4 C, in the dark, against 2 changes of 0.05 M sodium acetate at pH 4.0, 0.1 M NaCl and 2 changes of 0.3 M sodium borate at pH 9.0 9.3, 0.1 M NaCl. The solution was then made 0.4 M in 1,6 diaminohexane from a pH 9.3 stock solution of diamine. The mixture was incubated in the dark for 90 min. The resulting Schiff base was reduced with NaBH₄. NaBH₄, freshly dissolved in water to a concentration of 2 M, was added to the mixture at four 30 min intervals, producing incremental concentrations of NaBH₄ of 0.025 M to 0.1 M. Total incubation time was 3 h. The NaBH₄ was quenched by adjusting the pH to 5.0 5.5 by adding 4 M sodium acetate, pH 4.0. The DNA containing solution was dialyzed 12 h in 0.1 M sodium phosphate, pH 6.7, and then made 40 V V DMF and 20 mM biotin NHS ester. This solution was incubated for 12 h. Excess biotin and biotin NHS ester were removed by filtration through a G50 Sephadex column using 1 x SSC as the eluting buffer. Fractions containing DNA were collected from the column, combined, and dialyzed against a solution containing 0.1 M NaCl, 0.001 M EDTA, pH 7.0, and stored until further use at 20 C. In this example, a label is incorporated A thymine requiring mutant of Hybridization of labelled probe to analyte can be detected by reaction with monoclonal or polyclonal antibodies to BrdUR Gratzner, Monoclonal Antibody To 5 Bromo And 5 Iododeoxyuridine A New Reagent For Detection Of DNA Replication , In order to detect BrdUR labelled probe DNA, as produced in Example III, hybridized to an analyte, BrdUR labelled probe DNA may be further derivatized with biotin. Two hundred µg of triethylammonium salt of DNA containing bromodeoxyuridine residues was dissolved in 2 ml of anhydrous dimethylformamide DMF . To this solution, 0.5 ml of 50 mM thiobiotin in anhydrous DMF was added and the mixture was incubated for 2 h at 60 C under argon gas. The solvent was removed by evaporation under a reduced pressure at 40 C. The residue was dissolved in 0.5 ml 1 x SSC. Undissolved material was removed by centrifugation. Excess biotin was removed from the supernatant by filtration on a G50 Sephadex column utilizing 1 x SSC as the elution buffer. DNA containing fractions were collected from the column, combined, and stored at 70 C for future use. While we have hereinbefore presented a number of embodiments of this invention, it is apparent that our basic construction can be altered to provide other embodiments which utilize the processes and compositions of this invention. Therefore, it will be appreciated that the scope of this invention is to be defined by the claims appended hereto rather than the specific embodiments which have been presented hereinbefore by way of example.