# Plasmid vector for mammalian cells.

## Abstract
Plasmid vectors which possess restriction enzyme cut ting sites at 0.1 kb and 2.35 kb by Hind III, at 0.6 kb and 3.1 kb by Kpn 1, at 3.07 kb by Bam HI and 1.45 kb by Sal I, based upon the Eco RI cutting site at 0 kb.

## Claims
Claims 1. A plasmid vector which posesses restriction enzyme cutting sites at O.lOkb and 2.35kb by Hind III. 0.60kb and 3.10kb by kpn 1, 3.07kb by Bam H1, and, 1.45kb by Sal 1, the said cutting sites being based upon an Eco R1 cutting site at O.OOkb. 2. A plasmid vector LF1 with a molecular size of 5.3 kb as claimed in claim 1. 3. A plasmid vector LFII with a molecular size of 5.5 kb as claimed in claim 1. 4. A composite plasmid pdLFI as shown by the restriction map in Figure 2 of the drawings. 5. A composite plasmid pdLFII as shown by the restriction map in Figure 2 of the drawings. 6. A composite plasmid pdLFIl tk as shown by the restriction map in Figure 3 of the drawings. 7. A composite plasmid pdLFII1 tk as shown by the restriction map in Figure 3 of the drawings. 8. A composite plasmid pdLFI2 tk as shown by the restriction map in Figure 4 of the drawings. 9. A composite plasmid pdLFII2 tk as shown by the restriction map in Figure 4 of the drawings. 10. A composite plasmid pdLFI3 tk as shown by the restriction map in Figure 5 of the drawings. 11. A composite plasmid pdLFII3 tk as shown by the restriction map in Figure 5 of the drawings. 12. A composite plasmid pdLFI3a tk as shown by the restriction map in Figure 6 of the drawings. 13. A composite plasmid pdLFII3b tk as shown by the restriction map in Figure 6 of the drawings. 14. A method for the preparation of plasmid vectors which comprises 1 culturing mouse L cell line B82, 2 lysing the cells and, 3 isolating the plasmid vectors. 15. The method as claimed in claim 14. wherein the mouse L cell line B82 is a cell line B82 2 cloned from mouse L cell line B82. 16. The method as claimed in claim 15, wherein the plasmid vector is plasmid vector LFI. 17. The method as claimed in claim 15, wherein the plasmid vector is plasmid vector LFII. 18. A pure culture of mouse L cell line B82 2 in a medium, which culture contains plasmid vectors LFI andLFII. 19. A composition comprising a a substantially pure isolate of a double strandedDNA molecule which posesses restriction enzyme cutting sites at least at O.lOkb and 2.35kb by Hind III, 0.60kb and 3.10kb by kpn 1, 3.07kb by Bam H1, and, 1.45kb by Sal 1, the said cutting sites being based upon an Eco R1 cutting site at 0.O0kba and, b a medium. 20. The composition of claim 19, wherein the DNA molecule forms a closed loop. 21. The composition of either of claims 19 or 20, wherein the DNA molecule has a O.2kb insert beyond the kpn I site. 22. The composition of any of claims 19 21, wherein theDNA molecule has an insert at the Sal 1 site. 23. A method of cloning a nucleic acid sequence comprising a treating the composition of any one of claims 19 21 with a restriction enzyme specific to a specific cutting site, under conditions such that the DNA is cut at the said site, b treating the products of step a with a nucleic acid sequence. under conditions such that the said nucleic acid sequence ligates to the said DNA, c isolating those composite DNA molecules in which the nucleic acid sequence has become inserted into the original said DNA at the said specific cutting site, d introducing the composite DNA molecules into a host, and, e allowing the said host to replicate.

## Description
PLASMID VECTOR FOR MAMMALIAN CELLSBACKGROUND OF THE INVENTION 1. Field of the Invention This invention concerns novel vectors for mammalian cells which are isolated from mammalian cells. In more detail, the invention concerns vectors which can form composite DNA with vectors specific to prokaryotic and yeast cells, can multiply in these cells as well as in mammalian cells, can multiply extrachromosomally in mammalian cells, can allow expression of desired gene s integrated in the vectors in mammalian cells and can be used as shuttle vectors.2. Description of the Prior Art When attempts are made to produce useful materials by cloned genes in bacteria such asEscherichia coli, in most of the cases one encounters difficulties in producing the mammalian gene products due to the lack of the correct modification systems, for example for glycosylation and or export in bactria. Because of this, it is natural to attempt to produce useful materials in mammalian cells.However, at present moment, no such useful vectors like pBR322 for E. coli are available for mammalian cells.Several potentially useful vectors for mammalian cells are following 1 DNA from oncogenic viruses such as SV40 or polyoma. 2 Papilloma virus DNA. In case of 1 , these DNA either integrate into host chromosomes or kill the host cells. In case of 2 , sometimes there is no expression of integrated genes. Furthermore, the copy number of the exogenous DNA is only 50 100 copy cell.These characteristics limit the use of these DNA for large scale production of useful materials. The plasmid vectors in this invention have eliminated these shortcomings of the vectors described above.SUMMARY OF THE INVENTION It is therefore an object of this invention to provide useful novel vectors which eliminate the shortcomings inherent in the currently available vectors and represent following characteristics. 1 they do not integrate into host chromosomes, 2 they have a high copy number per cell and 3 desired gene s integrated in the vectors can express its function while the vectors exist and replicate extrachromosomally in the host cells. Other objects and advantages of this invention will be apparent from the following descriptions. BRIEF DESCRIPTION OF THE DRAWINGS Figure 1 shows restriction maps of plasmid LFI and LFII. Numbers within the circular maps express molecular size of the fragments between two adjacent cutting sites. Units are expressed as kilobases. Fig. 2 shows restriction maps of pdLFI and pdLFII. Fig. 3 shows restriction maps of cloned composite plasmids, pdLFI1 tk and pdLFII1 tk . Fig. 4 shows restriction maps of cloned composite plasmids, pdLFI2 tk and pdLFII2 tk . Fig. 5 shows restriction maps of cloned composite plasmids pdLFI3 tk and pdLFII3 tk . Fig. 6 shows restriction maps of composite plasmids, pdLFI3a tk and pdLFI3b tk . Numbers in the circles express molecular size bases of Bam HI fragments of HSV tk gene.DETAILED DESCRIPTION OF THE INVENTION The vectors of this invention are novel vectors named LFI and LFII and can be obtained from a mouse cell line which can be cultured in vitro. For example, a cell line B82 2, which was derived from mouse L cells, is grown in MEM Eagle s Minimum Essential Medium and after lysis of the cells with SDS sodium dodecyl sulfate , the vectors are isolated and purified by centrifugation. The cell line B82 2 was isolated from one of mouse L cells B82 , as a cell line which has a high copy number of the concerned vectors.B82 cells were cultured in MEM with 10 calf serum and approximately each 50 cells were plated on total 5 plates with 90 mm d ameter. After incubation for two weeks at 370C, each colony which consisted of 500 1000 cells was examined by the procedure as will be described in Example 1 4 and a cell was selected which had the highest copy number of the vectors. The cell line B82 is a useful tool in the recombinant DNA research and was first described byLittlefield BBA., 95, 14 22, 1965 and ExperimentalCell Research 41, 190 196, 1966 can be freely obtined from many research labolatories in the world e.g., Mass. Cell Culture Centers at MassachusettsInstitute of Technology, Harvard Medical School,University of Alabama, the mammalian cell stocks ofDivision of Enzymology, Institute of AppliedMicrobiology, University of Tokyo, etc. . Molecular size of the vectors, LFI and LFII, were calculated to be approximately 5.3 kb and 5.5 kb, respectively, by analysis on agarose gel electrophoresis. When the restriction maps of the vectors obtained by using a variety of restriction enzymes were compared with that of polyoma virus DNA, it was found that bothLFI and LFII consist of two kinds of DNA structures, one is present in polyoma virus DNA and the other is not present in polyoma virus DNA. LFII has an extra DNA structure of 0.2 kb which is inserted at approximately 500 bp clockwise from a Kpn I site near Bam HI site see Fig. 1 in addition to LFI struture. Restriction maps of LFI and LFII using restriction enzymes, Eco RI,Sal I, Bam HI, Hind III, Hpa II and Kpn I were compared with the restriction map of polyoma virus DNA with the same enzymes.LFI and polyoma virus DNA are cut approximately at the same sites where Eco RI, Bam HI,Hind III and Kpn I cut. On the other hand, LFI possesses a cutting site by Sal I which does not exist in polyoma DNA. Total DNA base pairs of LFI are almost equal to those of polyoma virus DNA. LFII also possesses a cutting site by Sal I which does not exist in polyoma virus DNA but it also has cutting sites byEco RI, Bam HI, Hind III, Kpn I which are located at almost the same places as located in polyoma virus DNA.The total base pairs of LFII 5.5 kb are roughly equal to those of polyoma virus DNA 5.292 kb . While polyomaDNA is cut at eight sites by Hpa II, LFI and LFII have 10 and 12 cutting sites by Hpa II, respectively. These results indicate that LFI and LFII differ from polyoma virus DNA in at least three base sequences. When we calculated copy numbers of the concerned vectors in the cell based upon the results obtained by agarose gel electrophoresis, the maximum copy number of LFI per cell is approximately 5,000 copies and that of LFII is approximately 1,000 copies.The number of polyoma virus genome per cell is only 10 to 50 copies. We have demonstrated that the concerned vec tors exist extrachromosomally without being integrated in B82 2 chromosomes. This was done by Southern hybr dization E. M. Southern J. Mol. Biol. 98, 503, 1975 of the total DNA isolated from cultured mouse L cellB82 2 after having treated the DNA by a restriction enzyme, Eco RI or Sal I. The concerned vectors LFI and LFII can form composite plasmids with bacterial plasmids. For example they can form composite plasmids with a derivative of pBR322 DNA, pML2d, and the composite plasmids can propagate in E. coli. This was demonstrated by following experiments. First, 100 ng of LFI and LFII digested with Sal I and 20 ng of pML2d 2.65 kb DNA digested likewise were mixed and ligated in the presence of 0.2 units of T4 DNA ligase and 2.5 mM ATP. When E. coliLE392 was transformed by the DNA by Caul2 procedure T. Maniatis, E. F. Fritsch, J. Sambrook Molecular Cloning, p250, Cold Spring Harbour Lab., 1982 , ampicillin 20 ml resistant transformants Ampr were obtained.DNAs were then prepared from these Amp cells and they were hybridized with 32p labeled LFI and LFII. Presence of 7.95 kb and X.15 kb circular DNA in Ampr cells was further demonstrated by agarose gel electrophoresis.These composite plasmids were named as pdLFI 7.95 kb and pdLFII 8.15 kb , respectively Fig. 2 . Furthermore, the concerned vectors, after integration of a gene s for useful materials into them, still can exist as extrachromosomal plasmids in mammalian cells, can replicate autonormously and can express the integrated gene s . The above described composite plasmids pdLFI and pdLFII, which were obtained from LFI or LFII and a derivative of pBR322, pML2d, were cut by a restriction enzyme Bam HI and ligated with herpes simplex virus thymidine kinase gene HSV tk which were cut by Bam HI. E. coli LE392 was then trans formed with the resulting DNA and from Ampr transfor mants six different plasmids were obtained Figs. 3, 4, 5 . These plasmids were named as pdLFI1 tk , pdLFI2 tk , pdLFI3 tk , pdLFIIl tk , pdLFII2 tk and pdLFlI3 tk , respectively.Among them, molecular size of pdLFI3 tk and pdLFII3 tk are approximately 9.45 and 9.65 kb, respectively and a portion of the original DNA sequences of pdLFI and pdLFII, approximately 2.10 kb, is deleted in these vectors. When mouse L cells defective in thymidine kinase were transfected with pdLFII3 tk DNA by the calcium phosphate method and incubated in HAT medium W. H. Lewis, P. R. Srinivason, N. Stock, L.Siminovitch Somatic Cell Genetics 6, 333, 1980 , colonies appeared after 2 or 3 weeks. The Tk colonies were then recultured, the cells were resuspended in TE buffer 10 mM Tris HCl, pH 7.4, 10 mM EDTA and lysed with SDS.The total DNA purified from the lysates by centrifuga tion were treated with a restriction enzyme, either EcoRI or Sal I or Xho I, electrophoresed on agarose gel and hybridized Southern hybridizacion with 32p labeled pdLFII3 tk DNA as a probe. Since hybridized band after autoradiography coincided with the position where pdLFII3 tk DNA migrates, it has become evident that all the pdLFII3 tk DNA exists extrachromosomally in Tk mouse cells. It was demonstrated from these results that while pdLFII3 tk DNA replicates extrachromosomally as plasmids in L cells, the integrated gene tk expresses its function. When E. coli was transformed with DNA prepared from the Tk cells, Ampr colonies appeared. DNA isolated from the Amp colonies D. S.Holmes, M. Quigley Anal Biochem 114, 193, 1981 was subjected to agarose gel electrophoresis. A DNA band was detected at the position where pdLFII3 tk DNA migrates. These results indicate that pdLFII3 tk DNA can be used as a shuttle vector from E. coli to L cells to E. coli, while maintaining its original DNA structure and expressing integrated genes amp in E. coli and tk in L cells . Similar results were obtained with pdLFI1 tk , pdLFI2 tk , pdLFI3 tk , pdLFII1 tk and pdLFII2 tk . As described above, the concerned vectors LFI and LFII are novel vectors, which can form composite plasmids with bacterial plasmids such as pBR322 pML2d , can propagate extrachromosomally in bacteria and mammalian cells, can allow the expression of a useful gene s integrated in the vectors and have a high copy number in the cells. Also, it is clear from the ralasionships between pdLFII1, pdLFII2 and pdLFII3, or between pdLFI1, pdLFI2 and pdLFI3 that functions which enable LFI and LFII to replicate autonormously reside in a part of LFI and LFII structure as in other plasmids. From these facts, it is easily deduced that derivatives of LFI and LFII with another DNA possess functions similar to those of LFI and LFII.Therefore, it is obvious that usefulness of LFI and LFII shuld not be limited to themselves but can be extended to the DNA which is obtained by modification of LFI and LFII.Example 1 1 Preparation of vector LFI and LFII. Cell line B82 from the cell collection,Division of Enzymology, Institute of AppliedMicrobiology, University of Tokyo was cultured in MEM with 10 calf serum and approximately each 50 cells were plated on total 5 plates of 90 mm diameter made of plastics. After incubation for two weeks at 370C, each colony which consisted of 500 1000 cells was examined by the procedure as will be described in Example 1 4 mentioned later and a cell line having the highest copy number of the vectors of this invention was selected as cell line B82 2. B82 2 cells were cultured in MEM containing 10 calf serum for 5 days at 370C.Approximately 2 x 107 cells were resuspended in 1 ml TE buffer 10 mM Tris HCl, pH 7.4, 10 mM EDTA and lysed witn SDS final concentration 0.6 . To the lysate,NaCl final 1.0 M was added and after leaving for 3 hrs at 40C, the sample was centrifuged 15,000 rpm 30min .The supernatant fraction was mixed with an equal volume of phenol chloroform mixture 1 1 , shaken and centrifuged. The aqueous phase was then mixed with chloroform isopropylalcohol 24 1 and DNA was precipitated with ethanol from the aqueous phase after centrifugation. the DNA precipitate was dissolved in TE buffer and electrophoresed on agarose gel with 0.5 ml of ethidiumbromide in Tris borate EDTA buffer 89 mMTris borate, pH 7.8, 89 mM boric acid, 2 mM EDTA for 4 hrs at 3 volts cm. After confirming two distinct bands under a UV lamp, the bands were transferred to a DEAE membrane filter paper by electrophoresis for 1 hr at 3 volts cm. The filter was then washed with TES buffer 10 mM Tris HCl, pH 8.0, 1 mM EDTA, 0.3 M NaCl and DNA was eluted with a TES elution buffer 10 mM Tris HCl, pH 8.0, 1 M NaCl for 2 hrs at 550C.The DNA was recovered as a precipitate from the eluate by adding 2.5 vols of 99.5 ethanol. The DNA thus obtained was dissolved inTE buffer 10 mM Tris HCl pH 8.0, 1 mM EDTA and kept at 40C. DNAs with smaller molecular weight faster migrating DNA and with larger molecular weight slower migrating DNA were named as LFI and LFII, respectively. 2 Determination of the molecular size. The DNA fraction 2 9 obtained in 1 was treated with a restriction enzyme, Sal I 15 units for 5 hrs at 370C in a buffer containing 10 mM Tris HCl, pH 7.5, 7 mM Mac12, 5 mM dithiothreitol DTT and 50 mMNaC1. After electrophoresis as described in 1 , two bands, as previously observed in 1 , were detected but at different locations. The molecular size of the DNA was determined to be approximately 5.3 kb and 5.5 kb after comparison with the positions of standard X DNA which was digested with Hind III and electrophoresed under the same condition. From these experiments, it became clear that B82 2 cells contain two circular DNA with 5.3 kb LFI and 5.5 kb LFII . Furthermore, 5.3 kb DNA was several fold more abundantly present than 5.5 kb DNA. 3 Construction of the restriction maps. 200 ng of LFI and LFII DNA was treated with a restriction enzyme 10 units per 1 g DNA including EcoRI, Bam HI, Hind III, Sal I, Kpn I, Bgl II, Xho I, andHpa II and electrophoresed as decribed in 1 . From the migration patterns of DNA, it has become clear that LFI and LFII have a common DNA structure and LFII has an extra DNA structure of 0.2 kb in addition to LFI DNA which is inserted at approximately 500 bp clockwise from a Kpn I site near Bam HI site . Molecular sizes of DNA obtained by treatment with a variety of restriction enzymes are shown in Table 1 LFI and Table 2 LFII . Based upon the cutting site by Eco RI at 0 , distances of cutting sites produced by the restriction enzymes Bam HI, Hind III, Kpn I, Sal I are shown in the following figure Figure 1 and Table 3. From the results of Tables 1, 2 and 3, restriction maps for LFI and LFII were constructed and they are shown in Figure 1. Table 1EMI12.1 tb Restriction SEP Number SEP of SEP cutting SEP Molecular SEP size SEP kb tb SEP enzymes SEP sites tb SEP SEP tb SEP Eco SEP RI SEP 1 SEP 5.3 tb SEP Bam SEP HI SEP 1 SEP 5.3 tb SEP Hind SEP III SEP 2 SEP 3j04 SEP 2.26 tb SEP Sal SEP I SEP 1 SEP 5.3 tb SEP Kpn SEP I SEP 2 SEP 2.8 SEP 2.5 tb SEP Bgl SEP II SEP 0 SEP tb SEP Hpa SEP II SEP 10 tb SEP Xho SEP I SEP 0 SEP tb Note a and b represent fragments, respectively. Table 2EMI12.2 tb Restriction SEP Number SEP of SEP cutting SEP Molecular SEP size SEP kb tb SEP enzymes SEP sites tb SEP lea SEP by SEP tb SEP Eco SEP RI SEP SEP SEP 1 SEP 5.5 tb SEP Bam SEP HI SEP 1 SEP 5.5 tb SEP Hind SEP III SEP 2 SEP 3.24 SEP 2.26 tb Cont d Table 2 Cont d EMI13.1 tb Sal SEP I SEP 5.5 tb Kpn SEP 1 SEP 2 SEP 3.0 SEP 2.5 tb Bgl SEP II SEP 0 tb Hpa SEP II SEP 10 tb Xho SEP I SEP SEP 0 SEP tb Note a and b represent fragments, respectively. Table 3EMI13.2 tb Restriction SEP Positions SEP from SEP Eco SEP RI SEP cutting SEP site SEP kb tb SEP enzymes tb SEP Eco SEP RI SEP SEP 0 SEP tb SEP Bam SEP HI SEP 3.07 tb SEP Hind SEP III SEP 0.1 SEP 2.3 SEP 5 SEP tb SEP Kpn SEP I SEP 0.6 SEP 3.1 tb SEP Sal SEP I SEP 1.45 tb SEP Bgl SEP II SEP No SEP cutting SEP site tb SEP Xho SEP I tb 4 Determination of the copy number. From approximately 2 x 107 cells cultured as described in 1 , DNA was extracted as described in 1 , electrophoresed on agarose gels with different amounts of SV40 DNA with molecular size of 5.2 kb as described in 2 . The intensities of the ethidiumbromide DNA revealed by W light were compared with those of 5.2 kbSV40 DNA. From the number of cells 2 x 107 used, the number of LFI and LFII per cell was calculated to be approximately 5,000 copies and 1,000 copies, respectively. 5 Evidence for the extrachromosomal existence of the vectors. B82 2 cells cultured as described in 1 were lysed with SDS and total DNA was extracted with phenolchloroform 1 1 . The DNA was purified by the procedure described in 1 and separated into four fractions. After treating them with no enzyme i , with EcoRI ii , with Sal I iii and with Xho I iv , they were electrophoresed on agarose gel as decribed in 1 .After alkaline denaturation T. Maniatis, E.F. Fritsch,J. Sambrook Molecular Cloning, P 383, C.S.H.L., 1982 and transfer to a nitrocellulose filter, DNA was then hybridized Southern hybridization with LFI and LFII as probes which had been labeled with 32p by nick translation using DNA polymerase I. The DNA bands which were hybridized with the probes were examined by radioautography. The hybridized DNA bands were detected at positions corresponding to 5.3 kb and 5.5 kb with ii and iii samples. The hybridized DNA bands with i and iv samples were located at the same position. No other band was detected. From these experiments, it was concluded that in B82 2 cells LFI and LFII exist and replicate extrachromosomally without being integrated in the chromosome.Example 2. Construction of composite plasmids with E.coli plasmid pBR322 pML2d shuttle vectors which can replicate in E. coli. a Composite plasmid with LFI and pML2d. Under the condition described in Example 1 2 , LFI 100 ng and pML2d DNA 20 ng which had been cut by Sal I were mixed, heated at 650C for 10 min, rapidly cooled, mixed with 2.5 vols of ethanol in the presence of 0.3 M sodium acetate and left for at least 2 hrs at 200C. The samples were centrifuged at 15,000 rpm for 20 min and the precipitate was dried in vacuo.To the precipitate, distilled water 7.5 , concentrated x 10 ligation buffer 1 p1, 660 mM Tris HC1, pH 7.5, 66 mM MgC12 and calf serum albumin solution 0.1 all, 2 mg 40 4 were added. After incubation at 420C for one hr, ATP 0.25 iii of 0.1 M , DTT 1 iil of 0.1 M and T4 DNA ligase 0.15 yl of 1.5 units per p1 were added and the samples were left for two days at 40C.Seventy iil of Tris buffer 10 mM Tris HCl, pH 8.0, 1 mM EDTA was then added and after electrophoresing a poriton of the samples to confirm appearance of new bands, the DNA was ued for bacterial transformation using E. coli LE392 as a recipient by CaC12 method T.Maniatis, E. F. Fritsch, J. Sambrook Molecular Cloning p250, Cold Spring harbour Lab. 1982 . Resultant ampicillin resistant 20 W ml colonies were replicaplated on nitrocellulose filter papers and the papers were placed for 7 min on Whatman 3MM papers in 10 ml of 0.5 M NaOH 1.5 M NaCl solution to denature DNA.The nitrocellulose papers were then neutralyzed by soaking them in Tris buffer 1 M Tris HCl, pH 6.7, 1.5 M NaC1 for 7 min, followed by soaking them in x 2 SSC which was prepared by 10 fold dilution of x 20 SSC 3 MNaCl, 0.3 M Na3 citrate, pH 7.0 . After drying in air for 30 min, the papers were heated for 2 hrs at 800C in an oven. The papers were then soaked in prehybridization solution 1 1 mixture of x 5 SSC and x 4 Denhardt solution which consists of 5 times diluted x 20 Denhardt solution 0.4 Ficoll, 0.4 calf serum albumin, 0.4 polyvinylpyrrolidone , and after incubation for 2 hrs at 700 C, the papers were dried in air. They were then placed and incubated at 650C for overnight in plastic bags with a mixture of a solution 100 p1 consisting of 15 iii of 32P labeled LFI DNA 2 x 105 cpm ml, prepared by nick translation, 3 p1 of salmon sperm DNA 10 mg ml and 82 1 of H2O, which had been heated at 1000C for 10 min and quickly cooled, and the prehybridizaiton solution 900 p1 described above. The papers were then washed three times with x 2 SSC solution at 550C for 20 min. After having dried in air, hybridization of the 32 probes P LFI DNA with the DNA from bateria was con firmed by radioautography colony hybridization . TheAmpr cells thus confirmed were then grown and lysed with lysozyme and SDS.Total DNA was then extracted by the procedures described in Example 1 5 , treated with restriction enzymes Sal I, Eco RI, Bam HI and electrophoresed on agarose. The presence of circularDNA with 7.95 kb in Ampr cells was then demonstrted bySouthern hybridization using P labeled LFI DNA as a probe. b Composite plasmid with LFII and pML2d. The presence of circular DNA with 8.15 kb inAmps cells were also confirmed by the same experiments as described in a except that LFII DNA was used in stead of LFI DNA. The composite plasmids obtained in a and those obtained in b were named as pdLFI and pdLFII, respectively.Example 3. Usefulness of pdLFI and pdLFII as shuttle vectors and evidence for extrachromosomal gene expression of a cloned gene in mammalian cells. The composite plasmid pdLFI and pdLFII obtained from LFI and LFII and pBR322 pML2d as described in Example 2 was cut by a restriction enzymeBam HI. Twenty ng of the DNA was then mixed with 100 ng of a Bam HI fragment of herpes simplex virus thymidine kinase gene drived from pFG5 Proc. Natl. Acad. Sci. USA 76, 3755, 1979 and ligated by the procedures described in Example 2. E. coli LE392 was then transformed with the DNA and Amp r transformants were subjected to colony hybridization as described in Example 1 5 using 32P labeled HSV tk as a probe. The presence of bac 32 terial DNA which was hybridized with 32P HSV tk DNA was confirmed by radioautography.Furthermore, DNA isolated from the Ampr transformants according to the procedures described in Example 1 1 was analyzed by various restriction enzymes Sal I, Bam HI, Eco RI, Hind III as decribed in Example 1 3 . Six different plasmids were then obtained from Ampr transformants and they were named as pdLFI1 tk , pdLFI2 tk , pdLFI3 tk , pdLFIIl tk , pdLFII2 tk and pdLFII3 tk Figs. 3, 4, 5 . Among them, molecular size of pdLFI3 tk and pdLFII3 tk are approximately 9.43 kb and 9.63 kb, respectively. The smaller sizes of these DNA were resulted by a deletion of a portion approximately 2.12 kb from plasmids pdLFI and pdLFII, respectively. Analyses of pdLFI3 tk by restriction enzymesSal I and Bgl II according to the procedures described in Example 1 5 have revealed that two kinds of forms exist in pdLFI3 tk . They are a , which produced approximately 3.1 kb and 6.4 kb fragments upon digestion with Sal I, followed by Bgl II and b , which produced approximately 4.4 kb and 5.0 kb fragments by the same treatment. In combining these results and an observation that the restriction enzyme Bgl II cuts HSV tk gene obtained by Bam HI treatment into two fragments with 825 bp and 2,775 bp, it was concluded that a composite plasmid ligated with HSV tk gene in the opposite direction also exists Fig. 6 . Naturally this situation shuld be applicable to pdLFI1 tk , pdLFI2 tk , pdLFIIl tk , pdLFII2 tk and pdLFII3 tk .Also it should be applicable to composite plasmids pdLFI and pdLFII, which were constructed from LFI and LFII and pML2d. When mouse L cells deficient in thymidine kinase tk were treated with pdLFII3 tk DNA calcium phosphate W. H. Lewis, P. R. Srinivason, N. Stock, L.Siminovitch Somatic Cell Genetics 6, 333, 1980 and incubated in HAT medium, colonies were formed after two weeks. The Tk colonies were then recultured and the total DNA was prepared by the procedure described inExample 1 1 and Example 1 5 . After electrophoresis on agarose gel, Southern hybridization was performed using 32P labeled pdLFII3 tk DNA as a probe. Results of autoradiography showed that the hybridized band was located at the position where pdLFII3 tk migrated, indicating that all of the pdLFII3 tk DNA locate extrachromosomally in mouse L cells. Furthermore, when DNA prepared from the Tk cells according to the procedure described inExample 1 1 was used as a donor for the transformation of E. coli LE392 as described in Example 2, Amp colonies appeared. DNA was prepared from theAmpr cells D. S. Holmes, M. Quigley Anal Biochem. 114, 193, 1981 and subjected to electrophoresis on agarose gel. A band corresponding to the molecular size of pdLFII3 tk was observed. From these results, it was demonstrated that pdLFII3 tk plasmid can express its genes Amp in E. coli, tk in L cells , while maintaining its original structure during transfers from E. coli L cells E. coli and can be used as a shuttle vector mammalian cells E. coli . Similar results were obtained with pdLFII tk , pdLFI2 tk , pdLFI3 tk , pdLFlI1 tk and pdLFII2 tk . From the experiments described above, it was proved that plasmids LFI and LFII can form composite plasmids with bacterial plasmids such as pBR322 pML2d .Furthermore, after integration of a gene s responsible for the production of useful materials, they can still replicate extrachromosomally and the integrated genes can be expressed in mammalian cells.