# SUPEROXIDE DISMUTASE CLONING AND EXPRESSION IN MICROORGANISMS

## Claims
Verfahren zur Herstellung einer acetylierten Human Superoxiddismutase, das das Einführen einer die Human Superoxiddismutase kodierenden und an ihrem 5 Ende eine ein Glycin oder Alanin in der ersten Position, gefolgt von einer polaren Aminosäure in der zweiten Position, kodierende Acetylierungs Signalsequenz aufweisenden DNA Sequenz in Hefe einsehließt, wobei die Aeetylierungs Signalsequenz den Hefewirt zum Acetylieren der N terminalen Aminosäure des Polypeptids veranlaßt.

## Description
Superoxide dismutase SOD is in fact a variety of different enzymes found in most living organisms. One function in mammals is to destroy superoxide, a material naturally produced during phagocytosis. The superoxide dismutases are characterized in families based on the metal associated with the enzyme, where the metals vary amongst iron, manganese, copper and copper zinc. Superoxide dismutase, e.g., from bovine liver, has found clinical use, particularly as an anti inflammatory agent in mammals including humans. Other utilities include scavenging superoxide anions due to exposure of a host to various superoxide inducing agents, e.g. radiation, paraquat, etc. prophylaxis or therapy for certain degenerative diseases, e.g., emphysema food preservation and the like. It is therefore important that stable supplies of physiologically acceptable superoxide dismutase be made available, particularly for use The amino acid sequence of human erythrocyte Cu Zn superoxide dismutase is described in Jabusch Polypeptides acetylated at their N terminus and methods for producing such acetylated polypeptides are provided. By providing a particular acetylation signal sequence at the 5 end of the structural gene for a desired polypeptide, the N terminal amino acid will be acetylated when the gene is expressed in yeast. The acetylation signal sequence encodes for at least the first two N terminal amino acids, where the first amino acid is either alanine or glycine, and the second amino acid is a polar amino acid, usually being threonine, serine or aspartate. Acetylation of human superoxide dismutase produced in yeast is demonstrated when the first two amino acids are alanine and threonine, respectively. Methods and compositions are provided for the efficient expression of polypeptides demonstrating the biological activities of human Cu Zn superoxide dismutase hSOD . The methods employ a DNA sequence hSOD gene encoding a substantial portion of the amino acid sequence of hSOD in conjunction with a translational initiation region optimized for expression in the expression host. The hSOD gene is inserted into an appropriate vector for expression in a host, conveniently under conditions which allow for secretion, so as to harvest the SOD product from the extracellular medium. Methods and compositions are also provided for the N terminal acetylation of hSOD and other polypeptides. Hereinafter, acetylation refers to addition at the amino terminus of polypeptides and proteins in contrast to modification of amino acid side chains, e.g., lysine, as is also observed naturally. Acetylation of polypeptides and proteins is useful for a number of reasons. Where the natural condition of the polypeptide includes acetylation, as is the case for cytoplasmic hSOD, methods of expression which include acetylation provide a product having the desired natural structure and conformation. Where the product finds pharmaceutical and or The structural gene for hSOD or other polypeptide includes an acetylation signal sequence at the 5 end thereof, which signal sequence causes a yeast expression host to effect actylation. The acetylation signal sequence encodes at least the first two N terminal amino acids in the polypeptide. The first amino acid will be either alanine, glycine or serine, while the second amino acid will be a polar or aromatic amino acid, usually being threonine, serine, aspartate or phenylalanine. The amino acids may be the natural N terminal amino acids normally present in the polypeptide to be expressed. This is the case with hSOD where the first two amino acids are alanine and threonine, respectively. Other naturally acetylated proteins which may be expressed and acetylated in yeast include The present invention is also useful for acetylating polypeptides and proteins which are not naturally acetylated. Acetylation may be achieved by joining the acetylation signal sequence to the 5 end of the structural gene for the polypeptide. The acetylation signal sequence will encode for at least two amino acids as described above , and may encode up to ten or more amino acids, preferably fewer than five amino acids. Fewer added amino acids is usually desirable to limit interference with or loss of a desired activity of the polypeptide. Conveniently, the signal sequence may be synthesized and joined to the structural gene using well known techniques. As an alternative to adding the acetylation signal sequence to the structural gene, it will sometimes be possible to modify the 5 end of the structural gene to substitute one or both of the first two amino acids of the polypeptide. Such modification may be accomplished by a variety of conventional methods. For example, the structural gene may be restricted near its 5 end to remove a known number of nucleotides. A synthetic oligonucleotide may then be joined to the cohesive end remaining after restriction. The oligonucleotide will restore and substitute the base pairs as necessary to provide the desired acetylation signal sequence. Alternatively, site specific mutagenesis employing, e.g., phage M13, can be used to effect an appropriate modification to the 5 end of the structural gene. In order to prepare hSOD, it is necessary to have a DNA sequence which encodes for hSOD. One manner of achieving such sequence, is to clone cDNA from messenger RNA from cells which produce hSOD. Conveniently, human liver cells may be used for this purpose. After the cDNA is cloned, where the DNA coding sequence is unknown, but at least a partial amino acid sequence is known, one may then screen the cDNA with mixtures of probes having all of the possible variations of nucleotides encoding for the particular series of amino acid residues. The choice of the residues for which the sequence encodes is somewhat arbitrary, although the residues chosen will usually be selected to minimize the number of different sequences which must be synthesized. For hSOD, conveniently a DNA sequence encoding for at least the amino acid residues 19 to 24 can be used, particularly a probe having at least about 15 bases and not more than about 20 bases, more conveniently about 17 bases. One may then restriction enzyme digest the clones which appear to hybridize with the labeled probes, fractionate the DNA fragments and repeat the hybridization, particularly by employing a second series of probes which hybridize to DNA sequences encoding for a different series of amino acid residues in hSOD. Conveniently, these amino acid residues may be 109 to 114. One or more clones may be found which are positive to both probes and these may be used as a source for cDNA encoding for at least a substantial proportion of hSOD. Quite surprisingly, it was found that the amino acid sequences which have been published for hSOD differed in a significant number of residues from the amino acid sequence encoded for by the cDNA. Specifically, where the two published sequences differed Jabusch Because of the uncertainties of the effect on translation of the separation between the ribosomal binding site and the translational initiation codon, normally AUG, the subject method provides a technique for varying the distance and nucleotides separating the ribosomal binding site from the initiation codon. Usually, there are from about 6 to 15, more usually about 6 to 12 nucleotides in the spacer between the ribosomal binding site and initiation codon. As the base sequence downstream from the initiation site may also affect translation efficiency, the subject method also provides for variation of nucleotide sequence but not length within the initial several 5 codons of the polypeptide as permitted by the redundancy constraints of the genetic code. Such degeneracy may intend up to 4 codons, more usually 2 codons, downstream from the initiation site. A plurality of linkers are prepared where at least 2 nucleotides, usually at least 3 nucleotides, and not more than 10 nucleotides, usually not more than about 6 nucleotides, are varied to include members having each of the 4 nucleotides if within the spacer or 2, 3, or 4 nucleotides as permitted by genetic code redundancy if within the structural gene for the polypeptide. In addition, the linkers are prepared, having differing numbers of nucleotides, so as to provide a group of linkers differing not only in the sequence, but also in length. The difference in length can be achieved by removal of portions of the support during the linker synthesis and, if appropriate, continuing synthesis at a subsequent stage, so as to provide for linkers having a graduated number of sequence lengths. Usually, the mixture of linkers will vary in length by at least one nucleotide and not more than over a range of six nucleotides, usually not more than four nucleotides. This can be conveniently illustrated where the absent bases are at the terminus. After each stage, a portion of the support is removed and the synthesis continued with the strands bound to the support, providing all four nucleotides dNTP at each stage. These single strands will then be hybridized to a single strand which is complementary in part, where the variable region will be an overhang. Thus, one will achieve a graduated series of linkers having overhangs differing in both their nucleotide sequences and lengths. At an appropriate point during subsequent hybridization, ligation or cloning operations the overhang region s is filled in to provide double stranded material amenable to further manipulation. This is usually and preferably performed The linkers include within their sequence, at least a portion of the region between the ribosomal binding site and the initiation codon, preferably the nucleotides proximal to the initiation codon. The linker may also include the initiation codon and portions of the structural gene, the ribosomal binding site, and bases upstream from the ribosomal binding site, which may or may not include transcriptional regulatory sequences. Usually the linker will be at least about 5 bases, more usually at least about 20 bases, and usually not exceeding about 200 bases, more usually not exceeding about 100 bases. Where the linker is greater than about 35 bases, it will usually be assembled by employing single stranded sequences of from about 10 to 35 bases, which have homology with only a part of a complementary strand, thus providing for complementary overlapping sequences with overhangs, so that the various single strands can be hybridized, ligated and the degenerate and or variable length overhang filled in as indicated above to produce the desired linker having cohesive and or blunt ends. Where the structural gene has a convenient restriction site, usually not more than about 50 bases downstream from the initiation codon, a fragment containing the structural gene may be restricted and joined to a complementary cohesive terminus of the linker or may be filled in to provide a blunt end terminus, which blunt end may be ligated to a blunt end of the linker. The linker is devised to ensure that the structural gene is complete and in reading frame with the initiation codon. As indicated, in preparing the linker, one provides that there are a series of linkers which have a randomized series of nucleotides, that is, each of the four possible nucleotides in the coding strand subject to the provision of genetic code limitations indicated above and which are graduated in size, lacking one or more of the nucleotides defining the region intermediate or bridging the ribosomal binding site and initiation codon. These linkers which are prepared from single strands may be joined to other single or double DNA strands to provide for extended linkers, which may include not only the ribosomal binding site, but bases upstream from the ribosomal binding site. Alternatively, the linkers may be relatively small, beginning at a site internal to or adjacent to the ribosomal binding site and extending downstream to a site at the initiation codon or internal to the structural gene. While the particular order of joining the various fragments to produce the constructs of this invention will usually not be critical, conveniently, the structural gene may be first joined to the linker. This DNA construct will include not only the structural gene, but also the ribosomal binding site and any additional nucleotides upstream from the ribosomal binding site. In addition, there will be substantial variety in the nucleotides and numbers of nucleotides between the ribosomal binding site and initiation codon. The subject DNA construct is inserted into an appropriate expression vector which has the necessary transcriptional initiation regulatory sequences upstream, as well as transcriptional termination regulatory sequences downstream from the insertion site of the subject DNA construct. Thus, the linker will be flanked at the 5 end with transcriptional initiation regulatory signal sequences and at the 3 end with at least a portion of a coding region and transcriptional and translational termination sequences. 5 and 3 intend the direction of transcription. After preparing the plasmid or viral DNA for introduction into an appropriate host usually including at an appropriate stage in the manipulations filling in of the variable overhang region , the host is transformed or transfected, respectively, cloned, the clones streaked and individual clones selected for efficient expression by assaying for production of the desired product, e.g., hSOD. The number of clones to be screened to determine the various levels of production of the product will depend upon and be proportional to the degreee of length variability and sequence degeneracy introduced into the synthetic linker. As exemplified in the present embodiment, with 4 length variables and 4 fold sequence degeneracy at each of 6 nucleotides in the linker, the number of possible recombinant sequences is 5440. Usually at least a few hundred, preferably several thousand or more, clones will be screened. Screening can be efficiently performed using Western blots antibody detection of product of host cell colonies or viral plaques transferred to filters of nitrocellulase or other suitable material. Alternatively, using electrophoresis and providing for a plurality of lanes, where each lane is an individual clone, an immediate and direct comparison can be made of which clones are most efficient in expression by visualization of staining intensity, autoradiography or Western blotting of the product band. This screen will usually be sufficient, although more quantitative immunoassays or enzyme assays can be employed, as appropriate. If desired, the construct can be transferred to a different host which recognizes the regulatory signals of the expression construct or the expression construct modified by introduction at appropriate sites of necessary regulatory signals to provide for efficient expression in an alternative host. If desired, the hSOD gene may be joined to secretory leader and processing signals to provide for secretion and processing of the hSOD. Various secretory leader and processing signals have been described in the literature. See for example, U.S. Patent Nos. 4,336,336 and 4,338,397, as well as copending application Serial Nos. 522,909, filed August 12, 1983 and 488,857, filed April 26, 1983. Of particular interest as hosts are unicellular microorganism hosts, both prokaryotes and eukaryotes, such as bacteria, algae, fungi, etc. In particular, A wide variety of vectors are available for use in unicellular microorganisms, the vectors being derived from plasmids and viruses. The vectors may be single copy or low or high multicopy vectors. Vectors may serve for cloning and or expression. In view of the ample literature concerning vectors, commercial availability of many vectors, and even manuals describing vectors and their restriction maps and characteristics, no extensive discussion is required here. As is well known, the vectors normally involve markers allowing for selection, which markers may provide for cytotoxic agent resistance, prototrophy or immunity. Frequently, a plurality of markers are present, which provide for different characteristics. In addition to the markers, vectors will have a replication system and in the case of expression vectors, will usually include both the initiation and termination transcriptional regulatory signals, such as promoters, which may be single or multiple tandem promoters, an mRNA capping sequence, a TATA box, enhancers, terminator, polyadenylation sequence, and one or more stop codons associated with the terminator. For translation, there will frequently be a ribosomal binding site as well as one or more stop codons, although usually stop codons will be associated with a structural gene. Alternatively, these regulatory sequences may be present on a fragment containing the structural gene, which is inserted into the vector. Usually, there will be one or more restriction sites conveniently located for insertion of the structural gene into the expression vector. Once inserted, the expression vector containing the structural gene may be introduced into an appropriate host and the host cloned providing for efficient expression of hSOD. In some instances, specialized properties may be provided for the vector, such as temperature sensitivity of expression, operators or activators for regulation of transcription, and the like. Of particular interest is the ability to control transcription by exogenous means, such as temperature, inducers, corepressors, etc., where transcription can be induced or repressed by an exogenous compound, usually organic. Where the hSOD is made intracellularly, when the cell culture has reached a high density, the cells may be isolated, conveniently by centrifugation, lysed and the hSOD isolated by various techniques, such as extraction, affinity chromatography, electrophoresis, dialysis, or combinations thereof. Where the product is secreted, similar techniques may be employed with the nutrient medium, but the desired product will be a substantially higher proportion of total protein in the nutrient medium than in the cell lysate. The hSOD which is formed has substantially the same amino acid sequence as the naturally occurring human superoxide dismutase, usually differing by fewer than 5 amino acids, more usually differing by fewer than 2 amino acids. The recombinant hSOD r hSOD displays substantially the same biological properties as naturally occurring hSOD. The biological properties include immunological properties, where antibodies raised to authentic hSOD cross react with r hSOD. Furthermore, in common bioassays employed for hSOD, the r hSOD product demonstrates a substantial proportion, usually at least about 10 , preferably at least about 50 , more preferably at least about 80 , of the enzymatic activity of the authentic hSOD, based on weight of protein. An illustrative assay technique is described by Marklund and Marklund, The following examples are offered by way of illustration and not by way of limitation. Total RNA was prepared from an adult human liver by the guanidinium thiocyanate lithium chloride method Cathala The cDNA insertion employed the oligo dG dC tailing method Maniatis Strain D1210 was retransformed with the liver cDNA library and about 40,000 clones were grown on nine 14cm diameter Petri dishes. After transfer of the colonies to nitrocellulose paper and chloramphenicol amplification of plasmid DNA, the cells were lysed and the filters prepared for hybridization Ish Horowicz and Burke, where all of the indicated possibilities for encoding the peptide sequence were prepared 32 fold degenerate . The probes were labeled with ³²P to a specific activity of 1 3x10⁸cpm µg and Millipore 0.45µm filtered before use. Filters were prehybridized for 6hrs at 30 C in 4x SSC, 2x Denhardts s solution, 40mM sodium phosphate, pH 7.5, 300µg ml sonicated salmon testes DNA. Hybridization was for 20hrs at 30 C in the same solution containing 2x10⁶cpm ml hSOD DNA probe residues 19 24 . Filters were washed in 4x SSC, once for 15min at r.t. and twice for 15min at 30 C, blotted dry and autoradiographed with an intensifying screen for 24hrs at 70 C. Areas on the master plates that corresponded to duplicate positive signals were picked into L broth and plasmid DNA prepared by the miniscreen procedure Maniatis One plasmid pool pSOD1 contained a cDNA insert of 520bp that hybridized with both probes and after colony purification, plasmid DNA was prepared from this clone and sequenced by the method of Maxam and Gilbert Plasmid p These fragments were ligated to gel purified pBR322 digested with Plasmid p Plasmid p The synthetic DNA molecules F 26 , C 16 , B 31 , D 11 , E 13 and 4 24 shown in Fig. 1, were synthesized by the phosphoramidite method. The single strand 4 24 was prepared by using all four bases, at each site where X is indicated. Furthermore, silica was withdrawn from the synthesis of the 24mer, such that single stranded 21mers, 22mers, and 23mers are obtained in addition to the 24mers. After removal from the silica support, the four mixtures are combined in appropriate proportions to provide for equimolar amounts of each of the possible single strands. This mixture was treated as a single product in the subsequent steps. Molecules F 26 , C 16 , B 31 and D 11 were mixed together in equimolar amounts and 10µg phosphorylated using T4 polynucleotide kinase. After phenol ether extraction, the additional non phosphorylated synthetic DNA molecules 4 24 and E 13 were added, such that all fragments were equimolar. The equimolar mixture contained 13µg of DNA in 133µl of 0.3x kinase buffer. After annealing by cooling at a uniform rate from 70 C to 20 C over 60min, the single strands were ligated together with T4 ligase in 200µl ligation mix at 14 C for 4hrs, phenol chloroform extracted, ethanol precipitated and the 5 ends of 4 24 and E 13 phosphorylated using T4 polynucleotide kinase Maniatis The above purified fragment mixture was then ligated to the 460bp For analysis of total After electrophoresis resolution Laemmli, For enzyme assays, 100ml cultures were induced as described above. Soluble cell extracts were prepared by vortexing cell pellets 10x20sec. with an equal volume of 0.45 0.5mm diameter glass beads Braun AG and an equal volume of assay buffer 50mM Triscacodylate, pH 8.2, 1mM diethylenetriaminepentaacetic acid in a 1.5ml microfuge tube at 4 C. Cell debris was pelleted by spinning the tube for 1min. in a microfuge Beckman . The supernatant was dialysed against the assay buffer containing 1mM each of zinc sulfate and copper sulfate for 4hrs at 4 C. SOD assays were performed using the pyrogallol method Marklund and Marklund, The following Table 2 indicates the results for one of the clones examined pSODx8 These data indicate that approximately 15 of the total soluble cellular protein was hSOD assuming that the pure human Cu Zn SOD used as a reference was fully active . Taken together with the electrophoretic data see above indicating that 5 10 of total soluble cellular protein migrated as hSOD, it appears that a substantial fraction, probably a majority of the hSOD produced is active. The correct sequence of the cloned gene was determined by the method of Maxam and Gilbert, Thus, the N formyl methionine was removed from the bacterial expression product and the material had the correct amino acid sequences, i.e, identical to that reported for cytoplasmic hSOD residues 1 10, but the N terminal ALA residue was not acetylated. Furthermore, the polypeptide made in For transfer of the r hSOD gene to a yeast vector, the p The plasmid phSOD was digested with Plasmid pC1 1 is a derivative of pJDB219 Beggs, Plasmids pC1 1GAPSOD and pC1 1GALGAPSOD were transformed into yeast strain 2150 2 3 available from Lee Hartwell, University of Washington as described previously Hinnen et al. Proc. Natl. Acad. Sci. USA 1978 75 1929 1933 , with the results of expression set forth in the following Table 3. hSOD levels were measured using a standard radioimmunoassay with iodinated authentic hSOD as standard. Constitutive synthesis from the GAP promoter leads to very high levels of hSOD production, of the order of 10 30 of the total cell protein. The induction with galactose works almost as well, yielding about 7 of the cell protein as hSOD. When hSOD is produced at these high levels, it is usually necessary to provide zinc and copper ion to the product protein as a prosthetic group in order to recover full enzymatic, i.e., catalytic, activity, e.g., by dialysis against 1mM solutions of both zinc and copper sulfate. Alternatively, zinc and or copper ion may be included in ten growth media this method also provides a means of selecting for strains producing high levels of hSOD and or avoiding the loss of plasmid vectors expressing hSOD in otherwise non selective media. pGAP1, a plasmid prepared by insertion of a pGAP128 was digested with Yeast expression vector pPGAP having a polyrestriction site linker between the GAPDH terminator and short promoter region was prepared as follows. Plasmid plot5pGAP128 was digested with to provide a pBR322 was digested with Plasmid pLGSD5 is prepared as described in Guarente This fragment was inserted into pBR322 which had been completely digested with Plasmid pBRGAL4 was completely digested with The fragment was then cloned in the plasmid plot5. plot5 was prepared by inserting the 40bp polylinker fragment of the following sequence into pBR322 as an The phSOD polypeptide made in yeast was shown to be identical to the natural human protein. Migration of hSOD made in yeast was identical to the protein in both polyacrylamide gel electrophoresis with and without sodium dodecyl sulfate and in agarose gel electrophoresis see above . Moreover, when highly purified yeast polypeptide was subjected to twelve cycles of Edman degradation, the sequence was the same as that reported for the human protein residues 1 10 made in To isolate the human SOD gene, a bacteriophage lambda library representative of the human genome R. Lawn et al. 1978 Cell 15 1157 1174 was screened with a radiolabelled DNA probe made from the human SOD cDNA. One million phage plaques were screened, and 13 positively hybridizing plaques were purified. Restriction endonuclease analysis of the phage DNAs indicated that there are at least 5 different genes, suggesting that there are other SOD related genes and gene products. One candidate for such a gene is the recently discovered extracellular Cu Zn SOD S. Marklund, 1982 Proc. Natl. Acad. Sci. USA 79 7634 7638 . To distinguish the authentic cytoplasmic Cu Zn SOD gene from the related ones we used synthetic DNA probes 5 AATGCTTCCCCACACC 3 and 5 CTCAGTTAAAATGTCTGTTTG 3 corresponding to amino acid residues 19 26 and nucleotides 193 213 3 from the terminator codon in the 3 untranslated region, respectively. Only one of the 13 genomic DNAs hybridized with these probes, indicating that it is the authentic human cytoplasmic SOD gene. This was confirmed by DNA sequence analysis of the N terminal region, as shown in Fig. 5 where the amino acid sequence determined by protein sequencing is confirmed. This also shows that no preprotein exists for SOD since an in frame termination codon exists nine nucleotides 5 from the initiator methionine codon. As shown in Figure 5, the human Cu Zn SOD gene contains intervening sequences. The map of the SOD gene shown in Figure 6 indicates that there is more than one intervening sequence. Although the foregoing invention has been described in some detail by way of illustration and example for purposes of clarity of understanding, it will be obvious that certain changes and modifications may be practiced within the scope of the appended claims.