# Transfer vector.

## Abstract
A recombinat DNA vector comprises a a replication origin and a selectable marker b , a nucleotide sequence containing a promoter and a transcription initiation site, the transcription initiation mucleotide being the first transcribed nucleotide, c a restriction endonuclease recognition sequence whose cleav age site is on the 3 side of the transcription initiation nucleo tide downstream from the promoter, and d a fragment of DNA inserted at the cleavage site. The inserted DNA fragment may code for rDNA, tDNA or snDNA.

## Claims
What is claimed is 1. A bacterial strain containing and replicating therein a recombinant DNA vector comprising a a replication origin and selectable marker, b a nucleotide sequence containing a promoter and a transcription initiation site wherein a transcription initiation nucleotide is the first transcribed nucleotide, c a restriction endonuclease recognition sequence whose cleavage site is on the 3 side of the transcription initiation nucleotide downstream from said promoter, and d a fragment of DNA inserted at said cleavage site. 2. A bacterial strain as recited in Claim 1 wherein the promoter nucleotide sequence is modified to contain the restriction endonuclease recognition sequence. 3. A bacterial strain as recited in Claim 1 wherein said strain is Escherichia coli K12 JM101 or Escherichia coli K12 JM103. 4. A bacterial strain as recited in Claim 1 wherein said replication origin and selectable marker is pUC9 or M13mp9. . 5. A bacterial strain as recited in Claim wherein said nucleotide sequence containing a promoter is pPM1. 6. A bacterial strain as recited in Claim 1 wherein said restriction endonuclease recognition sequence is a blunt ended restriction endonuclease site. 7. A bacterial strain as recited in Claim 1 wherein said restriction endonuclease recognition sequence is a SmaI site. 8. A bacterial strain as recited in Claim 1 wherein said fragment of DNA is a cDNA molecule derived by reverse transcription from a viral RNA molecule. 9. A bacterial strain as recited in Claim 1 wherein said fragment of DNA codes for rDNA, tDNA or snDNA. 10. A recombinant DNA vector comprising a a replication origin and selectable marker, b a nucleotide sequence containing a promoter and a transcription initiation site wherein a transcription initiation nucleotide is the first transcribed nucleotide, c a restriction endonuclease recognition sequence whose cleavage site is on the 3 side of the transcription initiation nucleotide downstream from said promoter, and d a fragment of DNA inserted at said cleavage site. 11. A recombinant DNA vector as recited in Claim 10 wherein the promoter nucleotide sequence is modified to contain the restriction endonuclease recognition sequence. 12. A recombinant DNA vector as recited in Claim 10 wherein said replication origin and selectable marker is pUC9 or M13mp9. 13. A recombinant DNA vector as recited in Claim 10 wherein said nucleotide sequence containing a promoter is pPM1. 14. A recombinant DNA vector as recited in Claim 10 wherein said restriction endonuclease recognition sequence is a blunt ended restriction endonuclease site. 15 A recombinant DNA vector as recited in Claim 10 wherein said restriction endonuclease recognition sequence is a SmaI site. 16. A recombinant DNA vector as recited in Claim 10 wherein said fragment of DNA is a cDNA molecule derived by reverse transcription from a viral RNA molecule. 17. A recombinant DNA vector as recited in Claim 10 wherein said fragment of DNA codes for rDNA, tDNA or snDNA. 18. A process for constructing a universal transcription vector comprising the steps of a isolating a DNA fragment containing a promoter, b inserting said DNA fragment into the replicative form of a filamentous single stranded phage strain to produce a recombinant filamentous single stranded phage, c transforming a bacterial strain with said recombinant phage d isolating ssDNA of said recombinant phage excreted from said bacterial strain, e synthesizing an oligonucleotide primer partially complementary to said promoter, the 5 end of said oligonucleotide primer corresponding to a transcription initiation site and said 5 end having the sequence of one half of a blunt ended restriction endonuclease site f replicating said ssDNA using said oligonucleotide primer to produce a modified complementary DNA strand containing a modified promoter, the modified promoter being the promoter comprising a sequence complementary to said oligonucleotide primer, g replicating said modified complementary DNA strand to produce a modified double stranded DNA fragment containing said modified promoter and h cloning said modified double stranded DNA fragment into a vector plasmid resulting in a universal transcription vector. 19. A process according to Claim 18 wherein said DNA fragment containing a promoter is a cI PRM PR fragment from phage lambda. 20. A process according to Claim 18 wherein said filamentous single stranded phage strain is pUC9 or M13mp9. 21. A process according to Claim 18 wherein said bacterial strain is Escherichia coli K12 JM101 or Escherichia coli K12 JM102. 22. A process according to Claim 19 wherein said oligonucleotide primer is 5 OH d GGGACCATTATCACC 3 OH. 23. A process according to Claim 18 wherein said blunt ended restriction endonuclease site is a SmaI site. 24. A process according to Claim 18 wherein said vector plasmid is pUC9.

## Description
TRANSFER VECTOR The nucleotide sequences of a DNA molecule carry an information code which can be transcribed into an intermediate messenger mRNA followed by translation into a protein. Such a protein may have a catalytic or a structural function in the organism. In some cases the nucleotide sequence is transcribed into an RNA molecule and this RNA molecule is the functional entity. Natural examples are the ribosomal RNA molecules rRNA , transfer RNA molecules tRNA and small nuclear RNA molecules snRNA . Some viruses e.g., retroviruses are reverse transcribed and the DNA product is then incorporated into the host genome from whence it can be transcribed once again into viralRNA.A useful artificial situation would be to isolate a DNA fragment of exactly the same sequence as a useful RttA molecule and then to insert such a fragment precisely at the transcription initiation site downstream from a strong promoter sequence which in turn has been inserted into a vector plasmid. In such a case large quantities of the desired RNA molecule could be produced by in vitro transcription. Prior vector systems have provided insertion sites downstream from a promoter, but the RNA transcribed therefrom contains extraneous sequence at the 5 end. The function of the resulting RNA may be modified in an undesired or uncontrolled manner since many RNA functions are affected by the nucleotide sequence at the 5 end.This prior art difficulty could be overcome by a transcription vector providing for precise initiation of transcription of the inserted DNA segment. In this manner it is possible to produce wild type viral RNA sequences, mutated viralRNA sequences, viral RNA sequences in which some non essential nucleotide sequences have been replaced by useful foreign genes and wild type or mutated rRNA, tRNA or snRNA molecules. Mutated viral RNA sequences may be used to infect plant cells thereby preventing superinfection or viral RNA sequences partly replaced by foreign genes may be used as vehicles for the introduction of those genes into plant cells.Alternatively the metabolism of microbial populations may be disrupted by transformation of the population with the recombinant plasmid carrying mutated rRNA, tRNA or snRN. A method of constructing a recombinant plasmid carrying a vector, a strong promoter and s restriction endonuclease site at the transcription initiation point is described. This restriction endonuclease site is used to insert the desiredDNA fragment.Summary of the Invention The techniques of recombinant DNA have made it possible to transcribe and translate a variety of open reading frames in a DNA sequence. The efficiency of such transcription and translation depends on many factors, e.g., the strength of the promoter region, the structure of DNA sequences upstream from the transcription initiation site, and, in some cases, the DNA structure on the 3 side of the termination signal. In some cases it is useful to produce an RNA transcript with precise initiation and termination points, e.g., for the in vitro production of viral RNA molecules, small nuclear RNA molecules and rRNA molecules.This invention describes the production by use of advanced genetic engineering techniques of a recombinant plasmid containing a restriction endonuclease site at a point downstream from a strong lambda virus promoter PR where transcription from various DNA sequences can be precisely initiated. In addition an EcoRI restriction site occurs imnediately after the transcription initiation site so the use of EcoRI immediately prior to transcription in vitro will also precisely locate the termination point within 7 nucleotides of any DNA fragment inserted into the initiation restriction endonuclease site.Background of the Invention In the past several years a very large potential has developed in the field of genetic engineering for the transfer of foreign genes to plants. In order to achieve such a transfer, it is necessary to find suitable vectors.The most advanced work on transfer of genes to a plant genome has been achieved by the use of Agrobacterium tumefaciens and, to a lesser extent,Agrobacterium rhizogenes Leemans, J., et al. 1982 In Kail , G. and J. S.Schell 1982 Molecular Biology of Plant Tumors Ch. 21 5375 45, AcademicPress, New York . A recent study Murai, N., et al. 1983j Science 222 476482 showed that sequences coding for the bean seed protein phaseolin were inserted into the sunflower genome by the transferredDNA regions of tumor inducing plasmids. In one instance the phaseolin encoding sequences were controlled by the octopine synthase promoter and in another instance by the phaseolin promoter region.In both cases the phaseolin genes were correctly transcribed, processed and translated, thus demonstrating the expression of a plant gene after transfer to a taxonomically distinct botanical family. In addition, there is the potential of genetically engineering the normal plasmids of Rhizobium strains prior to the infection and formation of nodules on plant roots. Much work has already been done on the structure and function of nitrogenase genes Scott, K. F., Rolfe, B. G. and J. Shine 1983 DNA 2 141 148 Scott, K. F., Rolfe, B. G. and J. Shine 1983 DNA 2 149 156 . The promoter regions of the nitrogenase genes of these organisms have been used to control the transcription and translation of a variety of foreign genes inserted into the symbiotic plasmids of Rhizobium in such a manner as to be under the control of these promoter regions U.S.Patent Application SerialNo. 506,676, filed June 22, 1983 . Both types of vectors, i.e., the T DNA ofAgrobacterium 5 pup. and the symbiotic plasmids of Rhizobium spp. involve the transfer of foreign genes as DNA. Another possibility for the transfer of foreign genes to plant cells is by the use of RNA or DNA viruses. In the case of RNA viruses, it would be necessary to reverse transcribe them into cDNA molecules because of the technical difficulties of handling RNA molecules and the lack of restriction endonuclease active on RNA. In some cases where amplification by secondary infection cycles is possible, e.g., poliovirus Racaniello, V. and D.Baltimore 1981 Science 214 916 919 and potato spindle tuber virus Cress,D., Kiefer, M. and R. A. Owens 1983 Nucleic Acids Res. 11 6821 6835 the method appears feasible but in many examples, infectivity of viral cDNA molecules is low or nonexistent, and this approach does not produce results. To understand why these negative results may occur, it is useful to list the following probable requirements for infectivity of directly inoculated viral cDNA 1 cDNA uptake into the cell 2 cDNA transport into the nucleus 3 transcription of at least full length viral RNA s vRNA 4 avoidance of vRNA splicing 5 processing of vRNA termini to infectious forms 6 vRNA transport to the cytoplasm 7 translation of at least some viral proteins and 8 effective interaction of vRNA with viral and cellular proteins for a first round of replication. Although effective transcription is often considered to be the most critical feature, viral cDNA infectivity probably involves a number of additional requirements. For example, linkage of viral cDNA to a strong plant promoter might be a useful strategy but would. not give infectivity if the viral RNA contains a cryptic splice site which leads to its effective processing to a non infectious form. Also, a number of steps, e.g. 4 to 6 above, involve cellular processes which are poorly understood. Consequently, the infectivity of directly inoculated cDNA could be severely limited at a number of steps, e.g., vRNA transport to the cytoplasm. These defects of infectivity would be extremely difficult to identify and so the various defects would be very difficult to correct without significant advances in basic molecular and cellular biology. Several features are needed to turn a fragment of nucleic acid into a plant vector 1 The nucleic acid should be capable of being cloned so that useful quantities can be recovered 2 The nucleic acid should be able to replicate within a plant cell and preferably, should be recognizable by selective methods 3 It should not be pathogenic 4 The nucleic acid must be capable of incorporating other nucleic acid sequences into its structure and of expressing such incorporated nucleic acid and 5 If heritable changes of the transformed plant are wanted, then the vector or a derivative of the vector should be stably maintained from one generation to the next. Further, in the case of T DNA, at some stage the transformed plant cell will have to be purified by cloning and regenerated into a plant.When these features are considered in relation to RNA plant viruses, a number of difficulties become apparent 1 The techniques of genetically manipulating and recombining RNA are much more difficult than the techniques of genetically manipulating and recombining DNA 2 Since most virus particles are packaged into protein capsids, there may be difficulties in accommodating more than limited quantities of foreign nucleic acid 3 RNA viruses need to be replicated within the plant cell so there will be difficulties in recovery of genetically engineered RNA 4 The strain of RNA virus used to infect the plant cells should cause minimal pathogenic effects.Detailed Description of the Invention The key feature of the present invention is the construction of a recombinant expression vector with a strong promoter element and a restriction endonuclease site positioned downstream from the promoter so that transcription initiation always occurs at a precise nucleotide. A critical feature of the present invention is the discovery that, following the conversion of viral RNA molecules to cDNA by the use of reverse transcriptase, these cDNA molecules can be transcribed in vitro into RNA. Furthermore, such viral RNA molecules transcribed in vitro from cDNA molecules are infectious, provided that transcription, initiation and transcription termination are essentially at the correct nucleotides. To be useful in the transmission of foreign genes into plants, DNA fragments are incorporated into suitable positions on the cDNA molecule. For example, using brome mosaic virus which has a tripartite genome i.e., RNA1, RNAZ and RNA3 , it may be possible to delete the nucleic acid sequence coding for the coat protein in RNA3 and replace it with the sequence coding for a foreign gene. Such a deletion and replacement could most easily be done by use of the cDNA molecule. This strategy has several attractions a Gene 3a occurs as one of the two open reading frames of RNA3 and it is necessary for this gene to be present for viral RNAreplication within plant cells. The requirement for gene 3a function would serve to maintain the engineered component in the virus b The foreign protein would presumably be strongly expressed by the same mechanisms which lead to strong coat protein production in normal infection c The size of the inserted sequences would not be limited by encapsidation constraints d The lack of viral coat protein should limit the escape of engineered virus from deliberately inoculated plants. If both the 3a and coat genes encoded by RNA3 were required for infection, they could be separated onto two different components, one or both of which could also carry foreign genetic information.Up to 2kb of foreign sequence should be insertable in a single such component without violating packaging constraints. Many other viruses are useful for the in vitro production of infectiousRNA molecules by transcription from cloned cDNA, including various other plant viruses Ee.g., tobacco mosaic virus TMV , poliovirus, influenza virus, foot and mouth disease virus, and others. There is yet another benefit to be obtained from the in vitro transcription of viral mRNA molecules from cDNA. It is possible to produce mutations in the viral cDNA molecules that convert a virulent strain to d symptomless strain. Such symptomless strains are used for in vitro synthesis of viral mRNA which is genetically engineered to contain foreign genes.Infection of plants by such symptomless strains of virus can also prevent later superinfection of these plants by virulent strains of virus. Another use of a recombinant vector carrying a strong promoter and a restriction site positioned precisely at the start of transcription is the use of mutated rRNA molecules to combat bacterial infections. rRNA molecules are transcribed from rDNA. Thus, it is possible to mutate the 3 end of the bacterial 16S rRNA molecule in such a way that the rRNA takes its place in the ribosomes but is incapable of mediating translation of mRNA molecules.Multiple transformation of bacteria by a recombinant vector containing such a mutated rRNA severely retards or eliminates bacterial infections. Similarly, mutated tRNA s which are transcribed from tDNA s could be used for the multiple transformation of bacterial populations. Such mutated tRNA s can cause a severe disruption of the bacterial capability to translate mRNA s into needed proteins. Yet another use of the transfer vector described herein is the enhancement of translation of a mRNA transcribed from a eukaryotic gene in a transformed bacterium. It is well known in the art that tRNA populations of bacteria occur in different proportions compared to the tRNA populations of eukaryotes and that codon usage is different in the mRNA s of prokaryotes and eukaryotes. Translation of a mRNA transcribed from a eukaryotic gene in a transformed bacterium can be optimized by further transforming a variety of tDNA s under the precise control of a promoter into the transformed bacterium. In order to clone these various DNA molecules from which precisely initiated and terminated transcripts are required, it is necessary to construct a recombinant plasmid where the DNA to be transcribed is inserted at a precise distance downstream from the regulatory signals of a strong bacterial or viral promoter. Bacteriophage promoters frequently provide strong initiation signals because an infecting virus must redirect the cellular metabolism of a host to its own purposes. It has been shown that lambda PR promoter initiates transcription many times more frequently than bacterial promoters such as gal or lac Queen, C. and M. Rosenberg 1981 Cell 25 241 249 . In previous work to increase the production of proteins Queen, C. 819 3 J. Mol. Appl. Genet. 2 1 10 an expression vector was created by placing thePR promoter and cro gene ribosome binding site on a plasmid, along with a repressor gene for PR in such a way that the cro ribosome binding site could be easily linked to foreign genes. The effectiveness of this vector was shown by producing two proteins, B galactosidase and SN 40 small t antigen, at the revel of 5 10X of total E. coli protein. The level of t antigen achieved 4 x 105 molecules cell was about 10 times higher than was obtained with the lac system Thummel, S., Burgess, T. L. and R. Tijian 1981 J. Virol. 37 683697 or a chemically synthesized ribosome binding site Jay, G., Khoury, G.,Seth, A. and Jay, E. 1981 Proc. Nat. Acad. Sci.U.S.A. 78 5543 5548 and is 20 times higher than the level of interferon obtained using a triple trp promoter Goeddel, D. V., Shepard, H. M., Velverton, E., Leung, D. and R. Crea 1980 Nucleic Acids Res. 8 4057 4073 . This expression vector which carried the PR phage lambda promoter Queen,C. 1983 J. Mol. Appl. Genet. 2 1 10 was the starting point for creating an entirely novel vector, namely, a vector carrying a strong promoter with a restriction site downstream from the regulatory sequences of the strong promoter precisely at the transcription initiation site. Since the restriction site can be used to insert foreign DNA, this means that transcription initiation of any foreign DNA can be precisely positioned. In the construction of these transcription initiation vectors, one is not confined to the use of the PR lambda phage promoter. Any suitable promoter may be used such a promoter may be strong or weak and subject to regulation by activators and or repressors. A representative list of such promoters would include but not be restricted to the following list a Bacterial promoters such as galP1 Musso, R. et al. 1977 Proc. Nat. Acad. Sci. U.S.A.74 106 110 or trp Bennett, G. N. et al. 1978 J. Mol. Biol. 121 113 137 b Other phage promoters such as T7 Al Siebenlist, U. 1979 Nucleic AcidsRes. 6 1895 1907 , or those of phage SP6 Green, M. et al. 1983 Cell 32 681694 c Plasmid and transposon promoters such as pBR322bla Sutcliffe, J. G. 1979 Proc. Nat. Acad. Sci. U.S.A. 75 3737 3741 and TnSneo Rothstein, S. J.et al. 1981 Cell 23 191 199 and promoters created by fusion or mutation such as lacPl15 Johnson, R. C. et al. 1981 Nucleic Acids Res. 9 1873 1883 and IS2 I tI Hinton, D. M. and R. E. Musso 1982 Nucleic Acids ReP. 10 50155031 . In addition, completely synthetic promoters or promoters recognized byRNA polymerases from biological sources other than bacteria may be used. The vector used in the present invention for the insertion, propagation and manipulation of the BamHI Clal DNA fragment from the expression vector pCQY2 Fig. 1 was M13mp9 Messing, J. and Vieira, J. 1982 Gene 19 269276 . The M13mp9 in this example was chosen for ease and convenience. The use of this strain is not to be interpreted as excluding the use of other filamentous single stranded phages e.g., fl or fd . Other filamentous, single stranded phages could equally well be utilized in these experiments The Single Stranded DNA phages 1978 Denhardt, D. T., Dressler, D. and D.S. Ray, eds. Cold Spring Harbor Laboratory, New York . M13mp9 was derived from another M13 engineered phage, i.e., M13mp7. The cloning of DNA into the replicative form RF of M13 has been facilitated by a series of improvements which produced the M13mp7 cloning vehicle Gronenborn, B. and J. Messing 1978 Nature, Lond. 272 375 377 Messing, J. 1979 Recombinant DNA TechnicalBulletin, NIH Publication No. 79 99, 2, No. 2 43 48 Messing, J., Crea, R. ancP. H. Seeburg 1981 Nucleic Acids Res. 9 309 321 . A fragment of the E. coli lac operon the promoter and N terminus of the ss galactosidase gene was inserted into the M13 genome. This lac promoter operator insert codes for the first 145 amino acid residues of the ss galactosidase gene.During the infection of certain cell lines e.g., E. coli K12 JM101 and JM103 this information will complement certain deletion mutants of the ss galactosidase gene and restore ss galactosidase activity a complementation . These cells show a lac phenotype and can be identified by a blue colored plaque on a medium containing IPTG and X gal Malamy, M. H., Fiandt, M. and Szybalski, W. 1972 Mol. Gen. Genet. 119 207ff . In addition, a small DNA fragment synthesized in vitro and containing an array of restriction cleavage sites Ea multiple cloning site MCS was inserted into the structural region ofthe ss galactosidase gene fragment. In spite of these insertions the M13mp7 DNA is still infective and the modified lac DNA is able to encode the synthesis of a functional 6 galactosidase a peptide Langley, K. E., Villarejo, M. R.,Fowler, A. V., Zamenhof, P. J. and I. Zabin 1975 Proc. Nat. Acad. Sci.U.S.A. 72 1254 1257 . This synthesized DNA fragment in M13mp7 contains two sites each for the EcoRI, BamHI, Sal I, AccI and HincII restriction enzymes arranged symmetrically with respect to a centrally located PstI site. By chance, either strand of a cloned restriction fragment can become part of the viral strand. This depends on the fragment orientation relative to theM13 genome after ligation. The insertion of a DNA fragment into one of these restriction sites is readily monitored because the insertion results in a nor functional a peptide and the loss of ss galactosidase activity. Under appropriate conditions, e.g., when strains E. coli K12 JM101 or 103 are infected by M13mp7, the functional a peptide results in blue plaques a nonfunctional a peptide results in colorless plaques Messing, J. and B.Bronenborn 1978 In Denhardt, D. T., Dressler, D. and D. S. Ray eds. The Single Stranded DNA Phages. Cold Spring Harbor Laboratory, Cold SpringHarbor, New York, pp. 449 453 . M13mp7 has found wide application in the dideoxy nucleotide sequencing procedure Sanger, F., Nicklen, S. and A. R.Coulsen 1977 Proc. Nat. Acad. Sci. U.S.A. 74 5463 5467 . A disadvantage of M13mp7 was that the multiple cloning site MCS contained two of each kind of restriction endonuclease site thus allowing DNA restriction fragments to be inserted in both orientations. Therefore, two new ssDNA bacteriophage vectors, M13mp8 and M13mp9, were constructed Messing, J.and Vieira, J. 1982 Gene 19 269 276 . The nucleotide sequence of M13mp7, containing the multiple restriction sites, was modified to have only one copy of each restriction site and, in addition, single HindIII, Sma I and XmaI sites. Thus, DNA fragments whose ends correspond to two of these restriction sites can be forced cloned11 by ligation to one of these new M13 cloning vehicles that has also been cut with the same pair of restriction enzymes.M13mp8 and M13mp9 have the modified multiple restriction site region arranged in opposite orientations relative to the M13 genome and a given restriction fragment can be directly oriented by forced cloning. The use of both vectorsM13mp8 and M13mp9 guarantees that each strand of the cloned fragment will become the strand in one or the other of the clones and thus be extruded as ssDNA in phage particles. Initially, fusion of the lambda phage PR promoter to the pUC9 polylinker resulted in 21 nucleotides of lambda sequence and some additional pUC9 nucleotides appearing at the 5 end of the synthetic brome mosaic virus cDNA3 transcript Figs. 2 and 3 . In order to allow the production of synthetic RNA transcripts including viral RNA transcripts with correct 5 termini, a universal transcription vector, now designated pPM1, was created. This transcription vector pPM1 retained most elements of the PR promoter but the transcription initiation site coincided with the cut site of a unique bluntended restriction site the SmaI site in Fig. 2b . The construction of such a transciption vector became possible when examination of a the large number of promoter sequences revealed that it might be possible to generate a bluntended SmaI site at the position of transcription initiation by specific site directed mutagenesis without altering the promoter strength. Transcription ofDNA fragments blunt end ligated into this SmaI site was later demonstrated to begin at the first nucleotide of the inserted fragment. In particular, viral cDNA s were primed by terminally complementary oligonucleotides, blunt end ligated into this site and transcribed to give the correct 5 ends. Further, the initial form of this vector Fig. 3b also provided a unique EcoRI site immediately downstream of the SmaI site where transcription could be runoffterminated to give product RNA s with no more than seven additional nucleotides at the 3 end. To construct the pPM1 vector, the lambda cI PRM PR cartridge from pCQV2 Fig. 2 was subcloned into M13mp9. The resulting ssDNA was used as a template for DNA synthesis from a mismatched oligonucleotide designed to fuse the initiation site to the left half of the SmaI recognition sequence. The mismatched oligonucleotide was synthesized by an in house facility. The DNA copied from the mismatch primer was made double stranded, isolated and ligated into M13mp9. After verifying the sequence of the modified PR promoter i.e.,PM promoter , the new cI PRM PM cartridge was subcloned into pUC9 to give the final desired vector pPM1. Several non trivial technical difficulties were encountered in the use of the mismatch primer.It was found at first that the primer was complementary to a nucleotide sequence which was 96 residues upstream from the PR promoter and that successful priming was not achieved 3 nucleotides upstream from the PR promoter initiation site as was expected. The solution to this unexpected result was finally reached after an exhaustive series of experiments which eliminated other possibilities and showed that the initial cloning protocol failed at the stage of oligomer binding. A comprehensive consideration of possible mechanisms of failure to bind oligomer at the desired site and a computer analysis of possible complementary pairing situations culminated in the conclusion that secondary structure in the template was the most likely problem Fig. 5 . This discovery then led to the development of a novel strategy to drive binding to the desired site despite competition from intra molecular interactions.This novel strategy included use of reverse transcriptase in place of DNA polymerase I Klenow fragment Fig. 4 and complete denaturation followed by slow renaturation of the template in the presence of a high effective concentration of the mismatched primer. Finally, direct testing of this newly developed strategy using a wide range of primer concentrations to determine the required levels of primer resulted in effective initiation of DNA synthesis at the desired position, namely, 3 nucleotides upstream from the promoter initiation site. The recognition sequence of the SmaI restriction endonuclease is Sma I Promoter 5 CCC GGG 3 transcription start nucleotide Reading strand 3 GGG CCC 5 upstream downstream cleavage siteThe cleavage site occurs in the center between C and G. The reading strand is 3 G G G C C C 5 . The cleavage site of the SmaI restriction endonuclease is therefore on the 3 side of the transcription initiation nucleotide with respect to the reading strand. Again, with respect to the reading strand, upstream is defined as 5 3 and downstream is defined as 3 5 . In general, the critical feature in the construction of transcription initiation vectors is the distance between the promoter sequence and the blunt ended cleavage site of a restriction endonuclease. The cleavage site of the restriction endonuclease must be on the 3 side of the nucleotide which is used for transcription initiation on the reading strand such that the cleavage site separates transcribed from untranscribed nucleotides. The term transcription initiation nucleotide , as used herein, means the first transcribed nucleotide, i.e., the nucleotide corresponding to or complementary to, depending on the DNA strand the 5 terminal ribonucleotide of the RNA transcript. The next step is to clone exact cDNA copies of the various RNA molecules, i.e., the three genomic RNA s of brome mosaic virus or other viruses see supra , the 165 RNA of various bacteria and the snRNA s from eukaryotic cells. These steps can be accomplished by synthesizing oligonucleotides complementary to each end of the RNA for use as first and second strand cDNA primers. Once the cDNA s have been synthesized and cloned into the uniqueSmaI site of the novel pPM1 vector plasmid, it is then possible to synthesizeRNA which differ from their authentic natural RNA counterparts only by the addition of a maximum of seven nucleotides to the 3 end. Example 1 Construction and Screening of Phage Recombinants M13 PR1 9 Plasmid pCQV2 Queen 1983 J. Mol. Appl. Genet. 2 1 10 was cleaved with restriction endonucleases ClaI and BamHI and the fragment containing the phage lambda cItS pRM pR ATG sequences was isolated from a low melting point agarose gel Sanger et al. 1980 J. Mol. Biol. 143 161 178 and ligated into calf intestinal phosphatased AccI BamHI cut M13mp9 RF DNA using T4 DNA ligase.This DNA was used to transform competent E. coli JM101 cells and plated. Nine white plaques were selected and screened by the ddATP sequencing technique Sanger et al. 1982 JMB 162 729 773 . The resulting autoradiograph confirmed that eight of the nine phage clones contained the expected phage lambda DNA insert, the remaining clone being a deletion mutant of M13mp9. One of the M13 k recombinants, designated M13 PR1, was selected for further work.Example 2 Optimum Primer Concentrations for Mismatch Priming on M13mp91pPRl ssDNA The 15 mer mismatched oligonucleotide 5 OH d GGGACCATTATCACC 3 OH which will be referred to as either the PR oligonucleotide or the PR primer, was synthesized by the phosphite triester method This PR primer was designed in such a manner as to allow a precise alteration of the phage lambda PR promoter. A number of ways were used to test the priming of this PR primer onPR DNA.Because of the presence of the complementary stem and loop in the nucleotide sequence of interest see supra , the following experiments were designed to test whether complete denaturation of the M13mp9 pPR1 DNA template plus vast excesses of the PR primer could achieve priming at the lambda PR promoter and whether such primers annealed at this site could be extended by reverse transcriptase without extension of PR primers bound at the upstream cIts site. It will be noted Fig. 5 that when the PR primer is bound at the upstream cItS site that it cannot then pair at the extreme 3 terminus. Such a lack of pairing at the extreme 3 terminus should prevent extension by reverse transcriptase which lacks a 3 exonuclease activity. Therefore,M12 PR1 ssDNA was mixed with PR oligomer at 1, 10, 100, and 1000 fold molar excesses, annealed by the boiling slow cooling method Anderson et al. 1980 Nude. Acids. Res. 8 1731ff and used as a template primer complex in ddTTP sequencing reactions using reverse transcriptase Ahlquist et al. 1981 Cell 23 183 189 . Autoradiography of the final gel showed that under these conditions DNA synthesis was primed exclusively from the desired site within the PR promoter sequence. Efficiency of DNA synthesis increased steadily withPR primer concentration, being at the highest primer template ratio 1000 1 as efficient as synthesis from a 5 fold excess of the normal M13 lac sequencing primer Duckworth et al., Nucl. Acids. Res. 9 1691 1706 .Example 3 Use of the PR Oligonucleotide to Synthesize and Clone an Altered Promoter Sequence A 1000 fold molar excess of PR oligomer was mixed with approximately 29 M13 PR1 ssDNA and annealed by the boiling slow cooling method Anderson et al. 1980 Nucl. Acids Res. 8 1731ff . The resulting template primer complexes were extended by reverse transcriptase in the presence of all four dNTPs. The product DNA was isolated by ethanol precipitation and used in a second DNA synthesis reaction primed with an M13 lac reverse primer as described byHong Bioscience Reports 1 243 252 1981 . The resulting DNA products were cleaved with PstI, electrophoresed on a low melting point agarose gel, and the 800 1000 bp fraction, as assayed by the mobility of defined marker DNA fragments, excised and eluted Sanger et al. 1980 , J. Mol. Biol. 143 161178 . This DNA was then mixed with SmaI PstI cut M13mp9 RF DNA, treated with T4 DNA ligase, and used to transform competent E. coli JM101 cells.After plating, 30 white plaques were selected and screened by the ddATP sequencing method Sanger et al. 1982 JMB 162 729 773 . The pattern obtained for 16 of these recombinants was consistent with the desired construct in which the sequence of the lambda PR promoter element was linked directly to the vector Smal site. Since the purpose of the PM clone constructions was to obtain a particular altered promoter sequence i.e., a SmaI restriction endonuclease site , two of these recombinants, designated M13 PM86 and M13 PM87, were sequenced by the 2 , 3 dideoxynucleoside triphosphate method Sanger, F. et al. 1977 Proc. Nat. Acad. Sci. U.S.A. 74 5463 5467 . The nucleotide sequence in the vicinity of the altered PR promoter in both strains was exactly as desired, namely SmaI ... .TGCGTGTTGACTATTTTACCTCTGGCGGTGATAATGGTCCCGGGAATTCACTGGCC PR EcoRI lambda sequence M13mp9 sequence The nucleotide sequence changes that have been made at the SmaI site can be seen by a comparison of the above sequence with that of PB3PR13 Fig. 2 .The sequence of M13 PM86 was read for a further 210 bases upstream of the sequence shown above. It was then computer matched to the sequence of pCQV2 Queen, C. 1983 J. Mol. Appl. Genet. 2 1 10 without any discrepancies.Close visual examination of the autoradiograph revealed no sequence differences between M13 PM86 and M13 PM87.Example 4 Subcloning M13 PM86.87 Inserts into pUC9 For each of the two clones M13 PM86 and M13 PM87, RF DNA was digested with the restriction endonucleases PstI and EcoRI and the EcoRI PstI fragment containing the desired modified PR promoter was recovered following electrophoresis in low melting point agarose. The appropriate DNA band as judged by size was excised, the DNA extracted as previously described supra and ligated to EcoRI PstI cut pUC9 Vieira and Messing 1982 Gene 19 259 268 . This DNA was used to transform competent E. coli MJ101 cells which were plated on media containing ampicillin, X gal and IPTG. EcoRI andEcoRI PstI digests of minilysate DNA samples Ish Horowicz and Burke 1981 Nucl. Acids. Res. 9 2989ff from six white colonies from each ligation mix were examined by agarose gel electrophoresis.In each case the digests confirmed the expected size of the insert fragment and that only one such fragment had been inserted. One of these clones, an M13 PM86 derivative designated pPM1, was selected for further work. Transcripts of pPM1 were obtained by standard methods. Runoff reverse transcription of these transcripts showed that initiation on the PR promoter occurred precisely at the SmaI site as planned illustrated below . transcript start TGATAATGGTCCCGGGAATTCACTGGCC SmaI Example 5 Insertion of a Complete cDNA Copy of Brome Mosaic Virus RNA3 BMV RNA3 Into the Smal Restriction Endonuclease Cleavage Site of pPM1 The nucleotide sequence of BMV RNA3 is known Ahlquist, P. et al. 1981 J. Mol. Biol. 153 23 28 . Starting from this information, synthetic oligodeoxynucleotide primers for the first and second strands of BMV RNA3 cDNA were designed, synthesized and used to prime production of ds cDNA to BMV RNA3 Fields and Winter 1982 Cell 28 303 313 . Full length ds cDNA was isolated from an agarose gel supra and inserted into the SmaI site of pPM1. Clones in which the cDNA was oriented to give strand transcripts i.e., transcripts corresponding in polarity to encapsidated viral RNA were selected after screening by restriction digestion. Plasmid DNA from such selected clones was cut with EcoRI and transcribed in vitro by standard methods.Results obtained using runoff reverse transcription of the transcripts showed that transcription initiation occurred on the first nucleotide downstream from the SmaI cleavage site of the pPM1 transcription vector.