# ISOLATION OF A NOVEL INTERFERON GENE AND EXPRESSION THEREOF

## Claims
Verfahren zur Identifizierung eines Gens, Genfragments oder Allels, das in einer genomischen oder cDNA Bibliothek in Form von rekombinanten Vektoren enthalten ist, die in einer Wirtszelle enthalten sind, durch

## Description
This invention relates to methods and compositions useful in the rapid screening and isolation of genes from genomic libraries. The methods and compositions of the present invention employ small DNA probes, approximately seventeen bases in length, comprising sequences complementary to the gene sought to be screened for or isolated from said genomic library. In one aspect of the present invention, a novel human interferon gene is identified and isolated from a genomic library by methods utilizing small DNA fragments containing nucleotide sequences complementary to at least four known human interferon genes. In another aspect of the present invention, compositions and methods employing recombinant DNA techniques are provided for the replication of the novel human interferon gene in microorganisms. Interferons are proteins with tremendous clinical importance and potential. Although a great deal of research has been directed toward an understanding of the molecular mechanism of interferon induction and activity, progress had been slow due largely to the small amounts of pure interferon protein available for study. Recombinant DNA technology offers an opportunity to produce a large amount of both the interferon proteins and the interferon gene sequences. Recombinant DNA technology, however, requires isolation of interferon DNA sequences which is a formidable task since it typically means preparing DNA recombinants from cellular messenger RNA mRNA which is present at very low levels and is further complicated by the existence of an as yet unknown number of heterologous interferon proteins and, hence, mRNA molecules. In humans, for example, at least three distinct classes of interferons have been identified, fibroblast, leukocyte and immune interferon. Each class in turn contains several distinct interferon protein species now known to be structurally related within a given class and structurally distinct as between classes. At the present time, there are two alternative sources from which human interferon genes may be isolated for use in recombinant DNA technology, cDNA libraries prepared from mRNA isolated from human leukocytes synthesizing interferon and genomic libraries prepared from human chromosomal DNA. These libraries are created by inserting either enzymatically synthesized DNA or native human DNA fragments which contain interferon gene sequences into an appropriate vector such as a bacteriophage or plasmid. The insertion of human DNA sequences into such a vector results in the formation of a recombinant or chimeric vector. In recent years, several general methods have been developed which enable construction of such recombinant vectors. For example, U.S. Patent No. 4,237,224 to Cohen and Boyer describes production of recombinant vectors employing restriction endonuclease enzymes and methods of ligation. U.S. Pat. No. 4,237,224 is hereby incorporated by reference into the present specification. The recombinant vector so produced is then used to transform or infect cells in which the vector is compatible resulting in the introduction of the foreign DNA or genes into the cell. The recombinant vector must be capable of autonomous replication in the host cell and should contain at least one selective marker, a limited number of known restriction endonuclease cleavage sites and a high copy number. The resultant replication of the recombinant vector allows for the generation of multiple identical copies or cloning of said recombinant vector and, hence, the foreign genes inserted into said vector. Each clone or replicated recombinant vector contains a unique fragment of foreign DNA or foreign gene. The term library refers to the generation of multiple, distinct recombinant vectors from which a single, specific gene sequence may be selected. To date, human interferon genes have been isolated from cDNA libraries employing a heterogeneous mixture of cDNA probes containing interferon specific DNA sequences and from genomic libraries employing similar cDNA probes. Pestka, Typically, cDNA libraries are constructed from interferon specific mRNA isolated from cells producing interferon. This mRNA is present at very low levels in these cells and represents only those interferon genes actively expressed. Interferon genes which are not actively expressed are not retrievable by this method. Additionally, due to the normal cell processes of generating interferon specific message, the resultant mRNA molecules may no longer contain as a complete complement of nucleotide sequences as compared to the native genes from which the mRNA is generated. Of the eukaryotic genes isolated to date, many of these genes carry sequences, called introns , within their respective gene sequences. These introns are excised during mRNA transcription and processing so that the resultant mRNA molecules no longer contain the introns. A further loss of gene sequences may occur during the isolation of these very labile and nuclease sensitive mRNA molecules and employment in the synthesis of cDNA molecules to be contained within the cDNA libraries. The need to transcribe the mRNA molecules into cDNA molecules and then double stranded DNA which is inserted into the library may result in copying errors and thus possible alterations in the structure and function of the final interferon gene products. Furthermore, construction of both cDNA and genomic libraries may suffer losses in gene sequences during the insertion of the gene sequences into the vector employed to carry the gene library. Alternatively, genomic libraries contain cloned DNA isolated from cell genomes. Given enough clones, one can generate a library with a 99 chance of containing any gene. Additionally, these cloned genes will be present in their native and genetically complete form along with regulatory signals which may prove advantageous in eukaryotic, as opposed to prokaryotic, cloning systems. Since genomic libraries are generated from native, chromosomal DNA, no further copying is required prior to their insertion into the chosen vector. Furthermore, these inserted double stranded DNA fragments are relatively non labile and nuclease resistant. Hence, the preferred source for genes not actively expressed, expressed in a low copy number, genes for which there exists an uncertainty regarding its site of synthesis, or genes present in multiple copies or forms is a genomic library. Genomic libraries are prepared by mechanically shearing chromosomal DNA or cleaving said DNA with restriction endonucleases into 16 20 kilobase kb fragments and thereafter inserting said DNA fragments into an appropriate DNA vector as described in The Isolation of Structural Genes from Libraries of Eucaryotic DNA , One of the major tasks in isolating the desired genes from genomic libraries involves the screening of these libraries for said gene or genes. A preliminary screening method useful in identifying and enriching for those vectors containing foreign DNA sequences or genes employs insertional inactivation . This method is described by Bochner et al., The identification and isolation of clones containing specific genes presently requires the use of gene sequence specific probes. One such technique involves the screening of bacterial clones described by Grunstein and Hogness, The use of synthetic DNA fragments based on known protein sequence information requires the isolation and sequencing of the gene specific protein product. Once the protein has been isolated and sequenced, a given region of the protein is selected to provide the gene sequence which is then synthesized by the phosphotriester method described by Agarwal et al., To date, identification and isolation of mammalian genes from genomic libraries has required the use of synthetic DNA probes of greater than 80 nucleotides in length. Anderson et al. Once the gene is identified within the genomic library, the bacteriophage or plasmid, recombinant vector, containing the gene is isolated and replicated by standard methodologies to generate multiple copies of the recombinant vector. The present invention provides methods and compositions for the identification, isolation, cloning of a novel, mature human interferon gene. In another aspect of the present invention, methods and compositions are provided which allow for the rapid screening and isolation of mammalian genes from genomic libraries. The method of the present invention comprises synthesis of small DNA probes, approximately seventeen nucleotides in length, comprising sequences complementary to the gene or genes sought to be identified and isolated, labeling said DNA probes, thereafter hybridizing the labeled DNA probes to the genomic library employing an improved method for hybridizing the labeled DNA probes to recombinant vectors contained within a host cell, the improvement comprising filtration of the labeled DNA probes prior to their addition to the hybridization solution, employing said labeled DNA probes in the hybridization reaction, employing a device to simultaneously rotate and incubate the nitrocellulose filters during prehybridization and hybridization and adding an effective amount of dextran sulfate to the prehybridization and hybridization solutions, and thereafter identifying said gene in a manner consistent with the genomic library and label chosen. In one example of the present invention, at least two small, labeled DNA probes, approximately 17 nucleotides in length, are synthesized to contain distinct complementary sequences present in at least four of the eight described cDNA alpha interferon genes. These DNA probes are then employed to screen human genomic libraries for human alpha interferon genes. In a further aspect of the present invention, an improved method for hybridizing radiolabeled DNA probes to recombinant vectors contained within a host cell and affixed to nitrocellulose filters, the improvement comprising employment of at least one radiolabeled DNA probe of at least 17 bases in length, filtration of the labeled probe before addition to the hybridization solution, use of a device to simultaneously rotate and incubate filters during prehybridization and hybridization and addition of ten percent dextran sulfate to the prehybridization and hybridization solution. The methods and compositions of the present invention are particularly useful in the isolation of mammalian genes present in low copy number from genomic libraries and subsequent generation of large quantities of said genes and gene products for clinical diagnosis and therapy. FIG. 1 is a perspective view of the combined incubation and rotational device with a break away view of the motor and right angle bevel gears. FIG. 2 is an enlarged fragmentary cross sectional view of FIG. 1 taken along lines 2 2. FIG. 3 is a perspective view of the platform unto which the containers for hybridization reaction mixtures are fastened. FIG. 4 shows the complete base sequence for the human alpha interferon IFWA gene carried on the vector M13 mp8 along with the corresponding amino acid sequence for the protein for which it codes. S₁ denotes the first of 23 amino acids of the interferon signal peptide. 1 denotes the start of the mature interferon protein. FIG. 5 shows the complete gene sequence of the mature human alpha interferon WA gene. FIG. 6 shows the restriction endonuclease cleavage sites present on the vector M13 mp8 employed in the insertion of human alpha interferon genes IFWA and IF77 into said vector and restriction endonuclease cleavage sites useful in determining the insertion of said genes into said vector. The present invention relates to methods and compositions which allow for the rapid screening and isolation of known and novel mammalian genes from genomic libraries. Unlike previously described methods for isolating genes from genomic libraries which required the synthesis and employment of large, greater than 80 nucleotides, DNA probes or a mixture of heterogeneous probes identifying a single region of the gene sought to be identified and isolated, the method of the present invention employs small, approximately 17 nucleotides, DNA probes containing sequences complementary to distinct regions on the gene or genes to be identified and isolated. The employment of small DNA probes, as described in the method of the present invention, provides several advantages over previously described methods for screening and isolating mammalian genes from genomic libraries. The smaller DNA probes are easily and relatively rapidly synthesized by conventional DNA synthetic techniques, allow for selection of gene regions possessing high G C content which results in a tighter and hence more efficient binding in conventional hybridization screening processes, and allows for the independent identification of desired genes when coupled to duplicate plating procedures which eliminates the high rate of false positive gene identification in conventional hybridization screening processes. In one example of the present invention, two probes for human alpha interferon were synthesized, one by Bio Logicals, Inc. Toronto, Canada and one using a Bachem Gene Machine obtained from Bachemgentec, Inc. Torrance, California . The 17 base 17 mer sequences of each probe were based on sequence information of eight cDNA interferon clones Goeddel et al., Probe A was synthesized by the solid phase phosphotriester method and purified as described by Itakura et al. The minimum length of a DNA probe employed to identify and locate a given gene sequence or fragment or allele thereof within a DNA library is determined by both the binding energy of the probe and signal of the label attached thereto used to detect the formation of a stable hybrid. To date, under present conditions employed in hybridization procedures, at least eleven nucleotides are required to establish a stable hybrid. In the method of the present invention, the preferred length of a DNA probe detectable by a radiolabel incorporated into said probe is at least about seventeen nucleotides in length. The end labeled DNA probes were then separated from unincorporated γ ³²P ATP by gel filtration through a Sephadex G 50 column. The columns are typically 30 x 0.6 centimeters and are contained within 2 milliliter disposable pipettes. The columns are equilibrated with a buffer comprising 10mm Tris tris hydroxymethyl amino methane HCl, pH 7.5, and 1mM Na₂EDTA sodium ethylene diaminetetraacetic acid . Approximately 40 fractions comprising 0.05 milliliters per fraction are collected and those fractions containing the first peak of radioactive material eluted from the column are pooled and adjusted to a final buffer concentration 6 x NET which comprises of 0.9M NaCl, 6.0mM Na₂EDTA and 0.09M Tris HCl, pH 7.5, by adding an appropriate amount of 20 x NET. The end labeled DNA probes are then filtered through a 0.2 micron ACRODISC filter purchased from Gelman Sciences, Inc. Ann Arbor, Mich. which have been pretreated by filtering 50 to 100 micrograms of yeast transfer RNA purchased from Boehringer Mannheim, W. Germany in 10 mM Tris HCl, pH 7.5, 1.0mM Na₂EDTA through the filter. The specific activities of the two probes may range from 1 to 5 x 10⁸ counts per minute cpm per microgram of DNA with a preferred specific activity ranging from about 3 to about 5 x 10⁸ cpm per microgram of DNA. The human genomic library in Lambda Charon 4A, Lawn et al. A total of about 1.8 x 10⁵ to about 1.9 x 10⁵ pfu were screened using the plaque hybridization method of Benton and Davis, In an alternative embodiment, the plaque hybridization method of Benton and Davis 1977 includes the following modifications 1 use of one DNA probe approximately 17 nucleotides in length and homologous to a region in the gene, gene fragment or genes, gene fragments to be identified, 2 separation of said probe from contaminating and or interfering substances prior to its addition to the hybridization mixture, 3 use of a device to rotate filters and control the environmental temperature during prehybridization and hybridization, 4 use of an effective amount of dextran sulfate in the prehybridization and hybridization solution, and 5 a parallel hybridization with said probe on duplicate filters of the plaques being screened. The separation of the radiolabeled DNA probes from contaminating and or interfering substances prior to their addition to the hybridization solution may be achieved by, for example, filtration which removes particles having a greater solution diameter than the probes. Such filtration appears to remove substances which otherwise cause non specific adherence of these probes to the nitrocellulose filters and thus reduces the background found when the probes are not previously filtered. These substances are hereinafter collectively referred to as interfering substances. A preferred method for separating interferring substances from said probes employs an inert filter with a pore size of approximately 0.2 micrometers or microns. An example of one such filter useful in separating DNA probes from such interfering substances is the 0.2 micron ACRODISC filter made by Gelman Sciences, Inc. Ann Arbor, Mich. . Dextran sulfate was added to the prehybridization and hybridization solutions to reduce non specific binding of the DNA probes and remaining interfering substances to the nitrocellulose filters. It is further known that the rate of association of nucleic acids is accelerated in the presence of dextran sulfate because the nucleic acids, herein the DNA probes, are excluded from the volume of solution occupied by the polymer. The effective concentration of the DNA probes is therefore increased. An effective amount of dextran sulfate was added to the hybridization solution to reduce the final solution volume resulting in an increased solution concentration of DNA probe or probes present in the hybridization solution. This increased DNA probe concentration presumably maximizes the contact between the probes and the complementary DNA sequences contained within the DNA on the nitrocellulose filters and thus increases the rate of association. Wahl et al. Duplicate nitrocellulose filter copies were made from each plate in accordance with the method described by Benton and Davis, The sealed pouches were placed in a rotation device shown in FIGS. 1 3 which simultaneously rotates the pouches 360 and provides a controlled temperature environment. As shown in FIG. 1, a conventional electric motor 10 with its own on off power switch 11 and power cord 12 and cover 13, is mounted by appropriate fastening means 14 midway on one outside wall 21A of a conventional oven 20, such as a Stable Therm Gravity Oven purchased from Fisher Scientific Pittsburgh, Pa. . Said oven 20 being capable of maintaining a constant temperature with a termperature range of from ambient to about 250 C. A Temperature indicator 30 denotes the temperature inside the oven 20. A front facing panel 22 located below a front opening door 23 contains a main power on off switch 22A for said oven 20, a temperature indicator on light 22B and a temperature selection and control device 22C. The oven 20 has its own power cord 28. As show in in FIG. 2, an axle 40 capable of continuous circular rotation about its central axis 72 and comprising, for example, a metal rod having a first end 41 and second end 42, runs horizontally through the center of the oven chamber 50 so as to run parallel to the floor 51 of the oven chamber 50 and perpendicular to the oven chamber 50 inner side walls 21B and 24B. The position and rotational movement of the axle 40 is maintained by securing the second end 42 to the inner side wall 24B by means of a bearing 45, said bearing 45 being attached to the inner side wall 24B by appropriate fastening means 45A, and by securing the first end 41 comprising driven beveled gear 43 to the inner side wall 21B by means of a bearing 46A, said bearing 46A being attached to the inner side wall by appropriate fastening means 48A, and to the outside wall 21A by means of a bearing 46B, said bearing 46B being attached to the outside wall 21A by appropriate fastenings means 48B, as shown in FIGS. 2 and 3 in the direction of arrow 74. The circular rotation, shown in FIG. 2, of the axle 40 is achieved by meshing the motor 10 powered driving beveled gear 60 with the driven beveled gear 43. The axial movement of the axle 40 along axis 72 is prevented by means of a collar 47A and set screw 47B positioned at the first end 41 of the axle 40 and a collar 44 and set screw 44A positioned at the second end 42 of the axle 40 as shown in FIG. 2. A platform 70 for mounting the sealed hybridization pouches 65 to the axle 40 is shown in FIGS. 2 and 3. The platform 70 comprises any suitable material, such as styrofoam or wood, to which the sealed hybridization pouches 65 may be securely mounted by such fastening means 66 as needles, pins, tacks, nails or the like, without puncturing the fluid filled portion of the sealed hybridization pouches 65. The platform 70 is attached to the axle 40 by a suitable locking means 67 as shown in FIG. 2, for example nuts and bolts and the like so that the platform 70 will circularly rotate in conjunction with the circular rotation of the axle 40. In one process conducted under the teachings of the present invention, the pouches were rotated at 10 revolutions per minute rpm at 37 C for about 12 to 24 hours. The rotation of the filters in the device of FIGS. 1 3 optimizes the movement of the prehybridization solution over the nitrocellulose filters. Following the incubation period, a corner of the pouch was cut open and the prehybridization solution was removed by aspiration. Two milliliters of solution, identical to the prehybridization solution, was added plus radiolabeled probe to a concentration of about one to about ten nanograms per milliliter with a preferred probe concentration of about two nanograms per milliliter of solution. In the modified hybridization procedure employing a single DNA probe, herein either Probe A or Probe B, a single probe is added to the hybridization solutions applied to the duplicate filters. In the present example of the invention, both Probes, A and B, were employed so that only Probe A was added to one filter hybridization reaction and Probe B was added to the duplicate filter hybridization reaction. Hybridizations were performed at 37 C for about 20 to about 24 hours with constant rotation as described for the prehybridization incubation above. Following the incubation period, the nitrocellulose filters were removed from the bags and placed in a suitable size pyrex dish containing 250 milliliters of wash buffer comprising 6X SSC, wherein 1X SSC comprises 0.15M sodium chloride, 0.015M sodium citrate. The filters were washed four times, twenty minutes per wash, in 250 milliliters of wash buffer, at room temperature with gentle rocking and, thereafter, washed four times in 250 milliliters 6X SSC containing 0.1 SDS at 45 C for about one hour. The nitrocellulose filters were then air dried at room temperature for about 30 minutes. Autoradiography was then performed using Kodak XAR film plus two intensifying screens, Dupont Cronex Hi Plus , for about 64 to 72 hours at minus 80 C. The filter and overlying Kodak XAR film were sandwiched between the two intensifying screens. Bacteriophage plaques which hybridized to the DNA probe were identified by common autoradiographic signals on duplicate filters hybridized with probes A and B, respectively. By employing and comparing the signals of a single DNA probe or two sequentially different DNA probes in parallel hybridizations conducted on duplicate filters, one may eliminate false positive hybridization signals typically occurring in a single hybridization reaction. The positive plaques were then isolated by several cycles of plaque purification as described by Benton and Davis, Due to the high density bacteriophage plating, approximately 1 x 10⁴ pfu per 80 square centimeter dish, during the initial screening of the Lambda Charon 4A genomic library, the isolated phage which hybridized to both probes, A and B, were subjected to a second infection and plating with The plaque purified clones, IF77, IFWA and IF 105, which hybridized to both probes, A and B, were then subjected to further analysis to localize and isolate the specific gene or genes, in the example herein, human alpha interferon. Localization of specific genes or gene fragments contained within a recombinant vector is typically conducted by fragmenting the recombinant vector with restriction endonuclease enzymes. Any restriction enzyme or combination thereof may be used. Ideally, the restriction enzymes employed should not destroy the gene or portion thereof sought to be isolated. The restriction enzymes used for gene localization and isolation are therefore selected with the gene location gene sequence, if known, and cloning vector in mind. Previous studies with alpha interferon genes isolated from cDNA libraries, Pestka, In the example of the present invention, clones IF77, IFWA and IF105 were subjected to analysis by restriction endonuclease cleavage with restriction enzymes BamHI, Ddel and EcoRI, purchased from Bethesda Research Laboratories, Inc. Gaithersburg, Md. followed by Southern hybridization, Southern Specifically, bacteriophage DNA from the clones was isolated as described by Maniatis et al. Once the gene, gene fragment or allele is isolated, the gene is inserted into an appropriate cloning vector which provides a means for replicating the gene. Any appropriate cloning vector containing a marker function may be used, for example, Insertion of the gene, gene fragment or allele into the cloning vector to create a recombinant cloning vector is accomplished by cleaving the cloning vector with restriction enzymes which yields cohesive ends that are complementary to those of the gene, or so modifying these ends to create complementary sequences, and then ligating the two fragments. Absent cohesive ends on the gene or gene fragment or following restriction enzyme cleavage, the gene may be blunt end ligated to the cleaved cloning vector. Such cloning vectors may also be employed to replicate wholly synthetic genes. Once ligated, the recombinant cloning vector is used to transform or infect the appropriate host cells. The host cells chosen must allow for the replication of the recombinant cloning vector and, preferably, in high copy number. In the example of the present invention, M13 mp8, describd by Messing et al. In the example of the present invention, the 2kb and 1.8Kb EcoRI fragments from bacteriophage clones λWA and λ77, respectively, were inserted into the unique EcoRI site on the M13 mp8 vector to create recombinant cloning vectors by the method described by Messing et al. Transfection of Approximately 100 to 500 M13 bacteriophage plaques were then isolated. The bacteriophage DNA from each plaque isolated was then individually screened for the presence of alpha interferon specific DNA by hybridization with one of the end labeled probes, probe A, as described for the initial bacteriophage screening. M13 mp8 bacteriophage DNA which hybridized to probe A was isolated, as described above, and subjected to restriction endonuclease cleavage with EcoRI to confirm the insertion of IFWA and IF77 into the M13 mp8 vector. The positive hybridization of probe A to the recombinant M13 mp8 vector DNA identifies only those recombinant vectors which contain alpha interferon gene sequences, IFWA or IF77, in the proper orientation. To further confirm the isolation of interferon specific gene sequences and to facilitate the subsequent insertion of said gene sequences into cloning and expression vectors, DNA sequence analysis of the entire isolated gene or gene fragment, was performed by sequencing the individual restriction endonuclease fragments generated from the isolated clones by the DNA sequencing method of Maxam and Gilbert, The partial DNA sequencing of IF77, approximately 80 of the gene sequence, strongly suggests that IF77 is identical to human alpha interferon L isolated from a human cDNA library as described by Pestka,