# NOVEL EXPRESSION PLASMIDS CONTAINING THE FULL CDNA SEQUENCE OF CALF PROCHYMOSIN

## Claims
Rekombinantes Expressionsplasmid mit der vollen cDNA Sequenz von Prochymosin und einem

## Description
This invention relates to recombinant expression plasmids capable of expressing prochymosin gene. More particularly, it relates to novel expression plasmids containing the full cDNA sequence of calf prochymosin and capable of efficiently expressing the full length cDNA in Prochymosin is secreted by the calf fourth stomach abomasum , and is a precursor protein of chymosin which is responsible for coagulating milk in the manufacture of cheese. Prochymosin has a molecular weight of about 40,000, and consists of 365 amino acid residues. A peptide consisting of 42 amino acids was cleaved from the N terminus of prochymosin by self digestion, resulting in the formation of chymosin. Recently milk coagulating enzymes, of microbial origin, have been discovered and these are substituting for chymosin. Nevertheless the need still exists for calf prochymosin in the cheese industry. Accordingly, it has been considered one of the primary objectives in the application of Genetic Engineering to utilize the recombinant DNA technology and to have microorganisms produce the calf prochymosin. It is one objective of this invention to provide novel expression plasmids capable of realizing the expression of the full length cDNA of prochymosin with high expression levels under the control of It is another objective of this invention to provide methods for the production of prochymosin which comprises transforming host organisms with the above stated expression plasmids and growing the thus obtained transformants. It is a still further objective of this invention to provide methods for the construction of the above stated expression plasmids. By employing the recombinant DNA technology, prochymosin cDNA is made to express by transformation in In one aspect of this invention there is provided a recombinant expression plasmid comprising the full cDNA sequence of prochymosin and an In another aspect of the invention there is provided a method for the preparation of novel plasmids with high expression efficiency which comprises slicing by restriction enzymes the base sequence of pCR 701 between the SD sequence and the ATG codon ligating the cut ends and thereby altering the constitution of the base pairs between said SD and ATG sequences e.g., the arrangement of base pairs and or the number of intervening base pairs . In this method, it is possible to lengthen the spacing between the SD sequence and the ATG initiation codon by cleaving with restriction enzymes, repairing the resultant single stranded termini, and ligating the cut ends. Conversely, it is also possible to shorten said spacing by removing the single stranded termini and ligating the cut ends. Preferably, the latter method i.e., shortening the spacing is employed in this invention. In another aspect of this invention there is provided a process for the production of prochymosin comprising the steps of introducing the herein described plasmids that have high expression levels into a host microorganism by transformation and cultivating the transformed host cells. Also included in this invention are the above transformed host microorganisms and prochymosin produced by them. Any microorganism capable of accepting, replicating the expression plasmids constructed by the processes of this invention, and producing prochymosin upon propagation of cells can be used as a host microorganism but for practical reasons, The prochymosin produced by a process according to this invention may be methionylprochymosin having methionine at the N terminus of prochymosin but may simply be referred to as prochymosin throughout the Specification. In the Drawings, Fig. 1 shows a restriction enzyme map of pCR 701. Fig. 2 shows a flow chart for the construction of recombinant plasmids, pCR 711, pCR 712, pCR 713, and pCR 714, beginning with the parental plasmid pCR 701 and base sequences between the SD sequence and the ATG initiation codon for each plasmid. This invention will be described in more detail by referring to preferred embodiments. Plasmid pCR 701 is digested with HindIII. The resultant single stranded portions at both cut ends are repaired by means of DNA polymerase. The resultant blunt ends are ligated together by T₄ DNA ligase. After these operations the spacing between the SD and the ATG initation codon is increased by 4 base pairs to become 18 base pairs. The thus produced plasmid, designated pCR 711, is identical with pCR 701 except the base sequence between the SD sequence and the ATG codon. the DNA sequence of this plasmid in the region between them, as analyzed according to the method of Maxam Gilbert, is Construction of pCR 711 from pCR 701 is schematically represented in Fig. 2. As previously described for pCR 711, plasmid pCR 701 is digested with HindIII. The resultant single stranded ends are removed by nuclease S1. The thus obtained blunt ends are ligated together by T₄ ligase. By varying the conditions for digestion with Nuclease S1 namely, composition of the buffered solutions , plasmid pCR 712 or pCR 713 is produced respectively. Both plasmids differ from each other in the distance between the SD and ATG sequences, but are identical with pCR 701 except said distance. The DNA sequence of pCR 712 in the region between the SD and the ATG initiation codon, as analyzed according to the method of Maxam Gilbert, is In like manner, the corresponding DNA sequence of pCR 713 is analyzed to be Construction of pCR 712 or pCR 713 from pCR 701 is schematically represented in Fig. 2. Among host cells transformed with pCR 712, amplicillin resistant clones are selected. These clones are all found to have pCR 712 and will produce prochymosin under the control of the A transformant, designated as Plasmid pCR 712 is digested with ClaI. The resultant single stranded portions at cut ends are partially repaired by DNA polymerase in the presence of dGTP and dCTP. When only the CG site is repaired and the partially repaired ends are ligated by means of T₄ DNA polymerase, plasmid pCR 714 is produced. After these operations, the number of base pairs between the SD sequence and the ATG codon is reduced by 2 base pairs to become 6 pairs. Except this base sequence, pCR 714 is exactly the same as pCR 712. The DNA sequence of pCR 714 in the region between the SD sequence and the ATG initiation codon, as analyzed according to the method of Maxam Gilbert, is Construction of pCR 714 from pCR 712 is schematially represented in Fig. 2. Among host cells transformed with pCR 714, ampicillin resistant clones are selected. These clones are all found to have pCR 714 and will produce prochymosin under the control of the The prochymosin production levels of the other plasmid containing strains are in the order of It is deduced from the above results that among pCR 701 derived expression plasmids the expression efficiency of prochymosin is not necessarily contigent upon the spacing between the SD sequence and the ATG initiation codon but also upon the base sequence itself. Prochymosin derived from microorganism can be activated to the active chymosin by exposure to acidic pH. This invention will be described in some more detail by way of illustration and example nevertheless, it should be understood that certain changes and modifications be practiced within the scope of the invention. Throughout the following examples, TEN buffer is meant a buffered solution containing Tris Hcl 20mM , NaCl 50mM and EDTA 1mM with a pH of 7.5. Nutrient media for transformation and expression experiments are Plasmid pCR 701 was digested with HindIII by incubation at 37 C for 2h in a 100µl solution containing Plasmid 10 µg in 32 µl TEN , HindIII 30 U, 5µl TEN , HindIII buffer 5 µl and H₂O 53 µl . The resultant DNA was precipitated with ethanol and dissolved in 20 µl of TEN. The digested pCR 701 was repaired at its cut single stranded portions by incubation at 30 C for 10 min in a 40 µl solution containing Plsmid 4 µg in 8 µl TEN , DNA polymerase I Klenow fragment 1.5 U, 1 µl TEN , polymerase buffer 4 µl , 5mM dNTP 4 µl TEN and H₂O 23 µl . The reaction was terminated by phenol treatment phenol was removed by ether extraction and the DNA was then precipitated with ethanol and dissolved in 20 µl H₂O. To this solution containing 4 µg of DNA was added T₄ DNA ligase 3.6 U, 4 µl TEN , 3mM ATP and x5 ligation buffer 8 µl , resulting in a final volume of 40 µl. After incubation at 22 C overnight, the reaction was terminated the DNA was precipitated with ethanol and the precipitated DNA was dissolved in 40 µl TEN. Transformation of the host The HindIII digest of pCR 701 obtained by the procedure of Example 1 was incubated at 16 C for 2h in a total volume of 50 µl containing the digest 1 µg in 2µl TEN , Nuclease Sl 50 U, 1 µl TEN , Nuclease S1 buffer 5 µl and H₂O 42 µl . The Nuclease S1 buffer x 10 contained 2.5M NaCl, 0.5M AcONa, 10 mM ZnSO₄, and 5 glycerol, with pH 4.5. The reaction was terminated by phenol treatment phenol was removed by ether treatment the resultant DNA was precipitated with ethanol and the DNA was then dissolved in 19 µl H₂O. A mixture containing the DNA 1 µg, 19 µl TEN , T₄ DNA ligase 0.9 U, 1 µl TEN , 3 mM ATP 3 µl, 50 PEG 6000 4 µl , and x10 ligation buffer 3 µl was incubated at 22 C for 90 min in a final volume of 30 µl. In a similar manner to Example 1, the ligated DNA was introduced into The HindIII digest of pCR 701 obtained by the procedure of Example 1 was incubated at 22 C for 30 min in a total volume 50 µl containing the digest 1 µg, 2 µl TEN , Nuclease S1 50 U, 1 µl TEN , Nuclease S1 buffer 5 µl , and H₂O 42 µl . The composition of the buffer solution was slightly different from that used in Example 2 namely, it consisted of 0.3M AcONa, 0.5M NaCl, and 10mM ZnSO₄, with pH 4.6. The reaction was terminated by phenol treatment phenol was removed by ether treatment and the DNA was then precipitated with ethanol. A mixture containing the DNA 3 µg, 20 µl TEN , 3mM ATP 8 µl , T₄ DNA ligase 3.6 U in 4 µl , and x5 ligation buffer 8 µl was incubated at 22 C overnight. The resultant DNA was precipitated with ethanol. As previously described in Example 1, the ligated DNA was introduced into Plasmid pCR 712 obtained in Example 2 was digested with ClaI by incubation at 37 C for 2h in a 50 µl solution containing Plasmid 2 µg, 15 µl , ClaI 18 U, 3 µl , ClaI buffer 5 µl , and H₂O 27 µl. The digested DNA was precipitated with ethanol and then dissolved in 41.5 µl of H₂O. To this solution was added 20mM dCTP 1.25 µl , 20mM dGTP 1.25 µl , DNA polymerase I Klenow fragment 1.5 U, 1 µl TEN , polymerase buffer 5 µl , resulting in a total volume of 50 µl. Incubation continued at 30 C for 10 min. The reaction was terminated by phenol treatment phenol was removed by ether extraction the resultant DNA was precipitated with ethanol and the precipitate was dissolved in 28 µl of H₂O. To this solution was added 3mM ATP 6 µl , T₄ DNA ligase 1.8 U, 2 µl and ligation buffer 4 µl to make up a final volume of 40 µl. The solution was incubated at 22 C overnight. The resultant DNA was precipitated with ethanol and the precipitate was dissolved in 40 µl of TEN. As previously described in Example 1, the ligated DNA was introduced into An The above sonicated cell free extract was treated with 4M urea. After centrifugation the supernatant solution was applied to SDS polyacrylamide gel electrophoresis. The migrated proteins were blotted onto nitrocellulose filters. Protein bands combined with the prochymosin antibody were detected by autoradiography. For each bacterial extract, a distinctive band corresponding to that of prochymosin was observed. By comparing the density of these protein bands with that of the authentic prochymosin, the amount of prochymosin produced by a specific strain was estimated. The ability of transformed strains containing the expression plasmids was thus in the order of pCR 712 pCR 701 pCR 711 pCR 714 pCR 713. As compared to the pCR 701 containing strain, the transformant containing pCR 712 showed the expression efficiency of about twenty five times.