# CLONING OF DNA FOR PROTOZOAL ANTIGENS

## Claims
Klonierte DNA Sequenz, die im wesentlichen der Sequenz entspricht, die bei Nucleotid 273 der DNA Sequenz der Fig. 1 beginnt und ein Polypeptid mit einem Molgewicht von 83 000 codiert, das vom Aminoterminus des P.195 Proteins von P.falciparum abgeleitet ist.

## Description
The present invention relates to the cloning of the P,195 gene of Malaria is an increasing health problem throughout the third world. Several hundred million people suffer from the disease and the most acute form, caused by the protozoan parasite In a rodent malarial model it has been shown that a protein antigen synthesised within the nature intraerythrocytic form, the schizont, and expressed on the surface of the merozoite, can be purified and that vaccination with this antigen can generate protective immunity against By molecular weight is meant the apparent relative molecular weight as determined by polyacrylamide gel electrophoresis in the presence of sodium dodecyl sulphate and standard molecular weight markers. The molecular weight of the antigenic proteins of the invention may thus be conveniently determined by the techniques described by U.K. Laemmli, When the protein was used to vaccinate mice the protection was adjuvant dependent and it appeared to be provided by a cell mediated effector pathway Freeman, R.R. and Holder, A.A. Clin. Exp. Immunol. Analogous protein antigens have also been described in other In The term P.195 is used herein to denote a protein of between 1.8 and 2.3 x 10⁵ MW which is localised in the erythrocytic schizont form of a Within The term epitope as used herein denotes an immunogenic determinant of an immunogenic molecule, the immunogenic determinant comprising a molecular configuration capable of eliciting a protective immune response in a susceptible animal, when presented in a suitable form. It will also be appreciated that P.195 is subject to allelic variation, wherein different strains of A comparison of the P.195 gene from the For the reasons outlined above, it is believed that the In order to obtain this protein or an antigenic fragment thereof,such as those produced We have now discovered that it is possible to clone the DNA sequence that substantially encodes the above described antigen P.195 from According to one feature of the present invention we therefore provide a cloned DNA sequence substantially encoding the P.195 protein of The term cloned is used herein to indicate any DNA sequence that has been substantially synthesised outside of its natural host. The term peptide as used herein pertains to any molecular structure composed essentially of amino acids comprising more than 2 amino acids. It will be appreciated that P.195 is a peptide by this definition. It will be appreciated that the DNA sequences of this invention may correspond to naturally occurring sequences, or they may be related to such sequences by mutation, including single or multiple base substitutions, deletions, insertions and inversions, always provided that the DNA molecule comprising such a sequence is capable of being expressed as the P.195 protein of The invention will be described in more detail hereinafter with reference to the accompanying drawings in which Figure 1 shows the base sequence of a stretch of Figure 2 illustrates a cDNA restriction map of the P.195 gene. Figure 3 illustrates a restriction map of the P.195 genomic sequence. Figure 4 illustrates the construction of plasmids used in the course of exemplifying the invention. Figures 1A to 11 show in sequence the nucleotide sequence of the P.195 gene, the amino acid sequence for which it codes and stretches of sequence at either end of the coding sequence. The lower line of each double line of letters represents nucleotides according to convention, while the upper line shows the amino acid sequence for which the open reading frame codes, the letters representing amino acids according to convention. The sequence was determined by methods described herein and it will be appreciated that, while it has been determined as accurately as possible within the margins of experimental error, there may be some variation in the P.195 genetic sequence. Figure 2 shows the gene, including sequence in the cDNA clones extending beyond either end of the coding sequence, at the top of the figure thick line with important restriction enzyme sites marked. All restriction enzyme codes in this figure are according to convention. Other restriction enzyme sites are indicated on parallel lines below, to simplify the diagram. The blocked in lines at the bottom of the figure indicate the positions relative to the P.195 gene from which various plasmid inserts derive, all being cDNA inserts except G1, which represents a genomic insert. X and Y indicate the putative 5 and 3 ends of the P.195 coding sequence, respectively. Figure 3 illustrates a restriction map of the stretch of genomic DNA including the P.195 gene. The scale is in Kbp and takes as its reference point a Hind III restriction site of the P.195 gene indicated by the bold capital H in the figure. The other restriction enzyme sites are coded as follows E EcoR1 , R Rsal , A Alul , M Mbol , Pv PvuII , N Ndel , T Taql , B BamHl and P Pstl . The stretches of sequence shown below the restriction map are those that were found to hybridise to the specific clone indicated. The number in brackets is the cDNA clone to which the segment hybridised, the number always corresponding to the relevant pPFc clone. Where there is also a number outside the brackets followed by two of the above indicated letters, this indicates that the probe used was less than the whole clone, the number indicating the length of the fragment in Kbp and the letters indicate the restriction enzymes used to generate the said fragment. Figure 4 illustrates the construction of 2 plasmids used to express fragments of P.195 DNA. pWRL507 has a number of characteristic restriction sites as indicated in the figure, and characteristic gene functions wherein Ptrp, The above mentioned DNA sequence may be characterised as having substantially all or part of the sequence shown in Figure 1, subject to the conditions noted above and further subject to considerations of experimental errors consequent upon determining such a sequence. The above mentioned DNA sequence may also be characterised as having substantially the restriction maps as shown in Figures 2 and 3, determined as described herein. The genetic sequence was determined by the chemical cleavage method Maxam, A., and Gilbert, W Meth. Enzymol Analysis of the sequence shown in Figure 1 reveals a likely start codon AUG at position 216 followed by an open reading frame of a further 1654 codons. The calculated molecular weight of the peptide gene product is 189, 953. The start codon is followed by a putative signal sequence of 18 codons coding for an amino acid sequence which would be cleaved off the protein before it matures. Nucleotides 447 527 code for alternate repeats of the tripeptide sequences serine glycine glycine and serine valine alanine occuring within the 83,000 MW fragment of P.195 Example 6 . The distribution of some of the amino acids within the translated sequence is asymmetric. For example, of the 19 cysteine residues, two are in the putative signal peptide and eleven are in the C terminal 97 amino acids the 42,000 MW Fragment . Eleven tripeptide sequences of the structure Asn X Ser or Thr where X can be any of the common amino acids except proline have been identified which are potential N glycosylation sites. Using the sequence data shown in Figure 1 it will be appreciated that a peptide corresponding to any part of the sequence may be synthesised using for example, the methods described by Merrifield, R.B., and Marglin, A. Ann. Rev. Biochem., Thus, in a further embodiment of the invention, we provide a synthetic polypeptide of molecular weight 42,000 derived from the carboxy terminus of the P.195 protein of The term synthetic as used herein relates to peptides produced by a chemical method as described above, for example. The identification and cloning of a fragment of the DNA sequence coding for P.195 may be carried out, for example, as follows. Whole messenger RNA mRNA was first extracted from synchronous cultures of Probes that recognised such mRNA Northern blot procedure were then characterised further by incorporation into a vector which, in a suitable host, was capable of expressing the cDNA sequence as a fusion peptide. To ensure expression of this cDNA, the fragments were treated with exonuclease prior to incorporation into the vector in order to give randomised reading frames. Peptides expressed were probed with polyvalent rabbit serum raised against P.195. The DNA fragment referred to above was detected using such a procedure. With regard to the cloning of full length cDNA for P.195 there are now available several methods for cDNA synthesis to represent the entire mRNA eg Heidecker, G. and Messing, J. 1983 Nucleic Acid Res. 11, 4891 4906 . Cloning may also be carried out by the use of total genomic DNA digest libraries which are prepared readily, and the relevant sequence, or fragments thereof, detected in such a library by the use of the above described fragment as a probe for example, Odink, K.G. The construction of a physical map of the gene for P.195 in genomic DNA by restriction endonuclease cleavage, gel electrophoresis, transfer to nitrocellulose or polyamide membranes and hybridisation to specific probes derived from the cloned DNA is extremely useful to confirm and facilitate the orientation and position of cDNA and genomic clones. In addition a comparison of the restriction sites within the cDNA clones with those in the genome can be used to detect features of the gene which may not be present in the mRNA from which the cDNA is synthesised. An example would be the presence of discontinuities within the coding sequence, introns, which may be spliced out of the transcribed RNA by specific splicing events. One such possible intron of about 700 b.p. has been located by the above techniques between nucleotides 221 and 313 of the coding sequence Fig.1 , that is, between the Mbol M and Hind III sites at these positions Fig.3 . As indicated above, P.195 is processed Those fragments of the naturally occurring P.195 most likely to elicit an immune response in a susceptible host are those that are present on the merozoite surface. To locate the position of the processing fragments Holder and Freeman, 1984b, The position of the 42,000 MW fragment was determined by the use of monoclonal antibodies in conventional manner and is described in Example 7. It has been established that the coding sequences for the 42,000 MW and 83,000 MW fragments are at opposite ends of the gene. Experiments to demonstrate allelic variation in the P.195 gene have shown that the greatest conservation occurs in the region 5 to the Hind III site in the 83,000 MW fragment and in the 3 non coding region Figure 2 . The most highly conserved sequence is at the 3 end of the gene corresponding to about 130 amino acid residues at the carboxy terminal of the 42,000 MW fragment, suggesting that this fragment may comprise at least one useful epitope. Thus in a further embodiment of the invention we provide a DNA sequence corresponding to the 42,000 MW fragment of P.195. The DNA sequence according to the invention may also be used to produce viruses containing the DNA sequence. For example, a strain of vaccinia virus Tk unable to confer upon infected cells the ability to grow on media not containing hypoxanthine is used to infect a tissue culture. The tissue culture may then be transformed with the P.195 gene, or fragment s thereof, linked to a Tk genetic determinant. Some of the subsequent viral progeny will have such transforming sequences in the form of inserts in their genomes. These can then be selected by their ability to confer upon tissue culture cells the ability to grow on media devoid of hypoxanthine. Such colonies as do grow are then further selected for production of P.195, or polypeptides of molecular weight of 83,000 153,000 or 42,000, for example by use of a relevant monoclonal antibody such as F111.2. Such vaccinia strains can then be used to infect animals susceptible to malaria, as the new vaccinia strain will cause production of an immunogenic malarial peptide s . It will be appreciated that such vaccines are also readily capable of providing immunity to other infections, such as smallpox, diphtheria, hepatitis B. rabies, herpes simplex virus, whooping cough and the like. Based on the above described characterisation of the DNA sequence for P.195, the cloning of any desired fragment of the sequence is possible by reference to the restriction map so gained. The insertion of a piece of foreign DNA into an Thus, examination of the restriction map of the expression system of choice and the sequences to be expressed, together with a knowledge of the translational frame, enables specific DNA fragments to be ligated into the expression vector and expressed, without further manipulation. For example, pWRL507 is a plasmid constructed from pAT153 and the By using suitable restriction enzyme sites, DNA fragments from the P.195 gene sequence may be cloned into a site within the Thus, in a further feature of the invention there is provided a fused DNA sequence comprising an amino terminal coding portion of a gene tandemly linked to any of the cloned DNA sequences defined herein. Thus, in another feature of the invention is provided a vector, containing a DNA sequence according to the invention tandemly linked to an amino terminal coding portion of a gene translatable by the relevant host and, optionally, any control sequence s associated therewith, which, when used to transform a suitable host will result in production of a fusion protein comprising the P.195 protein of An alternative method for expressing DNA fragments is to use an open reading frame ORF vector into which usually short pieces of DNA can be inserted, often within the sequence coding for the N terminal amino acid sequence of an A vector may be provided comprising a gene capable of translation in a suitable host, optionally provided with relevant control sequences, into which is inserted a DNA sequence according to the invention suitably altered such that the portion of said gene carboxy terminal to said DNA sequence is correctly translated on expression in a suitable host to produce a fusion protein comprising at least a part of the P.195 protein or a polypeptide encoded by one of the cloned DNA sequences defined herein. The peptide coded by part of the P.195 gene in a fusion protein may be cleaved from that fusion protein by enzymic or chemical cleavage of the appropriate peptide bond. It will be apparent to those skilled in the art which enzymic or chemical cleavage method should be employed, by examination of the amino acid sequence expressed. By insertion of a synthetic oligonucleotide linker between the P.195 DNA sequence and the bacterial gene sequence in the fusion protein expression system, a suitable site may be provided for enzymic or chemical cleavage between the P.195 sequence and the remainder of the expressed fusion protein. In this way the P.195 gene encoded fragments may be purified away from host peptides. Direct expression of the coding sequence for the P.195 protein or parts thereof can be achieved by placing the inserted DNA sequence directly after an AUG start codon in the correct reading frame such that the DNA insert replaces the coding sequence normally transcribed and translated by the bacterial control region. Such a control region includes a promoter and a ribosome binding site in the optimal position relative to the start codon. The DNA sequence to be expressed may be correctly positioned by the use of suitable restriction sites and if necessary by using a suitable synthetic oligonucleotide linker. At the end of the inserted DNA sequence a stop codon may be inserted in the correct reading frame to stop translation and a terminator sequence to stop transcription may be added. The inserted DNA to be expressed may be the entire coding sequence of P.195 or the entire sequence from which the amino terminal signal sequence has been removed, or preferably a part of the coding sequence corresponding to an immunogenic fragment of the protein. Suitable fragments may be prepared by restriction enzyme digestion of a suitable cDNA or genomic DNA clone after examination of the nucleotide sequence , and if necessary further treatment at either or both ends with Bal 31 to digest away in a controlled way parts of the DNA sequence. Controlled digestion is achieved by the selection of proper buffer, temperature, reaction time and amount of enzyme, for example as described in Example 8. At this stage a suitable synthetic linker may be added, preferably by blunt end ligation to the insert, to provide an AUG start codon or facilitate ligation into the expression vector. A host cell may be transformed with the above defined cloning vector and the host cell cultured to provide expression of the said P.195 protein, or a polypeptide encoded by one of the cloned DNA sequences defined herein. According to a further feature of the invention we provide a host transformed by a vector containing a DNA sequence substantially encoding the P.195 protein or a polypeptide encoded by one of the cloned DNA sequences defined herein. Controlled expression of any cloned fragment will be possible by use of the sequences at either end of the sequence, or by use of other sequences already known. Such sequences include promoters and enhancers. Examples of such promoters include According to a further feature of the present invention we provide a vector containing any one of the DNA sequences defined herein. The vector referred to above may be any appropriate vector which is suitable for the cloning of the DNA and which may be used to transform a host cell and thereby express the relevant protein. Such vectors include plasmids, bacteriophages and cosmids. Vectors which may be used for cloning cDNA include pUC8, pUC9, pAT153, pBR325 and pBR328 for use in Vectors for use in expression of the relevant protein will include control sequences, such as mentioned above. Such vectors include pXY460 and pWRL 507 for use in Examples of suitable host cells for use in the above described method may be prokaryotic, such as bacteria for example A method for synthesising at least a portion of P.195 or fragments thereof said P.195 or fragment being optionally covalently linked to further peptide sequence, comprises the steps of A polypeptide of molecular weight 42,000 is obtained by any of the above methods. This material may be incorporated into a vaccine for conferring immunity against malaria. For this purpose the antigenic polypeptide may be presented in association with a pharmaceutically acceptable carrier. The antigenic polypeptide may be used singly or in combination with other P.195 epitope containing peptides or with other proteins which will provide immunity against malaria. In a further aspect there is provided a vaccine for inducing immunity to malaria comprising a polypeptide of molecular weight 42,000 derived from the carboxy terminus of the P.195 protein of Pharmaceutically acceptable carriers, in this instance, are liquid media suitable for use as vehicles to introduce the antigen into the patient. An example of such a carrier is saline solution. The P.195 or peptide may be in solution or suspended as a solid in the carrier, or it may be solubilised by the addition of pharmaceutically acceptable detergent. The vaccine may also comprise an adjuvant for stimulating the immune response and thereby enhancing the effect of the vaccine. A convenient adjuvant for use in the present invention is aluminium hydroxide. Conveniently the vaccines are formulated to contain a final concentration of P.195 or peptide in the range of from 0.2 to 5 mg ml, preferably 0.5 to 2 mg ml, most preferably 1 mg ml. After formulation the vaccine may be incorporated into a sterile container which is then sealed and stored at a low temperature, for example 4 C, or it may be freeze dried. In order to induce immunity in vertebrate hosts to malaria one or more doses of the vaccine suitably formulated may be administered. It is recommended that each dose is 0.1 to 2 ml preferably 0.2 to 1 ml, most preferably 0.5 ml of vaccine. The vaccines may be administered by any conventional method for the administration of vaccines including oral and parenteral eg. subcutaneous or intramuscular injection. The treatment may consist of a single dose of vaccine or a plurality of doses over a period of time. The following examples are for illustration only and are not intended to limit the invention in any way. To purify the mRNA, oligo dT cellulose chromatography was performed by standard methods Maniatis, T., Fritsch, E.F. and Sambrook, J. 1982 , molecular cloning, a laboratory manual. Cold Spring Harbor Laboratory, New York . Size fractionation of RNA in sucrose gradients was as described previously Odink, K.G. Second strand synthesis was in 0.1 ml 0.1M HEPES pH6.9, 10mM MgCl₂, 2.5mM DTT, 70mM KCl, 0.5mM of each nucleoside triphosphate and 50 U of Bacterial extracts and an extract of A recombinant cDNA library was constructed from size fractionated cDNA. cDNA, prepared and tailed as described in Example 1, was centrifuged through a 5 ml 50 to 20 w v sucrose gradient in 25mM Tris HCl pH 7.4, 100 mM NaCl, 2.5mM EDTA for 3.5 hrs at 45,000 rpm in an SW50.1 rotor Beckman Instruments . cDNA in the 2 kbp to 8 kbp region about 100 ng was harvested, annealed with 300 ng dG tailed Pst 1 digested pUC9 and used to transform DHl cells to ampicillin resistance. 1200 recombinants were obtained on 6 agar plates. Replica filters were probed with the insert DNA from pPFc1017 which had been labelled by nick translation. Eleven clones were indentified as containing DNA hybridising to this probe. These clones were numbered pPFc1001 to 1011. This library was subsequently rescreened using a part of the insert in pPFc1007 which did not cross hybridise with pPFc1017. A further 8 clones, pPFc1028 to 1035, were isolated which hybridised to this probe. A further cDNA library was constructed using 100 ng dC tailed cDNA, annealed with 100 ng G tailed pUC9 and transformed into DHl cells. About 6000 recombinants were obtained and these were screened on replica filters with the 340 bp Hind III Pst 1 fragment from pPFc1017. Clones pPFc1013 to 1016 and 1018 to 1027 which hybridised to this probe were picked and purified. This library was also screened with the pPFc1007 derived probe and a further 11 colonies numbered pPFc1036 to 1046 were picked. Plasmid DNA from these cDNA clones was purified by centrifugation on caesium chloride gradients and characterized by restriction enzyme mapping and cross hybridisation. The cDNA clones were aligned in an overlapping linear sequence. The positions of the six cDNA clones used in the sequence analysis are shown in Figure 2. 100 µg The probe DNA was specific plasmid DNA or specific sequences excised from the cDNA or genomic DNA plasmid clones, and purified by agarose gel electrophoresis and elution. The DNA was labelled with ³²P by nick translation with An examination of the genomic map and the restriction enzyme map derived from the DNA sequence indicates that these two are colinear in the region corresponding to the coding sequence for the protein, to the right hand side of the Hind III site at nucleotide 313. However, there appears to be an additional 700 bp of sequence in the genomic DNA, not present in the cDNA clones, located between the Mbo 1 site and the Hind III sites at nucleotides 221 and 313 in the cDNA sequence. This may represent an intron in the very start of the coding sequence. The sites shown on the map are some of those for the enzymes A Alul , B Bam Hl , E EcoRl , H Hind III , M Mbol , N Ndel , P Pstl , Pv PvuII , R Rsal and T Taql . The probes were either total plasmids or specific fragments derived by digestion with specific enzymes at sites within the insert or within the plasmid polylinker region. Sequence analysis of the DNA was done using the chemical cleavage method of Maxam and Gilbert Maxam and Gilbert, 1980, Supernatants from The following partial amino acid sequence was obtained from the successive cycles of degradation starting at the N terminal amino acid of the polypeptide. The residues at positions 1 to 5 were not clearly identified N.I. and lysine and phenylalanine derivatives were not separated from each other in the HPLC system employed. By examination of the nucleotide sequence derived from the cDNA and genomic DNA clones it can be seen that this sequence of residues 6 to 20 corresponds to the translated sequence of nucleotides 288 to 332. On this analysis the 83,000 MW fragment is derived from the amino terminal sequence of the P.195 precursor. In the complete sequence the AUG methionine start codon at nucleotide 216 is followed by nucleotide sequence of 18 amino acids which may correspond to a signal sequence present on the primary translation product for many membrane or secreted proteins, and which is normally cleaved off during the passage of the protein into the lumen of the endoplasmic reticulum Kreil, G., Ann. Rev. Biochem, By direct amino acid sequencing, the amino terminus of the 83,000 MW fragment has been located close to the amino terminus of P.195 Example 6 . The other fragments produced by P.195 and processing fragments of it can be labelled From detergent extracts of ³⁵S methionine labelled schizonts a rabbit polyvalent antiserum against P.195 immunoprecipitated P.195 and fragments which have molecular weights of 153,000 110,000 93,000 45,000 42,000 and 29,000 as determined by SDS PAGE analysis. On non reducing gels in the absence of dithiothreitol the 45,000 and 42,000 MW species migrated with apparent molecular weights of 38,000 and 36,000 respectively. Antibody 89.1 immunoprecipitated P.195 and the 153,000, 110,000 and 83,000 MW species. Antibody 111.2 immunoprecipitated P.195 and the 45,000 and 42,000 MW species. From detergent extracts of surface labelled merozoites the polyvalent anti P.195 serum immunoprecipitated three fragments which have apparent molecular weights of 83,000, 42,000 and 19,000. The 83,000 MW fragment was immunoprecipitated by antibody 89.1. Antibody 111.2 immunoprecipitated the 42,000 and 19,000 MW species. An analysis of these fragments by peptide mapping showed that they were probably non overlapping pieces of P.195 Holder and Freeman, 1984b Based on these data and that described in examples 6 and 10 the linear order of the fragments within the P.195 coding sequence can be determined. The amino acid sequence determined for the amino terminus of the 83,000 MW fragment corresponds to the sequence at the 5 end of the coding region of the gene, immediately after the presumptive signal sequence. Therefore the 83,000 MW fragment is derived from the amino terminal 42 of P.195. The major intermediate processing fragment of 153,000 MW which is recognised by antibody 89.1 and not by antibody 111.2 is derived from the amino terminal 78.5 of the coding sequence, and therefore including all of the 83,000 MW fragment. The 42,000 MW fragment is derived from the carboxy terminal 21,5 of the coding sequence, since the antibody 111.2 does not react with the 153,000 MW fragment. The 19,000 MW fragment is presumably located within the sequence of the 153,000 MW species, but not within the 83,000 MW species. In separate experiments, pWRL507 Fig.4 of the accompanying drawings was digested with Ndel and EcoRl, EcoRl and BamHl or EcoRl and Hind III and the relevant fragments purified by agarose gel electrophoresis. This DNA 0.1 pmoles was ligated with 0.5 pmoles of the DNA fragments derived from the P.195 recombinants by the relevant pairs of restriction enzymes and then used to transform DHl cells to ampicillin resistance. Colonies were screened by restriction enzyme digests of small plasmid preparations. Strains with plasmid containing the insert were furthe grown in M9 minimal medium containing 100 µg ml ampicillin and 10 µg ml indole acrylic acid induced expression or 100 µg ml ampicillin and 10 µg ml tryptophan non induced . Bacteria were harvested by centrifugation at 10,000g for 1 minute and then lysed by the addition of SDS sample loading buffer for PAGE 62.5 mM Tris HCl pH 6.8 containing 2 w v sodium dodecyl sulphate, 10 v v glycerol, 0.1M dithiothreitol and 0.005 w v bromophenol blue . Aliquots were subjected to analysis by SDS PAGE and then the gels were stained with coomassie blue to detect total protein or the resolved proteins were transferred to nitrocellulose and used for Western blotting with a polyvalent antiserum raised against purified P.195. A strain containing the Nde I EcoRl fragment of pPFgl produced an inducible fusion protein of 135,000 MW which could be detected in lysates by coomassie blue staining and by reaction with the polyvalent antiserum. This corresponds to a fusion protein containing 233 amino acids from the N terminus of the A 750 bp Rsa I Hind III fragment derived from the cDNA clone pPFc1013, comprising nucleotides 863 to 1613 together with a G C tail and part of the pUC9 polylinker region from the Pst I site to the Hind III site was sub cloned into the plasmid pUC9 which had been cut with Hind II, treated with calf intestinal phosphatase and further digested with Hind III. The insert was subsequently cut out of pUC9 using EcoRI and Hind III and inserted in the correct orientation into pWRL507 cut with EcoRI and Hind III. To obtain inframe expression, 5µg of the plasmid was treated with EcoRI restriction enzyme to linearise the construct. The DNA was precipitated with ethanol, redissolved in 50 µl H₂O containing 500 µg ml BSA and then mixed with an equal volume of Bal 31 buffer Maniatis In a similar manner the Nde I Hind III fragment from pPFc1028 where the Hind III site is the plasmid polylinker was first subcloned as a blunt end Hind III fragment into pUC9 that had been cut with Hind II and Hind III. After inserting this fragment into pWRL507 cut with EcoRI and Hind III, the new construct was reopened with EcoRI, treated with Bal 31 and then recircularised as described above. One strain produced a fusion protein of approximately 56,000 MW which was detected on Western blots with anti P.195 serum, and containing the C terminal 190 amino acid sequence of the coding region for the P.195 gene. Relatively pure preparations of the individual fusion protein were produced from cell lysates. A single colony from a strain was grown up overnight at 37 C in 100 ml M9 medium containing 50 µg ml ampicillin. The following day the overnight culture was dilated with 400 ml M9 culture medium containing 50 µg ml ampicillin and 10 µg ml indole acrylic acid and incubated for 5 hrs at 37 C. The bacteria were harvested by centrifugation at 6000g for 10 minutes. The bacterial pellet was suspended in 10 ml 25mM Tris pH 8.0 containing 1mM EDTA, 1 mM PMSF, 0.2 v v NP40 and 1mg ml lysozyme and left on ice for 2 hrs. After this time 20 µl of 1M MgSO₄ and 200 µl of 1 mg ml DNAse were added and the sample was left to incubate for a further 2 hrs on ice. The insoluble material was harvested by centrifugation at 20,000g for 10 minutes and the supernatant S1 was retained. The pelleted material was washed by suspension in 10 ml 50mM Tris HCl pH 8.0 containing 5mM EGTA, 5mM EDTA, 1 mM PMSF and 1 NP40. The material was centrifuged at 20,000g for 10 minutes and the supernatant was retained S2 . The pellet was resuspended in 10ml 50mM Tris HCl pH 8.1, 5mM EGTA, 5mM EDTA, 0.5M KSCN, and then centrifuged at 20,000g for 10 min to yield a third supernatant S3 and a pellet fraction P . The pellet fraction was resuspended in 10ml H₂O with and without the addition of 0.1 w v sodium dodecyl sulphate, to solubilise the material and then dialysed extensively against 0.89 NaCl. Aliquots of each supernatant and pellet fraction were analysed by SDS PAGE and coomassie blue staining and by Western blotting. In instances where there was a high level of expression of fusion protein, this procedure resulted in a final pellet fraction that contained predominantly the fusion protein and therefore constituted an effective purification of the fusion protein. In instances where the level of expression was lower, the fusion protein was present in both S1 and P fractions. Two Dde 1 Fragments from pPFc1028 corresponding to nucleotides 2961 to 4754 1793 nucleotides and nucleotides 4753 to 5128 375 nucleotides were purified by agarose gel elctrophoresis, treated with 0.04 units of Bal 31 nuclease at 30 C for 2.5 minutes and then repaired with the Klenow Fragment of DNA polymerase I as described above. The DNA was ligated into PXY460 which had been cut with Sma 1 and treated with calf alkaline phosphatase and this was then used to transform JM105 cells to ampicillin resistance. Transformants were plated onto Agar plates containing Xgal and resultant blue colonies were picked. Strains obtained in this way were screened by restriction enzyme analysis of the plasmid DNA,coomassie blue staining and Western blotting of lysates from cells grown in the presence of IPTG. Strains containing the large Ddel fragment all produced a large fusion protein which reacted with the rabbit polyvalent anti P.195 serum. 10 strains containing the smaller Ddel fragment were investigated as described but only one appeared to produce a fusion protein which was significantly larger than wild type β galactosidase and this fusion protein reacted with the rabbit anti P.195 serum. Although the remaining strains contained the insert the gene product was the same size as the normal β galactosidase and did not react with the rabbit anti P.195 serum. The one strain that produced a fusion protein contained an insert of 310bp which covers the C terminal region of the protein. The EcoRI Nde I and EcoRI Hind III fragments derived from pPFc1028 were inserted into pWRL507 as described in Example 8, and were expressed directly by replacing the Pst I EcoRI fragment in the A rabbit polyvalent antiserum specific for the Polyvalent antiserum was produced by immunization with the purified fusion proteins Example 8 . Rabbits were immunized subcutaneously with 250 µg of the protein in FCA and then boosted with 250 µg of the protein in FIA on day 21. On day 35 after the primary immunization serum samples were collected. Mice were immunized intraperitoneally with 125 µg of the protein in FCA and boosted with the same dose on day 23. On day 30 after the primary immunisation serum samples were collected. The titration of an antiserum for the binding of antibodies to a protein can be quantified by a solid phase radioimmunoassay RIA in which the wells of a microtitre plate are coated with the protein and then serial dilutions of the antibody solution are added to the series of wells. After washing away the unbound antibodies the bound antibodies are detected using a highly labelled specific reagent for the first antibodies, such as protein A from The antigens were diluted to 20 µg ml in 0.05M NaHCO₃ pH 9.6 and 50 µl of this solution was added to each well of a 96 well PVC microtitre plate Dynatech Laboratories . After 90 minutes the plate was washed thoroughly with phosphate buffered saline supplemented with 0.5 v v Tween 40 and 0.2 w v bovine serum albumin wash buffer . 50 µl of each serum dilution was added to duplicate wells and after 30 minutes the plate was washed for 10 minutes in wash buffer. Specific antibody was detected by addition of 50 µl of ¹²⁵I labelled protein A 1.5 x 10⁵ cpm for 30 minutes, followed by extensive washing and determination of bound radioactivity using a Packard PED gamma counter. As can be seen from Table 1 rabbits immunized with the fusion proteins containing P.195 sequences produced antibodies which reacted with the purified P.195 in the RIA and the polyvalent antiserum against P.195 contained antibodies which reacted with the fusion proteins. In this instance, fusion protein 1 is the product of the pPFc1028 EcoRI Ndel fragment inserted into In Example 8 it was shown that the fusion proteins reacted with antibodies in the polyvalent anti P.195 antiserum on Western blots. The antiserum raised against the fusion proteins reacted with the fusion proteins and with the purified P.195 protein, on Western blots. The position of the processing fragments in the linear coding sequence was determined partially by immunoprecipitation from a detergent extract of ³⁵S methionine labelled