# Nitrogen fixation regulator genes.

## Abstract
The recombinant DNA plasmid comprises a vector, a promoter of a gene of a Rhizobium species, the gene coding for a product normally capable of activating nitrogen fixation, and a foreign structural gene or foreign DNA fragment under control of the promoter. The gene may, for example, be a fixD gene and the foreign structural gene may be a bacterial toxin gene of Bacillus thuringiensis. Also disclosed are recombinant DNA plasmids comprising a vector, a promoter of a constitutively expressed gene, and a coding sequence of a gene which codes for a product capable of activating nitrogen fixation.

## Claims
What is claimed is 1. A recombinant DNA plasmid comprising, a a vector, b a promoter of a gene of a Rhizobium species, wherein said gene codes for a product normally capable of activating nitrogen fixation, and c a foreign structural gene or foreign DNA fragment under control of said promoter. 2. A recombinant DNA plasmid as recited in Claim 1 wherein said vector is pSUP204. 3. A recombinant DNA plasmid as recited in Claim 1 wherein said gene is a fixD gene. 4. A recombinant DNA plasmid as recited in Claim 1 wherein said promoter comprises the nucleotide sequence 5 A A G C T T A A A C C T G C C T C G C G C T C A C G C G A G T C T T G C C C G G C C A A A T G C T A C G A G T T G A A T G A A A C T G G G C A A G T G G A A G T C A C T G C C G A T G G C T G C A T G G A G T G C G G C A C A T G C A G A G T G T T G T G C G A G G C A A A C G G T G A C G T C G A G T G G A G C T A T C C A C G A G G T G G C T T C G G T G T C C T C T T C A A G T T C G G A T G A G C C A C T C T A A G G T C G A T T C A C A A G C T A G A T C G G C G T T C A A T A G G G G G A C G A A G T G C C A G G G A T C C T T A C A A G A A C C A A C T T A C C T T C C G T A A C T T T A T C G C T C T C C G A C T G T C A A T A C G C A T A C C T C C T A A T A T T A A G C G G G C G A G A A A A T G A C T A A G G T G C T C C C A T C G C A A C T C G T T C A G G G G A G T T A G T G C C C T G T C T G T A C C T T C A C A A A G A G A C A T G C G C A A A C A G G A C A A G C G C T C C G C C G A A A T T T A C A G C A T A T C A A A G G C T C T G 3 or a functionally equivalent sequence hybridizable thereto under stringent conditions. 5. A recombinant DNA plasmid as recited in Claim 1 wherein said foreign structural gene is a bacterial toxin gene of Bacillus thuringiensis. 6. A recombinant DNA plasmid containing and replicating therein a recom binant DNA plasmid comprising a a vector, b a promoter of a constitutively expressed gene, and c a coding sequence of a gene which codes for a product capable of activating nitrogen fixation. 7. A recombinant DNA plasmid as recited in Claim 6 wherein said vector is a suicide vector. 8. A recombinant DNA plasmid as recited in Claim 7 wherein said suicide vector comprises a plasmid pSUP1011 and a transposon Tn5. 9. A recombinant DNA plasmid as recited in Claim 6 wherein said promoter is a promoter of a kanamycin resistance gene. 10. A recombinant DNA plasmid as recited in Claim 6 wherein said promoter of said kanamycin resistance gene comprises the nucleotide sequence 5 A C A G C A A G C G A A C C G G A A T T G C C A G C T G G G G C G C C C T C T G G T A A G G T T G G G A A G C C C T G C A A A G T A A A C T G G A T G G C T T T C T T G C C G C C A A G G A T C T G A T G G C G C A G G G G A T C A A 3 or a functionally equivalent sequence hybridizable thereto under stringent conditions. 11. A recombinant DNA plasmid as recited in Claim 6 wherein said coding sequence is a coding sequence of a fixD gene. 12. A recombinant DNA plasmid as recited in Claim 11 wherein said fixD gene comprises the nucleotide sequence 5 A T G G C C C C C A C T C G T C T T G A G A C C A C G C T T A A C A A T T T C G T G A A T A C C C T C T C T T T G A T T C T G C G C A T G C G C C G C G G C G G A C T C G A G A T T C C G G C G T C G G A A G G A G A G A C A A A G A T A A C A G C G G C T A C C C G C A A C A G C G G G T C T C C T T C T G C C G C T G A T T A T A C T G T A C C A A A G G C C G C A A T A G A C C A A G T C A T G A C T G C C G G G C G G C T G G T C G T A C C A G A C G T T T G C A A C T C T G A G C T G T T C A A G G A T C A G A T A A A A T G G C G C G G A A T T G G T C C G A C T G C C T T C A T C G C T G C G G C G G T G G A G G T C G A T C A C G A A A C G G G C G G A A T G C T G T G G T T C G A G T G C RTI ID 29.2 G C C G A A G A G T C C G A T T A T G A T T A T G A G G A G G A G G T A C A C T T T C T T T C T A T G G C C G C C A A T C T T G C G G G G A G G G C C A T T C G G C T T C A T C G C A C A A T C A G C A G G C G T C A G C G G A C A T T T G C C G A A G A G C A G C A A G A A C A A C A G A A T T C A C G T G A T G A G C A G A G C C A G A G T T C C G C C C G C C A G C G G C T G C T C A A G A A T G A C G G G A T C A T C G G G G A A A G T A C C G C C C T C A T G A C G G C G G T A G A T A C C G C C A A A G T C A T G G C A G A G A C C A A T T C A A T C G T T C T C C T T A G G G G A G A A A C A G G A A C T G G C A A G G A A T G C T T T C C G A A G C T A A T C C A C C A G C A T T C G A C T C G G C A A A A A A A G C C C T T C A T C A A G T T C A A T T G C C C C G C G C T G T C T G A G A G C C T T C T C G A A T C A G A G C T G T T T G G A C A T G A G A A A G G T G C G T T C A C C G G G G C T A T T G C T C A A C G A G T A G G C C G T T T C G A A T C G G C G A A T G G C G G A A C G T T G C T G C T C G A T G A A A T C G G C G A G A T T C C C C C G G C G T T C C A A G C A A A A C T G C T A C G C G T A A T A C A G G A A G G T G A A T T T G A G C G A G T C G G C G G C A C A A A G A C G C T G A A A G T C G A C G T C C G G C T C A T A T T C G C C A C A A A T A A G G A T C T C G A A A T G G C G G T CC A G A A T G G G G A G T T C A G G G A A G A C C T T T A C T A C C G C A T C A G C G G G G T G C C C C T C A T T T T G C C G C C C C T T A G G C A C C G C G A C G G T G A C A T T C C G C T C C T T G C A A G A G C A RTI ID 29.5 T T C C T T C A G C G G T T C A A C G A A G A G A A C G G T C G T G A T C T C C A T T T C G C G C C G T C T G C G C T T G A C C A C T T G T C GA A G T G C A A G T T C C C T G G A A A C G T T C G C G A G C T G G A AA A C T G T G T G C G G A G G A C T G C A A C T C T C G C C A G G T C A A A G A C G A T C A C T T C G T C A G A T T T C G C C T G C C A A A C G G A C C A G T G T T T T T C T T C T C G C C T C T G G A A A G G C G T T C A C T G T T C G C A T G G C C A C A T T G A G A T C C A T G C G C C CG C G G G T A C A A C A C C G T T G C T C G G A G C G C C A G C C A A T G A C G T T C C G C C G A A A G A G C C C G G A T C C G C A G G A G T G G C A T C C A A T C T G A T C G A G C G C G A C C G C T T G A T C A G T G C G C T G G A G G A G G C C G G T T G G A A T C A G G C A A A G G C A G C T C G C A T C C T C G A A A A A A C G C C C C G G C A G G T C G G G C T A T G C T C T A C G T C G G C A T G G T G T G G A C G T G A G A A A G C T C T A A G C T G C C G G T G A 3 or a functionally equivalent sequence hybridizable thereto under stringent conditions. 13. A method for activating expression of nitrogen fixation genes by placing said nitrogen fixation genes under control of a genetically manipulated constitutively expressed activating gene comprising the steps a combining a promoter of a constitutive gene in such a position as to control an activating gene thereby producing a constitutive expres sion of the activating gene, b transforming a strain of Escherichia coli with a constitutive gene transfer system comprising a suicide vector and a transposon wherein the constitutively expressed activating gene is inserted within the t ransposon, c transferring said constitutive gene transfer system to a strain of gram negative bacteria having said nitrogen fixation genes, and d selecting a recombinant strain of said gram negative bacteria where in said constitutively expressed activating gene is contained, replicated and expressed in said gram negative bacteria thereby activating fixation of nitrogen by activating expression of said nitrogen fixation genes. 14. A method for activating expression of nitrogen fixation genes as recited in Claim 13 wherein said constitutive gene is a kanamycin resistance gene. 15. A method for activating expression fo nitrogen fixation genes as recited in Claim 13 wherein said activating gene is a fixD gene. 16. A method for activating expressing of nitrogen fixation genes as recited in Claim 13 wherein said constitutive gene transfer system is pSUP1011 Tn5 ne Pt fixD. 17. A method for activating expression of nitrogen fixation genes as recited in Claim 13 wherein said strain of gram negative bacteria is a strain of a Rhizobium species. 18. A method for activating expression of nitrogen fixation genes as recited in Claim 17 wherein said Rhizobium species is Rhizobium meliloti. 19. A bacterial strain containing and replicating therein a recombinant DNA plasmid according to any of claims 1 to 12.

## Description
NITROGEN FIXATION REGULATOR GENESField of the Invention Biological nitrogen fixation in the root nodules of leguminous plants is a major component of world food production and therefore practical applications of this field are of major interest. Prokaryotes can use a wide variety of nitrogen compounds as sole sources of cellular nitrogen. This variety includes ammonia, dinitrogen and nitrate among the inorganic compounds, and proline, arginine and glutamine among complex organic compounds. Each species can utilize a different array of nitrogen compounds. Glutamine, glutamate and aspartate are the key nitrogen compounds in intermediary metabolism. The latter two are the starting compounds of many pathways of amino acid biosynthesis and serve as amino group donors in many reactions. In all other cases the amino group is donated by glutamine. The major enzyme required for the assimilation of ammonia produced by N2 fixation is glutamine synthetase, which catalyses the reaction Glutamate NH3 ATP glutamine ADP Pi.At high NH4 concentrations lmM glutamate dehydrogenase is also found.Utilization of the assimilated ammonia depends on the activity of glutamate synthase catalyzing Glutamine 2 ketoglutarate NADPH 2 glutamate NADP Since ATP is hydrolysed, these reactions have a favorable equilibrium and allow z or the use of ammonia in the medium or ammonia derived enzymatically from other nitrogen sources Meers, J., Tempest, D. and C. Brown 1970 J. Gen.Microbiol. 64 187 194 . The formation of ammonia is thus a key step in the biological nitrogen cycle. Biological nitrogen fixation can be achieved by a variety of microorganisms and occurs through the induction of an enzyme complex, nitrogenase, which converts atmospheric nitrogen to ammonia. This conversion occurs in a group of physiologically diverse prokaryotes, including facultative anaerobes e.g.,Klebsiella pneumoniae and Rhodospirillum rubrum , obligate anaerobes e.g.,Clostridium pasteurianum , obligate aerobes e.g., Azotobacter vinelandii and some strains of blue green algae e.g., Anabaena cylindrica Sprent, J. I. 197by The biology of nitrogen fixing organisms, London, McGraw Hill, pp. 811 . While this enzyme complex is common to all characterized nitrogen fixing organisms, the conditions under which it is expressed vary considerably between species Burns, R. C., Hardy, R. W. F. 1975 Nitrogen fixation in bacteria and higher plants, Springer Verlag, Berlin . The first stages of nitrogen fixation, conversion of nitrogen into ammonia, are achieved symbiotically in the root nodules of leguminous plants which contain the nitrogenfixing bacteria of the genus Rhizobium. Some non leguminous plants, e.g., alder, also have interactions with symbiotic bacteria which are nitrogen fixers. In addition, free living bacteria, e.g., Klebsiella pneumoniae and the photosynthetic blue green bacteria, also fix nitrogen.Biological nitrogen fixation in the root nodules of leguminous plants is a major component of world food production Burris, R. H. 1980 In Free Living Systems andChemical Models Nitrogen fixation, Newton, W. E., Orne Johnson, W. H., eds.Vol 1 Baltimore, University Park Press, pp. 7 16 . The symbiotic association between plants and bacteria of the genusRhizobium is the result of a complex interaction between the bacterium and its host, requiring the expression of both bacterial and plant genes in a tightly coordinated manner Vincent, J. M. 1980 In Symbiotic Associations andCyanobacteria, Nitroaen Fixation Vol. 2 W. E. Newton, W. H. Orne Johnson, eds. Baltimore, University Park Press pp. 103 129 and Verma, D. P. S.,Legocki, R. P. and S. Auger 1981 In Current Perspectives in NitrogenFixation A. H. Gibson, W. E. Newton, eds. Canberra Australian Academy ofScience, pp. 205 208 . In free living Rhizobia, nitrogenase synthesis is repressed and is only induced after the symbiotic relationship has been established. Furthermore, some Rhizobium species only interact with a narrow range of plant species, whereas other species interact with a wide range. Bacteria bind to the emerging plant root hairs and invade the root tissue through the formation of an infection thread. The plant responds to this infection by the development of a highly differentiated root nodule. These nodules are the site of synthesis of the nitrogenase complex. Following nitrogen fixation, the fixed nitrogen is exported into the plant tissue and assimilated by the plant derived enzymes Scott, D. B., Farnden, K. J. F. andRobertson, J. G. 1976 Nature 263 793,705 . Most Rhizobium symbioses are confined to leguminous plants, Furthermore,Rhizobium strains which fix nitrogen in associ3tion with the agriculturallyimportant temperate legumes are usually restricted in their host range to a single legume genus. However, some strains of Rhizobium have been isolated which can fix nitrogen in a diverse group of legume species but can also form an effective symbiosis with non legumes. Despite the ability of certain plants to induce nitrogenase activity in a symbiotic relationship with some species of Rhizobium, the genetic analysis of biological nitrogen fixation has previously been confined to free living nitrogen fixing organisms, in particular Klebsiella pneumoniae. There are 17 linked nitrogen fixation nif genes arranged in at least 7 transcriptional units in the nif cluster of lXlebsiella Kennedy, C., Cannon, F., Cannon, M.,Dixon, R., Hill, S., Jensen, 3., Kumar, S., McLean, P., Merrick, M., Robson,R. and Postgate, J. 1981 In Current Perspectives in Nitrogen Fixation A. H.Gibson, W. E. Newton, eds. Canberra Australian Academy of Science, pp. 146156 and Reidel, G. E., Ausubel, F. M. and F. M. Cannon 1979 Proc. Nat.Acad. Sci. U.S.A. 76 2866 2870 . Three of these genes, nifH, nifD and nifK encode the structural proteins of the nitrogenase enzyme complex viz. the Feprotein subunit dinitrogenase reductase and the a and B subunits of the MoFe protein dinitrogenase respectively. Dinitrogenase is an a262 tetramer in which the two non identical a and ss subunits have similar molecular weights of 55,000 to 60,000. Dinitrogenase reductase is a dimer of two identical subunits each having a molecular weight around 35,000. These genes are linked on the same operon in K. pneumoniae and are transcribed from a promoter adjacent to the nifH gene. A similar situation nifHDK was found in two fast growingRhizobia, R. meliloti Ruvkun, G. B., et al. 1982 Cell 29 551 559 and R.leguminosarum Schetgens, T. M. P. et al. 1984 Identification and analysis of the expression of Rhizobium leguminosarum PRE symbiotic genes, p. 699, InC. Veeger and W. E. Newton eds. Advances in nitrogen fixation research. Martinus Nijhoff Dr. W. Junk Publishers, The Hague . In the slow growingR. japonicum, it has been found that nifDK forms one operon and that nifH is located elsewhere on the genome Fuhrmann, M. and H. Hennecke 1982 Mol. Gen.Genet. 187 419 425 . A similar observation was made with another member of the slow growing rhizobia, Rhizobium sp. Parasponia a nifH region was found not to be linked to nifO Scott, K. F., et al. 1983 DNA 2 141 148 . Yet a different arrangement was detected in the cyanobacterium Anabaena sp. 7120, in which nifHD is separated from nifK Rice, D., et al. 1982 J. Biol. Chem. 257 13157 13163 . The remainder of symbiotic genes contain information required for bacterial attachment, root hair curling, initiation and development of nodules and establishment of symbiotic relationships. In addition, regulatory sequences such as promoters, operators, attenuators, and ribosome binding sites are found adjacent to the coding regions. These regulatory sequences control the expression of the structural genes, i.e., the coding sequences downstream in the 3 direction of the DNA reading strand. The discovery and study of plasmids, restriction enzymes, ligases and other enzymes involved in DNA synthesis has led to the rapidly developing field of genetic engineering. Use of these techniques has made it possible to transfer DNA across species boundaries, either from eukaryotic to prokaryotic organisms or vice versa. Alternatively, it has been possible to synthesize nucleotide sequences and to incorporate these synthetic sequences into living organisms where they have been expressed. For example, expression in E. coli has been obtained with DNA sequences coding for mouse dihydrofolate reductase Chang, A. C. Y., Nunberg, J. H., Kaufman, R. K., Ehrlich, H. A., Schimke, R.T. and Cohen, S. N. 1978 Nature 275 617 624 and for hepatitis B virus antigen Burrell, C. J., Mackay, P., Greenaway, P. J., Hofschneider, P. H. andK. Murray 1979 Nature 279 43 47 . Two mammal hormones have also been produced in bacteria by use of synthetic DNA Itakura, K., Hi rose, T., Crea, R.,Riggs, A. D., Heynecker, H. L., Bolivar, F., and H. h. Boyer 1977 Science 98 1056 and Goeddel, D. B., Kleid, D. G., Bolivar, F., Heynecker, H. L.,Yansura, D. G., Crea, R., Hirose, T., Kraszewski, A., Itakura, K. and A. D.Riggs 1979 Proc. Nat. Acad. Sci. U.S.A. 76 106 . The practical application of DNA recombination requires the success of a number of different features. First, it must be possible to recognize the DNA fragment coding for the compound of interest and it must be possible to isolate that DNA fragment. Second, it is necessary to understand the mechanisms which control the expression of the information on that DNA fragment and to be able to transfer that information to the control of regulatory sequences which will maximize the productive capabilities of that information. This increased productive capacity could be by rearrangement of coding information and regulatory information within the same organism or between different organisms.The organisms involved may be prokaryotic or eukaryotic. Third, the conversion of coding information into useful products, such as storage proteins and hormones, must occur in an environment where they are not subsequently degraded. Background of the Invention In bacteria of the genus Rhizobium, nitrogenase synthesis is normally repressed under free living conditions and is induced only within a complex symbiosis formed mostly with leguminous plants. R. trifolii is an example of a fast growing Rhizobium with a narrow host range and cannot normally be induced to fix nitrogen in culture. In contrast, a Parasponia Rhizobium species has been isolated and this species is a slow growing organism with a very broad host range capable of an effective symbiotic relationship with a broad variety of tropical legumes as well as the non legume Parasponia Ulmaceae Trinick, M. J. 1980 J. Appl. Bacteriol. 49 39 53 .ParasponiaRhizobium can be induced to fix nitrogen in culture although the level of this fixation is about 100 fold less than can be obtained from the free living bacterium Klebsiella pneumoniae. Other slow growing Rhizobia include the commercially significant R. japonicum, which nodulates soybeans. The genetics of biological nitrogen fixation have been well characterized in the free living organism Klebsiella pneumoniae. The structural genes for nitrogenase nifH, nifO and nifK encoding the Fe protein subunit and the a and ss subunits of the Mo Fe protein, respectively have been mapped both genetically and physically Kennedy, C. et al. 1981 In Current Perspectives inNitrogen Fixation eds. Gibson, A. H. and W. E. Newton Australian Acad.Science, Canberra, pp. 146 156 and Reidel, G. E., Ausubel, F. M. and F. M.Cannon 1979 Proc. Nat. Acad. Sci. U.S.A. 76 2866 2870 . Cloned DNA fragments carrying these sequences have been shown, by Southern blot analysis, to hybridize to homologous sequences in a wide range of nitrogen fixing organisms, including Rhizobium Ruvkun, G. B. and F. M. Ausubel 1980 Proc. Nat.Acad. Sci. U.S.A. 77 191 195 . In spite of the ecological diversity of nitrogen fixing organisms, the physiological structure of the nitrogenase enzyme complex appears to be very conserved. In all cases where the enzyme complex has been purified, two proteins are present. The larger protein dinitrogenase contains molybdenum, iron and acid labile sulfur, and carries the binding site for nitrogen and contains two subunit proteins a and encoded by the nifO and nifK genes respectively. The smaller protein dinitrogenase reductase contains iron and acid labile sulfur, and is required for the reduction of the dinitrogenase and for the binding of MgATP used in this reduction. The dinitrogenase reductase is coded by the nifH gene.Chemical and spectral analyses of the purified protein components support a conservation of protein structure between organisms Scott, K. F., Rolfe, B. G. and J. Shine 1981 J. Mol. Appl. Genet. 1 71 81 . In some cases the structures are sufficiently similar to allow formation of active hybrid enzymes between purified components, e.g.,Azotobacter vinelandii and Klebsiella pneumoniae Eady, R. R. and B. E. Smith 1979 In A treatise on dinitrogen fixation I, II, eds. Hardy, R. W.,Bottomley, F. and R. C. Burns, New York, Wiley Press pp. 399 490 . Not surprisingly, therefore, the region of the nif operon coding for dinitrogenase reductase and dinitrogenase a subunit nifH and nifD shows homology at the nucleic acid sequence level with the corresponding sequences in at least 19 other bacterial strains Ruvkun, G. B. and F. M. Ausubel 1980 Proc. Nat.Acad. Sci. U.S.A. 77 191 195 . Although this conservation of structure is generally true, significant differences between nitrogenases from different organisms also exist as can be shown by variable stability following purification and by the fact that active hybrid complexes do not form in all cases Eady, R. R. and B. E. Smith 1979 supra . A DNA fragment carrying the Klebsiella pneumoniae nifK, nifD and nifH genes has been isolated from the nif strain UNF841 Tn5 nifK Cannon, F.C.et al. 1979 Mol. Gen. Genet. 174 59 66 and cloned into the Escherichia coli plasmid pBR325. The nucleotide sequences of the nifH gene and of 622 nucleotides of the nifD gene were determined Sundaresan, V. and F. M. Ausubel 1981 J. Biol. Chem. 256 2808 2812 Scott, K. F., Rolfe, B. G. and J. Shine 1981 supra . In addition, the DNA sequence of the ff gene from Anabaena 7120 has been determined Mevarech, M., Rice, D. and R. Haselkorn 1980 Proc. Nat. Acad. Sci. U.S.A. 77 6476 6480 . A comparison of the two sequences demonstrates two interesting features 1 There is very little homology between the two sequences at the nucleotide sequence level although a few stretches up to 25bp are conserved, accounting for the observed interspecies homology of the nif genes Ruvkun, G. B. and F. M. Ausubel 1980 supra 2 In general, the promoter regions show very little sequence homology with the exception of a short region likely to be involved in common functions, e.g.,RNA polymerase recognition. In contrast, a comparison of the amino acid sequences of the dinitrogenase reductase and of the first 207 amino acids of the a subunit of dinitrogenase of the two species and of another species show a much greater conservatism. The three species used in this comparison are Klebsiella pneumoniae Kp Anabaena 7120 Ab and Clostridium pasteurianum Cp Tanaka, M.,Haniu, M., Yasunobu, T. and L. Mortenson 1977 J. Biol. Chem. 252 70937100 . The Kp and Cp proteins share 67 amino acid sequence homology, Kp andAb proteins share 71 homology, and the Cp and Ab proteins share 63 . This amino acid sequence homology is not spread evenly throughout the protein.Some regions are virtually identical 90 to 95 homology , while other regions are only weakly conserved 30 35 homology . The structural conservation appears to be centered around the five cysteine residues common to all three Fe proteins. These cysteine residues are believed to be ligands to the active center. Comparison of the N terminal amino acid sequence of the a subunit of dinitrogenase from Cp and Kp shows very little sequence homology in this region. This is in contrast to the very high conservation of amino acid sequence seen in the amino terminal region of the Fe protein. What little homology exists between Cp and Kp a subunits is confined to regions around cysteine residues, as in the Fe proteins. These homologous regions are thought to be involved in the catalytic functions of the nitrogenase enzyme complex. Therefore, this structural conservatism is thought not to be the result of recent evolution and dispersal of the nif genes Postgate, J. R. 1974 Sym. Soc. Gen. Microbiol. 24 263 292 but, rather, is postulated to be related to a conservation of function. The isolation of Klebsiella pneumoniae DNA which codes for the structural genes of the nitrogenase complex Ruvkun, G. B. and Ausubel, F. M. 1980 Proc. Nat. Acad. Sci. USA 77 191 195 has facilitated the identification of the corresponding structural genes of Rhizobium meliloti. The R. meliloti genes were found on an EcoRI fragment which was cloned in E. coli plasmid vectors Ruvkun, G. B. and F. M. Ausubel 1981 Nature 289 85 88 . Further studies using fragment specific mutagenesis in E. coli and transfer of the mutations to the R. meliloti genome confirmed that the cloned fragment carries nif specific genes Ruvkun, G. B. et al. 1980 Cold Spring Harb. Symp. Quant.Biol. 45 492 497 Ruvkun, G. B. et al. 1982 Cell 29 551 559 . This clonedR. meliloti fragment has been analysed by the minicell technique and it was demonstrated that the nifH gene of R. meliloti was expressed in minicells ofE. coli Weber, G. and A. Puhler 1982 Plant Mol. Biol. 1 305 320 . In free living R. meliloti, the nifH gene is not expressed. Further experiments have identified a number of R. meliloti genes involved in symbiotic nitrogen fixation and a preliminary map of R. meliloti nif and fix genes was published Puhler, A. et al. 1983 In Advances in Nitrogen Fixation Research ed. byVeeger, Newton . The Hague, Boston, Lancaster . This map contains the coding regions of the following R. meliloti genes nifK, nifH, nifD, fixA, fixB and fixC Figure 1 .This figure give a preliminary restriction map of theRhizobium meliloti nif and fix genes and their promoters. In addition, the coding regions of the various nif and fix genes are shown by black arrows.The direction of transcription and translation is also shown by the arrows. nifK and nifD are presented as hybrid genes. The indicated promoters p were identified in E. coli. The nifH as well as the fixA promoter can be activated by the K. pneumoniae nifAgp protein. The abbreviations used in the restriction map are C, ClaI E, EcoRI H, HindIII P, PstI S, SmaI and X, XhoI. Of special interest is the transcriptional regulation of the R. meliloti fix nif region. In Figure 1, two promoters are indicated one is located next to nifH and the other is next to fixA. For both the nifH promoter and the fixA promoter it has been demonstrated that they can be activated in E. coli by the Klebsiella nifA gene product Puhler, A. et al. 1983 see supra . The nifH and the fixA promoter are reading in opposite directions. Sundaresan, V.et al. 1983 Nature 301 728 732 and Sundaresan, V. et al. 1983 Proc.Nat. Acad. Sci. USA 80 4030 4034 identified and sequenced the nifH promoter and found some homology to the K. pneumoniae nifH promoter. Corbin, D. et al. r 1983 Proc. Nat. Acad. Sci. USA 80 3005 3009 also identified the fixA promoter as well as the nifH promoter of R. meliloti. In contrast, the K.pneumoniae nifH promoter cannot be activated by the E. coli glnG gene product, whereas the R. meliloti nifH promoter can be activated by the E. coli glnG gene product. These observations imply that the activator and or the promoter of nifH in Klebsiella pneumoniae and Rhizobium meliloti are different.Indeed, no endogenous activator of the nifH and fixA genes of R. meliloti was known. Sumnary of the Invention The isolation and characterization of a gene which activates nitrogen fixing genes of Rhizobium meliloti when that bacterium is in a symbiotic relationship with a plant is described. A method of altering the nature of this gene from an inducible state to a constitutive state is presented. In addition, it is possible to combine the promoter of the activating gene with the coding regions of other genes. This recombination is useful when the expression of that gene is only desirable during a symbiotic relationship between the bacterium and the plant. Detailed Description of the Invention For both the nifH promoter and the fixA promoter of R. meliloti, it has been previously demonstrated that they can be activated in E. coli by theKlebsiella nifA gene product Puhler, A. et al. 1983 supra . In the present study, the E. coli strains used in the isolation and identification of the R.meliloti fixD gene are listed Table 1 . Initially, plasmid pRmW54 was obtained by cloning a 3.5 kb HindIII fragment of pRm29D5 Ruvkun, G. B. et al. 1982 Cell 29 551 559 into the E. coli vector plasmid pAC177 C. pRm29D5 covers an 18 kb fragment adjacent to and surrounding the genes for nitrogenase nif and it was cloned from the genome of R. meliloti. The K. pneumoniae nifA probe Fig. 2 was used in hybridization experiments to detect any complementary DNA fragment in pRmW54, pRmR3 and pRmW52. In Figure 2, the coding regions of the K. pneumoniae nifA and nifL genes are shown in relation to the restriction map Puhler, A. and W. Klipp 1981 In Biology of inorganic nitrogen and sulfur ed. Bothe, H. and A. Trebst Berlin, Heidelberg . The two rectangles represent the restriction fragments which were used as radioactive DNA probes for the hybridization experiments with R. meliloti DNA carrying fix genes.The restriction endonucleases used were E, EcoRI S,SmaI P, PstI and H, HindIII. Restriction fragments of the R. meliloti plasmids pRmW54, pRmR3 and pRmW52 were separated on agarose gels and blotted to nitrocellulose filters Fig. 3a . This figure gives the results of Southern hybridization of R. meliloti fix DNA with the nifA and nifL specific DNA probes of K. pneumoniae following agarose gel electrophoresis of R. meliloti and K. pneumoniae restriction fragments.The results of various digestions are shown in the lanes of Figure 3 as follows Lane 1, an EcoRI HindIII digest of the EcoRI HindIII fragment shown in Fig. 2 subcloned in pUC8 is given Lane 2, x DNA digested with HindIII and EcoRI as molecular weight standards Lane 3, pRmW54 DNA digested with BamHI and BglII, Lane 4, pRmR3 DNA digested with HindIlI and XhoI and Lane 5, pRmW52 DNA digested with BamHI andXhoI. These filters were hybridized with radioactive labelled K. pneumoniae nifA fragments Fig. 3b . This figure 3b illustrates an autoradiograph of nitrocellulose filters after hybridization with the K. pneumoniae nifA probe in 6xSSC at 550C. Such conditions of hybridization are referred to herein as stringent conditions . The 1.8 kb BamHI fragment and the adjacent 0.9 kbBamHI BglII of pRmW54 are shown in Lane 3 Fig. 3b and they show homology to the K. pneumoniae nifA probe. The strong hybridization with other bands is due to vector vector homology or to the K. pneumoniae control DNA shown inLane 1. The hybridizing fragments are summarized in Figure 4. In addition Figure 3c an autoradiograph of the various fragments transferred to nitrocellulose filters is shown when the K. pneumoniae nifl probe was used for hybridization in 6ssC at 550C. It can be seen that there is no specific hybridization to the n L probe. The hybridizing R. meliloti gene region was covered by the 1.8 kb BamHI fragment, which contained the fixD gene Figures 3 and 4 . To a lesser extent, an adjacent 0.9 kb BamHI BglII fragment was also hybridizing. It was found that this lesser hybridization was caused by the fact that the nifD gene extends over to a slight extent from the 1.8 kb BamHI fragment to the 0.9 kb BamHI BglII fragment. In Figure 4, a restriction map of the R. meliloti fix gene region with fragments hybridizing to the nifA probe is shown. A black rectangle means strong hybridization, a hatched rectangle means weak hybridization and a white rectangle means no hybridization. This conclusion was confirmed by the nucleotide sequence of the coding region Figure 5 and by activation experiments Figures 6 and 7 . Figure 6 shows a restriction map of plasmid pGW6 see Table 2 and also the results of a control experiment in which the Rhizobium meliloti nifH promoter was activated by the Klebsiella pneumoniae nifA gene product. The plasmid pGW6 contains an EcoRI SmaI fragment of pRmR2 subcloned into pMC1403. The plasmid vector pMC1403 allows translational fusion to the eighth codon of the E. coli lacZ gene. 6 galactosidase activity is detectable if a second plasmid constitutively expressing the K. pneumoniae nifA gene is present in the same cell. Plasmid pGW7 Table 2 is a derivative of pGW6 where the reading frame of the fusion gene was corrected at the BamHI site byS1 nuclease digestion.In Figure 7, the activation of the Rhizobium meliloti nifH promoter in pGW7 by the fixed gene product of R. meliloti is documented.The B galactosidase activity of E. coli strain ET 8894 containing several plasmid combinations was measured. The results of six independent experiments are presented as black columns and the mean values are given as cross lines.The plasmid combinations were 1 pAc177 C pGW7 2 pRmW54 pGW7 3 pRmW54 Tn5 1 pew7 4 pRmW54 Tn5 2 pGW7 5 pRmW54 Tn5 5 pGW7 6 pRmW54 Tn5 7 pGW7 7 pRmW54 Tn5 8 pGW7 and 8 pRmW54 Tn5 9 pGW7. The plasmids are also shown schematically on the left side of the drawing. p stands for the aphI promoter i.e., the kanamycin resistance gene of the vector plasmid pAC177 C and the double line represents the R. melilotiHindIII restriction fragment of pRmW54. The triangles give the positions of the various Tn5 insertions and the black horizontal arrows show the fixD coding region. Note that the activation of the R. meliloti H promoter compared to the control No. 1 was not detectable when Tn5 was in the fixD region.In addition it was shown that the nifL probe of K. pneumoniae did not hybridize to pRmW54, pRmR3 and pRmW52 Table 2 . The fixD gene of R. meliloti was detected by expression of pRmW54 in E.coli minicells using methods already described in detail Puhler, A. and W.Klipp 1981 In Biology of inorganic nitrogen and sulfur Bothe, H. and A.Trebst eds. Berlin, Heidelberg and Weber, G. and A. Puhler 1982 Plant Mol.Biol. 1 305 320 . In E. coli minicells plasmid pRmW54 synthesized two polypeptides of 68K and 66K molecular weight Fig. 8 . In Figure 8, the restriction map of the HindIII fragment from pRmR29D5 Table 2 is shown on the top and also the position of the several Tn5 insertions arrows 1 through 9 . Restriction enzymes used are denoted as follows H, Hindlil B, BamHI X, XhoI E, EcoRI Bg, BglII and P, PstI. The promoter of the aphI gene kanamycin resistance gene of the vector plasmid pAC177 C is indicated by p. The mapping shown in Figure 8 is in good agreement with the fact that in minicells containing pRmW54 Tn5 1 to pRmW54 Tn54, the 68K 66K polypeptides were not detectable in SDS acrylamide gels. In their place, lower molecular weight polypeptides were found indicating that in pRmW54 Tn5 1 to pRmW54 Tn5 4 the Tn5 insertions were within the coding region of the fixD gene.Using several Tn5 insertions in the cloned HindIII fragment the coding region, designated fixD was again localized to the 1.8 kbBamHI fragment Fig. 3 and 4 . Especially, in the case of two Tn5 inserts pRmW54 Tn5 2 and pRmW54 Tn5 4 putative truncated polypeptides were detected and this finding was in good agreement with the postulated coding region. Evidently, the 68K and 66K molecular weight polypeptides were both encoded by the same DNA fragment and may have been caused by the usage of different ATG start codons.Both polypeptides are expressed on a very low level in E. coli mi ni cells, but this low level is not due to a different codon usage in R. meliloti and E. coli, since the same coding region is very well expressed in the form of a fusion polypeptide with an E. coli vector gene, i.e., aminoglycoside phosphotransferase aphI Example 6 . Thus, the experimental results disclosed in the present invention have defined the boundaries of the fixD gene of R. meliloti and have demonstrated that the protein of the fixD gene is responsible for the activation of the R.meliloti nifHDK and fixABC gene promoters leading in turn to the expression of the genes whose products are required for nitrogen fixation. However, the expression of the fixD gene requires that R. meliloti must first be in a symbiotic association with a specific host plant R. meliloti does not fix nitrogen when living in a free state. Furthennore, R. meliloti does not fix nitrogen even when in a symbiotic association with the same host plant if there is a substantial quantity of nitrate already present in the soil. A genetic manipulation of the fixD gene so that it is consistently expressed either while free living or while in symbiotic association with a specific host plant growing in soil containing high nitrate would therefore be immensely useful in the fixation of nitrogen.Firstly the high nitrate content of a rich soil would not be decreased when an alfalfa R. meliloti symbiotic association was present, but, to the contrary, the soil would become even richer in nitrates. In addition, the use of nif derepressed strains enables symbiotic nitrogen fixation to start earlier in nodule development and to last longer than in nodules with wild type Rhizobia. Secondly, nif derepressed Rhizobium strains have a big advantage when a poor symbiotic association with other plant species leads to ineffective nodulation by wild type Rhizobia. A likely explanation of ineffective nodulation is that a special plant signal normally necessary for transcription of nif genes is absent. The nif derepressed strains described herein have the ability to fix nitrogen without such a plant signal. Thirdly, Rhizobium strains can be induced to grow in close association with roots of economically important non leguminous plants. Since the nif derepressed Rhizobium strains can fix nitrogen in large amounts while in the free living state, such an association between nif derepressed Rhizobia and the roots of important non leguminous agricultural species has far reaching utility. The invention described here teaches how to make and use such nif derepressed Rhizobium strains. The present invention discloses another useful application of the fixD promoter region. When R. meliloti exists in the free living condition, the promoter of the fixD gene remains inactive and it is only when a symbiotic relationship is established that the fixD gene is activated most probably in direct or indirect response to a signal from the plant. It is often useful to incorporate a gene into a Rhizobium species but under such conditions that it is only expressed when the Rhizobia become symbiotically associated with the plant. This objective can be achieved in a genetic recombination by placing the coding region of a desirable gene under the control of the R. meliloti fixD gene promoter region.An example of such a gene, i.e., a gene whose expression is only desired when a symbiotic relationship has been established between a Rhizobium species and a plant, is the expression of the gene coding for an insecticidal protein such as the crystalline toxic protein of Bacillus thuringiensis see Examples 3 and 4 . Example 1 Sequencing of the R. meliloti fixD gene The two main techniques which exist for DNA sequence analysis are the chemical degradation method Maxam, A. W. and W. Gilbert 1977 Proc. Nat.Acad. Sci. USA 74 560 564 and the dideoxy sequencing method Sanger, F. et al. 1977 Proc. Nat. Acad. Sci. USA 74 5463 5468 . Both methods were used to determine the primary structure of the fix gene of R. meliloti. The M13 cloning and sequencing vectors mp8, mp9, mplO and mpll Messing,J. and Vieira, J. 1982 Gene 19 269 276 Norrander, J. et al. 1983 Gene 26 101 106 were used for subcloning of the HindlIl 3.5 kb insert of plasmid pRmW54. For this purpose the HindIII fragment was cleaved from the vector by digestion with HindIII and then separated and isolated by agarose gel electrophoresis. For a non random sequencing approach, the DNA was cleaved with various restriction enzymes EcoRI, BamHI, HindIII, SalI, XhoI and subcloned into the appropriate restriction sites of the M13 sequencing vectors. Shotgun cloning was carried out by digestion of the HindIII fragment with the restriction endonucleases Sau3A, HpaII and TaqI. The resulting fragments were again cloned into M13 sequencing vectors. Recombinant clones were sequenced by the method of Sanger but slightly modified Messing, J. et al. 1981 Nucleic Acids Res. 9 309 321 . The strategy for dideoxy sequencing is presented Figure 9 . To implement the sequencing strategy, subclones of a DNA fragment containing the fixD coding region were generated and sequenced by a non random and shotgun approach. In Figure 9, the position and extension of the subclones are presented in relation to the final restriction map of the fragment. The arrows Figure 9 indicate the direction of sequencing and the rectangle represents the only open reading frame of the analysed DNA fragment. Restriction mapping of the HindIII fragment revealed the presence of several convenient restriction sites suitable for sequencing by the chemical method Maxam, A. M. and W. Gilbert 1977 supra Figure 9 . The HindIII fragment was isolated as described supra , digested with the restriction enzymes EcoRI, BamHI, Sal and XhoI and subcloned into a pUC8 sequencing vector Vieira, J. and J. Messing 1982 Gene 13 259 268 . Determination of the sequence around several restriction sites of the fragment Figure 9 enabled further alignment and confirmation of the data obtained from the dideoxy sequencing. The combined results of the two sequencing methods yielded the sequence of 2127 bp of the 3.5 kb HindIII fragment Figure 10 . Inspection of the six possible reading frames of the nucleotide sequence revealed that only one frame was large enough to code the fix polypeptide.All other reading frames were closed by termination codons at many places.The large open reading frame which represents the fixD gene of R. meliloti started with an ATG 415 nucleotides from the HindIII site and spanned 1628 nucleotides. The coding region of fixD determined by the minicell procedure contained the following series of restriction sites XhoI, EcoRI and BamHI.The same series of restriction sites was also present in the open reading frame. In addition, the molecular weight determination of the polypeptide encoded by the open reading frame resulted in a value of 59K, comparable to the imprecise molecular weight determination of 66 68K determined by the minicell procedure. An overall map of the fix genes and the nif genes is shown Fig. 11 The hypothesis that the polypeptide double band of 66 68K detected in the minicell procedure is caused by different ATG start codons is supported by the nucleotide sequence. In addition to the ATG start codon at nucleotide 415, further ATG codons are located at positions 469 and 535.Example 2 Activation of the Rhizobium meliloti nifH promoter by the R. meliloti fixD gene products in Escherichia coli Following the hybridization experiments which localized the fixD gene on pRmW54, it was demonstrated that the R. meliloti fixD gene products activated the R. meliloti nifH promoter in E. coli. For these experiments plasmid pGW6, which contains the R. meliloti nifD gene fused on a translational level to a plasmid borne lacZ gene, was constructed Fig. 6 . Control experiments with plasmid pWK131 Table 2 which constitutively expressed the K. pneumoniae nifA gene showed that the nifD lacZ fusion of pGW6 was activated. The activation was detected by measuring the S galactosidase activity Figure 6 . In spite of the differences in activation of the R. meliloti nifH gene and the K.pneumoniae nifH gene by the glnG gene of E. coli, it was possible that the R.meliloti fixD gene product might activate the R. meliloti nifH promoter and be detectable in the same manner. For the actual experiment, an E. coli strainET8894 Table 1 which carries an IS1 insertion in the chromosomal lacZ gene and a deletion in the gln regulatory region, was used. This strain was transformed with plasmid pGW7 and plasmid pRmW54. The results Figure 7 were as follows ET8894 containing the nifD lacZ fusion pGW7 and the constitutive ly expressed fixD gene pRmW54 had a 1.5 fold higher B galactosidase activity compared to the control strain ET8894 containing the plasmid pAC177 C Table 3 Figure 7 . The significance of this activation on a very low level was shown by the use of Tn5 insertions in the cloned fixD gene.Only Tn5 insertions in the coding region resulted in the abolishment of the activation Table 3 Figure 7 .Example 3 Expression of foreign genes under the control of a fixD promoter DNA region. Construct a synthetic DlA primer which is complementary to the DNA sequence immediately upstream i.e., 5 to the ATG translation initiation codon of the Rhizobium meliloti fixD coding sequence. The synthetic DNA primer has the sequence 5 CAGACCCTTTGATATGCTGTA 3 . The HindIII XhoI DNA subclone of pRmW54 in pAC177 C designated pRmW69 Table 2 is then excised, purified, subcloned into a single stranded DNA phage and transformed into E.coli JM103 Fig. 11 . This HindIII XhoI subclone contains the complete promoter sequence of R. meliloti fixD gene and 82 nucleotides at the 5 end of the fixD coding sequence. The subcloned fragment is amplified therein and single stranded templates ca. lug are recovered from the supernatant following centrifugation of the bacterial host. A 10 fold excess of the synthetic DNA primer in the presence of the four deoxynucleotide triphosphates one of which is radioactive and DNA polymerase I Klenow fragment is now used as a primer on this fixD template to generate double stranded DNA dsDNA Fig. 12 . The mixture is incubated for a period e.g., 15 45 minutes at an appropriate temperature e.g., 250 370C during which period the complementary strand is extended beyond the HindIII site. The 468 nucleotide region between theHindIII site and the ATG initiation codon contains the complete promoter region of the fixD gene of R. meliloti. The remaining single stranded DNA is then removed by digestion with S1 nuclease Fig. 12 . EcoRI linkers GGAATTCC are then ligated to the dsDNA fragments followed by digestion withEcoRI Fig. 13 . The fragments are separated by agarose gel electrophoresis and the fragment containing the promoter sequence is eluted and cloned into the wide host range plasmid pSUP204 Fig. 13 , which has previously been restricted by the restriction enzyme EcoRI. The resulting recombinant plasmid is termed pRm fixD P SS204.Following transformation and amplification in a suitable E. coli host strain, e.g., 17 1, which is restriction negative, i.e., r , partial cleavage with EcoRI allows the addition of any foreign structural gene or foreign DNA fragment into the linearized plasmid downstream from the fixD promoter fragment. A foreign structural gene of foreign DNA fragment is herein defined as any structural gene or DNA fragment not found naturally under the direct or indirect control of a fixD promoter. For example, the gene coding for the crystalline toxic protein of Bacillus thuringiensis can be inserted resulting in a composite recombinant. A composite recombinant is herein defined as a recombinant DNA plasmid containing a vector, a promoter sequence and any foreign DNA whose expression is under the control of said promoter sequence.Example 4 Insertion of the bacterial toxin gene from Bacillus thuringiensis into the recombinant plasmid pRmfixD P SS204 Recombinant plasmids containing inserts of the gene encoding the toxic crystal protein of B. thuringiensis are obtained using the techniques described Wong, H. C., Schnepf, H. E. and H. R. Whiteley 1983 J. Biol. Chem.258 1960 1967 . The recombinant plasmid pES1 ATCC Number 31995 consisting of the plasmid vector pBR322 and DNA homologous to the 30, 32 and 37 megadalton plasmids, as well as DNA homologous to linearized forms of the very large plasmids of B. thuringiensis is partially cleaved with EcoRI to give linear molecules. These partial cleavage products are further restricted by the enzyme AvaI. The digestion conditions are as recommended by the manufacturer. A probe for the toxic crystal protein gene is isolated and radioactively labelled as previously described Wong, H. C. et al. 1983 supra .The restriction fragments are separated by agarose gel electrophoresis and the labelled probe is found to hybridize to one fragment of approximately 15 kilobases kb . This fragment includes the EcoRI fragments D and F Wong, H.C. et al. 1983 supra . The 15 kb fragment is then cloned into M13mp8 or M13 mp9 according to standard procedure Messing, J. and J. Vieira 1982 Gene 19 269 276 and transformed into E. coli JM103. The single stranded DNA from the extruded phage particles is purified and replicated in vitro by use of a synthetic primer 5 TGTTATCCATGGGTTACCTCC 3 The general method of site specific mutayenesis is described in Zoller, M. J. and M. Smith 1982 NucleicAcids Research 10 6487 6500 . The resulting double stranded recombinant plasmid is then transformed back into E. coli Jim103 and amplified. The amplified double stranded plasmid DNA is purified from the E. coli Jim103 cells and cleaved with the restriction endonucleases NcoI and AvaI.NcoI cleaves at the site of the synthetic primer which is the initiation site of the toxic crystal protein gene and AvaI cleaves at a site which is downstream from the 3 end of the toxic crystal protein gene. The overhangs are then filled in to blint ends Maniatis, T., Jeffrey, A. and D. G. Kleid 1975 Proc. Nat. Acad.Sci. U.S.A. 72 1184 1188 . Finally the pRmfixD P SS204 recombinant plasmid which is derived from pSUP204 Fig. 13 is cleaved with EcoRI and the overhangs filled in to blunt ends. HindIlI linkers are then added to both the B. thuringiensis toxic crystal protein gene fragment and to the pRmfixD P SS204 recombinant. Following the HindIII digestion of both cc . rnents, the toxic crystal protein gene and the pRmfixD P SS204 recombinant plasmid are ligated together to give a pRmfixD P SS204 B. thuringiensis toxic crystal protein gene composite. The mixture is transformed into a suitable E. coli host, e.g. K802, 17 1 or RR1.Plasmids are isolated from individual colonies and the orientation determined by restriction mapping. A colony containing a plasmid with the correct orientation is then conjugated to Rhizobium meliloti and the plasmid is transferred as already described Example 7 . The production of mRNA and or the toxic crystal protein is monitored as already described Wong, et al., supra .Example 5 Construction of a recombinant plasmid containing the fixD gene of R. meliloti under control of a constitutive kanamycin resistance gene promoter The source of the kanamycin resistance gene is the transposon Tn5 and the suicide vector pSUP1011 Simon, R., Priefer, U. and A. Puhler 1981 Proc. ofBielefeld Symposium, Springer Verlag, West Germany see Fig. 14 . TransposonTn5 is a DNA element of 5.7 kilobases kb in length, consisting of 1.5 kb inverted repeat sequences flanking a 2.7 kb central region. Encoded within the inverted repeats are the functions required for transposition. The central region of the transposon carries a gene conferring resistance to the antibiotic kanamycin Kmr .The coding region of the neomycin phosphotransferase gene neo alternatively named aminoglycoside phosphotransferase geneaph is deleted by digestion with the restriction enzymes BglII a BglII site is present 36 nucleotides upstream from the translational start codon ATG and Sali a SalI site is present 342 nucleotides downstream from the TGA termination codon see Fig. 15 . The sequence of the kanamycin resistance gene has been previously shown and is reproduced herein as Figure 15 Beck, E. et al. 1982 Gene 19 327336 . The nucleotide sequence of Tn5 DNA from position 1401 to position 2700 is shown. The aph coding region is boxed by solid lines. Some restriction sites are indicated in the sequence, as well as at the end of the left inverted repeat closing bracket and the promoter region. Such a deletion leaves the promoter region of the neomycin phosphotransferase gene neo intact. The single stranded overlaps generated by these two restriction endonucleases are converted to blunt ends and HindIII linkers are added, followed by digestion with HindIII restriction endonuclease. This construction is herein defined as pSUP1011 HlndIII Fig. 16 . In parallel, plasmid pRmW54 was obtained by cloning a 3.5 kb HindIII fragment of pRm29D5 Ruvkun, G. B. et al. 1982 Cell 29 551 559 into the E.coli vector plasmid pAC177 C Table 2 . After transformation into a suitable host strain and amplification, the recombinant plasmid is linearized by partial digestion with the restriction endonuclease HindIII. Linearization occurs either on the upstream side of the fixD gene or on the downstream side. These two alternatives can be distinguished by the use of restriction maps. Those linearizations where the cut occurs upstream i.e., 5 to the reading strand of the fixD gene are retained and treated with the exonucleaseBAL31 for a sufficient period to produce a blunt end at an optimal distance from the ATG initiation codon of the R. meliloti fixD gene.The linearized recombinant which is thus shortened towards the 5 terminus of the fixD gene is then further restricted with BglII. A single BglII site is present downstream i.e., 3 from the TGA termination codon of the reading strand from the termination codon of the fixD gene Figure 17 . This fragment which contains the fixD structural gene is converted to blunt ends and HindIII linkers are added. Following digestion with HindIII, the fragment containing the fixD gene is purified and ligated into the linearized pSUP1011 HindIII supra Figure 18 . Such a construction comprises the pSUP1011 suicide vector and a Tn5 transposon. Between the inverted repeats of the Tn5, the neomycin phosphotransferase promoter neo0 is included and, in addition, the fixD structural gene positioned so that it is under the control of next This construction is herein defined as pSUP1011 Tn5 neofl0fixD. Example 6 Construction of a fusion gene and product protein of fixD and aminoglycoside phosphotransferase In order to obtain such a fusion gene and protein product, the XhoIHindIII subfragment of pRmW54 was cloned into pAC177 C Table 2 . From the restriction mapping experiment Example 1 it was known that the XhoI site was located within the fixD coding region. On the other hand, the XhoI restriction site of the vector plasmid pAC177 C was located in the aphI gene kanamycin resistance gene . In order to obtain the correct reading frame of the fusion polypeptide, the XhoI site was treated with S1 nuclease in one case and with DNA polymerase I in the other. A strongly expressed 60K fusion polypeptide was obtained after treatment with DNA polymerase I.The recombinant plasnid described herein which contains such a constitutively expressed fixD gene is referred to as pRmW69 Table 2 .Example 7 Introduction of DNA sequences into the genome of gram negative organisms other than E. coli This example is based on the following general principles. Two basic components are employed. These are 1 a suicide vector and 2 a transposon. Suicide vectors are plasmid molecules which replicate stably in one bacterial host in this case E. coli but fail to replicate in a different bacterial species e.g., Rhizobium meliloti . Transposons are genetic elements which are able to move translocate from one location to another in DNA. The translocation process is mediated by gene products encoded on the transposon and is dependent on the integrity of repeated sequences directly or indirectly repeated located at each end of the transposon. Transposons generally carry a gene or genes encoding resistance to one or more antibiotics. The vector contains the inserted fixD gene under the control of a kanamycin resistance promoter pSUP1011 Tn5 neoProfixD see example 5 and is transformed introduced into Escherichia coli strain SM10. This type of recombinant plasmid comprising a suicide vector, a transposon and a constitutively expressed structural gene is herein defined as a constitutive gene transfer system . This strain is capable of mobilizing Mob pSUP1011 derivatives recombinant plasmids into other gram negative bacteria Simon, R., Priefer,U. and A. Puhler 1983 Proc. of Bielefeld Symposium, Springer Verlag, WestGermany . The resultant transformants are screened by the Grunstein andHogness colony hybridization procedure Grunstein, M. and D. S. Hogness 1975 Proc. Nat. Acad. Sci.USA 72 3961ff to detect those containing the desired cloned DNA fragment. Introduction of the cloned DNA fragment into the genome of any gramnegative bacterium e.g., Rhizobium meliloti is achieved via a process called bacterial conjugation. The E. coli SM10 derivative, carrying the desired pSUP1011 recombinant, is mixed with cells of R. meliloti carrying a symbiotic Sym plasmid on the surface of a nutrient agar plate. The plate is incubated for a suitable period 4 16 hours at 29 300 optimum temperature forRhizobium meliloti and during this time cells of each type come into physical contact conjugation and the pSUPlOil recombinant derivative is transferred from E. coli to R. meliloti. The cell mixture is washed off the plate and spread on an agar plate which is selective for organisms which can only grow if they are capable of constitutive nitrogen fixation.The resultant colonies which can constitutively fix nitrogen will be derivatives of R. meliloti in which the cloned DNA fragment, within Tn5, will be inserted at some point in the genome. Selection for ability to constitutively fix nitrogen insures maintenance of the inserted DNA. At this stage, it is unknown whether the DNA fragment, within Tn5, has been transferred to the chromosome of R. meliloti or to one of its several plasmids. This uncertainty can be resolved by visualization of the plasmids and the bacterial chromosome by ethidium bromide staining after horizontal agarose gel electrophoresis Djordjevic, M. A. et al. 1982 J. Bacteriol.151 560 568 . In either case, the transferred DNA fragment is fuctional, since the fixD gene product acts in trans to activate nitrogen fixation. E. coli C600 pRmW54 was placed on deposit with the American TypeCulture Collection, 12301 Parklawn Drive, Rockville, MD 20852 on June 4, 1984,Accession No. 39722. Example 8 Enhanced Expression of Foreign Gene Under nif Promoter Control Resulting from Enhanced Expression of fixD Gene Products Expression of B galactosidase was placed under nifHDK promoter control as described pGW7, Example 2, Fig. 6 . The amount of S galactosidase synthesized under nifHDK control was measured using E. coli ET8894 Table 1 containing pGW7. Synthesis of active B galactosidase in this system depends on activation of the nifHDK promoter. Introduction of pRmW54 expressing fixD as a fusion protein under control of the e I leads to a significant increase in ss galactosidase activity over control levels Table 3 .The described experiment was designed to test whether ss galactosidase expression could be enhanced by increasing the levels of fixD gene product, and whether the level of fixD gene product could be increased by shortening the distance, or number of amino acids, between the aphI promoter and the start of the fixD gene. A series of deletions was constructed in which the number of amino acids lying in the coding region between the aphI promoter and the fxD coding region was reduced. First, the larger of the two HlndIII Et3 it fragments of pRmW54 was cloned into pACYC177 C also termed pAC177 C herein , previously cut with HindIII and BamHI. The resulting plasmid, designated pRmW541, contained a unique HindIII site located between the aphI promoter and the fixD coding region. Deletions were introduced by cutting pRmW541 with HindII1 and digesting for various times with exonuclease Bal31.The resulting digestion mixture was religated after S1 nuclease digestion, and the mixture was used to transform a culture of E. coli ET8894 pGW7 . After incubation for 48 hours at 3O0C, the transformants were plated on agar plates supplemented with ampicillin, chloramphenicol, glutamine and X gal 5 bromo 4 chloro 3 indolyl ss D galactoside, a chromogenic ss galactosidase substrate . Although E. coliET8894 pGW7, pRmW54 produces detectable B galactosidase activity, the activity is low and incubation on X gal plates yields white colonies. Several transformants from the deletion mixture yielded blue colonies, indicating qualitatively enhanced expression of B galactosidase. Plasmids were isolated from strains with higher activity and retransformed into E. coli ET8894 pGW7 . The g galactosidase activity was quantitatively measured with results shown in Fig. 19. The structure of each of each of the pRmW541 deletion plasmids was further analyzed by sequencing XhoI fragments of the various pRmW54i isolates. An endonuclease XhoI site exists within the fixD gene and another is located just upstream from the translation start of the aphI coding region See Fig. 10 . The XhoI fragments were sequenced, after cloning them into the SalI site of the phage M13mp8.Nucleotide sequence determination of these deletions, which were contained within the XhoI fragments, revealed that all the nifHDK activated proteins were fusion polypeptides, with the exception of pRmW541 5. The structures are shown diagrammatically in Fig. 19. Fig. 19 diagrams the structure of the pRmW541 deletion plasmids, and the resulting level of activation in ET8894 pGW7 . For each of the plasmids the concluded polypeptide structure is shown schematically. The black closed circle indicates the aphI promoter. The open rectangles represent the APH I part of the translated polypeptide. The incomplete black triangle represents the FIX D part. Crosshatched regions indicated translated open reading frame regions generated by the construction, which are expressed only in the fusion polypeptides. The numbers in the boxes represent the amino acid residues.The plated right hand end of the hatched region of pRmW541 5 indicates a translation termination signal. The numbers on top of the FIX D part indicate the codon in which the fusion occurred. The right hand column shows the Bgalactosidase activity of each plasmid in ET8894 pGW7 . The effect of the deletions on the expression of the FIX D polypeptides was also measured. Deletions in plasmid pRmW541 which cause a high level of activation of the nifHDK promoter also lead to stronger expression of the FIXD polypeptide. The results demonstrate that deletions in the region between the aphl promoter and the fixD gene made it possible to increase the level of APH IFIX D fusion protein synthesized and to increase the activation of the nifHDKlacZ fusion. Control experiments have shown that activation of the nifHDK promoter by the fixD gene product was strictly dependent on the presence of an intact qlnF gene in the host organism. The glnF gene, also termed ntrA, is a positive regulatory gene for glutamine synthetase, and is active in ET8894. Table 1 Escherichia coli strainsStrain Reference RemarksE.coli DS410 Dougan et al. 1977 minicell isolationE.coli MCi00O Casabadan et al. 1980 lacZ deletion strainE.coli ET8894 McNeil et al. 1981 strain with a lacZ ISl insertion and glnALG deletion Table 2 PlasmidsPlasmid Reference Remarks pAC177 C Pühler et aL. 1983 E. coli vector plasmid pUC8 Vieria and Messing 1982 E. coli vector plasmid pMC1403 Casabadan et al. 1980 E. coli for IacZ translational fusions pRmR29D5 Ruvkun et al. 1982 R. meliloti nif fix region 25 kb ClaI pRmR2 Ruvkun et al. 1982 R. meliloti nifHD 3,9 kb EcoRT pRmR3 Ruvkun et al. 1982 R. meliloti fix genes 5 kb EcoRt pWK131 Pühler et al. 1983 K. pneumoniae nifA cloned in pAC177 C pWK130 Pühler et al. 1983 K. pneumoniae nifAL cloned in pAC177 C pMM14 Espin et al. 1982 K. pneumoniae ntrC cloned in pACYC177 pGW6 7 G. Weber, Bielefeld EcoRI SmaI subclone of pRmR2 into pMC1403 pRmW54 G. Weber, Bielefeld HindIII subclone of pRmR29D5 in pAC177 C pRmW69 G. Weber, Bielefeld HindIII XhoI subclone of pRmW54 in pAC177 pRmW52 G.Weber, Bielefeld EcoRI claI subclone of pRmR29D5 in pBR325 Table 3 Activation of the R.meliloti nifH promo G E.coli ET8894EMI26.1 plasmids SEP gene SEP products SEP B galactosidase SEP activity tb SEP units SEP per SEP cell tb SEP with SEP ammonium SEP without SEP ammonium tb pGW7 tb pWK131 SEP K.p.nifA SEP 276,0 SEP 235,0 tb pGW7 tb pWK130 SEP K.p.nifAL SEP 1,0 SEP 0,7 tb pGW7 tb pMM14 SEP K.p.ntrC SEP 13,6 SEP 11,4 tb pGW7 tb pRmW54 SEP 68K 66K SEP 1,4 SEP 1,4 tb pGW7 tb pRmW54 Tn5 tb SEP No SEP 1 SEP 0,8 SEP 0,8 tb pGW7 tb pRmW54 Tn5 tb SEP No SEP 2 SEP 0,8 SEP 0,9 tb DGW7 tb DRmW54 TnS tb SEP No SEP 5 SEP 0,6 SEP 0,7 tb DGW7 tb DRmW54 Tn5 SEP 68K 66K SEP 2,6 SEP 2,3 tb SEP No SEP 7 tb pGW7 tb pRmW54 Tn5 SEP 68K 66K SEP 3,2 SEP 2,2 tb SEP No SEP 8 tb pGW7 tb pRmW54 Tn5 SEP 68K 66K SEP 3,4 SEP 4,1 tb SEP No SEP 9 tb pGW7 tb pAC177 C SEP 1.0 SEP 0.9 tb The ss glactosidase activities of strains containing the indicated combinations of plasmids were measured in the pre sence or absence of ammonium in the medium each number re presents one measurement .