# Process for producing protein, and vector, recombinant DNA and transformant used therefor.

## Abstract
Disclosed is a process for producing a protein, which comprises 1 providing a gene which comprises a structural gene of a desired exogenous protein 2 providing a vector which comprises a gene comprising a gene A coding for a signal peptide capable of linking the structural gene of the desired exogenous protein to said gene immediately after the end on the downstream side of the gene A and is proliferable in a predetermined host cell 3 preparing a recombinant DNA proliferable in a predetermined host cell by use of the gene having the structural gene of the desired exogenous protein and said vector 4 transforming the host cell by use of the recombinant DNA to prepare a transfor mant and 5 culturing the transformant and recovering the desired exogenous protein produced. The exogenous or foreign protein produced within the host cell is excreted from the cell thanks to the signal peptide. The vector and the recombinant DNA for use in the process, and the resultant transformant are also claimed.

## Claims
WHAT IS CLAIMED IS 1. A process for producing a protein, comprising the steps of 1 providing a gene which comprises a structural gene of a desired exogenous protein 2 providing a vector which comprises a gene comprising a gene A coding for a signal peptide capable of linking the structural gene of the desired exogenous protein to said gene immediately after the end on the downstream side of the gene A and is proliferable in a predetermined host cell 3 preparing a recombinant DNA proliferable in a predetermined host cell by use of the gene having the structural gene of the desired exogenous protein and said vector 4 transforming the host cell by use of the recombinant DNA to prepare a transformant and 5 culturing the transformant and recovering the desired exogenous protein produced. 2. A process for producing a protein according toClaim 1, wherein the recombinant DNA is a recombinant plasmid, obtained as a result of linking a gene coding for the desired exogenous protein to the gene coding for a signal peptide immediately after the end on the downstream side thereof, which contains a base sequence comprising at least the gene coding for promoter, the gene coding for SD sequence, the gene coding for the signal peptide and the gene coding for the desired exogenous protein and yet contains no gene coding for an excretable protein at the portion other than said base sequence portion. 3. A process for producing a protein according toClaim 1 or 2, wherein the gene comprising a gene A coding for the signal peptide has a restriction endonuclease recognition site artificially created with at least one of its base pairs as being at least a part of the constituent members. 4. A process for producing a protein according to Claim 3, wherein the most upstream part of the restriction endonuclease cleavage site in the restriction endonuclease recognition site does not exist on the downstream side of the boundary between the gene coding for the signal peptide and the structural gene of the desired exogenous protein, which structural gene is or is to be linked to and immediately after the end on the downstream side of the gene coding for the signal peptide. 5. A process for producing a protein according to Claim 4, wherein the restriction endonuclease recognition site exists at the boundary between the gene coding for the signal peptide and the structural gene of the desired exogenous protein, which structural gene is or is to be linked to and immediately after the end on the downstream side of the gene coding for the signal peptide. 6. A process for producing a protein according to any one of Claims 1 to 5, wherein the gene coding for the signal peptide is derived from a natural product. 7. A process for producing a protein according to any one of Claims 1 to 5, wherein the gene comprising a gene A coding for the signal peptide is made up of two moieties, one of which lying on the upstream side from the restriction endonuclease cleavage site within the restriction endonuclease recognition site artificially created is derived from a natural product and the remaining of which lying or to be lying on the downstream side from the restriction endonuclease cleavage site is artificially synthesized. 8. A process for producing a protein according to any one of Claims 1 to 7, wherein the gene coding for the signal peptide is derived from alkaline phosphatase, and the restriction endonuclease recognition site is the Hind III recognition site and exists at the boundary between the gene coding for the signal peptide and the structural gene which is or is to be linked to and immediately after the end on the downstream side of the gene coding for the signal peptide. 9. A process for producing a protein according to any one of Claims 1 to 7, wherein the gene coding for the signal peptide is derived from 8 lactamase, and the restriction endonuclease recognition site is theNae I recognition site and exists at the boundary between the gene coding for the signal peptide and the structural gene which is or is to be linked to and immediately after the end on the downstream side of the gene for the signal peptide. 10. A vector, which comprises a gene comprising a gene A coding for a signal peptide capable of linking a structural gene of a desired exogenous protein to said gene A immediately after the end on the downstream side of said gene A and is proliferable within a predetermined host cell. 11. A recombinant DNA proliferable within a predetermined host cell, comprising 1 a gene of a desired exogenous protein, and 2 a vector which comprises a gene coding for a signal peptide capable of linking the structural gene of a desired exogenous protein to and immediately after the end on the downstream side of said gene A and is proliferable within a predetermined host cell. 12. A recombinant DNA according to Claim 11, wherein the recombinant DNA is a recombinant plasmid, obtained as the result of linking a gene coding for a desired exogenous protein to the gene coding for a signal peptide immedistely after the end on the downstream side thereof, which recombinant plasmid contains a base sequence comprising at least the gene coding for promoter, the gene coding for SD sequence, the gene coding for the signal peptide and the gene coding for the desired exogenous protein and yet contains no gene coding for an excretable protein at the portion other than said base sequence portion. 13. A recombinant DA according to Claim 11 or 12, wherein the gene comprising a gene A coding for the signal peptide has a restriction endonuclease recognition site artificially created with at least one of its base pairs as being at least a part of the constituent members. 14. A recombinant DNA according to Claim 13, wherein the most upstream part of the restriction endonuclease cleavage site in the restriction endonuclease recognition site does not exist on the downstream side of the boundary between the gene coding for the signal peptide and the structural gene of the desired exogenous protein which is or is to be linked to and immediately after the end on the downstream side of the gene coding for the signal peptide. 15. A recombinant DNA according to Claim 14, wherein the restriction endonuclease recognition site exists at the boundary between the gene coding for the signal peptide and the structural gene of the desired exogenous protein which is or is to be linked to and immediately after the end on the downstream side of the gene coding for the signal peptide. 16. A recombinant DNA according to any one of Claims 11 to 15, wherein the gene coding for the signal peptide is derived from a natural product. 17. A recombinant DNA according to any one of Claims 11 to 15, wherein the gene comprising a gene A coding for the signal peptide is made up of two moieties, one of which lying on the upstream side from the restriction endonuclease cleavage site within the restriction endonuclease recognition site artificially created is derived from a natural product and the remaining of which lying or to be lying on the downstream side from the restriction endonuclease cleavage site is artificially synthesized. 18. A recombinant DNA according to any one of Claims 11 to 17, wherein the gene coding for the signal peptide is derived from alkaline phosphatase, and the. restriction endonuclease recognition site is the Hind III recognition site and exists at the boundary between the gene coding for the signal peptide and the structural gene which is or is to be linked to and immediately after the end on the downstream side of the gene coding for the signal peptide. 19. A recombinant DNA according to any one of Claims 11 to 17, wherein the gene coding for the signal peptide is derived from 8 lactamase, and the restriction endonuclease recognition site is the Nae I recognition site and exists at the boundary between the gene coding for the signal peptide and the structural gene which is or is to be linked to and immediately after the end on the downstream side of the gene coding for the signal peptide. 20. A cell capable of producing a desired exogenous protein, which is a transformant transformed with a recombinant DNA proliferable within a predetermined host cell, formed by use of a gene comprising 1 the gene of a desired exogenous protein and 2 a vector which comprises a gene comprising a gene A coding for a signal peptide capable of linking the structural gene of the desired exogenous protein to said gene A immeidately after the end on the downstream side of said gene A and is proliferable within a predetermined host cell. 21. A cell according to Claim 20, wherein the cell belongs to the genus Escherichia. 22. A cell according to Claim 21, wherein the recombinant DNA is a recombinant plasmid, obtained as the result of linking a gene coding for a desired exogenous protein to the gene coding for a signal peptide immediately after the end on the downstream side thereof, which recombinant plasmid contains a base sequence comprising at least the gene coding for promoter, the gene coding for SD sequence, the gene coding for the signal peptide and the gene coding for the desired exogenous protein and yet contains no gene coding for an excretable protein at the portion other than said base sequence portion. 23. A recombinant DNA according to any one of Claims 20 to 22, wherein the gene comprising a gene A coding for the signal peptide has a restriction endonuclease recognition site artificially created with at least one of its base pairs as being at least a part of the constituent members. 24. A cell according to Claim 23, wherein the most upstream part of the restriction endonuclease cleavage site in the restriction endonuclease recognition site does not exist on the downstream side of the boundary between the gene coding for the signal peptide and the structural gene of the desired exogenous protein which is or is to be linked to and immediateLy after the end on the downstream side of the gene coding for the signal peptide. 25. A cell according to Claim 24, wherein the restriction endonuclease.recgn tion site exists at the boundary between the gene coding for the signal peptide and the structural gene of the desired exogenous protein which is or is to be linked to and immediately after the end on the downstream side of the gene coding for the signal peptide. 26. A cell according to any one of Claims 20 to 25, wherein the gene coding for the signal peptide is derived from a natural product. 27. A cell according to any one of Claims 20 to 25, wherein the gene comprising a gene A coding for the signal peptide is made up of two moieties, one of which lying on the upstream side from the restriction endonuclease cleavage site within the restriction endonuclease recognition site artificially created is derived from a natural product and the remaining of which lying or to be lying on the downstream side from the restriction endonuclease cleavage site is artificially synthesized. 28. A cell according to any one of Claims 20 to 27, wherein the gene coding for the signal peptide is derived from alkaline phosphatase, and the restriction endonuclease recognition site is the Hing III recognition site and exists at the boundary between the gene coding for the signal peptide and the structural gene which is or is to be linked to and immediately after the end on the downstream side of the gene coding for the signal peptide. 29. A cell according to any one of Claims 20 to 27, wherein the gene coding for the signal peptide is derived from ss lactamave, and the restriction endonuclease recognition site is the Nae I recognition site and exists at the boundary between the gene coding for the signal peptide and the structural gene which is or is to be linked to and immediately after the end on the downstream side of the gene coding for the signal peptide.

## Description
PROCESS FOR PRODUCING PROTEIN, AND VECTOR, RECOMBINANT DNA AND TRANSFORMANT USED THEREFOR BACKGROUND OF THE INVENTIONField of the Art This invention relates to a process for producing a protein according to a genetic engineering method.More particularly, the present invention relates to a process for producing a protein according to a genetic engineering method which comprises utilizing a vector having a DNA gene which has been so designed that a gene coding for a desired exogenous or foreign protein is linked to and immediately after a gene coding for a signal peptide, and recovering efficiently the protein produced within a host cell and excreted out of the cell thanks to the action of the signal peptide. Also, the present invention relates to a vector, a recombinant DNA and a cell which has been transformed by the recombinant DNA and which is thus capable of producing a desired protein to be used for the above process for production of a protein.Prior Art Techniques for producing desired genetic products in large amounts by utilizing the recombinant DNA techniques are being established as can be seen from the large number of reports and published patent specifications. With establishment of these techniques, genetic analyses are also carried out, and the knowledge that a series of peptides called signal peptides comprising 15 to 30 amino acid residues participate in secretion of the expressed proteins out of the host cells has also been found from among such analyses. This knowledge concerning secretion of the expressed protein outside of cells is called the Signal Hypothesis J. Cell Biol., 67, 835 1975 , and experimental data supporting this hypothesis are being accumulated Secretory Mechanisms, 9, Cambridge University Press, Cambridge 1979 Biochemistry, 52, 141 1980 , etc. . The outline and the function of signal peptides are described in, for example, Japanese Patent Laid Open PublicationNo. 69897 1983, and signal peptides are recognized as participating in permeation of a protein through cell membrane. At present, inventions utilizing signal peptides are disclosed in Patent Publications Japanese PatentLaid Open Publications Nos. 19092 1980, 45395 1980, 13786 1981, 145221 1981, 154999 1981, 192400 1982, 69897 1983, etc. . According to any of the methods disclosed in these Publications, the structural gene coding for the protein to be secreted from the host microorganism is cleaved at an appropriate restriction endonuclease recognition site, and an exogenous gene is linked thereto through a linker for matching a reading frame. As a result, the exogenous protein expressed from such a fused gene has a superfluous peptide at its N terminal side. Therefore, it may be considered probable that such a superfluous peptide may interfere with secretion of the exogenous protein or have a deleterious effect on the biological activity of the exogenous protein. On the other hand, according to Proc. Natl. Acad.Sci., USA, 77, 3988 3922 1980 or Science, 219, 620 625 1983 , all of these utilize the signal peptides inherently possessed by the proteins to be expressed, and therefore utilization of such signal peptides may be limited only to excretion of the proteins having such native signal peptides. Thus, the techniques of the prior art may be criticized as involving various problems, and it would be desirable to solve such problems. Meanwhile, in the prior art, it was difficult to produce a prbtein with a relatively low molecular weight e.g. Somatostatin Science, 198, 1056 1977 , but later it has been made possible to produce even a low molecular weight protein according to the hybridized protein method Japanese Patent Laid Open PublicationNo. 92696 1979 , namely, the method in which a desired protein is produced as a hybridized protein with E.coli protein. However, for obtaining a desired protein according to the hybridized protein method, it is necessary to utilize the method in which the gene coding for methionine is inserted at a fusing point between the gene coding for the E. coli protein and the gene coding for the desired protein, and then the gene is expressed according to the required steps, followed by cleavage with cyanogen bromide which is specific for methionine to obtain the desired protein Science supra or the method in which the desired protein is obtained by the trypsin digestion specific for lysine or arginine Nature, 285, 456 1980 . According to any one of these methods, depending on the amino acid composition of the desired protein, use of the method may be restricted. The reason for this is that in the case of the cyanogen bromide treatment, if methionine exists, or in the case of the trypsin digestion treatment, if lysine or arginine exists, in the amino acids constituting the desired protein, the protein will be cleaved at such an amino acid site, whereby a complete protein desired cannot be obtained. On the other hand, for a protein having a molecular weight of a certain order or higher, it is possible to produce a desired protein containing no amino acid sequence derived from E. coli according to the direct expression method Nature, 281, 544 1979 1. According to the direct expression method, a gene having an initiation codon ordinarily AUG coding for methionine attached to the gene coding for a desired protein immediately therebefore is expressed subsequently to the required steps, but a product having methionine attached at the N terminal may be produced depending on the kind of the protein e.g., SV40t antigen Nucleic Acids Res., 8, 4057 1980 1, ss globin Proo Natl. Acad. Sci.USA, 76, 5596 1979 , etc. , whereby no natural product can be obtained Thus, the prose for producing a protein accoxd ing to any of the above genetic engineering techniques may be restricted in its use depending on the desired protein to be obtained, and therefore it would be desirable to provide a process which can produce any desired protein without such restrictions as mentioned above. Further, J. Bechwith et al suggest that about 2 x 104 pores exist on the cell wall protein of E coli, and these pores are common to excretable proteins Cell., 24, 709 1981 . Accordingly, it may be considered that, during permeation through the membrane, a plural number of excretable proteins will compete with each other, whereby extracellular excretion is lowexed. Thus, according to the process for production of a protein by use of the genetic engineering technique as described above, when secreting the desired protein out of the microorganism cell, competition may occur with excretable proteins other than the desired protein, and therefore it would be desirable to provide a method to permit the desired protein to be excreted efficiently out of the microorganism cell without being subject to such competition. SUMMARY OF THE INVENTION An object of the present invention is to provide a solution to the point described above. We have found that this object can be accomplished by providing a process for producing a protein, which comprises utilizing a vector containing a DNA gene which has been so designed that a gene coding for a desired exogenous protein is linked to and immediately after a gene coding for a signal peptide, producing the desired protein in a host cell according to a genetic engineering technique, further permitting the desired protein to be excreted out of the cell thanks to the action of the signal peptide, and recovering the protein thus excreted. Accordingly, the process for producing a protein according to the present invention comprises the steps of 1 providing a gene which comprises a structural gene of a desired exogenous protein 2 providing a vector which comprises a gene comprising a gene A coding for a signal peptide capable of linking the structural gene of the desired exogenous protein to said gene immediately after the end on the downstream side of the gene A and is proliferable in a predetermined host cell 3 preparing a recombinant DNA proliferable in a predetermined host cell by use of the gene having the structural gene of the desired exogenous protein and said vector 4 transforming the host cell by use of the recombinant DNA to prepare a transformant 5 culturing the transformant and recovering the desired exogenous protein produced. In the preparation of the above recombinant DNA, it may sometimes be particularly preferable to use, for example, a recombinant plasmid, obtained as the result of linking a gene coding for a desired exogenous protein to the gene coding for a signal peptide immediately after the end on the downstream side thereof, which has a base sequence comprising at least the gene coding for promoter, the gene coding for SD sequence, the gene coding for the signal peptide, and the gene coding for the desired exogenous protein and yet contains no gene coding for an excretable protein at the portion other than said base sequence portion. Also, the present invention concerns a vector, a recombinant DNA and a cell transformed with the recombinant DNA capable of producing the desired exogenous protein which can be used in the above process. Thus, the vector according to the present invention comprises a gene comprising a gene A coding for a signal peptide capable of linking the structural gene of a desired exogenous protein to the gene A immediately after the end on the downstream side of that gene A , and is proliferable within a predetermined host cell. Thus, the recombinant DNA according to the present invention comprises a gene which comprises a gene of a desired exogenous protein, and a vector which comprises a gene comprising a gene A coding for a signal peptide capable of linking a structural gene of a desired exogenous protein to the gene immediately after tHe end on the downstream side of that and also is proliferable within a predetermined host cell. Thus, the cell capable of producing a desired exogenous protein according to the present invention comprises a transformant transformed with a recombinant DNA proliferable within the cell, formed by use of a gene which comprises the gene of a desired exogenous protein and a vector which comprises a gene comprising a gene A coding for a signal peptide capable of linking the structural gene of a desired exogenous protein to the gene A immediately after the end on the downstream side of that gene A , and is proliferable within a predetermined host cell. The term exogenous protein is herein used interchangeably with foreign protein . The present invention thus relates to a process for producing a protein according to a genetic engineering technique characterized, in order to permit the protein derived from an exogenous gene to be excreted out of the cell by a signal peptide, by the utilization of a vector a which contains a gene comprising a gene A coding for a signal peptide which is capable of linking the structural gene of a desired exogenous protein to the gene A at the end on the downstream side immediately thereafter, and b which is proliferable in a predetermined host cell, hereinafter referred to as vector having excretion function . According to a preferred embodiment of the present invention, for facilitating linking of the structural gene of a desired protein to the signal peptide gene immediately after the end on the downstream side thereof, the gene coding for the signal peptide employed has a restriction endonuclease recognition site artificially created, at least one of base pairs of a gene comprising the signal peptide gene as being at least a part of the constitutent members. In creating of such a site, it is possible to utilize the fact that there is degeneracy in the codon comprising DNA base pairs. Thus, if the restriction endonuclease recognition cleavage site is cleaved with the restriction endonuclease, when the cleavage side exists in contact with the signal peptide gene DNA at its end on downstream side, by preparing an exogenous gene having formed an end complementary to the end cleaved by the restriction endonuclease on the upstream side thereof and linking this gene to the signal peptide gene at the above cleaved end, the exogenous gene can be linked directly to the signal peptide gene on its downstream side. On the other hand, when the cleavage site of the signal peptide gene exists upstream of the downstream side end, the portion of the gene on downstream side of the cleavage sit. can be synthesized and linked to the upstream side of the exogenous gene to prepare a fragment, after which the fragment is linked simi larly as described above, whereby the signal peptide gene once cleaved can be restored on both chains ofDNA, and also a structure having the exogenous gene linked directly to its down stream side can be realized.Further, when the cleavage site is such that a cohesive end is produced upon enzymatic digestion, the single strand DNA part of the cohesive end thus produced will be hydrolyzed to form a blunt end by means of S1 nuclease or DNA polymerase, which blunt end will in turn be linked to a gene having an exogenous gene prepared as described above. Consequently, since the desired protein is linked immediately after the signal peptide, through the function of the signal peptide as mentioned above, the protein after expression within the host cell can be excreted out of the cell. In the excretion of the desired protein, the signal peptide is hydrolyzed by a membrane enzyme, viz. signal peptidase, and therefore, no superfluous amino acid is linked to the protein to produce a complete mature protein. Thus, the process for producing a protein of the present invention makes possible the production of a desired protein as a mature protein by utilizing skillfully the function of secretion out of the host cell by a signal peptide. The expression secretion out of the host cell as herein used means migration of the desired protein from within the host cell to the periplasm space between the cytoplasmic membrane and the cell wall when the host is a Gram negative microorganism such as E. coli, and migration out of the cell membrane in the case of a Gram positive microorganism such as B. subtilis. The process for producing a protein according to the present invention comprises incorporating a desired gene into a vector having secretion function, then transforming a host with the use of the gene on the vector, culturing the host microorganism, and thereafter recovering the desired protein. Furthermore, the present invention concerns the vector, the recombinant DNA, and the cell capable of producing the desired exogenous protein in which the recombinant DNA is incorporated. Accordingly, the present invention, while accomplishing the above objects, has the following advantages a Purification of the desired protein is simple.In the prior art, after growth of the host microorganism, the host microorganism was lysed, and the desired protein was obtained by extraction from miscellaneous substances inherently possessed by the microorganism, whereby an enormous amount of labor was required and some substances could not be easily purified. According to the process for production of protein of the present invention, through the function of the signal peptide as described above, the protein produced is excreted out of the cells, and the desired substance can be easily identified in and recovered from the culture medium because the culture medium for growth of the microorganism is constituted of known components. b The protein produced can be protected from decomposition with peptidase. That is, various peptidases proteases exist within the microorganism cell and proteins produced in and unnecessary to the cell will rapidly be hydrolyzed. If the desired useful proteins are immediately permitted to be excreted out of the microorganism cell without remaining in the microorganism cell, the protein desired can be protected from digestion by the above hydrolyzing enzymes. c Any desired exogenous protein can be expressed. That is, according to the hybridized protein method, for obtaining a desired protein in pure form, a super fluorous protein must be cleaved by the cyanogen bromide treatment specific for methionine Science, 198, 1056 1977 , or the tripsin digestion specific for lysine or alginine must be conducted Nature, 285, 456 1980 .In such cases, when these amino acids are contained in the amino acid composition of the desired protein, cleavage will also occur at that portion, whereby the desired protein cannot be obtained in its complete form.On the other hand, when a protein is to be produced according to the direct expression method Nature, supra , the gene is required to have an initiation codon methionine at its N terminal, and the protein obtained may sometimes have methionine at its Nterminal see Japanese Patent Laid Open Publication No. 68399 1981 . Since it is technically difficult to remove such terminal methionine by decomposition with the cyanogen bromide treatment, the protein produced will consequently become different in nature from the natural product. In contrast, when the exogenous gene is expressed in the host microorganism with the use of the recombinant plasmid according to the present invention as the vector, the protein once produced as the fused or hybridized protein with the signal peptide is cleaved specifically at the signal peptide portion with the signal pep tide within the host microorganism and excreted as a mature protein with a desired composition out of the host microorganism cell. Further, a preferred embodiment of the process for producing a protein of the present invention is a process for producing a protein by a genetic engineering technique, untilizing a recombinant plasmid, obtained as the result of linking a gene coding for a desired exogenous protein to the gene coding for a signal peptide immediately after the end on the downstream side thereof, which contains a base sequence comprising at least the gene coding for promoter, the gene coding for SD sequence, the gene coding for the signal peptide, and the gene coding for the desired exogenous protein and yet contains no gene coding for an excretable protein at the portion other than the base sequence portion. According to this process, the desired protein can be excreted efficiently out of the microorganism cell. That is, by use of a plasmid in which no gene coding for excretable proteins other than the gene coding for the desired protein exists, the desired protein produced can efficiently be excreted out of the microorganism cell without competition with the already existing excretable proteins derived from the plasmid. BRIEF DESCRIPTION OF THE DRAWINGS Fig. 1 is a schematic diagram showing the base sequence near the end of the signal peptide 1 of pTA529 and indicates the restriction endonuclease recognition site 2 , the restriction endonuclease cleavage site broken line and the signal peptide cleavage point 4 Fig. 2 is a flow chart of the production of pTA1529 Fig. 3 is a schematic diagram showing the base sequence of the structural gene of the hUG and Leu21 hUG synthesized Fig. 4 is a flow chart of the production of the structural gene of pTA1522 pTA1523 when containing .21 Leu hUG containing the structural gene of hUG 21 Leu hUG Fig. 5 is a graph indicating the results of theRRA method Fig. 6 is a Sephadex commat G 50 column chromatogram Fig. 7 is a DEAE Sepharose DE 52 column chromatogram Fig. 8 is a graph indicating an elution pattern by HPLC Fig. 9 is a flow chart of the production of pTA1502 Fig. 10 is a flow chart of the production of pTA1524 Fig. 11 is a flow chart of the production of pTA1504 Fig. 12 is a flow chart of the production of pTA1526 Figs. 13 15 are flow charts of the production of pTA1632 Fig. 16 is a flow chart of the production of pTA2529 Fig. 17 is a flow chart of the production of pTA2539 Fig. 18 is a flow chart of the production of pTA2522 containing the structural gene of hUG and Fig. 19 is a flow chart of the production of pTA2532 containing the structural gene of hUG. DETAILED DESCRIPTION OF THE INVENTION Gene coding for desired protein The gene structural gene coding for a desired protein contemplated may include those of eukaryotic cell proteins such as insulin, serum albumin, human growth hormone, interferon, and epithelial cell growth factor, which may be either prepared from a natural product chromosome DNA or synthesized. The methods for preparation of the genes coding for these desired proteins are described in various textbooks, literatures, laid open or published patent specifications, of which the principal ones are enumerated below. 1 Methods for preparing a desired gene directly from chromosome DNA Proc. Japan Acad., 55B, 464 1979 , Japanese Patent Laid Open Publications Nos.13099 1982, 166992 1982, 208994 1982 and 13599 1983 2 Methods in which messenger RNA hereinafter called mRNA is prepared, and a desired gene is prepared after the known necessary steps on the basis of the mRNA Japanese Patent Laid Open Publications Nos.56684 1983, 2998 1981, 36499 1981, 63996 1981, 85296 1981, 104897 1981, 131522 1981, 131598 1981, 150100 1981, 154499 1981, 158793 1981, 158799 1981, 24400 1982, 141287 1982, 171999 1982, 174085 1982, 206700 1982 and 56684 1983 3 Methods for preparing a gene coding for a desired protein by chemically synthesizing a DNA on the basis of an amino acid sequence of a protein produced by the desired gene Japanese Patent Laid OpenPublications Nos. 122096 1982, 200343 1982, 200400 1982 and 10600 1983 . From among the above methods, the most suitable for reference is determined according to the gene coding for the desired protein. In the modes of practice of the present invention set forth below, the chemically synthesized structural gene of human urogastrone hereinafter called hUG was employed. The details of its manipulation are disclosed in JapanesePatent Application No.175742 1983. Vector having secretion function In genetic manipulation, vector refers to a vehicleDNA having a role of transferring an exogenous gene a gene coding for a desired protein into a host cell and proliferating the exogenous gene in the cell, and a plasmid or a phage existing outside of chromosome has the above proliferating function in a microorganism cell. In the present invention, as the vector having the above function and the secretion function, it is possible to use the above plasmid or phage in which aDNA gene having the secretion function as previously proposed by us Japanese Patent Application No.140748 1983 is incorporated. A typical example of such a vector is the vector plasmid pTA1529 employed in the present invention. The plasmid pTA1529 was prepared from pTA529 made from pYK 283 deposited as E. coli K12C600 pYK283 FERM BP No.556 according to the method disclosed in Japanese Patent Application No.140748 1983 and pHSl. This plasmid was made by digesting pBR322 with restriction enconucleases EcoRI and HindIII and substituting theEcoRI HindIII portion with the following synthetic linker EMI14.1 See Japanese Patent Laid Open Publication No.71692 1984 about its details. As to the construction procedure, see the experimental examples presented hereinafter. Thus, the gene coding for the signal peptide in the above mode of practice is derived from an alkaline phosphatase.Otherwise, there may be employed any desired gene derived from 6 lactamase lipoprotein, etc. . Such a gene may be procured either from a natural product or synthesis, as a matter of course. en it is to be procured by way of synthesis, the gene can be designed and synthesized by appropriate selection of the codons corresponding to the amino acids, if necessary.Recombinant DNA 1 Construction A recombinant DNA is constructed by incorporating the gene coding for a desired protein at an appropriate position in a vector which is devised to express and excrete an exogenous desired protein gene.The incorporation manipulation itself can be performed according to a conventional manner known in the field of molecular biology. Typical specific methods are described in the experimental examples given hereinafter.2 Linker In incorporating the structural gene into a vector, use of various synthesized linkers is an effective measure in recombinant DNA technique for the purpose of matching in reading frame or introducing desired cleaved fragments of restriction endonuclease or ribosome binding site. In one mode of practice of the present invention, a linker is also used for the purpose of matching in reading frame. That is, a synthetic linker is used for the purpose. of utilizing the secretion function of the signal peptide and obtaining the desired protein without attachment of superfluous amino acids. As for practical application examples, see the examples given hereinafter. Synthesis of a linker can be accomplished by any desired method in which each of strands divided into several fragments are chemically synthesized, and the respective segments are ligated to each other.Examples of the synthetic method for fragments are the solid phase method Nucleic Acids Research, 8, 5491 1980 , the liquid phase method and respective diester method Science, 203, 614 1976 , the triester method Science, 198, 1056 1977 , and the method employing an enzyme J. Biol. Chem., 241, 2014 1966 . On the points of synthetic time, conversion, purification, etc., the use of the solid phase method by way of the triester method is preferred.3 Judgement of direction The direction of the gene coding for the desired protein incorporated in the vector can be judged by cleaving the specific site included within the structural gene with an enzyme capable of recognizing the site, cleaving a specific site outside of the structural gene with another enzyme and analyzing the size of the fragment obtained.Recombinant plasmid In manipulation of genes, plasmid refers to a genetically stable genetic factor which proliferates autonomously and independently of the host chromosome in the cytoplasma of a microorganism and is useful as a vehicle DNA having a role of introducing an exogenous gene a gene coding for a desired protein into the host cell and proliferating the exogenous gene in the cell. The plasmid according to one mode of practice of the present invention has the above function and also a secretion function, in which no gene coding for excretable proteins other than the gene coding for the exogenous protein to be incorporated exists at all. A plasmid having a secretion function as herein mentioned has a base sequence comprising at least a gene coding for a promoter, a gene coding for a SD sequence, a gene coding for a signal peptide, and a gene coding for the desired exogenous protein. a Promoter In a plasmid, the promoter is one of the most important regions related to expression efficiency of the plasmid. The gene coding for the promoter to be used in the present invention, namely the DNA region to be recognized by RNA polymerase for transcription, may be procured from a natural chromosome DNA, or it is also possible to use a synthesized one, since the base sequences of a large number of promoter genes have already been determined. Further, an entirely new promoter may be synthesized and used. See JapanesePatent Laid Open Publication No.71692 1984. Among these, preferable promoter regions are those with strong promoting action, such as Trp promoter,recA promoter, PL promoter, and lac promoter. ProteinNucleic Acid Enzyme, 26, 443 1981 . In the examples set forth hereinafter, a promoter derived from an alakline phsphatase is employed. b SD sequence This is also called the Shine Dalgarno sequence which refers to the sequence of mRNA existing immediately before the translation initiation codon AUG or GUG and linked to the 3 end sequence of 16sRNA of ribosome, which can either be procured from nature or chemically synthesized, similarly as the promoter. c Signal peptide The signal peptide in the plasmid in the present invention is preferably one possessing one restriction endonuclease recognition site for, e.g. Hind III, NaeI, etc. which has been created artificially in the gene containing the DNA portion corresponding to the gene coding for the signal peptide, as we have described inJapanese Patent Application No.140748 1983, and which has at least one of the base pairs of the DNA as at least a part of the constituent members. See the specification of the above application for details. The gene coding for such a signal peptide is as described above. The recombinant plasmid to be described in one mode of practice of the present invention is obtained by cleaving away a gene coding for an excretable protein other than the desired exogenous protein existing in the plasmid having the secretion function as described above, for example, an ampicillin resistant gene r hereinafter indicated as Amp , in a conventional manner. According to necessity, after removal of the gene coding for the excretable protein, non excretable drug resistant marker drug resistant marker coding for no excretable protein , for example, a kanamycin resistant gene hereinafter indicated as Kmr , and a tetracycline resistant gene hereinafter indicated as Tcr can be introduced as the selection marker. A typical example of such a plasmid is the plasmid pTA2529 used in the present invention. The plasmid pTA2529 was constructed by cleaving the pTA1529 as described above with restriction endonucleases PstI and rAatII in a conventional manner to remove Amp , which step was followed by introduction of a non excretable drug resistant marker, for example, kanamycin resistant gene Kmr thereinto. As to the construction operation, see the experimental examples set forth hereinafter. The recombinant plasmid, which is one mode of practice of the present invention, can be constructed by incorporating the gene coding for a desired protein at an appropriate site in the plasmid as prepared above which has been devised to express the exogenous desired protein gene and excrete the protein.The incorporation operation itself can be performed by following a known procedure known in the field of molecular biology.As to the details of the method, reference is made to the examples presented hereinafter. The excretable protein other than the desired protein as mentioned in one mode of practice of the present invention refers to a protein excretable out of the microorganism cell encoded by the plasmid used as the vector other than the gene coding for the desired exogenous protein to be incorporated, as exemplified by 6 lactamase Ann. Rev. Microbiol., 36, 435 1982 . The gene coding for such a protein is called, in the present invention, the gene coding for an excretable protein.Cell producing desired exogenous protein transformant In the present invention, the cell producing the desired exogenous protein is a microorganism cell transformed with the above recombinant DNA transformant .The microorganism into which the above recombinant can conveniently be transferred hereinafter sometimes referred to as host microorganism is preferably one in which the recombinant DNA as constructed above is proliferable, and of which genetic characteristics are clarified in detail.1. Host microorganism The host microorganism can be any microorganism, provided that the above recombinant DNA is proliferable in its cells. Specifically, any desired host such asE. coli, B. subtilis, yeast, etc. can be used. Specific examples of such host microorganisms are E. coli strains belonging to the genus Escherichia, K12C600 FERM BP No.115 , K12YK537 FERM P No.7941 and XA35 FERM BP No.116 . On the other hand, in one mode of practice of the process for production of a protein of the present invention, a recombinant plasmid is employed, in which the gene coding for a desired exogenous protein is linked to the gene coding for the signal peptide immediately thereafter, and the existing gene coding for excretable protein other than the gene coding for the exogenous protein is artificially removed.This is done for further enhancing the extracellular secretion of the desired protein, and the host microorganism to be used in this case is preferably one in which expression of excretable protein derived from the host chromosome is suppressed or defective. A specific example of such a host microorganism is anE. coli strain YK537 belonging to the genus Escherichia supra . The excretable protein derived from the chromosome as mentioned herein refers to a protein excretable out of the cell encoded by the chromosome gene of the host microorganism, as exemplified by alkaline phosphatase, cell wall principal lipoproteins, cell wall OmpC protein, cell wall ompF protein, and cell wall OmpA protein,Ann. Rev. Microbiol., 36, 435 1982 1. Another example of the host microorganism which can be contemplated is Bacillus subtilis. B. subtilis is a Gram positive bacterium, capable of forming spores, and is also utilized in industry as a microorganism capable of producing extracellular enzymes. Examples of strains suitable as the host are RM125, a mutant ofMarburg 168 Molec. Gen. Genet., 152, 65 1977 , the same strain deficient in spore forming ability NIH guideline, Federal Register 45, 6724 1980 1, and the strain deficient in recombination ability recE Molec. Gen. Genet., 165, 269 1978 . As to utilization of B. subtilis, reference may be made to various literatures e.g. Protein Nucleic Acid Enzyme, ExtraVolumes, Gene Manipulation , 26, 464 475 1981 , 28, 1468 1479 1983 , ibid. Gene ManipulationIntroduction of Recombinant Gene into Cell and Expres sion , 2.Transformant The cell producing the desired exogenous protein or transformant as mentioned in the present invention refers to a microorganism host microorganism transformed by transfer of the above secretion vector recombinant DNA in which the gene coding for a desired exogenous protein is incorporated, and the host microorganism. is one in which the vector is proliferable. The transformation operation itself may be conducted according to any of the conventional methods known in the field of molecular biology. For instance, theCushner method will be used when transformation of E.coli is desired genetic Engineering 1978, 17 1978 . The genetic method for transformation of E. coli can be used for transformation of B. subtilis, as specifically taught by a textbook Genetic Recombinant ApplicationTechnique 3 , published by Science Forum Co., Japan, p. 33 1982 and a textbook Molecular cloning, a laboratory manual , p. 247 268, published by ColdSpring Harbor Laboratory 1982 , U.S.A. The strain transformed by the transforming operation can be selected by means of an index of a marker of the recombinant DNA drug resistance, nutrient demand, etc. created by transfer of the recombinant plasmid into the transformant. A typical example of such a transformant is the transformant K12C600 pTA1524 obtained by transformation of the E. coli K12C600 with pTA1524 as described in detail hereinafter . This transformant is a micro organism strain different from the host microorganism K12C600 in the following properties Amp r Tc Therefore, with these properties as the index, the transformant can be selected. Another typical example of the transformant in the present invention is the transformant obtained by transformation of the E. coli K12YK537 with pTA2532 Ref. experimental example presented hereinafter .This transformant is a microorganism strain different in the property of kanamycin resistance Kmr from the host microorganism K12YK537. Therefore, the transformant can also be selected with this property as the index. Production and recovery of desired protein The desired protein can be produced by culturing the host microorganism namely the above transformant in a conventional manner. Specific details of the culturing method are disclosed in the experimental examples given hereinafter. For recovery of the desired protein from microorganism cells or the culture broth when the host microorganism is a Gram negative microorganism such asE. coli, since the desired protein is secreted into periplasm through the secretion function of the signal peptide, the desired protein can readily be recovered after it is released from the periplasm outside of the cell wall into the culture medium according to the osmotic shock method J. Biol. Chem., 240, 3685 1965 . When the host microorganism is a Grampositive microorganism such as B. subtilis, since the desired protein is released out of the cells through the secretion action of the signal peptide the method for producing a desired protein by use of B. subtilis is described, for example, in Gene, 22, 229 235 1983 1, it can be readily recovered from the culture medium.Details of the method for culturing and recovery of the desired protein after the osmotic shock method in the case of the host microorganism of E. coli, are described in the experimental examples set forth hereinafter. EXPERIMENTAL EXAMPLES Reference Examples Preparation of a vector having secretion function was carried out as follows. As the vector to be used in the process of the present invention, the plasmid pTA529 previously proposed by us Japanese Patent Application No.140748 1983 having the terminal codon on the downstream side of the DNA portion corresponding to the gene coding for the signal. peptide of a plasmid pYK283, deposited as E. coli K12C600 pYK283 at the FermentationResearch Institute, Agency of Industrial Science BR Technology FERM BP No. 556 , and the initial codon of the DNA portion directly linked thereto on its downstream side as shown in Fig. 1 was further improved to prepare a vector pTA1529 having a still stronger selfreplicating ability. In this connection, Fig. 1 shows a part of the DNA gene comprising a double stranded DNA, in which A, G,C and T represent adenine, guanine, cytosine and thymine, respectively, and Lys, Ala and Trp represent lysine, alanine and tryptophane, respectively. The region 1 in the double stranded DNA is the signal peptide gene portion, the region 3 the recognition site of the restriction endonuclease Hind III, and the broken line shows the HindIII cleavage site. The region 2 is theDNA portion linked to the signal peptide gene DNA immediately thereafter on the downstream side. Preparation of pTA1529 was carried out following the procedure as described below. In 50 p1 of a reaction mixture 50 mM Tris hydrochloride buffer hereinafter referred to Tris HCl pH 8.0 , 10 mM magnesium chloride hereinafter written MgC12 , 5 g of pTA529 Fig. 2 was hy drolyzed with 10 units of a restriction endonucleaseTha I BRL Co. hereinafter referred to as Tha I at 600C for one hour.Then, ethanol precipitation was carried out, and the precipitated fraction obtained was hydrolyzed in 50 p1 of a reaction mixture 10 mM Tris HCl pH 8.0 , 7 mM MgC12, 100 mM sodium chloride hereinafter referred to as NaCl with 4 units of a restriction endonuclease BamHI Takara hereinafter written BamHI and was then subjected to polyacrylamide gel electrophoresis to obtain a DNA fraction of about 220 base pairs hereinafter abbreviated as bp represented by Win Fig. 2 containing the desired alkaline phosphatase signal peptide region. On the other hand, 10 g of the plasmid pHSl seeJapanese Patent Laid Open Publication No. 71692 1984 was hydrolyzed with 4 units of a restriction endonuclease PstI Takara hereinafter referred to as PstI in 50 l of a reaction mixture 10 mM Tris HCl pH 7.5 , 10 mM MgCl2 and 100 mM NaCl at 370C for one hour.Then, ethanol precipitation was carried out, and the precipitate obtained was divided into two aliquots, one of which was hydrolyzed with 40 units of BamHI in 50 p1 of the above reaction mixture, which step was followed by agarose gel electrophoresis to obtain a DNA fragment of 330 bp represented by c in Fig. 2 S.On the other hand, the other aliquot of the ethanol precipitate was hydrolyzed similarly with 4 units of a restriction endonuclease SaaI Takara hereinafter referred to as SmaI in 50 ul of a reaction mixture 6 mM Tris HCl pH 8.0 , 20 mM potassium chloride hereinafter referred to as KC1 , 6 mM MgC12 at 370C for one hour, after which agarose gel electrophoresis was carried out to obtain a DNA fragment of 700 ba represented byl in Fig. 2 Q2 L. The three DNA fragments obtained in the above operations , S and L were allowed to react with the use of 30 units of T4 DNA ligase Takara in 30 ul of a reaction mixture 20 mM Tris HCl pH 7.5 , 10 mM MgC12, 10 mM dithiothreitol hereinafter referred to as DTT , 0.5 mM adenosine triphosphate hereinafter referred to as ATP at 140C for 16 hours.After the reaction, the E. coli K12C600 was transformed with the reaction product already deposited as E. coliK12C600 FERM BP No. 115 to obtain a transformed strain containing the desired plasmid pTA1529 . From the strains thus obtained, plasmids were prepared the details of the preparation method are disclosed inJapanese Patent Application No.140748 1983 , and the base sequences at the linking portions of the respective DNA fragments were confirmed according to the MaxamGilbert method Proc. Natl. Acad. Sci. USA, 74, 560 1977 1. Example 1 Human urogastrone hUG EGF and 21 leuchine human urogastrone 21 Leu hUG in which methyonine of the 21st amino acid in the human urogastrone was replaced with leucine were produced according to the process of the present invention and recovered. Since the procedures for preparation of hUG and 21 Leu hUG are similar, the preparation of hUG will be described primarily. Synthesis of the structural gene of hUG and 21 Leu EGF Synthesis of the structural gene of hUG and 21Leu EGF was carried out according to the process as disclosed in Japanese Patent Application No.123520 1983.More specifically, fragments of chain lengths of 10 17 were previously synthesized, and then these synthetic fragments were subjected to linking reactions for respective blocks, followed finally by binding of these blocks to synthesize a complete hUG structural gene.The base sequence of ahe hUG structural gene is as shown in Fig. 3, in which the mark indicates the initiation point, and the mark the termination point. In Fig.3, Met, etc. denote amino acids such as methionine, etc., and A, T, C and G denote bases, indicating abbreviations for adenine, thymine, cytosine and guanine, respectively, as approved in this field of the art. Further, the numerals in Fig. 3 indicate the number of the bases in the fragment, and the notation such as EcoRI indicates the restriction endonuclease recognition site. In the base sequence of 21 LeuhUG, the 21st codon is changed to that coding forLeu in place of Met, as written Leu above Met inFig. 3, the base sequence being otherwise the same as in hUG.Preparation of a plasmid containing the hUG 21 Leu hUG gene see Fig. 4 Hydrolysis of 5 pg of pTA1529 was conducted in 50 pl of a buffer 10 mM Tris HCl pH 7.5 , 10 mM MgCl2, 50 mM NaC1 with 4 units of a restriction endonucleaseHindIII Takara hereinafter referred to as HindIII at 370C for one hour. The precipitate obtained by ethanol precipitation was treated in 30p1 of a reaction mixture 67 mM Tris HCl pH 8.8 , 16.6 mM ammonium sulfate hereinafter referred to as NH4 2SO4 , 6.7 mMMgCl2, 10 mM 2 mercaptoethanol, 6.7 pM ethylenediaminetetraacetic acid hereinafter referred to as EDTA , 0.66 mM each of dATP, dCTP, dGTP and TTP with one unit of T4 DNA polymerase at 370C for 15 minutes.Subsequently, ethanol precipitation was carried out, and the resultant precipitate was hydrolyzed in 50 p1 of a reaction mixture 16mM Tris HCl pH 8.0 , 6 mM MgC12 and 150 mM NaCl with 4 units of a restriction endonucleaseSail Takara hereinafter referred to as SalI at 370C for one hour. After completion of the reaction, a DNA fragment of 3,900 bp Fig. 4 commat was obtained by agarose gel electrophoresis. Hydrolysis of 5 Vg of the plasmid pBR322 hUG obtained by incorporating the fragment of the hUG structural gene synthesized as described above formed by digestion with EcoRI and SalI into the pBR322 deposited as E. coli K12C600 pBR322 FERM P No.235 digested with EcoRI and SalI was conducted in 50 ul of a reaction mixture 1100 mM Tris HCl pH 7.5 , 50 mM NaCI and 50 mM MgC12 with 4 units of a restriction endonuclease EcoRI TTakaraf at 370C for one hour, followed by the T4 DNA polymerase reaction similarly as described above and further by the SalI treatment, and a DNA fragment of 160 bp Fig. 4 commat was obtained by agarose gel electrophoresis.In the case of 21Leu hUG, the structural gene was cleaved out similarly from the pLE6527 Japanese Patent Application No.123520 1983, deposited as E. coli XA35 pLE6527 FERM BP No. 514 . The two DNA fragments prepared as described above Fig. 4 0 and Q were subjected to reaction in 30 ul of a reaction mixture 20 mM Tris HCl pH 7.5 , 10 mM MgCl2, 10 mM DTT and 0.5 mM ATP with the use of 300 units of T4 DNA ligase Takara at 140C for 16 hours.After completion of the reaction, E. coli K12C600 was transformed with the reaction product to obtain a transformed strain containing the desired plasmid lhereinafter referred to as pTA1522 Fig. 4 3 E. coli K12C600 pTA1522 . Such a plasmid in 21 Leu hUG was pTA1523, and by following the same procedure as in the case of pTA1522, a transformed strain E. coliK12C600 pTA1523 was obtained. Similarly as in the above experimental example, plasmids were prepared from the strains obtained here see Japanese Patent Application No. 140748 1983 , and the base sequences at the linking portions of the respective DNA fragments were confirmed according to the Maxam Gilbert method supra .Production and recovery of hUG 21 Leu hUG 1 Cultivation of the transformed strain The transformed strain E. coli K12C600 pTA1522 was precultured in 80 ml of an L broth 1 bactotripton, 0.5 yeast extract, 0.5 NaCl, 0.1 glucose containing 20 pg ml of ampicillin. Then, the precultured broth was transferred into a synthetic medium Biochim. Biophs. Acta 38, 470 1960 containing 0.64 mM potassium dihydrogen phosphate hereinafter referred to as.KH2PO4 , and shaking culture was conducted at 370C overnight. After completion of cultivation, the microorganism cells were collected and suspended in the above mentioned synthetic medium, followed further by shaking culture at 370C for 5 hours.Also, for 21 Leu hUG, cultivation was carried out similarly. The culture medium had the following composition 0.1 M Tris HCl pH 7.2 , 80 mM NaCl, 2 mM KC1, 20 mMNH4C1, 3 mM Na2SO4, 1 mM MgCl21 0.2 mM Cacti2, 2 pM FeC13, 2 uM Zinc12, 0.2 glucose, 20 pg ml ampicillin, 40 pg ml leucine, 40 pg ml threonine and 10 pg ml thiamine. 2 Recovery of hUG 21 Leu hUG by the osmotic shock method The hUG or 21 Leu hUG excreted in the periplasm of the host microorganism was recovered by permitting it to be released out of the microorganism cell accprding to the osmotic shock method 1J. Biol. Chem. 240, 3685, 1965 1. That is, the microorganism cells collected were suspended in 120 ml of a reaction mixture 20 sucrose, 30 mM Tris HCl pH 8.0 , 1 mM EDTA and left to stand at room temperature for 10 minutes. Then, the cells were collected, suspended in 80 ml of cold water, left to stand in an ice bath for 10 minutes, and thereafter subjected to centrifugation, and the supernatant was recovered.3 Quantitative determination of hUG 21 Leu hUG A part of the supernatant obtained by the osmotic shock method was diluted 3 fold, and quantitative determination of hUG and 21 Leu hUG was conducted by use thereof the method of A. King et al J. B. C., 257, 3053 1982 1. More specifically, KB cells derived from human rhinopharynx epithelial cancer ATCC No. CCL17 were subjected to mono layer culturing in Dulbecco s modified Eagle medium DME Nissui inan 800 ml flask. The medium was removed, and the cells were peeled off with a phosphate buffered saline solution PBS containing 0.05 EDTA to prepare a cell suspension. Then, the cells were washed twice with Hanks buffered saline solution HBSS containing 20 mM HEPES pH 7.4 .After the cells were suspended in Binding solution DME medium, 20 mM Hepes pH 7.4 , 0.35 g l NaHCO3, 100 pg ml streptomycin , the number of cells was calculated and adjusted to 300,000 to 400,000 0.2 ml Binding solution, and 0.2 ml each of the suspension was apportioned into a tube. 0.2 ml of each sample solution containing hUG or 21 Leu hUG and I mEGF mouse EGF at various concentrations was added into the tube, after which incubation was carried out at 370C for one hour. The cells were washed with ice cooled HBSS x 3 and suspended in 10 trichloroacetic acid hereinafter referred to asTCA , and the cells were fixed with the use of a glass filter. After removal of TAC with acetone, counting was performed by means of a liquid scintillation counter. For comparison, the same experiments were conducted for the peptide fractions obtained by the cyanogen bromide treatment of mEGF Toyobo and S galacto sidase. Fig. 5 shows the results obtained in which the marks 0 denote mEGF and the marks 0 hUG . As is apparent from Figure 5, the peptide fraction containing hUG interferes competitively with binding between the 125I mEGF and the KB cell EGF receptor, its dosereaction curve being fairly identical with that of mEGF. As the result, it was confirmed that hUG contained a peptide having EGF activity of 6.24 ng l as calculated on mEGF. This means that 208 pg per liter of culture hereinafter written pg liter culture was produced by excretion. Also, for 21 Leu hUG, quantitative determination was conducted similarly to obtain the result that 185 pg liter culture was excreted.Purification of hUG 21 Leu hUG The supernatant obtained by the osmotic shock method was lyophilized into powder, which was dissolved in 6 ml of water and applied onto a SephadexCR G 50 column diameter 7.5 cm x length 90 cm equilibrated with 25 mM ammonium acetate hereinafter referred to asAcONH4 pH 5.8 . Elution was performed at a flow rate of 9.5 ml fraction 14 minutes, and 34 to 39 fractions having activities were collected. The elution pattern and the results of measurement of activities according to the above RRA method when subjected to Sephadex G 50 are shown in Fig. 6, in which o o indicate activities. Subsequently, the fractions 34 39 fractions were applied onto the DEAE Sepharose DE 52 Whatman Co. diameter 1.2 cm x length 12 cm equilibrated with 25 mM ACONH4 pH 5.8 . During the elution, the buffer of AcONH4 was employed at a concentration gradient of from 25 mM to 300 mM 300 ml , elution was performed at a rate of 3.12 fraction 7.6 min., and 39 to 45 fractions having activities were collected. The elution pattern and the results of measurement of activities according to the above RRA method are shown in Fig.6, in which o indicate activities. Subsequently, the fractions having activities thus obtained were purified by high performance liquid chromatography hereinafter referred to as HPLC UBondapak C 18, column diameter 0.6 cm x length 30 cm, elution 0.1 TFA, concentration gradient of acetonitrile from 208 to 50 , flow rate 1 ml min. . The elution pattern was as shown in Fig. 8, in which a single peak ascribable to hUG was obtained at the retention time of 24 minutes. Purification was also conducted similarly for 21 Leu hUG.Confirmation of hUG 21 Leu hUG Confirmation of hUG 21 Leu hUG was performed by analysis of.amino acid composition, N end analysis and C end analysis. Analysis of amino acid composition was carried out by hydrolyzing the fractions purified by the aboveHPLC with 6N HCl, drying the hydrolyzate, dissolving the dried product in 0.02 HC1 and subjecting the resultant solution to an amino acid analyzer HLC 803D OPA system Toyo oda . The results obtained were as shown in Table 1. On the other hand, N end analysis was conducted according to the Edman method Biological ExperimentalCourse 1, Chemistry of Proteins II, p. 132, edited bySociety of Biochemistry of Japan, published by TokyoKagaku Dojin to confirm the amino acid sequence from the N end to the 20th amino acid. Further, the C end analysis was carried out by the use of carboxy peptidase Y to confirm the amino acid sequence from the C end to the fourth amino acid. From these results, the product excreted was confirmed to be hUG. Also, concerning 21 Leu hUG previously proposed by us, it was confirmed according to the same series of operations as described above. Table 1EMI31.1 tb SEP Amino SEP acid SEP composition tb Amino SEP relative SEP ratio SEP Literature tb acid SEP Literature SEP tb SEP Measured SEP Integer SEP value tb SEP value tb SEP Asp SEP 7.0 SEP 7 SEP 7 tb SEP Thr SEP O SEP 0 SEP 0 tb SEP Ser SEP 3.2 SEP 3 SEP 3 tb SEP Glu SEP 5.1 SEP 5 SEP 5 tb SEP Pro SEP 1.4 SEP 1 SEP 1 tb SEP Gly SEP 4.3 SEP 1 SEP 4 SEP 4 tb SEP Ala SEP 2.3 SEP 2 SEP 2 tb SEP Cys SEP SEP SEP 3 tb SEP Val SEP 2.7 SEP 3 SEP 3 tb SEP Met SEP 0.4 SEP 1 SEP 1 tb SEP Ile SEP 1.9 SEP SEP 2 SEP 2 tb SEP Leu SEP 4.8 SEP 5 SEP 5 tb SEP Leu tb SEP Tyr SEP 4.5 SEP 5 SEP 5 tb SEP Phe SEP O SEP 0 SEP RTI ID 31.3 SEP O SEP tb SEP His SEP 2.2 SEP 2 SEP 2 tb SEP Trp SEP SEP SEP 2 tb SEP Lys SEP 1.7 SEP 2 SEP 2 tb SEP Arg SEP 2.9 SEP 3 SEP 3 tb In the above Table, Asp, etc. are, of course, symbols indicating amino acids such as aspartic acid as approved in this field of the art. Comparative Example 1 In Example 1, hUG 21 Leu hUG was produced by the use of a plasmid having a signal peptide. As a comparative test for examination of the secretion function of the signal peptide, a plasmid pTA1502 having no signal peptide was prepared and production of hUG was attempted similarly. In the following description, no detailed description is given relative to the restriction endonucleases, ligase reaction and preparation of DNA fragments, for which the same descriptions as inExample 1 are applicable. Also, a restriction endonuclease is denoted, for example, by AatIIPreparation of pTA1502 see Fig. 9 After pTA1522 supra was digested with AatII and RsaI, a small fragment was obtained in Fig. 9, Ql . . Similarly, after treatment of pTA 1522 with BglII and AatII, a large fragment was obtained inFig. 9, 02 . On the other hand, by treating pBR322 hUG withEcoRI and BglII, the structural gene of hUG partially deficient was obtained in Fig. 9, . Also, a linker for linking the above DNA fragments Ql through was prepared in the same Figure, . Subsequently, these fragments were bound according to the methods as required to obtain pTA1502 having the structural gene of hUG in the same Figure, 4 . Production, recovery, purification, etc. of hUG were all performed in the same manner as in the aboveExample 1. Quantitative determination of hUG was performed for the three fractions described below when production of protein was conducted by the use of pTA1522 and pTA1502. The results were as shown in Table 2. a The fraction prepared by destroying the microorganism cells after expression of the desired protein. b The supernatant fraction obtained by subjecting the host microorganism to the osmotic shock method after expression of the desired protein. c The fraction obtained by destroying the microorganism cells after the osmotic shock method. Table 2EMI33.1 tb SEP Plasmid SEP a SEP b SEP c tb SEP employed tb pTA1522 SEP 211 SEP g liter SEP 205 SEP g liter SEP 14.4 SEP g tb having SEP culture SEP culture SEP liter tb peptide tb pTA1502 SEP 4.2 SEP pg liter SEP 0 SEP 0 tb having SEP no SEP .culture tb signal tb peptide tb Example 2 According to the process of the present invention, an interferon hereinafter abbreviated INF was prepared. This INF is of the a type which is a novel IFN as proposed previously by us Japanese Patent ApplicationNo.175742 1983 . As the plasmid vector having the secretion function, the above pTA1529 was employed.Preparation of a plasmid having IFN structural gene Fig.10 After digestion of 5 ug of pTA1529 supra withSmaI and HindIII, a large fragment was obtained by agarose gel electrophoresis Fig. 10 . On the other hand, 5 pg of pIF202 having the IFN structural gene seeJapanese Patent Application No.175742 1983 FERM P No.7188 was digested with SmaI, and DdeI, a DNA fragment of the IFN structural gene deficient in a part of the tip end was obtained by agarose gel electrophoresis Fig. 10 C2 . Subsequently, in order to link the DNA fragments and Q2Th that is, so as to be followed by the complete IFN structural gene immediately after the signal peptide according to the object of the present invention , a linker having the following sequence was prepared AGCTTGTGATCTGCC Fig. 10 0 ACACTAGACGGAGT These three DNA fragments Gl, and Q3 were linked together according to the same procedure as in Example 1 and E. coli K12C600 was transformed with the DNA obtained to obtain the desired plasmid pTA1524 Fig. 10 .Also, for the pTA1524, the base sequence was confirmed following the Maxam Gilbert method Proc. Natl. Acad. Sci., USA, supra similarly as in the case of the above hUG. On the other hand, for comparison with pTA1524, a plasmid pTA1504 having no such signal peptide as mentioned above and having theIFN structural gene under its control by having the same drive unit as pTA1524 was prepared as described below see Fig. 11 . After digestion of pTA1529 with HaeII and RsaI, a DNA fragment of 180 bp was obtained by polyacrylamide gel electrophoresis Fig. 11 1 . Then, the fragment was subjected to bacterial alkaline phosphatase treatment to remove the 5 end phosphoric acid groups Fig.11 On the other hand, pIF202 supra was digested with XbaI and SmaI to recover the structural gene of IFN Fig. 10 , which step was followed by T4 DNA polymeraze treatment, to convert the sticky end at the XbaI cleaved site to the blunt end Fig. 10 S2 . Subsequently, the two DNA fragments , were ligated together Fig. 10 3 , , and the ligated product obtained was digested with ThaI to obtain the DNA fragment similarly as above Fig. 10 After insertion of the above DNA fragment into the pHSI supra, see Japanese Patent Laid OpenPublication No. 71692 1984 about its details digested with SmaI, a plasm it pTA1504 was obtained into which the promoter SD sequence and IFN structural gene had been correctly inserted in the same direction. The base sequence was confirmed according to the MaxamGilbert method.Production and Recovery of IFN Expression of IFN was carried out by culturing the microorganism containing the pTA1524 as prepared above E. coli K12C600 in a conventional manner, and confirmation of expression was done ty assay of antiviral activity. 1 Cultivation of microorganism Cultivation of the microorganism was carried out with the use of E. coli K12C600 similarly as in the above Example 1. 2 Assay of antiviral activity After completion of cultivation, the microorganisms were collected and subjected to osmotic shock treatment supra to obtain a supernatant, the antiviral activity of which was assayed Protein Nucleic AcidEnzyme, Etra Volume, No. 25, 360, 1980 . That is, FL cells established cell line derived from human amnion were plated and, after the culture broth was discarded, the supernatant obtained by the above osmotic shock treatment , appropriately diluted, was added to the plate of the FL cells, after which culturing was carried out overnight under the conditions of 370C, 5 CO2, hereinafter the same . After the culture medium was discarded, an appropriate amount of vesicular stomatitis virus Sindbis virus was inoculated onto the plate, and thereafter cultivation was carried out for 48 hours. After the culture medium was discarded, the cytopathogenic effect CPE was measured. The same cultivation as above was also conducted for the pTA1504 having no signal peptide prepared as a comparative example for pTA1524 to prepare a supernatant, the antiviral activity of which was measured. The antiviral activities when IFN was cultured by use of the two kinds of plasmids pTA1524 and pTA1540 as described above were as shown in Table 3. Table 3EMI36.1 tb SEP Plasmid SEP No. SEP Antiviral SEP activity tb SEP unit liter culture tb pTA1524 SEP having SEP 800 tb signal SEP peptide tb pTA1504 SEP having tb no SEP signal SEP O tb peptide tb Example 3 According to the process of the present invention, human growth hormone hereinafter abbreviated hGH was prepared. The structural gene of hGH was chemically synthesized. That is, we have cleaved out the structural gene from the plasmid phGHl containing the hGH structural gene Nucleic Acids Symposium Series, 12, 79 82 1983 and prepared a pTA1526 having incorporated the structural gene together with a synthetic linker in pTA1529, from which hGH was prepared according to the process of the present invention.Preparation of pTA1526 see Fig. 12 After digestion of 5 pg of pTA1529 supra withSalI and HindIII, a large DNA fragment was obtained by agarose gel electrophoresis Fig. 12 After digestion of 5 pg of phGHl supra containing the structural gene of hGH with ClaI and Salt, aDNA fragment of about 600 bp was obtained by agarose gel electrophoresis. Then, the fragment was digested with Hinf I to obtain a large fragment the structural gene of hGH deficient in a part of the tip end Fig. 12 On the other hand, the linker shown below Fig. 12 commat Q was synthesized in order to link the above two DNA fragments so that the complete structural gene of hGH will be linked to the signal peptide immediately thereafter according to the same solid phase synthetic method as in preparation of the previous linker Fig. 12 03, EMI37.1 The three DNA fragments were linked together to obtain the desired plasmid pTA1526. The base sequence of the pTA1526 was confirmed according to the Maxam Gilbert method supra similarly as in the foregoing Examples.Production and recovery of hGH Similarly as in the above Examples, the host microorganism was cultured in a conventional manner, after which hGH was recovered according to the osmotic shock method. Assay of hGH ativity was conducted by means of the kit of PhadebasRHGH Prist Shionogi . As a result, the supernatant fraction obtained by the osmotic shock method was found to contain 45 pg of hGH per liter of culture. Example 4 In the above Examples 1, 2, and 3, the process of the present invention was practiced with the use of pTA1529. In this Example 4, hUG was produced similarly as described above by constructing a plasmid pTA1632 having the secretion function similarly as pTA1529 the signal peptide of pTA1529 derived from alkaline phosphatase and containing the structural gene of hUG the signal peptide of pTA1632 derived from ss lactamase .Construction of pTA1632 1 Synthesis of the signal peptide of ss lactamase hereinafter referred to as bla signal The DNA gene coding for ss lactamase having the sequence as shown below was prepared according to a conventional method for synthesis of oligonucleotide.The amino acid sequence of bla used followed that ofProc. Natl. Acad. Sci., 75, 3737 3741 1978 .EMI38.1 In the above sequence, s  denotes the restriction endonuclease cleavage site.2 Cloning of the bla signal into pBR322 see Fig. 13 The gene of the bla signal synthesized above was constructed following the procedure shown below. After digesting pBR322 with HindIII and PstI, a large DNA fragment was recovered Fig. 13 . Also, after digesting separately pBR322 with PstI and EcoRI, a small DNA fragment was recovered Fig. 13 Q . These two DNA fragments were ligated with the bla signal DNA fragment to obtain a plasmid pBR bla Fig.13 3 . 3 Linking of hUG gene Fig. 14 Following the procedure shown below, the hUG structural gene was inserted into the above plasmid pBRbla . After digesting pBR322 with Sall and EcoRI, a large DNA fragment was recovered Q . After digesting the above pBR bla with EcoRI and NaeI, a DNA fragment of about 70 bp was.recovered . On the other hand, the pBR322 hUG supra was subjected toEcoRI digestion, then to the T4 DNA polymerase treatment and further to SalI digestion to obtain a DNA fragment of about 160 bp . Finally, the above DNA fragments , and were linked to obtain a plasmid pBR bla hUG having the structural gene of hUG linked immediately after the bla signal.4 Linking to tryptophan promoter Trp Fig.15 Following the procedure shown below, a plasmid pTA1632 with hUG being placed under the control of triptophan promoter was constructed. The pTS06l prepared by insertion of Trp promoter into pBR322 at the EcoRI recognition site in the direction of the tetracycline resistant gene, followed by removal of the attenuator portion was digested withClaI,then treated with T4 DNA polymerase and further digested with PstI to recover a small DNA fragment Ql . Also, the above pBR bla hUG was digested withEcoRI, treated with T4 DNA polymerase and further digested with PstI to recover a large fragment Finally, the above two and DNA were linked to obtain pTA1632. The method of obtaining pTA1632 and determination of its base sequence were performed similarly as in the foregoing Examples. Production and recovery of hUG Production and recovery of hUG were conducted similarly as in the above Example 1. As a result, 180 pg of hUG was found to be produced per one liter of culture. Example 5 Preparation of plasmid Preparation of a plasmid having the secretion function and having no gene coding for secretable protein derived from the plasmid was carried out as follows. The steps are as shown in Figs. 16 and 17. 1 Preparation of pTA2529 a After treatment of 5 g of the plasmid 1529 previously proposed by us Fig. 16 0 in 50 ml of a reaction mixture 50 mM Tris hydrochloride buffer hereinafter abbreviated Tris HCl , 50 mM sodium chloride hereinafter abbreviated NarCl , 10 mM magnesium chloride hereinafter abbreviated MgCl2 with 4 units of a restriction endonuclease PstI Takara Co. hereinafter abbreviated PstI 1 p1 and 4 units of a restriction endonuclease AatII Toyobo Co. hereinafter referred to as AatII at 370C for one hour, the product was extracted with phenol and subjected to ethanol precipitation to recover 2 in Fig. 16.The precipitate was treated in 50 pl of a reaction mixture 67 mM Tris HCl, 6.7 mM MgC12, 6.7 mM ethylenediaminetetraacetic acid hereinafter abbreviated EDTA , 16.6 mM ammonium sulfate hereinafter abbreviated NH4 2SO4 , 10 mM mercaptoethanol, 0.66 mM dATP, 0.66 mM TTP, 0.66 mM dCTP and 0.66 mM dGTPi with one unit of T4 DNA polymerase BRL Co. at 370C for 15 minutes, after which agarose electrophoresis was carried out, to recover the desired fragment Fig. 16 commat . b In 50 pl of a reaction mixture 500 mM TrisHC1 pH 7.4 , 10 mM MgCl2, 50 mM NaCl , 5 pg of pACYC177 Journal of Bacteriolqgy, 134, No. 3, 1141 1156 1978 Fig. 16 was treated with 4 units of a restriction endonuclease HaeII Takara Co. hereinafter referred to as HaeII at 370C for 60 minutes. The product was extracted with phenol and precipitated with ethanol to recover 0 in Fig. 16. The precipitate was treated with one unit of T4 DNA polymerase BRL Co. similarly as described above. By agarose gel electrophoresis, a fragment Fig. 16 was recovered. The two DNA fragments obtained in the above operations a and b Fig. 16 0 and were allowed to react in 30 p1 of a reaction mixture 20 mM Tris HCl pH 7.5 , 10 mM MgC12, 10 mM DTT, 0.5 mM ATP with 30 units of T4 DNA ligase Takara at 140C for 16 hours. After completion of the reaction, E. coliK12C600 FERM BP No. 115 was transformed with the reaction product to obtain a transformed strain containing the desired plasmid pTA529 . 2 Preparation of pTA2539 a After 5 pg of the pTA1529Eco Fig. 17 Q prepared according to the method described in the specification of Japanese Patent Application No.269998 1984 filed on December 21, 1984, was treated in 50 pl of a reaction mixture 10 mM Tris HCl pH 8 , 10 mMMaCl2, 100 mM.NaC1 with 4 units of a restriction endonuclease HpaI Takara hereinafter abbreviatedHpaI , the product was extracted with phenol and subjected to ethanol precipitation.The resultant precipitate was treated in 50 ul of a reaction mixture 1100 mM Tris HCl pH 7.5 , 50 mM NaCl, 10 mn1 MgC12 with 1 pl of 4 units of EcoRI Takara hereinafter referred to as EcoRI at 370C for one hour, which treatment was followed by acrylamide gel electrophoresis, to obtain the desired fragment Fig. 17 . b After 5 pg of the pBR bla prepared by us Fig. 17 0 was treated similarly in a reaction mixture 10 mM Tris HC1 pH 8 , 10 mM MgC12, 150 mMNaCl with 1 pl of 4 units of EcoRI and 1 pl of 4 units of a restriction endonuclease SalI Takara Co. hereinafter referred to as SalI at 370C for one hour, agarose electrophoresis was performed to obtain the desired fragment Fig. 17 Q4 . c After 5 pg of the pTA2529 as prepared above Fig. 17 was treated in 50 pl of a reaction mixture 100 mM Tris HCl pH 7.5 , 50 mM NaCl, 10 mM MgC12 with 1 pl of 4 units of EcoRI at 370C for one hour, the product was extracted with phenol and subjected to ethanol precipitation. The resultant precipitate Fig. 17 was treated similarly as described above with 1 ul of one unit of T4 DNA polymerase BRL Co. .Further, after treatment in 50 pl of a reaction mixture 110 mM Tris HCl pH 8 , 150 mM NaCl, 10 mM MgCl2 with 1 l of 4 units of SalI at 370C for one hour, the desired fragment Fig. 17 ommat was recovered by agarose gel electrophoresis. The three DNA fragments obtained in the above operations a to c were allowed to react in 30 pl of a reaction mixture 20 mM Tris HCl pH 7.5 , 10 mM MgCl2, 10 mM DTT, 0.5 mM ATP with 300 units of T4 DNA ligase Takara Co. at 140C for 16 hours. After completion of the reaction, E. coli YK537 deposited as E.coli K12YK537 FERM P No.7941 1 was transformed with the reaction product to obtain the desired transformed strain. From the strain obtained here, a plasmid was prepared see Japanese Patent Application No.140748 1983 , and the base sequences of the linking portions of the respective fragments were confirmed by theMaxam Gilbert method Proc. Natl. Acad. Sci. USA, 74, 560 1977 . Production of protein Human urogastrone hUG EGF was incorporated iji the above plasmid of the present invention, and its production and recovery were performed following the genetic engineering technique. The steps are as shown in Figs. 18 and 19.Synthesis of hUG structural gene Synthesis of the hUG structural gene was carried out similarly as in the foregoing Examples.Preparation of plasmid containing hUG gene 1 Preparation of transformed strain E. coli K12YK537 pTA2522 a Hydrolysis of 5 pg of pTA2529 Fig. 18 was carried out in 50 l of a buffer 10 mM Tris HCl pH 7.5 , 10 mM MgCl2, 50 mM NaCi with 4 units of a restriction endonuclease Hind III Takara hereinafter referred to as Hind III at 370C for one hour. Then, the precipitate obtained by ethanol precipitation was treated in 30 ul of a reaction mixture 67 mM Tris HCl pH 8.8 , 16.6 mM NS4 2SO4, 6.7 mM MgCl2, 10 mM 2mercaptoethanol, 6.7 mM EDTA, each 0.66 mM of dATP, dCTP, dGTP, TTP with one unit of T4 DNA polymerase supra at 370C for 15 minutes.Then, after ethanol precipitation, the resultant precipitate was treated in 50 U1 of a reaction mixture 100 mM Tris HCl pH 7.5 , 50 mM NaCl, 10 mM MgCl2 with 1 ul of 4 units of EcoRI at 37 C for one hour, after which polyacrylamide gel electrophoresis was carried out, to recover the desiredDNA fragment of 250 bp Fig. 18 b Treatment of 5 pg of the pTA2529 Fig. 18 was carried out in 50 ul of a reaction mixture 100 mMTris HCl, 50 mM Nail, 10 mM MgC12 with 1 p1 of 4 units of EcoRI at 370C for one hour.Then, after ethanol precipitation, the resultant precipitate was hydrolyzed in 50 ul of a reaction mixture 6 mM Tris HC1 pH 8 , 6 mM MgCl2, 150 mM NaC1 with 4 units of a restriction endonuclease SalI Takara hereinafter referred to asSalI at 370C for one hour. After completion of the reaction, a DNA fragment of 4,300 bp Fig. 18 was obtained by agarose gel electrophoresis. After 5 pg of the plasmid pBR322 hUG the pBR322 Fig. 18 , deposited as E. coli K12C600 pBR322 FERM BP No, 235 , digested with EcoRI and Sail, having the fragment of the hUG structural gene as synthesized above digested with EcoRI and SalI incorporated therein was hydrolyzed in 50 ul of a reaction mixture 100 mMTris HCl pH 7.5 , 50 mM NaCl, 50 mM MgC12 with 4 units of EcoRI Takara at 370C for one hour, the T4 DNA polymerase reaction was carried out similarly as described above, which procedure was followed further by the SalI treatment, and thereafter a DNA fragment of 160 bp was obtained by agarose gel electrophoresis Fig. 18 The three DNA fragments as prepared above Fig. 18 , and were allowed to react in 30 l of a reaction mixture 120 mM Tris HCl pH 7.5 , 10 mMMgCl2, 10 mM DTT, 0.5 mM ATP with the use of 300 units of T4 DNA ligase Takara at 140C for 16 hours.After completion of the reaction, E. coli YK537 deposited as E. coli K12YK537 FERM P No. 7941 1 was transformed with the reaction product toobtain a transformed strain E. coli K12YK5373 containing the desired plasmid hereinafter referred to as pTA2522 Fig. 18 2 Preparation of transformed strain E. coli S12YK537 pTA2532 3 a Treatment of 5 pg of the pTA2539 Fig. 19 was carried out in 50 p1 of a reaction mixture 6 mMTris HCl pH 8 , 20 mM NaCl, 6 mM MgCl2 with 4 units of restriction endonuclease NaeI N.E.B. Co. hereinafter referred to as NaeI at 370C for one hour. Then, the product was extracted with phenol and recovered by ethanol precipitation.This was treated in 50 ul of a reaction mixture 10 mM Tris HCl pH 7.5 , 10 mM MgC12, 50 mM NaC13 with 4 units of Hind III at 370C for one hour.After completion of the reaction, by the agarose gel electrophoresis, a DNA fragment of 800 bp was obtained Fig. 19 0 b Treatment of 5 pg of the pTA2539 Fig. lR 0 was carried out in 50 p1 of a reaction mixture 10 mMTris HCl pH 7.5 , 10 mM MgCl2, 50 mM NaCl with 1 p1 of 4 units of Hind III at 370C for one hour, which treatment was followed by ethanol precipitation.The resultant precipitate was treated similarly as above with 1 pl of 4 units of a restriction endonuclease SalI Takara at 370C for one hour, and then a DNA fragment of 3,800 bp was obtained by agarose gel electrophoresis Fig. 19 c After 5 pg of the pBR322 hUG Fig. 19 Q4 the pBR322 digested with EcoRI and SalI, having the fragment of the hUG structural gene as synthesized above digested with EcoRI and SalI incorporated therein was hydrolyzed in 50 pl of a reaction mixture 100 mMTris HCl pH 7.5 , 50 mM NaCl, 50 mM MgC121 with 4 units of a restriction endonuclease EcoRI Takara at 370C for one hour, the T4 DNA polymerase reaction was carried out similarly as described above, which reaction was followed further by the SalI treatment, and thereafter a DNA fragment of 160 bp Fig. 19 commat was obtained by agarose gel electrophoresis. The three DNA fragments prepared in the above operations a to c Fig. 19 , and were allowed to react in 30 ul of a reaction mixture 20 mMTris HCl pH 7.5 , 10 mM DTT, 0.5 mM ATP with the use of 300 units of T4 DNA ligase Takara Co. at 140C for 16 hours. After completion of the reaction, the YK537 was transformed similarly as described above to obtain a transformed strain fE. coli K12YK537 pTA2532 containing the desired plasmid hereinafter written as pTA2532 Fig. 19 . Similarly as in the above Examples, from the strain obtained here, a plasmid was prepared see Japanese Patent Application No.140748 1983 and the base sequences of the linking portions of the respective DNA fragments were confirmed according to the Maxam Gilbert method supra . Production, recovery and quantitative determination of hUG were conducted according to the same methods as in the foregoing Examples. For the purpose of comparison of the excreted amounts between the above transformed strain E. coliK12YK537 pTA2522 and E. coli K12YK537 pTA2532 , the above experiment was performed with YK537 transformed similarly by use of pTA1522 and pTA1532 in which Kmr of the plasmid was changed to Ampr coding for excretable proteins. The amounts of hUG after 2 hours cultivation were as shown in Table 4. The units g L indicate the amount of hUG excreted ug in one liter of the culture broth. Table 4EMI46.1 tb Transformed SEP strain SEP Amount SEP secreted SEP g L SEP tb coli SEP K12C SEP 537 SEP pTA2522 SEP 1511 tb SEP pTA2532 SEP 1614 tb SEP H SEP pTA1522 SEP 1200 tb SEP n SEP pTA1532 SEP 1106 tb From the above results, the transformed strains obtained by using the plasmid in which no gene coding for the excretable proteins derived from the plasmid exists E. coli K12C537 pTA2522 and E. coli K12C537 pTA 2532 1 were found to be increased in amounts of hUG excreted by 350 to 500 ug per one liter of the culture broth as compared with the transformed strains in which the gene coding for the excretable proteins derived from the plasmid exists E. coli K12YK537 pTA 2522 and E. coli K12C537 pTA2532 . Purification and confirmation of hUG were also practiced similarly as in the foregoing Examples by conducting amino acid analysis, N and C end analyses, whereby the hUG excreted was confirmed to be hUG.Bacteriological properties of the related microorganisms and deposition numbers The bacteriological properties and the deposition numbers of the microorganisms disclosed in the present invention are as follows.Representation of microorganism Deposition number 1 E. coli K12C600 FERM BP No. 115 2 E. coli X12C600 pYK283 FERM BP No. 556 3 . coli K12C600 pBR322 FERM BP No. 235 4 E. coli K12YK537 FERM P No. 7941, which is now FERM BP No. 822 5 E. coli XA35 pLE6527 FERM BP No. 514 6 E. coli K12C600 pIF202 FERM P No. 7188Date of deposition 1 June 9, 1981 2 April 30, 1983 3 June 9, 1981 4 November 14, 1984 5 July 2, 1983 6 August 2, 12 Deposition number at Fermentation Research Insti tute, Agency of Industrial Science and Technology, Ministry of International Trade and Industry, the Japanese Government, 1 3, Higashi l Chome, Yatabe Machi, Tsukuba Gun, Ibaraki Ken, 305, Japan. Strains 1 5 are on deposit under the BUDAPESTTREATY ON THE INTERNATIONAL RECOGNITION OF THE DEPOSITOF MICROORGANISMS FOR THE PURPOSES OF PATENT PROCEDURE, and will be maintained and released under the TREATY in compliance with Rule 11.3 and Rule 9 of the TREATY.Bacteriological properties 1 E. coli K12C600 This microorganism is a Gram negative rod bacterium, having the general properties of the genus Escherichia such as formation of no spores and facultative anaerobic characteristic, otherwise containing no F factor, and being deficient in the function of the suppressor geneE and defective in the recBC gene coding for the nuclease participating in recombination of the gene. As the nutrient demand, threonine and leucine are required for proliferation on its minimum medium. Also, in taxonomy, it belongs to the family of enteric bacteria and the genus Escherichia. The literatures concerning this microorganism are shown below. a Genetics, 39, 440 1954 b Nature, 217, 1110 1968 2 E. coli K12C600 pYK283 This microorganism is a Gram negative rod bacterium, having the general properties of the genus Escherichia such as formation of no spores and facultative anaerobic characteristic, otherwise containing no F factor, and being deficient in the function of the suppressor geneE and defective in the recBC gene coding for the nuclease participating in recombination of the gene. As the nutrient demand, threonine and leucine are required for proliferation on its minimum medium. It contains the plasmid pYK 283 constituted of the replication initiation region derived from the promoter operator region pACYC177 of the pho A gene and the bla gene of pBR322, exhibiting resistance to ampicillin. Also, in taxonomy, it belongs to the family of enteric bacteria and the genus Escherichia. Except for the properties derived from the plasmid pYK 283, the bacteriological properties of this microorganism are the same as those of the parent strain E.coli K12C600. 3 E. coli K12C600 pBR322 This microorganism is a Gram negative rod bacterium, having the general properties of the genus Escherichia such as formation of no spores and facultative anaerobic characteristic, otherwise containing no F factor, and being deficient in the function of the suppressor geneE and defective in the recBC gene coding for the nuclease participating in recombination of the gene.As the nutrient demand, threonine and leucine are required for proliferation on its minimum medium. It also contains a drug resistant pBR 322. As to the plasmid pBR322, reference may be made to Gene, 2, 95 1977 , and as to E. coli, to the above Nature. Except for the properties derived from the plasmid pBR322, the bacteria ological properties of E. coli K12C600 pBR322 are the same as those of the parent strain. 4 E. coli K12YK537 The E. coli K12YK537 was obtained by further modifying the E. coli R12RR1 Gene, 2, 95 1977 , Biochem,Biophs. Acta., 655, 243 1981 derived from the E. coliK12 strain which was a known strain MicrobiologicalReviews, 44, 1 56 1980 , exhibiting the properties as shown below, other properties being the same as those of K12RR1 rec Al, Pho A8, pro . That is, the YX537 is a Gram negative rod bacterium, having the general properties of the genus Escherichia such as formation of no spores and facultative anaerobi characteristic, and otherwise containing no F factor, and being deficient in the function of the suppressor gene E and defective in the recBC gene coding for the nuclease participating in recombination of the gene.As the nutrient demand, threonine and leucine are required for proliferation on its minimum medium. This strain is deficient in expression of alkali phosphatase from chromosome possessed by ordinary E. coli strains deposited as E. coli K12YK537 FERM P No. 7941 1. 5 E. coli XA35 pLE6527 E. coli XA35 pLE6527 is a transformant ofE. coli XA35 by means of a plasmid pLE6527. The host strain E. coli XA35 is a derivative of known E. coliK12 tSee Microbiological Reviews, 44, 1 56 1980 having the same characteristics as the E. coli K12 has except for the following Sm, Lac i3 , z The transformant E. coli XA35 pLE6527 has the same characteristics as the host E. coli XA35 has except for the following Ampr Lac The plasmid pLE6527 is such a hybrid that a DNA fragment comprising a part of lactose operon derived from X plac 5DNA and a gene of artificial human urogast rone 21Met  21Leu have been inserted into pBR322 plasmid. 6 E. coli K12C600 pIF202 E. coli K12C600 pIF202 is a transformant ofE. coli K12C600, which is referred to above, by means of a plasmid pIF202. The transformant E. coli K12C600 pIF202 has the same characteristics as the hostE. coli K12C has except for those due to the plasmid pIF202. The transformant E. coli K12C600 pIF202 r s r has a genotype of Ampr and of Tc at 30 C but Tc at 420C. The plasmid pIF202 is such a hybrid that a CI PR PL region derived from ACI85757DNA and an ainterferon gene derived from human lenkocyte have been inserted into pBR322 plasmid. The plasmid pIF202 can be constructed from E.coli K12C600 pIF101 , which is on deposit at the above captioned Japanese depository under the abovecaptioned BUDAPEST TREATY with an accession numberFERM BP No. 595 dated August 2, 1983, according to a procedure as set forth in Japanese Patent Laid OpenPublication No. 66992 1985.