# Recombinant saccharomyces.

## Abstract
A method for cloning and expressing DNA sequences derived from Candida in Saccharomyces, a method for characterizing the function of the DNA sequences, a Sacch aromyces host microorganism transformed by the DNA sequence, and a recombinant plasmid useful for performing such transformation.

## Claims
CLAIMS for the Contracting States BE CH DE FR GB IT LI LU NL SA 1. A process for preparing a transformed host microorganism of the genus Saccharomyces which comprises transforming said host microorganism with a functional DNA sequence derived from Candida. 2. The process of Claim 1 wherein the functionalDNA sequence is derived from Candida albicans, Candida tropicalis, Candida stellatoidea, Candida pseudotropicalis, Candida parapsilosis, or Candida guilliermondi, Candida utilis or Candida kruisii. 3. The process of Claim 2 wherein the functionalDNA sequence is derived from Candida albicans. 4. The process of Claim 1 wherein the functionalDNA sequence comprises a complete gene expression unit, a structural gene, a promoter, or a regulatory region. 5. The process of Claim 4 wherein the functionalDNA sequence is derived from a trpl gene, a his3 gene, a gene conferring resistance to the anti fungal agent benomyl, a functional DNA sequence allowing for growth on the carbon source sorbitol, a functional DNA sequence allowing for growth on soluble starch, a functional DNA sequence allowing for growth on isomaltose, or a galactokinase gene. 6. The process of Claim 1 which belongs to the species Saccharomyces cerevisiae. 7. The process of Claim 1 which comprises a recombinant plasmid, wherein said plasmid comprises a replicator region and the functional DNA sequence derived from Candida. 8. The process of Claim 7 wherein the replicator region is derived from Saccharomyces or Candida. 9. The process of Claim 7 wherein the recombinant plasmid is a Saccharomyces Escherichia coli shuttle vector. 10. The process of Claim 9 wherein the shuttle vector is a YEpl3 derivative. 11. A process for preparing recombinant plasmid comprising a replicator region, and a functional DNA sequence derived from Candida, which comprises ligating said replicator region directly or indirectly with the functional DNA sequence. 12. The process of Claim 11 wherein the replicator region is derived from Saccharomyces or Candida. 13. The process of Claim 11 wherein the functional DNA sequence is derived from Candida albicans,Candida tropicalis, Candida stellatoidea, Candida pseudotropicalis, Candida parapsilosis, Candida guilliermondi, Candida utilis or Candida kruisii. 14. The process of Claim 13 wherein the functional DNA sequence is derived from Candida albicans. 15. The process of Claim 11 wherein the functional DNA sequence comprises a complete gene expression unit, a structural gene, a promoter, or a regulatory region. 16. The process of Claim 15 wherein the functional DNA sequence is derived from a trpl gene, a his3 gene, a gene conferring resistance to the anti fungal agent benomyl, a gene allowing for growth on the carbon source sorbitol, a glucoamylase gene, an isomaltase gene, or a galactokinase gene. 17. The process of Claim 11 which is aSaccharomyces Escherichia coli shuttle vector. 18. The process of Claim 17 wherein the shuttle vector is a YEpl3 derivative. 19. A method of expressing a functional DNA sequence derived from Candida which comprises transforming a host microorganism of the genus Saccharomyces with the functional DNA sequence, and culturing the transformed host under suitable conditions such that the functionalDNA sequence is expressed. 20. The method according to Claim 19 wherein the functional DNA sequence is derived from Candida albicans,Candida tropicalis, Candida stellatoidea, Candida pseudotropicalis, Candida parapsilosis, Candida guilliermondi, Candida utilis or Candida kruisii. 21. The method according to Claim 20 wherein the functional DNA sequence is derived from Candida albicans. 22. The method according to Claim 19 wherein the functional DNA sequence comprises a complete gene expression unit, a structural gene, a promoter, or a regulatory region. 23. The method according to Claim 22 wherein the functional DNA sequence is derived from a trpl gene, a his3 gene, a gene conferring resistance to the anti fungal agent benomyl, a gene allowing for growth on the carbon source sorbitol, a glucoamylase gene, an isomaltase gene, or a galactokinase gene. 24. The method according to Claim 19 wherein the host microorganism belongs to the species Saccharomyces cerevisiae. 25. The method according to Claim 19 which further comprises effecting the transformation with a recombinant plasmid, wherein said plasmid comprises a replicator region, and the functional DNA sequence derived from Candida. 26. The method according to Claim 25 wherein the replicator region is derived from Saccharomyces or Candida. 27. The method according to Claim 25 wherein the recombinant plasmid is a Saccharomyces Escherichia coli shuttle vector. 28. The method according to Claim 27 wherein the shuttle vector is a YEpl3 derivative. 29. A method of characterizing a DNA sequence derived from Candida by its function which comprises a transforming a host microorganism of the genusSaccharomyces witch the DNA sequence, wherein said host microorganism lacks a known function, so that the CandidaDNA can complement the host microorganism s DNA and b assaying for the presence of the function in the transformed host microorganism. 30. A method of producing a polypeptide inSaccharomyces which comprises culturing a transformed host microorganism of the genus Saccharomyces, wherein said host microorganism comprises a promoter derived fromCandida fused, directly or indirectly, in phase and in proper orientation to a structural gene which codes for the polypeptide. Claims for the Contracting State AT 1. A method of expressing a functional DNA sequence derived from Candida which comprises transforming a host microorganism of the genus Saccharomyces with the functional DNA sequence, and culturing the transformed host under suitable conditions such that the functional DNA sequence is expressed. 2. The method according to Claim 1 wherein the functional DNA sequence is derived from Candida albicans. Candida troPicalis, Candida stellatoidea. Candida Pseudotrouicalis, Candida ParaDsilosis, Candida puil iiermondii, Candida utilis of Candida kruisii. 3. The method according to Claim 2 wherein the functional DNA sequence is derived from Candida albicans. 4. The method according to Claim 1 wherein the functional DNA sequence comprises a complete gene expression unit. a structural gene, a promoter, or a regulatory region. 5. The method according to Claim 4 wherein the functional DNA sequence is derived from a trpl gene, a his3 gene, a gene conferring resistance to the antifungal agent benomyl, a gene allowing for growth on the carbon source sorbitol, a glucoamylase gene, an isomaltase gene, or a galactokinase gene. 6. The method according to Claim 1 wherein the host microorganism belongs to the species Saccharomvces cerevisiae. 7. The method according to Claim 1 which further comprises effecting the transformation with a recombinant plasmid, wherein said plasmid comprises a replicator region, and the functional DNA sequence derived from Candida. 8. The method according to Claim 7 wherein the replicator region is derived from SaccharomYces or Candida. 9. The method according to Claim 7 wherein the recombinant plasmid is a Saccharomyces Escherichia coli shuttle vector. 10. The method according to Claim 9 wherein the shuttle vector is a YEpl3 derivative. 11. A method of characterizing a DNA sequence derived from Candida by its function which comprises a transforming a host microorganism of the genus Sac charomyces with the DNA sequence, wherein said host microorganism lacks a known function, so that the Candida DNA can complement the host microorganism s DNA and b assaying for the presence of the function in the transformed host microorganism. 12. A method of producing a polypeptide in Saccharomyces which comprises culturing a transformed host microorganism of the genus Saccharomyces, wherein said host microorganism comprises a promoter derived fromCandida fused, directly or indirectly, in phase and in proper orientation to a structural gene which codes for the polypeptide.

## Description
TITLE RECOMBINANT SACCHAROMYCES BACKGROUND OF THE INVENTION This invention relates to cloning and expression of DNA sequences from Candida in Saccharomyces. Candidiasis is an infection caused by any of several species of the yeast genus Candida. These organisms are members of the normal flora of the skin, mucous membranes, and gastrointestinal tract. The risk of endogenous infection is, thus, clearly ever present. MostCandida infections are caused by Candida albicans which is a common human fungal pathogen that causes both superficial and systemic mycosis. Candida albicans infections are most often associated with immune compromised patients. In such patients, the infection can be life threatening. Therefore, there is a real need for anti candidial agents. The isolation and expression ofCandida genes may enable the discovery of effective anti candidial agents. The understanding of the biochemistry and genetics of Candida has been hampered by the lack of an amenable genetic system. It is difficult to prepare and isolate mutants of Candida, in which no sexual stage has been described. In the absence of suitable mutants, it is difficult to analyze the biochemical and developmental pathways of Candida. Thus, the development of effective anti candidial agents has been slow. Saccharomyces cerevisiae is considered largely non pathogenic and is a favorite organism for cloning and expression of recombinant DNA. Struhl, Nature, 305 391 397 1983 , reviews cloning in Saccharomyces.Henikoff et al., Nature, 289 33 37 1981 , report the isolation of an adenine 8 gene from Drosophila by complementation in yeast. Dickson, Gene, 10 347 356 1980 , reports the expression of a ss galactosidase gene derived from Kluyveromyces lactis in Saccharomyces cerevisiae. Russell and Hall, J. Biol. Chem., 258 143 149 1983 , report the cloning of an alcohol dehydrogenase gene from Schizosaccharomyces pombe in Saccharomyces cerevisiae. Jellinghaus et al., J. Mol. Biol., 159 623 636 1982 , report the transcription in yeast of aDictyostelium gene cloned in an Escherichia coli yeast vector. SUMMARY OF THE INVENTION One aspect of this invention is a transformed host microorganism of the genus Saccharomyces which comprises a functional DNA sequence derived from Candida. Another aspect of this invention is a recombinant plasmid comprising a replicator region, and a functionalDNA sequence derived from Candida. This invention also relates to a method of expressing a functional DNA sequence derived from Candida which comprises transforming a host microorganism of the genus Saccharomyces with the functional DNA sequence, and culturing the transformed host under suitable conditions such that the functional DNA sequence is expressed. Yet another aspect of this invention is a method of characterizing a DNA sequence derived from Candida by its function which comprises a transforming a host microorganism of the genus Saccharomyces with the DNA sequence, wherein said host microorganism lacks a known function, so that the Candida DNA can complement the host microorganism and b assaying for presence of the function in the transformed host microorganism. This invention also relates to a method of producing a polypeptide in Saccharomyces which comprises culturing a transformed host microorganism of the genusSaccharomyces in suitable culturing media, wherein said host microorganism comprises a promoter derived fromCandida fused, directly or indirectly, in phase and in proper orientation to a structural gene which codes for the polypeptide. DETAILED DESCRIPTION OF THE INVENTION The transformed host microorganism of the subject invention is any member of the genus Saccharomyces into which a functional DNA sequence derived from Candida has been introduced. Saccharomyces cerevisiae is preferred because it is the most well studied member of the genus and has been the species most frequently employed for recombinant DNA purposes. The recombinant plasmid of the subject invention comprises a functional DNA sequence derived from any species of the genus Candida. DNA derived from Candida albicans is most preferred because it was the first DNA utilized in the subject invention since it is derived from the genus member most likely to cause candidiasis. DNA derived from any other Candida species, such as Candida tropicalis, Candida stellatoidea, Candida pseudotropicalis, Candida parapsilosis, Candida guilliermondi, Candida utilis and Candida kruisii, can also be utilized. By the term functional DNA sequence is meant any discrete region of DNA derived directly or indirectly from Candida which functions in Candida as a complete gene expression unit, a structural gene, a promoter, or a regulatory region. By complete gene expression unit is meant a structural gene and the promoter and regulatory regions required for its transcription and translation.By promoter1 is meant any region upstream of a structural gene which permits binding of RNA polymerase and transcription to occur. By regulatory region is meant any region which regulates transcription of the gene. By structural gene is meant a coding sequence for a polypeptide which serves to be the template for the synthesis of mRNA. The recombinant plasmid of the subject invention also comprises a replicator region, preferably aSaccharomyces or Candida replicator region. By replicator region is meant any DNA fragment, preferably derived directly or indirectly from yeast, such asSaccharomyces or Candida, which can be used to maintain, extrachromosomally, a functional DNA sequence derived fromCandida in the host Saccharomyces microorganism.Particularly preferred plasmids are those comprising aSaccharomyces Escherichia coli shuttle vector, such as the one described by Broach et al., Gene, 8 121 133 1979 .Illustrative of vectors which can be used in this invention by cloning therein a functional DNA sequence derived from Candida are the following plasmids which are publicly available, e.g., from the American Type Culture Collection, Rockville, Maryland, U.S.A. ATC 9 accession number in parentheses pBTI l 37087 , pBTI 7 37088 , pBTI 9 37089 , pBTI 10 37090 , pLC544 37013 , pRB3 37059 , pRB4 37058 , pRB5 37051 , pRB8 37109 , pRB9 37057 , pRB10 37056 , pRB18 37092 , pRB20 37054 , pRB21 37055 , pRB22 37053 , pRB12 37061 , pRB14 37063 , pRB15 37062 , pRB16 37060 , pRBl9 37064 , YEp2 37076 , YRpl7 37078 , YEpl3 37115 . The Candida DNA sequence can be prepared by known techniques such as the one described by Rigsby et al.,Molec. Cell. Biol., 2 853 862 1982 . The isolatedCandida DNA can then be further fragmented by treatment wit one or more restriction enzymes. A recombinant plasmid containing a functionalCandida DNA sequence and containing an Escherichia coli replicator region can be used to transform Escherichia coli to amplify the plasmid. Such transformation and amplification can be accomplished by known techniques.The recombinant plasmid is then retrieved from E. coli by known techniques. The retrieved recombinant plasmid can then be used to transform a host microorganism belonging to the genus Saccharomyces. Such transformation can be accomplished by the random insertion of recombinant plasmid DNA fragments into covalently closed circular yeast plasmids and a transforming spheroplasts therewith, such as by the method described by Hinnen et al., Proc. Nat l. Acad. Sci. USA, 75 1929 1933 1978 or b by using the lithium acetate or lithium chloride method of yeast transformation, such as described by Ito et al., J. Bacteriol., 153 163 1983 . DNA from Candida can also be transformed into a Saccharomyces microorganism by fusion of Candida and Saccharomyces protoplasts. The introduced Candida DNA can be integrated into the hostSaccharomyces microorganism s chromosomal DNA, such as by recombination, or can remain extrachromosomal. Thus, yet another exemplary method is transformation of a hostSaccharomyces organism directly with Candida DNA which, optionally, has flanking sequences which are homologous to a contiguous sequence in the host s chromosomal DNA thereby permitting recombination with the host s chromosomal DNA. Promoter regions derived from Candida by the process of the subject invention can be used to express coding sequences from Candida, Saccharomyces or other organisms, viruses or cells in the transformedSaccharomyces host. Such promoters can be fused to a structural gene, such as sequences coding for insulin, interferon, growth hormone, etc., directly by fusing all or a portion of the promoter s natural nucleotide to the coding sequence for the desired polypeptide, thereby producing a hybrid gene. When Saccharomyces host microorganisms, which have been transformed by a plasmid containing such hybrid gene, are cultured in suitable culturing media, they produce the polypeptide. In addition to those Candida promoters disclosed herein, additional promoters can be discovered by cloning random fragments of Candida DNA in Saccharomyces plasmids designed for promoter selection.Such plasmids can be designed by known techniques such as the one described byCasadaban et al., Methods in Enzymology, 100 293 308 1983 Structural genes derived from Candida can be expressed in Saccharomyces from their natural Candida promoters, from heterologous promoters derived fromCandida or Saccharomyces, or from other functional heterologous promoters which are derived from other organisms or produced from synthetic oligonucleotides and which function in Saccharomyces. Many known Saccharomyces mutants are defective in defined functions, such as a specific step in leucine or histidine biosynthesis. This function could be supplied if such mutants were transformed with DNA sequences derived from Candida which contain genes coding for such functions. Where the function to be characterized is not already defective in an available Saccharomyces strain, such mutations can be induced by known techniques, such as by ultraviolet radiation, chemical mutagenesis, or genetic manipulation. In some cases, the function displayed by the transformed host microorganism will be one which is naturally not present in Saccharomyces.For example, a host Saccharomyces cerevisiae microorganism can be transformed by a functional DNA sequence according to the method of this invention to display functions it does not naturally possess such as sorbitol utilization, maltose utilization by isomaltase catalyzed cleavage of CL 1 6 glucosidic bonds such as a me thy CL methyl D glucopyranoside hydrolysis , soluble starch utilization by glucoamylase catalyzed cleavage of CL 1 4 glucosidic bonds or resistance to an anti fungal agent. The development of a genomic library covering the entire genome of Candida enables the characterization of the functions of the DNA derived from Candida. This will permit the genetics of this asexual pathogen to be more closely analyzed than has been possible in the past and to facilitate the development of anti candidial agents. The preferred method of characterizing the functions of DNA derived from Candida is by expressing DNA derived fromCandida albicans in Saccharomyces cerevisiae so that theCandida genes can be studied in the background of a well characterized eukaryotic organism. Thus, the genetics ofCandida albicans can be studied by isolation of genes via complementation of functions displayed by transformed Saccharomyces cerevisiae.The e abundance of available Saccharomyces cerevisiae mutant strains, together with the possibility of selection of additional mutants, makes it possible to effect such complementation. Therefore, it is possible to select for Candida albicans genes associated with any one of a number of functions, such as those associated with defined biochemical reactions, or those involved in cell division or cell structure. The availability of the genomic library and a system for the identification of the selected genes offers a new approach to resolve some of the basic differences that make one organism economically useful and another organism a serious pathogen. Using the method of this invention, plasmids carrying one or more functional C. albicans genes which complement a Saccharomyces mutation or confer a new characteristic to the Saccharomyces strain transformed therewith can be isolated and characterized. Such functional C. albicans genes can include a gene complementing the S. cerevisiae trpl mutation, a gene complementing the S. cerevisiae his3 mutation, a gene conferring high level resistance to the anti fungal agent benomyl, a gene allowing for growth on sorbitol, a gene allowing for growth on isomaltose, a gene allowing for growth on soluble starch i.e., a glucoamylase gene , and a gene encoding the enzyme galactokinase which complements a S. cerevisiae gall mutation. Such genes are useful as markers for isolating Saccharomyces hosts successfully transformed with plasmids containing such genes.Each of the promoters of the C. albicans sorbitol, isomaltose and soluble starch utilization genes is also useful as a regulatable promoter for a Saccharomyces microorganism transformed with a plasmid containing such a promoter.Plasmids containing C. albicans functional DNA sequences are useful for production in Saccharomyces of gene products, such as pharmaceuticals, vaccinal antigens and hormones. In the following examples, specific embodimentsof the invention are more fully disclosed. These examples are intended to be illustrative of the subject invention and should not be construed as limiting its scope. EXAMPLE 1A. Isolation of DNA from C. albicans The DNA from C. albicans strain 792 1 courtesy of J. Kwon Chung, Natl. Inst. Health, U.S.A. was isolated by the procedure described by Rigsby et al. Molec. CellBiol., 2 853 862 1982 . The DNA purified by this procedure was on the average 100 kilobase pairs kbp in length.B. Construction of the genomic library of C. albicans A genomic library was constructed in theSaccharomyces Escherichia coli shuttle vector, YEpl3, described by Broach et al., Gene, 8 121 133 1979 .Calculation of the size of the genome of C. albicans basedon the data provided by Rigsby et al., cited above, suggests that the genome size of C. albicans is about 2 x 104 kbp. Therefore, some 8000 independent transformants carrying an insert of 10 kbp are required to provide a genomic library with a probability of 99 that each of the sequences of C. albicans will be represented at least once in this library. This was calculated by the method disclosed by Maniatis et al., Molecular Cloning ALaboratory Manual , Cold Spring Harbor Laboratory 1982 . The DNA was partially digested with the restriction endonuclease Sau3A and fractionated by centrifugation in a 5 20 sucrose gradient. The fractions containing DNA fragments in the range of 10 kbp were pooled and used for ligation. Plasmid YEpl3 DNA was cleaved at the unique BamHI site, and then treated with alkaline phosphatase to prevent self ligation. Equal concentrations 200 ng. in a 10 p L reaction mixture of C. albicans and YEpl3 DNA were ligated E. coli strains MC1061 Casadaban et al., J. Mol.Biol., 138 179 207 1980 and HB101 Bolivar et al.,Gene, 2 95 113 1979 were transformed with the ligated mixture according to Lederberg and Cohen, J. Bacteriol., 119 1072 1074 1974 , with the modification that a 45 min.expression time at 370C in LB Luria Broth was allowed.An estimate of the number of independent transformants was obtained by plating a small sample on LB plates containing 50 mcg ml of ampicillin. Simultaneously, the remaining sample was inoculated into LB broth containing 50 mcg ml of ampicillin and incubated overnight to use for DNA isolation. The plate results indicated that some 16,000 independent clones were obtained. To estimate the percentage of transformants which contained inserts, 100 transformants were tested for their resistance to ampicillin and to tetracycline. Since the BamHI site is in the tetracycline resistance gene tetR , transformants that contain plasmids with the inserted DNA should be sensitive to tetracycline and resistant to ampicillin, while transformants containing plasmids without inserts should be resistant to both antibiotics.The results obtained indicated that all the transformants were sensitive to tetracycline and resistant to ampicillin suggesting that at least 99 of the transformants were carrying plasmids containing inserted DNA of C. albicans. DNA was prepared from the overnight LB ampicillin culture of E. coli, either small scale or large scale by the method of Maniatis et al., cited above, using an aliquot of this original culture as inoculum.C. Expression of C. albicans genes in S. cerevisiae 1. General methodology To isolate cloned DNA sequences of C. albicans by expression in S. cerevisiae, genetically defined haploid strains of S. cerevisiae were employed as hosts for DNA mediated transformation. Transformation was performed a by the spheroplast procedure, as described by Sherman, cited above, or by the lithium chloride or lithium acetate method described by Ito et al., J. Bacteriol., 153 163 1983 . The yeast were plated on minimal medium Yeast nitrogen base Difco YNB containing all necessary supplements except one. Transformation of a defined S. cerevisiae mutant with a plasmid containing the wild type allele of the specific mutation allows the transformed host to grow under conditions restrictive to the non transformed mutant. In the examples following, the LEU2 gene was used as the first selectable marker. The transformed yeast were then subcultured onto additional types of media to select transformants that expressed other functions originally lacking in the S. cerevisiae host strain. The plasmids were recovered from S. cerevisiae by making a DNA preparation of the desired transformant, using the procedure of Sherman et al., cited above, and using this DNA to transform E. coli. Since the shuttle vector carried the ampicillin resistance gene, transformedE. coli were recovered on LB ampicillin plates. PlasmidDNA was then isolated from the E. coli by known techniques and used for further analysis. Such further study included a transformation of yeast to confirm the presence of a gene conferring the same phenotype as found in the original transformed yeast cell b hybridization analysis to confirm the presence of DNA of C. albicans in the hybrid plasmid and c further analysis of the , albicans DNA fragment both structurally and functionally. 2. Isolation of a Candida albicans gene that complements the trpl mutation of S. cerevisiae Spheroplasts of S. cerevisiae strain DBY746 a, hits3 leu2 3, leu2 112, ura3 52, trpl 289 were transformed with DNA isolated from the C. albicans library. S. cerevisiae strain DBY746 was obtained from Yeast Genetic Stock Center, Department of Biophysics andMedical Physics, University of California, Berkeley,California, U.S.A. The cells were plated on minimal medium Difco yeast nitrogen base containing all required supplements except leucine. The transformed yeast cells expressed the plasmid encoded LEU2 gene. The Leu cells were harvested from these plates and plated onto minimal medium lacking both leucine and tryptophane. Of the 500 transformants examined, two grew on these plates. To test whether the growth of these two transformants on medium lacking leucine and tryptophane was due to the presence of a C. albicans DNA insert, rather than due to the reversion of the trpl mutation, the transformants were grown in non selective yeast extract peptone dextrose YEPD medium for 10 generations, and then plated on minimal medium containing all supplements.The colonies were then replica plated onto minimal medium lacking only tryptophane or only leucine. In all cases, colonies arising from cells that had lost the plasmid, i.e., were Leu , also required tryptophane indicating co segregation of the LEU2 and TRP1 alleles. Plasmids were recovered from the two transformed isolates by transformation of E. coli HB101 with small yeast DNA preparations, prepared by the method of Sherman et al., Methods in Yeast Genetics Laboratory Manual ,Cold Spring Harbor Laboratory 1982 , selecting for ampicillin resistant transformants. The transformed E.coli cells were tetracycline sensitive, indicating that the plasmid contained an insert in the tetR gene. The two plasmids were designated pAR84 1 and pAR84 2. plasmid DNA of each was prepared by the method ofManiatis et al., cited above, and used to transform S.cerevisiae strain DBY746 available from Yeast GeneticStock Center . The selection of the transformed cells was performed on medium devoid of leucine but containing the supplements histidine, tryptophane and uracil. The Leu transformed cells were tested for prototrophy for tryptophane. Between 88 to 100 of the transformants expressed the unselected marker TRP confirming that both pAR84 1 and pAR84 2 contained a gene complementing the S. cerevisiae trpl mutation and indicating that a functional DNA sequence derived from Candida can be expressed in Saccharomyces.D. Further Characterization of the Plasmids To test for specific hybridization of the DNA insert in the recombinant plasmids to genomic DNA of C.albicans, DNA from non transformed and transformed S.cerevisiae strain DBY746 was isolated according to the procedure described by Sherman et al., cited above, for small scale preparations. The DNA of E. coli, with or without plasmid, was also prepared by the method ofManiatis et al., cited above. The DNA samples were digested with the restriction enzymes HindIII and BglII. Digested DNA of C. albicans strain 792 1, S. cerevisiae strain DBY746, plasmid YEpl3, and plasmid pAR84 2 was electrophoresed on agarose gels, and transferred to a nitrocellulose filter by the procedure of Southern, J. Mol. Biol., 98 503 517 1975 . The filter was then hybridized sequentially, first with 32P labelled YEpl3 DNA to test whether the vector used hybridized to the genomic DNA of C. albicans, and, second, with 32p labelled pAR84 2 DNA to determine whether the new plasmid hybridized to the genomic DNA of C. albicans. The nitrocellulose filters were prehybridized for 3 hrs. at 650C with a solution containing 6 x SSC and 3 xDenhardt 1 x Denhardt is 0.02 Ficol 400, 0.02 polyvinylpyrrolidone 360 and 0.02 bovine serum albumin according to the method of Maniatis et al. cited above. Hybridization solution was the same except the 32p labelled DNA probe prepared by nick translation according to the method of Maniatis et al., cited above was added.Hybridization was performed for 18 hours at 650C. After hybridization the filters were washed with 2 x SSC at 650C. The filters were then exposed to X ray film. YEp13 DNA hybridized only to the genomic DNA ofS. cerevisiae, whereas pAR84 2 DNA also hybridized to the genomic DNA of C. albicans. The hybridization of pAR84 2 to the S. cerevisiae genomic DNA is due to the S.cerevisiae LEU2 gene that is a part of the plasmid. The hybridization to C. albicans is due to homology of Candida insert with genomic sequences of Candida. The size of the inserts in each plasmid was estimated by electrophoresis of restriction digests on agarose gels. pAR84 1 contains a 15 kbp fragment of C.albicans DNA, while pAR84 2 contains a 10.4 kbp fragment.E. Other Specific Examples Using the same methodoldgy as that used for the isolation of a C. albicans DNA fragment that complements the S. cerevisiae trpl mutation, plasmids carrying other functional C. albicans genes have been isolated and characterized. These include 1 A gene complementing the his3 mutation. 2 A gene conferring high level resistance to the anti fungal agent benomyl methyl l butylcarbamoyl 2benzimidazole carbamate . Strains of C. albicans are more resistant to benomyl than S. cerevisiae. This gene thus confers a new characteristic to the transformed S.cerevisiae strain. 3 A gene allowing for growth on the carbon source sorbitol. Strains of C. albicans can grow on sorbitol butS. cerevisiae and other species of Saccharomyces cannot utilize this sugar. This gene thus confers a new characteristic to the transformed S. cerevisiae strain. 4 A gene allowing for growth on the carbon source cur methyl D glucopyranoside isomaltose . Strains of C.albicans can grow on this substrate but most strains of S.cerevisiae cannot naturally utilize this sugar. This enzyme has been found in other species of Saccharomyces.Thus, the C. albicans gene which codes for isomaltase which cleaves cr 1 6 glucosidic bonds confers a new characteristic to the transformed S. cerevisiae host. 5 A gene coding for an enzyme allowing for growth on soluble starch. Strains of C. albicans can utilize such soluble starch as a substrate but S. cerevisiae cannot naturally utilize this carbon source. This enzyme, glucoamylase, has been found in other species ofSaccharomyces. Thus, the C. albicans gene which codes for glucoamylase which cleaves a 1 4 glucosidic bonds confers a new characteristic to the transformed S.cerevisiae host. Furthermore, the C. albicans glucoamylase gene is of great potential utility, as the enzyme, when produced by C. albicans or by a S. cerevisiae host transformed therewith, is excreted into the medium.Thus, the glucoamylase gene may provide DNA sequences for a signal peptide which functions in protein secretion.The use of such a signal peptide for secretion of a cytoplasmic protein is well known, for example, seeSilhavy et al., U.S. Patent 4,335,336, issued June 22, 1982. This glucoamylase gene signal peptide can be used to develop a system for secretion of natural or heterologous proteins expressed by Saccharomyces or other yeast transformed therewith. 6 A gene encoding the enzyme galactokinase, which complements a S. cerevisiae gall mutation. EXAMPLE 2 COMPLEMENTATION OF THE S. CEREVISIAE his3 GENE WITH C. ALBICANS DNA a. Isolation of a C. albicans gene that complements the his3 mutation of S. cerevisiae The his3 locus of S. cerevisiae encodes imidazoleglycerol phosphate dehydrogenase. The isolation of a G albicans gene that complements the his3 mutation of S.cerevisiae was carried out according to the procedure ofExample 1. The Leu yeast transformants were plated on medium lacking leucine and histidine. Colonies which arose were selected.b. Recovery of a plasmid containing a C. albicans gene that complements the his3 mutation of S. cerevisiae Plasmids were recovered from the selected colonies, described above, by transformation of E. coli HB101 according to the method of Example 1. Plasmid DNA was isolated from the E. coli transformants according to the method of Example 1, and was designated pAR84 3.c. Transformation of S. cerevisiae strain DBY746 with a plasmid containing a C. albicans gene that complements the his3 mutation of S. cerevisiae Plasmid pAR84 3, prepared as described above, was transformed into S. cerevisiae strain DBY746 according to the procedure of Example 1.d. Selection of transformants and confirmation of the presence of C. albicans DNA in the transformant Host S. cerevisiae microorganisms, transformed with plasmid pAR84 3 according to the method described above, were selected according to the method described above, and tested for expression of the his3 gene. EXAMPLE 3 EXPRESSION OF THE C. ALBICANS BENOMYL RESISTANCE GENE IN S. CEREVISIAE The isolation of the C. albicans benomyl resitance gene was carried out according to the procedure of Example 1, except that S. cerevisiae strain JWG2 2C a, leu2 3, leu2 112 was employed as the host. S. cerevisiae strain JWG2 2C was derived from a cross of S. cerevisiae strain DC5 leu2 3, leu2 112, his3, ga12, canl ll , withS. cerevisiae strain YM146 at gall A152, trpl 289, ura352, rio . S. cerevisiae strain DCS was obtained fromJames R. Broach, S.U.N.Y., Stonybrook, New York, U.S.A. S.cerevisiae strain YM146 was obtained from Mark Johnson,Washington State University, Pullman, Washington, U.S.A.The Leu transformed cells were plated on miminal medium Difco yeast nitrogen base containing all required supplements except leucine, and also containing benomyl 50 vg ml . Several benomyl resistant colonies were obtained.b. Recovery of plasmids containing the C. albicans benomyl resistance gene Plasmids containing a C. albicans gene that codes for an enzyme conferring benomyl resistance were recovered according to the procedure of Example 1. Plasmid DNA was isolated from individual E. coli transformants according to the small scale method of Sherman et al., Methods inYeast Genetics Laboratory Manual , Cold Spring HarborLaboratory 1982 . The size of the C. albicans insert in the recovered plasmids was estimated by agarose gel electrophoresis. c. Transformation of S. cerevisiae strain JWG2 11D with a plasmid containing the C. albicans benomyl resistance gene The plasmid DNA was isolated from the E. coli transformants, and used to transform S. cerevisiae strainJWG2 11D a, his3, trpl 289, leu2 3, leu2 112, gall A152 1 according to the procedure of Example 1. S. cerevisiae strain JWG2 llD was derived by the same method as Sr cerevisiae strain JWG2 2C. d. Selection of transformants and confirmation of the presence of C. albicans DNA in the transformant Host S. cerevisiae microorganisms, transformed with a plasmid containing the C. albicans benomyl resitance gene according to the method described above, were selected as described above, and the presence of plasinid in the transformed hosts was confirmed according to the method of Example 1 by plating on medium containing benomyl. Three independent isolates, designated pCAbenl7, pCAben3 and pCAbenlS, were further analyzed by restriction mapping, and all were found to contain an identical 5.8 kilobase pair kbp DNA fragment from C. albicans. EXAMPLE 4 EXPRESSION OF THE C. ALBICANS SORBITOL UTILIZATION GENE IN S. CEREVISIAE a Isolation of the C. albicans sorbitol utilization gene The isolation of the C. albicans sorbitol utilization gene was carried out according procedure ofExample 1, except that S. cerevisiae strain JWG2 2C was employed as the host. Leu transformants were plated on minimal medium Difco yeast nitrogen base containing all required supplements except leucine and glucose, and also containing 2 sorbitol.b. Recovery of plasmids containing the C. albicans sorbitol utilization gene Plasmids containing a C. albicans gene that codes for an enzyme conferring sorbitol utilization ability were recovered according to the procedure of Example or PlasmidDNA was isolated from individual E. coli transformants according to the small scale method of Sherman et al., cited above, and the size of the C. albicans insert of the recovered plasmids was estimated by agarose gel electrophoresis. c. Transformation of S. cerevisiae strain JWG2 l1D with a plasmid containing the C. albicans sorbitol utilization gene Plasmid DNA was isolated from the E. coli transformants described above, and used to transform S.cerevisiae strain JWG2 llD according to the method ofExample 1.d. Selection of transformants and confirmation of the presence of C. albicans DNA in the transformant Host cells of S. cerevisiae, transformed with a plasmid containing the C. albicans sorbitol utilization gene according to the method described above, were selected as described above, and the presence of the plasmid in the transformed hosts was confirmed according to the method of Example 1. DNA of two independent isolates, designated pCAsor2 and pCAsor9, were further analyzed by restriction mapping. pCAsor2 was found to contain a 4.5 kbp DNA insert from C. albicans, while pCAsor9 was found to contain a 4.8 kbp DNA insert from C.albicans which showed little homology to the pCAsor2 C.albicans DNA insert. EXAMPLE 5 EXPRESSION OF THE C. ALBICAN GLUCOAMYLASE GENE IN S. CEREVISIAE a. Isolation of the C. albicans glucoamylase gene The isolation of the C. albicans glucoamylase gene was carried out according to the procedure of Example 1, except that S. cerevisiae strain JWG2 2C was employed as the host. Leu transformants were plated on minimal media Difco yeast nitrogen base containing all required supplements except leucine and glucose, and also containing 2 soluble starch. The E. coli transformants were probably secreting the glucoamylase since satellite colonies began to grow adjacent to the original large colonies. b. Recovery of a plasmid containing the . albicans glucoamylase gene Plasmids containing the C. albicans gene that codes for glucoamylase were recovered in E. coli according to the procedure of Example 1. Plasmid DNA was isolated from individual E. coli transformants according to the method of Example 1.c. Transformation of S. cerevisiae strain JWG2 llD with a plasmid containing the C. albicans glucoamylase gene Plasmid DNA, isolated from the E. coli transformants as described above, was used to transform S.cerevisiae strain JWG2 11D according to the method ofExample 1.d. Selection of transformants and confirmation of the presence of C. albicans DNA in the transformant Host S. cerevisiae microorganisms, transformed with a plasmid according to the method described above, were selected as described above, and the presence of the plasmid in the transformed host was confirmed according to the method of Example 1. Three independent isolates, designated as pSTA27 1, pSTAl 3 and pSTA15 1, were further analyzed by restriction mapping which revealed that pSTA27 1 and pSTAl 3 contain an identical Ç. albicans DNA insert while pSTA15 1 contains a possibly different C.albicans DNA insert. Plasmid pSTA27 1 was introduced by transformation Ito et al., J. Bacteriol., 153 163 1983 into three different S. cerevisiae strains, JCC4 11 a, leu2 hisl , JCC4 28 CL leu2, hisl and JCC4 29 , leu2, hisl . The same three strains were also transformed with the parent vector plasmid YEpl3 lacking any C. albicans sequences. The growth of all six transformed strains was tested on complete YEP medium containing 0.5 malto oligossacharides Mos Pfanstiel LaboratoriesInc. as the sole carbon source. This compound, Mos, consists of a mixture of glucose polymers linked by alpha 1,4 glucosidic bonds. It is devoid of glucose, maltose and maltotriose, and thus consists of higher order oligomers which can not be used as carbon sources bySaccharomyces cerevisiae strains.This is due to the fact that 1 these polysaccharides do not enter S. cerevisiae cells and 2 S. cerevisiae strains do not secrete an enzymatic activity amylase or glucoamylase capable of degrading Mos extracellularly. While all three strains transformed with plasmid YEpl3 failed to grow on Mos containing medium, the same three strains carrying plasmid pSTA27 1, as well as C. albicans strain 792 1 courtesy ofJ. Kwon Chung, Natl. Inset. Health, U.S.A. grow well on this medium. This result indicates that the C. albicansDNA fragment cloned in pSTA27 1 carries a gene encoding an enzyme capable of degrading Mos into a usable carbon source, that this gene is expressed in S. cerevisiae, and that its product of secreted into the culture medium. In order to confirm the expression and secretion of the C. albicans glucoamylase by a S. cerevisiae host transformed therewith, the culture media of all six transformed S.cerevisiae strains, as well as that of C. albicans strain 792 1, were tested for the presence of glucoamylase activity according to the method described by Searle et al., FEMS Microbiology Letters,, 111, 211 212 1981 . It was found that while the culture media of S. cerevisiae strains transformed with YEpl3 was devoid of any detectable glucoamylase activity, the culture media of C.albicans strain 792 1 and that of S. cerevisiae containing pSTA27 1 contained appreciable amounts of glucoamylase.In fact, it was found that S. cerevisiae strains carrying plasmid pSTA27 1 secrete 15 to 30 fold more glucoamylase activity into their culture medium than does C. albicans strain 792 1. The glucoamylase activity secreted by S. cerevisiae strains transformed with pSTA27 1 was further characterized and compared to that secreted by C. albicans strain 792 1 in the following manner a The pH optimum of the enzyme secreted by S. cerevisiae strains was determined to be between 4.5 and 6.0 and is identical to that of the enzyme secreted by C. albicans. b The temperature optimum of the S. cerevisiae secreted enzyme was found to be 60 0C and is identical to that of the C. albicans secreted enzyme. c Aliquots of the culture medium derived from S. cerevisiae strain JCC4 11, described above, transfomed with either YEpl3 or pSTA27 1 and of C. albicans strain 792 1 were analyzed by SDS, polyacrylamide gel electrophoresis. A diffuse protein band of a molecular weight higher than 94K d alton is present in JCC4 11 pSTA27 1 medium but absent from medium derived from JCC4 11 YEp13 . A similar, if not identical, protein band is detected, although in much lower amounts, in medium derived from C. albicans strain 792 1. Taken together, these results indicate the fragment of C. albicans DNA cloned in pSTA27 1 carries aC. albicans gene encoding a glucoamylase, that this gene is expressed in the yeast S. cerevisiae, and that the product of this gene is secreted into the culture medium by S. cerevisiae cells. EXAMPLE 6 EXPRESSION OF THE C. ALBICANS GALACTOKINASE GENE IN S. CEREVISIAE a. Isolation of the C. albicans galactokinase gene The isolation of the C. albicans galactokinase gene was carried out according to the procedure of Example 1, except that S. cerevisiae strain JWG2 llD was employed as the host. Leu transformants were plated on minimal media Difco yeast nitrogen base containing all required supplements except leucine and glucose, and also containing 2 galactose.b. Recovery of a plasmid containing the C. albicans galactok inase gene Plasmids containing the C. albicans gene that codes for galactokinase were recovered in E. coli according to the procedure of Example 1. Plasmid DNA was isolated from individual E. coli transformants according to the method of Example 1.c. Transformation of S. cerevisiae strain JWG2 11D with a plasmid containing the C. albicans galactokinase gene The plasmid DNA was isolated from the E. coli transformants as described above and used to transform S.cerevisiae strain JWG2 llD according to the method ofExample 1.d. Selection of transformants and confirmation of the presence of C. albicans DNA in the transformant Host S. cerevisie microorganisms, transformed with a plasmid containing the C. albicans galactokinase gene acccording the method described above, were selected as described above, and the presence of the plasmid in the transformed host was confirmed according to the method ofExample 1. Two independent isolates, designated as pCAgall l and pCAgall 4 were further analyzed by restriction mapping which revealed that the 7 kbp C.albicans DNA insert in pCAgall l is identical to the internal portion of the 13 kbp insert in pCAgall 4. The presence of a C. albicans DNA sequence encoding galactokinase in the S. cerevisiae transformants was further demonstrated by measuring the level of galactokinase in strain JWG2 llD with or without plasmid pCAgall 4. Enzyme activity, measured by the method ofButt et a1., Pt USA, 81, 3332 3336 1984 , was only detected in galactose induced cultures containing pCAgall 4. The promoter of the C. albicans galactokinase gene was demonstrated to be positively activated by the S.cerevisiae positive regulator, the GAL 4 gene product, as follows. Strain JWG2 llD containing pCAgall 4 was mated to strain JWG2 22B a trpl 289, leu2 3, leu2 112, gall 152, ura3 52 containing plasmid pCD21 4 a 2 vim Leu2 shuttle vector containing the S. cerevisiae GAL 4 gene .The resulting diploid was maintained on medium lacking both leucine and tryptophane. The induced level of galactokinase was significantly higher in the presence of the plasmid expressing the GAL 4 gene. This result indicates that the C. albicans sequence in pCAgall 4 contains a galactose regulated promoter which is responsive to S. cerevisiae regulatory proteins. EXAMPLE 7 EXPRESSION OF THE C. ALBICANS ISOMALTASE GENE IN S. CEREVISIAE a. Isolation of the C. albicans isomaltase gene Isomaltase cleaves ci 1 6 glucosidic bonds, and enables microorganisms expressing it to grow on the carbon source known as CL methyl D glucopyranoside. The isolation of the C. albicans isomaltase gene was carried out according to the procedure of Example 1.Leu transformants were plated on minimal media Difco yease nitrogen base containing all required supplements except leucine, and also containing 2 CL methyl D glucopyranoside. b. Recovery of a plasmid containing the C. albicans isomaltase gene Plasmids containing the C. albicans gene that codes for isomaltase were recovered according to the procedure of Example 1. Plasmid DNA was isolated from individual E. coli transformants according to the method of Example 1.c. Transformation of S. cerevisiae strain JWG2 llD with a plasmid containing the C. albicans isomaltase gene The plasmid DNA was isolated from the E. coli transformants described above and used to transform S.cerevisiae strain JWG2 llD according to the method ofExample 1.d. Selection of transformants and confirmation of the presence of C. albicans DNA in the transformant Host S. cerevisiae microorganisms, transformed with a plasmid containing the C. albicans isomaltase gene according to the method described above, were selected as described above, and the presence of the plasmid in the transformed host was confirmed according to the method ofExample 1. EXAMPLE 8 TRANSFORMATION OF S. CEREVISIAE WITH A RECOMBINANT PLASMID CONTAINING A C. ALBICANS REPLICATOR REGION This example reports the isolation of several C. albicans replicator regions that support autonomous replication of recombinant plasmids in heterologous hostS. cerevisiae microorganisms.A. Materials and Methods 1. Strains and Media Escherichia coli strain SF8 C600, leuB, thr, pro, B1, rk , mk , recBC , lopll, lig Cameron et al., PNAS, USA, 72, 3416 3420 1975 l, was used for bacterial transformation and propagation of plasmids.Bacteria were grown on LB rich media or M9 minimal media supplemented as required Miller, Experiments InMolecular Genetics , Cold Spring Harbor Laboratory, NewYork, 431 433 1972 1. Saccharomyces cerevisiae strain 483 a, leu2 3, leu2 112, his4 was used as host for yeast transformation. Yeast cultures were grown on YPG rich media 1 yeast extract, 1 peptone, 2 glucose or YNB minimal medium Difco yeast nitrogen base supplemented with histidine 2 p g ml and leucine 2 llg ml as required. Plasmid M851 Gorman et al., Mol. Gen. Genet., 183, 306 313 1981 1 was employed. C. albicans strain 792 1 was used as the source of genomic DNA. 2. DNA Preparations Covalently closed circular plasmid DNA was prepared from M9 glucose growri E. coli cultures amplified with chloramphenicol. DNA was extracted and purified by a cleared lysate technique essentially as described byClewell, J. Bacteriol., 110, 667 676 1972 . Crude E. coli plasmid preparations, used for yeast transformation and preliminary gel analysis, were made using the procedure of Birnbohm and Doly, Nucl. AcidsRes., 7, 1513 1523 1979 . Ten ml of bacterial culture 2x10 cells yielded 1 2 g g of crude plasmid DNA. For rapid screening of bacterial transformants for the presence of plasmid, single colony mini lysates, prepared by the same method, were used. For recovery of total DNA from S. cerevisiae, the small scale method of Sherman et al., cited above, was employed. C. albicans DNA was prepared by the method ofRigby et al., J. Mol. Biol., 113, 237 251 1977 . 3. Plasmid Construction Total Candida DNA was digested to completion with restriction endonuclease PstI. The DNA was mixed withPstI cut M851 plasmid DNA at concentrations estimated to give maximal recovery of plasmids containing Candida inserts by the method of Dugaiczyk et al., J. Mol. Biol., 96, 171 184 1975 , and ligated with T4 polynucleotide ligase. The ligated mixture was used to transform E. coli strain SF8 to leucine prototrophy. Leu transformants were then screened for ampicillin sensitivity. The Ampw colonies 5.8 percent of the Leu transformants , presumed to contain Candida inserts, were retained. The hybrid plasmids were designated by the prefix pCA followed by isolate number. 4. Gel Analysis and Hybridization Procedures All gel analysis and hybridization procedures employed were according to the method of Maniatis et al., cited above. 5. Transformation E. coli were transformed using the calcium heat shock method, as modified by Dagert and Ehrlich, Gene, 6, 23 28 1979 . The procedure used for yeast transformation was essentially as described by Beggs, Nature, 275, 104 109 1978 , using ss glucuronidase sulfatase Sigma for spheroplast formation, with the exception that a lower concentration of plasmid DNA 0.5 1.0 vg per 109 spheroplasts was used.B. Results 1 Plasmid Construction Candida DNA, digested to completion with restriction endonuclease PstI, was inserted by ligation into the unique PstI site of M851. Interruption of the ampr gene by an inserted fragment results in loss of drug resistance thus ampS Leu bacterial transformants were selected as harboring plasmids containing sequences from Candida designated pCA . 2. Transformation of Yeast Plasmids containing a yeast replicator sequence transform yeast at high frequency, reflecting autonomous replication of the plasmid in the eucaryotic cell. If the plasmid lacks a functional replicator, it also will transform, but at a low frequency. Presumably, this low frequency stable transformation reflects integration of the plasmid leu2 gene into chromosomal DNA by homologous recombination. If Candida chromosomal replicators are functional in yeast, hybrid plasmids containing them might be characterized by the ability to transform yeast at high frequency. Forty one 41 pCA plasmids were individually tested for their ability to transform leu2 S. cerevisiae to Leu . Five of the forty one pCA plasmids tested showed high frequency transformation. These results indicate that such plasmids can replicate autonomously inS. cerevisiae. 3. Detection of Plasmid DNA in S. cerevisiae If high frequency transformation indicates the presence of a Candida sequence that allows autonomous plasmid replication, it should be possible to recover covalently closed circular molecules from the unstableLeu yeast transformed with DNA from the pCA plasmids. Total yeast DNA was extracted from parental and transformed yeast strains, electrophoresed in a 1.0 agarose gel, and transferred to nitrocellulose paper. TheDNA was then hybridized with P32 labelled pBR322 probe.This probe has no homology with yeast chromosomal DNA.The DNA from transformed cultures was transferred to nitrocellulose filters and hybridized with P32 labelled pBR322. DNA sequences hybridizing to this probe could be detected in extracts of S. cerevisiae transformed with one of the five pCA plasmids giving high frequency transformation, as well as with control plasmid pYP42 Gorman et al., cited above . 4. Transformation of E. coli with Yeast Plasmid DNA The yeast plasmid preparations analyzed by hybridization techniques were also examined for ability to transform leuB E. coli to Leu . Bacterial colonies selected on leucine deficient medium were screened for the presence of plasmid by agarose gel electrophoresis of mini lysates. The five yeast extracts that were shown to contain plasmids by hybridization gave rise to Leu E. coli that contained plasmid DNA. Plasmid DNA of one of the five high frequency transformation pCA plasmids was compared with the plasmid isolated from E. coli following transfer through the S. cerevisiae host. The plasmids, containing a 4.0 kilobase pair kbp C. albicans insert, were found to be identical. Other replicator regions of Candida useful in the autonomous maintainence of plasmids used to transformSaccharomyces can be isolated according to the method ofKawamura et al., Gene, 24, 157 162 1983 , Hsu et al., J.Bacteriol., 154, 1033 1039 1983 , or Tikhomirova et al.,Mol. Gen. Genet., 189, 479 485 1983 . Plasmids containing such replicator regions can be prepared according to the method described in this Example. EXAMPLE 9 TRANSFORMATION OF S. CEREVISIAE WITH A PLASMID CONTAINING A C. ALBICANS REPLICATOR REGION AND A C. ALBICANS CODING SEQUENCE A plasmid containing a C. albicans replicator region and a C. albicans coding sequence can be prepared by combining the methods Nof Examples 1 and 8. Such a plasmid can be used to transform S. cerevisiae according to the method of Example 1, and expression of the C. albicans coding sequence by the transformed S. cerevisiae host can be detected according to the method of Example 1. While the above descriptions and Examples fully describe the invention and the preferred embodiments thereof, it is understood that the invention is not limited to the particular disclosed embodiments. Furthermore, this invention is not limited to the genes and promoters of Candida given in the examples above. Rather, this invention provides a general method for locating Candida gene expression units, promoters, regulatory regions and structural genes, as well as a general method for expressing such functional DNA sequences in Saccharomyces. Thus, the invention includes all embodiments coming within the scope of the following claims.