# Hybrid lymphoblastoid leukocyte human interferons.

## Abstract
DNA sequences encoding for novel human lymphoblastoid leukocyte hybrid interferons. Methods of making nucleotide sequences comprising ribosome binding sites, promoter regions, and segments of human lympho blastoid and leukocyte interferons are described. DNA se quences for hybrid interferons are cloned and expressed in plasmid vectors. Transformed microorganisms and novel lymphoblastoid leukocyte polypeptide interferons having biological or immunological activity are disclosed.

## Claims
Patent claims 1. An antiviral and antiproliferative composition of matter comprising a polypeptide of 166 amino acids and optionally having an additional methionine attached to the ordinary first amino acid at theN terminus, the amino acid sequence of said polypeptide compound having the formula H2N met cys1 X P Y B Z COOH wherein X consists essentially of amino acids 2 to 46 of human lymphoblastoid interferon LbIF Y consists essentially of amino acids 60 to 85 of said LbIF Z consists essentially of amino acids 112 to 166 of said LbIF P consists essentially of amino acids 47 to 59 of human alpha C , alpha F , or alpha I type leukocyte interferon LeIF and is gln47 phegln lys50 ala gln ala ileu ser55 val leu his glu59, or gln47 phe glnlys50 thr gln ala ileu ser55 val leu his glu59 and B consists essentially of amino acids 86 to 111 of human aC , aF , or aI type leukocyte interferon LeIF and is ser86 thr glu leu tyr90 gln gln leu asn asp95 leu glu ala cys val100 ileu gln glu val gly105 valglu glu thr pro110 leu, ser86 thr glu leu asn90 gln gln leu asn 95 100 105 asp met glu ala cys val ileu gln glu val gly val glu glu thr pro110leu, or ser86 thr glu leu tyr90 gln gln leu asn asn95 leuglu ala cys val100 ileu gln glu val gly105 met glu glu thr pro110 leu and substantial polypeptide fragments and or derivatives thereof having biological or antiviral activity of human interferon. 2. A hybrid interferon polypeptide according to Claim 1 wherein X is a LbIF segment having the amino acid sequence asp2 leu pro gln5 10 15 thr his ser leu gly asn arg arg ala leu ileu leu leu ala gln20 met gly arg ileu ser25 leu phe ser cys leu30 lys asp arg his asp35 phe gly phe pro gln40 glu glu phe asp gly45 asn Y is a LbIF segment having the amino acid sequence met60 ileu gln gln ileuphe65 asn leu phe ser thr70 lys asp ser ser ala75 ala trp asn gluser80 leu leu asp lys phe85 and Z is a LbIF segment having the amino acid sequence met112 asn val glu115 ser ileu leu ala val120 ser lys tyr phe gln125 arg ileu thr leu tyr130 leu ser glu lys lys135 tyr 140 145 150 ser pro cys ala trp glu ileu val arg ala glu ileu met arg ser leu thr leu 1eu155 thr asn ieu gln glu1 60 arg leu arg asn 165 asp 3. A deoxynucleotide sequence encoding for a hybrid lymphoblastoid leukocyte human interferon comprising 166 amino acids and optionally having an additional methionine attached to the ordinary first amino acid at the N terminus with a plus strand represented by the following formula 5 ATG W X P Y B Z wherein W is TGT or TGC X consists essentially of deoxynucleotides encoding for amino acids 2 to 46 of human lymphoblastoid interferon LbIF Y consists essentially of deoxynucleotides encoding for amino acids 60 to 85 of said LbIF Z consists essentially of deoxynucleotides encoding for amino acids 112 to 166 of said LbIF P consists essentially of deoxynucleotides encoding for amino acids 47 to 59 of human aC , a F , or al type leukocyte interferon LeIF and is CAG47 TTC CAA AAG50 GCT CAG GCA ATC TCG55 GTT CTG CAT GAG59, or CAG47 TTC CAA AAG50 ACC CAG GCA ATC TCG55 GTT CTG CAT GAG59 B consists essentially of deoxynucleotides encoding for amino acids 86 to 111 of said cr C , aF , or alpha I type LeIF and is TCC86 ACT GAA CTC TAT90 CAG CAA CTG AAC GAT95CTT GAA GCT TGC GTT100 ATC CAG GAG GTA GGC105 GTT GAA GAG ACT CCG110CTT, TCC86 ATC GAA CTC AAT90 GAG CAA CTG AAC GAT95 ATG GAA GCT TGC GTT100 ATC GAG GAG GTA GGC105 GTT GAA GAG ACT CCG110 OTT, or TCC86 ATCGAA CTC TAT90 GAG CAA CTG AAC AAC95 CTT GAA GCT TGC OTT100 ATC CAG GATGTA GGC105 ATG GAA GAG ACT COG110 CTT and transcriptional and translational equivalents, including single or multiple base substitutions, insertions and inversions and substantial fragments thereof permitting the expression of said hybrid interferon or biologicallyactive fragments thereof wherein A is deoxyadenyl, G is deoxyguanyl, C is deoxycytosyl and T is thymidyl. 4. A deoxynucleotide sequence according to Claim 3 whereinX is GAT2 TTG CCG CAG5 ACT CAT AGC TTG GGC AAC CGA AGA GCA CTC15 ATCCTG TTG GCC CAA20 ATG GGT CGC ATT TCC25 CTG TTC TCG TGC CTT30 AAA GACCGC CAC GAT35 TTC GGG TTT CCA CAG40 GAA GAG TTT GAC GGC45 AAT Y isATG60 ATT CAG CAA ATC TTC65 AAC CTG TTC TCC ACT70 AAA GAC TCT TCG GCT75 GCT TGG AAC GAA TCC80 TTG CTT GAT AAA TTC85 and Z is ATG112 AAT GTT GAG115 TCT ATC CTG GCT GTA120 TCT AAG TAT TTT CAG125 CGA ATT ACC CTC TAC130 CTC AGT GAG AAG AAA135 TAT TCA CCG TGT COG140 TGG GAA ATT GTGAGA145 GCC GAA ATC ATG CGT150 TCC CTG ACT CTT CTG155 ACG AAC CTC CAGGAA160 CGC CTG CGT AAT AAA165 GAC. 5. A recombinant DNA molecule comprising a DNA sequence according to Claim 1 or 2. 6. The recombinant DNA molecule according to Claim 5 wherein said DNA sequence is operatively linked to an expression control sequence. 7. A host transformed with at least one recombinant DNA molecule comprising a DNA sequence encoding for a hybrid lymphoblastoid leukocyte human interferon according to Claim 1, and wherein said DNA sequence is operatively linked to an expression control sequence. 8. The host of Claim 7 selected from the group consisting of Escherichia coli, Bacillus subtilis, or other bacteria, yeasts or other fungi, mouse or other animal or plant hosts and human tissue cells. 9. A host transformed with at least one recombinant DNA molecule encoding for a hybrid lymphoblastoid leukocyte human interferon comprising a hybrid promoter region, an operator region, a ribosome binding site and a polypeptide coding region coding for a polypeptide of 166 amino acids and optionally coding for an additional methionine attached to the ordinary first amino acid at the N terminus of said polypeptide, and a termination region with a plus strand represented by the following formula 5 L 0 Q ATG W X P Y B Z E wherein L consists essentially of the promoter region 0 consists essentially of the operator region Q is a portion of a ribosome binding site and is A A A A A T T A A G G A G G A T C A C T, T T A TT A C C C A A C T T G A G G A A T T T A T A,T T G A G G A T A A A T T W is TGT or TGC X consists essentially of deoxynucleotides encoding for amino acids 2 to 46 of human lymphoblastoid interferon LbIF Y consists essentially of deoxynucleotides encoding for amino acids 60 to 85 of said LbIF Z consists essentially of deoxynucleotides encoding for amino acids 112 to 166 of said LbIF P consists essentially of deoxynucleotides encoding for amino acids 47 to 59 of human aC , aF , or al type leukocyte interferon LeIF and isCAG TTC CAA AAG50 GCT CAG GCA ATC TCG55 GTT CTG CAT GAG, or CAG TTC CAA M G50 ACC CAG GCA ATC TCG55 GTT CTG CAT GAG B consists essentially of deoxynucleotides encoding for amino acids 86 to 111 of said a C , a F , or tI type LeIF and is TCC ACT GAA CTC TAT90 CAG CAA CTG AACGAT95 CTT GAA GCT TGC GTT100 ATC CAG GAG GTA GGC105 GTT GAA GAG ACT CCG110 CTT, TCC ATC GAA CTC AAT90 CAG CAA CTC AAC GAT ATG GAA GCT TGC GTT100 ATC CAG GAG GTA GGC105 GTT GAA GAG ACT CCG110 OTT, or TCC ATCGAA CTC TAT90 CAG CAA CTG AAC AAC95 CTT GAA GCT TGC GTT100 ATC CAG GATGTA GGC1 ATG GAA GAG ACT CCG110 CTT E consists essentially of the termination region and transcriptional and translational equivalents, including single or multiple base substitutions, insertions and inversions and substantial fragments thereof permitting the expression of said hybrid interferon or biologically active fragments thereof wherein A is deoxyadenyl, G is deoxyguanyl, C is deoxycytosyl and T is thymidyl. 10. The host of Claim 9 selected from the group consisting of Escherichia coli, Bacillus subtilis, or other bacteria, yeasts or other fungi, mouse or other animal or plant hosts and human tissue cells.

## Description
HYBRID LYMPHOBLASTOID LEUKOCYTE HUMAN INTERFERONS The invention herein described relates to a method for the construction of a nucleotide sequence encoding for novel hybrid lymphoblastoid leukocyte human interferons and including a hybrid promoter and ribosome binding sites useful for the expression of the hybrid interferons. The synthetic DNA sequences are introduced into plasmid vectors and subsequently introduced into microorganisms.Transformed microorganisms and .the expression of novel hybrid interferons are described. By way of background, it is well known that there are numerous commercial applications of microorganisms. For example, both eukarsotic and prokaryotic microorganisms have long been used for the treatment of wastes, as sources of protein and medicinal products, and for diverse biological conversions into both primary and secondary metabolites Sci. Amer. 245 67 74 1981 . More recently, it has become possible to transfer operative genes among microorganisms and between microorganisms and various higher forms of life. The modern techniques of genetic engineering, with particular reference to recombinant DNA RNA technology, now makes it theoretically possible to transfer any gene or group of genes to a plasmid capable of functioning in microorganisms. Cenetic recombination is the biological process whereby genetic information is exchanged between chromosomes. The fields of both classical and molecular genetics involve resolution of the complexities of genetic recombination. Recombination ordinarily does not occur between unrelated species of organisms. Nevertheless, the new recombinant DNA technologies now allow the transfer of genes between related or unrelated organisms Amer. Sci., 68 664 674 1980 . The genetic alteration of gene structure has particular application in the field of industrial microbiology Sci. Amer.245 91 102 1981 . Genetic recombination involves the breaking and splicing of molecules of DNA. It is now possible to gather diverse genetic information in vitro in the form of DNA fragments from various prokaryotic and eukaryotic sources and to introduce this genetic material into a self replicating genetic moiety known as a cloning vector. DNA fragments, or even intact genes, can be constructed by synthetic chemical means to correspond to desired theoretical genetic sequences, or to known genetic sequences, and then introduced into a selected vector. Cloning vectors are commonly bacterial plasmids or bacteriophages. Slasmids are autonomous extra chromosomal genetic units which consist of a circular strand of DNA and are present in most bacteria and in some eukaryotes. Bacteriophages are DNA viruses which parasitize only bacteria.The hybrid genetic vectors are then introduced into selected microorganisms in a process known as transformation. Thus transformed microorganisms are potential factories useful for the production of large amounts of the clonedDNA. The foreign genetic information present in the cloning vector is often not expressed in the transformed microorganisms. A non expressing cloning vector may be desirable in those instances where expression of the foreign genome could produce a product lethal to the host organism. A non expressing or low expressing cloning vector represents a ready source of the foreign genetic material which can then be isolated in vitro and introduced into an expression vector i.e., a specially prepared or selected plasmid .The foreign DNA is placed in a suitable location i.e., inserted within the lac operon in an expression vector where the indigenous genetic sequence is such that the foreign genetic information will be transcribed i.e., mRNA produced from the foreign DNA and translsted i.e., protein synthesis from the triRNA template and the desired product coded in the foreignDNA obtained. As with the manufacture of the cloning vector, the expression vector introduced into a selected microorganism and the resultant transformed microbe serve as a factory for the manufacture of the foreign DNA product. With the development of the class of enzymes known as restriction endonucleases, DNA can now be cut in vitro at specific locations in preparation for insertion into an appropriate transfer vector. Restriction endonucleases are enzymes which are site specific endonucleases which primarily cleave double stranded DNA but in some instances cleave single stranded DNA. All restriction endonucleases recognize specific DNA sequences. Class II restriction endonucleases cleave at specific sequences, whereas Class I restriction endonucleases appear to cleave DNA randomly and produce heterogeneous products. Various restriction endonucleases produce DNA fragments of different lengths and types.For example, some restriction endonucleases cleave both DNA strands at the same point and produce blunt end DNA fragments, in contrast to other enzymes which cleave one DNA strand several nucleotides away from the cleavage on the complementary strand and produce cohesive end DNA fragments. A practitioner in the recombinant DNA art by creative selection of endonucleases for treatment of subject DNA can obtain desired DNA fragments which can then be joined together by the action of a DNA ligase. In this manner the cutting and splicing of genetic information DNA is acconplished. Following the insertion of foreignDNA into a microbe, the exact sequences which have been cloned can be determined by DNA sequencing procedures. In order to effect a desired end sequence of subject DNA fragments i.e., for ligation purposes , nucelotides can be added to the terminal portion of such DNA. For example, in the process known as homopolymeric tailing, a series of identical deoxyribonucleotides is added to the 3 ends of the cloning vector and a series of complementary deoxyribonucleotides is added to the 3 ends of the DNA fragments to be cloned. These DNA sequences can then be joined and introduced into a microbial host. Another general method for obtaining a desired DNA fusion product is the addition of adapter or linker fragments to the ends of either or both the cloning vector or the DNA fragments to be cloned. Linker fragments are small sections of DNA that contain one or more recognition sequences for restriction endonucleases. Following the insertion of foreign DNA into a microbe, the exact sequences which have been cloned can be determined by DNA sequencing procedures. There are several ways in which foreign DNA material can be obtained for insertion into a transfer vector. DNA fragments directly obtained from the parental source can be inserted into the appropriate vector. Another method is to obtain mRNA from an active synthesis location product expression area in the parent system and to obtain a single stranded complementary DNA strand synthesized enzymatically reverse transcriptase from the isolated mRNA. A double stranded DNA molecule is then synthesized from the single stranded template.Double stranded DNA obtained in this fashion is known as complementaryDNA cDNA . Once a desired DNA sequence is known, genes can also be chemically synthesized in vitro for cloning expression purposes in microbial, tissue, or cell culture systems. Interferons are a large family of proteins released by various animal cells exposed to a virus which enable other cells to resist infection. Interferons vary from species to species. Although their mode of action is not yet well understood, they are generally believed to be antiviral in nature. Presumably, interferons may be effective against viral diseases and perhaps cancers. Interferon is known to have various biological activities other than those which contribute to antiviral effects. Three classes or groups of human interferons have been identified alpha or leukocyte interferon beta or fibroblast interferon and gamma or immune interferon. Alpha or leukocyte interferons are a family of related proteins having a chain length consisting of 165 or 166 amino acids.At least eight distant subtypes of alpha interferon have been detected by cloning cDNAs from leukocyte mRNAs. Many different alpha interferons have already been isolated and characterized. By use of recombinant DNA technology, it is now possible to isolate and characterize human interferon genes and to clone them into bacteria or other microorganisms for the production of quantitatively significant amounts of interferons for experimental and medical purposes. Lymphocytes are white blood cells leukocytes formed in lymphoid tissue, and lymphoblasts are immature lymphocytes. The chemical sequence of several human leukocyte interferons is known.However, the amino acid sequences of certain lymphoblastoid interferons have not yet been completely characterized. Among these partially characterized interferons is the lymphoblastoid interferon produced by the induction of Namalva cells with Newcastle disease virus, strain B1 Proc. Natl. Acad. Sci., 76 5601 5605 1979 J. Biol.Chem., 252 6585 6587 1977 . A portion of the amino acid sequence of this protein has been determined but the entire sequence is not among the known a interferons. This lymphoblastoid interferon is an attractive candidate for gene synthesis for several reasons. Firstly, the interferon is expressed naturally as the result of a viral infection in sufficient quantities for isolation and various biological and biochemical studies. Secondly, the gene for this interferon is not among known cloned interferon genes. Lastly, lymphoblastoid interferons which occur naturally are among the major classes of interferons currently undergoing clinical trials for medical purposes. The characterization, purification and manufacture of interferons using recombinant DNA procedures has been reviewed Sci.Amer. 249 37 43 1983 . Various hybrid human leukocyte interferons are described in United States Patent 4,414,150. In addition, gene sequences, recombinant DNA molecules, transformed microorganisms, and methods for producing human interferon like polypeptides are described in European Patent Application No. 32,134. In light of the foregoing discussion of some of the recent advances in the field of genetic engineering, and more specifically to the identification, chemical characterization and manufacture of human interferons, an improved method for the preparation of novel interferon genes is highly desirable. An object of this invention is to produce novel hybrid human interferons comprising polypeptide sections of lymphoblastoid interferon and leukocyte interferon.Another object of this invention is to synthesize the eucaryotic gene sequence coding for novel lymphoblastoid leukocyte hybrid proteins, including the synthesis of a gene section coding for appropriate ribosome binding sites and a promoter region. Yet another object of this invention is to clone synthesized hybrid lymphoblastoid leukocyte gene sequences in an appropriate vector and microbial system and to subsequently gain expression of novel hybrid interferons having biological activity. Yet another object of this invention is to develop novel transformed microorganisms containing the nucleotide sequence encoding for novel lymphoblastoid leukocyte hybrid interferons useful as antiviral agents. These and further objects are manifest in the following description and particularly delineated in the appended claims. The synthesis of a gene containing certain lymphoblastoid interferons and leukocyte interferon sequences is described.Embodiments comprise either alpha C, F, or I type leukocyte interferon sequences. The synthesis method is designed so that lymphoblastoid interferon sequences can be combined with either alpha C, F, or I leukocyte interferon sequences. The method of synthesis of the hybrid interferon gene sequences of the invention involves a combination of chemical and enzymatic procedures. In addition, the derivation of a hybrid promoter and three hybrid ribosome binding sites useful for the expression of this gene is described. The lymphoblastoid hybrid interferon gene is composed of a 598 base pair DNA duplex. The gene sequence codes for the 166 amino acids of the structural gene, an initiation codon, two termination codons and a control region containing a promoter, an operator and a ribosome binding site. The DNA base paired sequence for this gene, together with the amino acids for which it is coded, is presented inTable 1. Directly above the DNA sequence are numbers every 20 base pairs that designate nucleotides relative to the 5 EcoRI cut site on the sense strand which is number 1. Above the numbers are brackets which define the lambda PR T7 A2 promoter, lac operator, 1 ribosome binding site, and the structural gene. The brackets defining R1 extend through the eighth amino acid codon. The amino acid sequence for the lymphoblastoid hybrid gene is located above the appropriate codons of the DNA sense strand. Near the 3 end of the sense strand can be found two termination codons and at the 3 terminus, the EcoRI 3 cut site. Also shown by underlining are certain key restriction sites EcoRI twice , AluI, HaeIII, DdeI, and HindIII. The major elements of this gene promoter, ribosome binding site, and protein coding region are hybrid structures and do not correspond to any knows natural DNA sequences. TABLE 1. Lymphoblastoid Hybrid Interferon Gene Sequence EMI8.1 met cys asn leu pro gln thr his ser leu 80 100 Alul 120G G A T A A C A A T T A A A A A T T A A G G A G G A T C A C T A T G T G T G A T T T G C C G C A G A C T C A T A G C T TC C T A T T G T T A A T T T T T A A T T C C T C C T A G T G A T A C A C A C T A A A C G G C G T C T G A G T A T C G A A gly asn arg arg als leu lleu leu leu l gln met gly arg lleu ser leu he ser cys 140 lll 160 180G G G C A A C C G A A G A G C A C T C A T C C T G T T G G C C C A A A T G G G T C G C A T T T C C C T G T T C T C G T GC C C G T T G G C T T C T C G T G A G T A G G A C A A C C G G G T T T A C C C A G C G T A A A G G G A C A A G A G C A C leu lys asp arg hls asp phe gly phe pro gln glu glu phe asp gly as gln phe gln 200 220 240C C T T A A A G A C C G C C A C G A T T T C G G G T T T C C A C A G G A A G A G T T T G A C G G C A A T C A G T T C C AG G A A T T T C T G G C G G T G C T A A A G C C C A A A G G T G T C C T T C T C A A A C T G C C G T T A G T C A A G G T lys ala gln ala lleu set val leu his glu met lleu gln gln lleu phe asn leu phe ser de l 260 280 300A A A G G C T C A G G C A A T C T C G G T T C T G C A T G A G A T G A T T C A G C A A A T C T T C A A C C T G T T C T CT T T C C G A G T C C C T T A G A G C C A A G A C G T A C T C T A C T A A G T C G T T T A G A A G T T G G A C A A G A G thr lys asp ser ser als ala trp asn glu ser leu leu asn lys phe ser thr glu leu 320 340 360C A C T A A A G A C T C T T C G G C T G C T T G G A A C G A A T C C T T G C T T G A T A A A T T C T C C A C T G A A C TG T G A T T T C T G A G A A G C C G A C G A A C C T T G C T T A G G A A C G A A C T A T T T A A G A G G T G A C T T G A tyr gln gln leu asn asp leu glu ala cys val lleu gln glu val gly val glu glu thr 380 Hindlll 400 420C T A T C A G C A A C T G A A C G A T C T T G A A G C T T G C G T T A T C C A G G A G G T A G G C G T T G A A G A G A CG A T A G T C G T T G A C T T G C T A G A A C T T C G A A C G C A A T A G G T C C T C C A T C C G C A A C T T C T C T C TABLE 1.Lymphoblastiod Hybrid Interferon Gene Sequence pro leu met asn val glu ser lleu leu ala val ser lys tyr phe gln arg lleu thr leu 440 460 480T C C G C T T A T G A A T G T T G A G T C T A T C C T G G C T G T A T C T A A G T A T T T T C A G C G A A T T A C C C TA G G C G A A T A C T T A C A A C T C A G A T A G G A C C G A C A T A G A T T C A T A A A A G T C G C T T A A T G G G A tyr leu ser glu lys lys tyr ser pro cys ala trp glu lleu val arg ala glu lleu et 500 520 540C T A C C T C A G T G A G A A G A A A T A T T C A C C G T G T G C G T G G G A A A T T G T G A G A G C C G A A A T C A TG A T G G A G T C A C T C T T C T T T A T A A G T G G C A C A C G C A C C C T T T A A C A C T C T C G G C T T T A G T A arg ser leu thr leu thr asn leu gln glu arg leu arg asn lys asp term term 560 580 600G C G T T C C C T G A C T C T T C T G A C G A A C C T C C A G G A A C G C C T G C G T A A T A A A G A C T A A T A G C G C A A G G G A C T G A G A A G A C T G C T T G G A G G T C C T T G C G G A C A C G C A T T A T T T C T G A T T A T C T T A A The promoter 1 71 is primarily the lambda major rightward promoter lambda P and the lac operator. The objective was to design a transcriptionally very active promoter under lac control. At the time this project was initiated, no such promoter was known.However, a combination of the tryptophan operon promoter with the lac operator was independently developed at about the same time the tac promoter and these two constructions were published simultaneously tCaruthers, M. H., et al., In Promoters, Structure and Function R.Rodriguez and M. Chamberlin, eds. , Praeger, N.Y., 1982, pp. 432 451 DeBoer, H. A., et al., In Promoters, Structure and Function R.Rodriguez and M. Chamerlin, eds. , Praeger, N.Y., 1982, pp. 462 481 .Additionally, the specific selection of a hybrid promoter containing lambda PR in combination with lac operator was especially attractive as a model system for independent studies on protein DNA interaction mechanisms. This particular hybrid promoter would be recognized by E.coli RNA polymerase, lac repressor, cro repressor, and cI repressor.The interactions of these various proteins with the same DNA modified or unmodified could therefore be studied. Lambda PR was specifically selected in contrast to lambda PL because it could be extended to include the lambda PRM promoter. Once extended to contain two promoters, the DNA could be used to study promoter selection by E. coli RNA polymerase as a function of sequence modification. An early construction of the hybrid lambda PR T7 A2 lac promoter was simply the lac operator sequence position 51 71 combined with the lambda PR promoter sequence not shown between 6 50.However, a lambda PR sequence dAATGGT between the 10 region and the transcription initiation site was homologous with the lac operator sequence from 51 56 dAATTCT . The homology created difficulties when T4 DNA ligase was used in attempts to join synthetic oligodeoxynucleotides located in this region of the gene. The homology was eliminated by changing nucleotides 46 49 to the sequence found at these positions relative to transcription initiation in theT7 A2 promoter. Such a substitution was not considered detrimental to transcription activity of the hybrid promoter since among promoters the sequence in this region is highly variable. Therefore the final promoter DNA sequence is a composite of the lambda PR 5 45 , T7 A2 46 50 and lac 51 71 promoters.The transcription activity of this hybrid promoter was compared to the activity of the lac promoter.Both promoters were cloned into the same plasmid background and assayed for ss galactosidase activity in a lac deletion strain of E.coli. The lambda PR T7 A2 lac hybrid promoter is found to be approximately 50X more active than the fully induced lac promoter and approximately fourfold more active than the lac promoter under catabolite repression glucose as the carbon source . Ribosome binding sites still remain as undefined structures.Clearly translation from a ribosome binding site in E. coli starts from either an AUG or a GUG codon which is usually located approximately 6 8 nucleotides from a purine rich DNA segment called the Shine and Dalgarno sequence. Based on systematic comparisons of a large number of ribosome binding sites, several additional observations on important sequence elements within these sites have been postulated Scherer, G., et al. Nucleic Acids Res. 8 3895 3907 1980 . Table 2 presents these different ribosome binding sites. The first eight amino terminal amino acids coded by the lymphoblastoid hybrid interferon gene of the invention are shown. The clustered set of partial ribosome binding sites that will code for this protein sequence is shown directly below these amino acids. Also shown are ribosome binding sites R1, R2, and R3 and the idealized or known ribosome binding sites R.B.S. from which they were developed. TABLE 2. Ribosome Binding SitesAmino Acid Sequence met cys asp leu pro gln thr hisAmino Acid Codons A U G U G U G A U U U U C C U C A A A C U C A UDerived Rl Sequence A A A A A U U A A G G A G G A U C A C U A U G U G U G A U U U G C C G C A G A C U C A U dealized Sequence a a a a a u u a a g g a g g u a u a u u a u g a a a a a a a u u a a a a a a c u c a aDerlved R2 Sequence A U U A C C C A A C U U G A G G A A U U U A U A A U G U G U G A U U U A C C A C A A A C C C A ULll R. B. S. Sequence a u u a c c c a a c u u g a g g a a u u u a u a a u g g c u a a u a a a g u a c a a g c c u a uDerlved R3 Sequence C G G C C C C U U A C U U G A G G A U A A A U U A U G U G U G A U C U U C C A C A A A C C C A U xF R.B.S.Sequence c g g c c c c u u a c u u g a g g a u a a a u u a u g u c u a a u a u u c a a a c u g g c g c c At the time this research was initiated, certain observations were available to aid in the design of a hybrid ribosome binding site. Sherer et al. Nucleic Acids Res. 8 3895 3907 1980 1 had proposed a model ribosome binding site based on a computer analysis of 68 known E. coli ribosome binding sites. For R1 we chose to use this sequence after further modification. The first modification involved placing amino acid codons 3 to AUG that code for lymphoblastoid interferon amino acids. Wherever possible, model ribosome binding site nucleotide sequences were conserved.Additional sequence modifications were based on the analysis by Iserentant andFiers on the translation efficiency of a series of recombinant plasmids containing various ribosome binding sites 5 to the cro gene Gene, 9 1 112 1980 . They concluded that an ideal ribosome binding site consists of a folded structure having a base paired stem loop region with the AUG codon in the loop portion. Preferably the Shine and Dalgarno sequence would be 5 to the stem and in a single strand region. R1 was therefore modified to contain five base pairs formed from nucleotides on either side 86 90 and 96 100 of the AUG initiation codon. This was accomplished by further sequence changes in the model ribosome binding site sequence between 86 90. R1 as shown in Table 2 therefore contained wherever possible nucleotides corresponding to the model sequence.Modifications were however introduced to include the correct codons for the lymphoblastoid interferon gene, to position the AUG codon in the loop portion of the hair pin structure, and to locate the Shine and Dalgarno sequence in a single stranded region 5 to the base paired stem structure. Ribosome binding sites R2 and R3 were derived differently by modification of known ribosome binding sites. Early research by J. Steitz using translation initiation complexes demonstrated that approximately 40 ribonucleotides of mRNA centered on the AUG codon were protected from nuclease digestion by ribosomes Nature New Biol. 236 71 75 1972 .This protected sequence is now generally defined as the ribosome binding site. The 20 or so nucleotides located 3 to the AUG initiation codon are quite variable and dependent on the amino terminal amino acid sequence of a particular protein. The first step in deriving the nucleotide sequences of R2 and R3 was to identify known ribosome binding sites with potential homology to the lymphoblastoid interferon amino acid sequence. This was accomplished by screening known ribosome binding site sequences Scherer, G. F. E.et al., 1980 Nucl. Acid. Res. 8 3895 3907 from the AUG codon through the next seven codons 21 nucleotides for DNA sequence homology with a cluster of potential lymphoblastoid interferon gene ribosome binding sites. This cluster was derived from all possible codons for the first eight amino acids of the lymphoblastoid interferon. Therefore, each nucleotide of a known ribosome binding site was compared with all possible nucleotides that could be inserted at the same relative position of the interferon gene while keeping its amino acid sequence constant. For example, R3 was derived from the X F protein ribosome binding site. The third codon in the X F ribosome binding site is AAU which codes for asparagine. For the lymphoblastoid gene, the third amino acid is aspartic acid having codons GAU and GAC.Therefore, if GAU is selected as the aspartic acid codon, a fit is possible for two of the three nucleotides at the same relative positions within the fX F protein and R3 ribosome binding sites. Using this inspection procedure, all known ribosome binding sites were screened for possible homologies with the clustered set of potential lymphoblastoid gene sites. Several were identified where the maximum fit with the lymphoblastoid cluster over the first eight codons 24 nucleotides was approximately 50 . Two of these E.coli Lll protein and fX F protein sites , which presumably are used to initiate translation of large amounts of protein, were selected as lymphoblastoid interferon ribosome binding sites. Wherever possible within the protein coding regions of these known sites 24 nucleotides , the DNA sequence was maintained. Certain nucleotides were, however, substituted so that lymphoblastoid amino acid sequences could be expressed rather than either Lll or X F amino acids. The nucleotides 5 to the AUG codons in both the Lll and X F protein sites 26 nucleotides each were maintained without modification as part of the R2 and R3 sites, respectively.Both design strategies appear to be valid as interferon hybrid protein is produced in 0.1 2 mg OD578 nm liter quantities from all three ribosome binding sites. The lymphoblastoid gene sequence was obtained from the partial amino acid sequence of a human lymphoblastoid interferon produced by induction of Namalva cells with Newcastle disease virus, strain B1. The analysis covered 76 of the total protein and suggested that this lymphoblastoid interferon was not listed among the sequences of known human interferons derived via cDNA cloning or through DNA sequencing of genomic clones. Goeddel, D. V. et al. 1981 Nature 290 20 26 Stebbing, N. and Weck, P. K. 1983 . In Recombinant DNAProducts, Insulin, Interferons and Growth Hormones, A. Bollen, ed., CRCPress Weissmann, C. et al. 1982 UCLA Symp. Mol. Cell Biol.25 295 326. Sequenced regions were amino acids 2 46, 60 85, and 112 166. These polypeptides were most analogous to the DNA sequences of IFN aC, IFN cI, and IFN oF. In the region 2 46, there was only one amino acid difference between the lymphoblastoid peptides and the amino acid sequence of IFN aF leu 26 in the lymphoblastoid sequence was proline in IFN a F . However, near the carboxyl terminus 112 166 , the lymphoblastoid protein was more analogous to IFN aC and IFN aI. If this human lymphoblastoid interferon is analogous to all other cr interferons relative to size and general sequence, then a hybrid lymphoblastoid interferon can be derived. First the peptide sequences of the human lymphoblastoid interferon were alligned with the analogous positions in IFN aC and incorporated into the lymphoblastoid hybrid gene at these same relative positions. ThenIFN a C peptide sequences were first used in place of the unknown lymphoblastoid interferon sequences amino acids 47 59 and 86 to 111 .Corresponding sequences of IFN aF or IFN aI can conveniently be substituted in place of the IFN aC insertions at the 47 59 and 86 111 positions. However, it is known that IFN aC is equivalent to IFN a F at the 47 59 position. Thus the final interferon hybrid gene is a composite of these two presumably related proteins. The present disclosure relates the detailed procedure involved in the synthesis of a hybrid interferon gene having a C leukocyte interferon inserts at the 47 59 and 86 111 positions.However, the corresponding leukocyte IFN a F and IFN al sequences at these positions can be introduced by identical procedures. For example, new synthetic DNA fragments having sequences corresponding to leukocyte IFN a F and a I at positions 47 59 and 86 111 can be synthesized. These segments can then be ligated to the appropriate lymphoblastoid sections to form the additional hybrid interferons. In the construction of these new genes, new restriction sites corresponding to the new sections would have to be used. These new sites can be identified by simply screening the entire gene for all possible restriction sites and using those that can be used most conveniently. The DNA sequence of the lymphoblastoid hybrid interferon structural gene was based on several considerations. Initially, deoxyoligonucleotides were to be chemically synthesized and enzymatically joined to form the total gene. This total DNA duplex containing terminal EcoRI sites would then be cloned into the singleEcoRI site of pUC8. These oligodeoxynucleotides were generally designed so as to maximize overlap between complementary segments and minimize overlap between the single stranded regions. Since the gene was designed for expression in E. coli, codons preferred by this organism were selected wherever possible. Additionally, certain codons were chosen so as to minimize or eliminate inverted and direct repeat sequences which could create serious problems during enzymatic ligation reactions.Chemically synthesized oligodeoxynucleotides were joined enzymatically to form four DNA duplexes containing 108 to 214 base pairs each. These four duplexes were then amplified individually in plasmids, isolated, sequenced and then sequentially joined through convenient restriction sites, and cloned to form the final gene. This plan is quite general and can be used even for much larger genes, since only relatively short duplexes 250 base pairs or so are synthesized via chemical and enzymatic procedures. Certain restriction sites were introduced which led to the successful cloning of the total gene as four sections. A restriction map of the lymphoblastoid hybrid interferon gene of the invention is presented in Table 3. It is likely that additional restriction enzymes will become known. The top portion of Table 3 is a schematic representation of the gene. Each 100 base pairs and the approximate location of restriction sites are marked. The precise position and sequence for various restriction sites are also listed. TABLE 3. Restriction Map of the Lymphoblastoid Hybrid Interferon Gene EMI18.1 SEP 160 SEP 200 SEP 300 SEP 400 SEP 500 SEP 600 tb Enzyme Sequence SiteHincII GTTGAC 15Sau 3A GATC 05, 377HinfI GA T,C,A,G TC 109,277, 308, 329, 418, 437, 550Alu I AG CT 116, 305Hae III GG CC 149Dde I C T A,C AG 246, 456, 405HindIII AAGCTT 384EcoRII CC A,T GG 397, 445, 568Mbo II GAAGA N8 129, 215, 413, 493 Those restriction sites which were advantageously introduced are HaeIII position 149 DdeI position 246 and HindIII position 384 . Using these three sites, the gene was cloned as Sections I 1 149 , II 150 246 , III 247 384 , and IV 385 598 . After amplification in E. coli as part of plasmids pBR322 or pUC8, each section was isolated and sequenced.Sections I and II were then joined enzymatically, cloned, and amplified as an EcoRI and DdeI restriction fragment. This combined duplex was next joined enzymatically to Section III, cloned as a EcoRI and HindilI fragment, amplified in E. coli and isolated after restriction with EcoRI and BindIII. Finally, this EcoRI and HindIII duplex was joined enzymatically to Section IV. The final gene was sequenced and tested for expression of a interferon. The structural gene, hybrid promoter, and three ribosome binding sites required the synthesis of 92 oligodeoxynuclotides 9 to 26 mononucleotides each . The synthetic plans for the structural gene, hybrid promoter, and ribosome binding site RI are shown in Table 1 above. Two major DNA synthesis methodologies were used.Oligodeoxynucleotides for the hybrid promoter and all three ribosome binding sites were synthesized using 5 dimethoxytrityldeoxynucleoside 3 methoxytetrazoylphosphoramidites as synthons and silica gel containing a covalently attached deoxynucleoside as the polymer support. After removal of protecting groups, the oligodeoxynucleotides were purified of various intermediates by polyacrylamide gel electrophoresis. The products can then be eluted from the gel.After passage through a short G50 40 Sephadex column to remove salts, buffers and solubilized gel material, the oligodeoxynucleotides were 32 phosphorylated with y P ATP and T4 polynucleotide kinase. Results from analytical gel electrophoresis indicated that the products were homogeneous and ready to be used with T4 DNA ligase for preparing larger DNA duplexes. In summary, the lymphoblastoid hybrid interferon gene of the invention was synthesized and cloned in four sections. These four sections were then joined to form the total gene. Thus, oligodeoxynucleotides containing from 10 to 25 mononucleotides each were synthesized chemically. Complementary segments contained protruding single strand regions for guiding the next segment into place. These segments were then joined enzymatically to form DNA duplexes containing up to 214 base pairs each. Using appropriate restriction sites, these gene sections can be amplified through cloning, isolated, and sequentially joined to one another to form the completely assembled gene. This plan is quite general and can be used to assemble genes of any size. Each section should be designed similar to SectionsII and III.These two sections whose structural gene ends have high frequency restriction cutting sites were synthesized with linkers having restriction sites unique to the plasmid vector EcoRI andHindIII . If necessary the gene segment can then be excised and separated from the plasmid using gel electrophoresis. The vector would migrate as one large fragment and the gene section as a small duplex containing 108 214 base pairs. Alternatively, the section can be excised directly from the plasmid using the high frequency cutting sites in this case HaeIII and DdeI . This possibility can be used if the gene section duplex migrates in a unique region of the gel devoid of plasmid duplexes. Although not necessary, the synthesis would be somewhat simplified if the unique restriction sites were part of the structural gene rather than linker sequences. Section III is an example of this approach. Due to the redundancy of the genetic code and the large number of restriction enzymes that recognize different size base pair sequences, this alternative is usually possible. The biological activity of various control regions and the lymphoblastoid hybrid interferon are of considerable interest. The promoter is a composite of three different promoter elements and the ribosome binding sites were designed for the interferon gene and do not correspond to any natural sequences. The interferon protein is a hybrid molecule composed of sequences from lymphoblastoid and leukocyte interferons. Sections of the novel hybrid interferon from a C type interferons represent a preferred embodiment of the invention.Optionally, aF or aI type leukocyte interferon sections may be substituted in place of a C type interferon. All these structures i.e., the hybrid control elements and the hybrid interferon were quite active biologically. The hybrid promoter was designed to be transcriptionally active but easily controlled. It contains elements of the PR and T7A2 promoters and the lac operator. Experimental results also show that transcription can be controlled with lac repressor and IPTG Isopropyl iD thiogalactoside . Deoxynucleotide sequences of the invention encoding for hybrid lymphoblastoid leukocyte human interferons comprising 166 amino acids and optionally having an additional methionine attached to the ordinary first amino acid at the N terminus have plus strands represented by the following formula wherein A is deoxyadenyl, G is deoxyguanyl, C is deoxycytosyl, and T is thymidyl, and including transcriptional and translational equivalents, including single or multiple base substitutions, insertions and inversions 5 ATGW X P Y B Z wherein W is TGT or TGC X consists essentially of deoxynucleotides encoding for amino acids 2 to 46 of human lymphoblastoid interferon LbIF Y consists essentially of deoxynucleotides encoding for amino acids 60 to 85 of said LbIF Z consists essentially of deoxynucleotides encoding for amino acids 112 to 166 of said LbIF P consists essentially of deoxynucleotides encoding for amino acids 47 to 59 of human a C , a F , or al type leukocyte interferon LeIF and is CAG47 TTC CAAAAG50 GCT CAG GCA ATC TCG55 GTT CTG CAT GAG59, or CAG47 TTC CAA AAG50 ACC CAG GCA ATC TCG55 GTT CTG CAT GAG59 B consists essentially of deoxynucleotides encoding for amino acids 86 to 111 of said aC , aF , or al type LeIF and is TCC86 ACT GAA CTC TAT90 CAG CAA CTG AAC GAT95 CTT GAA GCT TGC GTT100 ATC CAG GAG GTA GGC105 GTT GAA GAG ACT CCG110 CTT, TCC86 ATC GAA CTC AAT90 CAG CAA CTG AAC GAT95 ATG GAA GCT TGC GTT100 ATC CAG GAG GTA GGC105 GTT GAA GAG ACT CCG110 CTT, or TCC86 ATCGAA CTC TAT90 CAG CAA CTG AAC AAC95 CTT GAA GCT TGC GTT100 ATC CAG GATGTA GGC105 ATG GAA GAG ACT CCG110 CTT.Consistent therewith are the following deoxynucleotide constructions 1. 5 ATG T TGT1 1 GAT TTG CCG CAG5 ACT CAT AGC TTG GGC10 AAC CGA AGA GCA CTC15 ATC CTG TTG GCC CAA20 ATG GGT CGC ATT TCC25 CTG TTC TCG TGC CTT30 AAA GAC CGC CAC GAT35 TTC GGG TTT CCA CAG40 GAA GAG TTT GAC GGC45 AAT CAG TTC CAA AAG50 GCT CAG GCA ATC TCG55 GTT CTG CAT GAG ATG60 ATT CAG CAA ATC TTC65 AAC CTG TTC TCC ACT70 AAA GAC TCT TCG GCT75 GCT TGG AAC GAA TCC80 TTG CTT GAT AAA TTC85 TCC ACT GAA CTC TAT90 CAG CAA CTG AAC GAT95 CTT GAA GCT TGC GTT100 ATC CAG GAG GTA GGC105 GTT GAA GAG ACT CCG110 CTT ATG AAT GTT GAG115 TCT ATC CTG GCT GTA120 TCT AAG TAT TTT CAG125 CGA ATT ACC CTC TAC130 CTC AGT GAG AAG AAA135 TAT TCA CCG TGT GCG140 TGG GAA ATT GTG AGA145 GCC GAA ATC ATG RTI ID 22.11 CGT150 TCC CTG ACT CTT CTG155 ACG AAC CTC CAG GAA160 CGC CTG CGT AAT AAA165 GAC 2. 5 ATG TGT1 z 1 GAT TTG CCG CAG5 ACT CAT AGC TTG GGC10 AAC CGA AGA GCA CTC15 ATC CTG TTG GCC CAA20 ATG GGT CGC ATT TCC25 CTG TTC TCG TCC CTT30 AAA GAC CGC CAC GAT35 TTC GGG TTT CCA CAG40 GAA GAG TTT GAC GGC45 AAT CAG TTC CAA AAG50 GCT CAG GCA ATC TCG55 GTT CTG CAT GAG ATG60 ATT CAG CAA ATC TTC65 AAC CTG TTC TCC ACT70 AAA GAC TCT TCG GCT75 GCT TGG AAC GAA TCC80 TTG CTT GAT AAA TTC85 TCC ATC GAA CTC AAT90 CAG CAA CTG AAC GAT95 ATG GAA GCT TGC GTT100 ATC CAG GAG GTA GGC105 GTT GAA GAG ACT CCG110 CTT ATG AAT GTT GAG115 TCT ATC CTG GCT GTA120 TCT AAG TAT TTT CAG125 CGA ATT ACC CTC TAC130 CTC AGT GAG AAG AAA135 TAT TCA CCG TGT GCG140 TGG GAA ATT GTG AGA145 GCC GAA ATC ATG CGT150 TCC CTG ACT CTT CTG155 ACG AAC CTC CAG GAA160 CGC CTG CGT AAT AAA165 GAC 3. 5 ATG TGT1 GAT TTG CCG CAG5 ACT CAT AGC TTG GGC10 AAC CGA AGA GCA CTC15 ATC CTG TTG GCC CAA20 ATC GGT CGC ATT TCC25 CTG TTC TCG TCC CTT30 AAA GAC CGC CAC GAT35 TTC GGG TTT CCA CAG40 GAA GAG TTT GAC GGC45 AAT CAG TTC CAA AAG50 GCT CAG GCA ATC TCG55 GTT CTG CAT GAG ATG60 ATT CAG CAA ATC TTC65 AAC CTG TTC TCC ACT70 AAA GAC TCT TCG GCT75 GCT TGG AAC GAA TCC80 TTG CTT GAT AAA TTC85 TCC ATC GAA CTC TAT90 CAG CAA CTG AAC AAC95 CTT GAA GCT TGC GTT100 ATC CAG GAT GTA GGC105 ATG GAA GAG ACT CCG110 CTT ATC AAT GTT GAG115 TCT ATC CTG GCT GTA120 TCT AAG TAT TTT CAG125 CGA ATT ACC CTC TAC130 CTC AGT GAG AAG AAA135 TAT TCA CCG TGT GCG140 TGG GAA ATT GTG AGA145 GCC GAA ATC ATG CGT150 TCC CTG ACT CTT CTG155 ACG AAC CTC CAG GAA160 CGC CTG CGT AAT AAA 165 GAC 4. 5 ATG TGT1 GAT TTG CCG CAG5 ACT CAT AGC TTG GGC10 AAC CGA AGA GCA CTC15 ATC CTC TTG GCC CAA20 ATG GGT CGC ATT TCC25 CTC TTC TCG TCC CTT30 AAA GAC CGC CAC GAT35 TTC GGG TTT CCA CAG40 GAA GAG TTT GAC GGC45 AAT CAG TTC CAA AAG50 ACC CAG GCA ATC TCG55 GTT CTG CAT GAG ATG60 ATT CAG CAA ATC TTC65 AAC CTC TTC TCC ACT70 AAA GAC TCT TCC GCT75 GCT TGG AAC GAA TCC80 TTG CTT GAT AAA TTC85 TCC ACT GAA CTC TAT90 CAG CAA CTG AAC GAT95 CTT GAA GCT TGC GTT100 ATC CAG GAG GTA GGC105 GTT GAA GAG ACT CCG110 CTT ATG AAT GTT GAG115 TCT ATC CTG GCT GTA120 TCT AAG TAT TTT CAG125 CGA ATT ACC CTC TAC130 CTC AGT GAG AAG AAA135 TAT TCA CCG TGT GCC140 TGG GAA ATT GTG AGA145 GCC GAA ATC ATC CGT150 TCC CTG ACT CTT CTG155 ACG AAC CTC CAG GAA160 CGC CTG CGT AAT AAA165 GAC 5. 5 ATG TGT1 GAT TTG CCG CAG5 ACT CAT AGC TTG GGC10 AAC CGA AGA GCA CTC15 ATC CTG TTG GCC CAA20 ATG GGT CGC ATT TCC25 CTG TTC TCC TCC CTT30 AAA GAC CGC CAC GAT35 TTC GGG TTT CCA CAG40 GAA GAG TTT GAC GGC45 AAT CAG TTC CAA AAG50 ACC CAG GCA ATC TCG55 GTT CTG CAT GAG ATG60 ATT CAG CAA ATC TTC65 AAC CTC TTC TCC ACT70 AAA GAC TCT TCG GCT75 GCT TGG AAC GAA TCC80 TTG CTT GAT AAA TTC85 TCC ATC GAA CTC AAT90 CAG CAA CTC AAC GAT95 ATC GAA GCT TCC GTT100 ATC CAG GAG GTA GGC105 GTT GAA GAG ACT CCG110 CTT ATG AAT GTT GAG115 TCT ATC CTC GCT GTA120 TCT AAG TAT TTT CAG125 CCA ATT ACC CTC TAC130 CTC AGT GAG AAC AAA135 TAT TCA CCG TGT GCG140 TGG GAA ATT CTC AGA145 GCC GAA ATC ATG CGT150 TCC CTG ACT CTT CTG155 ACG AAC CTC CAG GAA160 CGC CTC CGT AAT AAA165 GAC 6. 5 ATG TGT1 GAT TTG CCG CAG5 ACT CAT AGC TTG GGC10 AAC CCA AGA GCA CTC15 ATC CTC TTG GCC CAA20 ATC GGT CGC ATT TCC25 CTC TTC TCC TCC CTT30 AAA GAC CGC CAC GAT35 TTC GGG TTT CCA CAG40 GAA GAG TTT GAC GGC45 AAT CAG TTC CAA AAG50 ACC CAG GCA ATC TCG55 GTT CTC CAT GAG ATG60 ATT CAG CAA ATC TTC65 AAC CTG TTC TCC ACT70 AAA GAC TCT TCC GCT75 GCT TGG AAC GAA TCC80 TTG CTT GAT AAA TTC85 TCC ATC GAA CTC TAT90 CAG CAA CTC AAC AAC95 CTT GAA GCT TCC GTT100 ATC CAG GAT GTA GGC105 ATC GAA GAG ACT CCC110 CTT ATC AAT GTT GAG115 TCT ATC CTG GCT GTA120 TCT AAG TAT TTT CAG125 CCA ATT ACC CTC TAC130 CTC AGT GAG AAG AAA135 TAT TCA CCG RTI ID 24.9 TCI GT GCG140 TGG GAA ATT CTC AGA145 GCC GAA ATC ATC CGT, z 150 TCC CTC ACT CTT CTG155 ACG AAC CTC CAG GAA160 CGC CTC CGT AAT AAA165 GAC Deoxynucleotide sequences of the invention encoding for hybrid lymphoblastoid leukocyte human interferons comprising a hybrid promotor region, an operator region, a ribosome binding site and a polypeptide coding region coding for a polypeptide of 166 amino acids and optionally coding for an additional methionine attached to the ordinary first amino acid at the N terminus of said polypeptide, and a termination region are represented by a plus strand of the following formula 5 L O Q ATG W X P Y B Z E wherein L consists essentially of the promoter region 0 consists essentially of the operator region Q is a portion of a ribosome binding site and is A A A A A T T A A G G A G G A T C A C T, T T A TT A C C C A A C T T G A G G A A T T T A T A, or C G G C C C C T T A C T T G A G G A T A A A T T W is TGT or TGC X consists essentially of deoxynucleotides encoding for amino acids 2 to 46 of human lymphoblastoid interferon LbIF Y consists essentially of deoxynucleotides encoding for amino acids 60 to 85 of said LbIF Z consists essentially of deoxynucleotides encoding for amino acids 112 to 166 of said LbIF P consists essentially of deoxynucleotides encoding for amino acids 47 to 59 of human alpha C , aF , or aI type leukocyte interferon LeIF and isCAG TTC CAA AAG50 GCT CAG GCA ATC TCG55 GTT CTG CAT GAG, or CAG TTC CAA AAG50 ACC CAG GCA ATC TCG55 GTT CTG CAT GAG B consists essentially of deoxynucleotides encoding for amino acids 86 to 111 of said a C , a F , or aI type LeIF and is TCC ACT GAA CTC TAT90 CAG CAA CTG AACGAT95 CTT GAA GCT TCC GTT100 ATC CAG GAG GTA GGC105 GTT GAA GAG ACT CCG110 CTT, TCC ATC GAA CTC AAT90 CAG CAA CTG AAC GAT95 ATG GAA GCT TGCGTT100 ATC CAG GAG GTA GGC105 GTT GAA GAG ACT CCG110 CTT, or TCC ATCGAA CTC TAT90 CAG CAA CTG AAC AAC95 CTT GAA GCT TGC GTT ATC CAG GATGTA GGC105 ATG GAA GAG ACT CCG110 CTT E consists essentially of the termination region and transcriptional and translational equivalents, including single or multiple base substitutions, insertions and inversions and substantial fragments thereof permitting the expression of said hybrid interferon or biologically active fragments thereof.Consistent with this formula are the following constructions 1. 5 A A T T C A C C G T G C G T G T T G A C T A T T T T A C C T C T G G C G G T G A T A A T G A A A T C A A T T G T G A G C G G A T A A C A A T T A A A A A T T A A G G A G G A T C A C T ATG TGT1 GAT TTG CCG CAG5 ACT CAT AGC TTG GGC10 AAC CGA AGA GCA CTC15 ATC CTG TTG GCC CAA20 ATG GGT CGC ATT TCC25 CTG TTC TCG TCC CTT30 AAA GAC CGC CAC GAT35 TTC GGG TTT CCA CAG40 GAA GAG TTT GAC GGC45 AAT CAG TTC CAA AAG50 GCT CAG GCA ATC TCG55 GTT CTG CAT GAG ATG60 ATT CAG CAA ATC TTC65 AAC CTG TTC TCC ACT70 AAA GAC TCT TCG GCT75 GCT TGG AAC GAA TCC80 TTG CTT GAT AAA TTC85 TCC ACT GAA CTC TAT90 CAG CAA CTG RTI ID 25.19 AAC GAT95 CTT GAA GCT TGC GTT100 ATC CAG GAG GTA GCC105 GTT GAA GAG ACT CCG110 CTT ATG112 AAT GTT GAG115 TCT ATC CTG GCT GTA120 TCT AAG TAT TTT CAG125 CGA ATT ACC CTC TAC130 CTC AGT GAG AAG AAA 135 TAT TCA CCG TGT GCG140 TGG GAA ATT GTG AGA145 GCC GAA ATC ATG CGT150 TCC CTG ACT CTT CTG155 ACG AAC CTC CAG GAA160 CGC CTG CGT AAT AAA165 GAC T A A T A G 2. 5 A A T T C A C C G T G C G T G T T G A C T A T T T T A C C T C T G G C G G T G A T A A T G A A A T C A A T T G T G A G C G G A T A A C A A T T A T T A C C C A A C T T G A G G A A T T T A T A ATG TGT1 GAT TTG CCG CAG5 ACT CAT AGC TTG 10AAC CGA AGA GCA CTC15 ATC CTG TTG GCC CAA20 ATG GGT CGC ATT TCC25 CTG TTC TCG TGC CTT30 AAA GAC CGC CAC GAT35 TTC GGG TTT CCA CAG40 GAA GAG TTT GAC GGC45 AAT CAG TTC CAA AAG50 GCT CAG GCA ATC TCG55 GTT CTG CAT GAG ATG60 ATT CAG CAA ATC TTC65 AAC CTG TTC TCC ACT70 AAA GAC TCT TCG GCT75 GCT TGG AAC GAA TCC80 TTG CTT GAT AAA TTC85 TCC ACT GAA CTC TAT90 CAG CAA CTG AAC GAT95 CTT GAA GCT TCC GTT100 ATC CAG GAG GTA GGC105 GTT GAA GAG ACT CCG110 CTT ATG112 AAT GTT GAG115 TCT ATC CTG GCT GTA120 TCT AAG TAT TTT CAG125 CGA ATT ACC CTC TAC130 CTC AGT GAG AAG AAA135 TAT TCA CCG TGT GCG140 TGG GAA ATT GTG AGA145 GCC GAA ATC ATG CGT150 TCC CTG ACT CTT CTG155 ACG AAC CTC CAG GAA160 CGC CTG CGT AAT AAA 165 GAC T A A T A G 3. 5 A A T T C A C C G T G C G T G T T G A C T A T T T T A C C T C T G G C G G T G A T A A T G A A A T C A A T T G T G A GC G G A T A A O A A T T C G G O C C C T T A C T T G A G G A T A A A T T ATG TGT1 GAT TTG CCG CAG5 ACT CAT AGC TTG GGC10 AAC CGA AGA GCA CTC15 ATC CTG TTG GCC CAA20 ATG GGT CGC ATT TCC25 CTG TTC TCG TCC CTT30 AAA GAC CGC CAC GAT35 TTC GGG TTT CCA CAG40 GAA GAG TTT GAC GGC45 AAT CAG TTC CAA AAG50 GCT CAG GCA ATC TCG55 GTT CTG CAT GAG ATG60 ATT CAG CAA ATC TTC65 AAC CTG TTC TCC ACT70 AAA GAC TCT TCG GCT75 GCT TGG AAC GAA TCC80 TTG CTT GAT AAA TTC85 TCC ACT GAA CTC TAT90 CAG CAA CTG AAC GAT95 CTT GAA GCT TGC GTT100 ATC CAG GAG GTA GGC105 GTT GAA GAG ACT CCG110 CTT ATG112 AAT GTT GAG115 TCT ATC CTG GCT GTA120 TCT AAG TAT TTT CAG125 CGA ATT ACC CTC TAC130 CTC AGT GAG AAG AAA135 TAT TCA CCG TGT GCG140 TGG GAA ATT GTG AGA145 GCC GAA ATC ATC CGT150 TCC CTG ACT CTT CTG155 ACG AAC CTC CAG GAA160 CGC CTG CGT AAT AAA165 GAC T A A T A G 4. 5 A A T T C A C C G T G C G T G T T G A C T A T T T T A C C T O T C C C G C T C A T A A T C A A A T C A A T T G T C A G C G G A T A A C A A T T A A A A A T T A A G G A G G A T C A C T ATG TGT1 GAT TTG CCG CAG5 ACT CAT AGC TTG GGC10 AAC CGA AGA GCA CTC15 ATC CTG TTG GCC CAA20 ATG GGT CGC ATT TCC25 CTG TTC TCG TGC CTT30 AAA GAC CGC CAC GAT35 TTC GGG TTT CCA CAG40 GAA GAG TTT GAC GGC45 AAT CAG TTC CAA AAG50 GCT CAG GCA ATC TCG55 GTT CTG CAT GAG ATG60 ATT CAG CAA ATC TTC65 AAC CTG TTC TCC ACT70 AAA GAC TCT TCG GCT75 GCT TGG AAC CAA TCC80 TTG CTT GAT AAA TTC85 TCC ATC GAA CTC AAT90 CAG CAA CTC AAC GAT95 ATG GAA GCT TCC GTT10 ATC CAG GAG GTA GGC GTT GAA GAG ACT CCG110 CTT ATG AAT GTT GAG115 TCT ATC CTG GCT GTA120 TCT AAG TAT TTT CAG125 CGA ATT ACC CTC TAC130 CTC AGT GAG AAG AAA135 TAT TCA CCG TGT GCG140 TGG GAA ATT GTG AGA145 GCC GAA ATC ATC CGT150 TCC CTG ACT CTT CTG155 ACG AAC CTC CAG GAA160 CGC CTG CGT AAT AAA165 GAC T A A T A G 5. 5 A A T T C A C C G T G C G T G T T G A C T A T T T T A C C T C T G G C G G T G A T A A T G A A A T C A G C GA T A A C A A T T A T T A C C C A A O T T G A G C A A T T T A T A ATC TGT1 GAT TTG CCG CAG5 ACT CAT AGC TTG GGC10 AAC CGA AGA GCA CTC15 ATC CTG TTG GCC CAA20 ATG GGT CGC ATT TCC25 CTG TTC TCG TGC CTT30 AAA GAC CGC CAC GAT35 TTC GGG TTT CCA CAG40 GAA GAG TTT GAC GGC45 AAT CAG TTC CAA AAG50 GCT CAG GCA ATC TCG55 GTT CTC CAT GAG ATG60 ATT CAG CM ATC TTC65 AAC CTG TTC TCC ACT70 AAA GAC TCT TCG GCT75 GCT TGG AAC GAA TCC80 TTG CTT GAT AAA TTC85 TCC ATC GAA CTC AAT90 CAG CAA CTG AAC GAT95 ATG GAA GCT TCC GTT100 ATC CAG GAG GTA GGC105 GTT GAA GAG ACT CCG110 CTT ATG AAT GTT GAG115 TCT ATC CTG GCT GTA120 TCT AAG TAT TTT CAG125 CGA ATT ACC CTC TAC130 CTC AGT GAG AAG AAA 135 TAT TCA CCG TGT GCG140 TGG GAA ATT GTG AGA145 GCC GAA ATC ATG CGT150 TCC CTG ACT CTT CTG155 ACG AAC CTC CAG GAA160 CGC CTG CGT AAT AAA165 GAC T A A T A G 6. 5 A A T T C A C C G T G C G T G T T G A C T A T T T T A C C T C T G G C G G T G A T A A T G A A A T C A A T T G T G A G C G G A T A A C A A T T C G G C C C C T T A C T T G A G G A T A A A T T ATG TGT1 GAT TTG CCG CAG5 ACT CAT AGC TTG GGClo AAC CGA AGA GCA CTC15 ATC CTG TTG GCC CAA20 ATG GGT CGC ATT TCC25 CTG TTC TCG TGC CTT30 AAA GAC CGC CAC GAT35 TTC GGG TTT CCA CAG40 GAA GAG TTT GAC GGC45 AAT CAG TTC CAA AAG50 GCT CAG GCA ATC TCG55 GTT CTG CAT GAG ATG60 ATT CAG CAA ATC TTC65 AAC CTG TTC TCC ACT70 AAA GAC TCT TCG GCT75 GCT TGG AAC GAA TCC80 TTG CTT GAT AAA TTC85 TCC ATC GAA CTC AAT90 CAG CAA CTG AAC GAT95 ATG GAA GCT TCC GTT100 ATC CAG GAG GTA GGC105 GTT GAA GAG ACT CCG110 CTT ATG AAT GTT GAG115 TCT ATC CTG GCT GTA120 TCT AAG TAT TTT CAG125 CGA ATT ACC CTC TAC130 CTC AGT GAG AAG AAA135 TAT TCA CCG TGT GCG140 TGG GAA ATT GTG AGA145 GCC GAA ATC ATG CGT150 TCC CTG ACT CTT CTG155 ACG AAC CTC CAG GAA160 CGC CTG CGT AAT AAA165 GAC T A A T A G 7. 5 A A T T C A C C G TAC G T G T T G A C T A T T T T A C C T C T G G C G G T G A T A A T G A A A T C A A T T G T G A G C G G A T A A C A A T T A A A A A T T A A G G A G G A T C A C T ATG TGT1 GAT TTG CCG CAG5 ACT CAT AGC TTG GGC1o AAC CGA AGA GCA CTC15 ATC CTG TTG GCC CAA20 ATG GGT CGC ATT TCC25 CTG TTC TCG TGC CTT30 AAA GAC CGC CAC GAT35 TTC GGG TTT CCA CAG40 GAA GAG TTT GAC GGC45 AAT46 CAG TTC CAA AAG50 GCT CAG GCA ATC TCG55 GTT CTG CAT GAG ATG60 ATT CAG CAA ATC TTC65 AAC CTG TTC TCC ACT70 AAA GAC TCT TCG GCT75 GCT TGG AAC GAA TCC80 TTG CTT GAT AAA TTC85 TCC ATC GAA CTC TAT90 CAG CAA CTG AAC AAC95 CTT GAA GCT TGC GTT100 ATC CAG GAT GTA GGC105 ATG GAA GAG ACT CCG110 CTT ATG AAT GTT GAG115 TCT ATC CTG GCT GTA120 TCT AAG TAT TTT CAG125 CGA ATT ACC CTC TAC130 CTC AGT GAG AAG AAA135 TAT TCA CCG TGT GCG140 TGG GAA ATT GTG AGA145 GCC GAA ATC ATG CGT150 TCC CTG ACT CTT CTG155 ACG AAC CTC GAG GAA160 CGC CTG CGT AAT AAA165 GAC T A A T A G 8. 5 A A T T C A C C G T G C G T G T T G A C T A T T T T A C C T O T G G O G G T G A T A A T G A A A T O A A T T G T G A G C G G A T A A C A A T T A T T A C C C A A C T T G A G G A A T T T A T A ATG TGT1 GAT TTG CCG CAG5 ACT CAT AGC TTG GGClo AAC CGA AGA GCA CTC15 ATC CTG TTG GCC CAA20 ATG GGT CGC ATT TCC25 CTG TTC TCG TGC CTT30 AAA GAC CGC CAC GAT35 TTC GGG TTT CCA CAG40 GAA GAG TTT GAC GGC45 AAT46 CAG TTC CAA AAG50 GCT CAG GCA ATC TCG55 GTT CTG CAT GAG ATG60 ATT CAG CAA ATC TTC65 AAC CTG TTC TCC ACT70 AAA GAC TCT TCG GCT75 GCT TGG AAC GAA TCC80 TTG CTT GAT AAA TTC85 TCC ATC GAA CTC TAT90 CAG CAA CTG AAC AAC95 CTT GAA GCT TGC GTT100 ATC CAG GAT GTA GGC 105 ATG GAA GAG ACT CCG110 CTT ATG AAT GTT GAG115 TCT ATC CTG GCT GTA120 TCT AAG TAT TTT CAG125 CGA ATT ACC CTC TAC130 CTC AGT GAG AAG AAA 135 TAT TCA CCG TGT GCG140 TGG GAA ATT GTG AGA145 GCC GAA ATC ATG CGT150 TCC CTG ACT CTT CTG155 ACG AAC CTC CAG GAA160 CGC CTG CGT AAT AAA165 GAC T A A T A G 9. 5 A A T T C A C C G T G C G T G T T G A C T A T T T T A C C T C T G G C G G T G A T A A T G A A A T C A A T T G T G A G C G G A T A A C A A T T C G G C C C C T T A C T T G A G G A T A A A T T ATG TGT1 GAT TTG CCG CAG5 ACT CAT AGC TTG GGC10 AAC CGA AGA GCA CTC15 ATC CTG TTG GCC CAA20 ATG GGT CGC ATT TCC25 CTG TTC TCG TGC CTT30 AAA GAC CGC CAC GAT35 TTC GGG TTT CCA CAG40 GAA GAG TTT GAC GGC45 AAT CAG TTC CAA AAG50 GCT GAG GCA ATC TCG55 GTT CTG CAT GAG ATG60 ATT CAG CAA ATC TTC65 AAC CTG TTC TCC RTI ID 29.13 ACT70 AAA GAC TCT TCG GCT75 GCT TGG AAC GAA TCC80 TTG CTT GAT AAA TTC85 TCC ATC GAA CTC TAT90 CAG CAA CTG AAC AAC95 CTT GAA GCT TGC GTT100 ATC CAG GAT GTA GGC105 ATG GAA GAG ACT CCG110 CTT ACG AAT GTT GAG115 TCT ATC CTG GCT GTA120 TCT AAG TAT TTT CAG125 CGA ATT ACC CTC TAC130 CTC AGT GAG AAG AAA135 TAT TCA CCG TGT GCG140 TGG GAA ATT GTG AGA145 GCC GAA ATC ATG CGT150 TCC CTG ACT CTT CTG155 ACG AAC CTC CAG GAA160 CGC CTG CGT AAT AAA165 GAC T A A T A G 10. 5 A A T T O A C C G T G C G T G T T G A O T A T T T T A O C T C T G G C G G T G A T A A T G A A A T C A A T T G T G A G C G G A T A A C A A T T A A A A A T T A A G G A G G A T C A C T ATG TGT ATG TGT1 GAT TTG CCG CAG5 ACT CAT AGC TTG GGC10 AAC CGA AGA GCA RTI ID 30.4 CTC15 ATC CTG TTG GCC CAA20 ATG GGT CGC ATT TCC25 CTG TTC TCG TGC CTT30 AAA GAC CGC CAC GAT35 TTC GGG TTT CCA CAG40 GAA GAG TTT GAC GGC45 AAT CAG TTC CAA AAG50 ACC CAG GCA ATC TCG55 GTT CTG CAT GAG ATG60 ATT CAG CAA ATC TTC65 AAC CTG TTC TCC ACT70 AAA GAC TCT TCG GCT75 GCT TGG AAC GAA TCC80 TTG CTT GAT AAA TTC85 ACT GAA CTC TAT90 CAG CAA CTG AAC GAT95 CTT GAA GCT TGC GTT100 ATC CAG GAG GTA GGC105 GTT GAA GAG ACT CCG110 CTT ATG AAT GTT GAG115 TCT ATC CTG GCT GTA120 TCT AAG TAT TTT CAG125 CGA ATT ACC CTC TAC130 CTC AGT GAG AAG AAA135 TAT TCA CCG TGT GCG140 TGG GAA ATT GTG AGA145 GCC GAA ATC ATG CGT150 TCC CTG ACT CTT CTG155 ACG AAC CTC CAG GAA160 CGC CTG CGT AAT RTI ID 30.12 AAA165 GAC T A A T A G 11. 5 A A T T C A C C G T G C G T G T T G A C T A T T T T A C C T C T G G C G G T G A T A A T G A A A T C A A T T G T G A G C G G A T A A C A A T T A T T A C C C A A C T T G A G G A A T T T A T A ATG TGT1 GAT TTG CCG CAG5 ACT CAT AGC TTG GGC10 AAC CGA AGA GCA CTC15 ATC CTG TTG GCC CAA20 ATG GGT CGC ATT TCC25 CTG TTC TCG TGC CTT30 AAA GAC CGC CAC GAT35 TTC GGG TTT CCA CAG40 GAA GAG TTT GAC GGC45 AAT GAG TTC CAA AAG50 ACC GAG GCA ATC TCG55 GTT CTG CAT GAG ATG60 ATT CAG CAA ATC TTC65 AAC CTG TTC TCC ACT70 AAA GAC TCT TCG GCT75 GCT TGG AAC GAA TCC80 TTG CTT GAT AAA TTC85 ACT GAA CTC TAT90 CAG CAA CTG AAC GAT95 CTT GAA GCT TGC GTT100 ATC CAG GAG GTA GGC105 GTT GAA GAG ACT CCG110 CTT RTI ID 30.18 ATG112 AAT GTT GAG115 TCT ATC CTG GCT GTA120 TCT AAG TAT TTT CAG125 CGA ATT ACC CTC TAC130 CTC AGT GAG AAG AAA135 TAT TCA CCG TGT GCG140 TGG GAA ATT GTG AGA145 GCC GAA ATC ATG CGT150 TCC CTG ACT CTT CTG155 ACG AAC CTC CAG GAA160 CGC CTG CGT AAT AAA165 GAC T A A T A G 12. 5 A A T T C A C C G T G C G T G T T G A C T A T T T T A C C T C T G G C G G T G A T A A T G A A A T C A A T T G T G A G C G G A T A A C A A T T C G G C C C C T T A C T T G A G G A T A A A T T ATG TGT1 GAT TTG CCG CAG5 ACT CAT AGC TTG GGC10 AAC CGA AGA GCA CTC15 ATC CTG TTG GCC CAA20 ATG GGT CGC ATT TCC25 CTG TTC TCG TGC CTT30 AAA GAC CGC CAC GAT35 TTC GGG TTT CCA CAG40 GAA GAG TTT GAC GGC45 AAT CAG TTC CAA AAG50 ACC CAG GCA ATC TCG55 GTT CTG CAT GAG ATG60 ATT CAG CAA ATC TTC65 AAC CTG TTC TCC ACT70 AAA GAC TCT TCG GCT75 GCT TGG AAC GAA TCC80 TTG CTT GAT AAA TTC85 ACT GAA CTC TAT90 CAG CAA CTG AAC GATg5 CTT GAA GCT TGC GTT100 ATC CAG GAG GTA GGC105 GTT GAA GAG ACT CCG110 CTT ATG112 AAT GTT GAG115 TCT ATC CTG GCT GTA120 TCT AAG TAT TTT CAG125 CGA ATT ACC CTC TAC130 CTC AGT GAG AAG AAA135 TAT TCA CCG TGT GCG140 TGG GAA ATT GTG AGA145 GCC GAA ATC ATG GGT150 TCC CTG ACT CTT CTG155 ACG AAC CTC CAG GAA160 CGC CTG CGT AAT AAA165 GAO T A A T A G 13. 5 A A T T C A C C G T G C G T G T T G A C T A T T T T A G C T C T G G C G G T G A T A A T G A A A T C A A T T G T G A G C G G A T A A C A A T T A A A A A T T A A G G A G G A T C A C T ATG TGT ATG TGT1 GAT TTG CCG CAG5 ACT CAT AGC TTG GGC10 AAC CGA AGA GCA CTC15 ATC CTG TTG GCC CAA20 ATG GGT CGC ATT TCC25 CTG TTC TCG TGC CTT30 AAA GAC CGC CAC GAT35 TTC GGG TTT CCA CAG40 GAA GAG TTT GAC GGC45 AAT GAG TTC CAA AAG50 ACC CAG GCA ATC TCG55 CTT CTG CAT GAG ATG60 ATT CAG CAA ATC TTC65 AAC CTG TTC TCC ACT70 AAA GAC TCT TCG GCT75 GCT TGG AAC GAA TCC80 TTG CTT GAT AAA TTC85 TCC ATC GAA CTC AAT90 CAG CAA CTG AAC GAT95 ATG GAA GCT TGC GTT100 ATC CAG GAG GTA GGC105 GTT GAA GAG ACT CCG110 CTT ATG AAT GTT GAG115 TCT ATC CTG GCT GTA120 TCT AAG TAT TTT CAG125 CGA ATT ACC CTC TAC130 CTC AGT GAG AAG AAA 135 TAT TCA CCG TGT GCG140 TGG GAA ATT GTG AGA145 GCC GAA ATC ATG CGT150 TCC CTG ACT CTT CTG155 ACG AAC CTC CAG GAA160 CGC CTG CGT AAT AAA165 GAC T A A T A G 14. 5 A A T T C A C C G T G C G T G T T G A C T A T T T T A C C T C T G G C G G T G A T A A T G A A A T C A A T T G T G A G C GG A T A A O A A T T A T T A C O O A A O T T G A GG A A T T T A T A ATG TGT1 GAT TTG CCG CAG5 ACT CAT AGC TTG GGC10 AAC CGA AGA GCA CTC15 ATC CTG TTG GCC CAA20 ATG GGT CGC ATT TCC25 CTG TTC TCG TGC CTT30 AAA GAC CGC CAC GAT35 TTC GGG TTT CCA CAG40 GAA GAG TTT GAC GGC45 AAT CAG TTC CAA AAG50 ACC GAG GCA ATC TCG55 GTT CTG CAT GAG ATG60 ATT CAG CAA ATC TTC65 AAC CTG TTC TCC ACT70 AAA GAC TCT TCG GCT75 GCT TGG AAC GAA TCC80 TTG CTT GAT AAA TTC85 TCC ATC GAA CTC AAT90 CAG CAA CTG AAC GAT95 ATG GAA GCT TCC GTT100 ATC CAG GAG GTA GGC105 GTT GAA GAG ACT CCG110 CTT ATG AAT GTT GAG115 TCT ATC CTG GCT GTA120 TCT AAG TAT TTT CAG125 CGA ATT ACC CTC TAC130 CTC AGT GAG AAG AAA135 TAT TCA CCG TGT GCG140 TGG GAA ATT GTG AGA145 GCC GAA ATC ATG CGT150 TCC CTG ACT CTT CTG155 ACG AAC CTC CAG GAA160 CGC CTG CGT AAT AAA165 GAC T A A T A G 15. 5 A A T T C A C C G T G C G T G T T G A C T A T T T T A C C T C T G G C G G T G A T A A T G A A A T C A A T T G T G A G C G G A T A A C A A T T C G G C C C C T T A C T T G A G G A T A A A T T ATG TGT1 GAT TTG CCG CAG5 ACT CAT AGC TTG GGC10 AAC CGA AGA GCA CTC15 ATC CTG TTG GCC CAA20 ATG GGT CGC ATT TCC25 CTG TTO TCG TGC CTT30 AAA GAC CGC CAC GAT35 TTC GGG CCA CAG40 GAA GAG TTT GAC GGC45 AAT CAG TTC CAA AAG50 ACC CAG GCA ATC TCG55 GTT CTG CAT GAG ATG60 ATT GAG CAA ATC TTC65 AAC CTG TTC TCC ACT70 AAA GAC TCT TCG GCT75 GCT TGG AAC GAA TCC80 TTG CTT GAT AAA TTC85 TCC ATC GAA CTC AAT90 CAG CAA CTG AAC GAT95 ATG GAA GCT TGC GTT100 ATC CAG GAG GTA GGC105 GTT GAA GAG ACT CCG110 CTT ATG AAT GTT GAG1l5 TCT ATC CTG GCT GTA120 TCT AAG TAT TTT CAG125 CGA ATT ACC CTC TAC130 CTC AGT GAG AAG AAA135 TAT TCA CCG TGT GCG140 TGG GAA ATT GTG AGA145 GCC GAA ATC ATG CGT150 TCC CTG ACT CTT CTG155 ACG AAC CTC CAG GAA160 CGC CTG CGT AAT AAA 165 GAC T A A T A G 16. 5 A A T T O A O C G T G C G T G T T G A O T A T T T T A O C T C T G G C G G T G A T A A T G A A T O A A T T G T G A G C G G A T A A C A A T T A A A A A T T A G G A T C A C T ATG TGT ATG TGT1 GAT TTG CCG CAG5 ACT CAT AGC TTG GGC10 AAC CGA AGA GCA CTC15 ATC CTG TTG GCC CAA20 ATG GGT CGC ATT TCC25 CTG TTC TCG TGC CTT30 AAA GAC CGC CAC GAT35 TTC GGG TTT CCA CAG40 GAA GAG TTT GAC GGC45 AAT CAG TTC CAA AAG50 ACC CAG GCA ATC TCG55 GTT CTG CAT GAG ATG60 ATT CAG CAA ATC TTC65 AAC CTG TTC TCC ACT70 AAA GAC TCT TCG GCT75 GCT TGG AAC GAA TCC80 TTG CTT GAT AAA TTC85 TCC ATC GAA CTC TAT90 CAG CAA CTG AAC AAC95 CTT GAA GCT TGC GTT100 ATC CAG GAT GTA GGC105 ATG GAA GAG ACT CCG110 CTT ATG AAT GTT CAG115 TCT ATC CTG GCT GTA120 TCT AAG TAT TTT CAG125 CGA ATT ACC CTC TAC130 CTC AGT GAG AAG AAA135 TAT TCA CCG TGT GCG140 TGG GAA ATT GTG AGA145 GCC GAA ATC ATG CGT150 TCC CTG ACT CTT CTG155 ACG AAC CTC CAG GAA160 CGC CTG CGT AAT AAA 165 GAC T A A T A G 17. 5 A A T T C A C C G T G C G T G T T G A C T A T T T T A C C T C T G G C G G T G A T A A T G A A A T C A A T T G T G A G O G G A T A A C A A T T A T T A O C O A A O T T C A C C A A T T T A T A ATG TGT1 GAT TTG CCG CAG5 ACT CAT AGC TTG GGC10 AAC CGA AGA GCA CTC15 ATC CTG TTG GCC CAA20 ATG GGT CGC ATT TCC25 CTG TTC TCG TGC CTT30 AAA GAC CGC CAC GAT35 TTC GGG TTT CCA CAG40 GAA GAG TTT GAC GGC45 AAT CAG TTC CAA AAG50 ACC GAG GCA ATC TCG55 GTT CTG CAT GAG ATG60 ATT CAG CAA ATC TTC65 AAC CTG TTC TCC ACT70 AAA GAC TCT TCG GCT75 GCT TGG AAC GAA TCC80 TTG CTT GAT AAA TTC85 TCC ATC GAA CTC TAT90 GAG CAA CTC AAC AAC95 CTT GAA GCT TGC GTT100 ATC CAG GAT GTA GGC105 ATG GAA GAG ACT CCG110 CTT ATC AAT GTT GAG115 TCT ATC CTC GCT GTA120 TCT AAG TAT TTT CAG125 CGA ATT ACC CTC TAC130 CTC AGT GAG AAG AAA135 TAT TCA CCG TGT GCG140 TGG GAA ATT CTG AGA145 GCC GAA ATC ATG CGT150 TCC CTG ACT CTT CTG155 ACG AAC CTC CAG GAA160 CGC CTG CGT AAT AAA165 GAC T A A T A G 18. 5 A A T T O A O C G T 0 O G T G T T G A O T A T T T T A C C T O T G G O G G T G A T A A T G A A A T O A A T T G T 0 A G C G G A T A A C A A T T C G G C C C C T T A C T T G A G G A T A A A T T ATG TGT1 GAT TTG CCG CAG5 ACT CAT AGC TTG GGC10 AAC CGA AGA GCA CTC15 ATC CTG TTG GCC CAA20 ATG GGT CGC ATT TCC25 CTG TTC TCG TGC OTT 30 AAA GAC CGC CAC GAT35 TTC GGG TTT CCA CAG40 GAA GAG TTT GAC GGC45 AAT CAG TTC CAA AAG50 ACC CAG GCA ATC TCG55 GTT CTG CAT GAG ATG60 ATT CAG CAA ATC TTC65 AAC CTG TTC TCC ACT70 AAA GAC TCT TCG GCT75 GCT TGG AAC GAA TCC80 TTG CTT GAT AAA TTC85 TCC ATC GAA CTC TAT90 CAG CAA CTG AAC AAC95 CTT GAA GCT TGC GTT100 ATC CAG GAT GTA GGC105 ATG GAA GAG ACT CCG110 CTT ATG AAT GTT GAG115 TCT ATC CTG GCT GTA120 TCT AAG TAT TTT CAG125 CGA ATT ACC CTC TAC130 CTC AGT GAG AAG AAA135 TAT TCA CCG TGT GCG140 TGG GAA ATT GTG AGA145 GCC GAA ATC ATG CGT150 TCC CTG ACT CTT CTG155 ACG AAC CTC GAG GAA160 CGC CTG CGT AAT AAA165 GAC T A A T A G Recombinant DNA molecules comprising DNA sequences according to any of the above constructions can be used to transform various hosts Escherichia coli, i.e., E. coli W3110, E. coli 7118, E. coli 7902, etc. Bacillus subtilis other bacteria yeasts or other fungi mouse or other animal or plant hosts and including human tissue cells . Polypetides or fragments and derivatives thereof displaying immunological or biological activity of hybrid lymphoblastoid leukocyte human interferons can be produced by such transformed hosts. The lymphoblastoid hybrid interferon was derived from two proteins. Approximately 76 was based on the amino acid sequence of a lymphoblastoid interferon produced by induction of Namalva cells withNewcastle disease virus, strain B1. The remainder was from the C subtype of the leukocyte interferons. This hybrid protein was as active biologically as known natural interferons. Already many interferons have been identified through cDNA cloning and sequencing.Additionally two natural hybrids formed from the A and D subtypes Week, P. K., et al. 1981 Nucleic Acids Res. 9 6153 6166 and a consensus interferon have been synthesized and shown to be biologically active. Alton, K. et al. Production, Characterization and Biological Efforts of Recombinant DNA Derived from Human IFN a andIFN y Analogs. In The Biology of the Interferon System 1983 E.DeMaeyer and H. Schellekens, eds. Elsevier Science Publishers B.V. .All these results suggest that the a interferons are a rather diverse family of proteins. Additional modifications may therefore be possible in order to maximize biological activity, maximize yield, or perhaps simplify purification procedures. These modifications can best be accomplished if the gene is completely synthetic, cloned in sections, and readily accessible via various restriction enzymes.This same procedure will become generally useful for many different proteins. Questions of structure function relationships can most rapidly be approached if a protein can be readily modified through amino acid changes. This flexibility in turn requires synthesis of at least major sections of a protein gene perhaps between convenient, natural restriction sites . Polypeptide compositions of the invention comprising 166 amino acids and optionally having an additional methionine attached to the ordinary first amino acid at the N terminus have sequences represented by the following formula H2N met cys X P Y B Z COOH wherein X consists essentially of amino acids 2 to 46 of human lymphoblastoid interferon LbIF Y consists essentially of amino acids 60 to 85 of said LbIF Z consists essentially of amino acids 112 to 166 of said LbIF P consists essentially of amino acids 47 to 59 of human 47 aC , aF , or oI type leukocyte interferon LeIF and is gln phe gln lys ala gln ala ileu ser5 val leu his glu59, or gln47 phe gln lys50 thr gln ala ileu ser55 val leu his glu59 and B consists essentially of amino acids 86 to 111 of human aC , aF , or aI type leukocyte interferon LeIF and is ser86 thr glu leu tyr90 gln gln leu asn asp95 leu glu ala cys val100 ileu gln glu val gly105 val gluglu thr pro110 leu, ser86 thr glu leu asn90 gln gln leu asn asp95met glu ala cys val100 ileu gln glu val gly105 val glu glu thr pro110 leu, or ser86 thr glu leu tyr90 gln gln leu asn asn95 leu glu ala cys val100 ileu gln glu val gly105 met glu glu thr pro110 leu, and includ ing substantial polypeptide fragments and or derivatives thereof having biological or antiviral activity of human interferon. Consistent therewith are the following hybrid interferon polypeptide constructions 1. met cys asp leu pro gln5 thr his ser leu gly10 asn arg arg ala veu15 ileu leu leu ala gln20 met gly arg ileu ser25 leu 30 35 phe ser cys ieu lys asp arg his asp phe gly phe pro gln40 glu glu phe asp gly45 asn gln phe gln lys50 ala gln 55 60 ala ileu ser val leu his glu met ileu gln gln ileu phe05 asn leu phe ser thr10 lys asp ser ser ala15 ala trp asn glu ser80 leu leu asp lys phe85 ser thr glu leu tyr90 gln gln leu asn asp95 leu glu ala cys val100 ileu gln glu val g1y105 val glu glu thr pro 110 leu met asn val glu1 15 ser ileu leu ala val120 ser lys tyr phe gln125 arg ileu thr leu tyr130 leu ser glu lys lys135 tyr ser pro cys ala140 trp glu ileu val arg145 ala glu ileu let arg150 ser leu thr leu leu155 thr asn leu gln glu160 arg leu arg asn lys165 asp 2. met cys asp leu pro gln5 thr his ser leu gly10 asn arg arg ala leu15 ileu leu leu ala gln20 met gly arg ileu ser25 leu phe ser cys leu30 lys asp arg his asp35 phe gly phe pro gln40 glu glu phe asp gly45 asn gln phe gln lys50 ala gln ala ileu ser55 val leu his glu met60 ileu gln gln ileu phe65 asn leu phe ser thr70 lys asp ser ser ala75 ala trp asn glu ser80 leu leu asp lys phe85 ser thr glu leu asn90 95 100 gln gln leu asn asp met glu ala cys val ileu gln glu val gly105 val glu glu thr pro110 leu met asn val glu115 ser ileu leu ala val120 ser lys tyr phe gln125 arg ileu thr leu tyr130 leu ser glu lys lys135 tyr ser pro cys ala140 trp glu ileu val arg145 ala glu ileu met arg150 ser leu thr leu leu155 thr asn leu gln glu160 arg leu arg asn 1ys165 asp 3. met cys asp leu pro gln5 thr his ser leu gly10 asn arg arg ala leu15 ileu leu leu ala gln20 met gly arg ileu ser25 leu 30 35 phe ser cys leu lys asp arg his asp phe gly phe pro gln40 glu glu phe asp gly45 asn gln phe gln lys50 ala gln ala ileu ser55 val leu his glu met60 ileu gln gln ileu phe65 asn leu phe ser thr70 lys asp ser ser ala75 ala trp asn glu ser80 leu leu asp lys phe85 ser thr glu leu tyr9 95 100 gln gln leu asn asn leu glu ala cys val ileu gln asp val gly105 met glu glu thr pro110 leu met asn val glu115 ser ileu leu ala val120 ser lys tyr phe gln125 arg ileu thr leu tyr130 leu ser glu lys lys135 tyr ser pro cys ala140 trp glu ileu val arg145 ala glu ileu met rg150 ser leu thr leu leu155 thr asn leu gln glu160 arg leu arg asn lys165 asp 4. met cys asp leu pro glu5 thr his ser leu gly10 asn arg arg ala leu15 ileu leu leu ala gln20 met gly arg ileu ser25 leu 30 35 phe ser cys leu lys asp arg his asp phe gly phe pro gln40 glu glu phe asp gly45 asn gln phe gln lys50 thr gln ala ileu ser55 val leu his glu met60 ileu gln ileu phe65 asn leu phe ser thr70 lys asp ser ser ala75 ala trp asn glu ser80 leu leu asp lys phe ser thr glu leu tyr gln gln leu asn asp95 leu glu ala cys val100 ileu gln glu val gly105 val glu glu thr pro110 leu met asn val glu115 120 125 ser ileu leu ala val ser lys tyr phe gln arg ileu thr leu tyr130 leu ser glu lys lys135 tyr ser pro cys ala140 trp glu ileu val arg145 ala glu ileu met arg150 ser leu thr leu leu155 thr asn leu glu glu160 arg leu arg asn lys165 asp eu 5. met cys asp leu pro gln thr his eer leu gly asn arg arg ala leu15 ileu leu leu ala gln20 met gly arg ileu ser25 leu phe ser cys leu30 lys asp arg his asp35 phe gly phe pro gln40 glu glu phe asp gly45 asn gln phe gln lys50 thr gln 55 60 ala ileu ser val leu his glu met ileu gln gln ileu phe65 asn leu phe ser thr70 lys asp ser ser ala75 ala trp asn glu ser80 leu leu asp lys phe85 ser thr glu leu asn90 gln gln leu asn asp95 met glu ala cys val100 ileu gln glu val gly105 val glu glu thr pro110 leu met asn val glu115 ser ileu leu ala val120 ser lys tyr phe gln125 arg ileu thr leu tyr130 leu ser glu lys lys135 tyr ser pro cys ala140 trp glu ileu val arg145 ala glu ileu met arg150 ser leu thr leu leu155 thr asn leu gln glu160 arg leu arg asn 165 lys asp 6. met cys1 asp leu pro gln5 thr his ser leu gly10 asn arg arg ala leu15 ileu leu leu ala gln20 met gly arg ileu ser25 leu phe ser cys leu30 lys asp arg his asp35 phe gly phe pro gln40 glu glu phe asp gly45 asn gln phe gln lys50 thr gln ala ileu ser55 val leu his glu met60 ileu gln gln ileu phe65 asn leu phe ser thr70 lys asp ser ser ala75 ala trp asn glu ser80 leu leu asp lys phe85 ser thr glu leu tyr90 95 100 gln gln ieu asn asn leu glu ala cys val ileu gln asp val gly105 met glu glu thr pro110 leu met asn val glu115 ser ileu leu ala val120 ser lys tyr phe gln125 arg ileu thr leu tyr130 ieu ser glu lys 1ys135 tyr ser pro cys ala140 trp glu ileu val arg145 ala giu ileumet arg150 ser leu thr leu leu155 thr asn leu gln glu160 arg leu arg asn lys165 asp The following examples further serve to illustrate the invention and are not intended to be limitative thereof. EXAMPLE 1 TRANSCRIPTION ACTIVITY OF THE HYBRID PROMOTER The hybrid promoter of the invention is a composite of the lambda PR, the T7 A2 promoter, and the lac promoter i.e., positions 5 45, 46 50, and 51 71 respectively c.f., Table 1. The transcription activity of this hybrid promoter was compared to the activity of the lac promoter. Both promoters were cloned into the same plasmid background and assayed for ss galactosidase activity in a lac deletion strain 71 18 . Using this strain, the transcription activity of these two promoters was compared and the results are presented in Table 4. TABLE 4. Transcription Activitv of the Hybrid Promoter1 Activity in ss Galactosidase Units2 Promoter Glucose Glvcerol lac wild type 4800 10000 XPR T7 A2 lac 18800 15500 The synthetic lac promoter extended from position 74 to 21 of the natural lac promoter. This segment therefore included the lac promoter, lac operator, and the cyclic AMP preceptor protein binding site as a completely base paired duplex. The APR T7 A2 promoter was the base paired duplex extending from 1 to 71 of the gene sequence shown in Table 1. These synthetic promoters were cloned into pRZ5605. The plasmid contains the lac Z gene, including the ribosome binding site sequence but lacks a promoter.Upstream from the lac Z gene are three restriction sites tEcoRI, SmaI, and BamHI . These blunt end duplexes were introduced through the SmaI site and characterized by restriction mapping and DNA sequencing. 2 ss galactosidanse activity was measured on E. coli laboratory strain 71 18 using either glucose or glycerin as a carbon source. Within the limits of this assay where similar promoter constructions are being tested in the same plasmid vector, the lambdaPR T7 A2 lac hybrid promoter is approximately 50 more active than the fully induced lac promoter and approximately fourfold more active than the lac promoter under catabolite repression glucose as the carbon source . EXAMPLE 2 PREPARATION OF SECTION I OF THE HYBRID LYMPHOBLASTOID GENE Section I corresponding to the sequence from 1 149 c.f.,Table 1 was prepared by two different spproaches. The first approach involved isolation of the EcoRI HaeIII fragment from an aberrant attempt to clone an EcoRI to HindIII duplex nucleotides 1 384 . The second approach was more direct. A duplex containing nucleotides 1 158 was synthesized in order to clone the EcoRI to HaeIII fragment 1 149 . The sequences and corresponding synthetic oligodeoxynucleotides for the first approach are shown in Table 5. TABLE 5. Enzymatic Ligation of Section I EMI42.1 The DNA sequence from the EcoRI site 5 to the promoter through segment 33s is shown. The Hindill site is also shown.Brackets with enclosed letters and numbers define the various DNA segments that were chemically synthesized. The letters c and s designate control region and structural gene regions, respectively.The hybrid promoter, lac operator, and R1 ribosome binding site approximately occupy sequences lc 11c, 9c 12c, and 13c 20c, respectively. The ATG initiation codon sequence within R1 is found in 17c. The synthetic fragments shown in Table 5 were first joined to form three duplexes comprising segments 1c to 20c, 1s to 28s , and 29s to 33s. These three duplexes were then joined to form a duplex containing 1c to 33s. The synthesis of duplex 1c 20c was completed via three T4 DNA ligase catalyzed ligation steps. Segments 3c to lOc and llc to 20c were independently synthesized. Each duplex product was purified from unligated segments by gel permeation column chromatography on Sephadex G150 40 and analyzed by gel electrophoresis.The final step was a multiple ligation involving duplex 3c 10c 5 pmole , duplex 11c 20c 5 pmole , 5 32P segment 2c 10 pmole and unphosphorylated 1c 20 pmole . The gel analysis of this ligation reaction was performed. In addition to unreacted starting materials duplexes ic lOc and 11c 20c and product duplex lc 20c , two uncharacterized intermediates were also identified by gel electrophoresis. 51 32P Segment 2c migrates from the gel. Duplex 1c 20c containing segments composed of 113 and 115 mononucleotides was freed of duplexes 1c 10c and 1lc 20c by chromatography on a SephadexG150 40 column.Gel electrophoresis analysis of this purification step indicates that the two uncharacterized intermediates still remain in the sample of duplex 1c 20c. The yield of duplex 1c 20c was 1.5 pmole. The removal of duplex 11c 20c was especially critical. This duplex has the same single stranded region adjacent to the structual gene as does duplex lc 20c. If duplex llc 20c were not removed, it would compete with duplex ic 20c for the structural gene duplex during the next appropriate ligation step. Duplex 1s 28s containing structural gene segments is to 28s was synthesized in a stepwise manner. Segments is to 4s, 5s to 10s, ils to 14s , 15s to 18s , 19s to 24s , and 25s to 28s were joined to form intermediate duplexes and column purified. The first step in the final assembly was synthesis of duplex ls lOs by joining duplex ls 4s to 5s 10s followed by gel permeation chromatography onSephadex G150 40. Gel electrophoresis analysis of the column chromatography purified duplex ls lOs 90 pmole clearly shows that most of the duplexes 5s 10s and 1s 4s have been removed. This sample of duplex ls lOs was next joined to duplex 11s i4s to form duplex 1s 14s 45 pmole .After gel permeation column chromatography, excess duplex 11s 14s had been removed as can be seen by gel electrophoresis analysis. Duplex ls 14s however was contaminated with unreacted duplex is i Os. This growing duplex was then reacted sequentially with duplexes 1fs 18s , 19s 24s , and 25s 28s to form duplexes ls 18s 20 pmole , 1s 24s 45 pmole , and 1s 28s . Each intermediate duplex was purified by Sephadex G150 column chromatography and analyzed by gel electophoresis. Clearly the gel permeation technique did remove each new, relatively small duplex but failed to remove unreacted, partially extended duplexes. These incomplete duplexes were carried forward and constitute most of the DNA present in the final sample of duplex 1s 28s . The final step was purificationof duplex ls 28s by preparative gel electrophoresis. The appropriate band was cut from the gel, eluted, and purified from salts and buffers. The isolated yield of duplex ls 28s was 1 pmole. Duplex 29s 33s was prepared. After enzymatic joining of these segments, the duplex was purified by column chromatography inSephadex G 150 40 and analyzed by gel electrophoresis. Duplex lc 33s was prepared by joining duplexes lc 20c, ls 29s , and 29s 33s. Duplex lc 20e 1.25 pmole , duplex 1s 2Ss 0.53 pmole and duplex 29s 33s 5 pmole were each annealed from 75 C. Duplex ls 28s and duplex 29s 33s were then mixed, warmed to 45 C and colled to room temperature. Duplex 1c 20c was next added.The solution was warmed to 37 C and colled initially to room temperature and then to 4 C. The final volume was 15 l. After addition of T4 DNA ligase 2.5 units , the reaction was allowed to proceed overnight. The total reaction mixture was fractionated by gel electrophoresis. Size markers generated from pBR322 digestion by HDaII were used. The final yield of duplex lc 33s, as purified from this gel, was 50 femtomole. Analysis by gel electrophoresis of the reaction indicated that the ligation reaction only proceeded 10 20Z. Major bands of unreacted duplexes lc 20c, 1s 28s , and 29s 33s were present on the gel. The product was eluted from the appropriate gel slice and digested with HindIII in order to generate a duplex ready for cloning in pBR325. Synthesis of Section I via the second approach focused on the EcoRI to HaeIII region of the structural gene. The sequence and synthesis plan are shown in Table 6. TABLE 6. Synthetic Plan and Structural Gene Sequence for Section I from EcoRI to HaeIII Seetion EcoR Mee EMI46.1 The lymphoblastoid interferon hybrid gene from the 5 EcoRI site on the sense strand through the HaeIII site at 149 is depicted. Brackets with numbers enclose chemically synthesized segments. Segments ic through 20c are the same as described in Table 5. Therefore, the hybrid promoter, lac operator, and R1 ribosome binding site occupy approximately sequences 1c 11c, 9c 12c, and 13c 20c, respectively.The ATC initiation codon sequence within R1 is found in 17c. Duplex lc 20c 1.5 pmole , whose synthesis has been described, and duplex ls 4s 15 pmole were annealed from 37 C and mixed. After annealing from 370C to 40C, T4 DNA ligase was added and the reaction allowed to proceed overnight. Analysis of the reaction mixture by gel electrophoresis was performed. Size markers generated from pBR322 by restriction with Phall were used. The product, duplex lc 4s, as eluted from the gel, was isolated in 0.6 pmole yield. Once synthesized, fragment I can be cloned into pBR322 opened with the restriction enzymes EcoRI and BalI. Section I was synthesized from duplex 1c 20c and duplex ls 4s.Duplex ls 4s was synthesized in the usual manner and purified using a Sephadex G 150 column. The next step was joining duplex ls 4s in tenfold molar excess to duplex 1c 20c. An analytical gel of the reaction mixture suggested that conversion to the final product duplex 1c 4s was quite satisfactory.The isolated yield 40 confirmed these results. EXAMPLE 3 PREPARATION OF SECTION II OF THE HYBRID LYMPHOBLASTOID GENE Section II containing EcoRI and HindIll sites and extending from the HaeIII site position 149 to the DdeI site position 246 of the interferon gene is shown in Table 7. TABLE 7. Synthetic Design and Structural Gene Sequence for Section II from HaeIII to DdeI Section II Hae III del EMI48.1 This arrangement of restriction sites was used to cloneSection II between the unique EcoRI and Hindlll sites of pUC8.Restriction enzyme cleavage of the cloned gene fragment in pUC8 withHaeIII and DdeI generated Section II as a unique duplex containing the interferon gene sequences. Duplex 5s 16s corresponding to Section II was prepared in two steps. In the first step, 5 PJ Labeled segments 5s 8s, 9s 12s, and 13s to 16s were annealed and ligated in three separate test tubes as previously described 400 pmole of each segment . These three ligated duplexes were then purified on SephadexG 150 40 column, pooled, annealed together from 370C and joined usingT4 ligase 5 units . The results of this ligation were evaluated by gel electrophoresis. The final product, duplex 5s 16s containing 113 mononucleotides in each strand, was isolated and purified by preparative gel electrophoresis and characterized as to size against calibrated DNA markers.The isolated yield was 60 pmole. An aliquot of the purified duplex was analyzed by gel electrophoresis using pBR322DNA cut with EpaII as size markers. The product migrated as expected relative to these size markers. As determined by gel analysis, the major ligation product was an intermediate duplex which has not been characterized. Such intermediates can be generated by several mechanisms. The most trivial explanation is a stoichiometry problem.If unequal molar ratios of all 15 32p ,labeled oligodeoxynucleotides are used in a ligation reaction, then an intermediate corresponding in yield to the limiting reagent will be the major product. More often, however, the formation of either intramolecular or intermolecular secondary complexes lead to the accumulation of intermediates. These complexes are not easily predicted. Such problems are most readily solved by resynthesis using different positions for generating overlapping and complementary single strand oligodeoxynucleotides. EXAMPLE 4 PREPARATION OF SECTION III OF THE HYBRID LYMPHOBLASTOID GENE The DNA sequence of the structural gene of Section III, the duplex 17s 28s containing an EcoRI site and extending from the DdeI site at position 246 to the HindIII site at position 384 of the interferon sequence, is shown in Table 8. TABLE 8. Synthetic Design and Structural Gene Sequence for Section III from DdeI to HindIII Section III Ddel Hind III EMI51.1 The arrangement of restriction sites described in Table 8 was used to clone Section III between the unique EcoRI and HindIII sites of pUC8. Restriction enzyme cleavage of the cloned gene fragment in pUC8 with DdeI and HindIII generated Section III as a unique duplex. 5 32P Labeled segments 17s to 20s 100 pmole each , 21s to 24s 100 pmole each , and 25s to 28s 100 pmole each were annealed and ligated in three separate test tubes. These duplexes were then mixed without purification and enzymetically joined using T4 ligase. The reaction mixture was evaluated by gel electrophoresis. A major band corresponding to intermediates duplex 17s 24s or 21s 28s was observed. The generation of many intermediates was presumably due to the presence of unligated segments during the joining steps. The final product, as two oligodeoxynucleotides containing 146 mononucleotides each, was isolated in a purified form 13 pmole 26 yield by preparative gel electrophoresis. An aliquot of the purified duplex was characterized as to size by gel electrophoresis using pBR322 DNA cut with HpaII as size markers. The yield of purified duplex 17s 28s Section III was 13 pmole 26Z yield . EXAMPLE 5 PREPARATION OF SECTION IV OF THE HYBRID LYb POBLAST0ID GENE Section IV of the hybrid lymphoblastoid gene of the invention extends from the HindIII position 385, c.f., Table 1 site to theEcoRI site at the end of the gene. The DNA sequence of the structural gene is shown in Table 9 from position 351 to the end of the gene at the EcoRI site. TABLE 9. Synthetic Design and Structural Gene Sequence for Section IV from llindIII to EcoRI Seetion IV Hind III Eco RI EMI53.1 As a first step, 15 32P labeled segments 29s to 36s, 37s to 40s, and 41s to 54s were enzymatically joined independently to form the appropriate intermediate DNA duplexes. Each of these three duplexes was then purified and joined in stepwise fashion. Each duplex was purified from unreacted segments by column chromatography onSephadex G 150 40 and analyzed by gel electrophoresis after purification. The gel obtained after purification of 41s 54s showed that theDNA strands containing 135 and 123 mononucleotides have separated.Duplexes 37s 40s 45 pmole and 41s 54s 10 pmole were enzymatically joined using T4 DRA ligase. The product duplex 37s 54s of this ligation was fractionated from starting materials on a Sephadex G 150 40 column. The isolated yield was 2.5 pmole 25 . The purified duplex was analyzed by gel electrophoresis. Duplex 37s 54s 2.5 pmole and duplex 29s 36s 10 pmole were enzymatically joined and the reaction mixture fractionated by column chromatography on Sephadex G 150 40. The final product duplex 29s 54s, 1.6 pmole was analyzed by gel electrophoresis. A series of size markers generated by a HnaII cut of pBR322 was used in this gel analysis. Based on the limiting reagent duplex 41s 54s , the overall yield of partially purified duplex 29s 54s was 16 .The column purification step successfully removed unreacted duplex 29s 36s but failed to remove duplex 37s 54s from the product. EXAMPLE 6 CLONING THE LY.X2HOBLASTOID HYBRID INTERFERON GENE Successful cloning of the four subfragments of the 598 b.p.lymphoblastoid hybrid interferon gene and assembly of the total gene was accomplished using the procedure outlined in Figure 1. The top portion of Figure 1 shows schematically the interferon gene with the key restriction sites. The central portion shows the four sections as cloned in plasmids and the bottom portion schematically outlines the plan for stepwise joining these scctions. Thus, the gene was initially cloned in four sections. These sections were then joined sequentially and cloned in three steps to form the total gene. The results summarizing these steps are presented in Figure 2. Plasmid derivatives schematically outlining key gene sections are shown. Section I, which extends from the 5 EcoRI site to the HaeIII cut site at position 149, was isolated from an unsuccessful attempt to clone from EcoRI to theHindIII cut site at position 384.When the synthetic duplex extending from the 5 EcoRI site relative to the sense strand to the HindIII site was cloned into pBR325, only a clone containing part of the interferon sequence could be isolated. This clone had a deletion from position 223 to 372 or 225 to 374 . However, the sequence from the 5 EcoRI site to the HaeIII cut site was intact. This gene segment as a pBR325 derivative pMP19 was used to isolate Section I. Section II containing structural gene sequences from the HaeIII site to the DdeI site at position 246 was cloned pMP1 with EcoRI and HindIII linkers. In this way, cloning, isolation and sequencing as a unique EcoRI HindIII duplex could be easily accomplished in pUC8.Similarly, Section III containing structural gene sequences from the DdeI site at position 247 to Hindill was cloned pMP33 with an EcoRI linker, Section IV which extends from the HindIII cut site at 384 to the EcoRI site at the 3 end of the structural gene was inserted into pUC8 which had been opened with EcoRI and HindIll. The resulting plasmid pre29 was characterized to containSection IV by DNA sequencing. After amplification of the cloned gene fragments in vivo, the segments were isolated and assembled stepwise in pUC8. Section I was isolated from pbIP19 by treatment with EcoRI and HaeIII restriction enzymes followed by fractionation using 10 polyacrylamide gel electrophoresis. Similarly, Section II was isolated from pES1 after treatment with HaeIII and HindIIl.The two segments were recovered from the gel, joined using T4 DWA ligase, and cloned into pUCS that had been opened with EcoRI and HindIII enzymes. Clones containing the appropriate 946 base pair duplex extending from the 5 EcoRI site to the DdeI site Section I II were identified pMP456 . Section I II was recovered from pMP456 by first treatment of the plasmid with EcoRI and DdeI enzymes and then preparative gel electrophoresis. Similarly,Section III was recovered from pMP33 by enzymatic digestion with DdeI and HindIII.Section I II was next joined to Section III through the common DdeI site and the product Section I II III was cloned into pUC8. A plasmid containing Section I II III was isolated pMP18 and characterized by restriction analysis. The final step involved the ligation of the Section I II III with Section IV. Section I II III andSection IV were isolated from pMP18 and pbtP2, respectively, by enzymatic treatment with EcoRI and HindIII followed by isolation using preparative polyacrylamide gel electrophoresis. After joining these two sections with T4 DNA ligase and cloning into pUC8, the correct assembly of the gene in pMPlO5 was verified by restriction enzyme analysis and DNA sequence determination. Clones containing the interferon gene in both orientations with respect to pUC8 were obtained. EXAMPLE 7 INCORPORATION OF RIBOSOME BINDING SITES R2 AND R3 INTO THE HYBRID LYMPHOBLASTOID GENE The interferon gene as initially assembled c.f., Figs. 1 BR 2 contained ribosome binding site R1. In order to maximize the possibility for high level expression of this gene, two additional ribosome binding sites, R2 and R3, were constructed. In this way, translation initiation from three different ribosome binding sites could be rigorously checked since no changes in nucleotide sequence were introduced in either the promoter or structural gene. The chemically synthesized segments corresponding to R2 and R3 were assembled as EcoRI AluI duplexes.Therefore each duplex contained the first nine codons of the interferon gene and the entire control region including the APR T7 A2 promoter, lac operator, and the appropriate ribosome binding site. Preparation of the EcoRI Alul duplex containing ribosome binding site R2. The DNA sequence of the R2 duplex is shown in Table 10. TABLE 10. Preoaration of the EcoRI AluI Duplex Containing Ribosome Binding Site R2 EMI58.1 Table 10 shows the duplex extending from EcoRI through AluI and including segments is and 2s of the structural gene. As can be seen by comparison with the Rl control region c.f., Table 5 , segments 1c to lic and is, 2s corresponding to the promoter lac operator and the first two structural gene segments, respectively, are identical. The ribosome binding site sequences for R1 12c 20c and R2 21c 29c are different. Unphosphorylated 1c 200 pmole and 5 P labeled 2c were annealed from 7500 in 8 l ligase buffer.Preformed duplex 3c iOc carrying 5 32P labels 80 pmole was dissolved in 8 Ul ligase buffer, pooled with Ic and 2c, and the mixture annealed from 37 C. These segments were then combined with preformed duplex 21c 29c carrying 5 P labels 60 pmole and the mixture annealed from 35 C. The final volume was 40 l. T4 DNA ligase 5 units was added and the joining reactions allowed to proceed overnight. 5 32p labeled segments 1s and 2s 100 pmole each and T4 DNA ligase 2.5 units were added and incubation was continued for six additional hours at 4 C. The ligation mixture was dried in vacuo, redissolved in 40 l formamide, heated in a boiling water bath for 3 minutes, and loaded on to an 8X acrylamide gel containing 8 M urea.The band of acrylamide containing the R2 product duplex lc 2s was cut from the gel and the duplex extracted and precipitated with ethanol.The final yield was 6.5 pmole. Duplex 1c 2s from 75 C in 10 l restriction buffer and the units at 370C for 2 hours. The products obtained from this restriction cleavage were analyzed by gel electrophoresis. The small duplex generated from is and 2s by cleavage at the AluI site was found near the bottom of the gel. The slowest migrating band corresponded to the unrestricted duplex. The EcoRI AluI duplex migrated as separated strands and was the major P labeled band. Size markers generated from HDaII cleavage of pBR322 were used. The AluI digested duplex was fractionated on a Sephadex G 150 40 column and analyzed by gel electro phoresis. The column eluant contained the EcoRI AluI duplex of R2 and undigested duplex lc 2s. The recovery was 3.5 pmole. Preparation of the EcoRI AluI duplex containing ribosome binding site R3. The DNA sequence of the R3 duplex is shown in Table 11. TABLE 11. Preparation of the EcoRI AluI Duplex Containing Ribosome Binding Site R3 EMI61.1 Table 11 shows the duplex extending from EcoRI through AluI and including segments Is and 2s of the structural gene. As can be seen by comparison with the R1 control region c.f., Table 5 , segments 1c to 11c and ls, 2s corresponding to the promoter lac operator and the first two structural gene segments, respectively, are identical. The ribosome binding site sequences for RI 12c 20c and R3 30c 38c are different. The ligation of these segments to form a duplex lc 2s containing R3 was completed in the same manner as described above for the synthesis of the same duplex containing R2 sequences.Amounts of segments were as follows lc 320 pmole 2c 160 pmole 3c lOc 80 pmole 29c 38c 60 pmole 1s 100 pmole 2s 100 pmole . The final yield was 14 pmole. After digestion with AluI as described above, the EcoRI Alul duplex containing R3 was isolated in 3.5 pmole yield frqm 5 pmole of duplex lc 2s. The results of this synthesis were analyzed by gel electrophoresis The duplex lc 2s was purified by column chromatography as previously described.Using size markers generated by digestion of pBR322 with liaeli, the following were analyzed by gel electrophoresis ligated and column purified duplex lc 2s the reaction mixture from duplex lc 2s after digestion with Alum and column purified AluI digestion mixture. pBR322 was opened with the restriction enzymesEcoRI and PuuII. R1 and R2 were cloned separately into the opened plasmid as EcoRI AluI fragments. Assemblv of the hvbrid interferon oene containing ribosome binding sites R2 and R3. A schematic outline describing the assembly of the interferon gene containing ribosome binding sites R2 and R3 is presented in Figure 3. The original gene construction containing R1 is shown schematically at the top of the figure. By restriction first with HindIII and then Alul, an Alul HindilI duplex is generated. AnEcoRI AluI duplex containing either R2 or R3 sequences left part of figure is then joined to the AluI HindIII duplex. As indicated in the bottom part of Figure 3, the total gene is then resynthesized. The interferon gene with R1 and assembled in pMPiO5 was cut with EcoRI and HindIII. The 384 base pair EcoRI HindIII duplex was then cut with AluI. By first cutting the gene with HindIII, the only other AluI recognition site within the gene HindIII site was present as single strands and hence inactivated. The 267 base pair Alul FindIlI duplex was isolated and ligated to the EcoRI Alul duplex containing either R2 or R3. The new EcoRI HindIII duplexes were then cloned into pUC8, amplified, and reisolated. Finally, each 384 base pair EcoRI HindIII duplex containing either R2 or. R3 was ligated to the remainder of the gene HindIII EcoRI and inserted into pUC8. Plasmids having duplexes with both orientations of the gene with respect to the cloning vehicle could be identified. EXAMPLE 8 EXPRESSION IN E. COLI AND AND BIOLOGICAL ACTIVITY OF a C LYMPHOBLASTOID HYBRID INTERFERON The lymphoblastoid hybrid interferon gene was expressed in E.coli as a pUC8 plasmid derivative and found to be biologically active as determined by the cytopathic effect reduction assay. Three differ ent constructions of the gene containing ribosome binding sites R1, R2, and R3 were assayed. Results of various experiments are presented inTables 13 and 14. TABLE 13. A Summary of the Lymphoblastoid Hvbrid Interferon Assavs as Determined bv the Cvtopathic Reduction Effect Gene Control Region Interferon Yield2 mg 1 R1 0.3 R1 0.15 R1 0.15 R2 9 R2 5 R2 2 R3 0.2 R3 0.2 R3 0.2 Assay was with a bovine kidney cell line MDBK using vesicular stomatitis virus as challenge.2 Cells were assayed at OD578 nm of 4 5. TABLE 14. Titration of Lvmphoblastoid Leukocvte alpha C Hvbrid Interferon Activity Using Three Ribosome Binding Sites R1, R2, R3 Mode ofE. coli Cell Ribosome of Colonies Interferon Strain Binding Site Tested Determination RIA . CPE W3110 1 3 2 3 550 g 1 350 pg l 3 3 100 g 1 60 7118 1 3 75 Vg l 6 g 1 2 3 700 g 500 g 1 3 3 15 g 1 6 g 1 7902 1 3 50 g 1 200 g 1 2 3 2.5 pg l 5 g 1 I 3 3 50 g 1 200 g 1 RIA radioimmune assay using a standard a interferon antibody. CPE cytopatic effect reduction assay using a bovine kidney cell line MDBK and vesicular stomatitis virus as challenge. Specific activity with this assay was 5 10 x 108 units mg protein. The hybrid interfcron had a specific activity 1 1 x 108 units mg when assayed on a bovine kidney cell line MDBK when challenged with vesicular stomatitis virus. All three ribosome binding sites appear to be translationally active. The order of translation initiation activity is R2 R3 R1. The levels of expression were evaluated by gel electrophoresis of total cell protein. Cell lysates of strains of E.coli having R1, R2, and R3 controlled genes were assessed. Cell lysate material was obtained for this gel analysis by growing E. coli containing pMP105 to 0.4 0.6 OD578 in nutrient media supplemented with ampicillin and induced with 10 3 M IPTG. After 3 hours, cells were harvested, lysed by sonication and aliquots loaded on gels. The amount of interferon hybrid protein produced by each ribosome binding site is consistent with the results obtained using the virus challenge assay. More interferon appears to be produced from R2 than from eitherR3 or R1. EXAMPLE 9 CHEMICAL SYNTHESIS OF OLIGODEOXYNUCLEOTIDES Oligodeoxynucleotides containing control region sequences promoter, lac operator and ribosome binding sites were synthesized on silica gel polymer supports using 5 dimethoxytrityldeoxynucleotide 3tetrazoylphosphoramidites as synthons. All other oligodeoxynucleotides were synthesized similarly on silica gel polymer supports but used 5 dimethoxytrityldeoxynucleoside 3 N, N dimethylaminophosphoramidites as synthons. The chemical procedures have been published ChemicalSynthesis of Oligodeoxynucleotides Using the Phosphite TriesterIntermediates. M. H. Caruthers, In Chemical and Enzymatic Synthesis ofGene Fragments.Ed. by H. 0. Casson and Anno Lang. Verlag Chenie,Deerfield Beach, Florida, 1982 . Upon completion of a synthesis, each oligodeoxynucleotide was freed of protecting groups and isolated by polyacrylamide gel electrophoresis. Silica gel containing the synthetic deoxyoligonucleotide was first treated with triethylammonium thiophenoxide to remove the methyl groups from internucleotide phosphotriesters. The next step was hydrolysis of the ester that joins the oligodeoxynucleotide to the support using concentrated ammonium hydroxide for 3 hours at 20 C.After centrifugation and recovery of the supernatant containing the mixture of failure sequences and the correct oligodeoxynucleotide, the N benzoyl groups from deoxycytosine and deoxyadenosine and the N isobutyryl group from deoxyguanosine were removed by warming the concentrated ammonium hydroxide solution at 500C for 12 hours. The 5 0 dimethoxytrityl group was hydrolyzed using 80Z acetic acid. The final product was isolated from failure sequences by gel electrophoresis. The hydrolysate was dried in vacuo, dissolved in formamide, warmed at 100 C for 3 minutes and loaded on to a 122 polyacrylamide gel prepared in 7 M urea with a tris borate buffer pH 8.0 .After gel electrophoresis, oligodeoxynucleotides were visualized on the gel using short wavelength u.v. light with a fluorescent silica gel plate behind the gel. The u.v. absorbing band corresponding to the DNA segment was cut from the gel and the oligodeoxynucleotide eluted. The eluant was extracted four times with n butanol, dried, redissolved in 0.2 ml water, and freed of salts by column chromatography on a Sephadex G 150 40 column 5 ml syringe equilibrated with 10 mM triethylammonium bicarbonate TEAB . EXAMPLE 10 PREPARATION OF t5 32P LABELLED OLI CODEOxY .a UCLE6tIDES Oligodeoxynucleotides 100 500 pmoles were phosphorylated with T4 polynucleotide kinase and gamma P ATP having a specific activity of 2.5 10 x 10 cpm pmole. The reactions were completed in 5 15 l 33mM Tris HCl pH 7.5 , 3.3 mM MgCl2 and 5 mM DTT. After 30 40 minutes at 370C, reaction mixtures were warmed at 75 C to inactivate T4 kinase. Aliquots were assayed by polyacrylamide gel electrophoresis followed by autoradiography to visualize the results. EXAMPLE 11 LIGATION OF PHOSPHORYLATED OLIGODEOXYNUCLEOTIDES Reaction mixtures containing 5 phosphorylated oligodeoxynucleotides 100 500 pmole , y 3 P ATP, and inactivated T4 polynucleotide kinase were usually used without purification in ligation reactions. Each pair of complementary sequences was annealed by warming the combined solutions of phosphorylated segments to 95 C and slowly cooling to 4 C. Two sets of annealed duplexes having a complementary single strand region were then pooled, warmed to 37 C and cooled slowly to 40C.The solution was adjusted to 66 mM Tris HOl pH 7.5 , 6.6 mM MgC12 , 10 mM DTT and 400 pM ATP in a final volume of 20 30 p1. After addition of T4 DNA ligase 1 2 units , the reaction mixture was incubated for 12 15 hours and then assayed using polyacrylamide gel electrophoresis. Reaction mixtures were fractionated on Sephadex G 150 40 columns. For ligated DNA segments 40 to 50 nucleotides in length, the column size was 1 x 20 cm. For larger molecules, the size was 1 x 90 cm. Before fractionation, each column was washed extensively with 10 mM TEAB and preloaded with 0.5 ml of 95X formamide. The ligation mixture was dried, dissolved in 40 l of 95 formamide, warmed for 3 minutes at 100 C, and applied to the column. The DNA was eluted with 10 mM TEAB as 0.2 0.4 ml fractions, collected, and counted. Pooled fractions were dried and redissolved in 10 mM TEAB or a buffer appropriate for the next step. RYAdsLE 12 GEL PURIFICATION Polyacrylamide gels 8 10 containing 7 M urea were used to isolate completely homogeneous DNA duplexes from ligation mixtures.Routinely, ligation mixtures were dried, redissolved in 95 formamide, warmed at 100 C for 3 minutes and loaded on to the gel. After electrophoresis, the DNA was located by autoradiography and extracted from the gels. The DNA was precipitated from the extract using 2.5 volumes ethanol. After two additional precipitations, the DNA was dried and redissolved in 10 mM TEAB or a buffer appropriate for the next step.