# A positive retroregulatory element, plasmid and cells comprising said element and their production, a method of enhancing gene expression using said element and a method of enhancing RNA half life using said element.

## Abstract
The invention concerns a method for extending the half life of RNA s. The half life extension is conferred upon the RNA by a co transcribed positive retroregulatory element which is ligated to the 3 end of the DNA sequence coding the RNA. RNAs having an extended half life conferred by a co transcribed positive retroregulatory element are also claimed.

## Claims
CLAIMS 1. A method of enhancing expression of a gene product which comprises ligating a positive retroregulatory element to a DNA sequence able to express said gene product in a relationship whereby expression of said gene product is enhanced 2. A method as claimed in claim 1 wherein said DNA sequence is a non heterologous DNA. 3. A method as claimed in claim 1 wherein said DNA sequence is an heterologous DNA 4. A method as claimed in claim 3 wherein said heterologous DNA codes for a prokaryotic gene product, preferably penicillinase 5. A method as claimed in claim 3 wherein said heterologous DNA codes for a eurkaryotic gene product, preferably mammalian IL 2 6. A method as claimed in any one of claims 1 to 5 wherein said positive retroregulatory element is ligated downstream of said DNA sequence expressionable for a gene product 7. A method as claimed in claim 6 wherein a translation termination signal is located between said positive retroregulatory element and said DNA sequence coding for said gene product 8. A method as claimed in any one of claims 1 to 7 wherein said positive retroregulatory element comprises a portion of the 3 flanking sequence of the B. thurinaiensis crystal protein gene wherein said flanking sequence includes an inverted repeat sequence. 9. A method as claimed in claim 8 wherein said positive retroregulatory element comprises a portion of the DNA sequence AGATATAAGTAATTTGTTGTAATGAAAAACGGACATCACCTCCATTGAAACGGAGTGATGTCCGTTTTACTATGTTATTTTCTAGTAATACATATGTATAGAGA 10. The method as claimed in claim 9 wherein said inverted repeat sequence has the sequence 5 AAAACGGACATCACCTCCATTGAAACGGAGTGATGTCCGTTTT 3 11. A method as claimed in claim 1 wherein said positive retroregulatory element corresponds to an RNA transcript, a portion of said RNA forming a stem and loop structure, said stem and loop structure having a G of about 30.4 Kcal mol. 12. A method as claimed in claim 11 wherein said stem and loop structure comprises 40 to 45S preferably about 43 , guanosine and cytosine 13. A positive retroregulatory element which when ligated downstream of a DNA sequence expressionable for a gene product enhances the expression of said gene product. 14. A positive retroregulatory element as claimed in claim 13 sequence which when ligated at the 3 end of a DNA sequence expressionable for a gene product enhances the expression of said gene product 15. A positive retroregulatory element as claimed in claim 13 comprising a portion of the 3 flanking sequence of the B. thurinqiensis crystal protein gene, wherein said flanking sequence includes an inverted repeat sequence 16. A positive retroregulatory element as claimed in claim 13 comprising a portion of the DNA sequence AGATCTAAGTAATTTGTTGTAATGAAAAACGGACATCACCTCCATTGAAACGGAGTGATGTCCGTTTTACTATGTTATTTTCTAGTAATACATATGTATAGAGA 17. A positive retroregulatory element as claimed in claim 15 wherein said inverted repeat sequence comprises the sequence 5g AAAACGGACATCACCTCCATTGAAACGGAGTGATGTCCGTTTT 31 18. A positive retroregulatory element as claimed in claim 15 wherein the RNA transcript of said inverted repeat sequence can form a stem and loop structure 19. A positive retroregulatory element as claimed in claim 13 corresponding to an RNA transcript, a portion of said RNA transcript forming a stem and loop structure, said stem and loop structure having a t G of about 30.4 KcalXmole 20. A positive retroregulatory element as claimed in claim 19 wherein said stem and loop structure comprises 40 to 45t ,preferably 43, guanosine and cytosine 21. A plasmid comprising a DNA sequence expressionable for a gene product and a positive retroregulatory element ligated to said DNA sequence in a relationship thereto, whereby expression of said gene product is enhanced 22. A plasmid as claimed in claim 21 wherein said positive retroregulatory element is ligated downstream of said DNA sequence expressionable for a gene product 23. A plasmid as claimed in claim 2.1 wherein the positive retroregulatory element is ligated downstream of said DNA sequence expressionable for the gene product and wherein a translation terminator signal is located between said positive retroregulatory element and said DNA sequence expressionable for said gene product 24. A plasmid as claimed in claim 22 wherein the positive retroregulatory element comprises a portion of the 3 flanking sequence of the B. thurinqiensis crystal protein gene, wherein said flanking sequence includes an inverted repeat sequence 25. A plasmid as claimed in claim 24 wherein said inverted repeat sequence has the sequence 5 AAAACGGACATCACCTCC ATTGAAACGGAGTGATGTCCGTTTT 3 26. A cell transformed by a plasmid as claimed in any one of claims 21 to 25 27. A cell as claimed in claim 27 wherein said cell is a eukaryotic cell 28. A cell as claimed in claim 26 wherein said cell is a prokaryotic cell. preferably E. coli or B. subtilis 29. A process for producing a plasmid as defined in any one of claims 21 to 25 comprising inserting into a plasmid capable of transforming a host cell a positive retroregulatory element as defined in any one of claims 13 to 20 ligated to aDNA sequence in a position permitting expression of said gene 30. A process for producing a cell which is capable of expression of a selected protein regulated by a positive retroregulatory element as defined in any one of claims 13 to 20 which process comprises transforming a cell with a plasmid as claimed in any one of claims 21 to 25 31. A method of extending the half life of anRNA sequence comprising a providing a DNA sequence having a positive retroregulatory element ligated at the 3 end thereof b providing a means for transcribing said DNA sequence into RNA which it encodes and c transcribing said DNA sequence with said positive retroregulatory element 32. A method as claimed in claim 31 wherein said means for transcribing said DNA sequence is a host cell 33. A method as claimed in claim 32 wherein the positive retroregulatory sequence is heterologous to said host cell 34. RNA sequences having an extended half life conferred by a co transcribed positive retroregulatory element

## Description
A Positive Retroregulatory Element, Plasmid andCells comprising Said Element and their Production, a Method of Enhancing Gene Expression Using Said Element and a Method of Enhancing RNA half Life Using Said Element. Field of the Invention The invention concerns the fleld of recombinant DNA. More particularly the invention concerns positive retroregulatory elements, which when ligated to selected DNA sequences coding for a gene product, enhance the expression of the gene product. Plasmids carrying the positive retroregulatory element ligated to selected DNA sequence and cells transformed by such plasmids are provided. In addition, the invention relates to a method for enhancing expression of a gene product by ligating a positive retroregulatory element to a selected DNA sequence expressionable for a desired gene product. One of the fundamental stratagies forming the basis for the commercial utility of recombinant DNA technology is the production in relatively high volumes of gene products in the form of polypeptides or proteins that ordinarily occur in nature at very low concentrations or volume. By transforming cells that have relatively short generation or doubling times with recombinant molecules, which generally are in the form of plasmids, significant amounts of a desired gene product may be produced. Although each cell harboring the recombinant DNA molecule may in fact produce only a very small amount of the desired gene product, the rapid multiplication of the cell allows the production of significant amounts of the desired gene product. In general, the goal of producing larger amounts of a selected gene product is limited by the size of culture medium volumes required for growing the transformed celx that produces the desired gene product. Increasing the yield ot a desired gene product is clearly a major concern in the commercial production of a desired gene product. One approach to increasing yield is to improve the recovery rate of the desired gene product from the transformed cell or culture medium while leaving the amount of product produced by the cell unchanged. This approach entails the addition of processing steps to increase recovery of the desired gene product from a given fermentation run. Such additional processing steps can require enormous costs for added equipment and personnel. A different approach to increasing yield of a desired gene product is to increase the expression of the desired gene product by the transformed cell. Such increases in production of the desired gene product by each cell can lead to increases in yield for a particular fermentation run as well as improvements in purity of the product produced and a lowering of the cost per unit of product since further processing may be unnecessary. One of the important general approaches to improving the yield of desired gene product per cell is to provide optimal culture conditions for the cell. By enriching the nutrient content of the medium, providing optimal temperatures for fermentations, furnishing the optimum amount of trace factors and supplementing the culture medium with required amino acids, for example, yields can be significantly improved. A second general approach to the problem of increasing the yield of a desired gene product per cell is to manipulate the regulatory elements controlling the expression of the gene product by the cell. One method of manipulating the regulatory elements controlling expression of a desired gene product is by the selection of strong promoters. Promoters may be generally defined as regions of a DNA molecule to which RNA polymerase binds to initiate transcription of messenger RNA mRNA from a DNA sequence coding for a gene product. Strong promoters have the characteristic of initiating anRNA transcript by RNA polymerase with higher frequency than weaker promotors with the result that the DNA sequence with which the promoter is associated is transcribed to form more mRNA transcript coding for the desired gene product. It has become conventional in the recombinant DNA field to ligate a strong promoter to a selected DNA sequence coding for a desired gene product, in proper reading frame such that mRNA transcripts, initiated from the strong promoter are produced.Multiple strong promoters may be ligated to a selected DNA sequence, thereby increasing the opportunity for binding RNA polymerase to theDNA sequence and producing more mRNA transcript from the DNA sequence. An example of this last method is the use of tandam lac operon promoters at the beginning of a DNA sequence coding for a desired gene product. Another approach for increasing the binding of RNA polymerase to provide higher levels of mRNA transcript is the elimination of regulatory factors that tend to reduce the ability ofRNA polymerase to transcribe mRNA from a DNA sequence coding for a desired gene product. Certain strong promoters have associated with them attenuator regions which under certain conditions cause a boundRNA polymerase to cease transcription of DNA sequences with which the attenuator regions are associated. One such attenuator region is associated with the promoter of the tryptophan operon, a promoter which is known to be a strong promoter. By eliminating the attenuator region of the promoter of the tryptophan operon, the tryptophan promoter can serve as an unimpaired strong promoter. Another approach to increasing the yield of a desired gene product is to ligate strong ribosome binding sites within a selectedDNA sequence such that ribosomes bind with high efficiency to the mRNA transcript that has been transcribed from the selected DNA sequence.By increasing the affinity of an mRNA transcript for the ribosome through such strong ribosome binding sites, it is believed translation of the mRNA transcript occurs with a greater frequency, thus increasing the production of the desired gene product. All of the above mentioned techniques for increasing the expression of a desired gene product involve manipulation of regulatory sequences that appear at the 5 end of the DNA sequence coding for the desired gene product. In all cases, these regulatory sequences are involved in either initiation of transcription of theDNA sequence coding for the desired gene product, or initiation of translation of the mRNA transcript corresponding to the DNA sequence coding for the desired gene product. Another form of regulation of the expression of a gene has been observed in various viral systems. The term Zretroregulatlon has been coined for this form of expression control. Court et al.infra 1983 . As described in the prior art, the known form of retroregulatlon is uniformly found to decrease the expression of the gene product on which the retroregulator exerts its effect. Thus, known forms of retroregulation are negative retroregulation. In bacteriophage x, the regulation of the expression of the integrate gene int .mRNA sequence by a 3 terminal sequence designated S is described in Schindler and Echols, Retroregulation of the int gene of bacteriaphage k Control of translation completion Proc. Natl. Acd. of Sci. USA , 78 4475 4479 1981 . Schindler and Echols postulate that the sib region acts at the translation level of expression by preventing normal completion of protein synthesis from int mRNA. It is further postulated that the sib regulatory region corresponds to an mRNA region which is able to form a stem and loop secondary structure duplex. The sib region also is believed to include a region near the end of the gene which it regulates that provides a cleavage site for RNase III. This cleavage site for RNase III renders the mRNA sensitive to degradation by vartous exonucleases. The authors postulate that by degrading the mRNA transcript, protein synthesis of the int gene product is prematurely terminated. Thus, the reference discloses a negative regulatory role for the sib regulatory sequence, whereby the production of the polypeptide encoded by the int gene is decreased. Guarneros et al., Posttranscrtptional Control of Bacteriophage X intGene Expression From a Site Distal to the Gene, Proc. Natl. Acd. of Sci. USA , 79 238 242 1982 discloses that in bacterial hosts which lack functional RNase III, sib regulation is defective. The reference further concurs with Schindler and Echols, surpa, that the sib retroregulatory element functions post transcriptionally to prevent mRNA translation by increasing the degradation of int mRNA. Court et al., Detection Analysis of the Retroregulatory Site for the X fnt Gene, J. Mol.Biol., 166 233 240 1983 concurs with the general teachings of Guarneros et al. supra and further suggests that theRNase III sensitive structure of the retroregulatory element encoding the sib site is similar to other RNase III sensitive sites found in Escherichia coli and phage. In a general discussion of termination of transcription inE. coli, Holmes et al., Termination of Transcription in E. coli, Cell, 32 1029 1032 1983 , discloses that the primary structure of DNA sequences coding for transcription terminators includes runs of adenine and thymine base pairs on either side of symmetrical guaninecytosine rich sequences. mRNA transcripts from these sequences form secondary hairpin loop or stem and loop structures which are followed by runs of uradine residues on either side. Such terminators function as terminators in either orientation. The reference, however, emphasizes that the significance of this bidirectional terminator activity is unclear.The reference also discusses the fact that RNA transcripts of DNA sequences coding for transcription terminators that lack a stem and loop structure, are degraded more rapidly than terminators having such structures. Platt and Bear in Role of RNAPolymerase, p Factor and Ribosomes in Transcription Termination , GeneFunction in Prokaryots, Beckwith, et al. eds., Cold Spring HarborLaboratory, N.Y. 1983 generally reviews transcription termination and the role of secondary structures at the end of RNA transcripts in the regulation of transcription termination. The authors speculate on the possibility of several functions for secondary structures in mRNA, one of which includes the stabilization from degradation beyond a certain point by 3 exonucleases of completed mRNA transcripts.There is, however, no suggestion that enhancement of expression of the gene product for which the mRNA transcript codes is obtained. Gentz, R., Cloning and Analysis of strong promoters is made possible by the downstream placement of a RNA termination signal,Proc. Natl. Acad. of Sci. USA , 78 4936 4940 1981 shows that in plasmid pLBU3, addition of a strong transcription terminator derived from the bacteriophage fd, to the alpha fragment of the F galactosidase gene which lacked the lac promoter region, made possible the cloning of strong promoters for phage t5. Strong promoters were isolated by increased g galactostdase activity in an M15 a lac a fragment deletion mutant complementatlon assay.Strong transcription termination activity was shown by the fd terminator, which has a region of dyad symmetry, but this strong transcription termination activity was only reported in an orientation opposite to its native orientation in the fd genome in the system described by Gentz et al.The fd fragment in pLUB3, unlike its activity in the fd genome, cooperates with p factor to terminate transcription efficiently.There is however no teaching, nor is any evidence shown by Gentz et al., that the fd transcription terminator itself has an enhancing effect on expression of the ss galactosfldase gene. Gentz et al.attribute the assayable B galactosidase activity to the strong promoters derived from phage T5 which are ligated to 5 end of the DNA sequence coding for the fragment of the Blgalactosidase gene. Flock et al., Expression in Bacillus subtilis of the gene for human urogastrone using synthetic ribosome binding sites, Molecular and General Genetics, 195 246 251 1984 state that NplacRng a transcription terminator from bacteriaphage fd fmmedjately after the urogasterone gene in plasmid pFF810 improves the overall expression about 5 10 times in E. coli. No details of which fd transcription terminator signal, its sequence, orientation or placement relative to immediate end of the urogasterone gene are given. The fd transcription terminator of Flock et al. is used in conjunction with a putatively strong promoter which is expected to be highly active in B. subtil is. In some bacteria relatively stable high copy number mRNAs have been observed that appear to be associated with bacterial proteins or lipoproteins that occur in large amounts. For example, it has been reported that mRNA from the cry gene of Bacillus thuringiensis has a longer half life than other mRNAs of genes expressed during sporulation. Petit Galtron, M. F. and Rapoport, G.,Translation of a stable mRNA fraction from sporulating cells of B. thuringiensis in a cell free system from E. coli, Biochimie, 58 119 129 1976 . This increased mRNA stability has long been speculated as the contributing factor for the massive synthesis of the crystal protein during sporulation. In addition, the mRNA from the lipoproteln lop gene of E.coli which directs the synthesis of a major outer membrane protein is known to be relatively stable and has been shown to have sequences capable of forming extensive secondary structures in the form of stem and loop structures having AG values ranging from about 0.4 to about 21.1 kcal mole. Nakamura, K. et al., Messenger ribonucleic acid of the lipoproteln of the E. coli outer membrane II. The complete nucleotide sequence, J. Biol. Chem., 255 210 216 1980 . NeitherNakumura et al. nor Petit Galtron et al. suggest that these structures may be used to enhance expression of a desired gene product.Moreover, the nucleotide sequence for the cloned cry gene, as will be shown hereinbelow, lacks the extensive secondary structure associated with the mRNA of the gene. In summary, the prior art with respect to retroregulation in general, shows that known retroregulatory sequences have a negative effect on the expression of the gene which they retroregulate. With respect to terminators, Gentz et al. and Flock et al. indicate that in association with a DNA sequence under the control of a strong promoter, effective expression of the gene for which the DNA sequence codes may be obtained, but that in the absence of such strong terminators the activity of a putative strong promoter cannot be clearly demonstrated. Description of the Invention The inventors have discovered positive retroregulatory elements which, when ligated to a DNA sequence coding for a selected gene product, significantly increase the production of the selected gene product. As used herein, the term selected or desired gene product is meant to denote a polypeptide or protein produced by a prokaryotic or eukaryotic host by virtue of its transformation with a recombinant DNA molecule comprising a DNA sequence coding for the polypeptide or protein. Such a selected or desired gene product may be one otherwise ordinarily produced by the host, by a prokaryotic organism other than the host, or by a eukaryotic organism other than the host. The term genes as used herein means a DNA sequence coding for a polypeptide or protein. The term Nexpressionsi as used hereinbelow refers to the production of a polypeptide or protein coded for by a DNA sequence or gene.In general, the positive retroregulatory element is ligated downstream of the DNA sequence coding for the selected gene product. As used herein, the term downstream is used with respect to the direction of transcription of the messenger RNA to which the DNA sequence corresponds, transcription proceeds from upstream to downstream. The location of the positive retroregulatory element is generally 3 to the end of the coding strand of the DNA sequence coding for the selected gene product. It is well known that in order for a gene to be expressed, a translation termination codon is usually found at the 3 end of the DNA sequence coding for the selected gene product. Typically, the positive retroregulatory element is ligated to a DNA sequence coding for a selected gene product 3 to the translation termRnation sequence associated with the DNA sequence for the selected gene. Known translation termination sequences generally are nucleotide triplets.An example of such translation termination codons include those having the sequence TAG, TAA, and TGA, wherein the letters corXrespond to thymine, adenine and guanine residues which are components of the DNA molecule. Translation termination codons may appear singly, paired sequentially, or in pairs having a number of nucleotides in between. See for example Watson, J. D., MolecularBiology of the Gene, W. A. Benjamin, Inc., Menlo Park, CA 3rd ed.1977 . The translation termination signal may be native to the DNA sequence coding for the selected gene product or may itself be ligated at the 3 end of the DNA sequence coding for the selected gene product to provide a translation termination signal in a desired location. The distance of the positive retroregulatory element from the 3 end of the gene to which it is ligated may be varied while still exerting a positive or enhancing effect on the expression of the gene. As will be explained in greater detail herein below, a particular region of the positive retroregulatory element isolated from the 3 flanking region of the gene coding for the B. thuringiensis crystal protein, which is believed to form a system and loop structure at the end of the mRNA transcript for the selected gene, has been ligated from about 30 to about 300 nucleotides from the 3 end of the selected gene. In both constructions, a positive retroregulatory effect is shown on the expression of the gene to which the positive retroregulatory element was ligated. As will be shown in detail in the examples hereinbelow, positive retroregulatory elements have been shown to enhance expression of the gene to which they are ligated irrespective of the orientation of the positive retroregulatory element. It has been shown that so long as the positive retroregulatory element is ligated at the 3 end of a selected gene, the positive retroregulatory element may have a 3 5 orientation or a 5 3 orientation and still exert an enhancing effect on expression of the selected gene. The positive retroregulatory element according to the invention may be characterized structurally as a DNA sequence which transcribes for a corresponding RNA transcript which is capable of forming a stem and loop structure having a Gibbs free energy constant AG of about 30.4 Kcal as determined by the methods of Tinoco, I.et al., Nature New Biology, 246 40 41 1973 . In one effective embodiment, the positive retroregulatory element according to the invention is a DNA sequence that transcribes for a corresponding RNA transcript that is capable of forming a stem and loop structure which has a guanine cytosine residue content of about 43 . It is expected that such stem and loop structures having guanine cytosine contents of from about 40X to about 45X will also be effective positive retroregulatory elements. Such positive retroregulatory elements are intended to be within the scope of the invention. In a preferred embodiment of the invention, the positive retroregulatory element is characterized as a DNA sequence having an inverted repeat sequence including the deoxyribonucleotide sequence AtAAC66ACATCACÇTCCATTGAAACG6AGTGAT6TCCGTTTT wherein the underlined portion of the sequence comprises the inverted repeat sequence, The corresponding mRNA transcript for the above mentioned inverted repeat sequence has the ribonucleotide sequence AAAACGGAAUCACCUCCAUUGAAACGGAG UGAUGUCCGUUUU wherein the undermined portion of the ribonucleotfde sequence is the inverted repeat sequence. It is expected that some variants of the above mentioned deoxynucleotide sequence and the corresponding RNA transcript which can arise either through random mutation, point mutation, addition mutation, deletion mutation, synthetic ol i gonucl eotlde directed mutagenesis or construction of synthetic oligonucleotides may have inverted repeated sequences that will preserve the positive retroregulatory effect of the positive retroregulatory element. Such variants that preserve the positive retroregulatory effect of the positive retroregulatory element are clearly within the scope of the invention. As will be described in greater detail hereinbelow, the positive retroregulatory element according to the invention may be Ssolated from a portion of the 3 flanking region of the gene coding for the B. thuriniensis crystal protein cry gene . The 3 flanking region may be purified using conventional gel electrophoresis techniques and cloned into an appropriate host for multiplication. In one embodiment of the invention, as further described hereinbelow in the examples, a portion of the 3 flanking sequence of the cry gene which includes the transcription terminator signal thereof, was isolated, purified and cloned. This portion of the cry gene, about 382 nucleotides in length, includes a positive retroregulatory element as previously described hereinabove and portions of the 3 flanking region of the cry gene thus have a positive retroregulatory effect. When the positive retroregulatory element is ligated to aDNA sequence expressionable for a selected gene product in the proper relationship thereto, as described hereinabove, expression of the selected gene product is enhanced. Thus, the invention includes a method of enhancing expression of a selected gene product.As used herein the term DNA sequence expressionable for a selected gene product ls intended to mean a selected gene, for example betalactamase or interleukinw2, having associated therewlth a promoter, transcription start signal, ribosome binding site, translation start signal and translation termination signal all the proper relationship and reading frame such that the product for which the DNA sequence codes may be expressed.Numerous appropriate promoters are well known to those skilled in the art to which the invention pertains, and include promoters derived from E. coli including the tryptophan promoter, lac promoter operator and numerous other promoters well known to those skilled in the art Bacillus promoters including the penP promoter and various viral promoters such as for example the Sp82 promoter and PL promoter of phage k. Viral promoters may be recognized by various cells depending upon the particular cell it ordinarily infects. The selected DNA sequence may further include a signal sequence, for example, the penP signal sequence such that the expressed gene product is secreted. If the selected DNA including a signal sequence is expressed in E. coli, the gene product will generally be secreted into the periplasmic space of the cell, and can be released from the periplasmic space by sonication. If the selected DNA including a signal sequence is expressed by B. subtilis for example, the selected gene product will generally be secreted into the culture medium and may be recovered therefrom. The term DNA sequence expressionable for a selected gene product is furthermore intended to include factors required for expression of the gene product for which the DNA sequence codes, such as enzymes including functional RNA polymerase, transfer RNAs, amino acids, ribosomes and other elements necessary for the transcription and translation of the selected DNA sequence. From the foregoing it will be understood that a DNA sequence expressionable for a selected gene product includes a vector such as a plasmid and a host cell transformed thereby which is capable of expressing the gene and forming the gene product coded by the gene. Examples of such plasmids and hosts are well known to those skilled in the art and are exemplified in detail in the examples hereinbelow. Among appropriate hosts for plasmids carrying a DNA sequence expressionable for a selected gene product are prokaryotic and eukaryotic cells. Prokaryotes may be defined as organisms that lack a true nucleus, the nuclear membrane being absent and the nuclear structures being collected in a nuclear region or nucleold. The chromosomes of prokaryote are generally not associated with proteins. Among appropriate prokaryotic cells as hosts for plasmids are both Gram positive and Gram negative bacteria. By the terms Grampositive and Gram negative is meant cells capable of taking up and retaining Gram stain and cells incapable of retaining Gram statn respectively. Among appropriate Gram positive bacteria are those belonging to the genus Bacillus and in particular B. subtilis. B.subtilis strain PSL1 BGSC number IA510 are particularly preferred. Among appropriate Gram negative bacteria are those belonging to the genus EscherSchia, and in particular E. coli strain MM294 andCS412. Eukaryotic cells may be defined as cells having a true nucleus bound by a nuclear membrane within which are found chromosomes usually bound to proteins. Included in eukaryotic cells are plant animal and fungal cells. Among the fungal cells are yeasts, and in particular Saccharomyces and especially Saccharomyces cerevisiae are useful in the practice of the invention. Appropriate plasmids for carrying DNA sequences expressionable for a selected gene product are those capable of transforming a host cell such that the DNA sequence is expressed thereby. In general, a DNA sequence expressionable for a selected gene product ligated to a positive retroregulatory element may be placed in any plasmid capable of expressing the gene product in an appropriate transformed host. Thus the invention includes plasmids carrying DNA sequences expressionable for a selected gene product ligated to a positive retroregulatory element as well as the host cell transformed therewith. Plasmids capable of transforming E. coli tnclude for exampleCol El type plasmids in general. Other appropriate plasmids for transforming E. coli include pSC101, pSF2124, pMB8, pMB9, pACYC184, pACYCl77, pCKl, R6K, pBR312, pBR313, pBR317, pBR318, pBR320, pBR321, pBR322, pBR333, pBR341, pBR345, pBR350, pBR351, pML2, pML21, ColElAP, RSF1010, pVH51, pVHl51, and pVH153. See, Recombinant Molecules Impact on Science and Society, Beers, R. F. and Bassett, E. G., eds. RavenPress, New York 1977 . The plasmids of the type pBR include pBR325, pBR327 and pBR328. See, Soberon et al., Gene, 9 287 305 1980 .Other appropriate plasmids are described in DNA Insertion Elements, Plasmids and Episomes, Bukhari et al. eds Cold Spring HarborLaboratory 1976 . Plasmids capable of transforming B. subtilis include pC194, pC221, pC223, pUB112, pal27, pE194, pUB110, pSA0501, pSA2100, pTP4, pTP5 see Gryczan, T. J., Molecttlar Cloning in B. subtilis inThe Molecular Biology of the Bacilli, Dubnau, D., Ed., Academic Press,Inc., New York, 1982, p. 310 and their derivatives. Plasmids capable of transforming both B. subtilis and E.coli that may be used in the practice of the invention include pDH5060, pLP1201 Ostroff et al., infra, pO62165 Gray, 0. and Chang,S., Molecular cloning and expression of B. licheniformis ss lactamase gene in E. coli and B. subtilis , J. Bacteriol., 145 422 428 1982 , pHV11, pHV12, pHV14, pHVl6 Ehrlich, S. D., DNA cloning in B. subtilis , Proc. Natl. Acad. Sci. USA, 75 1433 1436 1978 , and pSYC310 2 McLaughlin et al., infra. See also Old, R. W. andPrimrose, S. B., Plasmid vectors for cloning in microbes other thanE. coli , Principles of Gene Manipulation 2nd ed., Carr, N.6,Ingraham, L. L., and Rittenberg, S. C., Eds, University of Ca. Press,Berkeley, 1981, p. 48 . Plasmids capable of transforming S. cerevisiae include pMP78, YEpl3, pBTIl, pLC544, YEp2, YRp17, pRB8 YIp30 , pBTI7, pBTI9, pBTI10, pAC1, pSLel, pJDB219, pDB248, and YRp7. The DNA sequence coding for an expressionable gene product which is ligated to the positive retroregulatory element may be cistronic, c, coding for a single polypeptide or polycistronic, i.e., coding for a plurality of polypeptides, the mRNA for the polycistronje DNA being under the transcriptional control of a single promoter. Such polycystronic DNA sequences are well known to those skilled in the art and include, for example, the genes for tryptophan biosynthesis under the control of the tryptophan operon in E. coli,Polycistronic genes may also be artificially formed by ligating a series of desired genes together under the control of a single promoter.See for example Flock, J. et al., supra 1984 . The selected DNA sequence expressionable for a desired gene product ligated to a positive retroregulatory element may be homologous to the DNA of the host or alternatively heterologous to theDNA of the host. Thus, for example, the selected DNA sequence may be derived from an organism that is of the same species as the host cell that has been transformedito express the gene product, in which case, the selected DNA sequence as used herein is termed homologous. An example of an homologous gene sequence expressionable for a selected gene product according to the invention is the enhanced expression in an E. coli host trassformed by a plasmid bearing a gene native to E.coli. An example is the production of E. coli p galactosidase wherein the enhanced expression is mediated by the ligation of a positive retroregulatory element at the 3 end of the ss galactosidase, As mentioned above, the selected DNA sequence expressionable for a desired polypeptide ligated to a positive retroregulatory element may alternatively be heterologous to the host. Numerous examples of such heterologous enhanced expression are given in detail hereinbelow and include the enhanced expression, under mediation of a positive retroregulatory element, of the B. f licheniformis penicillinase gene in B. subtilis eukaryotic genes such as mammalian interleukin 2 and mutated interleukin 2 in prokaryotic microorganisms such as B. subtilis and E. coli. In another aspect of the invention the positive retroregulatory element may be used in a method to extend the halflife of the mRNA transcript encoded by a selected DNA sequence. As is shown in detail in the examples, when the positive retroregulatory element is ligated at the 3 end of a selected DNA sequence, the halflife of the mRNA transcript encoded by the selected DNA sequence is extended. The mechanism by which the positive retroregulatory element exerts this mRNA half life extension, is not entirely understood. One possible explanation for the observed mRNA half life extension is that the RNA transcript of the positive retroregulatory element assumes a secondary structure that prevents enzymatic hydrolysis of the mRNA transcript initiated at the 3 end of the transcript.The method of extending the half life of mRNA transcripts has been demonstrated using the positive retroregulatory element in both its 3 to 5 and 5 to 3 orientations. The positive retroregulatory element may be used to the extend the half life of the mRNA transcript encoded by a selected DNA sequence when the DNA sequence is heterologous to the host organism.Furthermore, the half life extension of the mRNA transcript provided by the use of the positive retroregulatory element, occurs even where the promotor, selected DNA sequence and positive retroregulator are all heterologous to the host organism. Contemplated within the scope of the invention are methods for mRNA half life extension in which the promotor, selected DNA, or positive retroregulatory element are each either singly or taken together not heterologous to the host organism. mRNA transcripts encoded by a selected DNA sequence having the positive retroregulatory element at the 3 end thereof, have been demonstrated to have extended half lives in microorganisms in which they have been transcribed. Such microorganisms thus provide a means for transcribing a selected DNA into RNA. Other means for transcribing a selected DNA into RNA are contemplated to be useful within the scope of the invention. Thus, eukaryotic cells such asSaccharomyces or Xenopus oocytes are considered as such means for transcribing DNA.In addition, cell free systems having therein the components necessary for RNA transcription such as buffers, DNA dependent RNA polymerase, ions and ribonucleotide triphosphates are contemplated to be useful for transcribing a selected DNA sequence having a positive retroregulatory element ligated at the 3 end thereof to form an RNA having an extended half life. The extension of the half life of the mRNA transcribed from a selected DNA sequence having the positive retroregulatory element at the 3 end thereof, may be measured directly by hybridization of radiolabeled DNA probes to the mRNA of cells which have been treated with transcription inhibitors such as rifampicin. It has been shown that enchanced expression of a selected DNA sequence obtained by use of the positive retroregulatory element correlates substantially with the extension of the half life of tie mRNA encoded by the selected DNA sequence. From the foregoing summary of the invention, it will be clear to those ordinarily skilled in the art that the inventors have provided positive retroregulatory elements, plasmids carrying the positive retroregulatory elements ligated to a DNA sequence expressionable for a selected gene product such that expression of the selected gene product is enhanced, cells which transformed by such plasmids express the selected gene product at enhanced levels, and the selected gene products so expressed. It will furthermore be clear to those skilled in the art that the inventors have provided a general method for enhancing the expression of a selected gene product, the method comprising providing a DNA sequence expressionable for a selected gene product and ligating a positive retroregulatory element to the DNA sequence in a relationship thereto whereby expression of the selected gene product is enhanced. The general method, described hereinabove and in greater detail hereinbelow in the examples that follow, is effective to enhance expression of DNA sequences coding for a desired gene product in prokaryotic and eukaryotic hosts, and appears to be effective whether the DNA sequence to be expressed is homolgous or heterologous to the host cell that expresses it. The invention also includes the gene products produced by the general method for enhancing expression of a selected gene product. The following examples are intended by the inventors to be merely exemplary of the invention and are not intended by the inventors to be limiting. As mentioned above, the invention described herein and claimed below is broadly applicable to the enhanced expression of numerous gene products by numerous cell types, a fact which will be readily apparent to the ordinarily skilled practitioner. The examples hereinbelow are merely intended to provide a detailed and practical description of the invention as applied to the cells and plasmids exemplified below. Brief Description of the Drawings Figure 1 is a schematic representation of the restriction map of the recombinant plasmid pESl bearing the cloned cry gene fromB. thuringiensis. Transcription of the cry gene originates at the site about 350 bp upstream of the fourth EcoRI site and terminates about 360 bp downstream of the third PvuII site shown in Fig. 1. The nucleotide sequence of the PvuII NdeI restriction fragment carrying the transcriptional terminator of cry is also shown in Fig. 1. There is a inverted repeat sequence located approximately 40 bp upstream of the NdeI site. An mRNA transcript made from this region can potentially form a stem and loop structure. Figure 2 shows the expected stem and loop structure of theRNA coded by the inverted repeat sequences indicated in Fig. 1. The stem and loop structure is relatively rich in G C, about 44 G C, with a predicted AG of about 30.4 Kcal as calculated by the rules ofTinoco, et al., supra. Figure 3 is a schematic representation of plasmid pSYC667. Figure 4 is a schematic representation of plasmids pHYC70 and pHCW70. Figure 5 is a schematic representation of plasmids pHCW300 and pHCW301 and the plasmids from which they were constructed. Figure 6 is a schematic representation of plasmid pFC54.t.None of the plasmids shown in the figure are intended to be scale drawings. Relative positions of endonuclease positions and relevant coding regions of the plasmids are depicted. Figure 7 is a graph of the decay of the P mRNAs produced by E. coli and B. subtilis harboring the plasmids pSYC667 and pHCW A3. The following is a list of restriction endonucleases that were used in the examples appearing hereinbelow. The left column of the list is the commonly used name of the particular endonucleases and the right column is the organism from which it was originally derived. The listed restriction endonucleases were used under the buffer and temperature conditions recommended by the supplier.Restriction endonucleases are available from numerous commercial vendors including New England 810Labs, Beverly, Massachusetts, USA andBethesda Research Laboratories, Gaithersberg, Maryland USA. The identity of the nucleotide sequences at which particular restriction endonucleases cut may be found in product catalogues of suppliers of the enzymes, as well as standard reference texts.Restriction Enconuclease Organism SourceAvaI Anabaena variabilisAluII Arthobacter luteusBamHI Bacillus amyloltquefactens Bamil Bacillus aneurinolyticusBclI Bacillus caldolyticus Clal Caryophanon latumEcoRI Escherichia coilHhaI Haemophilus haemolyti cus HincII Haemophilus haemolyticus HindlIl Haemophilus haemolyttcusNdeI Neisseria denitrificansNruI Nocardia rubraPstI Escherichia coli PvuII Proteus vulgaris Restriction Enconuclease Organism SourceSau3AI Staphylococcus aureusSmaI Serratia marcescensSphI Streptomyces phaeochromogenesStuII Steptomyces RTI ID 19.2 tubertidicus XbaI Xanthomonas badrii Certain phrases and abbreviations are used herein. Unless otherwise noted, the meaning intended are as follows As used herein, the term penP is meant to denote the prepenicillinase gene of B. licheniformis strain 749 C, or, where clear from the context, a relevant portion thereof. The nucleotide sequence of penP has been published by Kroyer, J.s and Chang, S., Gene, 15 343347 1981 , and Neugebauer, K., Sprengel, R., and Schaller, H., Nucl.Acids Res., 9 2577 2589 1981 . As used herein, codon means, interchangeably, i a triplet of ribonucleotides in an mRNA which is translated into an amino acid in a polypeptide or a code for initiation or termination of translation, or 11 a triplet of deoxyribonucleotides in a gene whose complementary triplet is transcribed into a triplet of ribonucleotides in an mRNA which, in turn, is translated into an amino acid in a polypeptide or a code for initiation or termination or translation.Thus, for example, 5 TCC 3 and 5 UCC 3 are both codons for serine, as the term codon is used herein. As used herein, nucleotide , deoxynucleotide , and deoxyribonucleotide all mean deoxyrlbonucleotide. dNTP or NTP means any of the deoxyribonucleotide triphosphates, i.e., ATP, GTP, CTP or TTP. bp means base pair, and kb means kilobase pairs. ZPolypeptide means any peptide with two or more amino acids, including proteins. Coding sequence or DNA coding sequence means a DNA sequence encoding a polypeptide. ATCC means American Type Culture Collection, Rockville,Maryland USA. When used in connection with a number, for example ATCC 37017 , ATCC refers to the American Type Culture Collection accession number for an organism on deposit with the ATCC. Operably linked when used in regard to DNA sequence refers to the situation wherein the sequences are juxtaposed in such a manner so as to permit their ordinary functionality. For example, a promoter operably linked to a coding sequence refers to those linkages where the promoter is capable of controlling the expression of the sequence. The promoters operably linked to a ribosome binding coding sequence has the same significance i.e., it permits the ribosome binding site RBS to be positioned in the transcript so as to participate in the Initiation of the translation of the RNA transcript. An RBS operably linked to a start codon is positioned so as to permit the start of translation at this codon. The methods of the present invention make use of techniques or genetic engineering and molecular cloning. General techniques of genetic engineering and molecular cloning are included in Manlatis, T., Fritsch, E. F., and Sambrook, J., Molecular Cloning A LaboratoryManual, Cold Spring Harbor Laboratory, 1982, and Methods inEnzymology, Volume 68, Recombinant DNA, Wu, R., editor , AcademicPress, New York, 1979. Oligonucleotide synthesis may be carried out by a number of methods including those disclosed in U.S. Patent 4,415,734, and inMatteuci et al., J. Am. Chem. Soc., 103 11 3185 3191 1981 , Adams et al., J. Am. Chem. Soc., 105 3 661 663 1983, and Bemcage et al.,Tetrahedron Letters, 22 20 1859 1867 1981 . Example IPreparation of 3 Expresston Enhancement FragmentsA. pESI Plasmid pESI was prepared as described in published EuropeanPatent Application Publication 82302137.3, which is hereinIncorporated by reference, from plasmid pBR322 ATCC 37017 and a digest of the large plasmid fragment of B. thuringiensis var. KurstakiHD 1. pESI Is also described in H. Schnepf and H. Whiteley, Proc.Natl. Acad. Sci. USA , 78 2893 2897 1981 , and U.S. Patent 4,448,885 which is wherein incorporated by reference. pESl in E.. coli K 12 HB101 has been deposited by the assignee of the above cited European application at the American Type Culture Collection, Rockvllle, Maryland under the terms of the Budapest Treaty. The deposit number is ATCC 31995. B. thuringiensis var. Kurstaki strain HD 1 Dipel is available from the Northern Regional Research Laboratory CultureCollection in Peorla, Illinois under no. B3792. B. thuringiensis var. Kurstaki HD 1 is available from the American Type Culture Collection under accession number 1615. As described in Schnepf and Whiteley, supra, and in H. Wong et al., Jour. Bill. Chem., 258 1960 1983 , pES1 has inserted into the BamHI slate on its pBR322 parent, a Sau3Al fragment from the 30megadalton plasmid, the 47 megadalton plasmid, or both such plasmids from B. thuringiensis. The Sau3Al fragment includes a gene coding for the delta endotoxin crystal protein of B. thurlngiensis cry gene . This gene is expressed to form a biologically active crystal proteinIn E. coli transformed with the plasmid pESl. B. M13mp8 EP1 clone of 3 end of cry gene Plasmid pES1 was digested with PvuII and EcoRI and the resulting 2 kb fragment was isolated by agarose gel electrophoresis, Maniatfs et al. supra 1982 p. 164 with the single exception that 1 10X TEA 4 mM Tris. 0.1 mM Na EDTA 2H20, 0.5 mM sodium acetate was the buffer used. This fragment is from the 3 end of the crystal protein gene and includes the transcription termination signal of the crystal gene. The map location and a portion of the sequence of the fragment, including the translation termination codon to just after the transcription termination signal, is shown In Fig. 1. Using conventional ligation methods, the 2 kb fragment was then cloned into Replicative Form RF DNA of phage M13mp8 available from Bethesada Research Laboratories Inc., Gaithersberg, Maryland,USA that had been previously digested with SmaI and EcoRI. The resulting plasmid is designated M13mp8 EP1, E. coil strain JM103 was transformed with M13mp8 EPI plasmid.C. M13mp9 NP3 subclone of 3 end of cry gene M13mp9 NP3 was prepared as follows M13mp8 EP1 RF DNA was digested with NdeI. The NdeI ends were made blunt ended with E. coliDNA polymerase I Klenow fragment and dNTPs. The blunt ended DNA molecules were further digested with BamHI. A 382 base pair bp fragment designated NP3 carrying the transcription termination signal of the cry gene was then isolated by acrylamide gel electrophoresis using the method described in Maxim et al., Proc. Natl. Acad. Sci.USA, 74 560 564 1979 . NP3 was subcloned into a BamHI SmaI digestedM13mp9 RF DNA to form M13mp9 NP3. E. coli dM103 was transformed with M13mp9 NP3. The fragment designed NP3 cloned into M13mp9 NP3 is shown inFig. 1. As can be seen in the figure, the fragment cloned into M13mp9NP3 lacks approximately 1.6 kb of DNA present between the NdeI andEcoRI sites downstream of the transcription termination signal inM13mp8 EP1. Example IISynthetic Genes Comprising Positive Retroregulatory ElementsA. pSYC667 Plasmid pSYC667 is capable of replication in E. coli and B.subtilis see Fig. 3 . It contains the gene for prepeniclllinase ç P from B. licheniformis 749 C which is expressed under tne influence of a specifically mutated signal sequence for penP In E.coil and B. subtilis that have been transformed with the plasmid.Mature penicillinase is secreted extracellularly into the medium by B.subtilis transformed with pSYC667. Mature penicillinase is secreted into the periplasmle space of E. coll transformed with pSYC667 and can be released from the periplasmic space by osmotic shock according to the method of Nosel and Heppel, J. Biol. Chem., 241 3055 3062 1966 or Lunn and Piglet, J. B1ol. Chem., 257 11424 11430 1962 . pSYC667 was constructed as follows B. licheniformis penicillinase gene has been sequenced. SeeKroyer, J., and Chang, S., Gene, 15 343 347 1981 , and Neugebauer,K., et al., Nucl. Acids Res., 9 2577 2588 1981 . The native signal sequence contains the codon TGC which codes for a cysteine at amino acid position 27. The cysteine residue at position 27 is modified as part of a sequence of events leading to formation of the membranebound lipoprotein form of penicillinase. See Nielsen, J. B. K, Caulfield, M. P., and Lampen, J. O., Proc. Natl. Aca. Sci. USA , 78 3511 3515 1981 , and Lai, J. S., Sarvas, M., Brammar, W. J. Neugebauer, K., and Wu, H. C., Proc. Nati. Aca. Sci. USA , 78 35063510 1981 . To specifically alter this biosynthetic pathway, and shunt more of the protein to the extracellular form secreted from the cell in the case of Gram positive bacteria such as B. subtilis or into the periplasmic space in the case of Gram negative bacteria such as E. coil , it is necessary to mutate the sequence In the penicillinase signal sequence gene coding for cysteine at this position. Because of its simplicity and efficiency, the method of primer directed mutagenesis see Zoller, M. J., and Smith, M., Nucl. Acids Res., 10 6487 6500 1982 was used for the construction of the cysteine to serine mutation. A DNA fragment containing wild type penicillinase penP gene sequence was isolated. Specifically the DNA fragment located between the HindIll and BamHI sites was excised from plasmid pSYC310 2. See McLaughlin, et al. Nucl. Acids Res., 3905 3919 1982 . Plasmid pSYC31O 2 is capable of replicating in both B.subtilis and E. coll. It carries the wild type penP gene from B. licheniformis 749 C on the H1ndIII BamHI fragment. Those skilled in the art will realize that the wild type penP gene could have been excised from other engineered recombinant plasmids that carry it. One such plasmid is B. subtilis plasmid pOG2165. The excised HindIIIBamHI DNA fragment from pSYC310 2 was purified by acrylamide gel elution and then ligated to RF DNA of coliphage M13mp9. See Viera, J., and Messing, J., Gene, 19 259 268 1982 and Messing, J., and Vi era, J., Gene, 19 269 276 1982 . Specifically the purified HindIll BamMi fragment from pSYC310 2 was ligated to M13mp9, obtained from Bethesda ResearchLaboratories, Inc., P.O. Box 577, Gaithersburg, MD., that had previously been digested with restriction endonucleases HindIlI andBamHI. The double stranded phage DNA was transformed into E. coli JM103, and the cells were cultured. A clone transformed with recombinant phage carrying the penP gene, i.e., recombinant phage M13CM1, was identified and single stranded phage DNA was prepared from this clone. The methods used are described in Zoller, M. J., andSmith, M., Nucleic Acids Res., 10 6487 6500 1982 . A 15 nucleotide synthetic fragment 5 GTTAGCGGATCCTGC 3 , made by the phosphotriester method of Narang, S. A., Hsiung, H. M., and Brousseau, R., in Methods in Enzymology, 68 90 97 R. Wu, editor Academic Press 1979 , was first phosphorylated at the 5 end with ATP and T4 polynucleotide kinase and then employed as a primer to initiate the synthesis of the complementary strand in vitro after the 5 phosphorylated primer had been annealed to the template M13 CMl DNA.The primer was extended using DNA polymerase I Klenow fragment with all four dNTP s in the presence of T4 ligase. This primer complements the anti sense strand of the penP signal sequence gene segment corresponding to the codons for the five amino acids from positions 25 to 29, except that the middle nucleotide in the synthetic primer is a mismatched nucleotide, that does not complement the corresponding nucleotide, G, in the wild type penP gene template. Incorporation of the mismatched sequence into the penP gene causes conversion of the cysteine TGC codon to serine TCC codon at position 27. The alteration on the encoded peptide is essentially a conversion of the SH group on the cysteine27 to the OH group of the serine27. At the nucelotide level, a mutant gains a BamHI site GGATCC and loses the HhaI GCGC at the mutation locus. The presence of a new BamHI site was the phenotype used to identify the mutants carrying the G to cu nucleotide mutation. In constructing . the ,cysteine to serine mutation, complementary minus strands were synthesized by prtmer extension reaction using Klenow fragment of E. coil DNA polymerase I on the M13 CMl phage DNA template. See Zoller, M. J., and Smith, M., Nucl. AcidsRes., 10 6487 6500 1982 . In the presence of T4 DNA ligase in this reaction, a fraction of the DNA molecules was converted to doublestranded, covalently closed relaxed circles. These molecules were separated from other molecules, which either were incompletely extended by polymerase or failed to be ligated due to the Incomplete kinase reaction of the primer. Separation was accomplished by agarose gel electrophoresis.This was carried out by applying the reaction mixture on a 0.8 agarose gel in the presence of 2 micrograms ml of ethidium bromide. The band containing covalently closed circular DNA was excised and DNA recovered. Plasmid pSYC667 is the same as pSYC660 except that, in place of the BamHI recognition sequence 5 GGATCC 3 at the end of the approximately 1320 bp HindIII BamHI fragment in pSYC660, pSYC667 has the sequence 5 GGATCGATCC 3 . Plasmid pSYC667 retains the PstI and BglII recognition sites of plasmid pSYC660. Similarly, the HindIII BamHI fragment ofM13penPS27P28 that contains the penPS27P28 gene has a PstI site and BS1II it site at the same locations as the PstI site and Bgl II site, respectively, in the HindIII BamHI penPS27 containing fragment in pSYC667. pSYC667 has been deposited at the ATCC under the terms of the Budapest Treaty and has been assigned ATCC No. 39758. B. pHCW3 A3 Plasmid pSYC667 was digested with BclI, which cuts between the translation termination codon and transcription termination signalIn the B. lichenlformis penicillinase gene, The BclI ends were filled in using DNA polymerase I Klenow fragment and the four dNTP s. Such methods are well knows to those skilled in the art E Maniatis, et al., surpa 1982 . Enzymes were Inactivated by phenol extraction and the DNA was recovered by ethanol precipitation. The DNA was then further digested with Nrul, and extracted twice with phenol and then twice with ether. Nrul cuts downstream of the transcription termination signal of the B. llcheniformfsE penicillinase gene in pSYC667. Thus, the large NruI BclI cleaved fragment of pSYC667 contains the entire penicillinase gene through the translation termination codon but not including the transcription termination signal see Fig. 3 . M13mp9 NP3 was cut with EcoRI and BamHI and a 400 bp fragment was isolated with standard methods using a 1X agarose gel and purified with DE 52 chromatography. Such methods are described inManiatis, et al. supra, 104 1982 . The EcoRI BamHI 400 bp fragment from M13mp9 NP3 was then ligated into the large NruI 8clI fragment of pSYC667 using standard procedures. The resulting plasmids were transformed into E. coli K12 CS412 using the method of Cohen et al. Proc. Natl. Acad. Sci. USA , 69 2110 1973 see also Maniatis et al. supra, 1982, p.250 . Transformants were selected for resistance on LB medium plates containing 50 llg ml ampicillin. Maniatis et al. supra, 440 1982 .Plasmid DNA was prepared from selected transformants by the mini prep method of Birnboim and Doly Nucl. Acids Res., 7 1513 1979 and was screened for a plasmid with the expected 1.6 kb EcoRI fragment, approximately 0.4 kb of the fragment derived from M13mp9 NP3 and approximately 1.2 kb from pSYC667. One transformant harbors a plasmid which was designated pHCW A3. 6.1. Transformation of B. subtilis with pHCW A3 pHCW A3 was also transformed Into B. subtilis, PSL1,Bacillus Genetic Stock Center No. It510 Bacillus Genetic StockCenter, Department of Microbiology, Ohio State University, Columbus,Ohio, U.S.A. . B. subtilis PSL1 is Let , Arg , Thr and recE4. A culture of B. subtilis PSL1 cells was made competent for transformation using a technique related to that described byAnagnostopoulos and Spizizen, J. Bacteriol., 741 746 1961 . 10 X Spizizen I Minimal Solution was prepared by mixing, In a total solution volume of 1 liter made up with distilled water, 20 gm NH4 2504, 140 gm K2HP04, 60 gm KH2PO4 and 10 gm Na citrate. Spizizen I Medium was prepared by mixing 2.05 ml of 1 M MgS04 6 ml of 50X w w glucose 5 ml of 10 w w yeast extract 5 ml of 2X w w casein hydrolysate for each amino acid required by the strain to be transformed, 2.5 ml of 1X w w solution of the amino acid 50 ml of 10 x Spizizen I Minimal Solution and enough distilled water to bring the solution volume to 500 ml. For a. subtilis PSL1, the required amino acids are threonine, arginlneand leucine. Spizizen II Medium was prepared by adding 0.25 ml of 1 MCaCl2 and 1 ml of 1 M MgCl2 to 500 ml of Spizizen I Medium. 30 ml of Spizizen I Medium was Inoculated with a colony of spores of the B. subtilis strain to be transformed and was grown overnight 16 20 hours at 370C. 15 ml of the overnight culture were then inoculated into 135 ml of Spizizen I Medium in a 2800 ml flask and grown at 370C. The optical density at 600 nm O.D. of the culture was measured after 1.5 to 2 hours, and then every 15 minutes until the culture was found to be in late log phase on the basis of an increase in 0.D. of less than 5X between 15 minute O.D. readings. 50 ml of late log phase culture was then inoculated into 450 ml of Spizizen II Medium in a 2800 ml flask and grown at 370C for 1.5 hours. After the 1.5 hour growth, cells were spun down by centrifugatrion at 5000 rpm for 10 minutes at 40C. The pellet from centrifugation was then resuspended in 45 ml of supernatant, to which 6 ml of 80X v v sterile glycerol was then added, just prior to freezing the culture in a dry ice ethanol bath 700C . The cells in the frozen culture were competent cells, suitable for transforiation by the desired plasmid, as follows 0.5 ml 0.6 ml of the frozen, competent cell containing culture was thawed on ice, and 5 microliter to 50 microliter of solution containing the plasmid to be transformed into the cells was combined with the thawed culture. The resulting mixture was shaken at 370C for 2 hours, during which transformation of plasmids and expression of genes on them occurred. Finally, for selection, small aliquots such as about 5 microliter to about 200 microliter of the culture of transformed cells were transferred to plates containing the desired antibiotic or antibiotics for selection. A mini prep of plasmid pHCW A3 was prepared as described above from E. coli K 12 CS4 12 transformed with the plasmid. To l ml of culture of competent B. subtilis PSL1, 5 microliters of mini prep plasmid DNA solution was added. The mixture was incubated at 37dC for 2 hours. Aliquots of the mixture were then plated and incubated at 370C overnight on rich medium agar plates beef extract, 1.5 g liter yeast extract, 3.0 g liter peptone, 6.0 g liter agar 15.0 g liter to which 5 micrograms ml chloramphenicol had been added. A chloramphenicol resistant colony was picked and inoculated into 5 ml of 2 x LB medium containing 5 sg ml chloramphenicol the culture was incubated overnight at 370C with shaking. Subcultures of cultures of B. subtilis PSL1 transformed with pHCW A3, and prepared as above, have been deposited in the CMCC under collection number 2120.C. pLW1 Plasmid pLW1 is a pBR322 derivative capable of replication in E. coli containing a tetracycline resistance gene the E. coll trp promoter, ribosome binding site RBS fragment and a 706 bp HindIIt PstI DNA fragment which includes the gene for human interleukin 2 IL2 Rosenberg, S. A. et al. Science, 223 1412 1415 1984 . pLW1 has been deposited at the ATCC under terms of the Budapest Treaty and assigned ATCC No. 39405.D. pLW45 Plasmid pLW45 ib a pBR322 derivative capable of replication in E. coli containing a tetracycline resistance gene and the E. coli trp promoter. The plasmid contains, on a 706 bp HindIII Pstl fragment, a gene for a modified IL 2 protein. pLW45 has been deposited at the ATCC under terms of theBudapest Treaty and assigned ATCC No. 39629. The modified IL 2 it encoded by pLW45 and the uses of such modified IL 2 protein in treating human diseases involving suppression of the immune system are described in Belgian Patent Serial No.898,016, issued November 14, 1983, which is incorporated herein by reference.E. pHCW701 and pHCW702 The 400 bp EcoRI BamHI restriction fragment carrying the transcription termination signal of the cry gene was excised fromM13mp90 NP3 by digestion with EcoRI and BamHI restriction endonucleases under buffer conditions suggested by the supplier. TheEcoRI BamHI ends of the fragment were made blunt ended with KlenowPoll fragment and dNTPs. The blunt ended fragment containing the transcription terminal signal of the cry gene was isolated by acrylamide gel electrophoresis. The isolated blunt ended fragment was electroeluted and ligated using T4 ligase and ATP into plasmid pLW1 that had been previously digested with StuI restriction endonuclease. Regardless of the orientation in which the blunt ended fragment carrying the transcription termination signal is recombined with the Stul ends of pLW1, both of the original EcoRI and BamHI sites will be regenerated. One orientation results in the BamHI site being located nearer to the 3 end of the IL 2 gene and the plasmid so characterized is designated pHCW701. This orientation is similar to that found in the cry gene itself. The other orientation results in the EcoRI site being nearer to the 3 end of the IL 2 gene and this recombinant plasmid is designated pHCW702 Fig. 3 .Due to the a symmetrical location of the inverted repeat sequence in the EcoRIBamHI fragment as depicted in Fig. 1, the stem and loop structure of the positive retroregulatory element is located approximately 310 bp downstream of the BamHI site at the 3 end of the IL 2 gene in pHCW701. In pHCW702, the stem and loop structure of the positive retroregulatory element is approximately 30 bp downstream of the EcoRI site of the 3 end of the IL 2 gene. pHCW701 and pHCW702 can be easily distinguished by digesting the plasmid DNAs with restriction enzyme EcoRI and determining the size of restriction fragments using acrylamide or agarose gel electrophoresis. EcoRI digested pHCW701 releases a 960 bp restriction fragment which contains the trp promoter RBS cassette 108 bp , theIL 2 gene 450 bp and the terminator 400 bp . However, EcoRI digested pHCW702 releases a 560 bp restriction fragment which just contains only the trp promoter RBS cassette 108 bp and the IL 2 gene 450 bp .F. Construction of pHCW801 pHCW801 was constructed to assess the effect of the positive retroregulatory element on productin of the modified IL 2 by plasmid pLW45. Plasmid pHCW701 was digested with BamHI and the ends were made blunt with Klenow PolI fragment and dNTPs as previously described.The blunt ended BamHI fragment was digested with Aval. The largest resulting fragment was 2.7 kb and had one blunt BamHI end and one AvaI end. This fragment was purified by 0.8 agarose gel electrophoresis, and contains the 3 expression enhancement sequence. Plasmid pLW45 was digested with StuI and then Avail restriction endonucleases. An approximately 2.3 kb fragment was purified from the digest by agarose gel electrophoresis, electroeluted, and ligated using T4 ligase and ATP, to the fragment having a BamHI blunt end and an AvaI end, derived from plasmid pHCW701, which contained the positive retroregulatory element. Tetracycline resistant transformants were analyzed by mini prep isolation of plasmid DNA and screened for the presence of an EcoRI fragment including both the modified IL 2 gene and the retroregulatory element. Birnboim and Doly, supra 1979 . ss. pHCW301 1. Promoter 156 A promoter, recognized by 8. subtilis vegetative RNA polymerase, which is located on a 240 bp HhaI restriction fragment of the bacteriophage SP82, was first discovered by DNA restriction fragment probe analyses and B. subtilis RNA polymerase binding and initiation assay. Jones, B. B., Chan, H., Rothstein, S. Wells, R. D.and Reznikoff, W. S., Proc. Natl. Acad. Sci. USA , 74 4914 4918 1979 . The 240 bp HhaI fragment was isolated from the HhaI digested SP82 DNA by gel electrophoresis and the ends were made blunt by removal of unpaired nucleotides with Sl nuclease 2200 ml in pH 4.6 buffer containing 300 mM NaCl, 60 mM S04 and 50 mM Na acetate . Maniatis et al. supra at p. 140. The blunt end fragment was then cloned into the Hincli site of M13mp7 RF DNA obtained from BethesdaResearch Labs that had been previously digested with HincII restriction endonuclease. DNA sequence analysis indicated that a RBS sequence was located at the 3 end of the HhaI restriction fragment carrying the promoter sequence. To eliminate the RBS sequence, the 240 bp HhaI fragment was digested with Alul restriction endonuclease. A 150 bp HhaI AluI restriction fragment was isolated by acarylamide gel electrophoresis. The promoter was known to be located on this 150 bp restriction fragment by DNA sequence analysis.The 150 bp restriction fragment was then subcloned into the HincII site ofM13mp7 RF DNA which had been previously digested with HincII restriction endonuclease to form a phage designated M13mp7 p156. 2. The ribosome binding site Two synthetic oligonucleotides with the sequence 1 5 CGATAAGAGGAGGTA 3 and 2 5 AGCTTACCTCCTCTTAT 3 were made. 500 picomoles of each oligonucleotide were mixed and phosphorylated with polynucleotide kinase and ATP. The phosphorylated oligonucleotides were then annealed at 680C for 1 hour and then at 370C for 3 hours. The annealed oligomer having the sequence 5 AGCTTACCTCCTCTAAT 3 3 An GAGGAGATTAGC 5 was then cloned into the ClaI HindIII site of pUC8 41 using T4 ligase under ligation conditions at a molar ratio of the oligomer to vector of approximately 10 to 1. pUC8 is commercially available BethesdaResearch Laboratories, Gaithersberg, Maryland USA and was modified as follows to yield pUC8 41 pUC8 was digested with BamHI and the ends were made blunt with Klenow PolI fragment.White colonies of E. co7i JM103 containing the recircularized BamHI repaired pUC8 were selected. Plasmid DNA from these transformants was linearized withPstI and the ends were trimmed with Klenow PolI fragment. This procedure restores the correct reading frame for lac Z and after ligation blue transformants of Jim103 were selected. Since repair of the BamHI site generates a ClaI site in DNA prepared from a DNA methylase lacking Dam E. coli host, pUC8 41 was confirmed by ClaI linearization. Restriction enzyme analysis and nucleotide sequence determination were used to determine that the constructed recombinant plasmid designated pUC8 41 RBSl was indeed carrying a single copy of the synthetic ribosome binding site. 3. Construction of pLP1201 pl56 RBSl The synthetic ribosome binding site in pUC8 41 RBS1 was recovered by EcoRI HindlIl double digestion of the plasmid and was isolated by acrylamide gel electrophoresis. The EcoRI HindIlI fragment was subcloned into plasmid pLP1201 that had been previously digested with EcoRI and HindIII restriction endonucleases to form pLP1201 RBS1. Ostroff, G. R. and Pene, J. M. Mol. Gen. Genet., 193 306 311 1984 . E. coli strain CS412 was transformed with plasmid pLPl2Ol RBSl using conventional methods. Cohen et al. supra 1973 . E. coli transformants carrying the plasmid pLPl201 RBS1 were ampicillin resistant and tetracycline sensitive. Promoter p156 was then excised from M13mp7 p156 RF DNA usingEcoRI restriction endonuclease and subcloned into the EcoRI site of pLP1201 RBS1 that had been previously digested with EcoRI restriction endonuclease. The desired recombinant plasmid pLP1201 pl56 RBS1 confers ampicillin and tetracycline resistance to the E. coli host CS412 transformed with the plasmid. 4. Construction of plasmid pHCW300 pLP1201 RBS1 p156 IL 2 pHCW300 was constructed from pLW21 and pLP1201 RBS1 pl56. pLW21 is derived from pBR322 and contains a 570 bp EcoRI BanII sequence including a region coding for IL 2. pLW21 was constructed by ligating the 570 bp EcoRI Banli sequence, obtained by digesting pLW1 with EcoRI and Ban II endonucleases, into pBR322 previously digested to completion with EcoRI and Banal. This EcoRI BanlI fragment replaces the 485 bp EcoRI BanlI fragment in pBR322 containing a portion of the tetracycline resistance gene to yield pLW21 which is tetracycline sensitive. pLW21 was digested with HindlIl restriction endonuclease followed by digestion with NruI restriction endonuclease.Two fragments were generated and a HindIII NruI fragment approximately 400 bp in length was isolated by acrylamide gel electrophoresis. Plasmid pLP1201 RBS1 p156 was digested with HindIII and NruI restriction endonuclease to form a linearized plasmid with the region containing the tetracycline resistance gene removed. The llnearized plasmid was combined with the 400 bp HindIII NruI fragment from pLW21 and was ligated using T4 ligase to form a plasmid designated pHCW300 having the p156 promoter and RBS described above and a 400 bp sequence having a DNA sequence coding for IL 2.Tetracycline sensitive transformants of E. coli were obtained by replica plating of ampicillin resistance transformants and were screened for the presence of plasmid pHCW300 using HindIII and Avail restriction endonucleases to generate two fragments of about 7 kb and 0.85 kb. 5. Construction of plasmid pHCW301 pLP1201 RBS1 p156 IL 2 retroregulator Plasmid pHCW701 was digested with EcoRI restriction endonuclease and the ends were made blunt with Klenow Pol I fragment and dNTPs. Following digestion with HindIII restriction endonuclease, the approximately 0.85. kb fragment containing the IL 2 gene and the positive retroregulatory element was sol isolated by agarose gel electrophoresis. Plasmid pLP1201 RBS1 p156 was digested with HindIII and NruI restriction endonucleases resulting in the excision of the region containing the tetracycline resistance gene.The 0.85 kb EcoRI blunt HindIII fragment containing the It 2 gene and the positive retroregulatory element was ligated to the HindIII NruI digested pLP1201 RBS1 p156 vector using T4 ligase under ligating conditions.Tetracycline sensitive transformants of E. coli were obtained by replica plating of ampicllin resistant transformants and were screened for the plasmid pHCW301 using EcoRI restriction endonuclease. Three fragments of approximately 7 kb, .88 kb, and 156 bp corresponding to the expected sizes for the pLP1201 RBS1 fragment, IL 2 gene positive retroregulatory element fragment, and promoter fragment respectively, were generated. Example IIIEnhanced Protein Expression by Genes into Which a Positive Retroregulatory Element Has Been InsertedA. Assay procedure for B. licheniformis penicillinase expression 1. In E. coli Five ml of YT broth 8 g l tryptone, 5 g l yeast extract, 5 g 1 NaCl containing 50 Fgfml ampicillin were inoculated with individual colonies of E. coli K 12 CS412 carrying either pSYC667 or pHCW A3 and were grown overnight at 370C with shaking in a NewBrunswick rotary incubator. Cells were then pelleted at 5,000 rpm for 10 minutes in the JA 20 rotor of a Beckman Model J 21C centrifuge.The cell pellet was washed with 5 ml ice cold 50 nt phosphate buffer pH7.O. The washed cell pellet was resuspended into 0.5 ml 50 mM phosphate buffer and sonicated using a Branson 350 Sonifer at maximum output for 3 minutes in 0.5 second bursts at OOC. The sonicate was spun in an Eppendorf microfuge for 2 minutes. The resulting supernatant was used for the assay of enzyme activity. The production of B. licheniformis penicilllnase was assayed using the chromogenic B lactamase substrate PADAC available from CalBiochem by the method of Schindler and Huber. Schindler, P. andHuber, G., NUse of PADAC, A Novel Chromogenic, ss Lactamase Substrate, for the Detection of ss Lactamase Producing Organisms and Assay of ss Lactamase Inhibitors Inactivators , In Enzyme Inhibitors, Brodbeck,U., ed., Weinheim VerlagChemice, 1980, p. 169 176. PADAC substrate MW 562.7 was prepared by making a solution having OD573 1 about 27.4 SM In phosphate buffer at pH 7.0.The decrease in absorbance at 573 nm over time, after the addition of the cell extract, was measured using a Cary 219 spectrophotometer at room temperature. The results are shown in Table I. 2. In B. subtilis B. subtilis PSL1 was transformed with either pSYC667 or pHCW A3, according to the method related to Anagnostopolus andSpizizen supra 1961 described above, plated on rich medium agar described above, and grown overnight at 37 C. Five ml of 2 x LB medium containing 5 rg ml chloramphenicol was inoculated with individual transformants and grown at 370C with shaking on a NewBrunswick rotary incubator overnight. Cells were then pelleted at 5000 rpm for 10 minutes in a JA 20 rotor as above. The supernatant was used for the penicillinase assay by the same method described above for E. coli. The results are shown in Table I. TABLE I Production of the penicillinase polypeptide in E. coli and B. subtilis strains carrying penP recombinant plasmids with or without the positive retroregulatory elementHost Plasmid Specific Activit f Penicillinase 1E. coli CS412 0 pSYC667 1055 pHCW A3 2762B. subtilis lA510 0 pSYC667 3280 pHCW A3 17510 1 Specific activity is defined as nmoles of PADAC hydrolyzed per minute per mg of protein at 250CB. Assay procedure for IL 2 expression 1.In E coli A shake flask culture of E. coli K12 MM294 transformed with pHCi701, pHCW702, or pLW1 were grown in 10 ml of tryptophan trp containing N medium 0.7X Na2HP04, 0.3X KH2P04, 0.56 NaCl, 0.1X NH4Cl, 0.2X glucose, 0.5X casamlno acids, 40 zxg ml trp, and 10 Fg ml tetracycline at 370C overnight with shaking in a New Brunswick rotary incubator. Cells from a 5 ml overnight culture were pelleted by centrifugation at 5000 rpm for 10 minutes in a JA 20 rotor. The cell pellet was resuspended in 5 ml of N medium without trp containing 2 pg ml thiamine.Optical density was determined by absorbance at 600 nanometers 0D600 in a spectrophotometer. 25 ml subcultures having an 0D600 of 0.05 in N medium minus trp were set up and grown at 370C with shaking to a final OD600 of about 0.3. Controls were set up and grown under the same conditions except that the N medium lacked tryptophan. Cells were then pelleted and resuspended in IL 2 sonication buffer 50 mM Tris, pH 7.5, 50 vtM EDTA, 15X sucrose, 1XSDS to a final 0D600 10. Cells were sonicated as described above.Supernatants were assayed for the presence of IL 2 activity by the methods described in Gillis, S., et al. J. Immunol., 120, 2077 2032 1978 . Results are shown in Table II.2. Fermentation cultures E. coli K 12 MM294 1 transformed with either pHCW801 or pLW45 was fermented in a 10 liter fermentor at 370C and 350 1200 rpm with 0 2 liters per minute lpm air and 0 5 1pm oxygen dissolved oxygen at about 40 . The medium consisted of 72 mM NH4 2S04, 21.6 mM KH2PO4, 1.5 mM Na3 citrate, 1.5 mg l TK 9 Trace elements, and the following sterile additons 3 mM 0.5X MgS04, 20 mg l 1 thiamine HCl, 72 mM 0.2 M FeSO4, 5 g l 50X glucose, 70 mg l 0.5X L tryptophan, 5 mg l 1X tetracycline, and 100 ml l 20X casamino acids added at OD68o 15 20 . The inoculum was 20 mg l and the pH was controlled at 6.8 with 5N KOH. A glucose feed was also employed to maintain glucose concentration between 5 10 g l. Culture samples for SDS polyacrylamide gel electrophoretic analysis of total cell protein were withdrawn hourly from 13.7 hours to 19.7 hours. Densitometry of protein bands of the gels indicated a maximum production of IL 2 as 17.2X of total cell protein at 17.7 hours. The biological activity of IL 2 from these samples was determined as previously described for IL 2 expression in E. coli.Results are shown in Table II.3. In B. subtilis Cultures of B. subtilis lA510 transformed as described above with pHCW301 or pHCW300 were grown to an OD600 1.0 at 370C with shaking after single colony inoculation into 5 ml of LB medium containing 5 Fg ml chloramphenicol. Cells were pelleted, suspended in IL 2 sonication buffer, sonicated and assayed for IL 2 activity, as described for E. coli. Results are shown in Table II. TABLE II Production of IL 2 and modified IL 2 with and without the positive retroregulatory element 1. In E. coil 5 ml culture Specific Activity of IL 2 Produced Transformed with Induction U 150 g total Cellular Protein pLWl C 7 x 2 x 105 pHCW701 2.8 x 104 1.4 x 106 pHCW702 1.5 x 9.2 x 2. In E. coil 10 1 Fermentor Transforeed with IL 2 Produced U g dry weight pLW45 8.95 x pHCW801 1.6 x 108 3.In B. subtilis 5 al culture Specific Activity of IL 2 ProducedTransformed with U 150 g Total Cellular Protein pHCW300 1 x pHCW301 2.5 x 104 Example IVDeletion Mapping of the Positive Retrogugulatory Element Deletion studies were carried out to define more precisely the sequence within the cry terminator region responsible for the positive retroregulatory effect.Oligodeoxyribonucleotide directed site specific mutagenesis according to the method of Zoller, M. andSmith, M., Oligonucleotide directed mutagenesis of DNA fragments cloned into M13 vectors Methods in Enzymology, 100 468 500 1983 was employed to introduce two BglII restriction sites separately at the locations 80 and 20 bp upstream from the inverted repeat sequence in the cry terminator fragment depicted respectively as 81 and S2 in Fig.1. These shortened terminator containing fragments can be excised from the respective, modified M13mp9 NP3 phage genomes by BglII EcoRI digestion. They were cloned into pSYC667 at the BclI NruI slate by the procedures similar to that employed for the construction of pHCW A3, and generated plasmids pHCW A4 and pHCW A5. Data on the analysis ofB. subtilis as well as E. coli strains harboring these plasmids for their ability to express the cloned penP gene are presented In TableIII. It is clear that the shortened fragments still contain the positive retroregulatory element observed in the original fragment.Since the two newly created BglII sites are outside of the cry coding sequence, this data demonstrates that the locus that confers the enhancing activity is located in the non coding region of the cry gene, and it probably overlaps with the terminator of cry gene. B. subtilis strain PSL1 and E. coli strain CS412 were transformed with plasmids pHCW A3, pHCW A4, or pHCW A5 as described above. Penicillinase activity, assayed as described above is shown inTable III. TABLE III Synthesis of Penicillinase in E. coli and B. subtilis Strains Carry the penP Fusion Plasmids Length of cry Specific ActivityHost Strain Plasmid derived Fragments of Penicillinase 1 E. coli CS412 pHCW A3 380 2762 pHCW A4 158 2631 pHCW A5 78 2861B. subtilis lA510 pHCW A3 380 17510 pHCW A4 158 18600 pHCW AS 78 17822 1 r Penicillinase was assayed as described in Table I. Example VInsertion of Positive Retroregulatory Element Increases the Level ofProduction of a Novel Recombinant IL 2 Mutein using a PortableTemperature Regulated Control Cassette in a Temperature RegulatedColEl Copy Mutant Plasmid Vector A. Construction of plasmid pFC54.t Plasmid pFC54 encodes des Alanyl, serine125 Snterleukin 2 IL 2 under the control of the bacteriophage X PL promoter and gene N ribosome binding slate PL NRBS . The ColEl plasmid vector contains two mutations which confer a temperature sensitive copy number phenotype on the plasmid.E. coll cells harboringthis plasmid have been thermally induced to accumulate 20X of the total protein as a novel IL 2 mutein. Plasmid pFC54 was digested to completion with SphI and treated with DNA polymerase I Klenow fragment in the presence of 25SM dGTP to eliminate the 3 protruding single stranded tail. The blunt ended DNA was subsequently digested with XbaI. Plasmid pHCW801 was digested to completion with EcoRI, treated with DNA polymerase I Klenow fragment in the presence of dNTPs, and subsequently digested with XbaI. The 655 bp DNA fragment comprising the C terminal 225 bp of ser125 IL 2, 36 bp of 3 untranslated IL 2 cDNA and 394 bp of the fragment carrying the positive retroregulatory element were purified by gel electrophoresis. The pFC54 vector DNA fragments and purl fled pHCW801 fragment were ligated 1 3 molar ratio at a concentration of 30 pgDNA ml DNA using T4 DNA ligase under conditions favoring ligation of sticky ends, diluted 2.5 fold and then ligated using T4 DNA ligase under blunt end ligation conditions to favor intramolecular circle formation.The ligated DNA was digested with BanII to inactivate undesired ligation products comprised of the small and large XbaI SphI fragments of pFC54. E. coli K12 strain DG95 kN7N53cI857susP80 was transformed to AmpR with 60 ng of the ligated and digested DNA. This strain contains a lambda prophage which encodes a temperature sensitive X cI repressor, which at low temperature 30 320C is active. However, at high temperature 36 420C the repressor is inactive and transcription from the PL promoter can proceed. It is further characteristic of this strain that at elevated temperatures the prophage fails to incude. Transformants were selected for AmpR and AmpR colonies were screened for the desired 5.6 kb plasmid.Candidate plasmids were screened for release of an 1182 bp EcoRI fragment ligation of repaired EcoRI site in the fragment carrying the positive retroregulatory element to the repaired SphI site in the vector fragment was expected to regenerate an EcoRI site , retention of the XbaI site within the IL 2 coding sequence, acquisition of a uniqueBamHI site in the terminator fragment , and loss of the unique BanII site tn pFC54. A desired recombinant plasmid was isolated and was designated pFC54.t. As shown below, when E. coli K12 strain DG95 kN7N53cI857susP80 harboring plasmid pFC54.t is temperatureinduced under the proper conditions, 34X of the total cellular protein is des Alanyl, Ser125 IL 2. Plasmid pFC54.t has been deposited pursuant to the BudapestTreaty in the ATCC under accession number 39789. Plasmid pFC54.t is shown in Fig. 6. Beginning with theEcoRI restriction endonuclease site designed 5.61 0 and moving in a clockwise direction, plasmid pFC54.t comprises the components described immediately below. Coordinate 0 0.35 kb comprise an Eco EcoRI HindIII module encoding the temperature regulated promoter operator of the bacteriophage X PL promoter and the adjacent gene N ribosome binding site. The BglII recognition site at X coordinate 35715 Sanger, F. et al., J. Mol. Biol., 162 729 773 1982 has been converted to an EcoRI recognition site and the HinfI recognition site at X coordinate 35366 has been converted to a HindIII recognition site for insertion into plasmid pFC54.t. Coordinates 0.35 0.794 kb comprise the 444 bp Hindill Stul fragment of plasmid pLW46 encoding mature human des Alanyl, Serl25 IL2 mutein. Wang, A. et al., Science, 224 1431 1433 1984 . The 5 HindIII site immediately precedes the ATG initiation codon of the altered mutein Rosenberg, S. A. et al. Science, 223 1412 1416 1984 and the Stul recognition site 36 bp distal to the opal stop codon in human IL 2 has been converted to a BamHI recognition site in pFC54.t. Coordinates 0,794 1.188 kb comprise the 394 bp BamHI EcoRI DNA fragment from plasmid pHCWBOl and Includes the positive retroregulatory element from the B, thuringtensts delta enotoxin gene, Coordinates 1.188 2.05 kb comprise the 863 bp SphI repaired fAval repaired pBR322 DNA fragment pBR322 coordSnates 567 1429, respectively . Coordinates 2.05 3.48 kb comprise the 1.43 kb PuvII BamHIDNA fragment from plasmid pOP6 Gelfand, D. H., et al., Proc. Natl.Acad. Sci. USA, 75 5869 5873, 1978 Muesing, M., et al., Cell, 24 235242, 1981 encoding the 8 lactamase gene of bacterial transposonTn3. Heffron, F., et al., Cell, 18 1153 1163 1979 . Coordinates 3.48 3.95 kb comprise the .47 kb SstI PvuII DNA fragment of pOP6 Gelfand, supra 1978 . This DNA fragment contains the left boundary of bacterial transposon Tn3 coordinate 1 270,Heffron, F., et al. supra 1979 and a portion of the adjacent ColElHaeII C fragment. Coordinates 3.95 5.61 kb comprise the 1.66 kb PvuII EcoRIDNA fragment from plasmid pEW27 Wong, E. M., et al., Proc. Natl.Acad. Sci. USA, 79 3570 3574 1982 . This fragment encodes the coy kin El immunity region of plasmid Coin as well as the primer promoter region, RNA I region, and origin of replication. The wildtype Col n DNA sequence of pBGP120 Polisky, B., et al., Proc. Natl. Acad. Sci. USA, 73 3900 3904 1976 and Gelfand, D. H., et al., supra 1978 has been altered at two positions G A transitions in the DNA strand corresponding to nucleotides 125 and 135 of the primer RNA transcript conferring a temperature sensitive Cop phenotype on the plasmid pFC54.tB. IL 2 ivtitien produced by the transformed strains E. colS K12 strain DG95 XN7N53cI857susP80 , transformed with either pFC54 or pFC54.t, using methods described above were grown in 10 liter fermentors under the same conditions as described above for 18.5 hours to an OD680 of 28.7 or 28.1 respectively.Following measurement at 18.5 hours, the temperature of the culture medium containing transformed microorganism was raised to 420C to induce the temperature sensitive copy number plasmid and promoter. Samples were taken at half hourly or hourly Intervals after induction for 4 hours. Each sample was pelleted by centrifugation and was resuspended to a concentration of 10 mg dry weight dw ml in phosphate buffered saline PBS . Each sample was then diluted 10 X in 1X sodium dodecyl sulphate PBS, sonicated in a Heat Systems Model W 375 sonicator for a period sufficient to completely disrupt the cells and assayed. Units of IL 2 mutien ml was determined in the manner described above. Mg protein ml sonicate was determined by the Lowry method. Units IL 2 mutien mg protein was determined and is reported in Table IV below. The production of IL 2 as a percentage of total cellular protein produced for each sample was determined by SDS acrylamide gel electrophoresis. Approximately equal amounts of protein as determined by the Lowry assay were loaded onto the gel. Bands were stained withCoomasie Blue strain and were read using a Zeineh scanning densitometer attached a Hewlett Packard 3390A integrator. Pecent IL 2 mutien was determined by the integration program. I1 2 mutien production as a percentage of total cellular protein is reported inTable V below. Percent increase in I1 2 was found by determining the net increase in IL 2 mutien produced by the strain carrying the positive retroregulatory element and expressing the Increase as a percentage of the IL 2 mutien produced by the strain without the positive retroregulatory element. E. coli strain K12 DG95 N7N53c1857susP80 transformed with pFC54.t was grown under the same conditions as described above except that the culture was temperature induced at an OD680 of 14.0 rather than 28.1 as in the previous example. Samples were taken and determinatons were made as in the previous example. IL 2 mutien production as a percentage of total cell protein was determined to be 34 of total cellular protein. Increased production of IL 2 mutien by the microorganism transformed with the plasmid carrying the positive retroregulatory element is best accomplished when temperatureInduction is carried out at lower cell density as measured by 0D680 14 than higher OD680 28.1 . TABLE IV U IL 2 .g Protein Time after E. coli with E. coli with IncreaseInduction pCF54 pCF54.t U IL 2 mg Protein 0 715 x 104 8.60 x 104 10.3 1 hr. 1.44 x 105 1.62 x 105 12 1.5 hr. 1.99 x 105 2.26 x 105 13.5 2 hr. 2.54 x 105 2.68 xr105 5.5 3 hr. 2.23 x 105 3.02 x 105 26.1 4 hr. 2.85 x 105 2.52 x 105 TABLE V IL 2 Mutien X Total Cellular ProteinTime pFC54 pFC54.t X IncreaseO hr. 3.3 2.6 0.5 hr. 12.3 12.7 2 1 hr. 14.7 17.9 21 2 hr. 24.3 21.5 3 hr. 21.6 20.6 4 hr. 23.8 22.4 EXAMPLE VI Half life extension of mRNA transcripts by the positive retroregulatory element In order to determine whether the enhancement of the expression of the selected DNA sequence ligated to the positive retroregulatory element is the result of increased stability i ty and extended half life of the mRNA transcript encoded by the selected DNA sequence, the rates of decay of the penP transcripts produced by plasmid pHCW A3 and its parental plasmid pSYC667 in both E. coli andB. subtilis were measured. E. coli strain CS412 and B. subtilis strain lA510 each carrying plasmid pHCW A3 or its parental plasmid pSSYC667, were grown in L broth at 370C with shaking.When the culture reached mid log phase of growth A 600 0.7 , 1 mg ml rifampicin was added to block further initiation of transcription byRNA polymerase. Seven ml samples were then withdrawn from the culture at one minute intervals and the tubes containing the samples were rapidly chilled on ice water. The cells were harvested by centrifugation and resuspended into 500 l potassium acetate buffer, pH 5.5, containing 1 mM EDTA and 10 w v SDS. Total cellular RNA from E. coli was extracted and purified as described in Gabain et al.,Proc. Nat. Acad. Sci. USA, 80 653 657 1983 . RNA preparations from B. subtilis were extracted by the following procedure frozen cell pellets 1 g were suspended in 4 ml of 0.05 M potassium acetate, pH 5.0, containing 0.05 mg ml of bentonite per ml, and 2 ml of freshly distilled phenol saturated with the same potassium acetate buffer were added. The suspension was sonicated for three mintues using a Branson Sonifier Heat SystemsUltrasonics, Inc., Plainview, Long Island, NY equipped with a microtip, extracted at 6O0C for 10 minutes, and precipitated overnight with ethanol. The precipitated material was collected by centrifugation, washed with 200C ethanol, dissolved in 4 ml of 0.104M Tris, pH 7.0, 0.01 M MgC12. DNAse 30 pg ml, Worthington, RNAse free was added.The preparation was dialyzed for four hours at 370C against 400 volumes of 0.04 M Tris, pH 7.0, 0.01 M MgC12. The RNA was then extracted twice with phenol at room temperature for five minutes, precipitated with ethanol overnight, washed with ethanol, and dissolved in water. The purified mRNAs were immobilized on 82 mm nitrocellulose filters grade BA 85, Schleicher and Schuell, Inc. Prehybridization of the filters was carried out by the procedure of Woo et al. as described tn Methods 141 Enzymology, 68 389 1979 . Processed filters were then hybridized with penP specific probes 20 Fg filter in 10 ml hybridization buffer 5 x Denhards solution, 50 mM sodium phosphate, pH 7.0, 100 rg ml sheared singlestranded E. coli DNA, 1X SDS at 680C overnight. The probes used in the hybridization were made by completely digesting plasmid pSYC795 with Clay, and repairing the ends with dNTPs and DNA polymerase large fragment to form a linearized plasmid.Plasmid pSYC795 is a derivative of plasmid pSYC423, which is described in Hyashi et al., J. Biol. Chemistry, 259 10448 10454 1984 , which contains a G to C mutation at nucleotide 80 of the coding region of the penP gene. The linearized plasmid was further digested withEcoRI. The plasmid digest was run on a 5X polyacrylamide gel and a 727 bp EcoRI Clal fragment, containing the 5 portion of the penP gene was recovered.This fragment was subcloned into the SmaI EcoRI site of M13mpll. Single stranded DNAs from these recombinant phages were isolated according to the method of Messing, Methods in Enzymology, Vol. 101 20 78 1983 , and the circular DNA was digested with DNAse I 0.05 9 R9 of DNA for 20 mintues at 370C. The single stranded DNA fragments were phenol ether extracted, ethanol precipitated, resuspended in 50 mM Tris buffer, pH 8, dephosphorylated with bacterial alkaline phosphatase and labeled with y32P AlP and polynucleotide kinase to a specific radioactivity of 3 x 106 cpm g DNA. After hybridization, the filters were washed sequentially with 2 x SSC, and 1 x SSC containing 1.0 SDS at 680C for 15 minutes each, dried under a hot lamp, and counted in liquid scintillation fluid. To ensure that an excess of DNA probes was present in the hybridization solution, several filters were prepared which contain different amounts of immobilized mRNA samples. The results of these hybridization experiments are summarized in Figures 7A and 78. The half life of the penP mRNA produced from plasmid pSYC66 in either E.coli or B. subtilis was estimated to be about two minutes that from pHCW A3 was about stx minutes. Analysis also revealed that the cells carrying the plasmid pHCW A3 have a higher steady state level of PenP mRNA than the cell harboring the plasmid pSYC667. This result is expected if the positive retroregulatory element stabilizes the cotranscribed gene. Since the increase In the levels of mRNA matches the magnitude of increase in penicillinase enzyme in both of these bacterial hosts as shown In ExampleS III above, the positive retroregulatory element enhances gene expression through its influence on mRNA stability. Deposited Strains Deposits of strains listed in the following Table VI were made with the American Type Culture Collection, Rockville, Maryland, U.S.A., and were assigned the ATCC numbers listed in the Table. The ATCC deposits were made under the Budapest Treaty on the International Recognition of Deposits of MiCroorganisms for Purposes of Patent Procedures and the Regulations promulgated thereunder, and the strains will be available to the public in accordance with the terms of said Treaty and Regulations. TABLE VI ATCCStrain Plasmid Deposit NumberE. col1 pSYC667 39758 K 12 CS412 B. thurin 1615 gensis HD 1E. coli pHCW701 39757 K 12 MMZ94 E. coli pFC54.t 39789DG95 h E. coli pLW1 39405MM294E. coli 39626 MM294 1 E. coli 39452MM294 Various modifications of the invention, as described and exemplified in the present specification, will be apparent to persons of skill in the art. It is intended that such modifications are within the scope of the invention and the appended claims.