# RECOMBINANT PROTEINS OF VIRUSES ASSOCIATED WITH LYMPHADENOPATHY SYNDROME AND OR ACQUIRED IMMUNE DEFICIENCY SYNDROME

## Claims
Rekombinantes DNA Konstrukt zur Expression eines rekombinanten Polypeptids in einer das Konstrukt enthaltenden Zelle, wobei das Konstrukt Kontrollsequenzen umfaßt, die die Transkription und die Translation des rekombinanten Polypeptids in der Zelle regulieren und eine von den Kontrollsequenzen regulierte codierende Sequenz, wobei die codierende Sequenz eine DNA Sequenz von mindestens etwa 21 bp im Leserahmen umfaßt, dadurch gekennzeichnet, daß die DNA Sequenz eine gegen HIV I antigene Aminosäuresequenz von Fig. 2 codiert, wobei die Sequenz mit HTLV I und HTLV II immunologisch nicht kreuzreaktiv und mit HIV I reaktiv ist.

## Description
This invention is in the field of genetic engineering. More particularly, it relates to recombinant viral proteins associated with lymphadenopathy syndrome and or acquired immune deficiency syndrome. With the discovery of human T cell lymphotropic Virus I HTLV I as an infectious agent in humans, it was established that retroviruses could infect humans and could be the etiological agent of disease. After HTLV I was established, a second retrovirus of the same family, HTLV II was found in a hairy cell leukemia established strain. Since that time, other human retroviruses have been isolated which are associated with lymphadenopathy syndrome LAS and or acquired immune deficiency syndrome AIDS victims. Various retroviruses have been isolated from individuals with AIDS sometimes called HTLV III or LAS sometimes called LAV . See for example, Barre Sinoussi, et al, At the time of filing this application, these viruses HTLV III, LAV, and ARV were generically referred to as human T cell lymphotropic retrovirus hTLR . However, from 1986 onwards the equivalent generic term human immundeficiency virus HIV was adopted as the recognised term for such viruses. Subsequently, a sub division of the generic group HIV was necessary into HIV I and HIV II. Since the application relates to ARV 2 isolates which are HIV I isolates, HIV I has been substituted throughout the application for hTLR HIV and the claims are accordingly limited to HIV I. The HIVs hTLRS may be shown to be of the same class by being similar in their morphology, serology, reverse transcriptase optima and cytopathology, as identified in the above references. For example, the reverse transcriptase prefers Mg ², and has a pH optima of about 7.8. DNA clones containing HIV sequences are disclosed in EP A1 0173529, EP A1 0178978, EP A2 0185444 and WO 86 02383. The present invention provides the following A cell comprising a recombinant DNA construct described above wherein the cell expresses the antigenic HIV I amino acid sequence and is free from other cells which do not express the antigenic HIV I amino acid sequence. A method of producing a recombinant polypeptide comprising an antigenic HIV I amino acid sequence wherein a population of the cells described above is cultured under conditions whereby the recombinant polypeptide is expressed. An immunoassay for detecting antibodies to HIV I in a sample suspected of containing the antibodies, characterised in that at least one recombinant polypeptide is used to bind the antibodies and the recombinant polypeptide comprises an antigenic env, gag or pol HIV I amino acid sequence contained in the sequence shown in Figure 2 which is immunologically non cross reactive with HTLV I and HTLV II. A recombinant polypeptide characterised in that it is produced by a cell transformed by a recombinant DNA construct the construct comprising a DNA sequence which encodes an amino acid sequence from an env, gag or pol polypeptide of HIV I. An article of manufacture for use in an immunoassay for HIV I antibodies characterised in that it comprises a solid support having bound thereto a recombinant polypeptide described above. Figure 1 is a restriction map of proviral DNA ARV 2 . Figure 2 is the nucleotide sequence of ARV 2 9B . The amino acid sequences for the products of the gag, pol, and env genes are indicated. The U3, R, and U5 regions of the LTRs are also designated. The cap site is position 1. A 3 bp inverted repeat at the ends of the LTR, the TATA box at position 29, the sequence complementary to the 3 end of the tRNA Figure 3 is a flow diagram showing the procedures for making the plasmid pGAG25 10. Figure 4 is the nucleotide sequence of the p25 gag gene cloned in plasmid pGAG25 10 and the amino acid sequence encoded by that gene. Figure 5 is the coding strand of the nucleotide sequence cloned in pGAG41 10 for producing the fusion protein p41 gag and the corresponding amino acid. Figure 6 is a nucleotide sequence coding for ARV 2 p16 gag protein that was cloned into plasmid ptac5 to make an expression plasmid for producing p16 gag protein in bacteria. Figure 7 is a nucleotide sequence that encodes ARV 2 env protein that was used to prepare plasmid pDPC303. Figure 8 is a nucleotide sequence that encodes ARV 2 p31 protein and is contained in plasmid pTP31. The HIV I DNA sequences, either isolated and cloned from proviral DNA or cDNA or synthesized, may be used for expression of polypeptides which may be a precursor protein subject to further manipulation by cleavage, or a complete mature protein or fragment thereof. The smallest sequence of interest, so as to provide a sequence encoding an amino acid sequence capable of specific binding to a receptor, e.g., an immunoglobulin, will be 21 bp, usually at least 45 bp, exclusive of the initiation codon. The sequence may code for any greater portion of or the complete polypeptide, or may include flanking regions of a precursor polypeptide, so as to include portions of sequences or entire sequences coding for two or more different mature polypeptides. The sequence will usually be less than about 5 kbp, more usually less than about 3 kbp. Sequences of particular interest having open reading frames Figure 2 define the structural genes for the gag proteins p16 and p25 , the env protein, and the pol protein p31 . It is to be understood that the above sequences may be spliced to other sequences present in the retrovirus, so that the 5 end of the sequence may not code for the N terminal amino acid of the expression product. The splice site may be at the 5 terminus of the open reading frame or internal to the open reading frame. The initiation codon for the protein may not be the first codon for methionine, but may be the second or third methionine, so that employing the entire sequence indicated above may result in an extended protein. However, for the gag and env genes there will be proteolytic processing in mammalian cells, which processing may include the removal of extra amino acids. In isolating the different domains the provirus may be digested with restriction endonucleases, the fragments electrophoresed and fragments having the proper size and duplexing with a probe, when available, are isolated, cloned in a cloning vector, and excised from the vector. The fragments may then be manipulated for expression. Superfluous nucleotides may be removed from one or both termini using Bal31 digestion. By restriction mapping convenient restriction sites may be located external or internal to the coding region. Primer repair or The env domain of the HIV I genome can be obtained by digestion of the provirus with Digestion of proviral sequences with Digestion of the previous with The polypeptides which are expressed by the above DNA sequences may find use in a variety of ways. The polypeptides or immunologically active fragments thereof, may find use as diagnostic reagents, being used in labeled or unlabeled form or immobilized i.e., bound to a solid surface , as vaccines, in the production of monoclonal antibodies, e.g., inhibiting antibodies, or the like. The DNA sequences may be joined with other sequences, such as viruses, e.g., vaccinia virus or adenovirus, to be used for vaccination. Particularly, the DNA sequence of the viral antigen may be inserted into the vaccinia virus at a site where it can be expressed, so as to provide an antigen of HIV I recognized as an immunogen by the host. The gag, pol, or env genes or fragments thereof that encode immunogens can be used. Another alternative is to join the gag, env, or pol regions or portions thereof to HBsAg gene or pre S HBsAg gene or immunogenic portions thereof, which portion is capable of forming particles in a unicellular microorganism host, e.g., yeast or mammalian cells. Thus, particles are formed which will present the HIV I immunogen to the host in immunogenic form, when the host is vaccinated with assembled particles. As vaccines, the various forms of the immunogen can be administered in a variety of ways, orally, parenterally, intravenously, intra arterially, subcutaneously, or the like. Usually, these will be provided in a physiologically acceptable vehicle, generally distilled water, phosphate buffered saline, physiological saline, or the like. Various adjuvants may be included, such as aluminum hydroxide, and the dosages, number of times of administration and manner of administration determined empirically. In order to obtain the HIV I sequence, virus can be pelleted from the supernatant of an infected host cell. A 9 kb RNA species is purified by electrophoresis of the viral RNA in low melting agarose gels, followed by phenol extraction. The purified RNA may then be used as a template with random primers in a reverse transcriptase reaction. The resulting cDNA is then screened for hybridization to polyA RNA from infected and uninfected cells. Hybridization occurring from infected, but not uninfected cells, is related to the HIV I. Genomic DNA from infected cells can be restriction enzyme digested and used to prepare a bacteriophage library. Based upon restriction analysis of the previously obtained fragments of the retrovirus, the viral genome can be partially digested with HIV I DNA either provirus or cDNA may be cloned in any convenient vector. Constructs can be prepared, either circular or linear, where the HIV I DNA, either the entire hTLR or fragments thereof, may be ligated to a replication system functional in a microorganism host, either prokaryotic or eukaryotic cells mammalian, yeast, arthropod, plant . Micro organism hosts include For expression, expression vectors will be employed. For expression in microorganisms, the expression vector may differ from the cloning vector in having transcriptional and translational initiation and termination regulatory signal sequences and may or may not include a replication system which is functional in the expression host. The coding sequence is inserted between the initiation and termination regulatory signals so as to be under their regulatory control. Expression vectors may also include the use of regulatable promoters, e.g., temperature sensitive or inducible by chemicals, or genes which will allow for integration and amplification of the vector and HIV I DNA such as tk, dhfr, metallothionein, or the like. The expression vector is introduced into an appropriate host where the regulatory signals are functional in such host. The expression host is grown in an appropriate nutrient medium, whereby the desired polypeptide is produced and isolated from cells or from the medium when the polypeptide is secreted. Where a host is employed in which the HIV I transcriptional and translational regulatory signals are functional, then the HIV I DNA sequence may be manipulated to provide for expression of the desired polypeptide in proper juxtaposition to the regulatory signals. The polypeptide products can be obtained in substantially pure form, particularly free of debris from human cells, which debris may include such contaminants as proteins, polysaccharides, lipids, nucleic acids, viruses, bacteria, fungi, etc., and combinations thereof. Generally, the polypeptide products will have less than about 0.1, usually less than about 0.01 weight percent, of contaminating materials from the expression host. Depending upon whether the desired polypeptide is produced in the cytoplasm or secreted, the manner of isolation will vary. Where the product is in the cytoplasm, the cells are harvested, lysed, the product extracted and purified, using solvent extraction, chromatography, gel exclusion, electrophoresis, or the like. Where secreted, the desired product will be extracted from the nutrient medium and purified in accordance with the methods described above. The expression products of the env, gag, and pol genes and immunogenic fragments thereof having immunogenic sites may be used for screening antisera from patients blood to determine whether antibodies are present which bind to HIV I antigens. One or more of the recombinant antigens are used in the serological assay. Preferred modes of the assay employ a combination of gag, env, and pol antigens. A combination of p25, P31 and env recombinant antigens is particularly preferred. A wide variety of immunoassay techniques can be employed, involving labeled or unlabeled antigens or immobilized antigens. The label may be fluorescers, radionuclides, enzymes, chemiluminescers, magnetic particles, enzyme substrates, cofactors or inhibitors, ligands, or the like. A particularly convenient technique is to bind the antigen to a support such as the surface of an assay tube or well of an assay plate or a strip of material, such as nitrocellulose or nylon, that binds proteins and contact the sample with the immobilized antigen. After washing the support to remove non specifically bound antisera, labeled antibodies to human Ig are added. The support is then washed again to remove unbound labeled anti human Ig. The presence of bound analyte is then determined through detection of the label. ELISA and dot blot assays are particularly useful for screening blood or serum samples for anti HIV I antibodies. The ELISA assay uses microtiter trays having wells that have been coated with the antigenic HIV I polypeptides s . The wells are also typically post coated with a non antigenic protein to avoid non specific binding of antibodies in the sample to the well surface. The sample is deposited in the wells and incubated therein for a suitable period under conditions favorable to antigen antibody binding. Anti HIV I antibodies present in the sample will bind to the antigen s on the well wall. The sample is then removed and the wells are washed to remove any residual, unbound sample. A reagent containing enzyme labeled antibodies to human immunoglobulin is then deposited in the wells and incubated therein to permit binding between the labeled anti human Ig antibodies and HIV I antigen human antibody complexes bound to the well wall. Upon completion of the incubation, the reagent is removed and the wells washed to remove unbound labeled reagent. A substyate reagent is then added to the wells and incubated therein. Enzymatic activity on the substrate is determined visually or spectrophotometrically and is an indication of the presence and amount of anti HIV I antibody containing immune complex bound to the well surface. The dot blot procedure involves using hTLR antigen s immobilized on a piece or strip of bibulous support material, such as nitrocellulose filter paper or nylon membrane, rather than antigen coated microtiter trays. The support will also be treated subsequently with a non antigenic protein to eliminate non specific binding of antibody to the support. The antigen carrying support is dipped into the sample and allowed to incubate therein. Again, any anti HIV I antibodies in the sample will bind to the antigen s immobilized on the support. After a suitable incubation period the support is withdrawn from the sample and dipped repeatedly in wash buffer to remove any unbound sample from the paper. The support is then dipped into the enzyme labeled antibody to human Ig reagent for a suitable incubation period. Following treatment with the labeled reagent the support is dipped in wash buffer, followed by incubation in the substrate solution. Enzymatic activity, indicating the presence of anti HIV I antibody containing complexes on the support, causes color changes on the support which may be detected optically. Either of these techniques may be modified to employ labels other than enzymes. The reading or detection phases will be altered accordingly. The antigenic polypeptides of HIV I may also be used as immunogens by themselves or joined to other polypeptides for the production of antisera or monoclonal antibodies which may be used for therapy or diagnosis. The immunoglobulins may be from any mammalian source, e.g., rodent, such as rat or mouse, primate, such as baboon, monkey or human, or the like. For diagnosis, the antibodies can be used in conventional ways to detect HIV I in a clinical sample. The HIV I DNA sequences may also be labeled with isotopic or non isotopic labels or markers and be used as DNA probes to detect the presence of native HIV I nucleotide sequences in samples suspected of containing same. The following examples are offered by way of illustration and not by way of limitation. HUT 78 cells infected with ARV 2 ATCC Accession No. CRL 8597, deposited on August 7, 1984 were obtained from Dr. Jay Levy, University of California, San Francisco. Cultures were grown for two weeks in RPMI medium with 10 fetal calf serum. Cultures were centrifuged at 2 Krpm for 1 hr at 4 C using a SW 28 rotor. The pellet, containing the virus, was resuspended in 10 mM Tris HCl, pH 7.5 on ice. The resuspended pellet was treated with 10 µg of DNase Boehringer Mannhein and was layered onto a linear sucrose gradient 15 50 in 10 mM Tris HCl, pH 7.5, 1 mM EDTA. 20 mM NaCl . The gradient was spun at 34 Krpm for 4 hr at 4 C, in SW 41 rotor. Five 2.5 ml fractions were collected and an aliquot of each was electrophoresed in a 1 agarose, 5 mM methyl mercury hydroxide gel Bailey and Davidson, To purify the viral RNA from the nucleic acid pellet obtained as described above, a sample was electrophoresed in a low melting 1 agarose gel containing 5 mM Methyl mercury hydroxide. After electrophoresis, the gel was stained with 0.1 ethidium bromide and nucleic acid bands were visualized under UV light. The region corresponding to 9 kb was cut from the gel and the agarose was melted at 70 C for 2 to 3 min in three volumes of 0.3 M NaCl, 10 mM Tris, pH 7.5, 1 mM EDTA. The mixture was extracted with an equal volume of phenol. The aqueous phase was reextracted with phenol and was precipitated with ethanol. The pellet was washed with cold 95 ethanol, air dried, resuspended in water and stored at 70 C until use. One hundred ml of culture medium yielded 0.5 to 1 µg of purified RNA. A ³²P labeled cDNA was made to the gel purified viral RNA using random primers calf thymus primers prepared as described in Maniatis, et al, PolyA RNA was prepared from HUT 78 cells infected with ARV 2, ARV 3 or ARV 4 three different isolates from three different AIDS patients and from uninfected HUT 78 cells. The polyA RNA was electrophoresed on 1 agarose gels containing 5 mM methyl mercury hydroxide Bailey and Davidson, supra , was transferred to nitrocellulose filters, and hybridized with the homologous probe prepared as described in Section 2. Hybridizations were carried out in 50 formamide, 3 x SSC at 42 C. Washes were at 50 C in 0.2 x SSC. A 9 kbp band was present in all three samples of infected HUT 78 cells. This band was absent in polyA from uninfected cells. High molecular weight DNA chromosomal was prepared from cultures of HUT 78 cells infected with ARV 2 and from non infected HUT 78 cells following the procedure of Luciw, et al, The homologous ³²P probe to ARV 2 hybridized specifically to two bands in the DNA from infected cells restricted with In order to determine if different enzymes would cut the proviral sequence, several other restriction digestions of the cell DNA were carried out using High molecular weight cell DNA from infected HUT 78 cells was prepared following the procedure of Luciw, et al, supra. The DNA was digested with One of the 11 phage, designated λ ARV 2 9B , was deposited at the ATCC on 25 January 1985 and given Accession No. 40158. λ ARV 2 9B contained an insertion of full length proviral DNA along with flanking cell sequences. Digestion of λ ARV 2 9B DNA with In addition to λ ARV 2 9B , phage was obtained that 1 possessed the left half of the viral genome from the To measure the relatedness of independent ARV isolates, restriction enzyme digests of DNA from HUT 78 cells infected with ARV 3 and ARV 4 were analyzed with the probe made from cloned ARV 2 DNA. The Fragments or subfragments of ARV 2 DNA from λ phage 9B were prepared and cloned into M13 according to conventional procedures Maniatis, et al, supra . Sequencing was performed according to Sanger, et al, ARV 2 was prepared and purified as described in Section 1. The viral proteins were electrophoresed on an acrylamide gel, and the band corresponding to a 24,000 dalton or 16,000 dalton protein was excised from the gel and used for sequencing. Micro sequence analysis was performed using Applied Biosystems model 470A protein sequencer similar to that described by Hewick, et al, The amino acid sequence of Table 1 is predicted from the ARV 2 DNA sequence of Figure 2. Therefore, these results confirm that the indicated gag open reading frame is in fact being translated and identifies the N termini of p25 and p16. The p25 gag protein is synthesized by Plasmid pGAG25 10 is a pBR322 derivative which contains the sequence coding for p25 gag under transcriptional control of a hybrid tac promoter De Boer et al, Plasmid pGAG25 10 was constructed by cloning a 699 bp DNA fragment coding for p25 gag into plasmid ptac5, according to the scheme shown in Figure 3. The vector ptac5 is a pBR322 derivative which contains the tac promoter. Shine Delgarno sequences, and a polylinker as a substitution of the original pBR322 sequences comprised between the EcoRI and PvuII restriction sites. The 699 bp DNA fragment codes for the complete p25 gag protein amino acid residues 139 to 369 as numbered in Figure 2 , the only difference being that a methionine was added as the first amino acid in pGAG25 10 to allow for translational initiation. This change, as well as other changes in nucleotide sequence as indicated below, was achieved by using chemical synthesis of party of the DNA fragment. The DNA fragment also includes two stop codons at the 3 end of the sequence. Figure 4 shows the nucleotide sequence cloned in pGAG25 10 and the amino acid sequence derived from it. DNA sequences that are not underlined in the figure were derived directly from the ARV 2 9B cDNA. All other sequences were chemically synthesized or derived from vector ptac5. Changes were introduced in this DNA sequence, with respect to the original cDNA, to create or delete restriction sites, to add a methionine prior to the proline first residue of p25 or to include stop codons after the last codon of p25 gag. However, as previously indicated, all changes in the DNA sequence, except those in the first codon, do not alter the amino acid sequence of p25 gag. Single ampicillin resistant colonies are transferred into 1 ml L broth containing 100 µg ml ampicillin and grown at 37 C. Expression of p25 gag protein is induced by adding 10 µl of 100 mM IPTG Sigma to a final concentration of 1 mM followed by incubation at 37 C for 2 hr. Cells from 1 ml of induced cultures are pelleted and resuspended in 100 µl Laemmli sample buffer. After 3 cycles of boiling and freezing. portions of resultant lysates are analyzed on standard denaturing acrylamide gels. Proteins are visualized by staining with Coomassie blue. The extent of expression is initially determined by appearance of new protein bands for induced candidate samples compared with control. Proteins of molecular weights expected for the genes expressed comprised 2 5 of total cell protein in the highest expressing recombinants as determined by visual inspection with reference to a standard protein of known amount. Authenticity of the expressed proteins is determined by standard Western transfer of proteins to nitrocellulose and analysis with appropriate human or rabbit immune sera or mouse monoclonal antibodies see E.4.a. below or by ELISA assays of soluble Transformant cells from a culture expressing high levels 3 of p25 gag are streaked onto an L broth plate containing 100 µg ml ampicillin and the plate is incubated overnight at 37 C. A single colony is inoculated into 10 ml of L broth, 100 µg ml ampicillin and grown overnight at 37 C. An aliquot is used to verify plasmid structure by restriction mapping with The master seed stock is scraped with a sterile applicator which is used to streak an L broth plate containing 100 µg ml ampicillin. Single colonies from this plate are used to inoculate 20 50 ml of L broth amp, which is incubated at 37 C overnight. An aliquot of the overnight culture is used to inoculate larger volumes 1 6 liters of L broth amp. Cells are incubated at 37 C overnight and reach an O.D.₆₅₀ of approximately 5 prior to use as inoculum for the fermenter run. Fermenters capacity 16 liters containing 10 l of L broth and 1 ml of antifoam are inoculated with 100 500 ml from the inoculum culture. Cells are grown at 37 C to an O.D. of about 1. Expression of p25 gag is induced by addition of 100 ml of an IPTG solution 100 mM to yield a 1 mM final concentration in the fermenter. Cells are grown for 3 additional hours and subsequently harvested using continuous flow centrifugation. At this step cells may be frozen and kept at 20 C until purification of p25 gag proceeds. Alternatively, 250 l fermenters are inoculated with 1 5 l from the inoculum culture. Growth, induction, and harvest are as indicated before. Frozen The cell extract is made 30 NH₄ ₂SO₄ by slowly adding the ammonium sulfate at 4 C. The extract is stirred for 10 min after the final concentration is achieved, followed by contrifugation at 27.000 x g for 20 min. The pellet is resuspended in 1 M NaCl, 1 mM EDTA. 1 Triton X 100, and 5 SDS, and then boiled for 5 min. The fraction obtained by selective precipitation is submitted to gel filtration using a G50 Sephadex column equilibrated in 0.03 M NaPi, pH 6.8. Chromatography is developed in the same solution. Fractions are collected and absorbance at 280 nm is determined. Protein containing fractions are pooled and characterized by protein gel electrophoresis, Western analysis, and ELISA. The reactivity of purified recombinant p25 gag to various sera was compared to that of natural p25 gag protein purified by preparative polyacrylamide gel electrophoresis in an ELISA assay. For control, assays were also made using disrupted gradient purified virus 5 µg ml . PVC microtiter plates were incubated for 2 hr at 37 C with 10 µg ml 50 µl well in 0.1 M sodium borate, pH 9.0 of the Ig fraction of ascites from murine anti p25 gag monoclonal antibody 76C. The plates were washed with PBS and the wells were filled with 10 normal goat serum in PBS. Following a 30 min incubation at room temperature, the plates were washed with normal saline containing 0.05 Triton X 100 ST and dilutions of the test ARV protein 50 µl well in ST with 10 goat serum STGS were added to the wells. The plates were incubated for 2 hr at 37 C, washed with ST, and then incubated for 1 hr at 37 C with 50 µl well of rabbit antiserum raised against disrupted ARV 1 1000 dilution in STGS . The wells were washed, incubated for 1 hr with 50 µl of a 1 1500 dilution in STGS of HRP conjugated goat antiserum to rabbit immunoglobulin, washed, and then the wells received 50 µl well of substrate solution 150 µg ml 2,2 azino di 3 ethyibenzthiazolene sulfonic acid , 0.001 H₂O₂, 0.1 M citrate pH 4 . The reaction was stopped after incubation for 30 min at 37 C by the addition of 50 µl well of 10 SDS. The absorbance was read on a Flow Titertech ELISA reader at 414 nm. Samples were assayed in duplicate beginning at a dilution of 1 10 and by serial 2 fold dilutions thereafter. The table below summarizes the results of assays on 8 AIDS sera that scored positive in the assay with disrupted virus and 6 normal sera that were negative in the disrupted virus assay. These results show that p25 gag purified from bacteria behaves identically to similarly purified p25 gag from AIDS virus in an ELISA of the eight AIDS patient sera. The results of the ELISA show that there is a wide variation in the levels of anti p25 gag antibodies and suggests that antibodies to some virus encoded proteins may not be detected using conventional virus based assay systems. A fusion protein of the p25 gag and p16 gag proteins of ARV 2, designated p41 gag, was synthesized in The sequence shown in Figure 6 and coding for the p16 gag protein was chemically synthesized using yeast preferred codons. The blunt end A recombinant gag protein was also expressed in Cos mammalian cells. A partial env protein is synthesized by Plasmid pDPC303 contains an expression cassette described below for env cloned into the The expression cassette for env consists of the following sequences fused together in this order 5 to 3 yeast pyruvate kinase PYK promoter, env cDNA, and PYK terminator. The PYK promoter and terminator regions were derived from PYK cDNA isolated as described in Burke, et al, The env fragment cloned into the expression cassette was derived from ARV 2 cDNA and comprises a 1395 bp cDNA fragment which codes for env amino acid residues coded by nt 5857 to nt 7251 Figure 2 . In addition, there are 5 extra codons fused in reading frame in the 5 end, the first codon corresponding to a methionine, and 4 extra codons fused in reading frame at the 3 end followed by a stop codon. The extra codons were incorporated to facilitate cloning procedures exclusively. Figure 7 shows the coding strand of the nucleotide sequence cloned in pDPC303 and the amino acid sequence derived from it. DNA sequences that are not underlined in the figure were derived directly from the ARV 2 9B cDNA described above. All other sequences were either chemically synthesized or derived from the PYK vector. Yeast cells Frozen Pellet 1 is resuspended in 4 volumes of Tris HCl buffer 0.01 M Tris HCl, pH 8.0, 0.01 M NaCl, 0.001 M PMSF, 1 µg ml pepstatin, 0.001 M EDTA, 0.1 SDS and extracted for 2 hr at 4 C with agitation. The solution is centrifuged at 6,300 x g for 15 min. The insoluble fraction pellet 2 is resuspended in 4 volumes 360 ml of PBS per liter 0.2 g KCl, 0.2 g KH₂PO₄, 8.0 g Nacl, 2.9 g Na₂HPO₄.12H₂O , 0.1 SDS, 0.001 M EDTA, 0.001 M PMSF, 1 µg ml pepstatin, and centrifuged at 6,300 x g for 15 min. The pellet pellet 3 is suspended in 4 volumes of PBS, 0.2 SDS, 0.001 M EDTA, 0.001 M PMSF, 1 µg ml pepstatin and is extracted for 12 hr at 4 C with agitation on a tube rocker. The solution is centrifuged at 6,300 x g for 15 min. The soluble fraction is recovered for further purification as indicated below. The pellet can be reextracted by resuspending it in 4 volumes of 2.3 SDS, 5 β mercaptoethanol, and boiling for 5 min. After boiling, the solution is centrifuged at 6,300 x g for 15 min. The soluble fraction is recovered for further purification. The soluble fraction is concentrated by precipitation with 30 ammonium sulfate at 4 C. The pellet pellet 4 is resuspended in 2.3 SDS, 5 β mercaptoethanol, and chromatographed on an ACA 34 LKB Products gel filtration column. The column is equilibrated with PBS, 0.1 SDS, at room temperature. Chromatography is developed in the same solution with a flow rate of 0.3 ml min. Five ml fractions are collected, pooled and characterized by protein gel electrophoresis, Western analysis, and ELISA. If needed, pooled fractions are concentrated by vacuum dialysis on Spectrapor 2 MW cutoff 12 14K . SDS polyacrylamide gel analysis 12 acrylamide gels showed that a new 55,000 dalton protein was being synthesized in yeast cells transformed with the env containing vector. The 55,000 dalton protein is absent from cells transformed with control plasmid vector without env insert . The identity of env was confirmed by both ELISA see 9.E.4.c and Western analysis using AIDS patient serum. In both assays the 55,000 dalton protein showed immunoreactivity. No reactivity was obtained with serum from a normal individual. Recombinant env was also expressed in mammalian Cos cells. The C terminal region of the polymerase gene p31 pol is synthesized by A 5.2 kb DNA fragment was isolated from a The DNA sequence in 01100484 was altered by site specific mutagenesis to generate a restriction site recognized by A second oligomer was synthesized to insert sites for Replicative form RF of the 01031985 template was prepared by growing 6 clear plaques, each in 1.5 ml of 2 x YT at 37 C for 5 hr. Double stranded DNA was obtained as described by Maniatis, et al, The vector plot7 3 µg Hallewell, et al, The One hundred fifty picomoles each of dephosphorylated ARV 211, phosphorylated ARV 222 and ARV 223 were ligated to 20 µg of pRSP248 previously cut with Bacterial transformants containing either the vector alone, or the vector with the p31 DNA pTP31.2 were grown in L broth with 0.02 ampicillin to an OD₆₅₀ of 0.5. Cultures were induced by the addition of IPTG to a final concentration of 2 mM and grown for 3 more hr. Bacteria from 1 ml cultures were pelleted and resuspended in 200 µl of gel sample buffer. The cells were disrupted by three cycles of freezing and thawing. boiled, and the extracts loaded onto 12.5 polyacrylamide SDS minigels. Proteins were electrophoresed and transferred to nitrocellulose by electroblotting. The nitrocellulose filters were reacted with serum EW5111 diluted 1 100 positive reference serum from the CDC that reacts strongly with viral p31 , horse radish peroxidase conjugated goat anti human IgG and HRP substrate. A prominent band at 30,000 d and several lower molecular weight species were seen in gels of extracts from transformants with the p31 DNA, but not in extracts from bacteria transformed with the vector alone. Lysozyme NP40 extracts were prepared from bacteria transformed with pTP31.2 or vector alone. Five ml cultures were grown, the cells pelleted and resuspended in 1 ml of 50 mM Tris HCl pH 8, 0.5 mM EDTA, 1 mg ml lysozyme and incubated at 0 C for 15 min NaCl, MgCl₂, and NP40 were added to final concentrations of 0.4, 5 mM and 0.5 respectively, mixed and incubated with DNAse I 100 µg ml at 0 C for 30 min. When EW5111 serum diluted 1 100 was preincubated with a 1 10 dilution of the cell extracts from bacteria transformed with pTP31.2, prior to reaction with a virus blot, the viral p31 band was completely eliminated, while reactivity with other viral proteins remained unaffected. In contrast, extracts from bacteria transformed with the vector alone did not absorb out the p31 reactive antibodies. The viral p31 protein is thus the product of the C terminal or endonuclease region of the pol gene of ARV 2. The ARV248NL fragment was cloned into pBS100 previously cut with Three different induction procedures were done 1. Ura colonies patched on ura plates were induced for 24 hr in YEP 1 glucose. Both a Western and a polyacrylamide gel were run on these samples. Both results were negative. 2. Colonies from ura plates patched on leu plates were induced in either leu 3 ethanol or YEP 1 glucose for 24 hr. A Western and a polyacrylamide gel were run on these samples and the results were also negative. 3. Colonies from leu plates patched on leu plates were induced in either leu 3 ethanol or YEP 1 glucose for 24 hr. The polyacrylamide gel showed a negative result. No Western was run on these samples. For the construction of a gene for a fused protein SOD p31 to be expressed in yeast, a plasmid pSI4 39 2 was used. This plasmid contains the SOD gene fused to the proinsulin gene under the regulation of the ADH 2 GAP promoter. The proinsulin gene is located between Two µg of pSI4 39 2 iinearized with Three different kinds of inductions were tried 1. PO17 colonies were induced in either a 10 ml culture of YEP 1 glucose or a leu 3 ethanol culture for 24 hr. The yeast pellets were analyzed by both polyacrylamide gels and Westerns and even though the Coomassie stained gel showed a negative result, the Western did light up a band of the correct molecular weight with both induction methods. 2. PO17 colonies were induced in a 30 ml culture of YEP 1 ethanol for 48 hr. Aliquots were analyzed by PAGE at various time points during the induction. The Coomassie stained gel shows a band in the correct molecular weight range 47 50 kd that appears after 14 hr in YEP 1 ethanol and reaches a maximum intensity at 24 hr of induction. The Western result correlates well with the Coomassie stained gel, showing strong bands at 24 and 48 hr. 3. AB110 colonies were induced in either leu 3 ethanol or YEP 1 glucose for 24 hr. PAGE and Westerns were run and the results were negative for the PAGE and positive for the Western, in both induction methods. Frozen bacteria yeast cells are thawed at room temperature and suspended in 1.5 volumes of lysis buffer 20 mM Tris HCl, pH 8.0, 2 mM EDTA, 1 mM PMSF, for bacteria 50 mM Tris Cl, pH 8.0, 2 mM EDTA, 1 mM PMSF for yeast , and mixed with 1 volume of acid washed glass beads. Cells are broken for 15 min in a non continuous mode using the glass chamber of a Dynomill unit at 3,000 rpm, connected to a 20 C cooling unit. Glass beads are decanted for 2 3 min on ice, the cell lysate is removed. The decanted glass beads are washed twice with 30 ml of lysis buffer at 4 C. The cell lysate is centrifuged at 39,000 x g for 30 min. The pellet obtained from the above centrifugation is washed once with lysis buffer, after vortexing and suspending it at 4 C same centrifugation as above . The washed pellet is treated with 0.2 SDS for bacteria and 0.1 SDS for yeast in lysis buffer and is agitated by rocking at 4 C for 10 min. The lysate is centrifuged at 39,000 x g for 30 min. The pellet is boiled in sample buffer 67.5 mM Tris Cl, pH 7.0, 5 β mercaptoethanol, 2.3 SDS for 10 min and centrifuged for 10 min at 39,000 x g. The supernatant is recovered and further centrifuged at 100,000 x g for 60 min 60 Ti rotor . This step is replaced by a 0.45 µm filtration when yeast is used. The supernatant from the above centrifugation is loaded maximum 50 mg of protein on a gel filtration column 2.5 x 90 cm, ACA 34 LKB with a flow rate of 0.3 0.4 ml min, equilibrated with phosphate buffered saline PBS , 0.1 SDS. The fractions containing SOD p31 are pooled and concentrated either by vacuum dialysis or using a Gel electrophoresis analysis shows that the SOD p31 protein migrates having a molecular weight of about 46 kd and is over 90 pure. Stock solutions of purified p25 gag protein 1.25 mg ml in 20 mM sodium phosphate, 0.1 SDS, pH 7.2 , purified env protein 2 mg ml in 20 mM sodium phosphate, 0.1 SDS, pH 7.2 , and purified SOD p31 fusion protein 2 mg ml in 20 mM sodium phosphate, 0.1 SDS, pH 7.2 were prepared. For coating microtiter plates Dynatech Immulon I , 1 part each of the stock solutions of p25 gag, env, and SOD p31 were added to 997 parts of borate coating buffer 0.05 M borate, pH 9.0 . One hundred microliters of the coating solution was added to each well, and the plates were covered and incubated 2 hr at 37 C or 12 hr at 4 C. The coating solution was then aspirated from the wells and the plates washed 6 x with wash solution 0.137 M 0.8 NaCl, 0.05 Triton X 100 . Serum samples were diluted 1 100 in dilution solution 0.1 casein, 1 mM EDTA, 1 Triton X 100, 0.5 M NaCl, 0.01 thimerosal, pH 7.5 with yeast protein strain AB103.1 extract 1 40 dilution, approximately 2 mg protein per ml in PBS containing 1 Triton X 100, 2 mM PMSF, 0.01 thimerosal and Goat anti human Ig labeled with horseradish peroxidase Cappel diluted 1 8000 in dilution solution without added yeast and The following sera were tested The positive negative cut off used was 5 x average background signal signal with diluent alone and was determined to be 0.195. Thus, sera with signals below 0.195 were rated those above were rated . Each sample was also evaluated by the commercially available ABBOTT HTLV III EIA kit Abbott Labs and by Western analysis. Tests on the normal blood donor samples indicated all except one were negative in the invention ELISA. This normal serum scored negative in the ABBOTT HTLV III EIA test, but was actually positive, as confirmed by Western analysis. The results of the tests on the 52 sera from LAD and AIDS patients and contacts are tabulated below The above results show that the invention ELISA using recombinant ARV proteins is at least as good as the ABBOTT HTLV III EIA test or Western analysis. In the invention ELISA reported in this example the yeast and bacterial extracts were added to the serum to bind serum antibodies to yeast and bacteria to prevent such antibodies from binding to the recombinant ARV 2 proteins. Both yeast and bacterial extracts were required since the recombinant polypeptides included polypeptides expressed in yeast and polypeptides expressed in bacteria. If all the polypeptides were expressed in the same type of organism, only one extract would be needed. For instance, if a p25 gag polypeptide expressed in yeast was substituted for the bacterially produced p25 gag polypeptide of the example, only yeast extract would be added to the serum samples. Nitrocellular strips 0.5 x 5 cm are spotted with 50 ng polypeptide in PBS spotting volume 2 µl . After spotting the strips are dried at room temperature for 1 hr or more. The strips are then post coated in a 5 solution of Carnation non fat dry milk in PBS, 0.01 Thimerosal, for 15 60 min at room temperature. Each test solution sample is diluted 1 50 in 0.5 ml of the post coating solution in a test tube. A post coated strip is then placed in the tube and incubated in the sample with rocking at 37 C for 1 hr. The strip is then removed from the tube and washed with post coating solution. The strip is then incubated for 15 min at room temperature in goat anti human Ig reagent labeled with horse radish peroxidase diluted 1 500 in post coating solution. After incubation in the labeled antibody, the strip is washed serially with PBS, 1 Triton, and distilled water. The strips are developed by incubating them in substrate solution see 23 above for 15 min at room temperature. Positive samples will cause a visually perceptible color change at the spotting site. Normal negative sera sample yield no color change or give a faint signal that is discernible from a positive signal. Competition assays may be run on sera giving faint signals to verify that they are negative. In the competition assay, polypeptide 10 25 µg ml is added to the test sample and incubated from 1 hr at 37 C before the strip is incubated in the sample. With authentic positive sera the signal is completely blocked by the added polypeptide, whereas with normal negative sera there is no change in signal. Samples of organisms that express the above described ARV 2 p25 gag and ARV 2 env polypeptides and the fusion protein of ARV 2 p31 and SOD were deposited at the American Type Culture Collection ATCC , 12301 Parklawn Drive, Rockville, Maryland under the provisions of the Budapest Treaty. The accession numbers and dates of these deposits are listed below.