# Synthetic gene for human lysozyme.

## Abstract
The present invention presents a DNA comprising synthetic gene for human lysoryme, method of producing the same, cells transformed with the DNA, the method of producing the transformed cells and human lysozyme by cultivating the transformant cells. This invention has made it possible to provide a large quantity of human lysozyme which heretofore was produced only in small quantities although it has medical effects for anti inflammation, hemostatic, tissue regeneration, anti tumor etc. and it is also used for eyewash as anti inflammation enzyme and for food antiseptic and further it does not have a side effect depending on the immune response as does egg white lysozyme when it was used for medical purposes.

## Claims
CLAIMS What is claimed is 1. DNA comprising synthetic gene for expressing human lysozyme of the DNA sequence AAG GTT TTT GAG AGA TGC GAA TTA GCC AGA TTC CAA AAA CTC TCT ACG CTT AAT CGG TCT ACT TTG AAG AGA TTG GGT ATG GAC GGC TAC TGA AAC TTC TCT AAC CCA TAC CTG CCG ATG CGT GGT ATT TCT TTA GCC AAC TGG ATG TGT GCA CCA TAA AGA AAT CGG TTG ACC TAC ACA CTT GCT AAG TGG GAA TCC GGC TAT AAC ACT GAA CGA TTC ACC CTT AGG CCG ATA TTG TGA AGA GCT ACC AAT TAC AAC GCT GGC GAC eGT TCT CGA TGG TTA ATG TTG CGA CCG CTG GCA TCT ACA GAC TAT GGT ATT TTC CAA ATT AAC AGA TGT CTG ATA CCA TAA AAG GTT TAA TTG TCT AGA TAT TGG TGT AAC GAT GGC AAG ACT AGA TCT ATA ACC ACA TTG CTA CCG TTC TGA CCA GGT GCC GTC AAC RTI ID 52.8 GCC TGT CAC TTA TCT GGT CCA CGG CAG TTG CGG ACA GTG AAT AGA TGC TCA GCT TTG CTT CAG GAC AAC ATT GCT ACG AGT CGA AAC GAA GTC CTG TTG TAA CGA GAT GCT GTT GCC TGC GCT AAG AGA GTT GTC CTA CGA CAA CGG ACG CGA TTC TCT CAA CAG CGT GAC CCA CAG GGT ATT AGA GCC TGG GTC GCA CTG GGT GTC CCA TAA TCT CGG ACC CAG GCT TGG AGA AAC AGA TGC CAA AAT AGA GAT CGA ACC TCT TTG TCT ACG GTT TTA TCT CTA GTC AGA CAA TAC GTT CAA GGT TGT GGT GTT CAG TCT GTT ATG CAA GTT CCA ACA CCA CAA 2. The DNA according to claim 1 which is a plasm pPHO lyslOl. 3. The DNA according to claim 1 which is a plasmid pGLD 906 5. 4. The DNA according to claim 1 which is a plasmid pPHO lyes115. 5. The DNA according to claim 1 which is a plasmid pLYS 301. 6. A method of producing DNA, characterized by synthesizing and jointing plural number of oligo nucleotides and optionally, inserting them in vectors, said DNA comprising synthetic gene for expressing human lysozyme of the DNA sequence AAG GTT TTT GAG AGA TGC GAA TTA GCC AGA TTC CAA AAA CTC TCT ACG CTT AAT CGG TCT ACT TTG AAG AGA TTG GGT ATG GAC GGC TAC TGA AAC TTC TCT AAC CCA TAC CTG CCG ATG CGT GGT ATT TCT TTA GCC AAC TGG ATG TGT GCA.CCA TAA AGA AAT CGG TTG ACC TAC ACA CTT GCT AAG TGG GAA TCC GGC TAT AAC ACT GAA CGA TTC ACC CTT AGG CCG ATA TTG TGA AGA GCT ACC AAT TAC AAC GCT GGC GAC CGT TCT CGA TGG TTA ATG TTG CGA CCG CTG GCA TCT ACA GAC TAT GGT ATT TTC CAA ATT AAC AGA TGT CTG ATA CCA TAA AAG GTT TAA TTG TCT AGA TAT TGG TGT AAC GAT GGC AAG ACT AGA TCT ATA ACC ACA TTG CTA CCG TTC TGA CCA GGT GCC GTC AAC GCC TGT CAC TTA TCT GGT CCA CGG CAG TTG CGG ACA GTG AAT AGA TGC TCA GCT TTG CTT CAG GAC AAC ATT GCT ACG AGT CGA AAC GAA GTC CTG TTG TAA CGA GAT GCT GTT GCC TGC GCT AAG AGA GTT GTC CTA CGA CAA CGG ACG CGA TTC TCT CAA CAG CGT GAC CCA CAG GGT ATT AGA GCC TGG GTC GCA CTG GGT GTC CCA TAA TCT CGG ACC CAG GCT TGG AGA AAC AGA TGC CAA AAT AGA GAT CGA ACC TCT TTG TCT ACG GTT TTA TCT CTA GTC AGA CAA TAC GTT CAA GGT TGT GGT GTT CAG TCT GTT ATG CAA GTT CCA ACA CCA CAA 7. Cells transformed with the DNA comprising synthetic gene for expressing human lysozyme of the DNA sequence AAG GTT TTT GAG AGA TGC GAA TTA GCC AGA TTC CAA AAA CTC TCT ACG CTT AAT CGG TCT ACT TTG AAG AGA TTG GGT ATG GAC GGC TAC TGA AAC TTC TCT AAC CCA TAC CTG CCG ATG CGT GGT ATT TCT TTA GCC AAC TGG ATG TGT GCA CCA TAA AGA AAT CGG TTG ACC TAC ACA CTT GCT AAG TGG GAA TCC GGC TAT AAC ACT GAA CGA TTC ACC CTT AGG CCG ATA TTG TGA AGA GCT ACC AAT TAC AAC GCT GGC GAC CGT TCT CGA TGG TTA ATG TTG CGA CCG CTG GCA TCT ACA GAC TAT GGT ATT TTC CAA ATT AAC AGA TGT CTG ATA CCA TAA AAG GTT TAA TTG TCT AGA TAT TGG TGT AAC GAT GGC AAG ACT AGA TCT ATA ACC ACA TTG CTA CCG TTC TGA CCA GGT GCC GTC AAC GCC TGT CAC TTA TCT GGT CCA CGG CAG TTG CGG ACA GTG AAT AGA TGC TCA GCT TTG CTT CAG GAC AAC ATT GCT ACG AGT CGA AAC GAA GTC CTG TTG TAA CGA GAT GCT GTT GCC TGC GCT AAG AGA GTT GTC CTA CGA CAA CGG ACG CGA TTC TCT CAA CAG CGT GAC CCA CAG GGT ATT AGA GCC TGG GTC GCA CTG GGT GTC CCA TAA TCT CGG ACC CAG GCT TGG AGA AAC AGA TGC CAA AAT AGA GAT CGA ACC TCT TTG TCT ACG GTT TTA TCT CTA GTC AGA CAA TAC GTT CAA GGT TGT GGT GTT CAG TCT GTT ATG CAA GTT CCA ACA CCA CAA 8. Transformed cells according to claim 7 which is S. cerevisiae AH 22R pPHO lyslOl. 9. Transformed cells according to claim 7 which is S. cerevisiae AH 22R pGLD 906 5. 10. Transformed cells according to claim 7 which is S. cerevisiae AH 22R pPHO lys115. 11. A method of producing the transformed cells as claimed in claim 7, which is characterized by inserting into cells the DNA comprising synthetic gene for expressing human lysozyme of the DNA sequence AAG GTT TTT GAG AGA TGC GAA TTA GCC AGA TTC CAA AAA CTC TCT ACG CTT AAT CGG TCT ACT TTG AAG AGA TTG GGT ATG GAC GGC TAC TGA AAC TTC TCT AAC CCA TAC CTG CCG ATG CGT GGT ATT TCT TTA GCC AAC TGG ATG TGT GCA CCA TAA AGA AAT CGG TTG ACC TAC ACA CTT GCT AAG TGG GAA TCC GGC TAT AAC ACT GAA CGA TTC ACC CTT AGG CCG ATA TTG TGA AGA GCT ACC AAT TAC AAC GCT GGC GAC CGT TCT CGA TGG TTA ATG TTG CGA CCG CTG GCA TCT ACA GAC TAT GGT ATT TTC CAA ATT AAC AGA TGT CTG ATA CCA TAA AAG GTT TAA TTG TCT AGA TAT TGG TGT AAC GAT GGC AAG ACT AGA TCT ATA ACC ACA TTG CTA CCG TTC TGA CCA GGT GCC GTC AAC GCC TGT CAC TTA TCT GGT CCA CGG CAG TTG CGG ACA GTG AAT AGA TGC TCA GCT TTG CTT CAG GAC AAC ATT GCT ACG AGT CGA AAC GAA GTC CTG TTG TAA CGA GAT GCT GTT GCC TGC GCT AAG AGA GTT GTC CTA CGA CAA CGG ACG CGA TTC TCT CAA CAG CGT GAC CCA CAG GGT ATT AGA GCC TGG GTC GCA CTG GGT GTC CCA TAA TCT CGG ACC CAG GCT TGG AGA AAC AGA TGC CAA AAT AGA GAT CGA ACC TCT TTG TCT ACG GTT TTA TCT CTA GTC AGA CAA TAC GTT CAA GGT TGT GGT GTT CAG TCT GTT ATG CAA GTT CCA ACA CCA CAA 12. A process for the production of human lysozyme, comprising the steps of cultivating a cell transformed with DNA comprising synthetic gene for expressing human lysozyme of the DNA sequence AAG GTT TTT GAG AGA TGC GAA TTA GCC AGATTC CAA AAA CTC TCT ACG CTT AAT CGG TCTACT TTG AAG AGA TTG GGT ATG GAC GGC TACTGA AAC TTC TCT AAC CCA TAC CTG CCG ATGCGT GGT ATT TCT TTA GCC AAC TGG ATG TGTGCA CCA TAA AGA AAT CGG TTG ACC TAC ACACTT GCT AAG TGG GAA TCCsGGC TAT AAC ACTGAA CGA TTC ACC CTT AGG CCG ATA TTG TGAAGA GCT ACC AAT TAC AAC GCT GGC GAC CGTTCT CGA TGG TTA ATG TTG CGA CCG CTG GeATCT ACA GAC TAT GGT ATT TTC CAA ATT AACAGA TGT CTG ATA CCA TAA AAG GTT TAA TTGTCT AGA TAT TGG TGT AAC GAT GGC AAG ACTAGA TCT ATA ACC ACA TTG CTA CCG TTC TGACCA GGT RTI ID 57.6 GCC GTC AAC GCC TGT CAC TTA TCTGGT CCA CGG CAG TTG CGG ACA GTG AAT AGATGC TCA GCT TTG CTT CAG GAC AAC ATT GCTACG AGT CGA AAC GAA GTC CTG TTG TAA CGAGAT GCT GTT GCC TGC GCT AAG AGA GTT GTC CTA CGA CAA CGG ACG CGA TTC TCT CAA CAGCGT GAC CCA CAG GGT ATT AGA GCC TGG GTCGCA CTG GGT GTC CCA TAA TCT CGG ACC CAGGCT TGG AGA AAC AGA TGC CAA AAT AGA GATCGA ACC TCT TTG TCT ACG GTT TTA TCT CTA GTC AGA CAA TAC GTT CAA GGT TGT GGT GTTCAG TCT GTT ATG CAA GTT CCA ACA CCA CAA and recovering human lysozyme from the culture.

## Description
SYNTHETIC GENE FOR HUMAN LYSOZYME FIELD OF THE INVENTION This invention relates to recombinant DNA techniques for producing human lysozyme. More particularly, it relates to the chemical synthesis of the human lysozyme gene and their expression thereof, methods related thereto, recombinant plasmids, transformed cells, and their products. BACKGROUND OF THE INVENTION Lysozyme is a comparatively small enzyme protein with a molecular weight of approx. 14,000. Lysozyme is distributed in living tissue and is considered to play a role as a defensive substance against bacillus infections by dissolving various bacilli. The method of lysozyme function is thought to involve hydrolysis of polysaccharide of the bacillus cell wall by eglucosidase activity. A good deal of lysozyme is contained in egg white and lysozyme with high purity can be isolated therefrom comparatively easily. Lysozyme is thus added to cheeze, sausage and marine products for their preservation or used for the purpose of converting bovine milk to human maternal milk Katsuya Hayashi and Taiji Imoto, Lysozyme , Nankodo, Japan 1974 .Further, as a medicinal agent, lysozyme is accepted as a hemostatic, anti inflammation, tissue regeneration, anti tumor etc. see for instance Collection of 34 Recent New Medicines 107, Yakuji Nippo,Tokyo, Japan, 1983 and it is also on sale as anti inflammatory enzyme agent. When the egg white derived lysozyme is used for medical purposes, sensitive symptoms such as rash, redness often result. These are generally regarded as a side effect of the immune response to the presence of a foreign protein. To overcome the disadvantages of egg white derived lysozyme, the present inventors have discovered a means for the mass production of human lysozyme, by means of gene manipulation.Human milk lysozyme consists of 130 amino acids see Table 1 and the amino acids thereof are differ by 53 when compared with the amino acids of egg white lysozyme. Disulphide bonds exist at four sites and the active sites are assumed to be Glu 35,Asp 53 and Trp 64 Japan Biotechnology Association,Biotechnology Date Book volume 1, 189 1979 3. Table 1Human Milk Lysozyme Lys Val Phe Glu Arg Cys Glu Leu Ala Arg Thr Leu Lys Arg Leu 20Gly Met Asp Gly Tyr Arg Gly Ile Ser Leu Ala Asn Trp Met Cys KoLeu Ala Lys Trp Glu Ser Gly Tyr Asn Thr Arg Ala Thr Asn Tyr so 60Asn Ala Gly Asp Arg Ser Thr Asp Tyr Gly Ile Phe Gln Ile gSn 70Ser Arg Tyr Trp Cys Asn Asp Gly Lys Thr Pro Gly Ala Val Asn ao Ala Cys His Leu Ser Cys Ser Ala Leu Leu Gln Asp Asn Ile Ala oo Asp Ala Val Ala Cys Ala Lys Arg Val Val Arg Asp Pro Gln Gly o 120Ile Arg Ala Trp Val Ala Trp Arg Asn Arg Cys Gln Asn Arg Asp 30 Val Arg Gln Tyr Val Gln Gly Cys Gly Val SUMMARY OF THE INVENTION The present invention presents a DNA comprising synthetic gene for human lysozyme, method of producing the same, cells transformed with the DNA, the method of producing the transformed cells and human lysozyme by cultivating the transformant cells. The DNA sequence of the human lysozyme gene has not been elucidated so far, but the present inventors have designed a gene of human lysozyme based on the above amino acid sequence and chemically synthesized it. Though a DNA sequence may become varied due to degeneracy of codons in designing the structural gene, the most suitable sequence has been determined in accordance with the following standards i The most acceptable codon in yeast which are considered to fit for expression of synthetic gene is used. ii A specific recognition site of restriction enzyme Xba I in this case is prepared to serve as a marker for cloning or to fascilitate reconstruction of the gene after constructing an expression vector, iii Sequences which are palindromic, or complementary to other sequence at the cohesive ends except proper combinations of the cohesive ends are avoided. The following base sequence has been determined to satisfy the above conditions. AAG GTT TTT GAG AGA TGC GAA TTA GCC AGA TTC CAA AAA CTC TCT ACG CTT AAT CGG TCT ACT TTG AAG AGA TTG GGT ATG GAC GGC TAC TGA AAC TTC TCT AAC CCA TAC CTG CCG ATG CGT GGT ATT TCT TTA GCC AAC TGG ATG TGT GCA CCA TAA AGA AAT CGG TTG ACC TAC ACA CTT GCT AAG TGG GAA TCC GGC TAT AAC ACT GAA CGA TTC ACC CTT AGG CCG ATA TTG TGA AGA GCT ACC AAT TAC AAC GCT GGC GAC CGT TCT CGA TGG TTA ATG TTG CGA CCG CTG GCA TCT ACA GAC TAT GGT ATT TTC CAA ATT AAC AGA TGT CTG ATA CCA TAA AAG GTT TAA TTG TCT AGA TAT TGG TGT AAC GAT GGC AAG ACT AGA TCT ATA ACC ACA TTG CTA CCG TTC TGA CCA GGT GCC GTC AAC GCC TGT CAC TTA TCT GGT CCA CGG CAG TTG CGG ACA GTG AAT AGA TGC TCA GCT TTG CTT CAG GAC AAC ATT GCT ACG AGT CGA AAC GAA GTC CTG TTG TAA CGA GAT GCT GTT RTI ID 4.5 GCC TGC GCT AAG AGA GTT GTC CTA CGA CAA CGG ACG CGA TTC TCT CAA CAG CGT GAC CCA CAG GGT ATT AGA GCC TGG GTC GCA CTG GGT GTC CCA TAA TCT CGG ACC CAG GCT TGG AGA AAC AGA TGC CAA AAT AGA GAT CGA ACC TCT TTG TCT ACG GTT TTA TCT CTA GTC AGA CAA TAC GTT CAA GGT TGT GGT GTT CAG TCT GTT ATG CAA GTT CCA ACA CCA CAA BRIEF DESCRIPTION OF THE DRAWINGS Figure 1 and figure 2 show a scheme for constructing the plasmid pPHO17 and pGLD 906 described in Reference Examples 1 and 2 respectively. Figure 3 shows the restriction enzyme cleavage map of promoter cloning vector pBTM 126 and that of pBTM 128 in which promoter is inserted which are described in Reference Examples 3 and 4. Figure 4 shows the base sequence of a promotor DNA obtained from pBTM 128. Figure 5 shows a scheme for constructing the plasmid pBTM 134 described in Example 6. Figure 6 is the restriction enzyme cleavage map of pBTM 127 in which neutral protease gene is inserted. Figure 7 and 8 show sub cloning of neutral protease gene. Figure 9 is the restriction enzyme cleavage map of DNA containing neutral protease gene. In the drawing Bg indicatesBgl II, T Taq I, Sa Sac II, Ha Hae III, S Sau 3AI, A AccI,Bc Bcl I and H Hind III. Figure 10 is a part of the base sedquence of neutral proastase gene. Figures 11 and 12 show a scheme for constructing the plasmid pTEX 201 and pENX 374, which are described in ReferenceExample 10. Figure 13 and 14 show a part of sequence coding mature protein of penicillinase of pENX 374. Figure 15, 16 and 17 are drawings showing the synthetic formula of di nucleotide block, the extension of oligo nucleotide chain, and the synthetic formula of human lysozyme synthetic gene by arranging and linking of oligo nucleotides. Figure 18 shows a scheme for constructing the plasmid pPHOlys 115 described in Examples 7 and 8. Figure 19 shows a copy drawing of electrophoresis picture confirming the expression of human lysozyme synthetic gene of the present invention described in Example 10 and also a copy drawing of its comparative example and contrast example. Figure 20 shows a DNA sequence of Taq I Xho I fragment of human lyzosyme gene. Figure 21 shows a scheme for constructing the plasmid secreting human lysozyme. Figure 22 shows a copy drawing of electrophoresis picture of the culture supernatant of Example 19 and authentic human lysosyme. DETAILED DESCRIPTION This invention first relates to a DNA comprising the synthetic gene above shown for expression of human lysozyme.The DNA of the invention comprises not only the above synthetic gene itself but a DNA in which a certain kind of restriction enzyme recognition site, start codon and stop codon are connected at its 5 and 3 end. Such modifications make it convenient to manipulate recombining gene and the like. For example, at the 5 end of the above DNA sequence, the sequence has Xho I split site of TCGAGATG STAG,ATG as start codon, and at the 3 end, Xho I split site of TAATAGC ATTATCGAGCT TAA and TAG as stop codon are cited. Besides, recombinant plasmid containing above DNA comes under the category of the present invention. For instance, this gene can be produced by synthesizing plural number of oligo deoxy nucleotides ranging 2 to 80 and linking them each other. For example, as shown by Table 2, they are divided into 52 oligo nucleotide blocks and are able to be synthesized in accordance with the method mentioned in H. Ito, Y. Ike, S. Ikuta and K. Itakura NucleicAcids Res., 10, 1755 1982 1. Oligo nucleotide blocks are hybridized according to a known method and linked enzymatically for example, Agarwal et al, Nature 227, 27 34 1970 . Table 2Upper Chain Lower ChainNo. No. U1 TCGAGATGAAGGTTT L26 TCGAGCTATTAAACU2 TTGAGAGATGCGAAT L25 ACCACAACCTTGAACU3 TAGCCAGAACTTTGAAG L24 GTATTGTCTGACATCU4 AGATTGGGTATGGAC L23 TCTATTTTGGCATCT U5 GGCTACCGTGGTATT L22 GTTTCTCCAAGCGACU6 TCTTTAGCCAACTGG L21 CCAGGCTCTAATACCCTGU7 ATGTGTCTTGCTAAG L20 TGGGTCACGGACAACU8 TGGGAATCCGGCTATAAC L19 TCTCTTAGCGCAGGCU9 ACTAGAGCTACCAAT L18 AACAGCATCAGCAATU10 TACAACGCTGGCGAC L17 GTTGTCCTGAAGCUll CGTTCTACAGACTATGG L16 AAAGCTGAGCAAGATU12 TATTTTCCAAATTAACT L15 AAGTGACAGGCGTTGACU13 CTAGATATTGGTG L14 GGCACCTGGAGTCTTGCU14 TAACGATGGCAAGACTC L13 CATCGTTACACCAATATU15 CAGGTGCCGTCAACGCC L12 CTAGAGTTAATTTGGU16 TGTCACTTATCTTGC Lll AAAATACCATAGTCTGT U17 TCAGCTTTGCTTCAG LlO AGAACGGTCGCCAGCU18 GACAACATTGCTGAT L 9 GTTGTAATTGGTAGCU19 GCTGTTGCCTGCGCT L 8 TCTAGTGTTATAGCCGU20 AAGAGAGTTGTCCGT L 7 GATTCCCACTTAGCAAGU21 GACCCACAGGGTATT L 6 ACACATCCAGTTGGCU22 AGAGCCTGGGTCGCT L 5 TAAAGAAATACCACGU23 TGGAGAAACAGATGC L 4 GTAGCCGTCCATACCU24 CAAAATAGAGATGTC L 3 CAATCTCTTCAAAGT U25 AGACAATACGTTCAAGG L 2 TCTGGCTAATTCGCATCU26 TTGTGGTGTTTAATAGC L 1 TCTCAAAAACCTTCATC The method of division for fragments is not restricted to the above mentioned but may be varied if attention is paid to avoid self association. Next step is to insert this gene in a proper vector for sub cloning to enrich it. Though any kind of vector may be used so far as this gene can be inserted,Escherichia coli vector pACYC 177, Escherichia coli yeast shuttle vector pPHO17, pGLD 906 and Bacillus subtilis vector pTEX 201 are cited as specific examples. Various host organisms are transformed by using vectors containing synthetic genes thus obtained. The method of transformation itself is well known, but when Escherichia coli is used as a host organism, transformation is performed by the method of Cohen et. al., Cohen, S. N. et. al., Proc. Natl.Acad. Sci. USA, 69, 2110 1972 3, when yeast is used as a host organism, by the method of Hinnen et. al., Hinnen A et. al.,Proc. Natl. Acad. Sci. USA, 75, 1927 1978 3 and when Bacillus subtilis is used as a host organism, by protoplast method jChang, S. Cohen S. N. Gen. Genet., 168, 111 1979 or competent method Dubnau, D. Abelson, R. D, J. Mol. Biol. 56, 209 1971 1, respectively. As host organisms, for instance, E.coli 294, E. coli w3110, E. coli C600, B. subtilis lAl, B.subtilis 1A339, B. subtilis 1A340 may be used. Then, in order to express these genes, the plasmid DNA in which these genes are inserted is first isolated from the transformants by for example, an alkaline extraction method Birnboim, H. C. Doly, J. Nucleic Acids Res., 7, 1513 1979 . By treating plasmid DNA thus obtained with proper restriction enzyme the inserted genes are cutout and can be isolated by agarose gel electrophoresis or poly acrylamide gel electrophoresis. All of these processes are well known and they are mentioned in literature in detail Isee for exampleMolecular Cloning 1982 , Cold Spring Harbor Laboratory . The isolated gene is linked downstream from the promoter of a proper expression vector in the correct direction to construct an expression plasmid.Many expression vectors have already been known, so any one can be used so far as it is practically fit for an expression for this gene. To be more specific, pPHO17, pGLD906, ptrp771 Japanese Patent UnexaminedPublication No. 201796 83 , pRC23 Japanese Patent UnexaminedPublication No. 189197 83 , pBTM134, pTEX201 etc. are cited as previously mentioned, but pPHO17 and pGLD906 are preferable. Animal cells and other eukaryotic cells may be used as the host organisms for expression. Plasmids having a Xho I site downstream from a promoter are desirable as vectors. The host cells are transformed with the expression plasmid thus obtained to derive an intended recombinant.Transformants thus obtained are cultivated by a well known method. When Escherichia coli is used, M9 culture medium Millu, J., Experiments in Molecular Genetics, 431 433 ColdSpring Harbor Laboratory , New York, 1972 containing glucose and casamino acid, for instance, L broth and Penassay broth are listed. For better functioning of promoter reagents such as 3 indolylacryl acid may be added. They are generally cultivated at 1500 430C range, preferably at 280C 400C for 2 24 hours, preferably 4 16 hours, with aeration and or stirring if necessary. When yeast is used, Burkholder minimum medium Bostian, K. L. et. al., troc. Natl. Acad. Sci. USA, 77, 4505 1980 is cited as a culture medium, for example.In this instance the culturing temperature generally ranges from 1500 to 400C, preferably from 243C, to 370C for 10 to 96 hours, desirably 24 to 72 hours, srith aeration and or stirring if necessary. When Bacillus subtilis is used, L broth, BHI BrainHeart Infusion etc. ars cited as culture medium. The cultivating temperaturz generally ranges from 1500 to 4200, preferably 240C to 37JC, and as for pH of the culture medium the starting pH should be approx. 5.0 9.0, preferably approx. 6.5 7.5. The cu3cultivating time is to be generally about 2 72 hours, preferably 3 48 hours. At the incubation cells and the supernatant are separated by a well known process.After being suspended in proper buffer solution, cells are broken, in the case of the transformants of E. coli or B.subtilis by freeze melting, lytic enzyme treatment or ultrasonic wave treatment individually or jointly and in the case of the transformants of yeast by Zymolyase Kirin Beer,Japan which may be augmented by the use of mechanical destruction means such as glass beads. Further, if necessary, lysozyme produced may be extracted by adding surface active agents such as Triton X 100, deoxycolate etc. or protein denaturizing agents such as urea, guanidine hydrochloride etc.Lysozymes thus produced inside or outside of cells are purificated by the process of usual protein purification method such as salting out, isoelectric point precipitation, crystallization, ion exchange chromatography, high performance liquid chromatography HPLC, FPLC etc. , for instance, to obtain the desired lysozyme product. EXAMPLES The present invention is explained more concretely by reference examples and examples but it is not restricted by these examples.Reference Example 1 Constructing an Expression Vector Containing Yeast Repressible Acid Phosphastase Promoter PHO5 P After allowing 20 units of restriction enzyme BamHI Takara Shuzo, Japan and 20 units of restriction enzyme SalI Takara Shuzo, Japan to exert action on 50 pg of Escherichia coli plasmid pJAl Kramer, R. A. and Anderson, N., Proc. Natl.Acad. Sci. USA, 77, 6541 1980 3 containing 7.9 kb DNA fragments containing yeast Saccharomyces cerevisiae S288C strain derived repressible acid phosphastase gene PHO5 and constitutive acid phosphastase gene PHO3 in 100 pl reaction solution 10 mMTris HCl pH8.0 , 7 mM MgC12, 100 mM NaCl, 2 mM 2 mercaptoethanol3 at 37 C for 3 hours, this reaction mixture was electrophoresed in buffer solution 100 mM Tris HC1, 100 mM boric acid, 2 mM EDTA pH8.0 using a 1.0 agarose Sigma slab gel at 140V, for 2 hours. After electrophoresis, gels containing 0.63 kb DNA fragments were put into the dialysis tube and sunk in buffer solution for electrophoresis to elute these DNA fragments out of the gel electrically McDonell, M.W. et. al., J. Mol. Biol, 110, 119 1977 1. After extracting the solution in the dialysis tube with phenol and then ether,DNA was precipitated at 200C by adding NaCl up to 0.2 M to be followed by 2 volumes of ethanol. After allowing 2 units restriction enzyme Bam HI and 2 units restriction enzyme Sal I to exert action on 1 jig plasmid pSHl9 in 20 pl of reaction solution lOmM Tris HCl pH8.0 , 7mM MgC12, 100mM NaCl, 2mM 2 mercaptoethanol at 370C for 2 hours, this reaction solution was electrophoresed with a 0.88 agarose slab gel under the condition mentioned above. After electrophoresis, 8.0 kb DNA fragments were extracted from the gel in the same manner mentioned above and deproteinized to precipitate DNA with cold ethanol See Figure 1 . 400 ng of this 8.0 kb DNA fragments and 200 ng of 0.63 kb DNA fragments previously mentioned were mixed and ligated each other in 20 pl reaction solution 66mM Tris HCl pH 7.6 , 6.6mM MgC12, l0mM Dithiothreitol, lmM ATP, 2 unit T4DNA TakaraShuzo, Japan at 140C for one night. By using this reaction solution Escherichia coli 294 was transformed according to the method by Cohen et. al., mentioned previously. Plasmid DNA was isolated by the alkaline extraction method previously mentioned from transformants selected based on ampicillin resistance as a marker.Its molecular weight and cleavage pattern by restriction enzyme were studied, and then was separated plasmid pPHO 12 in which 0.63 kb DNA fragment isolated from pJA 1 was inserted at Bam HI Sal I site of pSH 19 See Figure 1 . After allowing 2 units restriction enzyme Sal I to exert action on 3 jig of plasmid pPHO 12 DNA in 20 oul of reaction solution 10 mM Tris HCl pH7.5 , 7 mM MgC12, 175 mMNaCl, 0.2 M EDTA, 7 mM 2 mercaptoethanol at 370C for 2 hours, this solution was deproteinized with phenol to precipitate DNA with cold ethanol. After allowing 12 units BAL 31 nuclease Bethesda Research Laboratories to exert action on 3 jig of this DNA in 50 rl of reaction solution 20 mM Tris HCl pH 8.1 , 12 mM CaC12, 12 mM MgCl2, 1 mM EDTA at 300C for 2 minutes, the solution was deproteinized to precipitate the DNA fragment with cold ethanol See Figure 1 . By allowing 3 units T4 polynucleotide kinase TakaraShuzo, Japan to exert action on 200ng of Xho I linker d CCTCGAGG New England Bio Labs in 50 Xul of reaction solution 50 mM Tris HCl pH 7.6 , 10 mM MgC12, 10 M 2 mercaptoethanol, 100 pM ATP at 370C for 1 hour, the 5 end was phosphorylated. 4ng of Xho I linker which was phosphorylated at the 5 end 5 P d CCTCGAGG and 400 ng of pPHO 12 DNA which was treated with the BAL 31 previously mentioned were mixed and combined by the action of T4 DNA ligase under the condition previously mentioned. With this solution Escherichia coli 294 was transformed according to the method of Cohen et. al.Plasmid DNA was isolated by the alkaline extraction method previously mentioned from transformants with ampicillin resistance as a marker and then plasmid pPHO 17 which gave 0.55 kb in size after double digestions with Bam HI and Xho I. As the result of the analysis of DNA base sequence in accordance with dideoxynucleotide method Sanger, F. et al, Proc. Natl. Acad. Sci. USA, 74, 5463 1977 it has been made clear that 20 bp upstream from the start codon ATG of PHO 5 were eliminated by BAL 31 nuclease treatment See Figure 1 .Reference Example 2 Construction of an Expression Vector Containing Yeast Glyceraldehyde 3 phosphate Dehydrogenase Promoter GLD p 1 Cloning of Glyceraldehyde 3 phosphate DehydrogenaseGene GLD In accordance with the method of Crea, R. et. al., previously mentioned synthesized was 5 AGCAACTCTAACCAT 3 which is complementary with oligo nucleotide coding for 5 amino acids from the N terminal end of pgap 491 Holland, J. P. et.al., J. Biol. Chem, 258, 5291 1983 of GLD. By allowing 10 iCi of 32P ATP Amersham and 10 units T4 polynucleotide kinase to exert action on 1 Xg of this oligonucleotide in 30 yul of reaction solution 50mM Tris HCl pH 7.6 , 10 mM MgC12, 10mM 2 mercaptoethanol at 3700 for 30 minutes and the 5 end was labelled with 32p. 10 1 of 200 mMEDTA pH 8.0 was added to this reaction solution. After being deproteinized with phenol, the reaction solution was applied toSepharose 4B Pharmacia column which was equilibrated with TEN buffer solution 10 mM Tris HCl pH 8.0 , 200 mM Nail, 1 mMEDTA and the oligo nucleotide eluted around the void volume was collected and used as a probe for screening of GLD genes.By using nitrocellulose filter Schleichu and Schull and this probe Southern blotting Southern, E. M., J. Mol. Biol., 98, 503 1975 was performed to find that the probe hybridized well with the sample of fraction No. 7 containing 2.0 2.3 kbDNA fragments. 0.ljig of pTR 262 Roberts, T. M. et. al., Gene 12, 123 1980 digested with Hind III and 0.2 jig of DNA of fraction No. 7 were mixed and ligated by reaction of T4 DNA ligase under the condition described at Reference Example 1.Escherichia coli DHl Maniatis, T. et. al., Molecular Cloning,Cold Spring Harbor Laboratory 254 255 1982 was transformed with this reaction solution and 1200 of tetracycline resistant transformants were obtained. Then transformants which hybridized well with 32p labelled probe were selected by colony hybridization. Plasmid pGLD 9 was isolated from these transformants by the alkaline extraction method previously mentioned and 2.2 kb insert DNA was detected by digestion with Hind III. As the result of inspection of this insert DNA by the method of Southern, it was confirmed that the insert DNA hybridized with the probe See Figure 2 . 2 Isolation of GLD Promoter Fragments After allowing 50 units of restriction enzyme Hind III to exert action on 100 fg of plasmid pGLD 9 DNA in 200 jil of reaction solution lOmM Tris HCl pH 7.5 , 7 mM MgCl2, 60 mMNaCl at 370C for 3 hours, the solution was electrophoresed with a 1.0 agarose slab gel under the condition described atReference Example 1 previously mentioned. After electrophoresis, a 2.2 kb DNA fragment was obtained from the gel in the manner described at Reference Example 1.After allowing 10 units restriction enzyme Hinf I Takara Shuzo,Japan to exert action on 10 jig of this 2.2 kb DNA fragment in 50 pl of reaction solution 10 mM Tris HCl pH 7.5 , 7 mM MgOl2, 0 100 mM NaCl, 7 mM 2 mercaptoethanol at 37 C for 2 hours, an experiment was performed with the probe for GLD according to the method of Southern to find hybrydization of this probe with the 0.5 kb DNA fragment See Figure 2 . After allowing restriction enzyme Hha I Takara Shuzo,Japan and Taq I New England BioLabs , 10 units of each, to exert action on the above 0.5 kb DNA fragment in 30 jil of reaction solution 10 mM Tris HCl pH 7.5 , 50 mM NaCl, 10 mM MgOl2, 1 mM dithiothreitol at 370C for 3 hours, the solution was electrophoresed with 1.5 agarose slab gel under the condition described at Reference Example 1. After electrophoresis, 0.36 kb DNA fragment was obtained from the gel in the manner described at Reference Example 1 See Figure 2 . The 0.36 kb DNA fragment was treated by the action ofDNA polymerase I large fragments under the condition described at Reference Example 1 to alter the adhesive end of Taq I to a blunt one. Then, 1 pg of the fragment and 50 ng of phosphorylated Xho I linker described at Reference Example 1 were mixed and ligated by the action of T4 DNA ligase under the condition described at Reference Example 1. After the reaction, an excess amount of Xho I was added to the solution to act at 37 C for 4 hours. Then 0.36 kb DNA fragment with linker jointed were separated with Sepharose 4B column Pharmacia, 0.25 X 25 cm . 10 ug of the above mentioned 2.2 kb DNA fragment was treated by the action of DNA polymerase I large fragments under the condition described at Reference Example 1 to alter the adhesive end to a blunt one. And then, by the action of T4 DNA ligase under the condition described at Reference Example 1 they were ligated with 50 ng of Bam HI linker 5 P d CGCGGATCCGCG New England BioLabs which was phosphorylated under the condition described at ReferenceExample 1. After the reaction, 20 units Bam HI was added to the solution to act at 370C for 3 hours. Then, the solution was applied to a Sepharose 4B column to separate 2.2 kb of linker joined DNA fragment.After allowing 2 units restriction enzyme Hha I to exert action on 6 jig of the DNA fragment in 50 jil of reaction solution 10 mM Tris HCl pH 7.5 , 50 mM NaCl, 10 mM MgC12, 1 mM dithiothreitol at 370C for 2 hours, the solution was electrophoresed with a 1.0 agarose slab gel under the condition described at Reference Example 1. After electrophoresis, a 0.75 kg DNA fragment was obtained from the gel in the manner described at Reference Example 1 See Figure 2 . 3 Construction of Expression Vectors After allowing 6 units of a restriction enzyme Sal I to exert action on 5 pg of Escherichia coli yeast shuttle vector pSH 19 in 20 jil of reaction solution 10 mM Tris HCl pH 7.5 , 7 mM MgC12, 175 mM NaC1, 0.2 mM EDTA, 7 mM 2 mercaptoethanol at 37 C for 2 hours, the solution was deproteinized and precipitated. 1 jig of this DNA was treated by the action of DNA polymerase I large fragments under the condition described in Reference Example 1 to alter the adhesive end of Sal I to a blunt one. 500 ng of this DNA fragment was mixed with 50 ng of phosphorylated Xho I linker described at Reference Example 1 and they were ligated by reaction of T4 DNA ligase under the condition described inReference Example 1. Escherichia coli DHl was transformed with this reaction solution and out of those ampicillin resistant transformants obtained were transformants containing plasmid pSH 19 1 whose Sal I site of pSH 19 was altered to Xho I site See Figure 2 . After allowing restriction enzyme Bam HI and Xho I, 10 units of each, to exert action on 10 ug of plasmid pSHl9 1 DNA in 50 pl of reaction solution 10 mM Tris HCl pH 7.5 , 7 mMMgC12, 100 mM NaCl, 7 mM 2 mercaptoethanol at 370P for 2 hours, the solution was electrophoresed with a 1.0 agarose slab gel under the condition described at Reference Example 1.After electrophoresis, an 8.0 kb DNA fragment was obtained from the gel in the manner described at Reference Example 1. 500 ng of the 8.0 kb DNA fragment, 200 ng of 0.36 kbDNA fragment and 200 ng of 0.75 kb DNA fragment were mixed and ligated by reaction with T4 DNA ligase under the condition described at Reference Example 1. With this reaction solutionEscherichia coli DHl was transformed and out of those ampicillin resistant transformants separated were recombinants containing plasmid pGLD 906 which consisted of three types ofDNA fragments.Reference Example 3 Construction of Promoter Cloning Vector pBTM 126 Plasmid pBTM 126 was prepared as below in accordance with the method of Williams et. al., DNA was prepared fromBacillus pumilus NCIB 8600 IFO 12089 obtained from theInstitute for Fermentation, Osaka, and the DNA 6.5 72g was reacted with 40 units of restriction enzyme Eco RI at 370C for 1 hour to be cleaved, and then precipitated with ethanol after being heated at 680C for 15 minuites. While, plasmid pUB 110 2.0 pg was reacted with 20 units restriction enzyme Eco RI at 370C for 1 hour to be cleaved, and then precipitated with ethanol after being heated at 680C for 15 minutes.Both precipitates were dissolved into water, mixed, and 60 n mole ofATP, 10 units of T4 DNA ligase Takara Shuzo, Japan and ligation buffer solution were added to the mixture. Reaction solution 100 pl thus prepared was kept for 30 hours at 1100 and precipitated with ethanol. The precipitate was dissolved into TE buffer solution 50 rl and with 25 Xul of this solutionBacillus subtilis MI 114 was transformed. Plasmid was prepared from transformants with chloramphenicol resistance and named pBTM 124. Then, plasmid pBTM 124 2.5 pg was treated by reaction of 14 units of restriction enzyme Pst I at 370C for 1 hour to be cleaved, and then precipitated with ethanol after being heated at 68 C for 15 minutes.The precipitate was dissolved into water and 66 n mole of ATP, 10 units of T4 DNA ligase Takara Shuzo, Japan and ligation buffer solution was added to it. Thus prepared reaction solution 100 pl was kept for 24 hours at 1100 and precipitated with ethanol. The precipitate was dissolved into TE buffer solution to transformBacillus subtilis MI 114 and plasmid was prepared from those kanamycin resistant transformants. Thus prepared plasmid was named pBTM 125.This plasmid is lacking in the promoter region of CAT gene Pst I fragment of plasmid pBTM124. Then, plasmid pBTM 125 2.5 fg was treated by reaction of 18 units of restriction enzyme Bam HI and 15 units of restriction enzymeBgl II at 370C for 1 hour to be cleaved, and then precipitated with ethanol after being treated with heat at 680C for 15 minutes. After the precipitate was dissolved into water the solution was kept at 1100 for 28 hours in a reaction solution 100 1 containing 66 n mole of ATP, 13 units T4 DNA ligase Takara Shuzo, Japan and ligation buffer solution. With this solution Bacillus subtilis MI 114 was transformed.Plasmid was prepared from kanamycin resistant transformants and named pBTM 126 See Figure 3 .Reference Example 4 Cloning of Promoter Promoter cloning vector pBTM 126 2.1 pg obtained byReference Example 3 was reacted with restriction enzyme Pst I 8 units at 370C for 1 hour to be cleaved. It was further reacted with restriction enzyme Eco RI 5 units at 370C for 1 hour to be cleaved, and then precipitated with ethanol after it was heated at 680C for 15 minutes and the reaction was stopped.A chromosome sample 6.2 ug of Bacillus subtilis JB 1 168 IFO 14144 was reacted with Pst I 24 units and with Eco RI at 370C for 1 hour respectively to be cleaved, and then precipitated with ethanol after heat treatment at 680C for 15 minutes. Both precipitates were dissolved in water, mixed and reacted each other at 1100 for 24 hours in the presence of ATP 66 n mole and T4 DNA ligase Takara Shuzo, Japan 10 units to be precipitated with ethanol. The precipitate was dissolved in TE buffer solution 10 mM Tris HCl buffer pH 8.0 , 1 mMEDTA and B. subtilis MI 114 was transformed in accordance with the protoplast method S. Chang and S. N. Cohen Mol. Gen.Genet., 168, 111 1979 . When transformants were selected with DM3 agar plate Mol. Gen. Genet., 168, 111 1978 containing 12.5 fg ml of chloramphenicol, 956 transformants were obtained. Further, when transformants were replicated with an agar plate of Brain Heart Infusion Difco, USA containing 200 ng ml of chloramphenicol, 20 strains were grown.These transformants harbor plasmids in which DNA fragments containing strong promoter activity are inserted. One of these plasmids which showed strong CAT activity measurment by the method by Williams et. al., J. Bacteriol., 146, 1162 1981 3 was named pBTM 128 See Figure 3 .Reference Example 5 Isolation of Promoter DNA Fragment and its Property Plasmid pBTM 128 221 jig described at ReferenceExample 4 was reacted with Pst I 208 units and Eco RI 220 units at 37 C for 1 hour respectively to be cleaved, and was then electrophoresed with a 10 polyacrylamide gel. The gel was soaked in ethidium bromide solution to be colored and promoter DNA fragments detected with ultra violet lamp were collected. After being eluted electrically from the gel, a DNA fragment was extracted with phenol and then with ether to be precipitated with ethanol. The precipitate was dissolved in TE buffer solution to isolate 3.55 rg of promoter DNA fragment. The size of promoter DNA fragment thus obtained was measured by electrophoresis with 4 polyacrylamide gel. On the assumption that the Hae III decomposition product of plasmid pBR 322 was used as a standard, the fragment was figured to be approx. 120 bp. The base sequence of this fragment has been determined as shown by Figure 4 in accordance with dinucleotide synthetic chain termination method previously cited . This fragment consists of 117 bp and has an Eco RI site at the 5 end and a Pst I site at the 3 end. In the fragment was found a base sequence which seemed to be 10 region and 35 region. Reference Example 6 Construction of Expression Vector pBTM 134 Plasmid pBTM 128 7.7 pg was reacted with restriction enzyme Pst I 51 units at 370C for 1 hour to be cleaved, and then treated with 0.75 units Escherichia coli alkaline phosphastase at 65 C for 30 minutes. Reaction products were collected by precipitation with ethanol after the reaction solution was extracted with phenol and then with ether. The precipitate was dissolved in a small amount of water, and synthetic nucleotide of 8 bp GGAGGTAT 200 ng with phosphorylated 5 end, synthetic nucleotide of 14 bpCGATACCTCCTGCA 350 ng with phosphorylate 5 end, 100 n mole of ATP, 28 units T4 DNA ligase Takara Shuzo, Japan and ligation buffer solution were added to this precipitate.The thus obtained reaction solution 100 1 was kept at 11 C for 20 hours to be precipitated with ethanol. After this precipitate was dissolved in a small amount of water and treated with 25 units Ola I at 370C for 1 hour, it was applied to a Sepharose 4B column to eliminate small oligo nucletides and intended object was collected by precipitation with ethanol. The precipitate was dissolved in water and 100 n mole of ATP, 28 units T4 DNA ligase Takara Shuzo, Japan and ligation buffer solution were added to this to prepare reaction solution.After this reaction solution 100 1 was kept at 1100 for 20 hours to ligate Ola I sites, Bacillus subtilis MI 114 was transformed with 50 1 of the reaction solution in accordance with protoplast method previously cited . Plasmid was isolated from kanamycin and chloramphenicol resistant transformants and this plasmid was named pBTM 134 see Figure 5 .Reference Example 7 Cloning of Neutral Protease Gene A chromosome DNA was prepared from Bacillus amvloliquefaciens IFO 14141 Institute for Fermentation,Osaka, Research Communications No. 11 1983 1 by the well known method Methods in Enzymology, 68, 342 1979 1. 8.8 pg of the chromosome DNA was reacted with 36 units of restriction enzymeBgl II at 370C for 50 minutes to be cleaved. After being treated at 650C for 15 minutes, the DNA was precipitated with ethanol. Cloning vector pBTM 119 Japanese Patent UnexaminedPublication No. 55897 84 2.9 pg was reacted with 30 units of restriction enzyme Bgl II at 370C for 50 minutes to be cleaved.After 0.5 units of alkaline phosphatase was added to the above product to keep it at 65 C for 30 minutes, the solution was extracted with phenol and then with ether to be precipitated with ethanol. Both precipitates were dissolved in water to be mixed. Reaction solution 100 pl prepared by adding 66 n mole of ATP, 28 units T4 DNA ligase Takara Shuzo, Japan and ligation buffer solution to this aqueous solution was kept at 1100 for 32 hours and then precipitated with. ethanol. The precipitate was dissolved in TE buffer solution to transform B.subtilis 1A274. Colonies forming halo 1H. Uehara et al., J.Bacteriol., 119, 82 1974 1 on the plate of CP agar medium yeast extract 0.2 , pepton 1 , salt 0.28, casein 1 , agar 1.5 , pH 7.3 were selected from kanamycin resistant transformants, and plasmid isolated from one of the strains was named pBTM 127. This plasmid is one with Bgl II fragment 3.9 kb containing neutral protease gene of B. amyloliguefaciens IFO 14141 inserted in the Bgl II site of plasmid pBTM 119 See Figure 6 . B. subtilis 1A 274 pBTM 127 containing this plasmid pBTM 127 has been deposited with the Institute forFermentation, Osaka under the designation IFO 14306.Reference Example 8 Sub cloning of Neutral Protease Gene After plasmid pBTM 127 80 Xug obtained in ReferenceExample 7 was reacted with restriction enzyme Bgl II 200 units at 370C for 1 hour to be cleaved and electrophoresed with 0.78 agarose gel, the gel of band with smaller molecular weight was cut out. Collecting DNA in the gel in accordance with electrophoresis elution method Y. Takagi, GeneManipulation Manual, Kodansha Scientific, Japan 1982 , approx. 10 fg of Bgl II fragment 3.9 kb was obtained. Thus obtained Bgl II fragments 3rig were treated with restriction enzyme Hind III 6 units at 370C for 1 hour to be cleaved in 3 fragments, A fragment Bgl II Hind III, approx. 0.8 kb , B fragment Hind III Hind III, approx. 1.1 kb and C fragment Hind III Bgl II, approx. 1.9 kb , and precipitated with ethanol. Plasmid pBTM 126 1 ug was reacted with restriction enzyme Hind III 5 units at 370C for 1 hour, cleaved and precipitated with ethanol. Both precipitates were dissolved in water. Reaction solution 100 1 prepared by adding 100 n mole of ATP, 8.4 units T4 DNA ligase Takara Shuzo, Japan and ligation buffer solution to the above aqueous solution was kept at 1100 for 24 hours to transform B. subtilis 1A 274.As the result of preparing plasmid from transformants of kanamycin resistance and chloramphenicol sensitivity, two types of plasmids were obtained and named pBTM508 6.2 kb and pBTM 515 7.7 kb respectively. Plasmid pBTM 515 is one with A fragment and C fragment inserted in the Hind III site of plasmid pBTM 126 mutually in reverse direction and plasmid pBTM 508 is one with B fragment inserted at the Hind III site of plasmid pBTM 126 see Figure 7 . Examining capability of forming halo of the plasmid harboring strain on the CP culture plate, strains harboring pBTM 515 showed the capability of forming halo but those harboring pBTM 508 did not show the capability. After plasmid pBTM 127 1 pg was reacted with Eco RI 3 units and Bgl II 2 units , and plasmid pUB 110 1 veg withEco RI 3 units and Bam HI 5 units at 370e for 1 hour respectively to be cleaved, they were heated at 650C for 10 minutes to stop the reaction, and precipitated with ethanol. Both precipitates were dissolved in water and mixed. To the above solution were added 100 n mole of ATP, 8.4 units of T4DNA ligase Takara Shuzo, Japan and ligation buffer solution, and the resulting reaction mixture was kept at 1100 for 24 hours to be used for transformation of B. subtilis 1A 274. A strain forming halo was selected from kanamycin resistant transformant strains and named plasmid pBTM 523. This plasmid is one with Eco RI Bgl II fragment of plasmid pBTM 127 inserted in the Eco RI Bam HI site of plasmid pUB 110 See Figure 8 .Besides, this Eco RI Bgl II fragment consists of a part of B fragment and whole C fragment of Bgl II fragment of pBTM 127. As the result of the above it was elucidated that neutral protease gene exists in C fragment 1.9 kb . pBTM 515 100 Xug was reacted with restriction enzymeBgl II 200 units and Hind III 200 units at 370C for 1 hour to be cleaved. The solution was electrophoresed with 18 agarose gel. As the result of collecting 1.9 kb DNA from this gel in accordance with electrophoresis elution method Y.Takagi, Gene Manipulation Manual, Kodansha, Japan approx. 10 ug of C fragment was obtained. Figure 9 shows the restriction enzyme cleavage map of this C fragment. B. subtilis 1A 274 Bacillus subtilis 1A 274 pBTM 515 containing plasmid pBTM 515 has been deposited with theInstitute for Fermentation, Osaka under the designation of IFO 14377 and also with Fermentation Research Institute, Agency of Industrial Science and Technology, Ministry of InternationalTrade and Industry, Japan FRI under the designation FERMBP 622 in accordance with the Budapest Treaty.Reference Example 9 Determination of Base Sequence A part of the base sequence of C fragment obtained inReference Example 8 was determined by using dideoxynucleotide synthetic chain termination method and Maxam Gilbert method.The base sequence of C fragment from the side of Bgl II site and the translated amino acid sequence are shown by Figure 10. In this sequence 35 region TTGCAG and 10 region TATTAT 55 recognized by E655, 35 region AGTTT and 10 region GAGATTGCT recognized by E 37, and 35 region AATTC and 10 region TAGTTTTATA predicted to be recognized by E 32 were found as promoters. As it was confirmed by using promoter cloning vector pFTB 281 Yoshimura et al., Summary of Speach of 1983 Japan Agricultural Chemistry Convention, 28 J. Bacteriol, 159, 905 1984 that Bgl II Sau 3A1 fragment 530 bp seeFigure 9 has promoter activity. Also, there are SD region and open reading frame downstream from the promoter, it is assumed that those between the first methionine and 27th or 28th alanine are the signal peptide of neutral protease.Amino acid sequence after 222nd alanine slightly differed from the neutral protease reported in literature P. L. Levy et. al., Proc.Natl. Acad. Sci., 72, 4341 1975 , but it was completely identical with the amino acid sequence of N end of neutral protease decribed in Example 3. As the result of this, it was predicted that approx. 200 peptides with unidentified function would exist between a sequence considered to be a signal peptide and mature protein.Reference Example 10 Penicillinase Secretion Plasmid Plasmid pTR 10 See Figure 11 prepared by insertingBgl II Sau 3A1 fragment approx. 530 bp containing a promoter region and a region coding for both signal peptide and upstream of propeptide of neutral protease gene on C fragment of plasmid pBTM 515 in the Bam HI site of pFTB 281 was cleaved with Eco RI to isolate approx. 530 bp of Eco RI fragment. This fragment was cleaved with restriction enzyme Hae III to isolate EcoRI Hae III fragment approx. 340 bp containing a region coding for both promoter and C terminal end of signal peptide. Xho I linker CCTCGAGG was linked to this fragment by using T4 DNA ligase. Sac I Pst I fragment 8.5 kb was isolated from plasmid pEN 1 obtained from Prof. T. Oshima of OsakaUniversity, Japan See Figure 10 in which penicillinase gene of Bacillus licheniformis was inserted in plasmid pMB 9 F.Bolivar et. al., Gene 2, 75 1977 . A region coding for the last half of signal peptide and mature protein of penicillinase is contained in the fragment and Pst I site is located at around the middle of the region coding for signal peptide.Then, the above mentioned Sac I Pst I fragments were treated with 5 units nuclease BAL 31 Betheda Research Laboratories at room temperature for 15 seconds and linked with added Xho I linker CCTCGAGG New England Biolabs by T4 DNA ligase to transform Escherichia coli JA 221 obtained from Prof. T.Oshima of Osaka University, Japan . By measuring the size ofBgl II Xho I fragment of plasmid of recombinant, which showed tetracycline resistance, with polyacryl amide gel electrophoresis, some clones whose majority of region coding for signal peptide was considered to be deleted were obtained.The base sequences of these clones were determined in accordance with dideoxynucleotide synthetic chain termination method. Regarding plasmid pENX 374 8.3 kb which is one of the clones it has been made clear that a nucleotide coding for Ser Arg Ala is joined to the 5 end of DNA coding mature protein of penicillinase See Figures 12, 13 and 14 . Then, pUB 110 was cleaved with restriction enzyme SphI and Eco RI to isolate large Sph I Eco RI fragment 3.5 kb with agarose gel electrophoresis and the electroelution method See Figure 11 . Eco RI Xho I fragment 340 bp, 0.5 Xug containing a region coding for promoter and signal peptide of neutral protease gene obtained as described above, pENX 374 derived XhoI Sph I fragment 1.9 kb, 0.5 pg and pUB 110 derived Sph I EcoRI fragment 0.5 pg were ligated by reaction with T4 DNA ligase to transform B. subtilis 1A 274. One of the plasmids containing all fragments among kanamycin resistant recombinants obtained was named pTEX 201 5.7 kb see Figure 11 . B. subtilis 1A 274 containing plasmid pTEX 201 Bacillus subtilis 1A 274 pTEX 201 has been deposited with the Institute forFermentation, Osaka under the designation of IFO 14375 and also with Fermentation Research Institute, Agency of IndustrialScience and Technology, Ministry of International Trade andIndustry, Japan FRI under the designation of FERM BP 619 in accordance with the Budapest Treaty.Example 1 Synthesis of 5 CATCGTTACACCAATAT L 13 Chain Heptadecanucleotides Each oligo nucleotide block was prepared by solid phase synthetic method, in which a resin which the supported nucleoside at the 3 terminal end was used. Thymidine resin, for instance, was prepared by joining 5 dimethoxytrityl thymidine 3 O succinate to aminomethyl polystyrene resin 1 divinyl benzene, amino radicals 0.21 meq g, 100 250 mesh in accordance with a well known method for example, Miyoshi et.al., Nucleic Acids Res., 8, 5507 1980 . As for a dinucleotide block, one which was prepared according to a well known method Figure 15, C.B. Reese, L. Xard Nucleic AcidsRes., 9, 4611 1981 or one on the market for instance, 16 kinds of the Protected Dimer Block Wako Pure Chemical, Japan was used. To lengthen the chain of oligo nucleotide corresponding to L13 thymidine supported resin 30mg, 5 pmol was used quoted as an example and the lengthening was completed by repeating the manipulation described in Table 4 See Figure 16 . TABLE 4EMI31.1 tb Manipu SEP Reagent, SEP Solvent SEP Quantity SEP Reaction SEP Recipro tb lation SEP ml SEP Time min SEP cation tb SEP No. SEP times tb SEP um SEP 1 SEP Dichloromethane Methanol SEP 2 SEP 0.1 SEP 3 tb SEP um SEP 2 SEP 2 SEP Benzenesulfonic SEP Acid SEP 2 SEP 1.0 SEP 1 tb SEP DCM M, SEP 7 3 tb SEP um SEP 3 SEP Dichloromethane Methanol SEP 2 SEP 0.1 SEP 1 tb SEP 7 SEP SEP 3 tb SEP um SEP 4 SEP 2 SEP Benzenesulfonic SEP Acid SEP 2 SEP 1.0 SEP 1 tb SEP DCM M, SEP 7 3 tb SEP SEP SEP 5 SEP Dichloromethane Methanol SEP 2 SEP 0.1 SEP 2 tb SEP 7 SEP SEP 3 tb EMI32.1 tb SEP 6 SEP Pyridine SEP 2 SEP 0.1 SEP 3 tb SEP 7 SEP Pyridine SEP 0.3 SEP Azeotropic SEP 1 tb SEP Dehydration tb SEP SEP 8 SEP Dimer SEP SEP Block Pyridine SEP 30mg SEP Azeotropic SEP 1 tb SEP 0.3ml SEP Dehydration tb SEP SEP 9 SEP Mesitylensufonyl nitro SEP 30mg SEP 20 SEP 1 tb SEP triazole Pyridine SEP 0.3ml SEP 40 C SEP tb SEP SEP 10 SEP Pyridine SEP 2 SEP 0.1 SEP 2 tb SEP SEP 11 SEP 0.1 SEP M SEP 4 Dimethylamino SEP 1.8 SEP ml tb SEP Pyridine Acetic SEP 0.2 SEP ml SEP 3 SEP 1 tb SEP anhydride tb um SEP 12 SEP Pyridine SEP 2 SEP 0.1 SEP 3 tb DCM M Dichloromethane methanol In the case of the fragment L13, the block was condensed in the order of TA, AA, CC, CA, TA, GT, TC, and CA.After condensation the resin was washed twice with dioxane 2 ml and 1 M N1, N1, N3, N3 tetramethyl guanidium syn pyridine 2 aldoxime 0.5 ml , dioxane 0.4 ml and water 0.1 nml were added to the resin and shaken at 300C for 16 hours. The resin was separated from the solution and washed with 50 pyridine water 2 ml four times. Then the separated solution and the washing solution were combined to be lyophilized. Pyridine 0.5 ml and then concentrated aqueous ammonia 10 ml were added to the residue which was then heated at 55 C 60 C for 5 hours in a sealed tube and lyophilized.The residual was separated and purified with C18 reversed phase chromatography carrier C18 silica gels Waters , column Econo Column 0 0.7 X 15 cm, Bio Rad to obtain the fraction which was eluted last. This fraction was concentrated and dissolved in 80 acetic acid 1 ml and left as is for 20 minutes. The acetic acid was completely removed to make the residue an aqueous solution.The solution was extracted with ethyl acetate 2 ml three times to remove trithanol, and purified with C18 reversed phase high performance liquid chromatography carrier Nucleosil 5C18 column 4 0 x 300mm,Umetani Seiki, Japan water containing 0.1 M triethyl ammoniumacetate acetonitrile system and Ion exchange high performance liquid chromatography carrier TSK GELDEAE 2SW column 04.6 X 250 mm, ammonium formate acetonitrile system to obtain 5.2OD26o of pure quality. The base sequence of this product was analyzed in accordance with the two dimension finger print method R. Frank, H. Blocker, Chemical andEnzymatic Synthesis of Gene Fragments Page 225, verlag Chemie, 1982 .Example 2 Formation of Hybrid of Oligo nucleotide Blocks and its Enzymatic Joining A series of manipulations concerning the construction of the double stranded human lysozyme gene are shown at Figure 17. 10 mM ATP 1 pl was added to each 0.1 OD260 of oligo nucleotide block except U1 and L26 chains that located in 5 end and the total quantity of solution was prepared to be 9.5 jil and to include 50 mM Tris hydrochloric acid pH 8.0 , 10 mM magnesium chloride, 10 mM mercaptoethanol and 1 mM spermine. Then, T4 polynucleotide kinase EC2. 7. 1. 78 6U 1, 0.5 1 was added to incubate at 370C for 90 minutes. The reaction solution was heated at 900C for 5 minutes and inactivated.Solution of the 5 phosphorylated oligomers 1 ul each were divided into five groups I to V as shown by Figure 17, the components of each group were mixed with each other and prepared so that the total quantity of solution would be 236 containing 66 mM Tris hydrochloric acid pH 7.6 , 6.6 mM magnesium chloride and 500 uM ATP. This was heated at 900C for 3 minutes and then cooled quickly with ice. Once again the solution was soaked in hot water of 750C and allowed to anneal slowly until it came to room temperature.Further, it was left as is at 15 C for 10 minutes and 0.2 M mercaptoethanol 13 1l and T4 DNA ligase EC 6.5.1.1 2.5U ,ul, l,nl were added to be incubated at 15 C for 20 hours. The reaction solution was heated at 650C for 5 minutes to inactivate the enzyme. As the result of applying a part 2.5 pal of the solution to electrophoresis of 10 polyacryl amidegel, it was made clear that group I and II approx. 78 bp were migrated to an area between 64 bp fragment, and 80 bp fragment of the pBR 322.Hae III digests used as a size marker group III approx. 82 bp an area between 80 bp fragment and 89 bp fragment and group 1V approx. 93 bp an area between 89 bp fragment and 103 fragment. However, so many bands were recognized about group IV approx. 74 bp as a result of side reaction that this group was further divided into two groups U16, U17, U18, L16, L17 and U19, U20, L18, L19, L20 to be joind separately by reaction with T4 DNA ligase. After ligation, they were mixed and ligated again to obtain group IV . The reaction solution was washed with saturated aqueous phenol chloroform 1 1 V V , 3 M sodium acetate pH 5.6 and then ethanol were added to precipitate the DNA after being kept at 80C for 5 minutes. DNA of each group was purified on 10 polyacrylamide gel under the degeneration condition and electrically eluted with 45 mMTris boric acid buffer solution pH 8.4 to obtain DNA 10 15 rg of each group.After the DNA was purified by precipitation with ethanol again, each group 1.5 2 jig was joined to each other with T4 DNA ligase to obtain perfect human lysozyme genes approx. 1 jig . This product correspondeds to approx. 404 bp on 5 polyacrylamide gel. Before sub cloning, both 5 emus of the DNA were enzymatically phosphorylated according to a conventional method.Example 3 Sub cloning of Human Lysozyme Gene After 1 ug of plasmid pGLD 906 constructed according to Reference Example 2 was reacted with 5 units restriction enzyme Xho I in 10 vul of reaction solution 100 mM NaCl, 50 mMTris HCl pH 7.5 , 10 mM MgC12, lmN DTT at 370C for 1 hour, the reaction solution was deprotenized and precipitated with cold ethanol. The DNA 10 ng and 50 ng of human lysozyme gene prepared in Example 1 were mixed and reacted each other in 10 of or reaction solution 66 mM Tris HCl pH 7.6 , 10 mM ATP, 10 mM spermidine, 100 mM MgC12, 150 mM DTT, 2 ng ml BSA, 5 unitsT4 DNA ligase Takara Shuzo, Japan at 140C for one night to join DNA.Using this reaction solution Escherichia coli DH1 was transformed according to the method of Cohen et. al., previously mentioned. Plasmid was isolated from transformants selected by ampicillin resistance as a marker in accordance with the alkaline extraction method previously quoted and molecular weight and cleavage pattern with restriction enzyme were examined to obtain plasmid pGLD 906 6, human lysozyme gene inserted at the Xho I site of pGLD 906. In this plasmid human lysozyme gene was inserted in opposite direction to GLD promoter.Example 4 Construction of Expression Plasmid of Human Lysozyme Gene After plasmid pGLD 906 6 10 pg stated in ReferenceExample 3 was reacted with restriction enzyme Xho I 50 units in 50 jil of reaction solution reaction solution for XhoI, previously quoted at 370C for 1 hour and then, applied to 1.5 agarose slab gel electrophoresis 150V, 1 hour . After the electrophoresis, gel containing the 409b DNA fragment was put into a dialysis tube and sunk in a buffer solution for electrophoresis. The DNA fragment was electrically eluted from gel McDonell, M. W. et. al., previously mentioned .The solution in the dialysis tube was extracted with phenol and then with ether, NaCl was added up to 0.2 M and then double quantity of cold ethanol to be precipitated at 20 C. Then, after plasmid pPHO17 1 pg constructed according to Reference Example 1 was reacted with the restriction enzyme Xho I 5 units in 10 ul of reaction solution reaction solution for Xho I previously mentioned at 370C for 1 hour, the solution was deproteinized with phenol andDNA 10 pg was precipitated with cold ethanol. 50 ng of theDNA fragment containing human lysozyme gene was added to this precipitated the DNA 10 ng to combine DNA according to the manner previously mentioned. Escherichia coli DHl was transformed according to the manner stated in Example 3 and out of the transformants was obtained a plasmid pPHOlys 101 with the human lysozyme gene inserted in the Xho I site of pPHO 17 in the correct direction to PHO5 promoter. In an exact same manner was obtained a plasmid pGLD 906 5 with the human lysozyme gene inserted in the Xho I site of pGLD 906 in the correct direction to GLD promoter.Example 5 Preparation of Yeast Transformants Saccharomyces cerevisiae AH22R was transformed with expression plasmid pPHOlys 101 and pGLD 906 5 obtained according to Example 4 in the manner by Hinnen et. al., previously mentioned to obtain transformants, S. cerevisiaeAH22R pPHOlys 101 and S. cerevisiae AH22R pGLD 906 5 respectively. Saccharomyces cerevisiae AH22R pPHOlys 101 has been deposited with the Institute for Fermentation, Osaka under the designation of IFO 10139 and with FRI under the designation of FERM BP 913 FERM P 7939 under the Budapest Treaty. AlsoSaccharomyces cerevisiae AH22R pGLD 906 5 has been deposited with the Institute for Fermentation, Osaka under the designation of IFO 10140 and with FRI under the designation ofFERM BP 914 FERM P 7940 under the Budapest Treaty. Saccharomyces cerevisiae AH22R has been deposited with theInstitute for Fermentation, Osaka under the designation ofIFO 10134 and with FRI under the designation of FERMBP 804 FERM P 7824 under the Budapest Treaty.Example 6 20 ml of Burkholder lAmer. J. Bot. 30, 206, 1943 low concentration of phosphate culture medium was partially poured into a 200 ml flask and inoculated with S. cerevisiaeAH22R pPHOlys 101 and shake cultured at 300C for 4 days.After this culture solution was centrifuged, the cells were rinsed with a physiological solution of sodium chloride. The cells were made spheroplast with Zymolyase Seikagaku Kogyo,Japan according to the method by Miyanohara et. al., Proc. Natl. Acad. Sci. USA 80, 1 1983 0.1 Triton X 100 was added to the spheroplast to extract lysozyme. The lysed solution was centrifuged at 5,000 rpm at room temperature for 10 minutes to obtain a supernatant 2 ml . After the supernatant was acidified according to the Meyer s method J. Biol. Chem. 113, 303 1936 , cold ethanol was added and the supernatent was separated again after a heat treatment, and ethanol was further added and the precipitate was collected by centrifugation. The precipitate was dissolved by adding 0.4 ml of 50 mM potassium phosphate buffer solution pH 7.4 , 0.1 ml of the solution was subjected to a quantitative analysis of activity.Examining lysis activity according to Worthington Enzyme Manual theWorthington Biochemical Corporation 1972 p 209 , decrease of turbidity due to the addition of the sample was confirmed and expression of lysozyme genes was also confirmed.Example 7 Improvement of Expression Plasmid pPHOlys 101 Based on information that a base in a location of 3 or 4, of all of yeast genes abundantly expressed, isA Adenine when A Adenine of translation start codon ATG is assumed to be 1 for instance, refer to J. Biol. Chem. 254, 9839 1979 , Nucleic Acids. Res., 10, 7791 1982 . etc. , the base sequence at the upperstream of translation start codon ATG of human lysozyme synthetic gene was reformed.First, 1 jig of pPHOlys 101 was treated with 1 unit of restriction enzyme Xho I in 10 jil of reaction solution 100 mM NaCl. 50 mM Tris HCl pH 7.5 , 10 mM MgOl2, 1 mM dithiothreitol at 37OC for 1 hour and then the solution was deproteinized with phenol to precipitateDNA with cold ethanol. After 1 jig of sample in which two types of linear DNA exist mingled was reacted with 25 units of nuclease S1 in 25 jil of reaction solution 200 mM NaCl, 50 mM sodium acetate pH 4.5 , 1 mM ZnSO4, 0.5 glycerol at 2500 for 5 seconds, 1 1 of 0.25 M EDTA was added to the solution and the reaction was stopped.Then the solution was treated with phenol in the conventional method to precipitate DNA with cold ethanol. While 5 TCGAGAAAAC 3 3 CTTTTG 5 was synthesized according to the method by Crea et. al., Proc.Natl. Acad. Sci. USA 75, 5765 1978 , 1 rg of the above adaptor was reacted with 20 units T4 polynucleotide kinase Takara Shuzo, Japan in 100 pl of reaction solution 50 mM Tris HCl pH 7.6 , 10 mM Mac12, 10 mM 2 mercaptoethanol, 100 uMATP at 370C for 1 hour to phosphorylate the 5 end. 400 ng of phosphorylated adaptor was mixed with 1 pg of DNA fragment treated with nuclease S1 previously mentioned and they were combined with T4 DNA ligase under the condition stated inReference Example 1 to be digested with Xho I.Using a portion of this reaction solution Escherichia coli DHl was transformed according to the method by Cohen et. al., Plasmid DNA was isolated out of transformants selected by ampicillin resistance as a marker according to the alkaline extraction method previsously mentioned to select plasmid pPHOlys 111 correct direction and pPHOlys 112 opposite direction respectively.The result of analysis of the base sequence of the insert in accordance with dideoxynucleotide method above mentioned was as follows in comparison with its original sequence.Original Sequence pPHOlys 101 TCGAGATG AAG TAA TAGCTCGA Assumed Sequence TCGAGAAAACGATG AAG TAA TAGCGTTTTCTCGAActual Sequence pPHOlys 111 TCGAGAAAAC ATG AAG TAA T GTTTTCTCGA indicates defect sequenceExample 8 Construction of Expression Plasmid pPHOlys 115 Since pPHOlys 111 lacked a stop codon, pPHOlys 115 was constructed as shown by Figure 18, which has the stop codon restored from pPHOlys 101 and pPHOlys 111. In other words, small fragment obtained by treating pPHOlys 111 with Xba I and large fragment obtained by treating pPHOlys 101 with Xba I were joined with T4 DNA ligase andEscherichia coli DHl was transformed with the reaction solution to obtain transformants containing aimed pPHOlys 115.Example 9 Transformation of Yeast with H1.an Lysozyme Expression Plasmid Saccharomyces cerevisiae AH22R was transformed according to the modified method of that of Ito et. al., Rodriquez, R. L Tait, R. C Recombinant DNA Techniques AnIntroduction, Addison Westey Publishing Company P186. 1983 to isolate yeast transformant AH22R pPHOlys 111 and AH22R ppHolys 115 containing the said plasmid.Example 10 Expression of Human Lysozyme Genes AH22R pPHOlys 111 and AH22R PHOlys 115 were inoculated on the culture medium stated in Example 5 and cultivated under shaking at 37 C for 3 days. After cells were collected from 1 ml of the culture solution and rinsed according to the method stated in Example 5, the cells were frozen in dry ice ethanol and stored at 800C. After the cells were fused, 200 ul of Laemmlis buffer 100 mM Tris HCl pH 6.8 , 2 SDS, 10 glycerol, 5 2 mercaptoethanol, 0.17 BPB was added and boiled for 10 minutes. 30 vul of cells dissolved solution thus obtained was applied to SDS PAGE to be electrophorased with 50 V for 3 hours. After the electrophoresis the protein produced was transferred to nitrocellulose filter of BIO RAD according to Trans Bolt Cell Operating Instructions. Namely7 the protein was transferred in transfer buffer 25 mM Tris HCl pH 8. 3 . 192 mM glycine, 20 V V methanol with 60V for 3 hours using the Trans Blot device of BIO RAD. After being soaked in TBS 20 mM Tris, 500 mM NaCl, pH 7.5 for 10 minutes, nitrocellulose filter thus obtained was put in a blocking solution gelatin is dissolved in TBS so much up to 3 and slowly shaken for 1.5 hour. Then, after being slowly shaken in fifty fold diluted solution of anti human lysozyme antibody prepared according to Example 12 to be mentioned later, the above nitrocellulose filter was rinsed quickly with deionized water and then rinsed slowly by shaking with TBS twice for 10 minutes each time.Nitrocellulose filter thus obtained was reacted in the second antibody solution Goat Anti Rabbit IgG BIO RAD marked with horseradish peroxidase HRP by shaking softly for 1 hour and then rinsed with deionized water to be followed by slow rinsing with TBS twice for 10 minutes each time. Then nitrocellulose filter thus obtained was soaked in the color development solution compound of 60 mg of HRP color development reagent BIO RAD dissolved in 20 ml of cooled methanol and 60 1 of cooled 30 H202 disolved in 100 ml ofTBS for 15 minutes. The nitrocellulose filter was soaked in distilled water for 10 minutes to suspend color development and take photograph. The result was as shown by Figure 19 and a distinct band of reacting anti human lysozyme antibody was confirmed at the position of 14.4 kd. It was made clear by comparison with controls human milk lysozyme Sigma that AH22R pPHOlys 115 produced 700 rg l of Human lysozyme by 3 days cultivation. Saccharomyces cerevisiae AH22R pPHOlys 115 has been deposited with the Institute for Fermentation, Osaka under the designation of IFO 10152 and with FRI under the designation ofFERM BP 912 FERM P 8318 under the Budapest Treaty..Example 11 Preparation of Antibody against Human Lysozyme Human lysozyme solution 500 Xug of human lysozyme Sigma , 500 1 of 0.85 NaCl, 500 rl of complete Freund s adjuvant was injected under the skin at five places of each of three New Zealand white rabbits weighing 2.5 kg the first immunization . Two weeks after the first immunization human lysozyme solution 500 jig of human lysozyme Sigma , 500 pl of 0.85 NaCl, 500 pl of complete Freund s adjuvant was injected under the skin, in the same manner as the first immunization the second immunization . Another two weeks later the third immunization was conducted in the same manner as the second immunization.One week after the third immunization 50 ml of blood per rabbit was taken from the ear of each rabbit. The blood was left as is to obtain supernatant which was centrifuged at 3000 rpm for 10 minutes and the supernatant was used as anti serum.Example 12 Purification of Anti Human Lysozyme Antibody NH4 2SO4 was added to the antiserum previously mentioned up to 40 saturation and the precipitate was collected by centrifugation at 10 K rpm for 10 minutes. This precipitate was dissolved in 5 ml of 20 mM borate buffer pH 8.0 and dialysed in the buffer at SOC for one night. 10 mg of human lysozyme previously mentioned was immobilized against 1g of CNBr activated Sepharose 4B Pharmacia according to Manual Affinity Chromatography, principles methods, Pharmacia Fine Chemicals pl5 . Then anti human lysozyme antibody was purified using a column filled with insoluble human lysozyme. Namely, fraction of the above precipitate with ammonium sulfate was put on the column equilibrated with borate buffer so that antibody would adhere. After rinsing with the above buffer until A280 became 0, the column was rinsed with 0.1 M acetate buffer pH 4.0 containing 0.5 MNaCl until A280 became 0. Then, anti human lysozyme antibody was eluted with 0.2 M glycine HCl buffer pH 2.0 and neutralized with 1 M NaOH after 0.2 M Na2HPO4 pf 1 50 quantity of the eluted solution was added.When IgG was calculated based on an general concept that A280 of 1 mg ml of IgG is 1.5, 1 2.77 mg, 2 2.43 mg and 3 2.83 mg of IgG were obtained from three rabbits 1 , 2 and 3 being identification numbers of rabbits . Merthiolate natrium Nakarai Chemicals was added to the anti human lysozyme antibody solution to be 0.01 and stored at 40C. Above was all manipulated at 5 C. Example 13 Preparation of Fab 2 After IgG solution of rabbit 1 purified according toExample 12 was concentrated to approx. 0.5 ml with collodion bag, it was dialysed against 0.1 M acetate buffer pH 4.5 for 1 hour. Pepsin Sigma was added to this IgG solution, 20 Zg per 1 mg of IgG to undergo reaction at 370C for 1 night. After removing the precipitate by centrifugation, the solution was adjusted to pH 7.5 8.0 with 1N NaOH and applied to a SephacrylS 200 Pharmacia column 0 1.6 cm X 100 cm equilibrated with 0.1 M borate buffer pH 8.0 , Fab 2 fractionation was obtained using molecular weight 100 kd of Fab 2 as a marker.Example 14 Preparation of Conjugation of Fab and Peroxidase by Maleimide i Preparation of Fab Fab 2 solution prepared according to Example 13 was concentrated with collodion bag to be approx. 1 ml and then dialyzed against 0.1 m acetate buffer pH 5.0 . 0.4 M 2 mercaptoethanol was added to be 20 mM and replaced with N2 to be left at 37 C for 90 minutes. This solution was applied toSephadex G 25 fine cplumn 1 cmX70 cm equilibrated with 0.1 M phosphate buffer pH 6.0 containing 4 mM EDTA to obtainFab . ii Preparation of Combination of Maleimido PeroxidaseMaleimide HRP was prepared according to the method by Yoshitake et. al., S. Yoshitake, et. al., J. Biochem., 92, 1413 1982 using horseradish peroxide HRP Boehringer Mannheim . This combination was obtained in dissolved form, in 0.1 M phosphate buffer pH 6.0 . iii Combination of Maleimido HRP and Fab Fab obtained at i and maleimide HRP obtained at ii were mixed and reacted each other at 40C for 1 night according to the above mentioned method by Yoshitake et. al.After the reaction the mixture was applied to Ultragel Act44 0 1 cmX 70 cm equilibrated with 0.1 M phosphate buffer pH 6.5 according to the above mentioned method to fractionate. Mol ratio of Fab to HRP from the value of A280 and A403 of obtained fraction was calculated and usable combination ofFab HRP was collected.Reference Example 11 Construction of Cloning Vector pBR 322 X After Syg of Escherichia coli vector pBR 322 was reacted with 1.5 units restriction enzyme Bal I in 40 Xul of reaction buffer 10 mM Tris HC1 pH7.5 . 10 mM MgC12. 1 mM Dithiothreitol at 37 C for 5 hours, the reaction mixture was extracted with phenol in a conventional method and DNA was precipitated with ethanol. 50 ng of phosphorylated Xho I linker d pCCTCGAGG New England Biolabs was added to this DN and they were combined with T4 DNA ligase according to a conventional method. With this reaction solution Escherichia coli DH I was transformed and plasmids were extracted from th obtained ampicillin resistant and tetracycline resistant colonies according to the alkaline extraction method previously mentioned to obtain plasmid pBR 322 X with Xho I site in place of Bal I site.Example 15 Extraction of Human Lysozyme 1 1 of culture of S. cerevisiae AH 22 R pPHO lys 115 cultivated according to the method mentioned at Example 10 was centrifuged at 4000 rpm and 40C for 15 minutes to collect cells. 12.7 g of wet cells thus obtained was frozen according to the method mentioned at Example 10 and stored at 8000. After these cells were dissolved by the addition of 30 ml of buffer 10 mM acetate buffer pH4.0 , 10 mM EDTA. 1 mM PMSF. 1 mM DTT , the cells were broken with French Press OtakeSeisakusho, Japan at 40C under pressure 1,500 kgf cm2 . Thus obtained solution containing broken cells was centrifuged at 10,000 rpm at 40C for 10 minutes to collect precipitate. The precipitate was rinsed twice with distilled cold water and then put into cooled acetone with stirring to obtain a precipitate.The precipitate was washed with acetone again to obtain acetone dried disrupted cells. The acetone dried cells were ground down in a mortar and mixed with 50 ml of 7 M urea pH 5.0 with stirring. This solution was centrifuged at 10,000 rpm and 40C for 10 minutes to obtain precipitate. 50 ml of 2M urea pH 7.0 was added to the precipitate and well stirred to obtain pricipitate as well. 100 ml of 7 M urea pH 9.0 was added to this precipitate and kept stirring at 4 C for one night to extract human lysozyme. As the result of measuring the amount of human lysozyme according to the manner mentioned in Example 10, more than 92 human lysozyme was extracted.Example 16 Preparation of Fragment of Human Lysozyme Gene comprising Taq I Site around 5 Terminal End In Table 2, CGAGAGATGCGAAT in place of U2 andTCTGGCTAATTCGCATCTCT in place of L2 were synthesized as U2 taq and L2 taq respectively according to the method stated inExample 1. Then using each fragment U2 taq, U3 U26, L2 taq,L3 L26, hybrid of oligo nucleotide was formed according to the method stated in Example 2. In this case,EMI48.1 was obtained as equivalent to group I of Figure 17 but other groups II V were completely identical with those of Figure 17. After each of these groups were linked according to the method stated inExample 2, both 5 ends were enzymatically phosphorylated.Example 17 Sub cloning of Fragment of Human Lysozyme Gene containing Taq I Site 2.6 jig of plasmid pBR 322 X constructed at ReferenceExample 11 was reacted with 6 units restriction enzyme Xho I and 6 units of restriction enzyme Ola I in 35 pl of reaction solution t33 mM Tris acetate pH 7.9, 66 mM potassium acetate, 10 mM magnesium acetate, 0.5 mM dithiothreitol 0.01 BSA at 370C for 1 hour and the solution was deproteinized to precipitate with cold ethanol.This DNA 200 ng was mixed with 100 ng of human lysozyme gene fragment prepared in Example 16 and allowed to react in 10 pl of reaction solution 66 mMTris HCl pH 7.6 , 10 mM ATP, 10 mM spermidine, 100 mM MgOl2, 150 mM DTT, 2 mg ml BSA, 5 units T4 DNA ligase at 140C for 1 night to be combined each other. Using this reaction solutionEscherichia coli DHl was transformed according to the method byCohen et. al. Plasmids were isolated from transformants thus obtained according to the alkaline extraction method previously mentioned . Their molecular weight and cleavage pattern with restriction enzyme were examined and pLYS 221 with human lysozyme gene fragment inserted in was obtained.As the result of isolating EcoR I Xho I fragment of pLYS 221 and determining its base sequence in accordance with dioxy nucleotide chain termination method, Taq I Xho I fragment of human lysozyme gene was obtained as shown by Figure 20 exactly assumed. This sequence codes from the fourth Glu to the 130 thVal of amino acid sequence of human lysozyme described in table 1.Example 18 Construction of secretion plasmid of Human Lysozydme Using a human lysozyme gene fragment obtained according to Example 17, a Bacillus subtilis secretion plasmid was constructed according to the manner shown in Figure 21. First, plasmid pTEX 201 See Figure 11 was prepared from Bacillus subtilis MI 114 pTEX 201 according to the alkaline extraction method previously mentioned. pTEX 201 has a Stu I site just before the Xho I site as shown in Figure 21.Then, as the first step, 20 units of restriction enzyme Stu I was added to 50 jig of pTEX 201 to react in 50 1 of reaction solution 33 mM Tris acetate pH 7.9, 66 mM potassium acetate, 10 mM magnessium acetate, 0.5 mM dithiothreitol, 0.018 BSA at 370C for 1.5 hours to be followed by a treatment at 65 C for 10 minutes to inactivate enzyme. Then 20 units of restriction enzyme Xho I was added to this reaction solution for another reaction at 370C for 15 hours. The solution was treated with phenol and precipitated with ethanol in accordance with the conventional method to obtain Stu I Xho I large fragment of pTEX 201.While, 1 pg of human lysozyme Taq I Xho I gene fragment obtained according to Example 17 and 0.9 yg of adapter CCAAGGTTT GCTTCCAAAAGC chemically synthesized according to triester method were ligated with T4 DNA ligase in accordance with the conventional method and treated with restriction enzymes Xho I and Hae III to obtain the Hae III Xho I fragment as shown by Figure 21. After 0.3 jig of this HaeIII Xho I fragment and 1.0 1g of the Stu I Xho I fragment previously mentioned were jointed with T4 DNA ligase, andBacillus subtilis MI 114 was transformed in accordance with the previously mentioned protoplast method. Many transformants which lost capability of producing lactamase were obtained among those kanamycin resistant transformants thus obtained. Some colonies were selected from them to confirm that human lysozyme gene were inserted in them as assumed. This secretion plasmid was named pLYS 301.Example 19 Cultivation of Bacillus subtilis Harboring Secretion Plasmid B. subtilis MI 114 pLYS 301 obtained at Example 18 was inoculated into a test tube containing 5 ml of LBK culture medium Bacttryptone DIFCO 10g l, yeast extract DIFCO 5g 1,NaCl 5g 1, kanamycin 5 mug 13 and incubated with shaking at 280C for 1 night. 0.5 ml of this culture was transferred to 200 ml flask containing 40 ml BHI culture medium Brain heart infusion DIFCO 37g 1, Kanamycin 5 mug 13 and cultivated at 250 rpm and 37 C for 7.5 hours. 0.5 ml of the culture broth was centrifuged and 0.5 ml of cooled 20 TCA was added to the supernatant to be left in ice for 30 minutes.Then the fluid was centrifuged at 12,000 rpm for 5 minutes to collect a precipitate which was rinsed with 0.5 ml of cooled 5 TCA and 0.5 ml of cooled acetone once each. 30 1 of Laemmli s buffer was added to the precipitate in accordance with the description of Example 10 and the mixture was applied to SDS PAGE. After the electrophoresis, human lysozyme was stained as shown byFigure 22 in accordance with the description of Example 10. It is considered that band 1 shows a mature human lysozyme and band 2 shows human lysozyme whose signal of neutral protease is not yet cleaved. Yield was 10 ng ml as a mature human lysozyme.Bacillus subtilis MI 114 pLYS301 carrying plasmid pLYS301 has been deposited with the Institute for Fermentation, Osaka under the designation of IFO 14470 and with FRI under the designation of FERM BP 920 under the Budapest Treaty. Possibility For Industrial Use Human lysozyme has medicinal uses for anti inflammation, hemostatic, tissue regeneration, anti tumor etc. It is also used for eyewash as anti inflammation enzyme and as a food antiseptic. When used for medicinal purposes, it does not have the immune response side effect as does egg white lysozyme. By virtue of the preparation of the human lysozyme synthetic gene of the present invention human lysozyme, it is now possible that human lysozyme, a useful medical agent, can be supplied in large quantities.