# Production of factor VIII and related products.

## Abstract
Preparations of recombinant DNA which code for cellular production of human and porcine factor VIII C and methods of obtaining such DNA and expression thereof in bacteria and eucaryotic cells is disclosed. New compounds, including deoxyribonucleotides and ribonucleotides which are utilized in obtaining clones which express factor VIII C are also disclosed.

## Claims
CLAIMS 1. A DNA sequence coding for human factor VIII C substantially free of other human genes. 2. The DNA sequence of Claim 1, wherein said sequence comprises one or more DNA sequences selected from the DNA nucleotide sequence depicted in Figure 7 or one or moreDNA sequences which hybridize to the DNA nucleotide sequence depicted in Figure 7 and which on expression codes for a polypeptide which exhibits factor VIII C activity. 3. The DNA sequence of Claim 1, wherein said sequence comprises or codes for a polypeptide having the amino sequence shown in Figure 7. 4. A transformed host containing a gene for human factorVIII C, said host being selected from bacteria, yeasts, and mammalian cells. 5. Isolated DNA coding for human factor VIII C, comprising a polydeoxyribonucleotide having the sequence 5 OGC AGC TTT CAG AAG AAA ACA CGA CAC TAT TTT ATT GCT GCA GTG GAG AGG 3 6. The DNA of Claim 5, excised from human genomic DNA. 7. The DNA of Claim 5, linked directly or indirectly toDNA from a non human source. 8. An expression vector for human factor VIII C, comprising the DNA segment of Claim 5. 9. A screening agent for identifying deoxyribonucleotide sequences and ribonucleotide sequences which encode for at least a portion of the human gene factor VIII C, comprising a deoxyribonucleotide having at least a ten nucleotide portion of the sequence or its inverse complement 5 TTT CAG AAG AGA ACC CGA CAC TAT TTC ATT GCT GCG GTG GAG CAG CTC TGG GAT TAC GGC ATG AGC GAA TCC CCC CGG GCG CTA AGA AAC AGG 3 . 10. A method of isolating a gene fragment with encodes at least a portion of protein, comprising a forming a genomic DNA library of an organism that produces said protein, b forming at least one pool of oligonucleotide probes whose sequence was selected solely based on the amino acid sequence contained in said protein and comprising a plurality of oligonucleotides which contain differing DNA sequences, each of which corresponds in DNA sequence to said amino acid sequence, c contacting the genomic library with said pool under conditions favoring DNA DNA hybridization, d identifying the genomic DNA which hybridizes to the oligonucleotide in said pool, and e isolating a segment of said genomic DNA which contains the oligonucleotide hyridizing DNA. 11. The method according to Claim 10, wherein at least on oligonucleotide pool contains a plurality of oligonucleotides, one of which contains each possible DNA sequence which corresponds with a selected amino acid sequence in the protein. 12. The method of Claim 10, wherein the oligonucleotides have a length of from about 10 to 200 nucleotides and preferably 10 to 20 nucleotides. 13. The method of Claim 10, wherein the oligonucleotides have a chain length of from about 30 to 200 nucleotides. 14. A method in accordance with Claim 10, wherein at least two oligonucleotide probes are formed corresponding in DNA sequence to amino acid sequences in the protein, and each oligonucleotide probe is contacted with the genomic DNA under hybridizing conditions. 15. The method of Claim 14, wherein one oligonucleotide probe comprises a plurality of oligonucleotides having a length of 11 to 20 nucleotides, and another oligonucleotide probe comprises at least one oligonucleotide having a length of 40 to 200 nucleotides. 16. A method of isolating a gene which encodes for human factor VIII C, comprising screening a cRNA library with a probe which corresponds to at least a ten nucleotide sequence of the following sequence or its inverse complement 5 CGC AGC TTT CAA AAG AAA ACA CGA CAC TAT TTT ATT GCT GCA GTG GAG AGG 3 constructing a cDNA library from the mRNA which hybridizes with the probe, and forming a gene which encodes for human factor VIII C by ligating AHF cDNA segments from the cDNA library. 17. A method for producing AHF, comprising a preparing one or more oligonucleotide probes which hybridize with human DNA which encodes AHF, or which hybridize with the complement of such DNA b using at least one such probe to identify cells containing mRNA transcripts having nucleotide sequences corresponding to AHF c preparing cDNA from the step b mRNA d assembling the step c cDNA into a replicable cDNA sequence encoding at least the amino acid sequence of mature human AHF e cloning the step d sequence f transforming a parental cell which does not express AHF with the step e sequence g culturing the step f transformed cell and h recovering AHF from the culture. 18. The method of Claim 17 wherein the cell is a mammalian cell. 19. A recombinant DNA sequence which codes for a polypeptide which produces factor VIII C activity. 20. The DNA sequence of Claim 19, wherein the polypeptide comprises at least a major portion of the amino acid sequence of Figure 7. 21. A polypeptide expressed by DNA which exhibits human factor VIII C activity comprising amino acid sequences selected from the amino acid sequence depicted in Figure 7 or amino acid sequences expressed by DNA sequences which hybridize to the DNA nucleotide sequence depicted inFigure 7. 22. The polypeptide of Claim 21, wherein said polypeptide comprises amino acids corresponding to a segment of the amino acid sequence of Figure 7 sufficient to provide factor VIII C activity, substantially free of human fibrinogen and fibronectin. 23. The polypeptide of Claim 21, wherein said polypeptide is substantially free from other human polypeptides. 24. The polypeptide of Claim 23, wherein said other human polypeptides are human factor VIII vWF, fibrinogen and fibronectin. 25. A pharmaceutical preparation useful for therapeutic treatment of Hemophilia A comprising a sterile preparation of the polypeptide of any one of Claims 21 to 24.

## Description
PRODUCTION OF FACTOR VIII AND RELATED PRODUCTSPrior Application This application is a continuation n part application ofU.S. Application Serial No. 546,650 filed on October 28, 1983.Background of the Invention This invention relates to the preparation of recombinant deoxyribonucleic acid DNA which codes for cellular production of human factor VIII C, and of DNA which codes for porcine factorVIII C, to methods of obtaining DNA molecules which code for factor VIII C, and to expression of human and porcine factorVIII C utilizing such DNA, as well as to novel compounds, including deoxyribonucleotides and ribonucleotides utilized in obtaining such clones and in achieving expression of human factor VIII C. This invention also relates to human AHF and its production by recombinant DNA techniques. Factor VIII C is a blood plasma protein that is defective or absent in Hemophilia A disease. This disease is a hereditary bleeding disorder affecting approximately one in 20,000 naples. Factor VIII C has also been known or referred to as factor VIII, the antihemophilic factor AHF , antihemophilic globulin AHG , hemophilic factor A, platelet cofactor, thromboplastinogen, and thrombocytolysin. It is referred to as Factor VIII C , to indicate that it is the compound which affects clotting activity.As used herein, factor VIII C and AHF are synonynous. Although the isolation of AHF fran blood plasma has been described in the literature, the precise structure of AHF has not previously been identified, due in part to the unavailability of sufficient quantities of pure material, and the proteolytic nature of many contaminants and purification agents. While some quantities of impure AHF have been available as a concentrated preparation processed fran fresh frozen human plasma, the extremely low concentration of AHF in human plasma and the high oost of obtaining and processing human plasma make the cost of this material prohibitive for any extensive treatment of hemo√ßhllia. The present invention makes it possible to produce humanAHF using recombinant DNA techniques. AHF, like other proteins, is comprised of some twenty different amino acids arranged in a specific array. By using gene manipulation techniques, a method has been developed which enables production of AHF by identifying and cloning the gene which codes for the human AHE protein, cloning that gene, incorporating that gene into a recombinant DNA vector, transforming a suitable host with the vector which includes that gene, expressing the human AHF gene in such host, and recovering the human AHF produced thereby.Similarly, the present invention makes it possible to produce porcine AHF by recoobinant DNA techniques, as well as providing products and methods related to such porcine AHF production. Recently developed techniques have made it possible to employ microorganisms, capable of rapid and abundant growth, for the synthesis of commerciAlly useful proteins and peptides, regardless of their source in nature. These techniques make it possible to genetically endow a suitable microorganism with the ability to synthesize a protein or peptide normally made by another organism. The technique makes use of fundamental relationships which exist in all living organists between the genetic material, usually DNA, and the proteins synthesized by the organism.This relationship is such that production of the amino acid sequence of the protein is coded for by a series of three nucleotide sequences of the DNA. There are one or more trinucleotide sequence groups called codons which specifically code for the production of each of the twenty amino acids most wtlnonly occurring in proteins. The specific relationship between each given trinucleotide sequence and the corresponding amino acid for which it codes constitutes the genetic code. As a consence, the amino acid sequence of every protein or peptide is reflected by a corresponding nucleotide sequence, according to a well understood relationship. Furthermore, this sequence of nucleotides can, in principle, be translated by any living organism. For a discussion of the genetic code, see J. D.Watson,Molecular Biology of the Gene, W. A. Benjamin, Inc., 1977 , the disclosure of which is incorporated herein by reference, particularly at 347 77 C. F. Norton, Microbiology Addison,Wesley 1981 , and U. S. Patent No. 4,363,877, the disclosure of which is incorporated herein by reference. The twenty amino acids fran which proteins are made, are phenylalanine hereinafter sometimes referred to as Phe or F , leucine Leu , L , isoleucine Ile , I , methionine Met , M , valine Val , V , serine Ser , S , proline Pro , P , threonine Thr , T , alanine Ala , A , tyrosine Tyr , Y , histidine His , H , glutamine Gln , Q , asparagine Asp , N , glutamic acid Glu , E , cysteine Cys , C , tryptophane Trp , W , arginine Arg , R and glycine Gly , G .The amino acids coded for by the various combinations of trinucleotides which may be contained in a given oodon may be seen in Table 1 TABLE 1 The Genetic Code First Second Position Third Position T C A G Position T Phe Ser Tyr Cys T Phe Ser t Tyr Cys C Leu Ser Stop Stop A Leu Ser Stop Trp G C Leu Pro His Arg T Leu Pro His Arg C Leu Pro Gln Arg A Leu Pro Gln Arg G A Ile Thr Asp Ser T Ile Thr Asp Ser r C Ile Thr Lys Arg A Met Thr Lys Arg G G Val Ala Asp Gly T Val Ala Asp Gly C Val Ala Glu Gly A Val Ala Glu Gly G The Stop or termination codon terminates the expression of the protein. Knowing the deoxyribonucleotide sequence of the gene orDNA sequence which codes for a particular protein allows the exact description of that protein a amino acid sequence. However, the converse is not true while methionine is coded for by only one codon, the other amino acids can be coded for by up to six codons e.g. serine , as is apparent from Table 1. Thus there is considerable ambiguity in predicting the nucleotide sequence from the amino acid sequence. In sum, prior to the present invention, very little was known about the structure of AHF, and, despite substantial work over many years, those skilled in this art were unable to detenine the structure of AHF, or of its gene, or provide any procedure by which AHF could be produced in substantially pure form in substantial quantities. The e method described herein by which the gene for humanAHF is cloned and expressed includes the following steps 1 Purification of porcine AHF 2 Determination of the amino acid sequence of porcine AHF 3 Formation of oligonucleotide probes, and use of those probes to identify and or isolate at least a fragment of the gene which codes A for porcine AHF 4 Use of the porcine AHF gene fragment to identify and isolate human genetic material which codes for human AHF 5 Using the previously described ARF DNA fragments to determine the site of synthesis of AHF from among the various mammalian tissues 6 producing cDNA segments which code for human and porcine AHF, using messenger RNA obtained fran the tissue identification in step 5 7 Constructing full length human and porcine cDNA clones from the cDNA segments produced in step 6, e.g. by ligating together cDNA segments which were cut by the restriction enzymes 8 Forming DNA expression vectors which are capable of directing the synthesis of AHF 9 Transforming a suitable host with the expression vectors bearing the full length cDNA for human or porcine AHF 10 Expressing human or porcine AHF in the host and 11 Recovering the expressed AHF. In the course of this work, a new technique of screening a genomic DNA library has been developed utilizing oligonucleotide probes h d on the amino acid sequences contained in the AHF molecule. be invention includes the above Ur, along with the various nucleotides, vectors, and other products Made in connection therewith.Brief Summary of The Drawings Figure 1 is a depiction of the amino acid sequence A determined for the amino terminal sequence of the 69,000 dalton thrombin cleavage product described in Example 1. The first residue tias not identifiable. Tiis sequence is compared with H deduced from the nucleotide sequence of the porcineAHF exon described in Example 3, and with the human AHF exon sequene C described in Example 4. Figure 2 illustrates the amino acid sequence, shown in single letter code, of bovine thrombin digestion fragments of A the amino terminus and B a 40 KS thrombin cleavage product of the 166 Kd porcine AHF fragment isolated by Fass et al., infra. Figure 3 illustrates the design of an oligonucleotide probe for the identification and isolation of at least a portion of the porcine gene which codes for AHF. Figure 4 illustrates the DNA sequence for a Sma I DNA fragment 34 S1 which contains a porcine AHF exon, derived from bacteriophage PB34 described in Example 3. Figure 5 is a representation of the DNA sequence of theHae III insert 34 H1 bearing the exon for porcine AHF, as described in Example 3. This sequence is included within the longer sequence shown in Figure 4 and corresponds to nucleotides 250 615 of the sequence in Figure 4. Figure 6 is a representation of the DNA sequence along with the deduced amino acid sequence for nucleotides 34 84 for a portion of the Sau 3AI insert of clone 25 S1, showing a portion of the exon for human AHF, as described in Example 4. Figure 7 illustrates the DNA nucleotide sequence shown in one strand only which contains the entire sequence coding for human AHF, along with the deduced amino acid sequence for humanAHF. retailed Descrlptlon of the invention The e following definitions are supplied in order to facilitate the understanding of this case. To the extent that the definitions vary from meanings circulating within the art, the definitions below are to control. Amplification means the process by which cells produce gene repeats within their chromc6aral DNA. Cotransformation means the process of transforming a cell with more than one exogenous gene foreign to the cell, one of which confers a selectable phenotype on the cell. Downstream means the direction going towards the 31 end of a nucleotide sequence. An enhancer is a nucleotide sequence that can potentiate the transcription of genes independent of the identity of the gene, the position of the sequence in relation to the gene, or the orientation of the sequence. A gene is a deoxyribonucleotide sequence coding for a given mature protein. For the purposes herein, a gene shall not include untranslated flanking regions such as RNA transcription initiation signals, polyadenylation addition sites, promoters or enhances. A selection gene is a gene that confers a phenotype on cells which express the gene as a detectable protein. A selection agent is a condition or substance that enables one to detect the expression of a selection gene. Phenotype means the observable properties of a cell as expressed by the cellular genotype. Genotype means the genetic information contained within a cell as opposed to its expression, which is observed as the phenotype. Ligation is the process of forming a phosphodiester bond between the 5 and 3 ends of two DNA strands. This may be acoomplished by several well known enzymatic techniques, including blunt end ligation by T4 ligase. Orientation refers to the order of nucleotides in a DNA sequence. An inverted orientation of a DNA sequence is one in which the 5 to 3 order of the sequence in relation to another sequence is reversed when oompared to a point of reference in theDNA fran which the sequence was obtained. Such points of reference can include the direction of transcription of other specified DNA sequences in the source DNA or the origin of replication of replicable vectors containing the sequence. Transcription means the synthesis of RNA from a DNA template. Transformation means changing a cell s genotype by the cellular uptake of exogenous DNA. Transformation may be detected in some cases by an alteration in cell phenotype. Transformed cells are called transformants. Pre transformation cells are referred to as parental cells. Translation means the synthesis of a polypeptide fran messenger RNA The present invention permits, for the first time, the identification and isolation of porcine Factor vIII C gene by recombinant DNA techniques. It also permits, for the first time, the isolation and identification of the gene which encodes human factor VIII C by recombinant DNk techniques, by taking advantage of the homology between porcine AHF DNA and human AHF DNA in order to locate and isolate the human gene for AHP. This route to cDNA clones for producing AHE via homology with the porcineAHF gene avoids the tedious time consuming and expensive need to purify human AHF, which is highly expensive and essentially unavailable. Porcine AHF is first highly purified, most preferably by a monoclonal antibody purification technique such as that disclosed by David Fass et al, Nonoclonal Antibodies to PorcineAHF Coagulant and Their Use in the Isolation of Active CoagulentProtein , Stood 59 594 600 1982 , and Knutsen et al., PorcineFactor VIII C prepared by affinity Interaction with VonWillebrand Factor and Heterologous Antibodies, Blood, 59 615 24 1982 , the disclosures of both of which are incorporated herein by reference. Porcine factor VIII C polypeptides are bound to anti VIII C monoclonal antibodies which are immobilized on a suitable affinity chratiatography column. Two large molecular weight polypeptides, having molecular sizes of about 166 and 130Kd, are eluted with ethylenediamine tetraacetic acid. Another protein segment having a molecular weight of about 76 Kd, is then eluted fran the column, utilizing about 50 ethylene glycol. The partial amino acid sequences of these polypeptides, and or of fragments of these polypeptides obtained in a known manner, e.g.from enzymatic digestion of the proteins, using bovine thrombin or other suitable agents to break up the proteins, are then determined by known methods of analysis. Based on the amino acid sequence of these materials, oligonucleotide probes are synthesized, at least some of which will hybridize with DNA segments which code for the corresponding segment of AHF. These oligonucleotides are then used to screen for segments of the gene which code for porcine AHF. On a portion of the porcine gene for AHF is obtained, that recombinant material is used to screen a human genomic DNA library to locate and isolate the gene which codes for human AHF.In this procedure, it is established that there are substantial similarities between human AHF and porcine AHF, and advantage is taken of those similarities to isolate and identify the human factor VIII C gene or gene segments. The e similarities in the porcine and human proteins are attributable to corresponding similarities in the DNA sequences which code for the amino acid sequences. The genetic materials coding for the AHF proteins in humans and pigs are identical at a high percentage of positions, and thus exhibit hybridization when subjected to the procedure of, for example, Benton and Davis, Screening gt Recombinant Clones by tybridization to SinglePlaques In Situ, Science. 196 180 1977 , the disclosure of which is incorporated herein by reference. me e cloned segments of the gene for human AHF are then used to identify a source for AHF mRNA fran among various human tissues. Similarly, one or more of the cloned segments for porcine AHF are used to screen potential tissue sources for porcine mRN. This step is accomplished by conventional procedures involving RNk extraction, gel electrophoresis, transfer to nitrocellulose sheets, and hybridization to radiolabelled cloned DNA as described for example in Maniatis, et. al., Molecular Cloning.A Laboratory Manual, Cold SpringHarbor Laboratory, 1982 . nce this tissue source is identified, cDNX libraries are prepared in either plasmid or preferably bacteriophage lambda vectors, both of which are generally available and screened for AHF cDNA clones as described below.The process of cDNA library screening is repeated until a set of cDNA clones is obtained which, by DNA sequence detennintion, is shown to comprise the entire DNA gene sequence encoding AHF. On the full length human or porcine AHF cDNA clone is obtained, known and appropriate means are utilized to express theAHF protein, e.g. insertion into an appropriate vector, and transfection into an appropriate host, selection of transformed cells transformants , and culture these transformants, to express AHF activity. Host vector systems for the expression of AHF may be procaryotic, but the complexity of AHF makes the preferred expression system eucaryotic, preferably at least for biologically active AHF having clotting activity a mammalian one. This is easily accomplished by eucaryotic usually mammalian or vertebrate cells transformation with a suitable AHF vector. Eucaryotic transformation is in general a well known process, and may be accooplished by a variety of standard methods. These include the use of protoplast fusion, DNA microinjection, chromosome transfection, lytic and nonlytic viral vectors For example, Mulligan et al., Nature London 277 109 114 1979 , cell cell fusion Fournier et al., Proc. Nat. Acad. Sci. li 319 323 1977 , lipid structures U.S. patent 4,394,448 and cellular endocytosis of DNA precipitates Bachetti et al., Proc. Nat. Acad. Sci. i. 74 1590 1594 1977 . Other eucaryotic cells, such as yeasts or insect cells, may also be used to advantage. Transformation which is mediated by lytic viral vectors is efficient but is disadvantageous for a number of reasons me maximum size of transfected DNA is limited by the geometry of viral rapid packing, the exogenous genes are frequently deleted during viral replication, there is a requirement for helper virus or specialized hosts, host cells must be permissive, and the hosts are killed in the course of the viral infection. Nonlytic transformations are based on the transcription and translation of virus vectors which have been incorporated into a cell line as a stable episome. Ihese systems generally require unique cell lines and suffer fran a number of disadvantages. See frends in Biochemical Sciences , June 1983, pp. 209 212. On the other hand, other transformation methods in which extrachromosomal DNA is taken up into the chranoscmes of host cells have been characterized by low frequencies of transformation and poor expression levels. These initial difficulties were ameliorated by transformation with genes which inheritably confer selectable phenotypes on the small subpopulation of cells that are in fact transformed selection genes . The e entire population of transformed cells can be grown under conditions favoring cells having acquired the phenotype, thus making it possible to locate transformed cells conveniently.Thereafter, transformants can be screened for the capability to more intensely express the phenotype. This is accomplished by changing a selection agent in such a way as to detect higher expression. Selection genes fall into three categories Detectably amplified selection genes, dominant selection genes, and detectably amplified dominant selection genes. Detectably amplified selection genes are those in which amplification can be detected by exposing host cells to changes in the selection agent. Detectably amplified genes which are not dominant acting generally require a parental cell line which is genotypically deficient in the selection gene. Examples include the genes for hydroxymetholglutanyl CoA reductase Sinensky, Biochem, Biophys, Res. Commun 78 863 1977 , ribonucleotide reductase Meuth et al. Cell i 367 1943 , aspartate transcarbamylase Kemp et al. Cell 2 541 1976 , adenylate deaminase tDeBatisse et al. Mol and Cell Biol. 2 11 1346 1353 1982 mouse dihydrofolate reductase DHFR and, with a defective promoter, mouse thymidine kinase TK . Dominant selection genes are those which are expressed in transformants regardless of the genotype of the parental cell.Most dominant selection genes are not detectably amplified because the phenotype is so highly effective in dealing with the selection agent that it is difficult to discriminate among cell lines that have or have not amplified the gene. Examples of dominant selection genes of this type include the genes for procaryotic enzymes such as xanthine guanine phosphoribosyltransferase Mulligan et al. Proc. Nat. Acad. Sci. 12 4 2072 2076 1981 and aminoglycoside 3 phosphotransferase Colbere Garapin et al., J. Mol. Biol. , 150 1 14 1981 . Some dominant selection genes also are detectably amplified. Suitable examples include the mutant DHFR gene described by Haber et al., Samatic Cell Genet. A 499 508 1982 , cell surface markers such as HIA antigens and genes coding for enzymes such as specific esterases that produce fluorescent or colored products from fluorogenic or chromogenic substrates as is known in the art. Detectably amplified, dominant selection genes are preferred for use herein. It should be understood that a dominant selection gene in sane cases can be converted to a detectably amplified gene by suitable mutations in the gene. Selection genes at first were of limited comnercial utility. While they enabled one to select transformants having the propensity to amplify uptaken DNA, most selection genes produced products of no commercial value. On the other hand, genes for products which were oommercially valuable generally did not confer readily selectable or even detectable phenotypes on their transformants. This would be the case, for example, with enzymes or hormones which do not provide transformed cells with unique nutrient metabolic or detoxification capabilities. MDst proteins of commercial interest fall into this group, e.g.hormones, proteins participating in blood coagulation and fibrinolytic enzymes. Subseguently it was found that eucaryotic cells having the propensity to be transformed with and amplify the selection gene would do the same in the case of the product gene. By following the selection gene one could identify a subpopulation of transformant cells which coexpress and ooamplify the product gene along with the selection gene. It has been the practice to culture the transformants in the presence of the selection agent and to conclude that transformants having increased expression of the selection gene will also show increased expression of the product gene. This is not always the case, as will be more fully explored below.Axel et al. U.S. patent 4,399,216 use the term cotransformation to describe the process of transforming a cell with more than one different gene, whether by vector systems containing covalently linked or unlinked genes, and in the latter case whether the genes are introduced into host cells sequentially or simultaneously. Cotransformation should allow the introduction and stable integration of virtually any defined gene into cultured cells Wigler et al. Cell , 777 785, 1975 , and by use of the ootransformation process it is possible to produce eucaryotic cells which synthesize desired proteinaceous and other materials U.S. patent 4,399,216, column 3, lines 37 42 .Transformation Vectors Vectors used in AHF cotransformation will contain a selection gene and the AHF gene. In addition there usually will be present in the transformation or cotransformation vectors other elements such as enhancers, pranoters, introns, accessoryDNA, polyadenylation sites and 3 noncoding regions as will be described below. Suitable selection genes are described above. It is preferred that the selection agent be one that prevents cell growth in the absence of the selection gene. That way, revertant cells in large scale culture that lose the selection gene and presumably the AHF gene as well will not over grow the fermentation. However, it would be desirable in the commercial production of AHF to avoid the use of cell toxins, thereby simplifying the product purification steps. thus, a desirable selection gene would be one that enables transformants to use a nutrient critical for growth that they otherwise would not be able to use. The TK gene described above is an example. Two classes of vectors have been employed in cotransformation. The e first class are the unlinked vectors.Here the selection gene and the AHF gene are not oovalently bound. This vector class is preferred because the step of ligating or otherwise bonding the two genes is not required. This simplifies the transformation process because the selection and product genes usually are obtained fran separate sources s and are not ligated in their wild type environment. In addition, the molar ratio of the AHF and selection genes employed during cotransformation can be adjusted to increase cotransformation efficiency. The e second class of cotransformation vectors are linked vectors. These vectors are distinguished fran unlinked vectors in that the selection and AHF genes are covalently bound, preferably by ligation. The vectors herein may also include enhancers. Enhancers are functionally distinct fran pranoters, but appear to operate in concert with promoters. Their function on the cellular level is not well understood, but their unique characteristic is the ability to activate or potentiate transcription without being position or orientation dependent. Promoters need to be upstream of the gene, while enhancers may be present upstream or 5 from the promoter, within the gene as an intron, or downstream fram the gene between the gene and a polyadenylation site or 3 fran the polyadenylation site. Inverted promoters are not functional, but inverted enhancers are. Fnhancers are cis acting, i.e., they have an effect on promoters only if they are present on the sameDNA strand.For a general discussion of enhancers see Khoury et al., Cell 313 314 1983 . Preferred enhancers are obtained from animal viruses such as simian virus 40, polyoma virus, bovine papillaria virus, retrovirus or adenovirus. Viral enhancers may be obtained readily from publically available viruses. The enhancer regions for several viruses, e.g., Bouts sarcoma virus and simian virus 40, are well known. See Luciw et al., Cell 33 705 716 1983 .It would be a matter of routine chemistry to excise these regions on the basis of published restriction maps for the virus in question and, if necessary, modify the sites to enable splicing the enhancer into the vector as desired. For example, seeKaufman et al, J. Mol. Biol. , 159 601 621 1982 and Mol. CellBiol. 2 11 1304 1319 1982 the disclosures of both of which are incorporated herein by reference. Alternatively, the enhancer may be synthesized from sequence data the sizes of viral enhancers generally less than about 150 bp are sufficiently small that this could be accooplished practically. Another element which should be present in the vector assembly is a polyadenylation splicing or addition site. This is a DNA sequence located downstream from the translated regions of a gene, shortly downstream fran which in turn transcription stops and adenine ribonucleotides are added to form a polyadenine nucleotide tail at the 3 end of the messenger RNA.Polyadenylation is important in stabilizing the messenger RNA against degradation in the cell, an event that reduces the level of messenger RNk and hence the level of product protein. Eucaryotic polyadenylation sites are well known. A concensus sequence exists among eucaryotic genes The hexanucleotide 5 AAUAAA 3 is found 11 30 nucleotides fram the point at which polyadenylation starts. DNA sequences containing polyadenylation sites may be obtained from viruses in accord with published reports. Exemplary polyadedylation sequences can be obtained fran mouse beta globulin, and simian virus 40 late or early region genes, but viral polyadenylation sites are preferred. Since these sequences are known, they may be synthesized in vitro and ligated to the vectors in conventional fashion. A polyadenylation region must be located downstream fran either the AHF and or selection gene, but may be ligated to either gene. It may be ligated to the selection gene only, and not the product gene, and this will be the case whether the vectors are linked or unlinked. The e sequence which separates the polyadenylation site fran the translational stop cordon is preferably an untranslated DNk oligonucleootide such as an unpromoted eucaryotic gene. Since such oligonucleotides and genes are not endowed with a promoter they will not be expressed.The oligonucleotide should extend for a considerable distance, on the order of up to about 1,000 bases, fran the stop codon to the polyadenylation site. This 3 untranslated oligonucleotide generally results in an increase in product yields. The vector may terminate fram about 10 to about 30 bp downstream fran the concensus sequence, but it is preferable to retain the 3 sequences found downstream fran the polyadenylation site in its wild type environment. These sequences typically extend about fran 200 to 600 base pairs downstream fran the polyadenylation site. The e vectors described herein may be synthesized by techniques well known to those skilled in this art. The components of the vectors such selection genes, enhancers, promoters, and the like may be obtained from natural sources or synthesized as described above. Basically, if the components are found in DNA available in large quantity, e.g. components such as viral functions, or if they may be synthesized, e.g.polyadenylation sites, then with appropriate use of restriction enzymes large quantities of vector may be obtained by simply culturing the source organism, digesting its DNA with an appropriate endonuclease, separating the DNA fragments, identifying the DNA containing the element of interest and recovering same. Ordinarily, a transformation vector will be assembled in small quantity and then ligated to a suitable autoncinously replicating synthesis vector such as a procaryotic plasmid or phage. The e pBR322 plasmid may be used in most cases.See Kaufman et al., . . The synthesis vectors are used to clone the ligated transformation vectors in conventional fashion, e.g. by transfection of a permissive procaryotic organsim, replication of the synthesis vector to high copy number and recovery of the synthesis vector by oell lysis and separation of the synthesis vector fran cell debris. The resulting harvest of synthesis vector may be directly transfected into eucaryotic cells, or the transformation vector may be rescued from the synthesis vector by appropriate endonuclease digestion, separation by molecular weight and recovery of the transformation vector. Transformation vector rescue is not necessary so long as the remainder of the synthesis vector does not adversely affect eucaryotic gene amplification, transcription or translation. For example, the preferred synthesis vector herein is a mutant of the E.coli plasmid pBR322 in which sequences have been deleted that are deleterious to eucaryotic cells. See Kaufman et al., pp. t. Use of this mutant obviates any need to delete the plasmid residue prior to cotransformation. Cotransformation. Selection and Detection of Amplification The cells to be transformed may be any eucaryotic cell, including yeast protoplasts, but ordinarily a nonfungal cell.Primary explants including relatively undifferentiated cells such as stem cells , and immortal and or transformed cell lines are suitable. Candidate cells need not be genotypically deficient in the selection gene so long as the selection gene is dominant acting. The e cells preferably will be stable mammalian cell lines as is discussed above. Cell lines that are known to stably integrate selection genes into their chranosanal DNA are best, for example Chinese hamster ovary CHO cell lines. Also useable are HeLa, aDS monkey cells, melanoma cell lines such as the Bowes cell line, mouse L cells, mouse fibroblasts and mouse NIH 3T3 cells. Cotransformation with unlinked vectors may be accomplished serially or simultaneously, see U.S. patent 4,399,216 . Methods for facilitating cellular uptake of DNA are described above. Microinjection of the vector into the cell nucleus will yield the highest transformation efficiencies, but exposing parental cells to DNA in the form of a calcium phosphate precipitate is most convenient. Considerably better cotransformation efficiencies result from cotransformation with a molar excess of product to selection gene, on the order of 100 1. The population of cells that has been exposed to transforming conditions is then processed to identify the transformants. Chly a small subpopulation of any culture which has been treated for cotransformation will exhibit the phenotype of the selection gene. The cells in the culture are screened for the phenotype. This can be acoomplished by assaying the cells individually with a cell sorting device where the phenotype is one that will produce a signal, e.g. fluorescence upon cleavage of a fluorogenic substrate by an enzyme produced by the selection gene. Preferably, however, the phenotype enables only transformants to grow or survive in specialized growth media as is further discussed above. Selection transformants then will be screened for ligation of the product gene into their chromosomes or for expression of the product itself. The former can be accooplished using Southern blot analysis, the latter by standard immunological or enzymatic assays. Once the tranformants have been identified, steps are taken to amplify expression of the product gene by further cloning in the presence of a selection agent such as MIX. SeeU.S. Patent 4,399,216. Cotransformants which my be produced in accordance with the processes described herein are suitable for in transfections of higher organisms in accordance with known techniques. Primary explants or stable cell lines from a potential host animal are cotransformedf and inoculated into the host or a substantially otherwise syngeneic host which is genotypically deficient in the product protein. The e invention will be further understood with reference to the following illustrative embo4imerts, wfiich are purely exemplary, and should not be taken as limitive of the true scope of the present invention. as described in the claims. Unless otherwise noted, restriction endonucleases are utilized under the conditions and in the manner recommended by their commercial suppliers. Ligation reactions are carried on as described by Maniatis et al., Molecular Cloning. A laboratory Manual, Cold Spring Harbor Laboratory 1982 at 245 6, the disclosure of which is incorporated herein by reference, using the buffer described at page 246 thereof and using a DNA concentration of 1 100 Rg ml, at a temperature of 230C for blunt ended DNX and 160C for sticky endd DN. Phosphatasing as described herein, refers to dephosphorylation of DNA, and is carried out in the manner described by Maniatis et al., supra, e.g. at page 133 et seq. Kinasing refers to phosphorylation ofDNA. Electrophoresis is done in 0.5 1.5 Agarose gels containing 90 wM Tris borate, 10 mM EDTA. Nick translating refers to the method for labeling DNA with 32p as described by Rigby et al., J.Mol. Biol., 113 237 1977 . All radiolabeled DNA is labeled with 32p, whatever labeling technique was used. By rapid prep is meant a rapid, small scale production of bacteriophage or plasmid DNA, e.g., as described by Maniatis et al., xEU at p. 365 373. In accordance with another aspect of this invention, there is provided a pharmaceutical preparation of AHF. A pharmaceutical preparation of human AHF produced in accordance with this invention may be prepared for parenteral administration in accordance with procedures well known in the art. me pharmaceutical preparation for human use comprises sterilized AHF recovered from transformed cells. In addition to the AHF polypeptide or sufficient portion thereof which producesAHF activity, there may be included one or more acceptable carriers therefor and optionally other therapeutic ingredients.The e carrier s must be acceptable in the sense of being compatible with other ingredients of the preparation and not deleterious to the recipient thereof. The e preparation may conveniently be presented in unit dosage form and may be prepared by any of the methods well known in the art of pharmacy. The pharmaceutical preparation of this invention, suitable for parenteral administration, may conveniently comprise a sterile lyophilized preparation of the AHF polypeptide which my be reconstituted by addition of sterilized solution to produce solutions preferably isotonic with the blood of the recipient. The preparation may be presented in unit or multi dose containers, for example sealed ampoules or vials. It should be understood that in addition to the sterileAHF or solution thereof, the pharmaceutical preparation of this invention may include one or more additional ingredients such as diluents, buffers, binders, surface active agents, thickeners, lubricants, preservatives including anti oxidants and the like.Gelatine, lactose, starch, inagnesi in sterate, micronized silica gel, cocoa butter, talc, vegetabilic and animalic fats and oils, vegetabilic rubber and polyalkylene glycol and other known carriers for pharmaceuticals are all suitable for manufacturing the pharmaceutical preparations of the present invention.Preparations for parenteral use include an ampoule of a sterile solution or suspension with water or other pharmaceutically acceptable liquid as the carrier therefor, or an ampoule of sterile solid AHF for dilution with a pharmaceutically acceptable liquid. The pharmaceutical preparation of the present invention is useful in the treatment of Hemophilia A. These preparations also provide an inportant tool in the in vivo as well as the in vitro study of the processes involved in clotting and in the study of the imnunulogical and biological characteristics of theAHF molecule.Example 1. Protein Sequence Analysis Porcine factor VIII C was purified by Dr. David Fass according to published procedures, Knutsen and Fass 1982 Fass et al. 1982 , supra. Amino acid sequence analysis is performed on a bovine thrombin digest product of the 76,000 dalton protein as described below. Porcine AHF polypeptides bound to the anti VIII C monoclonal antibody column, can be sucessively eluted in two steps Fass et.al. 1982 . The e two larger molecular weight species, 166,000 and 130,000 daltoos, can be eluted with ETTA. The retaining polypeptide, having a molecular weight of about 76,000 daltons, can then be eluted with 50 ethylene glycol. The 76,000 dalton protein is digested with thronbin after extensive dialysis in 50 nM Tris HCl PB 7.5 , 0.15 M NaCl.Bovine thrombin digests are performed at roan temperature for 60 minutes using 1 unit mi of bovine thrombin, followed by the addition of another 1 unit mA thrombin and incubation for an additional 60 minutes. The thrombin digestions are terminated by heating for 10 minutes at 900C in 0.01 SDS. The major thronbin digest product of the 76,000 dalton protein is a polypeptide, 69,000 daltons 69 Kd . Less than 1 ug of the 69 Kd polypeptide species described above is iodinated to serve as a radioactive marker after SDS gel electrophoresis, in accordance with the procedure of U. K. Laemmli, Nature, 227 680 1970 , the disclosure of which is incorporated herein by reference. The polypeptide, dissolved in mS buffer 50 mM Tris acetate pH 7.8 , 0.1t SDS is added to 100 l of the same buffer containing 5 mCi of carrier free iodine 125. 50 l of 2.5 mg ml chloramine T Baker in TAS buffer are added and the solution agitated for 1 minute. The reactions are stopped by adding 50 l 2.5 mg ml sodium metabisulfate in TAS buffer followed by 1 minute agitation.Labeled protein is separated from unincorporated I125 by chromatography using a small volume Sephadex G 25 M column PD 10, Pharmacia . The e coltznn is pre equilibrated with several column volumes of TAS buffer containing 2.5 mg ml sodium iodide.The void volume is collected and protein integrity analyzed bySDS gel electrophoresis in accordance with the procedures ofLaemmli, 1970 , supra. The radioactively labeled 69 Kd protein is added to its unlabeled counterpart for subsequent monitoring. The protein is then individually electrophoretically concentrated. The protein solutions are adjusted to 0.1 SDS, 10 ztt dithiothreitol and Coomassie brilliant blue Serva are added to make the solution a very pale blue. The solutions are then dialyzed briefly 1 2 hours in TAS buffer using Spectrapore dialysis tubing made by SpectrumMedical Industries, Inc. , with molecular weight cut off at about 14,000 . The dialyzed protein samples are then placed in an electrophoretic concentrator, of the design suggested byHunkapiller, et al Meth. Enzymol.. Enzyme Structures, Part I, 91 227 1983 , and electrophoretically concentrated in TAS buffer for 24 hours at 50 volts. The 69 Kd AHF polypeptide, concentrated by the above procedure, is electrophoresed through an SDS polyacrylamide gel in accordance with Laemli, supra. The purified protein, identified by autoradiography of the radioactively labeled tracer polypeptide, is excised from the gel and electroeluted and concentrated as described by Bunkapiller et al., supra. The concentrated sample is suitable for direct amino acid sequence analysis using the gas phase sequenator as described in Hewick et al., J. Biol. Chem. 256 7990 1981 . The amino terminal sequence extending from the 2 42 residue of the 69,000 dalton thranbin cleavage product is as depicted in Figure 1. The first residue X was not identifiable. The amino acid sequences of A the amino terminus and B a 40 kd bovine thrombin digestion product from the 166 kdAEF fragment noted by Fass et al., supra, are shown in Figure 2.The amino acid sequence of the amino terminus of the 76 Kd polypeptide is X Ile Ser lau Pro Thr Phe Gln Pro Glu Glu Asp LysMet Asp Tyr Asp Asp Ile Phe. Example 2 a Pentapeptide Probe Pool The partial amino acid sequence of a fragment of porcineAHF having been determined allows porcine AHF oligonucleotide probes to be designed and synthesized. From the genetic code Table 1 it is possible to predict the gene sequence that codes for this sequence of amino acids. Because the genetic code is degenerate, there are more than one possible DNk coding sequences for each amino acid sequence. Accordingly, a plurality or pool of oomplementary oligonucleotide probe sequences are provided for a region of the AHF molecule which required only a reasonable number of oligonucleotides to ensure the oorrect DSk sequence. Such regions are selected by searching for tracks of five to eight contiguous amino acids which have the lowest degeneracy.After the region is selected, a pool of oligonucleotides is synthesized, which would include all possible DNA sequences which could code for the five to eight amino acids in the selected region. In the 69,000 dalton thratin cleavage fragment of porcine AHF there is a pentapeptide sequence running fran the 18th through the 22nd amino acid fran the amino terminus, which could be coded for by up to 16 different DNA sequences, each having five codons, for a length of 15 nucleotides. 18 20 22 Trp Asp Tyr Gly Met Possible mRNA Sequences UGG GAC UAC GGU AUG or or or GAC UAC GGC or GGA or GGG The probes are made by synthesizing a limited number of mixtures of oligonucleotides with two to eight or more oligonucleotides per mixture. These mixtures are referred to as pools. Enough pools are made to encompass all possible coding sequences. These oligonucleotides can be synthesized manually, e.g., by the phospho tri ter method, as disclosed, for example in R.L. betsinger, et al., J. Am. Chem. Sc c. 98 3655 1967 , the disclosure of which is incorporated by reference. Other methods are well known in the art. See e also Matteucci and Caruthers, X Am. Chew. Soc. 103 3185 1981 , the disclosure of which is incorporated by reference. Preferably, however, the synthetic oligonucleotide probes for the desired polypeptide sequences are prepared by identical chemistry with the assistance of the corrp letely automatic AppliedBiosystems DNA synthesizer, Model 380A, as indicated above. The e oligonucleotides thus prepared can then be purified on a reverse HPLC column, as described by H. Fritz, et al., Biochemistry, 17 1257 1978 , the disclosure of which is incorporated herein by reference. After detritylation with 80 HOAc, the resulting oligonucleotide is normally pure and can be used directly as a probe. If there are any contaminants, the synthetic DNA can be further purified on the same HPLC column, preferably using a slightly different gradient system. Oligonucleotides are labelled, e.g. by using 32P ATP and T4 polynucleotide kinase, and their sequence checked either by two dimensional homochromatography as described by Sanger et al., PSAS U.S.A. 70 1209 1973 or by the Maxam Gilbert method,Meth. Enzymology, 65 499 1977 , the disclosures of both of which are incorporated herein by reference. b Forty Five Nucleotide Probes A unique aspect of the present invention has been the use of oligonucleotide probes to screen a genomic Pizza library for theAHF gene or fragments thereof. While oligonucleotides have been used for screening of cDNA libraries, see, e.g. M. Jaye, et al,Nucleic Acids Research 11 2325 1982 , the disclosure of which is incorporated herein by reference, genomic libraries have previously been screened successfully only with cDNA probes, i.e. probes which were generated only after the tissue source of the mRNA for a described protein had been identified and utilized to generate a cDNA clone which precisely natched the DNA sequence of the gene sought by the genomic search. In the present case, it has been shown possible to use oligonucleotides to identify gene segments in a genomic library which code for the amino acid sequence of the proteins of interest, and identification of such gene segments provides an exact probe for use in mRNk, cDNk or further genomic screening techniques. Preferably, as here, oligonucleotides corresponding to at least two segments of the amino acid sequence of the protein of interest are utilized. Preferably at least one of the oligonucleotide probes is used in the form of one or more pools of oligonucleotides which in the aggregate include every possible DNk sequence which could code for the amino acid sequences selected. Preferably one relatively short probe, e.g. 11 to 25 nucleotides, preferably 15 to 20 nucleotides is utilized in conjunction with a relatively long probe, e.g. 30 to 200 nucleotides, preferably 40 to 50 nucleotides. The second probe can be used for confirmation, and is not always necessary for identification of the DNA segment.Preferably at least one of the probes, and more preferably the longer of the probes is designed in accordance with the Rules 1 to 4 described below. Rule 1. Colon Preference In the absence of other considerations, the nucleotide sequence was chosen which matched prevailing or similar sequences in similar mamalien genes. SeeMechanisms of Ageing Dey., 18 1982 . RLle 2. Advantageous GT Pairing The nucleotide G, in addition to bonding to its compliment C, can also form weak bonds with the nucleotide T. See K. L. Agarwal et al., J. Biol. Chem.256 1023 1981 . Thus, faced with a choice of G or A for the third portion of an ambigious codon, it is preferable to chooseG, since if the resulting hybridization would occur even if the actual nucleotide in the position is a T, rather than a C, the hybridization would still be stable. If an A were chosen inoorrectly, the corresponding A C incompatability could be enough to destroy the ability of the probe to hybridize with the genomic DNA. Rule 3. Avoidance of 5 CG Sequences When selecting from among the possible ambiguities, select those nucleotides which will not contain the sequence, either intra codon or inter codon. Pulse 4. Mismatch Position In choosing the codon sequences to use, consideration was given to the postulation that mismatches near the ends of the molecule do not adversely affect the stability of the hybridization as do mismatches near the center of the molecule. Thus, for example, where substantial doubt occured concerning the sequence of a particular codon, and that codon was close to the center of the probe, the tendency was to test a pool of possible nucleotide sequences for that codon, whereas codon positions near the ends of the probe were more likely to be subject to determinations on the basis of codon preference. The e chosen sequence for the 45 mer probes, as well as the amino acid sequences, the possible DNA sequences, and the ActualProbe Sequence , i.e. the complement of actual coding strand determined for the AHF exon, as shown in Figure 3. Thus, out of nucleotide positions involving choices, three were covered by using pools containing both possible nucleotide alternatives, five were predicted correctly, one was predicted in a way to maintain neutrality though incorrect, and four others were in error. Despite the approximately 11 mismatch 5 45 the pool of 45 mer oligonucleotides are adequate to strongly identify a porcine AHF gene fragment, as described below. Example 3. Screening of Porcine Genomic Library A porcine genomic library is constructed using the bacteriophage vector Lambda Jl. Lambda Jl is derived from L47.1 Lcenen et al., Gene 20 249 1980 by replacement of the 1.37 kb and 2.83 kb Eco Rf Bam HI fragments with a 95 bp Eco R1 HindIII Xba I Dgl II B n HI polylinker. The 6.6 kb Bam HI fragment is then present as a direct repeat in reverse orientation relative to L47.1. The cloning capacity for Bam HI fragments is 8.6 23.8 kb. Barn HI cleaved porcine DE prepared as described by Piccini et al., Cell, 30 205 1982 is phenol extracted, ethanol precipitated and concentrated by centrifugation in aMicrofuge. 0.67 ug of Barn HI porcine DE is ligated to 2 g ofLambda JL Bam HI arms , prepared as described in Maniatis, et al., supra, pp 275 279, in a volune of 10 s1 ligation buffer with 10 units T4 DNA ligase Maniatis et al., , p 474 . The ligated DNA is packaged and plated as described in Maniatis et al., supra, p 291. Approximately 4 x 105 pfu are plated on E, coli stainC600 on 15 an plastic petri plates containing NZCYM agarose, at 8,000 pfu plate. These recombinant phage are screened by the method of Woo 1979 using the 45 mer probe described above, radioactively labeled with 32P as described above, as a probe.Filters are then hybridized in SxSSC, 5x Denhardt s, 0.1 SDS, and 5 x 106 cpm ml probe at 450 C for 16 hours, washed in 5 xSSC, 0.1 SDS at 500 C and subjected to autoradiography using intensifying screens DuPont Lightning Plus . Autoradiography reveals numerous phage which hybridized, to varying degrees, with the 45 mer. The e filters are then denatured in 0.SM NaOH, neutralized in 1.0M Tris pH7.5, 1.5M NaCl and hybridized to the 15 mer as described for the 45 mer except the hybridization and washing temperature is 370 C. One phage which hybridized to both probes is picked from the original plate and 100 pfu plated and the plaques screened as described above using the 15 mer as probe. A positive phage, named PB34, is picked as a plug and used to make a plate lysate as described in Maniatis et al, supra, pp 65 66. A small scale isolation of PB34 DNA is achieved using the procedure described in Maniatis et al., supra, pp 371 372. 10 1 of this DNa was cut with the restriction enzymeHae III and then phosphataeed using Calf alkaline phoaphatase Boehringer Mannheim . After phenol extraction, 20 ng of Sma I cut M13mp8 DNA is added, the solution is made 0.2M NaCI and nucleic acid precipitated by the addition of 2 volumes ethanol.Precipitated DNA is pelleted by centrifugation and redissolved in 2 ul of a ligation mixture and the DNA is ligated for 30 minutes at 230 C, diluted to 50 l with ligase buffer and ligated an additional 3 hours. 5 13 of this reaction is used to transformE. coli strain JMl0l TGl. Cells are made competent for transformation by growing to an 0.D.600 of 0.5 at 37 C in 50 ml SOBM medial SOBM is 2 tryptone, 0.5 yeast extract, 0.1M NaCl, 0.llg KOH per liter, 20mM MgSo4 . Cells are pelleted by centrifugation at 2500 rpn for 10 minutes at 40 C. The cells are resuspended in 3.5 ml 100mM RbCl, 45mM MnCl2, 50mM CaCl2, 10mM potassium MES pH 6.4 MES methylethane sulfonic acid . 200 l of competent cells are transformed with the DNA contained in 5 ml of the ligation reaction at 0 C for 30 minutes. The cells are then heat shocked at 420 C for 90 seconds after which 4 ini of 0.8 agarose SOBM containing 100 l stationary Je101 TG1 cells are added and plated on 10 cm SOBM agar petri plates. A subclone, containing a Hae III fragment from PB34, hybridizing to the 15 mer is identified by screening using the procedure or Benton and Davis, supra. This clone is isolated and prepared for use as a template. The DNA sequence of the Hae III fragment present in this clone, designated as 34 H1, is shown inFigure 5. M13 template DNA is prepared by growing 1.5 ml of infected cells 5 hours at 370 C. Cells are pelleted by centrifugation for 10 minutes in a Beckman microfuge. 1.0 ml of supernatant containing virus is remaved and 200 l of 20 polyethylene glycol, 2.5 M NaCI is added. This sample is then incubated at room temperature for 15 minutes followed by centrifugation for 5 minutes in a Beckman microfuge.The e pellet is dissolved in 100 l TE, 7.5 l of 4M NaCHCOO pH 4.5 is added and the sample extracted twice with a 1 1 mixture of phenol chloroform and once with chloroform. The e single stranded phage DNA is then precipitated by the addition of 2 volumes of ethanol. Precipitated DNA is pelleted by centrifugation in aBeckman microfuge and dissolved in 30 l 1mM Tris pH 8.0, 0.1 mMEDTA. The DNA sequencing is performed by the dideoxy chain termination technique, see, e.g. Sanger et al., PNAS U.S.A., 74 5463 1977 , utilizing the 15 mer as a primer. The e sequence observed, which is included in the sequences represented in Figs.4 and 5, confirmed the subclone as containing a porcine AHF exon as it includes the identical 14 amino acids encompassed by the region phenylalanine2 to glutaminels in the amino terminal sequence of the 69K fragment represented in Figure 1. Further confirmation came from sequencing fran the 5 end of the Hae III insert in the 34 z1 vector, by primping with the Universal primer Bethesda Research Labs at a point adjacent the polylinker in that vector. Also, the insert fram this clone, named 34 H1, was recloned into M13mp9 by restricting the DNA withEco RI and Hind III, phosphatasing with calf alkaline phosphatase, and ligating to Eco RI, Hind III cleaved M13mp9 DNA.This clone, which contains an inversion of the Hae III segment relative to the universal primer, was also sequenced as described above. The resulting sequence data for all of insert 34 H1 is shown in Figure 5. This sequence confirms that this subclone contains an exon of the porcine AEF gene that could encode, fran nucleotides 169 to 267, at least the thirty amino acids from the phenylanine2 through arginine31 of the 69K fragment Fig. 1 . It appears likely that arginine31 borders an intron because termination codons can be found downstream from nucleotide 267 Fig. 5 in all three reading frames, and a sequence similar to the consensus 5 splice site sequence is also found in that region between nucleotides 266 267. Further, the amino acid sequence which would be encoded by the downstream DNA differs completely fran that observed in the 69 Kd fragment of porcine AHF. PB34 DNA was cut with Bin HI and electrophoresed through an agarose gel and the bands visualized by ultraviolet light after staining the gel in 5 jjg ml ethidium bromide. Three inserts of approximately 6.6 kb, 6.0 kb, and 1.8 kb were observed. The DNA in the gel was transferred to nitrocellulose as described in Maniatis et al., supra, pp 383 386.Hybridization of the filter to the 15 mer and autoradiography were performed as described above. Autoradiography revealed that the 6.0 kb band contained an AHF gene fragment which hybridized to the 15 mer probe. Thus, for the first time, a section of the gene coding for porcine AHF had been isolated and identified. The bacteriophage ladd recombinant clone PB 34 is on deposit at the American Type Culture Collection under Accession Number ATCC 40087. Exairple 4. location of Humpn AHF Gene The hunan genanic library described by Maniatis et al., Cell, 15 687 1978 is screened for the human AHF gene by infecting E. coli strain LE392 publicly available with 6x105 pfu and plating on 15an MYCYM agar plates at a density of 20,000 pfu plate. These phage are screened using the procedure ofBenton and Davis, supra with the 6.0 kb porcine AHE fragment described in Example 3, labeled with 32p by nick translation, as the probe.A phage exhibiting a strong hybridization signal is picked and plated at about 100 pf u lOan plate and screened in duplicate as described above using the radioactively labeled 45 mer as one probe and the 6.0 kb Bam HI fragment of PB34 as the other. A phage, named HH25, which hybridizes to both probes is identified, a plate stock made and rapid prep DNA prepared as described above. The phage DNA is cut with Sau3A I, phosphatased with calf alkaline phosphatase phenol extracted, and co preicpitated with 20 ng of Bam HI cut M13 mp8Precipitated DNA is pelleted by centrifugation and redissolved in 2 l ligase buffer containing T4 DNA ligase.Ligation is performed for 2 minutes at 160 C, diluted to 50 1 in ligase buffer containing T4 DNA ligase and incubated an additional 3 hours at 160 C. 5 l of this reaction mixture is used to transform E. coli strain JM101 TGl as described in Example 3 above. The plaques are screened using the Benton and Davis procedure and probing with radioactively labeled 15 mer. A phage plaque, named 25 S1, exhibiting hybridization is isolated and single stranded phage DNA prepared for use as a DNA sequencing tenplate as described above. Sequencing is performed using the dideoxy chain termination technique described by Sanger et al., supra, utilizing the l5 mer as primer, and gives the information strand DNA sequence shown in Figure 6. 84 nucleotides sequenced in this manner demonstrated 85 homology with the homologous region of porcine AHF.There is also only one amino acid difference between the porcine AHF 69K region 2 16 as shown inFigure 1 and the corresponding region which was deduced fran the human nucleotide sequence of Figure 6. This high degree of hanology shows that the DNA of the recombinant phage HH 25 ennates fran the AHF gene. Thus, for the first time, an exon for the hunan AHF gene has been isolated and identified. A bacteriophage lambda recombinant HH 25 is on deposit at the American Type CultureCollection under the Accession Niter ACCC 40086. Example 5 Identifying Cells Actively Transcribing AHF The porcine and hunan AHF exons described above are useful for a variety of functions, one of which is as a screening agent which permits identification of the tissue which is the site or synthesis of AHF in vivo. A number of screening methods are available, based on the use of the exon as an exact complement to the mRNA which is produced during the course of natural expression of AHF. In the screening procedures, tissue fran various parts of the body is treated to liberate the mRNA contained therein, which is then hybridized to a DNA segment containing the exon for AHF, and if a molecule of mRNA does hybridize to that exon, the tissue which is the source of that mRNA is the source of AHF. 1. screening Procedure Porcine or human tissue fran various organs, including kidneys, liver, pancreas, spleen, bone marrow, lymph nodes, etc., is prepared by guanidine hydrochloride extraction as described byCox Methods Enzymol., 12B 120 1968 , the disclosure of which is incorporated herein by reference, with same modifications.Briefly, tissue is explanted into 8 M guanidine hydrochloride or 4M guanidine isothiocyanate as proposed by Chirgwin, et al., Biochemistryf 18 5294 1979 , the disclosure of which is incorporated herein by preference, see also Maniatis, et al., supra, at 189 et seq. , 50 mM Tris pH 7.5 , 10 mM EDTA and homogenized in an Omnimixer Sorvall at top speed for 1 minute.The homogenate is clarified at 5000 rEm for 5 minutes in a SorvalHB 4 rotor and the ffi is precipitated by the addition of 0.5 volunes of ethanol. The RNA is dissolved and precipitated 3 more times fran 6 M guanidine hydrochloride before being dissolved inH2O. Messenger RNA fran this pool is enriched by chromatography on oligo dT cellulose Collaborative Research . This rnPNA is then subjected to electrophoresis through an agarose gel containing formaldehyde, as described by Maniatis et al., supra, at 202 3. The mRNA in the gel is then transferred to a nitrocellulose filter Maniatis et al., ,at 203 4 . The thus obtained mRNA is hybridized with the radiolabeled porcine or human exon DNA obtained as described above, and the existence of hybrids detected by autoradiography.A radioactive signal indicates that the tissue source of the is a source of synthesis of AHF in the body. Alternatively, mRNA can be screened using the Sl protection screening method. S1 nuclease is an enzyme which hydrolyzes single strandedDNA, but does not hydrolyze base paired nucleotides, such as hybridized mRNA DNA. Thus the existence of a radioactive band after acrylamide gel electrophoresis and autoradiography shows that the single stranded DNA corresponding to the AHF exon has been protected by the complementary mRNA, i.e. the mRNA for AHF. Thus the tissue which was the source of that my is a site of synthesis of AHF ni vivo. That tissue can be used as a source for AEF S. This method of screening can be sanewhat more sensitive than screening method described above. A probe consisting of single stranded radioactively labeled DNA complementary to the mRNA is synthesized using the universal primer of M13 to prime DNA synthesis of the porcine genomic subclone 34 Hl, The reaction is performed in a 100 1 solution of 50 mM Tris pH 7.4, 5 mM MgC12, 1 mM 2 mercaptoethanol, 50 mM NaCl, 40 M dGTP, dTTP, dCTP, 60 Ci of 32p dATP 400Ci mmole , 10 ng universal primer, 200 400 ng of 34 53 template DNA, and the Klenow fragment of DNA polymerase I E. coli . The reaction is incubated at 230 C for 60 minutes, 10 minutes at 70 C, 50 units of PstI added and incubated an additional 60 minutes. The reaction is terminated by phenol chloroform extraction, NaCl added to 0.2M and then precipitated with two volunes 100 ethanol. The precipitated DNA is pelleted by centrifugation, redissolved in 20 sucrose, 50nM NaCH, 0.1 cresol green and then electrophoresed through 2 agarose in 50 mMNaOH, 10 mM En. The resulting single stranded fragment is localized by autoradiography, the band excised and DNA isolated by electroelution in dialysis tubing. Sable mRNA is prepared from liver, spleen, etc. tissue by the guanadine hydrochloride method described above. The probe is then hybridized to sample mRNA obtained fran the oligo dT chromatography enriching step in 50 formamide, 0,4M NaCl, 4 mM Pipes piperazine N,N bis 2 ethanesulfonic acid pH 6.5, 1 mM EDTA, 5 50 g mRNA, 2 g labeled DNA in a volune of 15 1. The hybridization is terminated by the addition of 200 a1 cold S1 nuclease buffer 0.25 M Nail, 0.3 M NaCH3COO pH 4.5 , 1 mM ZnSO4, 5 glycerol, and 1000 units S1 nuclease. The reaction is incubated at 450 C for 30 minutes.Samples are phenol extracted, ethanol precipitated with 10 g yeast tRNA carrier and subjected to analysis by electrophoresis through 5 polyacrylamide sequencing gels as described in Maxam Gilbert, PNAS USA, 74 560 1977 . Example 6 Use of AHF Exon DNA to Obtain AHF mRNA From Tissue Once a tissue which is a source of nHNA is identified, AHF mRNA fran that tissue is extracted and used to construct a cDNA library. The c library is then utilized for identifying and constructing a full length cDNk clone which encodes the amino acid sequence for AHF, without the introns contained in the genomic clone , as described below. Thereafter, the cDNA which encodes the AHF protein is inserted into an appropriate expression vector, in an appropriate host, for expression of AHF. Since human AHF is in great demand for treatment of hemophilia and other uses, the preparationg of human AHF cDNA is described. 1. Obtaining IIA for Roman EF mRNA from the human tissue responsible for AHF synthesis is prepared by the guanidine hydrochloride extraction method as described by Cox and modified by Chirgwin, et al. as disclosed above. Further fractionation of mRNA obtained fran the oligo dT cellulose chromatography column is obtained by sedimenting on 5 20 sucrose gradients containing 10 nR Tris SCl pH 7.4 , lnM EDTA and 0.2e SDS by centrifugation for 24 hours at 22,000 rpn in a Beckmen SW28 rotor. Fractions 1.0 ml are collected, sodium acetate was added to 0.2M, and the fractions are ethanol precipitated twice before dissolving in water. The size distribution of the fractionated RNA is determined by electrophoresis through 1.4 agarose gels containing 2.2 M formaldehyde. mRNA sedimenting with an S Svedberg value greater than 28 is pooled for the synthesis of double stranded cDNA. 10 g of this RNA is denatured at room temperature in 10 l of 10 mM methylmercuryhydroxide. 140 mM 2 mercaptoethanol is added to inactivate the methylmercuryhydroxide. The RNA is then diluted to 50 a1 containing 140 mM KCl, 100 mM Tris RCl pH 8.3 at 420C , 1 mM of each deoxynucleotide triphosphate, 200 g ml oligo dT 12 18, 10 mM MgCl2, and 0.1 Ci 32p dCTP ml. These reactions are performed at 420C for 1 hour after the addition of 3 l of 17 units l AMV reverse transcriptase Life Sciences . The reaction is terminated by the addition of 0.25 M EDTA pH 8.0 to 20 zM. The resulting mixture is extracted once with an equal volume of phenol chloroform 1 1 followed by one chloroform extraction. The sample is then chromatographed on a 5 ml Sepharose CL 4B column Pharmacia equilibrated in 10 mMTris HCl pH 8.00, 100 mM NaCl, 1 mM ETDA. The void volume is collected and the nucleic acids including any RNA cDNA hybrids are precipitated by the addition of 2.5 volumes of ethanol. Preferably in conjunction with the above prooedures anAHF exon oligonucleotide segment is also used in place of oligo dllr to prime the reverse transcription, as described in Ullrich et al., Nature, 303 821 1983 . The RNA cDNA hybrids are dissolved in 35 1 of deionizedH2O, made 100 mM potassium cacodylate pH 6.8 , 100 M dCTP, 1 mM 2 mercaptoethanol, 1 JIM cobalt chloride and enzymatically tailed by the addition of 10 units of deoxytidyl terminal transferase pH Biochemicals and incubating the reaction for 30 seconds at 37 C. The reaction is terminated by adding 0.25 MEDTA to 10 mM. Tris HCl pH 8.0 is added to 300 uM and the sample extracted once with an equal volume of phenol chloroform 1 1 and then with an equal volune of chloroform. Nucleic acids are precipitated fran this product by the addition of 2.5 volumes of ethanol. The dC tailed hybrid molecules are then annealed with 170 g ml oligo dG 14 18 cellulose in 10 mM KCl, 1 mM EDTA for 10 minutes at 430C and then an additional 10 minutes at 230C. This reaction product is then diluted to 100 l containing 100 mM ammonium sulfate, 1 mM 2 mercaptoethanol, 100 mM MgCl2, 100 g ml bovine serum albumin Sigma, Cohn fraction V , and 100 M nicotinamide adenine dinucleotide. Second strand cDNA synthesis is initiated by the addition of 1 unit RNase H P L Bioche nicals , 1 unit of . coli DNA ligase, and 10 units of DNA polymerase I and incubated at 160C for 12 hours. The sanple is then chranatographed over Sepharose CL 4B as described above. Double stranded DNA is ethanol precipitated and dC tailed as described for the RNA cDNA hybrid tailing. 2. Screening for Rumen AHF DNA dC tailed double stranded cA obtained as described above is annealed with an equimolar amOunt of dG tailed pBR322 New England Nuclear in 10 mM Tris HCl pH 8.0 , 1 mM EDrA, 100 nM NaCl at 370 for 2 hours. The annealed chimeric molecules are then frozen at 200C until use in bacterial transformation. Bacterial transformation is done using the MC1061 strain of E. coli source . Cells 50 ml are grown to an optical density of 0.25 at 600 nm. Cells are concentrated by centrifugation at 2,500 rpn for 12 minutes, washed in 10 ini of sterile 100 mM CaCl2, and again pelleted by centrifugation as described above. Cells are resuspended in 2 ml of sterile 100 mMCaCl2 and kept at 40 for 12 hours. The annealed chimeric molecules are incubated at a ratio of 5 ng of double stranded cDNA per 200 l competent cells at 40C for 30 minutes. The bacteria are then subjected to a two minute heat pulse of 420C.1.0 ml of L broth is then added and the cells incubated for 1 hour at 37 . Cells are then plated onto LB agar plates containing 5 ug ml tetracycline. Human AHF clones are identified using the colony hybridization procedure of Grunstein and Hogness PNAS U.S.A. 72 3961 1975 , the disclosure of which is incorporated herein by ref erence. The cDNA library is plated onto a nitrocellulose filter Schleicher and Schuell overlaying L broth 5 g ml tetracycline agar plates. Colonies are grown overnight at 370C and then the filter placed on sterile Whatman 3 M paper. A pre moistened nitrocellulose filter is then pressed against the master filter and the filters keyed using an 18 gauge needle.The replica filter is then grown on LD tetracyline plates at 370C until the colonies reached a diameter of 1 2 mn. The filters are then transferred to LB plates containing 150 g ml chloramphenicol and incubated at 370C for 16 24 hours. Filters are then removed and placed onto Whatman 3 M paper saturated with 0.5 M NaOH for 5 minutes at rocm temperature. Filters are then neutralized by placing onto Whatman 3 M saturated with 1 M Tris HCl pH 7.5 , 1.5 M Nail, and then batman 3 M saturated with 2x standard saline citrate SSC .SSC Ix is 0.15 M NaCl, .015 M sodium citrate. Filters are air dried and baked vacuo at 800C for 2 hours. Prehybridization of filters is done at 650C for 30 minutes in 10 n Tris HCl pH 8.0 , 1 mM EDTA, .1 Sfls followed by 30 minutes in 7x SSC, 5x Denhardt s lx Derhardt s is 0.02 polyvinylpyrollidine, 0.02 ficoll, 0.02 bovine serum albumin , 100 g ml denatured salmon sperm DNA, and 0.1 SDS. 32p labelled human exon DE, prepared as described above, are added to 106 cpm ml and the hybridization performed 12 16 hours at 370C.Filters are then washed in several changes of 7x SSC, .1 SDS for 1 2 hours at 370C. Filters are then air dried and subjected to autoradiography with Kodak zLAR film and a Dupont Lightning Plus intensifying screen. Those colonies showing a hybridization signal above background are grown in L broth containing 50 g ml tetracycline for rapid prep purification of plasmid DNA. Plasmid DNA is purified by the method of Holmes et al., Anal. Biochem., 114 193 1981 , the disclosure of which is incorporated herein by reference. An aliquot of this DNA is cleaved with restriction endonuclease Pst 1 and the fragments electrophoresed through 1 agarose TBE gels and blotted according to the procedure of E.Southern, J. Mol. Biol. 98 503 1975 , Methods Enzymol. 69 152 1980 , the disclosure of which is incorporated herein by reference. The nitrocellulose filters are hybridized with radiolabeled human AHF exon DNA as described for colony hybridization. Those plasmids which contained Pst I inserts hybridizing to the AHF exon DNA are used for tLTh sequencing analysis. For sequencing, plasmid DNA, purified fran 0.75 ml of culture by the procedure of Holmes, et al., supra , is digested to completion with the restriction endonuclease Sau 3aI. The resulting DNA segment, identified as 34 S1, has a DNA sequence which is depicted in Figure 4. The DNA is ethanol precipitated after extraction with phenolchloroform and redissolved in 10 lTE 10 mM Tris HCl pH 8.0, 1 mM EDTA . 5 l of the DNA solution is ligated with 20 ng of Bam H1 cleaved M13 mp 9 replicative formDNA in 100 ul of 50 mM Tris HCl pH 7.4 , 10 it MgCl2, 10 mM dithiothreitol, 1 mM ATP and an excess of T4 DNA ligase.Ligations are done at 150C for 2 4 hours. 5 l of the ligation reactions are used to transform 200 l of E. coli strain JM101 TGl as described above. Recombinants are identified as white plaques when grown on kagar plates containing X gal as an indication for beta galactosidase activity as described by Davies, et al., J. Mol. Biol. 36 413 1968 . Recombinant phage harboring sequences hybridizing to the human exon are identified by the procedure of Benton, et al.,Science 196 180 1977 , the disclosure of which is incorporated herein by reference, using radioactively labelled hunan AHF exon as a probe. Plaques showing a hybridization signal are picked and grown in 1.5 ml cultures of L4roth. Single stranded phageDNA prepared fran these cultures is used as template in oligonucleotide primed DNA synthesis reactions. Seuencing is done using the dideoey chain termination procedure, see, e.g.,Sanger et al., PNAS U.S.A. 74 5463 1977 . Human AHF recombinants are identified by comparing their nucleotide sequence with that which is known fran the human exon sequence of hunan AHF. 3. Porcine AHF mRNA Human AHF recornbinants are used to screen a porcine tissue library constructed exactly as described for the human tissue cDNA library. Prospective porcine AHF recombinant DNA clones are identified by the Grunstein Hogness procedure using the porcine 32p labeled exon fragment as a probe as described above. The probe is the porcine AHF exon segment labelled with 32P by nick translation as described by Rigby et al., J.Mol. Biol., 113 237 1977 , the disclosure of which is incorporated herein by reference. Colonies exhibiting hybridization signals are grown for rapid prep plasmid purification purposes as described above.Plaanid DNA is cleaved with the restriction endonuclease Pst 1, electrophoresed through 1 agarose TBE gels, and blotted according to the procedure of E. Southern 1975 . The blots are hybridized with nick translated porcine AHF recombinant DNA labelled with 32p. Full length clones may be constructed in a conventional manner, such as by ligation of DNA fragments fran overlapping clones at restriction enzyme sites coonon to both clones as is well knawn in the art gene walking . 4. Identifying full length clones from steps 2 or 3 The distance between the 5 end of an existing clone and the 5 end of the mRNA can be analyzed by the primer extension technique described in Agarwal et al., J.B.C. 256 2 1023 1028 1981 utilizing an oligonucleotide primer whose sequence canes fran the 5 amino terminal region of the existing AHF clone.If gels developed using this procedure show more than one transcript one should consider the most intense band as representing the full mES transcript. There are, however, many mRNAs which contain a long region of 5 untranslated sequence. Thus, expression of human DNa may not be contingent upon the acquisition of a complete cDNX clone. For example, a clone may be obtained which by DNA sequencing analysis demonstrates the existence of a methionine cordon followed by a sequence which is analogous to or identical to a known eucaryotic protein secretion signal and which is in frame with the remaining codons. Transformation and expression should be conducted for a clone which contains the met secretion signal sequence, which is of the expected size and which contains a poly T 3 terminus. 5. Eternative Procedure As an alternative to the methods in Sections 1 3 described above, it is preferred that the cDNA clones for porcine or human AHF be identified using a bacteriophage vector in the following manner. X fran human fetal liver tissue responsible for AHF synthesis was prepared by the guanidine hydrochloride method as described by Cox and modified by Chirgwin, et al. as set forth above in Example 5, Section 1. First strand X was synthesized from 10 ng of polyA fetal liver RNA by the procedure described in Example 6, Section 1, supra. Specifically, 10 sg of this RNA was denatured at roam tarperature in 10 l of 10 mM methylmercuryhydroxide. 140 mM 2 mercaptoethanol was added to inactivate the methylmercuryhydroxide.The RNA was then diluted to 50 l containg 140 mM KCl, 100 mM Tris HCl pH 8.3 at 42 C , 1 mM of each deoxynucleotide triphosphate, 200 g ml olio dT 12 18, 10 nM MgC12, and 0.1 Cl 32p dCTP ml. These reactions were performed at 420C for 1 hour after the addition of 3 1 of 17 units AMV reverse transcriptase Life Sciences . For the first strand synthesis of a primer extended library, 200 picomoles of a unique cooplinentary 38ner was included in the CH3HgOH denaturation step, and after 10 minutes at 23 C, the reaction was made 140mM betarmercaptoethanol, 0.7M KC1, 1mM ES, 20mM Tris HCl pH 8.3 at 420 , 1 unit ul of iNasin Biotec and incubated at 500C for 2 minutes and at 420C for 2 minutes. This was then diluted to 50 1 containing 140 mM KCl, 100 mM Tris HCl pH 8.3 at 420C , 1 mM of each deoxynucleotide triphosphate, 10 mM MgCl2, and 0.1 Ci 32p dCTP l. The reaction was incubated at 42 OC for 1 hour after the addition of 3 l ofAMV reverse transcriptase. After first strand synthesis the reactions were diluted to 150 l containing 10mM MgCl2, 50mM Tris pH 7.4, 5mM 2 mercaptoethanol, 7.5mM NH4SO4, 250 M of each deoxynucleotide triphosphate, and second strand synthesis initiated by the addition of 1 unit of RNase H E. coli and 45 units of DNA polymerse I. Reactions were incubated at 160C for 8 hours and then terminated by the addition of EDTA to 20mM and brought to a final volune of 200 l with H20. EcoRI methylation was then performed by addition of S adenosyl methionine to 50 M and 40 units of EcoRl rethylase. These reactions were incubated for 1 hour at 37 C, terminated by phenol chloroform extraction, and chromatographed using Sephadex G50 equilibrated in 10mM Tris HCl ph 8.0 , 1mM EDIA, .1M NaCl.The void volune was pooled and precipitated by ethanol addition. cDNA molecules were blunt ended in 200 l containing 5OmM Tris pH 8.3, 10mM MgCl2, 10mM 2 mercaptoethanol, 50mM NaCl, 50 M of each deoxynucleotide triphosphate, 100 g ml ovalbumin and 5 units of T4 polymerase. The reaction was incubated at 370 for 30 minutes and then terminated by phenol chioroform extraction.Nucleic acids were then precipitated by ethanol addition. EcoRl linkers Collabortive Research were then ligated to the blunted cDNA molecules under standard conditions, Manitis, et al., supra, at 243, in a total volume of 45ul. The reactions were terminated by the addition of EDTA to l5nM and extracted with phenol chloroform. Nucleic acids were precipitated by ethanol and pelleted by centrifugation. The linkered cDNt was redissolved in 200 l of 100 M Tris HCl pH 7.2 ,5mM MgCl2, 50mMNaCl and digested with 300 units of EcoRl for 2 hours at 370C. The reaction was then extracted with phenol chloroform and chromatographed over Seeharose CL 4B equilibrated with 10nM Tris pH 8.0 , 1mM EDDA, 0.1M NaCl. CDNx in the void volune was collected, precipitated by ethanol and pelleted by centrifugation. was was redissolved in 10mM Tris HCl pH 8.0 , lmM MEDIA and ligated to EcoRl cleaved, phosphatased lambda Charon 21A DNA at various ratios of cDNA to vector DNA using standard ligation conditions. Ligated DE was packaged and titered using established procedures, Manitis, et al., supra, at 64, 256. The library was plated and screened using 32P labelled human exon DNA under conditions described in Benton and Davis, aupra. Overlapping clones which spanned approximately 10,000 base pairs were obtained and a substantial portion thereof was sequenceed to reveal one long open reading frame encoding human AHF. The recombinent DE nucleotide sequence obtained thereof ran coding for human AHF is shown in Fig. 7 along with the deduced amino acid sequence for huaan AHF. The overlapping clones were assembled into vector pSP64 Pr nega Biotec using conventional techniques well known in the art, i.e. by ligation of DE fragments fran overlaping clones at resttriction enzyme sites anon to both clones.A pSP64 reccobinant clone containing the nucleotide sequence depicted in Figure 7, designated as pSP64 VIII, is on deposit at the American Type Culture Collection under AccessionNumber ATCC 39812. EXAMPLE 7 Expression of human or porcine AHF This example contemplates expression of AHF using the full length clone obtained by the method of Example 6 in a cotransformation system. Preparation of Transformation Vector A direct method for obtaining the AHF transformation vector is described below. The pCVSVL plasmid is partially digested with Pst I, the digest separated on gel electrophoresis.After visualization, the linear DNA fragment band corresponding to full length plasmid is isolated as pCVSVL Bl. The cDNA for AHF is rescued from the Example 5 cloning vectors by partial digest with Pst I. The partial digest is separated on gel electrophoresis, and the band corresponding to the molecular weight of the full length cDNA is isolated. pCVSV Bl is annealed with this DNA fragment, ligated with T4 ligase at 15 C, trasfected into E. coli strain HR101 and transformants selected for tetracycline resistance. The selected coli transformants are grown in the presence of tetracycline. Plasmid pCVSVL Bla is recovered in conventional fashion. Proper orientation of the cDNA in the plasnid may be determined in conventional fashion by assymetric digestion with an appropriate endonuclease s . 2. Cotransformation Selection and Amplification Plasmid pCVSVL Ala or pCVSVL Bla and pAdD26SVpA num 3 Kaufman et a., op. cit. are mixed together 50 g HP and .5 g pAdD265VpA 3 and precipitated by the addition of NaOAc pH 4.5 to .3M and 2.5 vols. of ethanol. Precipitated DNA is allowed to air dry, is resuspended in 2x HEBSS .5ml and mixed vigorously with .25 M CaCl2 .5in1 as described Kaufman et al., op. . me calcium phosphate DNa precipitate is allowed to sit 30 at room teperature and applied to CHO DUKX Bl cells Chasin andUrlaub, 1980, available fran Columbia University . The growth and maintenance of these cells has been described Kaufman et al., op. √ßit, Chasin and Urlaub 1980 . The DUKX Bl cells are subcultured at 5x105 10cm dish 24 hr. prior to transfection. After 30 minutes incubation at roan temperature, Smi of a media with 10 fetal calf serum is applied and the cells are incubated at 370 for 4.5 hr. The media is then removed fran the monolayer, 2ml of media with 10 fetal calf serum, 10 g ml of thymidine, adenosine, and deoxyadenosine, and penicillin and streptcguycin. Two days later the cells are subcultured 1 15 into media with 10 dialyzed fetal calf serum, and penicillin and streptarycin, but lacking nucleosides. Cells are then fed again with the same media after 4 5 days. Colonies appear 10 12 days after subculturing into selective media. Methotrexate MTX selection and detection of AHF gene and selection gene amplification are conducted in accordance with Axel et a., U.s. Patent 4,399,216, or Kaufman et al., op. cit. AHF yields may be it roved by cotransforming the permanent cell line EA.hy 926 Edgell et al., Proc. Natl. Acad. Schiz 80 3734 3737 1983 in place of DUKX Hl cells. In this case the cotransforming selection gene should be the dominant DHFR gene disclosed by Haber et al., Somatic Cell Genetics 4 449 508 1982 . Otherwise, the cotransformation and culture will be substantially as set forth elsewhere herein. 3. Production of NIP AHF producing CHO transformants selected in Section 2 are maintained in seed culture using standard techniques. The culture is scaled up to 10 liters by cell culture in conventional media. This medium need not contain and will not contain nucleosides so as to exclude selection gene revertants. The culture supernatant is monitored for clotting activity following standard assays for use with blood plasma samples reduction in clotting time of Factor VIII deficient plasma . The supernatant may be purified by conventional techniques such as polyethylene glycol and glycine precipitations as are known for use in purifying AHF fran blood plasma in order to increase the sensitivity of the FACTOR VIII assays. Fen FACTOR VIII activity has reached a peak the cells are separated from the culture medium by centrifugation. The Factor VIII is then recovered and purified by polyethylene glycol and glycine precipitations. Clotting activity is demonstrated inFactor VIII deficient plasma. 4. Altertutive Procedure for Productlon of AHF A full length cDNA containing the entire AElF coding region was constructed from the exon in HH 25 described above, two cX clones which overlapped the extrane 5 and 3 regions of that exon, and a third clone which overlapped the 3 cDNA clone and continued past the termination cordon. This was constructed such that synthetic Sal I sites were placed just 5 to the initiator methionine and at a position 3 to the translational stop signal at the end of the AHF coding sequences. This allowed the placement into and excision fran the polylinker Sal I site of pSP64.The Sal I fragment fran this clone pSP64 VIII was purified and ligated to pCVSVL2. pCVSVL2 is a plasmid which is identical to the expression vector pCVSVL Kaufman, et al., Mol. OellBiol., 2 1304 1982 the disclosure of which in incorporated herein by reference , except that it has deleted the Pst I site located 3 of the SV40 polyadenylation sequence which is accomplished by conventional procedures and that it oontains a duplication of the SV40 Ava II D fragment upstream fran the adenovirus major late promoter MLP . pCVSVL2 was derived fran pAdD26SVpA 1 See Kaufman et al., ja by adding XhoI linkers at each end of two SV40 Ava II D fragments, and inserting than into the XhoI site in pAdD26SVpA 1 .The Ava II D fragments are both inserted such that the SV40 late pranoter is in the same orientation as the Ad2 major late promoter. For plasmids pCVSVL and pAdD26SVpA 3 , see Kaufman et. al., supra. To insert the SalI fragment of pSP64 VIII into pCVSVL2, the unique Pst I site in pCVSVL2 was converted to a Sal I site by the procedure described in Rothetein et al., Methods Enzym., 69 98 1980 , and the Sal I fragment excised fran pSP64 VIII was inserted to yield pCVSVL2 VIII. pCVSVL2 VIII contains the correct 5 to 3 orientation which allows transcription of the AHF coding sequence from the adenovirus MLP of the vector. pCVSVL2 VIII is on deposit at the American Type Culture Collection under AccessionNumber 39813 The pCVSLV2 VIII was introduced into monkey C S 7 cells Mellon et al., 211 27 279 1981 using the DEAE dextran transfection protocol described by Sampsyrac and Danna, PNASU.S.A. 78 7575 1981 . The cells were cultured in serum free medium which was assayed for AHP activity 3 days after transfection. Factor VIII C activity was determined by the chranogenic substrate assay, described by Didisheim, Science 129 389 1959 , which confirmed that human Factor VIII C expression had been achieved at a level of about 0.05 units ml.