# REGULATORY REGION FOR HETEROLOGOUS GENE EXPRESSION IN YEAST

## Claims
Patentansprüche für folgende Vertragsstaaten BE, CH, DE, FR, GB, IT, LI, LU, NL, SE

## Description
This invention relates to the field of recombinant DNA biotechnology. In one of its aspects, the invention relates to DNA fragments which regulate the transcription of DNA into messenger RNA, and the initiation and termination of the translation of messenger RNA into protein. In another aspect, the invention relates to expression vectors which incorporate the above described DNA fragments. In yet another aspect, the invention relates to novel microorganisms transformed with the above described expression vectors. In a further aspect, the invention relates to the production of polypeptides. As recombinant DNA technology has developed in recent years, the controlled production by microorganisms of an enormous variety of useful polypeptides has become possible. Many eukaryotic polypeptides, such as for example human growth hormone, leukocyte interferons, human insulin and human proinsulin have already been produced by various microorganisms. The continued application of techniques already in hand is expected in the future to permit production by microorganisms of a variety of other useful polypeptide products. The basic techniques employed in the field of recombinant DNA technology are known by those of skill in the art. The elements desirably present in order for a host microorganism to be useful for the practice of recombinant DNA technology include, but are not limited to A basic element employed in recombinant DNA technology is the plasmid, which is extrachromosomal, double stranded DNA found in some microorganisms. Where plasmids have been found to naturally occur in microorganisms, they are often found to occur in multiple copies per cell. In addition to naturally occurring plasmids, a variety of man made plasmids, or hybrid vectors, have been prepared. Included in the information encoded in plasmid DNA is that required to reproduce the plasmid in daughter cells, i.e., an autonomously replicating sequence or an origin of replication. One or more phenotypic selection characteristics must also be included in the information encoded in the plasmid DNA. The phenotypic selection characteristics permit clones of the host cell containing the plasmid of interest to be recognized and selected by preferential growth of the cells in selective media. The utility of plasmids lies in the fact that they can be specifically cleaved by one or another restriction endonuclease or restriction enzyme, each of which recognizes a specific, unique site on the plasmid DNA. Thereafter, homologous genes, heterologous genes, i.e., genes derived from organisms other than the host, or gene fragments may be inserted into the plasmid by endwise joining of the cleaved plasmid and desired genetic material at the cleavage site or at reconstructed ends adjacent to the cleavage site. The resulting recombined DNA material can be referred to as a hybrid vector. DNA recombination is performed outside the host microorganism. The resulting hybrid vector can be introduced into the host microorganism by a process known as transformation. By growing the transformed microorganism, large quantities of the hybrid vector can be obtained. When the gene is properly inserted with reference to the portions of the plasmid which govern transcription and translation of the encoded DNA message, the resulting hybrid vector can be used to direct the production of the polypeptide sequence for which the inserted gene codes. The production of polypeptide in this fashion is referred to as gene expression. Gene expression is initiated in a DNA region known as the promoter region. In the transcription phase of expression, the DNA unwinds exposing it as a template for synthesis of messenger RNA. RNA polymerase binds to the promoter region and travels along the unwound DNA from its 3 end to its 5 end, transcribing the information contained in the coding strand into messenger RNA mRNA from the 5 end to the 3 end of the mRNA. The messenger RNA is, in turn, bound by ribosomes, where the mRNA is translated into the polypeptide chain. Each amino acid is encoded by a nucleotide triplet or codon within what may be referred to as the structural gene, i.e., that part of the gene which encodes the amino acid sequence of the expressed product. Since three nucleotides code for the production of each amino acid, it is possible for a nucleotide sequence to be read in three different ways. The specific reading frame which encodes the desired polypeptide product is referred to as the proper reading frame. After binding to the promoter, RNA polymerase first transcribes a 5 leader region of mRNA, then a translation initiation or start codon, followed by the nucleotide codons within the structural gene itself. In order to obtain the desired gene product, it is necessary for the initiation or start codon to correctly initiate the translation of messenger RNA by the ribosome in the proper reading frame. Finally, stop codons are transcribed at the end of the structural gene which cause any additional sequences of mRNA to remain untranslated into peptide by the ribosomes, even though additional sequences of mRNA had been formed by the interaction of RNA polymerase with the DNA template. Thus, stop codons determine the end of translation and therefore the end of further incorporation of amino acids into the polypeptide product. The polypeptide product can be obtained by lysing the host cell and recovering the product by appropriate purification from other microbial protein, or, in certain circumstances, by purification of the fermentation medium in which the host cells have been grown and into which the polypeptide product has been secreted. In practice, the use of recombinant DNA technology can create microorganisms capable of expressing entirely heterologous polypeptides, i.e., polypeptides not ordinarily found in, or produced by, a given microorganism so called direct expression. Alternatively, there may be expressed a fusion protein, i.e., a heterologous polypeptide fused to a portion of the amino acid sequence of a homologous polypeptide, i.e., polypeptides found in, or produced by, the wild type non transformed host microorganism so called indirect expression. With indirect expression, the initially obtained fusion protein product is sometimes rendered inactive for its intended use until the fused homologous heterologous polypeptide is cleaved in an extracellular environment. Thus, for example, cyanogen bromide cleavage of methionine residues has yielded somatostatin, thymosin alpha 1 and the component A and B chains of human insulin from fused homologous heterologous polypeptides, while enzymatic cleavage of defined residues has yielded beta endorphin. Up to now, commercial efforts employing recombinant DNA technology for producing various polypeptides have centered on The availability of means for the production of polypeptide products in eukaryotic systems, e.g., yeast, could provide significant advantages relative to the use of prokaryotic systems such as The development of poorly characterized yeast species as host vector systems is severely hampered by the lack of knowledge about transformation conditions and suitable vectors. In addition, auxotrophic mutations are often not available, precluding a direct selection for transformants by auxotrophic complementation. If recombinant DNA technology is to fully sustain its promise, new host vector systems must be devised which facilitate the manipulation of DNA as well as optimize expression of inserted DNA sequences so that the desired polypeptide products can be prepared under controlled conditions and in high yield. An object of our invention is therefore a novel regulatory region responsive to the presence of methanol. A further object of the invention is a novel catabolite sensitive regulatory region which is responsive to the presence of some carbon sources but which is not responsive to the presence of other carbon sources. Another object of the invention is a novel regulatory region responsive to carbon source starvation. Yet another object of our invention is novel vectors capable of expressing an inserted polypeptide coding sequence. Still another object of our invention is novel yeast strain of the genus A further object of our invention is a process for producing polypeptides employing the novel yeast strain as described hereinabove. These and other objects of our invention will become apparent from the disclosure and claims herein provided. In accordance with the present invention, we have discovered, isolated and characterized DNA sequences which control the transcription of DNA into messenger RNA and translation of the messenger RNA to give a polypeptide product. The novel DNA sequences of this invention are useful for the production of polypeptide products by a yeast strains which are capable of growth on methanol as a carbon and energy source, b yeast strains which are capable of growth on glucose, ethanol, fructose and the like and c yeast strains which are capable of growth on glycerol, galactose, acetate and the like. Figure 1 is a correlation of the relationship between the genomic clone pPG 6.0 and cDNA clone pPC 15.0 for protein p76. Figure 2 is a correlation of the relationship between the genomic clone pPG 4.0 and cDNA clones pPC 8.3 and pPC 8.0 for protein p72 alcohol oxidase . Figure 3 is a correlation of the relationship between the genomic clone pPG 4.8 and cDNA clone pPC 6.7 for protein p40. Figure 4 provides restriction maps of regulatory regions of the invention from clone pPG 6.0. Figure 5 is a restriction map of the regulatory region of the invention from clone pPG 4.0. Figure 6 is a restriction map of the regulatory region of the invention from clone pPG 4.8. Figure 7 is a restriction map of a sequence of DNA obtained from the 3 end of the p76 structural gene. Figure 8 provides restriction maps of sequences of DNA obtained from the 3 end of the p72 alcohol oxidase structural gene. Figure 9 is a restriction map of a sequence of DNA obtained from the 3 end of the p40 structural gene. Figure 10 is a restriction map of the protein p76 structural gene and the 5 regulatory region therefor. Figure 11 is a restriction map of the protein p40 structural gene and the 5 regulatory region therefor. Figure 12 is a restriction map of the protein p76 cDNA. Figure 13 is a restriction map of the protein p72 alcohol oxidase cDNA. Figure 14 is a restriction map of the protein p40 cDNA. Figure 15 provides restriction maps of two novel p76 regulatory region Figure 16 is a restriction map of a novel p72 alcohol oxidase regulatory region Figure 17 is a restriction map of plasmid pSAOH 1. Figure 18 is a restriction map of plasmid pSAOH 5. Figure 19 is a restriction map of plasmid pSAOH 10. Figure 20 is a restriction map of plasmid pTAFH.85. Figure 21 is a restriction map of plasmid pT76H 1. Figure 22 is a restriction map of plasmid pT76H 2. Figure 22a is a restriction map of plasmid pT76H3. Figure 22b is a restriction map of plasmid pT76H4. Figure 23 is a restriction map of plasmid pYA2. Figure 24 is a restriction map of plasmid pYA4. Figure 25 is a restriction map of plasmid pYJ8. Figure 26 is a restriction map of plasmid pYJ8Δ Figure 27 is a restriction map of plasmid pYJ30. Figure 28 provides a restriction map of plasmid pTAFH 1 and shows how the plasmid was derived. Figure 29 provides a restriction map of plasmid pTAO 12 and shows how the plasmid was derived. Figure 30 is a restriction map of plasmid pTAO13. Figure 30a is a restriction map of plasmid pT76U1. Figure 31 provides a restriction map of plasmid pTAO1 and shows how the plasmid was derived. Figure 32 provides a restriction map of plasmid pTAF.85 and shows how the plasmid was derived. Figure 33 provides a restriction map of plasmid YEp13. Figure 34 is a restriction map of pBPf1. The following abbreviations are used throughout this application to represent the restriction enzymes employed In accordance with the present invention, there is provided a novel DNA fragment comprising a regulatory region responsive to at least one of the following conditions the presence of methanol, carbon source starvation when cells are grown on some substrates other than methanol, and the presence of non catabolite repressing carbon sources other than methanol. The regulatory region of the DNA fragment of this invention is capable of controlling the transcription of messenger RNA when positioned at the 5 end of the DNA which codes for the production of messenger RNA. Also included within the scope of our invention are mutants of the above described DNA fragment. Further in accordance with the present invention, there is provided a DNA fragment which comprises a regulatory region which is capable of controlling the polyadenylation, termination of transcription and termination of translation of messenger RNA when positioned at the 3 end of the polypeptide coding region which codes for the production of messenger RNA, wherein the transcription and translation of the messenger RNA is controlled by a regulatory region which is responsive to at least one of the following conditions the presence of methanol, carbon source starvation when cells are grown on some substrates other than methanol and the presence of non catabolite repressing carbon sources other than methanol. Also included within the scope of our invention are mutants of the above described DNA fragment. Still further in accordance with a specific embodiment of the invention, there are provided DNA fragments which direct the incorporation of encoded polypeptide into peroxisomes. Peroxisomes are intracellular bodies present in large amounts in methanol grown yeast cells. These intracellular bodies serve to isolate the incorporated polypeptide product from intracellular fluids and enzymes such as proteases. In accordance with another embodiment of the invention, genes coding for the production of alcohol oxidase, a protein of about 40 kilodaltons and a protein of about 76 kilodaltons are provided. In accordance with yet another embodiment of the present invention, plasmids and transformed organisms containing the above described DNA fragments are provided. In accordance with still another embodiment of the invention, methods are provided for producing the plasmids and DNA fragments of the invention, as well as heterologous polypeptides, i.e., polypeptides not native to the host organisms. An approximately 20,000 member cDNA library was prepared in In an attempt to obtain cDNA clones of longer length, a second cDNA library was prepared using milder S1 nuclease digestion conditions than used for the preparation of the first cDNA library and the members of this new library screened individually with ³²P labeled cDNA clones pPC 6.4, pPC 8.0, and pPC 15.0. As a result, larger cDNA clones were isolated corresponding to cDNA clones pPC 6.4 and pPC 8.0. The larger clones, pPC 6.7 and pPC 8.3, were found to contain inserts measuring 1200 and 2100 nucleotides, respectively See Figures 2 and 3 . A cDNA clone possessing an insert larger than the 1100 nucleotides for pPC 15.0 has not been observed after screening more than 40,000 cDNA clones. The isolation of the genomic DNA fragments corresponding to each of these cDNA clones was accomplished by first cutting out and electroeluting from agarose gels The relationship of each cDNA clone to its corresponding genomic clone is illustrated in Figures 1, 2, and 3. pPC 15.0 is encoded within a 6000 nucleotide The cDNA clone pPC 8.3 is included within the genomic clone pPG 4.0 Figure 2 . pPG 4.0 contains an The cDNA clone pPC 6.7 is located entirely within a 4800 nucleotide In all of these above described genomic clones, there are at least 1.2 kilobase pairs of flanking genomic DNA sequence which are 5 to the structural genes copied in each of the cDNA clones. Each of the genomic and cDNA clones described above have been deposited with the Northern Regional Research Center of the United States of America, Peoria, Illinois. All clones have been deposited in All of the above organisms have been irrevocably deposited and made available to the public as of August 31, 1984. Each of the cDNA clones described above have been labeled and employed as probes of chromosomal DNA sequences from a number of methanol assimilating yeasts and a methanol non assimilating yeast. Homologous genes for all three of the cDNAs were found to exist in essentially all methanol assimilating yeasts, but were clearly not present in methanol non assimilating yeast The influence of methanol on the expression of each of these cloned genes can be observed by studying the effects on transcription of these genes. Isolated poly A RNA from To determine what protein products were encoded by each of the above identified cDNA clones, poly A RNA from A sample highly enriched for alcohol oxidase protein was prepared by dialysis of cleared cell lysates against H₂O See Example VII . The crystalline precipitate resulting from this dialysis was shown by SDS electrophoresis to contain predominantly two polypeptides of 76,000 and 72,000 daltons, respectively. The precipitate was subjected to additional purification by chromotography through Sephacryl 200 See Example VII , which demonstrated that alcohol oxidase activity corresponded to the activity of the purified 72,000 dalton polypeptide. The size of this polypeptide was identical to that of the protein product selected by cDNA clone pPC 8.3 See Example X . Additional support that clones pPC 8.3 and pPG 4.0 encode the alcohol oxidase structural gene was obtained by means of an immunological approach Example XI . The protein preparation isolated from To further verify that clone pPC 8.3 is in fact the cDNA clone encoding In parallel, the nucleotide sequence of the 5 end of the gene encoded in pPC 8.3 and pPG 4.0 was determined. The predicted amino acid sequence for amino acids 2 19 See Sequence B derived from the DNA sequences of both the genomic and cDNA clones agreed perfectly with the first 18 amino acids of the above determined amino acid sequence Sequence A for isolated In addition, the entire nucleotide sequence for the coding region of the alcohol oxidase gene was determined. The nucleotide sequence determined and the predicted amino acid sequence are set forth in Sequence B and are believed to be A comparison of the above nucleotide sequence with the published Ledeboer The nucleotide sequence for the first 51 nucleotides of the p76 gene was determined by standard techniques. From this sequence, the amino acid sequence for the amino terminal end of the p76 protein can be predicted This predicted amino acid sequence for p76 can be compared with the published amino acid sequence for the dihydroxyacetone synthase DHAS protein from Based on the significant degree of homology and the similar protein size about 76,000 daltons of As above with the alcohol oxidase gene, a comparison of the nucleotide sequence for the first 51 nucleotides of the The 5 regulatory regions of the invention are detailed in restriction maps presented in Figures 4, 5 and 6. The 5 regulatory region which controls the expression of polypeptide p76 is located within the DNA fragment depicted in Figure 4a. The approximately 2.9 kilobase pair The 5 regulatory region which controls the expression of polypeptide p72 alcohol oxidase is located within the approximately 2000 base pair Figure 6 provides a restriction map for the approximately 3 kilobase pair Figures 10, 2a and 11 provide restriction enzyme data for the regulatory regions plus structural genes for polypeptides p76, p72 alcohol oxidase and p40, respectively. Hence, Figure 10 provides detail for the 3.8 kilobase pair The genomic clones, pPG 6.0, pPG 4.0 and pPG 4.8 have also been characterized by restriction mapping. Thus, clone pPG 6.0 is detailed in Figure 1a. As a point of reference the 5 end of the DNA fragment is deemed the origin. Clone pPG 6.0 is a Clone pPG 4.0 is illustrated in detail in Figure 2a. The clone is an Clone pPG 4.8 is illustrated in detail in Figure 3a. The clone is a 4.8 kilobase pair The genomic clones pPG 6.0, pPG 4.0 and pPG 4.8 were manipulated by insertion into unique restriction sites on the Figures 7, 8 and 9 provide restriction map data for the 3 regulatory regions of the polypeptides p76, p72 alcohol oxidase and p40, respectively. The 3 regulatory regions are useful in controlling the polyadenylation, termination of transcription and termination of translation of messenger RNA which is coded for by preceding nucleotide sequences. Thus, the 3 regulatory region from the polypeptide p76 gene, a 2.7 kilobase pair Since the alcohol oxidase gene in pPG4.0 terminates within a few hundred base pairs of the AO gene transcription termination site, the additional 3 sequence detailed in Figure 8c was obtained as follows. The first step was to digest The 3 AO gene fragment was then cloned by recovering The cDNA clones for the regulatable genes from In Figure 13, a composite restriction map for p72 alcohol oxidase cDNA, constructed by overlap of clones pPC 8.3 and pPC 8.0, is presented. As above, the 5 end of the DNA sequence is referred to as the origin. Thus, treating alcohol oxidase cDNA with a variety of restriction enzymes gives the following size fragments Restriction enzyme mapping of the 3 end of the alcohol oxidase gene in clones pPC 8.0 and pPC 8.3 revealed that cDNA clone pPC 8.3 is missing approximately 250 nucleotides of the alcohol oxidase mRNA sequence Figure 2 . The sequences present at the 3 end of the alcohol oxidase mRNA are present in cDNA clone pPC 8.0 which overlaps pPC 8.3 by approximately 500 nucleotides. Figure 14 presents a restriction map for the cDNA of polypeptide p40, a 1.2 kilobase pair fragment. Referring to the 5 end of the cDNA clone as the origin, clone pPC 6.7 is cleaved by Each of the cDNA fragments have been cloned into pBR322, which is then transformed into Each of the above described cDNA clones are useful as probes for the identification and isolation of chromosomal DNA encoding the production of polypeptides unique to the growth of yeast on methanol as a carbon and energy source. Hence as already described, these clones were used to identify The 5 regulatory region of clone pPG 4.0 was further characterized by determining the nucleotide sequence of the clone upstream 5 of the point where the structural information for p72 alcohol oxidase is encoded. The first 250 nucleotides prior to the mRNA translation start site ATG codon are believed to be The promoter function of clone pPG 4.0 is believed to be contained within this sequence of nucleotide bases. In order to more fully describe this novel DNA fragment, an additional 301 nucleotides further upstream of the sequence detailed in Sequence C above have been determined. Thus, the first 551 nucleotides prior to the mRNA translation start site are believed to be The additional nucleotides contained in Sequence D compared to Sequence C are believed to impart, by an unknown mechanism, additional regulatory functions to the promoter region contained within Sequence C. It should be recognized that Sequence D represents only partial DNA sequencing for the 1.1 kbp DNA fragment shown in Examples XIV and XV to be capable of controlling gene expression in yeast. It may be that additional control functions are encoded in the portion of the 1.1 kbp DNA fragment not detailed in Sequence D. In order to further describe this novel 1.1 kbp DNA fragment, additional nucleotide sequencing was carried out to fully delineate the nucleotide sequence of the entire 1.1 kbp DNA fragment shown in Examples XIV and XV to be capable of controlling gene expression in yeast. The nucleotide sequence is set forth as Sequence D It is recognized by those of skill in the art that additional control functions, relative to Sequences C and D, may be encoded in that portion of sequence D which is further upstream i.e., in the 5 direction of the nucleotide sequence detailed in sequences C and D. To determine where RNA transcription for the alcohol oxidase gene is initiated, the DNA sequences around the 5 end of this gene from the genomic clone pPG 4.0 and the cDNA clone pPC 8.3 were compared. cDNA clone pPC 8.3 contains about 100 nucleotides of an untranslated region 5 to the alcohol oxidase gene. Based upon this sequence, an oligonucleotide of 15 bases 5 CTTCTCAAGTTGTCG 3 complementary with respect to nucleotides 29 to 43, where the A of the translation start site ATG codon is designated as 1 and the G in the 5 direction is designated as 1, was synthesized See Example IX and used as a primer to extend along the alcohol oxidase mRNA to reach the 5 end. The sequence of cDNA obtained from this primer extension experiment revealed three different transcriptional initiation points for The 55 nucleotides preceding the start of alcohol oxidase mRNA contain a putative Goldberg Hogness box TATAA box . The sequence TATAAA occurs at position 40 from the 5 end of the predominant transcript for alcohol oxidase mRNA and therefore 165 nucleotides upstream from the initiation codon for this protein. The 3 regulatory region of the alcohol oxidase gene was further characterized by determining the nucleotide sequence for about 120 nucleotides downstream of the point where the structural information for p72 alcohol oxidase is encoded. The sequence is set forth below as Sequence D The 5 regulatory region of the clone pPG 6.0 was also further characterized by determining the nucleotide sequence of the clone upstream 5 of the point where the structural information for p76 is encoded. The first 622 nucleotides prior to the mRNA translation start site ATG codon are believed to be The promoter function of clone pPG 6.0 is believed to be contained within this sequence of nucleotide bases, although those of skill in the art recognize that additional regulatory properties may be imparted by sequences further upstream than the sequences presented as Sequence D . The 3 regulatory region of clone pPG 6.0 was further characterized by determining the nucleotide sequence for about 180 nucleotides downstream 3 of the point where the p76 structural information is encoded. The sequence is set forth below as Sequence D The above described plasmids of the present invention have utility in yeast strains which can be transformed. Regulation of gene expression in yeast by the novel DNA fragments of the present invention can be accomplished by subjecting the transformed organisms to carbon source starvation. Carbon source starvation after growth on a variety of both catabolite repressing and non catabolite repressing carbon sources induces expression of the gene product maintained under the control of the regulatory regions of the invention. Another means to achieve expression of the desired gene product in appropriate species of transformed yeast is to grow transformed yeasts on methanol. Yet another means to induce expression of the desired gene product is to grow transformed yeast on media containing non catabolite repressing carbon sources. The regulatory regions of this invention are useful for expression in all yeast strains, since the regulatory regions have been shown to be induced under a variety of conditions. Thus, yeasts capable of growth on methanol or on non catabolite repressing carbon sources can be caused to produce foreign, i.e., heterologous, polypeptides directly while yeasts capable of growth on catabolite repressing carbon sources can be caused to produce foreign polypeptides by subjecting yeast cells so grown to conditions of carbon source starvation. Transformed yeast strains which are preferred in the process of the present invention include members of the genera Especially preferred yeast strains for use in one embodiment of the process of the present invention are those yeast strains which are capable of growth on methanol as carbon and energy source. Yeasts known to be capable of growth on methanol include members of the genera Since the regulatory regions of the present invention are also induced by growth on non catabolite repressing carbon sources as well as conditions of carbon source starvation, yeast strains which are capable of growth on such non methanolic substrates as In addition, since the regulatory regions of the invention are responsive to a variety of growth conditions, both in terms of induction and repression of expression, the regulated expression of a gene product under the control of the regulatory regions of the invention can be achieved. Thus, for example, cells can be grown on a carbon source which induces only low levels of foreign gene expression, then switched to methanol which will strongly induce gene expression. Alternatively, regulated gene expression can be achieved by employing mixtures of inducing repressing feeds such as, for example, methanol glucose mixtures. As yet another alternative, high expression levels produced by growth on methanol can be reduced as desired by addition to the growth media of a repressing carbon source such as glucose or ethanol. Of course, those of skill in the art recognize that other variations of feed mixtures and order of feed introduction are possible, and afford a great deal of control over the level of gene expression obtained from the invention regulatory regions. An especially preferred host yeast strain is the mutant Since the regulatory regions of the present invention have also been demonstrated to be useful for the regulated expression of heterologous gene products in yeast strains of the genus It is recognized by those of skill in the art that useful host strains are not limited to auxotrophic mutants. Thus, transformation of prototrophic strains with positive selection markers, such as, for example, antibiotic resistance genes, also provides a useful means for the detection and isolation of transformed strains. The transformation of GS115 NRRL Y 15851 can be transformed by enzymatic digestion of the cell walls to give spheroplasts the spheroplasts are then mixed with the transforming DNA and incubated in the presence of calcium ions and polyethylene glycol, then regenerated in selective growth medium deficient in histidine. The transforming DNA includes the HIS4 gene in which the host strain is deficient, thus only transformed cells survive on the selective growth medium employed. The HIS4 gene was isolated from the strain The ARG4 gene was isolated from Those of skill in the art recognize that other marker genes from Another useful component of the vectors of the present invention are To search for As a measure for the ability of the ARSs to maintain plasmids as autonomous elements in One of the putative To determine if plasmids pSA0H5 see Figure 18 and pT76H4 See Figure 22b , which contain PARS1 contributed by pYJ30 and pBPf1, respectively, display similar stability to the plasmids from which they were derived, cells containing these vectors were grown under selective conditions for about 50 generations under selective conditions His in the presence of glucose. The cells were then shifted to non selective conditions His and the loss of prototrophy was monitored. The stability of these plasmids was comparable to the stability of pYJ30, including the rapid loss of His protrotrophy upon shift to non selective media. Thus, it is believed that experiments carried out with plasmids containing the autonomous replication sequence, PARS1, provide results of gene expression from autonomous plasmid DNA. In order to demonstrate the ability of the regulatory regions of the present invention to control the production of protein products, novel DNA constructs were prepared. Thus the Although the introduction of the regulatory region β galactosidase gene fusions of the invention into host yeast cells is described herein employing plasmids as the vehicle for introduction, those of skill in the art recognize that it is not necessary for the regulatory region structural gene construct to be introduced into the cell via a plasmid. Hence, any molecule capable of being maintained in yeast can be employed. Therefore, the regulatory region structural gene constructs of the invention can be manipulated via vectors other than plasmids. Alternatively, the regulatory region structural gene construct can be integrated into the chromosome of the host yeast cell. Those of skill in the art also recognize that the scope of the present invention is not limited to the production of β galactosidase under the regulation of the regulatory regions disclosed herein. The variety of polypeptides which can be produced under the regulation of the regulatory regions of the invention is limited only by the imagination of the reader. Many procedures exist for the preparation of DNA sequences which code for desired polypeptides. For example, oligonucleotides of various lengths can be synthesized by known procedures. Several such oligonucleotides can be assembled, in consequence of the specific base pairing properties thereof, into longer, double stranded molecules. The component oligonucleotides of this double stranded molecule can be joined ligated by the enzyme DNA ligase. Alternatively, DNA molecules having the desired coding sequence can be synthesized by use of the enzyme reverse transcriptase, using messenger RNA related to the desired polypeptide as a template for the action of reverse transcriptase. Yet another possibility is the cloning of genomic DNA fragments and observing whether direct expression of the desired product occurs. The DNA sequence which codes for the desired polypeptide can be modified for preparation of the regulatory region structural gene construct by a variety of procedures. For example, the ends of the DNA prepared as described above can be ligated with the enzyme DNA ligase to short double stranded DNA molecules which contain the nucleotide sequence recognized by specific restriction endonucleases, so called linker molecules. Digestion of these molecules with a specific restriction endonuclease following the ligation will generate termini corresponding to the specified restriction endonuclease recognition site at the ends of the prepared DNA sequence. Three specific regulatory region β galactosidase gene constructs prepared in the course of this work are described in terms of restriction mapping data presented in Figures 15 and 16. The restriction map presented in Figure 15a describes a construct comprising a 0.85 kilobase pair Figure 16 is a restriction map of a construct comprising a 1.1 kilobase pair Plasmid pSAOH1 is illustrated schematically in Figure 17. In addition to containing the regulatory region β galactosidase gene fusion detailed in Figure 16, the plasmid is shown to contain Plasmid pSAOH5 is illustrated schematically in Figure 18. The plasmid is similar to pSAOH1 described above, except the Plasmid pSAOH10 is illustrated schematically in Figure 19. The plasmid contains Plasmids pTAFH.85, pT76H1 and pT76H2 are analogous to the above three described plasmids, except the regulatory region β galactosidase gene fusion employed was that described in Figure 15a instead of the fusion described in Figure 16 . Plasmids pT76H3 and pT76H4 are analogous to pSA0H1 and pSA0H5, respectively, except the regulatory region β galactosidase gene fusion employed was that described in Figure 15b instead of the fusion described in Figure 16 . Plasmid pTAFH.85 is illustrated schematically in Figure 20 and comprises Plasmid pT76H1 is illustrated schematically in Figure 21 and comprises Plasmid pT76H2 is illustrated schematically in Figure 22 and comprises Plasmid pT76H3 is illustrated schematically in Figure 22a and comprises Plasmid pT76H4 is illustrated schematically in Figure 22b and comprises The novel β galactosidase gene containing constructs were also used to transform It was found that cells grown on glucose as carbon source produced no detectable levels of β galactosidase or alcohol oxidase, while cells grown on methanol as sole carbon source expressed significant levels of both alcohol oxidase and β galactosidase. It was also found that the glucose grown cells, when subjected to conditions of carbon source starvation, also expressed measurable quantities of alcohol oxidase as well as β galactosidase. Thus, it is clear that the regulatory regions of the invention are responsive to both the presence of methanol as well as conditions of carbon source starvation. As verification that the regulatory regions of the invention are responsive to growth on non catabolite repressing carbon sources as well as conditions of carbon source starvation, a plasmid containing the alcohol oxidase regulatory region, pTA013 and a plasmid containing the p76 regulatory region, pT76U1, was used to transform a non methanol utilizing strain of yeast, The buffers and solutions employed in the following examples have the compositions given below Unless otherwise specified, the above solutions represent the basic 1x concentration employed. Throughout the examples, where the different concentration levels are employed, that fact is indicated by referring to the solution as a multiple of the basic 1x concentration. The following abbreviations are used throughout the examples, with the following meaning Several procedures carried out on a routine basis follow a standard protocol which will be detailed here. Centrifugation is carried out for a period of time and at a spin rate sufficient to provide a clear supernatant. Generally, centrifugation of yeast cells is carried out at at least 1500 g for at least 5 minutes. Nucleic acid extractions with phenol chloroform isoamyl alcohol involve contacting the nucleic acid containing solution with an equal volume of a 50 48 2 ratio by volume mixture of phenol, chloroform and isoamyl alcohol, respectively. Extractions with chloroform isoamyl alcohol involve contacting the solution to be treated with an equal volume of 48 2 ratio by volume mixture of chloroform and isoamyl alcohol. When gels, filters, etc. are described as being washed or soaked in a specified solution, the entire gel, filter, or the like was immersed in an appropriate vessel pan, dish, vial, etc. in order to contact the entire surface of the gel, filter, or the like with the solution of interest. Ethanol precipitation of nucleic acids involves first adjusting the salt content of the nucleic acid containing solution, then contacting the solution with two volumes of cold ethanol. Ten grams of fermenter grown Total cell RNA was prepared by pulverizing the frozen pellet prepared as described in Example I with a mortar and pestle and further disrupting the frozen pellet for about 2 5 minutes in a Waring Blender in the presence of liquid nitrogen. The pulverized pellet was added to PK buffer at a concentration of 7.5 mL per gram of cells. Proteinase K Boehringer Mannheim was added to the resuspended pellet to give a final concentration of 400 µg mL, and the suspension was incubated at room temperature for 10 minutes. This mixture was extracted with phenol chloroform isoamyl alcohol followed by a chloroform isoamyl alcohol extraction. Nucleic acids were precipitated by adjusting the solution to be 0.25 The nucleic acids were redissolved in a minimum volume of TE buffer. The RNA present in this solution was enriched either by centrifugation through a 4 mL CsCl cushion 1 g CsCl mL, 1 m Total RNA 5 to 10 mg was resuspended in ETS buffer at a concentration not greater than about 10 mg mL, placed in a 65 C water bath for 2 minutes and then placed immediately on ice. The RNA solution was then allowed to warm to room temperature and a stock solution of 5 Complementary DNA cDNA clones were synthesized as follows. Ten µg of poly A RNA prepared as described in Example II was resuspended in 7 µL H₂O and brought to a final concentration of 2.7 m The aqueous single stranded cDNA solution was adjusted to 2.5 After the final ethanol precipitation and the collection of the double stranded cDNA by centrifugation, the pellet was resuspended in 20.25 µL of water, then brought to a final volume of 50 µL with 50 m The double stranded cDNA pellet was resuspended in 42 µL of H₂O and the solution brought to a final volume of 47 µL by the addition of 5 µL of a stock solution containing 2.8 The double stranded, d C tailed cDNA was either reannealed directly to poly d G tailed pBR322 opened at the The annealed cDNA library was transformed into competent Replica filters were prepared by stamping nitrocellulose filters onto an original filter used to lift the colonies off of the plate. These replica filters were incubated on 2x LB Tet 15 µg mL of tetracycline plates. The colonies on the filters were prepared for probing by transferring the filters to 2x LB Tet plates containing 200 µg mL of chloramphenicol, incubating the filters at 37 C for at least 12 hours, then lysing the colonies by floating the filters on an aqueous pool which is 1.5 The vacuum dried nitrocellulose filters containing the cDNA library prepared as described in the previous example were prehybridized at 42 C. for 5 hours in prehybridization buffer. The filters were removed from the prehybridization buffer and lightly rubbed with a gloved hand in 5x SSPE in order to remove cell debris. The filters were placed in hybridization buffer same as prehybridization buffer except 1x Denhardt s . Either ³²P labelled single strand cDNA 10⁶ cpm mL or end labeled poly A RNA was hybridized to the filters for 17 hours at 42 C. After hybridization, the filters were washed briefly in 2x SSPE at 22 C, followed by two washes at 65 C in 0.1x SSPE, 10 minutes each. End labeling of poly A mRNA was performed by adding 2 µg of poly A mRNA to a volume of 50 µL containing 50m Two to five µg of poly A mRNA were heated at 65 C for 5 minutes in 10 m Ten µg of 200 ng of the cDNA clones were nick translated for 90 minutes at 14 C in 30 µL of a solution containing 50 m Genomic fragments which hybridized to the cDNA probes were isolated by digesting 200 µg of The electroeluted fragments were resuspended in water and 200 ng fragments were ligated to 500 ng of pBR322 which was cleaved at the appropriate restriction site and dephosphorylated when necessary. The ligation reaction was carried out in 300 µL of 66 m The plasmid bearing LE392 The plasmid band was visualized by UV fluorescence and drawn from the tube using a needle and syringe. The plasmid solution was n butanol extracted four times and ethanol precipitated at 20 C. The ethanol precipitation was repeated at least twice to remove all of the CsCl. The plasmid was stored at 20 C as an ethanol precipitate. Protein samples from Six 130 mL portions of the cell free supernatant were placed in cellulose acetate dialysis bags and dialyzed at 5 C. against about 8 liters of distilled water. After 4 days, the aqueous phase of each bag was decanted. The solids remaining in the bags consisted of two types of solid. The thin upper white layer was carefully removed and discarded. The bottom solid was brown yellow and was crystalline alcohol oxidase. A portion of the crystalline alcohol oxidase was dissolved in distilled water about 10 times the volume of the solid and an assay by the dye peroxidase method showed an activity of 94 EU mL. The specific activity of the alcohol oxidase was 10.4 EU mg of protein. The crystalline precipitate resulting from the above described dialysis was dialyzed against 0.05 The alcohol oxidase activity for reaction with methanol was determined by the following assay procedure dye peroxidase method . A dye buffer mixture was prepared by mixing 0.1 mL of an o dianisidine solution 1 weight o dianisidine in water with 12 mL of aerated 0.1 wherein 11.5 is a factor based on a standard curve prepared with known aliquots of H₂O₂ and ΔA is the change in absorbance during the experimental interval. A total of 0.1 µg of total protein from each fraction was also assayed for alcohol oxidase content by gel electrophoresis with SDS polyacrylamide 12 . Determination of DNA sequences was performed by the dideoxy chain elongation method using bacteriophage M13 Sanger et al, 1980 or by the chemical modification method Maxam and Gilbert, 1980 . The DNA fragments corresponding to the 5 end of the alcohol oxidase gene were inserted into the M13mp8 and M13mp9 vectors or end labelled for the chemical modification method using restriction enzyme sites available in this region. The 710bp The four base modification reactions were performed as follows a the G guanine reaction was incubated for 8 minutes at 22 C and contained 1 µL 50,000 cpm of the labelled DNA fragment, 4 µL of water, 200 µL of 50 m The DNA pellets were collected by centrifugation, resuspended in 250 µL of 0.3 The amino acid sequence of alcohol oxidase was determined by Sequemat, Inc. Watertown, Mass. using 2 mg of purified alcohol oxidase from To determine where the start of the mRNA for alcohol oxidase was located, a primer extension experiment was performed using a synthetic oligonucleotide copied from the DNA sequences of the 5 end of the alcohol oxidase gene as primer and 10 µg of poly A The gradient gel was prepared as follows 300 µL of 10 ammonium persulphate and 14 µL of TEMED were added to 30 mL of top gel 75 µL of 10 ammonium persulfate and 3.5 µL TEMED were added to 7 mL of bottom gel, 6 mL of top gel were drawn up into a pipet and then 6 mL of bottom gel were drawn into the same pipet. The gel was poured between the gel plates followed by 22 mL of top gel. Positive hybridization translation experiments were performed by linearizing twenty µg of cloned Prior to prehybridization, the filters with the bound DNA were placed in 1 mL of sterile water and heated for one minute at 100 C, cooled on ice, rinsed by vortexing in 1 mL of sterile water and rinsed with 1 mL of prehybridization buffer. The filters were prehybridized in 1 mL of prehybridization buffer, then 40 µg 2µg mL ETS of poly A mRNA was added directly to the prehybridization buffer. The hybridization mixture was heated at 65 C for 10 minutes and then incubated at 42 C for 24 hours. Following the hybridization, filters were washed briefly 2 times in 1x SSPE which contained 0.5 SDS at 22 C, 7 times in 1x SSPE which contained 0.5 SDS at 50 C for 5 minutes each, 3 times in 0.1x SSPE at 50 C for 5 minutes each, and once in 0.1x SSPE at 65 C for 10 minutes. The RNA was eluted from the filters by boiling for 1 minute in 300 µL of H₂O containing 15 µg of rabbit liver tRNA. The eluted RNA was quickly frozen in a dry ice ethanol bath. The RNA mixture was allowed to warm to room temperature and the filters removed. The RNA mixture was then precipitated by adjusting the medium to 2.5 Translations were performed according to standard techniques known by those of skill in the art, such as for example, instructions provided by New England Nuclear Antisera raised in rabbits against an extract from Purified rabbit anti p72 alcohol oxidase and anti p76 protein antibodies were prepared by affinity chromatography of whole antisera through a CNBr coupled p72 alcohol oxidase p76 Sepharose 4B column Pharmacia . One gram of CNBr activated gel was prepared by hydrating the gel for 15 minutes in 200 mL of 1 mM HCl followed by 3x 50 mL washes in coupling buffer 0.1 The affinity purified antisera was added to an The strains employed include pYA2 see Figure 23 consists of the YEp13 is available from the American Type Culture Collection and has been assigned accession number ATCC 37115. Both Five mL of yeast cultures in minimal medium were grown until A₆₀₀ equals 1 5 and harvested by centrifugation at 2,000 g for 5 minutes. Cells were suspended in 1 mL of SED and transferred to a 1.5 mL microfuge tube, washed once in 1 For small scale rapid plasmid isolations, 2 mL cultures in the supplemented 2B medium with antibiotic were grown overnight at 37 C with shaking and harvested by centrifugation in 1.5 mL microfuge tubes. Plasmids from all preparations were isolated by the alkaline lysis method described by Birnboim and Doly 1979 . Restriction enzymes were obtained from New England Biolabs and Bethesda Research Laboratories and digestions were performed by routine techniques. Restriction mappings were carried out by comparing parallel digestions of plasmids with and without insert DNA. Restriction fragments were purified by electroelution from agarose gels into Whatman 3 MM paper strips backed by dialysis tubing. The fragments were recovered from the paper and tubing by 3 4 washings with 0.1 0.2 mL volumes of a solution which contained 0.1 For the For the For transfer of large or supercoiled DNA molecules to nitrocellulose, DNA was first partially hydrolyzed by soaking agarose gels in 0.25 DNA sequencing was by the dideoxynucleotide chain termination method of Sanger The ability of each plasmid to be maintained as an autonomous element in DNA fragments which contained the To search for DNA fragments which contain Plasmid pPG 4.0, a pBR322 vector which contains the 4 kilobase pair This 1.12 kbp was ligated into the to give hybrid plasmid pTA011 See Figure 29 . Since the regulatory region Vector pTA011 is cleaved at the unique thus producing hybrid vector pTAO12, which has the following nucleotide sequence with respect to the regulatory region and thus, the regulatory region The vector pTAO13 was then used to transform Vector pTAO13 was then cleaved with Regulatory region The 1.35 kbp A 1.35 kb pair Vector pTAF.85 was then treated in an analogous fashion as vector pTAO13 to give plasmids pTAFH.85, pT76H1 and pT76H2. Thus, the following vectors were cleaved and ligated The production of β galactosidase by several Alcohol oxidase was determined by the dye peroxidase method described above See Example VII and β galactosidase was determined as follows Dissolve 400 mg ONPG Sigma N 1127 in 100 mL of distilled water to make a 4 mg mL ONPG solution. 1 U 1 nmole of orthonitrophenol ONP formed per minute at 30 C and a pH 7. The results presented in Table I indicate that a protein foreign to yeast, i.e., β galactosidase, can be produced in Cells of NRRL Y 15858 and SEY2102 pT76U1 were incubated at 30 C for about 3 4 generations in minimal medium containing 20 µg mL of histidine and leucine and 5 glucose. Cells were then shifted, i.e., collected by centrifugation and transferred into YP medium with 3 of the carbon source indicated in Table II and grown for about 5 generations at 30 C. Cells were then incubated for an additional 50 hours under carbon source starvation conditions, and periodically sampled for β galactosidase. The results are summarized in Table II. These results indicate that a protein foreign to yeast, i.e., β galactosidase, can be produced by The expression levels for β galactosidase in It is seen that the regulatory regions of the invention surprisingly are at least as effective as, or more effective as S. cerevisiae promoters than promoters native to Nine different methanol assimilating yeasts and one methanol non assimilating yeast were grown on minimal media IM1, See Example 1 plus 0.75 methanol or 1.0 glucose, respectively. Total chromosomal DNA was isolated as described in Example VI, i.e., total nucleic acids were isolated, treated with RNase A, extracted first with phenol chloroform isoamyl alcohol, then with chloroform isoamyl alcohol and finally ethanol precipitated. The precipitated DNA was resuspended in a minimum volume of TE buffer and centrifuged to clean the DNA solution. Southern hybridization filters were prepared as described in Example VI, i.e., 10 µg of total DNA from various yeast species was digested with excess The results presented in Table III indicate that genes for polypeptides analogous to p76, p72 and p40 are present in virtually all methanol assimilating yeasts. It is notable that none of these three genes were observed by hybridization of DNA from a methanol non assimilating yeast, The examples have been provided merely to illustrate the practice of the invention and should not be read so as to limit the scope of the invention or the appended claims in any way. Reasonable variations and modifications, not departing from the essence and spirit of the invention, are contemplated to be within the scope of patent protection desired and sought. Birnboim and Doly 1979 Nucl. Acids Res.