# Method of producing heterologous proteins.

## Abstract
A method of producing heterologous proteins comprises causing secretion of a heterologous protein into the periplasmic space by using, as the gene expression elements, a expression elements of a gram positive bacterial strain and, as the host, a gram negative bacterial strain. The invention has thus enabled production of heterologous proteins in gram negative bacteria through the use of gene expression elements of a gram positive bacterial strain and has contributed to technology diversification in the field of genetic engineering. On the occasion of the secretion of the heterologous protein into the periplasmic space of gram negative bacteria, the signal peptide is cleaved off, whereby the protein is obtained in the form of a mature protein free of methionine at the N terminal.

## Claims
What is claimed is 1. A method of producing a heterologous protein which comprises causing secretion of a heterologous protein into the periplasmic space by using, as the gene expression elements, expression elements of a gram positive bacterial strain and, as the host. a gram negative bacterial strain. 2. The method of claim 1, wherein the grampositive bacterial strain is a strain of the genusBacillus. 3. The method of claim 1, wherein the gramnegative bacterial strain is a strain of Escherichia coli. 4. the method of claim 1, wherein the heterologous protein is interferon at. 5. The method of claim 1, wherein the gene expression elements contain amylase promoter portion of Bacillus natto.

## Description
METHOD OF PRODUCING HETEROLOGOUS PROTEINS FIELD OF THE INVENTION This invention relates to a method of producing heterologous proteins by a genetic engineering technique. BACKGROUND OF THE INVENTION Genetic engineering techniques are widely used, for example in the production of useful heterologous proteins. Escherichia coli, Bacillus subtilis and yeasts, among others, are the hosts which current genetic engineering techniques mainly use, Among them,Escherichia coli may be used most easily as the host because of the most advanced elucidation of the genes in its expression system and so forth. However, the use ofEscherichia coli does not always result in efficient recovery of heterologous proteins since they remain in cells after production thereof and are exposed to the decomposing and other actions of intracellular enzymes.On the other hand, the use of Bacillus subtilis as the host might result in more efficient recovery of heterologous proteins produced since they are secrected out of cells. However, the expression system in Bacillus subtilis remains unknown in may respects, so that it is not yet always suitable to use Bacillus subtilis in commercial production of heterologous proteins. An object of the invention is to establish a system which is capable of causing secretion of heterologous proteins into the periplasmic space using gramnegative bacteria. SUMMARY OF THE INVENTION The invention provides a genetic engineering technique by which heterologous proteins are secreted into the periplasmic space using, as the gene expression elements, expression elements of a gram positive bacterial strain and, as the host, a gram negative bacterial strain. BRIEF DESCRIPTION OF THE DRAWINGS In the accompaning drawings, Fig. 1 shows the restriction enzyme map of pGC303, Fig. 2 shows the scheme of cloning the 2 amylase gene from Bacillus natto, Fig. 3 shows the process for constructing an expression vector for use in Escherichia coli, the numerical figures indicating the respective steps referred to in the detailed description of the invention, Fig. 4 shows the process for constructing pI alpha 2,amy2 by ligation of pY16 1 with the IFN alpha 2 gene, Fig. 5 shows the process for constructing pI alpha 2trap31, which carries the IFN alpha 2 gene, from pIztrpl Japanese Patent Application No. 250988 1984 , and Fig. 6 shows the process for preparation of pIa2 trpl, Fig. 7 shows the process for preparation of mature IFN a2, Fig. 8 shows the restriction cleavage map of DNA fragment 883 bp ,EMI3.1 indicating the region.coding for mature IFN alpha 2, indicating the region non coding for IFN alpha 2, indicating the trp promoter region, indicating the trp terminator region,RI indicating EcoRI and HIH indicating HindIII. Tcr Tetracycline resistance CM Chloramphenicol resistance Streptomycin resistance Apr Ampicillin resistance amy Gene coding for amylase BAP Bacterial alkaline phosphatase Ori Replication origin P O Promoter operator region trp P o Tryptophan promoter operator region DETAILED DESCRIPTION OF THE INVENTION a Preparation of gene expression elements The gene expression elements to be used in accordance with the invention is of the gram positive bacterial strain. The gram positive bacterial strain is preferably a strain belonging to the genus Bacillus1 such as a strain of Bacillus subtilis or Bacillus natto.The term gene expression elements includes within the meaning thereof the promoter1 SD sequence, signal sequence coding region and so on which are of the gram positive bacterial strain origin. A preferred example of the gene expression elements is of the amylase gene. The a amylase gene has been isolated by colleagues to the present inventors from Bacillus natto GC161 deposited at the Fermentation ResearchInstitute under the deposit number FERM BP 539 cf.European Patent Publication No. 135045 U.S. Ser. 633562 . A partial nucleotide sequence containing the alpha amylase promoter and signal sequence coding region is shown in Table 1. Tabel 1 GTAC AGTCTCGGGC zooAGTTTTTTTT ATAGGAACAT TGATTTGTAT TCACTCTGCC AAGTTGTTTT 150 GATAGAGTGA TTGTGATAAT TTAAAATGTA AGCGTTAACA AAATTCTCCA 100 GTCTTCACAT CAGCTTGAAA GGAGGAAGCG GAAGAATGAA GTAAGAGGGA soTTTTTGACTC CGAAGTAAGT CTTCAAAAAA TCAAATAAGG AGTGTCAAGA soATGTTTGCAA AACGATTCAA AACCTCTTTA CTGCCGTTAT TCGCTGGATT alpha amylase 100TTTATTGCTG TTTTATTTGG TTCTGGCAGG ACCGGCGGCT GCGAGTGCTG 150 AAACGGCGAA CAAATCGAAT GAGCTTACAG CACCGTCGAT CAAAAGCGGA 200 ACCATTCTTC ATGCATGGAA TTGGTCGTTC AATACGTTAA AACACAATAT 2 5 0 GAAGGATATT CATGATGCAG GATATACAGC CATTCAGACA TCTCCGATTA 300 ACCAAGTAAA GGAAGGGAT CAAGGAGATA AAAGCATGTC GAACTGGTAC A plasmid containing this gene expression elements is provided under the code pGC303 which is a derivative of the plasmid pGC200 derived from a strain of Bacillus natto deposited at the Fermentation Research Institute under the deposit number BERM BP 645 cf. European PatentPublication No. 135045 U.S. Ser. 633562 . The restriction enzyme map of pGC3O3 is shown in Fig. 1. b Preparation of expression vector for use in gram negative bacteria The gene expression elements prepared in step a is then inserted into such a plasmid as pBR322 derived from Escherichia coli, a gram negative bacterial species, to creat a plasmid capable of causing expression of heterologous proteins in gram negative bacteria by the action of the gram positive bacterial gene expression elements. c Ligation with heterologous protein gene A heterologous protein gene coding for a physio logically active substance of. significant use is ligated with the expression vector in gram negative bacteria as obtained in step b , at a site downstream from the promoter, SD sequence and signal sequence of said vector, by the known restriction enzyme and ligase treatment process. The ligation product is introduced into an appropriate host. Preferred exmples of the physiologically active substance are interferons, urokinase, TPA, IL 2 and HBs antigen. d Transformation As the host preferred is a gram negative bacterial strain, in particular a strain of Escherichia coli. The host is transformed by the known method then, S.N. et al., Proc. Natal, Acad. Sci. USA, 69, 2110 1972 , for instance. e Cultivation of host The transformant obtained in step d is cultivated on a per se known medium suited for growing the host. A preferred example of the medium is the medium used by Palva et al. for growing Bacillus subtilis Gene, 22, 229 1983 or a modification thereof. The cultivation is generally conducted at 15 43 t for 3 24 hours, with aeration and or stirring as necessary. f Recovery of heterologous protein After cultivation, cells are harvested by a known method e.g. centrifugation and the periplasmic fraction is recovered by a known method for recovering proteins secreted into the periplasm e.g. osmotic shock method Nossal, N.G. and eppel, L.A.,J. Biol. Chem., 241, 3055 1966 or a modification thereof. The protein is recovered from this fraction by a r s se known method, for example by affinity chromatography and or fractionation based on solubility differences. The invention has thus enabled production of heterologous proteins in gram negative bacteria through the use of gene expression elements from gram positive bacterial strains and has contributed to technology diversification in the field of genetic engineering. On the occasion of the secretion of the heterologous protein into the periplasmic space of gram negative bacteria, the signal pep tide is cleaved off, whereby the protein is obtained in the form of a mature protein free of methionine at the N terminal.Thus, the invention can provide an efficient method of producing hetero logous proteins using genetic engineering technology, In the following, the process for producing human interferon alpha 2 hereinafter, IFN alpha 3 as a preferred example of the heterologous protein using oc amylase promoter of the Bacillus natto as a preferred example of gene expression elements of gram positive bacterial strain and Escherichia coli as a preferred exmaple of the host will be described step by step. It is to be noted, however, this mode of practice is by no means limitative of the scope of the present invention. a Cloning of gene expression elements A one liter portion of a overnight culture ofBacillus natto GC161 deposited at the FermentationResearch Institute under the deposit number FERM BP 539 was centrifuged and the cells obtained were washed. To the cells were added 10 ml of 20 sucrose added 50 mM Tris hydrochioride buffer pH 8.0 and 100 mg of lysozyme, followed by reaction at 37 C for 15 minutes.Thereafter, 10 ml of 1 Sarkosyl l EDTA solution and 1 ml of Pronase E 100 mg ml were added. After standing at 37 C for 16 hours, the solution was made up to 35 ml with TES solution 30 mM Tris HCI, 50 mM NaCl, 5 mM EDTA, pK 8 . Thereto were added 35 g of cesium chloride and 1 ml of ethidium bromide 10 mg ml and the mixture was centrifuged at 60,000 rpm Beckman 70.1 Ti rotor for 16 hours. The chromosomal DNA band was collected, dialyzed against TES solution and treated twice with phenol for purification of the chromosomalDNA. The chromosomal DNA thus obtained was cleaved with the restriction enzymes PvuII, BglII, HpaI, BamHI, Hind III and so forth. Fragments were extracted from 0.8 agarose gel and ligated with various plasmids, pC194, pE194, pUB110, and pGC2OO deposited at theFermentation Research Institute under the deposit number FERM BP 645 which is a derivative of pC194, and the ligation products were used for transforming B. subtilis rec , amy . After selection based on drug resistance of each plasmid, the transformants selected were inoculated onto starch added agar plates L agar1 soluble starch .After incubation at 37 C for 2 days, iodine solution 0.1 iodine 1 KI was added to the agar plates for checking oc amylase activity on the basis of decoloration of iodine. When, among the five restriction enzymes mentioned above, Hindu was used for cleaving the chromosomal DNA prepared from Bacillus natto and a fragment of said DNA was inserted into pGC2OO at the Hind III site thereof, followed by trans formation with the resulting plasmid, there was obtained a strain having a amylase activity. The chromosomalDNA fragment inserted was a 7 kb fragment.Therefore, the 7 kb DNA containing the z amylase gene was excised using the restriction enzymes Hind E and RsaI and inserted into the EcoRV HindIII fragment of pBR322.The plasmid pGC302 thus obtained was then cleaved with Sacll, followed by ligation, which gave pGC303 7.9 kb Fig. 2 . Transformation was performed in the conven tional manner, selection was made based on the ampicillin resistance and the transformants selected were checked for z amylase activity in the same manner as above, whereupon expression of z amylase was confirmed. Thus was confirmed the presence of the whole alpha amylase gene series. In this manner, the plasmid pGC303 containing the alpha amylase promoter, signal sequence coding region and structural gene for are amylase was obtained. The DNA sequence of the promoter and signal sequence coding region of ar amylase gene was determined by the Maxam Gilbert method Maxam, A. and Gilbert, W., 1979, Methods in Enzymology, 65, 499 560 Table 1 . b Construction of expression vector for use in Escherichia coli An expression vector for use in Escherichia coli was constructed from pGC303 and pBR322 in accordance with the procedure shown in Fig. 3. Step 1 pGC303 was digested with the restriction enzymes BanIII and BanII to give a relatively large DNA fragment containing the promoter and signal sequence coding region of alpha amylase gene. The reaction was carried out using the following reaction mixture composition pGC303 1.0 g l 100 FQ 10 x BcII buffer 1 20 l Ban II 8U l, Toyobo 20 l Ban III 5U l, Toyobo 20 l HzO 40 FQ Total 200 lEMI10.1 60 mM Tris HCl pH 7.4 EMI11.1 tb SEP 100 SEP mM SEP MgCl2 tb SEP 10 SEP mM SEP dithiothreitol tb 750 SEP mM SEP KCl tb The reaction was carried out overnight at 37 CAfter completion. of the reaction, the whole reaction mixture was applied to 1 agarose gel, followed by electrophoresis. A 960 bp fragment was recovered following adsorption on DEAE paper. Thereto was added 60 g of tRNA as the carrier for ethanol precipitation.The mixture was extracted with phenol, and the DNA was recovered by ethanol precipitation. The DNA precipitate was dissolved in 90 l of H20. Step 2 The fragment obtained in step 1 was partially digested with AluI using the following reaction mixture composition DNA fragment solution obtained in step 1 50 l 10xAluI buffer 2 20 FQ AluI 2 U l, Takara Shuzo 20 l HzO 110 FQ Total 200 lEMI11.2 100 mM Tris HCl pH 7.5 70 mM MgCl2 200 mM NaCl 70 mM 2 mercaptoethanolThe reaction was carried out at 37 C C for 5 minutes, and the whole reaction mixture was applied to 5 polyacrylamide gel, followed by electrophoresis.Thereafter, a 621 bp DNA fragment was recovered by electroelution. To the fragment fraction was added 20 g of polyinosic acid was added as the carrier in ethanol precipitation. After extraction with phenol, the fragment was precipitated with ethanol. The DNA precipitate obtained was dissolved in 30 of HzO. Step 3 A BamHI linker was ligated with the DNA fragment obtained in step 2 at the AluI end thereof using the following reaction mixture composition Partial AluI digest fragment obtained in step 2 10 x ligation buffer 3 1 l BamHI linker 4 10 l Tt DNA ligase 175 U Takara Shuzo 2 N Total 23 lEMI12.1 660 mM Tris HCl pH 7.6 10 mM ATP 10 mM spermidine 100 mM MgCl2 150 mM DTT dithiothreitol EMI12.2 Product of Pharmacia P L BiochemicalsSequence 5 d CCCGGATCCGGG 3Used after phosphorylation at the 5 end. The reaction was carried out overnight at 4 c, followed by extraction with phenol and precipitation with ethanol. The DNA precipitate was dissolved in 42 Fe of HzO. Step 4 The DNA fragment obtained in step 3 was digested with BamHI using the following reaction mixture composition DNA obtained in step 3 42 FQ 10xBamHI buffer 5 5 l BamHI 12 U l, Takara Shuzo 3 l Total 50 FQ EMI13.1 100 mM Tris HCl pH 8.0 70 mIl MgClz 1 M NaCl 20 mM 2 mercaptoethanol The reaction was carried out overnight at 37 C and the whole reaction mixture was applied to 5 polyacrylamide gel for electrophoresis. After electrophoresis, a 533 bp fragment was recovered by electroelution, followed by extraction with phenol and precipitation with ethanol. The DNA precipitate recovered was dissolved in 50 l of H20. Step 5 For use as a vector for cloning the DNA fragment both ends being BamHI ends obtained in step 4, pBR322 was digested with BamHI overnight at 37 C using the following reaction mixture composition pBR322 1.0 g l 25 l 10xBamHI buffer mentioned above 5 l BamHI 10 U l, Takara Shuzo 3 l HzO 17 FQ Total 50 l and then the phosphoric acid residue at the 5 end was eliminated by using bacterial alkaline phosphatase BAP . After reaction, DNA was recovered by precipitation with ethanol. The DNA was dissolved in 180 l of HzO. To prevent self ligation of the vector pBR322 , this was treated with bacterial alkaline phosphatase BAP at 65 C for 40 minutes for eliminating the phosphoric acid residue at the 5 end. The reaction mixture composition was as follows BamHI digested pBR322 180 l 10xBAP buffer 6 20 FQ BAP 0.45 U l, Takara Shuzo 2 l Total 202 FQ EMI14.1 100 mM Tris HCl pH 7.5 70 mM MgCl2 600 mM NaCl After reaction, phenol extraction was repeated three times and DNA was recovered by precipitation with ethanol. The DNA precipitate was dissolved in 25 ss ofHzO. Step 6 The DNA fragment obtained in step 4 and the vector obtained in step 5 were ligated together by overnight reaction at 4 t. The reaction mixture composition was as follows DNA fragment from step 4 10 ss BamHI digested and BAP treated pBR322 5 l 10 x ligation buffer 7 4 l T. DNA ligase 175 U l, Takara Shuzo 1 oss Total 20 ss EMI15.1 660 mM Tris HC1 pH 7.6 50 mM MgC12 50 mM DTT dithiothreitol 10 mM ATP After reaction, the whole reaction mixture was used for transformation of E. coli RR1 by the method ofCohen et al. As a result, there were obtained 6,500 colonies, 200 of which were checked for sensitivity to ampicillin Ap and to tetracycline Tc .Fourteen strains were found to be resistant to Ap Apr and susceptible to Tc Tcs . Using these 14 strains, the plasmids were prepared by the alkaline extraction method Birnboim, H. C. and Doly, J., Nucleic Acids Research, 7, 1513 1979 , the plasmids were then examined for their structure using varous restriction enzymes. As a result, 2 colonies were found to contain a plasmid having the structure of pYI5 1 given in Fig. 3. One of these strains was used for large quantity plasmid preparation and the plasmid was named pYI5 1. Step 7 pYI5 1 is inconvenient for inserting heterologous genes such as the IFN alpha 2 gene since it has two BamHI sites. Therefore, for causing one BamHI site upstream from the amylase promoter to disappesr, pYI5 1 was first digested with AvaI at 37 t overnight. The reaction mixture composition used was as follows pY15 1 1.43 g l 14 l 20 g 10xAvaI buffer 8 5 FQ AvaI 8 U l, Takara Shuzo 3 l HzO 28 B Total 50 ss EMI16.1 100 mM Tris HC1 pH 8.0 70 mM MgCl2 600 mM NaCl 70 mM 2 mercaptoethanol After reaction, the whole reaction mixture was applied to 1 agarose gel and electrophoresis was performed.A DNA fragment of about 3.5 kbp was caused to be adsorbed on DEAE paper and then recovered. As the carrier to serve in precipitation with ethanol, there was added 20 g of tRNA. Following extraction with phenol, DNA was recovered by precipitation with ethanol. The DNA recovered was dissolved in 50 ss ofH2O. Step 8 The AvaI digested pYI5 1 obtained in step 7 was cyclized by allowing self ligation to take place at 4 C overnight. The reaction mixture composition used was as follows AvaI digested pYI5 1 obtained in step 7 2 l 10xligation buffer same in composition as in step 6 10 FQ T4, DNA ligase 175 U l, Takara Shuzo 1 PQ 1120 87 oN Total 100 l A 25 l portion of the reaction mixture 100 ss was used for transformation of E. coli RR1, whereby 18 ampicillin resistant Apr colonies were obtained. These colonies were subjected to plasmid extraction by the alkline extraction method and the plasmids extracted were examined for their structure using various restriction enzymes. As a result, 14 colonies were found to have a plasmid having the structure of pYI6 1 given in Fig. 3. One of these strains was used for largequantity plasmid preparation and the plasmid obtained was named pY16 1. The plasmid pYI6 1 thus obtained has, in addition to the BamHI site, one each of EcoRI, Clal same asBan III Hindu and EcoRV sites. Using these sites, heterologous genes such as the IFN alpha 2 gene can be ligated to said plasmid. Thus, DNA fragments having a Bamill site upstream from the gene coding for a heterologous protein such as IFN 22 and having a flush end site such as an EcoRI, ClaI, Hind III or EcoRV site downstream therefrom can be ligated with said plasmid. c Ligation of pYI6 1 and IFN alpha 2 gene construction of pI pI alpha 2amy2 For effecting expression of the IFN alpha 2 gene using pY16 1, a DNA fragment containing the IFN alpha 2 gene was ligated to pYI6 1 between the BamHI site and the EcoRV site thereof, whereby pI alpha 2amy2 was obtained. This process is summarized in Fig. 4. Step 1 pYI6 1 was digested with BamHI and EcoRV at 37 c overnight. The reaction mixture composition used was as follows pYI6 1 0.508 g l 100 ss 10xBamHI buffer 15 FQ BamHI 12 U l, Takara shuzo 10 FQ EcoR V 5 U l Takara Shuzo 20 ss H2O 5 l Total 150 FQ After reaction, the whole reaction mixture was applied to 1 agarose gel and electrophoresis was conducted. After electrophoresis, a DNA fragment of about 3.4 kbp was allowed to be adsorbed on DEAE paper. Following elutlon of DNA and one extraction with phenol, the DNA was recovered by precipitation with ethanol.The DNA was dissolved in 50 l of H2O, Step 2 An IFN alpha 2 gene bearing plasmid pI alpha 2 trp31 was prepared from pI alpha 3trp1 Japanese PatentApplication No. 250988 84 following the process given in Fig. 5. The IFN alpha 2 gene, with teh trp a terminator, was excised by digesting pI pI alpha 2trp31 with BamHI and SmaI at 37 C overnight. The reaction mixture composition used was as follows pI alpha 2trp31 2.22 g l 22.5 l 50 g 10xBamHI buffer 10 l BamHI 12 U l, Takara Shuzo 7 l Smal 8 U F , Takara Shuzo 10 ss HzO 50.5,a Total 100 , After reaction, the whole was applied to 1 agarose gel, followed by electrophoresis. A DNA fragment of about 950 bp was allowed to be adsorbed onDEAE paper, and then the DNA was recovered in the same manner as above. The DNA was dissolved in 50 Fe of HzO. pI alpha 2 trp1 was prepared by the following method see Fis 6 The cDNA fragment coding for IFN alpha 2 was prepared by a method similar to that of described in European Patent Publication No. 32134, and then ATG combined matured IFN alpha 2 gene, which was combined ATG, was prepared by the process given in Fig 7. The restriction cleavage map of DNA fragment 883 bp thus obtained was shown in Fig. 8. The pYN6 see European Patent Publication No. 152830 was digested with EcoRI. then digested product was treated with Bacterial Alkaline Phosphatase, and then digestion with BamHI was conducted. The reaction mixture was applied to 6 polyacryl amide gel PAGE , followed by electrophoresis, whereby a 69 bp fragment was isolated. The 69 bp fragment was ligated with IFN alpha 2 gene bearing 883 bp, and then a fragment corresponding to 952 bp was isolated by using 5 PAGE. Thus obtained fragment was ligated with a EcoBI PstI fragment 3,610 bp in pBR322. And E. coli RR1 was transformed by this obtained plasmid. Thus obtained Tcr transformants were screened and plasmid were extracted from these transformants. And these plasmid were examined using various restriction enzymes. Acontemplated plasmid pI acz trpl 4,562 bp was obtained. Step 3 The vector prepared in step 1 and the IFN alpha 2 gene containing DNA fragment prepared in step 2 were ligated together by overnight reaction at 12 t using the following reaction mixture composition BamHI EcoRV digested pYI6 1 2 BamHI SmaI digested DNA fragment 5 l 10xligation buffer 2.5 l T4 DNA ligase 175 U l, Takara Shuzo 2 l H2O 13.5 l Total 25 FQ d Transformation After the reaction in the above step 3, a 16 l portion of the reaction mixture was used for transformation of E. coli HB101. As a result, 9 Apr transformants were obtained. These strains were subjected to plasmid extraction by the alkaline method and the plasmids extracted were examined for their structure using various restriction enzymes.All the strains were found to have pI alpha 2amy2 shown in Fig. 4. One of these strains pI alpha 2amy2 carrying E. coli HB101 was checked for its ability to produce IFN alpha 2, e Production of IFN alpha 2 The pI ploc2amy2 carrying strain of E. coli HB101 was grown in a 2 liter erlenmeyer flask with shaking at 37 C using 500 ml of a medium specified below according to Palva et al. Gene, 22, 229 1983 with checking at timed intervals for bacterial growth in terms of measured A650 values. Yeast extract Difco 10 g liter Bactotryptone Difco 20 g liter Soluble starch mercy 10 g liter NaCl 10 g liter NH4 zSO4 1 g liter KzHPO4 7 g liter KHzPQ4 2 g liter Ampicillin 25 mg liter f Recovery of IFN alpha 2 The culture obtained in e was sampled 20 ml at 3 hour intervals and the samples were each fractionated according to the flowchart given later herein. The culture supernatant, cell washings 1 and 2 , periplasmic fraction and cell extract were assayed forIFN titer by the RPHA method. At 9 hours after start of cultivation, the IFN alpha 2 potencies thus measured are shown in Table 2. Table 2 IFN alpha 2 titer IU liter of medium Culture supernatant 1.88 x 106 Cell washings 6.25 x 106 Cell extract 2.65 x 106 Periplasmic fraction 1.88 x 106 Then, the periplasmic fractions and cell extract fractions collected at 3 hour intervals over the period of hour 6 to hour 27 were assayed for IFN alpha 2 potency.The results thus obtained are shown in Table 3. As is evident from the data given in Table 3, the yield increased with the growth of bacteria and became constant at 15 hours after start of cultivation. Not less than 95 of intracellular IFN 2 was found secreted in the periplasm. Table 3 IFN alpha 2 titer IU liter of medium EMI23.1 tb Cultivation SEP Periplasmic SEP Cell SEP A650 SEP tb SEP period SEP fraction SEP extract SEP adsorbance tb SEP hours tb SEP 6 SEP 6.63 SEP x107 SEP 2.65x106 SEP 3.748 tb SEP 9 SEP 1.88 SEP x108 SEP 2.65x106 SEP 7.27 tb SEP 12 SEP 2.66 SEP x108 SEP 5.30x106 SEP 9.65 tb SEP 15 SEP 3.75 SEP X108 SEP 5.30x106 SEP 11.03 tb SEP 18 SEP 3.75 SEP x108 SEP 5.30x106 SEP 11.43 tb SEP 21 SEP 3.75 SEP x108 SEP 7.50 SEP x106 SEP 11.71 tb SEP 24 SEP 3.75 SEP x108 SEP 1.50 SEP x SEP 107 SEP 11.13 SEP tb SEP 27 SEP 5.32 SEP x108 SEP 3.00x107 SEP 10.29 tb Flowchart illustrating the process for recovering IFN alpha 2 20 ml of culture EMI24.1 Centrifugation SEP 8.000 SEP rpm, SEP 4 C, SEP 10 SEP minutes tb tb Cell pellet Supernatant culture supernatant EMI24.2 Suspended SEP in SEP 5 SEP ml SEP of SEP cold SEP aqueous SEP solution tb containing SEP 10 SEP mM SEP Tris HCl SEP pH SEP 8.0 SEP and SEP 30 SEP mM SEP NaCl tb Centrifugation SEP 6.000 SEP rpm, SEP 4 C, SEP 5 SEP minutes tb tb Cell pellet Supernatant cell washings I EMI24.3 Suspended in 5 ml of aqueous solution containing 20 sucrose and 30 mM Tris HCl pH 8.0 Addition of 10 l of 0.5 M EDTA pH 8.0 final concentration 1 mM EDTA Shaking at room temperature for 10 minutesCentrifugation 6.000 rpm, 4 C, 5 minutes EMI25.1 Cell pellet Supernatant cell washings 2 EMI25.2 Suspended SEP in SEP 5 SEP ml SEP of SEP cold SEP distilled SEP water tb Shaking SEP on SEP an SEP ice SEP bath SEP for SEP 10 SEP minutes tb Centrifugation SEP 6.000 SEP rpm, SEP 4 C, SEP 5 SEP minutes tb tb Cell pellet Supernatant periplasmic fraction EMI25.3 Suspended SEP in SEP 1.5 SEP ml SEP of SEP lysis SEP buffer tb Allowde SEP to SEP stand SEP at SEP 0 C SEP for SEP 3 SEP minutes tb Five SEP repetitions SEP of SEP freezing thawing SEP cycle tb Centrifugation SEP 18.00 SEP rpm, SEP 4 C, SEP 90 SEP minutes tb tb Precipitate Supernatant cell extract 50 mM Tris HCl pH 8.0 , 3.0 mM NaCl, 6 mM EDTA, 10 g ml phenylmethylsulfonyl fluoride, 1 mg ml lysozyme