# PROCESS FOR PRODUCING ALCOHOL

## Claims
Verfahren zum Herstellen von Ethanol aus stärkehaltigem Material, wobei das stärkehaltige Material gleichzeitig der Zuckerbildung und anschliessenden Fermentation durch Verwendung eines Hefewirts ausgesetzt wird, der durch einen exprimierbaren, rekombinanten Vektor transformiert worden ist, der ein Glukoamylasegen umfasst, das für ein Polypeptid codiert, das die Aminosäuresequenz III umfasst

## Description
The present invention relates to a process for producing alcohol using a yeast which is engineered by recombinant DNA technology so as to acquire the ability to produce glucoamylase derived from Glucoamylase is an enzyme which hydrolyzes starch into glucose from the non reducing terminal and, therefore, is widely used for saccharifying starch in the poduction of ethanol from starchy materials. EP 126 206 describes host organisms transformed with a glucoamylase cDNA sequence obtained from the genus Aspergillus. Glucoamylase from Aspergillus differs, however, remarkably in its amino acid sequence from glucoamylase obtained from the genus Rhizopus. The glucoamylase derived from the genus However, The present inventors assumed that the above problem could be resolved if the glucoamylase gene of The present invention relates to a process for producing ethanol, wherein a glucoamylase gene derived from a filamentous fungus of the genus The starchy materials employable in the process of the invention include, for example, cereals, subterranean starches such as cassava, sweet potato, potato, as well as pure starch. In order to produce the The high temperature cooking process cooked at above 120 C , the low temperature cooking process cooked at 60 85 C , and the non cooking process are known steps for cooking or gelatinizing the raw material before saccharification in the production of alcohol. Any of the above processes may be employed in the alcohol production of the present invention. Instead of simultaneously producing alcohol and Rhizopus glucoamylase in yeast host, bacterial cells of Bacillus subtilis may be used as a host if According to the present invention, a method for obtaining DNA encoding the glucoamylase and the novel information of the nucleotide sequence of said DNA are also provided. The nucleotide sequence of the DNA fragment for the production of the glucoamylase in accordance with the process of the invention can be prepared by, for example, appropriately conjugating a part of the glucoamylase nucleotide sequence cloned from chromosomal DNA of Any fungi of the genus The present strain SAM 0034 has the following mycological properties. Colonies on Potato dextrose agar medium attain a diameter of 5 5.5 mm in one day at 28 C and cover 90 mm petri plates of Potato dextrose agar medium in two days, white. Colonies become grey with age. Stolons hyaline or yellowish brown rhzoids brown. Sporangiophores usually arising from rhizoids, occasionally arising directly from stolons, either single or in groups, occasionally divided, 220 1200 µm long. Sporangia globose or subglobose, dark brown, 60 150 µm in diameter columellae globose or subglobose. Sporangiospores globose, subglobose, or angular, striatae on the surface, 5 15 x 3 7 µm. Chlamydospores subglobose or cylindrical, 6 13 x 4 19 µm. No zygospores were observed. At 37 C, growth occurs. The present strain SAM 0034 can be accommodated in the fungus genus The mycological properties of the present strain SAM 0034 are compared with those of known species of the genus Rhizopus, referring to Inui, T., Y. Takeda H. Iizuka, 1965, Taxonomical Studies on genus The glucoamylase gene may be isolated in the form of cDNA prepared from the mRNA of Rhizopus or by cloning from the chromosomal DNA of Rhizopus using a synthetic origonucleotide corresponding to a part of amino acid sequence of Usually it is not easy to obtain the complete glucoamylase gene by the former method, while the gene obtained by the latter method usually contains intron sequences and thus cannot be expressed in host E. coli or yeast. In order to obtain a glucoamylase gene capable of expression in these hosts, an appropriate part of the cDNA from mRNA may be conjugated with the intron free part of the DNA sequence of the chromosomal gene. If one or both DNA fractions lack suitable sites to be cleaved by restriction enzymes, a technique of The glucoamylase gene used in the present invention involves not only the same nucleotide sequence coding for the bracketed amino acid sequence in Fig. 1 but also a nucleotide sequence corresponding to an amino acid sequence having an enzymatic activity comparable to that of the bracketed amino acids. The Rhizopus derived glucoamylase structural gene is a DNA fragment encoding the sequence of 26 604 amino acids from the N terminal in Fig. 1, or it corresponds to nucleotide sequence numbers 190 1926 designated in Fig. 1. The region of 1 25 amino acis from the N terminal is a signal peptide coding region involved in the extracellular secretion of glucoamylase from the host cell. When, as described below, glucoamylase was produced by a yeast using this signal peptide coding region, more than 90 of the glucoamylase produced was secreted in the culture medium. The secreted glucomaylase was purified by a routine method and the amino acid sequence at the N terminus was examined the amino acid sequence of the glucoamylase started at the 26th amino acid of the sequence given in Fig. 1 and this indicates that the region defined by 1 25 amino acids from the N terminus will function as a signal peptide in the yeast as well. A method that can be used with advantage for the purpose of isolating the glucoamylase gene shown above is described hereunder. The whole DNA is separated from a glucoamylase producing fungus of the genus A probe DNA oligomer described in Examples 1 and 2 for detecting glucoamylase gene is prepared and used in colony hybridization. Colonies that will hybridize with the probe are grown and the plasmid DNA is extracted. The present invention is hereunder described in greater detail with reference to the following examples. The whole DNA was isolated from glucoamylase producing Transformant selection was made by a method generally referred to as colony hybridization, using a nitrocellulose filter paper. The first step starts with the preparation of a probe for detecting the glucoamylase DNA. For this purpose, the purified Rhizopus glucoamylase was decomposed by a variety of proteases and the resulting peptides were separated and purified. These peptides were subjected to amino acid analysis and the primary structures were determined by routine methods. As a result, an amino acid sequence having the partial structure of Asp Leu Thr Trp Ser His Ala Ser was obtained. It was also found that this glucoamylase had an N terminal amino acid sequence of Ala Ser Ile Pro and a C terminal sequence of Ala Ala. In order to prepare the desired probe, 32 different synthetic DNA oligomers each consisting of 14 bases 5 ACNTGGTCNCAQGC 3 were produced by the triester solid phase method from the amino acid sequence of Thr Trp Ser His Ala which was part of the sequence identified above and wherein N is an arbitrary base and Q is T or C from pyrimidine. These DNA oligomers were labelled with γ ³²p ATP and T4 polynucleotidyl kinase and used as probes for detecting the glucoamylase gene. Transformed E. coli colonies that hybridized with these probes by colony hybridization were grown and plasmid DNAs were extracted. The extract was treated with restriction enzymes and the resulting DNA fragments were analyzed by agarose gel electrophoresis. As for the colonies that hybridized with the probes, the DNA fragment inserted in the plasmid had a size of 4.3 kb, as well as one cleavage site each of BamHI, KpnI, MluI and SacI, two sites for DraI and three sites for BglII, but had no cleavage sites for AccI, BalI, ClaI, EcoRI, HpaI, PstI, PvuII, ScaI or XhoI. The plasmid having this DNA fragment was named pRGA39. The whole RNA was isolated from the aerial hyphae of Rhizopus oryzae. For this purpose, known procedures including the use of guanidium thiocyanate were followed. Polyadenylated RNA was recovered from the whole RNA as a mRNA fraction by way of chromatography on oligo dT cellulose. Using this mRNA, a cDNA gene library was formed in E. coli WA802 by the method of Okayama and Berg described in Okayama, H. Berg, P., Mol. Cell Biol., Selection of the transformant having the c DNA of the aimed enzyme was made by the aforementioned method of colony hybridization. A DraI fragment 2.0 kb of the glucoamylase gene obtained in a ii was used as a probe for detecting the glucoamylase cDNA. For this purpose, this fragment was labelled by the technique of nick translation using α ³²P dCTP, DNA polymerase I and DNase I. The transformed colonies that would hybridize with this probe were allowed to grow and the plasmid DNA was extracted. The extract was treated with restriction enzymes and the resulting DNA fragments were analyzed by electrophoresis on agarose gel. The DNA fragment which had been inserted into the plasmid of the colonies that hybridized with the probe had a size of 1.7 kb. This plasmid was named pCGA239. Plasmids pRGA39 and pCGA239 were digested with restriction enzymes and DNA fragments were isolated on agarose gel. Their nucleotide sequences were determined by the dideoxy method using recombinant phage M13. Analysis of pRGA39 revealed that this gene contained four introns each having a length of several tens of bp for the intron sites, see Fig. 2 . The plasmid pCGA239 was not a cDNA corresponding to the full length of glucoamylase but lacked about 50 amino acids. Restriction maps of pRGA39 and pCGA239 are compared in Fig. 4. The cloned cDNA did not have the full length. A cDNA of the full length having in addition a glucoamylase promoter derived from Rhizopus was prepared by conjugating appropriate parts of pCGA239 and pRGA39. Since suitable restriction sites available for conjugation purposes were absent from the plasmids, the SalI site was introduced at the corresponding locations of pCGA239 and pRGA39 by the method of Because of the introduction of the SalI site, the 53rd amino acid codon from the N terminal of the treated plasmid pCGA39 was aspartic acid rather than lysine which was initially present. Therefore, the plasmid was again subjected to The full length glucoamylase gene in pCGA449 is the combination of the sequence appearing before the arrow in Fig. 2 a and the sequence appearing after the arrow in Fig. 3. e i With a view to ensuring efficient expression of pCGA449 in a yeast, a fragment ca. 8.3 kb cut out with EcoRI SalI from pYGIFLm212 having the acid phosphatase promoter Ppho5 disclosed in Japanese Patent Application No. 157037 1984 was used. In order to incorporate pCGA449 at the EcoRI SalI site, the PvuII site of pCGA449 was converted to XhoI site with XhoI linker, whereby plasmid pCGA450 was prepared Fig. 6A . This plasmid was cleaved with XhoI and EcoRI and a 2.2 kb fragment was separated by agarose gel electrophoresis. This fragment was ligated to the previously obtained 8.3 kb EcoRI SalI fragment with a T4 DNA ligase. The resulting plasmid containing Ppho5 as a promoter was named pYGA2149. This plasmid was grown in E. coli strain WA802 and then separated see Fig. 6A . e ii with a view to ensuring efficient expression of pCGA469 in a yeast, a fragment ca. 8.9 kb cut out with EcoRI SalI from pYGIFLm222 having the glyceraldehyde 3 phosphate dehydrogenase promoter P pYGA2169 was prepared from pYGIFLm212 using pCGA469 and repeating the same procedures as described above and the plasmid was separated Fig. 6B . pYGA2249 not shown was also prepared by the same procedures as described above and separated. The plasmid pYGA2149 was used to transform yeast Saccharomyces cerevisiae strains XS 30 2A MATα, leu2, his3, trp1, ura3 and XS 30 2B MATα, leu2, his3, trp1, ura3 and the transformed colonies were selected on the basis of the nutrient requirement for tryptophan as a marker. Transformation was performed by the method of Ito et al. Ito, H. et al., J. Bacteriol., 153, 1983 using LiCl. A platinum loopful of the transformed colonies were inoculated in 5 ml of γPD medium 1 yeast extract, 2 polypeptone and 2 glucose and sampling was made 48 hours later. Centrifugation 10,000 rpm x 5 min was conducted in an Eppendorf tube thus separating the sample into the supernatant and pellet. The activity of glucoamylase in the supernatant was measured by the following procedures 200 µl of the supernatant was added to 800 µl of a soluble starch solution 1.0 soluble starch in 20 mM acetate buffer solution, pH 5.0 and the mixture was left to stand at 37 C. The amount of released glucose was determined with a glucostat Fujisawa Pharmaceutical Co., Ltd. . The activity data for 2 hour reaction were as follows 0.004 U ml for pYGA2149 in XS 30 2A and 0.008 U ml in XS 30 2B, indicating the sex dependent difference by a factor of 2 see Fig. 7 . Activity of one unit U corresponds to 1 µmol of glucose released in 1 minute. No activity was observed in the supernatants obtained from yeast which did not contain pYGA2149. The glucoamylase activity for the plasmid using P It was confirmed by the following immunological techniques that the activities listed above were due to the Rhizopus derived glucoamylase. A rabbit antibody was prepared using a purified glucoamylase. This antibody was used in the analysis by a method commonly referred to as Western blotting, using a concentrate of 1.5 ml of the same supernatant from the 48 hour culture that was employed in the previous activity measurements. One third portion of the concentrate was subjected to 10 polyacrylamide electrophoresis and the protein in the gel was transferred and immobilized on a nitrocellulose filter paper electrophoretically. The glucoamylase on the nitrocellulose filter paper was then detected by the known technique in enzyme immunology using the reaction with peroxidase. A band that would react with the glucoamylase antibody emerged at a position substantially equal to the Rhizopus glucoamylase in terms of molecular weight. This fact did indicate the expression of the Rhizopus derived glucoamylase in yeasts. The data showing this fact are give in Fig. 8. The effect of pYGA2149 on the growth of yeast strain XS 30 2B was investigated using differnet carbon sources. First, XS 30 2B was shake cultured in YPD medium for 24 hours at 30 C and XS 30 2B pYGA2149 was shake cultured under the same conditions in a minimum nutrient medium 0.67 Difco Yeast nitrogen base and 2 glucose containing 1 Casamino acids and uracil. A 100 ml portion each of YPD medium or YPS medium 1 yeast extract, 2 polypeptone and 2 soluble starch was added in a 500 ml Sakaguchi flask and sterilized in an autoclave. Each of the pre culture suspensions 1 ml was added to these mediums at 30 C and the subsequent growth was evaluated in terms of absorbance at 660 nm. The strain XS 30 2B harboring no plasmid pYGA2149 was capable of little growth in the YPS medium, but the strain harboring pYGA2149 grew at equal rates on both YPS and YPD mediums. This fact clearly shows that XS 30 2B pYGA2149 produced glucoamylase and utilized the starch hydrolyzed by the enzyme see Fig. 9 . The medium used in this experiment was prepared by autoclaving 121 C, 15 minutes 200 ml of YPS medium 1 yeast extract, 2 polypeptone, and 1, 2 or 5 soluble starch in a 500 ml Erlenmeyer flask. The following yeast strains were employed. Starting pre culture was prepared by inoculating a platinum loopful of the yeast in a minimum nutrient medium 5 ml containing 1 Casamino acids, uracil and adenine and shake culturing at 28 C for 20 hours. This starting pre culture was inoculated in a YPD medium in an amount of two percent and subjected to stationary cultivation at 28 C for 24 hours to obtain a final pre culture. The final pre culture was inoculated in YPS medium at five percent and subjected to stationary cultivation at 28 C for the purpose of investigation ethanol production. The cultivation of yeast strain 3 on and after the final pre culture was conducted in a low phosphate YPD or YPS medium respectively with a view to inducing Ppho5. The same experiment was conducted for thee different starch concentrations 1, 2 and 5 . The results of ethanol production and yeast growth are shown in Tables 1 and 2. Ground corn 140 g was added to 402 ml of water, and after adding 0.5 g of α amylase preparation Termamil as a viscosity reducer and 160 ppm of potassium metabisulfite as a germicide, the mixture was held at 80 82 C for 5 minutes, and then rapidly cooled. Starting and final pre cultures were prepared using the same strain as employed in h i and the final pre culture was added to the low temperature cooked starch and the mixture was cultured at 28 C under three different conditions, i.e., in the absence of any additional component, in the presence of Casamino acids, uracil and adenine, and in the presence of Casamino acids, uracil, adenine and 0.4 glucose. The progress of fermentation as evaluated in terms of the decrease in CO₂ and alcohol production were investigated. A non transformed yeast using the routine amount of Rhizopus glucoamylase was employed as a positive control. The results are shown in Tables 3 and 4 and in Fig. 10, from which one can see that the yeasts obtained in accordance with the present invention were capable of direct alcohol production from non cooked or low temperature cooked LTC starch without addition of a Rhizopus glucoamylase preparation. a A 2.0 kb DNA fragment containing glucoamylase gene from pYGA2269 prepared in Example 1 e was inserted into the HindIII site of a known vector YCp19, whereby pYGA195 was obtained Fig. 6C . This plasmid, containing a centromere was present in yeast with a copy number of 1. The plasmids, pYGA2269 and pYGA195, were used to transform yeast strain XS 30 2B MATα, Leu2, his3, trp1, ura3 and the transformed colonies were selected by the nutrient requirement for tryptophan as a marker. Transformation was conducted by the method of Ito et al ibid using LiCl. The selected transformant colonies were cultured overnight at 30 C in 5 ml of a minimum nutrient medium 0.67 Yeast Nitrogen base, 2 glucose, 0.001 uracil, 0.0054 adenine, 0.0026 leucine and 0.0038 histidine . The culture solution was inoculated at one percent in 400 ml of the minimum nutrient medium in an Erlenmeyer flask 1,000 ml and shake cultured at 30 C. Sampling was conducted at approximate intervals of 3 hours, and the absorbance at 660 nm and the glucoamylase activity values at 24 and 48 hours were determined, by measuring the amount of free glucose that was released by reaction at 37 C of a mixture of the supernatant of the culture 50 µl and 950 µl of a soluble starch solution 0.5 soluble starch in 20 mM acetate buffer, pH 4.5 . The amount of glucose was determined by a glucostat of Fujisawa Pharmaceutical Co., Ltd. One unit U of glucoamylase activity corresponds to 1 µmole of glucose released in 1 minute. The yeasts transformed by pYGA2269 and pYGA195 produced glucoamylase activities in amounts of 0.2 U ml and 0.5 U ml, respectively, at 48 hours see Fig. 11 . This enzyme can be secreted in an adequate amount even from a plasmid with a copy number of 1 by expressing it in the presence of a strong promoter such as GAPDH promoter. S. cerevisiae XS 30 2B transformed by pYGA2269 was cultured for 3 days at 30 C in a medium containing 0.67 Yeast Nitrogen base, 2 Casamino acids, 2 glucose, 0.001 uracil, 0.0054 adenine, 0.0026 leucine and 0.0038 histidine. The supernatant of the culture was concentrated about 20 folds by an Amicon concentrator, dialyzed against an acetate buffer 20 mM CH₃COONa, pH 4.6 , and adsorbed on an SP Sephadex C 50 column equilibrated with the same buffer. A linear gradient from 0 to 200 mmol was applied to the column, whereby the active fractions eluted were recovered, dialyzed against water, freeze dried and stored for the following experiments. The amino acid composition of the glucoamylase produced by the recombinant yeast was in good agreement with the Gluc 1 having the highest molecular weight of the three glucoamylase molecules produced by the Rhizopus the N terminal sequence and isoelectric point of the glucoamylase were also the same as those of the Rhizopus produced glucoamylase. The apparent molecular weight of the glucoamylase as determined by SDS polyacrylamide electrophoresis was slightly higher than that of Gluc 1. The molecular weight of the glucomaylase was decreased by digestion with endoglycosidase H capable of cutting a certain sugar chain attached to the asparagine residue. However, the Rhizopus produced glucoamylase is not sensitive to the action of endoglycosidase H. The difference between the two glucoamylase molecules with respect to the molecular weight would be ascribable to the differences in the amount and mode of sugar chain attachment. No difference was observed with respect to pH dependency, heat stability or other parameters such as Vmax and Km for soluble starch. The Rhizopus produced glucoamylase had an r s value r activity on raw starch, s activity on gelatinized soluble starch of 0.47 while the value for the recombinant yeast produced glucoamylase was 0.56. It is therefore concluded that as far as the ability to decompose raw starch is concerned, a better glucoamylase preparation can be produced from the recombinant yeast, rather than from the Rhizopus. The Rhizopus produced glucoamylase contains three molecules having different molecular weights, Gluc 1, 2 and 3, the last two of which would be the product of limited proteolysis of the N terminal amino acid sequence of Gluc 1. The behavior of these three molecules in adsorption to starch was investigated. A sample of enzyme solution was mixed with an equal volume of raw starch and, after leaving the mixture to stand in iced water for 30 minutes, it was subjected to centrifugation. The supernatant was recovered as a fraction of the enzyme which was not adsorbed on the starch while the precipitate was recovered as a fraction which was adsorbed on starch. Both fractions were analyzed by SDS PAGE Gluc 1 and the recombinant yeast produced glucoamylase were found in the precipitate whereas Gluc 2 and 3 were found in the supernatant. The mixture of Gluc 2 and 3 had an r s value of 0.23, indicating that the Rhizopus glucomaylase lost its ability to be adsorbed onto raw starch by becoming deficient of the N terminal sequence. It is therefore concluded that the N terminal portion has a site which is adsorbed on raw starch. The Rhizopus glucoamylase was capable of hydrolysing gelatinized soluble starch as effectively as the glucoamylase of the full length even if it lost the N terminal portion. When a mixture of glucoamylase solution with gelatinized soluble starch was subjected to column chromatography on Ultrogel AcA 44, both Gluc 1 and the recombinant yeast produced glucomaylase were eluted in the void volumes in the column together with the starch, but neigher Gluc 2 nor 3 was adsorbed on the starch and they were eluted in the corresponding elution volumes. This provided a basis for the conclusion that Gluc 1 and the recombinant yeast produced glucoamylase are capable of adsorption not only on raw starch but also on gelatinized starch.