# Recombinant DNA vectors capable of expressing apoaequorin, apoaequorin peptides and their production, and microorganisms including said vectors.

## Abstract
A gene which codes for the protein apoaequorin is disclosed along with recombinant DNA vectors containing this gene. Apoaequorin is a single polypeptide chain protein which can be isolated from the luminous jellyfish

## Claims
1. Homogeneous peptide of the formula MET THR SER GLU GLN TYR SER VAL LYS LEU THR PRO ASP PHE ASP ASN PRO LYS TRY ILE GLY ARG HIS LYS HIS MET PHE ASN PHE LEU ASP VAL ASN HIS ASN GLY ARG ILE SER LEU ASP GLU MET VAL TYR LYS ALA SER ASP ILE VAL ILE ASN ASN LEU GLY ALA THR PRO GLU GLN ALA LYS ARG HIS LYS ASP ALA VAL GLU ALA PHE PHE GLY GLY ALA GLY MET LYS TYR GLY VAL GLU THR GLU TRY PRO GLU TYR ILE GLU GLY TRY LYS ARG LEU ALA SER GLU GLU LEU LYS ARG TYR SER LYS ASN GLN ILE THR LEU ILE ARG LEU TRY GLY ASP ALA LEU PHE ASP ILE ILE ASP LYS ASP GLN ASN GLY ALA ILE SER LEU ASP GLU TRY LYS ALA TYR THR LYS SER ALA GLY ILE ILE GLN SER SER GLU ASP CYS GLU GLU THR PHE ARG VAL CYS ASP ILE ASP GLU SER GLY GLN LEU ASP VAL ASP GLU MET THR ARG GLN HIS LEU GLY PHE TRY TYR THR MET ASP PRO ALA CYS GLU LYS LEU TYR GLY GLY ALA VAL PRO.

## Description
This invention relates to the field of genetic engineering and more particularly to the insertion of genes for the protein apoaequorin into recombinant DNA vectors and to the production of apoaequorin in recipient strains of microorganisms. Apoaequorin is a single polypeptide chain protein which can be isolated from the luminous jellyfish Natural apoaequorin is not a single compound but rather represents a mixture of molecular species. When pure natural aequorin, representing that of many thousands of individual Furthermore, it is difficult to produce sufficient aequorin or apoaequorin from jellyfish or other natural sources to provide the amounts necessary for use in bioluminescence assays. Accordingly, an improved means for producing apoaequorin in sufficient quantities for commercial utilization is greatly needed. Recently developed techniques have made it possible to employ microorganisms, capable of rapid and abundant growth, for the synthesis of commercially useful proteins and peptides, regardless of their source in nature. These techniques make it possible to genetically endow a suitable microorganism with the ability to synthesize a protein or peptide normally made by another organism. The technique makes use of a fundamental relationship which exists in all living organisms between the genetic material, usually DNA, and the proteins synthesized by the organism. This relationship is such that the amino acid sequence of the protein is reflected in the nucleotide sequence oif the DNA. There are one or more trinucleotide sequence groups specifically related to each of the twenty amino acids most commonly occuring in proteins. The specific relationship between each given trinucleotide sequence and its corresponding amino acid constitutes the genetic code. The genetic code is believed to be the same or similar for all living organisms. As a consequence, the amino acid sequence of every protein or peptide is reflected by a corresponding nucleotide sequence, according to a well understood relationship. Furthermore, this sequence of nucleotides can, in principle, be translated by any living organism. Key Each 3 letter triplet represents a trinucleotide of DNA having a 5 end on the left and a 3 end on the right. The letters stand for the purine or pyrimidine bases forming the nucleotide sequence. A adenine G guanine C cytosine J A or G K T or C L A, T, C or G M A, C or T T Thymine X T or C if Y is A or G X C if Y is C or T Y A, G, C or T if X is C Y A or G if X is T W C or A if Z is C or T W C if Z is C or T Z A, G, C or T if W is G Z A or G if W is A QR TC if S is A, G, C or T QR A, G if S is T or C S A, G, C or T if QR is TC S T or C if QR is AG The trinucleotides of Table 1, termed codons, are presented as DNA trinucleotides, as they exist in the genetic material of a living organism. Expression of these codons in protein synthesis requires intermediate formation of messenger RNA mRNA , as described more fully, infra. The mRNA codons have the same sequences as the DNA codons of Table 1, except that uracil is found in place of thymine. Complementary trinucleotide DNA sequences having opposite strand polarity are functionally equivalent to the condons of Table 1, as is understood in the art. An important and well known feature of the genetic code is its redundancy, whereby, for most of the amino acids used to make proteins, more than one coding nucleotide triplet may be employed. Therefore, a number of different nucleotide sequences may code for a given amino acid sequence. Such nucleotide sequences are considered functionally equivalent since they can result in the production of the same amino acid sequence in all organisms, although certain strains may translate some sequences more efficiently than they do others. Occasionally, a methylated variant of a purine or pyrimidine may be found in a given nucleotide sequence. Such methylations do not affect the coding relationship in any way. In its basic outline, a method of endowing a microorganism with the ability to synthesize a new protein involves three general steps 1 isolation and purification or chemical synthesis of the specific gene or nucleotide sequence containing the genetically coded information for the amino acid sequence of the desired protein, 2 recombination of the isolated nucleotide sequence with an appropriate vector, typically the DNA of a bacteriophage or plasmid, and 3 transfer of the vector to the appropriate microorganism and selection of a strain of the recipient microorganism containing the desired genetic information. A fundamental difficulty encountered in attempts to exploit commercially the above described process lies in the first step, the isolation and purification of the desired specific genetic information. DNA exists in all living cells in the form of extremely high molecular weight chains of nucleotides. A cell may contain more than 10,000 structural genes, coding for the amino acid sequences of over 10,000 specific proteins, each gene having a sequence many hundreds of nucleotides in length. For the most part, four different nucleotide bases make up all the existing sequences. These are adenine A , guanine G , cytosine C , and thymine T . The long sequences comprising the structural genes of specific proteins are consequently very similar in overall chemical composition and physical properties. The separation of one such sequence from the plethora of other sequences present in isolated DNA cannot ordinarily be accomplished by conventional physical and chemical preparative methods. Two general methods have been used in the prior art to accomplish step 1 in the above described general procedure. The first method is sometimes referred to as the shotgun technique. The DNA of an organism is fragmented into segments generally longer than the desired nucleotide sequence. Step 1 of the above described process is essentially by passed. The DNA fragments are immediately recombined with the desired vector, without prior purification of specific sequences. Optionally, a crude fractionation step may be interposed. The selection techniques of microbial genetics are relied upon to select, from among all the possibilities, a strain of microorganism containing the desired genetic information. The shotgun procedure suffers from two major disadvantages. More importantly, the procedure can result in the transfer of hundreds of unknown genes into recipient microorganisms, so that during the experiment, new strains are created, having unknown genetic capabilities. Therefore, the use of such a procedure could create a hazard for laboratory workers and for the environment. A second disadvantage of the shotgun method is that it is extremely inefficient for the production of the desired strain, and is dependent upon the use of a selection technique having sufficient resolution to compensate for the lack of fractionation in the first step. However, methods of overcoming these disadvantages exist, as will become apparent in later sections of this application. The second general method takes advantage of the fact that the total genetic information in a cell is seldom, if ever, expressed at any given time. In particular, the differentiated tissues of higher organisms may be synthesizing only a minor portion of the proteins which the organism is capable of making at any one time. In extreme cases, such cells may be synthesizing predominantly one protein. In such extreme cases, it has been possible to isolate the nucleotide sequence coding for the protein in question by isolating the corresponding messenger RNA from the appropriate cells. Messenger RNA functions in the process of converting the nucleotide sequence information of DNA into the amino acid sequence structure of a protein. In the first step of this process, termed transcription, a local segment of DNA having a nucleotide sequence which specifies a protein to be made, is copied into RNA. RNA is a polynucleotide similar to DNA except that ribose is substituted for deoxyribose and uracil is used in place of thymine. The nucleotide bases in RNA are capable of entering into the same kind of base pairing relationships that are known to exist between the complementary strands of DNA. A and U T are complementary, and G and C are complementary. The RNA transcript of a DNA nucleotide sequence will be complementary to the copied sequence. Such RNA is termed messenger RNA mRNA because of its status as intermediary between the genetic apparatus of the cell and its protein synthesizing apparatus. Generally, the only mRNA sequences present in the cell at any given time are those which correspond to proteins being actively syunthesized at that time. Therefore, a differentiated cell whose function is devoted primarily to the synthesis of a single protein will contain primarily the RNA species corresponding to that protein. In those instances where it is feasible, the isolation and purification of the appropriate nucleotide sequence coding for a given protein can be accomplished by taking advantage of the specialized synthesis of such protein in differentiated cells. A major disadvantage of the foregoing procedure is that it is applicable only in the relatively rare instances where cells can be found engaged in synthesizing primarily a single protein. The majority of proteins of commercial intest are not synthesized in such a specialized way. The desired proteins may be one of a hundred or so different proteins being produced by the cells of a tissue or organism at a given time. Nevertheless, the mRNA isolation technique is useful since the set of RNA species present in the cell usually represents only a fraction of the total sequences existing in the DNA, and thus provides an initial purification. In a more recent development, U.S. Patent 4,363,877 provides a process whereby nucleotide sequences can be isolated and purified even when present at a frequency as low as 2 of a heterogeneous population of mRNA sequences. Furthermore, the method may be combined with known methods of fractionating mRNA to isolate and purify sequences present in even lower frequency in the total RNA population as initially isolated. The method is generally applicable to mRNA species extracted from virtually any organism and therefore provides a powerful basic tool for the ultimate production of proteins of commercial and research interest, in useful quantities. The process takes advantage of certain structural features of mRNA and DNA, and makes use of certain enzyme catalyzed reactions. The nature of these reactions and structural details as they are understood in the prior art are described herein and are further detailed in the patent. The symbols and abbreviations used herein are set forth in the following table. In its native configuration, DNA exists in the form of paired linear polynucleotide strands. The complementary base pairing relationships described above exist between the paired strands such that each nucleotide base of one strand exists opposite its complement on the other strand. The entire sequence of one strand is mirrored by a complementary sequence on the other strand. If the strands are separate, it is possible to synthesize a new partner strand, starting from the appropriate precursor monomers. The sequence of addition of the monomers starting from one end is determined by, and complementary to, the sequence of the original intact polynucleotide strand, which thus serves as a template for the synthesis of its complementary partner. The synthesis of mRNA corresponding to a specific nucleotide sequence of DNA is understood to follow the same basic principle. Therefore a specific mRNA molecule will have a sequence complementary to one strand of DNA and identical to the sequence of the opposite DNA strand, in the region transcribed. Enzymatic mechanisms exist within living cells which permit the selective transcription of a particular DNA segment containing the nucleotide sequence for a particular protein. Consequently, isolating the mRNA which contains the nucleotide sequence coding for the amino acid sequence of a particular protein is equivalent to the isolation of the same sequence, or gene, from the DNA itself. If the mRNA is retranscribed to form DNA complementary thereto cDNA , the exact DNA sequence is thereby reconstituted and can, by appropriate techniques, be inserted into the genetic material of another organism. The two complementary versions of a given sequence are therefore inter convertible and functionally equivalent to each other. The nucleotide subunits of DNA and RNA are linked together by phosphodiester bonds between the 5 position of one nucleotide sugar and the 3 position of its next neighbor. Reiteration of such linkages produces a linear polynucleotide which has polarity in the sense that one end can be distinguished from the other. The 3 end may have a free 3 hydroxyl, or the hydroxyl may be substituted with a phosphate or a more complex structure. The same is true of the 5 end. In eucaryotic organisms, i.e., those having a defined nucleus and mitotic apparatus, the synthesis of functional mRNA usually includes the addition of polyadenylic acid to the 3 end of the mRNA. Messenger RNA can therefore be separated from other classes of RNA isolated from an eucaryotic organism by column chromatography on cellulose to which is attached polythymidylic acid. See Aviv, H. and Leder, P., Reverse transcriptase catalyzes the synthesis of DNA complementary to an RNA template strand in the presence of the RNA template, a primer which may be any complementary oligo or polynucleotide having a 3 hydroxyl, and the four deoxynucleoside triphosphates, dATP, dGTP, dCTP, and dTTP. The reaction is initiated by the non covalent association of the oligodeoxynucleotide primer near the 3 end of mRNA followed by stepwise addition of the appropriate deoxynucleotides, as determined by base pairing relationships with the mRNA nucleotide sequence, to the 3 end of the growing chain. The product molecule may be described as a hairpin structure in which the original RNA is paired by hydrogen bonding with a complementary strand of DNA partly folded back upon itself at one end. The DNA and RNA strands are not covalently joined to each other. Reverse transcriptase is also capable of catalyzing a similar reaction using a single stranded DNA template, in which case the resulting product is a double stranded DNA hairpin having a loop of single stranded, DNA joining one set of ends. See Aviv, H. and Leder, P., Restriction endonucleases are enzymes capable of hydrolyzing phosphodiester bonds in DNA, thereby creating a break in the continuity of the DNA strand. If the DNA is in the form of a closed loop, the loop is converted to a linear structure. The principal feature of a restriction enzyme is that its hydrolytic action is exerted only at a point where a specific nucleotide sequence occurs. Such a sequence is termed the restriction site for the restriction endonuclease. Restriction endonucleases from a variety of sources have been isolated and characterized in terms of the nucleotide sequence of their restriction sites. When acting on double stranded DNA, some restriction endonucleases hydrolyze the phosphodiester bonds on both strands at the same point, producing blunt ends. Others catalyze hydrolysis of bonds separated by a few nucleotides from each other, producing free single stranded regions at each end of the cleaved molecule. Such single stranded ends are self complementary, hence cohesive, and may be used to rejoin the hydrolyzed DNA. Since any DNA susceptible to cleavage by such an enzyme must contain the same recognition site, the same cohesive ends will be produced, so that it is possible to join heterogeneous sequences of DNA which have been treated with restriction endonuclease to other sequences similarly treated. See Roberts, R.J., It has been observed that restriction sites for a given enzyme are relatively rare and are nonuniformly distributed. Whether a specific restriction site exists within a given segment is a matter which must be empirically determined. However, there is a large and growing number of restriction endonucleases, isolated from a variety of sources with varied site specificity, so that there is a reasonable probability that a given segment of a thousand nucleotides will contain one or more restriction sites. For general background see Watson, J.D., Accordingly, it is an object of this invention to provide a microorganism capable of providing useful quantities of apoaequorin. It is a further object of this invention to provide a recombinant DNA vector capable of being inserted into a microorganism and expressing apoaequorin. It is still another object of this invention to provide a DNA segment of defined structure that can be produced synthetically or isolated from natural sources and that can be used in the production of the desired recombinant DNA vectors. It is yet another object of this invention to provide a peptide that can be produced synthetically in a laboratory or by microorganism that will mimic the activity of natural apoaequorin. These and other objects of the invention as will hereinafter become more readily apparent have been accomplished by providing a homogeneous peptide selected from 1 compounds of DNA molecules, recombinant DNA vectors, and modified microorganisms comprising a nucleotide sequence GTL₁ AAJ₂ XTY₃ ACL₄ CCL or QRS ₅ GAK₆ TTK₇ AAK or GAK ₈ AAK₉ CCL₁₀ AAJ or WGZ ₁₁ TGG₁₂ ATM₁₃ GGL₁₄ WGZ₁₅ CAK₁₆ AAJ₁₇ CAK₁₈ ATG₁₉ TTK₂₀ AAK₂₁ TTK₂₂ XTY₂₃ GAK₂₄ GTL₂₅ AAK₂₆ CAK₂₇ AAK₂₈ GGL₂₉ AAJ₃₀ ATM₃₁ QRS₃₂ XTY₃₃ GAK₃₄ GAJ₃₅ ATG₃₆ GTL₃₇ TAK₃₈ AAJ₃₉ GCL₄₀ QRS₄₁ GAK₄₂ ATM₄₃ GTL₄₄ ATM₄₅ AAK₄₆ AAK₄₇ XTY₄₈ GGL₄₉ GCL₅₀ ACL₅₁ CCL₅₂ GAJ₅₃ CAJ₅₄ GCL₅₅ AAJ₅₆ WGZ₅₇ CAK₅₈ AAJ₅₉ GAK₆₀ GCL₆₁ GTL₆₂ GAJ₆₃ GCL₆₄ TTK₆₅ TTK₆₆ GGL₆₇ GGL₆₈ GCL₆₉ GGL₇₀ ATG₇₁ AAJ₇₂ TAK₇₃ GGL₇₄ GTL₇₅ GAJ₇₆ GAK or GAJ ₇₈ TGG₇₉ CCL₈₀ GCL or GAJ ₈₁ TAK₈₂ ATM₈₃ GAJ₈₄ GGL₈₅ TGG₈₆ AAJ₈₇ AAJ or WGZ ₈₈ XTY₈₉ GCL₉₀ ACL₉₁ GAK, GAJ, or TGK ₉₂ GAJ₉₃ XTY₉₄ GAJ₉₅ AAJ or WGZ ₉₆ TAK₉₇ GCL or QRS ₉₈ AAJ₉₉ AAK₁₀₀ CAJ or GAJ ₁₀₁ ATM or CCL ₁₀₂ ACL₁₀₃ XTY₁₀₄ ATM₁₀₅ WGZ₁₀₆ ATM or XTY ₁₀₇ TGG₁₀₈ GGL₁₀₉ GAK₁₁₀ GCL₁₁₁ XTY₁₁₂ TTK₁₁₃ GAK₁₁₄ ATM₁₁₅ ATM or GTL ₁₁₆ GAK₁₁₇ AAJ₁₁₈ GAK₁₁₉ CAJ₁₂₀ AAK₁₂₁ GGL₁₂₂ GCL₁₂₃ ATM₁₂₄ ACL₁₂₅ XTY₁₂₆ QRS or GAK ₁₂₇ GAJ₁₂₈ TGG₁₂₉ AAJ₁₃₀ GCL₁₃₁ TAK₁₃₂ ACL₁₃₃ AAJ₁₃₄ QRS or GCL ₁₃₅ GCL₁₃₆ GGL₁₃₇ ATM₁₃₈ ATM₁₃₉ CAJ₁₄₀ ACL or QRS ₁₄₁ QRS₁₄₂ GAJ₁₄₃ GAJ or GAK ₁₄₄ TGK₁₄₅ GAJ₁₄₆ GAJ₁₄₇ ACL₁₄₈ TTK₁₄₉ WGZ₅₀ GTL₁₅₁ TGK₁₅₂ GAK₁₅₃ ATM₁₅₄ GAK₁₅₅ GAJ₁₅₆ QRS₁₅₇ GGL₁₅₈ CAJ₁₅₉ XTY₁₆₀ GAK₁₆₁ GTL₁₆₂ GAK₁₆₃ GAJ₁₆₄ ATG₁₆₅ ACL₁₆₆ WGZ₁₆₇ CAJ₁₆₈ CAK₁₆₉ XTY₁₇₀ GGL₁₇₁ TTK₁₇₂ TGG₁₇₃ TAK₁₇₄ ACL₁₇₅ ATG₁₇₆ GAK₁₇₇ CCL₁₇₈ GCL₁₇₉ TGK₁₈₀ GAJ₁₈₁ AAJ₁₈₂ XTY₁₈₃ TAK₁₈₄ GGL₁₈₅ GGL₁₈₆ GCL₁₈₇ GTL₁₈₈ CCL₁₈₉ wherein A is deoxyadenyl, G is deoxyguanyl, C is deoxycytosyl, T is deoxythymidyl, J is A or G K is T or C L is A, T, C, or G M is A, C or T X is T or C, if the succeeding Y is A or G, and C if the succeeding Y is C or T Y is A, G, C or T, if the preceding X is C, and A or G if the preceding X is T W is C or A, if the succeeding Z is G or A, and C if the succeeding Z is C or T Z is A, G, C or T, if the preceding W is C, and A or G if the preceding W is A QR is TC, if the succeeding S is A, G, C or T, and AG if the succeeding S is T or C S is A, G, C or T, if the preceding QR is TC, and T or C if the preceding QR is AG and subscript numerals refer to the amino acid position in apoaequorin, for which the nucleotide sequence corresponds according to the genetic code, the amino acid positions being numbered from the amino end or a nucleotide sequence coding a peptide previously mentioned, are also provided for use in carrying out preferred aspects of the invention relating to the production of such peptides by the techniques of genetic engineering. The accompanying figures and drawings are provided to demonstrate the results obtained in the specific examples which illustrate the invention but are not considered to be limiting thereof. The present inventor has obtained for the first time a recombinant DNA vector capable of expressing the protein apoaequorin in a microorganism and has additionally identified for the first time the amino acid sequence of apoaequorin, thereby providing access to homogeneous apoaequorin. Using this information a variety of recombinant DNA vectors capable of providing homogeneous apoaequorin in reasonble quantities are obtained. Additional recombinant DNA vectors can be produced using standard techniques of recombinant DNA technology. A transformant expressing apoaequorin has also been produced as an example of this technology. The amino acid sequence of apoaequorin is shown in TABLE 3. Since there is a known and definite correspondence between amino acids in a peptide and the DNA sequence that codes for the peptide, the DNA sequence of a DNA or RNA molecule coding for apoaequorin or any of the modified peptides later discussed can readily be derived from this amino acid sequence, and such a sequence of nucleotides is shown in TABLE 4. Since the DNA sequence of the gene has been fully identified, it is possible to produce a DNA gene entirely by synthetic chemistry, after which the gene can be inserted into any of the many available DNA vectors using known techniques of recombinant DNA technology. Thus the present invention can be carried out using reagents, plasmids, and microorganism which are freely available and in the public domain at the time of filing of this patent application. For example, nucleotide sequences greater than 100 bases long can be readily synthesized on an Applied Biosystems Model 380A DNA Synthesizer as evidenced by commercial advertising of the same e.g., Genetic Engineering News, November December 1984, p. 3 . Such obigonucleotides can readily be spliced using, among others, the techniques described later in this application to produce any nucleotide sequence described herein. Furthermore, automated equipment is also available that makes direct synthesis of any of the peptides disclosed herein readily available. In the same issue of Genetic Engineering News mentioned above, a commercially available automated peptide synthesizer having a coupling efficiency exceeding 99 is advertised page 34 . Such equipment provides ready access to the peptides of the invention, either by direct synthesis or by synthesis of a series of fragments that can be coupled using other known techniques. In addition to the specific peptide sequences shown in Table 3, other peptides based on these sequences and representing minor variations thereof will have the biological activity of apoaequorin. For example, up to 15 amino acids can be absent from either or both terminals of the sequence given without losing luciferin and calcium binding ability. Likewise, up to 10 additional amino acids can be present at either or both terminals. These variations are possible because the luciferin and calcium binding sites involve the amino acids in the middle of the given sequences. For example, the luciferin binding site appears to involve amino acids 40 100. Since the terminals are relatively unimportant for biological activity, the identity of added amino acids is likewise unimportant and can be any of the amino acids mentioned herein. Experimental data is available to verify that added amino acids at the amine terminal do not have a significant effect on bioluminscence. Nevertheless, preferred compounds are those which more closely approach the specific formulas given with 10 or fewer, more preferably 5 or fewer, absent amino acids being preferred for either terminal and 7 or fewer, more preferably 4 or fewer, additional amino acids being preferred for either terminal. Within the central portion of the molecule, replacement of amino acids is more restricted in order that biological activity can be maintained. However, all of the points of microheterogenity shown in Table 3 or Table 4 represent biologically functional replacements and any combination of the indicated replacements will represent a functional molecule. In both Tables, the main line Table 3 or first entry Table 4 represents the more prevelant amino acid or nucleotide for that location and is preferred. In addition minor variations of the previously mentioned peptides and DNA molecules are also contemplated as being equivalent to those peptides and DNA molecules that are set forth in more detail, as will be appreciated by those skilled in the art. For example, it is reasonable to expect that an isolated replacement of a leucine with an isoleucine or valine, an aspartate with a glutamate, a threonine with a serine, or a similar replacement of an amino acid with a structurally related amino acid will not have a major effect on the biological activity of the resulting molecule, especially if the replacement does not involve an amino acid at a binding site. Whether a change results in a functioning peptide can readily be determined by incubating the resulting peptide with a luciferin followed by contact with calcium ions. Examples of this process are described later in detail. If light is emitted, the replacement is immaterial, and the molecule being tested is equivalent to those of Table 3. Peptides in which more than one replacement has taken place can readily be tested in the same manner. DNA molecules that code for such peptides can readily be determined from the list of codons in Table 1 and are likewise contemplated as being equivalent to the DNA sequence of Table 4. In fact, since there is a 1 1 correspondence between DNA codons and amino acids in a peptide, any discussion in this application of a replacement or other change in a peptide is equally applicable to the corresponding DNA sequence or to the DNA molecule, recombinant vector, or transformed microorganism in which the sequence is located and vice versa . In addition to the specific nucleotides listed in Table 4, DNA or corresponding RNA molecules of the invention can have additional nucleotides preceeding or following those that are specifically listed. For example, poly A can be added to 3 terminal, short e.g., fewer than 20 nucleotides sequence can be added to either terminal to provide a terminal sequence corresponding to a restriction endonuclease site, stop codons can follow the peptide sequence to terminate transcription, and the like. Additionally, DNA molecules containing a promoter region or other control region upstream from the gene can be produced. All DNA molecules containing the sequences of the invention will be useful for at least one purpose since all can minimally be fragmented to produce oligonucleotide probes of the type described later in this specification. Peptides of the invention can be prepared for the first time as homogeneous preparations, either by direct synthesis or by using a cloned gene as described herein. By homogeneous is meant, when referring to a peptide or DNA sequence, that the primary molecular structure i.e., the sequence of amino acids or nucleotides of substantially all molecules present in the composition under consideration is identical. The term substantially as used in the preceeding sentence preferably means at least 95 by weight, more preferably at least 99 by weight, and most preferably at least 99.8 by weight. The presence of fragments derived from entire molecules of the homogeneous peptide or DNA sequence, if present in no more than 5 by weight, preferably 1 by weight, and more preferably 0.2 by weight, is not to be considered in determining homogeity since the term homogeneous relates to the presence of entire molecules and fragments thereof have a single defined structure as opposed to mixtures such as those that occur in natural apoaequorin in which several molecules of similar molecular weight are present but which differ in their primary molecular structure. The term isolated as used herein refers to pure peptide, DNA, or RNA separated from other peptides, DNAs, or RNAs, respectively, and being found in the presence of if anything only a solvent, buffer, ion or other component normally present in a biochemical solution of the same. Isolated does not encompass either natural materials in their native state or natural materials that have been separated into components e.g., in an acylamide gel but not obtained either as pure substances or as solutions. The term pure as used herein preferably has the same numerical limits as substantially immediately above. The phrase replaced by or replacement as used herein does not necessarily refer to any action that must take place but to the peptide that exists when an indicated replacement amino acid is present in the same position as the amino acid indicated to be present in a different formula e.g., when serine is present at position 5 instead of proline. Salts of any of the peptides described herein will naturally occur when such peptides are present in or isolated from aqueous solutions of various pHs. All salts of peptides having the indicated biological activity are considered to be within the scope of the present invention. Examples include alkali, alkaline earth, and other metal salts of carboxylic acid residues, acid addition salts e.g., HCl of amino residues, and zwitter ions formed by reactions between carboxylic acid and amino residues within the same molecule. The invention has specifically contemplated each and every possible variation of peptide or nucleotide that could be made by selecting combinations based on the possible amino acid and codon choices listed in Table 3 and Table 4, and all such variations are to be considered as being specifically disclosed. In a preferred embodiment of the invention, genetic information encoded as mRNA is obtained from It is preferred to use a cell extract from the light emitting organs of an Various forms of RNA may be employed such as polyadenylated, crude or partially purified messenger RNA, which may be heterogeneous in sequence and in molecular size. The selectivity of the RNA isolation procedure is enhanced by any method which results in an enrichment of the desired mRNA in the heterodisperse population of mRNA isolated. Any such prepurification method may be employed in preparing a gene of the present invention, provided that the method does not introduce endonucleolytic cleavage of the mRNA. Prepurification to enrich for desired mRNA sequences may also be carried out using conventional methods for fractionating RNA, after its isolation from the cell. Any technique which does not result in degradation of the RNA may be employed. The techniques of preparative sedimentation in a sucrose gradient and gel electrophoresis are especially suitable. The mRNA must be isolated from the source cells under conditions which preclude degradation of the mRNA. The action of RNase enzymes is particularly to be avoided because these enzymes are capable of hydrolytic cleavage of the RNA nucleotide sequence. A suitable method for inhibiting RNase during extraction from cells involves the use of 4 M guanidinium thiocyanate and 1 M mercaptoethanol during the cell disruption step. In addition, a low temperature and a pH near 5.0 are helpful in further reducing RNase degradation of the isolated RNA. Generally, mRNA is prepared essentially free of contaminating protein, DNA, polysaccharides and lipids. Standard methods are well known in the art for accomplishing such purification. RNA thus isolated contains non messenger as well as messenger RNA. A convenient method for separating the mRNA of eucaryotes is chromatography on columns of oligo dT cellulose, or other oligonucleotide substituted column material such as poly U Sepharose, taking advantage of the hydrogen bonding specificity conferred by the presence of polyadenylic acid on the 3 end of eucaryotic mRNA. The next step in most methods is the formation of DNA complementary to the isolated heterogeneous sequences of mRNA. The enzyme of choice for this reaction is reverse transcriptase, although in principle any enzyme capable of forming a faithful complementary DNA copy of the mRNA template could be used. The reaction may be carried out under conditions described in the prior art, using mRNA as a template and a mixture of the four deoxynucleoside triphosphates dATP, dGTP, dCTP and dTTP, as precursors for the DNA strand. It is convenient to provide that one of the deoxynucleoside triphosphates be labeled with a radioisotape, for example ³²P in the alpha position, in order to monitor the course of the reaction, to provide a tag for recovering the product after separation procedures such as chromatography and electrophoresis, and for the purpose of making quantitative estimates of recovery. See Efstratiadis, A., et al, The cDNA transcripts produced by the reverse transcriptase reaction are somewhat heterogeneous with respect to sequences at the 5 end and the 3 end due to variations in the initiation and termination points of individual transcripts, relative to the mRNA template. The variability at the 5 end is thought to be due to the fact that the oligo dT primer used to initiate synthesis is capable of binding at a variety of loci along the polyadenylated region of the mRNA. Synthesis of the cDNA transcript begins at an indeterminate point in the poly A region, and a variable length of poly A region is transcribed depending on the initial binding site of the oligo dT primer. It is possible to avoid this indeterminacy by the use of a primer containing, in addition to an oligo dT tract, one or two nucleotides of the RNA sequence itself, thereby producing a primer which will have a preferred and defined binding site for initiating the transcription reaction. The indeterminacy at the 3 end of the cDNA transcript is due to a variety of factors affecting the reverse transcriptase reaction, and to the possibility of partial degradation of the RNA template. The isolation of specific cDNA transcripts of maximal length is greatly facilitated if conditions for the reverse transcriptase reaction are chosen which not only favor full length synthesis but also repress the synthesis of small DNA chains. Preferred reaction conditions for avian myeloblastosis virus reverse transcriptase are given in the examples section of U.S. Patent 4,363,877 and are herein incorporated by reference. The specific parameters which may be varied to provide maximal production of long chain DNA transcripts of high fidelity are reaction temperature, salt concentration, amount of enzyme, concentration of primer relative to template, and reaction time. The conditions of temperature and salt concentration are chosen so as to optimize specific base pairing between the oligo dT primer and the polyadenylated portion of the RNA template. Under properly chosen conditions, the primer will be able to bind at the polyadenylated region of the RNA template, but non specific initiation due to primer binding at other locations on the template, such as short, A rich sequences, will be substantially prevented. The effects of temperature and salt are interdependent. Higher temperatures and low salt concentrations decrease the stability of specific base pairing interactions. The reaction time is kept as short as possible, in order to prevent non specific initiations and to minimize the opportunity for degradation. Reaction times are interrelated with temperature, lower temperatures requiring longer reaction times. At 42 C, reactions ranging from 1 min. to 10 minutes are suitable. The primer should be present in 50 to 500 fold molar excess over the RNA template and the enzyme should be present in similar molar excess over the RNA template. The use of excess enzyme and primer enhances initiation and cDNA chain growth so that long chain cDNA transcripts are produced efficiently within the confines of the sort incubation times. In many cases, it will be possible to further purify the cDNA using single stranded cDNA sequences transcribed from mRNA. However, as discussed below, there may be instances in which the desired restriction enzyme is one which acts only on double stranded DNA. In these cases, the cDNA prepared as described above may be used as a template for the synthesis of double stranded DNA, using a DNA polymerase such as reverse transcriptase and a nuclease capable of hydrolyzing single stranded DNA. Methods for preparing double stranded DNA in this manner have been described in the prior art. See, for example, Ullrich, A., Shine, J., Chirgwin, J., Pictet, R., Tischer, E., Rutter, W.J. and Goodman, H.M., The choice of restriction enzyme s capable of producing an optimal length nucleotide sequence fragment coding for all or part of the desired protein must be made empirically. If the amino acid sequence of the desired protein is known, it is possible to compare the nucleotide sequence of uniform length nucleotide fragments produced by restriction endonuclease cleavage with the amino acid sequence for which it codes, using the known relationship of the genetic code common to all forms of life. A complete amino acid sequence for the desired protein is not necessary, however, since a reasonably accurate identification may be made on the basis of a partial sequence. Where the amino acid sequence of the desired protein is now known, the uniform length polynucleotides produced by restriction endonuclease cleavage may be used as probes capable of identifying the synthesis of the desired protein in an appropriate The number of restriction enzymes suitable for use depends upon whether single stranded or double stranded cDNA is used. The preferred enzymes are those capable of acting on single stranded DNA, which is the immediate reaction product of mRNA reverse transcription. The number of restriction enzymes now known to be capable of acting on single stranded DNA is limited. The enzymes of HaeIII, HhaI and Hin f I are presently known to be suitable. In addition, the enzyme MboII may act on single stranded DNA. Where further study reveals that other restriction enzymes can act on single stranded DNA, such other enzymes may appropriately be included in the list of preferred enzymes. Additional suitable enzymes include those specified for double stranded cDNA. Such enzymes are not preferred since additional reactions are required in order to produce double stranded cDNA, providing increased opportunities for the loss of longer sequences and for other losses due to incomplete recovery. The use of double stranded cDNA presents the additional technical disadvantage that subsequent sequence analysis is more complex and laborious. For these reasons, single stranded cDNA is prefered, but the use of double stranded DNA is feasible. In fact, the present invention was initially reduced to practice using double stranded cDNA The cDNA prepared for restriction endonuclease treatment may be radioactively labeled so that it may be detected after subsequent separation steps. A preferred technique is to incorporate a radioactive label such as ³²P in the alpha position of one of the four deoxynucleoside triphosphate precursors. Highest activity is obtained when the concentration of radioactive precursor is high relative to the concentration of the non radioactive form. However, the total concentration of any deoxynucleoside triphosphate should be greater than 30 µM, in order to maximize the length of cDNA obtained in the reverse transcriptase reaction. See Efstratiadis, A., Maniatis, T., Kafatos, F.C., Jeffrey, A., and Vournakis, J.N., Fragments which have been produced by the action of a restriction enzyme or combination of two restriction enzymes may be separated from each other and from heterodisperse sequences lacking recognition sites by any appropriate technique capable of separating polynucleotides on the basis of differenes in length. Such methods include a variety of electrophoretic techniques and sedimentation techniques using an ultracentrifuge. Gel electrophoresis is preferred because it provides the best resolution on the basis of polynucleotide length. In addition, the method readily permits quantitative recovery of separated materials. Convenient gel electrophoresis methods have been described by Dingman, C.W., and Peacock, A.C., Prior to restriction endonuclease treatment, cDNA transcripts obtained from most sources will be found to be heterodisperse in length. By the action of a properly chosen restriction endonuclease, or pair of endonucleases, polynucleotide chains containing the desired sequence will be cleaved at the respective restriction sites to yield polynucleotide fragments of uniform length. Upon gel electrophoresis, these will be observed to form a distinct band. Depending on the presence or absence of restriction sites on other sequences, other discrete bands may be formed as well, which will most likely be of different length than that of the desired sequence. Therefore, as a consequence of restriction endonuclease action, the gel electrophoresis pattern will reveal the appearance of one or more discrete bands, while the remainder of the cDNA will continue to be heterodisperse. In the case where the desired cDNA sequence comprises the major polynucleotide species present, the electrophoresis pattern will reveal that most of the cDNA is present in the discrete band. Although it is unlikely that two different sequences will be cleaved by restriction enzymes to yield fragments of essentially similar length, a method for determining the purity of the defined length fragments is desirable. Sequence analysis of the electrophoresis band may be used to detect impurities representing 10 or more of the material in the band. A method for detecting lower levels of impurities has been developed founded upon the same general principles applied in the initial isolation method. The method requires that the desired nucleotide sequence fragment contain a recognition site for a restriction endonuclease not employed in the initial isolation. Treatment of polynucleotide material, eluted from a gel electrophoresis band, with a restriction endonuclease capable of acting internally upon the desired sequence will result in cleavage of the desired sequence into two sub fragments, most probably of unequal length. These sub fragments upon electrophoresis will form two discrete bands at positions corresponding to their respective lengths, the sum of which will equal the length of the polynucleotide prior to cleavage. Contaminants in the original band that are not susceptible to the restriction enzyme may be expected to migrate to the original position. Contaminants containing one or more recognition sites for the enzyme may be expected to yield two or more sub fragments. Since the distribution of recognition sites is believed to be essentially random, the probability that a contaminant will also yield sub fragments of the same size as those of the fragment of desired sequence is extremely low. The amount of material present in any band of radioactively labeled polynucleotide can be determined by quantitative measurement of the amount of radioactivity present in each band, or by any other appropriate method. A quantitative measure of the purity of the fragments of desired sequence can be obtained by comparing the relative amounts of material present in those bands representing sub fgragments of the desired sequence with the total amount of material. Following the foregoing separation or any other technique that isolates the desired gene, the sequence may be reconstituted. The enzyme DNA ligase, which catalyzes the end to end joining of DNA fragments, may be employed for this purpose. The gel electrophoresis bands representing the sub fragments of the desired sequence may be separately eluted and combined in the presence of DNA ligase, under the appropriate conditions. See Sgaramella, V., Van de Sande, J.H., and Khorana, H.G., The efficiency of reconstituting the original sequence from sub fragments produced by restriction endonuclease treatment will be greatly enhanced by the use of a method for preventing reconstitution in improper sequence. This unwanted result is prevented by treatment of the homogeneous length cDNA fragment of desired sequence with an agent capable of removing the 5 terminal phosphate groups on the cDNA prior to cleavage of the homogeneous cDNA with a restriction endonuclease. The enzyme alkaline phosphatase is preferred. The 5 terminal phosphate groups are a structural prerequisite for the subsequent joining action of DNA ligase used for reconstituting the cleaved sub fragments. Therefore, ends which lack a 5 terminal phosphate cannot be covalently joined. The DNA sub fragments can only be joined at the ends containing a 5 phosphate generated by the restriction endonuclease cleavage performed on the isolated DNA fragment. The majority of cDNA transcripts, under the conditions described above, are derived from the mRNA region containing the 5 end of the mRNA template by specifically priming on the same template with a fragment obtained by restriction endonuclease cleavage. In this way, the above described method may be used to obtain not only fragments of specific nucleotide sequence related to a desired protein, but also the entire nucleotide sequence coding for the protein of interest. Double stranded, chemically synthesized oligonuclease linkers, containing the recognition sequence for a restriction endonuclease, may be attached to the ends of the isolated cDNA, to facilitate subsequent enzymatic removal of the gene portion from the vector DNA. See Scheller, R.H., et al, In summary, genetic information can be obtained from Although the sequence of steps set forth above, when used in combination with the knowledge of those skilled in the art of genetic engineering and the previously stated guidelines, will readily enable isolation of the desired gene and its use in recombinant DNA vectors, other methods which lead to the same result are also known and may be used in the preparation of recombinant DNA vectors of this invention. Expression of apoaequorin can be enhanced by including multiple copies of the apoaequorin gene in a transformed host, by selecting a vector known to reproduce in the host, thereby producing large quantities of protein from exogeneous inserted DNA such as pUC8, ptac12, or pIN III ompA1, 2, or 3 , or by any other known means of enhancing peptide expression. In all cases, apoaequorin will be expressed when the DNA sequence is functionally inserted into the vector. By functionally inserted is meant in proper reading frame and orientation, as is well understood by those skilled in the art. Typically, an apoaequorin gene will be inserted downstream from a promotor and will be followed by a stop codon, although production as a hybrid protein followed by cleavage may be used, if desired. In addition to the above general procedures which can be used for preparing recombinant DNA molecules and transformed unicellular organisms in accordance with the practices of this invention, other known techniques and modifications thereof can be used in carrying out the practice of the invention. In particular, techniques relating to genetic engineering have recently undergone explosive growth and development. Many recent U.S. patents disclose plasmids, genetically engineering microorganisms, and methods of conducting genetic engineering which can be used in the practice of the present invention. For example, U.S. Pastent 4,273,875 discloses a plasmid and a process of isolating the same. U.S. Patent 4,304,863 discloses a process for producing bacteria by genetic engineering in which a hybrid plasmid is constructed and used to transform a bacterial host. U.S. Patent 4,419,450 discloses a plasmid useful as a cloning vehicle in recombinant DNA work. U.S. Patent 4,362,867 discloses recombinant cDNA construction methods and hybrid nucleotides produced thereby which are useful in cloning processes. U.S. Patent 4,403,036 discloses genetic reagents for generating plasmids containing multiple copies of DNA segments. U.S. Patent 4,363,877 discloses recombinant DNA transfer vectors. U.S. Patent 4,356,270 discloses a recombinant DNA cloning vehicle and is a particularly useful disclosure for those with limited experience in the area of genetic engineering since it defines many of the terms used in genetic engineering and the basic processes used therein. U.S. Patent 4,336,336 discloses a fused gene and a method of making the same. U.S. Patent 4,349,629 discloses plasmid vectors and the production and use thereof. U.S. Patent 4,332,901 discloses a cloning vector useful in recombinant DNA. Although some of these patents are directed to the production of a particular gene product that is not within the scope of the present invention, the procedures described therein can easily be modified to the practice of the invention described in this specification by those skilled in the art of genetic engineering. All of these patents as well as all other patents and other publications cited in this disclosure are indicative of the level of skill of those skilled in the art to which this invention pertains and are all herein incorporated by reference. The implications of the present invention are significant in that unlimited supplies of apoaequorin will become available for use in the development of luminescent immunoassays or in any other type of assay utilizing aequorin as a marker. Methods of using apoaequorin in a bioluminiscent assay are disclosed in Serial Number 541,405, filed October 13, 1983, and commonly assigned, which is herein incorporated by reference. Transferring the apoaequorin cDNA which has been isolated to other expression vectors will produce constructs which improve the expression of the apoaequorin polypeptide in Particularly contemplated is the isolation of genes from these and related organisms that express photoproteins using oligonucleotide probes based on the prinicpal and variant nucleotide sequences disclosed herein. Such probes can be considerably shorter than the entire sequence but should be at least 10, preferably at least 14, nucleotides in length. Longer oligonucleotides are also useful, up to the full length of the gene. Both RNA and DNA probes can be used. In use, the probes are typically labelled in a detectable manner e.g., with 32p, ³H, biotin, or avidin and are incubated with single stranded DNA or RNA from the organism in which a gene is being sought. Hybridization is detected by means of the label after single stranded and double stranded hydridized DNA or DNA RNA have been separated typically using nitrocellulose paper . Hybridization techniques suitable for use with oligonucleotides are well known. Although probes are normally used with a detectable label that allows easy identification, unlabeled oligonucleotides are also useful, both as precursors of labeled probes and for use in methods that provide for direct detection of double stranded DNA or DNA RNA . Accordingly, the term oligonucleotide probe refers to both labeled and unlabeled forms. Particularly preferred are oligonucleotides obtained from the region coding for amino acids 40 through 110 of the peptide sequences described herein, since these are the amino acids involved in binding to luciferin. Coelenterate luciferin is found in and binds to photoproteins from all the organisms listed in Table 5, and it is contemplated that oligonucleotides as described herein will be useful as probes in isolating photoprotein genes from all these species. The invention now being generally described, it will be more readily understood by reference to the following examples which are included for purposes of illustration only and are not intended to limit the invention unless so stated. Aequorin was purified according to the method of Blinks et al Blinks, J. R., P. H. Mattingly, B. R. Jewell, M. van Leeuwen, G. C. Harrer, and D. G. Allen, The purification of aequorin and green fluorescent protein GFP was done in Athens, Georgia Steps 5 9 . All steps were performed at 0 4 C. Aequorin containing fractions were stored at 80 C between steps aequorin seems to be stabile to freezing and thawing irrespective of protein concentration. Generally the pooled material from 6 to 10 initial G 50 steps was run on this column. Overall yield was improved by doing this, as was efficiency. This step was performed exactly as described by Blinks et al 1978 . After the column was loaded, the GFP was selectively eluted with a pH step 5 mM Na Ac, 5 mM EDTA, pH 4.75 . Aequorin was then eluted in a linear NaCl gradient in 10 mM EDTA, pH 5.5 500 ml total volume . The GFP was made 10 mM in Tris and the pH raised to 8.0 for storage at 80 until further purification. The aequorin pool was concentrated via ultrafiltration Amicon YM 10 membrane in preparation for the next step. Aequorin yield 80 . Amino acid sequence analysis was performed using automated Edman Degradation Edman and Begg, 1967 . The sequence analysis of relatively large amounts of protein or peptide 10 nmol or more was carried out using a Model 890 B Beckman sequencer Duke University updated as described by Bhown et. al. 1980 and employing a 0.55 M Quadrol program with polybrene Tarr et. al., 1978 . Two peptides, M3 and M5, which were small or appeared to wash out of the cup with the Quadrol method, were sequenced using a program adapted for dimethylallylamine buffer and polybrene as suggested by Klapper et al. 1978 . Phenylthiohydantoin PTH derivatives of amino acids were identified using reverse phase HPLC chromatography on a DuPont Zorbay ODS column essentially as described by Hunkapiller and Hood 1978 . Peptides which were available at the 2 10 nmol level were sequenced on a Model 890 C Beckman sequencer University of Washington using a program for use with 0.1 M Quadrol Brauer et al., 1975 and polybrene. PTH amino acids were identified using the reverse phase HPLC system described by Ericsson et. al. 1977 . An applied Biosystems Model 470 A gas phase sequencer University of Washington Hunkapiller et. al., 1983 was used for sequence analysis when there was less than 1.5 nmol of peptide available. PTH amino acids from the gas phase instrument were identified using an IBM Cyano column as described by Hunkapiller and Hood 1983 . Restriction enzymes were purchased from Bethesda Research Laboratories, New England Bio Labs and International Biotechnologies, Inc. and used according to conditions described by the supplier. RNasin and reverse transcriptase were obtained from Biotech and Life Sciences, respectively. Terminal transferase was purchased from PL Biochemicals. Coelenterate luciferin was synthesized as described Hori, K. Anderson, J.M., Ward, W.W. and Cormier, M.J. RNA was isolated according to the method of Kim et al Kim, Y J, Shuman, J., Sette, K. and Przybyla, A., Poly A RNA 1 µg and poly A RNA 20 µg were translated using the rabbit reticulocyte Double stranded cDNA was synthesized from total The The The filters were incubated at 55 C for 12 20 hours in 3 ml filter of a prehybridization solution 10X NET, 0.1 SDS, 3X Denhardt s after first wetting them in 1XSSC. The solution was poured from the hybridization bag and replaced with 1 ml filter of the hybridization solution 10XNET, 0.1 SDS, 3X Denhardt s, 1x10⁶ cpm ³²P labelled 17 mers per filter . The hybridization was carried out for 24 hours at 37 C after which the filters were washed four times in 10XSSC at 4 C for 10 min. The filters were air dried and then wrapped in plastic wrap. Kodak XAR 5 film was exposed to the filters at 70 C using a DuPont Cronex intensifying screen. The primary translation products immunoprecipitated with anti aequorin migrated on the SDS PAGE gel with an apparent molecular weight 23,400 daltons, lane 4 slightly greater than that for native aequorin isolated from The proteins immunoprecipitated from the poly A RNA translation migrated as a doublet or even a triplet lane 4, Fig. 1 if one studies the original autoradiogram. This result can be interpreted in two ways. Firstly, multiple apoaequorin genes may exist in The The Six transformants were identified which contained plasmids having inserts that hybridized to the synthetic oligonucleotides. The restriction map of the plasmid containing the largest Pst I insert, pAEQ1, is shown in Fig. 2. No hybridization of the synthetic oligonucleotides would occur if pAEQ1 was digested with The kinetics of formation of photoprotein activity from extracts of pAEQ1 is similar to that observed with native, mixed apoaequorin as shown in Fig. 3. Requirements for the formation of photoprotein activity in this extract is also indentical to that observed when authentic apoaequorin is used. As Fig. 3 shows, dissolved O₂ is required. Furthermore, the elimination of either β mercaptoethanol or coelenterate luciferin from the reaction mixture results in zero production of Ca² dependent photoprotein activity. Injection of the active component into Ca² free buffers produced no luminescence. The subsequent addition of Ca² resulted in a luminescence flash. To further characterize the active component in extracts of the pAEQ1 containing transformant, an extract of the transformant containing this recombinant plasmid was subjected to chromatography over DE 22 as described in Methods. The apoaequorin activity eluted at about 0.3M salt which is similar to that observed for authentic apoaequorin. The active fractions were then incubated in the presence of coelenterate luciferin, β mercaptoethanol and oxygen to generate photoprotein activity as described in Fig. 3. This mixture was then subjected to gel filtration. As Fig. 4 shows, the photoprotein activity generated from the partially purified component in pAEQ1 extracts eluted from the column with an M The pooled photoprotein fraction from Fig. 4 produces a luminescence flash upon the addition of Ca² . The kinetics of this flash was indistinguishable from the kinetics of the Ca² dependent aequorin reaction. Other recombinant plasmids did not express a light emitting protein when present in transformants, as is shown in Table 6 below. The above data show that the cDNA insert into pAEQ1 represents the full length cDNA coding for apoaequorin. The data also show that this cDNA is being expressed in pAEQ1 and that the protein product is indistinguishable in its biological properties from that of native, mixed apoaequorin. The level of expression was estimated to be about 0.01 of the total soluble protein. To 0.5 ml of each extract was added mercaptoethanol 2 mM and coelenterate luciferin 0.015 mM . The mixture was incubated at 4 for 20 hours. A 5 µl sample was removed, injected into 0.5 ml of 0.1 m A relatively low level of apoaequorin is expressed in an The 0.75 kb P Extracts of the six aliquots were assayed for apoaequorin activity. The Table 7 contains the results obtained from the six extracts of strains containing pAEQ7 and pAEQ8, an extract from a control The aequorin activity level in these extracts is dependent upon the charging efficiency with the luciferin. It is not possible to quantitate the charging efficiency in crude extracts. The observed charging efficiency was no higher than a 20 using apoaequorin isolated from The expression from pAEQ8 is significantly higher two hours post induction 600 fold when compared to expression from pAEQ1. No inducation of the aequorin gene is observed from pAEQ7 which contains the aequorin gene in the opposite orientation as in pAEQ8. This data demonstrates that the apoaequorin expression level can be significantly increased if the cDNA nucleotide sequence is positioned appropriately 3 to an inducible promoter. The structure of the P This identified sequence illustrates that selection of any amino acid sequence from the variations shown in Table 3 will produce an active apoaequorin since the sequence shown above contains both principal and minor amino acids at different positions of microheterogeneity. Accordingly, the different genes that code for the various peptides are likewise shown to represent biologically active molecules. The previous discussion in this application relating to other peptides, DNA molecules, vectors, etc., based on the general sequences disclosed in this application and representing minor variations thereof thereby having the biological activity of apoaequorin is equally applicable to the specific sequences disclosed in this Example. This Example discloses a peptide, a DNA sequence coding for the peptide, and a full DNA sequence between two restriction sites, among others. The coding DNA sequence optionally including a terminal stop codon is disclosed as a separate entity from the full DNA sequence and is to be considered separately therefrom, including its variations. Any DNA disclosed as double stranded DNA also is considered to disclose the individual single stranded DNA forming the same as well as RNA equivalent thereto by a transcription process. The present invention is not limited to the specific vectors and host bacteria disclosed in these Examples, although the vectors and hosts of the Examples are used in some of the preferred embodiments of the invention. All of the bacteria and plasmids used are readily available to those of ordinary skill in biotechnology through various sources. For example, The invention now being fully described, it will be apparent to one of ordinary skill in the art that many changes and modifications can be made thereto without departing from the spirit or scope of the invention as set forth herein.