# DNA ENCODING HUMAN INTERLEUKIN 1 ALPHA AND THE AMINO ACID CHAIN CORRESPONDING THERETO AND VECTORS AND HOSTS CONTAINING SUCH DNA AND THE PREPARATIION THEREOF

## Claims
Patentansprüche für folgende Vertragsstaaten BE, CH, DE, FR, GB, IT, LI, LU, NL, SE

## Description
The present invention relates to interleukin 1 hereinafter IL 1 , and more particularly to the cloning of a gene for human IL 1 by use of hybrid selection procedures and by use of radiolabeled natural and synthetic oligonucleotide probe derived from clones composing sections of the IL 1 gene to screen a complimentary deoxyribonucleic acid cDNA library synthesized from IL 1 messenger ribonucleic acid mRNA and to the characterisation of the screened IL 1 gene. The IL 1 molecule to which the invention relates is now known as IL 1α. IL 1, formerly known in the literature as lymphocyte activating factor or LAF, is a hormone secreted by macrophages while undergoing an immune response. This protein factor regulates a wide range of immunological and nonimmunological responses. For instance, it is considered that IL 1 mediates activities referred to as endogenous or leukocytic pyrogen, B cell activating factor BAF , epidermal cell thymocyte activating factor ETAF , leukocyte endogenous mediator LEM , bone resorption factor active in rheumatoid arthritis, and a variety of other activities. Although researchers have identified many of the biological properties of IL 1, the chemical nature of this hormone is not well understood. To date, this has been hampered, at least in part, by the unavailability of sufficient quantities of IL 1 in purified form to carry out necessary investigations. Attempts have been made in the past to purify and partially characterize IL 1 derived from both human and murine sources. For instance, Mizel, 122 In a subsequent communication Mizel et al., 126 Researchers have also investigated IL 1 produced from human peripheral blood leukocytes and monocytes. Blyden et al., 118 Crude IL 1 prepared from human leukocytes by Togawa et al., 122 In a more recent study, Lachman, 42 The availability of adequate quantities of homogeneous human IL 1 could be valuable in investigations and possible treatment of autoimmune disorders such as arthritis and lupus erythematosis. Also, human IL 1 in greater purity and larger quantities than heretofore available, could prove useful in achieving successful wound and burn healing. One potential method of providing relatively large quantities of homogeneous human IL 1 is through recombinant DNA techniques. Recombinant DNA techniques have been developed for economically producing a desired protein once the gene coding for the protein has been isolated and identified. A general discussion of such recombinant DNA techniques for protein production is set forth in the editorial and supporting papers in Vol. 196 of In accordance with the present invention, a gene coding for human IL 1 was isolated in several sections from a cDNA library with various hybridization techniques. Total human RNA was extracted from cells thought to produce relatively high levels of IL 1. Polyadenylated mRNA was isolated from the total RNA extract. A cDNA library was constructed by reverse transcription of size separated polyadenylated mRNA with reverse transcriptase. The DNA was rendered double stranded with DNA polymerase I and inserted into an appropriate cloning vector. Resultant recombinant cloning vectors were used to transform an appropriate host. Transformed hosts were identified and grouped into pools. Plasmid DNA prepared from these pools was hybridized with total RNA, which when translated in vitro gave rise to IL 1 biological activity IL 1 active mRNA . The pool s of clones that hybridized with IL 1 active mRNA was identified and then the putative pool subdivided and the hybrid selection screen repeated. By this procedure, a single transformant was eventually identified. Plasmid DNA was prepared from this transformant and sequenced to establish its nucleotide and amino acid composition. This first clone was radiolabeled and then employed as a hybridization probe to rescreen the entire cDNA library. Plasmid DNA prepared from the transformed hosts was hybridized with the radiolabeled first clone. Pools that gave a positive signal were plated out and rescreened with the probe. Through this process, a single positive colony was identified. Plasmid DNA was prepared from the colony and sequenced. This second clone was found to overlap a portion of the first clone and compose a further section of the IL 1 gene. The cDNA library was also screened with a synthetic oligonucleotide probe corresponding to a portion of the first clone. The screening procedure was essentially the same as used above with a probe composed of the entire first clone. Through this third screening process, two additional clones third and fourth were discovered which overlap portions of the first and second clones. These clones were then employed as probes to further screen the cDNA library and isolate further clones composing portions of the IL 1 gene. Due to the overlapping nature of the clones that were isolated, the nucleotide and amino acid sequences of the entire open reading frame and the nucleotide sequences of the 5 flanking region and a large 3 flanking region of the IL 1 gene were determined. Also, the entire open reading frame of the IL 1 gene and the coding region of the gene were cloned in mammilian and bacteria host systems, respectively, to express mature Il 1. Thereafter biological assays were conducted to confirm that the expressed protein product was IL 1 The invention further provides isolated mature human IL 1α, wherein said human IL 1α has the amino acid sequence extending from residue no. 113 to residue no. 271 inclusive in Figures 1A and 1B, as well as such human IL 1α for use in medicine. Pharmaceutical compositions comprising such human IL 1α, particularly for use in the regulation of immunological responses, in the treatment of autoimmune disorders and in the treatment of arthritis or wound or burn healing, are also provided. Human IL 1α having the sequence set out can be used in the preparation of medicaments for use in such indications. The details of typical embodiments of the present invention will be described in connection with the accompanyng drawings, in which Preferably, a cDNA library, from which genes coding for human IL 1 will be sought, is constructed from cells previously found to produce relatively high levels of other lymphokines, under the assumption that they might also produce human IL 1. These sources may include activated human peripheral blood adherent mononuclear cells. For use in the present invention, the peripheral blood mononuclear cells can be separated from whole blood by standard techniques, such as by Ficoll Hypaque centrifugation. The leukocytes removed from the blood are cultured Rather than being obtained from leukocytes removed from whole blood, IL 1 can alternatively be prepared from monocytes derived from any monocyte rich source. Such monocyte sources include monocytic leukemic spleen cells, lymph cells and alevolar macrophages. The medium used to culture the peripheral blood leukocytes may consist of commercially available media, such as Eagle s Minimum Essential Medium MEM or Roswell Park Memorial Institute RPMI medium. Additives, which may be individually or in combination added to the culture medium, include glutamine, HEPES buffer and various antibiotics, such as gentamycin, penicillin and streptomycin. In the past, serum also has been commonly used as an additive. Preferable stimulating agents used in conjunction with the present invention include The process of culturing the leukocytes to induce secretion of IL 1 may be carried out in various environmental conditions. Preferably, however, the cultures are maintained in the temperature range of approximately 35 38 C in a humidified atmosphere of approximately 5 10 CO₂ in air. The quantity of IL 1 released by stimulation of peripheral blood leukocytes with an activating agent varies with time. Applicants have found that optimum levels of IL 1 expression are reached at approximately 24 hours after stimulation. Total RNA from human potentially IL 1 producing cells is extracted by standard methods, such as disclosed by Chirgwin et al., 18 As is well known, when extracting RNA from cells, it is important to minimize ribonuclease RNase activity during the initial stages of extraction. One manner in which this is accomplished is to denature the cellular protein, including the RNase, at a rate that exceeds the rate of RNA hydrolysis by RNase. In the procedures of Chirgwin et al., Next, polyadenylated mRNA is separated from the extracted protein. Although several techniques have been developed to carry out this separation process, one preferred method is to chromatograph the polyadenylated mRNA on oligo dT cellulose as described by Edmonds et al., 68 The polyadenylated mRNA is then sized by electrophoresis through methylmercury agarose. Gel fractions corresponding to different size classes of mRNA and then translated In the As noted above, mRNA translation by the rabbit reticulocyte lysates or the By this thymocyte proliferation assay procedure, only the CD 1 thymocytes cultured in the presence of IL 1 incorporate ³H Tdr in a dose dependent manner. CD 1 cells cultured in the absence of IL 1 incorporate only background levels of ³H Tdr. IL 1 activity is calculated from the linear portion of the ³H Tdr incorporation data in a manner similar to the procedure used by Gillis et A second alternative assay for IL 1 activity may be employed which takes advantage of the fact that IL 1 was found by applicants to convert an interleukin 2 IL 2 nonproducer murine tumor cell line, LBRM 33 145, to an IL 2 producer. In this assay LBRM 33 1A5 cells, ATCC No. CRL 8079, are inactivated by addition of 50 µg ml of mitomycin C and incubated for 1 hour at 37 C. 100 µl of the inactivated LBRM 33 1A5 cells 5 x 10⁵ cells ml are cultured in 96 well flat bottomed plates in the presence of an equal volume of the mitogen, phytohemagglutinin PHA 1 final concentration together with serial dilutions of IL 1 containing fluid samples. At hourly time intervals the existence of IL 2 activity, generated by IL 1 triggered, mitomycin C inhibited LBRM 33 1A5 cells and thus IL 1 activity , is directly ascertained by adding 50 µl of IL 2 dependent CTLL 2 cells 8 x 10⁴ cells ml . The microwell cultures are then incubated for 20 additional hours followed by a 4 hour pulse with 0.5 µCi of ³H Tdr New England Nuclear, Boston, MA, 20 Ci mM specific activity . Thereafter, the thymidine pulsed cultures are harvested onto glass fiber filter strips with the aid of a multiple automated sample harvester MASH II Microbiological Associates, Bethesda, MD . ³H Tdr incorporation is measured by liquid scintillation counting. Details of this procedure are set forth in Gillis et al. A library of double stranded cDNA corresponding to the mRNA, as prepared and assayed above, is constructed by known techniques employing the enzyme reverse transcriptase. One such procedure which may be employed in conjunction with the present invention is detailed by Maniatis et al., It is to be understood that in accordance with the present invention, alternative standard procedures may be employed to prepare double stranded cDNA from mRNA. One such alternative technique is disclosed by Land et al., 9 Next, the double stranded cDNA is inserted within a cloning vector which is used to transform compatible prokaryotic or eukaryotic host cells for replication of the vector. Thereafter, the transformants are identified and plasmid DNA prepared therefrom. To carry out the present invention, various cloning vectors may be utilized. Although the preference is for a plasmid, the vector may be a bacteriophage or a cosmid. If cloning occurs in mammalian cells, viruses also can be used as vectors. If a plasmid is employed, it may be obtained from a natural source or artificially synthesized. The particular plasmid chosen should be compatible with the contemplated transformation host, whether a bacteria such as If If a bacteriophage is used instead of a plasmid, such phages should have substantially the same characteristics noted above for selection of plasmids. This includes the existence of a phenotypic marker and ligatable termini for attachment of foreign genes. Preferably, in the present invention, the double stranded cDNA, having blunt ends, is inserted into a plasmid vector by homopolymeric tailing. As is well known in the art, in this technique, complementary homopolymer tracks are added to the strands of the cDNA and to the plasmid DNA. The vector and double stranded cDNA are then joined together by hydrogen bonding between complementary homopolymeric tails to form open, circular hybrid molecules capable of transforming host cells, such as In one procedure for homopolymeric tailing, approximately 50 to 150 dA nucleotide residues are added to the 3 ends of linearized plasmid DNA. A similar number of dT nucleotide residues are added to the 3 ends of the double stranded cDNA and then the cDNA and plasmid joined together. In an alternative and preferred method, dG tails are added to the 3 ends of the cloning vector that has been cleaved with an appropriate restriction enzyme. For instance, if the pBR322 plasmid is employed, the restriction enzyme Pst I may be used to digest the plasmid at the ampicillin resistant gene. Complementary dC tails are added to the 3 ends of the double stranded cDNA prior to insertion of the cDNA segment in the plasmid with an appropriate annealing buffer. It is to be understood that the double stranded cDNA may be inserted within plasmid cloning vectors by other various standard methods. One such alternative technique involves attaching synthesized nucleotide linkers to the ends of the cDNA strands by using DNA ligase. The linkers are cleaved with a restriction enzyme to generate cohesive termini for insertion within a plasmid cleaved with the same restriction enzyme. Scheller et al., 196 The recombinant DNA plasmids, as prepared above, are used to transform host cells. Although the host may be any appropriate prokaryotic or eukaryotic cell, it is preferably a well defined bacteria, such as If In transformation protocols, including those disclosed by Maniatis et al., The transformants, prepared above, are initially screened for the presence of the IL 1 gene by a hybrid selection method. In this procedure, individual transformants are chosen for screening and then pooled into groups. The groups are grown in liquid culture, and then the replicated plasmids are extracted from the transformants using any one of several well known techniques, such as by alkaline lysis. Plasmid DNA is prepared by cleaving the plasmids at a unique restriction site s in the replicated plasmid. The resulting linearized DNA segments are filtered onto nitrocellulose. After baking of the plasmid DNA onto the nitrocellulose and removal of any loose DNA, the DNA that binds to the filter is hybridized with total IL 1 active mRNA, prepared above, in hybridization solution. Following hybridization, the unbound RNA is removed from the filter. Thereafter, the bound, hybrid selected mRNA is eluted and then translated The particular pool s of clones that gave a positive signal are subdivided, and then the above hybridization selection procedure repeated to identify positive subpools. Next, positive subpools are divided into subsubpools or, if possible, into individual colonies and then the hybridization selection procedure again repeated. By this process, applicants have discovered one positive colony. Plasmid DNA, designated as 2C1, is prepared from the particular positive colony identified. The 2C1 clone is then characterized by nucleotide sequencing using the chain termination method, discussed below under subheading Characterization of Screened cDNA. The DNA sequence of the 2C1 clone is set forth in FIGURE 1 which illustrates the DNA composition of the IL 1 gene. The 2C1 clone, extending from nucleotide number 860 to nucleotide number 1,202 in FIGURE 1, composes a portion of the 3 flanking region of the IL 1 gene. The 2C1 clone is employed as a probe to rescreen the entire cDNA library prepared above in an effort to obtain a larger clone. The cDNA insert from the 2C1 clone is prepared by Pst I digestion of the plasmid and agarose gel electrophoresis. The 2C1 cDNA insert as thus prepared is radiolabeled prior to being used as a probe to screen the cDNA library. Various labeling techniques may be employed however, because of its relatively large size, preferably the probe is labeled by nick translation. In this well known technique, as discussed by Rigby et al., 113 In the screening procedure, the transformants are pooled into groups each composed of approximately 2,000 transformants. The replicated plasmids are extracted by any one of several well known techniques, such as alkaline lysis. DNA is prepared by cleaving the extracted plasmids at the Pst I restriction site. The resulting DNA segments are fractionated by electrophoresis on agarose gel and then analyzed by Southern blotting as described by Southern, 98 The putative pool s of clones that discloses a strongly hybridizing band during autoradiography is plated out and used in direct bacterial colony hybridization on a nitrocellulose filter with the same labeled 2C1 cDNA probe. After completion of the hybridization, the nitrocellulose filter is monitored by autoradiography to identify a positive colony. By this procedure, applicants have discovered one such positive colony. Plasmid DNA, designated as 9H, is prepared from this particular colony. The 9H clone is characterized by nucleotide sequencing using the chain termination methods, described below. As illustrated in FIGURE 2, the nucleotide sequence of the 9H clone, which extends from nucleotide number 928 to nucleotide number 1,979 in FIGURE 1, comprises a rather large 3 flanking region of the IL 1 gene. A radiolabeled synthetic oligonucleotide corresponding to a portion of the 2C1 clone is used as a probe to screen the above prepared cDNA library. The hybridization of the synthetic oligonucleotide probe with plasmid cDNA prepared from the library clones is subsequently identified by autoradiography. Preferably, the probe is composed of the following composition from the antisense strand 5 GTTGGTTCCACTCTTAC 3 . This particular probe construct corresponds to nucleotide numbers 975 through 959 in FIGURE 1. This probe has the advantage of being short enough to be easily synthesized while being long enough to contain sufficient information to be useful as a probe for the IL 1 gene. Also, this same probe is employed as a primer in the chain termination procedure, discussed below, for sequencing the 2C1 fragment. Although the above described oligonucleotide sequence is a preferred composition of the synthetic probe of the present invention, it is to be understood that probes of other compositions corresponding to other segments of the nucleotide sequence of the 2C1 fragment can be employed without departing from the spirit or scope of the present invention. The synthetic oligonucleotide probe may be chemically synthesized by well known techniques, such as by phosphodiester or triester methods. The details of the triester synthesis technique are set forth in Sood et al., 4 The labeled oligonucleotide probe is employed to screen Pvu II and Hind III digested plasmid DNA prepared as discussed above, from pools of approximately 2,000 transformants using the Southern blotting method described above. The specific DNA fragments that hybridize to the probe are identified by autoradiography. The particular pool s of clones that give a signal following autoradiography are plated out and used in direct bacterial colony hybridization on a nitrocellulose filter with the same above identified oligonucleotide probe. After completion of the hybridization, the nitrocellulose filter is monitored by autoradiography to identify positive colonies. Applicants discovered two such positive colonies, from which plasmid DNA, designated as 5B and 8A, is prepared. The nucleic acid sequence of both the 5B and 8A clones is analyzed by chain termination method, as detailed below. As shown in FIGURE 1, the composition of the 5B clone extends from nucleotide number 591 to nucleotide number 995, and the composition of the 8A clone extends from nucleotide number 448 to 1,203. Also referring to FIGURE 2, both clones extend from within the open reading frame of the IL 1 gene to the 3 flanking region of the gene to overlap both the 2C1 and 9H clones previously isolated. A major 5 portion of the 8A clone was employed as a hybridization probe to further screen the cDNA library prepared above. Preferably, the probe is composed of the 5 Pst I Hae III fragment from the 8A clone corresponding to nucleotide numbers 448 through 771 in FIGURE 1 . This probe is prepared by digesting the 8A clone with the restriction enzymes Hae III and Pst I and then isolating the desired fragment by electrophoresis. For use in the screening procedure, the 5 Pst I Hae III fragment is radiolabeled by nick translation, as also discussed above. The probe as thus prepared is employed to screen the cDNA library by use of the Southern blotting technique, discussed above. The specific DNA fragments that hybridize to the labeled probe are identified by autoradiography. By this process, applicants discovered one positive colony from which plasmid DNA, designated as 5F, was prepared. The nucleotide sequence of the 5F clone, as also determined by chain termination method, is set forth in FIGURE 1 as nucleotide numbers 6 through 616. As shown in FIGURE 2, the 5F clone constitutes a major 5 portion of the open reading frame of the IL 1 gene, and overlaps the 5 portions of the 5B and 8A clones. The cDNA library prepared above was also screened with a probe composed of a 5 fragment of the 5F clone. This screening procedure was essentially the same as employed above with the 5 Pst I Hae III probe. This particular probe, composed of the 5 Pst I Hind III fragment, was prepared by digesting the 5 clone with Hind III and Pst I restriction enzymes and then employing electrophoresis to isolate the desired probe from the remaining plasmid fragments. By use of this further screening procedure, applicants isolated two further clones designated as 18E and 10A. From nucleotide sequence analysis, applicants have ascertained that the 18E clone extends from nucleotide number 45 to nucleotide number 652 in FIGURE 1. The 10A clones extend from about nucleotide number 45 to about nucleotide number 1,150. As further illustrated in FIGURE 2, both clones overlap the 5 terminal portions of the 5B, 8A, 5F clones and extend into the 5 flanking region of the IL 1 gene. Also, the 10A clone spans the entire open reading frame of the gene. As will be appreciated, FIGURE 1 was prepared by assembling each of the above identified clones, i.e., 2C1, 9H, 5B, 8A, 5F and 18E in correct order and orientation, which is made possible by the overlapping nature of these clones. The plasmid cDNA clones, isolated above, are sequenced using the chain termination method. This method of nucleotide sequencing was originated by Sanger et al., 70 As discussed above, the DNA and corresponding amino acid sequences of the various clones, assembled together, is illustrated in FIGURE 1. The overlapping nature of the clones enable them to be arranged in the proper order and orientation. The nucleotide bases and amino acid residues are numbered beginning from the 5 end of the open reading frame for the IL 1 gene, i.e., the ATG codon Met residue, marked with a star. The numbering of the amino acids extends to the Ala residue No. 271 located adjacent the termination codon TAG. The numbering of the nucleotide bases extends to the end of the sequence shown in FIGURE 1. Also, the portion of the nucleotide sequence extending into the 5 flanking region of the IL 1 gene is designated with minus numbers. The coding region of mature IL 1 protein, as determined from the murine IL 1 gene identified in Lomedico et al., 312 In preparation for the sequencing procedures, the clones shown in FIGURE 2 are digested with various restriction endonucleases and then the resulting DNA fragments cloned into M13 phage vectors to form single stranded DNA templates. A universal primer is used to sequence the sense and antisense strands of the clones. Rather than relying on the sequencing results obtained from sequencing the entire length of the fragments with a single chain termination procedure, additional synthetically produced primers are used to initiate the chain termination procedure from other intermediate locations along the lengths of the strands. By this process, both strands of the various clones identified above are sequenced in overlapping fashion, thereby serving to redundantly confirm the sequences. It is to be understood that rather than employing the chain termination technique outlined above, other known methods may be utilized to sequence the IL 1 gene without departing from the spirit or scope of the present invention. For instance, the chemical degradation method of Maxam and Gilbert as set forth in 74 To determine whether the cDNA coding region of the IL 1 gene would encode functional human IL 1, the gene is expressed in mammalian cells and then tested with the above described biological assays. A cDNA fragment containing the entire open reading frame of the IL 1 gene shown in FIGURE 1 is inserted into a plasmid vector derived in part from Simian virus 40 SV40 . The genome of this virus consists of a single, small covalently closed circular DNA molecule whose entire nucleotide sequence has been determined, Fiers et al., 237 The pMLSV IL 1α expression vector is transfected into mammalian cells. After subsequent incubation, the cells are harvested and assayed for expression of mature human IL 1 by use of the direct thymocyte mitogenesis assay and IL 1 conversion assay discussed, To investigate the essential portion of the IL 1 gene required for expression of biologically active IL 1, the C terminal portion of the IL 1 gene extending from codon TCA nucleotide numbers 337, 338 and 339 coding for Ser¹¹³ and extending to codon GCG nucleotide numbers 811, 812 and 813 coding for Ala²⁷¹ was inserted in an expression vector used to transform The processes and products of the present invention are further illustrated by the following examples. Leukocyte concentrates of a volume of 350 400 ml, obtained from human whole blood mixture from Portland, Oregon Red Cross , were mixed with and diluted in Ca , Mg free phosphate buffered saline PBS layered onto Histopaque Sigma Chemical Company, St. Louis, MO and then centrifuged at 600 x g for 30 minutes at room temperature. The interface layer, consisting of the leukocytes, was recovered, washed with PBS and centrifuged at 400 x g for 10 minutes at room temperature. The cells were washed two more times in Ca , Mg free PBS and centrifuged at 200 x g for 10 minutes after each washing. Cells were then added to plastic culture flasks in Roswell Park Memorial Institute RPMI 1640 medium together with 10 fetal bovine serum v v . Following a two hour incubation at 37 C, nonadherent cells were decantled and the flasks were then replenished with additional serum supplemented RPMI 1640 medium containing 20 µg ml Total RNA was extracted from the mononuclear cells generally by the method as described by Chirgwin et al., Thereafter, polyadenylated mRNA was separated from the extracted protein on an oligo dT cellulose chromatography column using the method disclosed by Maniatis et al., The polyadenylated mRNA was sized by electrophoresis through methylmercury agarose. Gel fractions corresponding to different size classes of mRNA were then translated A library of double stranded cDNA corresponding to the mRNA was prepared from the purified mRNA in Example 1 by employing the standard procedure detailed by Maniatis et al., The double stranded cDNA was fractionated into size classes by Sephacryl S 400 Pharmacia Fine Chemicals column chromatography and monitored by analysis using alkaline agarose electrophoresis employing end labeled fragments of pBR322 DNA as molecular weight markers. DNA strands having a length of less than 500 bp were culled out to avoid needless cloning of these undersized cDNA fractions. The double stranded cDNA fractions, as prepared above, were inserted into the Pst I site of the pBR322 plasmid Pharmacia Fine Chemicals by the method disclosed by Maniatis et al., The recombinant plasmids were transformed into The transformants, prepared in Example 2 above, were initially screened for the presence of the IL 1 gene by hybrid selection method. In this procedure, approximately 2,000 of the largest colonies of transformants were picked and grown in 96 well plates in liquid medium composed of Luria broth supplemented with tetracycline 25 75 µg ml . Of these approximately 2,000 colonies, 16 pools of 48 each were chosen for screening. Plasmid DNA was removed from the chosen samples of host bacteria by standard alkaline lysis method detailed by Ish Horowicz and Burke, 9 Next, 5 µg of plasmid DNA was prepared for nitrocellulose blotting according to the procedure of Panses et al., 78 Thereafter, the plasmid digests were filtered onto nitrocellulose with the aid of a Schleicher and Schuell manifold dot blot apparatus. After the transfer process, the filter was air dried and baked for two hours at approximately 80 C to bind the DNA fragments to the nitrocellulose. Next, individual circular discs of the nitrocellulose approximately 0.5 cm corresponding to an individual clone pool were cut out from the nitrocellulose sheet and boiled for ten minutes in 2 ml of H₂O. The nitrocellulose spots were air dried at 80 C for two hours. The DNA that bound to the nitrocellulose disc was next hybridized with total IL 1 active mRNA that previously had been ethanol precipitated and resuspended in a hybridization solution. To this end, approximately 25 µg of total mRNA was suspended in 2.5 µl of H₂O, 5 µl of formamide and 2.5 µl of 4 x hybridization salts 400 mM PIPES buffer pH 6.4 , 1.6 mM NaCl and 40 mM EDTA pH 8.0 . In the hybridization procedure, the circular nitrocellulose spots were placed into individual wells of a multiple well microtiter plate together with 10 µl of the above mRNA mixture and then covered with 100 µl of sterile paraffin oil. The microtiter plate was floated overnight on a water bath at 50 C. After overnight hybridization, the filters were removed and placed in individual polypropylene tubes and then washed ten times one minute per wash at 65 C with 1 x saline sodium citrate SSC 20 x SSC is composed of 175.3 g of NaCl and 88.2 g of sodium citrate in 800 ml of H₂O, with pH adjusted to 7.0 with 10 N NaOH and .2 sodium dodecyl sulfate SDS . The filter was then washed five times with .2 x SSC. Thereafter, the hybridized mRNA was eluted from the filter bound DNA by adding to the filter disc 200 µl of elution solution H₂O with 6.6 µg ml yeast tRNA Sigma Chemical Co., St. Louis, Missouri , then incubating in a boiling water bath for three minutes and then cooling by immersion of the tube in ice. The solution was then transferred to a sterile 1.5 ml centrifuge tube. An additional 100 µl of elution solution was added to the filters and then the incubating and cooling procedure was repeated. This was pooled with the previous eluate. Next, NaCl is added to the eluted solution 300 µl total volume to render it .3 M NaCl. Thereafter, two volumes of ethanol were added and the mixture was frozen at 70 C. The RNA was subsequently recovered by centrifugation. The resulting ethanol precipitated, hybrid selected mRNA was translated By the above procedure, one particular pool of clones gave a positive signal. This pool of 48 colonies was subdivided into eight pools of six colonies each and then the above described hybridization selection procedure repeated. As a result, two positive subpools were identified, which subpools were divided into individual colonies 12 total and the above hybridization procedure again repeated. By this process, applicants discovered one positive colony. Plasmid DNA, designated as 2C1, was prepared from this particular positive colony by standard alkaline lysis method, described The 2C1 clone was sequenced by the dideoxy chain termination method essentially as described in the Amersham Handbook, The synthetic universal primer 5 CCCAGTCACGACGTT 3 P L Biochemicals, Milwaukie, Wisconsin , was annealed to the single strand DNA templates and used to prime DNA synthesis as described above. Thereafter, the extension fragments were size separated by gel electrophoresis and autoradiographed from which the nucleotide sequences of the fragments were deduced. An additional primer was employed to prime synthesis from an intermediate location along the antisense strand of the DNA segment in FIGURE 2. A primer having the composition 5 GTTGGTTCCACTCTTAC 3 , corresponding to nucleotide numbers 959 through 975 FIGURE 1 , was used to prime synthesis of the antisense strand. The composition of this primer strand was established from the sequencing information previously obtained by use of the universal primer. By the above walk down method, both strands of the 2C1 clone were sequenced in an overlapping, redundant manner thereby confirming their nucleotide sequence. It is to be understood that other synthetic primers could have been employed to initiate chain extensions from other locations along the strands without departing from the scope of the present invention. The above primer strands were chemically synthesized by triester method as detailed by Sood et al., Deoxyadenosine 5 α ³⁵S thio triphosphate hereinafter dATP α ³⁵S was used as the radioactive label in the dideoxy sequencing reactions. Also, rather than using the gel set forth at page 36 of the Amersham Handbook, a 6 polyacrylamide gel was employed 6 polyacrylmide gel, 0.4 mm thick, containing 7 M, urea 100 mM Tris borate pH 8.1 , and 2 mM EDTA . As noted above, the nucleotide sequence of the 2C1 clone is illustrated in FIGURE 1 which includes the DNA composition of the IL 1 gene together with a large 3 flanking region of the gene as determined by combining together the individual clones isolated by the present invention. In FIGURE 1, the nucleotides are numbered from the beginning of the DNA sequence and the corresponding amino acids, as determined by the nucleotide sequence are set forth below the appropriate codons. The 2C1 clone extends from nucleotide number 860 to nucleotide number 1,202, and composes a portion of the 3 flanking region of the IL 1 gene. The 2C1 clone was employed as a probe to rescreen the entire cDNA library in an effort to obtain a larger clone. The 2C1 probe was radiolabeled by nick translation by the procedure set forth in Maniatis et al., The bound DNA was thereafter hybridized with the labeled 2C1 probe. Briefly, the baked nitrocellulose was presoaked in 6 x SSC and then incubated at 68 C for 2 4 hours in prehybridization buffer composed of 6 x SSC, 0.5 NP40 detergent, 0.1 sarcosyl, 5 x Denhardt s solution 0.02 Ficoll, 0.02 polyvinyl pyrrolidone, 0.02 BSA and 100 µg ml denatured salmon sperm DNA Sigma Type III, sodium salt . The filter was then incubated overnight at 68 C with the ³² P labeled 2C1 probe 5 X 10⁵ cpm ml in hybridization solution. After overnight hybridization, the filter was washed extensively with 6 x SSC at room temperature and then for 30 minutes at 68 C with 0.6 x SSC. After air drying, the filter was subjected to autoradiography at 70 C. From the radiography, applicants found several hybridizing bands. The pool of clones from which the plasmid DNA that produced the hybridizing bands was derived, was plated out and then used in direct bacterial colony hybridization on nitrocellulose paper with the labeled 2C1 probe under the same hybridization conditions as above. By this process, a single positive colony was identified. Plasmid DNA, designated as 9H, was prepared from this colony. Thereafter, the nucleotide sequence of the 9H clone was ascertained by using the chain termination method described in Example 4 above, with the variations set forth below. The 9H clone was prepared for sequencing by digesting with Rsa I, Hind III, Pst I and Sau 3A restriction endonucleases in various combinations prior to cloning the resulting DNA fragments into strains mp 18 and mp 19 of the M13 phage vector. In addition to synthesis with the universal primer described in Example 4 above, three additional primers were employed to prime synthesis from intermediate locations along the antisense and sense strands of the 9H clone. A primer having the composition 5 GTAATATGGGTAGAGTC 3 , corresponding to nucleotide numbers 1,098 through 1,114, FIGURE 1, was used to prime synthesis of the antisense strand in the upstream direction from nucleotide number 1,097. The composition of this primer strand was established from the sequencing information previously obtained by using the universal primer. A second synthetic primer of the composition 5 TCATCTTGAGGCTCGGC 3 , corresponding to nucleotide numbers 1,381 through 1,397, was used in sequencing the antisense strand in the upstream direction from nucleotide number 1,380. A third synthetic primer having the sequence 5 CATTTTGGTCCAAGTTG 3 , corresponding to nucleotide numbers 1,581 through 1,598, was employed to sequence the sense strand in the downstream direction from nucleotide number 1,598. As shown in FIGURE 1, the nucleotide in the sequence of the 9H clone as determined by the chain termination method extends from nucleotide numbers 927 through 1,979 and comprises a rather large 3 flanking region of the IL 1 gene. Also referring to FIGURE 2, the 9H clone overlaps a major portion of the 2C1 clone, thereby confirming the nucleotide sequence of the IL 1 gene in this area. A radiolabeled synthetic oligonucleotide corresponding to a portion of the 2C1 clone was employed as a probe in screening the cDNA library prepared as set forth above in Example 3. The probe was composed of the following composition 5 GTTGGTTCCACTCTTAC 3 . This particular construct corresponds to the composition of the synthetic primer employed above in Example 4 in sequencing the 2C1 clone by chain termination method. The oligonucleotide probe was chemically synthesized by a triester method as detailed by Sood et al., After chemical synthesis has been completed, the 5 ends of the oligonucleotide probes were labeled with ³²P. To facilitate labeling, the 5 ends of the oligonucleotide were synthesized with OH termini, thereby eliminating the phosphatase treatment which typically must be employed when labeling DNA fragments. The labeling protocol included adding 1 µl of synthetic oligonucleotides to 16 µl of ³²P ATP 7000 Ci mMl , 1 µl 10 U of T4 polynucleotide kinase and 2 µl of 10 X kinase buffer I. The 10 X kinase buffer I was composed of 0.5 M Tris Cl pH 7.6 , 0.1 M MgCl₂, 50 mM dithiothreitol, 1 mM spermidine and 1 mM ETDA. The reaction was carried out at 37 C for 30 minutes, and thereafter the synthesized oligonucleotides were extracted with phenol chloroform. The labeled probes were separated from unlabeled oligonucleotides by chromatography on or centrifugation through Sephadex G 50 columns Pharmacia Fine Chemicals . The labeled oligonucleotide probes were employed using the same Southern blotting method to screen linearized DNA prepared by digestion with Pvu II and Hind III and hybridization as detailed in Example 5 except that prehybridization was conducted at 45 C. Also, overnight probe incubation was conducted at 45 C followed by room temperature washing with 6 X SSC, and then a 5 minute wash with 0.6 X SSC. The particular pool s of clones that gave a positive signal following autoradiography were plated out and used in direct bacterial colony hybridization on nitrocellulose paper with the same labeled oligonucleotide probe under the same hybridizing conditions as above. By this process, applicants discovered two positive colonies from which a plasmid DNA, designated as 5B and 8A, was prepared. The 5B and 8A clones were analyzed by the dideoxy chain termination method described above in Example 4, with the following variations. The 5B and 8A clones were digested with Rsa I and Pst I and Hae III restriction endonucleases and then the resulting DNA fragments were cloned into strains mp 18 and mp 19 of the M13 phage vector. In addition to the synthetic universal primer employed as described in Examples 3 and 5, two additional primers were employed to prime synthesis from immediate locations along the antisense and sense strands of the 8A clone. A primer having the composition 5 CAATTGTATGTGACTGC 3 , corresponding to nucleotide numbers 568 through 584 FIGURE 1 , was used to prime synthesis of the sense strand in the downstream direction from nucleotide number 585. The composition of this primer strand was established from the sequencing information previously obtained by use of the universal primer. A second synthetic primer of the composition 5 TCAGCAGCACTGGTTGG 3 corresponding to nucleotide numbers 597 through 613 in FIGURE 1 was used in sequencing the antisense strand in the upstream direction from nucleotide number 596. Through this sequencing procedure, applicants ascertained that the com position of the 5B clone extends from nucleotide number 591 to nucleotide number 995, and the composition of the 8A clone extends from nucleotide number 448 to nucleotide number 1,203. Thus, as shown in FIGURE 2, the 8A clone overlaps both ends of the 5B clone. Also, both clones extend from within the open reading frame of the IL 1 gene to the 3 flanking region of the gene, and both clones overlap both the 2C1 and 9H clones, discussed above in Examples 3 and 5. The 8A clone in fact overlaps the entire length of the 2C1 clone, thereby redundantly confirming the nucleotide sequence of the IL 1 gene. The 8A clone, transformed into The 5 Pst I Hae III fragment from the 8A clone corresponding to nucleotide numbers 448 through 771 in FIGURE 1 , was employed as a hybridization probe to further screen the cDNA library, as prepared above in Example 3. This probe was prepared from the 8A clone by digestion with the restriction enzymes Pst I and Hae III followed by agarose gel electrophoresis. For use in the screening procedure, the 5 Pst I Hae III fragment was radiolabeled by nick translation as discussed above in Example 3 to a specific activity of approximately 3 X 10⁸ CPM µg DNA. Prior to use in the screening protocols, the labeled probe was denatured by boiling in H₂O at 100 C for ten minutes followed by chilling on ice. The probe as thus prepared was employed to screen the cDNA library in the manner described above in Example 5 by Southern blotting technique. By use of this procedure, one positive colony was identified from which plasmid DNA, designated as 5F, was prepared. The nucleotide sequence of the 5F clone was ascertained by use of the dideoxy chain termination method detailed above in Example 4, with the following variations. The 5F clone was digested with Pst I, Hind III and Eco RI, and restriction endonucleases in various combinations and then the resulting DNA fragments were cloned into mp 18 and mp 19 strains of the M13 phage vector. By the above sequencing procedure, the 5F clone was found to be composed of the portion of the IL 1 clone shown in FIGURE 1 extending from the nucleotide number 6 through nucleotide number 616. Additionally referring to FIGURE 2, the 5F clone was found to constitute a major 5 portion of the open reading frame of the IL 1 gene. Also, this clone overlaps the 5 portions of the 5B and 8A clones discussed above in Example 6. In a manner similar to that described above in Example 7, the cDNA library was also screened with a probe composed of the 5 Pst I Hind III fragment of the 5F clone. This particular probe, extending from nucleotide number 6 through nucleotide number 190 in FIGURE 1, was prepared by digesting the 5F clone with Pst I and Hind III restriction enzymes followed by agarose gel electrophoresis. The 5 Pst I Hind III probe was employed to screen the cDNA library by use of the same Southern blotting technique, discussed above in Examples 5 and 7. By this further screening procedure, applicants discovered two positive colonies from which plasmid DNA, designated as 18E and 10A was prepared deposited with the ATCC under Accession Nos. 39996 and 39997, respectively . The 18E clone was also sequenced by dideoxy chain termination method as described above in Example 4, with the following variations. The 18E clone was digested with Pst I, Eco RI and Hind III restriction endonucleases and then the resulting DNA fragments were cloned into strains mp 18 and mp 19 of the M13 phage vector. By the above sequencing procedure, the 18E clone was found to be composed of the portion of the IL 1 clone shown in FIGURE 1 extending from nucleotide number 45 of the 5 flanking region of the IL 1 gene to nucleotide number 652. Also referring to FIGURE 2, this clone spans a major 5 portion of the open reading frame of the IL 1 gene, completely spans the 5F clone and overlaps the 5 portion of the 8A and 5B genes. Similarly to the 18E clone, the 10A clone was found to begin in the 5 flanking region of the nucleotide gene, approximately nucleotide number 45. However, in the 3 direction, the 10A clone not only extends beyond the 3 end of the 18E clone, but also spans the entire length of the 5B clone and the open reading frame of the IL 1 gene, to about nucleotide number 1150. Because of the overlapping nature of each of the above discussed clones discovered by applicants, the entire open reading frame of the IL 1 gene, as well as the 5 flanking portion and an extensive 3 flanking portion of the gene was determined. The nucleotides in FIGURE 1 are numbered in each direction from the beginning of the open reading frame. The amino acids are numbered beginning from the open reading frame of the IL 1 gene, i.e., the Met residue, marked with a star, and extending to the termination codon TAG following residue No. 271. Based on the sequence of the murine IL 1 gene discussed in Lomedico et al., The entire open reading frame of the IL 1 gene shown in FIGURE 1 was inserted into a plasmid vector for a transfection of mammalian cells to ascertain whether functional human IL 1 could be expressed. The transfected cells and cell product was assayed for expression of IL 1 by their ability to promote thymocyte proliferation or induce IL 2 production in the IL 1 conversion assay. The 878 base pair Pst I Hinc II fragment from the 10A clone, containing the entire open reading frame of the IL 1 gene, was inserted into the pMLSV 6 expression vector to produce a plasmid expression vector designated as pMLSV IL 1α . The pMLSV 6 plasmid has been deposited with the ATCC under Accession No. 53081. The pMLSV 6 vector was derived principally from SV40 whose genome consists of a single, small covalently closed DNA molecule that has been entirely sequenced, Fiers et al., Downstream from the early promoter, the pMLSV 6 vector includes a synthetic polylinker of the following composition The polylinker has Xba I cohesive termini and contains the following restriction sites Xba I Cla I EcoRI Xho I Bgl II Pst I Pvu II Kpn I and, Hind III. The hatched box portion of the plasmid contains the SV40 small t antigen donor and acceptor splice junctions SV40 coordinates 4035 4656 and the SV40 polyadenylation signal SV40 coordinates 2469 2706 , originally derived from the pSV2 dhfr plasmid as a Bgl II BamHI fragment, Subramani, The long thin line portion of the pMLSV 6 plasmid is derived from the plasmid pML2d, a derivative of plasmid pBR322, that lacks sequences inhibitory to DNA replication in mammalian cells, Server et al., 79 The 10A clone was digested with Pst I and Hinc II restriction enzymes by standard techniques and inserted into the Pst I and Pvu II sites of the synthetic polylinker of the pMLSV 6 plasmid by standard techniques, for instance, as detailed in Maniatis et al., The pMLSV IL 1α plasmid as prepared above was transfected into COS 7 monkey kidney cells ATCC, Rockville, MD by standard techniques, for instance, by essentially using the procedures described by Luthman and Magnusson, 11 A 3 portion of the IL 1 gene, extending from the codon coding for Ser¹¹³ nucleotide numbers 337, 338 and 339 to the codon coding for Ala²⁷¹ nucleotide numbers 811, 812 and 813 was inserted into an expression vector to direct IL 1 expression in The 3 portion of the IL 1 gene from the Alu I site base number 351 in FIGURE 2 to the Nde I restriction site base 850 in FIGURE 2 in the 3 flanking region of the gene was removed from the 10A clone by use of Alu I and Nde I restriction enzymes in a standard protocol, for instance as set forth in Maniatis et al., Also, rather than cleaving the coding region of the IL 1 gene at the Alu I site, the 10A clone in FIGURE 1 could be cleaved at a restriction enzyme site in the 5 flanking region of the coding portion of the gene. Thereafter, the nucleotides of the flanking portion can be sequentially removed by standard techniques. Next, a synthetic oligonucleotide composed of the Eco RI 5 cohesive terminal, ribosome binding site and translation initiation codon, could be linked to the 5 end of the IL 1 gene. The pPLc28 expression vector was prepared for ligation to the synthetic oligonucleotide and the excised major portion of the coding region of the IL 1 gene by digestion of the vector to completion with restriction endonucleases Eco RI and Nde I by standard techniques, such as set forth in Maniatis et al., As shown in FIGURE 4, the synthetic DNA oligomer, the excised major portion of the coding region of the IL 1 gene and desired linearized pPLc28 fragment were ligated together in a reaction mixture composed of 50 nanograms ng of the pPLc28 vector fragment Eco RI Nde I , 50 ng of the major IL 1 cDNA fragment Nde I Alu I , 5 ng of synthetic oligonucleotide Alu I Eco RI , 1 unit of T4 DNA ligase and sufficient T4 ligase buffer 0.4M Tris pH 7.4 , 0.1M MgCl₂, 0.1M dithiothreitol, 10mM spermidine, 10mM ATP and 1 µg ml BSA to form a 20 µl reaction volume. The reaction was carried out by incubation at 25 C for 15 hours. The resulting recombinant plasmid, designated as pILP α, was then transformed into The cultures were then shifted to 43 C for 1 hour to promote derepression of the P As will be apparent to those skilled in the art in which the invention is addressed, the present invention may be embodied in forms other than those specifically disclosed above without departing from the invention. The particular embodiments of the present invention, described above, are therefore to be considered in all respects as illustrative and not restrictive.