# Method of producing interferon gamma.

## Abstract
A method of producing interferon gamma which comprises using, as the gene expression elements, a expression elements containing an alpha amylase promoter region derived from a gram positive bacterial strain and, as the host, a gram negative bacterial strain. The invention has thus enabled production of interferon gamma in gram negative bacteria through the use of gene expression elements of a gram positive bacterial strain and has contributed to technology diversification in the field of genetic engineering. On the occasion of the secretion of the interfteron gamma into the periplasmic space of gram negative bacteria, the signal peptide is cleaved off, whereby the protein is obtained in the form of a mature protein free of methionine at the N terminal.

## Claims
What is claimed is 1. A method of producing a interferon r which comprises using, as the gene expression elements, a expression elements containing an cc amylase promoter region derived from a gram positive bacterial strain and, as the host, a gram negative bacterial strain. 2. The method of claim 1, wherein the grampositive bacterial strain is a strain belonging to the genus Bacillus. 3. The method of Claim 2, wherein the strain belonging to the genues Bacillus is a strain of the species Bacillus natto. 4. The method of any of claims 1 4, wherein the gram negative bacterial strain is a strain ofEscherichia coli.

## Description
METHOD OP PRODUCING INTERFERON r FIELD OP THE INVENTION This invention relates to a method of producing interferon r by a genetic engineering technique. BACKGROUND OF THE INVENTION Genetic engineering techniques are widely used, for example in the productipn ofeuseful heterologous proteins. Escherichia coli, Bacillus subtilis and yeasts, among others, are the hosts which current genetic engineering techniques mainly use. Among them,Escherichia coli may be used most easily as the host because of the most advanced elucidation of the genes in its expression system and so forth. However, the use ofEscherichia coli does not always result in efficient recovery of heterologous proteins since they remain in cells after production thereof and are exposed to the decomposing and other actions of intracellular enzymes.On the other hand, the use of Bacillus subtilis as the host might result in more efficient recovery of heterologous proteins produced since they are secrected out of cells. However, the expression system in Bacillus subtilis remains unknown in may respects, so that it is not yet always suitable to use Bacillus subtilis in commercial production of heterologous proteins. An object of the invention is to establish a system which is capable of causing secretion of interferon r into the periplasmic space using gram negative bacteria. SUMMARY OF THE INVENTION The invention provides a genetic engineering technique by which interferon r are secreted into the periplasmic space using, as the gene expression elements, expression elements of a gram positive bacterial strain, in particular, a system containing aBacillus natto derived amylase and, as the host, a gram negative bacterial strain. BRIEF DESCRIPTION OF THE DRAWINGS In the accompaning drawings, Fig. 1 shows the restriction enzyme map of pGC303, Fig. 2 shows the scheme of cloning the ac amylase gene from chromosomal DNA prepared from Bacillus natto, Figs. 3 and 4 show the process for constructing an expression vector for use in Escherichia coli, the numerical figures indicating the respective steps referred to in the detailed description of the invention, Fig. 5 shows the process for constructing pI ramy5 by ligation of pYI7 1 with the IFN r gene, and Fig. 6 shows the results of Edman degradation. DETAILED DESCRIPTION OF THE INVENTION a Preparation of gene expression elements The gene expression elements to be used in accordance with the invention is of the gram positive bacterial strain. The gram positive bacterial strain is preferably a strain belonging to the genus Bacillus, such as a strain of Bacillus subtilis or Bacillus natto.The term gene expression elements includes within theMeaning thereof the promoter, SD sequence, signal sequence coding region and so on which are of the gram positive bacterial strain origin. A preferred example of the gene expression elements is of the cu amylase gene. The cc amylase gene has been isolated by colleagues to the present inventors from Bacillus natto GC161 deposited at the Fermentation ResearchInstitute under the deposit number FERn BP 539 cf.European Patent Publication No. 135045 U.S. Ser. No.633562 . A partial nucleotide sequence containing the a amylase promoter and signal sequence coding region is shown in Table 1. Tabel 1 GTAC AGTCTCGGGC zooAGTTTTTTTT ATAGGAACAT TGATTTGTAT TCACTCTGCC AAGTTGTTTT isoGATAGAGTGA TTGTGATAAT TTAAAATGTA AGCGTTAACA AAATTCTCCA 100 GTCTTCACAT CAGCTTGAAA GGAGGAAGCG GAAGAATGAA GTAAGAGGGA so 1TTTTTGACTC CGAAGTAAGT CTTCAAAAAA TCAAATAAGG AGTGTCAAGA 1 soATGTTTGCAA AACGATTCAA AACCTCTTTA CTGCCGTTAT TCGCTGGATT L amylase 100 TTTATTGCTG TTTTATTTGG TTCTGGCAGG ACCGGCGGCT GCGAGTGCTG 150AAACGGCGAA CAAATCGAAT GAGCTTACAG CACCGTCGAT CAAAAGCGGA 200 ACCATTCTTC ATGCATGGAA TTGGTCGTTC AATACGTTAA AACACAATAT 250GAAGGATATT CATGATGCAC GATATACAGC CATTCAGACA TCTCCGATTA 300 ACCAAGTAAA GGAAGGGAAT CAAGGAGATA AAAGCATGTC GAACTGGTAC A plasmid containing this gene expression elements is provided under the code pGC303 which is a derivative of the plasmid pGC200 derived from a strain of Bacillus natto deposited at the Fermentation Research Institute under the deposit number FERM BP 645 cf. European PatentPublication No. 135045 U. S. Ser. No. 633562 . The restriction enzyme map of pGC303 is shown in Fig. 1. b Preparation of expression vector for use in gram negative bacteria The gene expression elements prepared in step a is then inserted into such a plasmid as pBR322 derived from Escherichia coli2 a gram negative bacterial species, to creat a plasmid capable of causing expression of interferon r in gram negative bacteria by the action of the gram positive bacterial gene expression elements. c Ligation with interferon r gene A gene coding for interferon r is ligated with the expression vector in gram negative bacteria as obtained in step b at a site downstream from the promoter, SD sequence and signal sequence of said vector, by the known restriction enzyme and ligase treatment process. The ligation product is introduced into an appropriate host. d Transformation As the host preferred is a gram negative bacterial strain, in particular a strain of Escherichia coli.The host is transformed by the known method Cohen, S.N. et al., Proc. Natl. Acad. Sci. USA, 69, 2110 1972 , for instance. Cultivation of host The transformant obtained in step d is cultivated on a per se known medium suited for growing the host. A preferred example of the medium is the medium used by Palva et al. for growing Bacillus subtilis Gene, 22, 229 1983 or a modification thereof. The cultivation is generally conducted at 15 43 c for 3 24 hours, with aeration and or stirring as necessary. f Recovery of interferon r After cultivation, cells are harvested by a known method e.g. centrifugation and the periplasmic fraction is recovered by a known method for recovering proteins secreted into the periplasm te.g. osmotic shock method Nossal, N.G. and Heppel, L.A.,J. Biol. Chem., 241, 3055 1966 or a modification thereof. The interferon T is recovered from this fraction by a per se known method, for example by affinity chromatography and or fractionation based on solubility differences. The invention has thus enabled production of interferon T in gram negative bacteria through the use of gene expression elements from gram positive bacterial strains and has contributed to technology diversification in the field of genetic engineering. On the occasion of the secretion of the interferon r Into the periplasmic space of gram negative bacteria, the signal peptide is cleaved off, whereby the protein is obtained in the form of a mature protein free of methionine at the N terminal, Thus, the invention can provide an efficient method of producing interferon r using genetic engineering technology. In the following, the process for producing human interferon r hereinafter, IFN r using 2 amylase promoter of the Bacillus natto as a preferred example of gene expression elements of gram positive bacterial strain and Escherichia coli as a preferred exmaple of the host will be described step by step. It is to be noted, however, this mode of practice is by no means limitative of the scope of the present invention. a Cloning of gene expression elements A one liter portion of a overnight culture ofBacillus natto GC161 deposited at the FermentationResearch Institute under the deposit number FERn BP 539 was centrifuged and the cells obtained were washed. To the cells were added 10 ml of 20 X sucrose added 50 mM Tris hydrochloride buffer pH 8.0 and 100 mg of lysozyme, followed by reaction at 37 C for 15 minutes.Thereafter, 10 ml of 1 Sarkosyl l EDTA solution and 1 ml of Pronase E 100 mg ml were added. After standing at 37 C for 16 hours, the solution was made up to 35 ml with TES solution 30 mM Tris HCl, 50 mM Nail, 5 mn EDTA, pH 8 . Thereto were added 35 g of cesium chloride and 1 ml of ethidium bromide 10 mg ml and the mixture was centrifuged at 60,000 rpm Beckman 70.1 Ti rotor for 16 hours. The chromosomal DNA band was collected, dialyzed against TES solution and treated twice with phenol for purification of the chromosomalDNA. The chromosomal DNA thus obtained was cleaved with the restriction enzymes Pvu II , Bgl II , HpaI, Bamlil, Hind Ill and so forth. Fragments were extracted from 0.8 agarose gel and ligated with various plasmids, pC194, pE194, pUB11O, and pGC200 deposited at theFermentation Research Institute under the deposit number FERM BP 645 which is a derivative of pC194, and the ligation products were used for transforming B. subtilis rec , amy . After selection based on drug resistance of each plasmid, the transformants selected were inoculated onto starch added agar plates L agar1 soluble starch .After incubation at 37 C for 2 days, iodine solution 0.1 iodine 1 KI was added to the agar plates for checking amylase activity on the basis of decoloration of iodine. When, among the five restriction enzymes mentioned above, Hind Ill was used for cleaving the chromosomal DNA prepared from Bacillus natto and a fragment of said DNA was inserted into pGC200 at the Hind III site thereof, followed by transformation with the resulting plasmid, there was obtained a strain having 2 amylase activity. The chromosomalDNA fragment inserted was a 7 kb fragment.Therefore, the 7 kb DNA containing the cc amyiase gene was excised using the restriction enzymes Hind 111 and Rsal and inserted into the EcoRVHind III fragment of pBR322.The plasmid pGC302 thus obtained was then cleaved withSac II, , followed by ligation, which gave pGC303 7.9 kb Fig. 2 . Transformation was performed in the conventional manner, selection was made based on the ampicillin resistance and the transformants selected were checked for alpha amylase activity in the same manner as above, whereupon expression of alpha amylase was confirmed. Thus was confirmed the presence of the whole a amylase gene series. In this manner, the plasmid pGC303 containing the cc amylase promoter, signal sequence coding region and structural gene for 2 amylase was obtained. The DNA sequence of the promoter and signal sequence coding region of z amylase gene was determined by the Naxam Gilbert method Maxam, A. and Gilbert, W., 1979, Methods in Enzymology. 65, 499 560 Table 1 . b Construction of expression vector for use in Escherichia coli An secretion expression vector for use inEscherichia coli was constructed from pGC303 and pBR322 in accordance with the procedure shown in Figs. 3 and 4. Step 1 pGC303 was digested with the restriction enzymes Banfli and Banll to give a relatively large DNA fragment containing the promoter and signal sequence coding region of cc amylase gene. The reaction was carried out using the following reaction mixture composition EMI10.1 tb pGC303 SEP 1.0 SEP pg ,a SEP 100 SEP Q tb 10x SEP Bcll SEP buffer SEP 1 SEP 20 SEP FQ tb Ban SEP II SEP 8 SEP U ,zQ, SEP Toyobo SEP 20 SEP ,gQ tb BanIll SEP 5 SEP U ,ffQ, SEP Toyobo SEP 20 SEP ,sQ tb H20 SEP 40 SEP tb SEP Total SEP 200 SEP ya tb l 60 mM Tris HCl pH 7.4 100 mM MgClz 10 mM dithiothreitol 750 mM KCl The reaction was carried out overnight at 37 c. After completion of the reaction, the whole reaction mixture was applied to 1 agarose gel, followed by electrophoresis. A 960 bp fragment was recovered following adsorption on DEAE paper. Thereto was added 60 g of tRNA as the carrier for ethanol precipitation.The mixture was extracted with phenol, and the DNA was recovered by ethanol precipitation. The DNA precipitate was dissolved in 90 , of 1120. Step 2 The fragment obtained in step 1 was partially digested with AluI using the following reaction mixture composition DNA fragment solution obtained in step 1 50 10xAlu I buffer 2 20AluI 2 use , Takara Shuzo 20 H20 110 Total 200EMI11.1 EMI11.2 tb 2 SEP SEP 100 SEP mM SEP Tris HCl SEP pH SEP 7.5 tb SEP 70 SEP ,M SEP MgCl2 tb SEP 200 SEP mM SEP NaCl tb SEP SEP 70 SEP mM SEP 2 mercaptoethanol tb The reaction was carried out at 37 C for 5 minutes, and the whole reaction mixture was applied to 5X polyacrylamide gel, followed by electrophoresis. Thereafter, a 621 bp DNA fragment was recovered by electroelution. To the fragment fraction was added 20eg of polyinosic acid was added as the carrier in ethanol precipitation. After extraction with phenol, the fragment was precipitated with ethanol. The DNA precipitate obtained was dissolved in 30 l of HzO. Step 3 A BamHI linker was ligated with the DNA fragment obtained in step 2 at the AluI end thereof using the following reaction mixture composition Partial AluI digest fragment obtained in step 2 10 10xligation buffer 3 1BamHI linker 4 10T4 DNA ligase 175 U Takara Shuzo 2 Total 23EMI12.1 EMI12.2 tb 3 SEP SEP 660 SEP mM SEP Tris HCl SEP pH SEP 7.6 tb SEP 10 SEP mM SEP ATP tb SEP 10 SEP mM SEP spermidine tb SEP 100 SEP mM SEP MgCl SEP 2 SEP tb SEP SEP 150 SEP mM SEP DTT SEP dithiothreitol tb 4 SEP SEP Product SEP of SEP Pharmacia SEP P L SEP Biochemicals tb SEP Sequence SEP 5 d CCCGGATCCGGG 3 tb SEP SEP Used SEP after SEP phosphorylation SEP at SEP the SEP 5 SEP end. tb The reaction was carried out overnight at 4 C, followed by extraction with phenol and precipitation with ethanol. The DNA precipitate was dissolved in 42 L of H20. Step 4 The DNA fragment obtained in step 3 was digested with BamHI using the following reaction mixture composition DNA obtained in step 3 42 10xBamHI buffer 5 5BamHI 12 U l, Takara Shuzo 3 Total 50EMI13.1 EMI13.2 tb SEP 5 SEP SEP 1 SEP 00 SEP mM SEP Tris HCl SEP pH SEP 8.0 tb SEP 70 SEP mM SEP NgCl2 SEP tb SEP 1 SEP M SEP NaCl tb SEP SEP 20 SEP mM SEP 2 mercaptoethanol tb The reaction was carried out overnight at 37 c and the whole reaction mixture was applied to 5 polyacrylamide gel for electrophoresis. After electrophoresis, a 533 bp fragment was recovered by electroelution.followed by extraction with phenol and precipitation with ethanol. The DNA precipitate recovered was dissolved in 50 E of HzO. Step 5 For use as a vector for cloning the DNA fragment both ends being BamHI ends obtained in step 4, pBR322 was digested with BamHI overnight at 37 C using the following reaction mixture composition pBR322 1.0 g l 25 10xBamHI buffer mentioned above 5 lBamHI 10 U l, Takara Shuzo 3 l H2O 17 l Total 50EMI13.3 and then the phosphoric acid residue at the 5 end was eliminated by using bacterial alkaline phosphatase BAP . After reaction, DNA was recovered by precipitation with ethanol. The DNA Xas dissolved in 180 l of HzO. To prevent self ligation of the vector pBR322 , this was treated with bacterial alkaline phosphatase BAP at 65 C for 40 minutes for eliminating the phosphoric acid residue at the 5 end. The reaction mixture composition was as follows BamHI digested pBR322 180 10x BAP buffer 6 20BAP 0.45 U F , Takara Shuzo 2 Total 202EMI14.1 EMI14.2 tb 6 SEP SEP 100 SEP mM SEP Tris HCl SEP pH SEP 7.5 tb SEP 70 SEP mM SEP MgCl2 tb SEP SEP 600 SEP mM SEP NaCl tb After reaction, phenol extraction was repeated three times and DNA was recovered by precipitation with ethanol. The DNA precipitate was dissolved in 25 l of HzO. Step 6 The DNA fragment obtained in step 4 and the vector obtained in step 5 were ligated together by overnight reaction at 4 C . The reaction mixture composition was as follows DNA fragment from step 4 10BamHI digested and BAP treated pBR322 5 10x ligation buffer 7 4T4 DNA ligase 175 U oss, Takara Shuzo 1 Total 20EMI15.1 EMI15.2 tb 7 SEP SEP 660 SEP mn SEP Tris HCl pH SEP 7.6 tb SEP 50 SEP mM SEP MgC12 SEP tb SEP 50 SEP mM SEP DTT dithiothreitol tb SEP 10 SEP mM SEP ATP tb After reaction, the whole reaction mixture was used for transformation of E. coli RR1 by the method ofCohen et al.As a result, there were obtained 6,500 colonies, 200 of which were checked for sensitivity to ampicillin Ap and to tetracycline Tc . Fourteen strains were found to be resistant to Ap Apr and susceptible to Tc Tc . Using these 14 strains1 a plasmid was prepared in large quantities according to Clewell, D. B. and Helinski, D. R., Proceedings of the National Academy of Sciences USA, 62, 1159 1969 by the alkaline extraction method Birnboim, H. C., andDoly, J., Nucleic Acids Research, 7, 1513 1979 This plasmid was named pYI5 1. Step 7 pYI5 1 is inconvenient for inserting the IFN T gene since it has two BamHI sites. Therefore, for causing one BamHI site upstream from the cc amylase promoter to disappear, pYI5 1 was first digested with AvaI at 37 C overnight. The reaction mixture composition used was as follows pYI5 1 1.43 g l 14 10x AvaI buffer 8 5Aval 8 U l, Takara shuzo 3HzO 28 Total 50EMI16.1 EMI16.2 tb 8 SEP SEP 100 SEP mM SEP Tris HC1 SEP pH SEP 8.0 tb SEP 70 SEP mM SEP MgCl SEP 2 SEP tb SEP 600 SEP mM SEP NaCI tb SEP SEP 70 SEP mM SEP 2 mercaptoethanol tb After reaction, the whole reaction mixture was applied to 1 agarose gel and electrophoresis was performed.A DNA fragment of about 3.5 kbp was caused to be adsorbed on DEAE paper and then recovered. As the carrier to serve in precipitation with ethanol, there was added 20 g of tRNA. Following extraction with phenol, DNA was recovered by precipitation with ethanol. The DNA recovered was dissolved in 50 l of HzO. Step 8 The AvaI digested pYI5 1 obtained in step 7 was cyclized by allowing self ligation to take place at 4 C overnight. The reaction mixture composition used was as follows Aval digested pYI5 1 obtained in step 7 2 10 x ligation buffer same in composition as in step 6 10T4 DNA ligase 175 U Fss, Takara Shuzo 1HzO 87 Total 100EMI17.1 A 25 l portion of the reaction mixture 100 oB was used for transformation of E. coli RR1, whereby 18 ampicillin resistant Apr colonies were obtained. These colonies were subjected to plasmid extraction by the alkline extraction method and the plasmids extracted were examined for their structure using various restriction enzymes. As a result, 14 colonies were found to have a plasmid having the structure of pYI6 1 given in Fig. 3. One of these strains was used for largequantity plasmid preparation and the plasmid obtained was named pYI6 l. Step 9 pYI6 1 was digested with the restriction enzyme Hpa II to give a DNA fragment containing the amylase promoter and part of the signal sequence. For the digestion treatment, the following reaction mixture composition was used pYl6 1 0.53 g l 187 10xHpaII buffer 9 30Hpa II 6 U l, Toyobo 40HzO 43 Total 300EMI18.1 EMI18.2 tb 9 SEP 100 SEP mM SEP Tris HCl SEP pH SEP 7.5 tb SEP 70 SEP mM SEP MgClz SEP tb SEP SEP 70 SEP mM SEP 2 mercaptoethanol tb The reaction was carried out at 37 C for 5 hours.Thereafter, the whole reaction mixture was applied to a 5 polyacrylamide gel, followed by electrophoresis.A 348 bp DNA fragment was recovered by electroelution.To the fragment recovered was added 40 of tRNA as a carrier in ethanol precipitation. After extraction with phenol, the DNA was precipitated with ethanol, and theDNA precipitate obtained was dissolved in 60 l of HzO. Step 10 The DNA fragment obtained in step 9 was rendered blunt ended using DNA polymerase I in a reaction mixture having the following composition HpaII fragment obtained in step 9 24 10 x DNA polymerase I buffer 10 15 dNTP 1 6DNA polymerase I, large fragment 4 6 U l, Toyobo HzO 101 Total 150EMI18.3 EMI19.1 tb 0 SEP r500 SEP mM SEP Tris HCl SEP pH SEP 7.2 tb SEP 100 SEP mM SEP MgCl2 tb SEP L SEP 1 SEP mM SEP dithiothreitol tb Aqueous solution containing dATP, dTTP, dGTP and dCTP each in a concentration of 2 mM The reaction was carried out at 20 C for 45 minutes.After reaction, 120 l of 5 fl ammonium acetate and 540 l of cold ethanol were added for ethanol precipitation. The DNA precipitate obtained was dissolved in 20 l of H2O. Step 11 A mixture of a BamHI linker and a Pstl linker was allowed to add to both ends of the DNA fragment obtained in step 10 using the following reaction mixture composition DNA fragment obtained in step 10 10 10xligation buffer mentioned above 2BamHI linker 0.2 g l, Nippon Zeon 12 10 PstI linker 1.0 g l, Takara Shuzo 13 3T4 DNA ligase 175 U l, Takara Shuzo 2 Total 27EMI19.2 12 the sequence 5 d GCAGGATCCTCC 3 used after addition of phosphryl group at the 5 end. 13 the sequence 5 d pGCTGCAGC 3 , with a phosphoryl group already added in advance at the 5 end. The reaction was carried out at 4 C overnight.Thereafter, extraction with phenol was performed, followed by precipitation with ethanol, The DNA precipitate obtained was dissolved in 37 PQ of 1120. Step 12 The DNA fragment obtained in step 11 was digested with HI and AvaI using the following reaction mixture composition DNA fragment obtained in step 11 37 10xBamHI buffer mentioned above 5BamHI 12 U l, Takara Shuzo 3AvaI 8 U l, Takara Shuzo 5 Total 50EMI20.1 The reaction was carried out at 37 C overnight and, then, the whole reaction mixture was applied to a 5 polyacrylamide gel, followed by electrophoresis.Thereafter, a DNA fragment of about 300 bp was recovered by electroelution. After extraction with phenol, theDNA was precipitated with ethanol. On that occasion, 40 g of tRNA was added as a carrier. The DNA precipitate recovered was dissolved in 50 l of H 20. Step 13 pBR322, chosen as a vector for cloning the DNA fragment obtained in step 12 both ends beingAvaI and BamHI , was digested with BamHI and AvaI using the following reaction mixture composition pBR322 1.0 g l 25 10xBamHI buffer 5BamHI 12 U F2, Takara Shuzo 3AvaI 8 U oB. Takara Shuzo 6 H20 11 Total 50EMI21.1 The reaction was carried out at 37.C overnight.Then, for preventing self ligation of the vector pBR322 , the 5 end phosphoryl groups were removed using bacterial alkaline phosphatase BAP . Thus, 2 l of BAP 0.45 U l, Takara Shuzo was added to the above reaction mixture and the reaction was carried out at 65 C for 40 minutes. After reaction, the whole reaction mixture was applied to a 1 agarose gel, and electrophoresis was conducted. A 3,300 bp DNA fragment was allowed to be adsorbed on a DEAE paper, and then recovered. As a carrier in ethanol precipitation, 40 g of tRNA was added. The mixture was extracted with phenol and the DNA was recovered by precipitation with ethanol. The DNA precipitate thus obtained was dissolved in 90 of HzO. Step 14 The DNA fragment obtained in step 12 and the voctor obtained in step 13 were ligated together using the following reaction mixture composition DNA fragment from step 12 10Vector DNA from step 13 5 10 x ligation buffer mentioned above 4 T DNA ligase 175 U , Takara Shuzo 1H20 20 Total 40EMI22.1 The reaction was carried out at 4 c overnight.After completion of the reaction, the whole reaction mixture was used for transformation of E. coli RR1 by the method of Cohen et al. As a result, 3,500 colonies were obtained. Among them, 48 strains were used for plasmid preparation by the alkaline extraction method, and the structure of each plasmid was examined using various restriction enzymes. As a result, one strain was found to carry a plasmid of the structure of pYI7 1 as shown in Fig. 4, and this plasmid was named pYI7 1 and used in the subsequent steps. The plasmid pYI7 1 thus obtained has a BamHI site downstream from the signal sequence 31 amino acids for the Bacillus natto alpha amylase gene and, further downstream therefrom, one each of EcoRI, ClaI, Hind m andEcoRV sites. By making use of these sites1 it is possible to insert into the plasmid a heterologous gene such as an IFN T gene. Thus, a DNA fragment having aBamHI site upstream from a gene coding for a hetero logous protein such as IFN r and, downstream therefrom, a flush end site such as an EcoRI, Clal, Hind Ill and or further EcoR V site can be inserted into said plasmid. c Ligation of pYI7 1 with IFN r gene construction of pI gamma amys Utilizing pYI7 1 a DNA fragment containing anIFN r gene inserted into pYI7 1 between the BamHI site and the EcoRV site thereof to give pI r amys for expression of the IFN r gene and secretion of IFN r. The construction of pI r amys is schematically shown in Fig. Step 1 pYI7 1 was digested with BamHI and BcoRV using the following reaction mixture composition pY17 1 1.0 g l 50 10xBamHI buffer mentioned above 15BamHI 12 U l, Takara Shuzo 10EcoRV 5 U l, Takara Shuzo 20 H20 55 Total 150EMI23.1 The reaction was carried at 37 C overnight, Then, to prevent self ligation of the vector pYI7 1 , the 5 end phosphoryl groups were removed using bacterial alkaline phosphatase BAP . Thus, 2 l of BAP 0,45 U N, Takara Shuzo was added to the above reaction mixture and the reaction was carried out at 65 C for 40 minutes. Thereafter, the whole reaction mixture was applied to a 1 agarose gel and electrophoresis was performed.Then, a DNA fragment of about 3.7 Kbp was recovered following adsorption thereof on a DEAE paper.Thereto was added 40 g of tRNA as a carrier in ethanol precipitation. After extraction of the mixture with phenol, the DNA was recovered by precipitation with ethanol. The DNA precipitate obtained was dissolved in 50 l of HzO. Step 2 A genetically engineered, IFN r genecontaining plasmid, pYS61, having the construction shown in Fig. 5 was digested with NdeI and Hindu using the following reaction mixture composition EMI24.1 tb pYS61 SEP 1.2 SEP eg Fss SEP 42 SEP oB tb 10 SEP XNdeI SEP buffer SEP 14 SEP 10 SEP mQ tb NdeI SEP 8 SEP U oss, SEP NEB SEP 10 SEP ss tb Hind SEP m SEP 8 SEP U rss, SEP Takara SEP Shuzo SEP 10 SEP PQ tb B20 SEP 28 SEP PQ tb SEP Total SEP 100 SEP ss tb EMI24.2 tb 14 SEP 100 SEP mM SEP Tris HCl SEP pH SEP 7.8 tb SEP 1.5 SEP M SEP NaCl tb SEP 70 SEP mM SEP MgCl2 tb SEP SEP 60 SEP mM SEP 2 mercaptoethanol tb The reaction was carried out at 37 C overnight.Thereafter, the whole reaction mixture was applied to a 1 agarose gel and electrophoresis was conducted. A DNA fragment of about 700 bp was adsorbed on a DEAE paper and then recovered. After addition of 40 mg of tRNA as a carrier in ethanol precipitation, the mixture was extracted with phenol. Then1 the DNA was recovered by precipitation with ethanol. The DNA obtained was dissolved in 50 od of H20. The DNA fragment obtained in this step contains approximately the whole of the IFN r gene and the trp terminator but is lacking in that portion which code for the three terminal amino acids from the N terminal of IFN r . However, the DNA base sequence corresponding to these three amino acids can be reconstructed when said fragment is subsequently inserted into pYI7 1 following BamHI linker addition to said fragment. Step 3 The DNA fragment obtained in step 2 was rendered blunt ended using DNA polymerase I and the following reaction mixture composition DNA fragment obtained in step 2 25 10x DNA polymerase I buffer 15 mentioned above dNTP each 2 mfl mentioned above 6DNA polymerase I, large fragment 4 6 U Q, Toyobo H2O 100 Total 150EMI25.1 The reaction was carried out at 20t for 45 minutes. Thereafter, 120 l of 5 M ammonium acetate and 540 l of cold ethanol were added to thereby cause ethanol precipitation. The DNA precipitate obtained was dissolved in 20 2 of Hz0. Step 4 A BamHI linker was connected to both ends of the DNA fragment obtained in step 3 using the following reaction mixture composition DNA fragment obtained in step 3 10 10xligation buffer mentioned above 2BamHI linker 1.0 g l, Takara Shuzo 5 2 Tq DNA ligase 175 U l, Takara Shuzo 2HzO 4 Total 20EMI26.1 15 The sequence 5 d pCGGATCCG 3 having, at the 5 end, a phosphoryl group added beforehand The reaction was carried out at 4 C overnight.Thereafter, following extraction with phenol, precipitation was effected by addition of ethanol. The DNA precipitate obtained was dissolved in 30 PQ of HzO. Step 5 The DNA fragment obtained in step 4 was digested with BamHI and Hincll using the following reaction mixture composition DNA fragment obtained in step 4 30 0xBamHI buffer mentioned above 5BamHI 12 U F , Takara Shuzo 3 Hindu 6 U l, Takara Shuzo 6 H20 6 Total 50EMI27.1 The reaction was carried out at 37 C overnight and, then, the whole reaction mixture was applied to a 5 polyacrylamide gel, followed by electrophoresis.After migration, a fragment of about 700 bp was recovered by electroelution. After extraction with phenol, 30 pg of tRNA was added as a carrier and precipitation was caused with ethanol. The DNA precipitate thus recovered was dissolved in 25 of H20. Step 6 The vector prepared in step 1 and theIFN r gene containing DNA fragment prepared in step 5 were ligated together using the following reaction mixture composition Vector fragment obtained in step 1 2DNA fragment obtained in step 5 5 10xligation buffer mentioned above 3T4 DNA ligase 175 U l, Takara Shuzo 2HzO 18 Total 30EMI27.2 The reaction was carried out at 4 C overnight. d Transformation After the reaction in the above step 6, the whole reaction mixture was used for transformation of E. coli HB101. As a result, 24 Apr transformants were obtained.These strains were subjected to plasmid extraction by the alkaline method and the plasmids extracted were examined for their structure using various restriction enzymes. 22 Strains were found to have pI r amys shown in Fig. 5. One of these strains pI 7 amy5 carrying E. coli HB101 was checked for its ability to produceIFN r. e Production of IFN r The pi r amy5 carrying strain of E. coli HB101 was grown in a 2 liter erlenmeyer flask with shaking at 37 C using 500 ml of a medium specified below according toPalva et al. Gene, 22, 229 1983 , with checking at timed intervals for bacterial growth in terms of measured A650 values. Yeast extract Difco 10 g liter Bactotryptone Difco 20 g liter NaCl 10 g liter NH4 2SOI 1 g liter K2HP04 7 g liter KHzP04 2 g liter Ampicillin 25 mg liter f Recovery of IFN r The culture obtained in e was sampled 20 ml at 3 hour intervals and the samples were each fractionated according to the flowchart given later herein.The culture supernatant, cell washings 1 and 2 , periplasmic fraction and cell extract were assayed forIFN r titer by the RPHA method. At 12 hours after start of cultivation, the IFN r potencies thus measured are shown in Table 2. Table 2 IFN r titer IU liter of medium Culture supernatant 1.88 x 104 Cell washings 2.65 x 104 Cell extract 2.65 x 106 Periplasmic fraction 3.13 x 106 Then, the periplasmic fractions and cell extract fractions collected at 3 hour intervals over the period of hour 6 to hour 24 were assayed for IFN T titer. The results thus obtained are shown in Table 3. As is evident from the data given in Table 3, the yield increased with the growth of bacteria and became constant at 15 hours after start of cultivation. Not less than 50 of intracellular IFN r was found secreted in the periplasmic space. Table 3 IFN gamma titer IU liter of medium EMI30.1 tb Cultivation SEP Periplasmic SEP Cell SEP A650 SEP tb SEP period SEP fraction SEP extract tb SEP 6 SEP 5.22 SEP x105 SEP 6.63x10 SEP 3.70 tb SEP 9 SEP 1.04 SEP x SEP 10 SEP 1.33 SEP x SEP 10 SEP 7.15 tb SEP 12 SEP 3.13 SEP x106 SEP 2.65x10 SEP 9.40 tb SEP 15 SEP 3.13 SEP x10 SEP 3.98x10 SEP 10.90 tb SEP 18 SEP 3.13 SEP x10 SEP 3.98x104 SEP 11.00 tb SEP 21 SEP 3.13 SEP x106 SEP 3.98x106 SEP 11.01 tb SEP 24 SEP 3.13 SEP x106 SEP 3.98xl0 SEP 11.03 tb g Examination of IFN r secreted into periplasmic fraction for some characteristics The pI r amys carrying strain E. coli HB101 was shake cultured at 37 C in a 100 ml erlenmeyer flask using 20 ml of a medium according to Palva et al. men tioned above . Nine hours after start of cultivation, when A650 6.50, 200 Ci of H glutamine or H lysine Amersham was added and the shake culture was continued for additional 15 minutes. Thereafter, the erlenmeyer flask was cooled with ice to thereby terminate the labelling. Cells were harvested by cen trifugation and a periplasmic fraction was prepared accoding to the flowchart given later herein.The periplasmic fraction obtained was applied to a column packed with 2 ml of anti IFN 7 monoclonal antibody Sepharose to thereby cause adsorption of IFN r. The column was washed with 0.1 1.0 M NaCl for removal of impurities.Then, 3.5 M KSCN was poured into the column to cause elution of IFN r . The KSCN was removed by dialysis and the dialysate was concenterated to about 200 oX by ultrafiltration. Then, a 25 Fe portion of the thuspurified IFN r solution was subjected to SDS polyacrylamide gel electrophoresis. After fixation of the gel, sensitization and drying, autoradiography was performed.As a result, a single band was observed in the neighborhood of a molecular weight of about 18.000. In the next place, a 100 Fe portion of the purified IFN T was subjected to Edman degraddation using an Applied Biosystems model 470A protein sequencer.Each fraction from the N terminal as obtained was measured for radio activity using a liquid scintillation counter. The results are shown in Fig. 6. As is evident from the results, in the case of labelling with 3H glutamine, 3H uptake was observed in the first cycle whereas in the case of labelling with 3H lysine, 3H uptake was noted in the 6th, 12th and 13th cycles. This suggests that the IFN r secreted into the periplasmic fraction has the following amino acid sequence in the neighborhood of its N terminal 1 5 10 Glu X X X X Lys X X X X X Lys Lys X unknown .The above sequence is in agreement with the amino acid sequence of mature IFN r which starts with Gln shown inTable 4. Thus, it was confirmed that the IFN r secreted into the periplasmic space has the same amino acid sequence as that of native IFN r which is free of Met at the N terminal. Table 4 alpha amylase signal sequence and base sequence of linking portion of IFN gamma gene in pI gamma amys and corresponding amino acid sequence 27 28 29 30 31 1 2 3 4 5 6 7 8 9 10 11 12 13 Gly Pro Ala Ala Ala Gln Asp Pro Tyr Val Lys Glu Ala Glu Asn Leu Lys Lys GGa CCG GCT GCA GCG CAG GAT CCG TAT GTA AAA GAA GCA GAA AAc CTT AAG AAA CCT GGC CGA CGT CGC GTC CTA GGC ATA CAT TTT CTT CGT CTT TTG GAA TTC TTT EMI33.1 alpha amylase SEP SEP SEP IFN gamma SEP tb signal SEP sequence tb Flowchart illustrating the process for recovering IFN alpha 2 EMI34.1 20 SEP ml SEP of SEP culture tb SEP Centrifugation SEP 8.000 SEP rpm, SEP 4 C, SEP 10 SEP minutes tb SEP SEP tb Cell SEP pellet SEP Supernatant SEP culture SEP supernatent tb SEP Suspended SEP in SEP 5 SEP ml SEP of SEP cold SEP aqueous SEP solution tb SEP containing SEP 10 SEP mM SEP Tris HCl pH SEP 8.0 SEP and SEP 30 SEP mM SEP NaCl tb SEP Centrifugation SEP 6.000 SEP rpm, SEP 4 C, SEP 5 SEP minutes tb SEP SEP tb Cell SEP pellet SEP Supernatant SEP cell SEP washings SEP 1 tb Suspended in 5 ml of aqueous solution containing 20 sucrose and 30 mM Tris HCl pH 8.0 Addition of 10 l of 0.5 M EDTA pH 8.0 final concentration 1 mM EDTA Shaking at room temperature for 10 minutesCentrifugation 6.000 rpm, 4 C, 5 minutes9 to be continued EMI35.1 SEP SEP tb Cell SEP pellet SEP Supernatant SEP cell SEP washings SEP 2 tb SEP Suspended SEP in SEP 5 SEP ml SEP of SEP cold SEP distilled SEP water tb SEP Shaking SEP on SEP an SEP ice SEP bath SEP for SEP 10 SEP minutes tb SEP Centrifugation SEP 6.000 SEP rpm, SEP 4 C, SEP 5 SEP minutes tb SEP SEP tb Cell SEP pellet SEP Supernatant SEP periplasmic SEP fraction tb SEP Suspended SEP in SEP 1.5 SEP ml SEP of SEP lysis SEP buffer tb SEP allowed SEP to SEP stand SEP at SEP 0 C SEP for SEP 30 SEP minutes tb SEP Five SEP repetitions SEP of SEP freezing thawing SEP cycle tb SEP Centrifugation SEP 18.000 SEP rpm, SEP 4 C, SEP 90 SEP minutes tb SEP SEP tb Cell SEP pellet SEP Supernatant SEP cell SEP extract tb 50 mM Tris HCl pH 8.0 , 30 mM NaCl, 6 mM EDTA, 10 g ml phenylmethylsulfonyl fluoride, 1 mg ml lysozyme