# Cysteine depleted muteins of biologically active proteins, their preparation, formulations containing them, and structural genes, vectors and organisms, and their production, suitable for use in the preparation of said muteins.

## Abstract
The invention provides muteins of biologically active proteins in which cysteine residues that are not essential to biological activity have been deleted or replaced with other amino acids to eliminate sites for intermolecular crosslinking or incorrect intramolecular disulfide bridge formation. These muteins may be made via bacterial expression of mutant genes that encode the muteins and have been synthesized from the genes for the parent proteins by oligonucleotide directed mutagenesis.

## Claims
CLAIMS 1. A recombinant, synthetic biologically active mutein of a biologically active protein normally having at least one cysteine residue that is free to form a disulfide link and is non essential to said biological activity in which at least one of said cysteine residues has been deleted or replaced by another amino acid. 2. A synthetic mutein as claimed in claim 1. wherein there is only one of said cysteine residues. 3. A synthetic mutein as claimed in claim 1 or claim 2. wherein said cysteine residue s is are replaced by serine or threonine, or by glycine, alanine, valine, leucine, isoleucine, histidine, tyrosine, phenylalanine, tryptophan or methionine. 4. A synthetic mutein as claimed in any one of claims 1 to 3 and which is unglycosylated. 5. A synthetic mutein as claimed in any one of claims 1 to 4, wherein the protein is tumor necrosis factor. lymphotoxin, colony stimulating factor l, orIFN alphal. 6. A structural gene having a DNA sequence encoded for a synthetic mutein as defined in any one of claims 1 to 5. 7. An expression vector including a structural gene as defined in claim 6 in a position that permits expression thereof. 8. A host cell or organism which has been transformed with an expression vector as defined in claim 7, or progeny thereof. 9. E.coli which has been transformed with an expression vector as defined in claim 7, or progeny thereof. 1O. A method for making a gene as defined in claim 6, comprising a hybridizing single stranded DNA comprising a strand of a structural gene that is encoded for said protein with an oligonucleotide primer that is complementary to a region of said strand that includes the codon for said at least one cysteine residue or the antisense triplet paired with said codon. as the case may be, except for a mismatch with said codon or said antisense triplet that defines a deletion of the codon or a triplet that codes for another amino acid b extending the primer with DNA polymerase to form a mutational heteroduplex and c replicating said mutational heteroduplex. 11. A method as claimed in claim 1O, wherein the mismatch defines a triplet that codes for serine or threonine. 12. A method as claimed in claim 10 or claim 11, wherein the single stranded DNA is provided by a single stranded phage that includes said strand and the mutational heteroduplex of step b is converted to a closed circular heteroduplex. 13. A method as claimed in any one of claims 10 to 12, wherein said replicating is effected by trays forming a competent bacterial host with the closed circular heteroduplex and culturing the resulting transformants. 14. A method as claimed in any one of claims 10 to 13 also including the additional steps of isolating progeny of the mutant strand of the heteroduplex, isolating DNA from said progeny, and isolating said gene from the DNA from said progeny. 15. An oligonucleotide suitable for use in a method as defined in claim 10 and having a nucleotide sequence that is complementary to a region of the strand of the structural gene that includes the codon for the cysteine residue or the antisense triplet paired with said codon, as the case may be, except for a mismatch with said codon that defines a deletion of the codon or a triplet that codies for another amino acid. 16. A process for preparing an oligonucleotide suitable for use in a method as defined in claim 10 which comprises synthesizing by a manner known per se a nucleotide sequence that is complementary to a region of the strand of the structural gene that includes the codon for the cysteine residue or the antisense triplet paired with said codon, as the case may be, except for a mismatch with said codon that defines a deletion of the codon or a triplet that codes for another amino acid. 17. A process for producing a vector as defined in claim 7 comprising inserting into a vector capable of transforming a host cell a gene as defined in claim 6 in a position permitting expression of said gene. 18. A process for producing a cell which is capable of the production of a mutein as defined in any one of claims 1 to 5, which process comprises transforming a cell lacking such capability as aforesaid with an expression vector including a structural gene encoded for said mutein in a position that permits expression thereof. 19. A process for making a synthetic mutein as claimed in any one of claims 1 to 5, comprising culturing a host cell or organism as defined in claim 8 or E.coli as defined in claim 9, or progeny thereof, and harvesting the synthetic mutein from the resulting culture. 20. A method of preventing a protein having at least one cysteine residue that is free to form a disulfide link from forming said link comprising mutationally altering the protein by deleting the cysteine residue or replacing the cysteine residue with another amino acid, e.g., with serine or threonine. 21. A method as claimed in claim 20, wherein the protein is biologically active and the cysteine is not essential to said biological activity. 22. A method of making a pharmaceutical or veterinary formulation, which method comprises formulating for pharmaceutical or veterinary use. respectively, a synthetic mutein as defined in any one of claims 1 to 5. 23. A pharmaceutical or veterinary formulation comprising a synthetic mutein as claimed in any one of claims 1 to 5 formulated for pharmaceutical or veterinary use, respectively, and optionally also comprising a pharmaceutically acceptable or veterínarily acceptable diluent, carrier or excipient and or optionally being in unit dosage form. 24. A method of preventing a protein having at least one cysteine residue that is free to form a disulfide link from forming said link in a case where said link can be inhibitory of biological activity in said protein, comprising employing site specific mutagenesis to produce a vector which encodes said protein modified by said cysteine residue being deleted or replaced, cloning said vector using it to transform a host, using the resulting transformed hosts or progeny thereof to express protein encoded by the genotype thereof, assessing proteins thus expressed by comparative biological effectiveness analysis to determine the clone corresponding to the desired modified protein, and using said clone to produce said protein as a biologically effective entity. 25. A synthetic mutein as claimed in any one of claims 1 to 5 for use in prophylaxis, therapy or diagnosis practised on the human or animal body.

## Description
CYSTEINE DEPLETED HUTEINS OF BIOLOGICALLY ACTIVE PROTEINS,THEIR PREPARATION, FORMUL.STIOlIS CONT,kINTHS THEM, ANDSTRUCTURAL GENES, VECTORS AND ORGANISMS, AND THEIRPRODUCTION, SUITABLE FOR USE IN THE PREPARATION OF SAIDMUTEINS This invention is in the general area of recombinant DNA technology. More specifically it relates to mutationally altered biologically active proteins that differ from their parent analogs by one or more substitutions deletions of cysteine residues. Biologically active proteins that are microbially produced via recombinant DNA rDNA technology may contain cysteine residues that are nonessential to their activity but are free to form undesirable intermolecular or intramolecular links. One such protein is microbially produced human beta interferon IFN J3 . In the course of the preparation of IFN B by rDNA techniques, it has been observed that dimers and oligomers of o cYobially produced IFN B are formed inE.coli extracts containing high concentrations ofIFN B.This multiiner formation renders purification and separation of IFN B very laborious and timeconsuming and necessitates several additional steps in purification and isolation procedures such as reducing the protein during purification and reoxidizing it to restore it t its original conformation, thereby increasing the possibility of incorrect disulfide bond formation. Furthermore, microbially produced IFN B has also been found to exhibit consistently low specific activity due perhaps to the formation of multimers or of random intramolecular disulfide bridges.It would be desirable, therefore, to be able to alter microbially produced biologically active proteins such as IFN R in a manner that does not affect their activity adversely but reduces or eliminates their ability to fonn intermolecular crosslinks or intramolecular bonds that cause the protein to adopt an undesirable tertiary structure eg, a conformation that reduces the activity of the protein . The present invention is directed to producing by directed mutagenesis techniques mutationally altered biologically active proteins such proteins are called muteins , Glossary of Genetics andCytogenetics, 4th Ed, p 381, Springer Verlag 1976 that retain the activity of their parent analogs but lack the ability to form intermolecular links or undesirable. intramolecular disulfide bonds. In this regard Shepard, H.M., et al, Nature 1981 294 563 565 describe a mutein of IFN S in which the cysteine at position 141 of its amino acid sequence there are three cysteines in native human IFN ss at positions 17, 31, and 141, Gene 1980 10 11 15 and Nature 1980 285 542 547 is replaced by tyrosine.This mutein was made by bacterial expression of a hybrid gene constructed from a partial IFN B cDNA clone having aEMI2.1 transition at nucleotide 485 of the IFN B gene. The mute in lacked the biological activity of native IFN ss leading the authors to conclude that the replaced cysteine was essential to activity. Directed mutagenesis techniques are well known and have been reviewed by Lather, R.F. andLecoq, J.P. in Genetic Engineering Academic Press 1983 pp 31 50. Oligonucleotide directed mutagenesis is specifically reviewed by Smith, M. and Gillam, S.in Genetic Engineering rinciples and Methods,Plenum Press 1981 3 1 32. Reference is made to copending application no.83306221.9, which is specifically concerned with interleukin 2 and interferon beta muteins, and from which the present application is divided. In the present invention, however, one aspect is a recombinant, synthetic mutein of a biologically active protein normally having at least one cysteine residue that is free to orm a disulfide link and is nonessential to said biological activity in which at least one of said cysteine residues has been deleted or replaced by another amino acid. Another aspect of the invention are synthetic structural genes having DNA sequences that have been specifically designed designer genes to encode the above described synthetic muteins. Other aspects of this aspect are expression vectors that include such structural designer genes, host cells or organisms transformed with such vectors, and processes for making the synthetic mutein by culturing such transformants or their progeny and recovering the mutein from the culture. In the case of mute ins that have therapeutic utility, therapeutic fonxlaticns, whether for human or veterinary use and optionally in unit dosage form are another aspect of the invention. Another aspect of the invention is a method of preventing a protein having one or more cysteine residues that is free to form an undesirable disulfide link from forming such a link characterized in that the protein is mutationally altered by deleting the cysteine residue s or replacing ittthem with other amino acids. Still another aspect of the invention is a method for making the above describe synthetic structural gene by oligonucleotide directed mutagenesis comprising the following steps a hybridizing single stranded DNA compri sing a strand of a structural gene that encodes the parent protein with a mutant oligonucleotide primer that is complementary to a region of the strand that includes the codon for the cysteine to be deleted or replaced or the anti sense triplet paired with the codon, as the case may be, except for a mismatch with that codon or antisense triplet, as the case may be, that defines a deletion of the codon or a triplet that encodes said other amino acid b extending the primer with DNA polymerase to form a nutational heteroduplex and c replicating the mutational heteroduplex. The mutant oligonucleotide primers used in this process are another aspect of the invention. The invention will now be further described and illustrated withreference to the accompanying drawings, in which Figure 1 is a diagram of the amino acid sequence of IFN ss Figure 2 is a schematic illustration showing the preparation of a mutant IFN ss gene by oligonucleotide directed mutagenesis Figure 3 shows a diagram of plasmid pssltrp including the IFN ss gene Figure 4 is a diagram of the cloning vectorM13mp8 phage Figure 5 shows the restriction map of clone M13 f3l Figure 6 shows te sequencing gel pattern of the mutant serl7 gene showing a single base change in the coding region Figure 7 is a diagram of the expression plasmid pTrp3 Figure Sa shows the HinfI restriction pattern of clone pSY2501 and Figure 8b shows the resulting two 169bp and 28bp fragments thereof Figure 9 is a restriction map of clone pSY2501 Figure 10 shows the coding DNA sequence for the mute in ssserl7 with the corresponding amino acid sequence there for Figure 11 shows the single 18,000 dalton protein band corresponding to IFN Rserl7 in the extracts of clones pSY2501 and pBltrp Figure 12 is a diagram of the plasmid pLwl which contains the human interleukin 2 IL 2 gene under the control of the E.coli trp promoter Figure 13 is a restriction map of phage clone M13 IL2 Figure 14 is a restriction nap of the plasmid pLW46 and Figures 15a and 15b show, respectively, the nucleotide sequence of the coding strand of the clone pLW46 and the corresponding amino acid sequence of theIL 2 mutein designated IL 2Serl25. The present invention provides mute ins of biologically active proteins in which cysteine residues that are not essential to biological activity have been deliberately deleted or replaced with other amino acids to eliminate sites for intermolecular crosslinking or incorrect intramolecular disulfide bond formation mutant genes coding for such mute ins and means for making such muteins. Proteins that may be mutationally altered according to this invention may be identified from available information regarding the cysteine content of biologically active proteins and the roles played by the cysteine residues with respect to activity and tertiary structure. For proteins for which such information is not available in the literature this information may be determined by the skilled man by systematically altering each of the cysteine residues of the protein by the procedures described herein and testing the biological activity of the resulting muteins and their proclivity to form undesirable intermolecular or intramolecular disulfide bonds.Accordingly, while the nature of the invention is specifically illustrated below by reference to muteins of IFN ss and IL 2 it will be appreciated that the present invention is generally of applicability to any biologically active protein that contains a functionally nonessential cysteine residue that makes the protein susceptible to undesirable disulfide bond formation, and the specific teachings hereinafter are to be interpreted, accordingly, as clear guidance to the skilled reader in performing the generality of the present invention. Examples of proteins other than IFA ss and IL 2 that are candidates for mutational alteration according to the invention are tumor necrosis factor and colony stimulating factor, and IFN I. Candidate proteins will usually have an odd number of cysteine residues. The invention includes, of course, muteins of such proteins. In the case of IFN B it has been reported in the literature that both the glycosylated and unglycosylated IFNs show qualitatively similar specific activities and that, therefore, the glycosyl moieties are not involved in and do not contribute to the biological activity of IFN B. However, bacterially produced IFN B which is unglycosylated consistently exhibits quantitatively lower specific activity than native IFN B which is glycosylated. IFN B is known to have three cysteine residues at positions 17, 31 and 141. Cysteine 141 has been demonstrated by Shepard, et al, supra, to be essential for biological activity.In IFN a, which contains four cysteine residues, there are two intramolecular S S bonds one between cys 29 and cys 138 and another between cys 1 and cys 98. Based on the homology between IFN B and IFN as cys 141 of IFN B could be involved in an intramolecular S S bond with cys 31, leaving cys 17 free to form intermolecular crosslinks. By either deleting cys 17 or substituting it by a different amino acid, one can determine whether cys 17 is essential to biological activity, and its role in SS bond formation. If cys 17 is not essential for the biological activity of the protein, the resulting cys 17 deleted or cys 17 substituted protein might exhibit specific activity close to that of nativeIFN B and would possibly also facilitate isolation and purification of the protein. By the use of the oligonucleotide directed mutagenesis procedure with a synthetic oligonucleotide primer that is complementary to the region of theIFN B gene at the codon for cys 17 but which contains single or multiple base changes in that codon, a designer gene may be produced that results in cys 17 being replaced with any other amino acid of choice.When deletion is desired the oligonucleotide primer lacks the codon for cys 17. Conversion of cys 17 to neutral amino acids such as glycine, valine, alanine, leucine, isoleucine, tyrosine, phenylalanine, histidine, tryptophan, serine, threonine and methionine is the preferred approach. Serine and threonine are the most preferred replacenents because of their chemical analogy to cysteine. When the cysteine is deleted, the mature mutein is one amino acid shorter than the native parent protein or the microbially produced IFN R. Human IL 2 is reported to have three cysteine residues located at positions 58, 105, and 125 of the protein. As in the case of IFN ss, IL 2 is in an aggregated oligomeric form when isolated from bacterial cells and has to be reduced with reducing agents in order to obtain a good yield from bacterial extracts. In addition, the purified reduced IL 2 protein is unstable and readily reoxidized upon storage to an oligomeric inactive form. The presence of three cysteines means that upon reoxidation, the protein may randomly form one of three possible intramolecular disulfide bridges, with only one of those being the correct bridge as found in the dative molecule.Since the disulfide structure of the nativeIL 2 protein is not known, it is possible to use the present invention to create mutations at codons 58, 105 and 125 of the IL 2 gene and identify which cysteine residues are necessary for activity and therefore most likely to be involved in native disulfide bridge formation. In the same vein, the cysteine residue that is not necessary for activity can be modified so as to prevent the formation of incorrect intramolecular disulfide bridges and minimize the chance of intermolecular disulfide bridges by removal or replacenent of the free cysteine residue. The size of the oligonucleotide primer is determined by the requirement for stable hybridization of the primer to the region of the gene in which the mutation is to be induced, and by the limitations of the currently available methods for synthesizing oligonucleotides. The factors to be considered in designing oligonucleotides for use in oligonucleotidedirected mutagenesis eg, overall size, size of portions flanking the mutation site are described by Smith, M. and Gillam S., supra. In general the overall length of the oligonucleotide will be such as to optimize stable, unique hybridization at the mutation site with the 5 and 3 extensions from the mutation site being of sufficient size to avoid editing of the mutation by the exonuclease activity of the DNA polymerase.Oligonucleotides used for mutagenesis in accordance with the present invention usually contain from about 12 to about 24 bases, preferably from about 14 to about 20 bases and still more preferably from about 15 to about 18 bases. They will usually contain at least about three bases 3 of the altered or missing codon. The method for preparing the modified IFN ss gene broadly involves inducing a site specific mutagenesis in the IFN B gene at codon 17 TGT using a synthetic nucleotide primer which omits the codon or alters it so that it codes for another amino acid.When threonine replaces the cysteine and the primer is hybridized to the antisense strand of the IFN ss gene, the preferred nucleotide primer is GCAATTTTCAGACTCAG underlining denotes the altered codon . When it is desirable to delete cysteine, the preferred primer isAGCAATTTTCAGCAGAAGCTCCTG, which omits the TGT codon for cys. When cysteine is replaced by serine, a 17 nucleotide primer, GCAATTTTCAGAGTCAG, which includes an AGT codon for serine is the primer of choice. The T A transition of the first base in the cys 17 codon results in changing cysteine to serine.It must be recognized that when deletions are intro ducked, the proper reading frame for the DNA sequence must be maintained for expression of the desired protein. The primer is hybridized to single stranded phage such as M13, fd, or Xl74 into which a strand of the IFN R gene has been cloned. It will be appreciated that the phage may carry either the sense strand or antisense strand of the gene When the phage carries the anti sense strand the primer is identical to the region of the sense strand that contains the codon to be mutated except for a mismatch with that codon that defines a deletion of the codon or a triplet that codes for another amino acid then the phage carries the sense strand the primer is complementary to the region of the sense strand that contains the codon to be mutated except for an appropriate mismatch in the triplet that is paired with the codon to be deleted.Conditions that may be used in the hybridization are described by Smith, M. and Gillam, S., supra. The temperature will usually range between about oec and 700C, more usually about lO C to 500C. After the hybridization, the primer is extended on the phage DNA by.reaction with DNA polymerase I, To DNA polymerase, reverse transcriptase or other suitable DNA polymerase. The resulting dsDNA is converted to closed circular dsDNA by treatment with a DNA ligase such asT4 DNA ligase. DNA molecules containing singlestranded regions may be destroyed by S1 endonuclease treatment. Oligonucleotide directed mutagenesis may be similarly employed to make a mutant IL 2 gene that encodes a mutein having IL 2 activity but having cys 125 changed to serine 125. The preferred oligonucleotide primer used in matting this mutant IL 2 gene when the phage carries the sense strand of the gene is GATGATGCTTCTGAGAAAAGGTAATC. This oligonucleotide has a C G change at the middle base on the triplet that is paired with codon 125 of the IL 2 gene. The resulting mutational heteroduplex is then used to transform a competent host organism or cell. Replication of the heteroduplex by the host provides progeny from both strands. Following repli cation the mutant gene may be isolated from progeny of the mutant strand, inserted into an appropriate expression vector, and the vector used to transform a suitable host organism or cell. Preferred vectors are plasmids pBR322, pCRl, and variants thereof, synthetic vectors and the like. Suitable host organisms areE.coli, Pseudomonas, Bacillus subtilis, Bacillus thuringiensis, various strains of yeast, Bacillus thermophilus, animal cells such as mice, rat orChinese hamster ovary CHO cells, plant cells, animal and plant hosts and the like.It must be recognized that when a host of choice is transformed with the vector, appropriate promoter operator sequences are also introduced in order for the mutein to be expressed. Hosts may be prokaryotic or eukaryotic processes for inserting DNA into eukaryotic cells are described in PCT applications nos US81 00239 and US81 00240 published 3 September 1981 . E.coli andCHO cells are the preferred hosts. The mute ins obtained in accordance with the present invention may be glycosylated or unglycosylated depending on the glycosylation occurring in the native parent protein and the host organism used to produce the mute in. If desired, unglycosylated mutein obtained when E.coli or a Bacillus is the host organism, may be optionally glycosylated in vitro by chemical, enzymatic and other types of modifications known in the art. In connection with IFN ss, the cysteine residue at position 17 in the amino acid sequence of IFN B, as shown in Figure 1, is changed to serine by a T A transition of the first base of codon 17 of the sense strand of the DNA sequence which codes for the mature IFN D. The site specific mutagenesis is induced using a synthetic 17 nucleotide primer GCAATTTTCAGAGTCAG which is identical to a seventeen nucleotide sequence on the sense strand of IFN g in the region of codon 17 except for a single base mismatch at the first base of codon 17. The mismatch is at nucleotide 12 in the primer. It must be recognized that the genetic code is degenerate and that many of the amino acids may be encoded by more than one codon.The base code for serine, for example, is six way degenerate such that the codons, TCT, TCG, TCC, TCA, AGT, and ACG all code for serine. The AGT codon was chosen for the preferred embodiment for convenience. Similarly, threonine is encoded by any one of codons ACT, ACA, ACC andACG. It is intended that when one codon is specified for a particular amino acid, it includes all degenerate codons which encode that amino acid. The 17 mer is hybridized to single stranded M13 phage DNA which carries the antisense strand of the IFN B gene. The oligonucleotide primer is then extended on the DNA using DNA polymerase I Klenow fragment and the resulting dsDNA is converted to closed circular DNA with T4 ligase. Replication of the resulting mutational heteroduplex yields clones from the DNA strand containing the mismatch. Mutant clones may be identified and screened by the appearance or disappearance of specific restriction sites, antibiotic resistance or sensitivity, or by other methods known in the art.When cysteine is substituted with serine, the T A transition, shown in Figure 2, results in the creation of a new Hinfi restriction site in the structural gene. The mutant clone is identified by using the oligonucleotide primer as a probe in a hybridization screening of the mutated phage plaques, The primer will have a single mismatch when hybridized to the parent but will have a perfect match when hybridized to the mutated phage DNA, as indicated in Figure 2.Hybridization conditions can then be devised where the oligonucleotide primer will preferentially hybridize to the mutated DNA but not to the parent DNA. The newly generated HinfI site also serves as a means of confirming the single base mutation in the IFN ss gene. The M13 phage DNA carrying the mutated gene is isolated and spliced into an appropriate expression vector, such as plasmid pTrp3, and E.coli strain MM294 is transformed with the vector. Suitable growth media for culturing the transformants and their progeny are known to those skilled in the art. The expressed mutein of IFN B is isolated, purified and characterized. The following Examples are presented to help in the better understanding of the subject invention and for purposes of illustration and guidance. Examples 1 9 describe the preparation of a mutein of lFN 3. Examples 10 15 describe the preparation of a mutein ofIL 2. Example 1Cloning of the IFN ss Gene Into M13 Vector The use of 13 phage vector as a source of single stranded DNA template has been demonstrated byG.F. Temple et al, Nature 1982 296 537 540. Plasmid pssltrp Figure 3 containing the IFN B gene under control of E.coli trp promoter, was digested with the restriction enzymes HindIII and XhoII. The M13mp8 J. Messing, Third Cleveland Symposium onMacromolecules Recombinant DNA, Ed.A Walton,Elsevier Press, 143 153 1981 replicative form RF DNA Figure 4 was digested with restriction enzymes Hindill and BamHI, and mixed with the pssltrp DNA which had previously been digested with HindIII and XhoII. The mixture was then ligated with T4 DNA ligase and the ligated DNA transformed into competent cells ofE.coli strain JM 103 and plated on Xgal indicator plates J. Messing, et al, Nucleic Acids Res 1981 9 309 321 . Plaques containing recombinant phage white plaques were picked, inoculated into a fresh culture of JM i03 and minipreps of RF molecules prepared from the infected cells H.D. Birnboim and J. Doly, Nucleic Acid Res 1979 7 1513 1523 . The RP molecules were digested with various restriction enzymes to identify the clones containing the IFN B insert. The restriction map of one such clone M13 ssl is shown in Figure 5. Single stranded ss phage DNA was prepared from clone M13 ssl to serve as a template for site specific mutagenesis using a synthetic oligonucleotide. Example 2 Site Specific Mutagenesis Forty picomoles of the synthetic oligonucleotide GCAATTTTCAGAGTCAG primer was treated withT4 kinase in the presence of 0.1 mM adenosine triphosphate ATP , 50 mM hydroxymethylaninomethane hydrochloride Tris HCl pH 8.0, 10 mM MgCl2, 5 mM dithiothreitol DTT and 9 units of T4 kinase, in 50 pl at 37C for 1 hr. The kinased primer 12 pmole was hybridized to 5 yg of ss Ml 3 131 DNA in 50 ,ul of a mixture containing 50 mM NaCl, 10 mM Tris HCl, pH 8.0, 10 mM MgC12 and 10 mM 13 mercaptoethanol, by heating at 67 C for 5 min and at 42 C for 25 min.The annealed mixture was then chilled on ice and then added to 50 l of a reaction mixture containing 0.5 mM each of deoxynucleoside triphosphate dNTP , 80 mM Tris HCl, pH 7.4, 8 mM MgCl2, 100 mM NaCl, 9 units of DNA polymerase I, Klenow fragment, 0.5 mM ATP and 2 units of T4 DNA ligase, incubated at 37 C for 3 hr and at 25iC for 2 hr. The reaction was then terminated by phenol extraction and ethanol precipitation. The DNA was dissolved in 10 mM Tris HCl pH 8.0, 10 mM ethylenediaminetetraacetic acid EDTA , 50 sucrose and 0.05 bromophenylblue and electrophoresed on 0.88 agarose gel in the presence of 2 pg ml of ethidium bromide.The DNA bands corresponding to the RF forms of.M13 ssl were eluted from gel slices by the perchlorate method R.W. Davis, et al, AdvancedBacterial Genetics , Cold Spring Harbor Laboratory,N.Y., p. 178 179 1980 . The eluted DNA was used to transform competent JM 103 cells, grown overnight and ssDNA isolated from the culture supernatant. This ssDNA was used as a template in a second cycle of primer extension, the gel purified RF forms of the DNA were transformed into competent JM 103 cells, plated onto agar plates and incubated overnight to obtain phage plaques. Example 3Site Specific Mutagenesis The experiment of Example 2 above is repeated except that the synthetic oligonucleotide primer used is C CAATTTTCAGACTCAG to change codon 17 of the ISN B gene from one that codes for cysteine to one that codes for threonine. Example 4Site Specific Deletion The experiment of Example 2 above is repeated except that the synthetic oligonucleotide primer used is AGCAATTTTCAGCAGAAGCTCCTG to delete codon 17 of the IFN B gene. Example 5Screening And Identification of Mutagenized Plaques Plates containing mutated M13 81 plaques Example 1 as well as two plates containing unmutated M13 R1 phage plaques, were chilled to 4 C and plaques from each plate transferred onto two nitrocellulose filter circles by layering a dry filter on the agar plate for 5 min for the first filter and 15 min for the second filter. The filters were then placed on thick filter papers soaked in 0.2 N NaOH, 1.5 M NaCI and 0.28 Triton X 100 for 5 min, and neutralized by layering onto filter papers soaked with 0.5 M Tris HC1, pH 7.5 and 1.5 M NaCI for another 5 min. The filters were washed in a similar fashion twice on filters soaked in 2 x SSC standard saline citrate , dried and then baked in a vacuum oven at 80 C for 2 hr. The duplicate filters were prehybridized at 55iC for 4 hr with 10 ml per filter of DNA hybridization buffer 5 x SSC pH 7.0, 4 x Denhardtt s solution polyvinylpyrrolidine, ficoll and bovine serum albumin, 1 x 0.02 of each , 0.1 sodium dodecyl sulfate SDS , 50 mM sodium phosphate buffer pH 7.0 and 100 g ml of denatured salmon sperm DNA.32P labeled probe was prepared by kinasing the oligonucleotide primer with 32P labeled ATP. The filters were hybridized to 3.5 x 105 cpm ml of 32P labeled primer in 5 ml per filter of DNA hybridization buffer at 55iC for 24 hr. The filters were washed at 55 C for 30 min each in washing buffers containing 0.18 SDS and decreasing amounts of SSC.The filters were washed initially with buffer containing 2 x SSC and the control filters containing unmutated M13 B1 plaques were checked for the presence of any radioactivity using a Geiger counter. The concentration of SSC was lowered stepwise and the filters washed until no detectable radioactivity remained on the control filters with the unmutated M13 ssl plaques.The lowest concentration of SSC used was 0.1 x SSC.The filters were air dried and autoradiographed at 700C for 2 3 days. 480 plaques of mutated M13 ssl and 100 unmutated control plaques were screened with the kinased oligonucleotide probe. None of the control plaques hybridized with the probe while 5 mutatedM13 B1 plaques hybridized with the probe. One of the five mutated M13 B1 plaques M13 SY25Ol was picked and inoculated into a culture of JM 103. ssDNA was prepared from the supernatant and double stranded ds DNA was prepared from the cell pellet. The ssDNA was used as a template for the dideoxy sequencing of the clone using the M13 univer sal primer. The result of the sequence analysis is shown in Figure 6, confirming that the TGT cys codon has been converted to an AGT ser codon. Example 6Expression of Mutated IFN ss in E.coli RF DNA from rAl3 SY2501 was digested with restriction enzymes Hindili and XhoII and the 520 bp insert fragment purified on a 1 agarose gel. The plasmid pTrp3 containing the E.coli trp promoter Figure 7 was digested with the enzymes Hindili andBamHI, mixed with the purified M13 SY2501 DNA fragment, and ligated in the presence of T4 DNA ligaseThe ligated DNA was transformed into E.coli strainMM294. Ampicillin resistant transformants were screened for sensitivity to the drug tetracycline. Piasmid DNA from five ampicillin resistant, tetracylcine sensitive clones were digested with Hinfl to screen for the presence of the M13 SY2501 insert.Figure 8a shows the HinfI restriction pattern of one of the clones pSY2501 , comparing it with the HinfI pattern of the original IFN B clone, pBltrp. As expected, there is an additional HinfI site in pSY2501, cleaving the 197 bp IFN B internal fragment to a 169 bp fragment and a 28 bp fragment Figure 8b . A restriction map of the clone pSY2501 is shown in Figure 9. The complete DNA sequence of the mutant IFN ss gene is shown in Figure 10 together with the predicted amino acid sequence. The plasmid designated as clone pSY250lwas Geposite2 on 39 March 1933 with the Agricultural Research CultureCollection NRRL , Fermentation Laboratory, NorthernRegional Research Center, Science and Education Administration, U.S. Department of Agriculture, 1815North University Street, Peoria, Illinois 60604 and is assigned accession numbers CMCC No. 1533 and NRRL No. Ei 15356. Cultures of pSY25Ol and pBltrp, which include progeny thereof, were grown up to an optical density OD600 of 1.0. Cell free extracts were prepared and the amount of IFN ss antiviral activity assayed on GM2767 cells in a microtiter assay.Extracts of clone pSY2501 exhibited three to ten times higher activity than pssltrp Table I , indicating that clone pSY25û1 was either synthesizing more protein exhibiting IFN B activity or that the protein made had a higher specific activity. Table I EXTRACT ANTIVIRAL ACTIVITY U ml pSY2501 6 x 105 pssltrp 1 x 105 ptrp3 control 30 In order to determine if clone pSY2501 was synthesizing several times more active protein, the extracts of both clones were electrophoresed on a SDS polyacrylamide gel together with a control extract and the gel stained with coomasie blue to visualize the proteins. As shown in Figure 11, there was only one protein band corresponding to an apparent 18,000 dalton protein that was present in the extracts of clones pSY2501 and pnltrp but not in the control extract of ptrp3.This protein, which has a molecular weight of about 20,000 daltons but shows a gel migration pattern of an 18,000 dalton protein was previously shown to be IFN B by purification of this protein from extracts of pBltrp. Since there is less of this protein in extracts of pSY2501 than in extracts of pltrp, the specific activity of the protein in extracts of clone pSY2501 was higher than that of clone pssltrp. Example 7Purification of IFN ssser17 IFN ser17 was recovered from E.coli that had been transformed to produce IFN ssser17, TheE.coli were grown in the following growth medium to an ODof 10 11 at 680 nm dry wt 8.4 g l . Ingredient Concentration NH4Cl 20 mM K2S04 16.1 mM KH2PO4 7.8 mM Na2HPO4 12.2 mM MgSO4 7H2O 3 mM Na3 citrate .2H2O 1.5 mM MnSO4 4H20 30 pM ZnSOo .7H2O 30 pM CuSO4 .5H2O 3 pM L tryptophan 70 mg l FeSO4 .7K2O 72 pM thiamine HC1 20 mg l glucose 40 g l pH control with NH4OH A 9.9 1 9.9 kg harvest of the transformedE.coli was cooled to 20 C and concentrated by passing the harvest through a cross flow filter at an average pressure drop of 110 kpa and steady state filtrate flow rate of 260 ml min until the filtrate weight was 8.8 kg.The concentrate approximately one liter was drained into a vessel and cooled to 15 C. The cells in the concentrate were then disrupted by passing the concentrate through a Manton Gaulin homogenizer at 5 C, 69,000 kpa. The homogenizer was washed with one liter phosphate buffered saline, pH 7.4 PBS , and the wash was added to the disruptate to give a final volume of two liters. This volume was continuously centrifuged at 12000 x g at a 50 ml min flow rate. The solid was separated from the supernatant and resuspended in four liters PBS containing 2 by wt SDS.This suspension was stirred at room temperature for 15 min after which there was no visible suspended matervial. The solution was then extracted with 2 butanol at a 1 1 2 butanol solution volume ratio. The extract tion was carried out in a liquid liquid phase separator using a flow rate of 200 ml min. The organic phase was then separated and evaporated to dryness to yield 21.3 g of protein. This was resuspended in distilled water at a 1 10 volume ratio. The recovered product was assayed for humanIFN B activity using an assay based on protection against viral cytopathic effect CPE . The assay was made in microtiter plates. Fifty p1 of minimum essential medium were charged into each well and 25 p1 of the sample was placed in the first well and 1 3 volume dilutions were made serially into the following wells.Virus vesicular stomatitus , cell human fibroblast line GM 2767 , and reference IFN R controls were included on each plate. The reference IFN B used was 100 units per ml. The plates were then irradiated with UV light for 10 min. After irradiation 100 pl of the cell suspension. l.2 x 105 cells ml was added to each well and the trays were incubated for 18 24 hr.A virus solution at one plaque forming unit per cell was added to each well except the cell control. The trays were then incubated until the virus control showed 100 CPE. This normally occurred 18 24 hr after adding the virus solution. Assay results were interpreted in relation to the location of the 50 CPE well of the reference IFN ss control. From this point the titer of interferon for all samples on the plate was determined. The specific activity of the recovered product was determined to be 5 x 107 U mg. Example 8Acid Precipitation And Chromatographic Purification The process of Example 7 was repeated except that after extraction and separation of the aqueous and organic phases and mixing of the organic phase with PBS at a volume ratio of 3 1 the pH of the mixture was lowered to about 5 by addition of glacial acetic acid. The resulting precipitate was separated by centrifugation at 10000 17000 x g for 15 min and the pellet was redissolved in 10 w v SDS, 10 mM DTT, 50 mM sodium acetate buffer, pH 5.5, and heated to 80 C for 5 min, The solution was then applied to a Brown let RP 300, 10 pM, Aquapore column using a Beckman gradient system.Buffer A was 0.1t trifiuoroacetic acid TFA in H20 buffer B was 0.1 TFA in acetonitrile. Detection was by ultraviolet absorbance at 280 nm. The solvent program was linear gradient of 0 buffer B to 100 buffer B in three hr. Fractions containing highest interferon activities were pooled and the specific activity of the pooled interferon preparation was determined to be 9.0 x 107 to 3.8 x 108 international units per mg protein, as compared to about 2 x 108 U g for native IFN R. Example 9Biochemical Characterization of IFN B Ser17 Amino acid compositions were determined after 24 72 hr timed hydrolysis of 40 pg samples ofIFN in 200 pl of 5.7 N HCl, 0.1 phenol, at 108 C.Proline and cysteine were determined in the same fashion after performic acid oxidation in this case, phenol was omitted from the hydrolysis. Tryptophan was analyzed after 24 hr hydrolysis of 400 p1 samples in 5.7 N HCl, 10 mercaptoacetic acid no phenol .Analysis was performed on a Becknan 121MB anino acid analyzer using a single column of AA10 resin. The amino acid composition calculated from representative 24 ,48 , 72 hr acid hydrolyses of purified IFN B Serl7 agrees well with that predicted by the DNA sequence of the clones IFN gene, minus the missing N terminal methionine. The amino acid sequence of the first 58 residues from the amino acid terminus of purified IFN was determined on a 0.7 mg sample in a Beckman 890C sequanator with 0.1 M Quadrol buffer. PTH amino acids were determined by reverse phase HPLC on an Altex ultrasphere ODS column 4.6 x 250 mm at 45 C eluted at 1.3 min at 40 buffer X, and 8.4 min from 40 70 buffer B, where buffer A was 0.0115 M sodium acetate, 5 tetrahydrofuran THF , pH 5.11 and buffer B was 10 THF in acetonitrile. The N terninal amino acid sequence of IFN BSerl7 determined matches the expected sequence predicted from the DNA sequence, except for the absence ofN terminal methionine. As indicated above, the IFN Rserl7 preparation exhibits specific activity levels very close to or better than that of native IFN B. IFN B,,,17 has no free sulfhydryl groups but indicates one S S bond between the only remaining cysteines at positions 31 and 141. The protein does not readily form oligomers and appears to be substantially in the monomeric form.The IFN Bserl7 obtained in accordance with this invention may be formulated either as a single product or mixtures of the various forms, into pharmaceutically or veterinarily acceptable preparations, e.g. in inert, nontoxic, nonallergenic, physiologically compatible carrier media for clinical and therapeutic uses in cancer therapy or in conditions where interferon therapy is indicated and for viral infections. Such formulations may be in unit dosage form. Such media include but are not limited to distilled water, physiological saline,Ringer s solution, Hanks solution and the like. Other nontoxic stabilizing and solubilizing additives such as dextose, HSA human serum albumin and the like may be optimally included. The therapeutic formulations may be administered orally or parenterally such as intravenous, intramuscular, intraperitoneal and subcutaneous administrations. Preparations of the modified IFN ss of the present invention may also be used for topical applications in appropriate media normally utilized for such purposes. The principal advantages of the above described mutein of IFN ss lie in the elimination of a free sulfhydryl group at position 17 in IFN R, thereby forcing the protein to form correct disulfide links between cys 31 and cys 141 and to assume the conformation ostensibly required for full biological activity. The increased specific activity of the IFNJ3ser17 enables the use of smaller dosages in therapeutic uses. By deleting the cysteine at position 17 and eliminating the free Sl group, the IFN B,,,17 protein does not form dimers and oligomers so readily as the microbially produced IFN B. This facilitates purification of the protein and enhances its stability. Example 10 The nucleotide sequence for a cDNA clone coding for human IL 2, procedures for preparing IL 2 cDNA libraries, and screening same for IL 2 are described by Taniguchi, T., et al, Nature 1983 Vol 24, p 305 et seq. c.DBA libraries enriched in potential IL 2 clones were made from an IL 2 enriched mRNA fractions obtained from induced peripheral blood lymphocytes PBL and Jurkat cells by conventional procedures. The enrichment of the mRNA for IL 2 message was made by fractionating the mRNA and identifying the fraction having IL 2 mRNA activity by injecting the fractions in Xenopus laevis oocytes and assaying the oocyte lysates for IL 2 activity on HT 2 cells J. Watson, J Exp Med 1979 150 1570 1519 andS. Gillis et al, J Immun 1978 120 2027 2032. Example 11Screening and Identification of IL 2 cDNA Clones The IL 2 cDNA libraries were screened using the colony hybridization procedure.Each microtiter plate was replicated onto duplicate nitrocellulase filter papers S S type BA 85 and colonies were allowed to grow at 37 C for 14 16 hr on L agar containing 50 pg ml ampicillin. The colonies were lysed and DNA fixed to the filter by sequential treatment for 5 min with 500 mM NaOH, 1.5 M NaCl, washed twice for 5 min each time with 5 x standard saline citrate SSC . Filters were air dried and baked at 80 C for 2 hr.The duplicate filters were pre hybridized at 42 C for 6 8 hr with 10 ml per filter of DNA hybridization buffer 50 formamide, 5 x SSC, pH 7.0, 5 x Denhardt s solution polyvinylpyrrolidine, plus ficoll and bovine serum albumin 1 x 0.28 of each , 50 mM sodium phosphate buffer at pH 7.0, 0.28 SDS, 20 pg ml Poly U, and 5q pgfml denatured salmon sperm DNA. A 32P labeled 20 mer oligonucleotide probe was prepared based on the IL 2 gene sequence reported by Taniguchi, T., et al, supra. The nucleotide sequence of the probe was GTGGCCTTCTTGGGCATGTA. The samples were hybridized at 42 C for 2436 hr with 5 ml filter of DNA hybridization buffer containing the 32P cDNA probe. The filters were washed two times for 30 min each time at 50 C with 2 x SSC, 0.1 SDS, then washed twice with 1 x SSC and 0.18 SDS at 50 C for 90 min, air dried, and autoradiographed at 70 C for 2 to 3 days. Positive clones were identified and rescreened with the probe. Full length clones were identified and confirmed by restriction enzyme mapping and comparison with the sequence of the IL 2 cDNA clone reported by Taniguchi,T., et al, supra. Example 12Cloning of ll Gene into M13 Vector The IL 2 gene was cloned into M13mp9 as described in Example 1 using the plasmid pLWl Figure 12 containing the IL 2 gene under the control of theE.coli trp promoter. A sample of pLWl was deposited in the American Type Culture Collection, 12301Parklawn Drive, Rockville, Maryland 20852, USA, on 4 August 1983 and has been assigned ATCC number 39,405. The restriction map of one clone designated M13 IL2 containing the IL 2 insert is shown in Figure 13. Single stranded phage DNA was prepared from clone M13 IL2 to serve as a template for oligonucleotide directed mutagenesis. Example 13 Oligonucleotide directed Mutagenesis As indicated previously, IL 2 contains cys teine residues at amino acid positions 58, 105 and 125. Based on the nucleotide sequences of the por tions of the IL 2 gene that contain the codons for these three cysteine residues three oligonucleotide primers were designed and synthesized for mutating the codons for these residues to codons for serine. These oligonucleotides have the following sequences.CTTCTAGAGACTGCAGATGTTTC DM27 to change cys 58, CATCAGCATACTCAGACATGAATG DM28 to change cys 105 and GATGATGCTCTGAGAAAAGGTAATC DM29 to change cys 125. Forty picomoles of each oligonucleotide were kinased separately in the presence of 0.1 mM ATP, 50 mM Tris HC1, pH 8.0, 10 mM MgCl2, 5 mM DTT and 9 units of T4 kinase in 50 1 at 37µC for 1 hr. Each of the kinased primers 10 moles was hybridized to 2.6 pg of ss M13 IL2 DNA in 15 pl of a mixture containing 100 mM NaCl, 20 mM Tris HCl, pH 7.9, 20 mMMgC12 and 20 mM 5 mercaptoethanol, by heating at 67 C for 5 min and 42 C for 25 min.The annealed mixtures were chilled on ice and then adjusted to a final colume of 25 i l of a reaction mixture containing 0.5 mM of each dNTP, 17 mM Tris HCl, pH 7.9, 17 mM MgC12 83 mM NaCl, 17 m 13 mercaptoethanol, 5 units ofDNA polymerase I Klenow fragment, 0,5 mM ATP and 2 units of T4 DNA ligase, incubated at 37 C for 5 hr. The reactions were terminated by heating to 80 C and the reaction mixtures used to transform competentJM103 cells, plated onto agar plates and incubated overnight to obtain phage placques. Example 14Screening and Identification of Mutagenized PhagePlacques Plates containing mutagenized M13 1L2 placques as well as 2 plates containing unmutagenizedM13 IL2 phage placques, were chilled to 4 C and phage placques from each plate were transferred onto two nitrocellulose filter circles by layering a dry filter on the agar plate for 5 min for the first filter and 15 min for the second filter. The filters were then placed on thick filter papers soaked in 0.2 N NaOH, 1.5 M NaCl and 0.2 Triton for 5 min, and neutralized by layering onto filter papers soaked with 0.5 MTris HCl, pH 7.5, and 1.5 M NaCl for another 5 min.The filters were washed in a similar fashion twice on filters soaked in 2 x SSC, dried and then baked in a vacuum oven at 80 C for 2 hr. The duplicate filters were pre hybridized at 42 C for 4 hr with 10 ml per filter of DNA hybridization buffer 5 x SSC, pH 7.0, 4 x Denhardts solution polyvinylpyrrolidine, ficoll and bovin serum albumin, lx 0.02 of each , 0.1 SDS, 50 mM sodium phosphate buffer, pH 7.0 and 100 pg ml of denatured salmon sperm DNA. 32P labelled probes were prepared by kinasing the oligonucleotide primers with labelled ATP. The filters were hybridized to 0.1 x. 105 cpm ml of 32P labelled primers in 5 ml per filter of DNA hybridization buffer at 42 C for 8 hr.The filters were washed twice at 50 C for 30 min each in washing buffers containing 0.1 SDS and 2 x SSC, and twice at 500C for 30 min each with 0.1 SDS and 0.2 x SSC. The filters were air dried and autoradiographed at 70 C for 2 3 days. Since the oligonuclebtide primers DM28 andDM29 were designed to create a new DdeI restriction site in the mutagenized clones Figure 14 , RF DNA from a number of the clones which hybridized with each of these kinased primers were digested with the restriction enzyme DdeI. One of the mutagenized M13 IL2 placques which hybridized with the primer DM28 and has a new DdeI restriction site M13 LW44 was picked and inoculated into a culture of JM103, ssDNA was prepared from the culture supernatant and dsRF DNA was prepared from the cell pellet. Similarly, a placque which hybridized with primer DM29 was picked M13 LV146 and ssDNA and RF DNA prepared from it. The oligonucleotide primer DM27 was designed to create a new PstI restriction site instead of a DdeI site.Therefore, the placques that hybridized to this primer were screened for the presence of a new PstI site. One such phage placque was identified M13 LW42 and ssDNA and RF DNA prepared from it. The DNA from all three of these clones were sequenced to confirm that the target TGT codons for cysteine had been converted to aTCT codon for serine. Example 15Recloning of the Mutagenized IL 2 Gene for Expression in E.coli RF DNA from M13 LW42, M13 LW44 and M13 LW46 were each digested with restriction enzymes HindIII and BanII and the insert fragments purified from a 1 agarose gel. Sinilarly, the plasmid pTrp3 Figure 7 was digested with HindIII and Banal, the large plasmid fragment containing the trp promoter was purified on an agarose gel and then ligated with each of the insert fragments isolated from M13 LW42, M13 DW44 and M13 LW46. The ligated plasmids were transformed into competent E.coli K12 strain MM294.The plasmid DNAs from these transformants were analysized by restriction enzyme mapping to confirm the presence of the plasmids pLW42, ply44 and pLW46. Figure 14 is a restriction map of pLW46. µben each of these individual clones were grown in the absence of tryptophane to induce the trp promoter and cell free extracts analyzed on SDS polyacrylamide gels, all three clones, pLW42, pJV44 and pSi46, were shown to synthesize a 14.5 kd protein similar to that found in the positive control, pLW2l, which has been demonstrated to synthesize a 14.4 kd IL 2 protein.When these same extracts were subjected to assay for IL 2 activity on mouse HT 2 cells, only clones pç21 positive control and plow46 had significant amounts of IL 2 activity Table II below , indicating that cys 58 and cys 105 are necessary for biological activity and changing them to serines pLW42 and pLW44 respectively resulted in the loss of biological activity. Cys 125 on the other hand must not be necessary for biological activity because changing it to ser 125 pLW46 did not affect the biological activity. Table II Clones IL 2 Activity p ml pIL2 7 negative control 1 pLW21 positive control 113,000 pLW42 660 plow44 1,990 pLW46 123,000 Figure 15a shows the nucleotide sequence of the coding stzand of nlone pLW46. As compared to the coding strand of the native human IL 2 gene clone pLW46 has a single base change ofEMI31.1 at nucleotide 374. Figure 15b shows the corresponding amino acid sequence of the IL 2 mutein encoded by pLW46.This mutein is designated IL2ser125 As compared to native IL 2 the mutein has a serine instead of a cysteine at position 125. A sample of E.coli K12 strain MM294 transformed with pLW46 was deposited in the American TypeCulture Collection, 12301 Parklawn Drive, Rockville,Maryland 20852, USA on 26 September 1983 and has been assigned ATCC Number 39,452. Mute ins of IL 2 in which the cysteine at position 125 has been deleted or replaced with another amino acid, such as the mutein 1L2serl25 retain IL 2 activity. They may, therefore, be formulated and used in the same manner as native IL 2. Accordingly, such IL 2 mute ins are useful in the diagnosis and treatment of bacterial, viral, parasitic, protozoan and. fungal infections in manifestations of lymphokine or immunodeficiency for reconstitution of normal immunofunc tion in aged humans and animals in the development of diagnostic assays such as those employing enzyme amplification, radiolabelling, radioimaging, and other methods known in the art for monitoring IL 2 levels in the diseased state for the promotion of T cell growth in vitro for therapeutic and diagnostic purposes for blocking receptor sites for lymphokines and in various other therapeutic, diagnostic and research applications. The various therapeutic and diagnostic applications of human IL 2 have been investigated and reported in S.A. Rosenberg, E.A. Grimm, et al, A. Mazumder, et a3, an..RI.A. Grimm and S.A. Rosenberg. IL 2 mute ins may be used by themselves or in combina tion with other immunologically relevent B or T cells or other therapeutic agents. For therapeutic or diagnostic applications whether for human or veterinary use , they may, for example, be formulated in nontoxic, nonallergenic, physiolocally compatable carrier media such as distilled water, Ringer s solution,Hank s solution, physiological saline and the like. Administrations of the IL 2 mute ins to humans or animals may be oral or intraperitoneal or intramuscular or subcutaneous as deemed appropriate by the physician. Examples of relevant cells are B or T cells, natural killer cells, and the like and exemplary therapeutic reagents which may be used in combination with the polypeptides of this invention are the various interferons, especially gamma interferon, B cell growth factor, IL l and the like.