# Analogous interferon polypeptides, process for their preparation and pharmaceutical compositions containing them.

## Abstract
Analogues of human interferon alpha₂ in which an amino acid is present in at least one position selected from interferon alpha₂ positions 12 16, 21, 27 32, 37, 43a, 44, 59, 61, 64, 67, 98 115, 137 138 and 148 which differs from the amino acid in the corresponding position of interferon alpha₂, possess good antivi ral activity and may additionally be used as biochemical tools. The interferon analogues are prepared by culturing a microorgan ism, the microorganism having been transformed with a replic able plasmidic expression vehicle comprising genetic material coding for the said interferon analogue.

## Claims
Claims 1. A polypeptide of the formula I EMI65.1 in which XÚ, Xê, X , X4, X5, X6, X7, X8, X9, X10, X11, X155, X156,X157, X158, X159, X160, X161, X162, X163, X164 and X165 may each represent a naturally occurring alpha amino acid and X12 represents Arg or Ala, X13 represents Arg or Ala, X14 represents Thr or Ala, X15 represents Leu orAla, X16represents Met, Leu or Ile, X21 represents Met or Leu, X22 represents Ser, Arg or Gly and X23 represents Arg or Lys, X26 represents Leu or Pro, X27 represents Ser or Phe, X28 represents Ser or Ala X29 represents Ser or Cys, X30 represents Ile or Leu X31 represents Met or Lys, X32 represents Ala, Asn,Asp or Glu and X34 represents His or Pro, X37 representsGly or Ala, X38 represents Phe or Leu, X43a represents a single bond or Asp, X44 represents Gly or Ala, X50 represents Ala or Thr, X51 represents Glu, Pro or Glu X52 represents Ala or Thr, and X54 represents Ser orPro, X59 represents Leu or Met, X61 represents Glu orAla, X63 represents Ile or Thr, X64 represents Phe orAla, X67 represents Phe or Ala, X68 represents Thr orSer, X70 represents Lys or Glu, X77 represents Asp orGlu, X78 represents Glu or Gln, X79 represents Asp, Thr or Ser, X82 represents Asp or Glu X85 represents Cys,Tyr or Ser, X94 represents Asp or Asn, X98 representsCys or Leu r X99 represents Val or Thr, X100 represents Met, Ile or Asn, either X101 represents Gln,Tyr or Phe, X102 represents Glu, Gly, Ser or Ala, X103 represents Glu, Val or Lys, X104 represents Arg, Gly,Thr, Leu or Ala, X105 represents Val, Asp, Met or Thr, x106 represents Gly, Thr, Giti, Leu or Asn, X107 represents Glu, Asn or Tyr, X108 represents Thr or Val,X109 represents Pro or Gln, XllO represents Leu or Arg and xlii represents Met, Lys or Leu, or x101 x102 x103 x104 x105 x106 x107 X108 X109 X110 X111 represents Met, Lys or Leu, X1l2 represents Asn, Lys or Ala, xll3 represents Ala,Glu or Ile, X114 represents Asp or His, X115 representsSer or Phe, X120 represents Lys or Arg, X124 representsArg or Gln, X131 represents Thr or Lys, X137 represents Pro or Ala, X138 represents Cys or Ser, X148 represents Met or Leu, X151 represents Leu or Phe and X153 represents Leu or Phe, the amino acids being selected such that an amino acid is present in at least one position selected from alpha2 positions 12 16, 21, 28,29, 30, 32, 37, 44, 61, 64, 67, 98 115, 137, 138 and 148 which differs from the amino acid in the corresponding position of IFN alpha2, IFN alphal and IFN alphaI and or selected such that an amino acid is absent from at least one position selected from alpha2 positions 101 110 or the amino acids being selected such that an amino acid is present in at least one position selected from alpha2 positions 27, 31, 59, 151, 160 and 163 which is the same as the amino acid in the corresponding position of IFN alphal, the amino acids at alpha2 positions other than positions 1 to 11, 27, 31,43a, 59, 100, 102 104, 106, 112, 113, 151 and 156 to 165 being the same as the amino acids in the corresponding positions of IFN alpha2 or a N1 ll and or C1 ll truncated analogue thereof and the N3 1l truncated analogues of IFN alpha2, which may if desired be C1 ll truncated, with the proviso that a N3 truncated analogue of IFN alpha2 is also C711 truncated. 2. A polypeptide as claimed in claim 1 wherein X1 represents Cys, X2 represents Asp or Trp, X3 representsLeu or Cys, X4 represents Pro or Gln, X5 represents Glu,Gln or Asp, X6 represents Thr or Pro, X7 represents His or Tyr, X8 represents Ser, X9 represents Leu, X10 represents Asp, Gly or Ala, X11 represents Asn, Ser orAla, X155 represents Thr, X155 represents Thr, X156 represents Asn, x157 represents Leu, X158 representsGln, X159 represents Glu or Lys, x160 represents Arg,Ser or Ile, X161 represents Leu, X162 represents Arg,X163 represents Arg or Ser, X164 represents Lys and X165 represents Glu or Asp. 3. A polypeptide as claimed in claim 1 which is N1 3 and or C1 10 truncated. 4. A polypeptide as claimed in claim 1 of the formula II EMI68.1 Gln, XlOa represents Asp or Gly, Xlla represents Asn orSer, X27a represents Phe or Ser, X31a represents Lys orMet, X43a represents Asp or a single bond, X59a represents Met or Leu, XlO0a represents Met or Ile,X102a represents Glu or Gly, 103a represents Glu orVal, Xl04a represents Arg or Gly, Xl06a represents Gly or Thr, X1l2a represents Asn or Lys, X1l3a representsAla or Glu, K151a represents Leu or Phe, 160a represents Arg or Ser and X163a represents Arg or Ser with the proviso that the amino acids are selected such that at least one of X27a, X31a, X59a, 151a, X160a and X163a, represents the same amino acid as that in the corresponding position of IFN alphal . 5. A polypeptide as claimed in claim 1 of the formula 11 as defined in claim 4 in which x43a represents Asp, X100a represents Ile, Xl02a represents Gly, X103a represents Val, X104a representsGly, Xl06a represents Thr, X112a represents Lys andX113a represents Glu with the proviso that at least one of x5a XlOa, X11a, x151a, X160a and X163a represents an amino acid which differs from the amino acid in the corresponding position of IFN alpha2. 6. A polypeptide as claimed in claim 1 wherein X1 represents Cys, X2 represents Asp or Trp, X3 representsLeu or Cys, X4 represents Pro or Glu, X5 represents Gln or Asp, X6 represents Thr or Pro, X7 represents His orTyr, x8 represents Ser, X9 represents Leu, X10 represents Gly or Ala, X11 represents Ser or Ala, X16 represents Leu or Met, X22 represents Ary, X23 represents Lys, X26 represents Leu, X27 represents Phe,X31 represents Lys, X34 represents His, X38 representsPhe, X43a represents a single bond, X50 represents Ala,X51 represents Glu, X52 represents Thr, X54 representsPro, X63 represents lie, X68 represents Ser, X70 represents Lys, X77 represents Asp, X78 represents Glu,X79 represents Thr, X82 represents Asp, X85 representsTyr, X94 represents Asp, X105 represents Val, Asp orThr, X106 represents Thr, Leu or Asn, X120 represents Arg, X124 represents Gln, K131 representsLys, K151 represents Phe, X153 represents Leu, X156 represents Asn, X157 represents Leu, X158 representsGln, K159 represents Glu, X160 represents Ser, K161 represents Leu, X162 represents Arg, X163 representsSer, X164 represents Lys and X165 represents Glu. 7. A polypeptide as claimed in claim 1 of the formula EMI70.1 wherein X2 represents Asp or Trp, X3 represents Leu orCys, X4 represents Pro or Gln, X5 represents Gln or Asp,X6 represents Thr or Pro, X7 represents His or Tyr,XlOc represents Gly or Ala, X37 represents Gly or Ala,X44 represents Gly or Ala, X59 represents Met or Leu, X98c represents Cys or Leu, X99c represents Val or Thr,X100c represents lie, Met or Asn, X101c represents Gln or Tyr, xl02C represents Gly, Ala or Ser, X103c represents Val, x104c represents Gly, Ala or Thr, X105c represents Val or Asp, x106C represents Thr or Leu, xlO7c represents Glu or Asn, X108 represents Thr or Val,X109 represents Pro or Gln, X110 represents Leu orArg, xlllc represents Met or Lys, X112c represents Lys or Ala, X113c represents Glu or Ile, and X114 representsAsp or His and corresponding polypeptides in which Cys at positions 29 and 138 is replaced by Ser, the amino acids being selected such that at least in one of positions 2 7,10,37,44,59 and 99 114 an amino acid is present which differs from the amino acid in the corresponding position of IFN alpha2. 8. A polypeptide as claimed in claim 1 in which X99 X99 X100 X101 X102 X103 X104 X105 X106 X107 X108X109 X110 X111 X112 X113 X114 represents the moiety Leu Thr Asn Tyr Ser Val Thr Asp Leu Asn Val Gln ArgLys Ala Ile His . 9. A polypeptide as claimed in claim 1 in which at least one of X10 X37, X44, X102 and X104 represents alanine. 10. A polypeptide as claimed in claim 1 in whichX59 represents leucine. 11. A polypeptide as claimed in claim 1 selected from the group consistiny of IFN alpha2 4 155 endo Asp43a Leu151,Arg160,163 IFN alpha2 gamma 98 114 98 114 IFN alpha2 Met100, gamma 10l 114 101114 IFN alpha2 gamma 2 7 , gamma 98 114 IFN alpha2 Leu59 IFN alpha2 A1a10,37144,102,104 IFN alpha2 Ser29,138 IFN alpha2 12. A process for producing a polypeptide as defined in claim 1 which comprises culturing a microorganism, the microorganism having been transformed with a replicable plasmidic expression vehicle comprising genetic material coding for the said polypeptide whereby to effect expression of the said polypeptide and recovering the said polypeptide thereby expressed. 13. A transformant microorganism capable of expressing a polypeptide as defined in claim 1, the said microorganism comprising a replicable plasmidic expression vehicle, which vehicle comprises genetic material coding for the said polypeptide. 14. A process for the preparation of a transformant microorganism as defined in claim 13 which comprises transforming a microorganism by the insertion therein of a replicable plasmidic expression vehicle, which vehicle comprises genetic material coding for a polypeptide as defined in claim 1. 15. A replicable plasmidic expression vehicle capable, in a transformant microorganism, of expressing a polypeptide as defined in claim I. 16. A process for the preparation of a replicable plasmidic expression vehicle as defined in claim 15 which comprises inserting a gene coding for a polypeptide as defined in claim 1 into a vector therefor at an appropriate insertion site. 17. A process for the preparation of a replicable plasmidic expression vehicle as defined in claim 15 which comprises inserting a nucleotide sequence coding for a portion of the polypeptide as defined in claim 1 into a vector comprising a promoter sequence and a nucleotide sequence s coding for the remainder of the polypeptide as defined in claim 1 at an appropriate insertion site whereby a replicable plasmidic expression vehicle is obtained capable of directing the synthesis of the said polypeptide in a transformant microorganism. 18. A DNA sequence that encodes for a polypeptide as defined in claim 1. 19. A portable expression unit comprising theEcoRI SalI segment of the plasmid pSTPl in which a DNA sequence as defined in claim 18 is present between theClaI and SalI restriction sites of the said segment. 20. A pharmaceutical composition comprising as active ingredient at least one polypeptide as defined in claim 1 in association with a pharmaceutically acceptable carrier or excipient.

## Description
POLYPEPTIDES The present invention relates to polypeptides, particularly interferon analogues, such as analogues of human interferon alpha2, processes for their preparation and pharmaceutical compositions containing them as well as to the genetic modification of microorganisms to express the said analogues, the microorganisms thereby obtained, the genetic material employed in the modification and vectors therefor. The human interferon alpha2 hereinafter also referred to asIFN alpha2 analogues of the present invention possess interesting physiological activity. Natural interferons are known compounds which are produced by a variety of living cells both in vivo and in vitro in tissue cultures in response to certain inducers, such as viral infection, immune stimulation and a range of chemical inducers. They are defined as proteins able to inhibit the replication of a variety ofRNA and DNA viruses through cellular metabolic processes involving the synthesis of new RNA and protein Committee on Interferon Nomenclature, 1980 . Three main classes of natural interferons have been distinguished to date and are currently named alpha IFN K , beta IFN P and gamma IFN Y on the basis of antigenic types. Interferons, for example,IFN alpha possess antviral, antiproliferative and immunomodulatory activity. Information on the biological properties of interferons has come predominantly from studies of material produced by cell culture techniques. Recent results using recombinant DNA techniques have shown that human leucocyte interferon Hu IFN alpha is a family of at least 8 different molecules Goeddel et al, Nature, 1981, 290, pages 20 26 . Furthermore, Allen and Fantes Nature, 1980r 287 pages 408 411 have demonstrated that microheterogeneities in interferon preparations from Namalwa cells can be ascribed to the presence of at least five homologous proteins with different amino acid sequences and hence to different structural genes.Despite these advances, very little is known about the distinct physiological properties of individual subspecies of Hu IFN alpha. As stated above human leucocyte interferon is a family of a number of different molecules such as IFN alphal and IFN alpha2, the present invention being concerned inter alia with analogues of IFN alpha2. The preparation of certain truncated analogues of IFN alpha2 has been described in the literature. Thus it is stated in Weck, P. K. et al. J. Cell. Biochem.Suppl. 6 104, 1982 that studies with hybrid leukocyte interferons indicate that both the N and the Cterminal portions of the leukocyte interferons are involved in receptor binding. These studies involved certain hybrids and indicated that the C terminal 15 amino acids clearly have significant effects on the activity of these interferons. Nevertheless a number of analogues of interferon alpha2 have been prepared having a truncated C terminus and for example, IFN alpha2 1 160 , IFN alpha2 1 155 and IFN alpha2 1 152 have all been reported to retain their antiviral activity, but show no adequate separation of antiviral from antiproliferative or NK cell stimulating activities relative to IFN alpha2. IFN alpha2 4 160 and IFN alpha2 4 165 have also been disclosed, but these truncated analogues also showed no significant separation of antiviral from antiproliferative or NK cell stimulating activities relative to IFN alpha2. We have found, however, thatIFN alpha2 4 155 showed a consistent ten fold increase over IFN alpha2 in the number of antiviral units required to produce a 50 inhibition of Daudi cell proliferation. The literature also contains a number of references to the production of terminal hybrids ofIFN alpha2 with IFN alphal or with IFN alpha I.It will be understood that references herein to a terminal hybrid mean a hybrid of two interferon molecules in which the hybrid consists of two sections, the N terminal section of the hybrid comprising the amino acids in the sequence of one interferon and theC terminal section comprising the amino acids in the sequence of the other interferon. Thus IFN alpha 1 92 alpha2 92 165 , IFN alphal 1 62 alpha2 62 165 , IFN alpha2 l 91 alphal 93 166 andIFN alpha2 1 61 alphal 63 166 have all been described as possessing antiviral activity inWeck, P. K. et al 1981 Antiviral activities of hybrids of two major human leukocyte interferons ,Nucleic Acids Research 9, 6153 6166.IFN alpha2 1 149 alpha I 151 166 , IFN alpha I 1 150 alpha2 150 165 , IFN alpha2 1 149 alphal 151 166 and IFN alphal 1 150 alpha2 150 165 have all been described as possessing antiviral activity in Franke, A.E. et al 1982 . Carboxyterminal region of hybrid leukocyte interferons affects antiviral specificity .DNA, 1, 223 230. One internal X 1 22 2 hybrid in which an IFN alphal segment is flanked by segments of interferon alpha2 has been disclosed as possessing antiviral activity and this analogue is Thr68,Asp79,Cys85 IFN alpha2. This analogue is disclosed in Rehberg, E. et al 1982 Specific molecular activities of recombinant and hybrid leukocyte interferons Journal of Biological Chemistry, 257, 11497 11502. The present invention is based on the discovery that novel polypeptides possessing at least certain of the properties of natural interferons, but in particular good antiviral activity, may be prepared by effecting one or more amino acid or sequence changes as hereinafter defined to the structure of IFN alpha2 or its known analogues. Whilst the amino acid sequence forIFN alpha2 is known, it is not known, and it cannot be accurately predicted, what changes to the IFN alpha2 polypeptide may be made without losing antiviral activity. It is known that apparently small changes to polypeptide molecules made in one region of the polypeptide may destroy all biological activity whereas an improvement in biological activity may be obtained by an equally small change in a different region of the polypeptide. The polypeptides of the present invention may also be used as biochemical tools, for example, after immobilization on an appropriate support they may be used for the enrichment and purification of interferon receptors or they may be useful in discriminating between families of monoclonal antibodies raised against native interferons It is to be understood that in this specification the abbreviations used for amino acids are standard abbreviations used in the peptide art see Pure and Applied Chemistry, 1974, 40, 317 331, andNeuropeptides, 1981, 1, 231 235 . Thus for example the following amino acids are abbreviated as indicated Alanine Ala Leucine LeuArginine Arg Lysine LysAsparagine Asn Methionine MetAspartic acid Asp Phenylalanine PheGlutamic acid Glu Proline ProGlutamine Gln Serine SerGlycine Gly Threonine ThrHistidine His Tryptophan TrpIsoleucine Ile Tyrosine TyrCysteine Cys Valine Val The configuration of a particular amino acid, other than glycine, will be understood to be the naturalL configuration. According to one feature of the present invention we thus provide a polypeptide of the formulaI EMI6.1 in which X1, X2, X3, X4, X5, X6, X7 X8, X9, X10, X11, X155, X156, X156, X157, X158, X159, X160, X161, X162, X163,X164 and X165 may each represent a naturally occurring alpha amino acid and X12 represents Arg or Ala, X13 represents Arg or Ala, X14 represents Thr or Ala, X15 represents Leu or Ala, X16represents Met, Leu or Ile,X21 represents Met or Leu, X22 represents Ser, Arg or Gly and X23 represents Arg or Lys, X26 represents Leu orPro, X27 represents Ser or Phe, X28 represents Ser orAla, X29 represents Ser or Cys, X30 represents Leu orIle X31 represents Met or Lys, X32 represents Ala, Asn,Asp or Glu, X34 represents His or Pro, X37 representsGly, or Ala, X38 represents Phe or Leu, X43a represents a single bond or Asp, X44 represents Gly or Ala, X50 represents Ala or Thr, X51 represents Glu Pro or Glu X52 represents Ala or Thr, and X54 represents Ser orPro, X59 represents Leu or Met, X61 represents Glu orAla, X63 represents Ile or Thr, X64 represents Phe orAla, X67 represents Phe or Ala, X68 represents Thr orSer, X70 represents Lys or Glu, X77 represents Asp orGlu, X78 represents Glu or Gln, X79 represents Asp, Thr or Ser, X82 represents Asp or Glu X85 represents Cys,Tyr or Ser, X94 represents Asp or Asn, X98 representsCys or Leu, X99 represents Val or Thr, X100 representsMet, Ile or Asn, either X101 represents Gln, Tyr orPhe, X102 represents Glu, Gly, Ser or Ala, X103 represents Glu, Val or Lys, X104 represents Arg, Gly,Thr, Leu or Ala, X105 represents Val, Asp, Met or Thr,X106 represents Gly, Thr, Glu, Leu or Asn, X107 represents Glu, Asn or Tyr, X108 represents Thr or Val,X109 represents Pro or Gln, X110 represents Leu or Arg and X111 represents Met, Lys or Leu, or X101 X102 X103 X04 X105 X106 X107 Xl08 xl09 xll0 xlll represents Met, Lys or Leu, X1l2 represents Asn, Lys or Ala, X113 represents Ala,Glu or Ile, X114 represents Asp or His, X115 representsSer or Phe, X120 represents Lys or Arg, X124 representsArg or Gln, X131 represents Thr or Lys, X137 representsPro or Ala, X138 represents Cys or Ser, X148 representsMet or Leu, X151 represents Leu or Phe, X153 representsLeu or Phe, the amino acids being selected. such that an amino acid is present in at least one position selected from alpha2 positions 12 16, 21, 28,29, 30, 32, 37, 44, 61, 64, 67, 98 115, 137, 138 and 148 which differs from the amino acid in the corresponding position of IFNalpha2, IFN alphal and IFN alphaI and or selected such that an amino acid is absent from at least one position selected from alpha2 positions 101 110 or the amino acids being selected such that an amino acid is present in at least one position selected from alpha2 positions 27, 31, 59, 151, 160 and 163 which is the same as the amino acid in the corresponding position of IFN alphal, the amino acids at alpha2 positions other than positions 1 11, 27, 31, 43a, 59, 100, 102 104, 106, 112, 113, 151 and 156 165 being the same as the amino acids in the corresponding positions of IFN alpha2 or a N111 and or CI ll truncated analogue thereof and the N3 11 truncated analogues of IFN alpha2, which may if desired be C1 11 truncated, with the proviso that a N3 truncated analogue of IFN alpha2 is also C711 truncated. It will be appreciated that the amino acids at positions I to 11 and 155 to 165 are selected so as not to destroy the overall conformation of the molecule.Thus it is preferred that a variety of different amino acids should precede the N12 amino acid residue and follow the N154 amino acid residue. Polypeptides in which all amino acid residues preceding N12 and or following N154 are the same, especially where that amino acid is cysteine should be avoided. Similarly the presence of repeated for example more than six hydrophobic or repeated for example more than six charged amino acid residues in sequence in either the N1 ll or C1 ll portion of the polypeptide is preferably avoided. In the polypeptide of formula I preferably xl represents Cys, x2 represents Asp or Trp, X3 represents Leu or Cys, X4 represents Pro or Gln, X5 represents Glu, Gln or Asp, X6 represents Thr or Pro, X7 represents His or Tyr, X8 represents Ser, X9 representsLeu, X10 represents Asp, Gly or Ala, X11 represents Asn,Ser or Ala, X155 represents Thr, X156 represents.Asn, X157 represents Leu, X158 represents Gln, X159 represents Glu or Lys, X160 represents Arg, Ser or Ile,X161 represents Leu, X162 represents Arg, X163 represents Arg or Ser, X164 represents Lys and X165 represents Glu or Asp.It will be understood that references herein to an N1 11 truncated analogue mean an analogue of the polypeptide of formula I which is shortened by the absence from theN terminal end of the polypeptide of any of the first 111 amino acids without a sequence change. Preferably such references indicate an analogue of the polypeptide of formula I in which the first, first two or first three, amino acids are absent from the N terminal end of the polypeptide of formula I. Similarly references herein to a C1 11 truncated analogue mean an analogue of the polypeptide in which the C terminal end of the polypeptide is shortened by the absence of any of the last 1 11 C terminal amino acids without a sequence change being effected.Preferably such references indicate an analogue in which the last, last two, last three, last four, last five, last six, last seven, last eight, last nine or last ten amino acids are absent from the C terminal end of the polypeptide. It will be appreciated that the polypeptide of formula I may carry a methionine at itsN terminus. Moreover the polypeptide of formula I may carry additional amino acid residues at its N terminus which in turn may be preceded by methionine. Preferably the polypeptide of formula I will carry no more than 17 additional amino acid residues at its N terminus. It will also be appreciated that the polypeptides of the present invention may if desired be radiolabelled. The radiolabel may for example be present at the sulphur of a methionine or it may be present as an iodine substituent on tyrosine. The polypeptides of the present invention thus encdmpass certain N311, truncated analogues of IFNalpha2 per se such as IFN alpha2 10 165 and IFNalpha2 12 165 as well as such analogues which are additionally C1 ll preferably C1 10 C truncated such asIFN alpha2 4 155 . Indeed in our hands we have found that IFN alpha2 4 155 has a particularly good relative antiviral potency in terms of the ratio of antiviral activity to antiproliferative activity and thus IFNalpha2 4 155 is a preferred polypeptide of the invention. In one particular embodiment, the present invention relates to polypeptides of formula I other than truncated analogues The present invention also encompasses for example polypeptides of the formula II EMI10.1 EMI11.1 Gln, X10 represents Asp or Gly, X11 represents Asn orSer, x27a represents Phe or Ser, x3la represents Lys or Met, X43a represents Asp or a single bond, X59a represents Met or Leu, X100a represents Met or Ile, X102a represents Glu or Gly, X103a represents Glu orVal, X104a represents Arg or Gly, Xl06a represents Gly or Thr, X112a represents Asn or Lys, X113a representsAla or Glu, X15la represents Leu or Phe, Xl60a represents Arg or Ser and X163a represents Arg or Ser with the proviso that the amino acids are selected such that at least one of X27a, X31a, X59a, xlsla, X160a andX163a represents the same amino acid as that in the corresponding position of IFN alphal or a N1 11 or C1 11 truncated analogue thereof. Preferred polypeptides of formula II by virtue of their improved relative antiviral potency in terms of the ratio of antiviral activity to antiproliferative activity are compounds of formula II in which X43a represents Asp, X100a represents Ile, Xl02a representsGly, X103a represents Val, X104a represents Gly, Xl06a represents Thr, xll2a representsLys and X113a represents Glu with the proviso that at least one of X5a, xlOa, xlla, Xl5lar X160a and X163a represents an amino acid which differs from the amino acid in the corresponding position of IFN alpha2. An especially preferred compound of formula II by virtue of its good ratio of antiviral to antiproliferative activity is endo Asp43a Leu151, Arg160, 163 IFN alpha2. The polypeptides of the present invention relate in particular to analogues of IFN alpha2 of formula I in which an amino acid is present in at least one position selected from alpha2 positions 12 16,21,28, 29,30,32,37,44,61,64,67,98 115,137, 138 and 148 which differs from the amino acid in the corresponding position of IFN alpha2, IFN alphal and IFN alphaI. In a further embodiment of the present invention there are provided compounds of formula I in which X1 represents Cys, x2 represents Asp or Trp, X3 represents Leu or Cys, X4 represents Pro or Gln, X5 represents Gln or Asp, X6 represents Thr or Pro, X7 represents His or Tyr, X8 represents Ser, X9 representsLeu, X10 represents Gly or Ala, X11 represents Ser orAla, X16 represents Leu or Met, X22 represents Arg, X23 represents Lys, X26 represents Leu, X27 represents Phe,X31 represents Lys, X34 represents His, X38 representsPhe, X43a represents a single bond, X50 represents Ala,X51 represents Glu, X52 represents Thr, X54 representsPro, X63 represents Ile, X68 represents Ser, X70 represents Lys, X77 represents Asp, X78 represents Glu,X79 represents Thr, X82 represents Asp, X85 representsTyr, X94 represents Asp, X105 represents Val, Asp orThr, X106 represents Thr, Gly, Leu or Asn, X120 represents Arg, X124 represents Gln, X131 represents Lys, X151 represents Phe, X153 represents Leu, X156 represents Asn, X157 represents Leu, X158 representsGln, X159 represents Glu, X160 represents Ser, X161 represents Leu, X162 represents Arg X163 representsSer,X164 represents Lys and X165 represents Glu and the N1 ll and C1 ll truncated analogues thereof. Where such truncated analogues are obtained they are advantageously the N1 3 and or C1 10 truncated analogues. Generally it is preferred that the compounds of formula I of the present invention contain a Cys2 9, Cys 138 disulphide bridge as in natural IFN alpha2. We have found that Ser29,Ser138 IFN alpha2, however, not only possesses antiviral activity, but possesses a relative antiviral potency in terms of the ratio between antiviral and antiproliferative activity AV AP which is substantially greater than that reported forHu IFN alpha2. This is a particularly surprising finding in view of the belief in the literature that theCys29, Cys138 disulphide bridge is essential for activity. Compounds of the formula EMI13.1 EMI14.1 wherein X2 represents Asp or Trp, X3 represents Leu or Cys, X4 represents Pro or Gln, X5 represents Gln or Asp, X6 represents Thr or Pro, X7 represents His or Tyr, XlOc represents Gly or Ala,X37represents Gly or Ala, X44 represents Gly or Ala, X59 represents Met or Leu, X98c represents Cys or Leu, x99c represents Val or Thr, X100c represents Ile, Met or Asn, XlOle represents Gln or Tyr, XlO2c represents Gly, Ala orSer, xl03C represents Val, X104e represents Gly, Ala orTh r, X105c represents Val or Asp, xl06C represents Thr orLeu, xl07C represents Glu or Asn, X108 represents Thr orVal, X109 represents Pro or Gln, X110 represents Leu orArg, X111c represents Met or Lys, X112c represents Lys orAla, xll3c represents Glu or Ile and X114 represents Asp or His and corresponding polypeptides in which Cys at positions 29 and 138 is replaced by Ser, the amino acids being selected such that at least in one of positions 2 7,10,37,44,59 and 98 114 an amino acid is present which ditters from the amino acid in the corresponding position of IEN alpha2 and the N1 11 preferably N1 3 and C1 ll preferably C1 10 truncated analogues thereof are preferred because of their good relative antiviral potency in terms of the ratio of antiviral to antiproliferative activity. The following features either alone or in combination are preferred 1 the presence of gamma 98 114, especially gamma 101 114, in the Hu IFN alpha2 polypeptide at positions 98 114, especially 101 114 respectively 2 the presence of alanine in at least one of positions 10,37,44,102 and 104 in the Hu IFN alpha2 polypeptide 3 the presence of serine at both ppsitions 29 and position 138 of the Hu IFN alpha2polypeptide 4 the presence of leucine at position 59 of theHu IFN alpha2 polypeptide. When any one of these features is taken either singly or in combination, with the other general or particular features of the polypeptides of the present invention, preferred sub groups of polypeptides are obtained. Especially preferred polypeptides by virtue of their good relative antiviral potency include gamma 98 114 98 IFN alpha2, Met100, gamma 101 114 10l 114 IFN alpha2, gamma 2 7 , gamma 98 114 IFN alpha2, Leu59 IFN alpha2, Ala 10,37r44,102,104 IFN alpha2 and Ser29,138 IFN alpha2. In the drawings accompanying this specification Figure 1 illustrates the nucleotide sequence of a Hu IFN alpha2 gene and the corresponding amino acid sequence Figure 2 is a diagramatic representation of the structure of the plasmid pSTPl. The polypeptides of the present invention are conveniently prepared by genetic engineering techniques for example such as described in European PatentPublication No. 0062971A2, the disclosure of which is incorporated herein by way of reference. We thus provide according to a further feature of the present invention a process for producing a polypeptide of formula I as hereinbefore defined which process comprises culturing a microorganism, the microorganism having been transformed with a replicable plasmidic expression vehicle comprising genetic material coding for the said polypeptide of formula I, whereby to effect expression of the said polypeptide and recovering the said polypeptide thereby expressed. The above mentioned process may be effected by the use of any appropriate microorganism such as a gram negative E. coli or an autotrophic bacterium e.g. one capable of using as its sole or principal carbon source inorganic carbon e.g. carbon dioxide, or a bicarbonate , or a methylotrophic bacterium e.g. one capable of using as its sole or principal carbon source methane or methyl carbon, for example from methanol . However we do not exclude the use of any of the microorganisms referred to in European Patent Publication No.0062971A2. Both anaerobic and aerobic varieties of microorganisms are found to be interest in the process of the present invention, as well as both facultative and obligate organisms, and the selection of an organism will be made in the light of the preferred processing conditions as well as the biochemical or biosynthetic capabilities of an organism having otherwise acceptable characteristics. Preferred microorganisms for use in the above mentioned process of the present invention includeE. coli K 12 JA 221 and especially E. coli K 12 DS410 . These microorganisms, JA 221 and DS410, are well known having the genotype F thiA leuB6 trpE 5 recA and F ara azi ton A lac Y min A min B rps L mal A xyl mtl thi respectively and are freely available to the public, having been freely available to the public since before I February 1985. Moreover E. coli JA 221 andE. coli DS 410 have been deposited by us, under theBudapest Treaty, with The National Collections ofIndustrial Marine Bacteria Ltd, Torry ResearchStation, PO Box No. 31, 135 Abbey Road, Aberdeen,AB9 8DG Scotland under the deposition numbers NClB 12099 and NC1B 12100 respectively. The deposition date in respect of each microorganism being 7 June 1985. Typically, such micro organisms may be aerobically cultured in an aqueous medium containing as a source of assimilable carbon, together with necessary inorganic nutrients. Generally, where, for example, the interferon analogue is not passed out of the cell at a commercially useful rate, the organism may be cultured and harvested as the intact cell conveniently with subsequent extraction of the cells for example after separation from the medium. Where the interferon analogue is passed out of the cell into the surrounding culture solution, however, the analogue may be extracted therefrom in the conventional way. Such techniques are well known and require no further explanation. According to a further feature of the present invention there is thus provided a transformant microorganism capable of expressing a polypeptide of formula I as hereinbefore defined, the said microorganism comprising a replicable plasmidic expression vehicle, which vehicle comprises genetic material coding for the said polypeptide. The transformant microorganism is preferably an E. coli JA221 transformant, especially an E. coliDS410 transformant. According to a further feature of the present invention there is provided a process for the preparation of a tranformant microorganism as hereinbefore defined which comprises tranforming a microorganism by the insertion therein of a replicable plasmidic expression vehicle, which vehicle comprises genetic material coding for a polypeptide of formula I as hereinbefore defined. Methods for the introduction of foreign genetic material into microorganisms have been widely described in the literature. Such methods comprise formation of a replicable plasmidic expression vehicle which vehicle comprises a vector and the foreign genetic material, and introduction of the vehicle into the microorganism. Introduction of the vehicle into the microorganism may be advantageously facilitated by subjecting the microorganism to an appropriate treatment, for example treatment with a calcium chloride solution.Where the genetically modified microorganism of the present invention comprises a methylotroph or an autotroph the replicable plasmidic expression vehicle comprising the gene of an interferon analogue of the present invention is preferably introduced thereto by transfer from another organism by cell conjugation, more preferably by transfer of the replicable plasmidic expression vehicle from a strain of E. coli. According to a further feature of the present invention we provide a replicable plasmidic expression vehicle capable, in a transformant microorganism, of expressing a polypeptide of formula I as hereinbefore defined. We also provide according to a still further feature of the present invention a process for the preparation of such a replicable plasmidic expression vehicle which comprises inserting a gene coding for a polypeptide of formula I as hereinbefore defined or inserting a double stranded DNA fragment encoding a portion of the said polypeptide into a vector therefor at an appropriate insertion site whereby a replicable plasmidic expression vehicle is obtained capable of directing the synthesis of a polypeptide of formula I as hereinbefore defined in a transformant microorganism. Thus in one embodiment of the present invention the replicable plasmidic expression vehicle may be prepared by inserting a gene coding for a polypeptide of formula I into a vector therefor. In a further embodiment a vector comprising a promoter sequence and a nucleotide sequence or nucleotide sequences which code for a portion of the polypeptide of formula I is used and the nucleotide sequence fragment s coding for the remainder of the polypeptide of formula I is are inserted into the vector to form the replicable plasmidic expression vehicle. The replicable plasmidic expression vehicle of the present invention is advantageously prepared by the use of a plasmid vector comprising an E. coli lac or trp promoter, preferably a plasmid vector as described inEuropean Patent Publication No. 0062971 the disclosure which, as stated above, is for example incorporated herein by way of reference. Expression of the gene encoding for the polypeptide of formula I is for example effected under the influence of a trp promoter the promoter of the E. coli tryptophan operon . A synthetic trp promoter based on the tryptophan operon of E. coli K12 is conveniently employed and is present in the plasmid vector pSTPl Windass et al., Nucleic Acids Research, Vol. 10 1983 p 6639 illustrated in Fig 2. Such a plasmid vector is of particular interest since it allows the gene coding for the polypeptides of formula I to be introduced between the ClaI and SalI sites of the plasmid vector just downstream of a potentially strong promoter. The trp promoter together with such a gene the Eco RI SalI segment may be removed from the vector and used as a portable expression unit, which unit constitutes a further feature of the present invention. According to a further feature of the present invention there is provided a DNA sequence that encodes for a polypeptide of formula I as hereinbefore defined.Oligonucleotides selected from the aforesaid DNA sequence also constitute a further feature of the present invention. The aforesaid DNA sequence and oligonucleotides may be prepared by any convenient process known per se, but one method which we have found advantageous for preparing the aforesaid DNA sequence and which constitutes a still further feature of the present invention comprises ligating appropriate oligonucleotides whereby to form a DNA sequence of the present invention. The degeneracy of the genetic code permits substantial freedom in the choice of codons which can be used to construct a gene for the appropriate polypeptide of the present invention. Codons were normally chosen as those preferred in the expression of microbial genomes in a particular host such as E. coli. This consideration led to the preferred nucleotide sequences of the present invention which were selected to enable the polypeptides of the present invention to be expressed, such as the nucleotide sequence detailed in any one of the Examples hereinafter. We have found that the synthesis of the gene of the polypeptides of the present invention may be facilitated by synthesizing oligonucleotides, for example comprising about 14 or 15 nucleotides, and then ligating oligonucleotides, for example 4 6 oligonucleotides, in the appropriate sequence to form frayments, which fragments may then be assembled to form sequences and the sequences ligated to form the gene. The DNA sequence that encodes for the polypeptides of formula I is for example flanked by appropriate initiation and termination codons and appropriate single stranded termini to allow ligation at a suitable site in a plasmid vector. The DNA sequence is thus preceded by for example 5 CGACGATG 3 TGCTAC which sequence comprises the initiation codon for methionine and the linker nucleotides which enable theDNA sequence to be inserted at a ClaI TaqI restriction site. The DNA sequence may for example be followed by 5 TAAG 3 ATTCAGCT which sequence comprises a termination codon and the linker nucleotides which enable linkage to be effected to the vector for example at the SalI site.In the preparation of the DNA sequences of the present invention these sequences are preferably included in the appropriate 5 and 3 oligonucleotides where it is desired to prepare the DNA sequence by the above described technique of ligating oligonucleotides to form sequences which are then ligated to form fragments and the fragments ligated to form the DNA sequence or gene. We do not however exclude the possibility that alternative synthetic strategies may be used. For example, the degeneracy of the genetic code permits restriction sites to be present in the nucleotide sequence of the gene and this may be used to facilitate gene synthesis. According to a further feature of the present invention we therefore provide a vector comprising a promoter sequence and a nucleotide sequence or nucleotide sequences which code for a portion of the polypeptide of formula 1, the vector being such that the nucleotide fragment coding for the remainder of the polypeptide of formula I may be inserted into the vector whereby to form replicable plasmidic expression vehicle of the present invention. The nucleotide sequence of the vector at the site for insertion of the aforesaid gene fragment will thus be a restriction site s so that a restriction endonuclease may be used to cut the vector.at the appropriate site for insertion therein of the said gene fragment. Thus, for example, the nucleotide sequence set out in Figure 1 has a Bst E Il Eca I restriction site GGTAACC at the codons for amino acids 44 to 46.Thus, where it is desired to effect changes to theIFN alpha2 polypeptide within the first 43 amino acids, only the section of the gene spanning the restriction site at the 5 end of the gene and the Bst Ell Eca I restriction site need be prepared. This section of the gene or fragment is referred to herein as a Cla BstE fragment. The replicable plasmidic expression vehicle may thus be prepared directly by inserting the appropriate Cla Bst E fragment into a vector containing an appropriate promoter sequence, e.g. a trp promoter and the IFN alpha2 gene sequence between the Bst EII EcaI site and the 3 end of the gene at the appropriate site. The Cla Bst E fragments may, for example, be prepared by the production of a DNA sequence, which contains the desired restriction sites, the sequence then being cleaved by digestion with the appropriate restriction endonuclease but, may also be prepared using modified terminal fragments so that the single stranded cohesive end for Bst Ell Eca I is already present. The use of restriction site s in this way thus enables the genes for a series of polypeptides, modified in a particular region, to be produced in a replicable plasmidic expression vehicle without the need for the total synthesis of each gene. Moreover several vectors spanning any combination of restrictions site s may be generated and modified genes prepared by such procedures can be used to generate any combination of these modifications by transferring restriction fragments between plasmidic vectors.By way of example restriction sites introduced into the IFN alpha2 gene to facilitate modified gene synthesis are illustrated in theTable below EMI24.1 Oligonucleotds SEP New SEP sequence SEP Restriction SEP site SEP New SEP small tb Changed SEP 5 3 SEP introduced SEP sub fragment tb 4 SEP 4 SEP SauI SEP CTAAGGAATGAGTTG SEP SauI SEP A SauI tb 5 SEP 5 SEP SauI SEP TCCTTAGGTAGCCGT tb 29 SEP 29 SEP Xhol SEP AGCACTAAAGACTCG SEP XhoI tb 30 SEP 30 SEP XhoI SEP GCAGCACTCGAGTCT SEP F XHoI tb 31 SEP 31 SEP XhoI SEP AGTGCTGCATGGGAC SEP G XhoI tb SEP G Xho SstI tb 35 SEP 35 SEP Sst SEP AAATTCTACACCGAG SEP G SstI tb 36 SEP 36 SEP SstI SEP TGGTAGAGCTCGGTG SEP SstI tb 37 SEP 37 SEP SstI SEP CTCTACCAGCAACTG SEP SstI SEP H SstI tb SEP H Sst Xbal tb 38 SEP 38 SEP XbaI SEP AGATCGTTCAGTTGC tb 39 SEP 39 SEP XbaI SEP AACGATCTAGAAGCC SEP Xbal SEP H XbaI tb 40 SEP 40 SEP XbaI SEP ATGACGCAGGCTTCT tb 60 SEP 60BglII SEP ATCTCATGATTTCAG SEP LBglII tb 61 SEP 61BglII SEP ATGAGATCTTTCAGC SEP BglII SEP M BglII tb 62 SEP 62BglII SEP TGGACAGGCTGAAAG tb The polypeptides of formula I as hereinbefore defined may be used as obtained or purified in a known or appropriate manner and formulated into pharmaceutical compositions for example by admixture with an appropriate inert or active diluent, which typically will be liquid, but may be solid. The polypeptides of the present invention are potentially useful as antiviral, or anticancer agents or as modulators in the immune response of, for example, mammals and particularly man. Various aspects of the present invention will now be described with reference to the followingExamples which are illustrative of the invention.Examples 1 31 Preparation of HuIFN alpha2 analogues.A. Preparation of.the genes of HuIFN alpha2 analogues. IFN genes were prepared from oligodeoxyribonucleotides oligonucleotides according to the basic strategy outlined by Edge et al Nature 1981, 292 756 762 and Nucleic Acid Res. 1983 11, 6419 6435 and in European Patent Publication No. 62971A2. These Examples describe the preparation of the genes of HuIFN alpha2 analogues which can be ligated to the plasmid vector pSTP. 1, containing a synthetic trp promoter, to give expression of the gene in E. coli. 1.Oligonucleotide synthesis Oligonucleotides were prepared either by a solid phase synthesis procedure as described in EuropeanPatent Publication No. 62971 A2 or by using the reagents and solvents described in the aforementioned patent application with the composite polydimethylacrylamideKieselguhr support in a continuous flow assembly apparatus as described by M. J. Gait et al. J. Chem.Soc., Chem. Commun. 1982. p. 37 40 . Crude oligonucleotides were purified by ion exchange and reverse phase high performance liquid chromatography essentially as described by C. R. Newton et al. Anal.Biochem. 1983 129, 22 30 . For eh eurposes of these Examples the structure of the oligonucleotides referred to herein as 1 Taq, 2 Taq and by a number of from 3 to 68 are as identified by the horizontal arrows appearing above and below the relevant oligonucleotide sequence in Fig 1 which depicts a synthetic interferon alpha2 sequence.Where it is desired to refer to an oligonucleotide comprising an amino acid modification the same number will apply to the modified oligonucleotide as applied to the corresponding unmodified oligonucleotide with the proviso that additional notation is used to distinguish the modified oligonucleotide and its structure is identified hereinafter.It will further be noted that oligonucleotides 1 Taq and 2 Taq referred to herein differ from oligonucleotides 1 and 2 identified in Fig 2 of European Patent Publication No. 62971A2 and the designation 1 Taq and 2 Taq has been used herein to avoid confusion Thus oligonucleotides 1 Taq and 2 Taq possess the following sequences 1 Taq 5 CGACGATGTGTGAT 3 2 Taq 3 TGCTACACACTAGACGGC 5 which sequences may be further modified if it is desired to code for amino acid s which differ from any of the first five amino acids of IFN alpha2 or if it is desired to omit any such amino acids.2. Ligation of chemically synthesised oligonucleotides Oligonucleotides were assembled into small DNA sub fragments A M essentially as detailed in Table 1 below Table 1EMI27.1 tb I SEP Fragments SEP I SEP Oligonucleotides SEP I tb A SEP A SEP I SEP 1 5 tb B SEP 6 10 tb C SEP 11 15 tb D SEP 16 20 tb E SEP 21 25 tb I SEP F SEP I SEP 26 30 tb G SEP 31 35 tb H SEP 36 40 tb I SEP 41 45 tb J SEP 46 50 tb K SEP 51 55 tb L SEP 56 60 tb M SEP 61 68 tb The oligonucleotides were assembled into the small DNA sub fragments A M either as described by Edge et al. Nucleic Acids Res. 1983 11, 6419 6435 and inEuropean Patent Publication No. 62971A2 or by the following modification as illustrated in the preparation of small sub fragment B Oligonucleotides 6,7,8 and 9 400 pmol.each Fig. 1 were separately phosphorylated with polynucleotide kinase and gamma 32 P ATP as described in the aforementioned European Patent Publication then mixed in the ratio 1 0.9 0.8 0.9 and ethanol precipitated. The precipitate was dissolved in water 50 pl and the solution heated at 100 C. for 2 minutes to destroy residual kinase activity. Oligonucleotide 10 400 p mol. was added and the mixture extracted with butanol and then lyophilised. The residue was annealed and ligated to give fragment B as described in the aforementioned European Patent publication. Small sub fragment A was similarly prepared from oligonucleotides 1 Taq, 2 Taq, 3, 4, and 5 with oligonucleotide 1 Taq left unphosphorylated, to give a fragment subsequently referred to as A taq.3. Assembly of A to into the Interferon Gene Sequences I to III M were constructed by similarly ligating fragments A taq to M after phosphorylating the 5 hydroxy termini of fragments B toL, with T4 polynucleotide kinase and ATP containing 0.15 of gamma 32P ATP in groups as shown in Table 2. Table 2EMI28.1 tb I SEP Sequence SEP I SEP Fragments tb l SEP tb I SEP I SEP I SEP A Taq, SEP B, SEP C, SEP D SEP I SEP tb I SEP II SEP I SEP E, SEP F, SEP G, SEP H SEP I tb I SEP III SEP I SEP I, SEP J, SEP K, SEP L, SEP i SEP tb IIIM SEP I, SEP J, SEP K, SEP L, SEP M tb The sequences I, II and III or IIIM were purified by gel electrophoresis in a 10 polyacrylamide slab and either I II III M or I II IIIM ligated to give the gene of the appropriateHu IFN alpha2 analogue. After precipitation the product mixture was phosphorylated with T4 polynucleotide kinase as described previously. The following HuIFN alpha2 analogues were prepared by ligation of the oligonucleotides specified in the Table, according to the procedure described in this Example above and according to the basic strategy described in Edge et al Nature 1981, 292, 756 762 and,Nucleic Acids Res. 1983 Il, 6419 6435 and in European Patent Publication No. 62971A2. As stated above the structure of the oligonucleotides referred to below as 1Taq, 2 Taq and by a number of from 3 to 68 are identified by the horizontal arrows above and below the oligonucleotide sequence in Fig 1. References herein to gamma interferon refer to the amino acid sequence detailed in Structure of the human immune IFN gene P W Gray or DV Goeddel Nature 298 859 1982 . TABLE 3 EMI30.1 Ex SEP Name SEP New SEP oligonucletoide SEP New tb No. SEP Small SEP sub SEP oligonucletoides tb SEP Cod SEP sequence SEP 5 3 SEP Fragment SEP used. tb 27. SEP endo Asp43a Glu,Asp10,Asn11 IFN alpha 2 SEP 1 SEP taq SEP CGACGATGTGTGAT SEP A Bl SEP 1 taq, SEP 2Bl,x4,x5,x6 tb SEP 2 Bl SEP TCCGGCAGATCACACATCTC SEP B Bl SEP x7,x8,8,9,10 tb SEP x4 SEP CTGCCGGAGACCCAT tb SEP x5 SEP CCAGGCTATGGGTC tb SEP x6 SEP AGCCTGGATAACCGT tb SEP x7 SEP AGGGTACGACGGTTAT tb SEP x8 SEP CGTACCCTGATGCTG tb EMI31.1 SEP New SEP Oligonucleotide SEP New tb Ex SEP Name SEP Small SEP sub SEP oligonucleotides tb No. SEP Code SEP sequence 5 3 SEP Fragment SEP used tb 1. SEP 18F8 SEP TTACCGTCGAACTCTTCC SEP D F8 SEP 16, SEP 17, SEP 18F8, SEP 19F8, SEP 20 tb SEP 19F8 SEP TTCGACGGTAACCAGTTC tb 2. SEP endo lAsp43A leu151,Arg160,163 IFN alpha 2 SEP F18 SEP 19 SEP see SEP Example1 SEP DlF8 SEP 16, SEP 17, SEP 18F8, SEP 19F8, SEP 20 tb SEP x62 SEP ATGCGTTCCCTGTCTT SEP M B2 SEP x2, SEP x62, SEP x63, SEP x64, SEP x65, SEP x66, SEP x67 tb SEP x65 SEP GGTTGATAAAGACAGGG tb SEP x64 SEP TATCAACCAATCTTC tb SEP x65 SEP AAGAACGTTTAAGGCGC tb SEP x67 SEP TCGACTTATTCCTTGCGCCTT tb SEP x2 SEP AAGGAAATAAG tb 3. SEP gamma 2 7 2 7 LIFN alpha 2 SEP 1C5 SEP CGACGATGTGTTAC tb SEP 2C5 SEP CTGACAGTAACACATCGT SEP A C5 SEP lC5, SEP 2C4, SEP 3C5, SEP 4C5, SEP 5. tb SEP 3C5 SEP TGTCAGGACCCATAT tb SEP 4C5 SEP CCAGGCTATATGGGTC tb 4. SEP gamma 98 114 IFN alpha 2 SEP 40C4 SEP TTGGTCAGGGCTTCC SEP H C4 SEP 36, SEP 37, SEP 38, SEP 39, SEP 40C4 tb SEP 41C4 SEP CTGACCAACTACTCT SEP I C4 SEP 41C4, SEP 42C4, SEP 43C4, SEP 44C4, SEP 45C4 tb SEP 42C4 SEP TCAGTAACAGAGTAG SEP J C4 SEP 46C4, SEP 47C4, SEP 48, SEP 49,50 tb SEP 43C4 SEP GTTACTGACCTGAAC tb EMI32.1 SEP New SEP oligonucleotide SEP New tb Ex SEP Small SEP sub SEP oligonucleotides tb No. SEP Name SEP CODE SEP sequence SEP 5 3 SEP Fragments SEP used tb SEP 4. SEP 44C4 SEP CGCTGGACGTTCAGG tb SEP 45C4 SEP GTCCAGOGCAAAGCT tb SEP 46C4 SEP GAATGGATAGCTTTG tb SEP 47C4 SEP ATCCATTCCATCCTG tb SEP 5. SEP Met100, gamma 101 114 ifn alpha 2 SEP 41C6 SEP TGOGTCATGTACTCT SEP I C6 SEP 41C6,42C6,43C4,44C4,45C4. tb SEP 42C6 SEP TCAGTAACAGAGTAG SEP J C4 SEP Example SEP 4 SEP above. tb SEP 6. SEP gamma 98 107 98 107 IFN alpha 2 SEP 44C7 SEP ACOGGAGTGTTCAGG SEP H C4 SEP See SEP Example SEP 4 SEP above. tb SEP I C7 SEP 41C4,42C4,43C4,44C7,45. tb SEP 7. SEP gamma 95 101 101 107 IFN alpha 2 SEP 44C8 SEP TGOGTCATCTTOGAG tb SEP 42C8 SEP GTCAGTTTCTOGAAG tb SEP 43C8 SEP AAACTGACCAACTAC SEP I C8 SEP 41C8,42C8,43C8,44C8,45. tb SEP 44C8 SEP AGCGGAGTGTAGTTC tb EMI33.1 SEP New SEP oligonucleotide SEP New tb Ex SEP Small SEP sub SEP oligonucleotides tb No. SEP Name SEP CODE SEP sequence SEP 5 3 SEP Fragments SEP used tb SEP 8. SEP gamma 2 7 ,Met100,Glu102,103,Arg104, SEP 1 4C5 SEP see SEP Example SEP 3 SEP for SEP A C5 SEP 1C5,2C5,3C5,4C5,5. tb SEP Gly106,Asn112,Ala113 IFN alpha 2 SEP nucleotide SEP sequence SEP I C3 SEP 41Cl,42Cl,43Cl,44Cl,45C2. tb SEP 41 SEP Cl SEP TGOGTCATGCAAGAG tb SEP 42 SEP Cl SEP ACACGTTCCTCTTGT tb SEP 43 SEP Cl SEP GAACGTGTAGGTGAG tb SEP 45 SEP C2 SEP ACTCCGCTGATGAACG tb SEP 46 SEP C2 SEP GAGTCTGCGTTCATC SEP J C2 SEP 46C2,47C2,48,49,50. tb SEP 47 SEP C2 SEP CAGACTCCATCCTG tb SEP 9. SEP gamma 2 7 gamma 98 114 IFN alpha 2 SEP 40 47C4 SEP see SEP Example SEP 4 SEP and SEP 3 SEP A C5 SEP see SEP Example SEP 3 SEP or SEP 8. tb SEP 1 4CS SEP respectvely SEP for SEP H C4 SEP tb SEP nucleotide SEP sequences SEP I C4 SEP see SEP Example SEP 4 tb SEP J C4 SEP tb SEP 1C5 tb SEP 2C5 tb SEP 3C5 tb 10. SEP Net700, gamma 2 7 , gamma 100 114 IFN alpha 2 SEP 4C5 SEP see SEP Example SEP 3, SEP A C5 SEP see SEP Example SEP 3 SEP or SEP 8 tb SEP 41C6 SEP 4 SEP 5 SEP for SEP I 6 SEP see SEP Example SEP 5. tb SEP 43 47C4 tb 11. SEP Leu16,21,59,111,148 IFN alpha 2 SEP 7I2 SEP CGCACCCTGCTCGTC SEP B I2 3 SEP 6,7I2,8I2,9I3,10I3 tb SEP 8I2 SEP TGGGCCAGCAGGAGC tb EMI34.1 SEP New SEP oligonucleotide SEP New tb Ex SEP Small SEP sub SEP oligonucleotides tb No. SEP Name SEP CODE SEP sequence SEP 5 3 SEP Fragment SEP used tb 11. SEP 9I3 SEP CTGGCCCAACTGCGC tb SEP 10I3 SEP GAGATACGGCGCAGT tb SEP 24I4 SEP ATCAGTICGTGCAGT SEP E I4 SEP 21,22,23,2414,25I4. tb SEP 25I4 SEP GAACTGATtCAACAG tb SEP 45I5 SEP ACTCCGCTGCTGAAAG SEP I I5 SEP 41,42,43,44,45I5. tb SEP 46I5 SEP GAGTCTTCTTTCAGC SEP J 15 SEP 46I5,47,48,49,50. tb SEP 60I6 SEP AACGCAGGATTTCAG SEP L 16 SEP 56,57,58,59,60I6. tb SEP 61I6 SEP CTGCGTTCCTTCAGC SEP M I6 SEP 61I6,62,63,64,65,66,67,68. tb 12. SEP Leu16 IFN alpha 2 SEP 7I2 SEP see SEP exemple SEP 11 SEP B I2 SEP 6,7I2,8I2,9,10. tb SEP for SEP nucleotide tb SEP sequences. tb 13. SEP Leu21 IFN alpha 2 SEP 9I3 SEP see SEP exemple SEP 11 SEP for SEP B 13 SEP 6,7,8,9I3,10I3. tb SEP 10I3 SEP nucleotide SEP sequeces tb SEP 24I4 SEP see SEP exemple SEP 11 SEP for SEP E I4 SEP see SEP exemple SEP 11 tb 14. SEP Leu59 IFN alpha 2 SEP 25I4 SEP nucleotide SEP sequeces tb EMI35.1 SEP New SEP oligonucleotide SEP New tb Ex SEP Small SEP sub SEP oligonucleotides tb No. SEP Name SEP CODE SEP sequence SEP 5 3 SEP Fragment SEP used tb 15. SEP Leu111 IFN alpha 2 SEP 45I5 SEP see SEP exemnple SEP 11 SEP I 15 SEP see SEP exemnple SEP 11. tb SEP 46I5 SEP nucleotide SEP sequences SEP J 15 tb 16. SEP Ala10,37,44,102,144 IFN alpha 2 SEP 5D2 SEP AGCCTGGCTAGCCGT SEP A D2 SEP 1 SEP Tag,2Tag,3,4,5D2 tb SEP 6D2 SEP AGGGTGCGACGGCTAG SEP B D2 SEP 6D2,7,8,9,10. tb SEP 15D3 SEP CGCCATGACTTTGCT SEP C D3 SEP 11,12,13,14,15D3. tb SEP 16D3 SEP TGCGGGAAAGCAAAG SEP D D3 4 SEP 16D3,17,18D4,19D4,20. tb SEP 18D4 SEP TTAGCGAACTCTTCC SEP I D5 6 SEP 41D5,42D5 D6,43D6,44,45. tb SEP 19D4 SEP TTCGCTAACCAGTTC tb SEP 41D5 SEP TGCGTCATCCAGGCA tb SEP 42D5 D6 SEP ACAGCAACAGCCTGG tb SEP 43D6 SEP GTTGCTGTAACCGAA tb 17. SEP Ala137 IFN alpha 2 SEP 55K10 SEP AAGAAATACAGCGTC SEP K K10 SEP 51,52,53,54,55K10. tb SEP 56K10 SEP CAAGCGCAAGCGCTG SEP L K10 SEP 56KI0,57,58,59,60. tb 18. SEP Ser29,138 IFN alpha 2 SEP 12K1 SEP AGAGAGGAGAACAGG SEP C K1 SEP 11,12K1,13K1,14,15. tb EMI36.1 Ex SEP Name SEP New SEP oligonucleotide SEP New tb No. SEP Small SEP sub SEP oligonucleotides tb SEP CODE SEP sequence SEP 5 3 SEP Fragment SEP used tb 20. SEP 3K1 SEP TCCTCTCTGAAAGAC tb SEP 56K1 SEP CAAGCAGACGGGCTG SEP L K1 SEP 56K1,57K1,58,59,60. tb SEP 57K1 SEP TCTGCTTGGGAAGTT tb 19. SEP Glu32 IFN alpha 2 SEP 13K16 SEP TCCTGTCTGAAAGAG SEP C K16 SEP 11,12,13K16,14K16,15. tb SEP 14K16 SEP TCATGGCGCTCTTTC tb EMI37.1 Ex SEP Name SEP New SEP oligonucleotide SEP New tb No. SEP Small SEP sub SEP oligonucleotides tb SEP CODE SEP sequence SEP 5 3 SEP Fragment SEP used tb 20. SEP Ala61,64,67 IFN alpha 2 SEP 26G3 SEP TTAGCAATCTGTGCA tb SEP 2763 SEP ATTGCTAACCTGGCT SEP F G8 SEP 26G3,27G3,28G3,29,31 tb SEP 28G3 SEP TTAGTGCTAGCCAGG tb 21. SEP IFN alpha 2 12 165 SEP 5A8 SEP CGACGATGCGT SEP B A8 SEP 5A8,6A8,7,8,9,10Ú tb SEP 6A8 SEP AGGGTGCGACGCATCGT tb 22. SEP IFN alpha 2 10 165 SEP 4A7 SEP ACCCATCGT tb SEP 5A7 SEP CGACGATGGGTAGCCGT SEP B A7 SEP 4A7,5A7,6A7,8,9,10Ú tb SEP 6A7 SEP AGGGTGCGACGGCT tb EMI38.1 Ex SEP Name SEP New SEP oligonucleotide SEP New tb No. SEP Small SEP sub SEP oligonucleotides tb SEP CODE SEP sequence SEP 5 3 SEP Fragment SEP used tb 23. SEP Met100des 101 110 IFN alpha 2 SEP I SEP TGCGTCATGATGAAAG SEP oligonucleotide SEP I SEP use SEP in tb SEP place SEP of SEP small SEP sub SEP frag SEP fragment SEP I SEP N.B SEP also SEP converts tb SEP idle SEP 100 SEP to SEP Mat SEP 100 tb 24. SEP Ala10 15 IFN alpha 2 SEP 5Al1 SEP AGCCTGGCTGCAGCT SEP A Al1 SEP 1 Tag,2 Tag,3,4,5Al1. tb SEP 6Al1 SEP GCAGCAGCAGCTGCAG SEP B Al1 SEP 6Al1,7Al1,8,9,10. tb SEP 7Al1 SEP GCTGCTGCGATGCTG tb 25. SEP Phe115 IFN alpha 2 SEP 46J2 SEP AAGTCTTCTTTCATC tb SEP 47J2 SEP AAGACTTCATCCTG SEP J J2 SEP 46J2,47J2,48,49,50. tb EMI39.1 Ex SEP Name SEP New SEP oligonucletoide SEP New tb No. SEP Small SEP sub SEP oligonucletoides tb SEP Code SEP sequence SEP 5 3 SEP Fragment SEP used. tb 26. SEP IFN SEP 2 4 155 SEP 2Al SEP CGGCATCGT SEP A Al SEP 2Al,3Al,4,5 tb SEP 3Al SEP CGACGATGCCGCAAAGTCAT tb SEP 63A5 SEP CTGTCCACCTAAG tb SEP 64A5 SEP TCGACTTAGG SEP M A5 SEP 61,62,63A5,64A5 tb EMI40.1 Ex SEP Name SEP New SEP oligonucletoide SEP New tb No. SEP Small SEP sub SEP oligonucletoides tb SEP Code SEP sequence SEP 5 3 SEP Fragment SEP used. tb 27. SEP Ala32 IFN SEP 2 SEP 13K12 SEP TCCTGTCTGAAAGCT SEP C K12 SEP 11,12,13K12,14K12,15 tb SEP 14K12 SEP TCATGGCGAGCTTTC tb 28. SEP Ile30 IFN SEP 2 SEP 12K14 SEP ATACAGGAGAACAGG tb SEP 13K14 SEP TCCTGTATCAAAGAC SEP C K14 SEP 11,12K14,13K14,14K14,15 tb SEP 14K14 SEP TCATGGCGGTCTTTG tb 29. SEP Asn32 IFN SEP 2 SEP 13K17 SEP TCCTGTCTGAAAAAC SEP C K17 SEP 11,12,13K17,14K17,15 tb SEP 14K17 SEP TCATGGCGGTTTTTC tb 31. SEP Glu5,Ser24,Met31,Leu59 IFN SEP 3Cl4 SEP CTGCCGGAAACTCAT SEP SEP 1 SEP Taq,2Taq, SEP 3 Cl4, tb SEP 4Cl4SauI SEP CTAAGGAATGAGTTTC SEP A Cl4 SEP Sau SEP I SEP 4 Cl5 SEP Sau SEP I, SEP 5 SEP Sau SEP I tb SEP 5 SEP Sau SEP I SEP TCCTTAGGTAGCCGT SEP tb SEP 11Cl5 SEP CGTATCTCCCTGTCT tb SEP 12Cl5 SEP AGACAGGAAGACAGG tb SEP 13Cl6 SEP TCCTGTCTGATGGAC SEP C Cl5 16 SEP 11 C15, SEP 12 C15, SEP 14 C16, tb SEP 14Cl6 SEP TCATGGCGGTCCATC tb SEP 24I4 SEP See SEP Example SEP 11 SEP SEP E I4 SEP See SEP Example SEP 14 tb SEP 25I4 SEP See SEP Example SEP 11 SEP tb 1. In respect of this Example is no small sub fragment A for the gene. B. Cloning of the synthetic genes of HuIFN alpha2 analogues. The synthetic genes of the interferon alpha2 analogues listed in Table 3 were cloned into the plasmid vector, pSTPl Windass et al., Nucleic Acids Research, vol. 10, 1983 p6639 , which included a promoter based on that of the tryptophan trp operon of E. coliK12. For vector preparation, 200 ,ug. pSTPl was dissolved in 168 ul. water, 2 p1. of 10 mg. ml. BSA and 20 p1. of 10 x restriction buffer the restriction buffer is 6mM Tris. HC1 pH 7.4, 6mM beta mercaptoethanol 6mM MgC12, 10 ml. of the restriction endonuclease, Cla I New England Biolabs, 5 units ul. was added and the mixture was incubated at 37 C. for 3 hours or more until linearised plasmid was predominant over supercoiled and nicked circular forms. Linearised plasmid was separated from undigested plasmid molecules on a 1.4 preparative agarose gel. The band of linear plasmid was electroeluted into 5 ml. of 1 10 concentrated tris borate electrophoresis buffer from a 1 to 100 dilution of 108 g Tris base, 55 g. boric acid and 20 ml.0.5M EDTA pH8 made up to 1 litre with water . The eluate volume was reduced to about 50 pl. by repeated butanol extractions and diluted to 500 pl. with water.The eluate was extracted twice with 500 ul. of phenol chloroform 1 1 , twice with 1 ml. of water saturated ether and then concentrated to 20 p1. by butanol extractions. To this was added sequentially 160 ul. water, 20 pl. 3M sodium acetate pH5.2 and 500 1l. of ethanol. The linear plasmid was allowed to precipitate overnight at 20 C. and pelleted by centrifugation in anEppendorf microfuge for 15 minutes. The pellet was taken up in 140 p1. water and 20 pl. of 10 x restriction buffer was added followed by 20 ul. of 1 mg. ml. BSA and 15 p1 of 2M sodium chloride. 5 l. of the endonucleaseSalI Boehringer, 10 units ml. was added and the mixture was incubated at 37 C. for at least three hours until the large ClaI SalI fragment of pSTPl was predominant. This fragment was prepared as above from a 1.4 preparative agarose gel.The ClaI SalI vector fragment was then taken up in 1 ml. water and the DNA concentration was measured and adjusted to 47 ,ug. ml. For ligation, 14.9 jil. of vector DNA 0.7 g. was added to the synthetic interferon gene fragment in water and the DNA mixture was dried in vacuo. The DNA was taken up in 26.5 pl. of a mixture containing 525 pl. 2 x ligase buffer 132 mM Tris HC1. pH 7.6 ,13.2 mM MgC12 10 mM DTT and 0.4 mM ATP , 79 pl. ligase diluent 50 mMTris HC1, pH 7.6, l0mM 2 mercaptoethanol 500 ,ug. ml.BSA Z60 ul. 10 mM ATP, 33.5 p1. water and 30 p1. T4 DNA ligase BRL, 2.5 Weiss units l. . Ligation was for 2 hours at 5 C. and then 12 C. overnight. 10 ul. of the DNA mixes were used directly to transform the strain JA221 using a modification of the standard procedure described by Mandel and Higa, Journal of Mol. Biol., 53 1975 p154. The modifications involved the use of 0.1MCaCl2 in 5mM Tris. HC1 pH8 as the transformation buffer, a 45 minute incubation of the cells on ice following addition of DNA, a 30 minute incubation on ice after heat shock treatment and subsequent addition of L broth to 3.8 ml. Incubations prior to plating were overnight at room temperature without shaking. Aliquots of suspensions were plated onto L plates with 50 pg. ml. ampicillin. Transformants were screened for the presence of cloned synthetic genes of interferon analogues as follows 1. Colonies were streaked onto L plates with 10 g. ml. tetracycline in order to identify transformants including the intact vector plasmid, pSTPl which invariably did not include a gene for an interferon analogue.2. Tetracycline sensitive transformants were analysed by colony hybridisation analysis using standard methods described in Molecular Cloning A LaboratoryManual by Maniatis et al Cold Spring Harbor . For screening for recombinants including, for example, the synthetic interferon gene of Example 11, two separate hybridisations were performed using different small subfragments, one with a base sequence included in the natural interferon alpha2 gene group F , and the other with a sequence related to the synthetic interferon gene, for example to the synthetic interferon gene of Example 11 subfragment BI2 3 . 2 p moles of these sub fragments were dried down and dissolved in 50 ul. of a mixture composed of 80 ul. of nick translation buffer 50 mM potassium phosphate pH 7.4, 5 mM MgC12 , 32 ul. 200 pM. dATP, 32 ul. 200 pM dGTP, 32 ul. 200 pM dTTP, 25 p1. alpha32P dCTP l0mCi. ml., 3,000Ci. mmol., Amersham , 579 p1. water and 20 ,ul. DNA polymerase I 12 units pl. New EnglandBiolabs . After 2 hours at room temperature, the labelled sub fragments were isolated by Sephadex G50F chromatography using 10 mM Tris HC1. pH 8, 1 mM EDTA as running buffer. Lysis of colonies on filters and baking was performed as described in Molecular Cloning ALaboratory Manual by Maniatis et al. Cold SpringHarbor . Filters were prehybridised in 20 ul. Denhardts solution a 50 fold dilution of Ficoll, polyvinyl pyrrolidine and BSA, each at 10 g. l. in water at 65 C, for 1 hour.The 32P labelled subfragment probes were diluted into 2 volumes of 80 v v formamide, 10 mM NaOH, lmM EDTA, 0.05 w v xylene cyanol, and 0.05 w v bromophenol blue and heated for 2 minutes at 100 C. before cooling on ice. The probe was added directly to the prehybridisation solution on the filters, left to hybridise overnight at 65 C. and air dried before autoradiography. The recombinants, for example, of the interferon gene of Example 11 were identified by positive hybridisation to both group F and group BI 2 3 subfragments.3. The correct organisation of the interferon analogue gene was checked by restriction analysis using endonuclease HpaII. Thus, for example, analysis on a 6 polyacryamide gel showed that the interferon gene gave a pattern of HpaII fragments similar to that of the interferon alpha2 gene. 3 interferon recombinants giving positive hybridisation signals were grown up on a small scale in order to select a final clone expressing the interferon analogue of interest for subsequent use in production of the analogue. These small scale grows were conducted as follows Recombinants were grown overnight 250 rpm in a New Brunswick G24 shaking incubator at 37 C. in 10 ml. Spizizens medium 1 10 dilution of NH4 2S04 20 g. l., K2HP04 106 g. l., Na3 citrate 10 g. l.KH2P04 60 g. l., MgS04.7H20 2g. l. pH 7.0 combined with 2 100 volume of a solution containing casein hydrolysate 25 g. l., L leucine 1 g. l., thiamine 0.2 g. l., ampicillin 1.65 g. l. and glucose 100 g. l. containing a high tryptophan concentration 100 mg. i. and 50 ug. ml.ampicillin. Cells were pelleted and resuspended in 75 ml. Spizizens medium supplemented with a low tryptophan concentration of 2 mg. l. Incubation was for 5.5 hours at 37 C. at 250 rpm. Cells were pelleted at 4 C. and taken up in 2 ml. lysis buffer 15 sucrose, 50 mM.Tris. HC1. pH8 and 50 mM EDTA and frozen overnight at 20 C. The cells were then thawed and 40 pl. lysosyme 50 mg. per ml. of lysis buffer was added followed by 20 jil. 2.5 SDS. After incubation for 30 minutes on ice, 200 pl. of DNAase solution 66 pg. ml. DNAase in 0.15 M Tris. HC1. pH 7.5, 0.28 M MgC12, 40 mM CaC12 was, added and the lysate was incubated for 30 minutes on ice. The lysate was then sonisated for 90 seconds using the cup tip of a Heat Systems W 375 Sonicator set at full power.The lysate solution was clarified by centrifugation in a Sorvall SS34 rotor at 15,000 rpm for 20 minutes at 4 C. The supernatant was decanted and aliquoted for biochemical and biological evaluation of the interferon analogue. C. 1. Example of ProduSction of Interferon Analogue Proteins in E. coli JA221. A recombinant strain of E. coli JA221 expressing an IFN t2 related protein was recovered from stock, streaked on an L Agar plate supplemented with ampicillin 50 ,ug. ml. and incubated at 37 C. for 17 hours. A loop of cells was tranferred from this plate into each of 4 x 250 ml. Erlenmeyer flasks containing 75 ml. Spizizen Medium 1, and these flasks shaken at 300 rpm on an orbital incubator at 37 C., overnight.Bacterial cells were harvested by centrifugation in 6 x 37.5 ml. aliquots in the SS34 rotor of a sorval RC5B centrifuge at 10,000 rpm for 6 minutes at room temperature, and resuspended in 3 ml. of prewarmedSpizizen Medium 2. These resuspended cells were then added to 6 x 750 ml. aliquots of Spizizen Medium 2 in 2L Erlenmeyer flasks and shaken at 250 rpm on an orbital incubator at 37 C. for 5.5 hours. At the end of this time, bacterial cells were harvested by centrifugation in the H6000A rotor of a Sorval RC3B centrifuge at 5,000 rpm for 20 minutes at 4 C, and resuspended in a total volume of 40 ml. lysis buffer 50mM EDTA, 15 w v sucrose in 50 mM Tris HCl pH 8.0 . The resultant 40 ml cell suspension was stored at 20 C. prior to lysis.Media Spizizen Medium 1 contained per litre of distilled water NH4 2S04 2.0 g KH2P04 10.6 g Na3 citrate 1.0 g KH2P04 6.0 g MgS04. 7H20 0.2 g casein hydrolysate 0.5 g L leucine 20 mg thiamine 4 mg L tryptophan 100 mg ampicillin 33 mg glucose 2 g. Spizizen Medium 2 was identical except that it contained L tryptophan at 2 mg. l. and did not contain ampicillin. L Medium contained per litre of distilled water Bacto tryptone 10 g Bacto Yeast Extract 5 g NaCl 5 g Glucose 1 g and when necessary, Bacto Agar 15 g.2. Production of Interferon Analogue Proteins in E. coli DS410 A plasmid preparation from the recombinant strain of E. coli JA221 referred to in B above is used to transform E. coli DS410 and the transformant, which expresses an IFN alpha2 protein analogue, was streaked on an L Agar plate supplemented with ampicillin 50 ssug. ml. and incubated at 37 C. for 17 h. A loop of cells were transferred from this plate into each of 2 x 250 ml. Erlenmeyer flasks containing 75 ml. of L broth supplemented with ampicillin 50 ug. ml. , and these flasks shaken at 300 rpm on an orbital incubator at 37 C, overnight. Bacterial cells were harvested by centrifugation in 4 x 37.5 ml. aliquots in the SS34 rotor of a Sorval RC5B centrifuge at 10,000 rpm for 6 minutes at room temperature, and resuspended in 3 ml.prewarmed Spizizen Medium 2. These resuspended cells were pooled in 63 ml. prewarmed Spizizen Medium 2, and 3.75 ml of the resultant suspension inoculated into each of 12 x 75 ml aliquots of Spizizen Medium 2 in 250ml.Erlenmeyer flasks which were subsequently shaken at 300 rpm on an orbital incubator at 37 C. for 7h. At the end of this time bacterial cells were harvested and processed as described in C1.D. Lysis of Bacteria and Extraction of Interferon Analogue Proteins. 40 ml. of a frozen cell suspension was thawed to 4 C. and 800 p1. of a solution of lysozyme in lysis buffer 50 mg. ml. added, immediately followed by 400 pl. LDS in distilled water 2.5 w v . After incubation on ice for 30 minutes, 4 ml. DNAse solution ca. 5 pg. DNAse dissolved in 150 mM Tris. HC1. pH 7.5 containing 280 mM MgC12 and 4 mM CaCl was added and incubation on ice continued for a further 30 minutes. The resultant viscous material was subject to ultrasonic disintegration by 3 x 30s pulses from a 150 W MSE ultrasonic disintegrator set to deliver full power, and then centrifuged at 15,000 rpm for 15 minutes at 40C. in the SS34 rotor of Sorval RC5B centrifuge. The supernatant from this centrifugation was then further centrifuged at 40,000 rpm for lh. at 4 C. in the 50.2 Ti rotor of a Beckman L5 50B ultracentrifuge. The supernatant from this centrifugation contained the interferon analogue protein and was further purified as required.E. Purification 1. Purification by precipitation and size exclusion chromatography All operations were at 40 C. unless indicated otherwise. Clarified supernatant 40 ml. containing the interferon analogue was prepared as hereinbefore described. Trichloroacetic acid 50 v v, 2 ml. was added dropwise with stirring to give a final concentration of 2.5 . Alternatively saturated ammonium sulphate solution 80 ml. was added dropwise with stirring over 15 minutes to give a final concentration of 50 . After stirring for 30 min and standing for 30 min, the precipitated protein was collected by centrifugation at 10,000 rpm for 30 min in an MSE High Speed 18 centrifuge. The supernatant was discarded and the pellet extracted into O.lMK2HP04, pH8 20 ml. using a Ystral homogeniser. The milky suspension was adjusted to pH8 by dropwise addition of 2M NaOH with stirring using aRadiometer pH meter 26. The extract was ultracentrifuged at 34,000 rpm for 30 min in a 50.2.Ti rotor in a beckman L5 50B ultracentrifuge and the supernatant 21 ml. applied immediately to a column 5 x 90 cm. of Ultrogel AcA54 equilibrated in 0.1M K2HP04, pH8 at 7 C. The column was eluted with the same buffer at a flow rate of 40 ml. hr. and fractions ofDiaflo cell using an Amicon YMl0 membrane. The retentate was filtered through a Millex GV, 0.22 um.filter unit and the sterile filtrate 19 ml. was divided into multiple aliquots and stored at 20 C a new aliquot was used for each assay approximately 12 ml. were collected. Fractions 95 125, known to contain the interferon analogue and excluding over 80 of total bacterial protein, were pooled 400 ml. and concentrated to about 20 ml. in an Amicon Purification was monitored by antiviral andNK2 IRMA assays and the antiviral and NK2 IRMA titres for each relevant analogue are set out in Table 4 below.A hyphen in the NK2 IRMA assay column indicates that the relevant analogue was inactive in the NK2 IRMA assay. TABLE 4E. Gel filtration purified analogue profiles EMI49.1 Example SEP Protein SEP Profile SEP Sample SEP Titre SEP U Ml. tb No. SEP AV SEP x106 SEP NK 2IRMA SEP x106 tb SEP 3. SEP 2 7 2 7 IFN 2 SEP 0.26 SEP 3.9 tb SEP 4. SEP 98 114 98 114 IFN 2 SEP 0.42 SEP SEP 5. SEP Met100, SEP 101 114 101 114 IFN 2 SEP 0.09 SEP SEP 6. SEP , 9 107 98 107 IFN 2 SEP 0.22 SEP 5.3 tb SEP 8. SEP 2 7 , SEP Met100, SEP Glu102,103, SEP Arg104,Gly106 SEP Asn112Ala113 IFN SEP 2 SEP 0.09 SEP SEP 9. SEP 2 7 , SEP 98 114 IFN SEP 2 SEP 0.21 SEP 10. SEP Met100, SEP 2 7 , SEP 101 114 IFN SEP 2 SEP 0.05 SEP 11. SEP Leu16,21,59,111,148 IFN 2 SEP 0.02 SEP 0.36 tb 12. SEP Leu16 IFN 2 SEP 0.54 SEP 3.9 tb 13. SEP Leu21 IFN 2 SEP 0.28 SEP 9.5 tb 14. SEP Leu59 IFN 2 SEP 0.02 SEP 0.35 tb 15. SEP Leu111 IFN 2 SEP 0.59 SEP 1.4 tb 22. SEP IFN SEP 2 4 155 SEP 0.39 SEP 0.5 tb The interferon analogue protein still represents only about 0.1 of the total protein in the bioprofiling sample and as such cannot be distinguished directly by staining after polyacrylamide gel electrophoresis. The analogue protein was characterised by transforming the mini cell forming strain E. coli DS4I0 with the expression plasmid as described in D.Plasmid genes are expressed selectively in minicells allowing the proteins encoded by those genes to be radiolabelled selectively. Thus minicells were labelled with 35S methionine and the interferon analogue was characterised by polyacrylamide gel electrophoresis in the presence of SDS run under reducing conditions and revealed by fluorography.2. Purification by antibody affinity chromatography a on NK 2 Sepharose.Clarified supernatant 33 ml. containing the interferon analogue was prepared as hereinbefore described and was applied at a flow rate of about 10 ml. hr. to a column 200 pl. bed volume ofNK2 sepharose monoclonal antibody immuno adsorbent support Celltech Ltd, Slough, UK equilibrated at room temperature in phosphate buffered saline PBS as recommended by the manufacturer. After all the sample was loaded the column was washed with PBS 2 x 1 ml 1 x 40 ml and the interferon analogue eluted with 0.1M citric acid 0.3M NaCl pH2 . The fraction 400 pal. containing the analogue was adjusted to pH6 by addition of 450 p1. of 0.25M mixed phosphate buffer, pH 7.5 0.25M NaH2P04 0.25M Na2HP041 1 5 . This solution was diluted five fold and twenty fold with RPM1 1640 medium containing 10 foetal calf serum, 2mM glutamine, 2mM sodium pyruvate, 1 vitamins, 50 jig. ml 1 gentamycin and 5 x l05M 2 mercaptoethanol to give samples ready for bioprofiling These samples were divided into multiple aliquots and stored at 20 a new aliquot was used for each assay. The purification was monitored by antiviral and NK2 IRMA assays and the final product was characterised by polyacrylamide gel electrophoresis in the presence of SDS under reducing conditions.b. on YOK Sepharose Clarified supernatant 33 ml. containing the interferon analogue was prepared as hereinbefore described and was applied at a flow rate of about 10 ml. hr. to a column 200 pl. bed volume ofYOK Sepharose monoclonal antibody immuno absorbent support Celltech Ltd, Slough, UK equilibrated at room temperature in phosphate buffered saline PBS as recommended by the manufacturer. After all the sample was loaded the column was washed with PBS 2 x 1 ml 1 x 40 ml. and the interferon analogue eluted with 0.1M citric acid 0.3M NaCl pH2 . The fraction 400 ul. containing the analogue was adjusted to pH6 by addition of 450 pl. of 0.25M mixed phosphate buffer, pH 7.5 0.25M NaH2P04 0.25M Na2HP04, 1 5 .This solution was diluted five fold and twenty fold with RPM1 1640 medium containing 10 foetal calf serum, 2mM glutamine, 2mM sodium pyruvate, 1 vitamins, 50 jig. ml l gentamycin and 5 x l05M 2 mercaptoethanol to give samples ready for bioprofiling. These samples were divided into multiple aliquots and stored at 20 C. a new aliquot was used for each assay. The purification was monitored by antiviral and NK2 IRMA assays and the final product was characterised by polyacrylamide gel eletrophoresis in the presence of SDS under reducing conditions. The antiviral and NK2 IRMA titres for each relevant analogue are set out in Table 5 below. A hyphen in the NK2 IRMA column indicates no crossreactivity. The antiviral and NK2 IRMA titres for the analogues of Examples 27 30 were determined at the clone selection stage. TABLE 5Monoclonal antibody purified analogue profiles EMI52.1 Example SEP Protein SEP Profile SEP Sample SEP Titre SEP U Ml. tb No. SEP AV SEP x106 SEP NK 2IRMA SEP x106 tb SEP 1. SEP endo Asp43a Glu5,Asp10,Asn11 IFN alpha 2 SEP 6.3 SEP 61 tb SEP 2. SEP endo Asp43a Leu151,Arg160,163 IFN alpha 2 SEP 58.0 SEP 160 tb SEP 3. SEP gamma 2 7 2 7 IFN alpha 2 SEP 1 SEP 150 tb SEP 4. SEP gamma 98 114 98 114 IFN alpha 2 SEP 6.1 SEP SEP 5. SEP Met100, gamma 101 114 101 114 IFN alpha 2 SEP 0.7 SEP SEP 6. SEP gamma 98 107 98 107 IFN alpha 2 SEP 11.2 SEP 130 tb SEP 7. SEP gamma 95 101 1012 107 IFN alpha 2 SEP 8.0 SEP 36 tb SEP 8. SEP gamma 2 7 ,Met100,Glu102,103,Arg104,Gly106Asn112Ala113 IFN alpha SEP 3.1 SEP SEP 9. SEP gamma 2 7 , gamma 101 114 IFN alpha 2 SEP 4.7 SEP 10. SEP Met100, SEP gamma 101 114 IFN alpha 2 SEP 1.0 SEP Continuation.... 2 TABLE 5 EMI53.1 Example SEP Protein SEP Profile SEP Sample SEP Titre SEP U Ml. tb No. SEP AV SEP x106 SEP NK 2IRMA SEP x106 tb 11. SEP Leu16,21,59,111,148 IFN alpha 2 SEP 1.0 SEP 13 tb 12. SEP Leu16 IFN alpha 2 SEP 5.5 SEP 27 tb 13. SEP Leu21 IFN alpha 2 SEP 5.1 SEP 64 tb 14. SEP Leu59 IFN alpha 2 SEP 13.8 SEP 86 tb 15. SEP Leu111 IFN alpha 2 SEP 6.2 SEP 18 tb 16. SEP Ala10,37,44,102,104 IFN alpha 2 SEP 19.7 SEP 74 tb 17. SEP Ala137 IFN alpha 2 SEP 30 SEP 18. SEP Ser29,138 IFN alpha 2 SEP 0.18 SEP 5.5 tb 19. SEP Glu32 IFN alpha 2 SEP 5 SEP 41 tb 20. SEP Ala61,64,67 IFN alpha 2 SEP 0.02 SEP 0.29 tb 21. SEP IFN alpha 2 12 165 SEP 0.004 SEP 0.03 tb Continuation.... 3 TABLE 5 EMI54.1 Example SEP Protein SEP Profile SEP Sample SEP Titre SEP U Ml. tb No. SEP AV SEP x106 SEP NK 2IRMA SEP x106 tb 22. SEP IFN alpha 2 10 165 SEP 0.003 SEP 0.01 tb 23. SEP Met100des 101 110 IFN alpha 2 SEP 1.0 SEP 3.8 tb 24. SEP Ala10 15 IFN alpha 2 SEP 0.04 SEP 11 tb 25. SEP Phe115 IFN alpha 2 SEP 0.07 SEP 0.15 tb 26. SEP IFN alpha 2 4 155 SEP 1.6 SEP 6.4 tb 27. SEP Ala32 IFN alpha 2 SEP 0.0009 SEP 1.1 tb 28. SEP Ile30 IFN alpha 2 SEP 0.01 SEP 2.9 tb 29. SEP Asn23 IFN alpha 2 SEP 0.04 SEP 0.8 tb 30. SEP Ala28 IFN alpha 2 SEP 0.14 SEP 2.2 tb 31. SEP Glu5,Ser27,Met31,Leu59 IFN alpha 2 SEP 1.3 SEP 310 tb Example 32 This Example illustrates the preparation of Leu16,21,59,111,148 IFN alpha 2. The gene for this protein is hereinafter referred to for convenience as I 1. The procedure described in Examples 1 31 was repeated with oligonucleotides 7,8,9,10,24,25,45,46,60 and 61 inFigure 1 replaced by oligonucleotides 7 I2, 8 I2, 9 I3, 10 13, 24 14, 25 14, 45 15, 46 15, 60 16 and 16 16 Table 6 . TABLE 6EMI55.1 tb Oligonucleotide SEP I SEP Sequence SEP 5 3 SEP 1 SEP Amino SEP Acid SEP I SEP New tb SEP I SEP I SEP Amino SEP tb SEP changed SEP Acid SEP tb 7 12 SEP CGCACCCTGCTCCTG SEP 16 SEP Leu tb 8 12 SEP TGGGCCAGCAGGAGC tb 9 13 SEP CTGGCCCAACTGCGC tb 10 13 SEP GAGATACGGCGCAGT SEP 21 SEP Leu tb 1 SEP 24 14 SEP 1 SEP ATCAGTTCGTGCAGT SEP I SEP 59 SEP 1 SEP Leu SEP I SEP tb 1 SEP 25 14 SEP I SEP GAACTGATTCAACAG SEP I SEP I SEP I tb 1 SEP 45 15 SEP 1 SEP ACTCCGCTGCTGAAAG SEP I SEP 111 SEP i SEP Leu tb 1 SEP 46 15 SEP 1 SEP GAGTCTTCTTTCAGC SEP I SEP I tb I SEP 60 16 SEP I SEP AACGCAGGATTTCAG SEP RTI ID 55.8 I SEP SEP 148 SEP I SEP SEP Leu SEP I tb 61 I6 SEP CTGCGTTCCTTCAGC tb Fragments A to M were prepared by ligating oligonucleotide mixtures according to the strategy outlined previously. In this Example, fragments B I 2 3E I4, I I5, J IS, L I6, and M I6 replace fragments B,E,I,J,L and M in Examples 1 31. Sequences I, II and IIIM were constructed from the fragments shown in Table 7 after phosphorylation of the 5 hydroxy termini of fragments B to L. TABLE 7EMI56.1 tb Sequence SEP Fragments SEP used tb I 1 SEP A Taq, SEP B I SEP 2 3, SEP C,D SEP tb I SEP II SEP SEP I 1 SEP I SEP E I4, SEP F,G,H tb I SEP IIIM SEP I 1 SEP I SEP I 15, SEP J I5, SEP K, SEP L I6, SEP M I6 SEP I tb The gene I 1 was prepared by ligating sequences ItI 1 , II I 1 and IIIM I 1 and phosphorylated with T4 polynucleotide kinase. The gene I 1 was cloned as described inSection B of Examples 1 31, and expressed in E.coliJA 221 as described in Section Clr the interferon analogue was extracted by lysis of the bacteria as described in Section D and purified by precipitation and size exclusion chromatography as described inSection E of the aforesaid Examples. The purification was monitored by antiviral and NK2 IRMA assays and the final product characterised by polyacrylamide gel electrophoresis in the presence ofSDS under reducing conditions. The titres and recoveries for Leu16,21,59,111,148IFNx2 are summarized below TABLE 8 EMI57.1 SEP Antiviral SEP activity SEP NK2 IRMA SEP activity tb Sample SEP Vol. tb SEP units ml SEP total SEP units SEP units ml. SEP total SEP units tb lysate SEP supernatant SEP 40 SEP 1.3x104 SEP 5.2x105 SEP 4.0x105 SEP 1.6x107 tb extracted SEP protein SEP pellet SEP 21 SEP N D SEP SEP 3.7x105 SEP 7.8x106 tb final SEP sterile SEP filtrate SEP 19 SEP 2.3x104 SEP 4.4x105 SEP 3.6x105 SEP 6.8x106 tb recovery SEP SEP 85 SEP 43 tb Example 33Preparation of Ala10 15 IFN alpha 2 A. Oligonucleotide synthesis The procedure described in Examples 1 31 was repeated using as the new oligonucleotide sequences 5 All, 6 All and 7 All instead of the oligonucleotides 5,6 and 7 respectively.The sequences of 5 All, 6 All and 7 All are as follows oligonucleotide Sequence 5 3 5 All AGCCTGGCTGCAGCT 6 All GCAGCAGCAGCTGCAG 7 All GCTGCTGCGATGCTGFragments A taq to D were prepared by ligating appropriate oligonucleotide mixtures. In this example, fragment A All replaces A Taq and B All replaces B.A All comprises oligonucleotides 1 Taq, 2 Taq, 3,4 and 5 All and B All comprises oligonucleotides 6 All, 7 All, 8, 9 and 10. Ligation of fragments A A11, B All, C and D generates a DNA sequence equivalent to sequence I inExamples 1 31 and includes the Bst E II Eca I restriction site. Following isolation of the A D fragment from a polyacrylamide gel it was ethanol precipitated and resuspended in water 37.75 p1. . Bovine serum albumin 5 pg. in 5 p1 , Bst E II 3 units in 1 l. , a buffer 0.6M Tris HC1 pH 7.4, 0.6 M MgC12 and 0.6M mercaptoethanol 5 p1 and 2M NaCl 1.25 l. were added and the mixture incubated at 37 C. for 16 hours. The restriction digests were stopped by addition of 3M NaCl, pH 5.6 20 ul. and water 130 l. and theDNA precipitated with ethanol then fractionated in a 10 polyacrylamide non denaturing gel. The most slowly migrating band was eluted from the gel, ethanol precipitated and 5 phosphorylated to give the Cla Bst E fragment. B. Cloning of the lAla10 15 IFN X2 gene The vector for cloning the Ala10 15 IFN alpha 2 gene consisted of the large ClaI SalI fragment of the plasmid pSTPl see examples 1 31 ligated to the small BstEII SalI interferon gene fragment of the vector pIFS1205 Edge et al., NucleicAcids Research, Volume 11 1983 p 6419 . Fragments were isolated as for Examples 1 31 and ligated together prior to reisolation from an agarose gel as above. TheDNA concentration was adjusted to 10 g. ml. and mixed with a sample of the synthetic Ala10 15 IFN R 2 gene fragment. The mixture was dried and taken up in 30 ul. of a mixture containing 65 l. 10 x ligase buffer, 65 ul. 1 mg. ml. BSA, 510 ul. water and 10 l.T4 DNA ligase Boehringer, 1.8 units ul. . Incubations were for 1 hour at 4 C. and overnight at 12 C.Ligations were stopped by heating to 65 C. for 15 minutes. 6 p1. of the ligation mixes were used directly to transform the strain MM294 by the method described byHanahan Jour. of Mol. Biol 166 1983 p.557 . Screening for Ala10 15 IFN alpha 2 gene recombinants by colony hybridisation was as described for Examples 1 31 using nick translated probes for small subfragments as described in Examples 1 31. In addition, a colony hybridisation analysis was performed with oligonucleotide probes. For screening for Ala10 15 IFN alpha 2 gene recombinants, the 16 base long oligonucleotide 6 All was used. 5 x 10 4OD units of oligonucleotide was dried in vacuo and taken up in 20 pl. of a mixture containing 25 l. 32 P ATP lOmCi ml, 5000 ci mmol, New England Nuclear , 50 ul. 10 10 x kinase buffer 0.5M Tris pH9, lOmM EDTA , 50 rl lmM spermidine, 50 p1. 0.2M DTT, 50 p1. 100mM MgC12 5 p1. T4 polynucleotide kinase 2 units ul., New England Biolabs and 270 p1. water. The labelled oligonucleotide was diluted into 10 ml. 6 x SSC for hybridisation. The hybridisation temperature used was guided by the formula 4 G C 2 A T C. whereG C and A T denote the number of guanine cytosine or adenine thymine nucleotides in the oligonucleotide.Thus, for the probe 6 All G C ll, A T 5 , the formula yields the result 54 C., the hybridisation actually being performed at 56 C. for at least 1 hour and filters were subsequently air dried for autoradiography. Ala1015 IFN 2 recombinants were identified by positive hybridisation signals.The integrity of the Ala10 15 IFN alpha 2 analogue gene was checked by Hpa II restriction analysis as for Examples 1 31. Lysis of recombinants for biochemical and biological evaluations of interferon were as described in Examples 1 31. The vector containing the Ala10 15lIFN U2 gene was expressed in E.coli DS410 as described inSection D3 of Examples 1 31, the bacteria lysed as described in Section C and Alal0 15JIFN a2 purified by antibody affinity chromatography on NK2 Sepharose. The purification was monitored by antiviral and NK2 IRMA assays and the final product characterized by polyacrylamide gel electrophoresis in the presence ofSDS under reducing conditions. The titres and recoveries for Ala10 15 IFN alpha 2 are summarized below TABLE 9 EMI61.1 SEP Sample SEP Vol. SEP Antiviraol SEP Activity SEP NK2 IRMA SEP Activity tb SEP units ml. SEP total SEP units SEP units ml. SEP total SEP units tb lysate SEP supernatant SEP 36 SEP 2.1 SEP x SEP 10 SEP 7.6 SEP x SEP 106 SEP 4.9 SEP x SEP 105 SEP 1.8 SEP x SEP 107 tb column SEP eluate SEP pH6 SEP 0.85 SEP 3.2 SEP x SEP 104 SEP 2.7 SEP x SEP 108 SEP 1.1 SEP x SEP 107 SEP 9.4 SEP x SEP 106 tb recovery SEP 36 SEP SEP 52 SEP tb Example 34 The procedure of Example 33 was repeated except that the oligonucleotides 17 and 18 were replaced by the oligonucleotides 17 Bst E and 18 BstE Oligonucleotide Sequence 5 3 17 Bst E TTCCCGCAGGAAGAGTTCG 18 Bst E GTTACCGAACTCTTCCFragment D was thus replaced by fragment D Bst E, prepared from oligonucleotides 16, 17 Bst E and 18 BstE.Ligation of A taq, B, C Kll and D Bst E, followed by phosphorylation of the 5 ends with T4 polynucleotide kinase and ATP and isolation of the product by gel electrophoresis gives the Cla BstE fragment. The use of the modified fragment D ensures that the single stranded cohesive end for Bst E II Eca I is present.Example 35 The procedure described in Examples I to 31 was repeated in order to prepare the analogue of Example 21 except that oligonucleotides 4 and 5 as detailed below were replaced by alternative sequences which introduced restriction enzyme sites,but retained the protein sequence of the interferon analogue. Example 49TABLE 10 EMI63.1 Oligonucleotide SEP New SEP sequence SEP Restriction SEP Site SEP New SEP small tb Changed SEP 5 3 SEP Introduced SEP sub fragment tb SEP 4 SEP 4 SEP Sau SEP I SEP CTAAGGAATGAGTTG SEP Sau SEP I SEP A Sau SEP I tb SEP 5 SEP 5 SEP Sau SEP I SEP TCCTTAGGTAGCCGT tb Analogues of the aforementioned Examples were tested for antiviral and antiproliferative activity in the following assays i Antiviral assay Antiviral titers were determined in cytopathic effect inhibition assays by a dye uptake method, using the human amnion cell line WISH challenged with encephalomyocarditis virus. The assay was calibrated using the NIH human leukocyte IFN standard, G 023 901 527. An internal standard was employed in every assay. ii Growth inhibition assay The Daudi cell line was grown in RPMI 1640 medium supplemented with 20mM HEPES and 10 FCS. Cultures containing 5 x 104 Daudi cells in 100 l. of complete medium were incubated with 100 p1. of medium containing 0 to l000tI. of IFN for 72 hr. at 37 C. Four replicates were used for each sample dilution 3HjThymidine lssu.Ci in 50 pi. 5 Ci mmol was added, and the cultures were incubated for a further 24 hr. Cultures were subsequently processed using an automated cell harvester. Results were expressed as the percentage inhibition of 3H thymidine uptake of test sample compared with a medium control. The tested analogues wee all found to possess antiviral activity and the analogues of Examples 2, 4, 5, 11, 16, 18 and 26 were in particular found to possess a ratio of antiviral to antiproliferative activity AV AP substantially better than the AV AP of 3 reported in the literature for natural human IFN y2.