# CYCLIC DOUBLE STRANDED DNA, PROCESS FOR ITS PREPARATION, MICROORGANISM CONTAINING SAID CYCLIC DOUBLE STRANDED DNA, AND PROCESS FOR PREPARING PROTEIN CONTAINING ANTIGEN DETERMINANT SITE OF POLIOVIRUS CAPSIDE PROTEIN VP1 USING THE SAME.

## Abstract
A process for preparing a protein having an antigen determinant site of capside protein VPI utilizing cyclic double stranded DNA containing high expression promotor operator, SD sequence under its control, and a DNA portion which codes an antigen determinant site of capside protein VPI of attenuated poliovirus strain. This invention enables to produce the protein in a large amount as a blended or not blended protein.

## Claims
Claims 1. A cyclic double stranded DNA, comprising a high expression promoter operator, a DNA region coding for the amino group end portion of the metapyrocatechase gene containing the SD sequence of the metapyrocatechase gene positioned in the downstream region under the control thereof and a DNA portion subsequent thereto coding for a peptite corresponding to at least the amino acid number 92 to 105 of the capsid protein VP1 of the poliovirus attenuated strain Sabin type 1, Sabin type 2 or Sabin type 3. 2. A cyclic double stranded DNA according to Claim 1, containing a temperature sensitive gene portion which is positioned upstream on the 5 side of the high expression promoter operator in the opposite direction thereto and regulates its activity negatively. 3. A cyclic double stranded DNA according to Claim 2, containing a portion coading for ampicillin resistance downstream of the temperature sensitive gene portion. 4. A cyclic double stranded DNA according to Claim 1, wherein the high expression promoter operator is promoter operator, lac W 5 promoter operator, trp promoter operator or tac promoter operator. 5. A cyclic double stranded DNA according to Claim 2, wherein the high expression promoter operator is PL promoter operator. 6. A cyclic double stranded DNA, comprising a high expression promoter operator, a SD sequence positioned in downstream region under the control thereof, a translation initiation codon, a DNA portion connected thereto in series so as to coincide in reading frame coding for a peptide corresponding to at least the the amino acid number 92 to 105 of the capsid protein VP1 of the poliovirus attenuated strain Sabin type 1, Sabin type 2 or Sabin type 3 and a translation termination codon subsequent thereto. 7. A cyclic double stranded DNA according to Claim 6, wherein the SD sequence is the SD sequence of the metapyrocatechase gene. 8. A cyclic double stranded DNA according to Claim 6 orClaim 7, wherein the high expression promoter operator isPL promoter operator, and which is a plasmid containing a temperature sensitive gene portion for regulating its activity on the 5 side upstream thereof. 9. A process for preparing a cyclic double stranded DNA, comprising a high expression promoter operator, a SD sequence positioned in downstream region under the control thereof, a translation initiation codon, a DNA portion connected thereto in series so as to coincide in reading frame coding for a peptide corresponding to at least the the amino acid number 92 to 105 of the capsid protein VP1 of the poliovirus attenuated strain Sabin type 1, Sabin type 2 or Sabin type 3 and a translation termination codon subsequent thereto, which comprises cleaving an appropriate restriction endonuclease cleavage site after said translation initiation codon of the DNA containing the high expression promoter operator, the SD sequence positioned in downstream region under the control thereof and the translation initiation codon with said restriction endonuclease, and then, after optionally applying the treatment of smoothening the ends, inserting a DNA fragment coding for a peptide corresponding to at least the amino acid number 92 to 105 of the capsid protein VP1 of the poliovirus attenuated strain Sabin type 1, Sabin type 2 or Sabin type 3 after said translation initiation codon. 10. A process according to Claim 9, wherein the SD sequence is the SD sequence of the metapyrocatechase gene. 11. A process for preparing a cyclic double strandedDNA, comprising a high expression promoter operator, a SD sequence positioned in downstream region under the control thereof, a translation initiation codon, a DNA portion connected thereto in series so as to coincide in reading frame coding for a peptide corresponding to at least the the amino acid number 92 to 105 of the capsid protein VP1 of the poliovirus attenuated strain Sabin type 1, Sabin type 2 or Sabin type 3 and a translation termination codon subsequent thereto, which comprises imparting a translation termination codon to the carboxylic groun end by inserting an oligonucleotide designed so that the translation termination codons may be arranged at three frames at an appropriate restriction endonuclease corresponding to at least the the amino acid number 92 to 105 of the capsid protein VP1 of the poliovirus attenuated strain Sabin type 1, Sabin type 2 orSabin type 3, subjecting this to cleavage at an appropriate restriction endonuclease cleavage site after said translation initiation codon of the DNA containing the high expression promoter operator, the SD sequence positioned in downstream region under the control thereof and the translation initiation codon with said restriction endonuclease, and then, after optionally applying the treatment of smoothening the ends, inserting the DNA obtained after said translation initiation codon. 12. A process according to Claim 11, wherein the SD sequence is the SD sequence of the metapyrocatechase gene. 13. A bacterium of the genus Escherichia, comprising as the plasmid a cyclic double stranded DNA containing PL promoter operator, a DNA region coding for the amino group end portion of the metapyrocatechase gene containing the SD sequence of the metapyrocatechase gene positioned in the downstream region under the control thereof and a DNA portion subsequent thereto coding for a peptite corresponding to at least the amino acid number 92 to 105 of the capsid protein VP1 of the poliovirus attenuated strain Sabin type 1, Sabin type 2 or Sabin type 3. 14. A bacterium of the genus Escherichia, which comprises retaining a cyclic double stranded DNA containing a high expression promoter operator, a SD sequence positioned in downstream region under the control thereof, a translation initiation codon, a DNA portion connected thereto in series so as to coincide in reading frame coding for a peptide corresponding to at least the the amino acid number 92 to 105 of the capsid protein VP1 of the poliovirus attenuated strain Sabin type 1, Sabin type 2 or Sabin type 3 and a translation termination codon subsequent thereto. 15. A bacterium according to Claim 14, wherein the SD sequence is the SD sequence of the metapyrocatechase gene. 16. A bacterium according to Claim 14 or Claim 15, retaining a cyclic double stranded DNA wherein the high expression promoter operator is PL promoter operatore and a temperature sensitive gene portion for regulating its activity is contained upstream on the 5 side thereof. 17. A process for preparing a protein having the antigenic determinant of the polyovirus capsid protein VP1, which comprises culturing a bacterium of the genusEscherichia transformed with a plasmid, comprising a high expression promoter operator, a DNA region coding for the amino group end portion of the metapyrocatechase gene containing the SD sequence of the metapyrocatechase gene positioned in the downstream region under the control thereof, a DNA portion subsequent thereto coding for a peptite corresponding to at least the amino acid number 92 to 105 of the capsid protein VP1 of the poliovirus attenuated strain Sabin type 1, Sabin type 2 or Sabin type 3 and a portion coding for ampicillin resistance positioned upstream thereof, in a nutrient medium, and harvesting the fused protein accumulated in the microorganism cells. 18. A process according to Claim 17, wherein the high expression promoter operator is PL promoter operator, lacW5 promoter operator, trp promoter operator or tac promoter operator. 19. A process according to Claim 17, wherein the plasmid contains a temperature sensitive gene which controls negatiely the high expression promoter operator between said promoter operator and the portion coding for ampicillin resistance. 20. A process according to Claim 19, wherein the high expression promoter operator is PL promoter operator. 21. A process for preparing a protein having the antigenic determinant of the polyovirus capsid protein VP1, which comprises culturing a bacterium of the genusEscherichia transformed with a plasmid, comprising a high expression promoter operator, a SD sequence positioned in downstream region under the control thereof, a translation initiation codon, a DNA portion connected thereto in series so as to coincide in reading frame coding for a peptide corresponding to at least the the amino acid number 92 to 105 of the capsid protein VP1 of the poliovirus attenuated strain Sabin type 1, Sabin type 2 orSabin type 3 and a translation termination codon subsequent thereto, in a nutrient medium, and harvesting the VP1 protein accumulated in the microorganism cells. 22. A process according to Claim 21, wherein the SD sequence is the SD sequence of the metapyrocatechase gene. 23. A process according to Claim 21 or Claim 22, wherein the high expression promoter operator is PL promoteroperator and a temperature sensitive gene portion for controlling its activity is contained upstream on the 5 side thereof.

## Description
SpecificationCyclic double stranded DNA and process for producing the same, microorganism containing said cyclic doublestranded DNA, and process for preparing protein having antigenic determinant of poliovirus caps it protein VP1 by use of them Technical field This invention relates to a production technique, utilizing the recombinant gene technique, of a protein having the antigenic determinant of the capsid proteinVP1 of poliovirus which is a pathogen of acute anterior polyomyelits. Background art At present immunization against acute anterior polyomyelitis is effected by parenteral administration of poliovirus which has been treated with formalin or other chemical reagent to lose its infectiousness while permittin3 its antigenicity to remain, namely an inactivated vaccine, or by oral administration of attenuated poliovirus itself, namely live vaccine.Of these, the inactivated vaccine requires a large amount of virus fzr immunization and therefore high in cost, and also involves the drawback that immunization cannot be maintaine over the whole life.On the other hand, live vaccine has the probability that a highly ,vlthogenic strain may arise through mutation which may occur on the virus gene after repeated proliferation. zDr this reason, there are involved the problems of danger of outbreak of disease by inoculation of live vaccine and exhaustion of seed virus, particularly Sabin type 3. Also, it has the problems such as danger of administration to immuno deficient people, contaminant of monkey tumor virus by use of monkey cells, difficult availability of monkey under emergent circumstances, exhaustion and high cost of monkey source in the future.It has also been attempted to produce an immunizing protein in E. coli cells in place of live vaccine. That is, van der Werf, S., et al Gene, 23, 85 1983 3 andMarr Jirow Japanese Unexamined Patent Publication No. 173084 1984 produced the capsid protein VP1 in the form of a fused protein by expression of cDNA coading for the capsid protein VP1 of type 1 strongly poisonous strainMahoney type in E. coli. It is described there that corresponding DNA sequences of attenuated strains such asSabin type 1, 2 and 3 may also be used, but neither specific means nor possibility of practice is shown. On the other hand, Kuge, S., et al Journal of Biochemistry, 96, 305 1984 3 produced the capsid protein VP1 as the fused protein by expression of the cDNA coding for the capsid protein VP1 of Sabin type 1 incorporated in a plasmid containing an ampicillin resistant gene by use of trp promoter.However, in both of these methods, the amount of VP1 expressed is very minute, and it would be desirable to have an emergence of a method capable of producing a large amount of the capsid protein VP1.The present inventors studied intensively in order to cancel the problems of the prior art as mentioned above, and consequently successfully produced a protein having an antigen determinant of the specific protein of poliovirus in a large amount to accomplish the present invention.In the present specification, non fused means no fusion with a polypeptide other than the proteins derived from polyovirus, and the VP3 protein of poliovirus encoded.before VP1 may be attached, or the VP1 protein containing a part from its intermediate portion may be also included.The present invention is intended to provide a plasmid and a microorganism capable of producing a protein having the antigenic determinant of the capsid protein VPl of the attenuated strain of poliovirus as the fused protein or in the non fused form, and a process for producing said protein by use thereof. Disclosure of the invention A first cyclic double stranded DNA of the present invention comprises a high expression promoter operator, a DNA region coding for the amino group end portion of the metapyrocatechase gene containing the SD sequence of the metapyrocatechase gene positioned in the downstream region under the control thereof and a DNA portion subsequent thereto coding for a peptite corresponding to at least the amino acid number 92 to 105 of the capsid protein VP1 of the poliovirus attenuated strain Sabin type 1, Sabin type 2 or Sabin type 3.A second cyclic double stranded DNA of the present invention comprises a high expression promoter operator, a SD sequence positioned in the downstream region under the. control thereof, a translation initiation codon, aDNA portion connected thereto in series so as to coincide in reading frame coding for a peptide corresponding to at least the the amino acid number 92 to 105 of the capsid protein VP1 of the poliovirus attenuated strain Sabin type 1, Sabin type 2 or Sabin type 3 and a translation terrrination codon subsequent thereto. Examples of the above high expression promoter operator include phage A initial gene groups put promoteroperator, lac UVS promoter operator, trp promoteroperator and tac promoter operator. Pt promoter operator is contained in, for example, theHind III Bam HI fragment derived from phage . Such a fragment may be excised directly from phage A, but it may also be excised from another appropriate plasmid containing the fragment. Examples of such a plasmid include pKC30 plasmid Nature, 292, 128 1981 and pPL 10 plasmid commercial products .An example of the DNA sequence containing the Pt promoter operator is represented by the following formula I 50 5 GATCTCTCACCTACCAAACAATGCCCCCCTGCAAAAAATAAATTCATATA XXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX 100 AAAAACATACAGATAACCATCTGCGGIGATAAATTATCTCTGGCGGTGTT XXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX 150 GACATAAATACCACTGGCGGTGATACTGAGCACATCAGCAGGACGCACTG XXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX 200 ACCACCATGAAAGTGACGCTCTTAAAAATTAACCCCTGAAGAAGGGCAGC I XXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX 250ATTCAAACCAGAAGGCTTTGGGGTCTGTGATACGAACGAACCATTGGCC XXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX 300 RTI ID 5.2 GTAAGTGCGATTCCGGATTAGCTGCCAATGTGCCAATCGCGGGGGCTTTTXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX 350 CGTTCAGGACTACAACTGCCACACACCACCAAAGCTAACTGACAGGAGAAXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX 400 TCCAGATGGATGCACAAACACGCCGCCGCGAACGTCGCGCAGAGAAACAGXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX 450 GCTCAATGGAAAGCAGCAAATCCCCTGTTGGTTGGGGTAAGCGCAAAACCXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX AGTT 3 XXXX wherein A represents a deoxyadenylic acid group, G a deoxyguanylic acid group, C a deoxycitidylic acid group, T a thymidilic acid group, X represents T when the corresponding base in the other single stranded DNA is A, or A when it is T, or G when it is C, or C when it is G . Said DNA sequence contains a DNA sequence corresponding to the Bgl II HPaI fragment of the Hind III Bam HI fragment containing the Pt promoter operator.The DNA sequence coding for the amino group terminus hereinafter called Âµ N terminus of the metapyrocatechase gene containing the SD sequence of the metapyrocatechase gene in the first cyclic double stranded DNA is contained in, for example, the TOL plasmid of Pseudomonas putida mt 2 strain ATCC 23973 . Such a DNA fragment may be excised as the SsT II directly from the TOL plasmid or it may also be excised from another appropriate plasmid containing the fragment. Examples of such a plasmid include pTS 115 plasmid Inoue, S. et al Journal of Bacteriology J Bacteriol. , 145, 1137 1981 and pYT3 plasmid Ebina, Y., et al Journal ofBacteriology J. Bacteriol. , 156, 487 1983 .An example of the DNA sequence coding fo the N terminus of the metapyrocatechase gene containing said SD sequence is the DNA sequence of which single strand is shown by the following formula II 5 AATTCCGGTCACGTGCCGGCCAAGGCCGCCGCCGAAGGCTGGCCCTGGCC TCTCAAGTGTTTCCGCAAACCGACTTGACCATCGAGTACTTCGCCACC TT u GGCGGAAACAAACCTGACAACATGAACTATGAAGAGGTGACGTCATGAAC AAAGGTGTAATGCGACCGGGCCATGTGCAG 3 wherein A, G, C and T have the same meanings as defined above . The SD sequence in the second cyclic double stranded DNA in the present invention may include the SD sequence of the metapyrocatechase gene, the SD sequence ofcII, theSD sequence of Acro, the SD sequence of the lacZ gene, the SD sequence of the trpL, and the SD sequence of the lpp gene, of which the SD sequence of the metapyrocatechase gene is preferred.The DNA sequence containing the SD sequence of the metacatechase gene and the translation initiation codon ATG can be obtained by preparing this portion according to, for example, chemical synthesis, and recombining said DNA fragment with a suitable vector for expression having a high expression promoter operator. Alternatively, it may be prepared according to a genetic engineering method from a high expression promoter operator and a vector for expression containing the SD sequence of the metapyrocatechase gene.Examples of such a vector for expression include the pHT 3 plasmids deposited at Institute ofFermentation Research, Agency of Industrial Science BR Technology, Ministry of International Trade Industry hereinafter abbreviated as FERM as Escherchia coli HB 101 pHT3 FERM P 7776, Escherchia coli C 600 pHT3 FERMP 7777, and Escherchia coli W 3110 pHT3 FERM P 7778, and pHT 311 plasmid deposited at FERM as Escherchia coli HB 101 pHT3 FERM P 7994 .Further, an example of the vector having the SD sequence of the metapyrocatechase gene and the translation initiation codon, and also having a cleavable restriction endonuclease cleaving site thereafter is the pYTU3 plasmid deposited at FERM as Escherchia coli HB l0l pYTU3 FERMP 7992 .The DNA sequence in the SD sequence and the initiation codon region of the plasmid pYTU3 is represented by the following formula III TGAAGAGGTGCATCG ATG TCG A III EMI8.1 wherein A, G, C and T have the same meanings as defined above .In the present invention, the portion coding for a peptide corresponding to at least the amino acid number 92 to 105 of the capsid protein VP1 of the poliovirus attenuated strain Sabin type 1, Sabin type 2 or Sabin type 3 may be any DNA sequence, provided that it can code for the peptide represented by the following formula Val Asp Asn Ser Ala Ser Thr Lys Asn Lys Asp Lys Leu Phe Val Asp Asn Asp Ala Pro Thr Lys Arg Ala Ser Arg Leu Phe orVal Asp Asn Glu Gln Pro Thr Thr Arg Ala Gln Lys Leu Phe Of these DNA sequences, those related to Sabin type 1 are contained in the Taq I fragment, the qTa I Kpn I fragment and the Bam HI Hinc II fragment derived from the PBB2 plasmid Nomoto, Akio, et al ProceedingNational Academic Science U.S.A. Proc. Natl. Acad. Sci.USA. , 79, 5793 1982 .The DNA sequence of said Tag I fragment is represented by the following formula IV 50 5 CGACACCCAGAGAGATGGACATCCTTGGTTTTSTGTCAGCGTGTAATGAC XXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX 100TTCACCGTGCGCTTGATGCGAGATACCACACATATAGAGCAAAAAGC CT XXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX 150 AGCACAGGGGTTAGGTCAGATGCTTGAAAGCATGATTGACAACACAGTCCXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX 200GTGAAACGGTGGGGGCGGCAACGTCTAGAGACGCTCTCCCAAACACTwAA XXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX 250GCCAGTGGACCAGCACACTCCAAGGAAATTCCGGCACTCACCGCAQT.GA IV XXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX 300AACTGQGGCCACAAATCCACTAGTCCCTTCTGATACAGTGCAAACCAGAC XXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX 350 ATGTTCTACAACATAGGTCAACGTCAGAGTCTACCATAGAGTCTTTCTTCXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX 400CCCCCCCCTGCATCCGtGGCCATTATAACCCTCGATAACTCAGCTTCCACXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX 450CAAGAATAACGATAAGCTAtTTACACTCTGGAAGATCACTTATAAAGATA XXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX 500 CTGTCCAGTTACGGAGGAAATTGGAGTTCTTCACCTATTCTAGATTTGATXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX 550 AAATTTCAC CACACrAACAATCCATGGAATTTACCTTtGTGCTTACTGCAAATTTCACTGAGACTAAGAATGGXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX 600GCATGCCTTAAATCAAGtGTACCAAATTATGTACGTACCACCAGGCGCTCXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX 650 CAGTGCGACAAATGGGACGACTACACATGGCAAACCTCATCAAATCCATCXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX 700 700 AATCTTTTACACCTACGGAACAGCTCCAGCCCGGA TCTCGTA CGtATG XXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX XXXXXXXXXXXXXXX TTGGTATTT 3 XXXXXXXXXGC wherein A, G, C, T and X have the same meanings as defined above the DNA sequence of said Taq I Kpn I fragment is repre sented by the following formula V 50 5 CGACACCCAGAGAGATGGACATCCTTGGTTTTGTGTCAGCGTGTAATGAC XXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX X, 100 TTCAGCGTGCGCTTGATGCGAGATACCACACATATAGAGCAAAAAGCGCT XXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX,XX 150 AGCACAGGGGTTAGGTCAGATGCTTGAAAGCATGATTGACAACACAGTCC XXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX,XXXXXXXXXX 200 GTGAAACGGTGGGGGCGGCAACGTCTAGAGACGCTCTCCCAAACACTGAA XXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX 250GCCAGTCGACCAGCAGACTCCAACGAAATTCCGGCACTCACCGCAGTGGl XXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX 300AACTGGCCCACAAATCCATAGTCCCTTCTGATACAG TGCAAACCAGAC XXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX 350 ATGTTGTACAACATAGCTCAAGGTCAGACTCTAGCATAGAGTCTTTCTTC V xxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxx 400 RTI ID 12.4 GCGCGGGUTGCATGGGTGGCCATTATAACCCTGGATAACTCAGCTTCCACXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX,XXXXXXXXXXXXXX 450 CAAGAATAAGGATAAGCTATTTACAGTGTGGAAGATCACTTATAAAGATA XXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX 500CTGTCCAGTTACGGAGGAAATTGGAGTTCTTCACCTATTCTAGATTTGAT XXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX 550 ATGGAATTTACCTTTGTGGTTACTCCAAATTTCACTCAGACTAACAATGG XXXXXXXXXXXXXXXXX XXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX 800 GCATGCCTTAAATCAAGTGTACCAAATTATGTACGTACCACCAGGCGCTC XXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX 650 CACTCCCCACAAATGG ACGACTACACATGGCAAACCTCATCAAATCCA XXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX 700 TCAATCTTTTACACCTACGGAACAGCTCCAGCCCGGATCTCGGTACCGTAXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX 750 TGTTGGTATTTCGAACGCCTATTCACACTTTTACGAOGGTTTTTCCAAAG XXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX 800 TACCACTGAAGGACCAGTCGGCAGCACTAGGTGACTCCCTCTATGCTCCA XXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX 850GCATCTCtAAATGACTTCGTATTTTGGCT TTAGAGTAGTCAATGATCA XXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX 800CAACCCGACCAAGGTCACCTCCAAAATCAGAGTGTATCTAAAACCCAAAO XXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX 950ACATCAGAGTCTGGTGCCCGCGTCCAOCGAGGGCAGTGGCGTACTACGGCXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX 1000CCTCGAGtGGlTTACAAGGATGGTACGCTTACACCCCTCTCCACCAAGGA XXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX 1050TCTGACCACATATGGATTCGGACACCAAAACAAACCGGTGTACACTGCAG XXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX 1100GTTACAAAATTTGCAACTACCATTTGGCCACTCAGGAAGATTTGCAAAAC XXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX 1150 1150 GCAGTGAACGTCATCTGGAATAGAGACCTCTTACTCACAGAATCAAGAGC XXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX 1200 CCAGGGCACCGATTCAATCGCAAGGTGCAATTGCAACGCAGGGGTGTACT XXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX 1250 ACTGCGAGTCTAGAAGGAAATACTACCCAGTATCCTTCGTTCGCCCAACG XXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX TTCCAGTACATGGAGGCTAATAACTATTACCCAGCTAGGTAC 3 XXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX wherein A, G, C, T and X have the same meanings as defined above the DNA sequence of said Bam HI Hinc II fragment is represented by the following formula VI 50 5 ...CATCCATC ATCGCAACTCGCAAACTGTTGGTGTCATACGCGCCTCCTGGA XXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX 100GCCGACCCACCAXAGAAGCGtlAGGAGGCGATGTTGGGlACACATGTGAT XXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX 150CTGGGACATAGGACTGCAGTCCTCATGTACTATG TAGTGCOATG ATTA XXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX 200 GCAACACCACGTATCGGCAAACCATAGATGATAGTTTCACCGAAGGCGCAXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX 250TACATCAGCGTCTTCTACCAAACCAGAATAGTCGTCCCTCTTTCGACACC XXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX 300CAGAGAGATGGACATCCTTGGTTTTGTGTCAGCGTGTAATGACTTCAGCG XXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX 350 TGCGCTTGATGCGAGATACCACACATATAGAGCAAAAAGCGCTAGCACAGXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX 400GGGTTAGGTCACATGCTTGAAACATGATTGACAACACAGTCCGTGAAACXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX 450 GCTGGGGGCGGCAACGTCTAGAGACGCTCTCCCAAACACTGAAGCCAGTG XXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX 500 ACCACCACACTCCAAGCAAATTCCGGCACTCACCCCAGTGGAAACTGGG XXXXXXXXXXXXXXXXXXXXXX XXXXXXXXXXXXXXXXXXXXXXXXXXX,X 550 GCCACAAATCCACTAGTCCCTTCTGATACAGTGCAAACCAGACATGTTGTXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX 600ACAACATAGGTCAAGGTCAGAGTCTAGCATAGAGTCTTTCTTCGCGCGGG V XXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX 650GTGCATGCGTGGCCATTATAACCGTGGATAACTCAGCTTCCACCAAGAATXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX 700AAGGATAAGCTATTTACAGTGTGGAAGATCACTTATAAA ATACTGTCCA XXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX 750GTTACGGAGAAATTGGAGTTCTTCACOTATTCTAGATTTGATATGGAAT XXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX 800TTACCTTTGTGGTTACTGCAAATTTCACTGAGACTAACAATGGGCATGCC XXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX 850 TTAAATCAAGTGTACCAAATTATGTACGTACCACCAGGCGCTCCAGTGCCXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX 900CGAGAAATGGGACGACTACACATGGCAAACCTCATCAAATCCATCAATCTXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX 950 TTTACACCTACGGAACAGCTCCAGCCCGGATCTCGGTACCGTATGTTGGT XXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX 1000 ATTTCGAACGCCTATTCACACTTTTAC ACGGTTTTTCCAAAGTACCACT XXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX 1050GAAGGACCAGTCGGCAGCACTAGGTGACTCCCTCTATGGTGCAGCATCTC XXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX 1100TAAATGACTTCGGTATTTTGGCTGTTAGAGTAGTCAATGATCACAACCCG XXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX 1150 ACCAAGGTCACCTCCAAAATCAGAGTGTATCTAAAACCCAAACACATCAG xxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxx 1200AGTCTCCTCCCCGCGTCCACCGACGCCACTGCGTACTACGGCCCTGGA XXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX 1250TGGATTACAAGGATGGTACGCTTACACCCCTCTCCACCAAGGATCTGACC XXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX 1300ACATATGGATtCGGACACCAAAACAAAGCGGTCTACACtGCAGGTTACAA XXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX 1350 AATTTGCAACTACCATTTGGCCACTCAGGAAGATTTGCAAAACGCAGTGAXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX 1400ACGTCATGTGGAAtAGAGACCTCTTAGTCACAGAATCAAGACCCCAGGGC XXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX 1450 ACCGATTCAATGGCAAGGTGCAATTGCAACGCAGGGGTGTACTACTGCGAXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX 1500GTCTAGAAGGAAATACTACCCAGTATCCTTCGTTGGCCCAACGTTCCAGT XXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX 1550 ACATGGAGGCTAATAACTATTACCCAGCTAGGTACCAGTCCCATATGCTCXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX 1600 ATTGGCCATGGATTCGCATCTCCACGGGATTGTGGTGGCATACTCAGATCXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX 1650 TCACCACGGGGTGATAGGGASCATTACTGCTGGTGGAGAAGGGTTGGTTGXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX 1700CATTTACAGACATTAGAGACTTGTATGCCTACGAAGAAGAAGCCATGGAA XXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX 1750 CAAGGCATCACCAATTACATAGAGTCACTTGGGGCCCCATTTGGAAGTGGXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX ATTTACTCAGCTGATTGGAGACAAAATAACAGAGTT 3 XXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX wherein A, G, C, T and X have the same meanings as defined above . Those related to Sabin type 2 are contained in the Kpn I Pvu II fragment derived from the pVS 2 2503 plasmid Toyoda, Haruka, et al Journal of Molecular Biology J.Mol. Biol. , 174, 561 1984 .The DNA sequence of said Kpn I Pvu II fragment is represented by the following formula VII 50 5 ... CTTGGATCAGTAATACCACATACAGACAAACCATCAACGATAGTTTCACA CATGXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX 100 GAAGGTGGCTACATTAGCATGTTCTATCAAACTAGGGTTGTTGTCCCGTT XXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX 150 GTCCACACCCAGAAAGATGGACATCCTGGGTTTTGTGTCAGCTTGCAATG XXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX 200 ACTTCAGTGTGCGCTTACTGCGAGATACAACACACATTAGTCAAGAGGCT XXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX 250 ATGCCACAAGCAATTGGTCACATGATTGAAGCGGCCCTTCAAGGGATTACXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX 300TAAAAATGCATTGGTTCCCCCGACTTCCACCAATAGCCTGCCTGGACACA viz RTI ID 23.3 XXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX 350AGCCCAGCCGTCCAGCCCACTCCAAGGAGATACCTGCATTGACAGCCGTG XXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX 400 GAGACAGGGGCTACCAATCCGTTGGTGCCTTCGGACACCGTGCAAACGCGXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX 450 CCATGTCATCCAGAGACGAACGCGATCAGAGTCCACGGTTGAGTCATTCT XXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX 500TTGCAAGAGGGGCTTCCGTGGCTATCATTGAGGTGCACAATGATCCACCG XXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX 550 ACAAAGCGCGCCAGCAGATTGTTTTCGGTTTGGAAAATAACTTACAAAGAXXXxXXXXXXXXXXXXXXXXxXXXXXXXXXXXXXXXXXXXXXXXXXXXXX 600 TACTGTTCAACTGAGACGCAAACTGGAATTTTTCACATA TTCGAGATTTG XXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX 650 RTI ID 24.3 ACATGGAGTTCACTTTTGTGGTCACCTCAAACTACATTGATGCAAATAAC XXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX 700 GGACATGCATTGAACCAAGTTTATCAGATAATGTA TATACCACCCGGAGC XXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX 750 ACCTATCCCTGGTAAATGGAATGACTATACGTGGCAGACGTCCTCTAACC XXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX 800CGTCGGTGTTTTACACCTATGGGGCGCCcCCAGCAAAATATCAGTGCCCXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX 850 TACGTGGGAATTGCTAATGCGTATTCCCACTTTTATGATCOGTTTGCAAAXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX 900 AGTACCACTAGCGGGTCAAGCCTCAACTGAAGGCGATTCGTTGTACGGTGXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX 950 RTI ID 25.3 CTGCCTCACTGAATGATTTTGGATCACTGGCTGTTCGCGTGGTAAATGATXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX 1000CACAACCCCACGCGGCTCACCTCCAAGATCAGAGTGTACATGAAGCCAAA XXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX 1050GCATGTCAGAGTCTGGTGCCCACGACCTCCACGAGCAGTCCCATACTTCG XXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX 1100GACCAGGTGTTGATTATAAAGATGGGCTCACCCCACTACCAGAAAAGGGA XXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX 1150 TTAACGACTTATGGATTTGGACACCAAAACAAAGCTGTGTACACAG 3 XXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX wherein A, G, C, T and X have the same meanings as defined above .Further, those related to Sabin type 3 are contained in the Sph I Bgl II fragment derived from the pVS 2 2503 plasmid Toyoda, Haruka, et al Journal of MolecularBiology J. Mol. Biol. , 174, 561 1984 .The DNA sequence of said ph I Bgl II fragment is represented by the following formula VIII 50 5 CTGGGGTTTGTGTCAGCCTGTAATGATTTCAGTGTGGGATTGCTGCGAGAGTACXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX 100 CACCACTCACATTTCACAATCTGCGCTTCCACAGGGTATTGAAGATTTGA XXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX 150 TTTCTGAAGTTGCACAGGGCGCCCTAACTTTGTCACTCCCGAAGCAACAG XXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX 200 GATACCTTACCTGATACTAAGGCCAGTGGCCCGGCGCATTCCAAGGAGGT W XXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX 250ACCTGCACTCACTCCAGTCGACACTGCAGCCACCAATCCTCTGCACOAT xxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxx 300 RTI ID 27.2 CCGACACAGTTCAAACGCGCCACGTAGTCCAACGACGCAGCAGGTCAGAGXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX 350TCCACAATAGAATCATTCTTCGCACGCGGGGCGTGCGTCGCTATTATTGAXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX XXXXXX 400GGTGGACAATGAACAACCAACCACCCGGGCACAGAAACTATTTGCCATGTXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX 450 GGCGCATTACATACAAAGATACAGTGCAGTTGCGCCGTAAGTTGGAGTTTXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX 500 TTCACATACTCTCGTTTTGACATGGAATTCACCTTCGTGGTAACCGCCAAXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX 550CTTCACCAACGCTAATAATGGGCATGCACTCAACCAGGTGTACCAGATAA XXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX 800TGTACATCCCCCCAGGGGCACCCACACCAAAGTCATGGGACGACTACACT XXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX 650TGGCAAACATCTTCCAACCCGTCCATATTTTACACCTATGGGGCTGCCCC XXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX 700CGCGCGAATCTCACTGCCATACGTGGGGTTAGCCAATGCTTACTCGCACT XXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX 750 TTTACGACGGCTTCGCCAAGGTGCCATTGAAGACAGATGCCAATGACCAGXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX 800 ATTGGTGATTCCTTGTACAGCGCCATGACAGTTGATGACTTTGGTGTATTXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX 850GGCAGTTCGTGTTGTCAATGATCACAACCCCACTAAAGTAACCTCCAAAG XXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX 900TCCGCATTTACATGAAACCCAAACACGTACGTGTCTGGSGCCCTAGACCGXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX 950 CCGCGCGCGGTACCTTATTATGGACCAGGGGTGGACTATAGGAACAACTT XXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX 1000 GGACCCCTTATCTGAGAAACGTTTGACCACATATGGCTTTGGGCATCAGA XXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXATAAACTGTGTACACTGCTGGTTACAA... 3 XXXXXXXXXXXXXXXXXXXXXXXXXXXXCTAG wherein A, G, C, T and X have the same meanings as defined above , The first cyclic double stranded DNA of the present invention should preferably contain a temperature sensitive gene portion positioned upstream on the 5 side of the high expression promoter operator in the opposite direction thereto for regulating its activity.The second cyclic double stranded DNA of the present invention should preferably contain a temperature sensitive gene portion positioned upstream on the 5 side of the high expression promoter operator for regulating its activity.More specifically, an example of the temperature sensitive gene portion for regulating the activity of the high expression promoter operator is the cm857 mutant gene of the phage X cI gene. The cI857 hereinafter called cults mutant gene is a gene coding for the temeprature sensitive repressor of the Pt promoter operator.The DNA fragment containing such a gene may be taken out directly from, for example, phage A, but tit may also be excised and used from a plasmid containing such a fragment such as the pCQV2 plasmid Queen, C. Journal ofMolecular and Applied Genetics, 2 1 , 1 1983 .The DNA sequence of the cIts mutant gene portion derived from the pCQV2 plasmid may include, for example, the sequence of the following formula IX 50 5 AAtTCTCATGTTTGACAGCTTATCATATGAAGATTCTTGCTCAATTGTTA XXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX r00 TCAGCTATGCGCCGACCAGAACACCTTGCCGATCAGCCA AACGTCTCTTC XXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX 150AGGCCACTGACTAGCGATAACTTTCCCCACAACGGAACAACTCTCATTGC XXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX 200 ATGGGATCATTGGGTACTGT GGTTTACTCGTTGTAAAAACACCTGACCGXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX 250 CTATCCCTGATCAGTTTCTTGAAGGTAAACTCATCACCCCCAAGTCTGGC XXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX 300 IX TATCCAGAAATCACCTGGCTCAACAGCCTGCTCAGGGTCAACGAGAATTAXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX 350ACATTCCGTCAGGAAAGCTTGGCTTGGAGCCTGTTGGTGCCGTCATGGAA XXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX 400 TTACCTTCAACCTCAAGCCAGAATGCAGAATCACTGGCTTTTTTGGTTGTXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX 450GCTTACCCATCTCTCCGCATCACCTTTGGTAAAGGTTCTAAGCTTAGGTG XXXXXXXXXXXXRXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX 500 RTI ID 32.3 ACAACATCCCTGCCTGAACATGAGAAAAAACACGGTACTCATACTCACTTXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX 550CTAAGTCACGGCTGCATACTAACCGCTTCATACATCTCGTAGATTTCTCT XXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX 600 GGCGATTGAAGGGCTAAATTCTTCAACGCTAACTTTGAGAATTTTTGCAAXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX 650GCAATGCCGCGTTATAAGCATTTAATGCATTGATGCCATTAAATAAAGCAXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX 700CCAACGCCTGACTGCCCCATCCCCATCTTGTCTGCGACAGATTCCTGGGA XXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX 750TAAGCCAAGTTCATTTTTCTTTTTTTCATAAATTGCTTTAAGGCGACGTG XXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX 800CGTCCTCAAGCTGCTCTTGTGTTAATGGTTTCTTTTTTGTGCTCATACGT XXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX 850TAAATCTATCACCGCAAGGGATAAATATCTAACACCGTGCGTGTTGACTA XXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX 900 TTTTACCTCTGGCGGTGATAATGGTTGCATGTACTAAGGAGGTTGTATGXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXC 3 TAG wherein A, G, C, T and X have the same meanings as defined above . The cyclic double stranded DNA of the present invention should preferably contain a portion coding for ampicillin resistance downstream of the temperature sensitive gene portion, and said portion should more preferably be derived from the pBR322 plasmid.The first cyclic double stranded DNA can be prepared, for example, as follows.The pCQV2 plasmid is digested with the restriction endonuclease Cla I and the protruded end digested with S1 nuclease to make it a blunt end, which is cyclized again with T4DNA ligase, thereby extinguishing the Cla I site to obtain the pHT 1 plasmid. The cyclic DNA obtained was transformed into, for example, Escherichia coli HB 101 strain following the method of Cohen et al ProceedingNational Academic Science USA Proc. Natl. Acad. Sci.USA , 69, 2110 1972 , and the pHTl plasmid containing strains are screened with ampicillin resistance as the marker and amplified. From the transformed strain, the pHT 1 plasmid is isolated. Between the Bam HI site and the Eco RI site of the plasmid, there is contained the temperature sensitive gene cIts which is the repressor of the PR and PL promoters of phage A. By digesting the pHTl plasmid with Eco RI and Bam HI, the fragment containing cIts is isolated. This fragment comprises the DNA sequence shown by the above formula CG IX . The GC base pairs which existed at the Cla I site of the corresponding fragment of pCQV 2 which existed T the T A pair between the base pair at the 26 th and the TA base at the 27 th in the above formula IX has been removed.On the other hand, the pKC 30 plasmid is digested withEco RI and Bgl II to separate a long fragment therefrom. This fragment has one Bam HI site, and the DNA sequence from the Eco RI end to the Bam HI site corresponds to theDNA sequence anticlockwise from the Eco RI site of the pBR322 to the Bam HI site on the restriction endonuclease cleaved map. In this sequence, the portion coding for ampicillin resistance and the initiation point for duplication in E. coli are contained. There exists also one Hpa I site between the Bam HI site and the Bgl II end, and the PL promoter operator exists between the BglII end and the Hpa I site. The DNA sequence between theBgl II end and the Hpa I site is shown by the above formula I .The both fragments as obtained above are ligated withT4DNA ligase and cyclized to construct the pHT 2 plasmid.The plasmid is cloned similarly as the pHT 1 plasmid to be amplified.The pYT 3 plasmid is digested with the restriction endonuclease Sst II, the C 230 fragment containing the portion coding for metapyrocatechase is isolated and the 3 protruded end is digested with T4DNA polymerase to be made a blunt end.The pHT 2 plasmid is subjected to ring opening by digestion with the restriction endonuclease Hpa I and then theC 230 fragment with the blunt end is inserted thereinto, followed by ring closure, to construct the pHT 3 plasmid. E. coli is transformed with this plasmid and, with the use of ampicillin resistance as the marker, the colony colored in yelow with an aqueous catechol solution after elevation of the culture temperature to about 42 OC in the course of cultivation is collected, followed by cloning and amplification of the plasmid.In place of the C 230 fragment from the pYT 3 plasmid, the pSLMK 1 plasmid of the transformants having this plasmid, Escherichia coli HB 101 pSLMK 1 is deposited atFERM as the FERM P 7816, Escherichia coli W 31l0 pStMK 1 as the FERM P 7617, Escherichia coli RR l pStMK 1 as theFERM P 7618 and Escherichia coli RB 79l pSLMK 1 as theFERM P 7619, respectively or the pTCCM 1 plasmid of the transformants having this plasmid, Escherichia coli RB 791 pTCCM is deposited at FERM on August 17, 1984 as FERMP 77815, and transferred to the international deposition based on the Budapest Treaty on August 8, 1985, and given the number of FERM BP 862 may be digested with Eco RI, and the resultant C 230 fragment filled at both protruded ends with dNTP and inserted into the plamid pHT 2 to prepare the pHT 3 plasmid.The method for preparation of the pSLMK 1 plasmid is described in detail in Japanese Patent Application No.97106 1984.Next, the pHT 3 plasmid is digested with Bam HI, the protruded ends are repaired with T4DNA polymerase orKlenow fragment to be made blunt ends, and further digested partially with Pvu II, followed again by cyclization with T4DNA ligase to obtain the pHT 31 plasmid. At the ligated portion, the Bam HI portion is regenerated. The pHT 31 plasmid is digested with Sal I, the protruded portions are repaired with T4DNA ligase orKlenow fragment to make them blunt ends, and further digested with Pvu II, followed again by cyclization withT4DNA ligase to construct the pHT 311 plasmid. At the ligated portion, the Sal I is regenerated.The operations of the pHT 3 plasmid et seq are operations for making an appropriate restriction endonuclease site at the DNA portion coding for the N terminus portion of metapyrocatechase, and it is also possible to attempt insertion of the poliovirus cDNA fratment through utilization of the Pvu II site in the metapyrocatechase gene by digesting the pHT 3 plasmid partially with Pvu II. Next, the pHT 311 plasmid is digested with Sal I, the protruded ends are repaired with T4DNA polymerase orKlenow fragment to be made blunt ends to obtain a DNA fragment. On the other hand, a DNA fragment is prepared containing the VP1 gene from the cDNA derived from poliovirus.For example, in Sabin type 1, the pBB 2 plasmid is digested with Taq I, the protruded ends are repaired withT4DNA polymerase and a DNA fragment containing VP1 is isolated. Said DNA fragment and the DNA fragment obtained from the pHT 311 plasmid as described above are subjected to ligating cyclization with T4DNA ligase to construct the pHTP 31 plasmid.The pBB 2 plasmid is digested with Bam HI and Hin cII, the protruded ends are repaired with T4DNA polymerase, and the DNA fragment containing VP1 is isolated. SaidDNA fragment and the DNA fragment obtained from the pHT 311 plasmid as described above asre subjected to ligating cyclization with T4DNA ligase to construct the pYTP 100 plasmid. The pHTP plasmid is digested with Kpn I to prepare a DNA fragment.The pBB 2 plasmid is digested with Kpn I to separate aDNA fragment containing the carboxylic terminus hereinafter called C terminus of VP1. Said fragment and theDNA fragment obtained by digestion of the pHTP 31 plasmid with Kpn I are subjected to ligating cyclization withT4DNA ligase to construct the pYTP 11 plasmid.As for Sabin type 2, for example, the pVS 2 2503 plasmid is digested with Kpn I and Pvu II, the protruded ends are repaired with T4DNA to be made blunt ends, and the DNA fragment containing VP1 is isolated. Said DNA fragment and the DNA fragment obtained from the pHT 311 plasmid as described above are subjected to ligating cyclization to construct the pYTP 200 plasmid.As for Sabin type 3, for example, the pVS 3 2603 plasmid is digested with Sph I and Bgl II, the protruded ends are repaired with T4DNA polymeraze or Klenow fragment to be made blunt ends, and the DNA fragment containing VP1 is isolated. Said DNA fragment and the DNA fragment obtained from the pHT 311 plasmid as described above are subjected to ligating cyclization to construct the pYTP 300 1 plasmid. This does not coincide in frame of amino acids of a part of the metapyrocatechase gene and the DNA derived from poliovirus. Accordingly, for coincidence in frame, the pYTP 300 1 plasmid is digested with Sal I, the protruded ends digested with S1 nuclease, followed by recylization with T4DNA ligase to construct the pYTP 300 plasmid. The second cyclic double stranded DNA of the present invention can be prepared, for example, as follows.First, for making the N terminus of the poliovirus VPl protein in the non fused form, the vector pYTU3 for expression deposited at FERM as Escherichia coli HB101 pYTU3 FERM P 7982 can be utilized. The pYTU plasmid is digested with Sal I, the protruded ends are digested with S1 nuclease to be made blunt ends, whereinto the DNA fragment coding for the VP1 antigen region of poliovirusSabin type 1, Sabin type 2 or Sabin type 3 made to have blunt ends is inserted. In the case of Sabin type 1, for example, the fused type expression plasmid pHTP31 plasmid or the pYTPll plasmid can be used.The pHTP31 plasmid and the pYTPll plasmid are deposited at American Type Culture Collection ATCC as Escherichia coli C 600 pHTP31 and Escherichia coli C 600 pYTPll, respectively, and attached with the deposition numbers ofATCC 39976 and ATCC 39972, respectively.When using the pHTP31 plasmid, this is digested with the restriction endonuclease Sal I to obtain a DNA fragment of 713 base pairs. The DNA fragment may be blunted at the ends with T4DNA polymerase and insereted into the SalI site of the blunt end of pYTU3 with the use of T4DNA ligase, but without blunting of the ends, the Sal I digested DNA fragment of the pYTU3 plasmid and the DNA fragment of Sal I 713 base pairs containing the VP1 protein antigen site can be cyclized again with the use of T4DNA ligase to obtain the pYTP102. Next, the operation of making the C end non fused is conducted. The pYTP102 plasmid is digested with the restriction endonuclease Kpn I and then blunted the ends with T4DNA polymerase.The commercially available oligonucleotide of 16 base pairs having translation termination codon TAA in three frames hereinafter called translation termination unit represented by the following formula X 5... . GCTTAATTAATTAAGC .3 3 1 CGAATTAATTAATTCC 5 wherein A, G, C and T have the same meanings as defiend above , is phosphorylated at the 5 end with T4 polynucleotide kinase, and this was subjected to ligating cyclization with T4DNA ligase together with the DNA fragment obtained by digestion of the pYTP 102 plasmid with Kpn I, followed by blunting of the ends with T4DNA polymerase, to obtain the pYTP102K plasmid. On the other hand, non fused N terminus and C terminus can be also constructed at the same time. The pYTPll plasmid is digested with Pst I, followed by blunting of the ends with T4DNA polymerase. Said DNA fragment is ligated with the translation termination unit with T4DNA ligase. Further, after digestion of the restriction endonuclease Sal I followed by agarose gel electrophoresis, the DNA fragment of 1050 base pairs containing the region coding for the VP1 protein antigen site is recovered.Double digestion of the pYTU3 plasmid with the restriction endonucleases Sal I and Nru I gives a DNA fragment of 4000 base pairs, which is subjected together with the above DNA fragment of 1050 base pairs to ligating cyclization with T4DNA ligase to obain the pYTP102P plasmid.As for Sabin type 2, for example, after digestion of the fused type expressed plasmid pYTP200 with the restriction endonuclease Sal I followed by agarose gel electrophoresis, the DNA fragment of about 1160 base pairs containing the region coding for the VP1 protein antigen site is recovered. Said fragment and the DNA fragment obtained by digestion with the pYTU3 plasmid are subjected to ligating cyclization to construct the pYTP201. The pYTP201 plasmid is digested with the restriction endonuclease Nar I and blunted the ends with T4DNA polymerase.Said DNA fragment and the 5 end phosphorylated product obtained by treating the translation termination unit with T4polynucleotide kinase are subjected to ligating cyclization with T4DNA ligase to construct the pYTP201N plasmid. The pYTP plasmid is deposited at ATCC as Escherichia coli 600 pYTP200, and attached with the deposition number ofATCC 39977.As for Sabin type 3, for example, after digestion of the fused type expressed plasmid pYTP300 with the restriction endonuclease Pst I, it is blunted the ends with T4DNA polymerase and then digested with the restriction endonuclease Bam HI, followed by agarose gel electrophoresis, to obtain a DNA fragment of about 1500 base pairs.On the other hand, the pYTU3 plasmid is digested with the restriction endonuclease Sal I, followed by blunting the ends with S1 nuclease. Subsequently, after digestion with the restriction endonucelase with Bam HI followed by agarose gel electrophoresis, a DNA fragment of about 3800 base pairs is recovered. Said DNA fragment and the above fragment of about 1500 base pairs are subjected to ligating cyclization with T4DNA ligase to obtain a pYTP301 plasmid.The pYTP301 plasmid is digested with the restriction endonuclease Kpn I, blunted the ends with T4DNA polymerase, and then said DNA fragment and the 5 end phosphorylated product obtained by treatment of the translation termination unit with T4polynucleotide kinase were subjected to ligating cyclization with T4DNA ligase to obtain the pYTP301K plasmid.The pYTP300 plasmid is deposited at ATCC as Escherichia coli C 600 pYTP 300, and is attached with the deposition number of ATCC 39973.Of the plasmids as prepared above, the pYTP102K plasmid, the pYTPlO2P plasmid and the pYTP301K plasmid code only methionine coded by ATG at the N terminus and only 1 to 2 amino acids coded by the translation termination unit at the C terminus other than the polypeptide derived from the poliovirus Sabin type, while the pYTP201N plasmid codes methionine coded by ATG and one or two amino acids for coding the restriction endonuclease cleavage site at the N terminus and only one amino acid coded by the translation termination unit at the C terminus other than the polypeptide derived from the poliovirus Sabin type.In the above operations, for digestion with respective restriction endonucleases, ligation with T4DNA ligase, screening with the use of ampicillin resistance and amplification of respective plasmids, all conventional methods can be used. For purification by separation of respective DNA fragments and plasmids, conventional techniques such as extraction with phenol, precipitation with ethanol agarose gel electrophoresis and subsequent extraction techniques, separation by liquid chromatogrpahy, etc. can be utilized. By transformation of a bacterium of the genus Escherichia with the use of the cyclic double stranded DNA of the present invention as prepared above, a microorganism capable of producing a protein having the antigenic determinant of the capsid protein VPl hereinafter the fused type is called VP1 fused protein , and the nonfused type VP1 non fused protein can be obtained. As the bacterium of the genus Escherichia, it is preferable to use Escherichia coli.Typical microorganism strains capable of producing VP1 fused protein as obtained above are deposited at ATCC, and attached with the following deposition numbers. Deposition No. Escherichia coli C 600 pHTP31 ATCC 39976 Escherichia coli C 600 pYTPll ATCC 39972 Escherichin coli C 600 pYTP100 ATCC 39975 Escherichia coli C 600 pYTP200 ATCC 39977 Escherichia coli C 600 pYTP300 ATCC 39973 Escherichia coli C 600 pSLMKl ATCC 39974Typical microorganism strains capable of producing VP1 non fused proteins are as follows Escherichia coli C 600 pHTP102P Escherichia coli C 600 pYTP102K Escherichia coli C 600 pYTP201N Escherichia coli C 600 pHTP301KThese microorganisms were applied to be deposited atFERM, but rejected for the reason of corresponding to P2 level, but they were deposited at Deutsche Sammlung vonMikroorganismen DSM on September 12, 1985, and attached with the following deposition numbers. Deposition No. Escherichia coli C 600 pHTP102P DSM 3470 Escherichia coli C 600 pYTP102K DSM 3469 Escherichia coli C 600 pYTP201N DSM 3472 Escherichia coli C 600 pHTP301K DSM 3471The present invention also provides a process for preparing a poliovirus vaccine, which compriese culturing the above transformed strain in a nutrient medium and harvesting the fused protein or non fused protein accumulated in the microorganism cells.As the nutrient medium to be used in the present invention, conventional media such as LB medium, LB agar medium, SB medium, etc. may be used. These media may be added optionally with a chemical reagent such as ampicillin. In the process of the present invention, the transformed strain can be cultured according to the conventional method such as culturing on a solid medium, liquid aeration cultivation, etc. The culturing conditions such as culture temperature, liquid properties of the medium may be the same as those for, for example, Escherichia coli C600, Escherichia coli W3110, Escherichia coliRB791, Escherichia coli RR1, etc. The cultivation time may be generally from several hours to one day.The transformant to be used in the process of the present invention is inactivated in the cIts protein which is the repressor of the high expression promoter operator, such as the Pt promoter operator, by making the cultivation temperature from an ordinary temperature of 30 OC to about 42 OC in the course of or after growth, whereby said promoter operator can be activated and the gene portion coding for the VP1 fused protain or the VP1 non fused protein under its control can be potently transcribed and expressed.The grown microorganisms are collected in a conventional manner, and the cells are destroyed by sonication or the conventional cell treating method generally employed with enzymes or surfactants, optionally extracted with an aqueous solution containing a highly concentrated modifier such as urea or guanidine chloride, followed by restoration of the activity by dilution, etc., to give the desired VP1 fused protein or VP1 non fused protein.Also, by dialysing this extract against Tris hydrochloride buffer hereinafter called Tris HCl or phosphate buffer, the desired product can be obtained as a soluble protein. Brief description of the drawings Fig. 1, Fig. 2, Fig. 3, Fig. 4, Fig. 5, Fig. 6, Fig. 7 and Fig. 8 are diagrams illustrating preparation examples of the cyclic double stranded DNA of the present invention.Fig. 9 is a diagram illustrating a preparation example of the pHT3 plasmid. Best modes for practicing the invention Referring now to the Examples and Preparation examples, the present invention is described in more detail, but the present invention is not limited thereto within the spirit of the invention.The buffers and the media employed in the following description had the compositions as shown below.TE buffer lOmM Tris HCl lmM EDTA pH 7.6 or 8.0.Stopping solution 1 Sodium dodecyl sulfate, 0.1 Bromophenol blue, 100 mM Disodium ethylenediaminetetraacetate herein after called EDTA , and 50 Glycerine are contained produced by BethesdaResearch Laboratories Incorproated .LB amp medium NaCl 5g Peptone Bactotryptone, trade mark 5 g Yeast extract 5g Ampicillin 25 mg Water 1 literLB amp agar medium In addition to the respective components of the aboveLB amp medium, 15 g of agar is contained.For screening medium, LB amp agar medium is used and LB amp medium is used for the culture broth for amplification.Preparation example I Preparation of pHT 1 7 pg of the PCQV2 plasmid was incubated with 50 U of ClaI in 50 p1 of a reaction medium 6 mM Tris HCl pH 7.9, 6 mM MgC12, 50 mM NaCl at 37 OC for 2 hours. After phenol extraction and ethanol precipitation, incubation was carried out with 10 U of S1 nuclease in 50 ul of a reaction medium 0.2 M NaCl, 50 mM sodium acetate pH 4.5, 1 mM ZnSO4, 5 glycerine at 37 OC for 2 minutes.After the incubation, phenol extraction and ethanol precipitation were conducted, followed by incubation with the use of 2.5 U of T4DNA ligase in 50 ul of a reaction medium 66 mM Tris HCl pH 7.6, 6.6 mM MgC12, 1 mM ATP, 5 mM DTT at 15 OC for 17 hours. After the incubation, incubation was further carried out with addition of 6 p1 of 50 mMNaCl and 20 U of Cla I at 37 C for 2 hours, followed further by heat treatment at 70 C for 5 minutes for inactivation of the enzyme. After left to cool, theEscherichia coli HB 101 strain was transformed by use of this solution, the colony having the pHT 1 plasmid was screened with ampicillin resistance as the marker, and the plasmid was isolated following a conventional means. II Isolation of the cIts gene 10 ug of the pHT 1 plasmid was incubated with 100 U ofBam HI in 50 ul of a reaction medium 10 mM Tris HCl pH 8.0, 7 mM MgC12, 0.1 M NaCl, 2 mM ss mercaptoethanol at 30 C for 3 hours. After the incubation, the mixture was subjected to heat treatment at 70 C for 5 minutes, left. to cool and then incubated with addition of 10 ul 100 U of Eco RI, 10 l of a buffer 1M Tris HCl pH 7.5, 70 mMMgC12, 500 mM NaCl, 90 mM ss mercaptoethanol and 25 ul of water to a total amount of 100 l at 37 C for 2 hours,After the incubation, the mixture was heated at 70 C for 5 minutes and left to cool. To 30 l of this solution was added 5 l of a stopping solution, and electrophoresis was conducted 5,0 mA, 30 minutes on 0.7 agarose gel for separation. The fragment of 400 base pairs containing the cIts gene was excised and, after purification, dissolved ill 20 l of water. III Preparation of pHT 2 8 g of the pKC 30 plasmid was incubated with 80 U of BglII in 60 l of a reaction medium 10 mM Tris HCl pH 7.5, 7 mM MgC12, 60 mM t aCl, 7 mM ss mercaptoethanol at 37 C for 90 minutes. After the incubation, phenol extraction, ethanol precipitation were conducted and the precipitate was dissolved in 1. l of water. To the solution were added 10 l of a buffer 1M Tris HCl pH 7.5, 70 mM MgC12, 500 mM NaCl, 90 mM ss .mercaptoethanol , 15 ul 80 U ofEco RI and water to make up a total volume of 100 p1, and incubation was carried out at 37 for 90 minutes. After the incubation, phenol extraction, ethanol precipitation were conduct ed and the precipitate was dissolved in 20 l of water. To 1 l of this solution were added 5 l of the solution prepared in II , 2 l of a buffer 660 mM Tris HCl i 7.6, 66 mM MgC12, 50 mM DTT, 10 mM ATP , 7 l of water and 1 l 2.8 U of T4DNA ligase, followed by incubation at 15 C for 16 hours. After the reaction, 90 l of water was added to 10 l of this solution and the Escherichia coli HB 101 strain was transformed similarly as in the case of the pHTl plasmid.The colony having the pHT 2 plasmid was screened with arnpicillin resistance as the marker, and the plasmid was isolated. IV Preparation of metapyrocatechase gene 33 g of the pSLMK 1 plasmid was incubated with Eco RI 180 U in 50 Fil of a reaction medium 0.1 M Tris HCl pH 7.5, 7 mM MgC12, 50 mM NaCl, 9 mM B mercaptoethanol at 37 C for 3 hours. After the incubation, the reaction mixture was heated at 70 0C for 5 minutes. After left to cool, the reaction was added with 8 p of a stopping solution and subjected to electropboresis 150 mA, 2.5 hours on 0.7 agarose gel for separation. The DNA fragment of 1200 base pairs containing the metapyrocatechase gene was excised, purified iin a conventional manner and dissolved in 20 p1 of water.The fragment was blunted the ends by incubation wittlh 5 U of the DNA polymerase I Klenow fragment in 50 l of a reaction medium 50 mM Tris HCl pH 7.2, O InM MgCl2, 0.1 mM DTT, 0.04 mM dNTP containing 18 l of this solution at 25 C for 30 minutes. After the incubation, phenol extraction and chloroform extraction were conducted, followed by ethanol precipitation, and the precipitate was dissolved in 10 l of water. V Preparation of pHT 3 5.4 g of the plasmid pHT 2 obtained in III was incubated with 60 U of Hpa I in 50 ul o f a reaction medium 10 mM Tris HCl pH 7.5, 7 mM MgCl2, 0,1 M KC1, 7 mM ss mercaptoethanol at 37 C for 3 hours. After the incubation, phenol extraction and ethanol precipitation were conducted, and the precipitate was dissolved in 20 ul of water. To 15 l of this solution were added 8 ul of theDNA fragment solution obtained in IV , 5 ul of a buffer 660 mM Tris HC1 pH 7.6, 66 mM MgCl2, 50 mM DTT, 10 mMATP , 7 ul of water and 15 ul 42 U of T4DNA ligase, followed by incubation at 12 C for 18 hours.Then, after phenol extraction and ethanol precipitation, the precipitate was dissolved in 20 ul of water. To this solution were added 3 l of a buffer 100 mM Tris HCl pH 7.5, 70 mM MgC12, 1 M KCl, 70 mM 13 mercaptoethanol , 9 p1 of water and 8 ul 48 U of Hpa I, and incubation was carried out at 37 C for 3 hours, followed by heat treatment at 70 tC for 5 minutes.After left to cool, with the use of this solution, the Escherichia coli HB 101 strain was transformed similarly as in the case of the pHT 1 plasmid, and the microorganism strain having the pHT plasmid was screened with ampicillin resistance and the fact that the cells are tinted in yellow by spraying of 5 aqueous catechol solution after elevation of the cultivation temperature from 30 C to 42 C as markers. Further, the microorganism cells were cultured in a culture broth for amplification according to conventional means and purified to obtain 2 mg of the pHT 3 plasmid.The pHT 3 plasmid obtained as described above was introduced into various species of bacteria of the genusEscherichia, deposited at FERM on August 17, 1984, and attached with the deposition numbers as shown below Escherichia coli HBl0l pHT3 FERM P 7776 Escherichia coli C600 pHT3 FERM P 7777 Escherichia coli W3110 pHT3 FERM P 7778 VI Preparation of pHT311 5 g of the pHT 3 plasmid obtained in V and 15 U of BamHI were incuabed in 20 l of a reaction medium 10 mMTris HCl pH 8.0, 7 mM MgC12, 100 mM NaCl, 2 mM ss mer captoethanol for 3 hours.After ethanol precipitation, the precipitate was dissolved in 20 p1 of a reaction medium 50 mM Tris HCl pH 7.2, 10 mM MgC12, 0.1 mM DTT, 80 vM dNTP and incubated with addition of 1 U of Klenow fragment at 22 C for 0.5 hours. After deproteinization with phenol, ethanol precipitation was effected.Further, the precipitate was dissolved in 20 l of a reaction medium 10 mM Tris HCl pH 7.5, 7 mM MgC12, 60 mMNaCl, 7 mM S mercaptoethanol and incubated with 20 U ofPvu II at 37 C for 3 hours. After ethanol precipitation, the precipitate was dissolved in 20 l of a reaction medium 66 mM Tris HCl pH 7.6, 6.6 mM MgCl2, 10 mM dithiothreitol, 1 mM ATP and incubated with addition of 2.8 U of T4DNA ligase at 15 C for 20 hours. By use of this reaction product, Echerichia coliHB 101 strain was transformed.From among the ampicillin resistant transformants, the microorganism having the pHT 3i plasmid was isolated, Further, by use of a conventional method, the pHT31 plasmid was obtained. 5 g of the pHT plasmid was incubated together with 15 U of Sal I in 20 l of a reaction medium lOmM Tris HCl pH 7.5, 7mM MgCl2 150mM NaCl, 0.2mM EDTA, 7mM B mercaptoethanol at 37 C for 3 hours. After ethanol precipitation, the precipitate was dissolved in 20 p1 of a Klenow fragment reaction solution and incubated together with 1 U of Klenow fragment at 22 C for 0.5 hours. After ethanol precipitation, the precipitate was dissolved in 20 ul of a Pvu II reaction solution and incubated with 20 U of Pvu II at 37 C for 3 hours.After ethanol precipitation, the precipitate was dissolved in 20 1 of a T4DNA ligase reaction solution and incubated together with 2.8 U of T4DNA ligase at 15 C for 20 hours. By use of this reaction product,Escherichia coli HB 101 strain was transformed. From among the ampicillin resistant transformants, the pHT311 plasmid as shown in Fig. 1 was isolated. This plasmid has one Sal I site, and a fused gene with the metapyrocatechase gene can be prepared by inserting an extraneous gene here.Example 1 Preparation of pHTP31After 20 g of the pBB2 plasmid Nomoto, Akio Nomoto,A. , et al. Proceeding National Academic Science USA Proc. Natl. Acad. Sci. USA , 79, 5793 1982 was incubated with 20U of Taq I in 20 p1 of a buffer 10 mMTris HC1 pH 7.5, 7 mM MgC12, 100 mM NaCl, 7 mM B mercaptoethanol at 60 C for 2 hours, 4 polyacrylamide electrophoresis was carried out. The DNA fragment of about 700 base pairs was recovered following the method of Maxam and Gilbert Proceeding National AcademicScience USA Proc. Natl. Acad. Sci , 74, 560 1977 .This was incubated with 2.5 U of T4DNA polymerase in 20 l of a buffer 67 mM Tris HCl pH 8.8, 6.7 mM MgC12, 10 mM B mercaptoethanol, 6.7 M EDTA, 16.6 mM NH4 25O4, 330 pM dCTP, dATP, dGTP, dTTP at 37 C for 15 minutes. After phenol treatment, ethanol precipitation was effected, and the precipitate was dissolved in 10 l of 10 mM Tris HCl pH 7.5, 1 mM EDTA solution. This solution contained 0.08 g of DNA per 1 p1. This DNA fragment is of the 713 base pairs 2373 3085 containing the DNA region coding for the capsid protein VP1, of the cDNA obtained from the poliovirus Sabin 1 strain.On the other hand, 10 g of the pHT311 plasmid was incubated with 10 U of Sal I in 20 iji of a buffer 10 mMTxis HCl pH 7.5, 7 mM MgC12, 150 mM NaCl, 0.2 mM EDTA, 7 mM S mercaptoethanol at 37 C for 2 hours, After ethanol precipitation, the precipitate was incubated with 2.5 U of T4DNA polymerase in 20 l of a buffer 67 mMTris HC1 pH 8.8, 6.7 mM MgC12, 10 mM 13 mercaptoethanol, 6.7 mM EDTA, 16.6 mM NH4 2so4 330 M dCTP, dATP, dGTP, dTTP at 37 0C for 15 minutes. After phenol treatment, ethanol precipitation was conducted.The precipitate was incubated with 1 U of alkaline phosphatase bovine intestine in 20 l of a buffer 50 mM Tris HCl pH 9.0, 1 mMMgC12, 0.1 raM ZnC12, 1 mM spermidine at 37 C for 30 minutes. After phenol treatment, ethanol precipitation was conducted. The precipitate was dissolved in 20 p1 of 10 mM Tris HC1 pH 7.5, 1 mM EDTA solution. The solution contained 0.4 g of DNA per 1 1. 1 g of this DNA together with 0.4 g of the DNA fragment containing the DNA region coding for the poliovirus Sabin type 1 VP1 as described above were incubated with 2.8 U of T4DNA ligase in 20 p1 of a buffer 66 mM Tris HCl pH 7.6, 6.6 mM MgC12, 10 mM DTT, 1 mM ATP at 15 C for 15 hours. By use of this reaction product, Escherichia coliHB 101 strain was transformed, and the pHTP31 plasmid shown in Fig. 1 was isolated from among the ampicillin resistant strains.The cDNA of the poliovirus Sabin type 1 contained in this plasmid corresponds to the 713 base pairs from the 2373th base to the 3085th base shown in the literature Nomoto,Akio Nomoto, A. , et al. Proceeding National AcademicScience USA Proc. Natl. Acad. Sci. USA , 79, 5793 1982 .Example 2 Preparation of pYTP 11 15 g of the pBB2 plasmid was incubated with Kpn I in 20 p1 of a buffer 10 mM Tris HCl pH 7.5, 7 mM MgC12, 7 mM ss mercaptoethanol at 37 C for 3 hours, and then subjected to 1.2 agarose electrophoresis. The DNA fragment of about 600 base pairs was recovered following the method of Dretzen et al Analytical Biochemistry Anal. Biochem. , 112, 295 1981 . This was dissolved in 20 l of 10 mM Tris HCl pH 7.5, 1 mM EDTA solution.This solution contained 0.03 g of DNA per 1 p1. On the other hand, 5 g of the pHTP31 plasmid was incubated with 10 U of Kpn I in 20 l of a buffer 10 mMTris HCl pH 7.5, 7 mM MgC12, 7 mM 13 mercaptoethanol . After ethanol precipitation, the precipitate was incubated with 1 U of alkaline phosphatase bovine intestine in 20 p1 of a buffer 50 mM Tris HCl pH 9.0, 1 mM MgC12, 0.1 mM ZnC121 1 mM spermidine at 37 C for 30 minutes.After phenol treatment, ethanol precipitation was conducted. The precipitate was dissolved in 20 l of 10 mMTris HCl pH 7.5, 1 mM EDTA solution. This solution contained 0.25 g of DNA per 1 Li. 1 g of this DNA and 0.3 g of the DNA as described above were incubated with 2.8 U of T4DNA ligase in 20 l of a buffer 66 mM TrisHC1 pH 7.6, 6.6 mM MgC12, 10 mM DTT, 0.4 mM ATP . By use of this reaction mixture, Escherichia coli HB 101 strain was transformed, and the pYTPll plasmid shown in Fig. 2 was isolated from among the ampicillin resistant transformants.The cDNA of the poliovirus Sabin type 1 contained in this plasmid corresponds to the 1291 base pairs from the 2393th base to the 3683th base shown in the literature supra.Example 3 Preparation of pYTP100 15 g of the pBB2 plasmid was incubated with 30 U of BamHI in 20 l of a reaction medium 10 mM Tris HCl pH 8.0, 7 mM MgC12, 100 mM NaCl, 2 mM ss mercaptoethanol at 30 C for 3 hours, followed by ethanol precipitation. The precipitate was dissolved in 20 l of a reaction medium 10 mM Tris HCl pH 8.0, 7 mM MgC12, 60 mM NaCl, 7mM ss mercaptoethaol and incubated with 30 U of Hinc II at 37 C for 3 hours. After ethanol precipitation, the precipitate was dissolved in 20 21 of a T4DNA polymerase reaction solution and incubated with 2.5 U of T4DNA polymerase at 37 C for 15 minutes.After deproteinization with phenol, the product was subjected to 1.2 agarose gel electrophoresis, and a DNA fragment of about 1670 base pairs was recovered from the gel. This was dissolved in 20 l of 10 mM Tris HCl pH 7.5, 1 mM EDTA solution. This solution contained 0.07 vg of DNA per 1 l.0.5 ug of this DNA and 1 pg of the DNA fragment obtained from the pHT311 plasmid were incubated with 2.8 U ofT4DNA ligase in 20 1ll of a T4DNA reaction solution at 15 C for 15 hours. Escherichia coli HB 101 strain was transformed with this reaction product, and the pYTP100 plasmid as shown in Fig. 2 was isolated from among the ampicillin resistant transformants.The cDNA of the poliovirus Sabin type 1 contained in this plasmid corresponds to the 1666 base pairs from the 2150th base to the 3915th base shown in the literature supra.Example 4 Preparation of pYTP200 15 g of the pVS 2 2503 plasmid Toyoda, Haruka Toyoda,H. , et al. Journal of Molecular Biology J. Mol.Biol. , 174, 561 1984 was incubated with 30 U of Kpn I in 20 l of a buffer 6 mM Tris HCl pH 7.5, 6 mM NaC1, 6 mM Mgcl2, 6 mM ss mercaptoethanol at 37 C for 3 hours. After ethanol precipitation, the precipitate was incubated with 32 U of Pvu II in 20 l of a buffer 10 mMTris HC1 pH 7.5, 7 mM MgC12, 60 mM NaCl, 7 mM ss mercapto ethanol at 37 C for 3 hours. This was subjected to 1.2 agarose gel electrophoresis and a DNA fragment of about 1160 base pairs was recovered from the gel. This was incubated with 2.5 U of T4DNA polymerase in 20 1 of aT4DNA polymerase reaction solution 67 mM Tris HCl pH 8.8, 6.7 mM MgC12, 10 mM 13 mercaptoethanol, 6.7 mM EDTA, 16.6 mM NH4 2S04, 330 M dCTP, dATP, dGTP, dTTP at 37 C for 15 minutes. After phenol treatment, ethanol precipitation was conducted, and the precipitate was dissolved in 10 l of 10 mM Tris HCl pH 7.5, 1 mM EDTA solution.This solution contained 0.05 g of DNA per 1 p1. 0.5 g of this DNA and 1 ug of the DNA fragment obtained from the pHT311 plasmid as described in Example 1 were incubated with 2.5 U of T4DNA ligase in 20 1 of a buffer 66 mM Tris HCl pH 7.6, 6.6 mM MgC12, 10 mM DTT, 1 mM ATP at 15 C for 5 hours. Escherichia coli HB 101 strain was transformed with this reaction product and the pYTP plasmid shown in Fig. 3 was isolated from among the ampicillin resistant transformantsThe cDNA of the poliovirus Sabin type 2 contained in this plasmid corresponds to the 1157 base pairs from the 2286th base to the 3442th base as shown in literature supra. Example 5 Preparation of pYTP300 15 g of the pVS 3 2603 plasmid Toyoda, Haruka Toyoda,H. , et al. J. Mol. Biol., 174, 561 1984 was incubated with 30 U of SphI in 20 l of a buffer 6 mM TrisHC1 pH 7.7, 125 mM NaCl, 6 mM MgC12, 7 mM 6 mercapto ethanol, 0.01 Triton X 100 at 37 C for 3 hours.After ethanol precipitation, the precipitate was incu bated with 30U of Bgl II at 37 C in 20 l of a buffer 10 mM Tris HCl pH 7.5, 7 mM MgC12, 60 mM NaCl, 7mM ss mercaptoethanol at 37 C for 3 hours, and then subjected to 1.2 agarose gel electrophoresis. A DNA fragment of about 1040 base pairs was recovered from the gel. This was incubated with 2.5 U of T4DNA polymerase in a buffer 67 mM Tris HCl pH 8.8, 6.7 mM MgC12, 10 mM 13 mercapto ethanol, 6.7 mM EDTA, 16.6 mM NH4 2So4, 330 M dCTP, dATP, dGTP, dTTP at 37 C for 15 minutes. After phenol treatment, ethanol precipitation was conducted, and the precipitate was dissolved in 20 p1 of 10 mM Tris HCl pH 75, 1 mM EDTA solution.This solution contained 0.05 vg of DNA per 1 p1. 0.5 g of this DNA and 1 g of the DNA fragment obtained from the pHT311 plasmid as described inExample 1 were incubated with 2.5 U of T4DNA ligase in 20 p1 of a buffer 66 mM Tris HCl pH 7.6, 6.6 mM MgC12, 10 mM DTT, 1 mM ATP at 15 C for 15 hours. Escherichia coli HB 101 strain was transformed with this reaction product and the pYTP 300 1 plasmid shown in Fig. 3 was isolated from among the ampicillin resistant transformants.Next, 5 g of the pYTP 300 1 plasmid was incubated with 10 U of Sal I in 20 l of a buffer 6 mM Tris HCl pH 7.9, 150 mM NaCl, 6 mM MgC12, 6 mM B mercaptoethanol at 37 C for 3 hours. After ethanol precipitation, the precipitate was incubated with 50U of S1 nuclease in 20 ul of a buffer 50 mM CH3COONa, 200 mM NaCl, 1 mM ZnSO4, 0.5 glycerol at 37 C for 30 minutes. After phenol treatment, ethanol precipitation was conducted and the precipitate was incubated with 2.5 U of T4DNA ligase in 20 l of a buffer 66 mM Tris HCl pH 7.6, 6.6 mM MgCl2, 10 mMDTT, 1 mM ATP at 15 C for 15 hours.Escherichia coli HB 101 strain was transformed with this reaction product and the pYTP 300 plasmid shown in Fig. 3 was isolated from among the ampicillin resistant transformants.The cDNA of the poliovirus Sabin type 3 contained in this plasmid corresponds to the 1041 base pairs from the 2416th base to the 3456th base as shown in literature supra.Example 6 Preparation of pYTP102 5 pg of the pYTU3 plasmid was incubated with 5 U of Sal I in 20 p1 of a buffer 10 mM Tris HCl pH 7.5, 7 mM MgC12, 150 mM NaCl, 0.2 mM EDTA, 7 mM ss mercaptoethanol at 37 C for 3 hours. After ethanol precipitation, the precipitate was dissolved in 20 l of 10 mM Tris HC1 pH 7.5, 1 mM EDTA solution. On the other hand, of the cDNA of the poliovirus Sabin type 1, 15 vg of the pHTP31 plasmid containing the DNA region coding for the VP1 protein was incubated with 20 U of Sal I in 20 l of a buffer 10 mM Tris HCl pH 7.5, 7 mM MgC12, 150 mM NaCl, 0.2 mM EDTA, 7 mM ss mercaptoethanol at 37 C for 3 hours.After the incubation, 1.2 agrose electrophoresis was carried out, and a DNA fragment of 713 base pairs was extracted according to the conventional method. After ethanol precipitation, the precipitate was dissolved in 20 l of 10 mM Tris HCl pH 7.5, 1 mM EDTA solution. This DNA fragment corresponds to the 713 base pairs from the 2373th base to the 3085th base shown in the literature Nomoto, Akio Nomoto, A. , et al. Proceeding NationalAcademic Science USA Proc. Natl. Acad. Sci. USA , 79, 5793 1982 .0.5 ijg of this DNA fragment and 0.2 g of the DNA obtained by cleavage of the pYTU3 plasmid with Sal I were incubated with 2.5 U of T4DNA ligase in 20 l of a buffer 66 mM Tris HCl pH 7.6, 6.6 mM MgC12, 10 mM DTT, 1 mMATP at 15 C for 15 hours. With the use of this reaction mixture, Escherichia coliHB101 strain was transformed, and the pYTP 102 plasmid shown in Fig. 4 was isolated from among the ampicillin resistant microorganisms. Next, following a conventional method, Escherichia coli 600 strain was transformed with the pYTP 102 plasmid, and the Escherichia coli 600 pYTP 102 was isolated.Example 7 Preparation of pYTP102K 1 g of the pYTP102 plasmid was incubated with 5 U of KpnI in 20 l of a buffer 10 mM Tris HCl pH 7.5, 7 mM MgCl2, 7 mM ss mercaptoethanol at 37 C for 2 hours.After ethanol precipitation, the precipitate was incubated with 2.8 U of T4DNA polymerase in 20 l of a buffer 67 mM Tris HCl pH 8.8, 6.7 mM MgC12, 10 mM ss mercapto ethanol, 6.7 mM EDTA, 16.6 mM NH4 2SO4, 330 pM dCTP, dATP, dGTP, dTTP at 37 C for 15 minutes. After phenol treatment, ethanol precipitation was conducted, and the precipitate and 0.5 g of a commercially availableUniversal Translation Terminator Oligonucleotide sold byPharmacia P L Biochemical phosphorylated at 5 end by treatment with T4 polynucleotide kinase were incubated with 2.5 U of T4DNA ligase in 20 1 of a buffer 66 mMTris HCl pH 7.6, 6.6 mM MgC12, 10 mM DTT, 1 mM ATP at 15 C for 15 hours.With the use of this reaction mixture,Escherichia coli HB101 strain was transformed, and the pYTP 102K plasmid shown in Fig. 4 was isolated from among the ampicillin resistant microorganisms. Next, following a conventional method, Escherichia coli C600 strain was transformed with the pYTP 102K plasmid, and the Escherichia coli 600 pYTP 102K was isolated.Example 8 Preparation of pYTP102P 10 g of the pYTP plasmid was incubated with 30 U of PstI in 20 p1 of a buffer 10 mM Tris HC1 pH 7.5, 7 mM MgC12, 50 mM NH4 2S04 at 37 C for 3 hours. After ethanol precipitation, the precipitate was incubated with 2.8 U of T4DNA polymerase in 20 p1 of a buffer 67 mMTris HC1 pH 8.8, 6.7 mM MgC12, 10 mM 13 mercaptoethanol, 6.7 mM EDTA, 16.6 mM NH4 2SO4 330 vM dCTP, dATp, dGTP, dTTP at 37 C for 15 minutes.After phenol treatment, ethanol precipitation was conducted, and the precipitate and 0.5 g of a commercially available Universal Translation Terminator Oligonucleotide sold by Pharmacia P LBiochemical phosphorylated at the 5 end by treatment with T4polynucleotide kinase were incubated with 2.5 U ofT4DNA ligase in 20 l of a buffer 66 mM Tris HCl pH 7.6, 6.6 mM MgC12, 10 mM DTT, 1 mM ATP at 15 C for 15 hours.After ethanol precipitation, the precipitate was incubated with 30 U of Sal I in 20 l of a buffer 10 mMTris HCl pH 7.5, 7 mM MgC12, 150 mM NaCl, 0.2 mM EDTA, 7 mM ss mercaptoethanol at 37 C for 3 hours, followed by 1.2 agarose gel electrophoresis. A DNA fragment of about 1050 base pairs was recovered following a conventional method. On the other hand, 5 g of the pYTU3 plasmid was incubated with 16 U of Nru I in 20 l of a buffer 6 mM TrisHC1 pH 7.4, 6 mM MgC12, 100 mM NaC1, 5 mM ss mercapto ethanol at 37 C for 3 hours. After ethanol precipitation, the precipitate was incubated with 20 U of Sal I in 20 l of a buffer 10 mM Tris HCl pH 7.5, 7 mM MgC12, 150 mM NaC1, 0.2 mM EDTA, 9 mM S mercaptoethanol at 37 C for 3 hours. By carrying out 1.2 agarose gel electrophoresis, a DNA fragment of about 4000 base pairs was recovered.0.5 g of the above DNR fragment of about 1050 base pairs and 1.0 g of the DNA fragment of about 4000 base pairs were incubated with 2.5 U of T4DNA ligase in 20 l of a buffer 66 mM Tris HCl pH 7.6, 6.6 mM MgC12, 10 mM DTT, 1 mM ATP at 15 C for 15 hours. With the use of this reaction mixture, Escherichia coli HB101 strain was transformed, and the pYTP 102P plasmid shown in Fig. 5 was isolated from among the ampicillin resistant microorganisms. Next, following a conventional method,Escherichia coli C600 strain was transformed with the pYTP 102P plasmid, and the Escherichia coli 600 pYTP102P was isolated.Example 9 Preparation of pYTP201N 10 g of the pYTP200 plasmid was incubated with 30 U ofSal I in 20 p1 of a buffer 10 mM Tris HCl pH 7.5, 7 mMMgC12, 150 mM NaCl, 0.2 mM EDTA, 7 mM ss mercaptoethanol at 37 C for 3 hours. By carrying out 1.2 agarose gel electrophoresis, a DNA fragment of about 1160 base pairs was recovered.On the other hand, 5 g of the pYTU3 plasmid was incubated with 20 U of Sal I in 20 l of a buffer 10 mMTris HCl pH 7.5, 7 mM MgC12, 150 mM NaCl, 0.2 mM EDTA, 7 mM ss mercaptoethanol at 37 C for 3 hours. By carrying out 1.2 agarose gel electrophoresis, a DNA fragment of about 4500 base pairs was recovered.0.5 g of the above DNR fragment of about 1160 base pairs and 1.0 yg of the DNA fragment of about 4500 base pairs were incubated with 2.5 U of T4DNA ligase in 20 l of a buffer 66 mM Tris HC1 pH 7.6, 6.6 mM MgC12, 10 mM DTT, 1 mM ATP at 15 C for 15 hours. With the use of this reaction mixture, Escherichia coli HB101 strain was transformed, and the pYTP 201 plasmid shown in Fig. 6 was isolated from among the ampicillin resistant microorganisms.2 ijg of the pYTP201 plasmid was incubated with 10 U ofNar I in 20 l of a buffer 6 mM Tris HC1 pH 7.4, 6 mM MgC12, 50 mM NaCl, 6 mM B mercaptoethanol at 37 C for 3 hours. After ethanol precipitation, the precipitate was incubated with 2.8 U of T4DNA polymerase in 20 p1 of a buffer 67 mM Tris HC1 pH 8.8, 6.7 mM MgC12, 10 mM B mercaptoethanol, 6.7 mM EDTA, 16.6 mM NH4 2S04, 330 M dCTP, dATP, dGTP, dTTP at 37 C for 15 minutes, After phenol treatment, ethanol precipitation was conducted, and the precipitate and 0.5 g of a commercially available Universal Translation Terminator Oligonucleotide sold by Pharmacia P L Biochemical phosphorylated at 5 end by treatment with T4polynucleotide kinase were incubated with 2.5 U of T4DNA ligase in 20 ul of a buffer 66 mM Tris HCl pH 7.6, 6.6 mM MgC12, 10 mM DTT, 1 mM ATP at 15 C for 15 hours. With the use of this reaction mixture, Escherichia coli HB101 strain was transformed, and the pYTP 201N plasmid shown in Fig. 6 was isolated from among the ampicillin resistant microorganisms. Next, following a conventional method, Escherichia coli C600 strain was transformed with the pYTP 201N plasmid, and the Escherichia coli 600 pYTP 201N was isolated.Example 10 Preparation of pYT301K 10 g of the pYTP300 plasmid was incubated with 20 U ofPst I in 20 l of a buffer 10 mM Tris HCl pH 7.5, 7 mM MgC12, 50 mM NH4 2S04 at 37 C for 3 hours. After ethanol precipitation, the precipitate was incubated with 2.8 U of T4DNA polymerase in 20 l of a buffer 67 mMTris HCl pH 8.8, 6.7 mM MgC12, 10 mM 13 mercaptoethanol, 6.7 mM EDTA, 16.6 mM No4 2504, 330 pM dCTP, dATP, dGTP, dTTP at 37 C for 15 minutes. After phenol treatment, ethanol precipitation was conducted, and the precipitate was incubated with 30 U of Bam HI in 20 ul of a buffer 10 mM Tris HCl pH 8.0, 7 mM MgC12, 100 mM NaC1, 2 mM ss mercaptoethanol at 30 OC for 3 hours.By carrying out 1.2 agarose gel electrophoresis, a DNA fragment of about 1500 base pairs was recovered.On the other hand, 5 g of the pYTU3 plasmid was incubated with 20 U of Sal I in 20 p1 of a buffer 10 mMTris HCl pH 7.5, 7 mM MgC12, 150 mM NaC1, 0.2 mM EDTA, 7 mM ss mercaptoethanol at 37 C for 3 hours. After ethanol precipitation, the precipitate was incubated with 10 U of Sl nuclease in 100 1 of a buffer 50 mM CH3COONa, 200 mM NaCl, 1 mM ZnSO4, 0.5 glycerol at 37 C for 15 minutes. After phenol treatment, ethanol precipitation was conducted and the precipitate was incubated with 30 U of Bam HI in 20 p1 of a buffer 10 mM Tris HCl pH 8.0, 7 mM MgC12, 100 mM NaCl, 2 mM ss mercaptoethanol at 30 C for 3 hours. By carrying out 1.2 agarose gel electrophoresis, a DNA fragment of about 3800 base pairs was recovered.0.5 g of the above DNR fragment of about 1500 base pairs and 1.0 g of the DNA fragment of about 3800 base pairs were incubated with 2.5 U of T4DNA ligase in 20 l of a buffer 66 mM Tris HC1 pH 7.6, 6.6 mM MgCl2, 10 mM DTT, 1 mM ATP at 15 C for 15 hours. With the use of this reaction mixture, Escherichia coli HB101 strain was transformed, and the pYTP 301 plasmid shown in Fig. 7 was isolated from among the ampicillin resistant microorganisms.2 ug of the pYTP201 plasmid was incubated with 5 U of KpnI in 20 l of a buffer 10 mM Tris HCl pH 7.5, 7 mM MgC12, 7 mM ss mercaptoethanol at 37 C for 3 hours.After ethanol precipitation, the precipitate was incubated with 2.8 U of T4DNA polymerase in 20 l of a buffer 67 mM Tris HCl pH 8.8, 6.7 mM MgC12, 10 mM ss mercapto ethanol, 6.7 mM EDTA, 16.6 mM NH4 2SO4, 330 ssM dCTP, dATP, dGTP, dTTP at 37 C for 15 minutes. After phenol treatment, ethanol precipitation was conducted, and the precipitate and 0.5 ug of a commercially availableUniversal Translation Terminator Oligonucleotide sold byPharmacia P L Biochemical phosphorylated at 5 end by treatment with T4polynucleotide kinase were incubated with 2.5 U of T4DNA ligase in 20 ul of a buffer 66 mMTris HCl pH 7.6, 6.6 mM MgC12, 10 mM DTT, 1 mM ATP at 15 0C for 15 hours.With the use of this reaction mixture,Escherichia coli HB101 strain was transformed, and the pYTP 301K plasmid shown in Fig. 7 was isolated from among the ampicillin resistant microorganisms. Next, following a conventional method, Escherichia coli C600 strain was transformed with the pYTP 301K plasmid, and theEscherichia coli 600 pYTP 301K was isolated.Example 11 Preparation of VP1 fused proteinEscherichia coli C600 strain retaining the pHTP31 plasmid was subjected to shaking cultivation in a test tube containing 5 ml of L medium ampicillin conc. 25 pg ml at 28 OC for an overnight. When OD650 of the culture broth became 0.8, it was immediately transferred into a thermostat bath at 42 C, wherein shaking cultivation was carried out. Then, 1 ml of the culture broth was transferred into an Eppendorf tube and centrifuged. The supernatant was removed and 1 ml of a solution of 10 mMTris HCl pH 7.5, 1 mM DTT, 23 vM p methylsulfonic acid fluoride hereinafter called PMSF was added to the precipitated microorganism cells, and the cells were crushed by sonication on ice. The crushed solution was separated and, after removal of the supernatant, 20 p1 of 6 M urea solution was added, followed by thorough stirring. After left to stand on ice bath for 30 minutes, the mixture was subjected to centrifugation.The supernatant was added with an equal amount of a sample buffer for electrophoresis finally 10 mM Tris HCl pH 8.0, 1 mM EDTA, 10 8 sucrose, 40 mM DTT, 1 SDS and subjected to SDS polyacrylamide gel electrophoresis hereinafter called SDS PAGE . The same operations were conducted for the Escherichia coli strain C600 strain retaining no pHTP31 plasmid. Next, Coomassie blue staining and enzyme antibody staining were performed, with the result that a band attributable to the VP1 fused protein with a molecular weight of about 29,000 was detected for the C600 pHTP31 strain. For the C600 strain having no plasmid, this band was not detected. Also, by performing enzyme antibody staining, this protein was confirmed to react with the antibody against Sabin type 1 virus.The content of the VP1 fused protein was measured by a densitometer to be 24.8 . This corresponds to 19.4 mg as calculated on the expressed amount per 1 liter of the culture broth.Example 12 Preparation of VP1 fused proteinAfter 1 ml of the culture broth of the transformed strainEscherichia coli C600 pHTP31 cultured under the same conditions as Example 11 was centrifuged, microorganisms cells were treated with lysozyme chloride egg white andTriton X100 Biochemical Experimental Course Vol. 5,Enzyme Study Method part 1 , ed. by Society of Biochemistry of Japan, Tokyo Kagaku Dojin to be crushed. The precipitate after centrifugation was added with 6M urea solution, stirred well and then left to stand on ice bath for 30 minutes, followed by centrifugation. An equal amount of sample buffer for electrophoresis was added to the supernatant, and the mixture was subjected to SDSPAGE. Then, enzyme antibody staining was performed.The content of the VP1 fused protein was measured by densitometry to be 14.5 . This corresponds to 10.2 mg as calculated on the expressed amount per 1 liter of the culture broth.Example 13 Preparation of VP1 fused proteinFor the transformed strain Escherichia coli C600 retaining the pYTPll plasmid or the pYTP100 plasmid, according to the same method as Example 11, the VP1 fused protein was extracted from the precipitate after sonication with 6 M urea solution and subjected to SDS PAGE.In C600 pYTPll, some proteins were detected with a protein of about 46,000 molecular weight being the longest.On the other hand, in C600 pYTP100, some proteins were detected with a protein of about 69,000 molecular weight being the longest. Further, enzyme antibody staining was performed to confirm that these proteins reacted with the antibody against Sabin type 1 virus.The contents of the VP1 fused proteins were measured by densitometry to be 28.9 and 15.5 , respectively.These correspond to 22.6 mg and 12.1 mg, respectively, as calculated on the expressed amounts per 1 liter of the culture broth.Example 14 Preparation of VP1 fused proteinFor the transformed strain Escherichia coli C600 retaining the pYTP200 plasmid or the pYTP300 plasmid, according to the same method as Example 11, the VP1 fused protein was extracted from the precipitate after sonication with 6 M urea solution and subjected to SDS PAGE.In C600 pYTP200, a protein of about 52,000 molecular weight was detected. On the other hand, in C600 pYTP300, some proteins were detected with a protein of about 40,000 molecular weight being the longest. Further, enzyme antibody staining was performed to confirm that these proteins reacted with the antibody against Sabin type 1 virus.The contents of the VP1 fused proteins were measured by densitometry to be 14.9 and 5.0 , respectively. These correspond to 11.7 mg and 3.9 mg, respectively, as calculated on the expressed amounts per 1 liter of the culture broth.Example 15 Preparation of VP1 fused proteinShaking cultivation of a transformant Escherichia coliC600 strain retaining the pHTP31 strain was carried out in a 2 liter culture flask containing one liter of L medium ampicillin conc. 25 vg ml at 30 OC, and the culture broth, on reaching an OD650 value of 0.8, was immediately transferred into a thermostat bath at 42 C, wherein shaking cultivation was continued for 3 hours.After a predetermined period of time, the microorganism cells were collected by centrifuge and subjected to sonication with addition of 100 ml of 10 mM Tris HCl pH 7.5, 1 mM DTT, 23 vM PMSF solution to crush the microorganism cells.After centrifugation, the precipitate was well stirred with addition of 10 ml of 6 M urea solution, and left to stand on ice bath for 30 minutes. Further, after centrifugation, the supernatant was dialyzed against 50 fold volume of 10 mM Tris HCl pH 7.5, 1 mM EDTA, 1 mM DTT solution, which was repeated for 4 times, followed bySDS PAGE.The content of the VP1 fused protein was measured by densitometry to be 28.1 t. This corresponds to 21.6 mg as calculated on the expressed amount per 1 liter of the culture broth.Example 16 Preparation of pSLMPKl 14 g of the pBB2 plasmid was incubated with 100 U of BamHI in 120 p1 of a reaction medium 10 mM Tris HCl pH 8.0, 7 mM MgC12, 0.1 M NaCl, 2 mM B mercaptoethanol at 30 C for 5 hours. The reaction mixture was subjected to 0.7 8 agarose gel electrophoresis for separation, and a DNA fragment of 2500 base pairs was extracted according to a conventional means and dissolved in 20 p1 of water. To 16 p1 of this solution, 5 1 of a buffer 100 mM Tris HCl pH 8.4, 1 M NaC1, 60 mM MgC12, 60 mM ss mercaptoethanol , 27 l of water and 2 u1 20 U of Taq I, and incubation was conducted at 65 C for 3.5 hours.The reaction mixture was subjected to 0.7 agarose gel electrophoresis for separation, and a DNA fragment of about 700 base pairs was extracted and dissolved in 8 p1 of water.After 5.5 g of the pSLMKl plasmid was incubated with. 20U of Sal I in 100 l of a reaction medium 6 mM Tris HCl pH 7.9, 150 mM NaCl, 6 mM MgC12, 6 mM ss mercaptoethanol at 37 C for 3 hours, the reaction mixture was subjected to 0.7 agarose gel electrophoresis for separation, aDNA fragment of about 3200 base pairs was extracted and dissolved in 50 ul of water. To 10 l of this solution were added 2.5 p1 of a buffer 500 mM Tris HC1 pH 7.2, 100 mM MgC12, 1 mM DTT , 1 l of 2 mM dNTP, 10.5 p1 of water and 1 l 5.2 U of DNA polymerase I Klenow fragment, followed by incubation at 22 C for 30 minutes. The reaction mixture was heated at 70 C for 5 minutes.After left to cool, 25 ul of the solution was incubated with addition of 10 p1 0.005 OD of the Cla I linker d CATCCATG phosphorylated at the 5 end, 1 ul of a buffer 660 mM Tris HCl pH 7.6, 66 mM MgC12, 50 mM DTT, 10 mM ATP and 1 1 2.8 U of T4DNA ligase at 22 OC for 6 hours. After the reaction, phenol extraction and ethanol precipitation were conducted, and then the precipitate was dissolved in 90 p1 of water. To this solution were added 10 l of a buffer 60 mM Tris HCl pH 7.9, 60 mM MgC12, 500 ruM NaCl and 4 p1 20 U of Cla I, and incubation was carried out at 37 C for 4 hours.After the reaction, the reaction was stopped with addition of 4 1 of 0.25 M EDTA and, after phenol extraction and ethanol precipitation, the precipitate was subjected to 0.7 8 agarose gel electrophoresis for separation, and a DNA fragment of 2500 base pairs was extracted and dissolved in 20 l of water. To 18 1 of this DNA fragment solution were added 5 l of a buffer 500 mM TrisHC1 pH 9.0, 10 mM MgC12, 1 mM ZnCl2, 10 mM spermidine and 1 U of alkaline phosphatase bovine intestine , and incubation was carried out at 37 C for 30 minutes.After the incubation, the reaction was stopped with addition of 1 p1 of 0.25 M EDTA and, after phenol extraction and ethanol precipitation, the precipitate was dissolved in 2 p1 of TE buffer pH 8.0 . To 2 of this solution were added 4 l of the Tag I fragment obtained from the pBB2 plasmid as desdribed above, 1 l of a buffer 660 mMTris HCl pH 7.6, 66 mM MgC12, 50 mM DTT, 10 mM ATP , 2 p1 of water and 1 l 2.8 U of T4DNA ligase, and incubation was carried out at 12 C for 12 OC. After the incubation, Escherichia coli RR1 strain was transformed with the use of this solution, the colony was screened with ampicillin resistance as the marker, and the pSLMPKl plasmid as shown in Fig. 8 was isolated according to a conventional means.Example 17 Preparation of VP1 fused proteinShaking cultivation of Escherichia coli C600 strain retaining the pSLMPK1 plasmid was carried out in a test tube containing 5 ml of L medium ampicillin conc. 25 vg ml at 37 C, and the culture broth, on reaching anOD650 value of 0.8, was immediately transferred in an amount of 1 ml into an Eppendorf tube and centrifuged.Subsequently, the same operations as Example 11 were performed.By SDS PAGE, a band attributable to the VP1 fused protein with a molecular weight of about 60,000 dalton was detected. Also, it was confirmed by enzyme antibody staining that said protein reacted with the antibody against Sabin type 1 virus.Next, the content of the VP1 fused protein was measured by densitometry to be 24.1 . This corresponds to 18.6 mg as calculated on the expressed amount per 1 liter of the culture broth.Example 18 Preparation of VP1 non fused proteinShaking cultivation of Escherichia coli C600 strain retaining the pYTP 102P plasmid was carried out in a test tube containing 1 ml of L medium ampicillin conc. 25 g ml at 28 C for an overnight. When the OD650 value of the culture broth became 0.8, it was immediately transferred into a thermostat bath at 42 OC and shaking cultivation was conducted for 3 hours. After centrifugation, the supernatant was removed, and the precipited microorganism cells were crushed by sonication at 2 OC with addition of 100 ml of 10 mM Tris HCl pH 7.5, 1 mM EDTA solution. The crushed solution was separated and, after removal of the supernatant, stirred well with addition of 10 ml of 6 M urea solution.After left to stand on icebath for 30 minutes, the mixture was centrifuged.Further, the supernatant was dialyzed against 100 fold volume of 10 mM Tris HCl pH 7.5, 1 mM EDTA solution for 3 times. To a portion of the supernatant was added an equal amount of a sample buffer for electrophoresis finally 10 mMTris HCl pH 8.0, 1 mM EDTA, 10 sucrose, 40 mM DTT, 1 SDS , and the mixture was subjected to SDS polyacrylamide gel electrophoresis. Then, Coomassie blue staining was performed and a band attributable to the VP1 protein with a molecular weight of about 39,000 dalton was detected.The content of the VP1 fused protein was measured by densitometry to be 20.3 . This corresponds to 9.6 mg as calculated on the expressed amount per 1 liter of the culture broth. Also, it was confirmed by enzyme antibody staining that this protein reacted with the antibody against Sabin type 1 virus.The same operations were conducted also for Escherichia coli C600 pYTP102K, C600 pYTP201N and C600 pYTP301K, andVP1 non fused proteins with molecular weights of 25,500, 29,000 and 25,500 were obtained, respectively. These were confirmed by enzyme antibody staining to be reactive with antibodies against Sabin type 1 virus, Sabin type 2 virus and Sabin type 3 virus, respectively. The contents of the VP1 proteins in the solutions extracted of the precipitates after sonication with 6 M urea solution and the expressed amounts per 1 liter of the culture broth are shown in the following Table. TableEMI70.1 tb SEP Content SEP Expressed SEP amount tb SEP SEP per SEP liter SEP of tb SEP troth SEP mg tb Escherichia SEP coli tb C600 pYTP102K SEP 3.5 SEP 1.7 tb Escherichia SEP col SEP tb C600 pYTP2OlN SEP 22.9 SEP 10.8 SEP tb Escherichia SEP colt SEP 1 SEP tb C600 pYTP301K SEP 4.0 SEP 1.9 tb Utilizability in industry According to the present invention, the protein having the antigenic determinant of the polyovirus capsid protein VP1 can be supplied in a large amount in the form of fused protein or non fused protein. The active protein thus obtained can be used as the reagent for detection of polyovirus antibody, for example, for radioimmunoassay.Also, by administering parenterally to mammals, antibody against the poliovirus capsid VP1 protein can be produced and therefore it is useful for polyovirus vaccine. Further, the side effect of live vaccine can be reduced when it is used for preimmunization for administration of live vaccine. Format 2 NOTICE OF REFUSAL TO RECEIVE DEPOSITION Notice No. 60 FERM Doc. No. 270 Date of notice March 9, 1985Esq. Foundation Sagami Chemical Research Center Kiyosi Kondo, Vice chairman, and other 5 as per attached sheet President of Institute of Fermen tation Research, Agency of Industrial Science Technology Jiro Oyama sealed We hereby notice that the microorganism applied by you to be deposited cannot be received for the reason shown below. Remarks 1. Date of application February 27, 1985 2. Representation for discrimination of microorganism Escherichia coli C600 pYTP102K 3. Reason for refusal Because of outside of the range of microorganisms which can be received. Attached sheetNames of depositors 1. Foundation Sagami Chemical Research Center Vice chairman Kiyosi Kondo 2. Central Glass Co., Ltd. President Mitsuyoshi Ito 3. Hodogaya Chemical Co., Ltd. President Kenichi Fujioka 4. Nippon Soda Co., Ltd. President Takeo Sannomiya 5. Nissan Chemical Industries, Ltd. Representative Director Misao Kusano 6. Toyo Soda Manufacturing Co., Ltd. President Toshiaki Yamaguchi Format 2 NOTICE OF REFUSAL TO RECEIVE DEPOSITION Notice No. 60 FERM Doc. No. 271 Date of notice March 9, 1985Esq. Foundation Sagami Chemical Research Center Kiyosi Kondo, Vice chairman, and other 5 as per attached sheet President of Institute of Fermen tation Research, Agency of Industrial Science Technology Jiro Oyama sealed We hereby notice that the microorganism applied by you to be deposited cannot be received for the reason shown below. Remarks 1. Date of application February 27, 1985 2. Representation for discrimination of microorganism Escherichia coli C600 pYTP102P 3. Reason for refusal Because of outside of the range of microorganisms which can be received. Attached sheetNames of depositors 1. Foundation Sagami Chemical Research Center Vice chairman Kiyosi Kondo 2. Central Glass Co., Ltd. President Mitsuyoshi Ito 3. Hodogaya Chemical Co., Ltd. President Kenichi Fujioka 4. Nippon Soda Co., Ltd. President Takeo Sannomiya 5. Nissan Chemical Industries, Ltd. Representative Director Misao Kusano 6. Toyo Soda Manufacturing Co., Ltd. President Toshiaki Yamaguchi Format 2 NOTICE OF REFUSAL TO RECEIVE DEPOSITION Notice No. 60 FERM Doc. No. 272 Date of notice March 9, 1985Esq. Foundation Sagami Chemical Research Center Kiyosi Kondo, Vice chairman, and other 5 as per attached sheet President of Institute of Fermen tation Research, Agency of Industrial Science Technology Jiro Oyama sealed We hereby notice that the microorganism applied by you to be deposited cannot be received for the reason shown below. Remarks 1. Date of application February 27, 1985 2. Representation for discrimination of microorganism Escherichia coli C600 pYTP2O1N 3. Reason for refusal Because of outside of the range of microorganisms which can be received. Attached sheetNames of depositors 1. Foundation Sagami Chemical Research Center Vice chairman Kiyosi Kondo 2. Central Glass Co., Ltd. President Mitsuyoshi Ito 3. Hodogaya Chemical Co., Ltd. President Kenichi Fujioka 4. Nippon Soda Co., Ltd. President Takeo Sannomiya 5. Nissan Chemical Industries, Ltd. Representative Director Misao Kusano 6. Toyo Soda Manufacturing Co., Ltd. President Toshiaki Yamaguchi Format 2 NOTICE OF REFUSAL TO RECEIVE DEPOSITION Notice No. 60 FERM Doc. No. 273 Date of notice March 9, 1985Esq. Foundation Sagami Chemical Research Center Kiyosi Kondo, Vice chairman, and other 5 as per attached sheet President of Institute of Fermen tation Research, Agency of Industrial Science Technology Jiro Oyama sealed We hereby notice that the microorganism applied by you to be deposited cannot be received for the reason shown below. Remarks 1. Date of application February 27, 1985 2. Representation for discrimination of microorganism Escherichia coli C600 pYTP301K 3. Reason for refusal Because of outside of the range of microorganisms which can be received. Attached sheetNames of depositors 1. Foundation Sagami Chemical Research Center Vice chairman Kiyosi Kondo 2. Central Glass Co., Ltd. President Mitsuyoshi Ito 3. Hodogaya Chemical Co., Ltd. President Kenichi Fujioka 4. Nippon Soda Co., Ltd. President Takeo Sannomiya 5. Nissan Chemical Industries, Ltd. Representative Director Misao Kusano 6. Toyo Soda Manufacturing Co., Ltd. President Toshiaki Yamaguchi