# Improved expression using fused genes providing for protein product.

## Abstract
Novel methods and compositions are provided for en hanced yield of heterologous proteins in fungi. The method and compositions involve employing fusion sequences in volving a sequence encoding a heterologous product pro duced in relatively large amount as a stable polypeptide in the host fused to a second sequence in open reading frame with the prior sequence coding for a different heterologous polypeptide, where the two polypeptides are joined by a se lectively cleavable linkage. In particular, a sequence coding for superoxide dismutase is joined to another polypeptide of interest at either terminus of the superoxide dismutase in a yeast expression vector under transcriptional control of an active promoter and the vector introduced into a yeast host and the host grown. High yields of the fusion product are ob tained in this manner, where the fusion product can be se lectively cleaved so as to produce both the superoxide dis mutase and the other polypeptide in high yield. The S. cerevisiae strains 2150 2 3 pYASI1 and AB110 pYLUIGF2 14 were deposited at the A.T.C.C. on February 27, 1985 and March 19, 1986, respectively and given acces sion nos. 20745 and 20796, respectively. CROSS REFERENCE TO RELATED APPLICATIONS This application is a Continuation of application Serial No. 717 209, filed March 28, 1985, which disclosure is incor porated herein by reference.

## Claims
FSHAT IS CLAIMED IS 1. In a method for preparing a polypeptide in a cellular host, where the polypeptide is heterologous to the host and may be expressed in low percentage amounts of total protein, the improvement which comprises joining an open reading frame DNA sequence coding for said polypeptide with a second open reading frame DNA sequence coding for a heterologous polypeptide produced in high yield and stable in said host, where the two polypeptides are joined to form a fusion polypeptide introducing the sequence coding for said fusion polypeptide under conditions for expression in said host, whereby said fusion polypeptide is expressed and isolating said fusion polypeptide to provide said second polypeptide in high yield. 2. A method according to Claim l, wherein said host is a eukaryotic host and said two peptides are joined to form a selectively cleavable link or the two open reading frames are joined by a sequence coding for a selectively cleavable link. 3. A method according to Claim 2, wherein said host is eukaryotic. 4. A method according to Claim 3, wherein said eukaryotic host is yeast. 5. A method according to Claim 4, wherein said DNA sequences are under the transcriptional regulatory control of a transcriptional initiation regulatory region comprising a promoter region for a glycolytic enzyme. 6. A method according to Claim 5, wherein said transcriptional initiation regulatory region is inducible. 7. A method according to Claim 1, wherein said host is prokaryotic. 8. In a method for preparing a mammalian polypeptide in a yeast host, where the polypeptide may be expressed in low percentage amounts of total protein, the improvement which comprises joining an open reading frame DNA sequence coding for said polypeptide with a second open reading frame DNA sequence coding for heterologous superoxide dismutase SOD , where the two polypeptides are joined to form a fusion polypeptide introducing the sequence coding for said fusion polypeptide under conditions for expression in said yeast, whereby said fusion polypeptide is expressed and isolating said fusion polypeptide in high yield. 9. A method according to Claim 8, wherein said conditions for expression include an inducible transcriptional initiation regulatory region. 10. A method according to Claim 9, where said transcriptional initiation regulatory region consists essentially of a glycolytic enzyme promoter region and ADH2 control region. 11 A DNA sequence coding for superoxide dismutase joined to a DNA sequence coding for a mammalian polypeptide, where the two coding sequences are joined by bases coding for amino acids defining a selectively cleavable site or a link having at least one amino acid, wherein said link provides for a selectively cleavable site. 12. A DNA sequence according to Claim 11 wherein said link codes for methionine. 13. A DNA sequence according to Claim 11, wherein said cleavable link codes for K R. 14. A DNA sequence according to Claim 11, wherein said link codes for D 4K. 15. A DNA sequence according to Claim 11, wherein said cleavable link includes coding for hinge amino acids. 16. A DNA sequence according to Claim 11 wherein said link codes for an enzymatically removable link. An An expression sequence including in direction of transcription, an inducible transcriptional initiation regulatory region and a DNA sequence according to Claim 11. A A polypeptide encoded for by a DNA sequence according to Claim 11. 1 9 . A polypeptide according to Claim 18, wherein said mammalian polypeptide encodes for at least a portion of proinsulin. 20. A polypeptide according to Claim 14, wherein said mammalian polypeptide encodes for at least a portion of IGF 1 or IGF 2.

## Description
IMPROVED EXPRESSION USING FUSED GENES PROVIDING FOR PROTEIN PRODUCT BACKGROUND OF THE INVENTIONField of the Invention There are an increasingly large number of genes available for expression, where the expression product may find commercial use. In many instances, the initial expression has been observed in E. coli.Expression in E. coli has many disadvantages, one in particular being the presence of an enterotoxin which may contaminate the product and make it unfit for administration to mammals. Furthermore, there has not previously been an extensive technology concerned with the production of products in E. coli, as compared to such other microorganisms as Bacillus subtilis,Streptomyces, or yeast, such as Saccharomyces. In many situations, for reasons which have not been resolved, heterologous products, despite active promoters and high copy number plasmids, are produced in only minor amount, if at all, in a microorganism host. Since the economics of the processes are dependent upon a substantial proportion of the nutrients being employed in the expression of the desired product, the production of these products in unicellular microorganisms appears to be unpromising. There is, therefore, a substantial need for processes and systems which greatly enhance the production of a desired polypeptide without substantial detriment to the viability and growth characteristics of the host.Description of the Prior Art Villa Komaroff et al., Proc. Natl. Acad. Sci.USA 1978 75 3727 3731, describe a fusion sequence encoding proinsulin joined to the N terminus of penicillinase for expression in E. coli. Paul et al., European J. Cell Biol. 1983 31 171 174, describe a fusion sequence encoding proinsulin joined to the COOHterminus of a portion of the tryptophan E gene product for expression in E. coli. Goeddel et al., ibid. 1979 76 106 110, describe synthetic genes for human insulinA and B chains fused to E. coli s galactosidase gene to provide a fused polypeptide in E. coli. Stepien et al., Gene 1983 24 289 297, describe expression of insulin as a fused product in yeast, where the proinsulin gene was fused to the N terminus coding sequence ofGAL1 for expression in yeast. SUMMARY OF THE INVENTION Methods and compositions are provided for producing heterologous polypeptides in high yield in a eukaryotic microorganism host, whereby a completely heterologous fused product is expressed, one part of the peptide being a product shown to be expressed independently in high yield in such host and the remaining part of the product being a polypeptide of interest, resulting in production of the fused product in high yield. Sequences coding for the two polypeptides are fused in open reading frame, where the high yield polypeptide encoding sequence may be at either the 5 or 3 terminus. The two polypeptides contained in the expression product may be joined by a selectively cleavable link, so that the two polypeptides may be separated to provide for high yield of each of the polypeptides.Alternatively, the cleavage site may be absent if cleavage of the fused protein is not required for its intended use. Particularly, a yeast host is employed where the high yield polypeptide is superoxide dismutase SOD . DESCRIPTION OF THE SPECIFIC EMBODIMENTS Novel methods and compositions are provided for enhancing the production of heterologous products in eukaryotic organisms, particularly yeast, by employing sequences encoding for a polypeptide, which is a combination of two polypeptide regions joined by a selectively cleavable site. The two regions are a first region which is a polypeptide produced independently in high yield ih the host and a second polypeptide of independent interest and activity, particularly one which is only difficultly obtained in the host. Hosts of interest include eukaryotic unicellular microorganisms, particular fungi, such asPhycomycetes, Ascomycetes, Basidiomycetes andDeuteromycetes, more particularly Ascomycetes, such as yeast, e.g., such as Saccharomyces, Schizosaccharomyces, and Kluyveromyces, etc. Prokaryotic hosts may also be employed such as E. coli, B. subtilis, etc. The stable polypeptide to be used as the first region in the fusion may be determined empirically. Thus, as heterologous polypeptides are developed in various host organisms, the yield of the polypeptide as compared to total protein may be readily determined. As to those polypeptides which are produced in amounts of 5 or greater of the total protein produced by the host, those DNA sequences encoding for such polypeptides may be used in this invention. The DNA sequences may be identical to the heterologous gene encoding the sequence, may be mutants of the heterologous gene, or may have one or more codons substituted, whereby the codons are selected as being preferred codons by the host.Preferred codons are those codons which are found in substantially greater than the mathematical probability of finding such codon, based on the degree of degeneracy of the genetic code, in those proteins which are produced in greatest individual abundance in the host. Particularly, in yeast, the glycolytic enzymes may be the basis for determining the preferred codons. The entire gene or any portion of the gene may be employed which provides for the desired high yield of polypeptide in the host. Thus, where the stable polypeptide is of lesser economic value than the polypeptide of interest, it may be desirable to truncate the gene to a fragment which still retains the desirable properties of the entire gene and its polypeptide product, while substantially reducing the proportion of the total fused product which is the stabilizing polypeptide. As illustrative of a gene encoding a stable polypeptide product in the yeast, is the gene encoding for superoxide dismutase, more particularly human superoxide dismutase. The DNA sequences coding for the two polypeptides, the stabilizing polypeptide and the polypeptide of interest, may be obtained in a variety of ways. The sequences encoding for the polypeptide may be derived from natural sources, where the messenger RNA or chromosomal DNA may be identified with appropriate probes, which are complementary to a portion of the coding or non coding sequence. From messenger RNA, single stranded ss DNA may be prepared employing reverse transcriptase in accordance with conventional techniques. The ss DNA complementary strand may then be used as the template for preparing a second strand to provide double stranded ds cDNA containing the coding region for the polypeptide. Where chromosomalDNA is employed, the region containing the coding region may be detected employing probes, restriction mapped, and by appropriate techniques isolated substantially free of untranslated 51 and 3 regions.Where only portions of the coding sequence are obtained, the remaining portions may be provided by synthesis of adapters which can be ligated to the coding portions and provide for convenient termini for ligation to other sequences providing particular functions or properties. Where the two genes are obtained in whole or in part from naturally occurring sources, it will be necessary to ligate the two genes in proper reading frame. If cleavage of the fused protein is required, where their juncture does not define a selectable cleavage site, genes will be separated by a selectively cleavable site. The selectively cleavable site will depend to some degree on the nature of the genes. That is, the means for cleaving may vary depending upon the amino acid sequence of one or both genes. Alternatively, there will be situations where cleavage is not necessary and in some situations undesirable. Fused proteins may find use as diagnostic reagents, in affinity columns, as a source for the determination of a sequence, for the production of antibodies using the fused protein as an immunogen, or the like. The two genes will normally not include introns, since splicing of mRNA is not extensively employed in the eukaryotic unicellular microorganisms of interest. The polypeptide of interest may be any polypeptide, either naturally occurring or synthetic, derived from prokaryotic or eukaryotic sources.Usually, the polypeptide will have at least 15 amino acids gene of 45 bp , more usually 30 amino acids gene of 90 bp , and may be 300 amino acids gene of 900 bp or greater. Polypeptides of interest include enzymes, viral proteins e.g. proteins from AIDS related virus, such as p18, p25, p31, gp41, etc. , mammalian proteins, such as those involved in regulatory functions, such as lymphokines, growth factors, hormones or hormone precursors e.g., proinsulin, insulin like growth factors, e.g., IGF I and II, etc. , etc., blood clotting factors, clot degrading factors, immunoglobulins, etc. Fragments or fractions of the polypeptides may be employed where such fragments have physiological activity, e.g., immunological activity such as cross reactivity with the parent protein, physiological activity as an agonist or antagonist, or the like. One of the methods for selectable cleavage is cyanogen bromide which is described in U.S. Patent No.4,366,246. This technique requires the absence of an available methionine other than at the site of cleavage or the ability to selectively distinguish between the methionine to be cleaved and a methionine within the polypeptide sequence. Alternatively, a protease may be employed which recognizes and cleaves at a site identified by a particular type of amino acid. Common proteases include trypsin, chymotrypsin, pepsin, bromelain, papain, or the like. Trypsin is specific for basic amino acids and cleaves on the carboxylic side of the peptide bond for either lysine or arginine.Further, peptidases can be employed which are specific for particular sequences of amino acids, such as those pe.tidases which are involved in the selective cleavage of secretory leader signals from a polypeptide. These enzymes are specific for such sequences which are found with a factor and killer toxin in yeast, such as KEX 2 endopeptidase with specificity for pairs of basic residues Julius et al., Cell 1984 37 1075 1089 .Also, enzymes exist which cleave at specific sequences of amino acids. Bovine enterokinase Light et al.,Anal. Biochem. 1980 106 199 206 cleaves to the carboxylic side of lysine or arginine that is preceded by acid residues of aspartic acid, glutamic acid, or carboxymethyl cysteine. Particularly useful is the sequence Asp 4 Lys found naturally as part of the activation peptide of trypsinogen in many species.Other enzymes which recognize and cleave specific sequences include Collagenase Germino and Batia, Proc. Natl. Acad. Sci. 1984 81 4692 4696 factor X Nagai Thygersen, Nature 1984 309 810 812 and polyubiquitin processing enzyme Ozakaynak et al.,Nature 1984 312 663 666 . In addition to the amino acids comprising the cleavable site, it may be advantageous to separate further the two fused polypeptides. Such a hinge would allow for steric flexibility so that the fused polypeptides would be less likely to interfere with each other, thus preventing incorrect folding, blockage of the cleavage site, or the like. The hinge amino acid sequence could be of variable length and may contain any amino acid side chains so long as the side chains do not interfere with the mode of action employed to break at the cleavable site or with required interactions in either fused polypeptide, such as ionic, hydrophobic, or hydrogen bonding. Preferably the amino acids comprising the hinge would have side chains that are neutral and either polar or nonpolar and may include one or more prolines. The hinge region will have at least one amino acid and may have 20 or more amino acids, usually not more than 15 amino acids, particularly the nonpolar amino acids G, A, P, V, I, L, and the neutral polar amino acids, N, Q, S, and T. Exemplary hinge sequences may be, but are not limited to N S Q A N S G S P A A S T P N S G P T P P S P G S P S S P G A and the like. It is contemplated that such hinge sequences may be employed as repeat units to increase further the separation between the fused polypeptides. So that the hinge amino acids are not bound to the final cleaved polypeptide of interest, it is desirable, but not required to practice the invention, to place the hinge between the polypeptide that is produced independently at high yield and the sequence for the cleavable site. Where one or more amino acids are involved in the cleavage site, the codons coding for such sequence may be prepared synthetically and ligated to the sequences coding for the polypeptides so as to provide for a fused protein where all the codons are in the proper reading frame and the selectable cleavage site joins the two polypeptides. Instead of only a small portion of the fused coding sequence being synthetically prepared, the entire sequence may be synthetically prepared. This allows for certain flexibilities in the choice of codons, whereby one can provide for preferred codons, restriction sites, avoid or provide for particular internal structures of the DNA and messenger RNA, and the like. While for the most part, the fused coding sequence will be prepared as a single entity, it should be appreciated that it may be prepared as various fragments, these fragments joined to various untranslated regions, providing for particular functions and ultimately the coding sequences brought together at a subsequent stage. However, for clarity of presentation, the discussion will be directed primarily to the situation where the coding sequence is prepared as a single entity and then transferred to an expression vector. The various sequences comprising the parts of the fused coding sequence can be joined by introducing a first fragment into a cloning vector. The resulting clone may then be restricted at a site internal to the coding sequence and an adapter introduced which will replace any lost codons and which has a convenient terminus for joining to the next fragment. The terminus may be cohesive or blunt ended, depending upon the particular nucleotides involved. After cloning of the combined first fragment and adapter, the vector may be restricted at the restriction site provided by the adapter and the remaining coding sequence of the second fragment introduced into the vector for ligation and cloning. The resulting fused sequence should be flanked by appropriate restriction sites, so that the entire sequence may be easily removed from the cloning vector for transfer to an expression vector. The expression vector will be selected so as to have an appropriate copy number, as well as providing for stable extrachromosomal maintenance.Alternatively, the vector may contain sequences homologous to the host genomic sequences to allow for integration and amplification. The expression vector will usually have a marker which allows for selection in the expression host. In order to avoid the use of biocides, which may find use in certain situations, desirably, complementation will be employed, whereby the host will be an auxotroph and the marker will provide for prototrophy. Alternatively, the episomal element may provide for a selective advantage, by providing the host with an enhanced ability to utilize an essential nutrient or metabolite in short supply.The significant factor is that desirably the extrachromosomal cloning vector will provide a selective advantage for the host containing the vector as compared to those hosts which may spontaneously lose the vector during production of the fused polypeptide. The cloning vector will also include an active transcriptional initiation regulatory region, which does not seriously interfere with the viability of the host. Regions of particular interest will be associated with the expression of enzymes involved in glycolysis acid phosphatase heat shock proteins metallothionein etc. Enzymes involved with glycolysis include alcohol dehydrogenase, glyceraldehyde 3phosphate dehydrogenase, glucose 6 phosphate dehydrogenase, pyruvate kinase, triose phosphate isomerase, phosphofructokinase, etc. Various transcriptional regulatory regions may be employed involving only the region associated with RNA polymerase binding and transcriptional initiation promoter region , two of such regions in tandem, or a transcriptional initiation regulatory region control region , normally 5 to the promoter region, where the control region may be normally associated with the promoter or with a different promoter in the wild type host. The control region will provide for inducible regulation where induction may be as a result of a physical change, e.g.temperature, or chemical change, e.g. change in nutrient or metabolite concentration, such as glucose or tryptophan, or change in pH or ionic strength. Of particular interest is the use of hybrid transcriptional initiation regulatory regions. Preferably, the hybrid transcriptional initiation regulatory region will employ a glycolytic enzyme promoter region.The control region may come from the control regions of a variety of expression products of the host, such as ADHII, GAL4, PHOS, or the like. The transcriptional initiation regulatory regions may range from about 50 1000 base pairs bp of the region 5 of the wild type gene. In addition to regions involved with binding of RNA polymerase, other regulatory signals may also be present, such as a capping sequence, transcriptional initiation sequences, enhancer, transcriptional regulatory region for inducible transcription, and the like. The transcriptional initiation regulatory region will normally be separated from the terminator region by a polylinker, which has a plurality of unique restriction sites, usually at least two, and not more than about 10, usually not more than about six. The polylinker will generally be from about 10 50bp. The polylinker will be followed by the terminator region, which may be obtained from the same wild type gene from which the promoter region was obtained or a different wild type gene, so long as efficient transcription initiation and termination is achieved when the two regions are used. By digestion of the expression vector with the appropriate restriction enzymes, the polylinker will be cleaved and the open reading frame sequence coding for the fused polypeptide may be inserted.Where the polylinker allows for distinguishable termini, the fused gene can be inserted in a single orientation, while where the termini are the same, insertion of the fused gene will result in plasmids having two different orientations, only one of which will be the proper orientation. In any event, the expression vector may be cloned where it has a prokaryotic replication system for isolation and purification and then introduced into an appropriate eukaryotic host, such as a yeast host. Introduction of foreign DNA into eukaryotic hosts can be performed in a wide variety of ways, such as calcium polyethylene glycol treated DNA with spheroplasts, use of liposomes, mating, or the like. The host cells containing the plasmid with the fused gene capable of expression are then grown in an appropriate nutrient medium for the host. Where an inducible transcriptional initiation regulatory region is employed, the host cell may be grown to high density and initiation turned on for expression of the fused polypeptide. Where the promoter is not inducible, then constitutive production of the desired fused polypeptide will occur. The cells may be grown until there is no further increase in product formation or the ratio of nutrients consumed to product formation falls below a predetermined value, at which time the cells may be harvested, lysed and the fused protein obtained and purified in accordance with conventional techniques. These techniques include chromatography, e.g., HPLC electrophoresis extraction density gradient centrifugation, or the like. Once the fused protein is obtained, it will then be selectively cleaved in accordance with the nature of the selectively cleavable linkage. This has been described previously in relation to the description of the various linkages. In some instances a secretory leader and processing signal may be included as part of the fused polypeptide. Various secretory leader and processing signals are known, such as yeast a factor, yeast killer toxin and the like. The DNA sequence coding for these polypeptide signals may be linked in proper reading frame to the 5 end in direction of transcription of the sense strand of the DNA sequence coding for the fused polypeptide to provide for transcription and translation of a pre fused polypeptide. In accordance with the subject invention, the product is produced in at least a 5 weight percent, preferably at least 6 weight percent, and more preferably at least about 10 weight percent, of the total protein of the host. In this manner, the nutrients employed are efficiently utilized for conversion to a desired product. The following examples are offered by way of illustration and not by way of limitation. EXPERIMENTALEXAMPLE I Construction and Expression of Expression Vectors for SOD Proinsulin Fusion ProteinConstruction of pYSIl A yeast expression plasmid pYSIl, containing the human SOD gene fused to the amino terminus of human pro insulin gene, under the regulation of the GAP promoter and terminator was constructed. A triplet coding for methionine was included between the SOD and proinsulin genes to allow for chemical processing of the fusion protein. The SOD sequences correspond to a cDNA isolated from a human liver library, except for the first 20 codons which were chemically synthesized.The proinsulin sequence was chemically synthesized according to the amino acid sequence reported by Bell et al. 1979 , Nature 282 525 527 , but using yeast preferred codons. The GAP promoter and terminator sequences were obtained from the yeast GAP gene Holland BR Holland, J. Biol. Chem. 1979 254 5466 5474 isolated from a yeast library. Plasmid pYSIl was constructed as follows.Three fragments were employed which involve a 454bpNcoI Sau3A fragment isolated from phSOD also designated as pSODNco5 , where the fragment includes the entire coding sequence for human superoxide dismutase hSOD with the exception of the last three 3 codons a 51bp Sau3A HindIII synthetic adapter, which codes for the last three codons of hSOD, methionine, and the first 14 codons of proinsulin and a 231bp HindIII SalI fragment, isolated from pINS5, which encodes proinsulin excepting the first 14 amino acids. These fragments were ligated together and introduced into the plasmid pPGAP, which had been previously digested with NcoI andSalI and alkaline phosphatase treated. The resulting plasmid pSIl was digested with BamHI to provide an expression cassette which was cloned into plasmid pC1 1 to yield pYSIl. Plasmid phSOD also designated as pSODNco5 is a pBR322 derived bacterial expression vector which contains a complete cDNA coding except that the first 20 codons were chemically synthesized for hSOD as described in copending application serial number 609,412 filed on May 11, 1984. Plasmid pINS5 is a pBR322 derived vector which contains a proinsulin coding sequence chemically synthesized according to the amino acid sequence reported by Bell et al., Nature 1979 282 525 527. Plasmid pPGAP is a pBR322 derived vector described in copending application 609,412 supra which contains aGAP promoter and GAP terminator Holland and Holland,J. Biol. Chem. 1979 254 5466 5474 with a polylinker between them, which provides for single restriction sites for cloning. Plasmid pCl 1 is a yeast expression vector which includes pBR322 sequences, 2 plasmid sequences and the yeast gene LEU2 as a selectable marker. See EPO 83 306507.1, which relevant parts are incorporated herein by reference.Construction of pYS12 To prepare the fused gene having the hSOD coding sequence at the 3 terminus in the direction of transcription separated from the pro insulin gene by a spacer of codons coding for K R S T S T S, the following fragments were ligated. A 671bp BamHI SalI fragment containing the GAP promoter, the proinsulin gene and codons for the spacer amino acids a 14bpSalI NcoI synthetic adapter, which codes for the last spacer amino acids as a junction of both genes and a 1.5kb NcoI BamHI fragment isolated from pCl l GAPSOD described in copending application 609,412 supra , which includes the hSOD coding region, 56bp of hSOD terminator and 934bp of GAP terminator region. The resulting cloned fragment was isolated and inserted into BamHI digested, alkaline phosphatase treated pCl l. Plasmids pPKIl and pPKI2 Plasmids homologous to pYSIl and pYSI2, but using the yeast pyruvate kinase PYK gene instead of hSOD gene, were also constructed. pPKI1 contains thePYK coding sequence fused to the amino terminus of the human pro insulin gene under regulation of the yeast PYK promoter and yeast GAP terminator. pPKI2 contains thePYK coding sequence at the 3 terminus in the direction of transcription separated from the proinsulin gene by a spacer of codons coding for K R S T S. This fused gene is under regulation of the GAP promoter and PYK terminator.Construction of pYASIl This yeast expression plasmid is similar to pYSIl and contains the hSOD gene fused to the amino terminus of the human proinsulin gene, with a methionine codon at the junction between both genes.The fusion gene is under control of the hybrid inducible ADH2 GAP yeast alcohol dehydrogenase 2 promoter and the GAP terminator. An about 3kbp BamHI expression cassette was constructed by replacing theGAP promoter sequence from pYSIl with the hybridADH2 GAP promoter sequence. The ADH2 portion of the promoter was constructed by cutting a plasmid containing the wild typeADH2 gene plasmid pADR2, see Beier and Young, Nature 1982 300 724 728 with the restriction enzyme EcoRS, which cuts at a position 66 relative to the ATG start codon, as well as in two other sites in pADR2, outside of the ADH2 region. The resulting mixture of a vector fragment and two smaller fragments was resected with Bay31 exonuclease to remove about 300bp. SyntheticXhoI linkers were ligated onto the Bal31 treated DNA.The resulting DNA linker vector fragment about 5kb was separated from the linkers by column chromatography, cut with the restriction enzyme XhoI, religated and used to transform E. coli to ampicillin resistance. The positions of the XhoI linker additions were determined by DNA sequencing. One plasmid which contained an XhoI linker located within the 5 non transcribed region of the ADH2 gene position 232 from ATG was cut with the restriction enzyme XhoI, treated with nuclease S1, and subsequently treated with the restriction enzyme EcoRI to create a linear vector molecule having one blunt end at the site of the XhoI linker and an EcoRI end. The GAP portion of the promoter was constructed by cutting plasmid pPGAP supra with the enzymes BamHI and EcoRI, followed by the isolation of the 0.4Kbp DNA fragment. The purified fragment was cut with the enzyme AluI to create a blunt end near theBamHI site. Plasmid pJS104 was constructed by the ligation of the AluI EcoRI GAP promoter fragment to theADH2 fragment present on the linear vector described above. Plasmid pJS104 was digested with BamHI which cuts upstream of the ADH2 region and with NcoI which cuts downstream of the GAP region . The about 1.3Kbp fragment containing the ADH2 GAP promoter was gel purified and ligated to an about 1.7Kbp fragment containing the hSOD proinsulin fusion DNA sequences and GAP terminator present in pYSIl previously described . This 3Kbp expression cassette was cloned into BamHI digested and phosphatase treated pCl 1 to yield pYASIl.Construction of pYASIl Derivatives Containing Trypsin and Enterokinase Cleavage Sites A series of plasmids were constructed derived from pYASIl, in which the GAP terminator was replaced by the a factor terminator Brake et al., Proc. Natl.Acad Sci. USA 1984 81 4642 and the cleavage site between SOD and pro insulin was modified to code for trypsin or enterokinase processing sites. Sequences coding for Lys Arg were used to replace the methionine codon in pYASIl yielding a trypsin site.Alternatively, sequences coding for Asp 4Lys were used at the cleavage site to yield an enterokinase site. In addition, sequences coding for extra hinge amino acids were also inserted between the SOD and the cleavage site in other constructions. Expression of Fusion Proteins yeast strain 2150 2 3 Mat a, ade 1, leu 2 04, cairo or P017 Mat a, leu 2 04, cairo were transformed with the different vectors according toHinnen et al., Proc. Natl. Acad. Sci. USA 1978 75 1929 1933. Single transformant colonies harboring constitutive GAP regulated vectors were grown in 2ml of leu selective media to late log or stationary phase.Cells harboring inducible ADH2 GAP regulated vectors were grown to saturation in leu selective media, subsequently diluted 1 20 v v in YEP, 3 ethanol, with or without 2 3.5mM CuSO4 and grown to saturation in this medium. Cells were lysed in the presence ofSDS and reducing agent and the lysates clarified by centrifugation. Cleared lysates were subjected to polyacrylamide gel electrophoresis Laemmli, Nature 1970 277 680 . Following staining with Coomassie blue, a band of about 28kDal kilodaltons was observed, the size predicted for the fusion protein.This band was detected in those cells transformed with expression vectors, while being absent from extracts of cells harboring control pC1 1 plasmids. Amount of protein per band was determined by densitometric analysis of the Coomassie blue stained gels. The fusion protein accounts for over 10 of the total cell protein as estimated from the stained gels in those cells transformed with pYSIl, pYSI2 or pYASIl, while it accounts for less than 0.5 in pYPKIl or pYPKI2 transformants See Table 1 . TABLE 1 The Yield of SOD PI from 2150 or P017 Transformed with Different Expression Plasmids and Grown in the Absence Presence of 2 3.5 mM CuSO4 Strain Plasmid Description of sequences Expression percent contained in the expression of total cell protein cassette. Cu Cu 2150 pYPKI 1 PYK PYK M BCA5 GAPt 0.5 p 2150 pYPKI 2 GAP M BCA5 KRSTS 2PYK PYKt 0.5 p 2150 pYSI 1 GAP SOD M BCAS GAPt 10 p 2150 pYSI 2 GAP M BCAS KR ST 2S SOD GAPt 10 p SOD Met Proinsulin 2150 pYASI 1 ADH GAP SOD M BCA5 GAPt 10 p P017 pYASI 1 ADH GAP SOD M BCA5 GAP 20 30 20 30 p t SOD hinge Asp 4LysPIP017 pYSt12 ADH GAP SOD D4K BCA5 alpha factort 6 9 11 14P017 pYSI15 ADH GAP SOD NS D4K BCA5 alpha factort 5 6 9 14P017 pYS18 ADH GAP SOD NSGSP D4K BCA5 a factort 5 8 p 4P017 pYSI4 ADH GAP SOD NSGPTPPSPGSP D4K BCA5 alpha factort 9 12 9 16 SOD hinge LysArgPIP017 pYSI13 ADH GAP SOD KR BCAS alpha factort 8 10 8 10 p P017 pYS110 ADH GAP SOD NSGSP KR BGAS alpha factort 5 7 10 15 p P017 pYSI3 ADH GAP SOD NSGPTPPSPGSP KR BCA5 a factort 5 8 15 30 p Determined by scanning densitometer analysis of Coomassie Blue stained gels.Note Proinsulin PI accounts for less than 0.1 of total cell protein in cells transformed with pYGAPINS5, a plasmid containing the proinsulin gene under regulation of GAPDH promoter and terminator GAP M BCAS GAPt p PYK pyruvate kinase gene GAP GAP promoter SOD human SOD gene GAPt GAP terminator BCA5 proinsulin gene PYK PYK promoterP, G, D, N, M, K, R, S, T one letter PYKP PYK terminator amino acid code ADH2 GAP hybrid ADH2 GAP promoter alpha factort alpha factor terminator Results shown in Table 1 indicate that while expression levels of PYK proinsulin fusion are comparable to those obtained with pro insulin alone about 0.5 and 0.1 , respectively , the expression levels of hSOD proinsulin are about 20 to 100 fold higher.The inducible ADH2 GAP hybrid transcriptional initiation regulatory region is preferred, since it is noted that constitutive production in scaled up cultures results in unstable expression. The hSOD proinsulin proteins synthesized by yeast were also submitted to Western analysis. Cleared yeast lysates prepared as described above were electrophoresed on polyacrylamide gels Laemmli, supra and proteins were subsequently electroblotted onto nitrocellulose filters Towbin et al., Proc. Natl.Acad. Sci. USA 1979 76 3450 . Two identical filters were blotted. The filters were preincubated for lhr with 1 BSA in PBS and subsequently treated with rabbit anti hSOD or guinea pig anti insulin antibodies for 12hr at 40C. Both sera had been preadsorbed with pC1 1 control lysate in 10 goat serum. The filters were washed with 1 BSA PBS and a second goat anti rabbit or anti guinea pig antibody conjugated with horseradish peroxidase added. Finally, the filters were incubated with horseradish peroxidase color development reagent Bio Rad and washed. The Western analysis showed that the fusion protein reacted with both antibodies.Cleavage of the Fusion Proteins A saturated culture of 2150 pYASIl was grown in SDC minus leucine plus threonine and adenine, containing 2 glucose. This was used to inoculate a 10 liter fermentor containing YEP with 3 ethanol as carbon source. After 48hrs at 300C, the cells were harvested by centrifugation Sharples , weighed 124g , and washed with cold water. The cells were lysed by glass bead disruption Dyno mill using a buffer containing 10mM Tris C1, pH 7.0, lmM EDTA, lug ml pepstatin A and lmM PMSF. The mixture was centrifuged for 20min at 8,000rpm in a JA10 rotor Beckman . The pellet was resuspended in 100mls of buffer and the liquid was removed from the beads.This was repeated until 500mls of buffer was used to thoroughly remove all pellet material from the glass beads. The resuspended pellet was centrifuged, and the pellet washed a second time. The pellet was then extracted for 30min in buffer plus 1 SDS. The SDS soluble fraction was ion pair extracted using 500mls of solvent A Konigsberg andHenderson, 1983 Meth. in Enz. 91, pp. 254 259 , the pellet washed once with solvent A, and once with acetone. After drying in a vacuum desiccator, the powder was dissolved in 140mls 100 formic acid. Sixty mls of H2O and 20g CNBr were added. After 24hrs at room temperature, in the dark, an additional 20g CNBr was added, and the reaction continued for 24hrs. At this time, the material was dialyzed overnight against 4 liters H20 using 2000 MW cutoff tubing Spectrapor . A second dialysis against 0.1 acetic acid followed.After lyophilization, a powder consisting mostly ofSOD homoserine lactone and proinsulin was obtained, weighing l.lg. This powder was dissolved in a 200ml solution of 7 urea, 9 sodium sulfite, and 8.1 sodium tetrathionate 2H2O, pH 7.5. After incubation for 3hrs at 370C, the S sulfonate products were dialyzed twice versus 10mM Tris pH 8.0, and once versus 20mMTEAB triethylammonium bicarbonate , pH 7.3. The S sulfonates were recovered by lyophilization and dissolved in 240mls DEAE column buffer Wetzel et al., Gene 1981 17 63 71 and loaded onto a 60ml column. After washing with two column volumes, the proinsulin S sulfonate was eluted with a 600ml gradient of 0 to 0.4M NaCl in column buffer.Fractions containing proinsulin S sulfonate were pooled and dialyzed twice against 10mM Tris, pH 7.5, and once against lmM Tris. The product, 908 pure proinsulin S sulfonate, was shown to migrate as expected on pH 9 gel electrophoresis Linde et al., Anal. Biochem. 1980 107 165 176 , and has the correct 15 N terminal residues. On analysis, the amino acid composition was very close to that expected, not exactly correct due to the presence of a low level of impurities. The yield was 150mg. Preliminary results on renaturation have been obtained with the following procedure. The proinsulin S sulfonate can be renatured at pH 10.5, with ss mercap toethanol Frank et al. 1981 in Peptides Synthesis,Structure and Function, Proceedings of the SeventhAmerican Peptide Symposium, Rich and Gross, eds.,Pierce Chemical Co., Rockford, IL, pp. 729 738 . In preliminary experiments, the yield of correctly renatured proinsulin has been monitored by the production of insulin produced from digestion with trypsin and carboxypeptidase B. The proinsulin S SO3 produced by this process appears to renature as well as purified porcine proinsulin S SO3. This process has been reported to yield 70 of the expected amount of insulin.The insulin produced in this way has the correct N terminal 15 residues of each A chain and B chain as determined by amino acid sequencing.EXAMPLE II Construction and Expression of Expression Vectors for SOD p31 Fusion Protein A yeast expression plasmid pC1 1 pSP31 GAPADH2, containing the human SOD gene fused to the amino terminus of the endonuclease region p31 of the pol gene of the AIDS related virus ARV Sanchez Pescador et al., Science 1985 227 484 was constructed. Expression of SOD p31 is non constitutive and is under regulation of a hybrid ADH GAP promoter.Construction of pCl l pSP31 GAP ADH2 Derivative For the construction of a gene for a fused protein SOD p31 to be expressed in yeast, a plasmid pS14 39 2 was used This plasmid contains the SOD gene fused to the proinsulin gene under the regulation of the ADH 2 GAP promoter in the same manner as pYASl.The pro insulin gene is located between EcoRI and SalI restriction sites. To substitute the proinsulin gene with the p31 fragment, two oligomers designated ARV 300 and ARV 301, respectively, were synthesized using phosphoramidite chemistry. The sequences generate cohesive ends for EcoRI and NcoI on each side of the molecule when the two oligomers are annealed. ARV 300 and ARV 30l have the sequences ARV 300 5 AATTCAGGTGTTGGAGC GTCCACAACCTCGGTAC 5 ARV 301 Two pg of pS14 39 2 linearized with EcoRI were ligated to 100 picomoles each of phosphorylatedARV 300 and dephosphorylated ARV 301 in the presence ofATP and T4 DNA ligase in a final volume of 35 ul. The reaction was carried out at 140C for 18 hr.The DNA was further digested with SalI and the fragments were resolved on a 1 low melting point agarose gel and a fragment containing the vector plus the SOD gene 6.5kb was purified as described above and resuspended in 50 iil of TE 10mM Tris, lmM EDTA, pH 8 .Five 1 of this preparation were ligated to 5 vl of the p31 fragment ARV248NL, see below in 20 ul final volume for 18 hr at 140C and 5 ul used to transform competent HB101 cells. The resultant plasmid was called pSP3l. Twenty g of this plasmid were digested with BamHI and a fragment of about 2900 bp was isolated by gel electrophoresis, resuspended in TE and ligated to pCl 1 previously cut with BamHI. This DNA was used to transform HB101 and transformants with the BamHI cassette were obtained. Yeast strain P017 Mat a, leu2 04, cairo was transformed with this pCl 1 pSP3 1 GAP ADH2 derivative.Preparation of ARV248NL, the p31 Coding Fragment. The 800bp ARV248NL fragment codes for numbered amino acids 737 to the end of the pol protein as shown in Figure 2 of Sanchez Pescador et al. supra . The following procedure was used for its preparation. A 5.2kb DNA fragment was isolated from a digest of ARV 2 9B Sanchez Pescador et al., supra containing the 3 end of the pol gene, orf l, env and the 5 end of orf 2, that had been run on a 1 low melting point agarose Sea Pack gel and extracted with phenol at 650C, precipitated with 100 ethanol and resuspended in TE. Eight iil of this material were further digested with SstI for 1 hr at 370C in a final volume of 10 vl. After heat inactivation of the enzyme, 1.25 pl of this digest were ligated to 20 ng ofM13mpl9 previously cut with KpnI and SstI, in the presence of ATP and in a final volume of 20 p1. The reaction was allowed to proceed for 2 hr at room temperature.Five iil of this mixture were used to transform competent E. coli JM101. Clear plaques were grown and single stranded DNA was prepared as described in Messing and Vieira, Gene 1982 19 269 276. The DNA sequence in the M13 template was altered by site specific mutagenesis to generate a restriction site recognized by NcoI CCATGG . An oligodeoxynucleotide that substitutes the A for a C at position 3845 Figure 1 in Sanchez Pescador et al., supra and changes a T for an A at position 3851 was synthesized using solid phase phosphoramidite chemistry. Both of these changes are silent in terms of the amino acid sequence, and the second one was introduced to decrease the stability of the heterologous molecules. The oligomer was named ARV 216 and has the sequence 5 TTAAAATCACTTGCCATGGCTCTCCAATTACTG and corresponds to the noncoding strand since the M13 derivative template 01100484 is single stranded and contains the coding strand.The 5 dephosphorylatedM13 sequencing primer, 50 mM Tris HCl pH 8, 20 mM KC1, 7 mM MgC12 and 0.1 n EDTA. The polymerization reaction was done in 100 p1 containing 50 ng ul DNA duplex, 150 uM dNTPs, 1 mM ATP, 33 mM Tris acetate pH 7.8, 66 mM potassium acetate, 10 mM magnesium acetate, 5 mM dithiothreitol DTT , 12.5 units of T4 polymerase, 100 uglml T4 gene 32 protein and 5 units of T4 DNA ligase. The reaction was incubated at 300C for 30 min and was stopped by the addition of EDTA and SDS 10 mM and 0.2 respectively, final concentration . CompetentJM101 E. coli cells were transformed with 1, 2, and 4 p1 of a 1 10 dilution of the polymerization product and plated into YT plates. Plaques were lifted by adsorption to nitrocellulose filters and denatured in 0 2 N NaOH, 1.5 M Nail, followed by neutralization in 0.5 M Tris HCl pH 7.3, 3 M NaCl and equilibrated in 6 xSSC. The filters were blotted dry, baked at 800C for 2 hr and preannealed at 370C in 0.2 SDS, 10 x Denhardt s 6 x SSC. After 1 hr, 7.5 x 106 cpm of labelled ARV 216 were added to the filters and incubated for 2 additional hr at 370C. The filters were washed in 6 xSSC at 420C for 20 min, blot dried and used to expose film at 700C for 1 hr using an intensifying screen.Strong hybridizing plaques were grown and single stranded DNA was prepared from them and used as templates for sequencing. Sequencing showed that template 01021785 contains the NcoI site as well as the second substitution mentioned above. A second oligomer was synthesized to insert sites for SalI and EcoRI immediately after the termination codon of the pol gene position 4647,Figure 1, Sanchez Pescador et al., supra . This oligomer was called ARV 248 and has the sequence 5 GGTGTTTTACTAAAGAATTCCGTCGACTAATCCTCATCC. Using the template 01020785, site specific mutagenesis was carried out as described above except that the filter wash after the hybridization was done at 650C.As above, 8 strong hybridizing plaques were grown and single stranded DNA was sequenced. The sequence of template 01031985 shows that it contains the restriction sites for NcoI, SalI, and EcoRI as intended. Replicative form RF of the M13 01031098 template was prepared by growing 6 clear plaques, each in 1.5 ml of 2 x YT 0.5 yeast extract, 0.8 tryptone, 0.5 HaCl, 1.5 agar at 370C for 5 hr.Double stranded DNA was obtained as described byManiatis, et al., Molecular Cloning, a LaboratoryManual, Cold Spring Harbor, 1982, pooled and resuspended in 100 pl final volume. A 20 pl aliquot ofRF was cut with NcoI and SalI in a 40 pl volume of digestion buffer. This fragment was used for p31 expression in yeast. The samples were run on a 1 low melting point agarose Sea Pack gel and the DNAs were visualized by fluorescence with ethidium bromide. The 800 bp band was cut and the DNA was extracted from the gel as mentioned above and resuspended in 10 p1 of TE.The fragment was called ARV248NL. Induction of pCl l pSP31 GAP ADH2 Three different kinds of inductions were tried I P017 colonies were induced in either a 10 ml culture of YEP I glucose or a leu 3 ethanol culture for 24 hr. The yeast pellets from each mixture were analyzed for p31 by both polyacrylamide gels andWesterns using sera from AIDS patients. Even though the Coomassie stained gel showed a negative result, in both cases the Western did light up a band of the correct molecular weight. 2 P017 colonies were induced in a 30 ml culture of YEP 1 ethanol for 48 hr. Aliquots were analyzed by PAGE at various time points during the induction. The Coomassie stained gel shows a band in the correct molecular weight rar,ge 47 50 kd that appears after 14 hr in YEP 1 ethanol and reaches a maximum intensity at 24 hr of induction. The Western result for SOD p31 using sera from AIDS patients correlates well with the Coomassie stained gel, showing strong bands at 24 and 48 hr of induction.Purification and Characterization of SOD p31 from Yeast Frozen yeast bacteria cells were thawed at room temperature and suspended in 1.5 volumes of lysis buffer 20 mM Tris Cl, pH 8.0, 2 mtl EDTA, 1 mM phenylmethylsulfonyl fluoride PMSF , for bacteria 50 mM Tris Cl, pH 8.0, 2 tA EDTA, 1 mM PMSF for yeast , and mixed with 1 volume of acid washed glass beads. Cells were broken for 15 min. in a non continuous mode using the glass chamber of aDynomill unit at 3,000 rpm, connected to a 200C cooling unit. Glass beads were decanted for 2 3 min.on ice, and the cell lysate removed. The decanted glass beads were washed twice with 30 ml of lysis buffer at 40C. The cell lysate was centrifuged at 39,000 x g for 30 min. The pellet obtained from the above centrifugation was washed once with lysis buffer, after vortexing and suspending it at 40C same centrifugation as above . The washed pellet was treated with 0.2 SDS for bacteria and 0.1 SDS for yeast in lysis buffer and was agitated by rocking at 40C for 10 min. The lysate was centrifuged at 39,000 x g for 30 min. The pellet was boiled in sample buffer 67.5 mM Tris Cl, pH 7.0, 5 B mercaptoethanol, 2.3 SDS for 10 min and centrifuged for 10 min. at 39,000 x g. The supernatant was recovered and passed through a 0.45 ijm filter. The supernatant from the above filter was loaded maximum 50 mg of protein on a gel filtration column 2.5 x 90 cm, ACA 34 LKB with a flow rate of 0.3 0.4 ml min, equilibrated with phosphate buffered saline PBS , 0.1 SDS.The fractions containing SOD p31 were pooled and concentrated either by vacuum dialysis or using a YM5Amicon membrane at 40 psi. The protein was stored at 200C as concentrated solution. Gel electrophoresis analysis showed that theSOD p31 protein migrates having a molecular weight of about 46 kd and is over 90 pure. Similar constructions and results have been obtained by expressing an SOD p31 fusion under regulation of a bacterial trp lac promoter in E. coli. The SOD p31 fused protein finds use in immunoassays to detect the presence of antibodies against AIDS in body fluids. Successful results have been obtained using the SOD p31 fusion protein in ELISA as well as in strip assays. Example III Construction and Expression of Expression Vectors for SOD IG 2 Fusion Protein. A yeast expression plasmid pYLUIGF2 14, containing the human SOD gene fused to the amino terminus of the IGF2 gene see EPO 123 228 was constructed. Expression of SOD IGF2 is non constitutive and it is under regulation of a hybrid ADH GAP promoter.Construction of pYLUIGF2 14 For the construction of a gene for a fused protein SOD IGF2 to be expressed in yeast, plasmid pYS18 was used. Plasmid pYS18 contains the SOD gene fused to the proinsulin gene under the regulation of the ADH GAP promoter and a factor terminator see Table 1 . Plasmid pYS18 was digested with BamHI and EcoRI. The 1830 bp fragment containing the ADH GAP promoter and SOD gene was purified by gel electrophoresis. A second BamHI 460 bp fragment coding for amino acid residues 41 to 201 of IGF 2 and for the a factor terminator see EPO 123 228 was ligated to the following linker EcoRI SalI AATTCCATGGCTTACAGACCATCCGAAACCTTGTGTGGTGGTGAATTGG GGTACCGAATGTCT GGTAGGCTTTGGAACACACCACCACTTAACCAGCT The linker provides for an EcoRI overhang, an ATG codon for methionine and for codons 1 40 of IGF2 and SalI overhang. The resulting EcoRI BamHI 510 bp fragment containing the IGF 2 gene and a factor terminator was ligated to the 1830 bp BamHI EcoRI fragment containing the ADH GAP promoter and SOD see above . The resulting BamHI 2340 bp fragment was cloned intoBamHI digested and phosphatase treated pAB24 see below to yield pYLUIGF2 14. pAB24 is a yeast expression vector see Figure 2 which contains the complete 2 sequences Broach 1981, In Molecular Biology of the Yeast Saccharomyces 1 445, Cold Spring Harbor Press and pBR322 sequences.It also contains the yeast URA3 gene derived from plasmid YEp24 Botstein et al., 1979 Gene 8 17 and the LEU2 d yeast LEU2d gene derived from plasmid pC1 1 see EPO 116201 . Insertion of the expression cassette was in the BamHI site of pBR322, thus interrupting the gene for bacterial resistance to tetracycline.Expression of SOD IGF2 Yeast AB110 Mata, ura3 52, leu2 04 or both leu2 3 and leu2 112, pep4 3, his4 580, cairo was transformed with pYLUIGF2 14. Transformants were grown up on ura selective plate. Transformant colonies were transferred to 3 ml leu selective media and grown 24 hrs in 300C shaker. 100 p1 of a 1 X 10 4 dilution of this culture was plated onto ura plates and individual transformants were grown up for 48 72 hrs. Individual transformants were transferred to 3 ml leu media and grown 24 hrs in a 30 C shaker. One ml each of these cultures was diluted into 24 ml UEP, 1 glucose media and cells were grown for 16 24 hurs for maximum yield of SOD IGF2. Cells were centrifuged and washed withH2O.Cells were resuspended in 2 volumes of lysis buffer phosphate buffer, pH 7.3 50 100mM , 0.1 Triton X100 . Two volumes of acid washed glass beads were added and the suspension was alternately vortexed or set on ice 5x, 1 min. each cycle . The suspension was centrifuged and the supernatant decanted. The insoluble pellet was incubated in lysis buffer 1 SDS at room temperature for 30 min. The suspension was centrifuged and the supernatant was frozen and lyophilized. Two other constructions pYLUIGF2 15 and pYUIGF2 13 were used as controls for expression of a non fused IGF2. The former plasmid pYLUIGF2 15 for intracellular expression contains the IGF2 gene under control of the GAP promoter and a factor terminator.The latter plasmid pYUIGF2 13 for secretion of IGF2, the IGF 2 gene under control for the GAP promoter, a factor leader and a factor terminator. EXPRESSION OF IGF2 PAGE STAINConstruction in AB110 of total cell protein RRA 1. pYLUIGF2 15 NOT DETECTABLE NA GAPpIGE2 aft 2. pYUIGF2 13 BARELY DETECTABLE lOg Q GAP alpha fL IGF2 alpha fT 3. pYLUIGF2 14 10 15 NA ADH2 GAP SOD.IGF2 alpha fT NA Not available. By Coomassie blue staining, the SOD IGF2 fusion protein represents 10 15 of the total cell protein, i.e., 100 300 mg t culture equivalent. IGF2 represents 1 3 of the fusion protein, therefore it constitutes about 30 100 mg Q culture equivalent. Analytical CNBr cleavage reactions with the fusion protein have resulted in a band on PAGE which migrates to the position expected for IGF2. RRA IGF2 levels were measured by a placental membrane radioreceptor assay RRA according to Horner et al., J. of Clinical Endocrinology and Metabolism 1978 47 1287 and Marshall et al., J. of Clinical Endocrinology and Metabolism 1974 39 283. Placental membranes for the RRA were prepared by the method of Cuatrecasas, Proc. Natl. Acad. Sci. USA 1972 69 318.Protocol for CNBr Cleavage of SOD.IGF2 The insoluble fraction from glass bead lysis of yeast cells was dissolved in 70 formic acid. CNBr crystals g CNBr 100 mg fusion protein were added and incubation was carried out at room temperature for 12 15h in the dark. This step may be repeated after 2 hrs if cleavage is incomplete. It is evident from the above results that otherwise difficultly and inefficiently produced polypeptides may be produced in substantially enhanced yields by employing a fused protein, where the fusion protein includes a relatively short stable polypeptide sequence joined to the other polypeptide by a selectively cleavable site. Thus, high levels of the fusion protein are obtained in a eukaryotic host, such as yeast, allowing for the efficient production of desired polypeptides heterologous to the host. Although the foregoing invention has been described in some detail by way of illustration and example for purposes of clarity of understanding, it will be obvious that certain changes and modifications may be practiced within the scope of the appended claims.