# DEDICATED RIBOSOMES AND THEIR USE IN PRODUCING PROTEIN IN CELL CULTURE

## Claims
Patentansprüche für folgende Vertragsstaaten BE, CH, DE, FR, GB, IT, LI, LU, NL, SE

## Description
This relates to the field of protein synthesis. In particular, it is concerned with harnessing elements of cellular protein synthesis to the specialized production of a preselected protein. The initiation of protein synthesis SD sequences are believed to function as follows Preceding the initiation of translation, the 30S ribosomal subunit binds via its structural RNA 16S rRNA to the SD sequence on the messenger RNA. It is believed that this binding brings the 30S particle in the right position for binding of the fMet tRNA to the ATG in the so called P site of the 30S particle. After these events the 50S ribosomal subunit binds to form the 70S ribosomal particle. At this point protein initiation is completed and the elongation phase begins. The translational efficiency of mRNA in bacteria differs by several orders of magnitude. Although the parameters that affect translation efficiency are poorly understood, the nature of the SD sequence has been shown to be one of them. This knowledge has been derived from a study of fortuitous or predetermined, site directed mutations in SD sequences. For example, a mutation is known that improves the potential for base pairing of mRNA with the 16S rRNA and which also enhances the translational efficiency. However, another mutation which reduced SD ASD complementarity between mRNA and rRNA from 8 to 4 bases did not affect translation efficiency, perhaps because the residual 4 bases were in the most favorable position for interaction with 16S rRNA. For a review of such mutations see Kozak, Microbiological Reviews Further, it is known that an increase in SD ASD complementarity between mRNA and rRNA from 4 bases to 13 bases, achieved through the use of a portable SD region, reduced the translational efficiency about two fold de Boer It is known to insert site directed mutations into DNA encoding the central domain of 16S rRNA Stark Present methods for the recombinant synthesis of proteins in cells harness preexisting cellular mechanisms. Typically, vectors such as plasmids are used to transfect host cells. The DNA in these vectors is replicated and transcribed into mRNA by the cellular polymerases, whereafter the mRNA is translated by an endogenous ribosome population that simultaneously is engaged in the synthesis of normal cellular proteins. Since the objective of recombinant protein synthesis is to obtain high yields of a particular desired protein, various techniques have been developed to attempt to force the host cell to focus on synthesis of the desired protein in preference to other host cell proteins. For example, inducible promoters have been linked to DNA encoding heterologous protein to enable one to activate transcription of the DNA by exposure of the cells to an inducing agent. Thus the host cells are permitted to become well established before transcriptional activation of the foreign DNA. This notwithstanding, the new transcripts must compete with host cell mRNA for a limited ribosome population which, in A method is needed that will focus the translational machinery of host cells on the synthesis of one particular desired protein, to the exclusion of other normal cell proteins. In addition, improved methods are needed for optimizing the interaction of ribosome and messenger RNA to achieve maximal translation efficiency in the synthesis of desired proteins in host cells. Further, methods are needed for modifying the 16S rRNA ASD sequence and its flanking regions in a predetermined fashion. These and other objects of this invention will be apparent from consideration of this specification as a whole. The present invention provides a method which comprises The key feature is that the mutant 16S rRNA ASD sequence and the mutant mRNA SD sequence are complementary. The mutant 16S rRNA ASD sequence is selected so that, to the maximum extent, it is substantially unable to bind to or initiate protein synthesis from endogenous cellular mRNA, and Ribosomes assembled after the active transcription of DNA encoding the mutant 16S rRNA ASD sequence are believed to be responsible for growth inhibition in host cells. This is ameliorated by placing the mutant rRNA under the control of an inducible promoter in expression systems. The method herein is facilitated by novel DNA encoding 16S ribosomal RNA which has a predetermined mutation within the last approximately 20 nucleotides at the 3 end of the RNA. This region of the rRNA includes the ASD sequence. More specifically, DNA is provided that encodes 16S ribosomal RNA, the RNA having as its 3 terminus GGA N DNA also is provided that encodes mRNA complementary to the mutant rRNA. This mRNA is unique in that it is cytosine rich, on the order of greater than about 25 mole percent of cytosine nucleotide. Fig. 1 depicts the construction of a vector for expressing DNA which encodes mutant mRNA SD sequences. Fig. 2 is a schematic illustration of the construction of a vector comprising DNA that encodes a mutant 16S rRNA ASD sequence and complementary mutant mRNA. Fig. 3 is a graph showing the synthesis of human growth hormone HGH by transformants with the vectors of Fig. 2. It shows that dedicated ribosomes are synthesized in transformants and that they are active in producing the target protein. The DNA encoding the 16S rRNA ASD sequence and that which encodes the mRNA SD sequence are both mutated in order to establish them as complementary to one another. Complementarity is defined traditionally, i.e., RNA sequences are complementary when they are capable of forming heteroduplex complexes by hydrogen bonding. For the purposes herein, G and C, A and U, and G and U are nucleotides that form complementary base pairs, and complementary sequences will contain about from 3 to 7 contiguous complementary nucleotides. It is not necessary, however, for all of the nucleotides in complementary sequences to base pair. Typically, the existence of several internal nucleotides that do not base pair will not prevent the formation of heteroduplexes, particularly if complementarity is permissive with respect to the nucleotides in question, e.g. U with G. It should be understood that the terms Shine Dalgarno SD and anti Shine Dalgarno ASD sequence are used in their functional rather than the usual compositional sense. In fact, the mutant SD and ASD sequences of this invention are largely the opposite complements of wild type SD and ASD sequences. However, since the mutants act properly together in initiating protein synthesis they are nonetheless termed SD and ASD sequences. The complementary mutant rRNA and mRNA SD and ASD sequences are of sufficient length and character to minimize binding to, and protein initiation on, the endogenous host messengers. This is determined by preparing a series of complementary mutants under the control of an inducible promoter as is described The 3 end of the 16S rRNA carries the mutant ASD sequence. The mutant ASD sequence desirably is positioned in the same location as the native sequence. However, the mutant sequences may be longer or shorter than the native sequence which is typically 5 nucleotides long . The mutant sequences are about from 3 to 15 nucleotides long of which uncomplementary nucleotides may be present at a ratio of less than about 1 in 5 nucleotides . Preferably, the mutant sequences are about from 4 to 6 nucleotides in length. It is not necessary that the mutant rRNA and mRNA sequences be substituted precisely at the same location as the native ASD and SD sequences. For example, the gram negative microbial 16S rRNA 3 terminus concensus sequence is UGGAUCA 16S rRNA is encoded by 7 genes in Inducible promoters are well known and publicly available. Examples include the lambda phage PL, metallothionein, and lac UV5 promoters that are induced, respectively, by increasing the culture temperature, or by adding copper ions or isopropyl B D.thio galactoside to the culture medium. Other suitable promoters recognizable by the intended host cell will be apparent to the ordinary artisan, e.g. the tac promoter de Boer The starting DNA encoding the desired protein product may be obtained from eukaryotic or prokaryotic sources. The protein need not even be heterogeneous to the host cell. The starting DNA typically is a cDNA including the native 5 untranslated region. Where the cDNA is the reverse transcript of a eukaryotic mRNA, no SD sequence will be present. If this DNA is to be expressed in prokaryotes it is preferred to remove the eukaryotic 5 untranslated region and substitute a prokaryotic 5 sequence, including an SD sequence. This sequence is conveniently supplied by The DNA sequences encoding mutant rRNA and mRNA preferably are located on the same vector. This permits selection for transformation by both DNA sequences using a single selection agent, e.g. ampicillin in the case of amp The host cells for use herein preferably are prokaryotes because knowledge relating to the protein initiation systems of prokaryotes is more advanced than in the case of eukaryotes, particularly higher eukaryotic cells such as mammalian cells. Suitable prokaryotes include gram negative organisms such as Where the rRNA containing the mutant ASD is under the control of an inducible promoter the host cell must be able to repress the inducible promoter, e.g. it must be able to synthesize sufficient quantities of a repressor protein. For example, the lac or tac promoter optimally is to be used with host strains known in the art It is not critical that the cell cultures transformed with inducible rRNA be induced at any particular point in the growth cycle, although preferably the cultures are induced as they enter the logarithmic phase. This enables the culture to accumulate sufficient endogenous ribosomes to synthesize for example the ribosomal proteins and polymerases needed to support the dedicated ribosome population. If erythromycin resistant 16S rRNA is employed, erythromycin should be added to the culture only after a substantial population of dedicated ribosomes has been synthesized. In order to simplify the Examples certain frequently occurring methods may be referenced by shorthand phrases. Plasmids are designated by a low case p preceded and or followed by capital letters and or numbers. The starting plasmids herein are commercially available, are publically available on an unrestricted basis, or can be constructed from such available plasmids in accord with published procedures. In addition, other equivalent plasmids are known in the art and will be apparent to the ordinary artisan. Digestion of DNA refers to catalytic cleavage of the DNA with an enzyme that acts only at certain locations in the DNA. Such enzymes are called restriction enzymes, and the sites for which each is specific is called a restriction site. Partial digestion refers to incomplete digestion by a restriction enzyme, i.e., conditions are chosen that result in cleavage of some but not all of the sites for a given restriction endonuclease in a DNA substrate. The various restriction enzymes used herein are commercially available and their reaction conditions, cofactors and other requirements as established by the enzyme suppliers were used. Restriction enzymes commonly are designated by abbreviations composed of a capital letter followed by other letters and then, generally, a number representing the microorganism from which each restriction enzyme originally was obtained. In general, and unless otherwise provided, about 1 µg of plasmid or DNA fragment is used with about 1 unit of enzyme in about 20 µl of buffer solution. Appropriate buffers and substrate amounts for particular restriction enzymes are specified by the manufacturer. Incubation times of about 1 hour at 37 C are ordinarily used, but may vary in accordance with the supplier s instructions. After incubation, protein is removed by extraction with phenol and chloroform, and the digested nucleic acid is recovered from the aqueous fraction by precipitation with ethanol. Digestion with a restriction enzyme infrequently is followed with bacterial alkaline phosphatase hydrolysis of the terminal 5 phosphates to prevent the two restriction cleaved ends of a DNA fragment from circularizing or forming a closed loop that would impede insertion of another DNA fragment at the restriction site. Unless otherwise stated, digestion of plasmids is not followed by 5 terminal dephosphorylation. Procedures and reagents for dephosphorylation are conventional T. Maniatis Recovery or isolation of a given fragment of DNA from a restriction digest means separation of the digest on polyacrylamide gel electrophoresis, identification of the fragment of interest by comparison of its mobility versus that of marker DNA fragments of known molecular weight, removal of the gel section containing the desired fragment, and separation of the gel from DNA. This procedure is known generally. For example, see R. Lawn Southern Analysis is a method by which the presence of DNA sequences in a digest or DNA containing composition is confirmed by hybridization to a known, labelled oligonucleotide or DNA fragment. For the purposes herein, unless otherwise provided, Southern analysis was used for this purpose and was accomplished by separation of digests on 1 percent agarose, denaturation and transfer to nitrocellulose by the method of E. Southern, 1975, J. Mol. Biol. Transformation means introducing DNA into an organism so that the DNA is replicable, either as an extrachromosomal element or chromosomal integrant. Unless otherwise provided, the method used herein for transformation of Ligation refers to the process of forming phosphodiester bonds between two double stranded nucleic acid fragments T. Maniatis DNA is prepared from transformants by isolating cloned plasmid DNA from microbial culture. Unless otherwise provided, the alkaline SDS method of Maniatis Oligonucleotides are short length single or double stranded polydeoxynucleotides which are chemically synthesized by known methods and then purified on polyacrylamide gels. All literature citations are expressly incorporated by reference. The following examples should be considered illustrative of the invention and not limiting thereof. This construction is depicted in Fig. 1. pHGH207 1 , a pBR322 derivative having a single Oligonucleotide PSDR IX was prepared synthetically The foregoing procedure was repeated with PSDR X. Here the mutant mRNA SD sequence GTGTG was a less radical departure from the consensus GGAGG SD sequence than the CCTCC sequence in PSDR IX. Obviously, the same procedure can be used to generate a large bank of mutants in which the SD sequence, its spatial placement with respect to the start codon and or the nucleotides in the SD flanking regions are varied by simply preparing different synthetic fragments. Plasmid pKK3535 is prepared in accord with known procedures Brosius Plasmid pKK3535 was treated using the heteroduplex mutagenesis process of Morinaga The numbers indicate the corresponding positions in rrnB of pKK3535 as given by Brosius The next steps have as their objective the introduction of mutant ASD regions which are complementary to the PSDR IX or PSDR X sequences in Example 1. One approximately 1µg aliquot of plasmid pKK3535ΔP1P2 was digested with wherein the dots indicate bases complementary to the 3 region of the DNA encoding the wild type 16S rRNA. The transcription of this DNA encoding the 3 end of the modified 16S rRNA gene in pKK3535ΔP1P2ASDIX yields rRNA containing the following sequence The foregoing procedure is repeated using the following mutagenesis primer denominated ASDX. The rRNA transcribed from the resulting pKK3535ΔP1P2ASDX includes the sequence Another mutant DNA encoding 16S rRNA having the ASD 5 CAUAC differing from ASDX by one base was obtained in the same fashion. This mutant was denominated pKK3535ΔP1P2ASDXI. None of these plasmids express the 16S rRNA so they are not lethal and can be stably maintained in Any one of pKK3535ΔP1P2ASDX, XI or IX was treated as follows. For convenience, the latter shall be described and is depicted in Fig. 2. Plasmid pKK3535ΔP1P2ASDIX was digested simultaneously with pHGH PSDRIX was digested simultaneously with pkC30Δnut also referred to as pNO2678, Gourse This oligonucleotide was not phosphorylated Kinased as the converter is self complementary phosphate is supplied by the vector cohesive end, and only the top strand ligates. The residual gap between the bottom strand and the vector upon ligation as described below is filled The converted 1300 bp fragment was ligated to Aside from pASDIX PSDRIX PL and pASDX PSDRX PL, a plasmid pASDXI PSDRX PL was obtained in the same way as described above. Here, the 16S rRNA ASD mutant will complement the mRNA SD mutant, but the complementarity is not the result of perfect base pairing. HGH was synthesized by A plasmid pASDX PSDRX obtained in Example 2 having the incorrect orientation of the λ PL promoter was used as a negative control mutant 16S rRNA will be made by transformants with this plasmid, and the mutant mRNA PSDRX is essentially unrecognized by the wild type host cell ribosomes. Similarly, the transcripts of pHGH PSDRX Example 1 are unrecognized by the host cell and thus serve as an additional negative control. Three positive controls were employed. These were pHGH 207 1 to serve, in effect, as a commercial control, pHGH 207 1 ASDwt PL and pHGH 207 1 ASDwt. Plasmid pHGH 207 1 ASDwT PL was made in the same fashion as pASDIX PSDRIX PL except that the 16S rRNA encoded the wild type ASD sequence CCUCC rather than ASDIX. Plasmid pHGH 207 1 ASDwt was identical to pHGH 207 1 ASDwt PL except that the λ PL promoter had not been inserted. These three positive controls were expected to result in HGH synthesis because the transfected plasmids contain the trp promoted wild type HGH gene, resulting in transcripts recognized by normal host cell ribosomes, but not to result in the generation of dedicated ribosomes. The results are depicted in Fig. 3. As expected, pASDX PSDRX and pHGH PSDRX generated substantially no HGH since the mutant SD was unrecognized by the host cell ribosomes and mutant 16S rRNA was unpromoted or its encoding DNA was completely absent. It is believed that pHGH 207 1 ASDwt PL, containing the 16S rRNA wild type ASD under the control of the PL promoter, resulted in less HGH than pHGH 207 1ASDwt or pHGH 207 1 because of the presence of the operative, strong PL operator and the large size of the vector. Typically, the presence of a strong vector promoter and a large vector leads to a relatively lower vector copy number per cell, and accordingly to a lowered synthesis of product. Nonetheless, since all three plasmids contain DNA encoding HGH and the consensus SD sequence under the control of the trp promoter, it was expected that HGH would be synthesized prior to induction of λPL. Study of the results for the plasmids bearing complementary or substantially complementary mutant SD ASD encoding sequences demonstrates that dedicated ribosomes are produced and that the dedicated system is effective in the synthesis of protein. Even after 300 minutes the amount of HGH is continuing to increase, while the amount of HGH synthesized from conventional plasmids using endogenous ribosomes has reached a stationary level. Further refinements of the method herein, including optimizing the location and sequence of the mutant SD ASD system as well as increasing the culture period, are expected to result in further improvements in protein synthesis.