# Specific iodination of nucleic acid.

## Abstract
A probe comprising a nucleic acid fragment covalently linked to an iodinatable residue. Such nucleic acid probes can be readily radiolabeled such as by prelabeling the nucleic acid with a nonradioactive iodinatable residue, and then reacting with radioactive iodine.

## Claims
1. A probe comprising a nucleic acid fragment covalently linked to an iodinatable residue.

## Description
This application relates to novel nucleic acid probes which can readily be radiolabeled. Synthetic oligonucleotides have been widely used for different purposes in molecular biology. They have been used to introduce and detect specific mutation, to isolate specific genes, and for many other biological purposes Gillam S., Jahnake, P., Astell, C., Phillips, S., Hutchinson, C.A., and Smith, M., Defined Transversion Mutations At A Specific Position In DNA Using Synthetic Oligodeoxyribonucleotides As Mutagens. The detection of specific mutation sites using an oligonucleotide has been used for diagnostic purposes. Conner, B.J. Reyes, A.A., Morin, C., Itakura, K., Teplitz, R.L. and Wallace, R.B., Detection Of Sickle β To date the most efficient method of labeling of an oligonucleotide probe has been the use of polynucleotide kinase to introduce a radioactive phosphorous isotope at the 5 terminus of the nucleic acid. This procedure involves the use of an enzyme which is not stable enough to give reproducible results over a period of time also it is limited to the use of a phosphorous isotope which has a very short half life. The use of other enzymes for labeling purposes has similar disadvantages. For example, if one uses terminal deoxy nucleotidyl transferase TdT to label an oligonucleotide, one cannot control the extent of labeling without the use of dideoxynucleoside triphosphate. After the labeling, the purification of the labeled material also will create some problem where there is an enzyme present in the system. One approach to this problem involves chemically labeling the end of an oligonucleotide or a nucleic acid probe by oxidative coupling of nucleoside phosphates or nucleosides and also by reaction with Bolton Hunter reagent Bolton, A.E., and Hunter, W.M., Several radioactive isotopes of iodine atom can be used to label nucleic acids for specific purposes Commerford, S.L., It is accordingly an object of the invention to provide nucleic acid probes which are radiolabeled in a way which avoids the aforementioned disadvantages. These and other objects and advantages are realized in accordance with the present invention pursuant to which there is provided a method of iodinating a nucleic acid by prelabeling the nucleic acid with a cold non radioactive iodinatable residue. The nucleic acid probe which is to be iodinated is first chemically modified to provide a residue which can be catalytically or enzy matically iodinated, leaving the nucleic acid bases intact. Moreover, the present invention also provides a fast and efficient method of separation of the later iodinated material from the starting material and the other pre cursors, advantageously using a column chromatographic device under high pressure. The nucleic acid probe of the present invention in its iodinated form can be used in nucleic acid hybridization, in particular in nucleic acid hybridization assays. It is known that lactoperoxidase or chloramine T can be used to iodinate certain aromatic residues e.g., tyrosine, hydroxyphenyl residue . The present invention makes use of such chemistry. The present invention is a two step reaction. The nucleic acid probe to be iodinated is first reacted with a reagent which will provide a residue which can be catalytically iodinated leaving the nucleic acid structurally and functionally intact. The probe comprises a nucleic acid, e.g., an oligonucleotide, fragment covalently linked to an iodinatable residue, e.g., an aryl radical, e.g., phenyl, naphthyl, anthracyl, and the like, optionally substituted with radicals which do not interfere with iodination, e.g., halo, OH, NH₂, CN, alkyl and the like. Preferred aryl radicals for the present invention include hydroxyphenyl radicals, indole radicals and imidazole radicals. The linkage of the nucleic acid fragment to the aryl radical can be via a CON radical. A probe according to the present invention has the general formula Nucleic Acid R NH CO X, wherein R is an aliphatic hydrocarbon radical of up to 11 carbon atoms, preferably between 3 and 6 carbon atoms and X is an unsubstituted aryl radical, e.g., phenyl, naphthyl, anthracyl, or an aryl radical substituted by a radical which does not interfere with iodination, e.g., halo, OH, NH₂, CN and alkyl. A particularly prefered X is Moreover, the present method uses a fast and efficient method of separation of the iodinated material from the starting material and the other precursors. The present invention also involves a method of separation of such iodinated material and their precursors from the starting material and other reactants. The method utilizes a chromotagraphic device with a hydrophobic bonded phase for a reverse phase separation. It is known that on a reverse phase column system one can separate molecules primarily on the basis of their hydrophobic property. In order to utilize that property of a nucleic acid probe one must be able to discriminate the labeled material from the unlabeled material on the basis of their hydrophobic character. The present invention concerns a high pressure liquid chromatography HPLC method of separation of an oligonucleotide from a derivatized oligonucleotide and from an iodine labeled, modified oligonucleotide. The invention is illustratively described in the following scheme. The invention involves reacting a nucleic having a primary or secondary amino radical with An example of this reaction is described below starting with an amine containing oligonucleotide Amine containing nucleotide residue can be an 5 allylaminodU, or 8 or 6 hexylaminodA or A, or a 5 allylamino C or C, oxidatively coupled amino containing A, G, T, C, U. The bond between NH₂ and the iodinatable residue can be N C, C C, C S, S S, C O. In order to create linkages other than N C , the NH₂ residue has to be properly modified with known reagents, e.g., reaction of RNH₂ with m maleimidobenzoyl N hydroxysuccinimide ester will provide a third reacting group to form a thioether C S C linkage. The nucleic acid probe is first modified or prepared in a way such that it will carry a primary amine residue which is reactive with a reagent which carries a catalytically iodinatable molecule such as sulfo succinimidyl hydroxyphenylpropionate. After covalent attachment of the hydroxyphenyl residue to the oligo nucleotide the hydroxyphenyl residue can be iodinated under mild conditions in presence of chloramine T, or lacto peroxidase, or similar reagents. An iodinatable residue is a radical which can be covalently modified with iodine under specific conditions, such that the rest of the molecule nucleic acid part will not change the property. Suitable iodinatable radicals comprise hydroxyaryl, preferably hydroxyphenyl, radicals such as Specifically, a nucleic acid with primary aliphatic amine residue e.g., allylamino U or hexyl amino A will react with N hydroxy succinimide ester of a protective amino acid. Such reactions with typrosine ester will provide After the reaction of such iodinatable reagents with the oligonucleotide the product has to be purified from the starting reactants in order to get a pure derivatized oligonucleotide for subsequent reactions. The hydroxyphenyl residue labeled nucleic acid probe is then purified on a reverse phase system and then iodinated in the presence of chloramine T or other catalysts. After iodination in the presence of chloramine T or other catalysts, the major radioactive impurity is iodine. In order to remove iodine from the iodinated nucleic acid and non iodinated nucleic acid a small reverse phase column can be used. Routinely this column consists of a wide pore silica 300Å coated with a C 18 phase Commercially available from Brownlee Labs, Inc., Santa Clara, CA. as Aquapore RP 300 . The size of the column is 3 cm x 0.46 cm I.D. and has the absorptive capacity to bind a minimum of 5µg oligonucleotide in aqueous solution. After the major amount of sodium ¹²⁵iodide is removed, the separation of the iodinated oligonucleotide from the non iodinated oligonucleotide can be effected on a simple analytical column using acetonitrile gradient. Once the iodinated material is separated from the non iodinated material, substantially all oligonucleotides in the fraction containing the iodinated material will be labeled with at least one radioactive isotope. After the iodinated material is separated from the rest of the materials, the sample is tested by hybridization with an unknown sample and a known sample and also by running gel electrophoresis. Although the whole procedure normally involves reaction with an iodinatable residue, separation, iodination, and separation, the whole procedure can be carried out within a couple of hours. Moreover, all intermediate products are stable, allowing a large scale production and storage of the derivatized oligonucleotides which can be labeled with ¹²⁵I whenever necessary. The process can be automated in whole or in part to save time and material, and to improve the purity of the product and the effectiveness of the process, by choosing proper concentration conditions and proper solvent and buffer. Thus, one can use commercially available immobilized chloramine T as a catalyst, so that after the iodination reaction the only impurity will be sodium iodide which can be removed very easily on a simple guard column. The invention is also illustratively described in the following examples wherein all parts are by weight unless otherwise expressed. Although the example is given with a specific oligonucleotide, similar procedure can be adapted for a ribo oligonucleotide, deoxyribo oligo nucleotide, a DNA, or RNA probes. The reaction can be carried out with double stranded DNA, single stranded DNA, double stranded RNA, single stranded RNA, or RNA DNA hybrid or mixed nucleic acid probes or with a simple di or mono nucleotide or nucleoside phosphates, which can be used as a substrate for different enzymes. All the nucleic acid samples, ribo, dioxy ribo or mixed oligo or poly nucleotides can be modified with a primary amine containing nucleotide triphosphate, an enzyme, such as TDT, which will result in a nucleic acid containing primary NH₂ residues. In turn this will react with activated esters of aromatic acids to provide aryl residues on the nucleic acids. An amine containing nucleoside phosphate will also react similarly to produce iodinatable residue containing enzyme substrates for nucleic acid modification. As catalysts immobilized or soluble chloramine T, lacto peroxidase and others can be used. Synthesis of a reactive amine containing oligonucleotide Scheme for the synthesis of 1 to be added to 19A at the 5 end. 5 Chloromercuri 2 deoxyuridine 5 , prepared according to the method of D.E. Bergstrom and J.L. Ruth, A 19 unit oligonucleotide HB19A 3 GA GGA CTC CTC TTC AGA CG 5 was prepared using a DNA synthesizer three separate 1 µmole batches of each oligonucleotide were made and each was attached to a solid support and fully protected by a dimethoxy trityl radical. The dimethoxytrityl protecting group was removed from the 5 terminus and 1 was attached to the 19 unit chain without the DNA synthesizer, but using the same reagents and conditions the machine typically employs. The product of this process, after removal from the support, is an oligonucleotide with a 5 3 aminoallyl 5 4,4 dimethoxytrityl 2 deoxy uridine unit at the C 5 end, viz., The process of attaching an analogue of Bolton and Hunter A.E. Bolton and W.H. Hunter, The polynucleotide HB19A is a 19 unit poly nucleotide corresponding to a portion of human DNA which codes for the polypeptide β hemoglobin, specifically that region of the DNA wherein lies the mutation which manifests itself in the formation of sickle cell hemoglobin and the genetic disorder known as sickle cell anemia. Infrared IR spectra were obtained as solutions in CHCl₃ unless otherwise noted the 1602 cm ¹ band of polystyrene film was used as an external calibration standard. Proton magnetic resonance ¹H NMR spectra were obtained in CDCl₃ solution unless otherwise noted. Chemical shifts are reported in parts per million downfield from the internal standard tetramethylsilane, unless otherwise noted. Carbon 13 magnetic resonance ¹³C NMR spectra were obtained in CDCl₃ solution unless otherwise noted. Carbon shifts are reported in parts per million downfield from the internal standard tetramethyl silane, unless otherwise noted. Phosphorus 31 magnetic resonance ³¹PNMR spectra were obtained in CDCl₃ solution unless otherwise noted. Phosphorus shifts are reported in parts per million downfield from an external aqueous 15 H₃PO₄ stardard. Thin layer chromatograph TLC was performed using silica gel 60F 254 plates from E. Merck. Column chromatography was performed using E. Merck Silica Gel 60 70 230 mesh . A suspension of 5 chloromercuri 2 deoxyuridine 5 Bergstrom and Ruth, A solution of 8 0.60 g 1.58 mmol in anhydrous pyridine 8 ml was maintained under an inert gas atmosphere and treated at ambient temperature with 4,4 dimethoxytrityl chloride 0.67 g 1.25 equivalents . After stirring for 18 hours the reaction was poured into ice water 70 ml with vigorous shaking. On standing 1 3 hours at 0 C, a gummy solid separated, leaving a nearly clear solution which was decanted. The solid was washed once with H₂0 5 ml then taken up in CH₂Cl₂ 10 ml , washed once with brine 5ml then the CH₂Cl₂ solution was dried over K₂CO₃, filtered and evaporated to dryness A solution of 9 0.34 g 0.5 mmol in anhydrous CH₂Cl₂ 1.5 ml maintained under an Argon atmosphere at ambient temperature was treated first with anhydrous, diisopropylethylamine 0.35 ml 0.259 g 2mmol 4 equiva lents then dropwise, over 1 minute with N, N diisopropyl aminomethoxy chlorophosphine see McBride et al, The 19 unit oligonucleotides were synthesized using an Applied Bio systems Model 380A DNA Synthesizer on control pore glass solid support. Immediately prior to attaching 1 to the 5 end of the oligomer, the 5 0 4,4 dimethoxytrityl protecting group was cleaved on the machine with 3 CCl₃CO₂H in CH₂Cl₂ for 90 seconds. The support bound 5 deprotected oligomer was washed with CH₃CH and dried in an Argon stream. Subsequent steps were performed without the machine, but using the same chemistry The above ten steps were repeated for all batches of support bound oligonucleotide. Treatment of a portion of each on a silica gel TLC plate with 3 CCl₃CO₂H in CH₂Cl₂ produced the orange red color of dimethoxytrityl cation indicating the successful incorporation of 1 into the oligonucleotides. One batch of the modified HB19A oligonucleotide was detritylated with 3 CCl₃CO₂H in CH₂Cl₂, then purified by electrophoresis on polyacrylamide gel. Two micrograms of 19A amine from Example 1 were dissolved in 20 microliters of 10 mM borate buffer pH 8.16. To this, 5 microliters of a freshly prepared solution of sulfosuccinimyl 3 4 hydroxyphenyl propionate SHPP purchased from Pierce were added. The concentration of SHPP was 5 mg per ml in water. The reaction was allowed to proceed at room temperature for 30 minutes then it was kept in a freezer before HPLC separation was conducted. After the reaction, the extent of the reaction was monitored by running a 20 polyacrylamide gel in conventional manner. HPLC separation was conducted on a Brownlee RP300 guard column coupled to a Synchropack RPP 4.1 x 10 cm Synchrom Linden, Indiana column at ambient temperature with a gradient of .1 M triethylammonium acetate pH 7 to 0.1 M triethylammonium acetate pH 7, 50 acetonitrile was run over a period of 10 120 minutes depending on the sample. The detector was set at 254 nm and full scale was 0.15 absorbance unit. In order to determine the location of the derivatized oligonucleotide product, a blank run had been carried out with the reaction mixture without the oligonucleotide. A new peak appeared after adding the oligonucleotide and corresponded to the reaction product. After the product was separated and collected in a fraction collector, the product was analyzed by gel electrophoresis and from the next run an analytical determination of the proper peak was not deemed necessary. The yield and recovery of the product from the columns was between 80 to 90 . The HPLC fraction containing the SHPP labeled oligonucleotide was evaporated under reduced pressure and then was redissolved in 5 microliters of water. The 5 microliter aqueous solution containing 2 microgram of the SHPP labeled oligonucleotide was diluted to 50 microliter with phosphate buffer pH 7.1. The buffer contained 100 mmoles disodium hydrogen phosphate and 50 mmoles sodium dihydrogen phosphate. Then one microliter of 0.1 normal hydrochloric acid was added to this 55 microliter solution followed by one microliter of ¹²⁵I, New England Nuclear, 400 microcuries and one Iodobead immobilized chloramiue T from Pierce are added. The reaction was continued for 15 minutes in ice water. The reaction was terminated by removing the Iodobead, and injection of the solution onto a reverse phase chromatographic system. The mixture containing the oligonucleotide and the iodination reagents was diluted with triethylammonium acetate buffer then they are loaded onto a guard column Example 3 . First the column was washed with 0.1 molar triethylammonium acetate to remove sodium iodide and any other components which were not absorbed onto the reverse phase column. After the first large radioactive peak which did not contain any oligonucleotide eluted, the guard column was put in series with an analytical column Sychropack RP P and the gradient of 0 to 50 acetonitrile in 0.1 M triethylammonium acetate was run through the system. In a typical run the uniodinated material eluted prior to the iodinated SHPP labeled oligonucleotide. After the peaks were collected they were again analyzed on gel electrophoresis. This analysis was not necessary after establishing the chromatographic properties. The specific oligonucleotide fragment chosen for this example was usable for the detection of sickle cell anemia. This particular sequence of oligonucleotide 5 U GCA GAC TTC TCC TCA GGA G 3 U 5 allyamino U will hybridize with DNA extracted from normal blood, whereas under stringent conditions it will not hybridize to the DNA extracted from the blood of a person who has sickle cell anemia disease. The extraction of DNA from blood samples and the digestion with restriction enzymes and other treatment of the hybridization processes have been described in detail in a paper by Conner, B.J. et a., The material iodinated by the present method can be used for the detection of sickle cell anemia. It will be appreciated that the instant specification and claims are set forth by way of illustration and not limitation, and that various modifications and changes may be made without departure from the spirit and scope of the present invention.