# VACCINIA DNA

## Claims
Eine Transkriptionsregulations Sequenz mit der Formel

## Description
Vaccinia virus is the prototype for the orthopoxvirus of the poxvirus family. Its biology and replication has been described extensively by B. Moss Poxviruses , in Comprehensive Virology, eds. H. Fraenkel Conrat and R. Wagner, Plenum Press, New York, Vol. 4, pp. 405 474 1974 Poxviruses , in Molecular Biology in Animal Viruses, ed. D.P. Nayak, Marcel Dekker, New York, Vol. 2, pp. 849 890 1978 5 end labelling of RNA with capping and methylating enzymes , in Gene Amplification and Analysis, eds. J. G. Chirikjian and T. S. Papas, Elsevier, North Holland, Vol. 2, pp. 254 266 1981 Principles of virus replication poxvirus , in Human Viral Diseases, eds. B. N. Fields, R. Chanock, R. Shope and B. Roizman, Raven Press, New York, in press 1984 . While several types of animal DNA viruses with large genomes have been used as cloning vectors, including adenovirus, herpes simplex virus and vaccinia virus only recombinants of the latter have expressed foreign genes while retaining complete infectivity. Meanwhile great experience has been gained with the use of vaccinia virus as a live vaccine. Its wide host range, large capacity for foreign DNA and inability to induce oncogenic transformation are all features enhancing the potential of vaccinia virus recombinants as live vaccines. An updated review of the use of recombinant vaccinia viruses as live vaccines has been given by G. L. Smith et al. Biotechnology and Genetic Engineering Reviews 2, 383 407, 1984 , including a description of the biology of recombinant vaccinia viruses and the expression of foreign genes under the control of vaccinia promoters. The present invention deals with vaccinia DNA preferably transcriptional regulatory sequences from the 5 flanking region of the vaccinia virus late gene encoding a basic polypeptide with a molecular weight of 11000 11 kDa , recombination vectors useful for the insertion of foreign genes into poxvirus, recombinant infectious poxviruses containing foreign genes operatively linked to such transcriptional regulatory sequences and capable of effecting expression of the corresponding polypeptide and live vaccines on the basis of such recombinant infectious poxviruses. Preferred poxviruses, used in this invention are vaccinia viruses. The gene coding for a major late 11 kDa structural polypeptide of the vaccinia virus which has been mapped by R. Wittek et al. J. Virol. 49, 371 378 1984 has been sequences including its 5 flanking region. The DNA sequence of above gene and derived amino acid sequence is indicated below The 5 flanking region shows little homology to either the corresponding region of vaccinia early genes Weir, J. P. and B. Moss, J. Virol. 51, 662 669 1984 or to consensus sequences characteristic of most eukaryotic genes. Furthermore, it has been found that a DNA fragment of not more than about 100 base pairs from the 5 flanking region of the 11 kDa gene contains all necessary transcriptional regulatory signals for correct regulation of vaccinia virus late gene expression. Therefore, the present invention comprises a transcriptional regulatory sequence of the following formula or subunits fragments thereof which are capable of regulating expression of foreign genes. In addition it has been found that a DNA fragment of not more than 13 base pairs counted from position 2 at the 3 end of above transcriptional regulatory sequence constitutes a preferred element for correct regulation of vaccinia virus late gene expression. Therefore such 3 end transcriptional regulatory sequences fragments also belong to the present invention and are encompassed by the present application. Furthermore, the invention comprises variations of any of the foregoing transcriptional regulatory sequences including deletions, insertions, substitutions, inversions of single or several nucleotides and combinations thereof capable of functioning as a poxvirus late promoter functional variations . Specific examples of functional variations of the above mentioned transcriptional regulatory sequences are represented by the following formulas The invention also encompasses tandem repeats, e.g. from 2 to 5 times, of the above mentioned transcriptional regulatory sequences. The invention further comprises recombination vectors containing a chimeric gene consisting of at least one transcriptional regulatory sequence of the vaccinia major late 11 kDa gene operatively linked to a foreign gene encoding prokaryotic or eukaryotic polypeptides, DNA from a non essential segment of the poxvirus genome flanking said chimeric gene, the vector origin of replication and antibiotic resistence genes. The translational initiation site of the chimeric gene is provided either by the transcriptional regulatory sequence of the vaccinia major 11 kDa gene or by the foreign gene encoding prokaryotic or eukaryotic polypeptides. By using the translational initiation site of the foreign gene codon phasing and potential problems associated with biological activity of fusion proteins are avoided. Preferred recombination vectors using the translational initiation site of the transcriptional regulatory sequence of the vaccinia major 11 kDa gene or the translational initiation site of the foreign gene are described in the examples 1 4. The recombination vectors of this invention can be constructed by methods well known in the art D. Panicali and E. Paoletti, PNAS 79, 4927 4931 1983 D. Panicali et al., PNAS 80, 5364 5368 1984 G. L. Smith et al., supra M. Mackett et al., J. Virol. 49, 857 864 1984 comprising the steps of Intermediate recombination vectors comprising at least one transcriptional regulatory sequence of the vaccinia major late 11 kDa gene and still lacking a foreign gene encoding prokaryotic or eukaryotic polypeptides are also an object of the present invention and can be prepared by the above step a wherein the transcriptional regulatory sequence of the vaccinia major late 11 kDa gene includes the translational inition site of the 11 kDa gene or, optionally, is terminated in the region between the mRNA start and the translational inition site of the 11 kDa gene. The vector used to assemble the recombination vector may be any convenient plasmid, cosmid, or phage. Convenient vehicles of plasmid, cosmid or phase origin are mentioned e.g., in the laboratory manual Molecular Cloning by Maniatis et al., Cold Spring Harbor laboratory, 1982. Preferred vectors of plasmid origin, used to assemble the recombination vectors in this invention are pBR322 and pUC8. The DNA used to flank the chimeric gene may be derived from non essential regions of the poxvirus genome. Examples of such non essential regions include the thymidine kinase TK gene J. P. Weir and B. Moss, J. Virol. 46, 530 537 1983 . The preferred non essential regions used in this invention comprise a segment of the poxvirus thymidine kinase gene and DNA adjacent to said thymidine kinase gene. Especially preferred is a segment of the vaccinia virus thymidine kinase gene and vaccinia DNA adjacent to said vaccinia thymidine kinase gene. the preparation of the non essential regions is described more in detail in example 1. Foreign genes that may be inserted into the recombination vectors of this invention may be selected from a large variety of genes DNA genes or DNA copies of RNA genes that encode prokaryotic or eukaryotic polypeptides. For example, such genes may encode enzymes, hormones, polypeptides with immuno modulatory, anti viral or anti cancer properties, antibodies, antigens, and other useful polypeptides of prokaryotic or eukaryotic origin. Preferred foreign genes used in this invention are the genes encoding malaria antigens, in particular the 5.1 surface antigen of Plasmodium falciparum Hope, I.A. et al., infra and the mouse dihydrofolate reductase DHFR gene A.C.Y. Chang et al., Phenotypic expression in E. coli of a DNA sequence coding for mouse dihydrofolate reductase , Nature 275, 617 624 1978 . Plasmids of the pHGS family are specific examples of plasmid recombination vectors of the present invention. Their preparation is described more in detail in examples 1 4. E. coli strains containing plasmids useful for the preparation of the recombination vectors of the present invention E. coli HB 101 transformed with pHGS 1 pHGS 2 were deposited at Deutsche Sammlung von Mikroorganismen DSM in Göttingen on February 21, 1985 the accession nos. being DSM 3248 and DSM 3249 respectively. Other gram negative hosts such as E. coli C 600 E. coli DH 1 and E. coli RR1 ATCC No. 31343 can also be used and are described in the laboratory manual Molecular Cloning by Maniatis et al., supra. Suitable recombinant infectious poxviruses containing and expressing the above mentioned chimeric genes can be obtained by methods well known in the art G.L. Smith et al., Biotechnology and Genetic Engeneering Reviews 2, 383 407 1984 M. Mackett et al., supra comprising the steps of Suitable eukaryotic host organisms, which can be used for the manufacture of a recombinant infectious poxvirus include CV 1, RK 13, TK 143, or other cells. The preferred eukaryotic host cell used in this invention is RK 13. Recombinant infectious vaccinia viruses RVV 1 through 8 are specific examples of the present invention. Details of their preparation and isolation are indicated in examples 1 4. Examples of proteins which can be expressed by using the recombinant infectious vaccinia viruses of the present invention are mouse dihydrofolate reductase, chloramphenicol acetyltransferase and malaria surface antigens, in particular the 5.1 surface antigen of Plasmodium falciparum. Methods for expressing chimeric genes encoding prokaryotic or eukaryotic polypeptides using recombinant infectious poxviruses are well known. G.L. Smith et al., supra M.P. Kieny et al., Nature 312, 163 166 1984 G.L. Smith et al., Science 224, 397 399 1984 E. Paoletti et al., PNAS 81, 193 197 1984 D. Panicali et al., supra . They include infecting an appropriate host with a recombinant infectious poxvirus having the desired foreign gene operatively linked to the poxvirus transcriptional regulatory sequence, incubating the host under appropriate conditions and detecting the desired polypeptide by immunological, enzymatic, and electrophoretic methods. The recombinant infectious poxviruses can therefore be used as live vaccines by inoculating an animal or human with an inoculant containing a concentration of said recombinant infectious poxvirus sufficient to elicit an immunological response in said animal or human comprising the production of antibodies to at least the antigenic portion of the protein encoded by said foreign gene. Preferred recombinant infectious poxviruses used in this invention for protective immunization are recombinant infectious vaccinia viruses. Especially preferred are those expressing malaria surface antigens, in particular the 5.1 surface antigen of Plasmodium falciparum. Recombinant infectious vaccinia viruses for use in man can be prepared as described by C. Kaplan Br. med. Bull. 25, 131 135 1969 . Preparations suitable for vaccination must contain 10⁶ to 10⁸ plaque forming units per 0.05 ml. The vaccine can be stored frozen in aqueous buffer solution containing 40 50 glycerol, or in lyophilized form. This lyophilized form of the vaccine is essential for use in underdeveloped areas. Vaccination is achieved as a result of intradermal inoculation. A drop of the vaccine is applied to a small sterlized area of the skin and the epidermis beneath is then rapidly punctured by means of a sharp sterile needle or Knife. In mass vaccination compaigns jet guns are used. Having now generally described this invention, the same will be better understood on the basis of the following examples when considered in connection with the following figures The following methods were performed as described by Maniatis et al., supra, unless indicated differently Restriction endonuclease digestions at 37 C pp. 100 101 dephosphorylation with bacterial alkaline phosphates BAP at 37 C pp. 133 134 ligation with T4 DNA ligase at 14 C pp. 390 391 transformation of DNA into CaCl₂ cells of E.coli HB101 and selection of tranformants on agar plates containing LB medium plus 100 µg ml of ampicillin pp. 250 251 DNA plasmid preparation pp. 86 94 filling in single stranded DNA tails with the large fragment of DNA polymerase I Klenow fragment at 14 C pp. 113 114 DNA separation and fragment purification from agarose gels pp.164 167 the use of synthetic DNA linkers in subcloning pp. 392 397 removal of single stranded DNA tails with nuclease S1 at room temperature p. 140 isolation of mRNA from mammalian cells pp. 191 193 nuclease S1 mapping of mRNA pp. 207 209 sequencing of DNA by the Maxam Gilbert technique pp. 475 478 . Ten µg of Vaccinia virus VV DNA WR strain were digested to completion with 100 units of the restriction endonuclease Hind III. The Hind III J fragment approx. 5 kb containing the VV thymidine kinase TK gene Weir, J.P. and Moss, B., J.Virology 46, 530 537 1983 and F fragment approx. 14 kb were isolated from agarose. One µg of the plasmid pBR 322 J.G. Sutcliffe, Complete nucleotide sequence of the Escherichia coli plasmid pBR 322 , Cold Spring Harbor Symp. Quant.Biol., 43, pp 77 90 1979 was digested with one unit of the restriction endonuclease Hind III to completion, free ends were dephosphorylated with one unit of bacterial alkaline phosphatase BAP for 1 h. Twenty ng of Hind III linearized pBR 322 and 100 ng of the Hind III J fragment respectively F fragment were ligated with one unit of T4 DNA ligase, transformed into E.coli HB101 cells. In each case four transformants resistant to ampicillin were selected and cultures grown in LB medium containing 100 µg ml ampicillin. DNA from these cultures was isolated using standard procedures and analyzed for its size and presence of sites for restriction endonucleases Hind III, EcoRI and Kpn I in the case of the J fragment and Hind III, Sal I and Bgl I in the case of the F fragment. One plasmid displaying the expected patterns after electrophoresis on agarose gels was designated pBR J resp. pBR F. Two µg of the plasmid pBR J were digested to completion with 5 units of the restriction endonuclease Hpa II and single stranded DNA tails were filled in with 4 units of the large fragment of DNA polymerase I Klenow fragment . The DNA was subsequently digested to completion with 5 units of the restriction endocuclease Hind III, DNA separated on 0.8 agarose gel and a DNA fragment of 1310 bp was isolated from the agarose gel. Two µg of the plasmid pUC 8 J.Vierra and J.Messing. The pUC plasmids, an M13mp7 derived system for insertion mutagenesis and sequencing with synthetic universal primers , Gene 19, pp 259 268 1982 were digested to completion with 5 units of the restriction endonuclease EcoRI and single stranded DNA tails were removed with 2 units of nuclease S1 for 1 h at RT in S1 buffer. This DNA was subsequently digested to completion with 5 units of the restriction endonuclease Hind III, free ends dephosphorylated with 2 units of BAP for 1 h, the DNA was separated on a 0.8 agarose gel and a fragment of 2.7 kb was isolated from agarose as previously described. Fifty ng of this pUC 8 fragment were ligated with 200 ng of the purified 1310 bp VV fragment using one unit of T4 DNA ligase and the ligated DNA transformed into HP101. Eight transformants resistant to ampicillin were selected and cultures grown in LB medium containing 100 µg ml ampicillin. DNA from these cultures was isolated using standard procedures and analyzed for its size and presence of sites for restriction endonucleases Hind III, EcoRI and Xba I. One plasmid displaying the expected patterns after electrophoresis on agarose gels was designated pUC TK. Fifty µg of the plasmid pBR F1 were digested to completion with 200 units each of the restriction endonucleases EcoRI and Xba I, the DNA was separated on a 1.2 agarose gel. A DNA fragment of 104 bp was isolated, single stranded DNA tails were filled in with 4 units of Klenow fragment and synthetic EcoRI linkers CGAATTCG were attached. This new DNA fragment with Eco linkers 110 bp consist of the 11kDa transcriptional regulatory sequence of VV. Two µg of the plasmid pUC TK were digested to completion with 5 units of the restriction endocuclease EcoRI, free DNA ends were dephosphorylated with 2 units of BAP, the DNA was separated on a 0.8 agarose gel and the linear DNA fragment of 4 kb was isolated and extracted. Fifty ng of the EcoRI linearized pUC TK vector were ligated using 1 unit of T4 DNA ligase with 10 ng of the 110 bp promoter fragment and transformed into HB101. Eight transformants resistant to ampicillin were selected and cultures grown in LB medium containing 100 µg ml ampicillin. DNA from these cultures was isolated and analyzed for its size and presence of sites for restriction endonucleases EcoRI and Hph I. One plasmid displaying the expected patterns after electrophoresis on 6 acrylamide gels was designated pUC TK 11kDa. Ten µg of pUC TK 11kDa were digested partially up to 5 with one unit of the restriction endonuclease EcoRI at 37 C for 30 min, single stranded DNA tails were filled in with 4 units of Klenow fragment at 14 C. The DNA was separated on a 0.8 agarose gel and the linearized form of the pUC TK 11kDa plasmid 4104 bp was isolated and 10 ng were self ligated using 2 units of T4 DNA ligase and transformed into HB101. Sixteen transformants resistant to ampicillin were selected and cultures were grown in LB medium containing 100 µg ml of ampicillin. DNA from these cultures was isolated and analyzed for its size and presence of sites for restriction endonucleases EcoRI, Xmn I and Hind III. Plasmids displaying the expected patterns after electrophoresis on agarose gel were selected and the Cla I EcoRI fragment of 148 bp was sequenced. One plasmid displaying the sequence as indicated on page 3 preceeding and including the site for the restriction endonuclease EcoRI was designated pHGS 1. RK 13 cells adapted to 33 C were infected with 0.1 plaque forming units pfu per cell of the vaccinia virus temperature sensitive mutant t Ten µg of the plasmids pHGS 1, were partially digested up to 10 with 2 units of the restriction endonuclease Taq I. The DNA s were subsequently digested to completion with 20 units of the restriction endonuclease Hind III separated on an agarose gel and a DNA fragment of 1135 bp was isolated. Hundred ng of this fragment were ligated with 1 unit of T4 DNA ligase with 25 ng of the plasmid pBR 322 digested to completion with 1 unit each of the restriction endonucleases Hind III and Cla I and transformed into HB101. Four transformants resistant to ampicillin were selected and cultures grown in LB medium containing 100 µg ml of ampicillin. DNA from these cultures was isolated and analyzed for its size and presence of sites for restriction endonucleases Hind III, EcoRI and Cla I. Plasmids displaying the expected patterns on agarose gels were selected and used to prepare nuclease S1 DNA probes as followed Ten µg of the plasmid were digested to completion with 20 units each of the restriction endonucleases Cla I and Hind III, free ends were dephosphorylated with 10 units of BAP, a DNA fragment of 1.6 kb was isolated and 0.1 pmol was labelled before use with 1 unit of polynucleotide kinase PNK and 1 pmol of γ ³²P deoxyadenosine triphosphate, and the DNA was subsequently digested to completion with 1 unit of the restriction endonuclease RsA I. Five µg of the plasmid pUC TK were digested to completion with the restriction endonuclease ClaI, free ends were dephosphorylated with 5 units BAP, the DNA subsequently digested to completion with the restriction endonuclease Hind III, the DNA was separated on 0.8 low melting agarose and a DNA fragment of 737 bp was isolated. The S1 probe was labelled with PNK and γ ³²P deoxyadenosine triphosphate as described. Two petridishes 5 cm Ø with a confluent monolayer of RK 13 cells were infected each with 10 pfu per cell of the recombinant virus RVV 1 and after 3 respectively 7 h the RNA was extracted. The RNA s were resuspended in 100 µl of 0.2 SDS, divided into two equal parts and 20 000 cpm of the ³²P labelled S1 probe for the TK gene resp. 11KDa transcriptional regulatory sequence were added and the different mixtures were precipitated with 1 10 volume of NaOAc and 2 1 2 volumes of ethanol. The pellet was redissolved in 50 µl of S1 hybridisation buffer, heated for 3 minutes at 100 C and incubated for 16 hours at 42 C before addition of 450 µl of one times nuclease S1 buffer and 25 units of the enzyme nuclease S1. The digestion was stopped after one hour with EDTA final concentration at 50 mM and SDS final concentration of 0.5 . Sodium hydroxide was added to a final concentration of 0.25 M and after 1 hour at 45 C the mixture was neutralized to pH 7 using 3M HOAc. The residual DNA was precipitated using 5 µg of tRNA as a carrier, washed with 80 ethanol, dissolved in 20 µl formamide loading buffer and 10 µl were loaded on a 8 acrylamide 8M urea sequencing gel. The gel was exposed for 24 hours using Kodak x ray film and an intensifying screen. The x ray exposure is shown in Fig.6. At 3 hours post infection, only transcripts from the TK promoter can be detected slot 1 , whereas at 7 hours after infection a strong S1 protected band can be detected starting at the inserted 11kDa promoter 260 bp and only trace amounts of the TK transcripts. Therefore, it can be concluded that the inserted 11kDa regulatory sequence is functioning late after infection. Three µg of the plasmid pHGS 1 were digested to completion with 10 units of the restriction endonuclease EcoRI, the DNA was incubated at 37 C with 1 unit of T4 DNA polymerase in T4 polymerase buffer Maniatis et al., supra with deoxythymidine triphosphate at a final concentration of 100 µM. After 30 min the reaction was stopped by heating at 65 C for 10 min. The mixture was diluted 10 fold with nuclease S1 buffer and single stranded DNA tails were removed with 2 units of nuclease S1 for 1 hour. Synthetic Bam HI linkers 5 AAAGGATCCTTT 3 were ligated to the blunt DNA ends and 100 ng of the DNA was self ligated with 1 unit of T4 DNA ligase and digested with 10 units of the restriction endonuclease EcoRI for 1 hour and transformed into HB101. Eighty transformants resistant to ampicillin were selected and cultures grown in LB medium containing 100 µg ml of ampicillin. DNA from these cultures was isolated and analyzed for its size and presence of a site for restriction endonuclease Bam HI. DNA s containing a site for a restriction endonuclease Bam HI were selected. From each of these DNA s 20 µg were digested to completion with 50 units each of the restriction endonucleases Bam HI and Cla I. The DNA s were separated on acrylamide gels and DNA fragments of 156 resp. 141 bp were isolated and sequenced using the Maxam Gilbert technique after labelling the fragments at the site of the restriction endonuclease Bam I using 1 unit of Klenow fragment and α ³²P deoxyguanidine triphosphate. DNA s were selected having the following sequence preceeding the sequence 5 AAAGGATCC 3 representing the Bam HI linker Five µg of the DNA s having the sequence No 1 resp. 2 were digested to completion with 10 units each of the restriction endonucleases Bam HI and Hind III. The DNA was separated on an agarose gel and DNA fragments of 888 resp. 873 were isolated, and 100 ng each were ligated with a 50 molar excess each of the synthetic DNA fragments 5 GATCCCCGGG 3 and 5 AATTCCCGGG 3 using 1 unit of T4 DNA ligase. The DNA was subsequently digested with 300 units of the restriction endonuclease EcoRI for 8 h, loaded on a 6 acrylamide gel and fragments of 898 resp. 883 were isolated. The recombinant viruses RVV 2 resp. RVV 3 were constructed as described before using the plasmids pHGS 2 resp. pHGS 2Δ15. In each case one virus was selected for the presence of the mutated 11kDa transcriptional regulatory sequence No 1 resp. 2 inserted into the TK gene by blot hybridization Mackett et al., supra . The preparation of the nuclease S1 probes for transcripts of the transcriptional regulatory sequence No 1 resp. No 2 inserted into the TK gene was as described in example 1 section F.a using the plasmids pHGS 2 resp. pHGS 2Δ15. RNA transcripts 4 resp. 8 h after virus infection were prepared as described previously using the viruses RVV 2 resp. RVV 3 and the S1 probes for RNA transcripts from the TK promoter resp. 11kDa transcriptional regulatory sequences containing sequences No 1 resp. 2 . The x ray exposure is shown in Fig.7. At 4 hours post infection slot 1 and 4 only transcripts from the TK promoter can be detected. These transcripts are undetectable at 8 hours post infection, whereas cells infected with the virus RVV 2 pHGS 2 inserted a strong S1 protected band can be detected. However with RVV 3 pHGS 2Δ15 inserted no S1 protected band appears, demonstrating that the deletion of 15 basepairs 5 from the ATG results in an inactivation of transcripts starting from the mutated transcriptional regulatory sequence. Hundred ng of a Plasmodium falciparum cDNA fragment with flanking EcoRI linkers, containing the merozoite 5.1 surface antigen of Plasmodium falciparum Hope, I. A. et al., Nucleic Acids Research, 13, 369 379 1985 were ligated using 1 unit of T4 DNA ligase into 50 ng of pHGS 2 resp. pHGS 2Δ15, digested to completion with the restriction endonuclease EcoRI, free ends were dephosphorylated using 1 unit of BAP. The ligated DNA s were subsequently transformed into HB101 and in each case four transformants resistant to ampicillin were selected and cultures grown in LB medium containing 100 µg ml ampicillin. DNA from these cultures was isolated and analyzed for its size and presence of sites for restriction endonucleases EcoRI and Hinc II. In each case one plasmid displaying the expected patterns after electrophoresis on agarose gels was designated pHGS 2 5.1 resp. pHGS 2Δ15 5.1. The recombinant viruses RVV 4 and RVV 5 were constructed and selected as described before using the plasmids pHGS 2 5.1 and pHGS 2Δ15 5.1 RK 13 cells were grown upto 80 confluency and infected with 0.1 plaque forming units pfu per cell of recombinant vaccinia virus RVV 4 resp. RVV 5 with the malaria 5.1 Antigen stabely integrated in the virus genome through homologous recombination. One hour after infection the cells were washed with phosphate buffered saline PBS and fresh medium was added to the cells. Infection with the virus was allowed to continue for additional 16 hours. Cells were scraped of the culture dishes, harvested through centrifugation at 2000xg and washed once with PBS. Approximately 10⁴ cells were spotted on a microscope glass, air dried, fixed with 20 C acetone for 10 min and again air dried. The fixed cells were incubated at 37 C for 20 min with rabbit anti 5.1 antigen diluted in PBS, subsequently washed 3 times with PBS and again air dried. Next the fixed cells were incubated at 37 C for 20 min with goat anti rabbit FITC serum diluted with PBS, washed 3 times with PBS, once with distilled water and air dried. PBS glycerin 1 1 was spotted on the cells and covered with a microscope cover glass. The cells were analyzed for their fluorescence under a microscope at 450 490 nm. The result is shown in Fig.9. Panel A shows fluorescent cells infected with the virus RVV 4 and panel B infected cells in visible light. Cells infected with the virus RVV 5 did not show any fluorescence, demonstrating that deletions upstream of the 11kDa ATG causes inactivation of the regulatory sequences. Removal of the G residue of the ATG however has no effect on transcription or translation. Two µg of the plasmid pHGS 2 were digested to completion with 2 units each of the restriction endonucleases Bam HI and EcoRI. The DNA was separated on an 0.8 agarose gel and the Bam HI EcoRI vector was isolated 4 kb . Two µg of the plasmid pDS 1, to 1 deposited at Deutsche Sammlung von Mikroorganismen DSM in Göttingen on December 11, 1984, accession no. DSM 3135 were digested to completion with two units each of restriction endonucleases Bam HI and EcoRI and a fragment of approximately 920 bp containing the mouse dihydrofolate reductase DHFR gene was isolated. Twenty µg of the Bam HI and EcoRI digested vector pHGS 2 and 100 µg of the Bam HI EcoRI fragment containing the mouse dihydrofolate reductase gene were ligated using 1 unit of T4 DNA ligase and the DNA was transformed into HB 101. Eight transformants resistant to 100 µg ml of ampicillin were selected and cultures grown in LB medium containing 100 µg ml of ampicillin. DNA from these cultures was isolated and analyzed for its size and presence of sites for restriction endonucleases EcoRI, Bam H and SacI. One plasmid displaying the expected patterns was designated pHGS 2 DHFR. Two µg of the plasmid pHGS 2 were digested to completion with two units of the restriction endonuclease EcoRI, the free ends were dephosphorylated with one unit of Bacterial Alkaline Phosphatase BAP and the EcoRI digested vector was isolated from low melting LM agarose. Two µg of the plasmid pDS 1, to 1 were digested to completion with two units of the restriction endonuclease EcoRI and a fragment of approximately 920 bp containing the mouse DHFR gene was isolated from LM agarose. Twenty µg of the EcoRI digested vector pHGS 2 and 100 µg of the EcoRI fragment were ligated with one unit of T4 ligase and the DNA was transformed into HB 101. Eight transformants resistent to 100 µg ml of ampicillin were selected and cultures grown in LB medium containing 100 µg ml of ampicillin. DNA from these cultures was isolated and analyzed for its size and presence of sites for restriction endonucleases EcoRI and SacI. One plasmid displaying the expected patterns after electrophoresis on agarose gels was designated pHGS 2 DHFR E Fig. 11 . Ten µg of the plasmid pHGS 2 DHFR E were digested to completion with ten units each of the restriction endonucleases EcoRI and SacI and a fragment of approximately 275 bp was isolated from 8 acrylamide gel. Two hundred µg of the 275 bp EcoRI SacI fragment were ligated with 50 pmol each of the synthetic fragment 5 TATAAATA 3 and 5 AATTTATTTATA 3 in ligase buffer containing 50 mM NaCl using one unit of T4 DNA ligase. After ligation for 2 hours at 14 C, the DNA was digested to completion with one unit of restriction endonuclease SacI and a fragment of 283 bp containing the above mentioned synthetic sequences was isolated. Two µg of the plasmid pHGS 2Δ15 were digested to completion with four units of restriction endonuclease Bam H and the 5 overhangs were filled in with a large fragment of the E. coli DNA polymerase I Klenow fragment and the enzyme was inactivated through incubation at 65 C, for 10 minutes. Subsequently the DNA was digested to completion with one unit of ClaI and a fragment of 152 bp was isolated from a 8 acrylamide gel. Two µg of the plasmid pHGS 2 DHFR were digested to completion with 2 units each of the restriction endonuclease ClaI and SacI and the vector of approximately 4.6 kb was isolated from LM agarose. Hundred µg of the EcoRI SacI fragment containing the indicated synthetic fragments and hundred µg of the ClaI Bam HI fragment containing part of the 11 KDa transcriptional regulatory sequence were ligated with 20 µg of the ClaI SacI digested vector pHGS 2 DHFR using one unit of T4 DNA ligase and the DNA was transformed into HB 101. 16 transformants resistant to ampicillin were selected and cultures grown in LB medium containing 100 µg ml of ampicillin. DNA from these cultures was isolated and analyzed for its size and presence of sites for restriction endonucleases EcoRI and SacI. One plasmid displaying the expected patterns after electrophoresis on acrylamide gels was sequenced. The plasmid was designated pHGS A DHFR containing the sequence Ten µg of the plasmid pHGS 2 DHFR were digested to completion with 10 units each of the restriction endonucleases Bam HI and SacI and a fragment of 268 bp was isolated from a 8 acrylamide gel. Two hundred µg of the Bam HI SacI fragment were ligated with 50 pmol each of the synthetic fragments 5 TCTATCGATTAAATAAA 3 and 5 GATCTTTATTTAATCGATAGA 3 in ligase buffer containing 50 mM NaCl final concentration using one unit of T4 DNA ligase. After ligation for 2 hours at 14 C the DNA was digested with one unit of the restriction endonuclease SacI and the fragment of 290 bp containing the above mentioned sequence was isolated. Ten µg of the plasmid pHGS 2 Δ15 were digested to completion with 10 units each of the restriction endonucleases ClaI and DraI and a fragment of 130 bp was isolated from a 8 acrylamid gel. Hundred µg each of the Bam HI SacI fragment containing the indicated synthetic fragments and the ClaI DraI fragment containing part of the 11KDa transcriptional regulatory sequence were ligated with 50 µg of the ClaI SacI digested vector pHGS 2 DHFR using one unit of T4 DNA ligase and the DNA was transformed into HB 101. Sixteen transformants resistant to ampicillin were selected and cultures grown in LB medium containing 100 µg ml of ampicillin. DNA from these cultures was isolated and analyzed for its size and presence of sites for restriction endonucleases SacI and ClaI. One plasmid displaying the expected patterns after electrophoresis on acrylamide gels was sequenced. The plasmid was designated pHGS F DHFR containing the sequence The recombinant viruses RVV 6, RVV 7 resp. RVV 8 were constructed as described before using the plasmids pHGS 2 DHFR, pHGS A DHFR and pHGS F DHFR. In each case one virus was selected for the presence of the chimeric gene consisting of the mutated 11 kDa transcriptional regulatory sequence No.1, 3 resp. 4 and DHFR gene inserted into the VV TK gene by blot hybridization Mackett et al., supra . 10 µg of each of the plasmids pHGS 2 DHFR, pHGS A DHFR and pHGS F DHFR were digested to completion with 10 units of the restriction endonuclease AccI, the free ends were dephosphorylated using one unit of BAP and the enzyme was removed through subsequent extractions with phenol, phenol chloroform 1 1,v v and chloroform. The DNA s were precipitated and the free ends were phosphorylated using polynucleotide kinase in the presence of 2 fold molar excess of ³²P γ ATP. The enzyme was inactivated at 65 C for 10 minutes and the DNA s were digested to completion with 5 units of the restriction endonuclease HindIII. Fragments of approximately 1200 bp were isolated from LM agarose representing the S1 probes asymmetrically labeled at the AccI restriction endonuclease site. Monolayers of RK 13 cells were infected with 5 pfu per cell of the recombinant viruses RVV 6, RVV 7 resp. RVV 8. Early RNA was prepared 6 hours after infection of the cells incubated in medium containing 100 µg ml of cycloheximide Sigma . Late RNS s were isolated from virus infected cells 8 and 24 hours post infection. S1 mapping was performed as described in Example 1, section G, using the labelled S1 probes for the DHFR transcripts prepared as described in section F. The x ray exposure is shown in Fig. 14. DHFR transcripts are not detectable in RNA prepared from infected cells incubated in the presence of cycloheximide, indicating that the 11KDa regulatory sequences no. 1, 3 and 4 are not active early in transcription Fig. 14, lanes indicated with . DHFR transcripts indicated by arrow are detectable 8 and 24 hours after infection in cells infected with the viruses RVV 6, RVV 7 resp. RVV 8. Fig. 14, lanes indicated with 8 resp. 24 . The amounts of DHFR transcripts derived from the mutated 11KDa regulatory sequences no. 3 and 4 present in the insertion vectors pHGS A DHFR resp. pHGS F DHFR are at least 3 to 4 fold increased compared to the DHFR transcripts derived from the 11KDa regulatory sequence no. 1 present in the insertion vector pHGS 2 DHFR.