# Method for nucleic acid hybridisation.

## Abstract
A method of detecting a specific target polynucleotide se quence in a sample uses a labelled first probe and a second probe, both probes to hybridise to different sequences of the target and a solid phase carrying groups to bind the second probe. The method comprises contacting the sample under hybridising conditions with solutions of both probes, preferably in excess. Thereafter the mixture is contacted with the solid phase which is then washed. Target attached to the solid phase can be detected by means of the labelled first probe. The solid phase may be a dipstick carrying streptavidin to bind with the biotin groups attached to the second phase.

## Claims
CLAIMS 1. A method of detecting a specific target polynucleotide sequence in a sample, by the use of a a first polynucleotide probe having a singlestranded sequence complementary to a sequence of the target, and which is labelled, b a second polynucleotide probe having a singlestranded sequence complementary to a different sequence of the target, and c a solid phase, wherein the solid phase c and the second probe b carry groups reactive with one another to cause b and c to become bound together when the two are brought into contact, which method comprises the steps of i contacting the sample in solution under hybridising conditions with solutions of the probes a and b , ii contacting the resulting mixture with the solid phase c under conditions to cause the probe b to become bound thereto, and iii observing the presence or absence of the label of the first probe a bound to the solid phase. 2. A method as claimed in claim 1, wherein the second probe b carries biotin groups and the solid phase c carries streptavidin. 3. A method as claimed in claim 1, wherein the second probe b carries a hapten and the solid phase c carries a corresponding antibody. 4. A method as claimed in any one of claims 1 to 3, wherein the solid phase c is in the form of a dipstick which is dipped in the hybridisation mixture and subsequently removed and washed prior to observing the label. 5. A method as claimed in any one of claims 1 to 14, wherein each of the two probes is used in excess over the target. 6. A method as claimed in any one of claims 1 to 5, wherein the second probe b carries biotin groups, the solid phase c is in the form of a dipstick carrying streptavidin, and steps i to iii are i contacting the sample in solution under hybridizing conditions with solutions of probes a and b in excess over the target, ii contacting the resulting mixture with the dipstick c under conditions to cause the biotinylated probe b to become bound thereto, and iii observing the presence or absence of the label of the first probe a bound to the solid phase.

## Description
Method for Nucleic Acid HybridisationIntroduction There is increasing interest in the use of nucleic acid hybridisation for diagnostic purposes in clinical laboratories. A nucleic acid probe specific for a particular pathogen or group of pathogens can be used to detect the presence of the pathogen s in clinical samples which have been processed appropriately. Current methods for nucleic acid hybridisation were largely developed in research laboratories and are not ideal for clinical applications. In this invention, a method of hybridisation is proposed which would be much more suitable for such applications it could easily, for instance, be adapted to provide a dipstick procedure for detecting the formation of hybrids. Before describing the proppsed method, currently available methods will be reviewed briefly, with an assessment of their advantages and disadvantages.Currently Available Assays For the purpose of the ensuing discussion, hybridisation methods will be divided into two classes those in which the target nucleic acid is immobilised on a solid support, and those in which it is in solution.For the sake of textual clarity, the discussion will assume the use of a DNA probe to detect a DNA target either or both of the probe and target could of course b RNA rather than DNA. a Target immobilised. The great majority of research hybridisation assays depend on this type of methodology. In these assays, the target DNA is denatured into single strands which are then bound to a solid support such as a membrane filter made of nitrocellulose or nylon. To perform the hybridisation, this filter is immersed in a solution containing labelled probe. After incubation for an appropriate period, the filter is taken out of the probe solution and washed thoroughly to remove any probe which did not bind specifically to the targetDNA. The presence of labelled probe bound to the filter can then be detected by methods appropriate for the label used. This type of assay has a number of advantages Because the separated strands of the target DNA are immobilised, they cannot re anneal with each other in a reaction that would compete with the binding of labelled probe. As in all solid phase assays, separation of bound from unbound probe can be achieved very easily. Many samples can be spotted onto a single sheet of membrane filter and processed simultaneously. If required, topographical information present in the original sample can be preserved. Thus, the relative positions of electrophoretically separated fragments of DNA or of intracellular components can be maintained in, respectively, Southern blot or in situ hybridisations. Two disadvantages of this type of assay are Time is required to process the sample in such a way that the target DNA will bind to the solid phase. More seriously, the kinetics of the hybridisation reaction binding of probe to target DNA are adversely affected by having one of the partners in this reaction bound to a solid phase rather than in solution. To a certain extent, this problem can be overcome by using a large excess of probe. Six common types of assay in this category are Southern blot Southern E.M., 1975 J. Mol., Biol., 98 503 517. Northern blot Alwine J.C., Kemp D.J., and Stark G.R., 1977 Proc., Natl., Acad., Sci., U.S.A., 74, 5350 5354. Dot blot spot blot Brandsma I., and Miller G., 1980 Proc., Natl., Acad., Sci., U.S.A., 77 6851 6855. In situ hybridisation Gall J.G., and Pardue M.L., 1969 Proc., Natl., Acad., Sci., U.S.A, 63 378 383. Colony hybridisation Grunstein M., and Hogness D.S., 1975 Proc., Natl., Acad., Sci., U.S.A., 72 3961 3965. Plaque hybridisation Benton W.D., and Davis R.W., 1977 Science 196, 180 182. b Target in solution. In its simplest form, this type of assay involves mixing denatured target DNA with probe DNA in solution, and monitoring the formation of hybrids between target and probe. In practice, however, it can be difficult to separate bound from unbound probe. One approach, for instance, is to separate hybrids from unhybridised material by binding them to hydroxyapatite which has a higher affinity for double stranded than for singlestranded nucleic acids . Aside from the fact that the separation achieved may not always be absolute, this approach suffers from the inherent disadvantage that it relies on a physical property double strandedness which is not exclusive to the hybrid the probe may become involved in other double stranded structures which do not involve the target sequence, such as those formed by self annealing.This places unnecessary restrictions on the type of probe which can be used. More promising for clinical purposes is the sandwich hybridisation assay EDunn A. R. and HassellJ. A. 1977 Cell, 12, 23 36 Virtanen M., LaaksonenM., Soderlund H., Palva A., Halonen P. and Ranki M. 1983 The Lancet, Feb., 19, 1983, 381 383 and EPA 139489 which as its name suggests is the DNA version of one of the popular types of immunoassay. In sandwich immunoassays, two antibodies are required, each recognising a different epitope determinant of the target antigen similarly, in sandwich hybridisations, two different DNA probes are required, each hybridising to a different sequence in the target DNA.In its present state of development, the sandwich hybridisation assay has one of the probes bound, unlabelled, to a solid phase where it acts as a capture agent for any target DNA present in solution the second, labelled, probe is also present in solution and only becomes bound to the solid phase if target DNA has been captured by the first probe. To date, very little has been published on the use of sandwich hybridisation, which means that it is too early for the advantages and disadvantages of the technique to have emerged clearly. From a theoretical standpoint, however, one can speculate that they will be as follows Compared with the immobilised target type of assay, the sandwich hybridisation procedure should have one main advantage Sample preparation should be less time consuming because there is no need to immobilise the target DNA on a solid phase. In its present form, however, this type of assay could have several disadvantages A major problem could be that the assay is sample driven rather than reagent driven this could make it very difficult to force along the hybridisation reaction to give a reproducibly rapid and sensitive result. The reason for this is that the assay still features a kinetically undesirable reaction between a solid phase partner the first probe and a solution phase partner the target . In the immobilised target type of assay, this problem can be addressed by using an excess of solution phase partner, i.e. probe DNA. The same approach is not really possible in a sandwich hybridisation, because the solution phase partner is the target DNA, whose concentration will depend on the nature of the sample. The separated strands of the target DNA are free to re anneal with each other rather than to the probe DNAs in theory at least, this will reduce the sensitivity of this type of assay. The re annealing process could be slowed by reducing the concentration of the target DNA as has been shown, however, this would be very undesir able in a sample driven hybridisation assay. The Invention The invention provides a method of performing nucleic acid hybridisations which should combine the advantages of the two approaches so far discussed while eliminating their disadvantages. The reasoning behind this method is as follows.Hybridisations are most effective and sensitive when the probe and target sequences are both in solution, and each probe is present in excess over the target so as to drive the reaction. Separation and detection of hybrids, on the other hand, can be achieved most successfully when the hybrids are immobilised onto a solid phase. What is needed, therefore, is a method which allows a phase change between hybridisation and separation detection. In the method, this phase change is achieved by performing a sandwich hybridisation in which both of the probes are in solution one of them is, however, modified in such a way that it can easily be bound to a solid phase once the hybridisation is complete. Thus the invention provides a method of detecting a specific target polynucleotide sequence in a sample, by the use of a a first polynucleotide probe having a singlestranded sequence complementary to a sequence of the target, and which is labelled, b a second polynucleotide probe having a singlestranded sequence complementary to a different sequence of the target, and c a solid phase, wherein the solid phase c and the second probe b carry groups reactive with one another to cause b and c to become bound together when the two are brought into contact, which method comprises the steps of i contacting the sample in solution under hybridising conditions with solutions of the probes a and b , ii contacting the resulting mixture with the solid phase c under conditions to cause the probe b to become bound thereto, and iii observing the presence or absence of the label of the first probe a bound to the solid phase. The first probe a is labelled. By this is meant that the probe is isotopically or chemically modified in such a way that the person performing the hybridisation can, after further manipulation if necessary, detect the presence of the first probe. The nature of the label is not critical it may be a radioactive atom or a component of an enzyme or fluorescent or chemiluminescent system, or merely a chemical group by means of which such a component may subsequently be added. Techniques for labelling polynucleotide probes are well known and will not be described here. The single stranded sequence of the first probe that is complementary to the target sequence may, but need not be, labelled. The labelled part of the first probe can be partly or wholly double stranded.A requirement of the label is that it should not lead to nonspecific binding of the first probe a to the solid phase c , the second probe b or the sample. A variety of methods can be used to modify the second probe, provided that the method used allows the second probe b to be attached to the solid phase c after the hybridisation, but it does not generate any signal in the procedure used to detect hybridised first probe a , and that it does not lead to non specific binding between the second probe b and either the sample or the first probe a . For example, the second probe b may carry biotin groups and the solid phase c may carry streptavidin. Or the second probe may carry biotin or some other hapten and the solid phase may carry a corresponding antibody. Techniques for attaching complementary reactive groups to polynucleotide probes and to solid phases are well known in the art and will not be described here. The shape of the solid phase is not critical, and a variety of massive or particulate configurations may be used. One example is latex particles which may be dispersed in the hybridisation mixture and which may contain a magnetizable material for easy subsequent separation. Another example is a microtiter plate.A third example is a dipstick which may be dipped in the hybridisation mixture and subsequently removed and washed prior to observing the label. It is necessary, and is inherent in the method, that the two probes should hybridise to different regions of the target polynucleotide sequence. The sample may be hybridised with the two probes, either together, or separately in either order. It is an advantage of the invention that, being in solution, the two probes can be used in excess, so as to drive the hybridisation reaction in the forward direction.After incubation for a sufficient time to permit hybridisation of the two probes with the target polynucleotide sequence if present , the solid phase is added and the reaction conditions adjusted if necessary to effect binding between the probe b and any hybrids in which it is involved and the solid phase. Then the solid phase may be separated by known means from the hybridisation solution, washed, and further treated by known means to observe the presence of label thereon as indicating the presence in the sample of the target polynucleotide sequence. Thus, in the case where the first probe carries a radioactive label, the second probe carries biotin groups, and the solid phase is a dipstick coated with a material to which is bound streptavidin, the method may comprise the following steps Mix the sample in solution with an excess of each of the two probes, incubate this mixture to allow hybridisation to occur, bring the mixture into contact with the dipstick carrying immobilised streptavidin, and allow the biotinylated second probe and any hybrids in which it is involved to bind to the dipstick, recover and wash the dipstick, and then measure the amount of radioactivity bound to it there will be little if any bound if the sample did not contain the target polynucleotide sequence. The advantages of the method have already been alluded to the most important of them are a speed and sensitivity resulting from reaction in solution involving the use of an excess of each of the two probes to drive the hybridisation reaction this excess should also minimize any problems due to reanealing of target polynucleotide sequences.b easy separation and detection of hybrids.c easy adaptation to formats suitable for use in clinical laboratories, for example involving a dipstick. Experimental The practice of the invention can be illustrated using a model system based on plasmid pAT153, cosmid pJB8 and bacteriophageXCharon 4A. pJB8 DNA contains plasmid DNA sequences derived from pAT153 and X DNA sequences derived from Charon 4 Ish Horowicz, D. andBurke, J.G., Nucleic Acids Research, volume 9, page 2989ff, 1981 it should therefore hybridise with both a pAT153 DNA probe and a Charon 4 DNA probe, whereas these two probes should not hybridise directly with one another. For convenience, it is easier to use aCharon 4A DNA probe than a Charon 4 DNA probe, becauseCharon 4A DNA is commercially available the two phages are identical in the region which is represented in pJB8 . In the model system, denatured pJB8 target DNA is hybridised in solution with a biotin labelled pAT153DNA probe and a 32P labelled Charon 4A DNA probe, hybrids are captured using beads to which streptavidin had been linked covalently, the beads are washed to remove the labelled material not involved in specific hybrids, and the amount of bound 32P is determined by liquid scintillation counting. Restriction digests of pAT153 and pJB8 DNAs Typical preparations of plasmid and cosmid DNA contain significant proportions of highly supercoiled covalently closed circular ccc DNA. To ensure that the pAT153 and pJB8 target DNAs used in this experiment could be completely denatured prior to their immobilisation on a nylon hybridisation member, eachDNA was converted from ccc to linear form by digestion with restriction endonuclease Bam HI. In each case, 1 g of DNA was digested to completion under standard conditions for this enzyme 37 C, in 50mM Tris HCl pH 7.4, lOmM MgS04, 100mM NaCl and the reaction was terminated by heat inactivation of the enzyme at 650 C. Preparation of labelled probes Samples of Charon 4A DNAs were labelled with phosphorus 32 by nick translation using a commercially available kit and ti 32P dCTP from AmershamInternational . 1pg of the DNA was labelled to a specific activity of approximately 4 X 107 dpm g using procedure At from the protocol booklet supplied with the kit. The nick translation reaction was terminated by the addition of EDTA to a concentration of lOmM. The labelled probe was then separated from unincorporated nucleotides by gel filtration on a 10 X 1 cm column ofSephadex G501 the probe was eluted in TE buffer lOmMTris HC1, pH 8.0, 1mM EDTA in a volume of approximately 2 ml and stored frozen at 200C. Labelling of DNA by nick translation Nick translations were performed using the following reagents Nick translation kit All from 1 , 2 , 5 3H dCTP Amersham International biotin 11 dUTP synthesised according to the method of Ward and his colleagues Brigati, D.J., et al, Virology, volume 126, pages 32 50, 1983 and used as 0.3mM solution. These reagents were used to set up a nick translation reaction containing lug pAT153 DNA and each of the nucleotides dATP, dCTP, dGTP and biotin 11 dUTP at a concentration of 20 M. The unlabelled dCTP was supplemented with 5pCi of E3H dCTP, simply to permit the incorporation of nucleotides into DNA to be monitored. The nick translation was allowed to proceed for 3 hours at 15 0C, after which the reaction was terminated and the labelled DNA separated from unincorporated nucleotides as described above. From the quantity of tritium incorporated into DNA, it could be calculated that about 14 of the T residues had been replaced with bio dU the DNA from this reaction will henceforth be referred to as bio 3H pAT. Preparation of streptavidin coated beads Two samples of streptavidin coated beads were used. The first, henceforth referred to as PA SA beads, consisted of polyacrolein beads to which streptavidin had been linked as follows. The beads used had a mean diameter of 1.37 pm. 20 mg beads were washed three times in 0.1M carbonate bicarbonate buffer, pH 9.5, and resuspended in 1.0 ml of this buffer.1.3 mg of streptavidin at a concentration of Smg ml in 1OmM phosphate buffer pH 6 was added, and the mixture incubated for 2 hours at ambient temperature on a spiralling mixer. Next, 100 jal of a Smg ml solution of sodium borohydride Sigma was added to block unreacted aldehyde groups, and incubation continued at ambient temperature for a further two hours. The beads were then washed four times with lml of PBS containing 0.1 w v Tween 20 EPBS phosphate buffered saline 0.1M NaCl, 80 mM disodium hydrogen phosphate, l9mM sodium dihydrogen phosphate pH 7.4 , once with lml of 1M Nacl, and once with PBS 0.1 Tween 20.Finally, the beads were stored at LI 0C in 1 ml of PBS O.1 Tween 20 containing 1 w v BSA and 0.5 sodium azide. The second sample of streptavidin coated beads, henceforth referred to as a Amerlex M SA beads, consisted of magnetic Amerlex M beads to which streptavidin had been attached as follows.Carboxylated Amerlex M beads carboxylated polystyrene latex beads with a magnetic core , mean diameter 1 2 pm, were obtained from Amersham International plc.15mg of beads were washed three times in 10 mM phosphate buffer pH 6, and resuspended in 1 ml of the same buffer. To the beads were added 0.5 mg of streptavidin at a concentration of 6 mg ml in 50 mM phosphate buffer pH 7.4, followed by 5 mg of 1 ethyl 3 3 dimethylaminopropyl carbodiimide hydrochloride at a concentration of 10 mg ml in 10 mM phosphate buffer pH 6. The mixture was incubated at ambient temperature on a spiralling mixer for 2 hours, and the beads were then washed using the same sequence of washed described above for the PA SA beads, except that the fifth wash was with 0.1M glycine pH 3.5 rather than with 0.1M NaCl, and was prolonged for 20 minutes.Finally, the beads were stored in 1 ml of PBS O.1 Tween 20 containing 1 BSA and 0.5 sodium azide. Example 1 This example comprised three experiments. The first experiment demonstrated that pAT153 DNA and Charon 4A DNA both hybridise to pJB8 target DNA but not to each other. The second experiment demonstrated that PA SA beads can efficiently capture biotinylated DNA and hybrids containing biotinylated DNA, and that nonspecific binding of non biotinylated DNA is very low. In this third experiment, the model system was implemented in its entirety biotinylated pAT 153 DNA and 32P labelled Charon 4A DNA were hybridised in solution to samples containing various amounts of pJB8DNA, any hybrids formed were captured on PA SA beads, and the amount of 32P radioactivity which became attached to the beads was determined by LSC liquid scintillation counting . Ten solution hybridisation reactions were performed, consisting of five duplicate pairs.Reactions 1 4 were the model tests, and thus included the two probes bio 3H pAT and 32P Ch4A and various amounts of target pJB8 DNA. Reactions 5 10 were controls, 5 and 6 featured a pAT153 target to which 32P Ch4A should not bind , 7 and 8 lacked bio 3H pAT required for binding of hybrids to streptavidin coated beads , and 9 and 10 contained no target DNA at all. Table 1 below summarises which DNAs were included in which reactions, and in what quantity. One point worth commenting on is the large excess of 32P Ch4A used in each reaction 30 ng, compared with 2.5 ng of bio 3H pAT. The reason for this was that only about 1 30th of the Charon 4A genome is represented in pJB8, and so only about 1 30th of the input 32P Ch4A was capable of hybridising to the pJB8 target. The simplicity of the protocols used to perfrom the hybridisations and to assess the results provided a striking illustration of the utility of the invention.For each hybridisation, the DNAs to be included in the reaction were mixed together in a microcentrifuge tube in a total volume of 70 pl TE buffer, denatured by heated at 950C four 10 minutes., and cooled immediately by placing the tubes on ice. Concentrated stock solutions of SSC and SDS were added to give final concentrations of 6 X SSC 0.9M Nacl, 0.09M sodium citrate and 0.1 SDS sodium dodecyl sulphorate and a final volume of 100 pl. The tube was incubated overnight at 650C, then placed on ice for 2 minutes. To capture any hybrids formed, 25 Vl of the stock suspension ofPA SA beads were added, and the mixture was placed at ambient temperature for 30 minutes, with occasional vortex mixing.The beads were collected by centrifugation in a microfuge, and then washed, with washes 1 and 2 lasting for 20 minutes each, and washes 3 and 4 for 30 minutes each. After the final wash the beads were again collected by centrifugation and the amount of radioactivity bound to them was determined by the standard LSC procedures which allowed separate quantitation of 3H associated with bio 3H pAT and 32P associated with 32P Ch4A . The experimental design and results obtained are summarised in Table 1. TableTest of model system using PA SA beadsEMI15.1 tb SEP Quantities SEP of SEP DNA SEP included SEP Radioactivity SEP bound tb Reaction SEP pJB8 SEP pAT153 SEP bio 3H pAT SEP 32P Ch4A SEP to SEP PA SA SEP beads SEP cpm SEP tb SEP number SEP target SEP target SEP probe SEP probe SEP 3H SEP 32P tb SEP 1 SEP 500pg SEP 2.5ng SEP 30ng SEP 2024 SEP 2939 tb SEP 2 SEP 500pg SEP 2.5ng SEP 30ng SEP 2090 SEP 2587 tb SEP 3 SEP 50pg SEP 2.5ng SEP 30ng SEP 1919 SEP 444 tb SEP 50pg SEP 2.5ng SEP 30ng SEP 2014 SEP 525 tb SEP 5 SEP 500pg SEP 2.5ng SEP 30ng SEP 2084 SEP 270 tb SEP 6 SEP 500pg SEP 2.5ng SEP 30ng SEP 1585 SEP 80 tb SEP 7 SEP 500pg SEP 3ong SEP 0 SEP 19 tb SEP 8 SEP 500pg SEP 3ng SEP 0 SEP 46 tb SEP 9 SEP 2.5ng SEP 30ng SEP 1942 SEP 112 tb SEP 10 SEP 2.5ng SEP 30ng SEP 2036 SEP 136 tb These results demonstrate conclusively that the invention works in practice. The quantity of 32Plabelled Charon 4A which became attached to the PA SA beads was significantly greater following hybridisations in which pJB8 target DNA was present reactions 1 4 than the background level seen when any important component of the system was left out, as for instance when the target DNA was pAT153 rather than pJB8 reactions 5 and 6 , or when the biotinylated pAT153 probe was left out reactions 7 and 8 , or when no target DNA at all was present reactions 9 and 10 . Example 2 A preliminary experiment demonstrated thatAmerlex M SA beads can efficiently capture biotinylatedDNA and hybrids containing biotinylated DNA, and that non specific binding of non biotinylated DNA is very low. Model system featuring Amerlex M SA beads This Example is essentially a repeat of Example 1, but with hybrid capture by Amerlex M SA beads rather than PA SA, and with a new batch of 32P Ch4A probe labelled by nick translation to 4.8 X 10 7 dpm g , of which 75ng rather than 30ng was used per hybridisation.Reactions were set up as described in Example 1.After the overnight incubation, each reaction was placed on ice for 2 minutes, and 25 il added of the stock suspension of Amerlex M SA beads. Following 30 minutes incubation at ambient temperature, beads were collected using a magnetic separator and then taken through the standard sequence of washes. Finally, the radioactivity bound to the beads was determined by standard LSC procedures in this experiment, colourquenching by the Amerlex M beads significantly affected the measurement of bount 3H. The experimental design and results are summarised in Table 2. Table 2Test of model system using Amerlex M SA beadsEMI17.1 tb SEP Quantities SEP of SEP DNA SEP included SEP Radioactivity SEP bound SEP to tb Reaction SEP pJB8 SEP pAT153 SEP bio 3H pAT SEP 132P Ch4A SEP Amerlex M SA SEP beads SEP cpm tb SEP number SEP target SEP target SEP probe SEP probe SEP 3H SEP 32P tb SEP 1 SEP 500pg SEP 2.5ng SEP 75ng SEP 1200 SEP 1047 tb SEP 2 SEP 500pg SEP 2.5ng SEP 75ng SEP 1480 SEP 1279 tb SEP 3 SEP 50pg SEP 2.5ng SEP 75ng SEP 1394 SEP 221 tb SEP 4 SEP 50pg SEP 2.5ng SEP 75ng SEP 1430 SEP 234 tb SEP 5 SEP 500pg SEP 2.5ng SEP 75ng SEP 1310 SEP 67 tb SEP 6 SEP 500pg SEP 2.5ng SEP 75ng SEP 1465 SEP 145 tb SEP 7 SEP 500pg SEP 75ng SEP 0 SEP 67 tb SEP 8 SEP 500pg SEP 75ng SEP 11 SEP 61 tb SEP 9 SEP SEP 75ng SEP 1443 SEP 155 tb SEP 10 SEP 2.5ng SEP 75ng SEP 1401 SEP 78 tb The conclusions from this experiment with Amerlex M beads are exactly the same as those from the experiment with PA SA beads described in the previous Example.