# HEPATITIS A VIRUS VACCINES

## Claims
Patentansprüche für folgende Vertragsstaaten CH, DE, FR, GB, LI, NL

## Description
This invention relates to the field of recombinant vaccines and viral diseases. More particularly, the invention relates to use of the hepatitis A virus genomic sequence to obtain oligomeric probes for diagnosis, to obtain vaccines useful in protection of subjects at risk with respect to hepatitis A, and to obtain monoclonal antibody populations useful in treatment of infected subjects and as diagnostics. Hepatitis is a serious debilitating disease prevalent throughout the world. It may occur in acute form and causes extensive fatigue, liver damage, and, not infrequently, is fatal. The disease is caused by a viral infection. The infective agents are endemic in underdeveloped areas of the globe, where the majority of adults carry antibodies directed against viral antigens. In the United States, Western Europe, and Japan, the incidence of hepatitis is also significant especially among those exposed to conditions which are conducive to transmission of viral infection. However, because the probability of exposure in general is lower, most adults in these areas do not carry antibodies directed against the antigens of the virus causing hepatitis, and are at high risk when travelling to areas where the virus is endemic. Three very different and immunologically distinct forms of virus causing hepatitis are known Hepatitis A, Hepatitis B, and a third known at present only as Non A Non B. Hepatitis B virus HBV has been widely studied and vaccines protective against infection by HBV have been developed. Another infective species, hepatitis A virus HAV is a picornavirus belonging to the enterovirus genus. HAV was first characterized as the cause of acute illness by Feinstone, S. M., et al, As antibodies raised against hepatitis B are not cross reactive against hepatitis A or protective against HAV infection, there is a need for the development of vaccines specifically directed against infection by HAV. While the virus has been characterized morphologically, and the genome partially sequenced, the development of a vaccine has been hindered by insufficient knowledge of the nucleotide sequence of the genome. As will be further explained below, vaccine design is dependent on identification of suitable epitopes either as primary peptide sequences, or resulting from production of 3 dimensional configurations, which engender the formation of neutralizing antibodies. HAV has the typical icosahedral structure of a picornavirus, measuring 27 32 nm in diameter. The virions contain a single stranded genomic RNA molecule of approximately 7.5 kb, having positive strand polarity with a polyA stretch at the 3 end, and may be covalently linked to a protein at the 5 terminus. It has been assumed that the polypeptides encoded by the RNA are analogous to those of other picornaviruses, such as those responsible for polio and for foot and mouth disease FMD . While variations occur, sufficient similarity has been shown between such viruses to warrant devising a systematic nomenclature of picornavirus proteins, as set forth in Rueckert, R. R., et al, Some work relevant to vaccine preparation has been done. Monoclonal antibodies Mabs against HAV which are neutralizing in Partial cloning and sequencing of the HAV genome has been reported by others Ticehurst, J. R., et al, EP A 0154587, designating CH, DE, FR, GB, IT, LI and NL and published on 11 September 1985 discloses the partial sequence of the HAV genome. Certain fragments of this sequence are also disclosed. EP A 0175613, designating CH, DE, FR, GB, IT, LI and NL and published on 11 September 1985, discloses certain immunogenic peptides derived from HAV. W085 01517 designating EPO contracting states AT, BG, CH, DE, FR, GB, LU, NL and SE and published on 11 April 1985 discloses a 1489 nucleotide sequence of HAV CDNA. The invention provides a cDNA replica of the entire HAV genomic sequence. Portions of the genomic sequence are useful as probes to diagnose the presence of virus in clinical samples. The understanding of the genomic sequence also effectively makes available the entire viral polypeptide sequence and permits not only analysis, but production of portions thereof, if appropriate, in the correct configuration, so as to provide effective vaccines and permit the production of neutralizing antibodies. The availability of the entire HAV sequence thus permits the design and construction of polypeptides which may serve either as vaccines themselves, or as intermediates in the production of monoclonal antibody Mab preparations useful in passive immunotherapy against the disease, or as diagnostic reagents. Without the sequence of the entire genome at the disposal of the producer of the therapeutic or preventive compositions, successful production of optimally effective products would be impossible. Accordingly, in one aspect, the invention relates to a nucleotide sequence substantially identical with that representing the entire genome of HAV as represented in Figure 1. The remaining contributions of the invention relate to utilizing this sequence or portions thereof as oligomeric probes, to produce peptides which can serve as vaccines, or as reagents to produce monoclonal antibodies useful in treatment of the disease. Other aspects of the invention include expression systems which are capable of effecting the production of a desired protein encoded by sequences derived from the complete genome, to recombinant vectors containing such systems or portions thereof, to recombinant host cells transformed with such vectors, to proteins produced by the transformed cells, and to vaccines prepared from such proteins. In addition, the invention relates to specific peptide sequences representing epitopes encoded by the genome, and to such sequences covalently linked to carrier proteins. Such carrier proteins, in addition to more conventional carriers, include the 22 nm particle associated with hepatitis B infection, which carries polyalbumin receptor sites, and is 1000 fold more immunogenic than the unassembled subunit component. By inserting antigenic HAV determinants into the 22nm HBsAg particle, increased immunogenicity for these epitopes is obtained. The invention also relates to deleted virions capable of eliciting neutralizing antibodies in subject hosts, and to monoclonal antibodies useful in passive immunotherapy. The invention also relates to the methods of preparing these desired polypeptide vaccines and immunoglobulins. Insofar as the invention relates to portions of the sequence of Figure 1 and their use in expression vectors or as oligonucleotide probes, those portions of the sequences are those claimed in claim 3. Figure 1 shows the complete nucleotide sequence of the HAV genome. The cDNA contains the identical sequence, except, of course, that T will be substituted for U. The deduced amino acid sequence of the viral L434 polypeptide is also shown in Figure 1. Figure 2 shows a restriction map and the regions of overlap of the four cDNA clones which comprise the entire 7.5 kb HAV genome, pHAV47, pHAV8, pHAV1, and pHAV16. Figure 3 shows the construction of expression vectors for sequences encoding fusion mRNAs encoding portions of superoxide dismutase SOD and HAV capsid protein. Figure 4 shows a computer generated profile based on hydrophilicity and structural characteristics of the peptide chain as a function of position for regions of the polio virus polypeptide and the HAV polypeptide. Figure 5 shows the construction of a mammalian cell expression vector encoding deleted HAV virions. Figure 6 shows the construction of pDC 104 for production of hybrid HAV HBsAg in yeast. Figure 7 shows the construction of pWG 1 for production of hybrid HAV HBsAg in yeast. As used herein, a nucleotide sequence substantially identical to the exemplified HAV genome refers to a sequence which retains the essential properties of the exemplified polynucleotide. A specific, but non limiting example of such substantial equivalence would be represented by a sequence which encodes the identical or substantially identical amino acid sequence, but, because of codon degeneracy, utilizes different specific codons. Nucleotide changes are, indeed, often desirable to create or delete restriction sites, provide processing sites, or to alter the amino acid sequence in ways which do not adversely affect functionality. Nucleotide sequence refers both to a ribonucleotide and a deoxyribonucleotide sequence and includes the positive sense strand, as shown, and the negative sense strand as well. A DNA sequence derived from the nucleotide sequence which comprises the genome of HAV refers to a DNA sequence which is comprised of a region of the genomic nucleotide sequence, or a combination of regions of the sequence. These regions are, of course, not necessarily physically derived from the nucleotide sequence of the gene, but refer to polynucleotides generated in whatever manner which have the same or substantially identical sequence of bases as that in the region s from which the polynucleotide is derived. For example, typical DNA sequences derived from the HAV genome include fragments encoding specific epitopes, fragments encoding portions of the viral polypeptide, sequences encoding the capsid proteins, sequences encoding deleted virions, and sequences encoding the viral proteinase. Similarly, a peptide derived from the HAV polypeptide refers to an amino acid sequence substantially identical to that of this polypeptide or a portion thereof, having the same biological properties as that portion. The manner of synthesis of such derived peptide is not material it may be chemical synthesis or recombinant means, for example. Recombinant host cells , host cells , cells , cell lines , cell cultures , and other such terms denoting microorganisms or higher eucaryotic cell lines cultured as unicellular entities, are used interchangeably, and refer to cells which can be, or have been, used as recipients for recombinant vector or other transfer DNA, and include the progeny of the original cell transfected. It is understood that the progeny of a single parental cell may not necessarily be completely identical in morphology or in genomic or total DNA complement as the original parent, due to accidental or deliberate mutation. Progeny of the parental cell which are sufficiently similar to the parent to be characterized by the relevant property, such as the presence of the nucleotide sequence encoding a desired peptide, are included in the progeny intended by this definition, and are covered by the above terms. Control sequence refers to DNA sequences which are necessary to effect the expression of coding sequences to which they are ligated. The nature of such control sequences differs depending on the host organism in procaryotes, generally such control sequences include promoter and ribosome binding site in eucaryotes, generally, such control sequences include promoter, terminators, and, in some instances, enhancers. The term control sequences is intended to include, at a minimum, all components whose presence is necessary for expression, and may also include additional components whose presence is advantageous. Operably linked refers to a juxtaposition wherein the components so described are in a relationship permitting them to function in their intended manner. A control sequence operably linked to a coding sequence is ligated in such a way that expression of the coding sequence is achieved under conditions compatible with the control sequences. At the core of the present invention is the provision of a nucleotide sequence containing the entire genome of hepatitis A virus. The availability of this complete polynucleotide, first, permits the construction of oligomers of about 20 bp or more useful in disease diagnosis. Such probes may be used to detect the presence of the viral genome in, for example, fecal filtrates of subjects suspected of harboring the virus. Knowledge of the gene sequence also enables the design and production of vaccines effective against HAV and production of neutralizing Mabs. Sequencing information available from the genome allows the amino acid sequence of the polypeptide to be deduced and compared to that of other known picornaviruses for which locations of favorable epitopes is known. Further, once the desired sequences are chosen, fragments of the genome can be obtained and expressed independently, thus providing desired polypeptides using recombinant techniques. Both procaryotic and eucaryotic hosts are useful for such expression. Short polypeptide fragments may be chemically synthesized and linked to carrier proteins for use as vaccines. Recombinantly expressed polypeptides may be provided under conditions offering a favorable environment for processing into, for example, deleted virions, thus permitting three dimensional reconstruction of epitopic sites without danger of creating an infective virus. Mammalian and yeast cells provide particularly suitable environments for such expression. In addition, the epitopes may be produced linked to a particle forming protein. The proteins thus produced may, themselves be used as vaccines, or may be used to induce immunocompetent B cells in hosts, which B cells can then be used to produce hybridomas that secrete antibodies useful in passive immunotherapy. The genomic sequence of HAV was prepared using stool samples of infected patients as the source for the virus. The isolated viral RNA was fractionated, and those fractions containing RNA of sufficient length to contain the entire genome were pooled, ethanol precipitated, and used to obtain a cDNA library. Colonies from the library were hybridized to oligomeric probes complementary to the 3 proximal portions of the RNA. Several colonies hybridizing to these sequences were used to obtain additional portions of the RNA message from the cDNA library using walking techniques. Four cDNA clones were obtained from overlapping portions of the gene, and subjected to restriction mapping and sequencing. The entire genomic sequence was deduced from these four cDNA inserts. The amino acid sequence deduced from the viral genome is similar to that characteristic of picornaviruses, and the 5 and 3 non coding regions probably involved in viral replication were identified. The availability of the entire genomic sequence permits construction of expression vectors encoding presumptively antigenically active regions of the capsid proteins. Fragments encoding the desired proteins are obtained from the cDNA clones using conventional restriction digestion and ligated into vectors containing portions of fusion sequences such as β glactosidase or SOD, preferably SOD. Any desired portion of the HAV genome could be expressed as a fusion protein or co expressed with another readily produced protein in The availability of the entire genomic sequence also makes possible computer analysis of the structural characteristics of the deduced protein for comparison with the known sequence of polio virus. As the antigenic regions of polio virus are known, such comparisons make possible the designation of short peptide regions representing epitopes. These can then be synthesized using chemical methods, and provided with, for example, cysteine residues at the C terminus which provide means for linking the peptides to neutral carrier proteins. A number of techniques for obtaining such linkage are known in the art, including the formation of disulfide linkages using N succinimidyl 3 2 pyridylthio propionate SPDP and succinimidyl 4 N maleimido methyl cyclohexane 1 carboxylate SMCC obtained from Pierce Company, Rockford, Illinois. These reagents create a disulfide linkage between themselves and peptide cysteine residues on one protein and an amide linkage through the ε amino on a lysine, or other free amino group in the other. A variety of such disulfide amide forming agents are known. See, for example, Any carrier may be used, which does not itself induce the production of antibodies harmful to the host, such as the various serum albumins, tetanus toxoids, or keyhole limpet hemocyanin KLH . The conjugates, when injected into suitable subjects, will result in the production of antisera which contain immunoglobins specifically reactive against not only the conjugates, but also against fusion proteins carrying the analogous portions of the sequence, and against appropriate determinants within whole HAV. Virions which mimic the three dimensional epitopic configuration of intact HAV may also be prepared. It is generally surmised that antigenic sites may result from three dimensional folding of immunogenic proteins, and, specifically with respect to HAV, it has been reported that antibodies formed in response to injection with HAV capsid proteins are not reactive against the denatured viral peptides Gerlich et al, The deleted virions are prepared by providing DNA sequences which lack the 5 and 3 non coding regions of HAV putatively responsible for replicative functions. The sequences also lack a portion of the 3 coding region presumed, by homology with other picornaviruses, to encode the replicase origin. These fragments are preferably cloned into plasmid expression vectors appropriate for yeast or mammalian cells capable of the desired post transcription and post translation processing however, procaryotes can also be used. Deleted virions produced in hosts transformed with these vectors are immunoreactive against human convalescent HAV antisera, and can be used for immunization. The immunogenicity of the epitopes of HAV may also be enhanced by preparing them in mammalian or yeast systems fused with particle forming proteins such as that associated with hepatitis B surface antigen. Constructs wherein the HAV epitope is linked directly to the particle forming protein coding sequences produce hybrids which are immunogenic with respect to the HAV epitope. In addition, all the vectors prepared include epitopes specific to hepatitis B virus HBV , having various degrees of immunogenicity, such as, for example, the pre S peptide. Thus, particles constructed from particle forming protein which include HAV sequences are immunogenic with respect to both HAV and HBV. Hepatitis surface antigen HBsAg has been shown to be formed and assembled in In addition, portions of the particle forming protein coding sequences per se may be replaced with codons for an HAV epitope. In this replacement, regions which are not required to mediate the aggregation of units to form immunogenic particles in yeast or mammals can be deleted, thus eliminating additional hepatitis B antigenic sites from competition with the HAV epitope. Preparation of vaccines which contain peptide sequences as active ingredients is also well understood in the art. Typically, such vaccines are prepared as injectables, either as liquid solutions or suspensions solid forms suitable for solution in, or suspension in, liquid prior to injection may also be prepared. The preparation may also be emulsified or the protein encapsulated in liposomes. The active immunogenic ingredient is often mixed with excipients which are pharmaceutically acceptable and compatible with the active ingredient. Suitable excipients are, for example, water, saline, dextrose, glycerol, ethanol, or the like and combinations thereof. In addition, if desired, the vaccine may contain minor amounts of auxiliary substances such as wetting or emulsifying agents, pH buffering agents, or adjuvants which enhance the effectiveness of the vaccine. The vaccines are conventionally administered parenterally, by injection, for example, either subcutaneously or intramuscularly. Additional formulations which are suitable for other modes of administration include suppositories and, in some cases, oral formulations. For suppositories, traditional binders and carriers may include, for example, polyalkaline glycols or triglycerides such suppositories may be formed from mixtures containing the active ingredient in the range of 0.5 to 10 , preferably 1 2 . Oral formulations include such normally employed excipitents as, for example, pharmaceutical grades of manitol, lactose, starch magnesium stearate, sodium saccharine cellulose, magnesium carbonate and the like. These compositions take the form of solutions, suspensions, tablets, pills, capsules, sustained release formulations or powders and contain 10 95 of active ingredient, preferably 25 70 . The proteins may be formulated into the vaccine as neutral or salt forms. Pharmaceutically acceptable salts, include the acid addition salts formed with the free amino groups of the peptide and which are formed with inorganic acids such as, for example, hydrochloric or phosphoric acids, or such organic acids as acetic, oxalic, tartaric, mandelic, and the like. Salts formed with the free carboxyl groups may also be derived from inorganic bases such as, for example, sodium, potassium, ammonium, calcium, or ferric hydroxides, and such organic bases as isopropylamine, trimethylamine, 2 ethylamino ethanol, histidine, procaine, and the like. The vaccines are administered in a manner compatible with the dosage formulation, and in such amount as will be therapeutically effective and immunogenic. The quantity to be administered depends on the subject to be treated, capacity of the subject s immune system to synthesize antibodies, and the degree of protection desired. Precise amounts of active ingredient required to be administered depend on the judgment of the practititioner and are peculiar to each subject. The immunogenic proteins prepared as described above are used to immunize mammals. Lymphocytes from these animals may be used to prepare hybridomas capable of secreting monoclonal antibodies directed against these epitopes and cross reactive against the infective virus. The resulting monoclonal antibodies are particularly useful in diagnosis, and those which are neutralizing are useful in passive immunotherapy. The general techniques used in extracting RNA from the virus, preparing and probing a cDNA library, sequencing clones, constructing expression vectors, transforming cells, and the like are known in the art and laboratory manuals are available describing these techniques. However, as a general guide, the following sets forth some sources currently available for such procedures, and for materials useful in carrying them out. Both procaryotic and eucaryotic host cells may be used for expression of desired coding sequences when appropriate control sequences are used compatible with the designated host. Among procaryotic hosts, Eucaryotic hosts include yeast and mammalian cell culture. Mammalian cell lines available as hosts for expression include many immortalized cell lines available from the American Type Culture Collection, including HeLa cells, Chinese hamster ovary CHO cells, baby hamster kidney BHK cells, and a number of other cell lines. Suitable promoters for mammalian cells prominently include viral promoters such as that from Simian virus 40 SV40 Fiers, et al, The transformation procedure used depends on the host to be transformed. Bacterial transformation generally employs treatment with calcium or rubidium chloride Cohen, S. N., Vector construction employs techniques which are by now quite well understood. Site specific DNA cleavage is performed by treating with suitable restriction enzyme under conditions which generally are specified by the manufacturer of these commercially available enzymes see, e.g., The New England Biolabs Product Catalog . In general, about 1 µg of plasmid or DNA sequence is cleaved by 1 unit enzyme in about 20 µl buffer solution for an incubation time of about 1 2 hr at about 37 C. After incubation with the restriction enzyme, protein is removed by phenol chloroform extraction and the DNA recovered by reprecipitation with ethanol. The cleaved fragments may be separated using polyacrylimide or agarose gel electrophoresis techniques, according to the general procedures found in Sticky ended cleavage fragments may be blunt ended using Ligations are carried out using standard buffer and temperature conditions using T4 DNA ligase, and ATP sticky end ligations require less ATP and less ligase than blunt end ligations. When vector fragments are used as part of a ligation mixture, the vector fragment is often treated with bacterial alkaline phosphatase BAP in order to remove the 5 phosphate and thus prevent religation of the vector alternatively, restriction enzyme digestion of unwanted fragments can be used to prevent religation. Ligation mixtures are transformed into suitable cloning hosts, such as Synthetic oligonucleotides may be prepared using an automated oligonucleotide synthesizer as described by Warner, B.D., et al, DNA sequences including those isolated from genomic or cDNA libraries may be modified by site directed mutagenesis, as described by Zoller, M, et al, DNA libraries are probed using the procedure of Grunstein and Hogness For routine vector constructions, ligation mixtures are transformed into The following examples are intended to illustrate but not limit the invention. The procedures set forth, for example, in D.1 may, if desired, be repeated but need not be, as techniques are available for construction of the desired nucleotide sequences based on the information provided by the invention. Expression is exemplified in HAV virus was isolated from the stools of a hepatitis A patient collected during an epidemic outbreak in Los Angeles, California. Other isolates, available in the art could also have been used. See, for example, Daemer, R. J., et al, The isolate was inoculated onto PCL PRF 5 Alexander hepatoma cells followed by culture at 34 C. Detectable virus production was observed 3 4 weeks later, and the original infected cell cultures have been continuously passaged in DME 15 FCS for the past two years with no loss of virus titers. Virus titers were measured by immunofluorescence and or competitive radio immunoassay HAVAB, Abbott . Prehybridization and hybridization were essentially as described above. Probes were kinased at the 5 terminus with T4 kinase according to the method of Lillehaug, et al, These probes were thus validated by hybridization to the gels run on HAV RNA from partially purified virions. RNA from the sucrose fractions which contain full length HAV RNA were identified by the above described hybridization blotting techniques. Samples were pooled, ethanol precipitated and used to form a cDNA library. The cloning procedure used is a modification of that previously described by Wood and Lee, The resulting cDNA library was probed by transferring the colonies to replicate nitrocellulose filters, and hybridizing to HAV1 and HAV2 as described above. Using the foregoing probes, and selecting colonies hybridizing to both, 3 proximal cloners were obtained. Further portions of the genome were identified by walking techniques. An additional clone was obtained using the 5 terminal PstI PstI fragment from pHAV1307 as probe Ticehurst, J., et al, Supra . Four recombinant cDNA inserts designated pHAV16, pHAV1, pHAV8, and pHAV47 were selected and restriction maps prepared. The overlapping regions were confirmed by cross hybridization studies, and the total length of non overlapping regions was approximately 7.5 kb, the entire span of the HAV genome. The results of the restriction mapping and overlap are shown in Figure 2. pHAV16, pHAV1, pHAV8 and pHAV47 were deposited with the American Type Culture Collection ATCC on 29 March 1985 and assigned accession numbers 53077, 53078, 53076 and 53079, respectively. Complete sequencing of the series of four inserts shown in Figure 2 gave the results for the corresponding viral RNA set forth in Figure 1. Analysis of the open reading frames associated with this sequence indicated a deletion of 22 nucleotides at position 3530 of the HAV genome this was corrected by site directed metagenesis to obtain the sequence shown in Figure 1 the corrected plasmid was designated pHAV1 . Four bacterial expression plasmids were constructed which directed the simultaneous synthesis of human superoxide dismutase SOD Hallewell, et al, The expression plasmids were based on the tac promoter driven expression plasmid pSOD16 of Hallewell, et al, supra . Plasmid pSOD16cf2 was generated from pSOD16 by replacement of a portion of the carboxy terminal coding region of the SOD gene and downstream polylinker sequence through the EcoRI site by the new polylinker sequence The substitution of this polylinker sequence results in the removal of the natural carboxy terminal Gln of SOD. A 1390 base pair BgIII repair XmnI fragment isolated from a digest of pHAV8116SV7d see D.4 was cloned into the SmaI site of pSOD16cf2. DNA of a resulting clone having the HAV insert in the proper orientation was digested with KpnI, the 3 overhang was removed with T4 DNA polymerase, then digested the HindIII, and this site was filled in with reverse transcriptase. The resultant linear, blunt ended DNA was religated and transformed into pSODHAV2 2 was transformed into pSODHAVP1 1 contains, in addition to the sequences in pSODHAV2 2, the remainder of VP3 and the carboxy half of VP2. pSODHAV2 2 was digested with NcoI 5 to the HAV sequences and with SacI, which cuts within the VP1 sequence. The 3.4 kb fragment containing all the SOD sequences and a part of the VP1 sequences was gel isolated, ligated to a 1806 bp NcoI SacI fragment isolated from pHAV81167d see D.4 and transformed into When expressed in pSODHAV9 1 was derived from pSODHAVP1 1 and contains the additional half of VP2 to generate the full P1 precursor sequence. A 438 bp BstXI fragment from PHAV47 see D.1 containing the 5 end of VP2 was gel isolated and ligated to the single stranded synthetic DNA oligomer, pSODHAV10 5 was constructed in a manner analogous to that described above for pSODHAV9 1 except that the synthetic DNA oligomer For protein expression analysis, overnight cultures, in medium as above, were begun from glycerol stocks. These cultures were diluted 1 100 into the same medium and grown at 37 C to an OD₆₅₀ of 0.5 0.8. Uninduced controls were prepared by diluting a 1 ml portion of the growing culture with an equal volume of the same medium. Tac promoter driven protein synthesis was induced either by diluting of 1 ml of growing culture with medium plus 2 mM IPTG Isopropyl β thiogalactopyranoside or by addition of 1 100 volume of 100 mM IPTG to larger volume cultures, in either case, to give 1 mM IPTG, for 3 4 hr at 37 C. Large scale cultures had 50 ml medium in 500 ml flasks or 250 ml medium in liter flasks. Cells were lysed in the presence of SDS and DTT for analysis on denaturing polacrylamide gels Leammli, Expression was detected by the appearance of a protein band s of expected size present in induced sample lysates but not in uninduced samples induced proteins were detected at their respective, expected molecular weights for all plasmids. The HAV component of fusions and the free P1 of pSODHAV10 5 were detected by Coomassie staining and by specific reaction with anti HAV antibodies. The SOD component of fusions was detected by Coomassie staining and by anti SOD antibodies free SOD was detected both by Coomassie stain and by enzymatic activity in native gels. Gels were either stained with Coomassie blue or subjected to Western analysis after electroblotting onto nitrocellulose filters Towbin et al, SOD activity in non denaturing gels was identified by the method of Beauchamp, C., et al, Expressed products were approximately 1 3 of total cell protein for pSODHAV2 2 and pSODHAV1 1 and about 50 fold lower for pSODHAV9 1 and pSODHAV10 5. The SODHAV2 2 and SODHAVP1 1 fusion proteins were purifed on preparatory SDS acrylamide gels, located by Coomassie staining, and electroeluted. These purified proteins were injected into mice and the resulting sera were tested for reactivity with HAV virus on Western blots and in ELISA assays. Mouse antisera against the SODHAV2 2 react with a 34 kd protein from HAV virus on Western blocks that appears to be HAV VP1 and was therefore also designated anti SOD VP1. In a capture ELISA both mouse anti SODHAV2 2 and anti SODHAV1 1 react with HAV virus. In the ELISA assay, microtiter wells were coated overnight with IgG from an HAV seropositive patient. The coated wells were washed and incubated 1 hr with purified HAV. Dilutions of the mouse sera were the incubated in the well for 1 hr. The wells were then incuated 1 hr with peroxidase conjugated goat anti mouse Ig followed by reaction with the peroxidase substrate 2,2 azino di 3 ethylbenzthiazoline sulfonate ABTS and H₂O₂, and the plates were read on an ELISA reader at 414 nm the titer is expressed as the reciprocal of the dilution that gives 50 of the maximum absorbance. The results are as follows In additional assays, 20 ng of SODHAV2 2 and SODHAVP1 1 were spotted onto nitrocellulose strips, and the strips were then reacted with two HAV seropositive human sera and two seronegative human sera. The strips were washed and reacted with peroxidase conjugated goat anti human IgG, and then reacted with the peroxidase substrate 4 chloro 1 naphthol to visualize the binding of the peroxidase conjugated antibody. The results showed that the SODHAVP1 1 reacted with both the seropositive samples and neither of the seronegative samples the SODHAV2 2 reacted with one of two seropositive samples and neither seronegative sample. In another assay, SODHAVP1 1 was used as the coating antigen in an ELISA assay with HAV seronegative and HAV seropositive human and chimpanzee sera. All four positive sera showed a greater reaction with the SODHAVP1 1 than did the four negative sera. In this assay, microtiter wells were coated with solution of 5 µg ml of SODHAVP1 1. The coated wells were then reacted with a 1 50 dilution of the various chimp and human sera for 1 hr, and were then incubated with peroxidase conjugated goat anti human IgG which cross reacts with chimp IgG . The wells were finally incubated with ABTS and H₂O₂, and read on an ELISA reader at 414 nm. The results are given in absorbance units. Three short amino acid sequences were synthesized based on analysis of the deduced amino acid sequence of HAV and comparison with the known epitopes of polio virus protein. The hydrophilicity and structural characteristics as a function of position for both the polio polypeptide and the HAV polypeptide were compared with the aid of a computer based analysis. Peaks representing putative areas of structural homology have been identified with corresponding Roman numerals and are shown in Figure 4. Areas III and V of the polio virus polypeptide represent major antigenic sites of VP1 Minor et al, The three sequences constructed are as follows, where the numbers correspond to the positions designated in the deduced sequence of Figure 1. Each of these sequences ends with a C terminal cysteine for use in conjugation by a covalent bond to a carrier protein of sufficient size to confer immunogenicity of the peptide epitope. The synthesized proteins were conjugated to the carrier protein, keyhole limpet hemocyanin KLH by disulfide linkage through the cysteine of the peptide using m maleimido benzoyl N hydroxysuccinimide ester MBS as the coupling reagent Liu et al, The conjugated peptides were injected into rabbits. Sera obtained after 31 100 days contained antibodies against the injected conjugates as assayed by ELISA. Western blots using the antisera against gels run on tissue culture produced HAV show immunoprecipitates for the antisera produced in response to the protein corresponding to peak Va and to the protein with no polio homology. Immunoprecipitation was observed with the bands corresponding to VP1 protein. Deleted virions can be prepared using expression vectors compatible with procaryotic, yeast, and mammalian cells. The P1 precursor of several picornaviruses synthesized To obtain a similar effect Similar constructs expressing the P1 precursor and proteinase can be produced in yeast and mammalian cells. For production of non infectious virion like particles, mammalian cells can be transfected with the HAV genome under the control of appropriate regulatory elements. By deleting the 3 end portion of the genome beyond the sequences encoding 3C ie., the 3D region replicase gene is removed, thereby crippling the replication ability of the virus. The transfected cells will produce all virion proteins from the P1 and P2 region in addition to 3A, 3B and 3C. This is sufficient to generate virion like particles, but will not generate infectious virus. For example, an expression vector compatible with mammalian cells pHAV8161 SV7d, is constructed as shown in Figure 5. The HAV coding sequences are provided by digestion of pHAV1 , pHAV8, and pHAV16 D.1 . The mammalian replication and control systems are provided by pSV7d see below , which is a pBR322 derivative modified to contain the SV40 origin of replication, early promoter and polyadenylation site. It also has a polylinker with several restriction sites downstream from the SV40 promoter, as well as TAA stop codons in all three reading frames downstream from the polylinker. In more detail, pSV7d was constructed as follows the 400 bp BamHI HindIII fragment containing the SV40 origin of replication and early promoter was excised from pSVgtl Mulligan et al, which contains the restriction sites shown and stop codons in all three reading frames. The resulting 670 bp. fragment, containing SV40 origin of replication, SV40 early promoter, polylinker with stop codons and SV40 polyadenylation site was closed into the BamHI site of pML, a pBR322 derivative with an about 2.5 kb deletion Lusky and Botchan, pSV7c and pSV7d represent successive polylinker replacements. First, pSV7b was digested with StuI and XbaI, and the linker was ligated into the vector to yield pSV7c. pSV7c was digested with BgIII and XbaI, and the linker was ligated to yield pSV7d. In the construction of pHAV8161 SV7d, pHAV161 supra was digested with BamHI and SalI to release the 4.2 kb fragment encoding a portion of the HAV sequence. pHAV8 was digested with SalI and BamHI to provide the coding sequences upstream of the SalI site in the pHAV161 insert. These two fragments were ligated into the BamHI site of pSV7d. The resulting vector pHAV8161 SV7d places the ligated HAV sequences encoding amino acid 1 2076 under control of the SV40 promotor and directly behind the vector provided stop codon, as shown in Figure 8. pHAV8161 SV7d may be transfected into COS cells and transient expression tested by immunofluorescence using human convalescent serum for HAV. Plasmid DNA from successful COS transformants may then be used to construct vectors for stable expression. U.S. Serial No. 650,323, filed 12 April 1984 and assigned to the same assignee is incorporated herein by reference. This application describes the construction of hybrid particles of hepatitis B surface antigen HBsAg containing inserts of foreign immunogens into a pre surface pre S coding portion in reading frame with the codons for HBsAg. Plasmid pDC101, described therein contains a portion of the pre S HBsAg gene, including 55 codons of the pre S region, in a GAPDH controlled expression cassette cloned into the BamHI site of pBR322 derivative. The incorporated application describes the insertion of desired immunogens, such as the gD glycoprotein D antigenic site into a unique EcoRI site present in the pre S region of pDC101 to give the hybrid plasmid pDC103. Similarly, in accordance with the present invention, desired epitopes derived from the HAV genome may be provided with suitable EcoRI linkers and inserted in proper reading frames into the EcoRI site of pDC101, or used to replace the gD codons in the pDC103 hybrid. pDC103 is deposited with ATCC and has accession no. 20726. The construction of pDC 104, a yeast expression vector for a hybrid gene containing the sequence encoding HAV VP1 epitope shown in Figure 6 was as follows. The HindIII SalI fragment obtained from HAV1307 and containing codons for amino acids 445 657 of the VP1 region was provided with synthetic EcoRI linkers and ligated into the EcoRI site of pDC103 in place of the gD sequences. The resultant was transformed into yeast host pWG 1, a yeast expression vector containing the coding sequence for the hybrid gene harboring the VP 1 neutralizing epitope in the particle forming portion of the HBsAg encoding gene is constructed similarly see Figure 7 . The DNA corresponding to the HAV epitope of the following sequence codes for amino acids 792 848 and is synthesized using a commercially available automated oligonucleotide synthesizer. The resulting synthetic oligomer, containing BamHI sites, is then cloned into the BglII sites of the host vector pDC101, described in U.S. 650,323 supra , replacing a 150 bp fragment of the HBsAg coding for a dominant epitope. The resulting vector pWG 1 is transformed into Hybrid particle immunogens are thus prepared using fused coding sequences for HBsAg and HAV and provide enhanced immunogenicity for the HAV epitopes.