# Hybridization method and probes therefor.

## Abstract
A method for discriminating between two complementary nucleotide sequences differing by as little as 1 3 nucleotides is described in which an oligonucleotide signalling complex probe is used, which probe comprises a single stranded oligonucleotide of 8 to 30 nucleotide units having covalently attached thereto, at a sterically tolerant site, a residue capable of producing a non radioisotopic signal and having a molecular weight of at least 100 daltons. Oligonucleotide signalling complex probes and diagnostic kits for use in such a method are described and claimed.

## Claims
1. A method for discriminating between two complementary nucleotide sequences differing by as little as 13 nucleotides which comporises

## Description
The present invention relates to a method suitable for discriminating between two complementary nucleotide sequences which may differ by as little as a single nucleotide, nucleic acid hybridisation probes therefor and kits for use in such a method. The use of labelled polynucleotide probes is well known for a wide range of applications as described in, for example, Lewin, Science Conventional radiolabelled probes are efficient and sensitive, but are associated with several problems which mitigate against routine use for screening for example in clinical laboratories. Thus radiolabelled probes are potentially hazardous and pose problems of disposal. Furthermore radiolabelled probes are often unstable and have a limited shelf life as a result of the relatively short half life of the radioactive materials used as labels, especially ³²P. Moreover autoradiographic detection can be time consuming and the handling of radiolabelled probes requires personnel with the appropriate safety training. There is therefore a desire to use non radioactive methods of labelling probes and certain such methods have been described in the literature for example D.C. Ward at the 1981 ICN UCLA Symposium held in Keystone, Colorado on March 15 20, and published in Developmental Biology Using Purified Genes 1981 XXIII, 1981 pages 647 658 Academic Press, Editor Donald D Brown The present invention however is particularly concerned with the use of non radioactively labelled probes for use in hybridization analysis where it is necessary to discriminate between two complementary nucleotide sequences differing by relatively few nucleotides. It is well documented e.g. by J W Szostok et al., Methods in Enzymology, Volume 68, p419 428. Academic Press that the stability of hybrids of oligonucleotides with complementary nucleic acid sequences is sensitive to mismatched base pairs in the hybrid. These authors demonstrated that most oligonucleotide hybrids with single mismatches in base pairing are less stable to temperature than hybrids of the same oligonucleotides with correct base pairing. Thus subtle differences in oligonucleotide sequence complementarity can give rise to subtle differences in hybrid destabilisation which can be detected in practice by incubating matched and mismatched oligonucleotide hybrids to a specific temperature in aqueous solutions. It will therefore be appreciated that the present invention is concerned with non radioactively labelled probes which have sufficient selectivity to discriminate between a complementary sequence and a sequence with as little as 1 3 non complementary nucleotides. Because of the relatively small destabilisation effect of base pair mismatches on oligonucleotide hybrids, it would be expected that labelling of an oligonucleotide probe with anything but a low molecular weight species would alter the hybridization characteristics of the probe such that the desired hybridisation selectivity of the probe would be lost as a result of the presence of the label. Recently A Chollet and E H Kawashima in Nucleic Acids Research Vol 13 No. 5 1985 have demonstrated that a biotinylated probe may be used to detect a single A C mismatch whilst recognizing that even the low molecular weight species, biotin, may alter the hybridisation characteristics of the oligonucleotide probe. The belief that even the attachment of low molecular weight labels to oligonucleotide probes may adversely affect their ability to distinguish complementary and mismatched sequences involves the practical disadvantage that the detection system can only be attached to the probe after hybridisation. The operator of the probe is thus faced with conducting further reaction steps after the hybridisation which necessarily are time consuming, costly and conducive to operator error. The present invention is based on the suprising discovery that high molecular weight species may be attached to oligonucleotide probes prior to hybridisation without markedly altering the hybridisation selectivity of the probe. Thus for example a high moleuclar weight detection system may be attached to an oligonucleotide probe prior to hybridisation thus obviating the need for a series of reaction steps after hybridisation. Thus according to one feature of the present invention there is provided a method for discriminating between two complementary nucleotide sequences differing by as little as 1 3 nucleotides which comprises The residue which is capable of producing a non radioisotopic signal advantageously has a molecular weight of at least 5000 daltons and preferably of at least 20,000 daltons. Whilst we do not wish to be bound by theoretical considerations we have determined that in general a residue having a molecular weight as high as about 2 million daltons may for example be used with advantage in the method of the present invention. Preferably the residue will comprise a proteinaceous moiety. The method of the present invention is preferably effective to discriminate between two complementary nucleotide sequences differing by as little as only a single nucleotide. The method of the present invention is especially useful for detecting the presence of point mutations in genes, particularly where these point mutations are responsible for, or are associated with, particular disease states. Examples of such disease states are phenylketonuria, alpha₁ antitrypsin deficiency, alpha and beta thalassaemia and sickle cell anaemia. In addition the method of the present invention is useful for detecting the presence of polymorphisms in mammalian genomes, some of which are often associated with particular disease states. The method of the present invention is however useful in a number of other areas. For example, it is useful for testing the accuracy of a nucleotide sequence in a gene cloned into a plasmid or phage vector. The oligonucleotide signalling complex is capable of forming a hybrid with the complementary test nucleic acid sequence under such conditions that one or more single base change s in this complementary sequence reduces or precludes hybrid formation. The nucleic acids contained in the probe may be ribonucleotides, or deoxyribonucleotides, or a mixture of these two. The single stranded oligonucleotide is preferably composed of 12 to 25 nucleotide units, and most preferably of 17 to 20 nucleotide units when the test sample contains mammalian nucleic acids. The residue which is capable of producing a non radioisotopic signal may be covalently attached to a base, a sugar or preferably a phosphate of the oligonucleotide. Such attachment is preferably on a terminal base, sugar or phosphate, either at the 3 or 5 end of the oligonucleotide. When the attachment is via phosphate it is preferably at the 5 end. When the covalent attachment is on a base of the oligonucleotide, it is particularly important that the attachment is located at a sterically tolerant site, that is a position on the base at which modification can be effected by attachment thereto of a substituent group without causing interference with the ability of the modified oligonucleotide to hybridise with a fully complementary sequence. Generally, substitution on any hetero atom nitrogen or oxygen of the base is less desirable. Specific sites that are less desirable are N¹ and N⁶ of adenine N¹ being less desirable than N⁶ , N¹, N² and O⁶ of guanine and N³ and N⁴ of cytosine. Preferred sites of covalent attachment are C⁵ of cytosine and uracil and C⁸ of adenine and guanine. Methods of introducing a covalently bonded group in a base of an oligonucleotide are known from, for example, European Patent Publications 63879 Yale University and 97373 Enzo Biochem. and PCT Patent Publication WO84 03285 Molecular Biosystems . Methods of introducing a covalently bonded group in a phosphate of an oligonucleotide are known from, for example, European Patent Publications 97373 Enzo and 119448 Wakunaga Seiyaku Kabushiki Kaisha and T Kempe et al, The introduction of a covalently bonded group in a sugar residue of an oligonucleotide may, for example, be carried out as follows. The group may, for example, be bonded via a hydroxy group in the sugar, for example by an ether or ester link. This reaction may be carried out by any stanard method known in the art, for example that described by J Smrt, It will be understood that the residue which is capable of producing a non radioisotopic signal may itself be capable of signalling or may be capable of producing a signal by interaction with an appropriate agent according to methods known The detection system comprises a signalling moiety either alone or attached to a linkage for attachment to the spacer group. Thus the signalling moiety may either be directly attached to the spacer group by a direct covalent bond or may be attached to the spacer group via a linkage. Thus for example where a proteinaceous moiety is present it may be present in the signalling moiety of the detection system or, where a linkage is present, in the linkage between the signalling moiety and the spacer group or in both such linkage and signalling moiety. The spacer group may thus if desired be linked to the signalling moiety of the complex by a direct covalent link in which case the signalling moiety will preferably comprise a proteinaceous moiety such as a system for producing an enzymatically activated production of a colour change. In a further embodiment the spacer group may if desired be linked to the signalling moiety of the detection system by a proteinaceous moiety containing specific binding pair, for example a protein ligand or antigen antibody interaction for example an avidin biotin or dinitrophenyl antidinitrophenyl antibody interaction. If desired the signalling moiety may also comprise a proteinaceous moiety such as defined above in relation to the signalling moiety. In a still further embodiment the spacer group may for example simply be linked to a proteinaceous moiety containing specific binding pair or the proteinaceous partner of such a pair. A preferred signalling part of the residue incorporates a system for producing an enzymatically activated production of colour change. Preferred enzyme systems involve alkaline phosphatase AP , acid phosphatase, beta galactosidase, luciferase or horseradish peroxidase. Such enzyme systems are not themselves capable of signalling, but are capable of producing a signal in the presence of an appropriate substrate according to methods known In use it will generally be necessary for the nucleic acid probe to detect minute amounts of the fully complementary oligonucleotide sequence. In such circumstances it will be advantageous to incorporate within the probe a means of amplifying the signal. The amplification can be carried out by known techniques, for example using one or more of the systems described in European Patent Publications 27036 Self , 49606 Self , 58539 Self , and 60123 Self . The means employed to produce or observe the signal will depend on the signalling mechanism. Thus, for example, where the signal incorporates an enzymatically activated colour or colour change it will generally be necessary to supply the enzyme with a substrate. In employing the method of the invention it will generally be advantageous to employ a probe which is complementary to a nucleotide sequence in which the point mutation has occurred at or near the middle of the sequence. In this way the differentiation between normal sequence and the abnormal altered sequence will tend towards the optimal. It will be understood by those skilled in the art that discrimination of nucleotide sequences where a point mutation occurs towards a region opposite the terminus of an oligonucleotide probe is, in practice, more difficult as exemplified by Szostak et al., Methods in Enzymology, Vol. 68., p. 419 Academic Press . The method of the invention will be carried out using standard techniques and reagents, for example, employing the Southern blot , dot blot or Sandwich hybridisation techniques. The present invention also relates to hybridisation probes which are of particular interest for discriminating between two complementary nucleotide sequences differing by relatively few nucleotide sequences. Thus according to a further feature of the present invention there is provided an oligonucleotide signalling complex comprising a single stranded oligonucleotide of 8 to 30 nucleotide units in length and having covalently attached thereto at a sterically tolerant site, via a spacer group, either a residue comprising a specific binding pair optionally carrying an enzyme label, said residue having a molecular weight of at least 1000 daltons or an enzyme label with the proviso that where an enzyme label is linked to the oligonucleotide via a spacer group, the linkage of the spacer group to the oligonucleotide is via the terminal base, sugar or preferably phosphate. On the basis of experiments which we have conducted we have concluded that linkage of the spacer group to the terminal base, sugar or phosphate alone has negligible effect on the melting temperature of the oligonucleotide signalling complex. This is in contrast to a reduction in melting temperature which has been reported in the literature for substitutions at positions other than at the terminal base, sugar or phosphate alone. The spacer group is preferably linked to the oligonucleotide via phosphate at the 5 end. Where the enzyme label is directly linked to the oligonucleotide via a spacer group the spacer group is conveniently of formula A in which n is 2 to 16, preferably 6, a is 0 8, b is 0 8 and R represents a direct covalent link or a cyclohex 4 yl, 2 phenylene, 3 phenylene or 4 phenylene group optionally substituted by 1 to 3 radicals selected from C The oligonucleotide of the said oligonucleotide signalling complex preferably has a sequence complementary to a nucleotide sequence associated with a disease state or with the corresponding normal sequence. The specific binding pair is for example an immunological pair, conveniently an antigen antibody interaction for example a dinitrophenyl antidinitrophenyl antibody interaction or a protein ligand interaction preferably avidin biotin. Preferably a system for producing an enzymatically activated production of colour change or enzyme label is covalently attached to the said specific binding pair. Preferred enzyme systems include alkaline phosphatase, acid phosphatase, beta galactosidase, luciferase or horseradish peroxidase. Such enzyme systems are not themselves capable of signalling, but are capable of producing a signal in the presence of an appropriate substrate according to methods known It will generally be advantageous to incorporate within the probe a means for amplifying the signal. Such amplification can be effected by known techniques for example using one or more of the systems described in European Patent Publications 27036, 49606, 58539 and 60123. Preferably the obligonucleotide probes of the present invention possess the formula B wherein A represents avidin or an avidin enzyme moiety m is 4 or 5, p is 0 to 16 and X is a direct link, O P 0 OH O , S , O , CONH , CONHCO, or less preferably N R⁸ wherein R⁸ represents a straight chain or branched C If desired the group A in which A is an avidin enzyme moiety may additionally carry a substrate for the enzyme. In particular the oligonucleotide probe is especially a probe of formula B i in which X is a direct link, m is 5 and p is 0 or ii in which X is O P 0 OH O , m is 5 and p is 8. According to a further feature of the present invention there is provided a process for the preparation of an oligonucleotide signalling complex as hereinbefore defined wherein the residue comprises a specific binding pair optionally carrying an enzyme label, said residue having a molecular weight of at least 1000 daltons, the specific binding pair being covalently attached via a spacer group to the single stranded oligonucleotide, which process comprises reacting one member of said specific binding pair optionally carrying an enzyme label with an oligonucleotide complex comprising a single stranded oligonucleotide of 8 to 30 nucleotide units in length and having covalently attached thereto, at a sterically tolerant site, the other member of said specific binding pair via a spacer group, the reactants being selected such that the residue in the complex obtained has a molecular weight of at least 1000 daltons. Preferably at least one of the members of the specific binding pair comprises a proteinaceous moiety. The oligonucleotide complex containing only one member of said specific binding pair is conveniently prepared by reacting a protected oligonucleotide of 8 to 30 nucleotide units with a free OH moiety of an otherwise protected phosphate of one member of said specific binding pair and deprotecting the product thereby formed. The phosphate of one member of the specific binding pair is preferably prepared by reacting an OH group of the said member with a phosphate of the formula R¹ O P O R² R³ in which R¹ represents a protecting group for the phosphate and R² and R³, which may be the same or different each represents a leaving group, whereby a phosphate of one member of the specific binding pair is obtained in which the phosphate carries a single phosphate protecting group. It is essentially preferred to prepare an oligonucleotide signalling complex as hereinbefore defined in which the residue comprises a specific binding pair optionally carrying an enzyme label by first preparing the oligonucleotide complex containing only one member of the specific binding pair using the phosphate of the formula R¹ O P O R² R³ as hereinbefore defined and then reacting the complex obtained with the other member of the specific binding pair optionally carrying an enzyme label. Preferably the reaction is effected in the presence of an agent effective to introduce a phosphate protecting group whereby to isolate a phosphate of one member of the specific binding pair, which phosphate carries two phosphate protecting groups and subsequently removing one of said phosphate protecting groups. Such a process provides an advantageous method of purifying the product of the phosphorylation process. The above mentioned phosphate is preferably 2 chlorophenyl di 1,2,4 triazole phosphate and the phosphate protecting group is preferably introduced in the presence of a base such as pyridine. According to a further feature of the present invention there is provided a process for the preparation of an oligonucleotide signalling complex as hereinbefore defined in which an enzyme label is covalently attached via a spacer group of the formula A to the single stranded oligonucleotide, which process comprises reacting a compound of formula C in which E represents an enzyme such as alkaline phosphatase or horseradish peroxidase with a compound of formula D in which n is as hereinbefore defined. The compound of formula C is preferably prepared by reacting an amino group of the enzyme with a compound of formula F in which Y represents an activated ester and R, a and b are as hereinbefore defined. Preferably Y represents a group of the formula G in which R⁷ represents hydrogen or a sulphonic acid salt preferably the sodium salt. Preferably a compound of formula F is used in which R¹ is hydrogen, a is 0, b is 0 and R is the 3 phenylene moiety m phenylene . The compound of formula D in which n is as hereinbefore defined is preferably prepared by deprotection, for example by hydrolysis, of a corresponding compound of formula E. The compound of Formula E is preferably prepared by reacting the oligonucleotide in protected form, conveniently protected on a polymer support with a compound of formula H in which n is as hereinbefore defined, R⁴ and R⁵, which may be the same or different, each represents a C The compound of formula H is preferably prepared by reacting a compound of formula J in which n is as hereinbefore defined with a compound of formula K in which R⁴, R⁵ and R⁶ are as hereinbefore defined and Hal represents a halogen atom preferably chlorine. The compound of formula K is preferably chloro N,N diisopropylamino methoxy phosphine. When the oligonucleotide signalling complex and method of the invention are employed in the detection of a point mutation in a gene associated with a disease state, it will generally be advantageous to incorporate the oligonucleotide signal complex in a diagnostic kit and this kit is therefore regarded as a further feature of the invention. Thus according to a further feature of the present invention there is provided a diagnostic kit for discriminating between two complementary nucleotide sequences differing by as little as a single nucleotide, one of the said sequences being associated with a disease state and the other of said sequences being the corresponding normal sequence, which kit comprises at least one oligonucleotide signalling complex probe comprising a single stranded oligonucleotide of 8 to 30 nucleotide units in length, said oligonucleotide having a sequence complementary to a sequence associated with a disease state or the corresponding normal sequence, said oligonucleotide having covalently attached thereto, at a sterically tolerant site, a residue having a molecular weight of at least 1000 daltons, which residue is capable of producing non radioisotopic signal. The kit will preferably comprise 1 at least one oligonucleotide signalling complex probe comprising a single stranded oligonucleotide having a sequence complementary to a sequence associated with a disease state and 2 at least one oligonucleotide signalling complex proble comprising a single stranded oligonucleotide having a sequence complementary to the corresponding normal sequence which would be present in a human not subject to the said disease state. It will be appreciated in this regard that certain disease states are characterised by point mutations which appear in different regions of the human DNA genome. In such a case it would be important to determine that each relevant point mutation was present or absent and thus more than one oligonucleotide signalling complex probe would be employed, one probe to detect the presence or absence of each point mutation. It would also be advantageous to include in the kit the probe for the corresponding normal or abnormal sequence. The kit will also preferably comprise appropriate buffer solutions and or washing solutions and will contain written or printed instructions for use of the kit according to the method of the present invention. In the accompanying drawings The invention is illustrated, but not limited by the following Examples. In the Examples, unless otherwise stated, the solutions are aqueous and the values are w v. The constitution of various reagents is as follows PBS 150mM NaCl 10mM sodium phosphate at pH7.4 SSC 0.15M NaCl 0.015M sodium citrate Denhardt s reagent 0.02 BSA 0.02 Ficol 400,000 0.02 PVP The following contractions are used MeOH methanol EtOAc ethyl acetate DMF dimethylformamide DNA deoxyribonucleic acid tRNA transfer ribonucleic acid HRP horseradish peroxidase PBA phosphate buffered saline SDS sodium dodecyl sulphate 2 SSC double concentration SSC 0.5 SSC half concentration SSC BSA bovine serum albumin PVP polyvinyl pyrrolidone HEPES N 2 hydroxyethyl N¹ 2 ethane sulphonic acid Tris tris hydroxymethyl aminomethane AP alkaline phosphatase The following are trade marks A molecule of biotin was attached to a 15 base long oligonucleotide alpha₂ 55 with the nucleotide sequence of A solution of phosphorylating agent prepared from 2 chlorophenylphosphorodichloridate 0.32 ml. and 1,2,4 triazole 700 mg. in pyridine 3 ml. was added to anhydrous biotinol A solution of biotinyl 2 chlorophenyl 2 cyano ethyl phosphate A sample 10 mg. of a fully protected oligodeoxyribonucleotide sequence, known as sequence 55 in an interferon alpha₂ gene synthesis described by M. D. Edge After disconnecting the column from the solvent delivery system and removing the column top, a solution of biotinyl 2 chlorophenyl phosphate triethylammonium salt 23 mg. and 1 mesitylene 3 nitro 1,2,4 triazole 50 mg. in anhydrous pyridine 0.3 ml. was added to the dried resin described above. The column top was replaced and the column placed in an oven at 45 C for 45 minutes. Alternatively, the reaction can be performed at 20 C for 1.5 hours. The column was re attached to the solvent delivery system then the resin washed with pyridine 2 minutes , DMF 2 minutes , CHCl₃ MeOH 7 3 v v 2 minutes , ether 2 minutes and finally dried in a stream of nitrogen to give the biotinylated fully protected oligonucleotide sequence The biotinylated oligonucleotide sequence was cleaved from the solid support and deprotected by a standard procedure as described by M.D. Edge In order to establish that a biotin residue had been attached to the oligonucleotide sequence, an aqueous solution of the product 1.6 O.D.₂₆₀ units in 2 ml. of 1 PBS, 0.1 Tween 20 was added to a column 0.25 ml. of avidin agarose Sigma Chemical Company Ltd containing 8.3 units of avidin. The column was washed with 1 PBS, 0.1 Tween and fractions 1 ml. were collected. At least 85 of the u.v. absorbing material was retained by the column. In contrast, when a sample of ³²P labelled oligonucleotide sequence alpha₂ 55 was added to a similar column, all the radioactivity passed through the column in the first 2 ml. of eluant. A complex of avidin and HRP was attached to the biotinylated oligonucleotide using a Vectastain ABC kit Vector Laboratories, Burlingame, California 94010, USA . One drop of Vectastain reagent A was added to 2 ug. of biotinylated oligonucleotide in 1 ml. of PBS containing 0.1 Tween 20 Sigma Chemical Company . After mixing, one drop of Vectastain reagent B was added, mixed and the mixture allowed to stand at room temperature for 30 minutes. Plasmids for analysis contained either the interferon alpha₂ gene nucleotide sequence 3 TTCTTTATGTCGGGC 5 M. D. Edge et al. A gene was prepared from synthetic oligonucleotides essentially as described by M. D. Edge 1 Taq 5 CGACGATGTGTGAT 2 Taq 5 CGGCAGATCACACATCGT 55 F6 5 AAGAAATACAGCTGC 56 F6 5 CCGGCTTGGCAGCTG 57 F6 5 CAAGCCGGGCAGCTG The synthetic gene was cloned between the ClaI and SalI sites of the expression plasmid pSTP1 Windass et al, Serially diluted samples of plasmid DNA 1 ug maximum were dot blotted or slot blotted onto a nitrocellulose filter using equipment and instructions supplied by Schleicher and Schuell, Keene, New Hampshire, USA. The filter was air dried and baked Hybridised oligonucleotide protein complexes were visualized as follows. Filters were soaked for 30 minutes at 37 C in PBS containing 2 BSA and 0.1 Triton X 100 and then for 30 minutes at 37 C in 10 ml. PBS 0.1 Tween 20. Filters were then washed at room temperature for 5 minutes three times and twice respectively in each of the following solutions The HRP substrate was prepared by dissolving 5 mg. of diaminobenzidene in 10 ml. of 10mM Tris, pH 7.5. A solution of 200 ul. of 1 CoCl₂ was added, mixed well and the mixture left on ice in the dark for 10 minutes. Following this 10 min incubation, 15 ul of 30 H₂O₂ was added. Finally filters were placed in the substrate mixture resulting in visual localisation of hybridised oligonucleotide protein complexes as dark brown deposits. A hybridisation signal corresponding to a minimum of 300 ng of pIF55 alpha₂ DNA was detected while signal was absent from immobilised dots of pIF55 F6 alpha₂ DNA where three base pairs at one end of the oligonucleotide were non complementary to the corresponding sequence. This discrimination forms the basis for a test to distinguish alterations in a nucleotide sequence homologous to an oligonucleotide probe. Analysis of hybrids detected with complexes of avidin and biotinylated HRP indicates that the oligonucleotide protein complexes so formed include up to 30 molecules of biotinylated HRP and 10 molecules of avidin giving a maximum total molecular weight of about 1,880,000 daltons. Thus very large masses of protein complexed to oligonucleotide probes as exemplified do not interfere with the hydridisation selectivity of the oligonucleotide protein complex. The procedure described in Example 1 was followed except that the biotinylated oligonucleotide was bound with avidin alone for hybridisation. This avidin binding was performed as described in Example 1 except that Vectastain reagent B ie HRP addition was omitted. Following hybridisation and washing at temperatures to distinguish mismatches from homologous hybrids, filters were soaked in BPS, 2 BSA, 0.1 Triton X 100 for 30 minutes at 37 C as described in Example 1 and then in 10 ml of PBS 0.1 Tween 20 containing 2 drops of Vectastain reagent B only for 30 minutes at 37 C. Subsequent washes and HRP visualisation were carried out as described in Example 1. The results were similar to those obtained in Example 1 with a hybridisation signal corresponding only to pIF55 alpha₂ DNA. The ability of the probe of Examples 1 and 2 to distinguish mismatches is highlighted by the following comparative Example A The procedure described in Example 2 was repeated except that the biotinylated oligonucleotide was used directly for hybridisation, Vectastain reagent additions before hybridisation being omitted. After hybridisation, 2 drops of Vectastain reagent A were added to 10 ml. PBS 0.1 Tween 20, prior to Vectastain reagent B addition as described in Example 2. This mixture of Vectastain reagent A and reagent B was incubated at room temperature for ½ hour before use, to allow cross links to form between the avidin reagent A and the biotinylated HRP reagent B . A limit of hybrid detection corresponding to 30 ng pIF alpha₂ DNA was obtained compared to 1 ug of pIF55 F6 DNA. A molecule of biotin was attached to a 15 base long oligonucleotide alpha₂ 25 with the nucleotide sequence of Using a similar procedure to that described in Preparation 1 of Example 1, but with octane 1,8 diol replacing 2 hydroxypropionitrile, crude biotinyl 2 chlorophenyl 8 hydroxyoctyl phosphate was obtained. The crude mixture was slurried with EtOAc 100 ml. and filtered to remove excess octane 1,8 diol. The EtOAC extract was evaporated and the residue again slurried with EtOAC 100 ml. and filtered. After removal of solvents the residue was purified by flash silica gel chromatography Merck Kieselgel 60 ART 9385 using EtOAc EtOH 4 1 v v as eluant. After removal of solvents from appropriate fractions there was thus obtained biotinyl 2 chlorophenyl 8 hydroxyoctyl phosphate Using a similar procedure to that described in Preparation 1 of Example 1 but with biotinyl 2 chlorophenyl 8 hydroxyoctyl phosphate 0.61 g. replacing biotinol and EtOAc replacing chloroform there was obtained a crude sample of compound A solution of compound The procedure described in Preparation 3 of Example 1 was repeated with oligodeoxyribonucleotide sequence alpha₂ 25 M. D. Edge et al. Plasmids for analysis contained either the interferon alpha₂ gene nucleotide sequence 3 CTTTACTAAGTTGTC 5 M. D. Edge et al., A gene was prepared from synthetic oligonucleotides essentially as described by M. D. Edge The synthetic gene was then cloned by the method described in Example 1. The plasmid including the oligonucleotide 25 I4 will hereinafter be referred to as pIF25 I4 alpha₂. Plasmids were dot blotted as described in Example 1. An avidin alkaline phosphatase AP conjugate was attached to the biotinylated nucleotide as follows. An AP avidin D conjugate Vector Laboratories was reconstituted with water to 20,000 units of enzyme ml. This solution was diluted 5 fold with 10mM HEPES 0.15MNaCl pH7.5 and 1 ml. of this mixture was incubated with 2 ug. of the biotinylated oligonucleotide alpha₂ 25 The procedure described in Example 3 was repeated except that the biotinylated oligonucleotide was bound with avidin alone for hybridisation, analogous to the procedure described in Example 2. Following hybridisation and washing at temperatures to distinguish mismatches from homologous hybrids, filters were washed for 20 minutes at 37 C in 3 BSA in washing buffer 1 0.1M Tris HCl pH 7.5 , 0.1M NaCl, 2mM MgCl₂ and 0.05 Triton X 100 . Biotinylated AP was subsequently added either as a biotinylated AP polymer Bethesda Research Laboratories BRL at a concentration of 1 ug. ml. in washing buffer 1 or as biotinylated AP Vector Laboratories . After 10 minutes filters were washed twice for 2 minutes in washing buffer 1 at room temperature and then twice in washing buffer 2 0.1M Tris pH9.5, 0.1MNaCl, 5mM MgCl₂ prior to incubation in AP substrate mixture as described in Example 3. The visualisation results obtained were comparable with those of Example 3. The ability of the probes of Examples 3 and 4 to distinguish a single mismatch is highlighted by the following comparative Example B. The procedure described in Example 4 was repeated except that the biotinylated oligonucleotide was hybridised prior to the addition of streptavidin. Following hybridisation and washing at temperatures to distinguish mismatches from homologous hybrids, filters were washed as in Example 4 and incubated in streptavidin Bethesda Research Laboratories at a concentration of 2 ug. ml. in washing buffer for 10 minutes at room temperature. Following incubation with streptavidin, the filters were washed 3 times for 2 minutes in wash buffer 1 prior to the addition of biotinylated AP polymer BRL , as described in Example 4. Subsequent washing and detection of AP was carried out as described in Example 4. The visualisation results obtained were comparable with those of Examples 3 and 4. The procedure described in Preparation 3 of Example 1 was repeated using the triethylammonium salt of 9 instead of the triethylammonium salt of biotinyl 2 chlorophenyl phosphate to yield the biotinylated oligodeoxyribonucleotide sequence 12 . The hybridisation procedure described in Example 1 was repeated using the biotinylated oligodeoxyribonucleotide sequence 12 as the probe except that the plasmid pIFS1101 Edge et al., Nucleic Acids Res., 11 1983 p 6419 was used as a source of the normal interferon alpha₂ gene. As observed for Example 1, a hybridisation signal corresponding to 300 ng of pIFS1101 DNA was obtained but was absent from up to 1 ug of the mutant pIF55 F6 alpha₂ plasmid DNA. The procedure described in Example 2 was repeated using the biotinylated oligodeoxyribonucleotide sequence 12 bound with avidin alone for hybridisation to the plasmids pIFS1101 and pIF55 F6 alpha₂. 1 ug of pIFS1101 DNA was detectable in this experiment in the absence of signal from 1 ug of pIF55 F6 DNA. The ability of the probes of Examples 5 and 6 to distinguish mismatches is highlighted by the following comparative Example C The procedure described in Example A was repeated using the biotinylated oligodeoxyribonucleotide sequence 12 for hybridisation to plasmids pIFS1101 and pIF55 F6 alpha₂, hybridization being effected prior to the addition of streptavidin. A minumum of 3 ng of pIFS1101 DNA was detected by this procedure compared to a minimum of 300 ng of pIF55 F6 alpha₂ DNA. The procedure described in Preparation 3 of Example 1 was repeated using the oligodeoxyribo nucleotide sequence of alpha₂ 25 11 M. D. Edge et al, Nucleic Acids Res. 1983, The hybridisation procedure described in Example 3 was repeated using the biotinylated oligodeoxyribonucleotide sequence 13 as the probe. As shown in Figure 1, a hybridisation signal was observed for the immobilised pIF 25 alpha₂ DNA row B, 3 and 1 ug left to right but was absent from the mutant pIF25 I4 alpha₂ plasmid DNA row A, 3 and 1 ug . A molecule of alkaline phosphatase was covalently linked to a 15 base long oligonucleotide alpha₂ 55 with the nucleotide sequence of Chloro N,N diisopropylamino methoxyphosphine was added slowly to a solution of S trityl 6 mercaptohexanol 2.0 g and N ethyl isopropylamine 4.2 g at 0 C and the mixture stirred for 15 minutes. Ethyl acetate 100 ml was then added and the solution extracted with saturated sodium bicarbonate solution 100 ml then brine 100 ml . The ethyl acetate extract was dried MgSO₄ , concentrated to 4.0 ml and applied to a column of silica gel Kieselgel 60, 12.5 cm 5.0 cm and fractionated using petroleum ether b.p. 40 60 C triethylamine 9 1 v v . After evaporation of solvents from combined appropriate fractions N,N diisopropylamino methoxy S trityl 6 mercaptohexoxy phosphine The fully protected oligodeoxyribonucleotide sequence alpha₂ 55 was prepared on an Applied Biosystems 380A DNA synthesiser from 5 dimethoxytrityl N² isobutyryl 2 deoxyguanosine bound to Fractosil via 3 OH and a succinylglycylglycylaminopropyl spacer BDH Chemicals Ltd and 2 cyanoethyl N,N diisopropylamino phosphoramidites of 5 dimethoxytrityl N⁶ benzoyl 2 deoxyadenosine, 5 dimethoxytrityl N⁴ benzoyl 2 deoxycytidine, 5 dimethoxytrityl N² isobutyryl 2 deoxyguanosine and 5 dimethoxytritylthymidine BDH Chemicals Ltd . Alternatively, the fully protected oligodeoxyribonucleotide sequence may be prepared by the manual methods as described by Atkinson and Smith in Oligonucleotide Synthesis, a Practical Approach . M J Gait Editor, IRL Press, Oxford, Washington DC pp 35 81 . N,N Diisopropylamino methoxy S trityl 6 mercaptohexoxyphosphine 53.7 mg was dried by azeotropic distillation with anhydrous acetonitrile 3 1 ml then dissolved in an anhydrous mixture of 1,2 dichloroethane acetonitrile 3 2 v v 1.5 ml . This solution was used in place of a normal nucleoside phosphoramidite in a standard coupling reaction on an Applied Biosystems 380A DNA synthesiser and added to the 5 end of the fully protected oligodeoxyribonucleotide sequence alpha₂ 55 described above. The procedures briefly consisted of 1 removal of the dimethoxytrityl group with 3 trichloroacetic acid in 1,2 dichloroethane 2 coupling of the N,N diisopropylamino methoxy S trityl 6 mercaptohexoxyphosphine 5.37 mg in 0.15 ml of solvent after activation with tetrazole 3 iodine oxidation of the intermediate phosphite to a phosphate 4 a capping step with acetic anhydride. Partial deprotection and cleavage of the oligonucleotide sequence from the Fractosil support was performed automatically on the DNA synthesiser. Phosphate protecting groups were removed by treatment with thiophenolate then with ammonium hydroxide d.0.88 which also cleaves the oligodeoxyribonucleotide sequence from the support. The ammonium hydroxide solution was heated at 55 C for 6 hours then evaporated and the residue dissolved in ethanol water 3 7 v 1.5 ml . The trityl protected sequence 15 was purified initially by hplc on Partisil 10 SAX ion exchange resin as described in Preparation 3 of Example 1. Further purification was achieved using reverse phase hplc on a C18 u Bondapak column as described by C R Newton Compound 15 40 pmoles was³²P labelled with 2 ,3 dideoxyadenosine 5 alpha ³²P triphosphate using a 3 end labelling kit Amersham International PLC as instructed by the manufacturer except that a mixture of 2 ,3 dideoxyadenosine 5 triphosphate and the alpha ³²P derivative 4 1 was used. The labelled oligodeoxyribonucleotide was purified on a NENSORB 20 cartridge DuPont NEN Research Products as instructed by the manufacturer. Silver nitrate 61 nmoles in water 1.25 ml was added to a solution of 15 1.5 nmoles and the 3 end labelled 15 0.1 nmoles in water 246 µl . After 30 minutes at 20 C a solution of dithiothreitol 63 nmoles in water 1.25 ml was added. After a further 30 minutes at 20 C the mixture was filtered through a Millex GS filter unit 0.22 µm and evaporated to a volume of 250 µl to give a solution of compound 16 1.5 nmoles and the ³²P 3 end labelled derivative 0.01 nmoles . A solution of m maleimidobenzoyl N hydroxysuccinimide ester MBS 0.12 mg in dimethylformamide DMF 5.0 µl was added to a solution of calf intestine alkaline phosphatase Boehringer Mannheim 567752 3.0 mg in 0.05M phosphate buffer, pH 7.0 0.7 ml . The mixture was stirred at 20 C for 30 minutes then applied to a Sephadex G 25 column 5 1.5 cm equilibrated with 0.1M phosphate buffer, pH 5.6 and eluted with the same buffer at 4 C. Fractions 0.5 ml were collected and monitored for UV absorbance at 280 nm to give compound 17 . The solution of 16 250 µl containing the tracer amount of ³²P3 end labelled derivative from Preparation 3 was added to the solution of derivatised alkaline phosphatase 17 from Preparation 4 . After 1 hour at 20 C the solution was applied to a Sephadex G 50 column 26 cm 1.0 cm equilibrated with 0.1M phosphate buffer pH5.6 and eluted at 0.9 ml min 45 pmoles of alpha₂ 55 oligonucleotide alkaline phosphatase complex in 0.88 ml of 0.1M sodium phosphate buffer at pH5.6 was concentrated to 0.25 ml in the same buffer as follows BSA Fraction V, Miles was added to 1 ug ml and 2 equal aliquots of the sample were loaded onto a Centricon centrifugal microconcentrator Amicon Ltd., Stonehouse, UK . The samples were concentrated by spinning the devices in a Sorvall SS34 rotor at 1000 g for 30 minutes. As described in Example 1, serially diluted samples of either pIF55 alpha₂ or pIF55 F6 alpha₂ plasmid were dot blotted onto nitrocellulose. Prehybridisations were also as described in Example 1. The hybridisation solution consisted of 2 ml of 5 SSC, 350 ug ml tRNA containing, 45 pmoles of oligonucleotide alkaline phosphatase complex 18 in 250 ul buffer. Following an overnight hybridisation at room temperature, filters were washed twice for 5 minutes at room temperature in a mixture containing 6 SSC, 0.06 sodium pyrophosphate and 20mM sodium phosphate, pH7.0. Filters were then further washed in the above mentioned buffer at 39 C for 2 minutes under which conditions the oligonucleotide complex hybridised to the interferon alpha₂ gene sequence with three terminal mismatches was washed off the filter to a much greater extent than the oligonucleotide complex which had formed a perfect hybrid with the interferon alpha₂ gene. The filters were further washed three times for 5 minutes in 0.1M Tris. HCl pH7.5, 0.1M NaCl, 2mMMgCl₂ and 0.05 Triton X 100. Finally the filters were washed twice for 5 minutes in 0.1M Tris,pH9.5, 0.1M NaCl, 5mMMgCl₂ and subsequently incubated with the AP substrate mixture as described in Example 3. Figure 2 shows the result of this experiment and indicates discrimination by the oligonucleotide alkaline phosphatase complex between left to right 300 ng and 30 ng of the immobilised pIF55alpha₂ plasmid DNA row A and the pIF55 F6alpha₂ derivative plasmid DNA row B which has a three base pair deviation from the interferon alpha₂ gene see Example 1 . The pIF55alpha₂ plasmid DNA row C and its derivative pIF55 F6alpha₂ row D , were also mock hybridised without the oligonucleotide complex but with 14 nmoles of alkaline phosphatase Boehringer, code no 567752 included in 2 ml of hybridisation buffer as specified above. No marked signal was seen associated with immobilised DNA in rows C and D thus ruling out artefacts due to direct binding of alkaline phosphatase. A molecule of alkaline phosphatase was covalently attached to a 15 base long oligodeoxyribonucleotide alpha₂ 55F6 with the nucleotide sequence of 20 by the procedure described in Preparations 2,3,4 and 5 of Example 8 with the following modifications The fully protected oligodeoxyribonucleotide alpha₂55F6 with the nucleotide sequence 20 was prepared on an Applied Biosystems 380A DNA synthesiser from 5 dimethoxytrityl N⁴ benzoyl 2 deoxycytidine bound to controlled pore glass Pore size 500A Particle size 125 174 micron loading 20 50 micromol g from Cruachan Chemical Co Ltd. . N,N diisopropylamino methoxy S trityl 6 mercaptohexoxyphosphine was coupled to the partially deprotected oligodeoxyribonucleotide bound to controlled pore glass step 2 by 2 additions, each of 100 second duration, of the tetrazole activated phosphine. The capping step step 4 was omitted. The ammonium hydroxide solution containing the oligodeoxyribonucleotide after cleavage from the support was heated at 50 C for 4 hours. The amounts of silver nitrate and dithiothreitol were increased to 120 nmoles and 122 nmoles respectively. The amount of MBS was reduced to 0.08 mg in DMF 3.4 ul and the alkaline phosphatase reduced to 2.0 mg in 0.05M phosphate buffer, pH 7.5 0.23 ml . The reaction was carried out for 1 hour at 37 C and the column was equilibrated with 0.1M phosphate buffer at pH 5.0. Using this procedure the alpha₂ 55F6 oligonucleotide alkaline phosphatase complex 19 was obtained. The alpha₂ 55F6 oligonucleotide alkaline phosphatase complex was directly hybridised to dilutions of immobilised pIF55alpha₂ or pIF55 F6alpha₂ DNA. Prehybridisation, hybridisation and washing conditions were as described for Example 8 except that the hybridisation was conducted in 10 ml of buffer including 120 pmoles of alpha₂ 55 F6 oligonucleotide AP complex 19 in 0.91 ml 0.1M sodium phosphate, pH5.0. Figure 3 shows the results of this experiment and again indicates discrimination by the oligonucleotide AP complex between left to right 300 ng and 30 ng of pIF55 alpha₂ row A and pIF55 F6alpha₂ row B , signal intensity produced by the probe being stronger for the homologous plasmid pIF55 F6alpha₂. The procedure described in Example 9 was repeated with oligodeoxyribonucleotide sequence 11 replacing the oligodeoxyribonucleotide sequence 20 . In Preparation 2 of Example 9 the oligodeoxyribonucleotide was prepared from 5 dimethoxytrityl N² isobutyryl 2 deoxyguanosine bound to controlled pore glass. Preparation 3 was carried out as described in Preparation 3 of Example 8. As described in Example 8 serially diluted samples of either pIF25alpha₂ or pIF25 14alpha₂ DNA were immobilised onto nitrocellulose filters. The filters were subseqently prehybridised, hybridised and washed as described in example 9 except that hybridisation was conducted in 10 ml ofbuffer including 170 pmoles of alpha₂ 25 oligonucleotide AP comlex 21 in 1 ml 0.1M sodium phosphate, pH5.6. Figure 4 shows the results of this experiment indicating much stronger hybridisation of the probe to left to right 300 ng, 30 ng and 3 ng of the homologous plasmid pIF25alpha₂ row A than to 300 ng, 30 ng and 3 ng the plasmid pIF25 14alpha₂ row B which had a single base non complementary to the oligonucleotide probe. The procedure described in Example 8 was repeated except that hybridisation included 45 pmoles of alpha₂ 55 oligonucleotide alkaline phosphatase complex 18 in 0.4 ml buffer, 0.025 BSA, 5 SSC and 250 µg ml tRNA in 4 ml total volume. Following hybridisation, filters were washed as in example 9 except that the final 5 minute washes in 0.1M Tris pH9.5, 0.1MNaCl and 5 nM MgCl₂ were replaced by a 5 minute wash at room temperature in washing buffer working strength from an Ampak enzyme amplification kit IQ Bio Ltd., Cambridge, UK . A further 5 minute wash in 0.1M Tris pH9.5, 0.1MNaCl and 5mMMgCl₂ at room temperature was followed by 5 sequential room temperature washes of 5 minutes each in Ampak washing buffer concentrate. Finally individual immobilised DNA samples were cut out of the filters and placed in individual wells of a microtitre dish 24 well, Falcon 3047 . To each sample was added 100 µl reconstituted Ampak Substrate mixture and incubated at room temperature for 20 minutes. 200 µl reconstituted Ampak Amplifier mixture was added and the solution left for 10 minutes for slow colour development before adding 50 µl stopping solution per well. Absorbances were then read at 495 nm. There was a clear increase in absorbance for the oligonucleotide complex hybridised to the pIF55alpha₂ plasmid DNA compared to the derivative plasmid pIF55 F6 alpha₂ DNA. For example, 30 ng of the pIF55 alpha₂ DNA gave rise to an OD₄₉₅ of approximately 0.7 units above background while the signal from pIF55 F6 alpha₂ was not significantly above background. The oligonucleotide alpha₂55 alkaline phosphatase complex 18 , as described in example 9, was used for hybridisation to a plasmid DNA human DNA mixture after Southern blot transfer to nitrocellulose. DNA was extracted from human Daudi Burkitt Lymphoma cells by the method of Blin and Stafford Nucleic Acids Research, Volume 3 1976 page 2303 and was dissolved at 1 mg ml in 10mM Tris HCl pH8, 1mM EDTA. To 200 ul of this DNA solution was added 120 µl H₂O, 40 µl of 10 Core Buffer Bethesda Research Laboratories and 40 ul EcoR1 Cambridge Biotechnology Laboratores, Cambridge UK 20 µ µl . The solution was incubated at 37 C for 3 hours and then extracted once with phenol chloroform 1 1 , once with chloroform and three times with ether. DNA was then precipitated by addition of sodium acetate to 0.3M and 2 volumes of ethanol. The pelletted DNA was washed in 70 ethanol and redissolved in 200 µl H₂O. 1.5 µg of plasmids containing either the interferon alpha₂ gene pIF55alpha₂ or its derivative including the oligonucleotide alpha₂ 55 pIF55 F6 alpha₂ were similarly digested with EcoRI using 20 units for pIF55alpha₂ and 40 units for pIF55 F6alpha₂ in volumes of 10 µl and 20µl respectively. DNA was solvent extracted and ethanol precipitated as above and finally dissolved in 15 µl H₂O. 10 µg samples of Daudi cell DNA were mixed with either 300 ng and 30 ng of pIF55 alpha₂ or pIF55 F6 alpha₂ DNA and electrophoresed in a 1 agarose gel as described by Maniatis et al. Molecular Cloning A Laboratory Manual Cold Spring Harbor Laboratory, 1982 . DNA was then transferred to nitrocellulose using the Southern transfer procedure as described in Maniatis et al see above . The filter was air dried and baked The oligonucleotide alpha₂ 25 AP complex 21 as described in example 10 was used for hybridisation to a plasmid DNA human DNA mixture after Southern blot transfer to nitrocellulose. The plasmid DNA s utilised in the experiment were pIF25 alpha₂ and pIF25 I4 alpha₂ digested as described for pIF55 alpha₂ and pIF55 F6 alpha₂ respectively in Example 12. Samples of EcoRl digested plasmid DNA was mixed with EcoRI digested Daudi cell DNA and subjected to electrophoresis as in Example 12. Prehybridisation, hybridisation, washing and oligonucleotide AP detection conditions were as in Example 10. As observed for example 12, aliquots of pIF25 alpha₂ plasmid DNA, homologous to the probe, could be detected as a single band to an equal extent in the presence or absence of Daudi cell DNA. Under these same conditions, no signal corresponding to the plasmid pIF25 I4alpha₂ either in the presence or absence of Daudi cell DNA was detectable demonstrating discrimination, following Southern transfer, by the alpha₂ 25 oligonucleotide AP probe 21 between the homologous plasmid, pIF25 alpha₂, and the plasmid pIF25 I4 alpha₂ which has a single base non complementary to the probe. In the following formulae drawings Z represents an oligonucleotide and AP represents alkaline phosphatase. Throughout the specification the terminal phosphate of an oligonucleotide is considered to be part of the oligonucleotide, but in the drawings the terminal phosphate has been shown separately so as to illustrate the reactions which occur on this phosphate moiety.