# Cloned human serum albumin gene.

## Abstract
Disclosed are a synthetic human serum albumin gene, plasmids containing the gene, and microorganisms trans formed by those plasmids.

## Claims
CLAIMS FOR THE DESIGNATED STATES BE, DE, FR, IT, LU, NL, SE, CH and UK 1. A synthetic gene coding for human serum albumin. 2. An isolated human serum albumin gene. 3. An isolated prepro human serum albumin gene. 4. A human serum albumin gene as claimed in claim 1, comprising the following deoxyribonucleotide sequence which corresponds to the indicated amino acid sequence Asp Ala His Lys Ser Glu Val Ala GA GCX CAY AAM QRS GAM G T X G C X His Arg Phe Lys Asp Leu Gly Glu CAY LGN TTY AAM GAY YTZ G G X G A M Glu Asn Phe Lys Ala Leu Val LeuGAM AAY TTY AAM GCX YTZ GTX Y T Z Ile Ala Phe Ala Gln Tyr Leu Gln ATH GCX TTY GCX CAM TAY Y T Z C A M Gln Cys Pro Phe Glu Asp His Val CAM TGY CCX TTY GAM GAY CAY GTX Lys Leu Val Asn Glu Val Thr Glu AAM YTZ GTX AAY GAM GTX A C X G A M Phe Ala Lys Thr Cys Val Ala AspTTY GCX AAM ACX TGY GTX G C X G A Y Glu Ser Ala Glu Asn Cys Asp LysGAM QRS GCX GAM AAY TGY G A Y A A M Ser Leu His Thr Leu Phe Gly Asp ORS YTZ CAY ACX YTZ TTY G G X G A Y Lys Leu Cys Thr Val Ala Thr Leu A A M Y.T Z T G Y A C X G T X G C X A C X Y T Z Arg Glu Thr Tyr Gly Glu Met Ala LGN GAM ACX TA GGX GAM ATG GCX Asp Cys Cys Ala Lys Gln Glu Pro GAY TG TG GCX AAM CAM GAM CCX Glu Arg Asn Glu Cys Phe Leu GlnG AM LGN AAY GAM TGY TT Y Y T Z CAM His Lys Asp Asp Asn Pro Asn Leu CAY AAM GAY GA AAY CCX AAY YTZ Pro Arg Leu Val Arg Pro Glu Val CCX LGN YTZ GTX LGN CCX G A M GTX Asp Val Met Cys Thr Ala Phe His GAY GTX ATG TGY ACX GCX TTY CAY Asp Asn Glu Glu Thr Phe Leu LysGAY AAY GAM GAM ACX TTY Y T Z A A M Lys Tyr Leu Tyr Glu Ile Ala ArgAAM TAY YTZ TAY GAM ATH GCX LGN Arg His Pro Tyr Phe Thr Ala Pro LGN CAY CCX TAY TTY ACX G C X C C X Glu Leu Leu Phe Phe Ala Lys ArgG AM YTZ YTZ TTY TTY GCX A A M L G N Tyr Lys Ala Ala Phe Thr Glu Cys TAY AAM GCX GCX TTY ACX GAM TGY Cys Ala Gln Ala Asp Lys Ala AlaTGY GCX CAM GCX GAY AAM G C X G C X Cys Leu Phe Pro Lys Leu Asp GluTGY YTZ TTY CCX AAM YTZ G A Y G A M Leu Arg Asp Glu Gly Lys Ala Ser YTZ LGN GA GAM GGX AAM GCX QRS Ser Ala Lys Gln Arg Leu Lys CysORS GCX AAM CAM LGN YTZ AAM TGY Ala Ser Leu Gln Lys Phe Gly GluGCX QRS YTZ CAM AAM TTY G G X G A M Arg Ala Phe Lys Ala Trp Ala ValLGN GCX TTY AAM GCX TGG G C X G T X Ala Arg Leu Ser Gln Arg Phe Pro GCX LGN YTZ Q R S CAM LGN TTY CCX Lys Ala Glu Phe Ala Glu Val Ser AAM GCX GAM TTY GCX GAM G T X Q R S Lys Phe Val Thr Asp Leu Thr Lys AAM TT GTX ACX GAY YTZ ACX AAM Val His Thr Glu Cys Cys His GlyG T X C A Y A C X G A M T G Y T G Y C A RTI ID 29.11 Y G G X Asp Leu Leu Glu Cys Ala Asp AspGA YTZ YTZ GAM TGY GCX GAY GAY Arg Ala Asp Leu Ala Lys Tyr IleLGN GCX GAY YTZ GCX AAM TAT ATH Cys Glu Asn Gln Asp Ser Ile SerT G Y GAM AAY C A M G A Y Q R S ATH Q R S Ser Lys Leu Lys Glu Cys Cys GluQRS AAM YTZ AAM GAM TG T G Y G A M Lys Pro Leu Phe Glu Lys Ser HisAAM CCX YTZ TT GAM AAM Q R S C A Y Cys Ile Ala Glu Val Glu Asn Asp TGY ATH G C X GAM GTX GAM A A Y GAY Glu Met Pro Ala Asp Phe Pro SerGAM ATG CCX GCX GAY TTY CCX QRS Phe Ala Val Asp Phe Val Glu SerTTY GCX GTX GAY TT GTX G A M QRS Lys Asp Val Cys Lys Asn Tyr AlaAAM GAY GTX TGY AAM AAY T A Y G C X Glu Ala Lys Asp Val Phe Leu GlyGAM GCX AAM GAY GTX TTY Y T Z G G X Met Phe Phe Tyr Glu Tyr Ala ArgATG TTY TTY TAY GAM TAY GCX LGN Arg His Pro Asp Tyr Ser Val Val L G N C A Y C C X G A Y T A Y Q R S G T X G T X laeu Leu Leu Arg Leu Ala Lys ThrYTZ TZ YTZ LGN TZ GCX A A M A C X Tyr Glu Thr Thr Leu Glu Lys CysTAY GAM ACX ACX YTZ GAM A A M TGY Cys Ala Ala Ala Asp Pro His GluTG GCX GCX GCX GAY CCX C A Y G A M Cys Tyr Ala Lys Val Phe Asp Glu TGY T A Y GCX A A M G T X TTY G A Y GAM Phe Lys Pro Pro Val Glu Glu Pro TT AAM CCX CCX GTX GAM GAM CCX Gln Asn Phe Ile Lys Gln Asn Cys CAM AAY TT ATH AAM CAM A A Y T G Y Glu Leu Phe Glu Gin Leu Gly GluGAM YTZ TTY GAM CAM YTZ G G X G A M Tyr Lys Phe Gin Asn Ala Leu PheTAY AAM TTY CAM AAY GCX Y T Z T T Y Val Arg Tyr Thr Lys Lys Val Pro GTX LGN TA ACX AAM AAM G T X CCX Gln Leu Ser Thr Pro Thr Leu Val CAM TZ QRS ACX CCX ACX TZ GTX Glu Val Ser Arg Asn Leu Gly LysGAM GTX QRS laGN AAY YTZ G G X A A M Val Gly Ser Lys Cys Cys Lys His GTX GGX QRS AAM TGY TG AAM C A Y Pro Glu Ala Lys Arg Met Pro CysCCX GAM GCX AAM LGN ATG CCX TGY Ala Glu Asp Tyr Leu Ser Val ValGCX GAM GAY TAY YTZ QRS G T X GTX Leu Asn Gln Leu Cys Val Leu HisYTZ AAY CAM YTZ TGY GTX Y T Z CAY Glu Lys Thr Pro Val Ser Asp Arg GAM AAM ACX CCX GTX QRS GAY AYLGN Val Thr Lys Cys Cys Thr Glu SerGTX ACX AAM TG TG ACX GAM QRS Leu Val Asn Arg Arg Pro Gly Phe TZ GTX A laGN laGN CCX G G X T T Y Ser Ala Leu Glu Val Asp Glu ThrQRS GCX TZ GAM GTX GA GAM ACX Tyr Val Pro Lys Glu Phe Asn AlaTAY GTX CCX AAM GAM TTY A A Y GCX Glu Thr Phe Thr Phe His Ala AspGAM ACX TTY ACX TTY CA G C X G A Y Ile Cys Thr Leu Ser Glu Lys GluATH TG ACX TZ QRS GAM A A M G A M Arg Gln Ile Lys Lys Glu Thr Ala laGN CAM ATH AAM AAM GAM A C X G C X Leu Val Glu Leu Val Lys His Lys YTZ GTX GAM TZ GTX AAM CAY AAM Pro Lys Ala Thr Lys Glu Glu LeuCCX AAM GCX ACX AAM GAM G A M Y T Z Lys Ala Val Met Asp Asp Phe Ala AAM GCX GTX ATG GA GAY T T Y G C X Ala Phe Val Glu Lys Cys Cys Lys GCX TT GTX GAM AAM TG TGY AAM Ala Asp Asp Lys Glu Thr Cys PheGCX GAY GAY AAM GAM ACX T G Y T T Y Ala Glu Glu Gly Lys Lys Leu ValGCX GAM GAM GGX AAM AAM Y T Z G T X Ala Ala Ser Glu Ala Val Leu GlyGCX GCX QRS GAM GCX GTX Y T Z G G X LeuY T Z T A A wherein, the 5 to 3 strand, beginning with the amino terminus and the amino acids for which each triplet codes are shown, and wherein the abbreviations have the following standard meanings A is deoxyadenyl T is thymidyl G is deoxyguanyl C is deoxycytosyl X is A, T, C or G Y is T or C When Y is C, Z is A, T, C or G When Y is T, Z is A or G H is A, T or C Q is T or A When Q is T, R is C and S is A, T, C or G When Q is A, R is G and S is T or C M is A or G r. is A or C When L is A, N is A or G When L is C, N is A, T, C or G GLY is glycine ALA is alanine VAL is valine LEU is leucine ILE is isoleucine SER is serine THR is threonine PHE is phenylalanine TYR is tyrosine TRP is tyryptophan C S is cysteine MET is methionine ASP is aspartic acid GLU is glutamic acid LYS is lysine ARG is arginine HIS is histidine PRO is proline GLN is glutamine ASN is asparagine 5. A prepro serum albumin gene as claimed in claim 1 comprising the following deoxyribonucleotide sequence Met Lys Trp Val Thr Phe ATG AAM TGG GTX ACX TTY Ile Ser Leu Leu Phe Leu PheATH QRS YTZ YTZ TTY YTZ TTY Ser Ser Ala Tyr Ser Arg Gly Q R S Q R S G C X T A Y Q R S L G N G G X Val Phe Arg Arg Asp Ala His LysGTX TTY laGN LGN GAY GCX CAY A A M Ser Glu Val Ala His Arg Phe LysORS GAM GTX GCX CAY laGN TTY AAM Asp Leu Gly Glu Glu Asn Phe Lys GA YTZ GGX GAM GAM AA TTY AAM Ala Leu Val teu Ile Ala Phe AlaGCX YTZ GTX YTZ ATH GCX T T Y G C X Gln Tyr Leu Gln Gln Cys Pro PheC A M T A Y Y T Z C A M C A M T G Y C C X T T Y Glu Asp His Val Lys Leu Val AsnGAM GAY CAY GTX AAM YTZ GTX AA Glu Val Thr Glu Phe Ala Lys Thr GAM GTX ACX GAM TT GCX A A M A C X Cys Val Ala Asp Glu Ser ALa Glu TG GTX GCX GAY GAM QRS G C X G A M Asn Cys. Asp Lys Ser Leu His ThrAAY TGY GAY AAM ORS YTZ C A Y A C X Leu Phe Gly Asp Lys Leu Cys Thr YTZ TT GGX GAY AAM YTZ TGY ACX Val Ala Thr Leu Arg Glu Thr TyrGTX GCX ACX YTZ LGN GAM A C X T A Y Gly Glu Met Ala Asp Cys Cys AlaGGX GAM ATG GCX GA TG TGY GCX Lys Gln Glu Pro Glu Arg Asn GluAAM CAM GAM CCX GAM laGN A A Y G A M Cys Phe Leu Gln His Lys Asp AspTGY TTY YTZ CAM CAY AAM GAY GAY Asn Pro Asn Leu Pro Arg Leu ValAAY CCX AAY YTZ CCX LGN Y T Z G T X Arq Pro Glu Val Asp Val Met CysLGN CCX GAM GTX GAY GTX A T G T G Y Thr Ala Phe His Asp Asn Glu Glu ACX GCX TT CAY GAY AAY G A M GAM Thr Phe Leu Lys Lys Tyr Leu TyrACX TT TZ AAM AAM TA Y Y T Z T A Y Glu Ile Ala Arg Arg His Pro TyrGAM ATH GCX laGN LGN CA CCX TA Phe Thr Ala Pro Glu Leu Leu Phe TT ACX GCX CCX GAM YTZ Y T Z T T Y Phe Ala Lys Arg Tyr Lys Ala AlaTTY GCX AAM LGN TAY AAM G C X GCX Phe Thr Glu Cys Cys Ala Gln AlaTT ACX GAM TGY TG GCX CAM GCX Asp Lys Ala Ala Cys Leu Phe ProG A Y A A M G C X G C X T G Y Y T Z T T Y C C X Lys Leu Asp Glu Leu Arg Asp GluAAM TZ GAY GAM TZ laG GA GAM Gly Lys Ala Ser Ser Ala Lys Gln GGX AAM G C X Q R S Q R S G C X AAM CAM Arg Leu Lys Cys Ala Ser Leu Gln laGN YTZ AAM TGY GCX ORS Y T Z C A M Lys Phe Gly Glu Arg Ala Phe Lys AAM TT GGX GAM laGN GCX TTY AAM Ala Trp Ala Val Ala Arg Leu SerGCX TGG GCX GTX GCX LGN YTZ QRS Gln Arg Phe Pro Lys Ala Glu PheCAM LGN TTY CCX AAM GCX GAM T T Y Ala Glu Val Ser Lys Phe Val ThrGCX GAM GTX ORS AAM TTY G T X ACX Asp Leu Thr Lys Val His Thr GluGAY YTZ ACX AAM GTX CAY ACT G A M Cys Cys His Gly Asp Leu Leu GluTG TG CA GGX GA TZ TZ G A M Cys Ala Asp Asp Arg Ala Asp LeuTG GCX GA GAY laGN GCX G A Y Y T Z Ala Lys Tyr Ile Cys Glu Asn GlnG C X A. A M T A A T H T G G A M A A C A M Asp Ser Ile Ser Ser Lys Leu Lys GA QRS ATH QRS QRS AAM Y T Z A A M Glu Cys Cys Glu Lys Pro Leu PheGAM TGY TG GAM AAM CCX YTZ TTY Glu Lys Ser His Cys Ile Ala GluG A M A A M Q R S C A Y T G Y A T H G C X G A M Val Glu Asn Asp Glu Met Pro AlaG T X GAM AAY GAY GAM ATG CCX GCX Asp Phe Pro Ser Phe Ala Val Asp GAY TT CCX QRS TT GCX GTX GAY Phe Val Glu Ser Lys Asp Val CysTTY GTX GAM ORS AAM GA G T X T G Y Lys Asn Tyr Ala Glu Ala Lys AspAAM AAY TAY GCX GAM GCX AAM GAY Val Phe Leu Gly Met Phe Phe TyrGTX TTY YTZ GGX ATG TTY T T Y T A Y Glu Tyr Ala Arg Arg His Pro AspGAM TAY GCX LGN LGN CAY CCX GAY Tyr Ser Val Val Leu Leu Leu ArgT A Y QRS GTX G T X Y T Z Y T Z YTZ LGN Leu Ala Lys Thr Tyr Glu Thr ThrYTZ GCX AAM ACX TAY GAM A C X A C X Leu Glu Lys Cys Cys Ala Ala AlaYTZ GAM AAM TGY TGY GCX G C X G C X Asp Pro His Glu Cys Tyr Ala LysGAY CCX CAY GAM TGY TAY G C X A A M Val Phe Asp Glu Phe Lys Pro ProGTX TTY GAY GAM TTY AAM CCX CCX Val Glu Glu Pro Gln Asn Phe IleGTX GAM GAM CCX CAM AAY TTY ATH Lys Gln Asn Cys Glu Leu Phe Glu AAM CAM AAY T G Y GAM YTZ TTY GAM Gln Leu Gly Glu Tyr Lys Phe GlnCAM YTZ GGX GAM TAY AAM TTY CAM Asn Ala Leu Phe Val Arg Tyr ThrAAY GCX YTZ TTY GTX LGN T A Y A C X Lys Lys Val Pro Gln Leu Ser ThrAAM AAM GTX CCX CAM YTZ Q R S A C X Pro Thr Leu Val Glu Val Ser ArgCCX ACX YTZ GTX GAM GTX ORS LGN Asn Leu Gly Lys Val Gly Ser LysAAY YTZ GGX AAM GTX GGX Q R S A A M Cys Cys Lys His Pro Glu Ala LysTGY TGY AAM CAY CCX GAM G C X A A M Arg Met Pro Cys Ala Glu Asp TyrLGN ATG CCX TGY GCX GAM G A Y T A Y Leu Ser Val Val Leu Asn Gln LeuYTZ QRS GTX GTX YTZ AAY C A M Y T Z Cys Val Leu His Glu Lys Thr Pro TGY GTX TZ CAY GAM AAM ACT CCX Val Ser Asp Arg Val Thr Lys Cys GTX QRS GAY LiGN GTX ACX AAM TGY Cys Thr Glu Ser Leu Val Asn ArgTGY ACX GAM ORS YTZ GTX AWAY LION Arg Pro Gly Phe Ser Ala Leu Glu LiGN CCX GGX TTY QRS GCX YTZ G A M Val Asp Glu Thr Tyr Val Pro Lys GTX GAY GAM ACX TA GTX CCX AAM Glu Phe Asn Ala Glu Thr Phe ThrGAM TTY AAY GCX GAM ACX T T Y A C X Phe His Ala Asp Ile Cys Thr Leu T T Y C A Y G C X G A Y A T H T G Y A C X Y T Z Ser Glu Lys Glu Arg Gln Ile Lys QRS GAM AAM GAM LiGN CAM ATH AAM Lys Glu Thr Ala Leu Val Glu Leu AAM GAM ACX GCX TZ GTX G A M Y T Z Val Lys His Lys Pro Lys Ala ThrGTX AAM CAY AAM CCX AAM G C X A C X Lys Glu Glu Leu Lys Ala Val MetAAM GAM GAM YTZ AAM GCX G T X A T G Asp Asp Phe Ala Ala Phe Val GluGAY GAY TTY GOX GCX TTY G T X G A M Lys Cys Cys Lys Ala Asp Asp Lys AAM TGY TGY AAM GCX GA GAY AAM Glu Thr Cys Phe Ala Glu Glu Gly G AM ACX TG TTY GCX GAM G A M G G X Lys Lys Leu Val Ala Ala Ser Glu AAM AAM YTZ GTX GCX GCX Q R S G A M Ala Val Leu Gly Leu GCX GTX YTZ GGX TZ TAA wherein the 5 and 3 strand, beginning with the amino terminus, and the amino acids for which each triplet codes are shown, and wherein the abbreviations are defined as in claim 4. 6. A human serum albumin gene as claimed in claim 4 comprising the following deoxyribonucleotide sequence GAT GCA CAC AAG AGT GAG GTT GCT CAT CGG TTT AAA GAT TTGGGA GAA GAA AAT TTC AAA GCC TTG GTG TTG ATT GCC TTT GCTCAG TAT CTT CAG CAG TGT CCA TTT GAA GAT CAT GTA AAA TTAGTG AAT GAA GTA ACT GAA TTT GCA AAA ACA TGT GTT GCT GATGAG TCA GCT GAA AAT TGT GAC AAA TCA CTT CAT ACC CTT TTTGGA GAC AAA TTA TGC ACA GTT GCA ACT CTT CGT GAA ACC TATGGT GAA ATG GCT GAC TGC TGT GCA AAA CAA GAA CCT GAG AGAAAT GAA TGC TTC TTG CAA CAC AAA GAT GAC AAC CCA AAC CTCCCC CGA TTG GTG AGA CCA GAG GTT GAT GTG ATG TGC ACT GCTTTT CAT GAC AAT GAA GAG ACA TTT TTG AAA AAA TAC TTA TATGAA ATT GCC AGA AGA CAT CCT TAC TTT TAT GCC CCG GAA CTCCTT TTC TTT GCT AAA AGG TAT AAA GCT GCT TTT ACA GAA TGTTGC CAA GCT GCT GAT AAA GCT GCC TGC CTG TTG CCA AAG CTCGAT GAA CTT CGG GAT GAA GGG AAG GCT TCG TCT GCC AAA CAGAGA CTC AAG TGT GCC AGT CTC CAA AAA TTT GGA GAA AGA GCTTTC AAA GCA TGG GCG GTG GCT CGC CTG AGC CAG AGA TTT CCCAAA GCT GAG TTT GCA GAA GTT TCC AAG TTA GTG ACA GAT CTTACC AAA GTC CAC ACG GAA TGC TGC CAT GGA GAT CTG CTT GAATGT GCT GAT GAC AGG GCG GAC CTT GCC AAG TAT ATC TGT GAAAAT CAA GAT TCG ATC TCC AGT AAA CTG AAG GAA TGC TGT GAA AAA CCT CTG TTG GAA AAA TCC CAC TGC ATT GCC GAA GTG GAAAAT GAT GAG ATG CCT GCT GAC TTG CCT TCA TTA GCT GCT GATTTT GTT GAA AGT AAG GAT GTT TGC AAA AAC TAT GCT GAG GCAAAG GAT GTC TTC CTG GGC ATG TTT TTG TAT GAA TAT GCA AGAAGG CAT CCT GAT TAC TCT GTC GTG CTG CTG CTG AGA CTT GCCAAG ACA TAT GAA ACC ACT CTA GAG AAG TGC TGT GCC GCT GCAGAT CCT CAT GAA TGC TAT GCC AAA GTG TTC GAT GAA TTT AAACCT CCT GTG GAA GAG CCT CAG AAT TTA ATC AAA CAA AAT TGTGAG CTT TTT GAG CAG CTT GGA GAG TAC AAA TTC CAG AAT GCGCTA TTA GTT CGT TAC ACC AAG AAA GTA CCC CAA GTG TCA ACTCCA ACT CTT GTA GAG GTC TCA AGA AAC CTA GGA AAA GTG GGCAGC AAA TGT TGT AAA CAT CCT GAA GCA AAA AGA ATG CCC TGTGCA GAA GAC TAT CTA TCC GTG GTC CTG AAC CAG TTA TGT GTGTTG CAT GAG AAA ACG CCA GTA AGT GAC AGA GTC ACC AAA TGCTGC ACA GAA TCC TTG GTG AAC AGG CGA CCA TGC TTT TCA GCTCTG GAA GTC GAT GAA ACA TAC GTT CCC AAA GAG TTT AAT GCTGAA ACA TTC AnC TTC CAT GCA GAT ATA TGC ACA CTT TCT GAGAAG GAG AGA CAA ATC AAG AAA CAA ACT GCA CTT GTT GAG CTCGTG AAA CAC AAG CCC AAG GCA ACA AAA GAG CAA CTG AAA GCTGTT ATG GAT GAT TTC GCA GCT TTT GTA GAG AAG TGC TGC AAGGCT GAC GAT AAG GAG ACC TGC TTT GCC GAG GAG GGT AAA AAACTT GTT GCT GCA AGT CAA GCT GCC TTA GGC TTA TAA wherein the 5 to 3 strand, beginning with the amino terminus is shown, and wherein the abbreviations are defined as in claim 4. 7. A hunan Pre preseran albumin gene as claimed in claim 5 comprising the following deoxyribonucleotide sequence ATG AAG TGG GTA ACC TTT ATT TCC CTT CTT TTT CTC TTT AGCTCG GCT TAT TCC AGG GGT GTG TTT CGT CGA GAT GCA CAC AAGAGT GAG GTT GCT CAT CGG TTT AAA GAT TTG GGA GAA GAA AATTTC AAA GCC TTG GTG TTG ATT GCC TTT GCT CAG TAT CTT CAGCAG TGT CCA TTT GAA GAT CAT GTA AAA TTA GTG AAT GAA GTAACT GAA TTT GCA AAA ACA TGT GTT GCT GAT GAG TCA GCT GAA AAT TGT GAC AAA TCA CTT CAT ACC CTT TTT GGA GAC AAA TTATGC ACA GTT GCA ACT CTT CGT GAA ACC TAT GGT GAA ATG GCTGAC TGC TGT GCA AAA CAA GAA CCT GAG AGA AAT GAA TGC TTCTTG CAA CAC AAA GAT GAC AAC CCA AAC CTC CCC CGA TTG GTGAGA CCA GAG GTT GAT GTG ATG TGC ACT GCT TTT CAT GAC AATGAA GAG ACA TTT TTG AAA AAA TAC TTA TAT GAA ATT GCC AGAAGA CAT CCT TAC TTT TAT GCC CCG GAA CTC CTT TTC TTT GCTAAA AGG TAT AAA GCT GCT TTT ACA GAA TGT TGC CAA GCT GCTGAT AAA GCT GCC TGC CTG TTG CCA AAG CTC GAT GAA CTT CGGGAT GAA GGG AAG GCT TCG TCT GCC AAA CAG AGA CTC AAG TGTGCC AGT CTC CAA AAA TTT GGA GAA AGA GCT TTC AAA GCA TGGGCG GTG GCT CGC CTG AGC CAG AGA TTT CCC AAA GCT GAG TTTGCA GAA GTT TCC AAG TTA GTG ACA GAT CTT ACC AA GTC CACACG GAA TGC TGC CAT GGA GAT CTG CTT GAA TGT GCT GAT GACAGG GCG GAC CTT GCC AAG TAT ATC TGT GAA AAT CAA GAT TCGATC TCC AGT AAA CTG AAG GAA TGC TGT GAA AAA CCT CTG TTGGAA AAA TCC CAC TGC ATT GCC GAA GTG GAA AAT GAT GAG ATGCCT GCT GAC TTG CCT TCA TTA GCT GCT GAT TTT GTT GAA AGTAAG GAT GTT TGC AAA AAC TAT GCT GAG GCA AAG GAT GTC TTCCTG GGC ATG TTT TTG TAT GAA TAT GCA AGA AGG CAT CCT GATTAC TCT GTC GTG CTG CTG CTG AGA CTT GCC AAG ACA TAT GAAACC ACT CTA GAG AAG TGC TGT GCC GCT GCA GAT CCT CAT GAATGC TAT GCC AAA GTG TTC GAT GAA TTT AAA CCT CCT GTG GAAGAG CCT CAG AAT TTA ATC AAA CAA AAT TGT GAG CTT TTT GAGCAG CTT GGA GAG TAC AAA TTC CAG AAT GCG CTA TTA GTT CGTTAC ACC AAG AAA GTA CCC CAA GTG TCA ACT CCA ACT CTT GTAGAG GTC TCA AGA AAC CTA GGA AAA GTG GGC AGC AAA TGT TGTAAA CAT CCT GAA GCA AAA AGA ATG CCC TGT GCA GAA GAC TATCTA TCC GTG GTC CTG AAC CAG TTA TGT GTG TTG CAT GAG AAAACG CCA GTA AGT GAC AGA GTC ACC AAA TGC TGC ACA GAA TCCTTG GTG AAC AGG CGA CCA TGC TTT TCA GCT CTG GAA GTC GATGAA ACA TAC GTT CCC AAA GAG TTT AAT GCT GAA ACA TTC ACCTTC CAT GCA GAT ATA TGC ACA CTT TCT GAG AAG GAG AGA CAAATC AAG AAA CAA ACT GCA CTT GTT GAG CTC GTG AAA CAC AAGCCC AAG GCA ACA AAA GAG CAA CTG AAA GCT GTT ATG GAT GAT TTC GCA GCT TTT GTA GAG AAG TGC TGC AAG GCT GAC GAT AAGGAG ACC TGC TTT GCC GAG GAG GGT AAA AAA CTT GTT GCT GCAAGT CAA GCT GCC TTA GGC TTA TAA wherein the 5 to 3 strand, beginning with the amino terminus is shown, and wherein the abbreviations are defined as in claim 4. 8. A hunan prepro serum albumin gene as claimed in claim 7 ccrrpirised in the following deoxyribonucleotide sequence 5 TCTCTTCTGTCAACCCCACGCCTTTGGCACA ATG AAG TGG GTAACC TTT ATT TCC CTT CTT TTT CTC TTT AGC TCG GCT TAT TCCAGG GGT GTG TTT CGT CGA GAT GCA CAC AAG AGT GAG GTT GCTCAT CGG TTT AAA GAT TTG GGA GAA GAA AAT TTC AAA GCC TTGGTG TTG ATT GCC TTT GCT CAG TAT CTT CAG CAG TGT CCA TTTGAA GAT CAT GTA AAA TTA GTC AAT GAA GTA ACT GAA TTT GCAAAA ACA TGT GTT GCT GAT GAG TCA GCT GAA AAT TGT GAC AAATCA CTT CAT ACC CTT TTT GGA GAC AAA TTA TGC ACA GTT GCAACT CTT CGT GAA ACC TAT GGT GAA ATG GCT GAC TGC TGT GCAAAA CAA GAA CCT GAG AGA AAT GAA TGC TTC TTG CAA CAC AAAGAT GAC AAC CCA AAC CTC CCC CGA TTG GTG AGA CCA GAG GTTGAT GTG ATG TGC ACT GCT TTT CAT GAC AAT GAA GAG ACA TTTTTG AAA AAA TAC TTA TAT GAA ATT GCC AGA AGA CAT CCT TACTTT TAT GCC CCG GAA CTC CTT TTC TTT GCT AAA AGG TAT AAAGCT GCT TTT ACA GAA TGT TGC CAA GCT GCT GAT AAA GCT GCCTGC CTG TTG CCA AAG CTC GAT GAA CTT CGG GAT GAA GGG AAGGCT TCG TCT GCC AAA CAG AGA CTC AAG TGT GCC AGT CTC CAAAAA TTT GGA GAA AGA GCT TTC AAA GCA TGG GCG GTG GCT CGCCTG AGC CAG AGA TTT CCC AAA GCT GAG TTT GCA GAA GTT TCCAAG TTA GTG ACA GAT CTT ACC AAA GTC CAC ACG GAA TGC TGCCAT GGA GAT CTG CTT GAA TGT GCT GAT GAC AGG GCG GAC CTTGCC AAG TAT ATC TGT GAA AAT CAA GAT TCG ATC TCC AGT AAACTG AAG GAA TGC TGT GAA AAA CCT CTG TTG GAA AAA TCC CACTGC ATT GCC GAA GTG GAA AAT GAT GAG ATG CCT GCT GAC TTGCCT TCA TTA GCT GCT GAT TTT GTT GAA AGT AAG GAT GTT TGCAAA AAC TAT GCT GAG GCA AAG GAT GTC TTC CTG GGC ATG TTT TTG TAT GAA TAT GCA AGA AGG CAT CCT GAT TAC TCT GTC GTGCTG CTG CTG AGA CTT GCC AAG ACA TAT GAA ACC ACT CTA GAGAAG TGC TGT GCC GCT GCA GAT CCT CAT GAA TGC TAT GCC AAAGTG TTC GAT GAA TTT AAA CCT CCT GTG GAA GAG CCT CAG AATTTA ATC AAA CAA AAT TGT GAG CTT TTT GAG CAG CTT GGA GAGTAC AAA TTC CAG AAT GCG CTA TTA GTT CGT TAC ACC AAG AAAGTA CCC CAA GTG TCA ACT CCA ACT CTT GTA GAG GTC TCA AGAAAC CTA GGA AAA GTG GGC AGC AAA TGT TGT AAA CAT CCT GAAGCA AAA AGA ATG CCC TGT GCA GAA GAC TAT CTA TCC GTG GTCCTG AAC CAG TTA TGT GTG TTG CAT GAG AAA ACG CCA GTA AGTGAC AGA GTC ACC AAA TGC TGC ACA GAA TCC TTG GTG AAC AGGCGA CCA TGC TTT TCA GCT CTG GAA GTC GAT GAA ACA TAC GTTCCC AAA GAG TTT AAT GCT GAA ACA TTC ACC TTC CAT GCA GATATA TGC ACA CTT TCT GAG AAG GAG AGA CAA ATC AAG AAA CAAACT GCA CTT GTT GAG CTC GTG AAA CAC AAG CCC AAG GCA ACAAAA GAG CAA CTG AAA GCT GTT ATG GAT GAT TTC GCA GCT TTTGTA GAG AAG TGC TGC AAG GCT GAC GAT AAG GAG ACC TGC TTCGCC GAG GAG GGT AAA AAA CTT GTT GCT GCA AGT CAA GCT GCCTTA GGC TTA TAA CATCTACATTTAAAAGCATCTCAGCCTACCATGAGAATA RTI ID 42.1 AGAGAAAGAAAATGAAGATCAAAAGCTTATTCATCTGTTTTCTTTTTCGTTGGT GTTTTAATCATTTTGCCTCTTTTCTCTGTGCTTCAATTAATAAAAAATGGAAAGAA TCTAA wherein the 5 to 3 strand, beginning with the amino terminus is shown, and wherein the abbreviations are defined as in claim 4. 9. A plasmid having the capability of replication in a prokaryotic or eukaryotic organism, comprising a deoxyribo nucleotide sequence coding for human serum albumin. 10. A plasmid as claimed in claim 8 having the capability of replication in a prokaryotic organism, comprising a human serum albumin or human preproserum albumin gene as claimed in any one of claims 1 to 8. 11. A plasmid as claimed in claim 9 or claim 10 having the capability of replication in a prokaryotic organism of the genus Escherichia. 12. The plasmid of claim 10 designated pGX401 deposited in E. coli HB101 at the U.S. Dept. ofAgriculture Northern Regional Research Center,Peoria, Illinois under accession No. NRRL B 15784 and mutants thereof encoding human serum albumin. 13. A microorganism transformed by a plasmid as claimed in any one of claims 9 to 12. 14. A microorganism as claimed in claim 13 of the genus Escherichia. 15. A microorganism as claimed in claim 14 of the species coli. 16. A method of producing prepro human serum albumin which comprises cultivating on an aqueous nutrient medium containing assimilable sources of carbon, nitrogen and essential minerals and growth factors, under prepro human serum albumin producing conditions a prokaryotic organism as claimed in claim 13 transformed by a plasmid capable of replicating in said organism and having a deoxyribonucleotide sequence coding for prepro human serum albumin, and recovering the prepro human serum albumin so produced. 17. A method as claimed in claim 16 wherein the prokaryotic organism is E. coli. 18. A method as claimed in claim 17 wherein the prokaryotic organism is transformed by a plasmid substantially similar to plasmid pGX401 as claimed in claim 11. 19. E. coli strain NRRL No. 15784 pGX401 or a mutant thereof containing a human prepro human serum albumin gene. CLAIMS FOR THE DESIGNATED STATE AT 1. A process for preparing a gene coding for human serum albumin HSA which comprises obtaining HSA mRNA from HSA producing cells, in vitro synthesis of complementary DNA cDNA using said mRNA as a template and conversion of said cDNA to the double stranded form. 2. A process as claimed in claim 1 wherein said gene codes for prepro human serum albumin. 3. A process as claimed in claim 1 wherein said gene comprises the following deoxyribonucleotide sequence which corresponds to the indicated amino acid sequence Asp Ala His Lys Ser Glu Val AlaGAY GCX CAY AAM QRS GAM G T X G C X His Arg Phe Lys Asp Leu Gly GluCAY LGN TTY AAM GAY YTZ G G X G A M Glu Asn Phe Lys Ala Leu Val Leu G AM AAY TT AAM GCX YTZ GTX YTZ Ile Ala Phe Ala Gln Tyr Leu GlnATH GCX TTY GCX CAM TAY Y T Z CAM Gln Cys Pro Phe Glu Asp His Val CAM TGY CCX TTY GAM GAY CAY GTX Lys Leu Val Asn Glu Val Thr GluAAM YTZ GTX AAY GAM GTX ACT G A M Phe Ala Lys Thr Cys Val Ala AspTTY GCX AAM ACX TGY GTX G C X G A Y Glu Ser Ala Glu Asn Cys Asp LysG AM QRS G C X GAM AAY T G Y GAY AAM Ser Leu His Thr Leu Phe Gly Asp QRS YTZ CAY ACX YTZ TT GAG G X G A Y Lys Leu Cys Thr Val Ala Thr LeuAAM YTZ TGY ACX GTX GCX ACX YTZ Arg Glu Thr Tyr Gly Glu Met Ala LGN GAM ACX TA GGX GAM ATG GCX Asp Cys Cys Ala Lys Gln Glu Pro GAY TGY TGY GCX AAM CAM GAM CCX Glu Arg Asn Glu Cys Phe Leu GlnGAM LGN AAY GAM TGY TTY Y T Z C A M His Lys Asp Asp Asn Pro Asn LeuCAY AAM GAY GAY AAY CCX AAY YTZ Pro Arg Leu Val Arg Pro Glu ValCCX LGN YTZ GTX LGN CCX G A M GTX Asp Val Met Cys Thr Ala Phe HisGAY GTX ATG TG ACX GCX TTY CAY Asp Asn Glu Glu Thr Phe Leu LysGAY AAY GAM GAM ACX TTY Y T Z A A M Lys Tyr Leu Tyr Glu Ile Ala ArgAAM TAY YTZ TAY GAM ATH RTI ID 46.3 GCX LGN Arg His Pro Tyr Phe Thr Ala ProLGN CAY CCX TAY TTY ACX G C X C C X Glu Leu Leu Phe Phe Ala Lys ArgGAM YTZ YTZ TTY TTY GCX A A M L G N Tyr Lys Ala Ala Phe Thr Glu CysTAY AAM GCX GCX TTY ACX GAM TGY Cys Ala Gln Ala Asp Lys Ala AlaTGY GCX CAM GCX GAY AAM G C X G C X Cys Leu Phe Pro Lys Leu Asp Glu T G Y Y T Z TTY CCX AAM Y T Z GAY GAM Leu Arg Asp Glu Gly Lys Ala SerYTZ LGN GAY GAM GGX AAM GCX QRS Ser Ala Lys Gln Arg Leu Lys CysORS GCX AAM CAM LGN YTZ AAM TGY Ala Ser Leu Gln Lys Phe Gly GluGCX QRS TZ CAM AAM TTY G G X G A M Arg Ala Phe Lys Ala Trp ALa ValLGN GCX TTY AAM GCX TGG G C X G T X Ala Arg Leu Ser Gln Arg Phe ProGCX LGN YTZ QRS CAM LGN T T Y C C X Lys Ala Glu Phe Ala Glu Val Ser AAM GCX GAM TT GCX GAM G T X QRS Lys Phe Val Thr Asp Leu Thr Lys A A M T T Y G T X A C X G A Y Y T Z A C X A A M Val His Thr Glu Cys Cys His GlyGTX CAY ACX GAM TGY TGY CAY G G X Asp Leu Leu Glu Cys Ala Asp AspGAY YTZ YTZ GAM TGY GCX GAY GAY Arg Ala Asp Leu Ala Lys Tyr IleLGN GCX GAY YTZ GCX AAM T A Y A T H Cys Glu Asn Gin Asp Ser Ile SerTGY GAM AAY CAM GAY QRS A T H Q R S Ser Lys Leu Lys Glu Cys Cys GluORS AAM YTZ AAM GAM TGY T G Y G A M Lys Pro Leu Phe Glu Lys Ser HisAAM CCX YTZ TTY GAM AAM ORS C A Y Cys Ile Ala Glu Val Glu Asn Asp TG ATH GCX GAM GTX GAM A A Y G A Y Glu Met Pro Ala Asp Phe Pro SerG AM ATG CCX GCX GAY TTY CCX QRS Phe Ala Val Asp Phe Val Glu SerTTY GCX GTX GAY TTY GTX GAY ORES Lys Asp Val Cys Lys Asn Tyr Ala AAM GAY GTX TGY AAM AA T A Y GCX Glu Ala Lys Asp Val Phe Leu GlyGAM GCX AAM GAY GTX TT Y Y T Z G G X Met Phe Phe Tyr Glu Tyr Ala ArgATG TTY TTY TAY GAM TAY GCX LGN Arg His Pro Asp Tyr Ser Val ValLGN CAY CCX GAY TA ORS G T X GTX Leu Leu Leu Arg Leu Ala Lys ThrYTZ YTZ YTZ LGN YTZ GCX A A M A C X Tyr Glu Thr Thr Leu Glu Lys CysTA GAM ACX ACX YTZ GAM AAM TGY Cys Ala Ala Ala Asp Pro His GluTGY GCX GCX GCX GA CCX CA GAM Cys Tyr Ala Lys Val Phe Asp GluTGY TAY GCX AAM GTX TTY GAY G A M Phe Lys Pro Pro Val Glu Glu ProTTY AAM CCX CCX GTX GAM GAM CCX Gln Asn Phe Ilel Lys Gln Asn CysCAM AAY TTY ATH AAM CAM A A Y T G Y Glu Leu Phe Glu Gin Leu Gly Giu GAM TZ TTY GAM CAM TZ G G X G A M Tyr Lys Phe Gln Asn Ala Leu Phe TA AAM TT CAM AAY GCX YTZ TTY Val Arg Tyr Thr Lys Lys Val ProGTX LGN TAY ACX AAM AAM G T X C C X Gln Leu Ser Thr Pro Thr Leu ValCAM YTZ ORS ACX CCX ACX Y T Z G T X Glu Val Ser Arg Asn Leu Gly LysGAM GTX QRS LGN AA YTZ G G X A A M Val Gly Ser Lys Cys Cys Lys HisGTX GGX ORS AAM TG TG AAM CA Pro Glu Ala Lys Arg Met Pro CysCCX GAM GCX AAM LGN ATG C C X T G Y Ala Glu Asp Tyr Leu Ser Val ValGCX GAM GAY TAY YTZ QRS G T X G T X Leu Asn Gln Leu Cys Val Leu His YTZ AAY CAM YTZ TG GTX Y T Z C A Y Glu Lys Thr Pro Val Ser Asp ArgGAM AAM ACX CCX GTX QRS G A Y L G N Val Thr Lys Cys Cys Thr Glu Ser GTX ACX AAM TG TGY ACX G A M Q R S Leu Val Asn Arg Arg Pro Gly PheYTZ GTX AAY LGN LGN CCX G G X T T Y Ser Ala Leu Glu Val Asp Glu ThrQRS GCX YTZ GAM GTX GAY G A M ACX Tyr Val Pro Lys Glu Phe Asn Ala TA GTX CCX AAM GAM TTY A A Y GCX Glu Thr Phe Thr Phe His Ala Asp GAM ACX TTY ACX TT CAY G C X G A Y Ile Cys Thr Leu Ser Glu Lys GluATH TGY ACX YTZ QRS GAM A A M G A M Arg Gln Ile Lys Lys Glu Thr AlaLGN CAM ATH AAM AAM GAM A C X G C X Leu Val Glu Leu Val Lys His LysYTZ GTX GAM YTZ GTX AAM CAY AAM Pro Lys Ala Thr Lys Glu Glu LeuCCX AAM GCX ACX AAM GAM G A M Y T Z Lys Ala Val Met Asp Asp Phe Ala AAM GCX GTX ATG GA GAY T T Y G C X Ala Phe Val Glu Lys Cys Cys LysGCX TTY GTX GAM AAM TGY TGY A A M Ala Asp Asp Lys Glu Thr Cys PheGCX GAY GAY AAM GAM ACX TGY TTY Ala Glu Glu Gly Lys Lys Leu ValGCX GAM GAM GGX AAM AAM YTZ GTX Ala Ala Ser Glu Ala Val Leu GlyGCX GCX QRS GAM GCX GTX Y T Z G G X LeuY T Z T A A wherein, the 5 to 3 strand, beginning with the amino terminus and the amino acids for which each triplet codes are shown, and wherein the abbreviations have the following standard meanings A is deoxyadenyl T is thymidyl G is deoxyguanyl C is deoxycytosyl X is A, T, C or G Y is T or C When Y is C, Z is A, T, C or G When Y is T, Z is A or G H is A, T or C Q is T or A When Q is T, R is C and S is A, T, C or G then Q is A, R is G and S is T or C M is A or G L is A or C When L is A, N is A or G When L is C, N is A, T, C or G GLY is glycine ALA is alanine VAL is valine LEU is leucine ILE is isoleucine SER is serine THR is threonine PHE is phenylalanine TYR is tyrosine TRP is tyryptophan CYS is cysteine MET is methionine ASP is aspartic acid GLU is glutamic acid LYS is lysine ARG is arginine HIS is histidine PRO is proline GLN is glutamine ASN is asparagine 4. A process as claimed in claim 2 therein said gene comprises the following deoxyribonucleotide sequence Met Lys Trp Val Thr Phe ATC AAM TGG GTX ACX TTY Ile Ser Leu Leu Phe Leu PheATH QRS YTZ YTZ TTY YTZ TTY Ser Ser Ala Tyr Ser Arg GlyQRS QRS CCX TAY QRS LCN GX Val Phe Arg Arg Asp Ala His LysGTX TTY L G N LGN GAY G C X CAY AAM Ser Glu Val Ala His Arg Phe LysQRS CAM CTX CCX CAY LCN T T Y A A M Asp Leu Gly Glu Glu Asn Phe LysGAY YTZ GCX CAM CAM AAY TTY AAM Ala Leu Val Leu Ile Ala Phe AlaG C X Y T Z G T X Y T Z A T H G C X T T Y G C X Gln Tyr Leu Gln Gln Cys Pro PheC AM TAY YTZ CAM CAM T G Y CCX RTI ID 51.6 TTY Glu Asp His Val Lys Leu Val AsnCAM CAY CAY CTX AAM YTZ G T X AA Glu Val Thr Glu Phe Ala Lys ThrCAM GTX ACX CAM TTY CCX A A M A C X Cys Val Ala Asp Glu Ser Ala GluTC CTX CCX GAY CAM ORS CCX CAM Asn Cys Asp Lys Ser Leu His ThrAA TC GAY AAM QRS YTZ C A Y ACX Leu Phe Gly Asp Lys Leu Cys ThrYTZ TTY CCX CAY AAM YTZ T G Y ACX Val Ala Thr Leu Arg Glu Thr TyrCTX CCX ACX YTZ LCN CAM ACT T A Y Gly Glu Met Ala Asp Cys Cys AlaCCX CAM ATG CCX CAY TCY T G Y G C X Lys Gln Glu Pro Glu Arg Asn GluAAM CAM CAM CCX CAM LCN AAY CAM Cys Phe Leu Gln His Lys Asp AspTCY TTY YTZ CAM CAM AAM GAY CA Asn Pro Asn Leu Pro Arg Leu ValAAY CCX AAY YTZ CCX LCN TZ CTX Arg Pro Glu Val Asp Val Met CysLGN CCX CAM GTX GAY GTX A T G T G Y Thr Ala Phe His Asp Asn Glu GluACX CCX TTY CAY CAY AAT CAM CAM Thr Phe Leu Lys Lys Tyr Leu TyrACX TYy YTZ AAM AAM TAY Y TZ T A Y Glu lie Ala Arg Arg His Pro TyrCAM ATH CCX LCN LGN CAY C C X T A Y Phe Thr Ala Pro Giu Leu Leu PheTTY ACX GCX CCX CAM YTZ Y T Z T T Y Phe Ala Lys Arg Tyr Lys Ala AlaTTY CCX AAM LCN TAY AAM G C X G C X Phe Thr Glu Cys Cys Ala Gln AlaTTY ACX CM GZM TGY CCX CAM GCX Asp Lys Ala Ala Cys Leu Phe ProGAY AAM CCX CCX TGY YTZ TTY CCX Lys Leu Asp Glu Leu Arg Asp Glu AAM TZ GAY GAM TZ LGN G A Y CAM Gly Lys Ala Ser Ser ALa Lys GlnG G X AAM G C X QRS Q R S G C X AAM CAM Arg Leu Lys Cys Ala Ser Leu GlnLGN YTZ AAM TGY GCX QRS Y T Z C A M Lys Phe Gly Glu Arg Ala Phe LysAAM TTY CCX CAM LGN CCX T T Y A A M Ala Trp Ala Val Ala Arg Leu SerCCX TGG CCX CTX CCX LGN YTZ ORS Gln Arg Phe Pro Lys Ala Glu PheCAM LCN TTY CCX AAM CCX CAM TTY Ala Glu Val Ser Lys Phe Val ThrC C X C A M C T X Q R S A A M T T Y C T X A C X Asp Leu Thr Lys Val His Thr GLuCAY YTZ ACX AAM GTX CAY A C X G A M Cys Cys His Gly Asp Leu Leu GluTGY TGY CAY CCX CAY YTZ YTZ CAM Cys Ala Asp Asp Arg Ala Asp LeuTGY CCX GAY GAY LGN CCX G A Y Y T Z Ala Lys Tyr Ile Cys Glu Asn GlnC C X A. A M T A Y A T H T C C A M A A Y C A M Asp Ser Ile Ser Ser Lys Leu LysCA QRS ATH QRS QRS AAM Y T Z A A M Glu Cys Cys Glu Lys Pro Leu PheCAM TC TC CAM AAM CCX YTZ TTY Glu Lys Ser His Cys Ile Ala GluCAM AAM QRS CAY TGY ATH CCX CAM Val Glu Asn Asp Glu Met Pro Ala CTX CAM AAY CAY CAM ATC C C X GCX Asp Phe Pro Ser Phe Ala Val AspGAY TTY CCX ORS TTY CCX CTX G A Y Phe Val Glu Ser Lys Asp Val CysTTY CTX CAM ORS AAM CA CTX TGY Lys Asn Tyr Ala Glu Ala Lys Asp AAM A A Y T A Y G C X G A M G C X A A M G AY Val Phe Leu Gly Met Phe Phe TyrCTX TTY YTZ CCX ATC TTY TTY T A Y Glu Tyr Ala Arg Arg His Pro Asp G A M T A Y G C LGN LCN CA Y C C X G A Y Tyr Ser Val Val Leu Leu Leu ArgTA QRS CTX GTX TZ YTZ Y T Z L G N Leu Ala Lys Thr Tyr Glu Thr ThrTZ CCX AAM ACX TAY CAM A C X A C X Leu Ciu Lys Cys Cys Ala Ala Ala YTZ CAM AAM TC TC CCX CXG CX Asp Pro His Glu Cys Tyr Ala LysCAY CCX CAY CAM TC TA Y G C X A A M Val Phe Asp Glu Phe Lys Pro ProCTX TTY CA CAM TTY AAM CCX CCX Val Glu Glu Pro Gln Asn Phe IleCTX CAM CAM CCX CAM AAY T T Y A T H Lys Gln Asn Cys Glu Leu Phe GluAAM CAM AA TC CAM TZ T T CAM Gln Leu Gly Ciu Tyr Lys Phe GlnCAM TZ CCX CAM TA AAM TT CAM Asn Ala Leu Phe Val Arg Tyr ThrAA CCX TZ TT CTX LCN T A Y A C X Lys Lys Val Pro Gln Leu Ser ThrAAM AAM GTX CCX CAM YTZ ORS ACX Pro Thr Leu Val Glu Val Ser ArgCCX ACX yTZ CTX CAM CTX ORS LGN Asn teu Gly Lys Val Gly Ser LysAAY YTZ CCX AAM CTX CCX ORES ARM Cys Cys Lys His Pro Glu Ala LysTGY TGY AAM CAY CCX CAM RTI ID 54.5 CCX AAM Arg Met Pro Cys Ala Glu Asp TyrLGN ATG CCX TCY CCX CAM GAY TAY Leu Ser Val Val Leu Asn Gln LeuYTZ ORS CTX GTX YTZ AAY C A M Y T Z Cys Val Leu His Glu Lys Thr ProTC CTX TZ CA CAM AAM ACX CCX Val Ser Asp Arg Val Thr Lys CysGTX QRS CAY LGN GTX ACX A A M T G Y Cys Thr Glu Ser Leu Val Asn ArgTCY ACX CAM QRS YTZ CTX AAY LGN Arg Pro Gly Phe Ser Ala Leu GluLCN CCX CCX TTY QRS CCX YTZ CAM Val Asp Glu Thr Tyr Val Pro LysGTX CAY CAM ACX TAY CTX CCX A A M Glu Phe Asn Ala Glu Thr Phe ThrCAM TTY AAY CCX CAM ACX TTY ACX Phe His Ala Asp Ile Cys Thr LeuT T Y C A Y C C X C A Y A T H T G Y C X Y T Z Ser Glu Lys Glu Arg Gln Ile LysQRS CAM AAM CAM LCN CAM ATH AAMLys Glu Thr Ala Leu Val Glu LeuAAM CAM ACX CCX TZ CTX CAM Y T Z Val Lys His Lys Pro Lys Ala ThrCTX AAM CAY AAM CCX AAM G C X A C X Lys Giu Glu Leu Lys Ala Val MetAAM CAM CAM YTZ AAM CCX G T X A T G Asp Asp Phe Ala Ala Phe Val GluG A Y G A Y T T Y G C X G C X T T Y G T X C A M Lys Cys Cys Lys Ala Asp Asp Lys AAM TGY TCY AAM CCX GA GA AAM Glu Thr Cys Phe Ala Glu Glu Gly G AM ACX TGY TYY CCX GAM GAM GCX Lys Lys Leu Val Ala Ala Ser Glu AAM AAM TZ GTX CCX G C X ORS CAM Ala Val Leu Gly Leu G CX G T X Y T Z CCX Y T Z TAA wherein the 5 and 3 strand, beginning with the amino terminus, and the amino acids for which each triplet codes are shown, and wherein the abbreviations are defined as in claim 3. 5. A process as claimed in claim 3 wherein said gene comprises the following deoxyribonucleotide sequence GAT GCA CAC AAG AGT GAG GTT GCT CAT CGG TTT AAA GAT TTGGGA GAA GAA AAT TTC AAA GCC TTG GTG TTG ATT GCC TTT GCTCAG TAT CTT CAG CAG TGT CCA TTT GAA GAT CAT GTA AAA TTAGTG AAT GAA GTA ACT GAA TTT GCA AAA ACA TGT GTT GCT CATGAG TCA GCT GAA AAT TGT GAC AAA TCA CTT CAT ACC CTT TTTGGA GAC AAA TTA TGC ACA GTT GCA ACT CTT CCT GAA ACC TATGGT GAA ATG GCT GAC TCC TGT GCA AAA CAA GAA CCT GAG AGAAAT GAA TCC TTC TTG CAA CAC AAA GAT GAC AAC CCA AAC CTCCCC CCA TTG GTC AGA CCA GAG GTT GAT GTG ATC TCC ACT GCTTTT CAT GAC AAT GAA GAG ACA TTT TTG AAA AAA TAC TTA TATGAA ATT GCC AGA AGA CAT CCT TAC TTT TAT GCC CCC GAA CTCCTT TTC TTT GCT AAA AGC TAT AAA GCT GCT TTT ACA GAA TGTTGC CAA GCT GCT GAT AAA GCT GCC TGC CTC TTG CCA AAG CTCCAT GAA CTT CCC GAT GAA RTI ID 55.8 CCC AAG GCT TCC TCT GCC AAA CAGAGA CTC AAG TGT GCC AGT CTC CAA AAA TTT CGA GAA AGA GCTTTC AAA GCA TGG CCC GTG GCT CCC CTG AGC CAG AGA TTT CCCAAA GCT GAG TTT GCA GAA GTT TCC AAG TTA CTC ACA CAT CTTACC AAA GTC CAC ACG GAA TGC TGC CAT GGA GAT CTG CTT GAATGT GCT GAT GAC AGG CCC GAC CTT GCC AAG TAT ATC TGT GAAAAT CAA GAT TCG ATC TCC AGT AAA CTG AAG GAA TGC TGT GAA AAA CCT CTG TTG GAA AAA TCC CAC TGC ATT GCC GAA GTG GAAAAT GAT GAG ATG CCT GCT GAC TTG CCT TCA TTA GCT GCT GATTTT GTT GAA AGT AAG GAT GTT TGC AAA AAC TAT GCT GAG GCAAAG GAT GTC TTC CTC CCC ATC TTT TTG TAT GAA TAT GCA AGAAGG CAT CCT GAT TAC TCT GTC GTG CTC CTC CTC AGA CTT GCCAAG ACA TAT GAA ACC ACT CTA GAG AAG TCC TGT GCC GCT GCAGAT CCT CAT GAA TCC TAT GCC AAA CTC TTC GAT GAA TTT AAACCT CCT GTC GAA GAG CCT CAG AAT TTA ATC AAA CAA AAT TGTGAG CTT TTT GAG CAG CTT GGA GAG TAC AAA TTC CAG AAT CCC CTA TTA GTT CGT TAC ACC AAG AAA GTA CCC CAA GTG TCA ACTCCA ACT CTT GTA GAG GTC TCA AGA AAC CTA GGA AAA GTG CCC AGC AAA TGT TGT AAA CAT CCT GAA GCA AAA AGA ATG CCC TGTGCA GAA GAC TAT CTA TCC GTG GTC CTC AAC CAG TTA TGT CTC TTG CAT GAG AAA ACG CCA GTA AGT GAC AGA GTC ACC AAA TGCTGC ACA GAA TCC TTG GTC AAC AGG CGX CCA TGC TTT TCA GCTCTC GAA GTC GAT GAA ACA TAC GTT CCC AAA GAG TTT AAT GCTGAA ACA TTC Ar TTC CAT GCA GAT ATA TGC ACA CTT TCT GAGAAG GAG AGA CAA ATC AAG AAA CAA ACT GCA CTT GTT GAG CTC CTC AAA CAC AAG CCC AAG GCA ACA AAA GAG CAA CTC AAA GCTGTT ATG GAT GAT TTC GCA GCT TTT GTA GAG AAG TCC TGC AAGGCT GAC GAT AAG GAG ACC TCC TTT GCC GAG GAG GGT AAA AAACTT GTT GCT GCA ACT CAA GCT GCC TTA CCC TTA TAA wherein the 5 to 3 strand, beginning with the amino terminus is shown, and wherein the abbreviations are defined as in claim 3. o. A process as claimed in claim 4 wherein said gene comprises the following deoxyribonucleotide sequence ATC AAG TGG GTA ACC TTT ATT TCC CTT CTT TTT CTC TTT AGCTCG GCT TAT TCC AGC GGT GTG TTT CCT CCA GAT GCA CAC AAGAGT GAG GTT GCT CAT CCC TTT AAA GAT TTG GGA GAA GAA AATTTC AAR GCC TTG GTG TTG ATT GCC TTT GCT CAG TAT CTT CAGCAG TGT CCA TTT GAA GAT CAT GTA AAA TTA GTG AAT GAA GTAACT GAA TTT GCA AAA ACA TGT GTT GCT GAT GAG TCA GCT GAA AAT TGT GAC AAA TCA CTT CAT ACC CTT TTT GGA GAC AAA TTATGC ACA GTT GCA ACT CTT CGT GAA ACC TAT GGT GAA ATG GCTGAC TGC TGT GCA AAA CAA GAA CCT GAG AGA AAT GAA TCC TTCTTG CAA CAC AAA CAT G.AC AAC CCA AAC CTC CCC CGA TTG GTGAGA CCA GAG GTT GAT GTC ATG TGC ACT GCT TTT CAT GAC AATGAA GAG ACA TTT TTG AAA AAA TAC TTA TAT GAA ATT GCC AGAAGA CAT CCT TAC TTT TAT GCC CCC GAA CTC CTT TTC TTT GCTAAA AGC TAT AAA GCT GCT TTT ACA GAA TGT TGC CAA GCT GCTGAT AAA GCT GCC TGC CTG TTG CCA AAG CTC GAT GAA CTT CCC GAT GAA GGG AAG GCT TCG TCT GCC AAA CAG AGA CTC AAG TGTGCC AGT CTC CAA AAA TTT GGA GAA AGA GCT TTC AAA GCA TGG CCC GTG GCT CGC CTG AGC CAG AGA TTr CCC AAA GCT GAG TTTGCA GAA GTT TCC AAG TTA GTG ACA GAT CTT ACC AAA GTC CACACG GAA TGC TCC CAT GGA GAT CTC CTT GAA TGT GCT GAT GACAGC GCC GAC CTT GCC AAG TAT ATC TCT GAA AAT CAA GAT TCGATC TCC AGT AAA CTG AAG GAA TCC TGT GAA AAA CCT CTC TTGGAA AAA TCC CAC TCC ATT GCC GAA CTC GAA AAT GAT GAG ATGCCT GCT GAC TTG CCT TCA TTA GCT GCT GAT TTT GTT GAA AGTAAG GAT GTT TCC AAA AAC TAT GCT GAG GCA AAG GAT GTC TTCCTC CCC ATG TTT TTG TAT GAA TAT GCA AGA AGG CAT CCT GATTAC TCT GTC GTG CTC CTG CTC AGA CTT GCC AAG ACA TAT GAAACC ACT CTA GAG AAG TCC TGT GCC GCT GCA GAT CCT CAT GAATCC TAT GCC AAA CTC TTC GAT GAA TTT AAA CCT CCT GTG GAAGAG CCT CAG AAT TTA ATC AAA CAA AAT TGT GAG CTT TTT GAGCAG CTT GGA GAG TAC AAA TTC CAG AAT CCC CTA TTA GTT CCTTAC ACC AAG AAA GTA CCC CAA GTG TCA ACT CCA ACT CTT GTAGAG GTC TCA AGA AAC CTA GGA AAA GTG CCC AGC AAA TGT TGTAAA CAT CCT GAA GCA AAA AGA ATC CCC TGT GCA GAA GAC TATCTA TCC GTG GTC CTG AAC CAG TTA TGT GTC TTG CAT GAG AAAACC CCA GTA AGT GAC AGA GTC ACC AAA TCC TGC ACA GAA TCCTTG GTG AAC AGG CGA CCA TCC TTT TCA GCT CTG GAA GTC GATGAA ACA TAC GTT CCC AAA GAG TTT AAT GCT GAA ACA TTC ACCTTC CAT GCA GAT ATA TCC ACA CTT TCT GAG AAG GAG AGA CAAATC AAG AAA CAA ACT GCA CTT GTT GAG CTC GTG AAA CAC AAGCCC AAG GCA ACA AAA GAG CAA CTG AAA GCT GTT ATC GAT GAT TTC GCA GCT TTT GTA GAG AAG TCC TGC AAG GCT GAC GAT AAGGAG ACC TGC TTT GCC GAG GAG GGT AAA AAA CTT GTT GCT GCAAGT CAA GCT GCC TTA CCC TTA TAA wherein the 5 to 3 strand, beginning with the amino terminus is shown, and wherein the abbreviations are defined as in claim 3. 7. A process as claimed in claim 6 wherein said gene is comprised in the following deoxyribonucleotide sequence 5 TCTCTTCTGTCAACCCCACGCCTTTGGCACA ATC AAG TGG GTAACC TTT ATT TCC CTT CTT TTT CTC TTT AGC TCC GCT TAT TCCAGG GGT CTC TTT CCT CGA GAT GCA CAC AAG AGT GAG GTT GCTCAT CCC TTT AAA GAT TTG GGA GAA GAA AAT TTC AAA GCC TTGCTG TTG ATT GCC TTT GCT CAG TAT CTT CAG CAG TGT CCA TTTGAA GAT CAT GTA AAA TTA GTC AAT GAA GTA ACT GAA TTT GCAAAA ACA TGT GTT GCT GAT GAG TCA GCT GAA AAT TGT GAC AAATCA CTT CAT ACC CTT TTT GGA GAC AAA TTA TGC ACA GTT GCAACT CTT CCT GAA ACC TAT GGT GAA ATC GCT GAC TCC TGT GCAAAA CAA GAA CCT GAG AGA AAT GAA TCC TTC TTG CAA CAC AAAGAT GAC AAC CCA AAC CTC CCC CCA TTG CTC AGA CCA GAG GTTGAT GTG ATC TCC ACT GCT TTT CAT GAC AAT GAA GAG ACA TTTTTG AAA AAA TAC TTA TAT GAA ATT GCC AGA AGA CAT CCT TACTTT TAT GCC CCC GAA CTC CTT TTC TTT GCT AAA AGC TAT AAAGCT GCT TTT ACA GAA TGT TCC CAA GCT GCT GAT AAA GCT GCCTCC CTC TTG CCA AAG CTC GAT GAA CTT CCC GAT GAA CCC AAGGCT TCG TCT GCC AAA CAG AGA CTC AAG TGT GCC AGT CTC CAAAAA TTT GGA GAA AGA GCT TTC AAA GCA TGG CCC GTC GCT CCCCTC AGC CAG AGA TTT CCC AAA GCT GAG TTT GCA GAA GTT TCCAAG TTA GTG ACA GAT CTT ACC AAA GTC CAC ACC GAA TCC TCCCAT GGA GAT CTC CTT GAA TGT GCT GAT GAC AGC CCC GAC CTTGCC AAG TAT ATC TGT GAA AAT CAA GAT TCC ATC TCC AGT AAACTG AAG GAA TGC TGT GAA AAA CCT CTC TTG GAA AAA TCC CACTGC ATT GCC GAA GTC GAA AAT GAT GAG ATC CCT GCT GAC TTGCCT TCA TTA GCT GCT GAT TTT GTT GAA AGT AAG GAT GTT TCCAAA AAC TAT GCT GAG GCA AAG GAT GTC TTC CTC CCC ATG TTT TTG TAT GAA TAT GCA AGA AGG CAT CCT GAT TAC TCT GTC GTGCTC CTG CTC AGA CTT GCC AAG ACA TAT GAA ACC ACT CTA GAGAAG TGC TGT GCC GCT GCA GAT CCT CAT GAA TGC TAT GCC AAAGTG TTC GAT GAA TTT AAA CCT CCT GTG GAA GAG CCT CAG AATTTA ATC AAA CAA AAT TGT GAG CTT TTT GAG CAG CTT GGA GAGTAC AAA TTC CAG AAT CCC CTA TTA GTT CCT TAC ACC AAG AAAGTA CCC CAA GTG TCA ACT CCA ACT CTT GTA GAG GTC TCA AGAAAC CTA GGA AAA GTG CCC AGC AAA TGT TGT AAA CAT CCT GAAGCA AAA AGA ATG CCC TGT GCA GAA GAC TAT CTA TCC GTG GTCCTG AAC CAG TTA TGT GTG TTG CAT GAG AAA ACG CCA GTA AGTGAC AGA GTC ACC AAA TGC TCC ACA GAA TCC TTG GTG AAC AGGCGA CCA TGC TTT TCA GCT CTG GAA GTC GAT GAA ACA TAC GTTCCC AAA GAG TTT AAT GCT GAA ACA TTC ACC TTC CAT GCA GATATA TGC ACA CTT TCT GAG AAG GAG AGA CAA ATC AAG AAA CAAACT GCA CTT GTT GAG CTC GTG AAA CAC AAG CCC AAG GCA ACAAAA GAG CAA CTC AAA GCT GTT ATC GAT GAT TTC GCA GCT TTTGTA GAG AAG TGC TGC AAG GCT GAC GAT AAG GAG ACC TGC TTCGCC GAG GAG GGT AAA AAA CTT GTT GCT GCA AGT CAA GCT GCCTTA CCC TTA TAA CATCTACATTTAAAAGCATCTCAGCCTACCATGAGAATA AGAGAAAGAAAATGAAGATCAAAAGCTTATTCATCTGTTTTCTTTTTCGTTGGTGTTTTAATCATTTTGCCTCTTTTCTCTGTGCTTCAATTAATAAAAAATGCAAACAA TCTAA wherein the 5 to 3 strand, beginning with the amino terminus is shown, and wherein the abbreviations are defined as in claim 3. 8. A process for preparing a plasmid encoding human serum albumin which comprises inserting a deoxyribonucleotide sequence coding for human serum albumin into a plasmid having the capability of replication in a prokaryotic or eukaryotic organism. 9. A process as claimed in claim 8 wherein the deoxyribonucleotide sequence coding for human serum albumin is prepared by a process as claimed in any one of claims 1 to 7. 10. A process as claimed in claim 8 or claim 9 wherein the deoxyribonucleotide sequence coding for human serum alubmin is inserted into a plasmid having the capability of replication in a prokaryotic organism of the genus Escherichia. 11. The process of claim 10 wherein the deoxyribonucleotide sequence of claim 8 is inserted at thePst I site of plasmid pBR322 so as to prepare plasmid pGX401. 12. A process for preparing a microorganism containing a gene coding for human serum albumin which comprises transforming a microorganism with a plasmid capable of replicating in said microorganism and including said gene. 13. A process as claimed in claim 12 wherein said plasmid is prepared by a process as claimed in any one of claims 8 to 11. 14. A microorganism transformed by a plasmid containing a deoxyribonucleotide sequence as defined in any one of claims 3 to 7. 15. A microorganism as claimed in claim 14 of the genus Escherichia. 16. A method of producing prepro human serum albumin which comprises cultivating on an aqueous nutrient medium containing assimilable sources of carbon, nitrogen and essential minerals and growth factors, under prepro human serum albumin producing conditions, a prokaryotic organism transformed by a plasmid capable of replicating in said organism and having a deoxyribonucleotide sequence coding for prepro human serum albumin, and recovering the preprohuman serum albumin so produced. 17. A method as claimed in claim 16 wherein the prokaryotic organism is E. coli. 18. A method as claim in claim 17 wherein the prokaryotic organism is transformed by a plasmid substantially similar to plasmid pGX401. 19. E. coli strain NRRL No. 15784 pGX401 , or a mutant thereof containing a human prepro human serum albumin gene.

## Description
CLONED HUMAN SERUM ALBUMIN GENE This invention relates to a method for synthesising a human serum albumin gene. This invention further relates to a plasmid containing a cloned human serum albumin gene and a microorganism transformed with such a plasmid. Human serum albumin sometimes referred to hereinafter as HSA is the major protein component of plasma. The protein is produced in the liver and is primarily responsible for maintaining normal osmolarity in the bloodstream. It also is capable of binding and transporting various small molecules via the blood. HSA is administered in various clinical situations.Shock and burn victims, for instance, usually require doses of HSA to restore blood volume and thus ameliorate some of the symptoms associated with trauma. Persons suffering from hypoproteinemia or erythroblastosis fetalis also are likely to require treatment with serum albumin. To date, HSA is produced primarily as a by product from the fractionation of donated blood. A drawback to this is that the cost and supply of blood can vary widely. The blood also may contain undesirable agents such as hepatitis virus. It therefore would be advantageous to develop an alternative source of HSA. It accordingly is an object of this invention to produce human serum albumin in microorganisms. It is a further object of this invention to so produce HSA economically. It also is an object of this invention to develop a cloning procedure that can be applied to other serum proteins. Brief Description of the Figures Figure 1 shows a partial restriction map of a fulllength HSA cDNA cldne isolated by the procedures described herein. Figure 2 shows the DNA sequence of the 5 3 strand of the non coding and coding regions of the full lengthHSA cDNA, as well as the amino acid sequence specified by the DNA sequence. Figure 3 shows an A260 profile of sucrose gradient fractions of mRNA. Fraction group B was used as the template in the synthesis of HSA cDNA. Figure 4 shows pGX401, a recombinant plasmid containing a full length HSA cDNA insert. Figure 5 shows the DNA sequence in the region of codon 97 for HSA sequences derived from three different human livers. According to one aspect of the present invention, we provide a synthetic hunan serum albunin gene. The term 11synthetic as used herein should be understood to include DNA sequences produced by use of recombriant DNA techniques and or chemical synthesis. In accordance with the present invention, a novel hunan serum albumin HSA gene has been cloned and bacterial expression of the gene is described. The nucleotide sequence of the full length HSA gene and the amino acid sequence of the polypeptide specified by that gene also are reported herein. The procedure more fully described hereinafter which has been used to prepare an HSA producing microorganism can be divided into the following stages 1 obtaining HSA MRNA from a suitable source, e.g. by recovery and isolation of theHSA rreA fran HSA producing cells, 2 in vitro synthesis of camplementary DNA cDPa , using the lL as a template and conversion of the cDNA to the double stranded form and 3 insertion of the double stranded cDNA into a suitable cloning vector and transformation of microbial cells with that cloning vector. The procedures described herein resulted in the preparation of a full length clonedHSA cE. Eukaryotic genes are contained in the chroinosomalDNA of cell nuclei. This chromosomal DNA exists in a compact nucleoprotein complex called chromatin.Eukaryotic chromosomal DNA contains intervening sequences introns within the coding sequences exons , which would not permit correct expression in bacteria. For this reason a preferred method for producing contiguous coding blocks of a particular protein involves the use of messenger RNA mRNA . Messenger RNA has a ribonucleotide sequence corresponding to the gene of interest without the introns and conveniently can be recovered from eukaryotic cells that produce the protein specified by the gene. Human serum albumin mRNA can be recovered in useful quantities from human liver cells. The HSA mRNA produced by the liver cells is complementary to one of the two strands of the HSA gene and may be employed as a template for the synthesis of complementary DNA cDNA as hereinafter described. To effectively utilize the mRNA for the synthesis of cDNA, it advantageously is recovered from the cells in relatively pure form. The guanidine thiocyanate guanidine hydrochloride extraction procedure described by McCandliss et al., Methods in Enzymology 79 51 1981 , advantageously may be used to recover and purify the HSA mRNA. RNA is inherently less stable thanDNA, and is particularly subject to degradation by ribonucleases that are present in the cells. Therefore, mRNA recovery procedures generally employ means for rapidly inactivating any ribonucleases which are present. In general, recovery of total RNA is initiated by disrupting the cells in the presence of a ribonuclease inactivating substance. Disruption of the cells may be accomplished by subjecting the cells to a lysing reagent, freezing thawing, or mechanical disruption preferably a combination thereof. A mixture of guanidine thiocyanate and a reducing agent, such as mercaptoethanol, has been found to function effectively as a ribonuclease inactivator McCandliss, et al., supra . After disruption of the cells, the solid cell debris is removed, e.g. by centrifugation, and the RNA is precipitated from the resulting clarified solution.Precipitation is effected by known techniques, such as adding a water miscible alcohol, e.g. ethanol, to the solution in a precipitating amount. The RNA then is resuspended in a guanidine hydrochloride solution and precipitated with ethanol for two successive cycles. At this point the RNA is undegraded and free of proteins andDNA. The next step is the separation of mRNA from the total precipitated RNA. Human serum albumin mRNA is polyadenylated, therefore, it readily can be separated from non adenylated RNA by affinity chromatography with oligodeoxythymidylate oligo dT cellulose Aviv, H., et al., Proc. Natl. Acad. Sci. USk 69 1408 1972 McCandliss, et al., supra . Total RNA can be applied to a column in an approximately 0.5 M NaCl containing solution. Under these conditions only poly A RNA binds to the oligo dT cellulose and can be removed specifically by washing the column in a salt free solution. To enrich the preparation for HSA mRNA, the polyA RNA can be fractionated according to size by sucrose gradient centrifugation. Activity of the RNA in the various gradient fractions can be verified by in vitro translation in a reticulocyte lysate Pelham, H., et al.Eur. J. Biochem. 67 247 1976 and by electrophoretic analysis of the protein products Laemmli, U., Nature 227 680 1970 . Once a poly A RNA fraction able to synthesize proteins the size of HSA has been isolated, it can be used to provide a template for cDNA synthesis. This procedure involves enzymatically constructing doublestranded DNA, which has a nucleotide base pair sequence identical to the coding sequence of the original chromosomal gene. The cDNA does not contain any noninformational segments introns within the coding region which might be present in the eukaryotic gene, and thus can ultimately be transcribed and translated in prokaryotic systems. Synthesis of HSA cDNA employs the enzymes reverse transcriptase, Klenow fragment of DNA polymerase I and S1 nuclease Kacian, D., et al., Proc. Nat. Acad. Sci. USA 73 2191 1976 McCandliss, R., et al., Methods inEnzymology 79, p. 601 1981 . Reverse transcriptase catalyzes the synthesis of a single strand of DNA from deoxynucleoside triphosphates on the mRNA template. The poly r A tail of the mRNA permits oligo dT of about 12 to 18 nucleotides to be used as a primer for cDNA synthesis. The use of a radioactively labelled deoxynucleoside triphosphate facilitates monitoring of the synthesis reaction. Generally, a a32P containing deoxynucleoside triphosphate advantageously may be used for this purpose.The cDNA synthesis generally is conducted by combining the mRNA, the deoxynucleoside triphosphates, the oligo dT and the reverse transcriptase in a buffered solution. This solution is incubated at an elevated temperature, e.g., about 40 50 C, for a time sufficient to allow formation of the cDNA copy, e.g.about 5 20 minutes. The conditions of the reaction are essentially as described by Kacian, D.L., et al., supra.After incubation, disodium ethylenediaminetetraacetic acid hereinafter EDTA is added to the solution, and the solution is extracted with phenol chloroform 1 1 by vol. . The aqueous phase is advantageously purifed by gel filtration chromatography, and the cDNA mRNA complex in the eluate is precipitated with alcohol. The mRNA can be selectively hydrolyzed in the presence of the cDNA with dilute sodium hydroxide about 0.1 M at an elevated temperature, e.g., about 60 80 C for about 15 30 minutes. Neutralization of the alkaline solution and alcohol precipitation yields a singlestranded cDNA copy. The single stranded cDNA copy has been shown to have a 5 poly dT tail, and to have a 3 terminal hairpin structure, which provides a short segment of duplex DNA Efstratiadis, A., et al., Cell, 7, 279 1976 . This 3 hairpin structure can act as a primer for the synthesis of a complementary DNA strand. Synthesis of this complementary strand is conducted using the Klenow fragment of DNA polymerase I Klenow, H., et al., Eur. J.Biochem., 22, 371 1971 in a reaction mixture containing the deoxynucleoside triphosphates. The duplex cDNA recovered by this procedure has a 3 loop, resulting from the 3 hairpin structure of the single stranded cDNA copy. This 3 loop can be cleaved by digestion with the enzyme, S1 nuclease, using essentially the procedure ofMcCandliss et al., Methods in Enzymology 79 601 1981 .The S1 nuclease digest may be extracted with phenolchloroform, and the resulting cDNA precipitated from the aqueous phase with alcohol. The intact double stranded DNA about 2000 base pairs corresponding to a human serum albumin gene can be isolated by, for example, sucrose gradient centrifugation, using the procedure of McCandliss supra p. 51. In order to determine the sizes of the DNA in the sucrose gradient, aliquots of the gradient fractions are electrophoresed in a polyacrylamide gel with molecular weight markers. The resulting gel is first stained with ethidium bromide to visualize the markers and then autoradiographed to detect the radioactive cDNA. The fractions of the gradient containing DNA molecules larger than 1000 base pairs are pooled and the DNA is precipitated with ethanol. For purposes of amplification and selection, the double stranded cDNA gene prepared as described above is generally inserted into a suitable cloning vector, which is used for transforming appropriate host cells.Suitable cloning vectors include various plasmids and phages, with plasmids being preferred in this case. The criteria for selecting a cloning vector include its size, its capability for replicating in the host cells, the presence of selectable genes, and the presence of a site for insertion of the gene. With respect to its size, the vector is advantageously relatively small, to permit large gene insertions, and so as not to divert large amounts of cellular nutrients and energy to the production of unwanted macromolecules. The vector also includes an intact replicon which remains functional after insertion of the gene. This replicon preferably directs the desired mode of replication of the plasmid, i.e., multiple copies or a single copy per cell, or a controllable number of copies per cell.Genes specifying one or more phenotypic properties, preferably antibiotic resistance, facilitate selection of transformants. The insertion site is advantageously a unique restriction site for a restriction endonuclease. A cloning vector meeting all of these criteria is the plasmid pBR322. The cDNA can be conveniently inserted into this plasmid by a homopolymeric tailing technique. Homopolymer tails are added to the 3 hydroxyl groups of the human serum albumin double stranded cDNA gene, by reaction with an appropriate deoxynucleoside triphosphate, in the presence of terminal deoxynucleotidyl transferase. The plasmid is opened by digestion with the appropriate endonuclease,and complementary homopolymer tails are added to the 3 hydroxyl groups of the opened plasmid, using the homopolymeric tailing technique.Appropriate reaction conditions have been described for the addition of dC residues to ds cDNA McCandliss, R., et al., page 601 supra Roychoudhury, R., et al., Nucleic Acids Research 3 101 1976 and of dG residues to Pstl treated pBR322 Maeda, S., Methods in Enzymology 79 607 1981 . In a preferred embodiment, however, the molar excess of dXTPs to 3 ends is in the range of 3000 to 5000. Progress of the reactions is monitored until the chain length is approximately 15. The tailed cDNA and plasmids are recovered, e.g., by phenol extraction followed by alcohol precipitation. The homopolymeric ends of the two DNAs are complementary and will anneal together under appropriate conditions to yield a recombinant plasmid containing the HSA gene Maeda, S., Methods in Enzymology 79 611 1981 . A suitable strain of E.coli may be transformed with this recombinant plasmid, using essentially the method ofLederberg, J. Bacteriology 119 1072 1974 and be maintained indefinitely. Generally, several hundred to several thousand clones are produced by these procedures and can be screened for the presence of the HSA gene with, for example, rat serum albumin cDNA. A nick translated Maniatis, T., et al., Proc. Natl. Acad. Sci. USA 72 3961 1975 rat cDNA having 85 homology with human cDNA can be used to hybridize to plasmid cDNA attached to nitrocellulose filters Grunstein, M., et al., Proc.Natl. Acad. Sci. USA 72 396 1975 , Southern, E.M. J Mol. Biol., 98 503 1975 . In this procedure, DNA from each colony or from groups of colonies is fixed to discrete zones of a nitrocellulose filter and denatured. Alternatively, the DNA can be electrophoresed in a gel prior to fixing on a filter. A solution of the radioactively labeled rat cDNA is applied thereto under hybridizing conditions. Unhybridized rat cDNA is washed from the filter, and colonies containing DNA to which the rat cDNA hybridized are identified by autoradiography.One positive clone was identified but found to be an incomplete HSA cDNA by DNA sequencing. A portion of thisHSA cDNA was then nick translated in order to rescreen the entire bank of clones. Ninety positive hybridization signals were thus obtained. Positive clones may be cultivated on suitable growth media to obtain ample quantities of cells from which to extract the plasmid DNA. The plasmid DNA is extracted, using conventional techniques, such as disruption of the cells, followed by phenol extraction, and alcohol precipitation. The plasmid and chromosomal DNAs may be separated, e.g. by electrophoresis or cesium chloride equilibrium centrifugation. Plasmid DNA containing inserts of about 1500 to 2000 base pairs are selected for further characterization. The cloned gene can be excised from the plasmid DNA and then characterized by sequencing analysis Sanger,F., et al., Proc. Natl. Acad. Sci USA 74 5463 1977 Maxam, A., et al., Proc. Natl. Acad. Sci. USA 74 560 1977 . By these procedures a prepro HSA clone has been isolated. An E. coli HB101 culture transformed with the plasmid containing this prepro HSA gene has been deposited with the U.S. Department of AgricultureNorthern Regional Research Laboratory in Peoria,Illinois, as NRRL No. B 15784. A diagnostic partial restriction map of this HSA gene insert is shown inFigure 1 of the drawings and Figure 2 shows the 5 3 strand of the non coding and coding regions, along with the amino acid sequence specified by the gene. The cloned prepro HSA coag sequence consists of 2050 base pairs excluding the oligo dC tails added to the cDNA. The gene has noncoding regions at the 5 end base pairs 1 31 and at the 3 end base pairs 1858 2050 . The 5 end of the coding region 32 103 base pairs includes a 24 amino acid leader an 18 amino acid pre sequence followed by a 6 amino acid pro sequence and the mature human serum albumin protein is specified by the region from base pair number 104 to base pair number 1858. As used in Figure 2 and elsewhere herein, the abbreviations have the following standard meaning A deoxyadenyl T thymidyl G deoxyguanyl C deoxycytosyl GLY glycine ALA alanine VAL valine LEU leucine ILE isoleucine SER serine THR threonine PHE phenylalanine TYR tyrosine TRP tryptophan CYS cysteine MET methionine ASP aspartic acid GLU glutamic acid LYS lysine ARG arginine HIS histidine PRO proline GLN glutamine ASN asparagineIt will be appreciated that because of the degeneracy of the genetic code, the nucleotide sequence of the gene can vary substantially.For example, portions or all of the gene could be chemically synthesized to yield DNA having a different nucleotide sequence than that shown in Figure 2, yet the amino acid sequence would be preserved, provided that the proper codon amino acid assignments were observed. Having established the nucleotide sequence of the human serum albumin gene and the amino acid sequence of the protein, the gene of the present invention is not limited to a particular nucleotide sequence, but includes all variations thereof as permitted by the genetic code. It is believed that the amino acid sequence set forth in Figure 2 and claimed herein represents a genomicHSA allele that is widespread in the human population, in contrast to the sequences previously published in the scientific literature. Polymorphism is known for HSA.Protein electrophoresis has revealed over twenty genetic variants of HSA Weitkamp et al., Ann. Hum. Genet.London 36 381 1973 . Two differing amino acid sequences have been reported previously. See Lawn, R.M., et al., Nucl. Acids Res. 9 6103 1981 and Dugiaczyk,A., et al., PNAS 79 71 1982 . The DNA sequence ofFigure 2 differs from each of these published sequences.Although some of the differences occur in third base position of codons or in the noncoding regions, and as such do not cause amino acid changes, conflicting nucleotide sequence data suggest different amino acids at positions 97 and 396. In Figure 2, the amino acid represented by codon 97 GAG is glutamic acid. The same was reported by Lawn, et al., supra. Dugiaczyk, however, reported that codon to be GGG glycine . Codon 396 in Figure 2, also is designated GAG gluatmic acid .Dugiaczyk reported the same however, Lawn reported codon 396 to be AAG lysine . Thus, each of the three DNA sequences would encode a different polypeptide. ExampleIV below sets forth the procedures followed to determine that these differences represented true protein polymorphism and not merely experimental artificats. The present invention has been described in connection with the use of E. coli as the bacterial host for recombinant DNA containing the HSA gene, but skilled molecular biologists will appreciate that other gramnegative bacteria, such as Pseudomonas gram positive bacteria, such as Bacillus higher unicellular organisms, such as yeasts and fungi, and mammalian cells can be employed for cloning and or expression of the HSA gene. The invention is further illustrated by reference to the following examples, which are not intended to be limiting. EXAMPLE I Isolation of HSA mRNA from Human Liver Tissue Messenger RNA mRNA was isolated from human liver tissue taken from a 10 year old accident victim.Extreme care was taken throughout the procedures to avoid ribonuclease contamination of the mRNA preparation.These measures included the use of new, sterile laboratory glassware, treatment of solutions with diethylpyrocarbonate when appropriate, followed by autoclaving, keeping the preparation cold when possible and using gloves to avoid contact of the preparation with skin. Frozen human liver tissue 10.5 grams was homogenized in 210 mls lysis solution 4M guanidine thiocyanate 0.lM Tris HCl, pH 7.5 0.1M 2 mercaptoethanol using a Virtis homogenizer. Cellular debris was pelleted by centrifugation at 8750 rpm, 4 C, for 10 minutes in aSorvall GSA rotor, and the supernatant was transferred to a new centrifuge bottle. To the supernatant were added 0.04 volume 1M acetic acid and 0.5 volume 95 ethanol.After 2 hours at 200C, the mixture was centrifuged at 7500 rpm, 10 minutes, 40C and the pellet resuspended in 50 mls wash solution 6M guanidine hydrochloride l0mM Na2EDTA, pH 7.0 10mM dithiothreitol. Centrifugation at 5500 rpm, 10 minutes, pelleted particulate debris, and the supernatant was transferred to a new centrifuge bottle. To the supernatant were added 0.04 volume 1M acetic acid and 0.5 volume 95 ethanol. After 2 hours at 20iC, the mixture was centrifuged at 7200 rpm 20 minutes. The pellet was resuspended in 20 mls wash solution, and 0.04 volume 1M acetic acid and 0.5 volume 95 ethanol were added. The mixture was kept at 20iC for 12 hours, then centrifuged at 8,000 rpm for 10 minutes at 40C in a Sorvall SS 34 rotor.The pellet was resuspended in 15 mls sterile distilled H2O dHzO and extracted with an equal volume of 4 1 chloroform butanol. The aqueous phase was transferred to a fresh tube and 0.1 volume 2.4 M sodium acetate and 2.5 volumes 95 ethanol were added. After 2.5 hours at 20 c, theRNA was pelleted by centrifugation and the pellet was resuspended in 2 mls sterile dH2O . A total of 19.2 msRNA was recovered. mRNA was then separated from the total RNA using generally, the oligo dT cellulose affinity chromatography procedure described in Aviv et al.. supra and McCandliss, et al., supra. A column of 5 qrams oligo dT cellulose was washed with one column volume 0.1M NaOH to denature any ribonuclease present, then equilibrated with high salt buffer l0mM Tris HCl, pH 7.4 0.5M NaCl 0.5 sodium dodecyl sulfate . The totalRNA preparation, dissolved in two mls dH2O above, was heated at 70 C for 1 minute, then cooled on ice to room temperature.Next, 0.1 volume 5M NaCl, 0.04 ml 0.5MTris HCl, pH 7.5, and 0.1 ml 10 sodium dodecyl sulfate SDS were added to the RNA. 8 mls high salt buffer were then added to the RNA and the solution was applied to the column with a flow rate of about 10 drops minute. After the sample had passed through, unbound RNA was washed from the column with high salt buffer. Fractions 1 2 ml each were collected and the optical density at 260 nm A260 of each fraction was measured in a spectrophotometer. The column was washed until the A260 readings of fractions dropped below 0.05. Undesired RNA was further washed from the column with low salt buffer 1OmM Tris HCl, pH 7.4 0.2M NaCl 0.1 SDS and fractions were collected as above until the 60 had dropped to 0.05. Next, the mRNA was eluted from the column with elution buffer lOmM Tris HC1, pa 7.4 1mM EDTA 0.1 SDS and 1 ml fractions were collected until the A260 was less than 0.05. The first 15 fractions those having the highest OD260 readings were pooled and the mRNA was precipitated by adding 0.1 volume 2.4M sodium acetate and 2.5 volumes 95 ethanol, and placing at 20eC for 12 hours. The eluted mRNA was then pelleted by centrifugation and resuspended in 800 ul elution buffer. After heating the resuspended pellet at 700C for 90 seconds then cooling on ice 0.1 volume 5M NaCl and 0.05 volume 10 SDS were added. The eluted mRNA prepared above was then further purified by passage over a second oligo dT celluose column. A column containing 0.1 gram oligo dT cellulose was washed with NaOH, then with hiqh salt buffer as previously described. The RNA was applied to the column and fractions were collected with high salt, low salt, and elution buffers as with the first column. The peak fractions from the elution buffer step were pooled and the twice purified mRNA was precipitated and pelleted as before. The mRNA was then size fractionated on a 12 ml sucrose gradient as described in McCandliss et al.,Methods in Enzymology, 79, pp. 56 58. A 5 20 sucrose gradient was prepared in gradient buffer 0.02M sodium acetate, pH 5.6 and chilled at 40C for 3 hours. 100us of the mRNA was resuspended in 100oil gradient buffer, heated at 800C for 2 minutes, quick cooled in an ice bath, then layered on top of the gradient. A second 520 gradient had E. coli 16 and 23S rRNA 100pg total loaded on it to serve as molecular weight markers. The two gradients were centrifuged in a BeckmanSW40 rotor at 38,000 rpm for 12.5 hr at 4 0C. Fractions of about 0.5 ml were then collected and the A260 measured fraction 1 is that collected from the bottom of the gradient tube. The A260 peak was divided into 6 groups of fractions, groups A through F as shown in Figure 3. The fractions in each group were pooled and the mRNA precipitated with 0.1 volume 2.4 M sodium acetate and 2.5 volumes 95 ethanol. Fraction groups containing mRNA which encodes protein of the size expected for HSA were identified by in vitro translation using a rabbit reticulocyte lysate kit available from Bethesda Research Laboratories and used according to manufacturer s instructions supplemented with 35S methionine. A reaction mixture for each fraction group contained the components necessary for translation of the mRNA into radioactively labeled proteins which were visualized by electrophoresis on a 12.5 polyacrylamide SDS gel, followed by fluorography. The fluorogram showed a prominent protein band of the size expected for HSA 68,000 daltons among the translation products of fraction groups B and C. Group B had a much lower percentage of protein products in undesirable low molecular weight range so the mRNA in group B was chosen for use as a template in the synthesis of cDNA. EXAMPLE II Synthesis of HSA cDNA Generally, the cDNA synthesis procedure ofMcCandliss et al., Methods in Enzymology, 79, pp. 601607 1981 was used. Incorporation of a radioactively labeled deoxynucleotide allowed monitoring of the synthesis and calculation of yields at each step. The first strand of cDNA was synthesized on the mRNA template, using oligo dT as a primer, as follows. Prepared mix and kept on ice 0.5 M Tris HCl, pH 8.3 20it1 1.4 M RC1 10oil 0.25M Mac12 8p1 0.05M dATP, pH 7.0 21 0.05M TTP, pH 7.0 2 p 1 0.05M dCTP, pH 7.0 2iil 0.05M dGTP, pH 7.0 2V1 0.01M dithiothreitol 4p1 sterile distilled H2O 45V1 aqueous label, a32P dCTP 10pCi p1 Sul 1 00us Added remaining components oligo dT 12 t8 250pg ml 20pl actinomycin D 500 pg ml, aqueous 16p1 10pg mRNA, B fraction 201 sterile dH2O 37 l AMV reverse transcriptase 16u ul 7ul Total volume 200p1 Avian myeloblastosis virus AMV reverse transcriptase is kept at 800C and thawed briefly to add as last component The reaction mixture was kept on ice 5 minutes and 2 l were removed and counted in ASC scintillation fluid in order to determine the specific activity of the dCTP.The reaction mixture was then incubated 10 minutes at 460C. 20 l 0.2M EDTA pH 8.0 was added to stop the reaction, and the mixture was then extracted with an equal volume 1 1 phenol chloroform. 0.14 volume 80 glycerol was added and sample was chromatographed on a 0.7 x 17 cm. Sephadex G 100 column.Once the sample had entered the column, G100 buffer 10mMTris HCl, pH 8.0 1mM EDTA 100mM NaCl was added to the column and 5 drop about 275 l fractions were collected.The radioactive fractions were Cerenkov counted and the cDNA fractions comprising the peak counts per minute were pooled. The mRNA cDNA hybrids were precipitated by adding 0.1 volume 2.4M sodium acetate and 2.5 volumes 95 ethanol, placing in a dry ice ethanol bath for 30 minutes, then pelleting by centrifugation at 10,000 rpm, 4 C, for 20 minutes. The pellet was resuspended in 300 l 0.1M NaOH and heated at 700C for 20 minutes to hydrolyze the RNA, leaving single stranded cDNA. 30 l 1M HC1 were added to neutralize the solution. The DNA was precipitated by adding 5g tRNA, 1 10 volume 2.4M sodium acetate, and 2.5 volumes 95 ethanol, placing in a dry ice ethanol bath 10 minutes, and centrifuging in a microfuge 10 minutes at 40C. The pellet was resuspended in the following mix 40 l 0.5M potassium phosphate, pH 7.4 8 l 0.25M MgCl2 2 P 1 0.1M dithiothreitol 1 l 0.05M dATP, pH 7.0 1 l 0.05M dCTP, pH 7.0 1 l 0.05M dGTP, pH 7.0 11 0.05M TTP, pH 7.0 124 l sterile dH2O 178 lNext, added 22 l DNA polymerase I Klenow fragment 5 l, available from Boehringer Mannheim. The reaction mixture was then incubated in a 150C water bath for 12 hours. 20p1 0.2M EDTA pH 8.0 was added to stop the reaction and the mixture was extracted with an equal volume 1 1 phenol chloroform. 0.14 volume glycerol was added to the aqueous phase. The sample, which now contains double stranded cDNA, was run over a Sephadex G100 column and the peak cDNA fractions were pooled and precipitated as before.The double stranded DNA has a 3 hairpin loop as previously described, which was removed with S1 nuclease as follows. The pellet was resuspended in 72 l sterile distilled water and then 18 l 5X S1 buffer 1M NaCl 0.25M sodium acetate, pH 4.5 SmM ZnS04 2.5 glycerol were added. An enzyme mix was prepared by adding 2.5 l 50 units of S1 nuclease 20 g l to 47.5 ul 1X S1 buffer. 10 l of enzyme mix was added to the 90V1 DNA solution then incubated at 37iC 20 minutes.Addition of 20 ul 0.2M sodium EDTA stopped the reaction, and the reaction mixture was extracted with an equal volume 1 1 phenol chloroform. The aqueous phase was loaded onto a 5 258 sucrose gradient and spun at 38,000 rpm 17.5 hours 5iC in an ultracentrifuge. One ml fractions were collected and Cerenkov counted. Fractions were pooled with fractions 1 6, 7 9.and 10 12 comprisinq the 3 pools. Fraction 1 was the fraction taken from the bottom of the qradient. DNA was precipitated bv adding 0.1 volume 2.4M sodium acetate, 12 ug tRNA, and 2.5 volumes 95 ethanol to each pool, then placing them at 20eC overnight. The DNA was pelleted by centrifugation at 25K for 30 minutes at 4C. After slightly dessicating pellets, the DNA from each pool was resuspended in 200 l dH2O and precipitated again with ethanol and sodium acetate. Pellets were resuspended in 22P1 dH2O and spun in a microfuge 5 minutes to pellet insoluble matter. 2P 1 of each cDNA containing supernatant were analyzed by electrophoresis on a 6 polyacrylamide gel.Autoradiography of the gel showed that the DNA in the pool of fractions 1 6 had an average size of 1100 base pairs bp and included DNA in the 200 bp range and this pool was chosen for addition of polyC tails to the 3 ends of the cDNA, using, generally, the homopolymeric tailing procedure described in McCandliss et al., page 601 et seq., supra. A 5000 molar excess of dCTP over 3 cDNA ends was found to give good results. The reaction mixture was as follows 20P1 cDNA about 43 ng 3H dCTP 645 pmol, lyophilized 2.4oil 10X TdT buffet 1.6us dH2O 24.0 l 10X TdT buffer 1.4M potassium cacodylate 0.3M Tris HC1, pU 7.0 10mM CoC12 lmM DTT The reaction mixture was preincubated to 37 C for 2 minutes, 2P1 were removed for use in calculations, then 2P 6.66 units P L Biochemicals terminal deoxynucleo tidyl transferase were added and incubation at 37 C was continued for 5 minutes.Calculations based on incorporation of 3H dCTP indicated that the 3 ends of the cDNA now carried polyC tails an average of 14 nucleotides in length. 80 l T.E. buffer l0mM Tris HCl, pH 7.6 lmM EDTA were added to the DNA and the solution was extracted with an equal volume of 1 1 phenol chloroform. The organic phase was then retracted with 100p1 dH2O and the two aqueous phases were combined. The C tailed double stranded cDNA was then annealed to plasmid pBR322 DNA which had been linearized with the restriction endonuclease PstI, then G tailed by the homopolymeric tailing method. The complementary singlestranded C and G tails will anneal, producing recombinant plasmids with cDNA inserts at the PstI site. 200 l cDNA, C tailed 39.2 ng 10.5p1 pBR322 PstI, G tailed 302 ng 93 l 10X buffer 626.5 iil dH2O 930 l The reaction mix was placed in an insulated water bath at 70 C. The bath was then transferred to a 37 C room and allowed to cool slowly to 370C overnight, then transferred to room temperature, where the bath cooled to 30 C over several hours. The reaction mixture was then stored at 4 C. lOX annealing buffer 1.5M NaCl 100mM Tris HCl, pH7.5 l0mM EDTA E. coli HB101 cells were made competent for transformation by known calcium chloride treatment procedures. 200 l aliquots of competent HB101 cells were each combined with 40 l of the annealing reaction mixture and kept on ice 20 minutes, then heat shocked at 42 C for 2 minutes. 2.8 mls Luria broth were added to each tube and incubated at 370C for 1 hour. The tubes contents were aliquoted 1 2 ml aliquots into tubes containing Luria broth plus 0.7 agar, and then were poured onto Luria broth agar plates containing 25g ml tetracycline and incubated at 37 C until colonies appeared. Only those cells transformed by pBR322 with or without a cDNA insert can grow on tetracycline plates.Approximately 2500 transformant colonies grew on the plates. EXAMPLE III Isolation of a Full Length HSA cDNA The transformants were initially screened with a rat serum albumin RSA cDNA fragment. The RSA cDNA fragment was obtained from a pBR322 plasmid containing a 2000 bp RSA cDNA insert. This recombinant plasmid is similar to, but contains a longer cDNA insert than, the plasmid prAlbI described in Proc. Nat l. Acad. Sci. USA, 76, 4370 1979 . A 1480 bp rat serum albumin RSA fragment was isolated by digesting the plasmid carrying the RSA cDNA with the restriction endonuclease BstII all restriction endonucleases used in these examples were used according to manufacturer s specifications. The fragment was then radioactively labeled with a32P by the nick translation procedure Maniatis et al. PNASUSA, 72 3961 1975 . About 80 10 ml cultures of individual transformants were grown and plasmid DNA was isolated by known plasmid mini prepn procedures. The partially purified plasmidDNAs were subjected to electrophoresis on 0.8 agarose gels. The DNA was transferred from the gels to nitrocellulose filters using the Southern blotting technique Southern, E.M. J. Molec. Biology 98, 503 1975 . The nitrocellulose filters were immersed for 2 hours at 42C in prehybridization solution 50 formamide 5X SSC 0.05M NaPO4, pB 6.5 5X Denhardt s 100g ml salmon sperm DNA . The filters were then transferred into hybridization solution 50 formamide 10 dextran sulfate 5X SSC 20mM NaPO4, pH 6.5 1X Denhardt s 50jig ml salmon sperm DNA. The nicktranslated 1480bp RSA fragment prepared above was heated at 100 C for 5 minutes, then quick cooled on ice, and this probe was added to the hybridization solution at 2 X 105 cpm probe per ml of solution. The filters were incubated in the hybridization solution at 420C for 18 hours, then washed twice in 2XSSC and once in 0.1X SSC at room temperature. Autoradiography of the filters revealed nonspecific hybridization of the probe to all plasmid DNAs.Therefore, several Southern blot filters were washed in 2XSSC at various temperatures from 65 C to 80 C. DNA from one plasmid on a filter washed at 65 C hybridized strongly with the probe. DNA sequencing revealed that the positive clone, called 6C3, was a partial length human serum albumin clone. Plasmid DNA was isolated from a culture of 6C3 and digested with the restriction endonuclease PstI. One of the resulting HSA cDNA fragments, about 475bp in length, was isolated and nick translated for use as a 50X Denhardt s stock 1 polyvinylpyrrolidone l ficoll 1 bovine serum albumin. lXSSC 150mM NaCl 1SmM sodium citrate, pH 6.8 with citric acid probe. The entire bank of approximately 2500 clones was screened with this probe using a modification of the hybridization procedure of Grunstein et al., supra. The transformant colonies were individually picked from the plates into separate wells in 96 well microtiter plates containing Luria broth plus 0.2 glucose plus 25pg ml tetracycline and incubated at 370C ovenight.Using a transfer device with 48 metal prongs, samples of each culture were transferred to two Luria broth agar tetracycline plates, one plate previously overlaid with a nitrocellulose filter, and incubated at 37 C 2 days. The filters were then placed successively on Whatman filter paper soaked in one of the following solutions 0.5M NaOH lMTris, pH7.4 1M Tris, pH7.4 2XSSC 90 ethanol, and 90 ethanol in that order, 7 minutes per solution. The nitrocellulose filters were then baked in vacuo at 80eC for 2 hours. Prehybridization and hybridization procedures were as described above, except that the three washes were at room temperature. 90 positive hybridization signals were detected by autoradiography. Some of the positive clones were further analyzed by restriction analysis e.g. PstI digestion and hybridization of Southern blots as above. A clone bearing a full length HSA cDNA was identified and confirmed by DNA sequencing. The recombinant plasmid containing this HSA cDNA insert was termed pGX401 and is shown in figure 4. A partial restriction map of the HSA cDNA is shown in Figure 1, while Figure 2 shows the DNA sequence 5 3 strand of the cloned gene and the amino acid sequence it specifies. A sample of E. coli HB101 transformed with pGX401 has been deposited at the U.S. Dept. of Agriculture Northern Regional Research Center in Peoria, Illinois, under accessin number NRRL B 15784. EXAMPLE IV DNA Sequence Analysis of HSA cDNA Prepared from Human Liver Samples Taken from Different Individuals In comparing the DNA sequence of the HSA cDNA insert in pGX401 Example III with the cDNA sequences published by Lawn et al., supra, and Dugaiczyk et al., supra, two codon differences were found that predict amino acid differences. The pGX401 sequence and the sequence reported by Lawn et al. indicated that codon 97 of the mature protein was GAG glutamic acid , whileDugaiczyk et al. reported it to be GGG glycine . In the pGX401 sequence and the sequence reported by Dugaiczyk codon 396 also was reported to be GAG glutamic acid , and Lawn et al. reported that codon to be AAG lysine . To gain some insight into whether these differences represented true protein polymorphisms or merely experimental artifacts, the DNA sequence in the regions of codons 97 and 396 was determined for several new independent HSA genes. Messenger RNA mRNA was isolated from normal human liver tissue taken from four different individuals. The procedures of Example I were followed except that sucrose gradient fractionation of oligo dT cellulose purified mRNA was omitted. Double stranded cDNA was synthesized from this mRNA template by the procedures described in Example 11 and poly dC tails were added according toDeng and Wu, NAR 9 4123, 1981. The vector into which the dC tailed cDNA was inserted was plasmid pGX1066. This plasmid comprises the phage AtRl transcription terminator upstream of a bank of ten closely spaced unique restriction sites, which in turn is upstream of the A4S transcription terminator. E. coli strain GX1170 F leu hsdR thi supE gal 1,2 lac xyl ara trpC9830 lacIq transformed with pGX1066 has been deposited with the American Type Culture Collection,Rockville, Maryland, as ATCC No. 39955. Plasmid pGX1066 was linearized with PstI and poly dG tails were added using the homopolymeric tailing method described by Deng and Wu Nucleic Acids Res., 9 4173 1981 . The vector DNA and cDNA were then annealed as described in Example II. E. coli strain DH1 cells F , endAl, hsdR17 Rk , Mk , supE44, thil, X , recAl, gyrA96, relAl were made competent and transformed with the annealing reaction mix. Both E. coli strain DHl and the transformation procedure used are described by D.Hanahan J. Molec. Biol., 166 557 1983 . Transformants were plated on LM plates 1 w v Bacto tryptone, 0.5 w v yeast extract, l0mM Nail, l0mM MgS04B7H20, 1.5 w v Bacto agar with 35pg ml ampicillin added. Transformed E. coli colonies were screened for the presence of HSA sequences by Grunstein Hogness filter hybridization Gergen et al., 1979, Nuc. Acids. Res.7 2115 Wallace et al., 1981, Nuc. Acids Res. 9 879 using kinased oligomers or nick translated HSA cDNA fragments as probes. For identification of clones carrying HSA cDNA containing codon 396, a synthetic oligonucleotide, 5 TTGTACTCTCCAAGCTGC 3 , corresponding to codons 397 402 and the last nucleotide of codon 396 was used. For detection of clones carrying HSA cDNA containing codon 97, either of two synthetic oligonucleotides, 5 TCTCTTCATTGTCATGAAAAGC 3 , corresponding to codons 126 132 and one nucleotide of codon 133 , or 5 TTCTTGTTTTGCACAGC 3 , corresponding to codons 90 last 2 nucleotides 95, or a nick translatedHSA fragment derived from pGX401 , corresponding to codons 1 to 364 was used.Upon identification of clones containing the HSA sequence of interest, restriction fragments were subcloned into an M13 phage. HSA cDNAcarrying phage were identified by screening plaques according to the procedure of Benton and Davis Science, 196 180 1977 . The DNA sequence was determined with these M13 clones by the dideoxy method Biggin et al.,Proc. Nat. Acad. Sci., U.S.A. 80 3963 1983 . By the procedures described above, transformants containing HSA cDNA that included codon 396 were derived from all four human livers. Transformants containing HSA cDNA that included codon 97 were derived from only two of the four livers. The DNA sequence in all cases including 60 to 100 base pairs on each side of the codon in question matched the sequence determined for pGX401. Messenger RNA then was isolated from normal human liver samples taken from two more individuals, and the sequence at codon 97 was determined using a modification of the Sanger sequencing procedure in which reverse transcriptase was used to copy the single stranded RNA template. A synthetic oligonucleotide, 5 TGTCTCTTCATTGTCATGAAAAGC 3 , corresponding to codons 126 133, was used as a primer.The mRNA, purified by oligo dT cellulose chromatography as previously described, was incubated in a reaction volume of 2111 containing 10 mM Tris . HC1 pH 8.3 , 140 mM KC1, 10 mMMgCl2, 20 mM 6 mercaptoethanol, 1.6 mM dNTP, 0.2 mM ddNTP, 250 ng RNA, 5 ng kinased primer and 1.88 units reverse transcriptase Life Sciences, Inc. . After overlaying the solution with 4 iil of mineral oil the reaction was incubated at 42 C for fifteen minutes and was terminated by the addition of 7 lll of 250 mM Na2EDTA. The mineral oil was extracted with ether and removed with a drawn out pasteur pipette. Formamide loading buffer was added to the samples prior to electrophoresis on a urea sequencing gel. The gels were run until the bromphenol blue tracking dye had migrated to the bottom. They then were dried under vacuum and exposed to X ray film with two intensifying screens for periods between twelve hours and several days. The HSA sequence at codon 97 for both liver samples was identical to the sequence at codon 97 in pGX401. See Figure 5. The reliability of the technique to determine nucleotide sequence from mRNA was evaluated using polyA RNA prepared from the liver that was the source of the cDNA originally cloned in pGX401. The results Figure 5 showed that the sequence determined in this manner was identical to the sequence originally determined in pGX401.