# DNA SEQUENCES, RECOMBINANT DNA MOLECULES AND PROCESSES FOR PRODUCING HUMAN INTERFERON ALPHA LIKE POLYPEPTIDES

## Claims
Patentansprüche für folgende Vertragsstaaten BE, CH, DE, FR, GB, IT, LI, LU, NL, SE

## Description
This invention relates to DNA sequences, recombinant DNA molecules and processes for producing interferon and interferon like polypeptides. More particularly, the invention relates to DNA sequences expressed in appropriate host organisms. The recombinant DNA molecules disclosed herein are characterized by DNA sequences that code for polypeptides having an immunological or biological activity of human leukocyte interferon. As will be appreciated from the disclosure to follow, the DNA sequences, recombinant DNA molecules and processes of this invention may be used in the production of polypeptides useful in antiviral and antitumor or anticancer agents and methods. In this application the interferon nomenclature announced in Two classes of interferons IFN are known to exist. Interferons of Class I are small, acid stable glyco proteins that render cells resistant to viral infection A. Isaacs and J. Lindenmann, Virus Interference I. The Interferon , Human interferons HulFN have been classified into three groups α, β and γ. HuIFN α or leukocyte interferon is produced in human leukocyte cells and together with minor amounts of HuIFN β fibroblast interferon in lymphoblastoid cells. HuIFN α has been purified to homogeneity and characterized HuIFN α has also been reported to exist in several different forms, e.g. British patent application 2,037,296A. These forms appear to differ from each other structurally and physiologically. No accepted nomenclature has been adopted for these forms of HuIFN α. Therefore, in this application each form will be referred to by a number after the general HuIFN α designation, HuIFN α may, like many human proteins, also be polymorphic. Therefore, cells of particular individuals may produce HuIFN α species within the more general HuIFN α group or forms within that group which are physiologically similar but structurally slightly different than the group or form of which it is a part. Therefore, while the protein structure of an HuIFN α may be generally well defined, particulalar individuals may produce a HuIFN α that is a slight variation thereof, this allelic variation probably being less severe than the difference between the various forms of HuIFN α. HulFN is usually not detectable in normal or healthy cells Interferon therapy against viruses and tumours or cancers has been conducted at varying dosage regimes and under several modes of administration The Interferon System, pp. 305 321 . For example, interferon has been effectively administered orally, by innoculation intraveneous, intramuscular, intranasal, intradermal and subcutaneous , and in the form of eye drops, ointments and sprays. It is usually administered one to three times daily in dosages of 10⁴ to 10⁷ units. The extent of the therapy depends on the patient and the condition being threated. For example, virus infections are usually treated by daily or twice daily doses over several days to two weeks and tumors and cancers are usually treated by daily or multiple daily doses over several months or years. The most effective therapy for a given patient must of course be determined by the attending physician who will consider such well known factors as the course of the disease, previous therapy, and the patient s response to interferon in selecting a mode of administration and dosage regime. As an antiviral agent, HulFN has been used to treat the following respiratory infections HuIFN has other effects in addition to its antiviral action. For example, it antagonizes the effect of colony stimulating factor, inhibits the growth of hemopoietic colony forming cells and interferes with the normal differentiation of granulocyte and marcrophage precursors Therefore, in addition to its use as a human antiviral agent, HulFN has potential application in antitumor and anticancer therapy Today, HuIFN α is produced either through human cells grown in tissue culture or through human leukocytes collected from blood donors. 2.6 10⁹ IU of crude HuIFN α have been reported from 800 l of cultured Namalva cells P. J. Bridgen Recent advantages in molecular biology have made it possible to introduce the DNA coding for specific nonbacterial eukaryotic proteins into bacterial cells. In general, with DNA other than that prepared via chemical synthesis, the construction of such recombinant DNA molecules comprises the steps of producing the single stranded DNA copy cDNA of a purified messenger RNA mRNA template for the desired protein converting the cDNA to double stranded DNA linking the DNA to an appropriate site in an appropriate cloning vehicle to form a recombinant DNA molecule and transforming an appropriate host with that recombinant DNA molecule. Such transformation may permit the host to produce the desired protein. Several non bacterial proteins and genes have been obtained in None of these recombinant DNA processes, however is directed, as is this invention, toward the synthesis of HuIFN α. This is the problem to which the present invention is addressed. Its solution is not facilitated as were the above described recombinant DNA schemes by the availability of the sequence information required to prepare a synthetic gene e.g., somatostatin or of a cell type or virus rich in a particular DNA sequence e.g., hepatitis viral antigen of mRNA species e.g., rat insulin which allows preparation and identification of bacterial clones containing the desired hybrid DNA, or of a system allowing the selection of Finally, it should be recognized that the selection of a DNA sequence or the construction of a recombinant DNA molecule which hybridizes to a mRNA from polyA RNA, that mRNA producing HulFN activity in oocytes, is not sufficient to demonstrate that the DNA sequence or the hybrid insert of the recombinant DNA molecule corresponds to HulFN. Instead, only the production of a polypeptide that displays an immunological or biological activity of HulFN can actually demonstrate that the selected DNA sequence or constructed recombinant DNA molecule corresponds to HulFN. More importantly, it is only after HulFN activity is shown that the DNA sequence, recombinant DNA molecule or sequences related to them may be employed to select other sequences corresponding to HulFN in accordance with this invention. It will therefore be appreciated from the foregoing that the problem of producing HuIFN α with the use of recombinant DNA technology is much different than any of the above described processes. Here, a particular DNA sequence of unknown structure that coding for the expression of HuIFN α in an appropriate host must be found in and separated from a highly complex mixture of DNA sequences in order for it to be used in the production of HuIFN α. Moreover. this location and separation problem is exacerbated by the predicted exceedingly low concentration of the desired DNA sequence in the complex mixture and the lack of an effective means for rapidly analyzing the many DNA sequences of the mixture to select and separate the desired sequence. The present invention solves the problems referred to by locating and separating DNA sequences that code for the expression of HuIFN α in an appropriate host and thereby providing DNA sequences, recombinant DNA molecules and methods by means of which a host is transformed to produce a polypeptide displaying an immunological or biological activity of human leukocyte interferon. By virtue of this invention, it is possible to obtain polypeptide s displaying an immunological or biological activity of HuIFN α for use in antiviral, antitumor or anticancer agents and methods. This invention allows the production of these polypeptides in amounts and by methods hitherto not available. As will be appreciated from the disclosure to follow, the DNA sequences and recombinant DNA molecules of the invention are capable of directing the production, in an appropriate host, of a polypeptide displaying an immunological or biological activity of HIFN α. Replication of these DNA sequences and recombinant DNA molecules in an appropriate host also permits the production in large quantities of genes coding for these polypeptides. The molecular structure and properties of these polypeptides and genes may be readily determined. The polypeptides and genes are useful, either as production in the host or after appropriate derivatization or modification, in compositions and methods for detecting and improving the production of these products themselves and for use in antiviral and antitumor or anticancer agents and methods. This process is therefore distinguishable from the prior processes, above mentioned, in that this process, contrary to the noted prior processes, involves the preparation and selection of DNA sequences and recombinant DNA molecules which contain appropriate DNA sequences which code for at least one polypeptide displaying an immunological or biological activity of HuIFN α. It will be appreciated from the foregoing that a basic aspect of this invention is the provision of a DNA sequence which is characterized in that it codes for a polypeptide displaying an immunological or biological activity of an α type interferon for use in cloning or expression in bacteria, yeasts or animal cells or in selecting DNA sequences that code for an α type interferon, the DNA sequence being selected from These sequences permit the production of interferon and interferon like polypeptides in hosts selected from bacteria, yeasts or animal cells according to the processes of this invention. Figure 1 is a schematic outline of one embodiment of a process of this invention for preparing a mixture of recombinant DNA molecules. some of which are characterized by inserted DNA sequences that code for polypeptides of this invention. Figure 2 is a schematic outline of the initial clone screening process of this invention. Figure 3 is a schematic outline of one embodiment of a clone screening process using DNA sequences prepared in accordance with the invention. Figure 4 is a restriction map of one of the clones of the invention. The positions of the restriction sites are based on fragment sizing by agarose gel electrophoresis. Figures 8 10 display the position of these restriction sites as determined by nucleotide sequence data. Figure 5 is a schematic outline of the process of determining the orientation of a DNA insert in one recombinant DNA molecule of this invention. Figure 6 displays the partial nucleotide sequence of some cloning vehicles useful in accordance with this invention. Figure 7 displays the results of a Sephadex G 100 fractionation of supernatant prepared from a bacterial culture of this invention. Figures 8 10 display the nucleotide sequence of a DNA insert to a recombinant DNA molecule of this invention. The sequence is numbered from the nucleotide following the polyG 5 tail to the nucleotide before the polyA residues and polyC 3 tails. Nucleotides 57 125 represent a signal sequences and nucleotides 125 626 represent the mature interferon and the stop codon. The amino acid sequence of the signal sequence is depicted above its nucleotide sequence in lower case letters and the amino acid sequence of the mature interferon is depicted above its nucleotide sequence in upper case letters. Various restriction endonuclease recognition sites in this gene are also depicted in Figures 8 10, these sites being determined by analysis of nucleotide sequence data. Figure 11 is a schematic comparison of the restriction maps of two DNA inserts of recombinant DNA molecules of this invention. Figures 12 16 display the nucleotide sequences of two DNA inserts of recombinant DNA molecules of this invention. The sequences are numbered from the nucleotide following the polyG 5 tail to the nucleotide before the polyA residues and polyC 3 tails. The amino acid sequence of the signal sequence for each of these inserts is depicted above its respective nucleotide. sequence in lower case letters and the amino acid sequence of the mature interferon is depicted above its nucleotide sequence in upper case letters. Streuli et al., Science, 209 19 September 1980 , pages 1343 to 1347 discloses the displayed DNA and amino acid sequences for clone 206 II . Figure 17 displays a partial restriction map of Z pBR322 Pst Hc F II 206 and the sequencing strategy employed to determine the nucleotide sequence of the Hif II 206 fragment displayed in Figures 12 16. Figure 18 displays the partial restriction maps of a series of hybrid phages which hybridize to the Hif 2h fragment. Figure 19 disolays a partial restriction map of the hybrid insert of Z paR322Pst Hchrlf 35HBα and the sequencing strategy employed to determine its nucleotide sequence. Figures 20 23 display the nucleotide sequence of the HchrlF 35HBα framant and the amino acid sequence derived from it. Figure 24 displays partial linkage maps for HuIFN α related genes. The arrows show regions which form R loops with induced leukocyte poly A RNA. The hatched box chr 16 indicates the sequence which was inferred from blotting experiments. but was not revealed by R loop mapping. Figure 25 is a schematic outline of the construction of plasmid C8 IFN α1. Figure 26 is a schematic outline of the construction of plasmid LAC AUG α2 . Figure 27 displays the reconstruction of the AUG initiation codon and the CYS initial codon in the construction of LAC AUG α2 . Figure 28 is a schematic outline of the construction of a plasmid C8 IFN α2 and the hybrid molecules I, II, III and IV. Figures 29 32 display the nucleotide sequence and amino acid sequence encoded thereby for IFN α4b and its signal sequence. In order that the invention herein described may be more fully understood the following detailed description is set forth. In the description the following terms are employed Referring now to Figure 1, we have shown therein a schematic outline of one embodiment of a process for preparing a mixture of recombinant DNA molecules, some of which are characterized by inserted DNA sequences that code for polypeptides having an immunological or biological activity of human leukocyte interferon. Human leukocytes were induced for 5 hours at 37 C with Sendai virus and extracted to yield a poly A RNA mixture containing human leukocyte interferon mRNA HuIFN αmRNA . Induction was by the Cantell procedure After storage overnight at 20 C, the fibrous nucleic acid precipitate was removed by filtration through a plastic tea sieve. This material was then stirred with 200 ml TNE 50 mM Tris HCI pH 7.5 , 100 mM NaCI, 5 mM EDTA containing 0.5 SDS. It subsequently dissolved on addition of a further 350 ml of that solution. The non fibrous precipitate was collected by centrifugation in 1 l Sorvall bottles in a Sorvall RC 3 centrifuge for 15 min at 5,000 rpm and dissolved in 350 ml TNE containing 0.5 SDS. The two TNE solutions were combined, extracted 3 times with 1 vol phenol, 3 times with 1 2 vol ether and 3 times with 1 vol ether. RNA recovery from the aqueous phase totalled 775 mg, as measured by absorbance at 260 nm. Isolation of the poly A RNA mixture was achieved by repeated batch adsorption to oligo dT cellulose type 7 P L Biochemicals, Inc. . 2.7 grams oligo dT cellulose were added to 500 ml, RNA was assayed for HulFN αmRNA by injection into An assay for HuIFN αmRNA activity was carried out as described above on a portion of the poly A RNA precipitate. It had a specific activity of 3600 IU interferon µg of RNA injected. Therefore, the sucrose gradient had enriched the poly A RNA about 10 fold in regard to HuIFN αmRNA. In a subsequent, similar preparation about a 40 fold enrichment was obtained. Preparation B was purified similarly and, since it had a similar specific activity as Preparation A, the two were pooled. At this point it should be recognized that even the poly A RNA product obtained from the sucrose gradient contains a very large number of different mRNA s. Except for the mRNA specific for IFN α, the other mRNAs are undesirable contaminants Figure 1 . Unfortunately, these contaminant RNAs behave similarly to HuIFN αmRNA throughout the remainder of the cloning process of this invention. Therefore, their presence in the poly A RNA will result in the ultimate preparation of a large number of unwanted bacterial clones which contain genes that code for polypeptides other than IFN α. This contamination presents complex screening problems in the isolation of the desired IFN α hybrid clones. In the case of IFN α, the screening problem is further exacerbated by the lack of a sufficiently purified sample of HuIFN αmRNA or DNA or portion thereof to act as a screening probe for the identification of the desired clones. Therefore, the screening process for the IFN α clones is very time consuming and difficult. Further, because only a very small percentage of IFN α clones themselves are expected to express IFN α in a biologically active or immunologically active form, the isolation of an active clone is a needle in a haystack screening process. Advantageously, we may use recombinant DNA technology to provide a purified sample of HuIFN αmRNA or cDNA or a portion thereof. This purified mRNA or cDNA can be used to screen rapidly very large numbers of bacterial clones and thereby markedly increase the probability of isolating a clone which expresses IFN α in an active form. The poly A RNA enriched for IFN αmRNA Prepartion A B was used as a template to prepare single stranded complementary DNA cDNA Figure 1 Again, it is to be understood that the single stranded cDNA product prepared above is in reality a complex mixture of a large number of different cDNAs transcribed from the corresponding mRNAs present in the poly A RNA mixture Figure 1 . Only a very few of these cDNAs are IFN α related, The sizes of the various single stranded cDNAs were determined by electrophoresis of a small aliquot on a alkaline 2 agarose gel using 30 mM NaOH, 2 mM EDTA as electrolyte M. W. McDonell et al., Analysis Of Restriction Fragments of T7 DNA And Determination Of Molecular Weights By Electrophoresis In Neutral And Alkaline Gels , The single stranded cDNA may be rendered double stranded by treatment with DNA polymerase I T. Maniatis The DNA was treated with nuclease S₁, prepared by the method of R. C. Wiegand Again, it must be recognized that the double stranded cDNA produced above is a mixture of a large number of cDNAs and fragments thereof, only a very few of which are HuIFN αcDNA or its fragments Figure 1 . A wide variety of host cloning vehicle combinations may be employed in cloning the double stranded cDNA prepared as above described. For example, useful cloning vehicles may consist of segments of chromosomal, non chromosomal and synthetic DNA sequences, such as various known bacterial plasmids, Furthermore, within each specific cloning vehicle, various sites may be selected for insertion of the double stranded DNA. These sites are usually designated by the restriction endonuclease which cuts them. For example, in pBR322 the The vector or cloning vehicle and in particular the site chosen therein for attachment of a selected DNA fragment to form a recombinant DNA molecule is determined by a variety of factors, e.g., number of sites susceptible to a particular restriction enzyme, size of the protein to be expressed, susceptibility of the desired protein to proteolytic degradation by hose cell enzymes, contamination of the protein to be expressed by host cell proteins difficult to remove during purification, expression characteristics, such as the location of start and stop codons relative to the vector sequences, and other factors recognized by those of skill in the art. The choice of a vector and an insertion site for a particular gene is determined by a balance of these factors, not all selections being equally effective for a given case. Although several methods are known in the art for inserting foreign DNA into a cloning vehicle or vector to form a recombinant DNA molecule, the method preferred for a first construction in accordance with this invention is described in Vila Komaroff Of course, other known methods of inserting DNA sequences into cloning vehicles to form recombinant DNA molecules are equally useful in this invention. These include, for example, direct ligation, synthetic linkers, exonuclease and polymerase linked repair reactions followed by ligation, or extension of the DNA strand with DNA polymerase and an appropriate single stranded template followed by ligation. It should, of course, be understood that the nucleotide sequences or cDNA fragment inserted at the selected site of the cloning vehicle may include nucleotides which are not part of the actual structural gene for the desired polypeptide or may include only a fragment of the complete structural gene for the desired protein. It is only required that whatever DNA sequence is inserted, a transormed host will produce a polypeptide having a biological or immunological activity of HuIFN α or that the DNA sequence itself is of use as a hybridization probe to select clones which contain DNA sequences useful in the production of polypeptides having an immunological or biological activity of HuIFN α The cloning vehicle or vector containing the foreign gene is employed to transform a host so as to permit that host to express the protein or portion thereof for which the hybrid DNA codes. The selection of an appropriate host is also controlled by a number of factors recognized by the art. These include, for example, compatibility with the chosen vector, toxicity of proteins encoded by the hydrid plasmid, ease of recovery of the desired protein, expression characteristic, biosafety and costs. A balance of these factors must be struck with the understanding that not all hosts may be equally effective for expression of a particular recombinant DNA molecule. In the present synthesis, the preferred initial cloning vehicle is the bacterial plasmid pBR322 and the preferred initial restriction endonuclease site therein is the Plasmid pBR322 20 µg was digested with 21 units Addition of homopolymeric dGMP tails Figure 1 by terminal deoxynucleotidyl transferase TdT purified according to F. J. Bollum, Deoxynucleotide Polymerizing Enzymes From Calf Thymus Gland , in Double stranded DNA was elongated with dCMP residues by standard procedures A single colony of The annealing of the tailed, The product is, of course, a large mixture of different recombinant DNA molecules and some cloning vehicles without inserted DNA sequences. However, each recombinant DNA molecule contains a cDNA segment at the The transfection of Since plasmid pBR322 includes the gene for tetracycline resistance, After 48 h at 37 C, individual colonies were picked and suspended in 100 µl tryptone medium supplemented as above in the wells of microtiter plates Dynatech . After incubation at 37 C overnight, 100 µl 40 glycerol were mixed into each well. The plates were stored at 20 C and a library of 100,000 individual clones of transformed These 100,000 clones contain a variety of recombinant DNA molecules representing complete or partial copies of the mixture of mRNAs in the poly A RNA preparation from IFN producing leukocytes Figure 2 . The majority of these will contain only a single recombinant DNA molecule. Only a very few of these recombinant DNA molecules are related to IFN. Accordingly, the clones must be screened to separate the IFN related clones from the others. There are several approaches to screen for backterial clones containing human leukocyte interferon cDNA HuIFN αcDNA . These include, for example, RNA selection hybridization Alwine Referring now to Figure 2, recombinant DNA was isolated from a culture of a mixture of 512 clones from the above library of clones two mixtures of 2 clones shown in Figure 2 Step A . The reason for selecting this batch size will be explained below. The recombinant DNA molecules were cleaved, denatured and hybridized to leukocyte poly A RNA containing IFN αmRNA prepared as before Step B . All recombinant DNA molecule poly A RNA hybrids were separated from the non hybridized poly A RNA Step C . The poly A RNA was recovered from the hybrids and purified Step D . The recovered RNA was assayed for IFN αmRNA activity as above Step E . If, and only if, the mixture of recombinant DNA molecules contains a recombinant DNA molecule having an inserted nucleotide sequence capable of hybridizing to the IFNmRNA in the poly A RNA under stringent hybridization conditions, will the mRNA released from that hybrid cause the formation of IFN α in oocytes, because mRNA released from any other recombinant DNA molecule poly A RNA hybrid will not be IFN α related. If a group of 512 clones gave a positive response, the clones were regrouped in 8 lots of 64, and each lot assayed as before. This process was continued until a single clone responding to this assay was identified. There is no assurance that the recombinant DNA molecules and bacterial clone transformed therewith, which are thus identified, contain the complete IFN αcDNA sequence of IFN α or even that the DNA sequence actually codes for IFN α. However, the recombinant DNA molecules will certainly contain extensive nucleotide sequences complementary to the IFN αmRNA coding sequence. Therefore, the recombinant DNA molecule may at least be used as a source of a probe to screen rapidly other recombinant DNA molecules and clones transformed with them to identify further sets of clones which may contain an authentic and complete IFN α nucleotide coding sequence. The conditions for the hybridization Step B are critical. The absolute concentrations and the ratio of recombinant DNA molecule and poly A RNA must be chosen so as to take into consideration reaction rate and stoichiometry. The proper choice is difficult to make, because the proportion of IFN αmRNA in the poly A RNA is not known. In order to assure controlled and adequate kinetics, the hybridization was carried out under conditions where the concentration of DNA sequences from the recombinant DNA molecules was in excess as compared to the estimated IFN αmRNA concentration. In a mixture of 512 possible different recombinant DNA molecules, an IFN α related DNA sequence IFN αR DNA will either not occur giving a negative assay , or it will constitute at least about 1 512 of the recombinant DNA molecule mixture and therefore the concentration of the IFN αR DNA, if any, can thus be adjusted in the hybridization step to ensure adequate hybridization rates. In addition, the amount of the IFN αR DNA in the reaction mixture must be sufficient to bind enough IFN αmRNA from the poly A RNA to allow detection of IFN α after injection into oocytes of the mRNA recovered from the recombinant DNA molecule poly A RNA hybrid. In order to detect IFN α by the assays available, its concentration should be 100 IU ml or higher. Because 0.5 ml aliquots are required for replicate determinations, 50 IU should be generated in the oocytes. The poly A RNA from induced leukocytes, used previously, generates about 500 IU IFNr α upon injection of 1 µg into oocytes. Therefore, at least 0.1 µg poly A RNA has to be injected to generate the needed 50 IU. Model experiments with rabbit globin mRNA and rabbit β globin cDNA clones showed that the overall recovery of ¹²⁵I globin mRNA in the oocyte relative to ¹²⁵I globin mRNA added to the hybridization mix was about 10 , and the recovery of mRNA activity about 5 . Therefore, at least 0.1 0.05 2 µg of leukocyte poly A RNA should be used for the hybridization assay. To ensure an adequate safety margin, 12 µg of poly A RNA were used per assay. To calculate how much DNA from the recombinant DNA molecules is required to bind the IFN αmRNA in 12 µg of poly A RNA, the IFN αmRNA content of poly A RNA was estimated. One µg of poly A RNA generates 500 IU of IF. The specific activity of IFN α lies between 2 10⁸ and 10⁹ IU mg protein. 500 IU of IFN α therefore correspond to between 500 2 10⁸ 2.5 10 ⁶ mg 2.5 ng and 500 10⁹ 5 10 ⁷mg 0.5 ng of interferon. The relationship between the amount of IFN αmRNA injected into an oocyte and the amount of IFN α produced is unknown. In the case of β globin mRNA, about 30 molecules of protein per molecule mRNA are produced per hour this valve is about 6 for β globin J.B. Gurdon Should the translation ratio of the IFN αmRNA in the oocytes be lower by an order of magnitude than the average for globin mRNA, the IFN αmRNA content of the poly A RNA would be 10 times higher than calculated above, or between about 5 1 1000 to 1 5000. And, 12 µg of poly A RNA would then contain about 12 ng to 2 ng of IFN αmRNA. On the other hand, should the translation ratio of the IFN αmRNA in the oocytes be higher by an order of magnitude than the average for globin mRNA, the IFN αmRNA content of the poly A RNA would be 10 times lower than calculated above, or between about 1 100,000 and 1 500,000. And, 12 µg of poly A RNA would then contain 0.1 ng to 0.02 ng IFN αmRNA. Plasmid pBR322 has 4361 b.p. The complete cDNA of IFN αmRNA would add about 800 1000 b.p. to pBR322 on formation of pBR322 IFN αcDNA to a total of about 5200 5400 b.p. Its molecular weight would thus be about 12 times 2 5200 800 that of the IFN αmRNA alone. Therefore, in order to bind the IFN αmRNA calculated above to be present in 12 µg poly A RNA required for the assay, an amount of recombinant DNA molecules equal to 12 times the amount of the IFN αmRNA will be required stoichiometric amount . Because the IFN αmRNA content of the poly A RNA used to prepare the recombinant DNA molecules had been increased 10 to 40 fold over that of the crude poly A RNA, the group of 512 clones should have 10 to 40 times more clones containing the desired IFN αmRNA than calculated from the above. If IFN αmRNA is 1 part of 1000 of the crude poly A RNA, then 12 µg of poly A RNA contain 12 ng IFN αmRNA and the stoichiometric amount of IFN αcDNA plasmid is 144 ng. Since a group of 512 clones will contain at least 5 with IFN αcDNA inserts, the amount of total hybrid plasmid DNA required is 14.8 µg 144 512 5 10 ³ . If IFN αmRNA is 1 part in 10,000, then 12 µg of poly A RNA contain 1.2 ng IFN αmRNA and the amount of IFN αcDNA plasmid required is 14.4 ng. A group of 512 clones will contain either O or 1 IFN αcDNA insert, so that the amount of total hydrid plasmid DNA required is 7.4 µg 14.4 512 10 ³ . If IFN αmRNA is 1 part in 100,000, then the amount of total hybrid plasmid DNA required is 0.74 µg 1.44 512 10 ³ . In order to ensure that the hybridization reaction will proceed under DNA excess conditions i.e., excess recombinant DNA as compared to poly A RNA , 20 µg of the mixture about 1.4 to 30 fold excess was chosen for the assay. Hybridization must be conducted under conditions which ensure a that the hybridized portion of the poly A RNA is recovered intact and in a biologically active form, b that non specific DNA mRNA association is prevented, and c that the hybridization reaction goes to at least 75 completion. These conditions are most likely to be met by hybridization in 80 formamide, 0.4 M NaCI J. Casey and N. Davidson, Rates Of Formation And Thermal Stability Of RNA DNA And DNA DNA Duplexes At High Concentrations Of Formamide , In the present hybridization, self hybridization of DNA is not a major problem because the mixture of DNA s being used consists of the same vector pBR322 and a variety of cDNA inserts. Therefore, most of the DNA sequences will be heteroduplexes in which the inserts are available for hybridization to poly A RNA. It is very unlikely that complementary cDNA inserts which form part of different duplexes will interact because of topological constraints. In any event, DNA DNA reassociation is minimized under the reaction conditions used J. Casey and N. Davidson, To determine the hybridization time required to ensure at least 75 reaction, a second order rate equation was employed where The desired number of bacterial clones was inoculated onto tryptone medium agar plates supplemented as above, by transferring to it an aliquot from each microtiter well with use of a mechanical device. After incubation at 37 C, each clone had given rise to a colony of several mm diameter. All colonies were washed off the plate s and pooled to give an inoculum used to inoculate 1 l of tryptone medium supplemented as above in a 2 l Erlenmyer flask. The culture was shaken at 37 C to an apparent OD₆₅₀ of about 0.8 estimated visually . One volume of supplemented tryptone medium and chloramphenicol to 170 µg ml were added to the culture which was further shaken at 37 C for 16 h. 20 ml chloroform were added and the culture shaken again for 10 min at 37 C to kill the bacteria C. Weissmann and W. Boll, Pancreatic RNAase A 5 mg ml, preheated 10 min at 85 C was added to the Form I DNA to a concentration of 20µg ml and the mixture incubated 60 min at 37 C. 1 5 vol 5 M NaCI were added and the mixture adjusted with 30 polyethylene glycol 6000 Union Carbide, autoclaved 20 min at 120 C up to a final concentration of 7.5 PEG. After 2 16 h at 10 C, the precipitate was collected in a Sorval SW Rotor for 20 min at 8,000 rpm and 0 C, dissolved in 0.075 M NaCl, 0.0075 M Na citrate to an absorbance of 20 at 260 nm, and adjusted to 0.5 SDS. The solution was incubated for 30 min at 37 C with 0.5 mg ml Pronase self digested at 20 mg ml, 2 h at 37 C and extracted 3 times with 1 vol distilled phenol and 2 times with 1 vol chloroform. The sample up to 2 ml of a 1 mg ml DNA solution was centrifuged through a 5 to 23 sucrose gradient in 50 mM Tris HCI pH 7.5 ,1 mM EDTA for 15 h at 21,000 rpm and 15 C using an SW 27 Beckman Rotor. Fractions were collected and the OD₂₆₀ monitored. DNA containing fractions were pooled and the DNA precipitated with sodium acetate and ethanol. 20 to 100µg of the Form I DNA mixture were recovered by centrifugation. Twenty µg of purified Form I DNA were digested in 150 µl 10 mM Tris HCI pH 7.5 , 6 mM MgCl₂, 50 mM NaCI, 6 mM 2 mercaptoethanol, 200 µg ml BSA or gelatin and 20 units Two hybridization mixtures were prepared. Mixture I contained 4 µl of 10 fold concentrated hybridization buffer 4 M NaCI, 0.1 PIPES pH 6.4, 1,4 piperazinediethane sulfonic acid, Sigma , 50 mM EDTA, 0.5 µl about 5 ng ¹²⁵I globin mRNA 5000 cpm and 6 µl induced leukocyte poly A RNA 2 µg µl , an amount sufficient to generate 6000 IU of IFN when injected into oocytes. Mixture II contained 10 µg of the After dilution to 1 ml with cold 0.9 M NaCI, 0.09 M Na citrate and formamide 100 to 4 by volume the solution was filtered at 0.5 ml min through a Millipore filter 0.45 µm pore size , the filter having been first tested for its capacity to retain RNA DNA hybrids, because not all filters obtained from the manufacturer were equally efficient. The above filter, with poly A RNA hybrids attached, was immersed in 1 ml 0.15 M NaCI, 0.015 M Na citrate, 0.5 SDS for 10 min at 37 C, rinsed with 50 mM Tris HCI pH 7.5 , 10 mM MgCl₂, 2 mM CaCl₂ and placed in 0.6 ml of fresh buffer. After the addition of 5 µl iodoacetate treated DNAase 5 mg ml S. B. Zimmermann and G. Sandeen, The filter was removed and the solution extracted with 1 vol phenol and 1 vol ether and passed through a 0.1 ml Chelex column. 5 µg of carrier RNA purified yeast RNA were added to the solution and the RNA precipitated with sodium acetate and ethanol. The precipitate was collected by centrifugation at 10,000 xg, dissolved in 100 µl 1 mM EDTA, heated for 90 sec at 100 C, and TNE and SDS added to 2 TNE and 0.5 SDS. The RNA was adsorbed to a 100 µl oligo dT cellulose column, eluted with four washes of 0.3 ml distilled water and precipitated with sodium acetate and ethanol. After 16 h at 20 C the precipitated RNA was separated by centrifugation and dissolved in 2 µl TNK buffer. The poly A RNA solution from above was injected into 40 oocytes about 50 nl per oocyte . The oocytes were incubated at 23 C for 24 48 hours, homogenized and centrifuged or the incubation medium recovered and assayed as described previously for IFN α. Most subsequent assays of a recombinant DNA molecule from a single clone were carried out with DBM or DPT paper bound DNA, because the assay conditions were no longer critical and the assay is more convenient. DPT paper gave lower backgrounds and was used preferentially. DBM paper was prepared as described J. C. Alwine DNA up to 15 µg was bound to 50 mm² diazotized ABM DBM or diazotized APT DPT paper as described by J. H. J. Hoeijmakers Hybrid plasmid DNA was digested with endonuclease The assays from 8 groups of 512 clones Group λ was subdivided into 8 subgroups of 64 clones and hybridized and assayed as before. The subgroups gave the following results, presented in the same format as above Subgroup λ III was subdivided into 8 sets of 8 clones, and hybridized and assayed Because the first positive result was achieved with the set λ III 4, the individual colonies of this set designated A to H were hybridized and assayed The recombinant DNA molecule in this clone is designated Z pBR322 Pst HclF 4C Hif 4C , and the bacterial strain containing it Since primary clones of transformed cells occasionally contain more than one species of recombinant DNA molecule Efstratiadis To determine the capacity of Hif 4c and its cDNA insert to hybridize to IFN αmRNA, Hif 4c 115 µg was digested to completion with 125 units of The unlabeled The Hif 4c fragment 120 ng was bound to DPT paper 0.5 0.5 cm as described above. As a control, 120 ng β globin cDNA fragment excised with Thus, Hif 4c contains an insert capable of hybridizing to IFN αmRNA. Since the cDNA insert in recombinant DNA molecule Hif 4c was only about 320 b.p., or a third of the estimated size of IFN αmRNA, the purified Hif 4c fragment described above was used as a probe to screen for bacterial clones containing recombinant DNA molecules having related hybrid DNA inserts Figure 3 . The 64 bacterial clones constituting subgroup λ III described above were stamped onto a Millipore membrane 8 cm diameter , placed on an agar plate supplemented with diaminopimelic acid, nalidixic acid and tetracycline, as above and incubated for 24 h at 37 C. The filter was placed onto a 0.75 ml drop of 0.5 M NaOH and after 2 3 min transferred onto a paper towel to remove excess liquid the step was repeated. The filter was neutralized, using 1 M Tris HCI pH 7.5 , and washed with 1.5 M NaCl 0.5 M Tris HCI pH 7,4 in a similar fashion as above and air dried. The filter was dipped in 0.3 M NaCI, air dried and heated at 80 C for 2 h in a vacuum. Hif 4c Small cultures were prepared from the Hif 4c related clones. Form I DNA was purified, cleaved with Hif 2H was tested for its capacity to bind IFN αmRNA by binding it to DPT paper 4 µg 100 mm² and hybridizing it to poly A RNA 0.3 µg µl , all as described above, for 16 h and determining IFN αmRNA activity Hif 2H was recloned as described for Hif 4C and designated Hif 2h. In a further experiment an additional set of It is, of course, evident, given the ability of Hif 2h to produce a polypeptide displaying an immunological or biological activity of HuIFN As described above recombinant DNA molecule Hif 2h contains an insert of about 900 b.p., and hybridizes to human leukocyte interferon nRNA. The following additional characteristics were determined. If mRNA is hybridized to a cloned, complementary cDNA, the translation of the mRNA is inhibited. however heat denaturation of the hybrid releases translatable mRNA B. M. Paterson Therefore, Hif 2h, when hybridized with poly A RNA, inhibited the translation of the IFN αmRNA in the poly A RNA after denaturing the hybrid, the IFN αmRNA was again translatable. This experiment confirms that Hif 2h contains sequences complementary to IFN αmRNA. Digestions of Hif 2h with various restriction enzymes New England Biolab were carried out, and the resulting products analyzed by agarose gel electrophoresis. The underlined fragments are not common to pBR322. and Hif 2h In addition, 5 terminally ³²P labeled From these data a restriction map of Hif 2h was deduced Figure 4 . The position of the restrictions sites in Figure 4 are based on fragment sizing by agarose gel electrophoresis and may also be incomplete in regard to Although the actual structure of the Hif 2h fragment or other inserts in clones of this invention or the amino acid sequence or structure of the polypeptides coded therefrom is not required for one of skill in the art to make and use the invention described and claimed herein, the above data and restriction map were included in the original application hereto as the best available information on the structure of fragment at the time of filing the original application. Since that time, as expected Restricted DNA 20 µg was extracted with phenol, precipitated with ethanol, dissolved in 0.05 M Tris HCI pH 8 , and passed over a small column of Chelex 100. Fragments with flush or 5 overhanging ends were dephosphorylated by treatment with 0.2 units calf intestinal alkaline phosphatase Boehringer per pmol DNA 5 ends in 200 µl 0.05 M Tris HCI pH 8 for 60 min at 37 C. The enzyme was inactivated by heating 60 min at 65 C. For DNA fragments with 3 overhanging ends, bacterial alkaline phosphatase Worthington was used as described A. M. Maxam and W. Gilbert, After ethanol precipitation, the DNA was 5 terminally labeled with γ ³²P ATP 12 34 µCi per pmol DNA 5 ends and polynucleotide kinase New England Biolabs or P L Biochemicals Inc. essentially as described by A. M. Maxam and W. Gilbert, For sequencing, labeled fragments were cleaved with a second restriction enzyme and the products separated by electrophoresis through a 5 polyacrylamide gel in tris borate EDTA buffer. The desired fragments were extracted from the gel and purified Muller The fragments were degraded according to the method of A. M. Maxam and W. Gilbert, Each stretch of the cDNA insert was sequenced from both strands and each restriction site which served as labeled terminus was sequenced using a fragment spanning it. The nucleotide sequence thus obtained is depicted in Figures 8 10. As is to be expected the positions of the various restriction sites in this insert are more absolutely located than those determined by restriction enzyme cleavage alone and depicted in Fig. 4. Referring now to Figs. 8 10. the heteropolymeric part of the insert is flanked by 23G residues at the 5 end and by 7A residues probably reflecting the poly A terminus of the mRNA followed by 15C residues at the 3 terminus. For reference, the insert is numbered from the first nucleotide following the dG tail to the last nucleotide before the polyA residues. An ATG initiation triplet in position 57 59 and a TAA termination triplet at position 624 626 define a reading frame uninterrupted by nonsense codons. Both other reading frames in this region of the insert contain 18 and 12 nonsense codons respectively. Moreover, the only other sequences, It should of course be understood that cloned cDNA from polyA RNA by usual procedures A. Efstratiadis By comparing the polypeptide coded by this region of the insert with that sequence of 35 amino terminal amino acids of authentic human Iymphoblastoid interferon SerAspLeuProGlnThrHisSer LeuGlyAsnArgArgAlaLeulleLeuLeuAlaGlnMetGlyArglleSerLeuPheSerCysLeuLysAspArgHisAsp determined by K. C. Zoon In eukaryotic mRNAs the first AUG triplet from the 5 terminus is usually the initiation site for protein synthesis M. Kozak, How Do Eukaryotic Ribosomes Select Initiation Regions In Messenger RNA , Cell, 15, pp. 1109 25 1978 . The codon in the Hif 2h fragment corresponding to the first amino acid of Iymphoblastoid interferon is 22 codons from the first AUG and 14 codons from the second one indicating that the sequence coding for interferon may be preceded by a sequence determining a signal peptide of 23 or less likely 15 amino acids. The longer of the presumptive signal sequences contains an uninterrupted series of 1 1 hydrophobic amino acids and the shorter one, one of 6 hydrophobic amino acids . This accumulation of hydrophobic residues is characteristic of signal sequences The nucleotide sequence apparently corresponding to mature IFN a polypeptide comprises 498 nucleotides, which code for 166 amino acids. Assuming that there is no carboxyterminal processing, the molecular weight of the interferon polypeptide is 19,388. The base composition of the coding sequence is 50 GC. The codon usage within the interferon coding sequence is in reasonable agreement with that compiled for mammalian mRNAs in general R. Grantham, The structure of the polypeptide depicted in Figure 8 10 for the Hif 2h fragment, of course, does not take into account any modifications to the polypeptide caused by its interaction with The DNA strand that has the same sequence as the mRNA is designated as plus strand, and its complement as minus strand. The plus strand of the IFN αcDNA insert was identified as outlined in Figure 5. Hif 2h DNA was cleaved with the restriction enzyme An experiment identical to that described in the proceding section was carried out, however, the poly A RNA was from non induced human leukocytes, prepared by the same procedure as in the case of Sendai virus induced leukocytes. No detectable amount of labelled DNA was protected. By comparison to the results of the preceding section the poly A RNA from non induced cells contains less than about 1 20 the amount of mRNA hybridizable to Hif 2h than does poly A RNA from induced cells. The Extracts of some of the above strains as well as of some of the strains Z pBR322 Pst HcIF SN1 to 95 were tested for IFN a activity. Bacteria were grown in Tryptone medium to stationary phase, harvested, washed with 1 20 vol based on the vol of the culture 50 mM Tris HCl pH 8 ,30 mM NaCI and frozen. After thawing, the cells were resuspended in the volume indicated below of the previous buffer and lysozyme was added to 1 mg ml. After 60 min at 0 C the suspensions were frozen in an ethanol dry ice bath and thawed at 37 C 5 times, and centrifuged 10 min at 12,000 rpm in a GSA Sorvall rotor. In come cases part of the supernatant S30 was further centrifuged at 100,000xg in a Type 65 Spinco rotor and the supernatants S100 recovered. Such supematants were screened for IFN α activity by the cytopathic effect reduction assay Expt.1 . The colonies showing a positive response in Expt. 1 were reassayed as well as 49 clones from the set Z pBR322 HclF SN 1 to SN 95 described above at a lower dilution Expt. 2 . Some of the more active producers from above were examined in more detail. Cultures were grown to late log phase apparent OD₆₅₀ about 0.9 and the cells lysed as above, in 1 50 of the culture volume. The following activities were found, using Z pBR322 Pst HclF SN32 as a negative control It is to be understood that the actual protein produced by these strains has not been analyzed structurally to determine whether or not it is produced fused to amino acids unrelated to IFN or with all or part of IFN s signal sequence. However, whatever protein is produced, it displays an immunological or biological activity of IFN. Therefore, the protecin as expressed is useful. Most importantly, the activity of the protein demonstrates that the DNA sequence which codes for it is a DNa sequence related to HuIFN α. It therefore is within the skill of the art to employ that DNA sequence as is demonstrated herein to select other like HuIFN α related DNA sequences and to provide the basis for other constructions that will express mature interferon or other variants thereof or will improve the yield of the particular protein expressed. The level of production of a protein is governed by two major factors the number of copies of its gene within the cell and the efficiency with which those gene copies are transcribed and translated. Efficiency of transcription and translation which together comprise expression is in turn dependent upon nucleotide sequences, normally situated ahead of the desired coding sequence. These nucleotide sequences or expression control sequences define, Several expression control sequences may be employed as described above. These include the operator, promoter and ribosome binding and interaction sequences including sequences such as the Shine Dalgarno sequences of the lactose operon of Further increases in the cellular yield of the desired products depend upon an increase in the number of genes that can be utilized in the cell. This is achieved, for illustration purposes, by insertion of recombinant DNA molecules engineered in the way described previously into the temperate bacteriophage λ NM989 , most simply by digestion of the plasmid with a restriction enzyme, to give a linear molecule which is then mixed with a restricted phage λ cloning vehicle Particularly useful λ cloning vehicles contain a temperature sensitive mutation in the repression gene In an initial attempt to increase the yield of polypeptide, displaying a biological or immunological activity of human leukocyte interferon, produced from hosts transformed with Z pBR322 Pst HcIF SN35 by the process above described, a restriction map of the DNA insert in the hybrid was determined. This mapping revealed that as compared to Hif 2h, Hif SN35 was lacking a The plasmid Hif SN25 was opened with Referring now to Figure 25, another attempt to improve the yield of polypeptide, displaying an immunological or biological activity of human leukocyte interferon, produced by hosts transformed with Z pBR322 Pst HcIF SN35 SN35 in Figure 25 is depicted. The hybrid plasmid was cleaved using standard procedures with To ensure that the treated plasmid contained the complete IFN αl coding sequence, the plasmid was then cleaved with DNA sequence analysis of C8 IFN αl revealed that the coding sequence following the initiator triplet determined the first seven amino acids of β galactosidase, a Pro residue generated by the fusion, amino acids 16 to 23 of the IFN αI signal sequence and the IFN αI SN35 sequence. Further, examples of various constructions to improve the protein yields in accordance with this invention are discussed in connection with other forms of IFN α 50 µl samples of authentic HuIFN α specific activity, 1.2 10⁶ U mg 5OU , and the S100 extracts described above of Extract Hif 35 1 ml and the S100 extract of HuIFN α specific acitivity 1.2 10⁶ IU mg , and the Hif 35 Hif 287 6 S100 extracts were incubated with various dilutions of sheep antiserum against HuIFN α prep. K. Cantell, Feb. 24, 1976. specific acitivity 450,000 units ml in 100 µl Modified Eagles Medium MEM with 10 calf serum for 30 min at 37 C and 45 µl were assayed for IFN α activity by the cytopathic effect reduction assay. The antibody itself did not cause a cytopathic effect To show that the action of the antibody was not due to an unspecific effect, such as proteolytic degradation, a similar experiment was performed with the mouse interferon system Thus, antibodies directred against HuIFN α specifically inhibit the IFN α activity of polypeptides produced in Human CCL23 or Mouse L929 cells were treated with These results show that Hif 35 and Hif 287 6 extracts have a protective action on human cells and only a slight effect 10 on mouse cells, as is typical for human interferon. Extracts from IFN α without amino terminal sequences has also been made in To construct the appropriate recombinant DNA molecule, plasmit Hif 2h Some of these plasmids were sequenced. One began at the fifth amino acid of IFN αl and one at the tenth amino acid of IFN αI. In The comparison of the first 35 amino acids of authentic lymphoblastoid interferon Zoon The hybrid plasmid Z PBR322 Pst HcIF II 206 HclF II 206 of this clone and its DNA insert Hif II 206 fragment are weakly hybridizing to Hif 4c and Hif 2h fragment. Referring now to Figures 12 16, the nucleotide sequences of the inserted DNA sequence Hif II 206 fragment of culture HcIF G and the inserted DNA sequence Hif 2h fragment of culture HcIF E Restricted DNA usually about 10 µg was 5 terminally labelled as described by N. Mantei et al., Referring now to Figure 17, the various fragments for sequencing were prepared as follows 25 and 26 cleavage of Hif II 206 with 21, 22 and 23 cleavage of Hif II 206 with 11, 12, 13, and 14 cleavage of Hif II 206 with 27L, 27U, 41, 43, 44 and 45 cleavage of Hif II 206 with All segments, except 27U and 27L, were sequenced on both strands and across the restriction sites that served as origins for sequencing, except for the From a comparison of the amino acid sequence coded for by the two inserts it is apparent that the Hif II 206 fragment codes for an interferon like protein having one less amino acid than does the Hif 2h fragment amino acid 44 Asp present in Hif 2h is missing in Hif II 206 . Moreover, 10 of the nucleotide positions and 17 of the derived amino acid residues of the two fragments are different. In addition, when compared to the 35 amino acids determined for the amino terminus of lymphoblastoid interferon K. C. Zoon et al., The differences between IFN α1 and IFN α2 are also reflected in their varying activities on human CCL23 and bovine embryonic kidney BEK cells Therefore, IFN α1 is about 30 times less active on human cells than IFN α2. Yet, they are both about equally active on bovine cells. Therefore, IFNs may, in addition to their use as antiviral and antitumor or anticancer agents in humans, also be useful in treating these conditions in cattle. For example, preparations of HuIFN α could be employed in a standard manner Because of the improved yield attained in the case of IFN α1 with construction C8 DNA sequencing of hybrid plasmid C8 IFN α2 revealed that it like C8 IFN α1 had a coding sequence following the initiator triplet that determined the first seven amino acids of β galactosidase, a Pro residue generated by the fusion and amino acids 16 to 23 of the IFN α22 signal sequence. Therefore, again a fused protein containing IFN α2 is likely to be expressed. Minicells transformed with this plasmid gave 100 200 millions units per 1 of IFN or 20000 to 40000 times higher yields of IFN α2 than minicells transformed with unmodified Z pBR322 Pst Hif II 206. A comparison of the relative yields of C8 IFN α1 50 10 units liter and C8 IFN α2 100 200 million units per liter is at first surprising because IFN α2 has been shown to be about 30 times more active than IFN α1 on human cells Referring now to Figure 26, another construction in an attempt to improve the yield of IFN α2 is described. First, an expression plasmid containing the LAC Another IFN α2 construction based on similar linking principles has also been made. Here, the penicillinase expression control sequence of pBR322 has been connected via an AUG initiator codon to the IFN α2 gene from Hif II 206. This plasmid designated as β lac AUG α2 when used to transform host cells affords the production of IFN α2 without fusion to other protein sequences. In minicells, yields of 50 100 million units per liter have been observed. This plasmid is the most preferred plasmid for use in accordance with this invention. It is also preferred for use with the other IFN a genes disclosed herein. The preferred host in accordance with this invention is Other constructions using various promoter sequences, ribosime binding sites. Shine Dalgarno sequences and DNA sequences between the promoter and the AUG initiator codon and using various of the IFN α genes disclosed herein can also be constructed using similar methods and principles. These constructions are of course envisioned by this invention and are within the scope thereof. A number of hybrid molecules of IFN α1 and IFN α2 have been constructed. Surprisingly these hybrid constructions have quantitatively different properties and activities as compared either of their parents IFN α1 or IFN α2. Referring now to Figure 28, a schematic outline of the construction of four of these hybrid molecules is displayed. For convenience these hybrid molecules are designated as plasmids I, II, III and IV. In these constructions digestions with restriction enzymes obtained from Biolabs, with the exception of Hybrid molecule 1, an α 1 Hybrid molecule II, an α 1 Hybrid molecule III, an α 2 Hybrid molecule IV, an α 2 The biological activities of the different interferon species relative to each other were also determined. Cultures of minicells DS410 transformed with the various plasmids were grown and the bacteria harvested by centrifugation, washed with PBS, suspended in PBS about 1 20 of the original vol , incubated for 60 min at 0 C with 1 mg ml lysozyme, 10 mM EDTA, freeze thawed four times, sheared by passing five times through a syringe and cleaved by centrifugation. The activities on human, mouse, guinea pig and bovine cells were as follows Surprisingly, all of the interferons have about the same activity on bovine cells, yet IFN α1 and the two hybrid IFN s I and II which have the amino terminal moiety of IFN α1 have about a 10 to 1000 fold lower activity on human cells than IFN α2 and the two hybrid IFN s III and IV having the amino terminal moiety of IFN α2. It is even more striking that the two hybrid IFN s I and II with the amino terminal part of IFN α1 have more than a 10 fold lower activity on human cells than IFN α1 itself. Yet, one of the hybrids III with the amino terminal part of IFN α2 has about the same activity on human cells as IFN α2. A collection of hybrid phage derived from fragments of fetal human chromosomal DNA which had been generated by partial cleavage with Referring now to Figures 18 and 24, it can be seen that clones chr 3 and chr 26 may represent DNA segments which overlap over much of their length because they have several Accordingly, it appears that the 13 hybridizing portions of the 11 hybrid chromosomal DNAs fall into not less than 10 distinct classes chr 1, chr 3, chr 12, chr 13, chr 16 left hand, labelled 1 in Figure 18 chr 26, chr 30, chr 35, chr 19 and chr 27. Referring now to Figure 24, the overlapping of chr 1, chr 3, chr 10 and chr 26 and that of chr 16 and chr 35 are displayed. The above data suggests that the genome of an individual human contains not less than 10 different DNA sequences that cross hybridize to Hif 2h. This conclusion is reinforced by the fact that the proportion of fragment Hif 2h related sequence detected in the clone bank is about 1 in 16,000. Assuming a value of 3 10⁹ bp for the haploid human genome, the expected value for a single gene copy with an average DNA fragment size of 16kb the average value of the clones examined is about 1 190,000. Therefore, the frequency of Hif 2h related fragments is 12 times higher than expected for a single gene. In comparison to these data, when Lawn et al As an illustration only, the hybridizing sequence of chr 35 Hif chr35 was further characterized. It is to be understood that the hybridizing portions of the previously described chromosomal hybrid phages could similarly be characterized and handled without departing from the scope of this invention. The hybridizing portion of chr 35 Hif chr35 and the right hand segment of Hif chr16 to which it is most likely identical, To analyze the Hif chr35 fragment further, a Clones of these transformants were screened by in situ colony hybridization D. Hanahan and M. Meselson, Cultures of these positive clones were grown to an apparent OD₆₅₀ 0.8 and the bacteria harvested and lysed by the lysozyme freeze thaw method described in S. Nagata et al., The DNA insert of one of these 7 IFN producing clones For nucleotide sequence determination, Hchrlf 35HBα was cleaved with various restriction enzymes, the products separated by electrophoresis through a 5 polyacrylamide gel in Tris borate EDTA buffer A. C. Peacock and C. W. Dingman, The sequencing strategy employed is depicted in Figure 19 and described as follows The various fragments were sequenced by the Maxam Gilbert procedure A comparison of the nucleotide sequence of the coding region of HchrIF 35HBα and that of Hif 2h coding region Figures 8 10 compared to Figures 20 23 reveals that they are identical. In particular, it is surprising that there is no indication of the presence of introns within the coding sequence of the HchrIF 35HBα fragment, i.e., between the The gene containing segments of chr 3 and chr 26, which appear identical by heteroduplex analysis but differ in at least one A comparison of Figures 29 32 with Figures 8 10, 12 16 and 20 23 reveals that the proteins encoded by each of the sequences differ from each other in about 15 of their residues. This divergence is typical for products of non allelic genes which have diverged 20 90 million years ago. Plasmid Z pBR322 Pst HchrIF 35HBα It should of course be understood that the nucleotide sequences depicted in Figures 8 10, 12 16, 20 23 and 29 32 do not take into account any modifications to the nucleotide sequences such as mutation, including single or multiple, base substitutions, insertions, inversions or deletions which may have already taken place or which may subsequently be employed. Moreover, the sequence also does not take into account the possible substitution of other codons coding for the same amino acid as a codon depicted in these figures. Therefore, it should be understood that such modified sequences as code for polypeptides displaying an immunological or biological activity of IFN α are also within this invention. In addition, it is to be understood that the amino acid sequences depited in Figures 8 10, 12 16, 20 23 and 29 32 do not take into account any modifications to the polypeptides caused by their interaction with Since the cytopathic effort reduction assay W. E. Stewart II and S. E. Sulkin, We have isolated a set of recombinant DNA molecules containing cDNA prepared from poly A RNA from Sendai virus treated induced human leukocytes, representatives of which have the following properties These properties demonstrate that the recombinant DNA molecules described by this invention contain at least a part of the coding sequence for human leukocyte interferon and that some of these plasmids lead to expression in Micro organisms and recombinant DNA molecules prepared by the processes described herein are exemplified by cultures deposited in the culture collection Deutsche Sammlung von Mikroorganismen, in Gottingen, West Germany on January 7, 1980. and identified as HcIF A to E These cultures were assigned accession numbers DSM 1699 1703, respectively. In addition, micro organisms and recombinant DNA molecules prepared by the processes described herein are exemplified by cultures deposited in the culture collection of the American Type Culture Collection, Rockville, Maryland on March 27, 1980 and identified as HcIF G to H, and assigned ATCC accession numbers 31633 and 31634 respectively Other micro organisms prepared by the processes described herein are exemplified by cultures deposited in the culture collection Deutsche Sammlung von Mikroorganismem, in Gottingen, West Germany on October 1, 1980 and identified as HchrIF A through J, and assigned accession numbers DSM 1941 1923 While we have hereinbefore presented a number of embodiments of this invention, it is apparent that our basic construction can be altered to provide other embodiments which utilize the processes and compositions of this invention. Therefore, it will be appreciated that the scope of this invention is to be defined by the claims appended hereto rather than the specific embodiments which have been presented hereinbefore by way of example.