# GENETICALLY ENGINEERED MICROORGANISMS FOR MASSIVE PRODUCTION OF AMYLOLYTIC ENZYMES AND PROCESS FOR PREPARING SAME

## Claims
Rekombinante DNA, enthaltend ein Amylase kodierendes Gen als einziges Fremdgen, das exprimiert werden soll, hergestellt mit einem in vitro Verfahren, bei dem DNA, die aus einem bakteriellen Donor Mikroorganismus stammt, gespalten wird und die entstehenden DNA Fragmente mit einem Vektor, der auf dieselbe Weise gespalten wurde, kombiniert wird,

## Description
This invention is in the area of genetic engineering and relates specifically to the production of recombinant DNA deoxyribonucleic acid containing an amylase coding gene as the only foreign gene desired to be expressed, and to the utilisation of same to produce microorganisms for the massive production of amylolytic enzymes. Although the term genetic engineering is frequently used to describe a large number of techniques for the artificial modification of the genetic information of an organism, throughout the specification and claims it is employed only in reference to the The terms amylase and amylolytic enzyme are synonymous, and as used throughout the specification and claims refer broadly to those enzymes capable of catalyzing the hydrolysis of starch, such as alpha amylase, beta amylase, iso amylase including the alpha 1,6, glucosidases such as pullulanase , glucoamylase, etc. There has been, of course, a great deal written on the subject of genetic engineering within recent years just two of the many excellent reviews being DNA Cloning and the Analysis of Plasmid Structure and Function by K.N. Timmis, S.N. Cohen and S.C. Cabello, In accordance with our invention, genetically engineered microorganisms are prepared which are capable of producing, under appropriate conditions of cultivation, substantially greater amounts of amylolytic enzymes than can be produced by the donor microorganism. In a preferred embodiment, the amylases so produced will have one or more characteristics making them particularly useful in industrial enzymatic starch hydrolyses e.g. in the enzymatic liquefaction and or saccharification of starch to produce adhesives, sisings, malto dextrins, starch syrups of various compositions, maltose, dextrose, etc. such as resistance to high temperature, resistance to heavy metal poisoning, etc. The invention is accomplished by first extracting the DNA from a bacterial microorganism the donor microorganism which is capable of producing at least one amylolytic enzyme, cleaving with an appropriate restriction enzyme the DNA plus, as the vector, the DNA of a derivative of phage lambda, and combining and ligating the resulting fragments to form recombinant DNA s, some of which will contain an amylase coding gene. The recombinant DNA s are then made biologically active by insertion into suitable host cells i.e. The resulting clones are then screened for the presence of an amylase coding gene, and one or more positive clones are selected and multiplied, thereby providing new, genetically engineered bacterial microorganisms which are capable of producing, under suitable conditions of cultivation, substantially greater quantities of amylase than can be produced by the donor microorganisms. Optionally the DNA of the new phage is extracted, cleaved and sub cloned into a second vector, which may be either a plasmid or another phage, and the new clones screened and selected on the basis of the presence of an amylase coding gene. Successive sub sub clonings may also be performed. In addition to producing new, genetically engineered microorganisms which are over producers of amylase, the invention has the further advantage that it results in the transfer primarily of the gene for the production of a single amylolytic enzyme, thereby minimizing greatly the purification necessary with cultures of non genetically engineered microorganisms. Once the genetically engineered microorganism containing the desired recombinant DNA has been produced the microorganism is cultivated in such a way as to amplify the recombinant DNA and thereby provide amylase in substantially greater amounts than can be provided by the donor microorganism. When the vector is a derivative of phage lambda, in which case the host microorganism will necessarily be When the vector host system is an appropriate lambda lysogenic When the vector is a multicopy plasmid such as pBR 322 or pACYC 184 amplification of the foreign DNA is achieved per se. Alternatively, the genetically engineered microorganism containing plasmid DNA is first cultivated to multiply the bacterial cells to the desired density, after which chloramphenicol is added because the antibiotic inhibits protein synthesis it prevents further cell multiplication and amylase production but permits multiplication amplification of the plasmid DNA within the cells. Ultimately, the cells are separated from the culture medium and washed to eliminate the chloramphenicol. For amylase production the cells are then recultivated, in the absence of chloramphenicol of course, for the production of massive amounts of amylase. It is essential, of course, in any genetic engineering work, to be able to mark , and thereby select, those clones containing the desired genetic information. In the practice of the instant invention this can readily be accomplished by plating on a medium containing starch,if alpha amylase, beta amylase or glucoamylase activity is being sought. The culture medium is then stained with iodine clones displaying amylase activity on a starch containing medium are surrounded by a white area. A specific, preferred staining method will be described hereinafter. If pullulanase activity is being sought the clones are plated on a BBL Trypticase medium containing pullulan. This technique is described in more detail hereinafter. We shall now describe the invention in more precise detail, including the specific materials and techniques which we have employed in our work. It will be recognized that many of the techniques employed are standard and well known to practitioners in this field nevertheless, some of these will be described in detail, to ensure clarity. Other advantages of using a phage, rather than a plasmid, for the primary cloning are the following 1 the percentage of recombinants having foreign DNA inserts is higher 2 the bacteria are lysed, thus liberating the cell contents, and therefore any amylase, into the medium, thus facilitating the detection by the iodine staining technique 3 the resistance of a phage toward iodine is higher than that of any living bacterium. The selection of a suitable plasmid as a vector for a sub cloning is well within the skill of the competent geneticist, one criterion being, of course, the existence of a single or a limited number of restriction sites for the restriction enzyme to be employed. The plasmids employed in most of our work are pBR 322 and pACYC 184. In its intact state plasmid pBR 322 confers resistance to both ampicillin and tetracycline, and contains a single restriction site for each of Pst I, Eco RI, Hind III, Bam HI and Sal I enzymes. Cutting and insertion at the Pst I site destroys the capacity to confer resistance to ampicillin, while insertion into the Bam HI and the Sal I sites will destroy the resistance to tetracycline. Insertion into the Hind III site sometimes destroys the resistance to tetracycline, providing that the cloned gene does not possess its own promotor. In its intact state plasmid pACYC 184 confers resistance to both tetracycline and chloramphenicol, and contains single restriction sites for each of Eco RI, Hind III, Bam HI and Sal I enzymes. Cutting and insertion at the Eco RI site destroys the capacity to confer resistance to chloramphenicol, while insertion into the three other sites for Hind III, Bam HI and Sal I have the same effects as in plasmid pBR 322, since this region is common to both plasmids. For sub cloning into For practical reasons we have used As is described in Example II A, a recombinant plasmid can be subcloned into a plasmid such as pC 194, capable of replicating in Having selected the donor microorganism, the restriction and ligating enzymes and the specific derivative of phage lamda, the DNA s are extracted, restricted, mixed, and the joined pieces are ligated, all by conventional techniques which need no description here. The DNA is then made biologically active in an Many several thousand of the clones which have accepted foreign DNA are then plated onto media containing starch and screened for the presence of an amylase coding gene by the iodine staining method mentioned previously. Care must be taken, of course, not to employ so high a concentration of iodine to kill the phage, and this can be a problem if the iodine is added in the form of a solution. Our greatly preferred staining technique is to expose the plates to iodine vapors for a short time we have employed this technique with very good success. A preferred method for detecting clones having pullulanase activity, which method does not employ iodine staining, will now be described. The phages are plated on petri dishes containing BBL Trypticase plus pullulan at a concentration of about 0.25 . The pullulan in the medium around the positive plaques is hydrolyzed to maltotriose by the pullulanase, which maltotriose is used by the bacteria. Therefore those bacteria feeding on the maltotriose grow better than the others in the lawn, with the consequence that the plaques producing pullulanase are surrounded by an opaque ring of growing bacteria and can readily be detected. This technique is described in detail in Example IV. One or more positive clone is then picked and multiplied. This can, of course, comprise the final genetically engineered microorganism, and it can be used to produce large amounts of amylase by appropriate cultivation as described previously. Alternatively, the clone can be used as a source of DNA for a second cloning into a plasmid or another, more suitable phage. The process of sub cloning into a plasmid will now be described. Again using standard techniques the DNA of the clones is extracted and cleaved with a restriction enzyme, the plasmid is similarly cleaved, the fragments are mixed and the recombined fragments are ligated. When using plasmid pBR 322 identification of clones which have accepted DNA containing an amylase coding gene is accomplished by making the DNA biologically active in a host such as The drawings present maps of wild type bacteriophage lambda Figure 1, a and all vectors used, and new recombinant plasmids produced, in accordance with the following examples. Specifically, Figure 1 also shows maps of phages lambda NM 590 b and lambda NM 781 c , MURRAY, N.E., BRAMMAR, W.J., MURRAY, K., 1977 Molec. gen. Genet, 150, 53 61. Fig. 2 comprises maps of plasmids pBR 322 BOLIVAR, F., RODRIGUEZ, R.L., GREENE, P.J., BETLACH, M.C., HEYNEKER, H.L., BOYER, H.W. 1977 , Gene 2, 95 113 and pAYC 184 CHANG, A.C.Y., COHEN, S.N. 1978 J. Bact. 134, 1141 1156 . Fig. 3 is a map of new plasmid pCP 1 and Fig. 4 is new plasmid pCP 2. Figure 5 illustrates Example II A, showing plasmids pC 194, pCP 2.3, and final new plasmid pCH 1. The examples will exemplify the practice of the invention. They are presented for illustrative purposes only, and should not be construed as limiting the invention in any way. Percentages are by weight unless specified otherwise. All experiments were performed following the NIH U.S.A. guidelines for containment. The following materials were used T4 DNA ligase. Phage lambda NM 590. The phage DNA was prepared by phenol extraction from highly purified phage particles. Plasmid pBR 322. Plasmid DNA was purified from lysozyme lysed Host microorganism, The donor microorganism was a strain of The microorganism was identified as Following is a description of the process used. About 7 µg of Two tests were performed to determine the efficiency of the cleavage The cleaved DNA s were mixed and incubated for 5 hours at 10 C with 0.15 units of T4 DNA ligase, to allow random reannealing and covalent sealing of the DNA fragments. At the end of this incubation, the DNA was mixed with an To assess the efficiency of this process, two controls were performed Thus one could assess that the preparation contained a random sample of all The remainder of the encapsidation mixture was used to inoculate This was achieved both to obtain an overproducing strain and to prepare easily a large amount of DNA with the amylase gene. 1 µg of DNA from the lambda NM 590 Amy 1 and 0.3 µg of plasmid pBR 322 DNA were cleaved, mixed and treated with ligase, as described in section A. This preparation was used to transform using usual methods the strain HB 101. Selection was for ampicillin ap resistance, a property conferred to the cells by the presence of this plasmid. This plasmid confers normally also resistance to tetracycline Tc . However, the introduction of foreign DNA in this plasmid splits the This demonstrates that re cloning a gene with the same enzyme in this case Hind III is a very efficient process 16 instead of 1 10⁵ . The plasmid containing microorganism is designated as Enzyme production in plasmid pCP 1 containing cells was improved in two ways, as follows Cultures were incubated overnight at 37 C on a rotary shaker in 5 l. baffled erlenmeyer flasks containing 1 l. of culture medium LB yeast extract tryptone . Cultures were incubated at 37 C on a rotary shaker in 5 l. baffled erlenmeyer flasks containing 1 l. of culture medium LB until a density of 0.8 650 nm was reached, at which point 150 µg ml of chloramphenicol was added. This prevented further replication of the chromosomal DNA but not that of the amylase gene containing plasmid pCP 1 . After amplification up to 3,000 copies of the plasmid, compared to chromosomal DNA, chloramphenicol was removed to allow protein synthesis to resume. Specifically, after amplification the cells were separated from the medium by centrifugation, washed to eliminate chloramphenicol, and recultivated for amylase production. We employed the osmotic shock method to recover the alpha amylase in very pure form. The process, which is reported by H.C. Neu and L.A. Heppel in The cells were first cultured overnight, after which they were suspended in a 25 sucrose solution in 0.5 volume of culture and shaken for 10 minutes at 24 C, which treatment plasmolyses the cells. EDTA was then added to 1mM of final to render the cell walls permeable , and the material was shaken for another 10 minutes at 24 For comparison the donor microorganism The enzyme produced by The microorganism herein called The donor microorganism was an original strain of The strain has been deposited with the NCIB on February 12, 1980, as NCIB No. 11571. The DNA was extracted and submitted to the action of Eco RI Hind III Pst I Sal I Bam HI and Bgl II. Only Bgl II was able to generate many fragments of a wide range of molecular weights. It was, however,possible to generate fragments with Eco RI, when the NaCl concentration was decreased to 50 mM. We therefore decided to use Eco RI to generate fragments for cloning into an Eco RI lambda DNA vector lambda NM781 . 1.25 µg of lambda NM 781 DNA were cut by one unit of Eco RI in 25 µ λ of the following buffer The restricted DNA s were mixed and ligated using two units of T₄ DNA ligase in a mixture containing 60 mM Tris HCl pH 8 , 10 mM MgSO₄, 10 mM 2 mercaptoethanol and 0.1 mM ATP. The reaction was done for 10 to 15 hours at 10 C. After the ligation, aliquots of 0.2 µg of DNA were mixed with ATP to give a final concentration of 10 ²M. These aliquots were submitted to A rather high proportion of amylase producing phages was observed 1 in 400 . One was selected and is designated as lambda NM 781 alpha Amy 1 . It has been deposited with the NCIB on February 12, 1980, as NCIB No. 11572. 1 µg of lambda NM 781 alpha Amy 1 DNA and the same amount of DNA from the plasmid pBR 322 were cut with Eco RI, using the usual conditions. The ligation and transformation were done as explained in the Example I. Colonies of Amplification of the amylase coding gene of lambda NM 781 alpha Amy 1, and amylase production, were accomplished as follows. The host bacteria Amplification and enzyme production with The amylase activity was determined using the DNS method both in the phage lysate and in cultures of In the case of the phage lambda NM 781 alpha Amy 1 all of the enzyme is liberated upon cell lysis and consequently is present in the culture medium. In the case of the plasmid, most of the enzyme is cell bound 96 . To illustrate the preparation of a vector host system comprising a lambda lysogenic Bacteriophage lambda T4 lig. CI 857 Wam Eam Sam lambda NM 989 was employed. It is described in This phage contains the DNA ligase gene of bacteriophage T4 and is able to lysogenize The DNA of the phage and plasmid pCP 2 were cut with Ecc RI and the fragments ligated. The phage DNA resulting from the ligation was then in vitro packaged and the plaques visualized by plating on a strain of This phage was then used to lysogenize a strain of Amplification of the gene product was accomplished in the following way. The lysogen was grown at 32 C in LB medium until a density of 0.8 at 650 nm was reached, then centrifuged and the cells were suspended in fresh LB medium which had been pre warmed at 45 C. The culture was then incubated in a 45 C bath for 15 minutes in order to induce the lytic cycle since the immunity gene product is thermosensitive . The incubation was then continued with vigorous shaking at 37 C for 3 hours. The amylase activity was then measured in the supernatant and in the cells. The values are reported in Table I. This particular experiment illustrates a sub sub cloning technique from the plasmid into a new lambda phage. It will be readily understood that the DNA from lambda NM 781 alpha Amy 1 could just as easily have been sub cloned into phage lambda T4 lig. CI 857 Wam Eam Sam. Alternatively the donor DNA could have been cloned directly into the latter mentioned phage, any number of variations of the exemplified process being feasible. As in Example I, we employed the osmotic shock method for recovery of the enzyme. In addition, because the enzyme of this example is thermostable it could be further purified by adding to the aqueous solution 10 mM Ca and warming the solution to 80 C and holding for 10 minutes this treatment precipitates all of the The specificity of the alpha amylase was determined it is active on amylose and starch, the products of hydrolysis are glucose, maltose and maltotriose with traces of higher molecular weight components. This enzyme is not active on cyclodextrin. The thermo resistance of the enzyme was also studied, and was shown to be very high. The optimum temperature for activity with 0.5 amylose is between 80 C and 90 C. With 8 soluble starch, the optimum is situated around 100 C. It is possible that the microorganism herein called This example illustrates a technique whereby recombinant DNA prepared in accordance with the invention can be expressed in a host bacterium other than Plasmid pCP 2, from Example II, contains a 3.31 Kb fragment from the New plasmids, designated by us as pCP 2.3, were formed, having a 2.64Kb fragment of the For the next step plasmid pC 194 was selected, which is a plasmid known to be capable of replicating itself in The mutant strain of The protoplasts were then incubated in a rich medium for 1.5 hr. in order to permit the plasmid in the bacteria to express the resistance to chloramphenicol. The protoplasts were then plated on a regeneration medium to which had been added chloramphenicol in an amount of 20 µg. ml. After 2 days of incubation at 37 After an overnight culture in the presence of chloramphenicol, the quantity of alpha amylase produced was determined as follows The donor microorganism was a strain of The methods used for the restriction, ligation and recovery of the recombinant phages were the same as for Example II. The DNA 2 µg of In order to increase the production of enzyme this first beta amylase clone was used as a source of beta amylase coding DNA to subclone it into the multicopy plasmid pBR 322. 2 µg of lambda NM 781 beta amy 1 DNA and 1 µg of pBR 322 were cut with Eco RI, mixed and treated with T4 ligase. Ligation and transformation were realized as described previously. One colony on 200 ampicillin resistant colonies showed a starch degradating activity. One was isolated and designated The vector employed was the thermosensitive lambda T4 lig phage CI 857 Wam Eam Sam lambda NM 989 described in Example II. About 1 µg of the plasmid pCP 3 DNA and 0.5 µg of the phage DNA were cleaved, then the fragments ligated together and the resulting pieces of DNA packaged in vitro. The phage particles displaying an amylase activity were isolated and utilized to obtain lysogenic colonies by the same method as before. The strain In all clones and subclones, the amylase activity was detected by the DNS method. The beta amylase activity was confirmed on TLC chromatograms by the presence of a single spot of maltose after digestion of amylose. Amplification of the beta amylase gene in different clones was performed as described before. Table II presents the enzymatic activity in the original strain compared to that in the three clones this activity was recovered either from the phage lysate or by the method of osmotic shock described in Example II. The donor microorganism was a 2.25 µg of donor DNA were extracted and fragmented with Eco RI, using standard conditions, and 1.25 µg of lambda NM 781 DNA was cut by the same enzyme, under the same conditions. The incubation was conducted for three hours at 37 C, after which the reaction was stopped by heating for 10 minutes at 75 C. The completeness of the restriction was determined as in Example II. The restricted DNA s were ligated, recovered, and used to infect strain HB 101 of After two days incubation at 37 C on plates containing BBL Trypticase plus 0.25 pullulan some plaques showed pullulanase activity, that is to say they were surrounded by opaque rings characteristic of over growing bacteria. The proportion of pullulanase producing phages was in the order of 1 2,500. One was selected and is designated by us as lambda NM 781 Pul 1 it was deposited with the NCIB on April 9, 1980, as NCIB No. 11593. Pullulanase activity was detected by the DNS method and the product of the hydrolysis of pullulan by the phage lysates was identified as maltotriose by thin layer chromatography. Lambda NM 781 Pul 1 contains a large fragment of DNA from The 6.1 Kb subset was further subcloned in the multicopy plasmid pACYC 184 a derivative of pBR 322 with the Tc A third subclone was constructed by inserting the 6.1 Kb fragment into phage lambda NM 989, as described in Example II. This clone is called The level of expression and the amplification of the pullulanase was studied in all clones. The results are summarized in Table III. As can be seen from the results, the pullulanase activity is induced by maltose, 0.04 maltose having been added to the LB medium. Furthermore, a Triton X 103 treatment of the membrane fraction was necessary to recover the pullulanase, indicating that the activity is localized in the membranes.