# Adaptor molecules for DNA and their application to synthesis of gene derived products.

## Abstract
Adaptor molecules have been prepared to comprise either start or stop signals for protein synthesis, in addition to recognition sites for restriction endonucleases. Separate adaptors may be used in a symmetrical duplex form. The start adaptor may include nucleotide base inserts to provide the correct reading frame of the triplet code in a DNA sequence with inappropriate reading frame. Insulin A chain and B chain genes of the human type, have been synthesized with the appropriate adaptor molecules provided on each end. The adapted DNA genes have been joined to replicable cloning vehicles and the hybrid DNA transferred to a host cell. The transformed host cell has been shown to contain the desired insulin gene.

## Claims
CLAIMS 1. Adaptor oligodeoxynucleotide molecules, for facilitating insertion of DNA information sequences into a cloning vehicle, and transcription and translation of said sequences in a host cell each adaptor comprising a a recognition site for a restriction endonuclease, and b a signal for protein synthesis, selected from a start signal and a stop signal, said start signal being downstream of said recognition site and said stop signal being upstream of said recognition site. 2. The start signal adaptor molecules of claim 1 including at least one additional base between each start signal and the recognition site to correct the frame shift of a DNA information sequence with inappropriate reading frame. 3. Adaptor molecules of claim 1 which are symmetrical DNA duplexes having duplicate signals at each end, each complementary nucleotide strand having complete palindromic symmetry to its opposite strand, with both halves of the duplex being identical on 1800 rotation of one half. 4. Start signal adaptor molecules of claim 3 including additional nucleotide bases between each start signal and the central recognition site, to correct the frame shift of a DNA information sequence with inappropriate reading frame. 5. The adaptor molecules of claim 3 wherein from 8 to 20 nucleotide base pairs are present. CLAIMS cont. 6. The start signal adaptor molecule of claim 1 wherein said start signal comprises the ATG codon or theGTG codon. 7. The stop signal adaptor molecules of claim 1 wherein said stop signal comprises at least one codon selected from the group TGA, TAA, and TAG. 6, The adaptor molecule of claim 3 comprising one of the duplex structures EMI27.1 tb SEP Start tb 5 SEP CAT SEP recognition SEP ATG tb 3 SEP GTA SEP site SEP TAC tb SEP Start tb 5 SEP CAC SEP recognition SEP GTG tb 3 SEP GTG SEP site SEP CAC tb 9. The start signal adaptor of claim 8 comprising the structure EcoRI site start 5 C A T G A A T T C A T G 3 G T A C T T A A G T A C 10. The start signal adaptor of claim 4 comprising the structure EcoRI site start 5 C A T A G A A T T C T A T G 3 G T A T C T T A A G A T A C start CLAIMS cont. 11. The start signal adaptor of claim 4 comprising the structure Eco RI site start 5 C A T A A G A A T T C T T A T G 3 G T A T T C T T A A G A A T A C start 12. The start signal adaptor of claim 6 including a strong promoter sequence for mRNA transcription located upstream from the start signal and the recognitionsite. 13. The stop signal adaptor of claim 7 comprising one of the duplex structures EMI28.1 tb SEP stop tb SEP TCA tb 5 SEP TGA SEP recognition SEP tb ACT SEP site SEP AGT tb SEP stop tb SEP 5 SEP TAA SEP recognition SEP TTA tb 3 SEP ATT SEP site SEP AAT tb SEP stop tb 5 SEP TAG SEP SEP recognition SEP CTA tb 3 SEP ATC SEP site SEP GAT tb 14. The stop signal adaptor of claim 13 wherein the recognition site has the following structure 5 G G A T C C 3 C C T A G G Barn HI site CLAIMS cont. 15. The stop signal adaptor molecule of claim 7 having the structure EMI29.1 tb 5 SEP TGA SEP TAG SEP recognition SEP CTA SEP TCA SEP tb 3 SEP ACT SEP ATC SEP site SEP GAT SEP AGT tb 16. The stop signal adaptor molecule of claim 7 having the structure 5 TGA TAG GGATCC CTA TCA 3 ACT ATC CCTAGG GAT AGT Bam HI site 17. An adaptor molecule of claim 1 ligated to a DNA information sequence to give an adapted DNA. 18. The adapted DNA information sequence of claim 17 wherein said sequence is a gene or a partial gene. 19. An adapted DNA information sequence having upstream a first adantor molecule as in claim 1 with a start signal, and downstream a second adaptor molecule as in claim 1 with at least one stop signal. 20. The adapted DNA information sequence of claim 19 wherein at least one of the adaptors is a symmetrical DNA duplex. 21. The adapted DNA sequence of claim 20 comprising the structure EMI29.2 tb SEP start SEP stop tb 5 SEP CAT SEP SEP SEP ATG SEP SEP TAG SEP SEP TCA tb SEP recognition SEP duplex SEP DNA SEP recognition tb 3 SEP GAT SEP SEP SEP TAC SEP SEP SEP ACT SEP SEP SEP AGT tb CLAIMS cont. 22. The adapted DNA sequence of claim 21 wherein the start adaptor has the structure EMI30.1 tb SEP 5 SEP CAT SEP X SEP recognition SEP Y SEP start tb 3 SEP GTA SEP Y SEP site SEP X SEP ATG tb SEP Z SEP Z SEP TAC tb where X and Y are nucleotide bases inserted to correct the frame shift, and Z is 1 or 2 or is the end product of digestion thereof with the enzyme for the recognition site. 23. The adapted DNA sequence of claim 19 with at least one additional nucleotide base between the start signal and the recognition site to correct the frame shift. 24. The adapted DNA sequence of claim 21 comprising the structure EMI30.2 tb SEP Eco SEP RI tb SEP we SEP start SEP stop tb 5 SEP CAT SEP GAATTC SEP ATG SEP DNA SEP duplex SEP TGA SEP GGATCC SEP TCA SEP tb 3 SEP GTA SEP CTTAAG SEP TAC SEP sequence SEP SEP ACT SEP CCTAGG SEP AGT tb SEP Bam SEP III tb SEP site tb 25. The adapted DNA sequence of claim 19 comprising the structure EMI30.3 tb SEP Eco SEP RI tb SEP site tb 5 SEP CAT SEP GAATTC SEP ATG SEP DNA SEP duplox SEP TGA SEP TAG SEP GGATCC SEP CTA SEP TCA tb 3 SEP GTA SEP CTTAAG SEP TAC SEP sequence SEP ACT SEP ATC SEP CCRAGG SEP GAT SEP AGT tb SEP Bam SEP HI tb SEP site tb 26. The adapted DNA sequence of claim 23 ligated at both ends into a replicable cloning vehicle having the appropriate restriction enzyme recognition sites. 27. The adapted DNA sequence cloning vehicle combihation of claim 26 in a transformed host cell. CLAIMS cont. 28. A transformed host cell comprising as part of its genetic information sequences, an adapted DNA sequence of claim i3. 29. The partial duplex DNA molecule, adapted for forming insulin A chain gene, having the structure 5 GGC.ATT.GTG.GAG.CAG.TGC.TGC.ACC.AGC.ATC.TGC.TCC.CTC.TAC.C 21 3 TAG.ACG.AGG.GAG.ATG.G 20 3 cont. TT.GAC.CTC.TTG.ATGACG.TTG. 30. The duplex DNA molecule, gene for insulinA chain, baving the structure 5 GGC.ATT.GTG.GAG.CAG.TGC.TGC.ACC.AGC.ATC.TGC.TCC.CTC.TAC.CAA. 3 CCG.TAA.CAC.CTC.GT C.ACG.ACG.TGG.TCG.TAG.ACG.AGG.GAG.ATC.GTT. 63 CTG.GAG.ACC.TAC.TGC.AAC. GAC.CTC.TGG.ATG.ACG.TTG. 31. . The duplex DNA gene as in claim 30 including in addition adaptor DNA segments at each end, said adaptor segments including adjacent the upstream end of the gene a start signal, and adjacent the downstream end of the gene a stop signal, each adaptor segment also including a recognition site for a restriction endonuclease. 32. The duplex DNA of claim 31 wherein said start signal is the ATG codon or the GTG codon. 33. The duplex DNA of claim 31 wherein the said stop signal is at least one of the codons TGA, TAA and TAG. CLAIMS cont. 34. The duplex DNA of claim 31 wherein the recognition sites are selected from those selective for the endonucleases Eco RI, Bam HI, and Hin dIII. 35. The duplex DNA of claim 31 wherein the start signal adaptor contains an Eco RI site and comprises the structure Eco RI site start 5 C A T G A A A T T C A T G 3 G T A C T T A A G T A C or the end product thereof after Eco RI enzyme digestion. 36. The duplex DNA of claim 31 wherein the stop signal adaptor contains a Ham HI site and has the structure stoo 5 T G A G G A T C C T C 3 A C T C C T A G G A G T Bam HI site .or the end product thereof after Pam HI enzyme digestion. 37. The duplex DNA molecule as in claim 31 wherein the two adaptors are in partial duplex form.and both have protruding cohesive ends which are part of the recognition sites and capable of being rejoined to complementary sequences. 38. The adapted insulin A chain gene of claim 37 comprising the structure EMI32.1 tb SEP 5 SEP RI SEP site SEP start SEP stop SEP 3 tb A SEP A SEP T SEP T SEP C SEP A SEP T SEP G SEP 63 mer SEP duplex SEP T SEP G SEP A SEP G tb 3 SEP G SEP T SEP A SEP C SEP A chain SEP gene SEP A SEP C SEP T SEP C SEP C SEP T SEP A SEP G SEP 5 tb SEP Bam SEP site tb CLAIMS cont. 39. The insulin A chain gene of claim 37 ligated to a replicable cloning vehicle. 40. The gene cloning vehicle hybrid of claim 39 wherein the cloning vehicle is a plasmid or a plasmid fragment resulting from cutting a plasmid with restriction endonuclease. 41. The hybrid of claim 40 wherein the plasmid is of the type pBR322, pBGPl20, or a hybrid of pBGPl2O and pKN402. 42. The hybrid of claim 39 in a transformed host cell 43. Transformed bacterial cells containing the hybrid of claim 39. 44. Transformed bacterial cells of claim 43 wich are a solected strain of B. coli. 45. A culture of the transformed host cells of claim 42, capable of producing insulin A chain protein. 46. A culture of the transformed bacterial cells of claim 43, capable of producing insulin A chain protein. 47. The partial duplex DNA molecule, adapted for forming a complete insulin B chain gene, having the structure 5 TTT GTC AAT CAG CAC CTT TGT GGT TCT CAC CTG GTG GAG 3 TA GTC GTG GAA ACA CCA AGA GTG GAC CAC CTC 5 GCT CTG TAC CTG GTG TGT GGG GAA CGT GGT TTC TTC TAC 3 CGA GAC ATG GAC CAC ACA CCC CTT GCA CCA AAG AAG ATG 5 ACA CC 3 TGT GGG TTC TGG CLAIMS cont. 48. The duplex DNA molecule, gene for theB chain of insulin, having the structure 5 TTT GTC AAT CAG CAC CTT TGT GGT TCT CAC CTG CTC GAG 3 AAA CAG TTA GTC GTG GAA ACA CCA AGA GTC GAC CAC CTC 5 GCT CTG TAC CTG GTG TGT CCC GAA CGT GGT TTC TTC TAC 3 CGA GAC ATG GAC CAC ACA CCC CTT GCA CCA AAG AAG ATG 5 ACA CCC AAG ACC 3 TGT CCC TTC TGG 49. The duplex DNA gene as in claim 48 including adaptor DNA segments at each end, said adaptor segments including a start signal adjacent the upstream end of the gene, and a stop signal adjacent the downstream end of the gene1 each adaptor segment also including a recognition site for a restriction endonuclease. 50. The duplex DNA of claim 49 wherein said start signal is the ATG codon or the GTG cordon. 51. The duplex DNA of claim 49 wherein said stop signal is at least one codon selected from the group TGA, TAA and TAG. 52. The duplex DNA of claim 49 wherein the recognition sites are chosen from those selective for the endonucleases Eco RI, Bam HI, and Hin dill. 53. The duplex DNA of claim 49 wherein the start signal adaptor has the structure Eco RI site start 5 C A T C A A T T C A T C 3 G T A C T T A A G T A C or the end product thereof after Eco RI enzyme digestion. CLAIMS cont. 54. The duplex DNA of claim 49 wherein the stop signal adaptor has the structure stop 5 T G A G G A T C C T C A 3 A C T C C T A G G A C T Ham HI site or the end product thereof after Bam HI enzyme digestion. 55. The DNA gene as in claim 49 wherein the two adaptors are in partial duplex form and both have protruding cohesive ends which are recognized partial recognition sites capable of being rejoined to complementary sequences. 56. The insulin B chain gene of claim 55 with adaptor molecules having the structure EMI35.1 tb 5 SEP AAT SEP C SEP ATG SEP m SEP DNA SEP duplex SEP TGA SEP G tb 3 SEP G SEP TAC SEP SEP sequence SEP i SEP ACT SEP C SEP CTAG tb 57. The insulin B chain gene of claim 55 ligated to a replicable cloning vehicle. 58. The B chain gene cloning vehicle hybrid of claim 57 wherein the cloning vehicle is a plasmid or a plasmid fragment resulting from cutting a plasmid with restriction endonuclease. 59. The hybrid of claim 58 wherein the plasmid is of the type pBR322, pasp120. or a hybrid of pBGPl2O and pKN401. 60. The hybrid of claim 57 in a transformed host cell. CLAIMS cont. 61. Transformed bacterial cells containing the hybrid of claim 57. 62. Transformed bacterial cells of claim 61 which are a selected strain of E. coli. 63. A culture of the transformed host cells of claim 60 for producing insulin B chain protein. 64. A culture of the transformed bacterial cells of claim 61, for producing insulin B chain protein. 65. In Åa method of inserting a genetic information sequence into a cloning vehicle and translating said sequence, the steps comprising enzymatically ligating a first start signal containing adaptor molecule as in claim 1 to the upstream end of said genetic information sequence and ligating a second stop signal containing adaptor molecule as in claim 1 to the downstream end of said genetic information sequence. 66. In a method.of inserting genetic information sequences into a cloning vehicle and translating said sequences, the steps comprising enzymatically ligating separate aliquot portions of said genetic information sequence separately to a a start signal containing adaptor molecule as in claim 1, and b to at least one start signal containing adaptor molecule as in claim 2, said start adaptors b containing at least one additional nucleotide base inserted between said start signal and the endonuclease recognition site to correct a possible inappropriate reading frame, thereby to form a multiplicity of adapted DNA sequences, inserting each of said adapted DNA sequences into a cloning vehicle and subsequently selecting CLAIMS cont. 66. cont. an adapted DNA sequence which provides a desired product upon translation. 67. In a method of forming an insulin A chain gene adapted for insertipn into a cloning vehicle, the step of enzymatically ligating a start signal comprising one of EMI37.1 tb SEP 5 SEP CAT SEP recognition SEP ATG tb SEP 3 SEP GAT SEP Z SEP SEP site SEP TAC tb and tb SEP 5 SEP CAC SEP recognition SEP GTG tb SEP site tb SEP 3 SEP CTC SEP SEP site SEP SEP CAC tb proximate to the head end of an insulin A chain gene whose 5 terminus at the tail end is in the 5 OH form. 68. In a method of forming an insulin A chain gene adapted for insertion into a.cloning vehicle the step which comprises enzymatically reacting the 5 OH tail end of the product of the process of claim 67 with ATP to convert the 5 OH to 5 OP and then enzymatically ligating a stop signal adaptor selected from the group consisting of andEMI37.2 tb SEP 5 SEP TGA SEP recognition SEP TCA tb 3 SEP ACT SEP site SEP AGT tb 5 SEP TAA SEP recognition SEP TTA tb 3 SEP ATT SEP site SEP AAT tb SEP 5 SEP TAG SEP recognition SEP CTA tb 3 SEP ATC SEP site SEP GAT tb to said 5 OP tail end. CLAIMS cont. 69. In a method of forming an insulin A chain gene adapted for insertion into a cloning vehicle the step of enzymatically digesting the product of the process of claim 68 to form cohesive ends at both ends of said adapted A chain gene. 70. A method of inserting an adapted insulinA chain gene into a cloning vehicle which comprises enzymatically cutting and reacting a replicable DNA to form cohesive ends complimentary to the cohesive ends of the product of the process of claim 69 and then attaching each of the cohesive ends of said product of claim 69 to the corresponding complimentary cohesive ends of said enzymatically reacted replicable DNA. 71. In a method of forming an insulin B chain gene adapted for insertion into a cloning vehicle, the step of enzymatically ligating a start signal adaptor comprising one of andEMI38.1 tb SEP 5 SEP CAT SEP recognition SEP ATG tb 3 SEP CAT SEP Site SEP TAC tb SEP 5 SEP CAC SEP recognition SEP GTG tb 3 SEP GTG SEP site SEP CAC tb proximate to the head end of an insulin B chain gene whose 5 terminus at the tail end is in the 5 OH form. 72. In a method of forming an insulin B chain gene adapted for insertion into a cloning vehicle the step which comprises enzymatically reacting the 5 OH tail end of the product of the process of claim 71 with ATP to convert the 5 OH to 5 OP and then enzymatically ligating a stop CLAIMS cont. 72. cont. a signal adaptor selected from the group consisting of andEMI39.1 tb 5 SEP TGA SEP recognition SEP TCA tb 3 SEP ACT SEP site SEP AGT tb 5 SEP TAA SEP recognition SEP TTA tb 3 SEP ATT SEP site SEP AAT tb 5 SEP TAG SEP recoggnition SEP CTA SEP tb SEP 5 SEP TAG SEP recognition SEP CTA tb 3 SEP ATC SEP site SEP GAT tb to said 5 OP tail end. 73. In a method of forming an insulin B chain gene adapted for insertion into a cloning vehicle the step of enzymatically digesting the product of the process fo claim 72 to form cohesive ends at both ends of said adapted B chain gene. 74. A method of inserting an adapted insulinB chain gene into a cloning vehicle which comprises enzymatically cutting and reacting a replicable DNA to form cohesive ends complimentary to the cohesive ends of the product of the process of claim 73 and then attaching each of the cohesive ends of said product of claim 73 to the corresponding complimentary cohesive ends of said enzymatically reacted replicable DNA. 75. A method of producing in vitro a particular genetically produced organic material which comprises culturing transformed host cells containing a genetic sequence which produces said organic material, which sequences have been inserted into a host cell contained cloning vehicle by CLAIMS cont. 75. cont. means of the adaptor molecules of claim 1, under conditions which promote said organic material formation. 76. A method of producing insulin A chain protein which comprises culturing transformed host cells containing an insulin A chain protein producing genetic sequence inserted into a host cell contained cloning vehicle by means of the adaptor molecules of claim 1, under conditions which promote said protein formation. 77. A method of producing insulin B chain protein which comprises culturing transformed host cells containing an insulin B chain protein producing genetic sequence inserted into a host cell contained cloning vehicle by means of the adaptor molecules of claim 1, under conditions which promote said protein formation.

## Description
ADAPTOR MOLECULES FOR yA X THEIR APPLICATION TO SYNTHESIS OF GENE DERIVED PRODUCTSField of the Invention This invention is concerned with DNA information sequences, novel adaptor molecules for these sequences and the joining of the resulting adapted sequences into replicable cloning vehicles. Of particular concern are genes and adapted genes coding for the A and B chains of insulin, the insertion of these adapted genes into cloning vehicles, and the transferring of the hybrid DNA molecules into host cells, the transformed cells thus having the ability to produce the specified insulin chains.Background and Prior Art In very recent years, methods have been developed see Molecular Cloning of Recombinant DNA , eds., W.A. Scott and R. Werner, Academic Press Inc., 1977 1 for the in vitro joining by DNA ligase of a DNA segment to be cloned scheme 1, structure I to a cloning vehicle DNA capable of independent replication, structure II , 2 for introducing the hybrid DNA molecule recombinantDNA, structure III into a suitable cell, 3 for selecting and identifying the transformed cells carrying the desired hybrid DNA cloned DNA as a hybrid DNA, structure IV , 4 for amplifying the desired cloned DNA in the transformed cells, and 5 for expressing the cloned DNA as a,protein product.In most reported cases, DNA molecules isolated from cells or viruses have been fragmented by restriction enzyme digestion or by physical shearing before cloning. Scheme 1. The cloning of a DNA fragment in a bactarium.The dots at the ends of DNA molecules represent the proper sequence that allows matching and ligation of the ends with other DNA molecules. EMI2.1 Protein synthesis in bacteria at a site located within a transferred DNA segment derived from mouse was shown by Chang et al Cell 6, 231 244, 1975 . Still other examples of the cloning of natural foreign DNA have been described recently. Methods for the total chemical syntheis of oligodeoxynucleotides of up to 20 nucleotides long, have been well established by using either the phosphodiester method Khorana, H.G., J. Mol. Biol. 72, 209, 1972 or the improved phosphotriester method Hsiung, H.M., and Merang, S.A, Nucleic Acids Res. 6, 1371, 1979 Narang, S.A.et al, Methods in Enzymology, Vol. 65, IC , 1979, andVol. 68, 90, 1979 . The latter method is now the preferred method because of its higher speed, better yield and purity of products, and has been used to prepare defined DNA sequences of longer length. A few chemically synthesized DNA sequences, such as the lactose operator Marians, Wu et al, Nature 263, 744, 1976 and the tyrosine tRNA gene Khorana, Science 203, 614, 1979 , have been successfully cloned in E. coli and the expression of the cloned DNA detected in subsequent cultures. Recent reports have indicated that human brain hormone somatostatin 14 amino acids has been produced in a transformed bacterial host which had the transferred synthesized gene Itakura et al, Science 198, 1056, 1977 .More recently, reports have appeared that human growth hormone has been produced in bacteria transformed with transferred genetic material comprising the appropriate gene.This latter gene was prepared by copying to DNA part from isolated mRNA from human pituitary and synthesizing the remaining part, then joining the two Goedell et al,Nature 281, 544, 1979 . In the pancreas of animals, preproinsulin Chan, S.J. and Steiner, D.F., Proc. Nat. Acad. Sci. 73, 1964, 1976 is synthesized as the precursor of insulin.The general structure of proinsulin Formula 1 isNH2 B chain C peptide A chain COOH it is converted to insulin by the action of peptidases in the pancrearic islet tissue which removed the C peptide by cleavage at the positions of the two arrows shown in Formula 1. TheB chain and A chain of insulin are held together by two disulfide cross linkages. Using a biological method, Ullrich et al, Science 196, 1313, 1977 and Villa Komoroff, et al, Proc. Nat. Acad. Sci. 75, 3727, 1978 succeeded in cloning the coding region of rat proinsulin I. However, the cloned gene included extraneous sequences and production of biologically active insulin was not achieved. Using a chemical method, Crea et al Proc. Nat. Acad. Sci. 75, 5765, 1978 synthesized, and Goeddel et al Proc. Nat. Acad.Sci. 76, 106, 1979 cloned, an insulin A chain gene and aB chain gene. The codons selected for these synthetic genes were arbitrary and quite different from the natural humanDNA sequence. The construction of these genes was tedious.On culturing, the bacteria.produced an insulin A chain protein and B chain protein which were separately treated to remove the extraneous B galctosidase and methionine.The production of A chain and B chain proteins was rather poor. In United States Patent Application Serial Formula 1. The structure of human proinsulin. B chain, amino acids 1 30 C peptide, amino acids 31 65 A chain, amino acids 66 86. EMI5.1 No. 843,422, filed October 19, 1977, by R. Wu, C.P. Bahl and S.A. Narang, adaptor molecules were described for attachment to the ends of DNA sequences for joining to cloning vehicles or other DNA. These adaptors compriseDNA oligonucleotide sequences having particular nucleotide segments which are recognition sites for restriction endonucleases. These adaptors can be used to provide an enzyme recognition site on a duplex DNA sequence or to change from one type of site to another. Summary of the Invention Adaptor oligodeoxynucleotide molecules are provided for facilitating insertion of DNA information sequences into ,a cloning vehicle and transcription and translation of said sequences in a host cell each adaptor comprising a a recognition site for a restriction endonuclease, and b a signal for protein synthesis, selected from a start signal and a stop signal, said start signal being doam.stream of said recognition site and said stop signal being upstream of said recognition site. In each start adaptor, the start signal s will always be located downstream of the restriction endonuclease recognition site in each stop adaptor, the stop signal s will always be located upstream of the recognition site. Each start or stop signal is one codon base triplet long, but more than one codon may be used together to reinforce the signal. The start or stop codons are selected and positioned to cause the desired DNA information sequence to be read or translated correctly and exclusively in every case thus to assure a high yield of mRNA and ultimate protein in the host cell. In many cases, start signal adaptor molecules will need to include at least one additional base between each start signal and the recognition site to correct the frame shift of a DNA information sequence with inappropriate reading frame. The ATG codon is the basic start signal although the GTG codon may be used instead. The stop signal is one or more codon s selected to ensure that the translation in the host cell ceases at precisely the right location to give the correct protein without subsequent modifications being required .Codons found to be particularly suitable are at least one from the group TGA, TAA and TAG. The adaptors are preferably formed and used as separate molecules Bahl et al, Gene 1. 81, 1976 , but in some cases, may be built up on the end of a DNA information sequence Marians, Wu et al, Nature 263, 744, 1976 . Many of those separate adaptor molecules which are to be attached to a DNA information sequence, preferably will be prepared as short duplex DNA sequences including the recognition site and signal, the signal being at both ends of the duplex so that the complete adaptor molecules have rotational symmetry along a two fold axis X and Y . In other words, these symmetrical adaptors areDNA duplexes having duplicate signals at each end, each complementary nucleotide strand having complete palindromic symmetry to its opposite strand, with both halves of the duplex being identical on 1800 rotation of one half. Either end of the adaptor can attach to the DNA with increased efficiency see example with insulin A chain below .Such start signal adaptors can include additional nucleotide bases between each start signal and the central recognition site, to correct the frame shift of a DNA information sequence with inappropriate reading frame. Synthetic genes coding for the A chain andB chain of human type insulin have been prepared, provided with selected start and stop adaptors as described above, and inserted into a replicable cloning vehicle. This vehicle or vector is extrachromosomal DNA which can enter a cell and replicate itself, such as a plasmid. The synthetic gene cloning vehicle hybrid has been inserted into a host cell, and the transformed host cell progeny was shown to contain the exact input insulin A chain or B chain gene byDNA sequence analysis.Detailed Description and Preferred ESbodiments Typical adaptors having. start or stop signals can be depicted respectively as follows in duplex form Formula 2aEMI8.1 tb SEP start tb 5 SEP CAT SEP recognition SEP ATG tb SEP 3 SEP GTA SEP site SEP TAC tb SEP Formula SEP 2b tb SEP stoo tb SEP 5 SEP TGA SEP recognition SEP TCA tb SEP 3 SEP ACT SEP site SEP AGT tb SEP Formula SEP 2c tb SEP stop tb SEP 5 SEP TAA SEP recognition SEP TTA tb SEP 3 SEP ATT SEP AAT tb Formula 2dEMI9.1 tb SEP stop tb SEP 5 SEP TAG SEP recognition SEP CTA tb 3 SEP ATC SEP site SEP GAT tb These duplexes are seen to have rotational symmetry so that either end can attach to the genetic information sequence without side product formation. The recognition sites themselves have this rotational symmetry. A typical symmetrical duplex adaptor having a double stop signal and a Bam HI site is shown in Formula 2e Formula 2e 5 TGA TAG GGATCC CTA TCA 3 ACT ATC CCTAGG GAT AGT Bam HI site The adaptors can have from about 8 to about 20 or more nucleotide base pairs or incomplete pairs in their structure. The symmetrical adaptors will require an even number. The enzyme recognition sites can be selected from those known for suitable restriction endonucleases over 100 are now known . In the examples below, we have used as the recognition site the boxed in region theBam HI site GGATCC and the Eco RI site GAATTC CCTAGG CTTAAGThe part of this sequence which serves as the protruding cohesive end after cleavage, is 5 GATC for Bam HI, and 5 AATT for Eco RI the complete recognition sequence less the two end bases . The cohesive end portions of the recognition sequences of other restriction endonucleases are derived similarly depending on the digestion cleavage location.For example, for Hind III the protruding cohesive sequence would be AGCT for Xba I, CTAG and forAva II, GAC or GTC. Other restriction endonucleases and their recognition sequences are described in, for example, the article by H.O. Smitk in Science, Vol. 205, p. 455, 3 August 1979 see Table 1, p, 458 , and by R.J. Roberts, in Gene, Vol. 4, p. 183, 1978. Preferably the enzyme recognition sequence duplex has a cleavage pattern that, on digestion, leaves a protruding part of at least 3 bases, and more preferably 4. When used herein, recognition site is meant to include the partial site which is recognized and adhered to by the complementary portion. The DNA information sequence may be a duplexDNA which can be isolated from a natural source such as from microorgansims, plant cells or animal cells or it may be synthesized chemically or enzymatically. It will code for some desired gene derived product. In some cases, the genetic information sequence to be transferred may have a faulty frame shift.By attaching one or more base pairs between the start signal and the enzyme recognition sequence on the adaptor molecules, the reading frame of the triplet code can be corrected by means of the adaptors. For example, the reading frame of the non overlapping triplet genetic code as it follows through the cloning vehicle and adaptor sequence may end up at the start signal, e.g. as inFormula 3a. Formula 3a startEMI10.1 tb 5 SEP ...C1ATG SEP IDNA SEP duplex tb 3 SEP ... SEP TAA SEP GiTA SEP C tb This will cause the DNA information duplex to be read incorrectly. By inserting one or two nucleotide pairsX in the adaptor between the start signal and the y recognition site the reading frame can be shifted as shown in Formula 3b Formula 3b startEMI11.1 tb SEP ATT SEP CiXX SEP aN iS tb DNA SEP DNA SEP duplex tb ... SEP TAA SEP G1YY SEP TAC tb In some cases it may be desirable to insert one complete codon plus one, or plus two nucleotide pairs i.e. 4 or 5 nucleotide pairs . The X and Y can be any nucleotides with bacterial host cells T and A usually are preferred. These inserts will enable the DNA to be read correctly.In the case where the rotationally symmetrical adaptor is used, corresponding base pairs should be inserted on both sides of the enzyme recognition site to maintain the twofold rotational symmetry of the adaptor molecule. The advantage of such symmetrical adaptors is to assure the correct joining to DNA in every case and thus give good yields of the desired adapted DNA sequence where full duplex is attached to DNA duplex .An example of such a symmetrical start signal adaptor is given in the following formula 3c Formula 3c Eco RI site start 5 C A T A A G A A T T C T T A T G 3 G TA T T C T T A A G A A T A C start These adaptor sequences have been prepared by an improved phosphotriester method as described inUnited States Patent Application 843,422, referred to above, and including techniques as described in NucleicAcid Res. 7, 2199,.1979, and in the Hsiung et al andNarang et al references mentioned above. An outline of the synthesis of a 12 nucleotide long start signal adaptor is given below for formula 5a and for its digested formCleft side of formula 3d . The duplex adaptors can be prepared, then joined to each end of the gene or DNA duplex to be cloned and digested to give the protruding cohesive ends.In some cases particularly where the gene is being synthesized , steps can be designed to yield the protruding 5 recognition site and signal without digesting the full adaptor duplex, as illustrated below for the insulin Chain. When attaching the first adaptor duplex to the gene duplex as by DNA ligase , it has been found desirable, in order to minimize side products, to make the 51 terminus at the opposite end of the gene 5 OH. After joining this first adaptor, the 5 OH end is made 5 OP by using ATP and polynucleotide kinase, and the second adaptor joined. When the two adaptor duplexes are joined, the product is digested with the appropriate restriction enzymes to produce the cohesive ends. The resulting digested adapted DNA sequence or gene will have the structure 3d for example, where the Eco RI and Bam HI enzyme sites were used with the start and stop adaptors respectively . X and Y are the optional nucleotide inserts to correct the reading frame and Z is 0, 1 or 2 or 4 or 5 . Formula 3dEMI13.1 tb SEP RI SEP site SEP start SEP stop tb 5 SEP A SEP A SEP T SEP TC SEP XATG SEP DNA SEP duplex SEP T SEP G SEP A SEP C SEP 3 SEP tb 3 SEP G SEP SEP T SEP A SEP C SEP SEP seguence SEP A SEP C SEP T SEP C SEP C SEP T SEP A SEP G SEP 5 tb SEP BamHI SEP site tb The adapted genetic information sequence having protruding cohesive ends which are part of the recognition site and are recognized by the complementary part of the site , is now ready for insertion into a replicable cloning vehicle having complementary sites for cohesion.Suitable cloning vehicles may be plasmids, plasmid fragments, viruses, viral DNA fragments, and other cellular DNA pieces usually at least about 3000 nucleotides long . The adapted DNA molecule gene and the cloning vehicle which has been treated with appropriate endonuclease, can be joined by use of a polynucleotide ligase such as T4 ligase. Preferably the cloning vehicle DNA will include a transcription promoter sequence Such as the lac promoter or tryptophan promoter placed upstream of the inserted gene. A wn Insulin Gene Of particular concern are genes coding for human insulin A and B chains and these will be described in detail as preferred embodiments of the invention. The amino acid sequence of human insulin A chain is shown in formula 4a upper line, from amino acid number 66 to 86.This sequence is identical to that of rat insulin A chain except that amino acid no. 69 is aspartic acid in rat insulin. Ullrich et al cited above described the codon sequence of mRNA and DNA for rat insulin A chain. Drawing in part from this sequence and from other knowledge, and using instead the codon GAG for glutamic acid, a mRNA and partial duplex DNA sequence was put together as shown in formula 4a and 4b. This sequence we selected is much closer to the natural human DNA sequence than that selected byCrea et al, mentioned above. We synthesized the DNA. H strand of 43 nucleotides and separately the DNA L strand of 36 nucleotides as in formula 4a using the phosphotriester method as described previously. The two strands were partially complementary and formed the partial duplex as shown.To complete the synthesis of the 63 nucleotide long duplex DNA, we made use of primer extension repair synthesis techniques to complete the full duplex DNA gene. These repair synthesis techniques are described by R. Wu et al, in Methods in Cancer Research 12, 87, 1976. The overlapping of 16 nucleotides between the H and L strands was sufficient to allow efficient repair synthesis by AMV reverse transcriptase at 370C, or by E. coli DNA polymerase at 5 150C.In this way, we reduced the rather time consuming effort required for the chemical synthesis of the entire two strands. The sequence of the resulting DNA was analyzed in Formula 4a Amino acid sequence of human insulin A chain and the corresponding mRNA and DNA sequence. The designation of the H and L strands of DNA are arbitrary.66 70 76 80 86Amino acids A chain Gly.Ile.Val.Glu.Gln.Cys.Cys.Thr.Ser.Ile.Cys.Ser.Leu.Tyr.Gln.Leu.Glu.Asn.Tyr.Cys.Asn A chain human insulin mRNA 5 GGC.AUU.GUG.GAG.CAG.UGC.UGC.ACC.AGC.AUC.UGC.UCC.CUC.UAC.CAA.CUG.GAG.AAC.UAC.UGC.AAC 3 DNA H 5 GGC.ATT.GTG.GAG.CAG.TGC.TGC.ACC.AGC.ATC.TGC.TCC.CTC.TAC.C 3 5 10 15 20 25 30 35 40DNA L 3 TAG.ACG.AGG.GAG.ATG.GTT.GAC.CTC.TTG.ATG.ACG.TTG. 5 36 1 Formula 4b Construction of human insulin A chain gene. The roman numerals indicate the specific oligodeoxynucleotide fragments synthesized chemically. EMI16.1 I SEP II SEP III SEP VIII tb DNA SEP H SEP PGGC.ATT.GTG.GAG.CAG.TGC.TGC.ACC.AGC.ATC.TGC.TCC.CTC.TAC.C SEP G.GAG.AAC.TAC tb 5 SEP 5 SEP 10 SEP 15 SEP 20 SEP 25 SEP 30 SEP 35 SEP 43 tb 36 SEP 1 tb DNA SEP L SEP 3 SEP TC.GTC.ACG.ACG.T. SEP TAG.ACG.AGG.GAG.ATG.GTT.GAC.CTC.TTG.ATG.ACG.TTG OH tb 5 tb VII SEP IV SEP V SEP VI tb part and confirmed by the chemical method of Maxam andGilbert Proc. Nat. Acad. Sci. 74, 560, 1977 . This human A chain gene differs from that described by Crea et al cited above by having some different codons which are closer to the natural codons used for human insulin gene and having no extragenous DNA sequences so that this was synthesized gene can be joined directly to insulin chain B C gene to construct a proinsulin gene chain B C A with no improper codons intervening. In constructing the synthetic human insulinA chain gene, each of the chemically synthesized oligodeoxynucleotides fragment I through VIII, formula 4b was phosphorylated at its 5 ends with 32p. The phosphorylation reaction for each oligodeoxynucleotide 2 nmoles includes 50 mM glycine NaOH pH 9 5 , 10 mM MgClp, 10 mM dithiothreitol, 5 nmoles of Y32P ATP of specific activity of around 20 mCi pmole, and 3 units of T4 polynucleotide kinase in a final volume of 50 l. The reaction was caried out at 37 C for 2 hours during this time from 50 to 70 percent of the oligodeoxynucleotide was phosphorylated. The desired product was purified by polyacrylamide gel electrophoresis 15 gel, under denaturing conditions . The 43 nucleotide long upper strand see formula 4a, H strand was assembled by annealing 500 pmoles each of 5 phosphorylated oligonucleotide fragments I, II,III, IV and VII in 15 ul at 650 and slowly cooled to OOC. Concentrated ligase buffer was added to give the following final concentrations 50 mM Tris HCl pH 7.6 , 10 mM MgCl2, 10 mM dithiothreitol, 100 uM ATP. T4 polynucleotide ligase 0.4 unit was added and the mixture incubated at 120C for 24 hours. The ligation product was purified on a 15 polyacrylamide gel denaturing condition . The desired 43 long oligodeoxynucleotide migrated to about 46 that of the bromophenol blue dye marker. The yield of purified 43 long product after elution from the gel was around 50 pmoles. The 36 nucleotide long lower strand formula 4a,L strand was assembled in a similar manner using phosphorylated oligonucleotide fragments III, IV, V, VIII and non phosphorylated fragment VI. The 63 nucleotide long duplex 63 mer A chain insulin gene was constructed by annealing about 40 pmoles each of the 51 32P 43 long and the 5 OH 36 long oligodeoxynucleotides. Repair synthesis was carried out, to fill in the single stranded regions, in 50 mM Tris HCl pH 8.3 , 50 mM KCl, 10 mM MgC12, 10 mM dithiothrp.itol, 100 pg ml bovine serum albumin, 100 uM each of dGTP, dATP, dTTP and o 32P dCTP specific activity around 4 mCi pmole , and 9 units of avian myeloblastosis virus reverse transcriptase in a final volume of 20 ul. Incubation was at 370C for 3 hours. The desired products were purified by polyacrylamide gel electrophoresis and two strands of the 63 mer were separated migrated to about 72 that of the xylene cyanol dye marker . DNA sequence analysis of the eluted 63 long products pilot scale with high specific activity 3 2P dCTP showed the correct sequence for the A chain insulin gene. The cloning of this insulin A chain gene was controlled by adding both a start signal and a stop signal for protein synthesis in the adaptors described above to the two ends of the synthetic gene. Not only can this adapted DNA be joined to the cloning vehicle but after cloning and transcription, the initiation and termination of insulin protein A chain synthesis can be achieved. The dodecadeoxynucleotide adaptor duplex containing a start signal downstream of the restriction endonuclease recognition site was synthesized using the phosphotriester methpds to have the structure Formula 5a EMI19.1 tb SEP Eco SEP RI SEP site SEP start tb H SEP 5 SEP C SEP A SEP T SEP G SEP A SEP A SEP T SEP T SEP C SEP A SEP T SEP G tb L SEP 3 SEP G SEP T SEP A SEP C SEP T SEP T SEP A SEP A SEP G SEP T SEP A SEP C tb Similarly, a stop signal adaptor was synthesized to have the structure Formula 5b EMI19.2 tb SEP stop tb H SEP 5 SEP T SEP G SEP A SEP G SEP G SEP A SEP T SEP C SEP C SEP T SEP C SEP A tb L SEP 3 SEP A SEP C SEP T SEP C SEP C SEP T SEP A SEP G SEP G SEP A SEP G SEP T SEP tb SEP Bam SEP HI SEP site SEP The 5 terminus at the tail end of the A chain gene was kept in its 5 OH form and the start signal adaptor 5a joined to the head end using Tq polynucleotide ligase by blunt end ligation as follows.The reaction mixture contained 50 mM Tris HCl pH 7.6 , 10 me MgCl2, 10 mM dithiothreitol, 100 uM ATP, 200 pmoles of the selfcomplementary 12 mer start signal adaptor 5a , 25 pmoles of sa mer duplex A chain gene, and 0.25 units of T4 polynucleotide kinase in a final volume of 15 ul. Incubation was at 12 0C for 40 hours. Small scale gel electrophoresis of an aliquot of the reaction mixture with and without Eco RI enzyme digestion showed that about 50 of the 63 mer duplex was ligated to a start signal adaptor. TheDNA in the main portion of the reaction mixture was extracted with chloroform isoamyl alcohol 24 1 v v and precipitated with 2 volumes of ethanol.The tail end 5 OH was then converted to 5 OP using ATP32 and polynucleotide kinase as before . Next.the.stop signal adaptor 5b was ligated to the tail end of the A chain gene to give the adapted gene. The adapted gene was digested successively withEco RI and Bam HI restriction enzymes to produce the respective cohesive ends and to remove any additional start or stop signals ligated to the gene . A cloning vehicle was prepared, by cutting out and purifying on an agarose gel, a 4000 nucleotide long fragment from pBR 322 plasmid with both Eco RI and Bam HI enzymes. This fragment 0.5 pg had cohesive ends which were then ligated by DNA ligase as before to the complementary part of the enzyme recognition sites on the adapted insulin A chain gene about 1 pmole . This gene cloning vehicle hybrid one half of the sample was used to transform cells of E. coli strain 5436 . This strain and the containment facilities used conformed to current recombinant DNA regulations. Onehundred and ten ampicillin trademark resistant and tetracycline sensitive transformants were isolated. On being hybridized with 32P labelled short synthetic insulinA chain DNA fragments 3Q 35 bases long by colony hybridization Grunstein Hogness, Proc. Nat. Acad. Sci.72 , 3961, 1975 22 positive clones were obtained. DirectDNA sequence analysis Maxam and Gilbert method was carried out on several of the positive clones to confirm the presence of the insulin A chain gene in the hybrid plasmidDNA. The DNA fragment excised, by digestion with bothEco RI and Bam HI enzymes, from three clones showed by direct DNA sequence analysis the exact nucleotide sequence as synthesized. The cloned insulin A chain gene on pBR 322 plasmid was prepared in increased quantities by growing E.coli cells carrying this hybrid plasmid in M9 medium until the optical density of the cell culture equalled about 1 at 600 nm. Chloramphenicol was added to give a final concentration of 150 mg L. The cells were shaken at 370C for another 12 to 16 hours. The hybrid plasmid DNA in the cells was isolated according to standard procedure such as that of Arians et al, Nature 263, 744, 1976 . The cloned insulin A chain gene was next excised by digestion of the cloned hybrid plasmid DNA withEco RI and Bam HI endonucleases. The Bam HI end was converted to an Eco RI and by using an Eco RI Bam HI conversion adaptor of the type reported by Bahl et al Biochem. Biophys. Res. Commun. 81, 695, 1978 except that the double stranded portion of the adaptor was increased from 6 long to 8 or 10 long to increase the efficiency of ligation.The conversion adaptor has the structure Formula Sc BanHI site 5 GATC CGA CTG TTA G 3 GCT GAC AAT C TTAA EcoRI siteThe resulting adapted insulin A chain gene carrying anEco RI site at each end, with the following type of structure Formula 5dEMI22.1 tb SEP EcoRI tb SEP site SEP start SEP stop tb 5 SEP AATTC SEP ATG SEP ATG SEP DNA SEP duplex SEP TGA SEP G SEP GATC SEP CGA SEP CTG SEP TTA SEP G SEP 3 tb 3 SEP G SEP TAC SEP SEP sequence SEP ACT SEP C SEP CTAG SEP GCT SEP GAC SEP AAT SEP CTTAA SEP 5 SEP tb SEP BamHI SEP EcoRI SEP tb SEP site SEP site tb can be ligated to a plasmid such as pBGPl20 Polisky et al,Proc. Nat. Acad. Sci. 73, 3900, 1976 , or a high copy number plasmid such as pKN402 of Uhlin et al, Gene 6, 91, 1979 carrying a strong promoter such as the lactose promoter, the strain being constructed in our laboratory for expression.This reconstructed hybrid plasmid DNA can be ligated at the Eco RI sites and used to transform E.coli cells to direct the synthesis of large amounts of the human insulin A chain protein. The flow of information fromDNA to protein is by transcribing the DNA sequence into a complementary messenger RNA mRNA sequence and then translating the mRNA sequence using the 3 letter genetic code in the synthesis of a protein. Still other plasmids or replicable DNA sequences would be operative as cloning vehicles. For good yields of such transferred genederived products, the culture medium for the host cells should be one that favors the reproduction of the insertedDNA hybrid, as well as transcription, translation and expression of desired product. Such culture media can be of the type designated M9 as described in J. Bacteriol. 119, 450, 1973, L. Katz and D.R. Helinski. These media contain nutrients that favor the replication of plasmids and other replicable cloning vehicles as well as protein synthesis . Further such media are described in Experiments in Molecular Genetics byJ.M. Miller, page 431, 1972, Cold Spring Harbour Lab., N.Y.B Chain Insulin Gene The chemical synthesis of.a gene coding for the B chain of human insulin is outlined in formula 6, the roman numerals indicating the portions as synthesized by the phosphotriester method H.M. Hsiung, S.A. Narang et al,Nucleic Acids Res. 6, 1371, 1979 and their sequence of joining by DNA ligase. The portions can be joined by DNA ligase, and the DNA partial duplex completed by repair synthesis to form the full duplex, in a manner similar to the assembly of the insulin A chain fragments. Both a start signal and a stop signal adaptor can be added to the two ends of the synthetic gene as in the construction of the insulin A chain gene.The synthesized B chain gene, with protruding cohesive ends as in formula 6, can be ligated to the same 4000 nucleotide long duplex plasmidDNA as for the A chain and the hybrid used to transformE. coli strain 5346 cells as before. An adapted B chain gene based on the structure as in formula 6, was constructed by preparing Formula 6. Sequence of fragments I to VII and X to XVI which were completed to constitute human like insulin B chain DNA coding sequence.EMI24.1 tb SEP I SEP II SEP III tb 5 SEP TTT SEP GTC SEP AAT SEP CAG SEP CAC SEP CTT SEP TGT SEP GGT SEP TCT SEP CAC SEP CTG SEP GTG SEP GAG tb 3 SEP TA SEP GTC SEP GTG SEP GAA SEP ACA SEP CCA SEP AGA SEP GTG SEP GAC SEP CAC SEP CTC tb SEP X SEP XI SEP XII tb SEP IV SEP V SEP VI SEP VII tb 5 SEP GCT SEP CTG SEP TAC SEP CTG SEP GTG SEP TGT SEP GGG SEP GAA SEP CGT SEP GGT SEP TTC SEP TTC SEP TAC tb 3 SEP CGA SEP GAC SEP ATG SEP GAC SEP CAC SEP ACA SEP CCC SEP CTT SEP GCA SEP CCA SEP AAG SEP AAG SEP ATG tb SEP XIII SEP XIV SEP XV tb 5 SEP ACA SEP CC tb 3 SEP TGT SEP GGG SEP TTC SEP TGG tb SEP XVI 1 SEP tb segments and joining in sequence as shown by roman numerals, ligated to the plasmid, and transferred into host cells as described above. Transformed cells and the presence of the B chain gene therein, were identified and confirmed as before. Purified insulin A chain protein, isolated from E. coli carrying the A chain gene, can be converted to the S sulfonate form and combined in vitro with theSH form of similarly isolated B chain protein, or conversely, the B chain in the S sulfonate form can be combined in vitro with the SH form of similarly isolated A chain protein, according to the method of Xatsoyannis et al, in J. Am. Chem. Soc. 95, 6427, 1973, to form human type insulin. The host cells any cell that allows theDNA to enter, to replicate and to produce gene directed product into which the adapted gene cloning vehicle hybrid is transferred can be chosen from among various other bacteria, such as Bacillus subtilis various fungi, e.g.yeasts, particularly baker s or brewer s yeasts or plant or animal cells. The transformed E. coli cells carrying the insulin A chain and B chain gene are maintained at theBiochemistry Department, Cornell University, Ithaca, N.Y., and are available upon request subject only to present or future Federal Government regulations on the handling of such materials. The transformed E. coli cells carry ing the insulinA chain and B chain genes have also been deposited with the americanType Culture Collection under ATCC nos. 31850 and 31851 respectively.