# Expression of polypeptides in yeast.

## Abstract
In order to make a yeast strain capable of expressing a desired polypeptide e.g. leukocyte interferon D which it does not normally make or use, it is transformed with an expression vector. This is obtained by producing a transfer vector which has respective origins of replication and phenotypic selection genes derived from both bacteria and yeast, and inserting into it DNA fragments i coding for the desired polypeptide and ii comprising a yeast derived promoter. The insertion is effected so that start and stop signals are provided, and these and the promoter can control expression of the polypeptide, and so that the replication origins and phenotypic selection genes of the transfer vector are still functional. Thus the expression vector and its precursors can replicate, and their presence can be recog nised, both in yeast and bacteria. Bacteria are used for amplification. The yeast derived phenotypic selection gene may complement a mutation carried by the strain to be transformed.

## Claims
CLA111S 1. A method of forming a transformant of a given yeast strain, which transformant is capable of expressing a biocompetent polypeptide ordinarily exogenous to yeast and not required for growth of said strain, which comprises a providing a D tA transfer vector having bacterial and yeast origins of replication and genes for phenotypic selection of both bacterial and yeast moieties transformed with said genes b providing a DNA fragment comprising a structural gene encoding said polypeptide c providing a DNA fragment comprising a yeast promoter genetically distinct from said structural gene d inserting the fragments of steps b and c into said transfer vector together with appropriately positioned translation start and stop signals for said structural gene to form an expression vector in which said structural gene is under the control of said promoter, while maintaining said origins of replications and genes for phenotypic selection and e transforming said strain with the resulting expression vector. 2. The method of Claim 1 which additionally comprises provision of a transcription terminator between the 3 end of said coding strand and said origins of replication in the direction in which the structural gene is transcribed. 3. The method of Claim 1 wherein the DNA fragment of step b also comprises translation start and stop codons respectively adjacent to the 5 and 3 ends of the coding strand of said structural gene. 4. The nethod of Claim 3 wherein the bacterial selection phenotype encodes antibiotic resistance and wherein the yeast selection phenotype complements a mutation carried by the yeast strain to be trjnsformed. 5. The method of Claim 4 wherein the yeast stra n transfcrmed is strain RH218. 6. The method of Claim 3 wherein the amino acid sequence of said polypeptide is selected to correspond to the amino acid sequences of polypeptides selected from the group consisting of the normal and hybrid human Interferons, human proinsulin, the A and B chains of human insulin, human growth hormone, somatostatin and thymosin alpha 1. 7. A DNA expression vector capable, in a transformant strain of yeast, of expressing a biologically competent polypeptide ordinarily exogenous to yeast under the control of a genetically distinct yeast promoter, said polypeptide being unrequired for growth of the transformant. 8. A DNA expression vector according to Claim 7 having a transcription terminator between the 3 end of the structural gene encoding said polypeptide and the origin s of replication of said vector, in the direction in which said gene is transcribed. 9. A DiA expression vector according to Claim 8 which comprises one or more genes for phenotypic selection of yeast transformants. 10. The Dt expression vector of Claim 9 which additionally comprises one or more genes for phenotypic selection in bacteria. 11. The DNA expression vector of Claim 10 wherein the phenotypic basis for selection in bacteria is a gene encoding antibiotic resistance and wherein the phenotype for selection in yeast complements a mutation carried by a yeast strain suitable for transformation with the expression vector. 12. The expression vector of claim 8 wherein the amino acid sequence of said polypeptide is selected to correspond to the amino acid sequences of polypeptides selected from the group consisting of normal and hybrid human interferons, human proinsulin, the A and B chains of human insulin, human growth hormone, somatostatin and thymosin alpha 1. 13. A yeast strain transformed with a DNA expression vector according to any of claims 7 to 12. 14. A method of producing a biocanpetent polypeptide comprising culturing a yeast strain according to claim .13 or a yeast strain produced by the method of any one of claims 1 to 6, and isolating fran the culture system a biocompetent polypeptide ordinarily exogenous to the yeast. CLAIMS TK 1. A method of forming a transformant of a given yeast strain, which transformant is capable of expressing d biocompetent polypeptide ordinarily exogenous to yeast and not required for growth of said strain, which comprises a providing a D tA transfer vector having bacterial and yeast origins of replication and genes for phenotypic selection of both bacterial and yeast moieties transformed with said genes b providing a DNA fragment comprising a structural gene encoding said polypeptide c providing a DNA fragment comprising a yeast promoter genetically distinct from said structural gene d inserting the fragments of steps b and c into said transfer vector together with appropriately positioned translation start and stop signals for said structural gene to form an expression vector in which said structural gene is under the control of said promoter, while maintaining said origins of replications and genes for phenotypic selection and e transforming said strain with the resulting expression vector. 2. The method of Claim 1 which additionally comprises provision of a transcription terminator between the 3 end of said coding strand and said origins of replication in the direction in which the structural gene is transcribed. 3. The method of Claim 1 wherein the DNA fragment of step b also comprises translation start and stop codons respectively adjacent to the 5 and 3 ends of the coding strand of said structural gene. w. The method of Claim 3 wherein the bacterial selection phnnotype encodes antibiotic resistance and 81herein the yeast selection phenotype colnplcments a mutation carried by the yeast strain to be transformed. 5. The method of Claim 4 wherein the yeast stra n transfcrmed is strain RH218. 6. Tile method of Claim 3 wherein the amino acid sequence of said polypeptide is selected to correspond to the amino acid sequences of polypeptides selected from the group consisting of the normal and hybrid human interferons, human proinsulin, the A and a chains of human insulin, human growth hormone, somatostatin and thymosin alpha 1. 7. A method of preparing a DNA expression vector capable, in a transformant strain of yeast, of expressing a biologically competent polypeptide ordinarily exogenous to yeast under the control of a genetically distinct yeast promoter, said polypeptide being unrequired for growth of the transformant, which comprises steps a to d as defined in claim 1, optionally with the further steps or features defined in any of claims 2 to 6. 8. A method of preparing a biocompetent polypeptide comprising forming a transformant yeast capable of expressing said polypeptide by a method according to any of claims 1 to 6, culturing said yeast, and isolating the polypeptide from the culture system. 9. The use in forming a transformant of a given yeast strain of a DNA expression vector capable of expressing in the transformant a biologically competent polypeptide ordinarily exogenous to yeast under the control of a genetically distinct yeast promoter, said polypeptide being unrequired for growth of the transformant.

## Description
EXPRESSION OF POLYPEPTIDES IN YEAST FIELD OF THE INVENTION This invention relates to the production, via recombinant DNA technology, of useful polypeptides in Saccharomyces cerevisiae yeast , and to the means and methods of such production. BACKGROUND OF THE INVENTION The publications and other materials referred to herein to illuminate the background of the invention and, in particular cases, to provide additional detail respecting its practice are incorporated herein by reference, and, for convenience, are numerically referenced and grouped in the appended bibliography.Recombinant DNA Technoloqy With the advent of recombinant DNA technology, the controlled microbial production of an enormous variety of useful polypeptides has become possible. Already in hand are bacteria modified by this technology to permit the production of such polypeptide products as sornatostatin 1 , the component A and B chains of human insulin 1 , human proinsulin 2 , thyinosin alpha 1 3 , human growth hormone 4 , human 5 and hybrid 6 leukocyte and fibroblast 7 interferons, as well as a number of other products.The continued application of techniques already in hand is expected in the future to permit bacterial production of a host of other useful polypeptide products, including other hormones, enzymes, immunogens useful in the preparation of vaccines, inmune modulators and antibodies for diagnostic and drug targeting applications. The workhorse of recombinant DNA technology is the plasinid, a non chromosomal loop of double stranded DNA found in bacteria and other microbes, oftentimes in multiple copies per cell. Included in the information encoded in the plasmid DNA is that required to reproduce the plasmid in daughter cells i.e., an origin of replication and, ordinarily one or more selection characteristics such as, in the case of bacteria, resistance to antibiotics, which permit clones of the host cell containing the plasmid of interest to be recognized and preferentially grown under selective conditions. The utility of plasmids lies in the fact that they can be specifically cleaved by one or another restriction endonuclease or restriction enzyme , each of which recognizes a different site on the plasmid DNA.Thereafter heterologous genes or gene fragments may be inserted into the plasmid by endwise joining at the cleavage site or at reconstructed ends adjacent to the cleavage site.DNA recombination is performed outside the cell, but the resulting recombinant plasmid can be introduced into it by a process known as transformation and large quantities of the heterologous gene containiny recombinant plasmid are then obtained by growing the transformant.Moreover, where the gene is properly inserted with reference to portions of the plasmid which govern the transcription and translation of the encoded DNA information, the resulting expression vehicle can be used to actually produce the polypeptide sequence for which the inserted gene codes, a process referred to as expression. Expression is initiated in a region known as the promoter which is recognized by and bound by RNA polyinerase. The polymerase travels along the ARIA, transcribing the information contained in the coding strand from its 5 to 3 end into messenger RNA which is in turn translated into a polypeptide having the amino acid sequence for which the DNA codes. Each amino acid is encoded by a nucleotide triplet or codon within what may for present purposes be referred to as the structural gene i.e., that part which encodes the amino acid sequence of the expressed product.After binding to the promoter, the RNA polymerase, transcribes a 5 leader region of niessenger RNA, then a translation initiation or start signal ordinarily ATG, which in the resulting messenger RNA becomes AUG , then the nucleotide codons within the structural gene itself.So called stop codons are transcribed at the end of the structural gene whereafter the polymerase may form an additional sequence of messengerRNA which, because of the presence of the stop signal, will remain untranslated by the ribosomes. Ribosomes bind to the binding site provided on the messenger RNA, and themselves produce the encoded polypeptide, beginning at the translation start signal and ending at tile previously mentioned stop signal. The resulting product may be obtained by lysing the host cell and recovering the product by appropriate purification from other microbial protein or, in particular instances, possibly by purification from the fennentation medium into which the product has been secreted. Plasmids employed in genetic manipulations involved in the construction of a vehicle suitable for the expression of a useful polypeptide product are referred to as DNA transfer vectors, Thus, employing restriction enzymes and associated technology, gene fragments are ordered within the plasmid in in vitro manipulations, then amplified in vivo in the transformant microbes into which the resulting, recombinant plasmid has been transferred . A DNA expression vector comprises not only a structural gene intended for expression but also a promoter and associated controls for effecting expression from the structural gene. Both transfer and expression vectors include origins of replication. Transfer vectors must and expression vectors may also include one or more genes for phenotypic selection of transfonnant colonies. Thus far, the useful products of expression from recombinant genes have fallen into two categories. In the first, a polypeptide having the amino acid sequence uf a desired end product is expressed directly, as in the case of human growth hormone and the interferons referred to above.In the second, the product of expression is a fusion protein which includes not only the amino acid sequence of the desired end product but also one or more additional lengths of superfluous protein so arranged as to permit subsequent and specific cleavage away of the superfluous protein and so as to yield the desired end product. Thus, cyanogen bromide cleavage at methionine residues has yielded somatostatin, thymosin alpha 1 and the component A and B chains of human insulin from fusion proteins enzymatic cleavage at defined residues has yielded beta endorphin 8 . A biocompetent polypeptide , as that term is used herein, refers to a product exhibiting bioactivity akin to that of a polypeptide innately produced within a living organism for a physiological purpose, as well as to intermediates which can be processed into such polypeptides, as by cleavage away of superfluous protein, folding, combination as in the case of the A and B chains of human insulin , etc.Saccharomyces cerevisiae The cells of Saccharomyces cerevisiae, or yeast, are, like those of mammalian organists, eukaryotic in nature as distinguished from the prokaryotic nature of bacteria. With regard to mechanisms for the expression of genetic information, eukaryotes are distinguished from bacteria by 1 chromosomes which are organized in 140 base pair units, each containing two molecules each of histones H2A, H25, H3, and H4. 2 Transcription of the protein encoding gene by the alpha amanitin sensitive RNA polymerase II. 3 Post transcriptional addition of Gppp and polyadenylic acid to the 5 and 3 termini of mRNA molecules. 4 Transport of newly completed mRNA from the nuclei wnere they are transcribed to the cytoplasm where they are translated. 5 Some but not all eukaryotic genes contain intervening sequences introns which make tnein non colinear with the corresponding mature mQÃ¬4A molecule. The initial transcription products of these genes contain the intron sequence which is spliced out subsequently in the formation of a finished mRNA molecule. The nucleotide sequences of all eukaryotic cells are transcribed, processed, and then translated in the context described above. There are reasons to believe that expression of eukaryotic genes may proceed with greater efficiency in yeast than in E. coli because yeast is a eukaryote cell. A number of workers have previously expressed, or attempted to express, foreign genes in yeast transformants. Thus, attempted expression from a fragment comprising both a promoter and structural gene for rabbit globin is reported 9 to have yielded partial mRNA transcripts, seemingly unaccompanied either by translation into protein or maturation intron elimination of the message. A gene coding forDrosophila GAR tranformyias yeast ADEN , an enzyme in the adenine synthesis pathway, is reported to have been expressed under the control of its own promoter 10 . A number of yeast proteins have hitherto been expressed in yeast via recombinant plasmids see, eg., 12 . In the experiments, as in the Ade 8 case earlier discussed, expression occurred under the selective pressure of genetic complementation.Thus, each expression product was required for growth of the host strains employed, mutants whose chromosomal DNA was defective in the structural gene s from which expression occurred. The availability of means for the production in yeast of proteins of choice could provide significant advantages relative to the use of bacteria for the production of polypeptides encoded by recombinant DNA. Yeast has been employed in large scale fermentations for centuries, as compared to the relatively recent advent of large scale. coi fermentation. Presently, yeast can be grown to higher densities than bacteria, and is readily adaptable to continuous fermentation processing. Many critical functions of the organism, e.g., oxidative phosphorylation, are located within organelles, and hence not exposed to the possible deleteriocts effects of the organism s overproduction of foreign proteins. As a eukaryotic organism, yeast may prove capable of glycosylating expression products where important to enhanced bioactivity.Again, it is possible that as eukaryotic organisms, yeast cells will exhibit the same codon preferences as higher organisms, tending toward more efficient production of expression products from mammalian genes or from complementary DNA cDNA obtained by reverse transcription from, e.g., mammalian messenger RNA. Until the present invention, however, attempts to produce biocompetent expression products other than those required for cellular growth have proven largely unsuccessful. BRIEF SUMMARY OF THE INVENTION T le present invention provides DNA expression vectors capable, in transformant strains of yeast, of expressing biologically competent preferably pharmacologically active polypeptides under the control of genetically distinct yeast promoters, the polypeptides being ordinarily exogenous to yeast and other than those required for growth of the transformant. The invention also provides DNA transfer vectors for the transformation of yeast strains with genes encoding biocompetent polypeptides, as well as novel yeast organisms and cultures thereof incorporating such vectors and methods for the formation of the same.The structural genes incorporated in the expression vectors and transformant organisms of the invention are under the control of genetically distinct yeast promoters, i.e., promoters different from those evolutionarily associated with the subject structural genes. The manner in which these and other object and advantages of the invention are obtained will become apparent froin the detailed description which follows, and froin the accompanying drawings in which Figure 1 schematically illustrates the construction of a DnA transfer vector having a single Eco RI restriction site for the subsequent insertion of a yeast promoter and comprising both bacterial and yeast origins of replication and selection phenotypes Figure 2 schematically illustrates the construction of alcohol dehydrogenase promoter fragments for insertion into the transfer vector of Figure 1 Figure 3 illustrates the coding strand sequence and end points 904, 906, etc. of a series of yeast promoter fragments made by digestion with the exonuclease Bal 31 and attachment of EcoRI molecular recombinational linkers. Figure 4 schematically illustrates the insertion of yeast promoter fragments into the transfer vector of Figure 1 and subsequent insertion in two orientations of a structural gene encoding human leukocyte interferon D. In the Figures and throughout, the letters A, T, C and G respectively connote the nucleotides containing the bases adenine, thynine, cytosine and guanine. Only the coding strands of plasmids and gene fragments are depicted. Though obviously not to scale, the representations of plasmids depict the relative position of restriction enzyme cleavage sites EcoRI , Hindlll etc. and other functions such as tetracycline resistance Tcr and ampicillin resistance apt . DESCRIPtIOn OF PREFERRED EM30DI.MENTS Preferred embodiments of the invention are obtained by bringing an exogenous gene under the control of a yeast promoter carried by a plasmid suitable for the transformation of yeast. Essentially any yeast strain suited for the selection of transformants may be employed. in order to achieve direct expression of the desired end product or an intermediate therefor, rather than a fusion comprising portions of the yeast protein whose expression is controlled by tie promoter in wild type strains, the parental plasmid is resected toward the promoter in the direction opposite that of transcription, so as to excise the ATG triplet which initiates translation of mRNA encoding the yeast protein referred to. An ordinarily exogenous gene, with its associated start signal, may then be inserted at the endpoint of the resection, and thus positioned for direct expression under the control of the yeast promoter. This and other aspects of the invention are illustrated in the description of preferred embodiments which follow. METHODSMaterials All DNA restriction and metabolism enzymes were purchased from New England Biolabs except for exonuclease Bal 31 and bacterial alkaline phosphatase, which were obtained from Bethesda ResearchLaboratories. DNA restriction enzyme and metabolic enzymes were used in conditions and buffers described by their respective manufacturers. ATP and the deoxynucleoside triphosphates dATP, dGTP, dCTP and dTTP were purchased from PL Biochemicals. Eco RI, Bam HI, Hind III and Xho I linkers were obtained from Collaborative Research, Inc. a 32P was obtained from New England Nuclear Corp. DtlA Preparation and Transformation Purification of covalently closed circular plasmid ONAs from E. coli 13 and yeast 14 plus the transformation of E. coli 15 was as previously described.Transformation of yeast was as described by Hsiao and Carbon 16 with the exception that 1.2 M Sorbitol was used instead of 1.0 M Sorbitol. E.Coli miniscreens were as described by 17 .Strains and Media E. coli strain JA300 thr leu36 thi thyA trpC1117 hsdM hsdR strR 18 was usd to select for plasmids containing functional trpI gene. Ecoli K 12 strain 294 ATCC no.31446, deposited 28 Oct. 1978 1gwas used for all other bacterial transformation. Yeast strains RH218 having the genotype a trpl qal2 suc2 mal CUPI 20 and GM 3C 2 a, leu 2 3, leu 2 112, trp 1 1, his 4 519, cyc 1 1, cyp 3 1 21 were used for yeast transformations. Yeast strain RH 213 was deposited without restriction in theAmerican Type Culture Collection, ATCC no. 44076 on 8 Dec. 1980. M9 minimal medium with 0.25 percent casamino acids CAA and LB rich medium were as described by Miller 22 with the addition of 20 pg ml ampicillin Sigma after media is autoclaved and cooled. Yeast were grown on the following media YEPD contained 1 percent yeast extract, 2 percent peptone and 2 percent glucose 3 percent Difco agar.YNB CAA contained 6.7 grams of yeast nitrogen base without amino acids YNB Difco , 10 mg of adenine, 10 mg of uracil, 5 grams CAA, 20 grams glucose and 30 grams agar per liter. The selection of ADH promoter active fragments occurred on YEPGE plates containing 3 percent glycerol and 2 percent ethanol substituted for glucose in the YEPD formula.Leucine prototrophy was determined on plates containing 6.7 gms YNB, 20 gms glucose, 50 mgs histidine and 50 mgs trytophan and 30 gms Difco agar per L. Construction and Identification of ADH Promoter Deletions Fig. 2 pY9T6 was digested with Sau3A then run on a preparative 1 percent agarose gel. The 1600 bp fragment containing the ADil promoter region .sas cut from the gel, electroeluted then purified on a diethylamino cellulose DE52, hsshatman column before ethanol precipitation. Fragment DI4A was resuspended in urIA Polymerase I Klenow fragmcnt buffer supplemented with the four deoxyribonucleoside triphosphates in a final concentration of 80 pM. Polymerase I was added and the thirty minute room temperature reaction was terminated by ethanol precipitation of the DNH. An equal molar amount of BamHI and HindllI linker was added to the resuspended Sau3A fragment so thdt each linker was in a 20 1 molar excess to the large DtlA fragment. T4 DNA ligase was added and the 12 hour reaction occured at 12 degrees centigrade. After ethanol precipitation and resuspension In the appropriate buffer, tile DNA was digested with BamHI, then HindIII. The now larger promoter containing fragment was purified away from the unattached linkers by passage through a 10ml sizing column before ethanol precipitation. This DNA fragment was then ligated in pBR322 previously isolated as missing the Hindlll to BaSHI restriction fragment. E. coli strain RR1 was transformed to ampicillin resistance using part of this ligation mix. After quick screen analysis of a number of recombinant plasmids, pJD221 which had the insert with the HindlIl linker added to the end of the fragment closest to the ATG of the ADH structural gene was isolated by plasmid preparation. pJD221 was linearized with HindllI and the resulting fragment than successively treated with exonuclease III and S1 nuclease. The ends of these deleted plasmids were then made blunt using the Klenow fragment ofDNA Polymerase I see procedure above . After ethanol precipitation the ends of the DNA were ligated with XhoI linkers in a 12 hour reaction mixture. After digestion of resulting ligation mix with XhoI, plasmid solution was run in a 0.5 percent preparative agarose gel. DNA bands were cut from the gel, electroluted, then passed through a DE52 column before ethanol precipitation. Linear plasmid was circularized using T4DNA Ligase. The resulting ligation mix was used to transform E. coli strain RR1 to ampicillin resistance. All such colonies were pooled together. The resulting single plasmid pool was cut with XhoI and Bamill1 then run on a preparative 0.7 percent agarose gel. The 1500bp bands containing the ADH promoter region were cut froin the gel, electroeluted then passed through a DE52 column before ethanol precipitation and ligation into the vector pYecylax f1. This plasmid had previously been isolated from an agarose gel as having lost the XhoI to Bag restriction fragment described in the Figure. The resulting ligation was used to transform E. coli strain RRI to ampicillin resistance Colonies were mixed for preparation of a plasnid pool which was then used to transform yeast strain GM 3C 2 to leucine prototrophy.Plasmids were then isolated from leucine prototrophs able to grow on glycerol plates. One plasmid, pACF 301, was found to contain a deletion extending toward the ATa of theADH1 structural gene, leaving intact the first five triplets of the structural gene and the AC of the ACC of Thr6 Fig. 2b . This plasmid was digested with XhoI then treated with exonuclease Ba131 for 15 and 30 seconds two different aliquots . Resulting plasmids were pooled, ethanol precipitated and then treated with DNA Polymerase I reaction described above so that all DNA ends were made blunt. EcoRI linkers were then added to the DNA solution and ligation allowed to proceed for 12 hours. After digestion with EcoRI and Basil, ligation mix was run on a preparative agarose gel.A DNA band about 1500 bp in size was cut from the gel, electroeluted then passed through a sizing column before ethanol precipitation. This DNA was then ligated into the linear pBR322 DNA previously isolated as missing the EcoR1 to BamHI restriction fragment.This ligation mix was used to transform E. coli strain 294 to ampicillin resistance. Plasmids isolated from these colonies are referred to as the pGBn plasmid series. Miniscrcen analysis of 2 number of different recomoinant plasmids from the pGBn plasmid series indicatcd that nine particular plasmids had small Bal 31 generated deletions toward the ADI promoter region through the ATG of the ADH structural gene. All nine plasmids were digested withEcoRI, then end labeled by incubation with a32P dATP and DNA polymerase I conditions as described above . After ethanol precipitation, seven plasmids were digested smith Alul. then electrophoresed on a 20 percent acrylamide urea sequencing gel.32p labelled plasmid DflAs from pGB904 and pGB906 were cut with BamHI then run on a preparative gel. Labelled fragments containing the ADH promoter region were excised from the gel, electroluted, passed through aDE52 column before ethanol precipitation. These two resuspended fragments from plasmids pGB904 and pGB906 were then subjected to theG A and T C sequence specific degradation reactions described by Maxam and Gilbert procedure 11 and 12 respectively 23 . These sequencing reaction products were electrophoresed along with labeled fragments from pGB905, pGB914, pGB917, pGB919 and pGB921 on the thin 20 percent acrylamide sequencing gel described in the sequencing reference .Autoradiography was as described. This procedure allowed the determination of the extent of deletion of ADH promoter region as this region had previously been sequenced using all four Maxam Gilbert sequencing reactions J. Bennetzen, Ph.D Thesis, University ofWashington, 1980 . ExpressionVector Construction Oig.l lOpg of YRp7 24 26 was digested with EcoRi. Resulting sticky DNA ends were made blunt using DNA Polymerase I Klenow fragment . Vector and Insert were run on 1 percent agarose SeaKem gel, cut from the gel, electroeluted and 2X extracted with equal volumes of chloroform and phenol before ethanol precipitation. The resulting blunt end DNA molecules were then ligated together in a final volume of 50 l for 12 hours at 12 C. This ligation mix was then used to transform E. coli strain JA300 to ampicillin resistance and tryptophan prototrophy. Plasmids containing the TRPI gene in both orientations were isolated. pFRW1 had the TRPI gene in the same orientation as YRp7 while pFRW2 had the TRPI gene in tile opposite orientation. 10 g of pFRW1 and 10 g of YRp7 were digested with Hindlil then run in separate lancs on a 1 percent agarose gel. The large Hindlil fragment from the pFRW1 Iane and the small fragment from the YRp7 lane were eluted from the gel, extracted with phenol and chloroform, ethanol precipitated, then ligated for 12 hours at 15 C in a final volume of 50 pl. This ligate mix was used to transform JA300 to tryptophan prototrophy and ampicillin resistance. Plasmid pFRL4 containing a single EcoRI site was then purified. Referring now to Fig.2, the pGBn plasmid series was digested with BamHI andEcoRI then run on a 1 percent agarose gel. The 1500 bp promoter containing fragment from each lane was cut from the gel, electroeluted, then purified on a 10 ml diethylamino cellulose WRatman column before ethanol precipitation. 20 pg of pFRL4 was digested with BamHI and EcoRI then run on a 1 percent agarose gel. The large Skb fragment was cut from the gel, electroeluted, 2X extracted witi phenol and chloroform before ethanol precipitation. 3 g of this fragment was then separately ligated with each of the promoter containing fragments for 12 hours at 150C in 50 ligation mix. E. coli K 12 strain 294 was transformed with the ligation mix to ampicillin resistance and plasmids from each of these different transformation mixtures were purified pFRPn plasmid genes . 10 g of pLelF D 5 was digested with EcoRI then run on a 6 percent acrylamide gel. The 560 bp leukocyte interferon D gene was cut from the gel, electroeluted and 2X extracted with phenol chloroform before ethanol precipitation. This interferon gene was then ligated into the uniqueEcoRI site in the pFRPn plasmids previously cut with EcoRI and treated with bacterial alkaline phosphatase. These vectors were then used for EglII restriction analysis and yeast transformations.Interferon Assay Extracts of yeast were assayed for interferon by comparison with interferon standards by the cytopathic effect CPE inhibition assay 27 . Yeast extracts were prepared as follows Five ml cultures were grown in YNB CAA until reaching A660 l.2. Cells were collected by centrifugation then resuspended in 600 pl of 1.2 14 sorbitol, 10 mH KI 2P04, pH 6.8 and 1 percent zymolyase 60,000 then incubated at 30iC for 30 min. Spheropiasts were pelleted at 3000 xg for 10 min., then resuspended in 150 l of 7 M guanidine hydrochloride plus lm14 phenylmethylsulfonylfluoride PMSF . Extracts were diluted 1,000 fold inPBS buffer 20 mM NaH2PO4, pH 7.4, 150 mM NaCl, 0.5 percent BSA immediately before the assay. RESULTSConstruction of a Vector for Insertion of a Series of Proraoter Fragments and for insertion of a Gene to be Expressed To design a plasmid vector for autonomous replication in yeast, it is necessary to have both an origin of replication and a gene present for selection in yeast. furthermore, the plasmid must contain a bacterial plasmid origin of replication and a means of selection in bacteria e.g., an antibiotic resistance gene . With these requirements a plasmid can be constructed and modified in vitro using recombinant DNA techniques, amplified in bacteria, preferably E. coli, and finally transformed into yeast. Such a vector is shown in Fig. 1 and is designated YRp7 24 26 .It contains a chromosomal origin of replication from yeast arse as well as the TRP1 gene which codes for W 5 phosphoribosyl anthranilate isomerase 23 . The TRP1 yeast gene can complement allow for growth in the absence of tryptophan trpl mutations in yeast e.g., RX218, see Methods and can also complement the trpC1117 mutation of E. coli e.g.JA300 18 . The plasmid is pBR322 29 based so it also permits growth and selection in E. coli using antibiotic resistance selection. Since it was necessary to clone into this vector BamHl EcoRi restriction fragments containing a yeast promoter, it proved convenient to first remove one EcoRI site from the vector. This was done as shown in Fig. 1. The vector YRp7 was cut with EcoRI followed by filling in of the sticky EcoRI ends of both fragments with Klenow DNA polymerase I.The fragments were then blunt end ligated and the resulting DNA was used to transform E. coli JA300 to Trp and ampicillin resistance air . In such a way plasmid pFR W1 was isolated with both EcoRI sites removed. One EcoRI site was then restored to the plasmid in order that anEcoRI BamHI fragment could be later cloned into the vector. This was done by cutting both YRp7 and pFRW1 with HindlII followed by the isolation of the fragments indicated. When the small Hindlil fragment of YRp7 was put together with the large HindIII fragment of pFRWI, pFRL4 was obtained. It was selected for in E. coli JA300 using Trp and ApR phenotypes.Construction of Yeast Alcohol Dehydrogenase ADH Promoter Fragments Since It is not known whether certain specific sequences in the leader region preceding structural genes are required for RNA polymeraseII binding or what OI4A is necessary for ribosome recognition tribosome binding sites of the mRNA, promoter fragments from the ADH gene ADCI were obtained as described in Fig. 2. The first step was to show that the G leader DNA sequence of the ADH gene could be used to express another structural gene from yeast without its leader sequence CYC1 . Thus a plasmid which can complement a cycl mutation in yeast can be used to isolate the ADH promoter fragment that will result in cycl expression. This promoter fragment could then be used to express other eukaryotic genes eg., the Leukocyte Interferon D gene . As shown in Fig. 2, pY9T6 containing the ADC1 locus Bennetzen, supra was cut with Sau3A to isolate tile 5 flanking sequence of the ADH gene on an approximately 1600 bp fragment. The ATG translation start for the ADH coding sequence is shown with the A at position 1, and transcription goes from left to right as shown. This fragment was blunt ended using Klenow DNA polymerase I followed by a ligation with a mixture of Bam9I and Hindill linkers. After cutting with BamHI and Hindill, the fragments were ligated with the large BamHI HindIII fragment of pBR322.The ligation products were used to transform E. coli to ApR and the desired pJD221 was isolated from a transformant colony using a standard miniscrcen procedure see Methods . pJD221 was cut with HindIII and then with exonuclease III and S1 nuclease to remove base pairs toward but not through the ATG of the ADH structural gene. This procedure also removes base pairs in the opposite direction toward the EcoRi site at approximately the sane rate. The reaction was designed so as to not remove the ATG of ADH since the ATG of CYC1 was not present in the fragment to be expressed under AO promoter control.Therefore a complementation of cycl yeast would require a functional ADIS1 CYC1 fusion protein. The end deleted products were treated with Klenow DNA polymerase I to ensure blunt ends followed by the addition of XhoI linkers by blunt end ligation. After XhoI cutting a circular plasmid was regenerated by ligation. Plasmids containing gel determined, properly sized EcoRI to BamHI restriction fragments were digested with XhoI , then ligated with the large XhoI to BamHI restriction fragment of plasmid pYecyc 1 ax 1. After amplification in E. coli RR1 resulting plasmids were used to transform a cycl cyc3 leul yeast strain to leucine prototrophy on minimal glucose plates. Growing colonies were patched onto glycerol ethanol plates. Yeast able to grow on such plates require the presence of functional cytochrome c protein.This can only occur on this plasmid if fragments containing ADH promoter deletions can initiate in the correct reading frame translation of tt cytochrome c coding region. Plasmid pACF301 was isolated from one such transformant. The junction between ADH1 and CYC1 is shown at the bottom of Fig. 2b. Six amino acid codons from the ADH sequence were present with 3 new amino acid codons due to the XhoI linker, and the rest represented the CYC1 structural gene. Thus the ADH promoter fragment is expressing a fusion gene product that produces a phenotypically active CYC1 gene fusion product. In the construction of a yeast expression plasmid, it is desirable that the ATG codon of the non yeast gene to be expressed be the one belonging to the same non yeast gene rather than a vector ATG which would lead to the synthesis of an undesired fusion protein. Therefore, it proved appropriate to remove nucleotides through the ATG of the ADH promoter fragment by another series of deletions and supply a new translation start signal with the gene to be expressed. Since the functionality of upstream D.4A sequence 1 to 1500 during the expression process is not known, it was desirable to remove as little sequence as possible upstream from the ATG and to try different fragments lacking both the initially present ATG and various amounts of additional DWA sequence. These additional promoter fragments were Isolated as shown in Fig.2b. pACF301 was cut with XhoI and Bal31. After blunt ending, addition of EcoRI linker, BamlI EcoRI cutting, and sizing fragments the correct size class of fragments were ligated with EcoR1 BJmllI cut pBR322.Specific recloned AUDIT promoter fragments were isolated from plasmids from various E. coli ApR transformants. Fig. 3 shows the DNA sequences of the transcribed strand of 8 of the resulting, variously sized and numbered promoter fragments. The numbered lines show where the right end of the fragment ends and where the EcoRI linker sequence begins. The ends of fragments 904 and 906 were exactly determined by sequencing. The EcoRI sticky ends of these fragments were labelled withKlenow DIA polymerase using a 32P dATP. A sequencing gel was used to read from the A s into the linker through the junction. The other 6 fragment ends were approximated to within about 1 2 base pairs by labelling as above, cutting with Alum, followed by sizing on the same denaturing gel.Construction of Plasmids that Express Biologically Active LeukocyteInterferon D in Yeast In order to optimize for successful expression of LelF D in yeast, eigllt different promoter fragments Fig. 3 were ligated into the pFRL4 vector as shown in Fig. 4. The vector was designed to have ADIl promoter transcription in the same direction as TRP1 gene transcription 31 . Since the LeIF D gene was to be inserted in the EcoRI site and was not known to contain proper 3 termination and processing sequences for yeast recognition, the TRP1 gene flanking sequence was aligned to perform these functions. The resulting pFRPn series where n is the promoter fragment number was obtained as shown. The preferred embodiment of these, pFRP6 in a transformant strain of E. coli 294, has been deposited in the AmericanType Culture Collection ATCC no. 31814, deposited 24Feb.1981 . These vectors were cut with EcoRI, alkaline phosphatase treated to avoid premature recircularization , and ligated with the EcoRI LelF D gene fragment. Tic ATC of this gene Immediately follows the C of the EcoRI linker GAATTCATG as shown Fig. 4 . Ampicillin resistant transfornants of E. coli K 12 strain 294 were screened to filld plasmids containing both orientfitions of the LeIF D fragment pFRSn series n refers to screening number . Orientations were determined by agarose gel electrophoresis using Bgill digestion which cuts both in the vector and in the LeIF D gene as shown. Three of the plasmids demonstrated unpredicted restriction patterns.pFRS7 and pFR.35 have an extra uglil fragment at 560 bp. This results from having two fragments of LelF D in line with ADH transcription. pFRS16 has no proper orientation fragment but has a 1700 bp fragment which apparently resulted from the ligation of two vector fragments together two TRP1 containing tails together with one LeIF D fragment in between two heads containing ADH promoter fragments. Thus in this ligation product the interferon gene is in the proper orientation for expression by one of the ADH promoter fragments.Evidence for Leukocyte Interferon D Expression in Yeast First the E. coli 294 strains containing the various plasmids pFRSn were grown and extracts prepared see Methods . No interferon activity was observed in the extracts using the cytopathic effect inhibition assay see Methods . However, when plasmids were purified and used for yeast transformations by selection of.TRP phenotype using yeast RH218 trpl mutation , all plasmids with orientation I produced interferon activity in yeast while no plasmids with orientation II produced interferon. Table 1 shows the results of interferon assays which measure antiviral activity effects on VSV virus challenge of MDBK tissue culture cells see Methods . Seven of the promoter fragments definitely express the LelF D gene when the gene is in the proper orientation I . This is demonstrated by comparing units ml of extract for the orientation I plasmids with the orientation II plasmids. All orientation II plasmids expressed 1900 units ml of extract , a value 1 to 4 percent of the values for orientation I plasmids actually background values are probably much lower than this since the 1900 value is a function of the assay procedure . Table 1 Interferon activity in yeast extracts Units Unitsb l Cellsc 1 Pct.c cell Moleculesd ADH Unitsa ml Units l of cells of of cells containing cellOrien promoter of of cells at Abs 1 culture with plasmid containing tation fragment Plasmid extract x10 6 X10 6 x10 10 plasmid x104 plasmidI 904 pFRS3 47,000 1.4 1.2 2.1 19 3.5 53,000I 905 pFRS7 47,000 1.4 0.54 3.4 10 1.4 21,000I 913 pFRS16 187,500 5.6 2.0 5.1 32 3.4 51,000I 906 pFRS12 125,000 3.8 1.8 2.7 19 7.4 110,000I 906 pFRS36 187,500 5.6 2.3 3.0 16 12 180,000I 915 pFRS23 125,000 2.1 0.81 3.6 21 2.8 42,000I 921 pFRS35 250,000 7.5 2.8 4.5 18 9.2 140,000I 919 pFRS34 93,750 2.8 1.4 2.2 17 7.6 110,000II 904 pFRS2 1900 22II 905 pFRS6 23II 913 pFRS17 36II 906 PFRS11 13II 915 pFRS22 46II 921 pFRS26 25II 919 pFRS33 31 a Yeast cells were grown in 5 ml of YNB CAA Trp selection to an absorbance Abs of 1.2 to 2.8 at 660 mn and spheroplasted with zymolyase as described in Methods. The final pellet was resuspended in 0.15 ml of 7 M guanidine HCl and 1 mH PHSF. The extracts were assayed using 1 100 or 1 1000 dilutions in dilution buffer using and VSV virus challenge of MDBK cells see Methods . b Units liter of cells at Abs660 1 x 10 6 is a normalization since different cultures were stopped beween 1.2 to 2.8 Abs660.c The number of yeast cells per culture was determined by dilution and plating onYNB CAA tryptophan 50 g ml plates. To find out how many cells have plasmid Trp complementation cultures were also plated on YNB CAA without tryptophan . Thus percentage of cells containing plasmid was determined.d Molecules cell containing plasmid was calculated assuming purified leukocyte interferon D has a specific activity of 2 x 108 units mg 27 in the same assay and a molecular weight of 20,000 g mole. Since arsi chromosomal origin of replication containing plasmids have previously been shown to be unstable and lost in a high percentage of the cells even under selective maintenance pressure 24, 25 , the percent of cells containing the plasmid at the time of extract preparation was measured. This was done by plating diluted cultures on plates with and without tryptophan. The results of this demonstrate that the plasmid is somewhat unstable in yeast though not in bacteria , but can be maintained by growth under selective pressure. These results also are evidence for the presence of the plasmid, since RH218 trpl yeast do not grow on plates without tryptophan and since a revertant to TRP would plate with equal efficiency on plates with and without tryptophan.Furthermore, the percentages of cells containing plasmid are similar comparing yeast with orientation I and II plasmids. This suggests that the production of interferon in the yeast cell does not result in increased instability of the plasmid due to interferon toxicity to the cell. The fact that all the promoter fragments express interferon when up to 32 bp are removed upstream from the ATG suggests that the DNA sequence in this region is relatively unimportant in transcription and translation. These results also suggest that precise spacing between the promoter and the ATG may be relatively unimportant for expression in yeast. In addition, Table 1 shows molecules cell values which are very much higher than the 10,000 molecules cell observed.for interferon D expression in E. coli on a high copy plasmid with a strong promoter trp promoter 32 . Assessment of this extreme difference up to 18 fold in molecules per cell should recognize that the yeast cell volume ls probably 2 orders of magnitude higher than that of E. coli however, the amount of expression from only r 2 copies of the yeast plasmid versus the high copy number of plasmids producing interferon in E. coli is dramatic. Co inarison sf the Size of Interferon Produced in Yeast versus E. coli Since the interferon gene uses its own ATG initiation codon and since the alcohol dehydrogenase ATG has been removed in the construction, one would expect to find that the interferon expressed in yeast is the same size as the interferon in E. coli 32 . SDS potyacrylamide gel electrophoresis was accordingly done on an E. coli extract containing interferon D versus a yeast extract containing interferon D. After running the gel, two lanes containing yeast extract versus E. coli extract were simultaneously sliced. The slices were put into assay dilution buffer and left at 4 C for 3 days. Interferon assays were then performed to compare sizes of the peptides.Both appear to be about 20,000 daltons, the size expected for Interferon D. However, there does appear to be a slight difference in the molecular weights, with yeast interferon D being about 7 percent larger, possibly owing to glycosylation. Despite the size difference, the products of yeast expression exhibited interferon activity Table 1 . The preceding data clearly demonstrates that a yeast 5 fidnking DlIA sequence, without the translation start signal of the structural gene, can efficiently promote the expression of an inserted mammalian or other structural gene for a biocompetent polypeptide, and do so without the aid of selective pressure for the product of expression i.c., the expression product is not required for cell growth . The availability of yeast promoter containing plasmids pFRPn series having both yeast and bacterial phenotypical genes and origins of replication, and a site downstream from the promoter convenient for the insertion of translation start and stop bearing structural genes permits the creation of D1IA expression vectors for a wide variety of polypeptides. Thus, into such a site may be inserted, for example, structural genes for both normal 5 and hybrid 6 human leukocyte interferons, fibroblast interferon 7 , somatostatin or the A or B chains of human insulin 1 , human proinsulin 2 , thymosin alpha 1 3 , human growth hormone 4 and, indeed, virtually any other biocompetent polypeptide. Following expression, product may be extracted and purified as in the case of bacterial expression, mutatis mutandis. It will be appreciated that the Invention is not limited in its application to the particular expression vector exemplified above. For example, use of the so called two micron origin of replication would provide additional stability, making unnecessary resort to selective pressure for maintenance of the plasmid in the yeast cell, particularly if the host strain is CIR , I.e., contains normal two micron plasmid 33 . Such an expression vector would be stable in yeast in the rich medium ordinarily best for large scale fermentations. At the same time, use of the two micron origin of replication could significantly increase plasmid copy number in each cell. Stability of the expression vector in yeast may also be enhanced by inclusion within the plasmid of a yeast centromere 34 , an element involved in maintenance of the yeast chromosome. The resulting plasmid will behave as a mlnichromosome, such that selective pressure will not be required during growth or maintenance of the plasmid. As many as 17 different yeast centromeres have been identified to the present date. Transcription terininators other than that present on the TRP1 gene may be employed, e.g., other 3 flanking sequences from yeast such as the 3 flanking sequence contained on a Hinc ll ssa H1 fragment of the ADH 1 gene. Optimization may also result from alteration of the sequence between the yeast promoter fragment and the inserted gene fragment. Thus, an A adenine base is found at position 3 the third base before the translation start signal of all twenty different mRNA coding yeast genes heretofore sequenced. A variety of means eg., use of linkers for including such an element In the plasmids of the invention will appear to those skilled in the art. Of course, promoters other than the ADH promoter exemplified above may be employed In variants of the invention. For example, the promoter of the yeast 3 phosphoglycerate kinase gene may be employed, doubtless increasing expression levels significantly over those observed for theADH system. Again, one or more of the promoters for yeast glyceraldehyde 3 phosphate dehydrogenase may be employed. This system is nonfunctional in the absence of glucose, but induced 200 fold in its presence, and could accordingly be employed for fine control of expression. From the foregoing, it will be apparent that the invention provides new means for the expression of valuable polypeptides. In particular instances, efficiency of expression relative to that in recombinant bacteria may result from the different codon usage patterns as between yeast and bacteria, such that eukaryotic genes may be better expressed in yeast. The yeast expression systems of the invention may also provide advantage in the glycosylation of biocompetent polypeptides, an ability bacteria lack. The glycosylation system of yeast is very similar to that of higher eukaryotes, and glycosylation may prove to have profound effects on the functions of proteins. As will be apparent to those skilled in tlic art in the light of the foregoing discussion, the invention is not to be limited to the preferred embodiments thereof exemplified above The contributions of certain of the coinventors hereof arose in the course of work funded in part by the United States Department of Health and Human Services. BIBLIOGRAPHY 1. United States patent application Ser. No. 90,980 filed llov. 5, 1979 by A. Riggs, assignee Genentech, Inc. EPA Publication 000 1930. 2. United States patent application Ser. No. 222,044, Atty. Dkt. Ilo. 1980 109, filed Jan. 2, 1981 by D. Goeddel et al., assignee Genentech. Inc. Sures et al. Science 208, 57 1980 . 3. United States patent application Ser.No.125,685, filed Feb.28, 1980 by Crea et al, assignee Genentech, Inc. Wetzel et al, Biochemistry 19, 6096 1980 . 4. United States patent application Ser. No. 55120, filed July 5, 1979 by D. Goeddel et al., EPA Publication 00 22 242. 5. United States patent application Ser. No. 205,578, filed flow. 10, 1980 by D. Goeddel et al, assignees Genentech, Inc. and Hoffman La Roche Inc. Goeddel et al., Nature 287,411 1980 . 6. United States patent application Ser. No. . Atty. Dkt. No. 100 42, filed Feb. 23, 1981 by D. Goeddel et al., assignee Genentech, Inc. Weck et al., Nucleic Acids Research, 9, 6153 1981 . 7. United States patent application Ser.No.190,799, filed Sept.25 1980 by D.Goeddel, assignee Genentech, Inc. Goeddel et al, Nucleic Acids Research, 8, 4057 1980 . 8. Shine et al. Nature 285, 456 461 1980 . 9. Beggs, J.D., van den Berg, J., van Ooyen, A., and Weissmann, C. Nature 283, 835 840 1980 .10. Henikoff, S., Tatchell, K., Hall, B.D. and Nasmyth, K.A. Nature 289, 33 37 1981 .11.12. Carbon, J., Clarke, L., Chinault, C., Ratzkin, B., and Walz, A. 1978 in Biochemistry and Genetics of Yeast Bacila, M., Horecker, B.L., and Stoppani, A.O.t4., eds pp. 425 443, Academic Press, New York.13. Clarke, L. and Carbon, J. Cell 9, 91 99 1976 .14. Davits, R.W., Thomas, 14., Cameron, J., St. John, T.P., Scherer, S., and Padgett, R.A. Methods ln Enzymology, Vol, 65, Part 1, p. 404 411. 15. Clare, L. and Carbon, J. PNAS 72. 4361 4365 1975 .16. llsiao, C. L. and Carbon, J., Proc. Matl. Acad. Sci. USA 76, 3829 3833 1979 .17. Blrnboim, H.C., and Doly, J. Nucleic Acids Res. 7, 1513 1523 1979 . 18. Hitzeman, R.A., Clarke, L. and Carbon, J. Journal of Biological Chemistry 255, 12073 12080 1980 .19. Backman, K., Ptashne, M., and Gilbert, W., Proc. that. Acad. Sci. USA 73, 4174 4178 1976 .20. Miozzari, G., Neideiberger, P., and Hutter, R. Journal of Bacteriology 134, 48 59 1978 .21. Faye, G., Leung, D.W., Tachell, K., Hall, B.D., and Smith, M. Proc. Natl. Acad. Sci. USA, in press 1981 .22. Miller, J.H. Experiments in Molecular Genetics, pp. 431 433, Cold Spring Harbor Laboratory, Cold Spring Harbor, NY.23. Maxim, A.M., and Gilbert, W. Methods in Enzymol. 65, 490 565 1980 .24. Stinchcomb, D.T., Struhl, K., and Davis, R.W. Nature 282, 39 43 1979 .25. Kingsman, A.J., Clarke, L., Mortimer, R., and Carbon, J. Gene 7, 141 153 1979 . 26. Tschumper, G., and Carbon, J. Gene 10, 1980 .27. Stewart, W.E. II The Interferon System Springer, New York, 1979 .28. DeMoss, J.A. Biochem. Biophys. Res. Common. 18, 850 857 1965 .29. Bolivar, F., Rodriguez, R.L., Green, P.Y., Betlach, M.C., Heyneker, H.L., Boyer, H.W., Crosa, Y.H., and Falkow, S Gene 2, 95 113 1977 .30. Williamson, Y.M., Bennetzen, J., Young, E.T., Nasmyth, K., and Hall, B.D. Nature 283, 214 216 1900 .31. Tschumper, G., and Carbon, J. Gene Amst. 10, 157 166 1980 .32. Goeddel, D.Y. et al. Nature 287, 411 416 1980 . 33. Beggs, J.D. Nature 275, 104 109 1978 .34. Clarke, L., and Carbon, J. Proc. Natl. Acad. Sci. USA 77, 2173 2177 1980 .