# Preparation of hepatitis B surface antigen in yeast.

## Abstract
The hepatitis B surface antigen HBsAG for use in hepatitis vaccine is produced in particle form like the natural product by yeast transformed with a vector. The use of yeast has advantages over prokaryotic hosts, e.g. in yields. A preferred vector is plasmid pYeHBs. This includes a promoter derived from the yeast PKG promoter region a LEU2 gene allowing for selection in yeast and the HBsAg gene. This last war derived from a plasmid containing the complete HBV gene inserted into the EcoRI site of pBH322.

## Claims
Cldims 1. A DNA expression vector capable of replication and phenotypic selection in yeast host strain comprising a promoter compatible with a yeast host strain and a DNA sequence encoding hepatitis B surface antigen, said sequence being positioned together with translational start and stop signals in said vector under control of said promoter such that in a transformant yeast strain it is expressed to produce hepatitis B surface antigen in discrete particle form. 2. The plasmid pYeHBs. 3. A yeast strain transformed with the DNA expression vector according to Claim 1 or the plasmid according to claim 2. 4. The yeast strain according to Claim 3 obtained by transforming a leu2 auxotrophic yeast strain. 5. The yeast strain according to Claim 3 obtained by ansforming strain XV610 8C. 6. A fermentation culture comprising a transformed yeast according to Claim 3, 4 or S. 7. A method of producing hepatitis B surface antigen in particle form suitable for use in conferring immunogenicity to hepatitis B virus in a susceptible human which comprises a providing a DNA transfer vector capable of replication and phenotypic selection in yeast host strains, b providing a DNA fragment comprisir3 a promoter compatible with a yeast host strain, c providing a DNA fragment encoding hepatitis B surface antigen, d assembling the fragments of steps a , b and c together with appropriately positioned translational start and stop signals to form a replicable expression vector characterized in that said sequence of step c is under control of said promoter such that it is expressible to produce discrete hepatitis B surface antigen, e transforming a yeast strain with the vector of step d , f allowing the yeast transformant to grow until said hepatitis B surface antigen is produced therein, and g recovering said hepatitis B surface antigen in discrete particle form. 8. The method according to Claim 7 wherein the DNA sequence of step c comprises, in order from the 5 end of its coding strand, ATG translational start codon, the nucleotides encoding hepatitis B surface antigen of the hepatitis B genome, and translational stop signal s . 9. The method according to Claim 7 or 8 wherein the yeast strain of step e is XV610 8C. 10. The method according to any one of Claims 7 to 9. wherein the promoter of step b is derived from the yeast PGK promoter region. 11. The use of hepatitis B surface antigen, produced by the method according to Claim 8, for conferring immunogenicity to hepatitis B virus in a susceptible human or for preparing a vaccine therefor.

## Description
PREPARATION OF HEPATITIS B SURFACE ANTIGEN IN YEASTThis invention relates to the use of recombinant DNA technology for the production of hepatitis B surface antigen HBsAg in yeast organisms for use in the preparation of vaccines against hepatitis B virus HBV . In one aspect, the present invention relates to tlie construction of microbial expression vehicles containing DNA sequences encoding hepatitis B surface protein antigens operably linked to expression effecting promoter systems and to the expression vehicles so constructed. In another aspect, the present invention relates to yeast organisms transformed with such expression vehicles, thus directed in the expression of the DNA sequerces referred to above.In yet other aspects, this invention relates to the means and methods of converting the end products of sucr yeast expression to entities, such as vaccines, useful against hepatitis B virus. In preferred embodiments, this invention provides for particular expression vectors that utilize potent yeast promoters and DNA sequences distal to that encoding the hepatitis B surface antigen protein which, in combination, provide attractive yields of desired protein, produced in particle form of about 22 nm, and containing antigenic determinant s of hepatitis B virus. The publications and other materials hereof used to illuminate the background of the invention, and in particular cases, to provide additional details respecting its practice are incorporated herein by reference, and for convenience, are numerically referenced in the following text and grouped in the appended bibliography. Hepatitis B Virus Hepatitis B serum hepatitis virus is transmitted among humans and manifests as chronically debilitating infections which can result progressively in severe liver damage, primary carcinoma and death. In most cases, complete recovery from hepatitis B infections can be expected however, large segments of the population, especially in many African and Asian countries, are chronic carriers with the dangerous potential of transmitting the disease pandemi call y. Hepatatis is caused by d virus vector hepatitis B virus orHDY which in its whole state the so called Dane particle represents the virion and consists of a 27nm nucleocapsid enclosing a DNA molecule and an envelope surrounding the nucleocapsid. Proteins associated with the virion include tbo core antigen li3cAg , a DNA polynerdse and the surface antigen HBsAG which has been found in serum of infected and carrier humans. Antibodies to HBsAG have also been found in serum HBV infected people. It is believed that HBsAG is the HBV antigen that can induce immunogenic production of antibody anti HBs and thus it would be the principal in an HBV vaccine. Attention is directed to Dane et al., Lancet 1970 I , 695 1970 Hollinger et al., J. Immunology 107, 1099 1971 Ling et al., J.Immunology 109, 834 1972 Blumberg, Science 197, 17 1977 Peterson et al., Proc. Nat. Acad. Sci USA 74, 1530 1977 and Viral Hepatitis, A ContemporaryAssessment of Etiology, Epidemiology, Pathogenesis andPrevention. Vyas et al., eds. Franklin Institute Press,Philadelphia, 1978, each of which is hereby incorporated by this reference to further illustrate the background of this invention.HBsAg is present in infected plasma predominantly in the form of sphencal particles having diameters ranging from about 16 to 25 nm the so called 22 nin particle. These are thought to represent a nonintectious viral envelope.Because antibodies against HBsAg are protective against H V infection, these non infectious particles can effectively be used s a vaccine. inasmuch as the hepatitis B virus has not been infectious in cell culture and can only be obtained from infected humans or higher primates, means have not been available for obtaining and maintaining sufficient supplies of HBV for use in producing antigen for immunization against HBV.The present invention provides the means and methods for producing a vaccine effective against HBV. By means of recombinant DNA technology, the gene encoding HBsAg was inserted together with appropriate translational start and stop signals under the control of an appropriate expression promoter into a replicable expression vector and the latter used to transform yeast cells. The cells, thus genetically directed, produced HBsAg directly and in particle form which, when purified, is suitable to immunize against HBV.Recombinant DNA TechnologyWith the advent of recombinant DNA technology, the controlled microbial production of an enormous variety of useful polypeptides has become possible. Many mammalian polypeptides, such as human growth hormone and leukocyte interferons, have already been produced by various microorganisms. The power of the technology admits the microbial production of an enormous variety of useful polypeptides, putting within reach the microbially directed manufacture of hormones, enzymes, antibodies, and vaccines useful against a wide variety of drug targeting applications.A basic element of recombinant DNA technology is the plasmid, an extrachromosomal loop of double stranded DNA found in bacteria oftentimes in multiple copies per cell.Included in the information encoded in the plasmid DNA is that required to reproduce the plasmid in daughter cells i.e., a replicon or origin of replication and ordinarily, one or more phenotypic selection characteristics, such as resistance to antibiotics, which permit clones of the host cell containing the plasmid of interest to be recognized and preferentially grown in selective media. The utility of bacterial plasmids lies in the fact that they can be specifically cleaved by one or another restriction endonuclease or restriction enzyme , each of which recognizes a different site on the plasmidDNA. Thereafter heterologous genes or gene fragments may be inserted into the plasmid by endwise joining at the cleavage site or at reconstructed ends adjacent to the cleavage site. Thus formed are so called replicable expression vehicles. DNA recoinbination is performed outside the microorganism, and the resulting recombinant replicable expression vehicle, or plasmid, can be introduced into microorganisms by a process known as transformation and large quantities of the recombinant vehicle obtained by growing the transformant. Moreover, when the gene is properly inserted with reference to portions of the plasmid which govern the transcription and translation of the encoded DNA message, the resulting expression vehicle can be used to direct the production of the polypeptide sequence for which the inserted gene codes, a process referred to as expression.Expression is initiated in a DNA region known as the promoter. In the trancription phase of expression, the DNA unwinds exposing it as a template for initiated synthesis of messenger RNA from the DNA sequence. The messenger RNA is, In turn, bound by ribosomes, where the messenger RNA is translated into a polypeptide chain having the amino acid sequence encoded by the mRNA. Each amino acid is encoded by a nucleotide triplet or codon which collectively make up the structural gene , i.e., that part of the DNASequence which encodes the amino acid sequence of the expressed polypeptide product. Translation is initiated at a start signal ordinarily ATG, which in the resulting messenger RNA becomes AUG . So called stop codons define the end of translation and, hence, the production of further amino acid units.The resulting product may be obtained by lysing the host cell and recovering the product by appropriate purification from other bacterial proteins.In practice, the use of recombinant DNA technology can express entirely heterologous not ordinarily found in, or produced by, a given micro organism , polypeptides so called direct expression or alternatively may express a heterologous polypeptide fused to a portion of the amino acid sequence of a homologous found in, or produced by, the corresponding wild type microorganism polypeptide. In the latter cases, the intended bioactive product is sometimes rendered bioinactive within the fused, homologous heterologous polypeptide until it is cleaved in an extracellular environment. See, eg., British PatentPubl. No. 2 007 676 A and Wetzel, American Scientist 68, 664, 1980 .If recombinant ONA technology is to fully sustain its promise, systems must be devised which optimize expression of gene inserts, so that the intended polypeptide products can be made available in controlled environments and in high yields.British Patent Publication 2034323A, publishedJune 4, 1980, describes the isolation of the HBV genome comprising about 3200 nucleotides and the incorporation of this DNA into a vector taking advantage of an EcoRI digest and subsequent ligation. It is stated that the resultant cloned HBV DNA can be labelled and used as a probe to detect Dane particles in serum and that the vector could be used to express a polypeptide containing a hepatitis B protein fragment.In a related article from the same laboratories Proc.Natl. Acad. Sci USA 77, 4549 1980 , it is reported that tandem cloned hepatitis B genomes were introduced into mouse cells and integrated into the mouse chromosome.Polypeptide identified as hepatitis surface antigen was excreted from the cells as particles similar to those found in the sera of infected humans.European Patent Application Publication No. 13828 describes the production of HBV core antigen by read through translation of an mRNA transcript from the genomic DNA. No surface antigen was detected.In European Patent Application Publication No. 20251 there is described the production in E. coli of a fusion protein purportedly containing a portion of the surface antigen protein of HDV. Because the surface antigen determinants were present as d fusion protein, the structure could not be native hence, the reported low ingnunoloyical activity.The present invention is based upon the discovery that recombinant DNA technology can be used to successfully and efficiently produce the hepatitis B surface antigens, directly and in discrete particle form. The product is suitable for use in conferring immunogenicity to hepatitis B virus in a susceptible human or for preparing a vaccine for such use. The product is produced in a eukaryotic host yeast and thus carries the advantages such systems confer on corresponding, proteinaceouz products, to wit, glycosylation and sugar and lipid association more closely related to human produced proteins versus bacterial produced proteins whose hosts are incapable of such sophisticated processing.In addition, the eukaryotic yeast cells tolerate the protein product better than prokaryotic systems, doubtless increasing expression levels and yields significantly over bacterial systems, in which the hepatitis protein per se proved lethal.The present invention comprises the hepatitis B surface antigen HBsAG produced and the methods and means of its production.Specifically, the present invention is directed to HBsAg in particle form comprising immunogenic determinant s of hepatitis B virus, produced in yeast. The HRsAg hereof is produced in discrete particle form, devoid of any additional, fused polypeptide artifact, whether encoded by another portion of the HBV genome or by DNA homologous to the yeast strain employed. The present invention is further directed to replicable DNA expression vehicles harboring gene sequences encoding HBsAg in directly expressible form. Further, the present invention is directed to yeast strains transformed with the expression vehicles described above and to microbial cultures of such transformed yeast strains, capable of producing HBsAg.In still further aspects, the present invention is directed to various processes useful in preparing said HBsAg gene sequences, DNA expression vehicles, yeast strains and cultures and to specific embodiments thereof.Still further, this invention is directed to the use of thus produced HBsAg for the preparation of vaccines useful to confer immunogenicity to HBV in susceptible humans, to such vaccines and tithe method of using them to inoculate and confer immunogenicity to HBV in susceptible humans. The preferred promoter herein is one tailored from the yeast 3 phosphoglycerate kinase PGK promoter region which is described infra. The PGK gene was analyzed for restriction sites which would be useful for sequencing. DNA sequence analysis around one of these restriction sites indicated that translation beginning with a specific ATG would yield a protein whose N terminal amino acids corresponded to the sameN terminal amino acids of human and horse PGK, this flankingDNA being considered the yeast PGK promoter region. The PGK promoter region was analyzed for a region which would logically support a unique restriction site. A primer repair reaction was carried out and this unique site was placed at the 3 end of the PGK promoter region. This unique site allowed the PGK promoter region to effect the direct expression of the thus fused gene in yeast. A yeast bacteria shuttle plasmid, pCV13 or YEpl3 described infra, had certain restriction fragments removed. Replacement of these fragments by PGK promoter region fused to sur ace antigen allowed reclosure of the vector. This vector was used to transform yeast which were grown up under customary fermen tation conditions and permitted to produce the desired HBsAg product. Recovery of product followed by breaking open the transformed yeast. Supernatants from centrifuged solution were used as a substrate in hepatitis B surface antigen radioimmune assays.A particular DNA sequence, found to be suitable herein as a transcription terminator, was placed after the gene sequence encoding HBsAg in order to terminate mRNA synthesis and provide a Poly A addition site for proper mRNA processing and translation. This terminator is based upon the 232bp Hind lII to BglII restriction fragment from the Trpl gene contained in pFRL4. This fragment was isolated containing some coding sequence from the yeast TRPI gene and the 3 flanking sequence required for proper transcription termination and polyadenylation. This terminator fragment was fused via its HindlII site to the HindilI site of the HBsAg gene.Transcription of the HBsAg gene by a fused yeast promoter in yeast is designed to terminate in the fused TRP1 terminator fragment. pCV13 1 contains the ampicillin and tetraycline resistance genes allowing for plasmid selection in E. coli, an E. coli origin of DNA replication, the LEU2 gene allowing for selection of the plasmid in yeast and the 2 micron origin allowing forDNA replication of the plasmid in yeast This plasmid is found in the nucleoplasm of transformed yeast. The small HindllI to Baf I fragment from the tetracycline resistance gene was removed from the vector. The resulting large vector fragment was then ligated with the 1.6 kbpHindIII to EcoRI assembled PGK promoter fragment and a I.lkbpEcoRI to BglII HDsAg TRP1 terminator fragment so as to produce the pYcHBs vector in Figure 6.This HBsAg expression vector was constructed to have the PGK promoter fragment initiate transcription of the HBsAg structural gene and then terminate this transcript in the TRP1 terminator previously fused to theHBsAg gene. Preferred embodiments of the invention will now be described in greater detail with reference to the accompanying drawings in which Figure A depicts the construction of plasmid pSVR containing SV4O DNA with a deletion of the coding region for the VP 1 protein.Figure 1 schematically illustrates the restriction map of the 3.1 kbp HindlII insert of vector pB1 from which the PGK promoter was isolated. Indicated is the Insertion of an EcoRi site and an XbaI site in the 5 flanking DNA of the PGK gene.Figure 2 illustrates the 5 flanking sequence plus the initial coding sequence for the PGK gene before Insertion of XbaI and EcoRI sites.Figure 3 schematically illustrates techniques used to insert anXbaI site at position 8 in the PGK promoter and to isolate a 39bp fragment of the 5 fl an king sequence of PGK containing this XbaI end and a Sau3A end.Figure 4 schematically illustrates the construction of a 300 bp fragment containing the above 39bp fragment, additional PGK 5 flanking sequence 265bp from Pvur to Sau3A see Fig. 1 , and a EcoRI site adjacent to XbaI.Figure 5 schematically illustrates the construction of the 1500 bp PGK promoter fragment HindlIl EcoRI which contains, in addition to the fragment constructed in Fig. 4, a 1300bp Hindlil to PvuI fragment from PGK 5 flanking sequence seeFig 1 .Figure 6 schematically illustrates the construction of an expression vector for hepatitis surface antigen in yeast, containing the modified PGK promoter, the HBsAg gene, and the terminator region of the yeast TRP1 gene, as described in more detail herein.Figure 7 depicts A sucrose gradient sedimentation of yeast produced HBsAg compared with 22 nm particle and B a corresponding CsCl gradient centrifugation thereof.Yeast Host Organisms In the preferred emodiments hereof, the expression system was placed in the plasmid pCV13 1 , which is capable of selection and replication in both E. coli and yeast,Saccharomyces cerevisiae. For selection in yeast the plasmid contains the LEU2 gene 2 which complements yeast allows for growth in the absence of leucine containing mutations in this gene found on chromosomes III of yeast 3 . The strain used here was the strain XY610 8C which has the genotype a leu2 trpl ade6 ade2 lysl canl. This strain is deposited in theAmerican Type Culture Collection without restriction ATCCNo. 20622 . However, it will be understood that any Saccharolnyces cerevisiae strain containing a mutation which makes the cell Leu2 should be an effective environment for expression of the plasmid containing the expression system.An example of another strain which could be used is ATCC No. 38626 which has the genotype, a leu2 3 leu2 1I2 his4 519 cani 4 .This strain has two point mutations in leu2 which restricts reversion and allows for tighter tranformation of Leu toLeu .Yeast Promoters The 5 flanking DNA sequence from a yeast gene for alcohol dehydrogenase 1 can promote the expression of a foreign gene eg. leukocyte interferon D when placed in a plasmid used to transform yeast. Another yeast gene has been placed at the 3 end of this construction to allow for proper transcription termination and polyadenylation in yeast.Indeed, we have shown that the mRNA transcript does end 3 end as expected in the termination adenylation region of the yeast TRP1 gene Hitzeman et.al., Nature, 293, 717 1981 . This promoter can be suitably employed in the present invention as well as others cf. infra. In the preferred embodiments, the 5 flanking sequence of the yeast 3 phosphoglycerate kinase gene 5 was placed upstream from the structural sequence for hepatatitis surface antigen followed again by DNA containing the TRP1 gene termination polyadenylation signals. Since both the yeast alcohol dehydrogenase I 5 flanking sequence and the 3 phosphoglycerate kinase 5 flanking sequence infra can function to promote expression of foreign genes in yeast, it seems likely that the 5 flanking sequences of any highly expressed yeast gene could be used for the expression of important gene products.Since under some circumstances yeast expresses up to 65 percent of its soluble protein as glycolytic enzymes 6 and since this high level appears to result from the production of high levels of the individual mRNAs 7 , it should be possible to use the 5 flanking sequences of any other glycolytic genes for such expression purposes e.g., enolase, glyceraldehyde 3 phosphate dehydrogenase, hexokinase, pyruvate decarboxylase, phosphofructokinase, glucose 6 phosphate isomerase, 3 phosphoglycerate mutase, pyruvate kinase, triosephosphate isomerase, phosphoglucose isomerase, and glucokinase. Any of the 3 flanking sequences of these genes could also be used for proper termination and mRNA polyadenylation in such an expression system cf. Supra.Some other highly expressed genes are those for the acid phosphatases 8 and those that express high levels of production due to mutations in the 5 flanking regions mutants that increase expression usually due to the presence of a Tyl transposon element 9 . All of the genes mentioned above are thought to be transcribed by yeast RNA polymerase II 9 . It is possible that the promoters for RNA polymerase I and III which transcribe ribosomal DNA, 5S DNA, and tRNA DNA 9, 10 , may also be useful in such expression constructions. Finally, many yeast promoters also contain transcriptional control so that they can be turned off or on by variation in growth conditions. Some examples of such yeast promoters are the genes that produce the following proteins Alcohol dehydrogenase II, isocytochrome, acid phosphatase, degradative enzymes associated with nitrogen metabolism, glyceraldehyde 3 phosphate dehydrogenase, and enzymes responsible for maltose and galactose utilization 9 . Such a control region would be very useful in controlling expression of protein products especially when their production is toxic to yeast. It should also be possible to put the control region of one 5 flanking sequence with a 5 flanking sequence containing a promoter from a highly expressed gene.This would result in a hybrid promoter and should be possible since the control region and the promoter appear to be physically distinct DNA sequences.Detailed DescriptionIdentification of PGK Promoter DNA The six N terminal amino acids of the 3 phosphoglycerate enzyme purified from humans are as follows 1 2 3 4 5 6 SER LEU SER HSM LYS LEU One of the translational reading frames generated from theDNA sequence of the 141 bp Sau3A to Sau3A restriction fragment containing the internal HincII site see PGK restriction mapFigure 1 produces the following amino acid sequence. 1 2 3 4 5 6 MET SER LEU SER SER LYS LEU After removal of initiator methionine, it is seen that PGKN terminal amino acid sequence has 5 of 6 amino acid homology with N terminal amino acid sequence of human PGK. This sequencing result suggested that the start of the yeast PGK structural gene is coded for by DNA in the 14 bop Sau3A restriction fragment of pB1. Previous work 5 has suggested that the DNA sequences specifying the PGK mRNA may reside in this area of the HindIII fragment. Further sequencing of the 141 bp Sau3A fragment gives more DNA sequence of the PGK promoter Shown in Figure 2 .Protocol For the insertion of XbaI EcoRI sites in the 5 flanking sequence of the PGK gene and the Construction of aPromoter Fragment see Fig. 1 Step 1 Primer repair reaction and cloning of 39bp XbaI to Sau3A fragment as shown in Fig. 3 .Step 2 Clone PvuI to XbaI partial promoter fragment as shown in Fig. 4 Step 3 Clone EcoRI to EcoRI promoter fragment as shown in Fig. 5 contains the HindIII to EcoRI promoter fragment used in Fig. 6 for construction of expression pl asmid. Insertion of EcoRI Restriction Site In the PGK Promoter andPromoter Reassembly The use of the PGK promoter for direct expression of heterologous genes in yeast requires that there be a restriction site on the 3 end of the PGK promoter fragment which does not contain the PGK initiator ATG. Figures 1, 3, 4 and 5 show how we inserted an EcoRI site in the PGK promoter.We used a primer repair reaction 11 to first insert an XbaI site in the 3 end of promoter sequence. This XbaI sticky end was ligated to the XbaI site of a LEIFA 12 gene containing an EcoRI site adjacent to the Xbal site. This connection sequence at the 5 end of the LEIFA gene is as follows 5 TCTAGMTTCATG 3 3 AGATCTTAAGTAC 5 This connection on the LEIFA gene is how an EcoRI site was inserted next to the XbaI site on the PGK promoter. Use of this XbaI to EcoRI connnection allows for placement of structural genes containing either of these restriction ends at the 3 end of the PGK promoter for expression purposes.Construction of expression vectorIsolation of HBsAg Structural Gene The structural gene coding for the HBsAg was recovered from a plasmid pHBV T 1A containing the entire genome of HBV cloned into the EcoRI site of pBR322. This clone was obtained by methods similar to those recently published by Valenzuela et. al. 14 and 15 . The structural gene was modified in two ways 1 to incorporate a unique restriction site directly in front of the initial ATG methionine codon and 2 to blunt end ligate the HBVHpa I site located distal to the HBsAg gene to the filled inEcoRI site of pBR322. These two modifications to the DNA fragment containing the HBsAg structural gene were accomplished as described below 1. 50 g of pHBV T IA DNA was first digested with Hpa II 80 units in 200 rl reaction mixture according to enzyme supplier s BRL reaction condition to obtain a 1.7 kb DNA fragment, in which the initiation codon for the coding sequences of HBsAg was located close to the 5 end of the sense strand about 400 bp .The DNA was purified by electrophoresis on polyacrylamide gels. PAGE . The purified HpalI fragment was then treated with x exonuclease 2 units in 100 pl reaction mixture New England BioLab for 30 minutes at 37 C. A exonuclease is a 5 exonuclease which digests double stranded DNA. This reaction degraded the 5 half of the sense strdnd DNA from HBsAg coding sequences and exposed the antisense strand for pairing with added primer.The x exonuclease treated DNA was deproteinized and resuspended in 50 l of reaction mixture containing 40 mM potassium phosphate buffer pH 7.4 , 1 mM DTT, 50 rg ml BSA, 6mM MgCl2, 0.5 tnM each of dNTPs, and 0.2 n mole of dATGGAGAACATC 32p labelled at 5 end by polynucleotide kinase . The mixture was first heated at 90 C for 1 minute, annealed at O C for 30 minutes and then incubated at 37 C for 3 hours in the presence of 2 units of E. coli DIJA polymerase I Klenow fragment 16 .The DNA polymerase synthesized DNA primed by the added primer and degraded single stranded DNA with 3 OH termini and, therefore, created blunt ended DNA molecules. The resultant DNA was then deproteinized, digested with XbaI 45 units at a site located within the HBsAg gene in a 100 rl reaction mixture and fractionated by PAGE. A 91 base pair DNA fragment containing the first 30 condors of HBsAg gene was isolated after autoradiographic detection fragment A . To create a unique restriction site site immediately 5 to the liBsAg gene, we took advantage of a derivative of the plasmid pBR322 called pNCV which contains a synthetic DNA segment with the sequence AATTCTGCAG GACGTCTTAA located at the EcoRI site. Incorporated into this synthetic DNA sequence is a Pstl site. Ten 9 of pNCV DNA was first cut with 24 units of PstI enzyme in a 100 ,l reaction mixture and then treated with 2 units E. coli DNA polymerase Klenow fragment in 50 ,1 reaction mixture as described above at 8 C for 1 hour.The DNA polymerase treatment removed the 4 base pair 3 overhang created by PstI digestion to leave a blunt ended DNA with an intact EcoRI restriction site giving the fragment 8, containing the origin of replication of pox322. The blunt ended HBsAg gene fragment A prepared above was ligated to the EcoRI site of fragment B. This was accomplished in a three fragment ligation to create a plasmid pHS94. The third fragment fragment C was prepared as follows 2. The HBsAg gene from the plasmid pHBV T lA was cleaved withHpaI at a site distal to the HBsAg gene. The HpaI site was ligated to a EcoRI site of pBR322 previously filled in with DNA polymerase I Klenow fragment 16 .This was accomplished by subcloning the derivative of pBR322 to give pHS42. This plasmid was cleaved with XbaI which cleaves at codon 31 and with Dam HI which cleaves 375 base pairs from the EcoRI site of pBR322 to give a DNA fragment containing most of the HBsAg gene, ca. 150 base pairs distal to the HBsAg gene, and the promotor operator and the first 200 base pairs of the tetracycline resistance gene. The DNA fragment C, bounded byXbaI and BamHI was isolated by PAGE and used in the three fragment ligation described above to give the plasmid pHS94. Fig. A . Successful expression of a heterologous gene in yeast requires that transcription of that gene end with a sequence ofDNA which allows for proper processing polyadenylation and transport of that transcript. DNA RESTRICTION AND MOOIFICATIOtl ENZYMES Restriction enzymes EcoRI and HindIII along with bacterial alkaline phosphatase were purchased from Bethesda ResearchLaboratories. DNA polymerase Klenow fragment was purchased from Boehringer Mannheim. T4 polynucleotide kinase, ATP and the deoxynucleoside triphosphates dATP, dGTP, dCTP and dTTP were purchased from PL Biochemicals. All other DNA restriction and metabolic enzymes were purchased from New England Biolabs. Ey 32P ATP was purchased from Amersham Corp. DNA restriction and metabolic enzymes were used in conditions exactly described by their respective manufacturers.DNA Preparation and Transformation Purification of covalently closed circular plasmid DNAs from E. coli 17 and yeast 18 plus the transformation ofE. coli 19 and yeast 20, 21 were performed as previously described. E. coli miniscreens were done as previously described 22 .Strains and Media E. coli K 12 strain 294 ATCC No. 31446 23 was used fpr all bacterial transformations. Yeast strain XV610 8C having the genotype leu2 trpi ade6 and or ade2, lysl canl was used as yeast cloning host. ATCC No. 20622 . LB rich medium was prepared as described by Miller 24 with the addition of either 20 g ml ampicillin Sigma or 20 g ml tetracycline Sigma . Restriction Map and Partial Sequencing of 3.1 kbInsert of pB1 300 g of pod1 5 was exhaustively digested with HindIII in a 5001 reaction volume, then electrophoresed on a 1 percent agarose preparative agarose Sea Kem gel. The 3.1 kb Hindlil insert was cut from the ethidium stained gel, electroeluted 25 , 2x extracted with equal volumes of buffer saturated phenol and chloroform before ethanol precipitation. Portions of the resuspended DNA fragment was divided up and subjected to restriction cuts with a group of different restriction enzymes to yield the partial restriction map depicted in Figure 1. 30 ,ag of the purified 3.1 kb insert was cut with Sau3A then run on a 6 percent acrylamide gel. Fragments corresponding to the 265 bp and 141 bp were separately purified by elecroelution as described above. Each DNA fragment was then subjected toDNA sequence analysis 25 . A portion of this DNA sequence is shown In Figure 2. Amino acids corresponding to the N terminal amino acids of the PGK structural gene are printed above the DNA sequence.Insertion of a Restriction Site in the PGK 5 Promoter Region A synthetic oligonucleotide with the sequenceS ATTTGTTGTAAA3 was synthesized by standard methods 26 .100 ng of this primer was labelled at the S end using 10 units of T4 polynucleotide kinase in a 20 ul reaction also containing 200 Ci of Ey32 P ATP. This labelled prinersolution was used in a primer repair reaction designed to be the first step in a multi step process to put an EcoRI restriction site in the PGK S flanking DNA just precedingPGK structure gene sequence. The multistep process is explained below Step 1 Figure 3 Primer repair reactions and cloning of 39bp XbaI to Sau3A PGK piece 100 u9 of pB1 was completely digested with HaeIII then run on a 6 percent polyacrylamide gel. The uppermost band on the ethidum stained gel containing PGK promoter region was isolated by electroelution as described above.This 1200 bp HaeIII piece of DNA was restricted with HindII then run on a 6 percent acrylamide gel. The 650 bp band was isolated by electroelution. 5 9 of DNA was isolated. This 650 bp HaelII to HindII piece of DNA was resuspended in 20 iil dIH20, then mixed with the 20 rl of the phosphorylated primer solution described above. This mixture was 1X phenol chloroform extracted then ethanol precipitated. Dried DNA was resuspended in 50 gl of H20 and then heated in a boiling water bath for seven minutes. This solution was then quickly chilled in a dry ice ethanol bath 10 20 seconds then transferred to an ice water bath. To this solution was added 50 ssl of a solution containing 10 Rl of lOX DNA polymerase I buffer Boehringer Mannheim , 10 pl of a solution previously made 2.5mM in each deoxynucleoside triphosphate dATP, dTTP, dGTP and dCTP , 25 gl of dIH20 and 5 units of DNA Polymerase I, Klenow fragment. This 100 gl reaction was incubated at 37 C for 4 hours. The solution was then 1X phenol chloroform extracted, ethanol precipitated, dried by lyophilization then exhaustively restricted with 10 units of Sau3A. This solution was then run on a 6 percent acrylamide gel. The band corresponding to 39 bp in size was cut from the gel then isolated by electroelution, as described above. This 39 bp band has one blunt end and oneSau3A sticky end. This fragment was cloned into a modified pFIFtrp69 vector 11 . 10 g of pFIFtrpG9 was linearized withXbaI, 1X phenol chloroform extracted, then ethanol precipitated. The XbaI sticky end was filled in using DNAPolymerase I Klenow fragment in a 50 all reaction containing 250 M in each nucleoside triphosphate.This DNA was cut with BarrHI then run on a 6 percent acrylamide gel. The vector fragment was isolated from the gel by electroelution then resuspended in 20 sil dIH20. 20 ng of this vector was ligated with 20 ng of the 39bp fragment synthesized above for 4 hours at room temperature. One fifth of the ligation mix was used to transform . coli strain 294 to ampicillin resistance on LB 20 glg ml amp plates. Plasmids from the transformants were examined by a quick screen procedure 22 . One plasmid, pPGK 39 was selected for sequence analysis. 20 g of this plasmid was digested with XbaI, ethanol precipitated then treated with 1000 units of bacterial alkaline phosphase at 58 C for 45 min. The DNA was 3X phenol chloroform extracted, then ethanol precipitated.The dephosphorylated ends were then labeled in a 20 l reaction containing 200 zCi cf 32 P ATP and 10 units of T4 polynucleotide kinase. re plasmid was cut with Sall and run oa a 6 percent acrylamide gel. The labeled insert band was isolated from tie ge and sequenced by the chemical degradation method 25 . he ttA sequence at the 3 end of this promoter piece was as expected. Step 2 Figure 4 Construction of 312 bp PvuI to EcoRI PGK Promoter Fragment 25 g of pPGK 39 Fig. 3 was simultaneously digested withSal and XbaI 5 units each then electrophoresed on a 6 percent gel. The 390 bp band countaining the 39 bp promoter piece was isolated by electroelution. The resuspended DNA was restricted with Sau3A then electrophoresed on an 8 percent acrylamide gel. The 39 bp PGK promoter band was isolated by electroelution. This DNA contained 39 bp of the S end of thePGK promoter on a Sau3A to Xbal fragment. 25 g of pB1 was restricted with PvuI and KpnI then electrophoresed on a 6 percent gel. The .8 kbp band of DNA was isolated by electroelution, then restricted with Sau3A and electrophoresis on a 6 percent gel. The 265 bp band from thePGK promoter figure 1 was isolated by electroelution. This DNA was then ligated with the 39 bp promoter fragment from above for to hours at room temperature. The ligation mix was restricted with XbaI and PvuI then electrophoresed on a 6 percent acrylamide gel. The 312 bp Xba to PvuI restriction fragment was isolated by electroelution, then added to a ligation mix containing 200 ng of pBR322 previously isolated missing the 162 PvuI to PstI restriction fragment and 200 ng of the XbaI to pst I LeIFA cDNA gene previously isolated from 20 tig of pLEIFtrpA. This 3 factor ligation mix was used to transform E. coli strain 294 to tetracycline resistance.Transformant clonies were miniscreened and one of the colonies, pPGK 300 was isolated as having 304 bp of PGK 5 flanking DNA fused to the LeIFA gene in a pBR322 based vector. The 5 end of the LeIfA gene has the following sequence 5 CTAGAAATTC 3 , thus fusion of the XbaI site from the PGK promoter fragment into this sequence allows for the addition to the XbaI site anEcoRI site. pPGK 300 thus contains part of the PGK promoter isolated in a PvuI to EcoRI fragment.Step 4 Fig. 5 Construction of a 1500 bp EcoRI to EcoRI PGK Promoter Fragment 10 up of pB1 was digested with PvuI and EcoRI and run on a 6 percent acrylamide gel. The 1.3 kb PvuI to EcoRI DNA band from the PGK 5 flanking DNA was isolated by electroelution.10 yg of pPGK 300 was digested with EcoRI.and PvuI and the 312 bp promoter fragment was isolated by electroelution after electrophoresing the digestion mix on a 6 percent gel. 5 gg of pFRL4 was cut with EcoRi, ethanol precipitated then treated with bacterial alkaline phosphatase at 68 for 45 minutes.After 3X phenol chloroform treating the DNA, ethanol precipitation, and resuspension in 20 ml of dIH20 200 ng of the vector was ligated with 100 ng of 312 bp EcoRI to PvuI DNA from pPGK 300 and 100 ng of EcoRI to Pvul DNA from pB1. The ligation mix was used to transform E. coli strain 294 to ampicillin resistance. From one of the ApR colonies was obtained pPGK 1500. This plasmid contains the 1500 bp PGK promoter fragment as an EcoRI to EcoRI or HindIll to EcoRI piece of DNA. Construcion of expression vector Figure 6 20 019 of pFRL4 was digested with HindlII and BglII then electrophoresed on a 6 percent arylamide gel. The 230 bp fragment from the T.I gene containing part of the structural gene dnd the 3 untranslated region 28 was isolated by electroelution from the gel slice. This is the yeast terminator restriction fragment. 10 9 of pHS94 was restricted with HindIII and EcoRI, run on a 6 percent acrylamide gel. The 870 bp band representing the HBsAg gene was isolated by electroelution. 3 9 of the HbsAg gene was ligated wth 1 9 of the TRP1 terminator for 2 hours at room temperature. This solution was then restricted with EcoRI and II and electrophoresed on a 6 percent acrylamide gel. The 1.1 kb band corresponding to 5 HBsAg TRP1 3 DNA fragment was isolated by electroelution.50 9 of pPGK 1500 was restricted with HindIlI and EcoRI then electrophoresed on a 6 percent acrylarnde gel. The 1.5 kbp PGK promoter fragment was isolated by electroelution. 10 pg of pCV13 was restricted with HindIII and BamHI then electrophoresed on a 1 percent agarose gel. The large vector fragment was isolated by electroelution. 0.9 rg of pCV13 HindlII to BamHI , 150 ng of HindIII to EcoRI PGK promoter fragment, and 100 ng of EcoRI to BglII HbsAg TRP1 fusion were ligated for 12 hours at 16 C and then used to transform E. coli strain 294 to ampicillin resistance. Miniscreen analysis of a number of transformants indicated that some colonies contained the proper vector construction with promoter HBsAg structural gene TRP1 terminator. Several of these plasmids were used td transform yeast to Leu . Transformation of yeast and assay for HbsAg Yeast strain XV610 8C was grown in YPD 29 , prepared for transformation using standard procedures 20, 21 , and transformed to leucine prototrophy Leu using plasmid pCV13 or pYeHBs on minimal YNB leu See Strains and Media . Transformant colonies were prepared for HBsAg radioimmune assay Abbott Labs as follows 1 Grow 10 mis of yeast in YNB leucine at 30 C with aeration to an absorbance at 660 m of 1.0. 2 Spin culture 5 min at 5000 xg. Discard supernatant. 3 Resuspend cells in 500 wl of PBS 20my sodium phosphate at pH 7 plus 0.14 m Nail , add 1.5 gram sterile glass beads .45 .50 mm and vortex 5 min keeping cold by putting on ice intermittently . 4 Spin cell extract 5,000 kg for 3 min at 4 C. Use 200 pl of cell extact for standard radioimmune assay. Cell extract was diluted using PBS. Evidence for HBsAg Expression in Yeast by pYeHBs pYeHBs and pCV13 were used to transform yeast strain XV610 8C to leucine prototropy in YNB leu minimal media. The presence of the plasmid was verified by growing the yeast on plates with no leucine selection YPD to colonies followed by repli aplating to YNB leucine plates 2 percent agar . Fifty to 60 percent of the colonies that grew on YEPD grew on YNB leu demonstrating that 50 60 percent of yeast contained pCV13 or pYeHBs. It should be noted that a Leu revertant would show growth of all colonies on YNB leu. PCV13 stability in the yeast was no greater than pYellBs, suggesting that the production of HBsAg as discussed infra. does not put pressure on cells to restrict copy number. Transformant yeast were grown in liquid leucine selection media YNB leu then broken open by vortexing with a glass bead suspension infra. . A radioimmune assay was used to assessHBsAg production in the cleared yeast extract. The 22nm HBsAg particle is roughly 1000 times more antigen then the HBsAg monomers alone and we used antibody prepared against this particle.The RIA on the yeast extract indicates the production of HBsAg from pYeHBs in yeast. EpCVl3 transformed yeast extracts produce no HBsAg antigen material. Characterization of HBsAg In Yeast 1 200 l of yeast extract prepared as described above was layered on top of a 5 ml linear density gradient consisting of 5 2G percent sucrose w v in 20 mM tris HCl pH 7.4 , 0.5 rM EDTA, and 0.5M NaCl then centrifuged in a SWS0.1 rotor at 45, 000 rpm for 2 hours.Fractions were collected and assayed for HBsAg by radioimmune assay. 2 203 yl of extract from the same source as above was layered on top of a 5 ml solution containing 20mM. Tris HC1 ph 7.4 , Q.5mM EDTA, and CsCl density 1.2g cc then centrifuged in a SW50.1 rotor at 45,000 rpm for 70 hours. HBsAg was assayed as described above and the density of CsCl was determined by recording the refractive index of each gradient fraction. Hepatitis surface antigen is synthesized and secreted from liver cells as a particle 22 nm particle . In the construction described above, only the coding region representing the mature structural gene plus some 3 untranslated sequences were included. We assessed the particulate nature of the hepatitis surface antigen monomer within the yeast ceil. To this end, aiquots of a yeast cell extract containing HBsAg were subjected to sedimentation velocity and sedimentation equilibrium analysis. As shown inFigure 7, the hepatitis surface antigen synthesized in yeast cell has a sedimentation rate that is virtually identical to that of the 22 nm particle form of HBsAg obtained from a cell line PLCWIII secreting authentic 22 nm particles. HBsAg synthesized in yeast has a buoyant density of 1.18 g cm3 as determined by sedimentation equilibrium analysis.This value is slightly lower than that of HBsAG synthesized in the infected liver cell.Vaccine Preparation Vaccines incorporating theHBsAg produced as herein described, can be prepared according to known methods, wherein said HBsAg is combined in admixture with a suitable vehicle. Suitable vehicles include, for example, saline solutions, various known adjuvants, or other additives recognized in the art for use In compositions applied to prevent viral infections. Such vaccines will contain an effective amount of the HBsAg hereof and a suitable amount of vehicle in order to prepare a vaccine useful for effective administration to the host. Attention is also directed to New Trends and Developments in Vaccines, Editors A. Roller andH. Friedman, University Park Press, Baltimore, 1978, which is hereby incorporated by reference, for further background details on the preparation of vaccines. Bibliography 1. Broach, J. R., Strathern, J. N., and Hicks, J. a. Gene 8, 121 133 1979 .2. Ratzkin, 8., and Carbon, J. Proc. Natl. Acad. Sci. U.S.A. 74, 487 491 1977 .3. Chinault, A. C., and Carbon, J. Gene 5 111 126 1979 . 4, Hinnen, A., Hicks, J. B., and Fink, G. R. Proc. Natl. Acad. Sci. U.S.A. 75, 1929 1933 1978 . 5. Iitzeman, R. A., Clarke, L., and Carbon, J. J. Biol. Chew. 255, 12073 1980 . 6. Hess, B., Boiteux, A., and Kruger, J. Adv. Enzyme Regul. 7, 143 167 1958 . 7. Holland, J. J., and Holland, J. P. Biochemistry 17, 4900 4907 1978 . 8. Bostian, K. A., Lemire, J. M., Cannon, L. E., and Halvorson, H. 0. Proc. ttatl. Acad. Sci. U.S.A. 77, 4504 4508 1980 .9. Abstracts of papers presented at the 1981 meeting on The Molecular Biology of Yeast August 11 August 15, 1981 Cold Spring Harbor Laboratory, Cold Spring Harbor, t ew Ycrk. 10. Chambon, P. Ann. Rev. Biochen. 44, 613 538 1975 . il. Goeddel, D. V., Shepard, H. M., Yelverton, E., Leung, D., and Crea, R. Nucleic Acids Research 8, 4057 4073 1980 .12. No reference.13. Goeddel, D. V. et. al. Nature 287 411 416 1980 .14. P. Valenzuela, et. al. Nature, 280, 815 819. 1979 .15. P. Charnay et. al. Proc. Natl. Acad. Sci. U.S.A. 76, 2222 2226 1979 .16. H. Jacobsen et. al. Eur. J. Biochem. 45, 623. 1974 .17. Clarke, L., and Carbon, J. Cell 9, 91 99 1976 .18. Davis, R. N., Thomas, M. Cameron, J., St. John, T. P., Scherer, S., and Padgett, R. A. Methods in Enzymology 65, 404 411.19. Clark, L., and Carbon, J. Proc. Natl. Acad. Sci. U.S.A. 72, 4361 4365 1975 .20. Beggs, J. D. Nature 275, 104 109 1978 .21. Hinnen, A., Hicks, J. B., and Fink, G. R. Proc. Natl. Acad. Sci. U.S.A. 75, 1929 1933 1978 .22. Birnbo√≠m, H. C., and Poly, J. Nucleic Acids Res. 7, 1513 1523. 23. Bachnan, K., Ptashne, M., and Gilbert, W. Proc. Natl. Acad. Sci. U.S.A. 73, 4174 4178 1976 .24. Miller, J. H. Experiments in Molecular Genetics, pp. 431 433, Cold Spring Harbor Laboratory, C,ld Spring Harbor, N. Y.25. Maxim A. M., and Gilbert, W. Methods in Enzymol. 65, 490 565 1980 .26. Crea, R. and Horn, T. Nucleic Acids Res. 8, 2331 2348 1980 .27. Bolviar, F., Rodriguez, R. L. Green, P. Y., Betlach, M. C., Heyneker, H. L., Boyer, H. W., Crosa, Y. H., and Falkow, S. Gene 2, 95 113 1977 .28. Tschumper, G., and Carbon J. Gene 10, 157 166 1980 . 29. Laboratory Manual for a course Methods In Yeast Genetics Cold Spring Harbor Laboratory, Sherman, Fink, and Hicks, Cold Spring Harbor, New York.