# DNA molecules comprising the genes for preprochymosin and its maturation forms, and microorganisms transformed thereby.

## Abstract
The present invention relates to recombinant DNA and plasmids comprising specific structural genes of mammalian origin coding for the various allelic and matu ration forms of preprochymosin, particularly those of bovine origin, and the use of said recombinant plasmids to transform microorganisms in which said genes are expressed.

## Claims
CLAIMS 1. DNA sequences encoding various allelic and maturation forms of mammalian preprochymosin according to Fig. 1 and 2. 2. DNA sequences encoding various allelic and maturation forms of mammalian preprochymosin according to Fig. 19, 20, 21 and 21. 3. A recombinant plasmid comprising i a structural gene coding for the various allelic and maturation forms of mammalian preprochymosin according to Fig. 1 and 2 or 19 22. ii specific DNA sequences regulating the expression of said gene in a microbial host, consisting of an inducible or constitutive regulon. 4. A recombinant plasmid according to Claim 2, wherein the regulon consists of a double lac UV5 system according to Fig. 8. 5. A recombinant plasmid according to Claim 3, comprising a regulon consisting of at least one modified tryptophan system according toFig. 9, wherein the information coding for the trp attenuator protein is eliminated while its ribosome binding site is maintained. 6. A recombinant plasmid according to Claim 5, comprising two trp regulons linked in a head to tail fashion according to Fig. 11. 7. A recombinant plasmid according to Claim 3, comprising a regulon consisting of at least one modified promotor ribosome binding site of en VIII of bacteriophage M13, according to Fig. 10. 8. A recombinant plasmid according to Claim 2, comprising a thermosensitive replication mutant derived from the cloacin D 13 pla iid pVU 208 according to Fig. 12 and 13. 9. A recombinant plasmid according to Claim 1, selected from pUR 1521 ATCC 39120 , i531, 1541, 1731, 1821, 1831, 1841, 1931 pll 152, 1532, 1542, 732, 1822, 1832 ATCC 39197 , 184., 1932 jUP 1523, 1533 ATCC 39121 , 1543, 1733, 1823, 1833, 1843, 1933 pUR 1524, 1534, 1534, 1544, 1734 ATCC 3919, 1824, 1834, 4, 1944 10. A bacterial culture comprising E coli cells comprising a according tu Claim 9. 11. Micro organisms, particularly non toxic, edible micro organisms such as streptococci or lactobacilli, or micro organisms of bacillus or yeast origin, in which are incorporated a DNA sequence according to Claim 1 or Claim 2, and an appropriate regulon, suitable for the production of bovine calf preprochymosin or any one of the various allelic and maturation forms thereof. CLAIMS 1. A process for producing D sequences encoding various allelic art maturation forms of l reprochyl losin according to Fig. 1 and 2 orFig. 19 22, characterized in that a the message RNA mRNA cf preprochymosin is isolated and purified b mRNA is converted into double stranded DNA dsDNA c said dsDNA is incorporated in an appropriate plasmid d said plasmid is transferred into bacterial cells and hybrid plasmid DNA molecules are selected to yield a plasmid containing a substantially complete copy of the preprochymosin mRNA e this DNA is further genetically engineered so as to code exactly for preprochymosin or its maturation forms and brought under transcriptional control of appropriate regulons. 2. A process fo producing a recombinant plasmid, characterized by combining i a structural gene coding for the various allelic and maturation forms of mammalian preprochymosin according to Fig. 1 and 2 or 19 22 with ii specific DNA sequences regulating the expression of said gene in a microbial host, consisting of an inducible or constitutive regulon. 3. A process according to Claim 2, characterized in that the regulon consists of a double lac UV5 system according to Fig. 8. 4. A process according to Claim 2, characterized in that a regulon is used which consists of at least one modified tryptophan system according to Fig. 9, wherein the information coding for the trp attenuator protein is eliminated while its ribosome binding site is ma ntai ned. 5. A process according to Claim 4, characterized in that 2 trp regions are used which are linked in a head to tail fashion according to Fig. 11. 6. A process according to Claim 2, characterized in that a regulon is used which consists of at least one modified promotor ribomome binding site of en V 11 of bacteriophage M13, according no J. 10. 7. A process according to claim characterized in that a thermo sensitive replication mutant is used which is derived from the cloacin DP13 plasmid pVU 208 according to Fiy. 12 and 13. 8. A process according to Claim 2, characterized in that a plasmid is used which is selected from pUR 1521 ATCC 39120 , 1531, 1541, 1731, 1821, 1831, 1841, 1931 pUR 1522, 1532, 1542, 1732, 1822, 1832 ATCC 39197 , 1842, 1932 pUR 1523, 1533 ATCC 39121 , 1543, 1733, 1823, 1833, 1843, 1933 pUR 1524, 1534, 1534, 1544, 1734 ATCC 39198 , 1824, 1834, 1844, 1944 9. A process for producing mammalian chymosin, characterized in that a a plasmid according to Claim 2 is incorporated in a microorganism b the transformed micro organism is cultivated and c chymosin produced by said micro organism is isolated. 10. A process according to Claim 9, characterized in that the microorganisms consist of E coli cells. 11. A process according to Claim 9, characterized in that non toxic edible micro organisms such as streptococci or lactobacilli, or microorganisms of bacillus or yeast origin are used.

## Description
DNA MOLECULES COrlPRISING THE VARIOUS ALLELIC FORMS OF THE STRUCTURALGENES ENCODING THE MtNALIAN PROTEINS PREPROCHYMOSIN, PROCHYMOSIN,PSEUDOCHYMOSIN AND CHYMOSIN RECOMBINANT DNA CLONING VEHICLES COMPRISING SAID DNA MOLECULES, MICROORGANISMS TRANSFORMED THEREBY ANDSYNTHESIS OF SAID PROTEINS IN THESE MICROBIAL HOSTSThe present invention relates to recoribinant DNA and plasmids comprising specific structural genes of mammalian origin coding for the various allelic and maturation forms of preprochymosin, particularly those of bovine origin, and the use of said recombinant plasmids to transform microorganisms in which said genes are expressed. Chymosin is a protein originating from the stomach of newborn mammals.The bovine type EC 3.4.23.4 is secreted as an inactive precursor, prochymosin, which consists of a single polypeptide chain of 3G5 amino acid residues B. Foltmann et al. Proc. Natl. Acad. Sci. USA 74, 2321 2324, 1977 B. Foltmamct. al. J. Biol. Chemistry 254, 8447 8456, 1979 . Applicants have found that the precursor of bovine prochyniosin, preprochymosin, consists of a single peptide chain of 381 amino acid residues and differs from prochyinosin in an amino terminal of 16 amino acids. This extension very much resembles a signal sequence which is involved in the process of cotranslational excretion G.Blobel BR U.Dobberstein, J. Cell. Biol. 67, 835 851, 1975 .Prochymosin is irreversibly converted into active enzyme chymosin by limited proteolysis, during which a total of 42 amino acid residues are released from the amino terminal part of the peptide chain. The activation is effected through a pH dependent two step autocatalytic conversion.The intermediate, pseudochymosin, is formed by proteolytic cleavage of bond 27 28 at pH 2 3. The final product, chymosin, is formed by activation of pseudochymosin at pH 4 5. V. Barkilolt Petterseil t al., Eur. J. ioc iem., 94, 573 50, 1979 .The enzyratic activity of chymosin consists of the specific proteolysis of K casein. This property makes chymosin widely used as milk clotting enzyme in cheese manufacture.Chymosin is the essentidl milk clotting component of rennet, the crude extract of the abomasum of bovine calves. Punnet is used in the produc tion of several types ofi cheeses in almost all parts of the worldBecause of the ever increasing shortage of calf rennet created by the increasing 4emand tor cheese, many laboratories have been searching for substistutes of microbial origin. Many microbial proteases have been screened, only a few could be used in cheese making, and even these substitutes exhibit a different specificity and therefore may cause an unacceptable texture and or bitter taste of the cheese.So the production of chymosin by recombinant DNA containing microorganisms is expected to become of great economical importance.Developments in recombinant DNA technology have made it possible to isolate or synthesize specific genes or portions thereof from higher organisms, such as man and other mammals, and to transfer these genes or fragments to microorganlsms such as bacteria or yeasts. The transferred gene is replicated and propagated as the transformed microorganism replicates. As a result, the transformed microorganism may become endowed with the capacity to make whatever protein the gene or gene fragment encodes, whether it is an enzyme, a hormone, an antigen, an antibody, or a portion thereof. The microorganism passes on this capability to its progeny, so that in effect, the transfer has resulted in a new microbial strain, having the described capability.See, for example, Ullrich, A. et al., Science 196, 1313 1977 , andSeeburg, P.H. et al., Nature 270, 486 1977 . A basic fact underlying the application of this technology for practical purposes is that DNA of all living organisms, from microbes to man, is chemically similar, being composed df the same four nucleotides. The significant differences lie jin the sequences of these nucleotides in the polymeric DNA molecule. The nucleotide sequences are mainly used to specify the amino acid sequences of proteins that comprise the organism.Although most of the proteins of different organisms differ from each other, the coding relationship between nucleotide sequence and amino acid sequence is fundanBntallyr the came for Cell organisms. For example, the same nucleotide sequence which codes for the amino acid sequence cf HGH in human pituitary cells, will, when transferred to a microorganism, be recognized as coding for the same amino acid sequence.For ecdnomic asoi s it is important tiat proteins encoded by the recoiiibinant DNA gene are produced under optimal conditions in approved edible micro organisms as host cells. The main routes to achieve this are 1 integration of the structural gene down stream of an effective regulon, in such a way that under selected growth conditions, the amount of protein produced per cell by an optimal number of cells is as high as possible.For that purpose regulons like the double lac UV5, the trp regulon, the double trp regulon of E. coli and the regulon of the gene VIII product of the bacteriophages M13, fd and fl are, amongst others, adequate in their natural state or in their modified form s .Another factor influencing the yield of required protein per cell is the increase amplification of the copy number of the plasmids containing the above described regulons and the structural genes. Amplification can be effected by the use of a thermosensitive replication mutant derived from the cloacin DF 13 plasmid pVU 208 . 2 excretion of said protein by microbial host cells into their periplasmic space and or into the culturing medium, talus preventing said protein from intracellular degradation or preventing the disturbance of normal cellular processes due to too high an intracellular level of said protein. It is now generally accepted that in many prokaryotic and eukaryotic cells a special NH2 terminal amino acid sequence of the unprocessed form of the proteins is involved in the protein excretion process.G.Blobel B.Dobberstein 1975 , J. Cell Biol. 67, 835 851.In the present invention use is made of recombinant DNA and other molecular biological techniques to construct recombinant DNA molecules that fulfil the above described requirements. The present invention is also related to the change of the yenetic information of structural genes using site directed mutagenesis. For a better understanding of the invention the most important tenns used in the description will be definedAn operon is a gene comprising a particular D see once structural gene s for polypeptide s expression and a control reniol or reculo regulating said expression ai ic niostly consisting of a promoter sctlu2rlce, an operator sequence, a ribosonle binding or interactior UNA sequence.Structural genes are DNA sequences which encode through a template mRNA a sequence of amino acids characteristic for a specific polypeptide.A promoter is a DNA sequence within the requlon to which RNA polymerase binds for the initiation of the transcription.An operator is a DNA sequence within the regulon to which a repressor protein may bind, thus preventing Ri 4A polymerase from binding to the adjacent promoter.An inducer is a substance which deactivates a repressor protein, freeing the operator and permitting RNA polymerase to bind to the promoter and start transcription.Cloning vehicle. A non chromosomal double stranded DNA, plasmid or phage, comprising a DNA sequence intact replicon that allows selfreplication after transformation into suitable host cells.Phage or bacteriophage. Bacterial virus which can replicate in a suitable bacterial host cell.Reading frame. The grouping of triplets of nucleotides codons into such a frame that at mRNA level a proper translation of the codons into the polypeptide takes place.Transcription. The process of producing RNA from a structural gene.Translation. The process of rroducing a polypeptide from mRNA.Expression. The process undergone by a structural gene to produce a polypeptide. It is a combination of many processes, including at least transcription and translation.Thermosensitive replication mutant. A plasmid containing a mutation in its replication origin which causes. its copy number to be temperature dependent. Allelic foci. Ore of two or more naturally occurring alternative forms of a gene product.Maturation form. One of two or more naturally occurring forms of a gene product procured by specific processing, e.g. specific proteolysis.Plus strand. DNA strand whose nucleotide sequence is identical with the nRNA sequence, with the proviso that uracil is replaced by thymidine.By maturation forms of preprochymosin are meant prochymosin, pseudochymosin and chymosin.Prochymosin arises through the action of signal peptidase on preprochymosin, which causes the loss of the amino terminal excretion related signal sequence.The chemical structure of bovine preprochymosin is given in Fig. 1.Bovine prochymosin corresponds with residues 1 365 Fig. 1 .Pseudochymosin arises through the autocatalytic proteolysis of prochymosin at pH 2, causing the loss of an amino terminal portion of prochymosin. The chemical structure of bovine pseudochymosin corresponds with residues 28 365 of preprochymosin given in Fig. 1.Chymosin arises through the autocatalytic proteolysis of pseudochymosin at pH 4 5, causing the loss of an amino terminal portion of pseudo chyiaosin. The chemical structure of bovine chymosin corresponds with residues 43 365 of preprochymosin given in Fig. 1.According to the invention there is provided structural gene s coding for the various allelic and maturation forms of mammalian preprochymosin, particularly bovine preprochymosin according to Fig. 1 and 2 and further a recombinant plasmid comprising i structural gene s coding for the various allelic and maturation forms of ammalian preprochymosin, particularly bovine preprochymosin according to Fig. 1 and 2 ii specific DNA sequences which regulate the expression of said structural genes in a microbial host. These specific DNA sequences consist of either an inducible or a constitutive regulon. A preferred inducible rejulon consists of a double lac UV5 system as described by D.V.Goeddel ct .4C.s Nature 281, 544 548 1978 . See Fig. 8 . Anotner nreferred inducible regulon is a constituent of the tryptophan system described by F. Lee et al., J. Mol. Biol 121, 193 217 1978 and t. Bertrand S ., Science iS9, 22 20 1975 . Applicants have od fied this tryptophan system to obtain a more adequate system according to Fig. 9 . In this modified system the information coding for the trp attenuator protein is eliminated while ilaintaining its ribosome binding site. Expression is highly increased when two trp regulons in a head to tail fashion are being used.Synthesis of this system is illustrated in Fig. 11.The recombinant plasmid according to the invention may comprise DNA sequences which regulate the expression of the structural genes, preferably consisting of a modified promoter ribosome binding site of gene VIII of bacteriophage M13, fd or fl P.M.G.F. van Wezenbeek ot aZ., Gene 11, 129 148 1980 .Efficiency of required protein per cell was highly increased when in conjunction with the regulon systems described above, the copy number of the recombinant cloning vehicle was increased. This was effected by the use of a thernosensitive replication mutant derived from the cloacin DF13 plasmid pVU 208 A. Stuitje, thesis, V.U. Amsterdam, 1981 .Increase in temperature results in a ten to hundred fold increase in copy number. The construction of such plasmids is illustrated in Fig.12 and 13.In the recombinant plasmid according to the invention the regulon may be either directly linked to the structural gene or indirectly through a novel start codon and EcoRI site containing DNA linker comprising the nucleotide sequence 5 p CAT N nGAATTC N nATG 3 wherein n 0, 1, 2 or 3 and OH N and N are any of the nucleotides A, T, G or C, with the proviso that in the double stranded structure N and N are such that a rotational symmetrical structure is present. By a rotational symmetrical structure is meant that where N is e.e represented by A, N should be represented by the complementary base T.In some instances it turned out that the yield of expression imprcved when the sequence AATT between the regulon and the structural gene has been eliminated. Aílelic forms of bovine preprochymosin have been constructed departing from the nucleotide sequence given in Fig. 1. Examples of these con struotions are outlined in Fig. 18 and construction step 10e described further in the Specification, whereby use is made of site directed mutagenesis. The latter procedure could also be applied usefully so as to produce preprochymosin with a specifically altered signal sequenece which allows efficient excretion in microbial hosts and prochymosin with an i nip roved acid induced autocatdlytic proteolysis characteristic.The microbial cloning vehicles containing the structural genes encoding the various allelic and maturation forms of preprochymosin according to the invention are produced by a number of steps, the most essential of which are 1. Isolation and enrichment of the messenger RNA nlRNA of prepro chymosin. 2. Conversion of this mRNA into double stranded DNA dsDNA 3. Construction of dsDNA having a poly dC tail 4. Incorporation of the dsDNA poly dC molecules in PstI cleaved and poly dG tailed pBR 322 DNA.5. Transformation of competent E. coZi cells and selection of tetra cycline resistant colonies.6. Determination of the nature of the inserts by RNA DNA hybridization and o71 vitro translation and by DNA DNA hybridization using a specific 32P labelled cDNt probe.7. Double checking the nature of the cloned PstI inserts by DNA and RNA sequence analysis.8a. Producing DNA encoding the amino terminal part of pseudochymosin plus a translational initiation ATG codon added at the amino terminus.8b. Producing DNA encoding the amino terminal part of chymosin plus a translational initiation ATG codon at the amino terminus.8c. Producing DNA encoding the amino tenninal part of prochymosin plus a translational initiation ATG codon at the amino terminus.8d. Producing DNA encoding the amino terminal part of preprochymosiii plus a transitional initiation ATG codon at the amino terminus.9. Construction of plasmids comprising a constitutive or inducible regulon, with or without a thermosensitive replication mutation. 10. Construction of plasmids consisting of plasmids described under 9 and the ligated preproehymosin gene or its various maturation forms, and transformation of said plasmids into microbial host cells. 11. Culturing of microbial cells, e.g. P . 0 it cells, containing recombinant plasmids described under 10 and detection and iso lation of preprochymosin, prochymosin, pseudochymosin or chymosin therefrom. Culturing conditions were optimized as to the yield of preprochymosin or its maturation forms per cell. Conversion into enzymatically active chymosin of the various precursors was also optimized.The following description will illustrate the abo e l,lentioned steps in detail.1. Isolation and purification of bovine preprochymosin mRNA The fourth stomach of a preruminant calf abomasum, Frisian cow was ground under liquid nitrogen, extracted with phenol and a selective precipitation of the RNAs with LiCl was performed following the procedure described by K.S. Kirby, Biochem. J. 96, 266 269 1965 , U. Wiegers H. Hilz FEBS. Letters, 23, 77 82 1972 . PolyA containing mRNA was recovered by several passages over oligo dT cellulose columns as described by H. Aviv P. Leder Proc. Natal. Acad. Sci. USA, 69, 1408 142 1972 .2. Conversion of prepro chymosin mRNA into double stranded DNA Purified prepro chymosin mRNA was copied with AMV reverse tran scriptase to yield a single stranded DNA molecule, according to the procedure described by G.N. Buell et aZ., J. Broil. Chem. 253, 2471 2482 1978 . This cDNA was subsequently converted into a double stranded molecule using DNA polymerase, according to the procedure described by A.R. Davis et al., Gene 10, 205 218 1980 . Thereafter the loop structure of the double stranded DNA copy Wa removed by SI nuclease digestion.3. Construction of double stranded DNA with poly dC tails DNA molecules of the desired length were obtained hy nol acryll ii. ie gel electrophoresis, extracted from the gel and tailed with poly dC by terminal transferase according to the procedure described by R. Roychoudhury v d ., Nucleic Acids Research 3, 863 877 1976 . 4. Integration of the dsDNA poly dC molecules in plasmid pBR 322 Plasmid pBR 322 was treated with restriction endonuclease PstI, that cleaves the plasmid at d recognition site that lies in the gene encoding tie ampicillin resistance, whereafter the linearized DNA of pBR 322 was supplied at the PstI site with poly dG tails by terminal transferase. The poly dC tailed DNA molecules were annealed to the poly dG tailed plasmid pBR 322.5. Transformation and cloiie selection The plasmids thus obtained were transferred into CaCl2 treated W. coLi cells. After transformation, cells containing hybrid plasmid DNA molecules were selected on their resistance to tetracycline. M. Mandel A. Higa, J. Mol. Biol., 53, 159 162 1970 .6. Determination of the nature of the inserts I DNA DNA colony hy bridization and RNA DNA hybridization in vitro translation Positive colonies were further selected by colony hybridization R.E. Thayer, Anal. Biochem., 98, 60 63 1979 with a radioactively labelled, chymosin specific cDNA. The latter product was obtained by priming reverse transcription see 2 of the mRNA preparation see 1 with the oligonucleotide 5 dTTCATCATGTToH 3 . This oligonucleotide was designed by the Applicants, because it represents one of the two theoretically possible nucleotide stretches coding for the unique amino acid sequence asN met met asN , which occurs in the prepro chymosin molecule at position 183 186 Fig.1 . Upon priming cDNA synthesis with this undecanucleotide a distinct cDNA product with a chain length of circa 650 nucleotides was ob tained. This product was isolated and used as a probe in the colony hybridization experiments. Several positive colonies could be ident ified and, as a double check on the identity of the cloned DNA, the plasmid DNA from some of these colonies was isolated and used in the hybridization in Vi 8 translation procedure described by J.G. Williams ct ai. Cell, 17, 903 913 1979 .7. Determination of the nature of the inserts it by DNA RNA sequence analysis Fig. 1 The nucleotide sequence analysis of the prepro chymosin inserts was performed by the chemical degradation procedure as outlined by A.M. Maxam W. Gi1bert in Methods in Enzymology, Q. Grossman BR K. Moldave editors, New York, Acad. Press, 1980, Vol. 65 1 , pages 499 560, and the dideoxy nick translation procedure as out lined by J. Maat A.J.H. Smith, Nucleic Acid Research, 5, 4537 4545 1978 .Further information on the nucleotide sequence of the prepro chymosin mRNA was derived indirectly by primed synthesis by AMV reverse transcriptase on the prepro chymosin mRNA template in the presence of chain terminating inhibitors, as outlined by D. Zimmern P. Kaesberg, Proc. Natl. Acad. Sci. U.S.A. 75, 4257 4261 1978 . This screening yielded into aiia plasmid pUR 1001 containing an almost complete copy of preprochymosin mRNA.8a. Production of DNA encoding the amino terminal part of pseudo chymosin ATG Fig. 3, 4 Numbers refer to the preprochymosin mRNA sequence in Fig. 1, unless indicated otherwise. Plasmid pBR 322 was cleaved with the restriction enzyme HaeIII and the resulting fragments were subsequently blunt end ligated with synthetic HindIII linkers 5 dCCAAGCTTGG 3 . The mixture was subsequently incubated with HindIII and phosphatase. The reactions were terminated by protein extraction with phenol chloroform 50 50 v v and the DNA was then cleaved with PstI. The resulting mixture was subjected to polyacrylamide gel electrophoresis and a 148 bp fragment fragment A, Fig. 3 extending from position 3608 3756 in the pBR 322 DNA sequence J.G. Sutcliffe, Cold Spring Harbor Symposia on Quantitative Biology, 43, 77 90 1978 was isolated from the gel by electroelution. Plasmid pUR 1001 was cleaved with EcoRI and treated with calf phosphatase. The mixture wds extracted with phenol chloroform and subsequently PstI was added. Resulting fragments were separated by agarose electrophoresis. Fragment B see Fig. 3 extending from the EcoRI site at position 549 to the PstI site in the noncoding sequence of the preprochymosin gene at the carboxy terminal end of the pUR C01 doe, ciao isolated. Fragments A and B were ligated with the large EcoRI HindIII frag ment of pUR 201, 301, 401, 303, 210, 310, 410, 311 combined called fragment C yielding pUR 1520, 1530, 1540, 1730, 1820, 1830, 1840, 1930, respectively. Subsequently plasmid pUR 1001 DNA was cleaved with PstI and result ing DNA fragments were ligated to the synthetic pentanucleotide 5 dHOCTGCA 3 . Following ligation, the mixture was incubated with E coli DNA polymerase, large fragment, in the presence of dGTP in order to make blunt ends. The DNA was phosphorylated using T4 kinase and ATP, and subsequently supplied with synthetic EcoRI linkers of the structure 5 dCAT N nGAATTC N nATG 3 , wherein n 0, 1, 2 and 3 and N and N are any of the deoxynucleotides A, C, G or T, with the proviso that in the double stranded structure N and N are such that a rotational symmetric structure is present. The DNA was then treated with EcoRI and resulted in fragment I of circa 400 bp in length, which was isolated Fig. 4 .8b. Production of DNA encoding the amino terminal part of chymosin Fig. 5 Plasmid pUR 1001 was cleaved with PstI and the 1300 bp PstI insert was isolated. This DNA fragment was heat denaturated to produce single strands. The synthetic primer 5 dGGGGAGGTGG 3 was used to produce complementary DNA synthesis starting from position 198 in the direction of the carboxy terminus by the action of EcoRI DNA polynierase, large fragment. Subsequently the DNA was treated with nuclease S1 to procure blunt ended dsDNA. To this dsDNA the synthetic EcoRI linker 5 dCAT N nGAATTC N nATG 3 was ligated. After digestion with EcoRI and treatment with phos phatase, the DNA was cleaved once more by BglII. The resulting fragment II Fig. 5 was isolated.8c. Production of DNA encoding the amino terminal part of prochyinosin Fig. 6 Plasmid pUR 1001 was treated with HphI, followed by nuclease S1. A 202 base pair long fragment III was isolated Fig. 6 . Fragment III was then ligated to the synthetic EcoRI linker 5 dCAT N nGAATTC N nATG, cleaved with EcoRI and dephospho rylated with phosphatase. The resulting DNA was digested once more by BglII and the resulting fragment IV was isolated.8d. Production of DNA encoding the amino terminal part of preprochymosin Fig. 7 Plasmid pUR 1001 was cleaved with PstI and EcoRI. A fragment of 396 bp was isolated. This fragment was then treated with exonuclease III to procure single stranded non complementary DNA A.J.H. Smith 1979 , Nucleic Acids Res. 6, 831 841 . rills DNA was hybridized to preprochymosin mRNA under conditions described by G. Akusjrvi and U. Petterson 1978 , Proc. Natl. Acad. Sci. U.S.A., 75, 5822 5826. The cDNA synthesis was performed as described under 2. Following heat denaturation, dsDNA was made using DNA polymerase, large fragment, with 5 dAGGTGTCTCGOH 3 acting as a primer. The dsDNA was treated with nuclease S1 and ligated to the synthetic EcoRI linker dCAT N nGAATTC N nATG. After EcoRI cleavage and dephosphorylation with calf intestinal phosphatase, the DNA was split once more with BglII. The resulting circa 230 bp long fragment V was isolated.9. Construction of plasmids comprising a constitutive or inducible regulon, with or without a thermosensitive replication mutation 9a. Construction of a plasmid pUR 201 Fig. 8 A fragment containing 285 base pairs comprising double lac regulon lac UV5 was obtained by restriction endonuclease EcoRI cleavage of pKB 268, described by K. Backman M. Ptashne, Cell 13, 65 71 1978 . This fragment was ligated in the EcoRI site of pBR 322 DNA. Plasmid DNA with the lac regulon in the right orientation, pUR 200, Fig. 8 was partly cleaved by EcoRI in the presence of 1 . Z i RNA polymerase. The EcoRI cleavage site most distant from the restriction endonuclease HindIII cleavage site was preferentially attacked. The linearized DNA was treated with S1 nuclease, purified by agarose gel electrophoresis, circularized by ligation with T4 DNA ligase and subsequently transformed into E. ctt. From the tetracycline resistant transformants pUR 201 with the correct structure Fig. 8 was obtained. 9b. Construction of plasmid pUR 301 Fig. 9 A DNA fragment of about 510 base pairs containing the trp reoulon was obtained by restriction endonuclease HinfI cleavage of ptrp ED5, as described by R.A. Hallewell S. Emtage, Gene 9, 27 47 1980 . This fragment was cleaved with restriction endonuclease TaqI in the presence of E. coli RNA polymerase. The TaqI site in the trp regulon described by K. Bertrand et at., Science 189, 22 26 1975 and F. Lee et aZ., J. Mol. Biol. 121, 193 217 1978 was selectively protected, thus yielding a fragment containing 234 base pairs comprising the trp regulon Fig. 9 . This fragment was then treated with S1 nuclease, blunt end ligated with the EcoRI linker 5 dGGAATTCCOH 3 , cut with EcoRI and subsequently cloned in the EcoRI site of pBR 322. Plasmid pUR 300 with the trp regulon in the correct orientation Fig. 9 was isolated. The EcoRI cleavage site most distant from the HindIII site was removed by partial cleavage of pUR 300 DNA by EcoRI in the presence of ethidium bromide and S1 nuclease treatment. Linear DNA molecules were recircularized by T4 DNA ligase. From the tetracycline resistant transformants pUR 301 with the structure as outlined in Fig. 9 was obtained.9c. Construction of plasmid pUR 401 Fig. 10 A 269 base pairs fragment comprising the gene VIII promotor was obtained by digestion of RF M13 DNA DNA sequence 1128 1379 see P.M.G.F. van Wezenbeek et aZ. Gene 11, 129 148 1980 , with the restriction nucleases TaqI and HaeIII the TaqI site was made blunt ended by a repair reaction with E. coli DNA polymerase the fragment was subsequently partly digested with restriction enzyme MnII. The partial products were treated with T4 DNA polymerase and 51 nuclease and subsequently blunt end ligated with the EcoRI linker 5 dGGAATTCCOH 3 , then treated with EcoRI and ligated in the EcoRI site of the pBR 322.By restriction enzyme analysis and DNA sequencing, a plasmid was isolated in which the EcoRI cleavage site was located just beyond the ribosome binding site of the M13 gene VIII DNA sequence. Applicants have found that the plasmids having the M13 regulon from nucleotide 1128 to nucleotides 1291 to 1297 were appropriate regulons for expression. The EcoRI cleavage site most distant from the HindIII site was removed essentially as described for pUR 301. The complete construction of pUR 401 is outlined in Fig. 10.9d. Construction of plasmid pUR 303 Fig. 11 Plasmid pUR 300 9b, Fig. 9 was digested with EcoRI and the 234 bp fragment comprising the trp regulon was isolated. This fragment was ligated to pUR 301 DNA, which previously had been cleaved with EcoRI and dephosphorylated with phosphatase. The ligation mixture was used to transform competent R iI cells and from the ampicillin resistant transformants pUR 302 was obtained. This plasmid comprises two trp regulons with identiXdz transcriptiol7 polarity Fiy. 11 . pUR 302 was partially cleaved with EcoRI, in the presence of ethidium bromide, treated with nuclease S1 to generate blunt ends and the cleaved plasmid DNA s were religated. The ligation mix was used to transform competent E. col cells and from the ampicillin resistant transformants pUR 303 was isolated, wherein the EcoRI site in between the two trp regulons in pUR 302 had been removed.9e. Construction of pUR 10 Fig. 12 Plasmid pBR 322 was cleaved with PstI and PvuII, and subsequently dephosphorylated with phosphatase. The 2817 bp long fragment D, Fig. 12 was isolated. Plasmid pBR 322 was also cleaved with MboII, treated with nuclease S1 and phosphatase and then cleaved once more with PstI. The 400 bp long fragment E, Fig. 12 extending from position 3201 3608 J.G. Sutcliffe, Cold Spring Harbor Symposia on Quantitative Biology, 43, 77 90 1978 was isolated. Plasmid pVU 208 A.R. Stuitje, thesis, V.U. Amsterdam 1981 was cleaved with BamHI and treated with nu clease S1. The 760 bp fragment F, Fig. 12 containing the replication origin of clo DF 13 with the cop ts mutation, was isolated. To construct pUR 10, fragments D and E were ligated first, followed by ligation with fragment F. The ligation mix was used to transform competent F. coli cells. From ampicillin and tetracycline resistant transformants pUR 10 containing cells were isolated. The replication origin containing fragment is oriented such that the unidirectional replication is in a counter clockvJise direction.9f. Construction of pUR 210, pUR 31q, pUR 311, pUR 410 Fig. 13 These plasmids are derived from pUR 201, pUR 301, pUR 303 and pUR 401, respectively, and contain the cop ts replication origin of pUR 10. pUR 10 was digested with PstI and BamHI and the 2841 bp long fragment G, Fig. 13 was isolated by agarose gel electrophoresis and electro elution. Each of the plasmids pUR 201, pUR 301, pUR 303 and pllR 401 was digested with PstI and BamHI and the regulon containing fragments collectively called H, Fiy. 13 were isolated. Fragment G and each of the fragments H in turn were ligated using T4 DNA ligase and the ligation mixes were used to transform competent 2g. coZi cells. From ampicillin and tetracycline resistant colonies pUR 210 , pUR 310 , pUR311 and pU R 410 containing cells were isolated.10. Construction of plasmids comprising a constitutive or inducible regulon and the ligated preprochymosin gene or its various allelic and maturation forms, the latter being under transcriptional control of said regulons, and transformation of said plasmids into E. coli 10a. Construction of expression plasmids giving rise to the synthesis of bovine pseudochymosin Fig. 14 Plasmids pUR 1520, 1530, 1540, 1730, 1820, 1830, 1840 and 1930 were cleaved with EcoRI and dephosphorylated. Each preparation in turn was ligated with fragment I 8a, Fig. 4 .This ligation mix was used to transform competent E.coli RRI and from the ampicillin resistant transformants, cells containing pUR 1521, 1531, 1541, 1731, 1821, 1831, 1841 and 1931 were selected which contained fragment I inserted such that the genetic information coding for pseudochymosin was present as a continuous uninterrupted entity. lOb. Construction of expression plasmids giving rise to the synthesis of chymosin Fig. 15 Plasmid pUR 1521 was cleaved with HindlII, dephosphorylated and then cleaved once more with BglII. The resulting X 1300 bp long fragment VI Fig. 15 was purified. The vector fragments C 8a, Fig. 3 , in turn, were ligated with fragments II 8b, Fig. 5 and fragment VI. The ligation mix was used to transform competent E. coli cells and from ampicillin resistant transformants cells containing pUR 1522, 1532, 1542, 1732, 1822, 1832, 1842 and 1932 were selected.10c.Construction of expression plasmids giving rise to the synthesis of prochymosin Fig. 16 Vector fragments C 8a, Fig. 3 , fragment N 8c, Fig. 6 and fragment VI 9b, Fig. 15 were listed and the resulting iiydiiurr mix was used to transform competent I. f cells. From the tlnp cillin resistant t. ansfs,rlluant cells containing pJR 1523, 1533, 1543, 1733, 1823, 1833, 1843 and 1933 were selected. i d. Construction of expression iJlasaiids giviilg rise to Xhe expression of preprochymosin Fig. 17 Vector fragments C 8a, rig. 3 , fragment V 8d, Fig. 7 and frag ment VI 9b, Fig. 15 were ligated and the resulting ligation mix was used to transform competent . voLi cells. From the ampicillin resistait transformants cells containing pUR 1524, 1534, 1544, 1734, 1824, 1834, 1844 and 1934 were selected Fig. 17 .10e. Example demonstrating the use of site directed mutagenesis to create allelic forms of the bovine preprochymosin or its maturation forms, departing from the chemical structure given in Fig. 1 and thereby converting residues 202 and 286 into aspartic acid residues, such in turn or in combination Fig.18 22 . Plasmid pUR 1001 was cleaved with PstI and the resulting DNA frag ments were ligated to the synthetic pentanucleotide 5 HOdCTGCAOH 3 . Following ligation, the mixture was incubated with E. coti DNA polymerase, large fragment in the presence of dGTP in order to make blunt ends and phosphorylated with T4 kinase and ATP. The DNA was subsequently supplied with EcoRI linker 5 dCATGAATTCATG 3 and then treated with EcoRI. A circa 880 bp long fragment extending from the EcoRI site at position 549 to the carboxy terminal end of the chymosin encoding DNA was isolated. This fragment was subsequently cloned in the EcoRI site of RFM13 mp 2. Two clones were isolated, M13 1020 and M13 1021, which were different in the orientation of the EcoRI insert with respect to each other cf. Fig. 18 . M13 1020 contained the coding strand plus strand M12 1021 contained the non coding strand minus strand . ss.Phage DNA of M13 1020 was converted into double stranded DNA using E. uoli DNA polymerase large fragment, dNTP s and 5 dTGGCCATCCCTGTCC 3 i 675 or 5 dAAACTCATCGTACTG 3 ii 928 in turn as primers, using procedures described by S. Gillam et ai. 1979 Nucleic Acids Revs., 6, 2973 2985. The underlined bases represent mismatches in tie primer template hybrid. Following tra lsformation of competent E. coli JM101.7118 cells B. Gronenborn J. Messing, Nature, 272, 375 377 1978 , phages ere screened for the required conversion into the chynosin encoding sequence by plaque hybridization with the 32p labelled pentadecanucleotides i, ii, as probes and DNA sequence analysis.Two phage isolates, M13.1022 and 1023 contained the DNA sequences asp arg asp gly 5 G GAC AGG GAT GGC CA 3 M13.1022 675 glN tyr asp glu phe 5 CAG TAC GAT GAG ITT 3 M13.1023 928RF M13.1022 and RF M13.1023 were cleaved with EcoRI, dephosphorylated and then cleaved with PstI. From each preparation an 888 bp long fragment was isolated by agarose gel electrophoresis and electroelution. Apart from the required mutations, this fragment corresponds with fragment B Sa, Fig. 3 . Using procedures identical with those described in 8a 8c, expression plasmids were constructed which gave rise to the synthesis of specifically altered pseudochymosin, chymosin, prochymosin and preprochymosin, respectively.Culturing of E. coli cells containing recombinant plasmids described under 10 and detection and isolation of preprochymosin, prochymosi n, pseudochymosi n or chymosin coii cells containing one of the plasmids pUR 1521 ATCC 39120 , 1531, 1541, 1731, 1821, 1831, 1841, 1932 pUR 1522, 1532, 1542, 1732, 1822, 1832 ATCC 39197 , 1842, 1932 pUR 1523, 1533 ATCC 39121 , 1543, 1733, 1823, 1833, 1843, 1933 pUR 1524, 1534, 1544, 1734 ATCC 39198 , 1824, 1834, 1844, 1944 with or without the AATT sequence in the linker netween the regulon and tie preprochymosin ge or its m,tur ation forms were cultured under optin al conditions for their growth. These culturing condi tions ary with the type cif plasinid present in the cells, JUL at suitable antibiotic ampicillin was always present to maintain selection pressure . tinder these conditions the cells containing either plasmids pJR 1521, 1531, 1541, 1731, 1821, 1831, 1841, 1931 or pJR 1522, 1532, 1542, 1755, 1822, 1832, 1842, 1932 or pJR 1523, 1533, 1543, 1733, 1823, 1833, 1843, 1933 or pJR 1524, 1534, 1544, 1734, 1824, 1834, 1844, 1944 produced considerable amounts of pseudochymosin, chymosin, prochymosin or preprochymosin, respectively. These amounts varied from 103 10 molecules cell.E. coZi cells which contained preprochymosin or modified preprochymosin encoding plasmids contained modified preprochymosin in the cytoplasm and prochymosin in their periplasmic space.The bacterially produced preprochymosin, prochymosin and pseudochymosin could be converted into chymosin using the procedures described by 2. Barkholt Pedersen et aZ. Eur. J. Biochem., 94, 573 580 1979 . The chymosins which were thus obtained and bacterially produced chymosin were shown to be fully biologically active in proteolysis. The presence of the proteins was further demonstrated by SDSpolyacrylamide gel electrophoresis with or without immunoprecipitation, and by immunological ELISA and RIA tests. The antisera for this test were generated by injecting bovine calf chymosin supplemented with Freund adjuvant into sheep as well as rabbitsThe above description was focussed on the synthesis of chymosin inE. coZi cells it is of course possible and desirable to use for that purpose non toxic, edible micro organisms, such as streptococci or micro organisms of Bacillus or yeast origin. AbbreviationsDNA deoxyribonucleic acid cDNA complementary DNA dsDNA double stranded DNARNA ribonucleic acid mRNA messenger RNAA adenineG guanineC cytosineT thymineRBS ribosome binding siteN any nucleotide bp pase pairM13 bacteriophage M13RF replicative form p plasmid 32p phosphorus 32 n number p o promotor operator trp tryptophan lac lactoseEcoRI rest tans endonuclease derived from Escherichia ccli RYI Hindlil restriction endonuclease derived from Haemophilus Influenza RdIIIHinfI restriction endonuclease derived from HaemophiZus InfZuenzae HaeIII restriction endonuclease derived from llaemcphilus aegypticusBglII restriction endonuclease derived from Bacillus glcbii HphI restriction endonuclease derived from Haemophilus parahaemolyticusPstI restriction endonuclease derived from Providencia stuartfi MnlI restriction endonuclease derived from Moraxella nonliquefaciens TaqI restriction endonuclease derived from ThermophiZus aquaticusBamHI restriction endonuclease derived from Bacillus amylolitique faciens HE. coZi Escherichifw coli ts thermosensitiveDdeI restriction endonuclease derived from Desufovibrio desuifwicans HpaII restriction endonuclease derived from Haemophilus parainfluenzae ELISA enzyme linked immuno sorbent assayRIA radioimmune assaySDS sodium dodecyl sulphate Abbreviations continued met methionine leu leucine ile isoleucine ala alanine asp aspartic acid asN asparagine glu glutamic acid glN glutamine val valine thr threonine phe phenylalanine tyr tyrosine cys cysteine arg arginine ser serine his histidine pro proline gly glycine lys lysineAp ampicillin resistanceTc tetracycline resistance Legends to the Figures Fig. 1. Amino acid sequence of one of the allelic forms B of bovine preprochymosin and DNA sequence corresponding to the bovine preprochymosin B mRNA. The S numbering denotes numbering of the signal sequence amino acid residues.Fig. 2. Schematic representation of several examples of allelic forms of bovine prenrochymosin. The upper drawing corresponds with Fig. 1. The numbering in parentheses denotes the amino acid residues in the Dreprochymosin molecule. Other numbers refer to the nucleotide sequence of the mRNA.Fig. 3. Construction route of pUR 1520, 1530, 1540, 1730, 1820, 1840 and 1930, described in 8a.Fig. 4. Construction route of dsDNA encoding amino terminal end of pseudochymosin plus a transcriptional initiation triplet, described in 8a.Fig. 5. Construction route of dsDNA encoding the amino terminal end of chymosin plus a transcriptional initiation triplet, described in 8b.Fig. 6. Construction route of dsDNA encoding the amino terminal end of prochymosin plus a transcriptional initiation triplet, described in 8c.Fig. 7. Construction route of dsDNA encoding the amino terminal end of preprochymosin, described in 8d.Fig. 8. Construction route of pUR 201, described in 9a.Fig. 9. Construction route of pUR 301, described in 9b.Fig. 10. Construction route of pUR 401, described in 9c.Fig. 11. Construction route of pUR 303, described in 9d.Fig. 12. Construction route of pUR 10, described in 9e.Fig. 13. Construction route of pUR 210, 310, 311 and 410, described in 9f.Fig. 14. Construction route of pUR 1521, 1531, 1541, 1731, 1821, 1831, 1841 and 1931, described in 10a.Fig. 15 Construction route of pUR 1522, 1532, 1542, 1732, 1822, 1832, 1842 and 1932, described in 10b.rig. 16. Schematic representation of pUR 1523, 1533, 1543, 1733, 1823, 1833, 1843 and 1933, described in 10c.Fig. 17. Schematic representation of pUR 1524, 1534, 1544, 1734, 1824, 1834, 1844 and 1934, described in 10d. Fig. 18. Construction route of M13.1020, M13.1021 and part of dsDNA encoding an allelic variation of bovine preprochymosin, described in 10e.Fig. 19. General representation of the plus strand DNA sequence corre sponding to the structural gene encoding preprochymosin wherein A is deoxyadenyl, G is deoxyguanyl, C is deoxycytosyl, T is thymidyl, J is A or G K is T or C L is A, T, C, or G M is A, C or T X is T or C if Y is A or G, and C if Y is C or T Y is A, G, C or T if Xis C, and A or G if Xis T W is C or A if Z is G or A, and C if Z is C or T Z is A, G, C or T if W is C, and A or G if W is A QR is TC if S is A, G, C or T, and AG if S is T or C and S is A, G, C or T if QR is TC, and Tor C if QR is AG.Fig. 20. General representation of the plus strand DNA sequence corre sponding to the structural gene encoding prochymosin plus a transcriptional i nitiation ATG triplet wherein A is deoxyadenyl, G is deoxyguanyl, C is deoxycytosyl, T is thymidyl, J is A or G K is T or C L is A, T, C, or G M is A, C or T Xis T or C if Y is A or G, and C if Y is C or T Y is A, C, C or T if Xis C, and A or G if Xis T W is C or n if Z is G or A, and C if Z is C or T Z is A, G, C or T if W is e, and A or G if W is A QR is TC if S is A, G, C or T, and AG if S is T or C and S is A, G, C or T if OR is TC, and T or C if QR is AG.Fig. 21. General reuresentation of the plus strand DNA sequence corre spondinq to the structural gene encoding pseudochymosin plus a transcriptional initiation ATG triplet wherein A is deoxyadenyl, G is deoxyguanyl, C is deoxycytosyl, T is thymidyl, J is A or G K is T or C L is A, T, C, or G M is A, C or T Xis T or C if Y is A or G, and C if Y is C or T Y is A, G, C or T if Xis C, and A or G if X is T W is C or A if Z is G or A, and C if Z is C or T Z is A, G, C or T if W is C, and A or G if W is A QR is TC if S is A, G, C or T, and AG if S is T or C and S is A, G, C or T if QR is TC, and T or C if QR is AG.Fig. 22. General representation of the plus strand DNA sequence corre sponding to the structural gene encoding chymosin plus a transcriptional initiation ATG triplet wherein A is deoxyadenyl, G is deoxyguanyl, C is deoxycytosyl, T is thymidyl, J is A or G K is T or C L is A, T, C, or G M is A, C or T Xis T or C if Y is A or G, and C if Y is C or T Y is A, G, C or T if X is C, and A or G if X is T W is C or A if Z is G or A, and C if Z is C or T Z is A, G, C or T if W is C, and A or G if W is A OR is TC if S is A, G, C or T, and AG if S 55 T ui C and S is A, G, C or T if QR is TC, and T or C if QR is AG. General remarks on the figuresGenerally plasmid DNA is drawn as a single lined circle, still this represents double stranded DNA bacteriophage M13. DNA is single stranded the replicative form RF, however, is double stranded . 5 ends of c cuvc DNiA at restriction enzyme cleavage site are phosphorylated unless indicated otherwise 3 ends are always dephosphorylated. ihe numbers given in italics refer to the bovine preprochymosin DNA sequence given in Fig. 1 otherwise they refer to plasmid DNA sequences.EMI24.1 dA dT stretch dG dC stretch regulon, the arrow indicates transcription direction