# Improved gene splicing method and products produced therefrom.

## Abstract
An improved gene splicing and recombinant plasmid transformation method is described. The method includes mechanical fragmenting of chromosomal DNA followed by conventional digestion with a restriction enzyme and gene splicing into a vector to provide recombinant plasmids in a bank of at least about 100 different plasmids. The plasmids in the bank are provided for transformation into a suitable host, particularly a plasmid free bacterium of the same species from which the chromosomal DNA or the vector is derived. The method provides high transformation frequencies because of the presence of multiple super coiled or closed coiled recombinant plasmids in the bank. The method also allows for the direct selection of many different phenotypic traits in a pool of the transformed hosts. The selected hosts are useful for the production of various gene products.

## Claims
I CLAIM 1 In a gene splicing method wherein chromosomalDNA is partially and randomly cleaved into fragments with a restriction enzyme and spliced into a cleaved vector thereby joining the chromosomal DNA fragments to the vector to form a recombinant plasmid, the improvement which comprises a mechanically fragmenting concentrated chromosomal DNA in the absence of bacterial cells into multiple fragments of randomly varying lengths and b partially cleaving with a restriction enzyme and joining the mechanically fragmented and cleaved chromosomal DNA with the cleaved vector to form a bank of recombinant plasmids of varying sizes. 2 The method of Claim 1 wherein the vector is a derivative of plasmid pR01600 as carried in NP2 B l2l24. 3 The method of Claim 1 wherein the vector is plasmid pRO1601, 1613, 1614 as carried in NRRL B 12125, 12,126 and 12127 respectively. 4 The method of Claim 1 wherein the chromosomalDNA is harvested from a Pseudomonas and wherein the vector is a derivative of pRO1600 as carried in NRRL B 12124. 5 The method of Claim 4 wherein the chromosomalDNA is harvested from Pseudomonas cerzigi7çosa, 6 The method of Claim 1 wherein the restriction enzyme is an endonuclease selected from BamHI, HindIII,SalI, PstI, and m II. 7 The method of Claim 1 wherein the mechanical fragmenting is accomplished using a vortex mixer. 8 The method of Claim 1 wherein the chromosomalDNA is cleaved into segments having an average length of between about 10 and 20 daltons x 106. 9 The method of Claim 1 wherein the fragmenting is accomplished by whipping the chromosomal DNA fragments. 10 The method of Claim 1 wherein the fragmenting is accomplished in the presence of lysozyme, detergent and a chelating agent. 11 The method of Claim 1 wherein the recombinant plasmids are transformed into a receptive microorganism and wherein selection is made for a particular useful gene trait present in the chromosomal DNA. A preserved recombinant plasmid bank adapted for transforatio into bacteria and then selection for particular recombinant plasmids, the bank containing multiple random length mechanically fragmented chromosomal DNA fragments having an average length between 0.1 and 20 daltons x 106 recombined with a vector and the plasmids being adapted for transformation into a bacterium having a particular mutant gene trait allowing for selection. 13 The recombinant plasmid of Claim 12 wherein the bank is a concentrate containing at least about 100 different recombinant plasmids in aimixturer The recombinant plasmid of Claim 12 wherein the bank is preserved by being frozen in the presence of a freezing stabilizing agent. The recombinant plasmid of Claim 14 wherein the freezing stabilizing agent is a 2 amino 2 hydroxymethyl 1, 3 propanediol and disodium ethylene diamine tetraacetic acid mixture with a pH of about 8. 16 A bacterial strain containing various reccibinant plasmids prepared by the method of Claim 1. 17 A bacterial strain containing various recombinant plasmids prepared by the method of Claim 1 wherein the vector is a derivative of pRO160û. 18 A bacterial strain containing various recombinant plasmids prepared by the method of Claim 1, wherein the vector is seledted from pRO1601, 1613 and 1614 as carried in NRRL 3 l2125, 12126 and 12127, respectively,

## Description
IMPROVED GENE SPLICING METHOD AND PRODUCTS PRODUCED THEREFROMBACKGROUND OF THE INVENTION 1. Field of the Invention The present invention relates to an improved gene splicing method, to recombinant plasmids and to transformed hosts produced therefrom. In particular, the present invention relates to a method whereby the selection of transformed hosts having a particular phenotypic trait is greatly simplified.2. Prior Art The basic prior art is described in U.S. PatentNo. 4,237,224 to Cohen and Boyer. The pioneering effort described in this patent provided the starting point for the present invention and is familiar to all those skilled in the art. There is a large body of prior art which is directly related to this patent. The patent terminology is generally used herein. In my U.S. patent application Serial No. 147,563 and related foreign applications, I described new plasmid vectors derived from plasmid pRO1600. These plasmid vectors are used in the present invention. U.S. Patent No. 4,278,765 to Debabov et al describes other vectors and recombinat plasmids. U. S. Patent No. 3,813,316 toChakrabarty describes multiple plasmid containing Pseudemonas and the method for their preparation. In the method of application Serial No. 147,563 as in the method of thc Cohen and Boyer patent, indirect selection of recombinant plasmids which are transformed into a recipient host is described. This method involves the selection for a marker in the transformed bacteria and the selection for the particular phenotypic trait. The selection method is laborious and time consuming in obtaining a particular phenotypic trait and provides one important reason why genetic research has been expensive. The reason for this result is that the transformation frequencies are lowCor non existent in relation to a particular gene in a recombinant plasmid.Objects It is therefore an object of the present invention to provide a method for rapidly and inexpensively obtaining a particular desired phenotypic trait in a transformed host. It is further an object to provide novel recombinant plasmid compositions as banks for obtaining the desired trans for mants. Finally it is an object of the present invention to provide novel, improved transformed hosts. These and other objects will become increasingly apparent by reference to the following description and the drawings.In the Drawings Figures 1, 3, 4, 5 and 6 are electrophoresis gels showing various plasmid prow ilex Figures 2 and 7 are physical maps of plasmids.In particular Fibre 1 is an agarose gel electrophoresis showing the origin and derivation of plasmids RP1, pROl 600r pRO1601, pR01613 and pRO 1614. DNA was prepared as described in Materials and methods. A 25 ttl sample of each DNA preparation was mixed with 10 ill of dye and subjected to electro phoresis. Samples were as follows. A PAO2 RP1 11 B pAo2 RPl pROlE00 C PA02tpR01601 D DNA from Escherichia coli V517, a multi plasmid containing strain used as a size standard Macrina, F., D. Kepecko, K. Jones,D. Ayers and S. McCowen.Plasmid 14 7 420 1978 E PA02 pROl6l3 F PAO2 pRO1614 . Gels which contained 0.7 percent agarose were constructed and run as described previously Hansen, J., and R. Olsen. J. Bacteriol. 135 227 238 1978 . Figure 2 is a physical map of plasmids A pR01601, B pRO16113, C pRO1614, and D pRO1600. Numerical values in parentheses represent molecular size in daltons x 106 for restriction enzyme cleavage fragments. These were determined using HindIII cleavage of phage lambda DNA and plasmid pBR322 DNA cleaved with combinations of two enzymes double digest . Figure 3 is an agarose gel electrophoresis of plasmids formed with pRO1613 and cleaved with PstI. Samples were cleaved as described in materials and methods and run as described for Figure 1. Samples were as follows A pRO1613 B pRO1665, a lys 12 reccmbinant plasmid C lambda cleaved with HindIII D pRO1661, a met 28 recombinant plasmid. Figure 4 is an agarose gel electrophoresis of recombinant plasmids formed with pRO1614. Samples were cleaved with BamHI unless otherwise noted as described in materials and raethods and run as described for Figure 1.Samples were as follows A pRO1655, a proA recombinant plasmid B and CF lambda cleaved with HindlIl C pRO1657, a proA recombinant plasmid D pRO1614, the vector cleaved with HindIII E pRO1658, a his II recombinant plasmid G pRO1615, an ilvB,C recombinant plasmid cleaved with HindIII. Figure 5 is an agarose gel electrophoresis of cleaved PAO chromosomal DNA. Samples were cleaved as described in materials and methods and run as described inFigure 1. Samples were as follows A un cleaved DNA B chromosome cleaved with HindIII C chromosome cleaved with SalI D lambda cleaved with Hindlil E chromosome cleaved with BamHI F chromosome cleaved with PstI. Figure 6 is an agarose gel electrophoresis of recombinant plasmid gene banks derived from the PAO chromosome. DNA for the pooled recombinant plasmids was prepared as described in the text. Samples in files F and J are the vectbr, pRO1614. Other files in the figure show plasmid DNA obtained from cultures inoculated from mixed suspensions which contained independently isolated recombinant plasmids as described in the specification, Figure 7 is a chromosomal map of PAO showing the map location of markers complemented by representative recodbinant plasmids listed in Table 6. The map locations are those of Royle, Matsumoto and Hollowly, 3. Bacteriol. 145 145 155. Marker abbreviations are listed in footnote a of Table 1.Summary of the Invention The present invention relates to an improvement in the gene splicing method wherein chromosomal DNA is partially and randomly cleaved into fragments with a restriction enzyme and spliced into a cleaved vector thereby joining the chromosomal DNA fragments to the vector to form a recodbinant plasmid which comprises a mechanically fragmenting concentrated chrorac somal DNA in the absence of bacterial cells into multiple fragments of randomly varying lenths and b partially cleaving with a restriction enzyme and joining the mechanically fragmented and cleaved chromosomal DNA with the cleaved vector to form a bank of recombinant plasmids of varying sizes. Further the present invention relates to a preserved recombinant plasmid bank adapted for transformation into a bacteria and then selection for particular recombinant plasmids, the bank containing multiple random length mechanically fragmented chromosomal DNA fragments having 6 an average length between 0.1 and 20 daltons x 10 recom bined with a vector and the plasmids being adapted for transformation into a bacterium having a particular mutant gene trait allowing for selection. Finally the present invention relates to certain novel trans, ormed hosts of the genus Pseudronss into which the recombinant plasmids prepared by the method are transformed. The recombinant plasmids in a gene bank formed as a result of the initial fragmenting of the chromosomalDNA in the method of the present invention are highly transformable. Virtually any gene from the chromosomalDNA can be found in the collection of recombinant plasmids and can be selected using conventional selection techniques with plasmid free, auxotrophic bacteria for the transformation and defined media The recombinant plasmids of the present invention are closed coiled or super coiled like a spring which promoteshighly efficient transformation. Because of the high, randomly sheared chromosomal transformation frequency, the selection can be made directly for almost any particular phenotypic or gene trait from the pool of super coiled DNA recombinant plasmids.This contrasts with the prior art method wherein indirect selection is made based upon a particular phenotypic marker in the chromosomal DNA usually an antibiotic marker or directed selection of rare closed circular DNA plasmids.The transformants in the prior art method are then tested for acquisition of a particular associated marker within the transformants along with the antibiotic marker, such as the ability to produce an essential amino acid. Many times the desired marker is not formed. In the method of the present invention the probability is great that the particular marker can be selected directly. The result is a considerable savings of time and money in obtaining desirable transformants. A vortex mixer tsuch as a Vortex Genie FisherScientific Company is preferably used to fragment the chromosomal DNA. An eccentric cup oscillates, causing a vortex in a test tube shaped container. The chromosomalDNA because of its length and relatively greater inertia with respect to the moving liquid breaks into relatively small fragments which have an average length between about 10 and 2Q x 106 daltons. The fragmenting is accomplished essentially by whipping the long strands of chro nosomal DNA. Description of Specific Embodiments A host vector system particularly for Peumonas aerugtnosa PAO has been developed. Scattered regions of the strain PAO chromosome have been cloned by direct selection from a DNA gene bank which contains over 1,000 independently isolated chromosome vector recombinant plasmids The use of partially digested chromosomal DNA facili tated the selection of a variety of strain PAO chromosomal markers. As described in Serial No, 147,563 the progenitor of the vector was a small, multi copy plasmid, pRO16û0, found in one PAO strain which had acquired RP1 in a mating experiment. The bacterial host range of vectors produced from pR01600 resemble that for plasmid RP1. Two derivative plasmids were formed one, pR01613, for cloning DNA cleaved with restriction endonuclease PstI. Another, pRO1614, was formed by deleting part of pR01613 and fusion with plasmid pBR322. Plasmid pROl6l4 utilizes cloningsites within the tetracycline resistance region of pBR322. Molecular cloning of chromosomal DNA using plasmids developed for that purpose has been reported for a variety of bacterial species In most instances, the utility of the plasmid cloning vector derives from the inactivation of an antibiotic resistance determinant of the vector as a consequence of the insertion of a piece of hetero DNA from the chromosome into a site on the vector cleaved by a restriction endonuclease Thus, the continuity of the vector gene and hence its expression is interrupted by insertion of foreign pieces of DNA. One of the most widely used plasmid vectors of this kind is plasmid pBR322 which has the advantage of a single restriction endonuclease site within its antibiotic resistance determinants for ampicillin resistance and tetracycline resistance Bolivar, R, et al Gene 2 95 113. .However, the host range of pBR322 is limited to Escherichia coli and related bacterial strains prohibiting its usefulness in bacteria with disparate properties. Bagdasarian et al Developments inGenetics 1 411 422 1979 and recently Wood et al J.Bacteriol. 145 1488 1451 1981 have reported the construction of a cloning vector for Pseudomonas aeruginosa to overcome this difficulty. Another plasmid vector has also been developed by Ditta et al which has been derived from the broad host range plasmid RK2 Proc. Natl. Acad. Sci. USA 77 7347 7351 . This system offers the advantage of potential for use in a wide variety of Gram negative bacterial species in view of its broad host range The present invention uses a preferred cloning vector system from a particular Pseudomonas aeruginosa strain which contains plasmid RP1. Plasmid RP1 is similar if not identical to RX2 used previously for this purpose and also plasmid RP4. One of the vectors, designated pR01613 allows cloning of DNA previously digested with PstI using direct selection for the desired phenotype. Another vector, designated pRO1614, was derived from a deleted pRO1613 and the insertion of pBR322.Using these plasmids, we show cloning of widely scattered regions of the Pseudomonas aeniginosa strain PAO chromosome using a recombinant plasmid gene bank derived from the PAO chromosome. This gene bank has been used for the selection of a wide variety of recombinant plasmids which complement mutations in the chromosome of strain PAO.MATERIALS AND MET tODS Bacterial strains and plasmids. The relevant properties of the bacterial strains and plasmids used in this study are listed in Table 1. Table 1. Bacteria and plasmidsBacterial Reference orStrain Genotype a SourcePAO1 Prototroph 1 PAO2 ser 3 1 PA025 argF leu 10 1 PA036 his II ilvB,C lys 12 2 met 23 proA trpC,D,E,PA02003 argH recA 3 PA02178 catAl met 9020 nar 9011 4 bPAO2198 arg 90310 catAl chu 9002 b leu 9014 lys 9015 met 9020 nar 9011 trpA,BPA02324 catAl met 9020 nar 9011 b puuD6 tyu 9009PAO 2369 catAl cnu 9001 met 9020 b nar 9011 puuR8 tyu 9025GMA052 hisV aGMA057 hisIV cGMA065 hisIII cGMA253 cys 5605 his I 1 cPseudomonas putida PP0131 his 1 dPseudomonas fluorescens his 1 5 e PF0141Escherichia coli ED8654 met rk mk fKlebsiella pneumonias Prototroph g KPM 100PlasmidRP1 Cbr, Tcr, PRRs, PRD1s 6, 7 PBR322 Cbr, Tcr 8 pRO1600 Cryptic parent application pRO1601 pRO1600 Tn1 Cbr parent application pRO1613 Cbr parent application pRO1614 Cbr, Tcr parent application . arker abbreviations Bacterial Strains arg, arginine cat, catechol utilization chu, choline utilization cnu, carnosine utilization cys, cysteine his, histidine ilv, isoleucine valine leu, leucine lys, lysine met, methionine nar, nitrate reductase pro, proline puu, purine utilization rec, recombination rk mk , Escherichia coli K12 restriction and modification ser, serine tryp, tryptophan tyu, tyrosine utilization. Plasmids Cbr, carbenicillin resistance Tc , tetracycline resistance Kmr, kanamycin resistance PRR1s, sensitive to phagePRR1 PRDLs, sensitive to phage PRD1. bMatsumoto, H. et alt Mol. Gen. Genet. 167 165 176 1978 . cMee, B., and B. Lee. Genetics 62 637 696 1969 . mutant of ATCC 12633. mutant of strain PFO14 isolated from milk and received from J. Jezeski, University of Minnesota. strain K12, received from D. Jackson. Stock culture, Department of Microbiology andImmunology, University of Michigan Medical School. 1 Royle, P., H. Matsumoto, and B. Holloway.J. Bacteriol. 145 145 155 1981 . 2 Haas, D., and B. Holloway. Mol. Gen. Genet.144 243 251 1976 . 3 Chandler, P., and V. Krishnapillai. Mutat.Res. 23 15 23 1974 . 4 Royle, P., H. Matsumoto, and B. Holloway.J. Bacteriol. 145 145 155 1981 . 5 Olsen, R., E. Metcalf, and J. Todd. J. Virol.2 357 364 1968 . 6 Olsen, R., and P. Shipley. J. Bacteriol.113 772 780 1973 . 7 Olsen, R., J. Siak, and R. Gray. J. Virol.14 689 699 1974 8 Bolivar, R., R. Rodriguez, P. Greene, M.Betlach, H. Heyneker, H. Boyer, J. Crosa, and S. Falkow.Gene 295 113 1977 . Media. Minimal medium VBG and complex medium 10 mare prepared as described in Olsen, R., and P. Shipley. J.Bacteriol. t13 772 7B0 1973 . When nutritional selection for transformants was done, amino acid requirements were satisfied by the addition of these components to a final concentration of 5mM. Antibiotic supplements were as described in the tables. Selection or indirect testing for the acquisition of catabolic markers was done using a minimal medium MMO described previously for this purpose Stanier, R., N. Palleroni, and M. Doudoroff, 3. Gen. Microbiol 43 159 271 1966 . Carbon sources were incorporated into teIO at a final concentration of 0.2 percent. Preparation of DNA. Plasmid DNA was prepared using a modification of the method of Guerry, LeBlanc andFalkow, J. Bacteriol. 116 1064 1066 1973 . Cells were grown overnight on the surface of TN agar medium plates.In some cases, carbenicillin Cb, 0.5 mg per ml was included in the medium to maintain selection for plasmids. The cells were harvested from the surface of the plates by adding 5 ml sterile water to each plate and scraping with a glass rod. These suspensions were decanted and pooled into a bottle which was shaken vigorously to disperse clumps of cells. The cell suspension was then centrifuged at ambient temperature and the pellets suspended inTS buffer 9 sucrose 0.05M Tris, pH8 10 ml buffer for each centrifuge tube with about 1.5 x 1010 cells . The following additions and manipulations were then done with each suspension.Na2EDTA 0.5M, pH8.O which functions as a chelating agent for metal ions was added to a final concentration of 0.08M. This was followed immediately by the addition of lysozyme 10 mg per ml in 0 25M trips, pH 8.0 to a final concentration of 0.77 mg per ml. The tubes were then briefly mixed on a vortex mixer Vortex GenieFisher Scientific Company and incubated at 370C for 5 min. Following this, sodium dodecyl sulfate SDS, 15 percent in distilled water was added to a final concentration of 2 percent.The tubes were inverted slowly several times to mix and then incubated in a water bath for 5 min at 370C. During this time they were removed several times and slowly inverted to promote lysis of the cells. Finally, 5M NaCl in distilled water was added to a final concentration of 0.95M and mixed into the suspension by several gentle inversions. The tubes were then placed into an ice water bath for 10 min followed by storage overnight at 40C. Precipitation of chromosome, harvest of plasmid DNA and CsCl ethidium bromide centrifugation was then done as described previously by Hansen and Olsen, J. Bacteriol. 135 227 238 l98 . DNA was stored frozen in TO buffer Tris l0M ImM Na2 EDTA, pH 8 and thawed slowly in ice water when used. Recombinant plasmids were surveyed for their size using cells harvested from a path of growth on selective medium and lysis of cells by the procedure of Hansen andOlsen. Chromosomal DNA was harvested from Pseudomonas aeruginosa strain PAO1 which had been selected for resistance to rifampin 50 g per ml in the laboratory. The procedure described above modification of Guerry et al. was used for chromosomal DNA too except that the lysed cell suspensions were given two 30 sec pulses at full speed on a vortex mixer prior to the addition of SM NaCl. This was done to fragment the chromosome. These lysed suspensions were then salt precipitated followed by DNA pre capitation and CsCl ethidium bromide centrifugation as described previously for plasmid DNA. Transformation. pseudomonads were trans formed using a modification of the method of Mercer and Loutit,J. Bacteriol. 140 37 42 979 . Bacteria were grown overnight on TN agar and a portion then inoculated into TN broth with incubation for 2 to 3 hours reaching a cell density of 1 x 108 per ml. The cells were centrifuged at 4 C and the pellet suspended in one half volume cold sterile MgC12 0.l5 in distilled water . The pellet was dispersed and held in an ice water bath for an additional 5 min. The cells were centrifuged and the pellet suspended as before but then held in the ice water bath for 20 min. The cells were centrifuged again and the pellet suspended in one tenth volume cold MgCl2 ta . 15M .Transforming DNA 10 to 50 p1 was placed in a cold centrifuge tube and 0.2 ml of the above cells added with mixing.This mixture was incubated in an ice water bath for 6,0 min followed by a heat pulse in a water bath at 370C for 3 min while gently swirling the tube The DNA cell mixture was then placed immediately in an ice water bath and incubated for S min. After this, 0.5 ml TN broth was added and the suspension incubated at 370C for 1 to 2.5 hours.The cells were then plated on selective medium and the plates were incubated at appropriate temperatures for 48 hours. Escherichia coli and Klebsiella pneumoniae were transformed using a modification of the method of Davis, Botstein and Roth, In Advanced Bacterial Genetics a manual for genetic engineering. Cold Spring Harbor Laboratory,Cold Spring Harbor 1980 pp 134 137 . For this, bacteria were grown as above and cultures chilled at the end of the growth period in an ice water bath for 10 min. The cells were ten centrifuged at 40C and pellets were suspended in one half volume CTG buffer CaC12, 50rM glycerol, 10 percent thymidine, 50 g per ml .The cells were dispersed and incubated 5 min in a bath at OOC. They were then centrifuged and pellets were suspended at one twentieth volume of the original culture. 0.2 ml cells were added to tubes which contained 20 to 100 ul DNA and mixed these were held 3 min in an ice water bath. The tubes were then transferred to a 450C water bath and slowly swirled for 2 min they were then placed in an ice water bath forS min. Following this, 0.5 ml of TN broth was added and the mixture was incubated at 370C for 0.5 2.5 hours.Samples of these transformation mixtures were then plated at 370C on selective medium. For both transformation procedures, all materials, including pipettes were at 40C unless otherwise noted. Enzymes. Restriction endonuclease Bethesda Research Laboratories, Rockville, MD digestion and ligation were done as is shown in the prior art. Digested DNA to be ligated was incubated for 20 hours at 170C. Ligations were done in a volume of 20 ijl or less. In some experiments, this required concentration of cleaved DNA by ethanol precipitation prior to the addition of ligase and buffer. These ligation mixtures were used for transformation as described above and in the Tables. Development of a cloning vector system. Over the years, the inventor has routinely transferred plasmid RP1 from one bacterium to another and, on some occasions, examined the transconjugants for the acquisition of a plasmid whose size and phenotypic characteristics correspond to that of the donor bacterium which contains RP1.On one such occasion, following the preparation of plasmidDNA from the recipient, it was noted the presence of two plasinids one corresponding to RP1 and another smaller plasmid of approximately 2 Mdaltons in molecular size.The recipient bacterium used for these experiments never before has been shown to contain such a small plasmid so this result suggests the rare occurrence and fortuitous observation of a RP1 mutation which occurred during transfer to a recipient. This is the invention described in application Serial o. 147,563. The relationship of these plasmids is shown in Figure 1. In Figure 1, file A is shown RP1 plasmid DNA isolated from a PA02 RP1 transconjugant. In file B, DNA from another PAO2 RP1 transconjugant used for the preparation of plasmid DNA showed two plasmids RP1 at the top of the gel and a smaller plasmid near the bottom which we have designated pRO1600. Its size was 2Mdaltons in relation to plasmid DNA standards prepared from Escherichia coZi V517 Macrina, F., D.Kepecko, K. Jones, D. Ayers, and S. McCowen, Plasmid 14 7 420 1978 shown in file D. We used the DNA preparations shown in files A or B to transform PAO25 with selection for the acquisition of either carbenicillin Cbr , tetracycline Tcr or kanamycin Kmr resistances.The rationale here was that one of these determinants from RPl might be included within p,R01600, or alternatively, a derivative might be obtained of the small plasmid designated pRO1600 which had acquired Tn1, a Cbr transposon, from RPl in the mixed DNA. The results of tis are shown inTable 2. As can be seen from Table 2, no transformants with only Tn1 were obtained from PAO2 RP1 11 DNA whereas 8 percent were Cbr only of those which obtained when PAO2 RPl pR0I600 DNA was used to transform. One sucr transformant is shown in Figure 1, file C. It was designated pR01601 and its size is about 5.2 Mdaltons.This result was consistent with the acquisition of Tnl 3.2 Mdaltons by pRO1600 2 Mdaltons . Plasmid pRO1601 and other similarTnl transposed transformants which were only Cbr derived from the experiment shown in Table 2 were used to prepare plasmid DNA and were cleaved by DstI or BglI restriction endonucleases and the fragments were subjected to agarose gel electrophoresis. Table 2. Selection of pR01600 Tnl a Percent nonselected Source of Number of markers acquired byTransforming DNAb Transformants transformants Tcr KmrP. aeruginosa PAO2 190 100 100 RP1 11 P. aerugirsa PA02 397 92 92 RP1 pRO1600 aApproximately 0.5 g of either plasmid DNA preparation was used for transformation as described in materials and methods. Strain PA02 was used as the recipient.Selection for transformants resistant to Cb was on TN agar medium with 0.5 mg of carbenicillin per ml. Cbr transformants were picked onto TN agar medium with Tc 60 ug per ml or VBG with serine and Xm 60 g per ml . bThese strains were transconjugants derived from mating PA025 RPl x PA02 and selecting for the acquisition of Cbr. Partial digests were also done to determine contiguity of the fragments. The results of one of these determinations are shown in Figure 2, panel A. Since the digest pattern for Tnl was shown from previous work, the map of pRO1600 shown in Figure 2, panel D was deduced. Similar determinations on eleven other Cbr transformants yielded different regions of pRO1600 altered as a consequence of aTnl insertion. However, none showed a Tnl insertion within the 0.83 Mdalton BglI fragment. Therefore, the results of this survey suggest that this 0.83 Mdalton region includes the replication functions for plasmid pR01600. Pernaps then, other regions of pRO1600 may be dispensible and either the size of pRO1601 may be reduced by PstI digestion and ligation of the replication region to the fragment withCbr, or that the replication region could be added to plasmid pBR322, which is non viable in PAO to provide replication functions in Pseudomonas. Accordingly, pR01601 was digested with PstI and a portion of this digest was added to pBR322 digested with PstI and the mixture was ligated. These preparations Cwith and without pBR322 were then used to transform PA02 and the results of this are shown in Table 3. Table 3. Derivation of cloning plasmids Plasmid Number of b AntibioticDNA used Treatment of DNA transformants resistance pBR322 none 0 Cbr pRO1601 none 69,000 Cbr pRO1601 PstI cleavage 70 Cbr ligation pR01601 PstI cleavage 2 Cb pBR322 ligation 4 Cbr, Tcr About 0.5 Vg of plasmid DNA was treated as described in the Table and transformed into PA02 as described in materials and methods. Selection was as described in Table 2.Undigested pRO1601 and pBR322 were also used as controls in this experiment. Several transformants from the PstIdigested pR01601 and the mixture of pRO1601 plus pBR322 were purified and DNA was prepared and analyzed by restriction endonuclease cleavage as before. The results of this are shown in Figure 2, panels B and C. In the former case, a plasmid designated pRO1613 which contains only two PstI sites was obtained. In the latter case, a plasmid designated pRO1614 which contained only a portion of pRO1601 and all of pBR322 was obtained In the latter instance, apparently part of the beta lactamase gene of Tnl, known to have a PstI site within it Bolivar, R., et al, Gene 2 95 113 1977 , has matched up with an analogous contiguous region of pBR322 to reconstitute 2 viable beta lactamase determinant. I next determined the bacterial host range of pRO1613 or pR01614 derived from the experiments shown in Table 3. For this, I used several bacterial straIns of disparate physiological properties. The choice of either pRO1613 or pRO1614 for this was influenced by the intrin sic Cbr or Tcr of these strains which would allow unambiguous scoring of transformants for their acquisition of the antibiotic resistances encoded by the plasmids. The data resulting from this preliminary host range determination are shown in Table 4. Table 4. Host range of cloning plasmids pRO1613 and pRO1614a Number of Plasmid Recipient transformants DNAb bacteria Selection c per g DNA pRO1613 P. aeruginosa PAO2 Cbr 132,000 pRO1614 P. aeruginosa PhOZ Tcr 38,000 pRO1614 P. putida PPO131 TCr 960 pRO1614 P. fluorescens PFO141 Tc4 24,000 pRO1614 E. coZi ED8654 Tcr 5,500 pRO1614 K. pneumoniae KPM100 Tcr 1,300 a Escherichia coli and Klebsiella pneumoniae were trans formed using the CTG procedure described in materials and methods. Pseudomanus strains were transformed using theMercer and Loutit procedure. DNA was prepared from PAOZ which contained either pRO1613 or pRO1614. c Selection was as described in Table 2. Pseudomonas putida or Pseudomonas fuorescens were incubated at 250C. Other strains were at 370 C. The data in the Table indicate the broad host range of the plasmids among pseudomonads, E. coZiand KZebsielZa. Perhaps this list will be extended as transformation procedures are developed for other gram negative bacterial species of interest for molecular cloning. Although the same DNA preparation for pR01613 or pRO1614 was used in all cases here, the number of transformants obtained varied with the recipient. This may reflect restriction of the plasmics in view of its preparation from strain PAO and perhaps the use of sub optimal transformation procedures for the bacterial strains shown here. Cloning the PAO chromosome. 1 tested the utility of pR01613 and pRO1614 for cloning using PstI cleaved pRO1613 and PstI cleaved PAO chromosome. Similarly, pR01614 and chromosome were cleaved with EcoRI, HindII, BamHI or SalI. The DNA suspensions were mixed after cleavage, ligated and transformed into PA0236. Strain PA0236 is a multiple auxotroph and transformants were selected for their nutritional independence Usually, one or two nutritionally independent transformants were obtained this way from an experiment and these were purified, characterized for antibiotic resistance, and used to prepared plasmid DNA for analysis.Plasmid DNA for typical transformants was cleaved with the enzyme used for their preparation and the electorpheregrams for some of these digests are shown inFigures 3 and 4. File A, Figure 3 siiows a PstI digest of the vector, pRO1613. The upper band is the PstI fragment internal to parft of Tnl which was retained by pRO1613 when pR01501 was reduced in size. This piece is lost during PstI digestion and cloning of pstl cleaved hetero DWA and therefore is absent in digests of recorbinant plasmids shown in files B and D.File B shows a recombinant plasmid designated pR01665 which complements the lys 12 mutation inPA0236 file D is a recombinant plasmid designated pR01661 which complements the met 28 mutation in, this strain. In addition to the replicator region lower band, file A of pRO1613, file B sows two bands derived from chromosomal cloning. It is uncertain whether these two bands were derived from a partially digested contiguous region of the chromosome or whether they represent the random association of two different fragments during the cloning process. File D, on the other hand shows a single fragment associated with the met 28 phenotype of this recombinant plasmid.To further confirm the auuhenticity of these recombinant plasmids, plasmid DNA was extracted from the strains shown in Figure 3 and used to re transfora PA0236. As expected, a high frequency of transformation, corresponding to that observed for per01613 using Cbr selection, occurred, using direct selection for nutritional independence with the recombinant met 28 or lys 12 plasmids. When pR01614 was used as the vector plasmid, transformants with markers corresponding to PA0236 were isolated using HindIII or BamHI. However, no transformants were obtained for plasmid and chromosome DNA cleaved withEcoRI or Salt. Figure 4 shows transformant DNA cleaved with HindIII or BamHI for several of the recombinant plasmids made using these enzymes. File D shows the vector, pR01614, cleaved at a single site as expected. A similar band is observed for files A, C, E, and G which contain recombinant plasmi ds complementing respectively proA, proA, hisil and ilvB,C mutations of PA0236. The ilvB,C recombinant plasmid was produced using HindIII the others were made using BamHI. The plasmids shown in files A andC are another example of varied sizes obtained for a given selection. The lower band is common to both and likely encodes for proA. The upper band in file C, then, is either a randomly associated fragment derived from another region of the chromosome or is the result of chromosomal DNA partially digested with BamHI. Table 5 summarizes the composition of recombinant plasmids shown in Figures 3 and 4. Table 5. Composition of PAO recombinant plasmids.Recombinant Plasmid PAO236 Enzyme Fragment sizesDesignation Marker Used x 106 daltons a pRO1613 vector PstI 1.8, 1.1 pR01665 lys 12 PstI 6.1, 1.1, 0.74 pRO1661 met 28 PstI 2.4, 1.1 pR01614 vector BamHI 3.9 pR01655 proA BamHI 3.9, 2.3 pRO1657 proA BamHI 12.1, 3.9, 2.4 pRO1658 hisII BamHI 13.6, 3.9, 0.82 pRO1615 ilvB,C HindIII 3.9, 1.8, 1.1 0.95, 0.74, 0.50 a Fragment sizes were calculated from data shown in Figures 3 and 4.The sum of the fragment size for pR01658 is about 18.3 Mdaltons representing 14.4 daltons of the PAO chromosome. Recombinant plasmids of this size are near the upper limit of those obtained to date using direct selection for the acquisition of nutritional markers. Not all the recombinant plasmids described here are stable in the absence of nutritional and antibiotic selection in appropriate mutant bacteria.For example, pRO1615 requires both selection for ilvB,C andCbr for maintenance in PAO236. If Cbr selection is omitted, a nutritionally independent chromosomal recombinant occurs if ilvB,C selection is omitted, a deleted version of pRO1615 occurs data not shown . However, the other plasmids described in Table 5, and for the most part all others obtained to date, are stale in recA PAO bacteria in the absence of nutritional or antibiotic selection on TN agar maintenance medium. Derivation of a chromosomal gene bank for PAO.Over a period of years, the number of recombinant plasmids.obtained using direct selection and the enzymes described in the foregoing work was small in comparison to the total yield of transformants obtained using antibiotic selection. When these transformants selected for antibiotic resistance were scored for insertional inactivation of an antibiotic resistance marker resulting from the addition of chromosomal DNA, few were found. Therefore, the cloning efficiency was low.Two bases for this may be relevant the donor chromosomalDNA may be poorly cleaved by the enzymes used and or the use of recA transforming recipient bacteria may diminish the recovery of recombinant plasmids. I investigated the former possibility by digesting PAO chromosomal DNA with the enzymes used in the foregoing work and the results of this are shown in Figure 5. These digests show variable cleav age of the fragmented chromosomal DNA preparation described in Materials and Methods . On the basis of the result shown in Figure 5, fileE I prepared a B mEI digest of fragmented DNA from PAO to be used in evaluating a possible enhancement of PAO which is recA Chandler, P., and V. Krishnapillai. mutant. Res. 23 15 23 1974 . For this I also cleaved pRO1614 withBamHI and selected PA02003 transformants for their acquisition of Cb r These Cbr transformants were then streaked onto medium which contained Tc to determine insertional inactivation of the TCr region of the vector as a consequence of the inclusion of a chromosomal fragment at its BamHI cleavage site. When this was done, 132 of 552 colonies picked were Tcs s C24 percent . In other experiment, where DNA from PAO was prepared from a rifampin resistant mutant 50 c nl , 952 of 30aO transformant colonies picked were Tc s 31 percent .Therefore, these data support the utility of recA strain PA02003 for cloning in Pseudomonas aeruginosa as suggested previously by Carbon et al for cloning in Escherichia Coli Carbon, J. et al, RecomMinant molecules impact on science and society. Raven Press,New York t1977 . In the second set of experiments described above, I also did direct selection for the acquisition of rifampin resistance or an ariR reconbinart plasmid complementing the argH mutation in the chromosome ofPA02003. Two recombinant plasmids of different size were found for rifampin resistance one argli recombinant plasmid was observed.As in all previous examples of recombinant plasmids discussed here, DNA from PA02003 transformants containing these plasmids was used to retransform PA02003 and the relevant markers were acquired at a high frequency independent of selection for either Cbr or the selective marker. Therefore, these were authentic recombinant plasnids for the markers relevant to their initial selection. Fourty four of the 132 Tcs transformants from the first experiment described above were patched onto each of 3 TN agar plates which contained Cb 0.5 mg ml and 100 TcS clones per agar plated from the 938 obtained in the second experiment described above were patched on the same media.Following overnight incubation, the cells were washed from each agar plate and 0.5 ml used to inoculate 10 nutrient agar plates which contained Cb. These ten TN agar plates per inoculum source were harvested and extracted for plasmidDNA which was subsequently centrifuged in CsCl ethidium bromide gradients. The plasmid DNA band was harvested from the gradient and electrophoresed to determine their content of recombinant plasmids resulting from the inocula derived from mixed suspensions of transformants. These are shown inFigure 6. In Figure 6, files A, B and C are shown plasmidDNA preparations derived from a mixed culture resulting from the inoculum which contained 44 independently isolated pRO161 Tc5 transformants. The other mixed culture DNA preparations were derived from inocula prepared from 100 Tc5 transformant colonies. None of the files show unique plasmids corresponding to the number of transformants used for their preparation. However, throughout the files, some plasmid bands are more prominent than others perhapsindi cative of either higher copy number recombinant plasids at those locations or alternatively, several different recombinant plasmids of approximately the same size.I favor the second alternative in view of the utility of these plasmid DNA gene banks for selecting almost any marker tested to date when various mutants are used as transforming recipient bacteria. Also, when these DNA gene banks are used as a source of transforming DNA with direct selection for a given marker, the frequency of transformation varies widely for various markers, This suggests that some recombinant plasmids may be present in low concentration and not visualized on the agarose gel shown inFigure 6. A summary of the recombinant plasmid phenotypes isolated from the earlier work with direct selection and also from the gene banks shown in Figure 6 is presented in Table 6. Table 6. Representative recombinant plasmids of the PAO chromosomea PAO strain and plasmid designation Markers complemented PA025 pR0l703 arg PAO236 pR1615 b ilvB,C PA0236 pR01616 b met 28 PAO236 pRO1654 b lys 12 PA0236 pR01657 proA PA0236 pR01658 hisII PAO2003 pRO1700 b rifA b PA02003 pR017Q2 argH PAO2178 pRO1669 b catA met 9011 PAO2198 pRO1687 chu 9002 PAO2198 pRO1689 lys 9015 PA0219 8 pROl69O chu 9002 trpA,B PA02l98 pR01694 leu lO PAO2324 pRO1677 puuD PAO2324 pRO1683 met 9011 PAO2369 pRO1679 nar 9011 PAO2369 pRO1680 nar 9011 tyu 9009 PA02369 pRO1682 catA GMA052 pRO1707 hisIV GMA57 pRO1705 hisV GMA065 pRO1713 hisIII GMA253 pRO1704 hisI Table 6 continued a The derivation of the mixed plasmid DNA suspensions is discussed in the text. These recombinant plasmids were obtained using appropriate PAO nutritional mutants. They were isolated using direct selection from cleaved chromosome and vectorDNA that was ligated.In Table 6, only one recombinant plasmid for each phenotype is listed. However, in most instances, several recombinant plasmids for a given phenotype were isolated and they vary in size for their independent isolations the larger versions show more than one BamHI cleavage site when plasmid DNA is isolated and digested from the various isolates of the same phenotype. As shown earlier in Figure 4, however, a band unique to the marker in question is found in all recombinant plasmids of the same phenotype. In some cases, selection for one marker results in the isolation of recombinant plasmids which also include adjacent markers. For example, plasmids pR01680, pR01669 and pRO1690 which respectively were selected for nar 9011, met 901l,and trpA,B were subsequently tested for adjacent markers shown on the Royle et al. map Royle, P., H.Matsumoto, and B. Holloway. J. Gen. Microbiol. 43 159 271 1981 and were found to complement these too. On the other hand, recombinant plasmids have been selected which encode for these markers singly. These results suggest that the conditions used for digestion of chromosomal DNA result in incomplete digestion of the chromosome allowing for the isolation of fragments of variable length from the same region of the chromosome. The recombinant plasnids listed in Table 6 correspond to scattered regions of the PAO chromosome. This is evident from Figure 7 where their map location corresponding to chromosomal mutations they complement is shown.For this, I have used the most recent map of Royle et al as a guide to position the markers in relation to a time of entry map using PAO sex factors and transductional analysis linkage determinations. Most of the plasmids listed inTable 6 and Figure 7 were obtained from the gene pools shown in Figure 6. This result suggests that procedures for the preparation of chromosome, cleavage with BamHI and indirect selection for the isolation of recombinant plasmids followed by the preparation of recombinant DNA from mixed cultures is suitable as a general protocol. These procedures apparently do not bias against the molecular cloning of significantly extensive regions of the PAO chromosome. Clarke and Carbon sheared the Escherichbc coli chromosome and then formed hybrid plasrnids in vitro using pOly dA.dT connectors Clarke, L., and J. Carbon. Proc. Nat. Acad. Sci. USA 72 4361 4365 1975 . This was done to diminish the likelihood of recovering chromosomal fragments which contained cleavage sites in structural genes and to obviate the need in the vector for z cloning site which would be insertionally inactivated with the addition of a cloned fragment of chromosome. In the present invention the recovery of DNA fragments cleaved in structural genes was minimized by the method of the present invention which reproducibly results in partially digested chromosomalDNA. The utility of this is suggested by the isclation of recombinant plasmids with additional BamHI sites within the BatHI cloned DNA.Therefore, if a particular allele contained a cleavage site within its structure, partial digests enhanced the recovery of an intact allele. In this regard, the nap distribution Figure 6 of recombinant plasmids listed in Table 6 suggests that partially cleaved fragments of chromosomal DNA were isolated from disparate regions of the PAO chromosome. Furthermore, the use of BamHI for this purPose may have promoted the recovery of diverse recombinant plasmids in view of the extent of cleavage cf this enzyme of the chromosome and the distribution of BamHI cleavage sites on the chromosome. The following table shows the recombinant plas mids from application Serial No. 147,563 and the numbers assigned by the Northern Regional Research Laboratory inPeoria, Illinois. Plasmid NRRL No. RP1 B 12123 PR0 RP1 1600 B 12124 pRO1601 3 12125 pRO 1613 B 12126 pRO 1614 B 12127 pRO 1615 B 12149 pRO 1616 B 12148