# A precursor of a C terminal amidated peptide and production thereof.

## Abstract
A precursor of a C terminal amidated peptide repre sented by the general formula P X Gly Y

## Claims
What is claimed is 1. A precursor of a C terminal amidated peptide, which is represented by the general formula P X Gly Yn wherein P is a peptide residue, X is an amino acid residue the C terminal on the Gly side of which is capable of being converted in vivo to a CONH2 group,Gly is a glycine residue, Y is a basic amino acid residue, n is an integer of 2 to 4, and a further amino acid residue other than Z or a peptide residue may optionally be attached to Yn 2. The precursor according to Claim 1, wherein P is a peptide residue of the general formula Cys B2 Asn Leu Ser Tnr Cys B8 Leu B10 B11 B27 Gly B29 B30 B31 wherein represents the 12th to 26th Bs, B in each occurrence is a natural amino acid residue and X isPro. 3. The precursor according to Claim 1, wherein P X is a calcitonin residue. 4. The precursor according to Claim 2, wherein 32 3 10 is Ser or Gly, B3 is Met or Val, B10 is Gly or Ser, B is Lys, Thr or Ala, B12 is Leu or Tyr, B13 is Thr, Ser or Trp, B14 is Gln, Arg or Lys, B15 is Glu, Asp or Asn,B16 is Leu or Phe, B17 is His or Asn, B18 is Lys or Asn,B19 is Leu, Phe or Tyr, B20 is Gln or His, B21 is Thr or Arg, B22 is Tyr or Phe, B23 is Pro or Ser, B25 isThr or Met, B26 is Asn, Asp, Ala or Gly, B27 is Thr, Iteu or Phe, B29 is Ser, Ala, Val or Pro, B30 is Gly orGlu and B31 is Val, Thr or Ala. 5. The precursor according to Claim 1, wherein P X is a calcitonin residue having the formula Cys Gly Asn Leu Ser Thr Cys W Leu Gly Thr Tyr Thr Gln Asp Phe Asn Lys Phe Ris Thr Phe Pro Gln Thr Ala Ile Gly Val Gly Ala Pro wherein W is Val or Met. 6. The precursor according to Claim 1, 2, 3, 4, 5 or 6, wherein the basic amino acid residue Y is a lysine or arginine residue. 7. The precursor according to Claim 1, 2, 3, 4, 5 or 6, wherein Yn is Lys Lys Arg OH. 8. A gene comprising a chemically synthesized doublestranded polynucleotide coding for the amino acid sequence represented by the formula Cys Gly Asn Leu Ser Thr Cys W Leu Gly Thr Tyr Thr Gln Asp Phe Asn Lys Phe His Thr Phe Pro Gln Thr Ala I le Gly val Gly Ala P ro Gly Lys Lys Arg OH wherein W is Val or Met. 9. The gene according to Claim 8, wherein a codon for methionine is added to the head of the nucleotide sequence of the gene and a translation terminating codon is added to the tail end. 10. The gene according to Claim 9, wherein a sequence capable of being cleaved by a restriction enzyme is further added to each end of the nucleotide sequence. 11. The gene according to Claim 8, wherein an EcoRI cleavage site is added upstream Con the S side and a BamHI cleavage site downstream on the 3 side . 12. The gene according to Claim 8, 9r 10 or 11, wherein at least one of RsaI, KpnI, SmaI and XmaL cleavage sites is provided in the nucleotide sequence. 13. A unicellular organism derived or animal cellderived plasmid or hybrid plasmid with an inserted DNA containing the gene according to Claim 8. 14. The plasmid according to Claim 13, wherein the gene according to Claim 9, 10 or 11 is ligated to a structural gene in a unicellular organism or animalderived, promotor region containing operon so that correct reading frame placement can be obtained. 15. The plasmid according to Claim 14, wherein the structural gene is the Escherichia coli APase or betagalactosidase structural gene. 16. The plasmid acccording to Claim 14, which is pAHPCT38 as shown in Fig. 6. 17. The plasmid according to Claim 14, which is pHPCT4 as shown in Fig. 6. 18. A unicellular organism or animal cell transformed with the plasmid according to Claim 14. 19. The unicellular organism or animal cell according to Claim 18, wherein the plasmid is pAHPCT38 as shown in Fig. 6. 20. A method of producing a precursor of a C terminal amidated peptide which comprises cultivating host cells transformed with a plasmid containing an inserted polydeoxyribonucleotide having an added code for Met on the 5 end side, followed by a nucleotide sequence coding for the amino acid sequence of a polypeptide of the formula P X Gly Yn wherein P is a peptide residue, X is an amino acid residue the C terminal Con the Gly side of which is capable of being converted in vivo to a CONH2 group, Gly is a glycine residue, Y is a basic amino acid residue, n is an integer of 2 to 4, and a further amino acid residue other than Y or a peptide residue may be attached to Ynt and further followed by a nucleotide sequence coding for the termination of translation, and thereby causing formation of the above polypeptide as a protein component in the culture broth. 21. The method according to Claim 20, wherein P X is a calcitonin residue.

## Description
A precursor of a C terminal amidated DeDtide and oro diction thereof This invention relates to a precursor of a peptide the C terminal of which is amidated to a CONH2 group and a method of producing the same. Among C terminal amidated peptides, some are pharmacologically active. For instance, calcitonin is a peptide composed of 32 amino acid residue with theC terminal being in the amide form and is used for the treatment of hypercalcemia, osteoporosis and Beh√ßet disease of the bone or for promoted osteogenesis. The amide form C terminal structure is considered to be essential to activity development. So far, pig, ox, sheep, human, rat, salmon and eel calcitonins have been isolated and the structure of each calcitonin has been elucidated. Among them, human, eel, pig and salmon calcitonins are now available on the market for therapeutic use. These are either extraction products from animal bodies or chemically synthesized products. However, the calcitonin content in animal bodies is low and the synthesis is difficult. The present inventors intended to produce calcitonin on a large scale and at a low cost by using the gene manipulation technology. However, the present day gene manipulation technology cannot affordC terminal amidated peptides. Unexpectedly, it was found that precursors resulting from addition of glycine Gly to the amino acid which is to be amidated and further addition of two or more basic amino acids such as lysine Lys or arginine Arg also can produce pharmacological effects. The reason is presumably that the precursors are cleaved and modified in vivo under enzymatic actions to form C terminal amidated peptides. The present invention, which has been completed on the basis of this novel finding, provides, in one aspect thereof, a precursor of a C terminal amidated peptide, which is represented by the general formula P X Gly Yn I wherein P is a peptide residue, X is an amino acid residue the C terminal ton the Gly side of which is capable of being converted in vivo to a CONH2 group,Gly is a glycine residue, Y is a basic amino acid residue, n is an integer of 2 to 4, and a further amino acid residue other than Y or a peptide residue may optionally be attached to Since, in vivo, X Gly is presumably cleaved in the middle to yield a terminal amidated peptideP X NH2, adequate selection of the peptide residue P can give precursors corresponding to the desired terminal amidated peptides, for instance calcitonin, oxytocin, vasopressin, LH RH, melittin, eledoisin, secretin, kassinin, a MSH, substance P, ranatensin, pancreatic polypeptide and VIP. P X may preferably be a peptide residue of theFollowing formula II Cys B2 Asn Len Ser Thr Cys B8 Leu B10 B11 B27 Gly B29 B30 B31 Pro II wherein, represents the 12th to 26th Bs, each B being independently a natural. amino acid residue. P X may be a calcitonin residue and, in the above formula, B2 may be Ser or Gly, B8 may be Met or Val, B10 may be Gly or Ser, B11 may be Lys, Thr or Ala,B12 may ba Leu or Tyr, B13 may be Thr, Ser or TRp, B14 may be Gln, Arg or Lys, B15 may be Glu, Asp or Asn,B16 may be Leu or Phe, B17 may be His or Asn, B18 may be Lys or Asn, B19 may be Leu, Phe or Tyr, B20 may be Gln or His, 321 may be Thr or Arg, B22 may be Tyr orPhe, 323 may be Pro or Ser, B25 may be Thr or Met,B26 may be Asn, Asp, Ala or Gly, B27 may be Thr, Val,ILeu or Phe, B29 may be Ser, Ala, Val or Pro, B30 may be Gly or Glu and B31 may be Val, Thr or Ala. More specifically, P X may be a calcitonin residue of the following formula III Cys Gly Asn Leu Ser Thr Cys W Leu Gly Thr Tyr Thr Gln Asp Phe Asn Lys Phe His Thr Phe Pro Gln Thr Ala Ile Gly Val Gly Ala Pro CIII wherein W is Val or Met. The basic amino acid residue Y is a lysine or arginine residue. A preferred example of Yn is LysLys Arg OH. The above precursors can be produced by the method to be described hereinlater. In another aspect, the present invention provides a gene which is a chemically synthesized double stranded polynucleotide coding for the amion acid sequence of the above mentioned calcitonin residue. It is preferable that a codon for methionine is added to the head of the nucleotide sequence of the above gene and a translation terminating codon to the tail thereof Furthe ore, it is preferable that a restriction enzyme cleavable is further added to each end for instance, an EcoRI cleavage site upstream Con the 5 side and a BamHI cleavage site downstream on the 3 side . Furthe ore, it is possible to introduce one or more restriction enzyme cleavage sites, for instance one or more RsaI, KDnI, SmaI and or XmaI site or sites, into the nucleotide sequence. Referring to the drawings, the figures are concerned with an embodiment of the present invention, which is to be described later by way of example. Thus, Fig. 1 illustrates, in the upper row, the amino acid sequence on which the synthetic gene designing is based and, in the lower row, the structure of mature calcitonin, with the portions in the lower row which are the same as those in the upper row being represented by line segments.Fig. 2 shows the design of a gene, HPCT,Fig. 3 shows the chemically synthesized fragments CFl to F20 of the HPCT 1 36 gene, Fig. 4 shows the process for immobilizing a 3 terminal mononucleotide unit on an aminomethylated polystyrene resin, Fig. 5 shows the structure of 3 phosphodiester dimer tor trimerl blocks, Fig. 6 shows the process for constructing a recombinant plasmid containing the HPCT gene,Fig. 7 shows the HPLC elution pattern for HPCT produced by a bac erLal strain transformed with the abovementioned plasmid, and Fig. 8 shows the HPLC elution pattern for the fraction presenting a peak in Fig. 7.Fig. 9 shows the amino acid sequence of HPCT isolated by the above HPLC. In Fig. 9, Cp means carboxypeptidase,EMI5.1 denotes the amino acid sequence as determined byEdman s degradation method,EMI5.2 denotes the carboxy terminal portion amion acid sequence as determined by the carboxllpeptidase method,EMI5.3 denotes the carboxy terminal portion amino acid sequence as estimated by the carboxypeptidase method, andEMI6.1 denotes the carboxy terminal as determined by the hydrazine decomposition method. Fig. 10 shows the HPLC elution pattern for the tryptophan decomposition product fromHPCT, and Figs. 11 and 12 are the 8 polyacrylamide gel electrophoresis mappings for the gene synthesized from the oligonucleotide fragments F1 to F20. A preferred example of the design of the doublestranded gene DNA is given in Fig. 2. The chemical synthesis of said gene is performed by first synthesizing the twenty DNA fragments F1 to F20 shown in Fig. 3 by the solid phase method Miyoshi, K., Nucleic AcidsRes., 8, 5507, 1980 and then preparing the doublestranded DNA from these fragments by the ligase reaction method. In a still another aspect,the present invention provides a unicellular organiem derived or animal cellderived plasmid or hybrid plasmid with a DNA containing the above gene and inserted therein. An embodiment includes plasmids produced by connecting the above gene to a structural gene e.g.APase or beta galactosidase in an operon containing a promoter region derived from aunicellular organism, such as a microbe e.g. bacterium, yeast, mold or an animal cell such as monkey kidney cell, in a manner sucb that correct reading frame placement can be obtained. A preferred example is pAHPCT38 as illus heated in Fig. 6 or pHPCT4 as shown in Fig. 6. pHPCT4 can be obtained by inserting the above chemically synthesized gene in between the EcoRI and BamHI cleavage sites of pKO13 cf. Japanese Patent ApplicationSho 56 163303 , while pAHPCT38 can be obtained by inserting the above chemically synthesized gene in between the EcoRI and BamHI cleavage sites of the plasmid pA NE1 cf.Japanese Patent Application Sho 56 170543 derived from an Escherichia coli alkaline phosphatase APase gene containing plasmid pBR322. In a further aspect, the present invention relates to unicellular organisms or animal cells transformed with the above plasmid. In an embodiment thereof, a transformant strain WA802 pBPCT could be obtained by transformation of anEschrichia coli strain K12 WA802 with the plasmid PHPCT4, and a transformant strain E15 pAHPCT could be obtained by transformation of an Escherichia coli strain K12.E15 with the plasmid pAHPCT38. The strain E15 pAHPCT38 has been given a code nameSBMC138 and deposited with the Fermentation ResearchInstitute, Agency of Industrial Science and Technology under Deposit No. FERM BP 283. In a still further aspect, the present invention provides a method of producing a precursor of a C terminal amidated petite, which comprises cultivating host cells transformed with a plasmid with athereininserted gene having a nucleotide sequence coding for the amino acid sequence of a peptide of the above formula I with a code for Met added on the S end side thereof and having a subsequent nucleotide sequence coding for the termination of translation, and thereby causing formation of the above peptide as a protein component in the culture broth. An example of the transformed host cells is the above mentioned El5 AHPCT. Taking this as an example, the method of cultivation is described. First, in a typical case, said transformant is cultured in a high phosphoric acid content medium and then transferred to a low phosphoric acid content medium, whereby APase is induced and the desired peptide precursor is formed as a protein component in the cells. Preferred procedure and conditions of cultivation should preferably be selected depending on the plasmid species and host cell species. After cultivation, the cells are separated from the culture, and treated with formic acid cyanogen bromide. The desired fraction containing the desired peptide can be collected, for example, by density gradient fractional elution on an SP Sephadex column using a pyridine acetic acid buffer and purified byHPLC. The thus obtained precursor of C terminal amidated peptide is as active as the C terminal amidated peptide.For instance, the precursor HPCT shown in the upper part of Fig. 1 showed calcitonin activity, markedly reducing the calcium level in the rat blood. The abbreviation HPCT stands for a calcitonin precursor in which Met8 of human calcitonin is replaced by Val. ExampleChemical svnthesis of HPCT gene fragment The chemical synthesis of 20 oligodeoxyribonucleotides respectively designated F1 through F20 in Fig. 3, except for the improvements mentioned hereinafter, was carried out basically by the solid phase method reported by Ken ichi Miyoshi et al Nucl. Acids Res. 8, 5507 1980 1. The outline of chemical synthesis of these oligodeoxyribonucieotides by the improved technology is as follows. a First, as shown in Fig. 4, the 3 end mononucleoside unit was bound to an aminomethylated polystyrene resin. Thus, the aminomethylated polystyrene resin 1 shown in Fig. 4 is a resin cross linked with 1 divinylbenzene, and the resin 2 was prepared by adding 2 units of beta alanine to 1 . Then, three different compounds 3 B represents thymine, N benzoylcytosine or Nisobutyrylguanine were reacted with the above resin 2 to provide 3 different polystyrene resins 4 each carrying the 3 end nucleoside. The following is an example of synthesis of the same resin 4 relevant to thymine for B in Fig. 4. Fifteen grams of aminomethylated polystyrene HCl divinylbenzene 1 , 100 200 mesh, NH2 0.63 m mol g was swollen well with 75 ml of CH2Cl2 and N tBOC beta alanine 1.01 g, 11.34 m moles was added.The condensation reaction was conducted in the presence of DCC 2.34 g, 11.34 m moles at room temperature for 3 hours. The resin was filtered through a glass filter and washed 4 times with 150 ml portions of CH2C12, 4 times with 150 ml portions of DMF and finally 4 times with 50 ml portions of pyridine. Then, the resin was treated with 75 ml of 25 acetic anhydride pyridine for 30 minutes to acetylate the unreacted amino groups. Thereafter, the resin was treated with 90 ml of 20 CF3COOHCH2C12 at room temperature for 25 minutes to eliminate the t BOC groups. The resin was washed with 150 ml of 5 triethylamine CH2C12 and further washed 4 times with 50 ml portions of CH2C12. The above procedure was repeated for a second time to give the resin 2 of Fig. 4, i.e. the aminomethylated polystyrene resin having two ss alanine units.In 30 ml of DMF was suspended 5 g of the above resin Fig. 4, 2 i, followed by the addition of 5 m moles of compound 3 Fig. 4, 3 where B means N thymine and 500 mg of triethylamine. The mixture was shaken at room temperature for 12 hours.After the reaction, the resin was filtered through a glass filter, washed with DMF and, then, with pyridine, and finally treated with 75 ml of 10 phenyl isocyanate pyridine for 3 hours to protect the unreacted amino groups. The resin was subsequently washed with pyridine methanol and dried under reduced pressure in the presence of P205, whereby the resin 4 of Fig. 4 B means thymine was obtained. Using a portion of this resin Fig. 4, 4 , where B is thymine , the amount of 3 end nucleoside secured to 1 gram of the resin was determined by the assay method of M.J. Gait et al Nucl. Acids Res. 8, 1081 1980 . The result was 0.153 m mole. The resin 4 wherein B is N benzoylcytosine or Nisobutylquanine was also prepared in the same manner as above. The amount of 3 end nucleoside secured to 1 g of resin 4 was 0.150 m mole for B N benzoylcytosine and 0.147 m mole for B N isobutylguanine. b Using these three kinds of resin 4 and the 3 phosphoric diester dimer block Fig. 5, n 0 and 3 phosphoric acid diester trimer block Fig. 5, n l , 20 oligodeoxyribonucleotides F1 through F20 having the specified predetermined base sequences.Thus, the resin 4 was treated with 2 benzenesulfonic acid CHCl3 MeOH 7 3, v,v to remove the dimethoxytrytyl group briefly, DMTr and, then, 4 equivalents of 3 phosphoric acid diester dimer Fig. 5, n O or 3 equivalents of 3 phosphoric acid diester trimer Fig. 5, n l per equivalent of the 3 nucleoside unit secured to resin 5 was sequentially condensed with the aid of 20 to 50 equivalents of mesitylenesulfonyltetrazolide MsTe .The excess 3 phosphoric acid diester block and condensing agent could be easily removed by filtering the reaction mixture through a glass filter. Then, to protect the unreacted 5 hydroxy group, the resin was treated with 10 acetic anhydride pyridine at room temperature for 1 hour, whereby the unreacted 5 hydroxy groups were acetylated. The resin was then treated with 2 benzenesulfonic acid CXC13 MeOH 7 3, v v to remove the 5 end DMTr group and, then, the next condensation reaction was conducted. The above procedure was repeated until the required length was obtained. An exemplary sequence of synthesis is given below with reference to the F3 fragment. 200 mg of resin 4 Fig. 4, 4 , B N benzoylcytosine was treated twice with 20 ml of 2 benzenesulfonic acid CHC13 MeOH 7 3, v v hereinafter abbreviated to 2 BSA at room temperature for 1 minute each to give the resin 5 free from DMTr. To this resin 5 was added 90 V moles of the 3 phosphoric acid trimer block GAG Fig. 5, n 1, B 1 N isobutyrylguanine, B2 N benzoyladenine, B3 N isobutyrylguanine and the azotropic distillation with 3 ml of pyridine was repeated 3 times. Then, 2 ml of pyridine was added to the resin and the condensation reaction using 1 m mole of the condensing agent MsTe was carried out at room temperature for 2 hours.After the reaction, the excess 3 phosphoric acid trimer and condensing agent could be easily removed by filtering the reaction mixture through a glass filter and washing the resin well with pyridine. Then, to protect the unreacted 5 hydroxy groups, the resin was suspended in 20 ml of 10 acetic anhydride pyridine and the suspension was shaken at room temperature for 1 hour. After this reaction, the resin was separated by filtration through a glass filter and washed well with pyridine and CHCl3 in the order mentioned.Then, the resin was treated twice with 20 ml of 2 BSA CHCl3 MeOH 7 3, v v for 1 minute each, whereby the DMTr group was removed from the resin.The resin was washed well with CHC13 and pyridine in that order. The above procedure was repeated until the required length was obtained. Thus, to synthesize the oligodeoxyribonucleotide of F3, the next 3 phosphoric acid blocks CCT, AA and GGT were sequentially employed. By these sequential condensation reactions, a completely protected oligodeoxynucleotde resin having the base sequence of F3 was obtained. The other frangments F1, F2 andF4 to F20 were also produced in the same manner except that different combinations of nucleotides were employed. c The severence of the completely protected oligodeoxyribonucleotide having such a definite base sequence from the resin and the complete deprotection thereof were carried out by treating the resin with concentrated aqueous ammonia pyridine 2 1, v v at 550C for 10 hours and, then, with 80 acetic acid at room temperature for 10 minutes. This deprotection procecure is described below with reference to F3 fragment. In a sealed tube, 50 mg of the F3 resin was shaken with 1 ml of pyridine and 2 ml of concentrated aqueous ammonia at 550C for 10 hours, at the end of which tie the resin was filtered off and the filtrate was cncentrated under reduced presure. To the residue was added toluene and the azeotropic distillation was repeated to remove the pyridine. The residue was treated with 2 ml of 80 acetic acid at room temperature for 10 minutes, whereby a completely deprotected oligodeoxyribonucleotide was obtained. d The isolation and purification of the final product was carried out by high performance liquid chromatography briefly, HPLC . Thus, ALTEX Model 110 A Liquid Chromatograph, a linear gradient method using Solvent A 0.05 M KH2PO4, pH 4.0 and SolventB 0.05 M KH2PO4 1.0 M KC1, pH 4.0 , and aPermaphase AAX Du Pont column 0.21 x 50 cm were employed. The gradient was established by adding 3 of Solvent B to Solvent A at the intervals of 1 minute. Elution was carried out at 580C and a flow rate of 2 ml min. The fractions containing the desired compound were pooled, desalted by dialysis and lyophilized.Each of the fragments thus obtained was further purified by HP on a linear gradient of Solvent A 0.01 M EDA 2.5 acetonitrile, pH 7.0 and Solvent B 0.01 M EDA 25 acetonitrile, pH 7.0 in a p Bondapak C18 column 0.4 x 25 cm .The gradient was established by adding 3 of SolventB to Solvent A at intervals of 1 minute. Elution was carried out at room temperature and a flow rate of 2 ml min. The fractions containing the desired compounds were respectively pooled and lyophilized to obtain 20 DNA fragments F1 through F20. e The homogeniety of the 20 completely deprotected oligodeoxyribonucleotides F1 through F20 thus obtained was confirmed by labeling their 5 ends with gamma 32PiATP using T4 polynucleotide kinase and carrying out polyacrylamide gel 20 electrophoresis. The base sequence of each oligodeoxyribonucleotide was confirmed by the two dimensional mapping method Bambara, J.E. et al 1974 Nucl. Acids. Res. 1, 3313. Thus, the 5 end was was labeled with gamma 32P ATP using T4 poly nucleotide kinase and, then, partial hydrolysis was carried with nuclease. This partial hydrolysate was subjected to cellulose acetate electrophoresis pH 3.5 and DEAE cellulose homochromatography. Libation of HPCT aene fragment The 51ends of the chemically synthesized 18 oligonucleotide fragments F2 through F19 Fig. 3 were phosphorylated and after F1 and F20 fragments were annealed, they were ligated by means of T4 DNA ligase to synthesize a gene co esponding to HPCT consisting of 122 base pairs. Details of the procedure are given below. Excepting F1 and F20, 4.2 pg of each or the remaining 18 fragments was dissolved in 22 1 of distilled water and the solution was heated at 900C for 2 minutes and, then, immediately cooled to OOC. To this aqueous solution was added 10 yCi of y 32P ATP 5100 Ci m mole . It was then adjusted to give a 50 mM Tris HCl pH 7.5 10 mM MgC12 2 mM spermine100 mM KCl 10 mM DTT oligonucleotide kinase buffer solution. Then, 3 units of polynucleotide kinase was added to make a total of 30 p1. By conducting the reaction at 370C for 15 minutes, the 5 ends were labeled with 32p Then, to phosphorylate all the 5 ends, 4 n moles of ATP and 3 units of polynucleotide kinase were further added and the reaction was continued at 370C for 45 minutes.The reaction was terminated by heating the mixture at 900C for 5 minutes. 2.1 g each of the above 18 phosphorylated fragments F2 through Flg and unphosphorylated fragments F1 and F20 were respectively mixed and dialyzed in a dialysis tube against distilled water at 40C for 16 hours to remove the unreacted ATP and kinase buffer components. ThisDNA dialyzate was concentrated to dryness and dissolved in 14.5 1 of ligase buffer 20 rn1 trisHCl, pH 7.6, 10 mM MgCl2 . This solution was put in a 0.5 ml Eppendorf tube and heated at 950C for 2 minutes. Thereafter, the temperature was gradually lowered to room temperature in 30 minutes to anneal the 20 fragments.The reaction mixture was then held at OOC. This DNA solution was made into a ligase buffer containing 0.4 mM of ATP and 10 mM ofDTT. Then, 30 units of T4 DNA ligase was added to make a total volume of 80 iil and the reaction was allowed to proceed at 110C for 16 hours. A portion of the reaction mixture was taken and subjected to electrophoresis with 18 polyacrylamide gel. Using two kinds of gels, one containing 7 M of urea and the other not containing urea, electrophoresis was carried out and the reaction product was analyzed by radioautography. It was found that a DNA corresponding to 122 base pairs had been produced at the rate of about 20 percent Figs. 11 and 12 .Then, the entire amount of the above reaction mixture was fractionated by electrooresfs through 8 polyacrylamide gels. After the 122 base pairs of DNA were confirmed by radioautography, the corresponding portion of the gel was cut out and placed in dialysis tubes. Each tube was filled with 9 mM Tris borate pH 8.3 0.25 mM EDTA buffer and after closing both ends of the tube, electrophoresis was conducted in the buffer at a constant voltage of 200 V for 3 hours to elute the DNA from the gel. The DNA solution was taken out from the dialysis tube and lyophilized. The lyophilisate was dissolved in 100 iil of 0.3 M sodium acetate pH 7.0 , and 2.5 volumes of ethanol was added. The mixture was allowed to stand at 80 C for 30 minutes, at the end of which time it was centrifuged to recover the DNA sediments. TheDNA thus obtained was dissolved in 30 1 of kinase buffer, 6 units of polynucleotide kinase and 0.8 nmolATP were added, and the reaction was conducted at 370C for an hour. Thereafter, 3 p1 of 3 M sodium acetate was added, followed by 2.5 volumes of ethanol. There was thus obtained about 2 vg of theHPCT gene comprising 122 base pairs as the DNA precipitate. Construction of recombinant plasmids Fig. 6 illustrates the method of constructing recombinant plasmids containing the HPCT. The method is described hereinbelow in more detail. A Construction of plasmid pHPCT The method of construction of the plasmid pKO13 is as distinctly disclosed in the specification of Japanese Patent Application No. 163303 1981.The plasmid pKOl3 is a plasmid having most of the beta galactosidase genes from the promoter region. of the Escherichia coli lactose operon. The plasmid has one EcoRI and one BamHI cleavage site, and therefore genes obtained chemically or from natural sources and having EcoRI and BamH cohesive ends can be easily inserted into the plasmid pKO13. Furthermore, it is possible to cause the genes inserted downstream to the EcoRI cleavage site to express in terms of hybrid proteins with beta galactosidase under the control of the lactose promoter. Following decomposition of the hybrid proteins with cyanogen bromide the desired peptides can be separated and purified. Thus, 5 g of pKOl3 was reacted at 370C for 60 minutes with 1 unit each of the restriction enzymesEcoRI and BamHI in 40 1 of I x TA mixture containing 33 mM Tris acetate buffer, pH 7.6, 66 mM potassium acetate, 10 mM magnesium acetate and 0.5 mM dithiothreitol . The reaction mixture was heated at 650C for 30 minuites so as to inactivate the enzymes.Then, 0.7 agarose electrophorests was carried out and large DNA fragments were recovered by electrophoresis. The resulting EcoRI BamHI cleaved fragmentDNA of pKOl3 1 pg and 0.6 pg of the 5 OH phosphorylated HPCT gene previously obtained were dissolved in 40 ul of T4 DNA ligation mixture 20 mM Tris HCl buffer pH 7.6, 10 mM MgC12, 10 mM DTT, 0.4 mM ATP and reacted with 2 units of T4 DNA ligase at 50C for 16 hours. To this reaction mixture was added 2.5 volumes of cold ethanol to give a DNA precipitate. This DNA was dissolved in 15 pl of distilled water and using 10 pl of the solution, it was introduced into E. coli WA802.The transformant was cultivated on a nutrient agar medium containing 40 pg ml of ampicillin at 370C overnight and the colony formed was replicated on a nutrient agar medium containing 10 pg ml of tetracycline to obtain tetracycline sensitive colonies.The ampicillin resistant, tetracycline sensitive transformants thus obtained were designated asWA802 pHPCTl through WA802 pHPCT5. The DNA base sequence of the small EcoRI BamHI cleaved DNA fragment obtained from WA802 pHPCT4 was analyzed by the procedure described below to find that the pHPCT4 plasmid had the DNA base sequence of the desired HPCT gene. In this WA802 pHPCT4 strain, HPCT is expressed as a ss galactosidase HPCT hybrid protein under the control of the lactose promoter and the HPCT can be separated as a complete peptide desired from this hybrid protein by methionine decomposition with cyanogen bromide. However, since the ss galactosidase protein contains 23 methionine residues, a number of unnecessary impurity peptides are produced in the cyanogen bromide decomposition, thus interferring with the separation and purification of the desired peptide. Therefore, we developed as a vector free from the above disadvantage a plasmid derived from the E. coli alkaline phosphatase gene and established its usefulness Japanese Patent Application No.170543 1981 . Therefore, in the thought that for the separation and purification of the HPCT peptide from Escherichia coli, it is more desirable to construct a plasmid by recombination of the alkaline phosphatase gene based plasmid with the HPCT gene, we performed the construction of such a plasmid as follows. The HPCT base sequence in the plasmid carried by the above transformant strains was analyzed in the following manner.HPCT base seauence Each 150 uS of the plasmid DNAs, pHPCTl, pHPCT2, pHPC3 and pHPCT4, was digested with 200 units of the restriction enzyme EcoRI and 200 units of the restriction enzyme SalI, and a DNA fragment comprising about 400 base pairs inclusive of thePHCT gene was separated and purified. Then, the 5 end was dephosphorylated with bacterial alkaline 32 phosphatase, and the 5 end was labeled with 32p using y 32P ATP and polynucleokinase. Thereafter, digestion was conducted with 100 units of the restriction enzyme 3amHI, and a DNA fragment comprising 122 base pairs was separated and purified and subjected to base sequencing by the Maxam Gilbert method Proc. Natl. Acad. Sci., U.S.A., 74, 560 564, 1977 .As a result, it was confirmed that all the plasmids except for pHPCT1 had the base sequence as initially designed. In pHPCTl, the third member C of the codon AAC corresponding to the 17th amino acid of HPCT, namely asparagine, had been substituted by T. Nevertheless, the codon AAT also corresponds to asparagine. B Construction of plasmid pAHPCT38 The method of constructing plasmid pAaNEl is detailed in the specification of Japanese PatentApplication No. 170543 1981. pAaNEl is a plasmid prepared by incorporating an a neoendorphin gene downstream of the EcoRI cleavage site of an E. coli alkaline phosphatase structural gene, and is a vector which expresses an alkaline phosphatase alpha neoendorphin hybrid protein under the expression and modulation control of alkaline phosphatase.The procedure for construction of plasmid pAHPCT38 containing an HPCT gene in place of the alphaneoendorphin gene is described in detail below. In 100 pl of 1 x TA mixture, 10 1 of plasmid pAaNEl was partially cleaved using 10 units of the restriction enzyme EcoRI at 370C for 30 minutes. Then, 0.78 agarose gel electrophoresis was carried out and the DNA fragment cleaved by EcoRI at one site only was recovered by electrophoresis. The DNA fragment thus recovered was completely cleaved in 30 1 of 1 x TA using 5 units of the restriction enzyme BamHI at 370C for 60 minutes, after which 2.5 volumes of cold ethanol was added thereto so as to precipitate the DNA. This DNA precipitate was dissolved in 30 pl of distilled water. In 30 1 ofT4 DNA ligation medium was dissolved 0.5 pg of thisEcoRI BamHI cleaved DNA fragment together with 0.5 pg of an HPCT gene DNA prepared from pHPCT4 using EcoRI and BamHI, and ligation was conducted using 2 units of T4 DNA ligase at 50C for 16 hours.To the reaction mixture was added 2.5 volumes of cold ethanol to give a DNA precipitate. This DNA precipitate was dissolved by addition of 10 pl of distilled water and introduced into Escherichia coliE15 Hayashi et al J. Biol. Chem. 239, 3091 1964 1. Each transformant strain was incubated on a nutrient agar medium containing 40 pg ml of ampicillin at 37oC overnight. The 72 ampicillin resistant strains obtained in the above manner were examined for the existence of SmaI cleaved sites in plamids. It was found that 3 strains are found to carrv plasmids having Smal cleaved sites within the HPCT gene. One of these strains was represented asE15 pAHPCT38 and used in the next separation and purification of HPCT peptide. The above Escherichia coli E15 pAHPCT38 has been deposited with the Fermentation ResearchInstitute, the Agency of Industrial Science andTechnology under the deposit number of FERM BP 283.Purification of a human calcitonin precursor from E.coli ElS oAHPCT38 In E coli E15 pAEPCT38, the HPCT gene was inserted into the alkaline phosphatase structural gene. Therefore, HPCT was expected to be expressed as a protein hybridized with alkaline phosphatase.However, as shown in the initial HPCT gene design, by inserting methionine at the N terminal of EPCT and by cleaving this alkaline phosphatase EPCT hybrid protein with cyanogen bromide, HPCT can be easily obtained. Therefore, the separation and purification of HPCT from E, coli K12E15 pAHPCT38 was performed by the procedure described in detail hereinafter. As to the cultivation of E. coli K12E15 pABPCT38 and the induction of the alkaline phosphatase HPCT hybrid protein, these were conducted in accordance with the methods described in the specification of Japanese Patent Application No. 170543 1981. The E. coli E15 pAHPCT38 strain was incubated in 100 ml of TG 20 phosphate rich medium containing 40 pg l of ampicillin at 370C for 16 hours and the culture was used to inoculate 10 t of TG l phosphate lean medium and further incubated at 37 C for 24 hours. The cells were collected by centrifugation to recover 2.36 g of moist cells.The cells were suspended in 20 ml of a 70 formic acid solution containing 500 mg of cyanogen bromide and allowed to stand in the dark at room temperature for 24 hours. Purification of HPCT from E. coli E15 pAHPCT1 The cyanogen bromide treated cell suspension was lyophilized and after addition of 60 ml of distilled water, it was ultrasonicated at room temperature for about an hour. The resulting suspension was centrifuged 10,000 r.p.m., 15 min. and with the supernatant kept undiscarded, the sediment was diluted with 30 ml of distilled water, ultrasonicated again and centrifuged. The supernatant was combined with the previous supernatant to give 90 ml.This solution was made up to 100 ml with 10 ml of distilled water and passed through an SP Sephadex C 25 column 1.4 x 20 cm . The column was previously equilibrated with 0.025 M acetic acid and after application of the sample, it was washed with 230 ml of the same acetic acid solution and, then, with 200 ml of 1.5 pyridine acetate buffer briefly, PA buffer pH 4.47 and 160 ml of 2.5 PA buffer pH 4.65 .Thereafter, elution with 60 ml of 2.5 PA buffer pH 4.65 and 60 ml of 3.75 PA buffer pH 4.76 was carried out by the linear gradient method, followed by further elution with 60 ml of the latter buffer and still further elution on a linear gradient using 160 ml of 3.75 PA buffer pH 4.76 and 160 ml of 5.0 PA buffer pH 4.87 , the eluate being collected in 6 ml fractions. At the interval of several tubes, a 100 pl portion of each fraction was subjected to HPLC analysis. The conditions of HPLC are mentioned hereinafter. The fractions corresponding to the main peaks on HPLC were respectively evaporated to dryness, dissolved in 150 ml of 6 N hydrochloric acid containing 0.1 phenol, and hydrolyzed at 110 0C for 24 hours.The nydrolysate was evaporated to dryness and submitted to amino acid analysis Hitachi 11835 50 Amino Acid Analyzer to monitor the state of elution from the column. As a result, the presence of HPCT was confirmed in the entire 2.58 3.75 gradient eluate total, 120 ml , the entire 3.7S PA buffer eluate 60 ml , and the first 10 fractions 60 ml in the 3.75 5.0 gradient eluate. These HPCT containing fractions were pooled and lyophilized, and the purification thereof by HPLC was carried out. The above dried product was dissolved in 8.0 ml of 0.1 N acetic acid and 100 to 300 1 portions thereof were purified by means of p Bondapack C18 Waters columns 0.39 x 30 cm . Elution was carried out by the linear gradient method using 0.1 trifluoroacetic acid containing 108 of acetonitrile and 0.1 trifluoroacetic acid containing 50 of acetonitrile and the state of elution was monitored at the W wavelength of 210 nm. As shown by the typical chromatogram in Fig. 7, the peak corresponding to HPCT can be clearly detected. The fractions containing such peaks were pooled and HPLC under the same conditions was performed again Fig. 8 to obtain 2.15mg 0.56 u mole of pure HPTC.Structural identification of HPCT The amino acid composition of HPCT was determined on the basis of the amino acid analysis of the hydrolysate of HPCT prepared by hydrolyzing HPCT with 6 N hydrochloric acid containing 0.1 of phenol in a sealed tube and the result of an amino acid analysis conducted after HPCT was oxidized with performic acid and hydrolyzed with 6 N HC1 at 110 C for 24 hours in the conventional manner 1 Table 1 .The amino acid sequence of the performic acid oxidation product of HPCT was analyzed by Edman s degradation method 2 , whereby the sequence to the 15th residue from the N te inal could be determined Fig. 9 . Then, in order to determine the amino acid sequence of total HPCT, HPCT was degraded with trypsin in the hope of obtaining fragment peptides. In 30 p1 of 0.1 N PA buffer pH 7.80 was suspended 160 pg of HPCT and after 30 minutes of ultrasonic treatment, 10 p1 of TPCK treated trypsin solution 3 1 mg ml was added. The mixture was allowed to stand at 370C for 2 hours. The reaction mixture was then lyophilized, dissolved in 400 pl of 0.1 N acetic acid, and subjected to HPLC. The conditions of HPLC used here were exactly identical with those used in the purification of HPCT and the chromatogram monitored at the W wavelength of 210 nm was as reproduced in Fig. 10.Thus, the amino acid compositions of the peaks T1, T2 and T3 suggested that T1 was a peptide from the 19th amino acid unit of HPCT through the 34th amino acid unit,T2 was a peptide from the N te inal through the 18th amino acid unit, and T3 was a peptide from the terminal through the 34th amino acid unit Table 1 . The literature references mentioned in the foregoing are as follows. 1 C. H. W. Hirs, Methods in Enzymology , Academic Press, New York, Vol. 11, P. 197 1967 2 J. P. Van Eerd et al., Biochemistry, 15, 1171 1976 3 S. S. Wang et al., J. Biol. Chem., 240, 1619 1965 Then, the amino acid sequence analysis was carried out using T1 and the performic acid oxidation product T2 Fig. 9 . Moreover, the amino acid sequence analysis of the C terminal ends ofHPCT, T1, and T2 was carried out by the carboxypeptidase method T. Isobe et al. J. Mol. Biol., 102, 349 1976 , and the C terminal analysis thereof was carried out by the hydrazine method 1A. Tsugita et al. Proc. Nat. Acad. Sci., U.S.A., 46, 1462 1960 Fig. 9 . Bsed on results of the above analyses, the amino acid sequence of HPCT was identified as shown in Fig. 9.Purification of HPCT Cp 1 33 , the carboxypeptidase degradation product In 300 pl of 0.1 N PA buffer pH 7.80 was suspended 500 iig of HPCT and after 1 hour of ultrasonication, 7.5 1 of carboxypeptidase B solution 0.67 mg ml was added. The mixture was allowed to stand at 370C for 6 hours. The liberated amino acids 2 moles of lysine and I mole of arginine were detected by amino acid analysis. The reaction mixture was then lyophilized, dissolved in 500 1 of 0.1 acetic acid and purified by HPLC. The conditions of HPLC were identical with those used in the purification of HPTC and the chromatogram was monitored at the W wavelength of 210 nm. Amino acid analysis of the peptide thus purified showed that it is a peptide from the N terminal through the 33rd amino acid, i.e. HPCT Cp 1 33 Table 1 . The yield calculated from the amino acid analysis data was 302 pg or 68 . Table 1The Amino Acid Compositions of HPCT, T1, T2 and HPCT Cp 1 33 HPCT T1 T2 HPCT Cp 1 33 CySO3H 1.6 1.8 1.9 Asp 3.1 3.0 3.0 Thr 4.7 1.8 2.8 4.7 Ser 1.1 1.1 1.0 Glu 2.0 1.0 1.1 2.1 Pro 2.0 1.9 1.9 Gly 4.8 2.8 2.0 5 Ala 2.2 1.9 2.1 Val 1.9 1.0 1.0 2.0 Ile 1.1 1.0 1.1 Leu 2 1.9 2.0 Tyr 1.0 1.0 1.0 Phe 2.7 1.9 1.1 3.0 Lys 2.9 1 1 1.1 His 0.9 0.9 1.0 Arg 1.0 36 16 18 33 In the table, the underscored figures were used to calculate the figures for other amino acids. Biological activit of HPCT The biological activities of HPCT and HPCTCp 1 33 purified by the above mentioned procedure were assayed using the serum calcium lowering effect in rats as an indicator. Procedure MaLe Wistar strain rats weighing 300 to 350 grams were fasted overnight and used. Each animal was secured in supine position and under ether anesthesia, a polyethylene tube Intramedic PE 50,Clay Adams filled with heparin sodium injectionJ. P. Japan Upjohn was inserted into the femoral artery of the right hind leg. The rat was fixed in a Boleman cage, and 30 minutes after awakening of the animal, 0.5 ml of the blood was withdrawn through the above polyethylene tube inserted into the femoral artery into a polyethylene centrifugal tube containing 50 units of heparir. sodium.Immediately thereafter 0.2 ml of the under mentioned physiological saline, HPCT, HPCTCp l 33 or swine calcitonin solution was administered subcutaneously at the back of the animal. Then, 1, 2, 4 and 6 hours after the administration, 0.5 ml each of the blood was taken through the above mentioned polyethylene tube. The blood thus taken was promptly centrifuged on an Eppendor mic ocentrfuge Sppendorf 5414 at 13,000 r.p.m. for 1 minute and the supernatant was used as serum fraction. The blood calcium level was measured by the o cresolphthalein complexon method using a commercial calcium assay kit RM117 S Ca SET Yatron . Fifty iil of the serum was taken in a test tube and stirred with 0.5 ml of the color reagent containing o cresolphthalein complexion and 8 hydroxyquinoline RM117 2 . Then, 5.0 ml of monoethanolamine borate buffer RM117 1 was added dnd stirred. Within 90 minutes thereafter, the absorbance was measured with a self recording spectrophotometer Ritachi 320 at the wavelength of 575 mu. The concentration of calcium was calculated from this absorbance value. The methods used in the preparation of the physiological saline solution, HPCT solution, HPCTCp 1 33 solution and swine calcitonin solution employed are as follows. a Physiological saline solution Bovine serum albumin Sigmae was dissolved in physiological saline J. P. Ohtsuka Pharmaceutical Co. at room temperature to the albumin concentration of 0.1 . b HPCT solution EPCT was diluted with the above mentioned physiological saline solution containing 0.1 of albumin to the HPCT concentration of 4.5 p 0.2ml. c HPCT Cp l 33 solution HPCT Cp 1 33 was diluted with the above mentiond physiological saline solution containing O of albumine to the HPCTCp l 33 concentratn of 20 yg 0.2 ml. d Swine calcftonin solution Calcital forInjection Yamanouchi Pharmaceutical Co. containing 160 I.U. of calcitonin per vial was dissolved in 40 ml of a vehicle containing 16 of purified gelatin and 0.5 phenol J. P. and the solution was diluted with the above mentioned physiological saline solution containing 0.1 of albumin. This dilution contained 0.2 or 1 I.U. of swine calcitonin per 0.2 ml. Results The results are shown in Table 2. The serum calcium level of the rat treated subcutaneously with 4.5 g of HPCT drcpped with statistical significance in 1 and 2 hours after the administration. The time course of serum calcium concentration after EPcm treatment was very similar to the time course of serum calcium observed in the rat subsutaneously given 0.2 I.U. of swine calcitonin. It was therefore considered that under the above experi mental conditions 4.5 g of EPcm is equivalent in activity to 0.2 I.U. of swine clacitonin. The serum calcium level of the rat treated subcutaneously with 20 g of HPCT.Cp 1 33 dropped wit statistical significance in 1 hour after the administration, however, no dropping of the serum calcium level was observed since 2 hours after the administration. With respect to calcitonins, it is known that compared with naturally occurring peptides having amidated C terminals, peptides with free C terminals desamidopeptides are generally very low in biological activity.However, HPCT a precursor of Val8 human calcitonin and HPCT Cp 1 33 suggest their stronger physiological activity as compared with swine calcitonin as shown in Table 2, though they can not be compared in an equal amount because the weight of the swine calcitonin in the Table is not clear. Moreover, although much research has been done inn the subs, on of amino acids in the amino acid sequence of calcitonin and the partial elimination of peptides or amino acids from the sequence, it has never been known that the addition of a peptide as in this invention has ever given rise to an active substance. Table 2Infuluence of HPCT on Rat Serum Calcium Change in serum Ca level mg dl 1 hour 2 hours 3 hours 4 hoursPhysiological saline 0.01 0.12 0.31 0.45 0.03 0.08 0.08 0.07HPCT 4.5 g rat 0.85 b 0.64 b 0.47 0.63 0.09 0.08 0.05 0.11HPCT.Cp 1 33 20mg rat 0.82 a 0.28 0.21 0.25 0.20 0.17 0.08 0.11Swine calcitonin 0.2 I.U. rat 0.70 a 0.71 b 0.49 0.60 0.19 0.07 0.06 0.12Swine calcitonin 1.0.I.U. rat 0.67 b 0.93 b 1.25 b 0.88 0.08 0.06 0.06 0.14The figures are the mean values for 4 rats per group standard deviations.a Significant difference at 5 level from physiological saline control b Significant difference at 1 level from physiological saline control