# A METHOD OF PRODUCING HUMAN GAMMA INTERFERON LIKE POLIPEPTIDE

## Claims
Patentansprüche für die Vestragsstaaten BE, CH, DE, FR, GB, IT, LI, LU, NL, SE

## Description
This invention relates to the production of a human gamma interferon like polypeptide hereinafter abbreviated to γ IFN . The chemically synthesized DNA molecule as disclosed herein is characterized by the DNA sequence coding for a polypeptide substantially agreeing in the amino acid sequence and composition with human gamma interferon and having the immunological or biological activity of human gamma interferon. Interferon includes three classes, namely alpha, beta and gamma, of proteins reportedly capable of inducing an antiviral condition in cells also showing antitumor activity. It is known that γ IFN can be produced by the genetic engineering technique using a gene prepared by reverse transcription of a messenger RNA in the presence of a reverse transcriptase Nature, Chemical synthesis of the γ IFN gene may presumably overcome the above disadvantage and also may lead to construction of a chimeric polypeptide through addition of arbitrary restriction enzyme cleavage sites. However, the synthesis of said gene is not easy since it is composed of several hundreds of base pairs. The present inventors endeavored to overcome this difficulty and succeeded in synthesizing a γ IFN gene. Thus, they developed a method of producing γ IFN by the genetic engineering technique using said gene. The present invention thus provides a method of producing human gamma interferon like polypeptide which comprises cultivating in a nutrient medium host cells transformed with a plasmid with a polydeoxyribonucleotide, which has a translation initiating codon at the 5 end followed by a nucleotide sequence consisting of Referring to the drawings, Figs 1 to 5 and Figs. 8 to 11 are given by way of supplement to the description of the example. Thus, sequence of each of 62 oligodeoxyribonucleotides used for the synthesis of a synthetic γ IFN gene, Fig.1 2 shows the amino acid sequence of human γ IFN and the nucleotide sequence of the γ IFN gene, Figs. 2 and 3 illustrate the oligodeoxyribonucleotide synthesis reaction process, Fig. 4 shows the oligodeoxyribonucleotide ligation scheme, Fig. 5 illustrates the scheme of determining the base sequence of the γ IFN gene synthesized, and Fig. 8 shows the process of construction of plasmid pGIF. Fig. 9 is the 14 polyacrylamide gel electrophoresis mapping for the polydeoxyribonucleotide blocks A. B. C. D. E and F synthesized and for the molecular weight marker M A preferred example of the above mentioned polydeoxyribonucleotide is the one having a sequence coding for the amino acid sequence 1 146 as shown in Fig. 1 2 and having a translation initiating codon, such as ATG, at the 5 end and a translation terminating codon, such as TAA, at the 3 end. Because ATG codes methionine, γ IFN of present invention naturally comprises a peptide added with formyl methionine or methionine at the 5 end of the amino acid sequence 1 142 mentioned above, as shown in Fig. 1 2 . For the sake of convenience in gene manipulation, it is preferable to add a nucleotide sequence capable of being cleaved with a restriction enzyme to each of the translation initiating and translation terminating codon endings. In a preferred embodiment, an The above polydeoxyribonucleotide can be synthesized, for example, in the following manner. First, the 62 gene fractions oligodeoxyribonucleotides shown in Fig. 1 1 are synthesized by the In the next place, the above gene is inserted into a vector plasmid. The following is an example of the insertion procedure. First, a plasmid, pKO703, is constructed by inser ting the promoter region 99 base pairs of the As the host cells, there may be used bacteria, yeasts and other fungi as well as cultured human and other animal cells and cultured plant cells. The bacteria are, for example, of the genus The yeasts are, for example, those belonging to the genus A strain belonging to the genus Upon inductive culture of the above transformant strain in a medium, such as L broth, using isopropyl B D thiogalactopyranoside, a substance having interferon activity is produced in the cells. As an example of a clone of the strain WA 802 pGIF, the strain A mode of practice of the present invention is described hereinbelow by way of example. The chemical synthesis of 62 oligodeoxyribonucleotides respectively designated F¹ through F₆₂ in Fig. 1 1, except for the improvements mentioned hereinafter, was carried out basically by the solid phase method reported by Ken ichi Miyoshi et al Nucl. Acids Res. The outline of chemical synthesis of these oligodeoxyribonucleotides by the improved technology is as follows. The base sequence of each oligodeoxyribonucleotide was confirmed by the two dimensional mapping method Bambara, J.E. et al 1974 Nucl. Acids. Res. Using the 62 oligodeoxyribonucleotide fragments 1 to 62 synthesized in the above manner, the gene Fig. 1 2 corresponding to γ IFN consisting of 450 base pairs were ligated according to the ligation schema shown in Fig. 4. Details of the procedure are set forth below. A to F blocks were synthesized. Excepting the 5 end fragments of each block 1, 14, 15, 24, 25, 32, 33, 44, 45, 54, 55, 62 , 1 n mole of each of the remaining 50 fragments was dissolved in 22 µl of distilled water and the solution was heated at 90 C for 2 minutes and, then, immediately cooled to 0 C. To this aqueous solution was added 5 µCi of γ ³²P ATP 5100 Ci m mole . It was then adjusted to give a 70 mM Tris HCl ph 7.5 10 mM MgCl₂ kinase buffer solution. Then, 3 units of polynucleotide kinase was added to make a total of 30 µl. By conducting the reaction at 37 C for 15 minutes, the 5 ends were labeled with ³²P. Then, to phosphorylate all the 5 ends, 4 n moles of ATP and 3 units of polynucleotide kinase were further added and the reaction was continued at 37 C for 45 minutes. The reaction was terminated by heating the mixture at 90 C for 5 minutes. 2.1 µg each of the above phosphorylated fragments 2 through 13 and unphosphorylated fragments 1 and 14 A Block , phosphorylated fragments 16 through 23 and unphosphorylated fragments 15 and 24 B Block , phosphorylated fragments 26 through 31 and unphosphorylated fragments 25 and 32 C Block , phosphorylated fragments 34 through 43 and unphos phorylated fragments 33 and 44 D Block , phosphorylated fragments 46 through 53 and unphosphorylated fragments 45 and 54 E Block , and phosphorylated fragments 56 through 61 and unphosphorylated fragments 55 and 62 F Block were respectively mixed and dialyzed in a dialysis tube against distilled water at 4 C for 16 hours to remove the unreacted ATP and kinase buffer components. This DNA dialyzate for each of A to F blocks was concentrated to dryness and dissolved in 14.5 µl of ligase buffer 20 mM Tris HCl, pH 7.6, 10 mM MgCl₂ . This solution was put in a 40 µl fine gauge glass tube and after both ends of the tube were sealed, it was heated in a 16.5 mm test tube containing 20 ml of water at 95 C for 2 minutes. Then, the test tube as a whole was allowed to stand at room temperature for 2 hours to aneal the DNA fragment. It was then placed at 0 C. This DNA solution was taken out of the fine gauge glass tube and made into a ligase buffer containing 0.4 mM of ATP and 10 mM of DTT. Then, 30 units of T4 DNA ligase was added to make a total volume of 80 µl and the reaction was allowed to proceed at 15 C for 16 hours. A portion of the reaction mixture was taken and subjected to electrophoresis with 14 polyacrylamide gel containing 7 M of urea and the state of ligation was investigated by radioautography. As a result, the largest ligation reaction products were found to agree in size with the desired products for 6 blocks A through F. Then, the entire amount of the above ligation product was fractionated by electrophoresis through 14 polyacrylamide gel containing 7 M of urea. After the positions of A through F were confirmed by radioautography, the gel portions corresponding to the positions were cut out and placed in dialysis tubes. Each tube was filled with 9mM Tris borate buffer pH 8.3, 0.25 mM EDTA added elution buffer and after closing both ends of the tube, electrophoresis was conducted in the elution buffer at a constant voltage of 200 V for 3 hours to elute the DNA in the gel. The DNA was taken out from the dialysis tube and lyophilized. The lyophilisate was dissolved in 100 µl of 0.3 M sodium acetate pH 7.0 , and two volumes of ethanol were added. The mixture was allowed to stand at 80 C for 30 minutes, at the end of which time it was centrifuged to recover the DNA as a sediment. The DNA sediments for A through F blocks were dissolved in 20 µl of distilled water and a 1 µl portion of the solution was investigated by electrophoresis with 14 polyacrylamide gels containing 7 M of urea. As shown in Fig. 9, substantially uniform bands corresponding to the sizes of DNAs for the respective blocks A through F were obtained. The DNA solution for each of A through F blocks was adjusted to the composition of the kinase buffer, and 2 n moles of ATP and 6 units of polynucleotide kinase were added. This reaction was conducted at 37 C for 1 hour, whereby the 5 ends of the blocks were phosphorylated. The DNA for A , B and C blocks 20 p moles each were mixed and ATP and DTT were added in the concentrations of 0.4 mM ATP and 10 mM DTT and to make a total of 50 µl. Then, using 8 units of T4 DNA ligase, the mixture was reacted at 15 C for 16 hours. The DNA s for D , E and F blocks were also mixed in the proportions of 5 p moles each and the ligation reaction was conducted at 15 C for 16 hours. After the above reaction, a portion of each reaction product was taken and the state of ligation was examined by electrophoresis with 8 polyacrylamide gels containing 7 M of urea. As shown in Fig. 10, it was found that I and II blocks corresponding to 225 bp had been synthesized. Then, the entire amount of the above reaction mixture was fractionated by electrophoresis through 8 polyacrylamide gels containing 7 M of urea. After the position of the 225 base pair DNA was confirmed by radioautography, the corresponding portion of the gel was cut out and the DNA was eluted by the above mentioned electro phoretic elution method. The DNA solution was lyophilized and dissolved in 100 µl of 0.3 M sodium acetate pH 7.0 , followed by addition of 2.5 volumes of ethanol, whereby the DNA was precipitated. To this DNA were added 0.5 n mole of ATP and 3 units of polynucleotide kinase and the reaction conducted at 37 C for 1 hour in the same manner as described hereinbefore. The DNA s 1 p mole each of blocks I and II were mixed and adjusted to the ligase buffer composition. Then, ATP and DTT were added to the mixture in the concentrations of 0.4 mM ATP and 10 mM DTT to make a total of 20 µl. Using 5 units of T4 DNA ligase, the ligation reaction was conducted at 15 C for 16 hours. After the reaction, a portion of the reaction mixture was taken and the state of ligation was investigated by electrophoresis with 6 polyacrylamide gel containing 7 M of urea. The result is shown in Fig. 11. In Fig. 11, 1 and 11 stand for the isolated and purified products for I and II blocks, respectively. Further, G denotes the result of ligation of the total γ IFN gene, indicating that a DNA molecule corresponding to 450 base pairs was obtained. To the whole amount of the above reaction mixture was added 1 10 volume of 3 M sodium acetate, followed by addition of 2.5 volumes of ethanol, whereby the DNA was recovered as a precipitate. This precipitate was dissolved in 100 µl of TA buffer 33 mM Tris acetate, pH 7.9, 66 mM potassium phosphate, 10 mM magnesium acetate, 0.5 mM DTT , and 20 units of restriction enzyme For the expresion of the chemically synthesized gene as a complete protein in The details of construction of the series of plasmids employed in this experiment are set forth below Fig. 8 . The plasmid pKO703 containing the UV5 promoter operator region of To separate the UV5 promoter operator region of lactose operon, the following experiment was conducted. Plasmid pKO601 is a plasmid having the Lactose operon promoter operator region between two 80 µg of this plasmid pKO601 was treated with 150 units of restriction enzyme After ethanol precipitation, the DNA precipitate was dissolved in 40 µl of T4 DNA polymerase solution and the solution was heated at 65 C for 30 minutes and, then, quenched. Thereafter, 5 µl each of dATP, DTTP, dCTP and dGTP from the respective 4 mM solutions as well as 5 µl of 100 mM B mercaptoethanol was added. The mixture was reacted using 10 units of T4 DNA polymerase at 37 C for 60 minutes. The reaction mixture was heated at 65 C for 30 minutes to inactivate the enzyme and the DNA was precipitated with ethanol. Finally, the DNA precipitate was dissolved in 20 µl of distilled water. Using 0.8 µg of the pBR 322 DNA with After the ligation, the DNA was precipitated from ethanol and dissolved in 10 µl of distilled water. Using this DNA solution, transformation of The transformant was cultivated on a M9S X gal medium The recognition site of restriction enzyme Therefore, the WA802 pKO701 through WA802 pKO710 strains obtained above were cultivated by single colonization on the ordinary nutrient agar medium containing 10 µg ml of tetracycline to examine their tetracycline sensitivity. It was found that WA 802 pKO701, WA802 pKO703, WA802 pKO706 and WA802 pKO709 were tetracycline insensitive. Therefore, plasmid DNA was separated from each of these strains and 2 µg of DNA was reacted in 1 x TA mixture with 5 units of restriction enzymes In 30 µl of 100 mM tris HCl buffer pH 7.3 5 mM MgCl₂ 50 mM Nacl, 15 µg of plasmid pKO703 was reacted with 10 units each of restriction enzymes After separation of plasmid DNA s, each plasmid DNA was cleaved with restriction enzymes The base sequence in the region corresponding to the 450 base pair γ IFN gene obtainable by the In E. Using a Coaster s 96 well plastic multiplate and a Eagle s minimal medium containing 5 of fetal calf serum, a dilution series of interferon samples were prepared by multi pipetting. Then, 100 µl of FL cells 5 x 10⁵ cells ml were added to each well and the cultivation was conducted at 37 C overnight in a 5 CO₂ gas atmosphere. The culture supernatant was discarded and 100 µl of Sindbis virus in a concentration 100 times as high as TCID₅₀ was added to each well and the incubation was carried out at 37 C overnight in a 5 CO₂ gas atmosphere. After 17 hours, the supernatant was discarded and the residue was stained with 1 crystal violet solution containing 20 of ethanol. As to interferon potency, the dilution factor of the interferon sample showing the intensity of stain a half that of the difference in the intensity of stain between the virus free well and the interferon free, virus added well was expressed in laboratory units ml. This assay system affords a sensitivity 7 to 25 times as high as α interferon for which an international standard is available. Whether the interferon activity was certainly human gamma interferon activity was confirmed immunologically using an anti human gamma interferon antibody. In this case, 10 µl of anti human gamma interferon antibody with a sufficient activity to inactivate 2,000 units ml of human gamma interferon activity was added to 50 µl of an interferon active sample, and after standing at room temperature for 3 hours, the mixture was centrifuged at 10,000 r.p.m. for 5 minutes. The supernatant was separated for use as a sample. As shown in Table 1, the transformed The